{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "collapsed_sections": [
        "5YyrgSxQnB06",
        "sBfZGn3YA_TK",
        "wLhpAAvwnWit",
        "-bqDjH1ppth1",
        "eYDUJwU4nrxy"
      ]
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "[Apache Spark](https://spark.apache.org/) es un motor de procesamiento y análisis de datos distribuidos de propósito general.\n",
        "\n",
        "Spark puede utilizarse en modo autónomo o utilizando gestores de recursos externos como YARN, Kubernetes o Mesos.\n",
        "\n",
        "Spark trabaja sobre el concepto de RDDs (conjuntos de datos distribuidos resistentes) que representa los datos como una colección distribuida.\n",
        "\n",
        "El motor de Spark puede aplicar operaciones para consultar y transformar el conjunto de datos en paralelo sobre múltiples ejecutores de Spark.\n",
        "\n",
        "Los DataFrames son una extensión de los RDD que impone un esquema a la colección distribuida de datos. Los dataframes pueden definirse para consumir desde múltiples fuentes de datos, incluyendo archivos, bases de datos relacionales, bases de datos NoSQL, flujos, etc.\n",
        "\n"
      ],
      "metadata": {
        "id": "G-dTPrx6tMJW"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Instalación y Monitorización\n"
      ],
      "metadata": {
        "id": "5YyrgSxQnB06"
      }
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "sBfZGn3YA_TK"
      },
      "source": [
        "## Instalación"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "En primer lugar instalamos y configuramos todas las dependencias de Spark para Python. De esta forma enlazaremos nuestro entorno con el servidor de Spark. Además configuraremos el entorno Spark con las variables que sean necestarias.\n",
        "\n",
        "**NOTA: la última versión de PySpark es la 3.5.1 [link](https://pypi.org/project/pyspark/#history)**"
      ],
      "metadata": {
        "id": "cDbDDJ9snxav"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "1. Descargar y localizar en `/usr/local/`"
      ],
      "metadata": {
        "id": "4po8zx7un1vP"
      }
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "nqld9bk_-mF7"
      },
      "source": [
        "!wget -q https://dlcdn.apache.org/spark/spark-3.5.1/spark-3.5.1-bin-hadoop3.tgz\n",
        "!tar xf spark-3.5.1-bin-hadoop3.tgz\n",
        "!mv  spark-3.5.1-bin-hadoop3 /usr/local/"
      ],
      "execution_count": 1,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "%%bash\n",
        "wget http://ftp.ntu.edu.tw/MySQL/Downloads/Connector-J/mysql-connector-java-5.1.49.tar.gz\n",
        "tar -zxf mysql-connector-java-5.1.49.tar.gz\n",
        "cd mysql-connector-java-5.1.49\n",
        "mv mysql-connector-java-5.1.49-bin.jar /usr/local/spark-3.5.1-bin-hadoop3/jars\n"
      ],
      "metadata": {
        "id": "ktdCR1TbwG0S",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "590a7d27-e526-4bde-9ac7-ecbaab85bd9a"
      },
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "--2024-02-23 12:40:59--  http://ftp.ntu.edu.tw/MySQL/Downloads/Connector-J/mysql-connector-java-5.1.49.tar.gz\n",
            "Resolving ftp.ntu.edu.tw (ftp.ntu.edu.tw)... 140.112.36.185\n",
            "Connecting to ftp.ntu.edu.tw (ftp.ntu.edu.tw)|140.112.36.185|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 3362563 (3.2M) [application/octet-stream]\n",
            "Saving to: ‘mysql-connector-java-5.1.49.tar.gz.1’\n",
            "\n",
            "     0K .......... .......... .......... .......... ..........  1%  309K 10s\n",
            "    50K .......... .......... .......... .......... ..........  3% 46.5M 5s\n",
            "   100K .......... .......... .......... .......... ..........  4%  309K 7s\n",
            "   150K .......... .......... .......... .......... ..........  6%  309K 8s\n",
            "   200K .......... .......... .......... .......... ..........  7%  309K 8s\n",
            "   250K .......... .......... .......... .......... ..........  9% 91.5M 6s\n",
            "   300K .......... .......... .......... .......... .......... 10%  310K 7s\n",
            "   350K .......... .......... .......... .......... .......... 12% 58.6M 6s\n",
            "   400K .......... .......... .......... .......... .......... 13%  310K 6s\n",
            "   450K .......... .......... .......... .......... .......... 15% 51.0M 5s\n",
            "   500K .......... .......... .......... .......... .......... 16%  310K 6s\n",
            "   550K .......... .......... .......... .......... .......... 18% 47.0M 5s\n",
            "   600K .......... .......... .......... .......... .......... 19%  310K 5s\n",
            "   650K .......... .......... .......... .......... .......... 21% 49.5M 5s\n",
            "   700K .......... .......... .......... .......... .......... 22%  151M 4s\n",
            "   750K .......... .......... .......... .......... .......... 24%  310K 5s\n",
            "   800K .......... .......... .......... .......... .......... 25% 68.1M 4s\n",
            "   850K .......... .......... .......... .......... .......... 27%  310K 4s\n",
            "   900K .......... .......... .......... .......... .......... 28% 99.9M 4s\n",
            "   950K .......... .......... .......... .......... .......... 30% 44.2M 4s\n",
            "  1000K .......... .......... .......... .......... .......... 31%  310K 4s\n",
            "  1050K .......... .......... .......... .......... .......... 33% 72.7M 4s\n",
            "  1100K .......... .......... .......... .......... .......... 35%  310K 4s\n",
            "  1150K .......... .......... .......... .......... .......... 36%  133M 3s\n",
            "  1200K .......... .......... .......... .......... .......... 38% 51.8M 3s\n",
            "  1250K .......... .......... .......... .......... .......... 39%  310K 3s\n",
            "  1300K .......... .......... .......... .......... .......... 41% 83.3M 3s\n",
            "  1350K .......... .......... .......... .......... .......... 42%  311K 3s\n",
            "  1400K .......... .......... .......... .......... .......... 44% 50.6M 3s\n",
            "  1450K .......... .......... .......... .......... .......... 45% 58.0M 3s\n",
            "  1500K .......... .......... .......... .......... .......... 47%  311K 3s\n",
            "  1550K .......... .......... .......... .......... .......... 48% 78.1M 3s\n",
            "  1600K .......... .......... .......... .......... .......... 50%  312K 3s\n",
            "  1650K .......... .......... .......... .......... .......... 51% 26.1M 2s\n",
            "  1700K .......... .......... .......... .......... .......... 53% 74.9M 2s\n",
            "  1750K .......... .......... .......... .......... .......... 54%  312K 2s\n",
            "  1800K .......... .......... .......... .......... .......... 56% 48.1M 2s\n",
            "  1850K .......... .......... .......... .......... .......... 57% 72.9M 2s\n",
            "  1900K .......... .......... .......... .......... .......... 59%  310K 2s\n",
            "  1950K .......... .......... .......... .......... .......... 60%  176M 2s\n",
            "  2000K .......... .......... .......... .......... .......... 62%  312K 2s\n",
            "  2050K .......... .......... .......... .......... .......... 63% 26.4M 2s\n",
            "  2100K .......... .......... .......... .......... .......... 65%  104M 2s\n",
            "  2150K .......... .......... .......... .......... .......... 66%  310K 2s\n",
            "  2200K .......... .......... .......... .......... .......... 68% 83.2M 1s\n",
            "  2250K .......... .......... .......... .......... .......... 70%  312K 1s\n",
            "  2300K .......... .......... .......... .......... .......... 71% 26.3M 1s\n",
            "  2350K .......... .......... .......... .......... .......... 73% 96.5M 1s\n",
            "  2400K .......... .......... .......... .......... .......... 74%  312K 1s\n",
            "  2450K .......... .......... .......... .......... .......... 76% 28.9M 1s\n",
            "  2500K .......... .......... .......... .......... .......... 77%  314K 1s\n",
            "  2550K .......... .......... .......... .......... .......... 79% 23.1M 1s\n",
            "  2600K .......... .......... .......... .......... .......... 80% 45.2M 1s\n",
            "  2650K .......... .......... .......... .......... .......... 82%  313K 1s\n",
            "  2700K .......... .......... .......... .......... .......... 83% 28.9M 1s\n",
            "  2750K .......... .......... .......... .......... .......... 85%  315K 1s\n",
            "  2800K .......... .......... .......... .......... .......... 86% 19.8M 1s\n",
            "  2850K .......... .......... .......... .......... .......... 88% 33.3M 1s\n",
            "  2900K .......... .......... .......... .......... .......... 89%  314K 0s\n",
            "  2950K .......... .......... .......... .......... .......... 91% 22.5M 0s\n",
            "  3000K .......... .......... .......... .......... .......... 92%  316K 0s\n",
            "  3050K .......... .......... .......... .......... .......... 94% 25.4M 0s\n",
            "  3100K .......... .......... .......... .......... .......... 95% 26.6M 0s\n",
            "  3150K .......... .......... .......... .......... .......... 97%  315K 0s\n",
            "  3200K .......... .......... .......... .......... .......... 98% 19.6M 0s\n",
            "  3250K .......... .......... .......... ...                  100%  105M=4.5s\n",
            "\n",
            "2024-02-23 12:41:04 (723 KB/s) - ‘mysql-connector-java-5.1.49.tar.gz.1’ saved [3362563/3362563]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "2. Establecer las variables de entorno necesarias"
      ],
      "metadata": {
        "id": "FKovJcBAoFy1"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "os.environ[\"JAVA_HOME\"]=\"/usr/lib/jvm/java-11-openjdk-amd64\"\n",
        "os.environ[\"SPARK_HOME\"] = \"/usr/local/spark-3.5.1-bin-hadoop3\"\n"
      ],
      "metadata": {
        "id": "Md8rfmP87Wlz"
      },
      "execution_count": 4,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "os.environ[\"CLASSPATH\"] = \"/usr/local/spark-3.5.1-bin-hadoop3/jars\"\n",
        "os.environ['PYSPARK_SUBMIT_ARGS'] = '--jars /usr/local/spark-3.5.1-bin-hadoop3/jars/mysql-connector-java-5.1.49-bin.jar pyspark-shell'"
      ],
      "metadata": {
        "id": "6Y_UjALuzuCe"
      },
      "execution_count": 5,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "3. Instalar [findspark](https://pypi.org/project/findspark/)"
      ],
      "metadata": {
        "id": "peZLB8oZoI2R"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install -q findspark\n"
      ],
      "metadata": {
        "id": "PKJdUuukn6GW"
      },
      "execution_count": 6,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "Ejecutamos `findspark.init()` para hacer que pyspark sea importable como una biblioteca normal para ello añade pyspark al sys.path en el entorno de ejecución."
      ],
      "metadata": {
        "id": "HyTBGMZAo_RZ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import findspark\n",
        "findspark.init()"
      ],
      "metadata": {
        "id": "aw0xbBn37f3w"
      },
      "execution_count": 7,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "findspark.add_packages('mysql:mysql-connector-java:5.1.49')"
      ],
      "metadata": {
        "id": "F2TSVJCJ5v8Q"
      },
      "execution_count": 8,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "1ARyob50A8CR"
      },
      "source": [
        "4. Lanzamos una sesión Spark con la librería `pyspark`\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Es necesario crear un contexto Spark para ejecutar la aplicación\n",
        "\n",
        "> NOTA: Un SparkContext representa la conexión al cluster de Spark, y puede utilizarse para crear RDDs y otros elementos. **Sólo puede haber un SparkContext activo**. Se debe detener (*stop()*) el SparkContext activo antes de crear uno nuevo.\n",
        "\n",
        "Si creamos el contexto sin ningún argumento al respecto  se utilizará la variable de entorno SPARK_HOME, y si no está establecida, se comprobarán otras posibles ubicaciones de instalación.\n",
        "\n",
        "Más consideraciones sobre en arranque en el siguiente [enlace](https://github.com/minrk/findspark)\n",
        "\n",
        "Más detalle sobre las opciones de configuración en el siguiente [enlace](http://spark.apache.org/docs/latest/configuration.html)"
      ],
      "metadata": {
        "id": "W_A9DVotoVUh"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import pyspark\n",
        "from pyspark.sql import *\n",
        "from pyspark.sql.functions import *\n",
        "from pyspark import SparkContext, SparkConf"
      ],
      "metadata": {
        "id": "D09Rj23C7oeE"
      },
      "execution_count": 9,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 219
        },
        "id": "wHFAANWX_hHe",
        "outputId": "9b60466b-1ccc-4e45-b987-579c0714b6fa"
      },
      "source": [
        "conf = SparkConf()\n",
        "conf.set(\"spark.ui.port\", \"4050\")\n",
        "conf.set(\"spark.appName\", \"SQL\")\n",
        "conf.set(\"spark.executor.instances\", \"4\")\n",
        "conf.set(\"spark.executor.cores\", 4)\n",
        "conf.set(\"spark.executor.memory\",\"10g\")\n",
        "conf.set(\"spark.driver.extraClassPath\", \"/usr/local/spark-3.5.1-bin-hadoop3.2/jars/mysql-connector-java-5.1.49-bin.jar\") \\\n",
        "\n",
        "# create the context\n",
        "sc = pyspark.SparkContext(conf=conf)\n",
        "spark = SparkSession.builder.getOrCreate()\n",
        "spark"
      ],
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<pyspark.sql.session.SparkSession at 0x7ad3181642e0>"
            ],
            "text/html": [
              "\n",
              "            <div>\n",
              "                <p><b>SparkSession - in-memory</b></p>\n",
              "                \n",
              "        <div>\n",
              "            <p><b>SparkContext</b></p>\n",
              "\n",
              "            <p><a href=\"http://c406b32af55a:4050\">Spark UI</a></p>\n",
              "\n",
              "            <dl>\n",
              "              <dt>Version</dt>\n",
              "                <dd><code>v3.5.1</code></dd>\n",
              "              <dt>Master</dt>\n",
              "                <dd><code>local[*]</code></dd>\n",
              "              <dt>AppName</dt>\n",
              "                <dd><code>pyspark-shell</code></dd>\n",
              "            </dl>\n",
              "        </div>\n",
              "        \n",
              "            </div>\n",
              "        "
            ]
          },
          "metadata": {},
          "execution_count": 11
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Se puede comprobar fácilmente la versión actual de Spark y el acceso a la interfaz web simplemente mediante la variable de la sesión (*spark*).\n",
        "\n",
        "En la interfaz de Spark se puedes supervisar el progreso del  trabajo. El enlace es directo si se trabaja en un entorno local."
      ],
      "metadata": {
        "id": "WhGOQzSpq8TG"
      }
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "xTcpKV6TqaM1"
      },
      "source": [
        "También podemos consultar la configuración por defecto que se nos crea sobre todo en cuanto a particiones y paralelismo.\n",
        "\n",
        "Las siguientes propiedades indican el número por defecto en los RDDs devueltos por transformaciones como join, reduceByKey, y parallelize cuando no es establecido por el usuario. El valor $2$  representa un paralelismo \"mínimo\", que puede ayudar a detectar errores que sólo existen cuando ejecutamos en un contexto distribuido."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "5Aw6dDbxrQpu",
        "outputId": "01f31aee-7a2e-44fc-8208-4085c06edcca"
      },
      "source": [
        "print(\"Default Min Partitions \" + str(spark.sparkContext.defaultMinPartitions))\n",
        "print(\"Default parallelism \" + str(spark.sparkContext.defaultParallelism))"
      ],
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Default Min Partitions 2\n",
            "Default parallelism 2\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#spark.stop()"
      ],
      "metadata": {
        "id": "ZKdrjpT4wh1N"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Monitorización\n"
      ],
      "metadata": {
        "id": "wLhpAAvwnWit"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "Como no se puede utilizar un teórico localhost sobre Google Colab entonces hay que hacer accesible el puerto desde el exterior mediante un tunel ngrok. **NGROK** es una herramienta de uso gratuito que nos permite exponer nuestro entorno local a la web, es decir, podemos \"publicar\" nuestro trabajo en local para que el resto del mundo lo pueda ver sin la necesidad de subir la aplicación a un servidor."
      ],
      "metadata": {
        "id": "VEBUhbrg9afw"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "**Subir fichero authtoken.txt con la clave de acceso que proporciona ngrok al registrarse.**"
      ],
      "metadata": {
        "id": "6qyJXuz_8spW"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install pyngrok\n",
        "!ngrok authtoken $(<authtoken.txt)\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "L8gy8jdx8ebA",
        "outputId": "628ec05c-e903-4e82-d303-f2f29cfe8509"
      },
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting pyngrok\n",
            "  Downloading pyngrok-7.1.2-py3-none-any.whl (22 kB)\n",
            "Requirement already satisfied: PyYAML>=5.1 in /usr/local/lib/python3.10/dist-packages (from pyngrok) (6.0.1)\n",
            "Installing collected packages: pyngrok\n",
            "Successfully installed pyngrok-7.1.2\n",
            "/bin/bash: line 1: authtoken.txt: No such file or directory\n",
            "authtoken - \n",
            "\n",
            "USAGE:\n",
            "  ngrok authtoken TOKEN [flags]\n",
            "\n",
            "AUTHOR:\n",
            "  ngrok - <support@ngrok.com>\n",
            "\n",
            "COMMANDS: \n",
            "  config          update or migrate ngrok's configuration file\n",
            "  http            start an HTTP tunnel\n",
            "  tcp             start a TCP tunnel\n",
            "  tunnel          start a tunnel for use with a tunnel-group backen\n",
            "\n",
            "EXAMPLES: \n",
            "  ngrok http 80                                                 # secure public URL for port 80 web server\n",
            "  ngrok http --domain baz.ngrok.dev 8080                        # port 8080 available at baz.ngrok.dev\n",
            "  ngrok tcp 22                                                  # tunnel arbitrary TCP traffic to port 22\n",
            "  ngrok http 80 --oauth=google --oauth-allow-email=foo@foo.com  # secure your app with oauth\n",
            "\n",
            "Paid Features: \n",
            "  ngrok http 80 --domain mydomain.com                           # run ngrok with your own custom domain\n",
            "  ngrok http 80 --allow-cidr 2600:8c00::a03c:91ee:fe69:9695/32  # run ngrok with IP policy restrictions\n",
            "  Upgrade your account at https://dashboard.ngrok.com/billing/subscription to access paid features\n",
            "\n",
            "Upgrade your account at https://dashboard.ngrok.com/billing/subscription to access paid features\n",
            "\n",
            "Flags:\n",
            "  -h, --help      help for ngrok\n",
            "\n",
            "Use \"ngrok [command] --help\" for more information about a command.\n",
            "\n",
            "ERROR:  accepts 1 arg(s), received 0\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from pyngrok import ngrok\n",
        "public_url = ngrok.connect(4050)\n",
        "public_url"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Avv4q6WF8lUV",
        "outputId": "57a771f9-26ee-4ec3-e0d6-425e8cc03d6d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<NgrokTunnel: \"http://7fb9-34-125-216-10.ngrok.io\" -> \"http://localhost:4050\">"
            ]
          },
          "metadata": {},
          "execution_count": 13
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Paralelización de procesos"
      ],
      "metadata": {
        "id": "-bqDjH1ppth1"
      }
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "QeZ3vsFxpMmT"
      },
      "source": [
        "1. Plantear el problema a resolver. En este caso la aproximación de Pi por el método de Montecarlo [enlace](https://www.geogebra.org/m/cF7RwK3H)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 14,
      "metadata": {
        "id": "1lcoGXBzkU5s"
      },
      "outputs": [],
      "source": [
        "import random\n",
        "num_samples = 1000000\n",
        "def inside(p):\n",
        "  x, y = random.random(), random.random()\n",
        "  return x*x + y*y < 1\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "eRxOanjhpCoJ"
      },
      "source": [
        "2. Paralelizar el cálculo con **parallelize**: Este método se utiliza para distribuir la colección de elementos del mismo tipo (datos u operaciones) para poder funcionar en paralelo."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 15,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "JzoJEvPtoUNB",
        "outputId": "d147b44d-bec4-4526-8503-c83e778cb18f"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "785328"
            ]
          },
          "metadata": {},
          "execution_count": 15
        }
      ],
      "source": [
        "count = sc.parallelize(range(0, num_samples)).filter(inside).count()\n",
        "count\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "po4pk0sJprwq"
      },
      "source": [
        "3. Obtenemos el resultado final"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 16,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "672qv1phpq8g",
        "outputId": "b66dde0e-516d-432f-aed8-4015df1b7b4b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "3.141312\n"
          ]
        }
      ],
      "source": [
        "pi = 4 * count / num_samples\n",
        "print(pi)\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Momento de revisar el enlace de la monitorización"
      ],
      "metadata": {
        "id": "NK9ybppeqLWZ"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Manejo de Datos JSON"
      ],
      "metadata": {
        "id": "eYDUJwU4nrxy"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "1. Descargamos y renombramos los datos json procedentes de OpenData"
      ],
      "metadata": {
        "id": "-veqYaeUsTR3"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "%%bash\n",
        "wget https://ias1.larioja.org/opendata/download?r=Y2Q9ODYxfGNmPTA0\n",
        "mv /content/download?r=Y2Q9ODYxfGNmPTA0 /content/pruebas_localidades.json"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "gdXQpSbz-RX4",
        "outputId": "daf87395-f28b-4f03-a96a-13520c095a98"
      },
      "execution_count": 17,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Process is interrupted.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "PfqSLQlJyoLx"
      },
      "source": [
        "2. Spark para lectura de JSON"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "3KBigFfgE1Z5",
        "outputId": "12dc4bf5-f835-442f-827c-f2436e8a7bfa"
      },
      "source": [
        "df = spark.read.json(\"/content/pruebas_localidades.json\")\n",
        "df.printSchema()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "root\n",
            " |-- data: array (nullable = true)\n",
            " |    |-- element: struct (containsNull = true)\n",
            " |    |    |-- ACUMULADO NEGATIVO: long (nullable = true)\n",
            " |    |    |-- ACUMULADO POSITIVO: long (nullable = true)\n",
            " |    |    |-- Lat: double (nullable = true)\n",
            " |    |    |-- Lon: double (nullable = true)\n",
            " |    |    |-- NEGATIVOS: long (nullable = true)\n",
            " |    |    |-- NUEVOS CASOS ACUMULADO 14 DIAS: long (nullable = true)\n",
            " |    |    |-- NUEVOS CASOS ACUMULADO 7 DIAS: long (nullable = true)\n",
            " |    |    |-- PCR REALIZADOS: long (nullable = true)\n",
            " |    |    |-- POSITIVOS: long (nullable = true)\n",
            " |    |    |-- TASA IA 14: double (nullable = true)\n",
            " |    |    |-- TASA IA 7: double (nullable = true)\n",
            " |    |    |-- TOTAL ACUMULADO: long (nullable = true)\n",
            " |    |    |-- Title: string (nullable = true)\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "3. Consultar datos: Select/Explode"
      ],
      "metadata": {
        "id": "ewcBZcKrsao0"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "from pyspark.sql.functions import explode\n",
        "df2 = df.select(explode(\"data\")).select(col(\"col.*\"))\n",
        "df2.printSchema()\n",
        "df2.show()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "vduWGRafDMIw",
        "outputId": "f1d16268-c990-4dd8-c9da-5818af70bbac"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "root\n",
            " |-- ACUMULADO NEGATIVO: long (nullable = true)\n",
            " |-- ACUMULADO POSITIVO: long (nullable = true)\n",
            " |-- Lat: double (nullable = true)\n",
            " |-- Lon: double (nullable = true)\n",
            " |-- NEGATIVOS: long (nullable = true)\n",
            " |-- NUEVOS CASOS ACUMULADO 14 DIAS: long (nullable = true)\n",
            " |-- NUEVOS CASOS ACUMULADO 7 DIAS: long (nullable = true)\n",
            " |-- PCR REALIZADOS: long (nullable = true)\n",
            " |-- POSITIVOS: long (nullable = true)\n",
            " |-- TASA IA 14: double (nullable = true)\n",
            " |-- TASA IA 7: double (nullable = true)\n",
            " |-- TOTAL ACUMULADO: long (nullable = true)\n",
            " |-- Title: string (nullable = true)\n",
            "\n",
            "+------------------+------------------+----------+----------+---------+------------------------------+-----------------------------+--------------+---------+----------+---------+---------------+---------+\n",
            "|ACUMULADO NEGATIVO|ACUMULADO POSITIVO|       Lat|       Lon|NEGATIVOS|NUEVOS CASOS ACUMULADO 14 DIAS|NUEVOS CASOS ACUMULADO 7 DIAS|PCR REALIZADOS|POSITIVOS|TASA IA 14|TASA IA 7|TOTAL ACUMULADO|    Title|\n",
            "+------------------+------------------+----------+----------+---------+------------------------------+-----------------------------+--------------+---------+----------+---------+---------------+---------+\n",
            "|              null|              null|42.5727159|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|-2.7100606|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|   ABALOS|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|             0|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|             96|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|        0|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|        0|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|                14|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|                82|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                             0|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                            0|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      -1.0|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     -1.0|           null|     null|\n",
            "|              null|              null|42.4451704|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|-2.2913654|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|AGONCILLO|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|             1|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|            357|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|        0|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|        1|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "+------------------+------------------+----------+----------+---------+------------------------------+-----------------------------+--------------+---------+----------+---------+---------------+---------+\n",
            "only showing top 20 rows\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "B3dVIL8Syqta"
      },
      "source": [
        "4. Spark Hive para datos Json"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "NUHekGp-_siT",
        "outputId": "95292c5f-f62f-4837-ef6a-1c7f5ce4ad79"
      },
      "source": [
        "from pyspark.sql import HiveContext\n",
        "hiveCtx = spark.builder.enableHiveSupport().getOrCreate()\n",
        "ex1 = hiveCtx.read.json(df2.toJSON())\n",
        "ex1.registerTempTable(\"datos\")\n",
        "results = hiveCtx.sql(\"SELECT * FROM datos\").show()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/spark-3.2.1-bin-hadoop3.2/python/pyspark/sql/dataframe.py:140: FutureWarning: Deprecated in 2.0, use createOrReplaceTempView instead.\n",
            "  FutureWarning\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "+------------------+------------------+----------+----------+---------+------------------------------+-----------------------------+--------------+---------+----------+---------+---------------+---------+\n",
            "|ACUMULADO NEGATIVO|ACUMULADO POSITIVO|       Lat|       Lon|NEGATIVOS|NUEVOS CASOS ACUMULADO 14 DIAS|NUEVOS CASOS ACUMULADO 7 DIAS|PCR REALIZADOS|POSITIVOS|TASA IA 14|TASA IA 7|TOTAL ACUMULADO|    Title|\n",
            "+------------------+------------------+----------+----------+---------+------------------------------+-----------------------------+--------------+---------+----------+---------+---------------+---------+\n",
            "|              null|              null|42.5727159|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|-2.7100606|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|   ABALOS|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|             0|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|             96|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|        0|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|        0|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|                14|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|                82|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                             0|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                            0|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      -1.0|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     -1.0|           null|     null|\n",
            "|              null|              null|42.4451704|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|-2.2913654|     null|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|           null|AGONCILLO|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|             1|     null|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|     null|      null|     null|            357|     null|\n",
            "|              null|              null|      null|      null|     null|                          null|                         null|          null|        0|      null|     null|           null|     null|\n",
            "|              null|              null|      null|      null|        1|                          null|                         null|          null|     null|      null|     null|           null|     null|\n",
            "+------------------+------------------+----------+----------+---------+------------------------------+-----------------------------+--------------+---------+----------+---------+---------------+---------+\n",
            "only showing top 20 rows\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Ingesta de Datos"
      ],
      "metadata": {
        "id": "tderNCLTtwZg"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "![image.png](data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA2sAAAGRCAIAAAB0QbyRAAAgAElEQVR4nOzdd5xU1d0G8N85t0yf7b3AFnqXIiDSFHuPNfbESKLRGNuboibGxNii0SR2jRpbErtBwQoo0nsvC+wubO/T5957zvvHsOuKtAG2sc/3w5vXvXP33jOzM3eeeyqTUhIAAAAAwCHjXV0AAAAAAOhhkCABAAAAID5IkAAAAAAQHyRIAAAAAIgPEiQAAAAAxAcJEgAAAADigwQJAAAAAPFBggQAAACA+CBBAgAAAEB8kCABAAAAID5IkAAAAAAQHyRIAAAAAIgPEiQAAAAAxAcJEgAAAADigwQJAAAAAPFBggQAAACA+CBBAgAAAEB8kCABAAAAID5IkAAAAAAQHyRIAAAAAIgPEiQAAAAAxAcJEgAAAADigwQJAAAAAPFBggQAAACA+CBBAgAAAEB8kCABAAAAID5IkAAAAAAQHyRIAAAAAIgPEiQAAAAAxAcJEgAAAADigwQJAAAAAPFBggQAAACA+CBBAgAAAEB8kCABAAAAID5IkAAAAAAQHyRIAAAAAIgPEiQAAAAAxAcJEgAAAADigwQJAAAAAPFBggQAAACA+CBBAgAAAEB8kCABAAAAID5IkAAAAAAQHyRIAAAAAIgPEiQAAAAAxAcJEgAAAADigwQJAAAAAPFBggQAAACA+CBBAgAAAEB8kCABAAAAID5IkAAAAAAQHyRIAAAAAIgPEiQAAAAAxAcJEgAAAADigwQJAAAAAPFBggQAAACA+CBBAgAAAEB8kCABAAAAID5IkAAAAAAQHyRIAAAAAIgPEiQAAAAAxAcJEgAAAADigwQJAAAAAPFBggQAAACA+CBBAgAAAEB8kCABAAAAID5IkAAAAAAQHyRIAAAAAIgPEiQAAAAAxAcJEgAAAADigwQJAAAAAPFBggQAAACA+CBBAgAAAEB8kCABAAAAID5IkAAAAAAQHyRIAAAAAIiP2tUFAICjb1fQP7+2YkFd5f92lzLW1aU5ehTGp6Rnn5iWNSMjL9fp7uriAAD0XkxK2dVlAICjwxDii+pdD25cEbTMPKd7kDdpQkrmsfQJF1Kubqrb2NJYFvT5TfO+YeNOyczXONpSAAA6GxIkwDEibJk3rfhqeUPtk6MnF7kTNM4Vxo69jzcjkkRRYdWEQ9ct/TLb4Xrp+JPcqtbV5QIA6F1w7w5wLCgP+MZ88t+AYcyafObQhBSdc34sxkciij0pjfFcp3v2lLNTdfsJn71T4m/u4mIBAPQyqIMEOBZcufCzwQlJN/UbLkhKKdmx1PnxgFTGXty+aU5V2YeTz+zqsgAA9CKogwTo8T6pKl/ZVDuzaIglBRH1nvhIRIYQl/Xp12JE/7F1bVeXBQCgF0GCBOjZygK+mUvn/nXUJJXzXpUdYxhjOudPj5ny6KZV65sburo4AAC9BRIkQM82p6p8TFLa9IzcTu6RYgmx2de43d8cFVZnnvf7hJSF7oQL84pWNtZ2bUkAAHoPzAcJ0INJKTe2NExKy4oKq9MqIDmxDS0Nf9+69v3dO+yKOjE18+ERE9Ptjs45+/cxxkwpxqdmzqksu6LvgK4qBgBAr4I6SIAezJJyu79lfGqmKTqpApIRrW2uP3Xeh59WlbtUTWFscX31L1d+bQjROQXYJ1OIwd6kzb6mLiwDAECvggQJ0IMZUlSFg4O9SbExNJ2AEfu0qlznSvsqzzXN9Yvrqzl1WS9MxphDUeuj4a4qAABAb4MECdCDSSkVxkUnTt8TFuai+uq9LhwRy7pr7eJOS7H7pDJudmk9KABAr4IECQBxaIxGSvzNewVWzlhZwHfPuiUC88sCAPQOGEkDAHFoiEZqI2G7osR+7O9JtFpT4/LG2g0tjUMTkruudAAA0EmQIAHgUKmMrWmqj1U/SilVzi/ILbx/w/LYREKC5H3rl70+YYayryb1sGXWR8J1kfBXdZVza3Y3GdFCl/fWASMGe5MFoeYSAKCHQYIEgEOlML6isYYzRkSCZKHbe2Px0FkVO1c11SmMK0SL6qv+vnXtHQNHRlu7JEYsa2VT7fyaygV1lXWRcKMRiVpW7Ag7/S2L6qs+nXpOht3Zlc8KAADihwQJAIdKZWx1U32sitEU8viUDCK6Z8jYCxbMji3G7VDUF7ZvmJCSkeN0lwV8H1WWflxZ1mJEdc5VxomRzrnOv+1+bQq5qaUx0+5EJSQAQM+CBAkAh8qQYmNLo50rRGRKMSk1KyKs45LSZhYNfq5kw559hPjpsnlExBgpjGucp9rs+zyaKeX09KzhiamIjwAAPQ4SJAAcEka0qaUpNnOQlFJIOTIx1ZJSSHFl3wHv7d5RGw4xxiRR0DL3+t1YDeVe/SN1rtw2YGSybkOCBADocTCbDwAcEs7YqsZale25aOQ53Um6LRYocx3uX/QfEdnXAtlCylyn+9ycgusKBzkVNUHT2/7d3G9YtsOJCYAAAHoi1EECwMFJKYmxNc0NbfWIBW6vJIpNDBkV1lV9+39UsfPrukqFcSJiRGk2x3m5hRflFfZxegOWceWiz1qMaKz+0pTiwrziG/sNC5hGp82FDgAARxESJADsl5Bym795fXPDkoaar2srK0J+je+ZCbLQ5W3bjTEWsqzbB46iTdTfkzQiMWVUUlqx22tKaQiLM/bnjSvWNNXHhmAzxi7MLX7iuEmIjwAAPRcSJADsQ4sRfWfX9v+UbysP+E0pYm3NbU3YRFTg9rbfX0o52Jv03NhpCmOxXBiwTCmlTVHuXrf4tZ1b1NgcQFIWuL33Dz8+aJqIjwAAPRcSJADswYi2+1vm11bMq634qrYiKoSNK5yIWkMh7RkTQ/kuT393omy3HnfsP9TvhkLG2D93bPrXji0a57Hf7ePyfDDpdJVxdH4EAOjRkCABgIhISvlm+bY/b1gRtkxJxIk5FHWvHTyafm5OwcV5xZl2h1NVD1yJqDI2t6bid2uXtHWdTLHZ/zluuk1RRbvoCQAAPRESJABQ0DRvXbVgdmWZXVFYu3AXS41pNvvwxNRzcvpOT89lRBFhHUoN4lZf8xWLPrUrauw4bk17buy0vi6PifgIANDzIUEC9HY2rtyzacnHlaX21lEyRGQKMT414+SMvMlp2Wk2e6Juk1Luc76efTKEdfmiT22tB/Rq+gvjpg1PTDGEQHwEADgGIEEC9HaMyG8YKuPts50kcqvaQE9ioq47VdWKZ9ZGr6b3+fCVSOv610LKkzJyj0tKQ+0jAMAxAzOKA/R2EWHdPnDkIG9S+8m9Nc4/r9p15eLPLlow51erF31TV6UccvjzGdE5U852qZqUkog4Y9/UVVWGgx1SegAA6ApIkAC9nSTKsjufHD05xWaX7UKkwrnC+O5Q4P3dO3648JPzvv74i+pdEcs66CoykijT7jw5I7dty66Q/43Sre0nAwIAgB4NF3QAIMZYodv74rjpU9NzdL73ZYEzZlfUNY11Vy3+/KyvZt26aoF2sCzIGftZ8VC19VAa48+WrN/qb0IbNgDAsQEJEgCIiEwpB3oSnxs77Y0Jp5yQmhnr+LhXlaRb1coCvi9rdvODtWjHJhj/YZ9+EcuKTRsZEdZDG1dq34unAADQE+FqDgB7MMYsKYYlpLw6fsa7k047P7cgy+Haax9JVORKOGhDNmMsLKxb+o/IdbpjWxTGv66tXN5Q2yFFBwCAzoUECQDfYowJkhFhDU1I+dOw8R+eeMZVBQMi1ncm8SlwewR9J0FyYssaau7fsLwpGmnbKKV0qdpPi4a0jb+OCOupbetVDMcGAOj5MJsPAOwbI/Jo+n1Dx6Xrjoc2rYy1XEuSRe4E+d06yDXN9T9ZOrcpGnl8y5piT8LpmfmjktIK3d58p/us7L4v7thYHvRzxjhjb5Vvu7n/sMHe5L0yKAAA9CxIkACwb7G6w5BlGVK0bZSSCl3etp8ZUUUo8LNl84KmYVMUm6LUhUOv7Nz88s7NHlVzqdq4lPQJqZlqQ02s6rHQ7b1/w/Jnxk7da8lEAADoWXARB4ADYUS1kVDbjwpjDkWNDY6RUtZEwlcv/qIyFGgbdt22KGLQMoOW+d6uHUJKlbO2OsdSRdniaxqRmNrJTwQAAI4i9IMEgAPZK0HaFTVZt8VSokXyphVflQZa1P2PsNY4tymKwrjC9/wzhVjf3NAZRQcAgA6DOkgAODBWFwm3DX6xK0qiphORkPLHS75YUl9tUxQpJWN0RlZBut2+w9/iNw2faUi53wHbybq9c4oOAAAdBAkSAA6EMaoJt6uD5EqibjOE+OXKrxfUVtkUhYgYsZlFg28dMFLn3GcYZUH/Zl/jV7WVX9bsboiGGX1n8LVb1SanZXf20wAAgKMKCRIADoQR1UXDbT/aFCXFZv/V6oX/q9hpV9RYPePpWfk39x8hpIwIoStKkdtb7Ek4J6dA43xOZfmLOzasaWqQUgqSCZpt7rTzFI4JfQAAejYkSAA4EElUHwm7FJWIpJQa579ft/SVnZvtiiqJTCkvyiv60/DjlXazPO7pJSmlaZonZeScnpXvM4yykM8QotidwBmLDcTpsqcEAABHDAkSAA4kYBqWEExlRCSIiNgmX9OJrc3QXk2/d+g4TvuOg4wxU0rTMhXOClze9ts7vNwAANCRkCABYL8YUbMRbQt8CmNbfI2yhYjIkOLUzLxHR54Q6woJAAC9ChIkAOwXI9ZiGO0rDBXGiZGUclxK1pOjp6ico0kaAKAXwnyQAHAAkn83HUophZTTMnKfGzs11vcR8REAoBdCHSQA7Jcg6udOKHB7a8OhWFLUuHLLgOHXFgxSEBwBAHox1EECwIFIkmdk9XGrml1RMmyOR0ZOvLF4GC4cAAC9HOogAeBAhJS/Hjz6kvxiU8g0mz3N7ghbJlquAQB6OSRIADiQWFgscifEfrQwbgYAANCKDQAAAADxQoIEAAAAgPggQQIAAABAfJAgAeA7pJRSyq4uBQAAdGsYSQMAe4TD4Xmff7FtyxZN1wuLi6ZMm6bpOmPslRdeUFWViDRNT05NGTZiRFp6OhFJKdetXr1uzRrLsmacdnpGViYRmYbx6ezZTY2NsWN6ExKL+/frP2gQI4pGo5989JHf5zth8pS8Pvl1tbWfz/nENM2pJ5+Um5eH2AoA0IMgQQIAEZFlWT/+4eXr1qwRQhAR57xPQcGb773r8XrvuuNOb0ICETHGuKIY0ejH8+b2LSiwLOul55//5KOPDcNobm6+5Y47DSMajUafe/LJrZu3xA7LOVcU5errrrvlzjvCodDTT/ytvKzs4Scez83Pe/Kvj//3jTf6Fhaedd65iI8AAD0LWrEBgIhoycKFa9esUTXt3At/8MOrr87v02fchPHexEQikkSmaf7w6qvO+cEFubm5jLHLzjtfURTLsr6aO9fj8ei6/p/XXlfaHY0xNu3kk6+9/idDhg+3LOu/b7xRWVHR9pCqal9++tk/n3tOVdW77r03KTm5K54xAAAcPtRBAgARUXNzs2kYbpfr8quvHjJsWFNTk9vtNg1DURQiMk3zkiuuGDBw4MIFC354/gWmYdTV1q5ZtaqqovKa66/fvGH9ssVLVq5aMXjY8NjRGGPTT5lxwcUXb928+YfnX9Dc1ORr8Tmdztij69eufen551VFuf8vj5wwZbJpml32tAEA4LCgDhIASEpZUFjo8Xp8Pt+MEybd9vObtm/d2r5lmTHW0tRUVVn59bz5CudcUdLS0l998Z+Kopx06injJkzQdP3tf/9H4bztgPV1dSVbt346e044HE5NS0tI8MYesizr748+akSjw48bNeP00w3D6IInDAAARwZ1kABAjLEBgwY9/MQTf7z7HinlR++//9ns2ZOmTPnbc8/GdtB1/f9u+aVpWTVVVZZl3fCLm1tamhcuWJCQlDTquNFGNKpp2qIF3/j9/rZjPvv3fzz35FNNjY0K52ecfXZqerrf54udy2a3Syk3rF33yccfn3P++aiDBADocVAHCQBERFLKydOnf7Fo4R8eeCAtIz3Wx3Hj+vXEGBExxgKBgLCs3Ly8+x566LqZP533xZdSStMwpowde/vPbyKi+rq6yt272w5os9s1TeWc5/ctuO/ee3m76klN16fPmEFEt91wYyAQ6IqnCwAARwQJEgCIiAKBwJ/uvmfb1q2XXnHFW7M+0nVdSlm6Y2fs0Ugk8tenn/rg008+mvvl5ddcHYqEv5o7l4hSUlMK+xX3GzBA1/WmxsbVK1fGkiJj7M67fvva229LKbdv2/rKG6+3JUjG2M233frYU08W9+unqOp9d90dG/0NAAA9CBIkABBj7I1XXnnt5ZdPnzzlvNNPO23KlHA4rGpacf/+RJKISMqkpCRvQgJjzDRNIxrdsG4dY+yWO+748JNPZ8+ZM3rcOCL65KOPVU2LHdMwjD6FhRdeeqlus734zLMtLS2x7ZzzrOxsxti1M69XFGX2rFkVu3Z10fMGAIDDhAQJAERE48ZPmH7KDLfHs271mkg4XNSv+L4HHxw8ZMieAEnEGIvtyRirra1dv3YtER03dmzUiAaF9YNLLjEN87M5c/w+H2OciBgxKcTl11zjcDi2bdkyZ9YsRVFjbeLEGGPsvAsv7D9ggL+l5de33WbT9K564gAAcBgYJvIF6LmCpjFj7ocfTT6zLd4dHiklY0xKKSxrx/btSUnJKWmpsY2xR2O7tZ1lr2Hae21p/9ABrjDtHz3C8hOR3zBOnvv+hjN+eITHAQCAQ4Gx2BAXKYVFVoQi9cJXTqJLhtBKUuzcW0C6hyl2IkZHHD6AtQ6XUVS1/8CBUkohRPtKx33uf+AtB95+KI8CAEC3hQQJh0paUWvL68K3k3v7Mlcec2UR17qgHIyR4bdql5Nvu/BVKMUXKCmjiCsH/0U4NLFxLch2AABwAEiQcHBSmLKlxNr1BU8brQ24nHVJcPwunjqSiKQZsra/azZtU3KmMUcaMfTr3Yfampr1a9a0b0t2OJxjxo2d98WXRLL/wEH5BX1Jyi8/+8yyrOJ+/bOys+bPnRtbiibG5XIdN3asqqpEFAwG169Zu7u8XLfpfQoKBg0ZIqX84pNPGOMnTJnscDiklOvWrKmrqe1bVJifn//l55+3PxQRMcaGDh+RmpYaDofXr1mzq3wXkczOyRk8bJjL5SIiIcSqFSuaG5uIEUnpdLkys7Pz+/RBqAUA6D7QDxIORpjmltdZQiFPGcF0D8UGVbT+35EeXEqKRknXD6Mluu30MlRnrH5UKbpQSTvuSMvT0xxKP8jZ/5t15y9+0X7GnPy+fed//fWM6dM3rF/ff8CAL+bPX7Rs6XmnniaFmLd0iW63Txwx0uFwtO1fWFT0xvvvud3uqorKmVdfXbpzZyQS4Yy53O7Lrrzy+ptvGpiTq6rq/OXL8vLzTdO8+fqZ87744oZf/OLKH107tKDQ4/W2L4+iKI8/8/SkKVNu/PGPly5eEg6FiMhmsxUVFz//+qtJScmGYVx76WUrly+P7a9pmsPhuPDSS++467eWZe3vaaIfJABAZ0IdJByEVb2QbIlK1iQpJUnZ2mOOjkJ8JCLGgu9/4DjnbGa3x/2rsf8nJXOk6uPvN9c9LXQvTyg6OgU7pkjGmN1uHz1urKpqRJSWnh4h+cBjj11wxhk7tm9fuHTJow88qCrKZddeM6C43/ayUs6YEGLCpBPsDgdJysjKVBTFMIwbfvzjrVu2JKekjJswgSt83eo1AwYP5pwzxhj/tgKYMWKMESNFVaefcorL7dq4bn1NdbXT5Ro3fjwRZWVnP/7wXz6bPScxKenEKVNUVV25fPn6tWtvu+Hn/3zzDaJY71Y2ZPjwfv37rV6xsnTnzn+/9tp5F19UWFTURa8h7MNrpVtWNdYpHV83bEoxPCHlqoKBHX0iADh0SJBwIKJpi2wuUQdeQx3WMc6qqDA3btJGjTzM32csNpBYGXClufF5PuwmDKzZp6ycnCeefdbtdhORlDIajfYbOOCkU0/58tPP7v3Nb7dv25aekTHz5zeFzD2rVAf8/mdfftnhcEqSUkrTNJcvXbpl40aH03n/Xx45acYplrS2b91WUFQUDIX2eUYppdPpfPWttxSixx555G+PPpqVlfXSSy+FSdbV1v3z2WcUVf35bbf99IYbiOh/H3zwyxtuWLt69cb1G4r6FceOcMrpp/305zeV7txxybnnNjU21VbXIEF2KxWhQLJuOyenoEPPojL2j23rNrQ0duhZACBeSJCwX1JaVuksbfBMScSk7KBkxj3elgcfTnnztcM/BGNMStJcPG20VTpL6Xv20SvdsaOmuvqe//uVpmnhcPjXv/9damqqlPKqH/147mefb964kTE2buKErJzstv0dTudvb79D1TQp5VnnnXfC5BNLt+80LSs7NeX4iROjRpSI+hQUCCH2TDm+H4YRFYoSa32WUkaJDMMo3bmDMaYoyqVXXB6ORoho5OjjMjIz6+vqqqsq2xJkwB+orqlevXJlS3OLy+VKSU3pwBcI4seIUm2OIQnJHX2iBE1X0csZoJtBgoT9C1YyRwapjthELx10EmN7iVFaeqRHYUySVLImGSv/wvOijGuoidxLMBD48J13JJHf57/ljtspNZWIho4YRkScc9MwZpx2Gue8rWO0ruvvv/MOEUkphwwbNnna1F3lpYwxVdUcDkf7/ojff6FjhzhApXVjQwMReTye2OkYY6qmarpumqbf52vb7fWXX37rzTcrdu3SdP3k007rU9CxdV1whKSUftPc7GvUuDLIm6gxjsFPAMcwJEjYL6tyMU8f3aGnEP5AZN5XztNPpXYjYw4PI0ZEPGuSbNzAUkYcpQIeO7Kysx96/HGH0yksKz0rMxbd/vv6G7QnwGlv/uvVk087TWtdk9Dv8707e3ZsbHVaRoZlWTm5eVJKwzACgYC9Xb9Vj9sbO0hLSzMxJoSIRqKMMYfDub/CZGRlEVFzU1MkEnE6nUQUDoXDoaCm60kp31Y0RqNRwzBsdntWdvbTTz/tN6Id8tLAUVIfjfxm7eK6cCgirJFJqb8fMlbFtFAAxy60C8B+ycb1PKFfRw1MkVIK0XL//VZ1tfPaa+konYYnFFmVCzC/wPfZ7PYBgwYOGjJ4yIhhuqYTUTAYfO2ll6WUV193nW6zrVy+fNXy5W0DYizLGjRk8KChQwYNHZKSmiKEKB7Q32az1dZUz5k1KxYL3n/77fq6OiIq7t9PVdVPP54thaisqNi5fTtjLDc/b3+FKSwqcjidjLEH/3AfESmK8uE779ZU1zgcjry8b3/r5ttve2f2R5Zl7SgpefWN1xUVd7zd2tu7tk9KzXxn0mmzJp9JJF/ZubntIZ8R9ZtGXP98rV1yAaB7whUZ9ktGGztuhkUpZeD5F0PvvGebdIKSkd62gN6R0j0MY7H3ZVdZ2aXnnhebmjElNfWZl196+81/by8psdlsd99999LFi1ctX/67X/967ldfx/Z3ezwXnnW2qihE5HK7n3zxhaHDhxcUFW3bsuU3t93++iv/aqqvr6ysPPXMMx5+4olzf3DhM3//+/NPPvX5nE+2b9vGOdc0bfzEifsrjMvl+vXv7rnz5l989MEHi7/5xuVy7dyxg6ScPG1aUVFxIBSM7WZEjaLifpdfc80br7zyj8f+On3GKd4E7/6OCV3u06ryf4w5MfbfNxYPu3fd0msLBsY+jS/v3Hzf+qWH3pdRSJnrcC899aKOKSnsW004tKCu4qPKsrk1FQHTwJW0o9m4ck5uweS0rKlpOcm2uCck6XJIkLBvUpjMltpBBxd+v//pZ/zPPMdT07x33MZ0nYikJazSUunzKX378ISEwzw0U8mde8RN4scg0zS3bd0a++/M7KzKioo///73Qog/P/qoSfTI356YMXHSlo2bnnvh+VPPOIOIFEUpad3fm5BgWZaqqs+8/NIffnvXogUL1q1eLaXMzc8ff8IJQogfXn3V5k2bvvryy5KtWxVFycnNveXOO1JSUiPRSOt6id8pTDQavfDSS3eVlb/15ps11dVElJiYeMoZZ9x13x8iRpTamj4ZM03zymuvmfX++5UVFR++++4V116DKWy7rakZOQvrqs/K7kOS3i7fPiIxte3PfmZWn4vyig445uq7GFn4Q3eWFiP69LZ1/y0vsXEl3+ke6E16bOQJdkXBH6BDMaIWI7q8sfa/ZSUPb1xFjH5UMOjyPv2catev2XGIMKM47JsUUXP989qwG47+kYOhxl/eGpn/Falqyhuv6kOHElF02XLfY39lbnfCPXfzrEzG+eGFQCksq3yOkn8a6x0jNw9lRnEjGt1rwh3OmN3hCAYCksjr9XLOiailuVlKqWqaw+HwtbS0D32MMbfbzRhjjFmWFQgEynbuTEhITElNcbpce85iGL6WlvKysqSkpLSMjFgHyrZShcPhSDisKIrb4yGKzb/EhBChYLC2piYaiebk5zkcjrahPMFAwDQtu8Nus9mklH6fXwhL0/VYp8l9wozine+RTSsTNNtPigbHfiwN+H61ZlGW3RmwjIgQT42eYuecMXbYLQxtv3j32sUq478bOvZolh6IiGh30H/hgtlpNsdTY6boXNE4VxhDLOg0jMiU0hQiKqyfLptXHw3/e+KpeU53V5frkKAOEvZBEjGmacNuOGqNy7HDCmFt31F//c+s0p1KdnbCvb/TBgwIz5vve+QvwtfimTnTedmle65cRzJ5EC5+36XpeoKuf397QmJi+x+97ep993qoPc651+MZPnKklLL9OjeapiWnpKSmpbVtb//Osdvt7QffxB7inDtdroKiIiISQrTWVjIicrnd7Xf2eD2H/nyhS0gp+7g8L42bPre2wqGok9Oy2j7Fh30NwSicDiWlfHf3jrvWLL6+aPAN/YaGLJMkMcTHziWJFMYURdE5f23CjD+uX3bGvP89O3bqhNTMri7awSFBQhtJlimtMAmDDL8w/WRFSXOT6iLVQYrOFMcRTpFjLF/ReOttVnWNNrt3rzsAACAASURBVGRI4uOPcZut8cabIgsX2SZOTL7vOZ6WRm3VjvjmOFSd/ULFvtS/nxFj9rf9AEdrm9DnCAsmSBpSHHw/6Bixv6BNUU7NzGvbdHRvQeHoqomEfrV64X8mntLPkxi2LEboQ96V9lS3DxkzNT1n5rK5X047L6Xb94xEguz1pJTSFPVryFcu6teYZR9RcxlJISk2UzRnjJHq5Jljef6Z3JnBU4aTM7NtZcNDZ2zd2vCT64U/oPbrl/TsU0xRG2+6ObpipfPccxIeegCR8fConKmMVYWD2Q5XL685CFtmvhNVlZ0tYBlV4WBbbyhGLGiZ1pFFeY1zO1dla2uCxrkhhKLgEnGU3bz8qyv69C90J5itLQDQ5STRtIyc0Ulp1y398p0TTuvmfxckyN5O+HYYi34rqhZJI0BExBQiRowz4kQkY23CZtgqn2eVfUmMeEKxUnS+Nur2+M4SCDTecqvwB5SU5JR/vaSkpjZcNzPyzULneeclPPQAY+yImq17MY3x4YkpsyvLZhYPiYreWwPHiUr8LQM9+218hw7yVW3l8obath8Vxja1NDYZUe1wOyJbJBJU2wBvovVtKqWwZY1IwopER9Pjm1fvDgf+NvpEQm+BbsYQ4q+jJp0678OPKkvPzO7b1cU5ECTIXk1aRnTuDaJ2JWMK4/sY/9VuMIVCpBCRaNkplj3IEvurBece+olEdY2ormGK4rnzdiU11di4KbJkCbPbXD++pm3U7ZE+mV6JMdbPk/hVbcWN/YZFqfcmSJXzRfXVwxIRMjqPlPL2gaO+v/23axeNSky7MO8wVzC/a83iQrf3R4WD9nlGZJ2jImxZb+/a/sDwCS5VM6XEa9rdMMYeHjHxnzs2IUF2hpBllgf9ZUGf3zCiQkSEFRUiYlmGFDrnOld0zm2KojMlUdfzne58lwerrBKR9O8QjZsZUw49wDEiybko+5jiSZDEOTFGNpvavz8RGatXy0iELGGWlukjsH7METk5I/exzas3tjT2cfXeNtyacOjtXSXPj52GkNFp9vc6m0JG2i16GS9TiqjY96/jL3u0BE3Db0RHJ6UZUuA17YaklEm6bUNLQ1cX5CB6ZIK0pLSkCFnmf8tLvqmrWtNUXxpokUR872EYjO0ZmCuJSLb+lyQppBzgTRqWmDIlNfusnD4ORVUZ573w8iTjvygzRoLi7QTJ01KVnGxzy1aKRIlIyc/jNpsQovme39unTD782R97PSnl4ITk64sG37d+2TNjpyi99b7ozlXf9PckjkvJ6OqCAPQAActM6vajNHozxphb0+oiYUMIjXffq3oPS5C7gv4FdZVzayrWNtWvba7njKmMccZVxg+YhFjb/7TZ7m/Z6mv6T+m221YpI5NSB3uTJ6ZmTk3PSdFtuNM96rjL5b7+usZf3hF46y19zGjbxInakCGRZctkMNh8358S//wnpvWYOVS7ldhY5l8PHn3mvP/9r6L0B7lFZi8bj8yIvqjeVR0JfTzlrK4uC0DPsM3X3N+TYGHms24sUbPlO91zKsvOyum+Ddk9JkEGTOPRzav+sXUdb53+w6moRxL1FMYUpuicpJSrG+tWN9a9WbY1UbNdVzTousLBXm0f8+cdgzQnKTqZ4Th+JTbkxZYc76kcZ5xhrNvgf+bZhP+7kycleX93d92FF8moEf7sc3H7bUomao+OyL3Dxp3/9cdCyvNyCnrPLRAntqyx5hcrF3xw4hl2RUUTNsChqAwHUnQ7YT2Rbkzn3Klq1eFgVxfkQHpGglxYV3XhgtlBy3QoKh3t3jDtj9YQDd+3ftkzJevfPeH0wQlxh6SeRRJxV64+4X5j9ROyaeuh/Y4kxabmTddG3hbnyaQk8txys2hoqDn1jPTP5mgDB3h/9X8tf/qzDASkaRxG+SEm9gYek5z+xoRTfr9uyZKGmplFgwtc3mP7y4ERVYQCT5esX9ZQ8/SYKcMTUwj95DpdZShQGQ62f9FtXKmPhNc1139cWXoYB+TEKkOBiGVtaG6ItOsNKYky7c5sh+uIiwzQY3T/y1kPWNWwLOA7+6tZNeFQp309CCmTdfvCky9I0G2dc8auInw7RdlsXnyJqFwoymZZZZ9QNLCPUTVSSkbcW6AUXcjzTmKMicpv1CEz4zpXrH5ImmbzH+/nXq/31ltEc3PN5GksMSH1nbeUlKMzilYKyyqbo/TpLasatom9vBHL+tuWNf/YtnZSatbxqZknZ+Qee1+6FaHA/NqKJfXV82oqzs4puG/YOFfPWUb2GPPXLasf2rjSxpW2LYzIkEJIedjdyqWUksjGv7MosyXFJX36PTxi4pGVF/Z4vXTLhuaGXw06rnd1eelRVMYu+eaT83IKfty6amg31N3rIP2m8dNl8ypDQbUTO5NyxuqjoV+s/PqZMVNtinLwX+i5rKix9AG28WW1+CKl3+XqiFtltFEGqynSLK0IScG4SpqL2ZOZM4uIZNNWc8MLVsk7at8zD3psKWUgalU2h0vrg3X+aEVT+MT+qX1THGl3/7b53j9YFRVWRSUpiueGn7XFR0vI91dVfLaprqYlnO6xXTep78j8hN44wil+sfsrnfPbB406PbvPm2VbP68qf2nHJiYp0Wbr7reJh4YR+Q3DlCLL4ernSXh67NSTMnK7ulC9minEwyMnXt6nf0ef6I/rlyHr9FrV4eDrpVvN1vluLSkHeJMuzivCGlRdrrsnyJtWfLWkoVrnnR3jFMZnV5Y9tGnl3UPGdPKpOxVjUtFky05jxSOkaKR5yZbIbAlM9zBuI8aksKQZlEaLjDRTpImMgJQWCZMOVsPXEIj+a1HZeysryxqC54/KOW1Y+pg+ifnJTk1hxJj3N78xSrY13fmrxEcesk+fRkSmJZpCRl0g2hKxbppWuLMuePnzS95fXbng/6b0TXF2ymtxLIgNrBmSkHzfsOOJyGdE/aZhdft2hkPHGEtQdaeqxu4r0PGxywVNsxPOEhEW5l/rtaJCvFG6tdHY018/bFm/GzqubaYV6ELdPUEure+C+BhjSdkQjWeISQ/EvUVK5jhr13xJglkGWfUUrhckW3tYS9qzUir7dsFUxpk9SSk87wCHNSxxyl+/Xr6jiav8B6OyzxiaYdcUX9jcVOUTkgzLChjSX2VFZv427OlXPnvLxoqW5rDZEjLzkuwPXTQsw2MblOW5fmrhg7M2PzVvx4MXDOno1+FY0j5ReTTdc0yPCUN87FaklNsCzSsa6lTOT0jJzHQ4ZWtfrvXNDV5Nz3O623b+6+bVp2f3GdC6jNDT29b9qHBQ29W+MhR8s2zLLweM7OznAN0SZ9Q2VRlnmMOyu+juCVLBN0RHksLUJjzId31mrv0HhRqkFWVSMEZ7VzFKSYxLxpnqVNJGaKPuYJ4DzS/AGP3louFPz9+xfGfjNyX1n6yv1hRm1xSFMynJsETUkoyRXVe1VetyEu2XT8gfnOkpSnflJDqotWIpxaUT6wFdiQEgZn1Lw0ObVp2XUxC0zB8t/fKJ4yYVubyxftWzKkqHJ6a0T5Bv7Srp70lsS5APblxxcV5xsm1Pgtwd8v9x/XIkSIDurLsnSOhQoqUk+vEP9Bn/sp8/T0ZbzNV/sXZ9TVaEvq2GbK2D9OZrA69U8k8jyzRWPEihKn36i/s7rMLYlP6pU/qnxn5sDBjEZIJTJ0n8gJGwtcaC+cPmwx9vcdnVqyfmH60nCwAd6t3dO37Zf8SopFQiSrM7/rVz8++HjI09pDC212c/5bvjFDWutL9flJKK3d6OLjAAHAkkyF6NMUUaocgHpzNHGs+ayBP6a8N+SpqL2RKk6mSMk2VI008Rn4w0yro1kQ0vyvp1MupT+19ywMPu+SqImOLWf695fsHOj24+YdqA1L3GxBiWqGqOrChrKm8I/fD43GSXHnu4vDF4+QvLgob11BUjB6S7JUnURQJ0f9XhULK+Z6WTPg7PbKNMHvKMJCpj7bu1oZMbQPeHBNm76R6muaQVpnCjtWOWRf8jinV6VCm2WLaUJE0S1rf1g0TEFHKkHcrhDVM8M38HV/hlzy7RFO7UlaxEh1PngYiobAqGDGFYwh8xo5Z8cm7JwxcNO2Vw+sLtDVc+vywnyb7u9yf3SXGgoxtATzE1Pee93Tt+UjiQMfbYllVT03MO8OkVJNs/akjRPmxyxkqDvg4rKQAcBUiQvRpzZmnj/2iu/qtsKmHSIsaJJEkiaRJ9d4glY7HZQ5lq57lnqsNuPJTjO3Rl430zXltUvrHav7XGv7sh1FTZElvUhjOWkWArSHVlem2DMz15yc7RfRJfWVS+trz5qStGnj40MxYdMdgWoKc4JSP3he0bH9y8KmiZ45IzLsvv1zbfsM75XoOp6yOR9o0SKbq9/ZwBgmS6DZMwwNHRYkQ3tjSOT8lAxfbRhQTZe0kiJkkpOFfJnmRte9va+aFs3ibMMBMmkfi2xlFK4lwqdqa51awJSv/LecZ4xpSDtk9JKRXO+qW77zprYFlDsMYXOb5vUsQUliSFkaJw3m6Md2wammsn5LOJ3zaCS4nBtgA9g5TSq+q39B/e2oGaSMq2j++N/Ybt9Un+6qQL2m9ZcPIFvN0VZXRS2qrTLu7QAkMv4VDUKxZ9Orem4t8TTpmcnn0szW7W5bp7gvxJ0ZB71i62K11QTkmy8Jjuys2IiBGTRPZUdehMdehMEaySzVulv4KizVIYJCVjXCoOZk9kicVK8hAi/p1fP/DxW2fs++276x+as/V3Zw+0LJnptTttnIgCEbGzPrihosUfNc8ZkTUgw61yxr/b2R7p8QhFLKs6HKyPho+NWdmlJK+m5zhcx/g8/z1T7PPO9vOp/f47cK95NpTv1lAyxhT0foYj5jOi1yz+fEFtlVvVrlr8+ZOjJ5+ahdGZR013T5A/7zfsq9rKuTW7O39an74uz8V5xZ180i7Q7oXlzkxyZh5g38NoU2aMfbyuSlXYnz/e8tjn2zSuqJwkkSUoalmhqCWJHvl024A0139/enxukuMwnwW0UxUKflpdPruydE1TQ0SYpjh27rkVxjhjJ6RmjU/NODUzv4/L09Ul6u1WN9U9uW1t24+c2MaWxiYjetjXa0mUZrMXuxNE61gaTmxHoCVRO8bXmIWjTuf8Z8vnLamvid1zqozduHz+c2OnzcjMNVETeTR09wQppXxk5IQz5s2qi3Tyuti23w4ek2HvvR1xpDBlw3qeOqL9xkP5E0RMsbXaX94Y3FwTCETMq8bnr7h7+p9mbf5iU21TyAwaZjgqhJQum+LUbU6dp7lsV07MP3VQepJLjzWtW0Ks3uWzhBiVn6geePof+C6fEX171/Y/b1he6PaOTEo7L6dweGLKAG9SV5frqKkOBxfWVa1orH1v144/rl9228CR1xYM8h7Ts6Z3Z4XuhG/qqqrDwbYtCmObWpoSND3HeXgLsrONLQ12rpYH/eK73/FTcrKPrLDQuwRM46pl876urWxrw2SMqYxdvfizF8ZNPzkjF12kjlx3T5CMsVyn560TTn1sy+r/7S4VHT/Dg5RyYmrWX0ZOLHB7e/cwDmHt/GivBHlQa3Y3/+adDWt3NZ9QnPLTKQUFqc4kp6Zy/ruzB/3qtP7BqPBFzGDUElI4NNVtUxya4tSVbxuvJZOMYkN2Lnhy0cp7TsK84oduU0vjj5d8UeROmD3l7FSbnTMmpJREzdHIsfE2llI6FPWkjNyTM/IkydpI6JFNq06b9+EjIydOTM3q6tL1OlLKC3ILL8gt3Gv7r9csHJ6Qelmffod32F+vXljg8l5fvI+VqHr3BRniwIh+umze4vrq73eBcyjqTSu++ttxJ56elY+ayCPU3RNkWcCXbncM8CY9PWbqooKqBzeuLPE3N0QjlhBH8VIipdS5kmyz9XF5f1Qw8JycAiIKmMayhpop6TlH6yw9T5yvcDBqXf/yiqXlzRqx4kx32LCaQ4ZhyYDd1BWuKdyuc4eux7pLCSktIS0h6wKRUFSEolbYFNW+SEVjaPb66jXlzeeOzOKMzVpXddawAzWsQ8yG5obT5n14bcGgOweNMoWQRG0dxo+ZL909PWv3zBdAKbr98eMmPb1t/TWLv3hjwozRyeldXcDeZX/vKyFJHMEXsyTaX03BMfNOhg4VFda1i7/4pq5qnx2mY++iqxd//ur4k6el5+BNdSS6e4K85JtP+ro8tw4YOTYlfXxK5ruTTt8V9JcH/e/v3jG3ZveOgO8I+0dqjA9PTDk+JWN6Rk5flzfL7oy9n76o3nXX2sXHp2T06gQZJ5vKr57Yp2Br/fLSxr99tu1JxjhnqsqTnbpd46rCVM7aYqkgKQRZQoYNyx8xhZScqCjdPTDTfcnYnN+cMWBFadOFTy+6cdreNRywTz9fPv/BERPPzy2IHtWbq+6MMRY0zZ8UDe7j8sxcNm/ZKRd1dYkAoIuZQly35MslDTUHHm/n1fSbVnz1yIiJZ+X0xejsw9bdE2TQMr6s2f1Bxc7rCgfd2G9YjsOV43DlOt0TUjOJyJLi69rKRXXVC+qrSgM+IaUlxT7rpTkRY4wTUzlLtTmGJSYP9aZOTM0YnJDcto8lRcAyVzXWPbd9w0cVpYzo+JSMznuqPR9n9LOphT+bWkhElpTzttT/e+kuS0ghZShq+iNWMGpaFgmSKuceu+LQFLdddahKmtd+2rD0nAR7QyC6urz543W1L3+zLDvBdtup/S4YlY2mq4N6aONKu6KcmpkXsaxe9VoxxqJCTE7LTrc5frZs3lNjpnR1iQCgK121+LNF9dU6P/h0DZaUP1ryxZsTT0E90WHr7gmSiDhjCZr+WumWt8pLhiYkT03PHZWUOtiblOd0K4xPSc9p+/OHLdNnGj4juleIZEQ2RbFxxcaVRH3vAX1SSkOKz6t3f1xZuqS+ZnNLo8q5yhh6SMSrdfoeIiYVxorTnCNyvdW+6M7agFNXPTZV4Q7OKdY/T0iyLBkxrVDUKm0Ivrdq96ZKnyXo1GEZMwamv37dmDOGZzk1ftCTwsrG2qe2rXv/xNNV3ktfLoWxZ8ZOOXXu/5bUV4/DXR9ArxQ0zdjI60OJjzEJuu1ny+Y9NGLiOTkFnTDK4tjTAxJkjM4VSbSmqX51U72U0pSywO2dkpZ9SmbelPRszhgjpjKerNtTdHv7ShgZ+yclETFGphBEJEgaQqxtbviqpmJJQ82yxpqAYSiMMcYcautrggR5WPbMCkcyP9n5symFhhBRUzQFjYZANGRYYVOEopZhSSJSFXKoql3jdo0nOLQEh+bQFZVztbVrQmz1GjiwhXVVIxNThniT/abRaRWQUkqN8/9bvWhwQtLFeUXaIV+yO6QwROk25+lZ+Wub65EgO9OG5obKULD9m07nSkUoUBsONUYjh3FARlQa8FWEAkMTUqLCatsuJA3yJuY43UdeZjgm6ZxftvSLpfU17W+kpZQK4/KA0VAS/XT5vETdNjktS3R8OY8xPSZBxrA9s9YyhWhX0P/yzk3Pb9/AGevj8qTo9mTdlqLbnapqV1SHouqcG0I0GZFmI9oUjYQty6PpphARYVWGAzv9vrCwVMYUxlXGem39zVEnTZOIGOeSM8akrnBd4W6bGsdcj63t1oiPByWlXN1Uf1ZO387s/qgyNqd61583rtjubzaE+Kyq/KZ+w0cnp3XhLVdUWGdn9321dMuPCwd3XSl6nQ8qdiyorZqQmvntl7Sk4YkpROQ3jcM7ZuyNNK9md9sUDArx2VVlo5PS/jLqhKNQaDjmtBjRny+fv/i7jdeWFKMS0yanZ7+7a7vSOgjPxpUL8oreLN3S/mLFiH69ZtEv+g+/KK8I9UZx6e4JMtPuqg6H9rmiBmdMZ4rOFSlldShYHQp+f58DUBhzq9oBdpAk022Y4DpuDddeZ1VWJj70gH7cqL1XroktpPi9jXKv+Xp6VVe+I2NIscXXdHP/4abspPtnRrSwvnrm0rkKYzpXdK58U1fVEI28PmGG64AfqA4lpMx1urf7W7qqAL2Tyvg5OQU/Kerw1B6wjL2W1YZjlcb4u7u2c87OyOpzKPvbuXLp0rkrGmv3arwe4El6efxJr5VuKW034jZZt/+oYGBDJPz89g2yXTOjJNrQ3MCQIOPU3T+TD4wYH7JMecAGZXa4DnzqJN32o8JBBz41fJ9ZUmKsW2fV1+/jsT0N3HtvRF48bJaQPsPId7qPZP6UeM2uLOPtPkGMsY0tjbMqStWuS/6MMbui1EfDXVUAICIppSVlwDSCptmZb0g4ZlhSvrRj08xlc3+1euH82oqDXlJajOiF38xeXF/dvprJkmKgJ/GFcdPcqharHG//jc+I7hg48so+/WPfPIwxU8pzcwruGjIGg7Lj1d0T5KiktJ8VD+v8JQ0tKc/LKUy12VEdFjfOSVFYrFeAlGQJsqw9/2KfTyFIiG+7mcZ+FGLPf7fbWQoho1GyrP2cCUiStCuK6MTh6n7TWNpQs1ezACP6zZpFJYGWLvy0cGKGwFulK0mi10u3XvzNJ5ct/HR+XWVXFwd6GJXxt8tLfr9+iUfTI5b1s2Xz3t+94wCXFJWx65fOXVr/nYl7pJQjElNfmzAj0+7cXyCURL8ZPPoXA4YzxkwhflI0+OGRE/kh1CvBXrp7K7aU8oER439UOPAP65curq/2Gx0+VkBKmed03z5w1MX5vWBR7A6x51MrQ6Hw/K/MkhJz02biXCsq0qdOVgv6Bp7/p5TSdeXlSmoqEYXmfGpu26pk5+hjR0eXLTO3lZg7dvIErzpokOL2NNzyS9cVP0z88/1d+oy6tU6+a26KRr7fWBy79P5l46rHjpvU+fd70E18UlX+Rc2uZ8ZObYiEb135tW348eOTM/CtDIdCSPlq6eZbVn4dmy+FMWZJ+avVi5J1e1+X5/v7B0zzqsWfLa6vbh8fLSkHeBKfHD3FpWrf6y+1txuKh1aGAo3RyJ0DR2HOuMPT3RNkQzSSYrP39yS+On7G5pamx7esfntXidJhHWKK3N6L84qvKRjo0XQiWt/cMKTdhJEQD2Zs3NR0y61M14lzaVmhWR+pH3yQ8srLLY8+RlIqKSmuq66QUgZffyMyf777lpuNjRsC/3qN2+3EmYxE5Vvv2KdPS3n2KXXgwK5+LrAHI6qNhv2mYW+9avdxeYSUKmMuVasIB+dW7z45MxdNQb3Th5Wl9wwZk+tw5Tpcjx134ovbNxyfnBH7Wv5g945Y54dDP5olJSb47D04YzuDvr0GJ0SE9ZOlX84sGrLXl74k+snSL5c31LaPj0LKMUlpz46dmqDp1sESIWNMEt0zZKwpZVxvS2ivuyfI87/+eGbRkHNzC9yqNsCb+OSYKRfmFb2wfeNWX5PfNHymYR7BEFQppcq5S1Hdmj4mOe26wsFjktOJyBSiIhR4rmRDsxF5dNSko/qEeoHWKki1sCDpb4/rxx3H3G6zZFv9lddYFZVWXZ1n5vX+Z58Lvvee88orZChklmyTluW64nLR2GifPl0fOYIUJfDyy77HnrB27bZNncocDtwgdhMKY6ub6mLXcimlxpU7B466cfl8SwopSZB8dIsxJT37+zMbWFK0GEaLEV3bXD+3pmJOVVldOJTtdP9x2Lgzs/piJrZjg0fVApYZ+7S2GFFHuyWJXao2OCE5rlt/9KTsVaSU9w8fH7TM/5Rta9vIGTOlfKpkfZ7T5db21EQyoue2r0/W7P08iW3fN0JSrtN195AxblUThzwaU+W8u2eg7q27v3pN0cjtqxa8sGPjrwaNOjEt26Go0zNyp2fkthjRmnCoKhxc21y/I+Ar9bfsDPoqQoFD6QmbaXcO9CYN9CYO9CT1cXkSND3d7kjW7bFH1zTVv7B9w6dV5btDgWsLB3Xw8zt2MeKJibaTpsv6enPTRnNnKSmciMg0nJdf5n/+BXPjJnPbVmLM3FnKk5KUtDQlLU0WGKKm1qqsoKhJnBORNE2Gwdndhsr48oba2C27JeXxSWnn5PR9u7xkTlW5xjkRrW9u/N36JX8eNqEtFK5srF1QV/VNXVV9JFwdCbYYUU5MYSxBt/mM6C0rvi6anDDAk4iwcAw4N6fv3WsW3znouIqQ/8mt654fO611Yld5UkbuSRm58R4Qt469B2OsORq5b+g4r6o9W7LhOyNjhNjh931nZ6LGyLcTjgopRyenPTh8QprdgdEwnam7J0jGiDG2qaXh0m8+yXW6r+w74KK8ojSbw6moRW5vsSdhUlpW+/1LA76qULDZiDSakYBpto7DoiTdnqjqXk0fkpC8V69bxlhUWC1GtCESeXb7+qe3rdO4EpuppNdfug7ro9j6qkkh/I//zf/CizIQIJuNSUmqSpJ4UhJLTKBgyFiz1li3jmma2r8fEYmGhsZf3BpZvIgsQZpGQmCQdnejcr6qqS52cTelOD4lXUj5myGj59ZWWELEBkR/sHvnOdkFhhT/Kdv2v4qdnJjGucJ47O3kbFcvRYwZQhz2xIHQrUgpJ6Rk/mbImMe3rHYr2ivjT8px7JkA/LBTIOJjr8IYE1L+YejxjdHIf8tL2uoO9jnCpW2LlLLI7X1mzJ7G684sMHT3BNmK2RW1Jhx6YMOKJ7asGehJyne5hyakDE5IGpWYlm53UGsW7OPy9NlXr9v2Yns2RSM7Ai07A74tvqbVTfVbfE07Az6S0sYVXLaIiJhC8rBGtgpJRCQp8NwLvr/9XR87xnX5D0nTWu69T/h8RJJ5PM7zzgu89npk/lfm1q2kKIl/uFcEAg03/Dy6bLnzkottUyZbu3b5/vLYYSVYSRKJpKO0GNHt/haXohKRKeXE1MyoEH2cnl8POu7+Dctjf66oZV235Eu7oqic5zhcsSlAW5eFItpzXyKJyJDy7IzcQpcXV/1jQOyyOTYp7dXjT45taV0JDJdTOFSMsWYj8tCIiSk2+7MlGw66vyXFiMTUJ0dPSdbt6AzT+XpKgiQi4ozZFMWScn1Lw7rm+lkV4ZRBwQAAIABJREFUpZKkJWWKzdHfnVDsSejnTuzn8abojkTdlqjpLlULWuauoH9XyF8e9NdFwkTkM4zSoG9NU11lKBhbxpC3XuN0LEvTSkrJmCLDdYf320REUkSXLSNV1UeNcpxztlVfTzadfEREjHO1oICIQh/PJsaUjHSelSmqasxNm5XUVPdPrlP65EcXLWaH9+eQFvl3HWwQHhwOTrSmuYETY4xJKRnRyMRUS0op5VnZfZ8r2VAdDsbmXTOkMMxvZziP3bMpjFlStv/DJOn23w8Zi89djyZJSilF+1uEVrE6pEOfUld+b9fYEThjiAa9R6xV8K7BY+oi4X+XbVP3P8mOlLLA5f3nuJPcqor42CW6e4L0avo+16RpfU8xhVFzNLK4vnphXZVFUkgpW69EscH8nBhjFFs4m2JbGFMYa9+WvT8upbu/Ph1kT9uBI10KkzE1rjwmLUFExBXmdhNRdO3awJv/MdauE9U11Pqa206YyF0uEQiQlLbJJ/LkZBGNMo9b1NYFXnpZLS4KvfueiEYPpxtBpFmyrlyj+RjGGVvZWKvyPX+V/p7EPVGSsXSb495h43685Ast9lFrJaXs70kcnJA8NCGZiP1j69r2efGPw4/XOUcdVY/2dvn21U31EctqiIbbR0DGiBHLsjsP8XbOlKI2EvreEXiG3aEytqKxbmxKekeUH7ohxljANP46alKKzf5CycZ9pkNLikHe5KfHTPFqGsJjV+nuCen2gaOuWvTZgVdL44x1xGh8lbGrCnr1PDIscaBs3sYSB8RVpSd9PpKSJyfbTzopMndedOkyY9lylpTI01JFU3NsrWu1qFAtLo6uXs00zX7mmYxznp3tOO+cwAsvBd94kxRFGzSQ2+2kKPHWJgpfiZI1AZGkI/hM4//Zu+/wKIo2AODvbLl+KZfeOylAQgiEEnoRkCZdBCwUFQSliGBBQAVEP5UidnoRUEpAegsQekkIPRBIhfRc6t1tm++PhfMEEZSSEOb3+PiQ3dl+t/fuzsw7F8uN1O0rEqC1g9sPGzyWOrn79PIOjM++ziAkYqmevVMv74BeXkEahkaA8i2mfod2lNweM6ZKFJY0ad/O1Usg7ZaeZuNCo96uEwkAV8tLJ6ccWdnsObMkAADCqJAzzzh/4qvouAe8OVcJQo+DW1Y176SiaQwYAfCS9N6ZIzPqN9GxLABQpGLhWSIHkZPDGpbx3PL0y3eMWIgxDtTar272nIZhnuR4CsQdanoE2dMroKdXQEJ+zhP+pcEYN3fx8FJrn+RGaxrao7mYuRM51HnQcQcxNickYIuFsrdnvH0UDaKYsFD+1CmgKEXjxlJ5uVRYyAQGymXtZ30mpmdgmlY2bwYACMBuwgRlXAsxIwNpNarnnrMcO45YFqlVD7y/GABJNw6zDcb+h4Ml7oYxLhf4tIqy82VFe/NyjhbmmUTB+vL+jjS/FlF8PzzGXaWJdnBp7uzuqlJzkshJEgIo5bkxpw7eNFXKWX4ogI/rNu7k7sM/RCouoiag0a3+hgxFsRStZ1g93HraRwAIkIKiHzCCFCgJEHJVqq1vqQUsKSmapag7ogfiGSFXZ3/TIE7LsEuvX7L2khExDtLZ/dy4rYZhgDS0rVY1OoKUK8gWxbY7WZz/bvLhqxWlTybzJyeJrwaEfR7Z7O60ds8OjAHUrpgrA74SFPfpnCTjTp4yTpwMNK15cQBlcAAANjiIDQ66e9UYgA0OZoODbSYAQkjZtAk0bSJPVLVsYS0PD3TdkZiznzKEA6nFfmgSxinGoo051/fmZZfynFm6ldbAGj4ihEL1Dncs5axUvRcWLd/my3kO3e6yPeDwzqsVpfK3iZPEwX6hbwbXJeFj7YVvBZAPuRb8xAdcImoYhFApz00Oixax9EvaRZai5J7X6+I66xiWvH2sdjU6gpQ/HDRCTZ3dEzv0npt6ZlXGlayqise3RR3DeKt1E8Oju3j4PXgD8FoJIQCgaP/nhatr2fBXHyQsE3JykEKhfq6jfuzbQFH3jPzu+tLf5y7wYPcIzJVJhUls1FgMgB406CT+QsL4QlnxtpuZu/Oyz5cWKyiKRRSS2xDbnE49y3qptKF2jrbLyhcR//VPBDAnNeVaRak81KGcF3BOwxalnIXc+p8KnCTeMFXet9gNU2Ulz6eWGy3yKPYISjjOJArplWUP+NhvEkVelM6WFikpGmMMCAmSVCZwmVUVWuY+v1M0Qh4q7bP8wF+LycMbzopshjEsS7/so9H90rithmYweftYA9ToCFJm/ZS8UydqoG+dFemXl6Rfyv+77jX/GcY4UGc/xL9OBzdvN5VWz7JAPp0AGICyCwRDnnB9Mx3Q859OCMaAkLpbV0VMDO3piWjqSacCxpJ4dS0d8AIARhiR8PE/KOEs08+fSMjPqeB5CiEtzdxxBRUU1czZo4Obd6zB1UmpsmNY6V7rAgAAJUV9ffnM91fOWUfK9tLovotpTcLHp0iVIIw6uf++jYg4ScysquhxcKt1CgZMA/Xmyf0PuCGMcSFneunILttPhoild04fvO+t3kutXRDTikSQtZWcbPyDiBhOEt+pE+VEEvfUGE9BBGnLVaUeH9ZgfFiDFGPRvvzsA/k3c0wVZlG0SKL8/wdZiYKi1DSjoRkNzbirNdGOLm1dvZo5u8tzn/FXj7bkfraUWxMpdaV08wDlFIUU9vd6rQgYA8MwPt4YAJ5k+IgxNuXzZ+YwIQMoO38ATBrc/1sIIM9c1SFhU5UgMBRl+0usoml7VuGn1Xf39O/k7uuu0lgkUW6Q9M/hI8Y4Pif9m8tn5LVhjLUMu6dNDxI7PkUwxg4K5dbW3at7R4hnnZz2YVZkM7MokPCx5njKIkirSAenSAend+pEVQp8Gc+V83yZwBl5LquyvJizVAh8hcCZRVH+oFEI6RhWx7BamvHX2XmqNA4KlaNCYVCo/jZP0JM/nBrqVrIWigkdLBmvimlrkSGSMkQgWg00c2c7p9snTh5D4LHvG8YgcVioEjN3AUhs9LtI5YQBHrTTD2FDQdHzr5wr53lrM0eMsZ5VvBYQ1srF00OtdVWqEQIJ40qBf5AvCIVQUknhe2cOW98+uijVy5p2oBAiTZeeIuRKETUHQsgiieQzWaM8rRGklZZhtQzrof4XI6iSsVb/hdt5N5FDCGMfLFxYyF+PpwzhSOeL1M6A/inL0mMkmrApH5emSsZrdOgQ2iUa4Mm++KxdEEClwNs+TSGEynnuq0vJAPCib0iVKCgpCt07te8dlBS1NjutQuDlXrQilnp4B4To7UVyjQiCIGqLpz6CtHrwXybyG/YfyH1TmLrDAQCLFjAVSGXpgIVq2BWMgVFRLg3Bqy1DsSD3/CRdZx6CRRI/iGiYVVVx5vaY1wCAEGIp6n+XkuempjR0dGnj6tXezbuO3v5BKpBMojg1otGB/Jx8swkhRCNqQ/a1V/xD3VSax3ogBEEQxBNTeyJI4vGyfUFFK0HnTeu8q3F3bD2hevPaCwM4KVTfxrTsnbi90GICa0/q20M3JRsLk0oKvklN7uDm80ZQ3UgHp/uuU0HTfX2Cv7t6Tv4z32xamZ46Kbwhj/+5/SRBEATxdCCd1wiCAISQp1q7vGmH3t5BHmrt34xxjBBg2H4z84WDW189todF9791DA8Md2AV8qoYhH6+diG9qoxE+gRBELUDiSAJggAAECQpUGc3M7LJrjY93g2LdmAVdxSQ67VZijpXWnzfBCsYYwdWOTEs2iSKcDup28TkIzR5W0wQxJNCIaSkaAWiFIhSUjS5/zxapBabIAgAm/bBLIVGh9Qf7F9n642M9dnXUkqLOPHPLpAYIFBnJ90v6ZU8ItlgvzrL0y9fKitBCFEInS8tOlGcH+3o8niPhCAIAkDE0sjgumNDo/6chHHp7eGyiIdH3kESBPEnhBAAMouCmmb6+gQtbdJ+WZP2flq9tV77VgT515RsNEJbb2T0Tty242YWBX/2xTFL4jt1Iq0JJk2i+N2Vc8wD1IATBEE8JISQRZLKeO7P/x4sHxnxgMg7SIIg7mQdopBGKMbR9c2gupNSjt4e1xIHau98B3mo8ObEM4d5SRp+Yq+EcUd3n04evnXtHN1UmsYGNx+N7lpFGYUQhdCmG+mHC3ObObuJJHU/QRDE04xEkARB3BNCSAKcWVVhfWzHGPw0emv0RyN0srhg+PEEQZIohLQMizE+XJibWHBTSdNOCpWXRhesc9DYjJH4U9p5H43WQ6198odD/K3k5OSZM2YIgsAwTHh4eJMmTZ7v2tW2wPfff79zxw6GYb7++mtXN7cpH3109epV2wKiKHbr1m3E669LktSta1eVSsWy7Oo1a+SLnp2VNXzECI1abS0fEhIy+4svfvrxx+3btwMABnBxdm7UqNHgIUM0mlspn06cODH144+VSuVzzz03ctQoeeLKlStX//qrQqFYuGiRg4MDAOTk5AwfNoyi6YkTJ7Zp02bt2rXLly1j2VupagVBGDlyZJfnn39c544gnmE1OoK8ceNGZWWl/G+WZfV6vcFgkMdJuZGTU2UyydPVarXBYJBvGRjjvLy88vJy61I6rdbg5ERRlCRJ2dnZFotFp9O5u7vLtzaj0Zifn0/TtK+vr7yGwsLCGzduGEtKvH183N3drbcz4gHl5eUZjUY7OzsPDw+z2ZyRkUFRFEJIpVJptVoHB4db77cwTk1NpSgK5MbOSqXBycn2bFdVVubm5mZmZbm5ubm5uhqc7p9BhngcEEChxWz9k6EoDXPrvoExvlRufOXYHsFmrAg57TiFkIhxvsWUZ64SMLatN2IoKq2ijESQNUdhQcHJEydEUWQY5lBi4rfz57ds2XLh4sVOt790q1asyMjI4Hl+//79A1588fLly2dTUiRJkts20DTN83xkZCQAJOzbd/rUKaVSSdP02ZSUyKgoADCZzcePHtXr9fKtAABEQQCAjIyMkydOMAyjUChOVlRsio9fsGBB4qFDer0eAH7+6afkpCSWZS1m89Bhw5RKJQBkZWaeOH5cqVTu27u3V+/eALBk8eLkpCRRFPPy8wEgOyvrxLFjSpWKpmkA4Hg+Pz+/Ok4qQdR+NTqCnPTee3t270YIYQCtRuPi4tKvX79xEyYAwLhx4w4lJgJCGo3GwcEhPCxs6vTpQUFBGOPPPv10w/r18hrUarWLi0vvvn0nTpzIcdyI4cPPnT0b1aBBfHw8q1AAwPRp035dtcrRYNi2fbu/v/+unTunT59+9coVAFCqVGGhoXPmzQsPD6/Gk/DU+WL27EWLFvXu3XvhokWXL11q2bKlXqejKMrBwcHJyan/gAGj3noLACRJahQTY6fXAwBN03q93tPTc9mKFR4eHgBw9uzZjz744ExystlioWnaz89v8gcf9OjRo5qP7ZmEAPItJuufKoo2KFTy9GxT5ejTBy2i8A9NixBC7F/nShhfqyht4eLx+PaZ+LcQQi6urp9//nlubu4Xs2efOnVq9ueff/HllwBw+fLlc+fOxcbGnj17dvmyZYOHDPn000/LKyqSk5K+mD1br9fPmz9foVTK39xVK1dqNBp7B4eSoqL9+/fXj4y05hatqqraEB9PIYQBdFqtPDYYQiimUaOPpky5fOnSZ59+WlJcvHvXrl69e1dWVu7ds8fd3b2srCwjIyMvL8/X11dej7zC7du39+jZk+O4TZs2KRQKs9mMbh+IhLGPj8/8b7+tMpmwJAUFBVXfeSWI2qxGN2mnaZphGB8fnw7t26tUqhs3bixYsOBsSgoA0BTFMIynh4eTwVBeVrZ///5ePXuazWbrUiEhIf3797ezt7958+bKFStycnIAgKFplmWPHD5cUFgob2LFihVKpZJhGECI5/k5c+ZkZWZ6eHh0fO45nVZ7+fJldze3ajwDTyOKoliGQXLnCYRYhqEoKqZRI4ZhMjIyPvzgg/iNG+WSDEVRFNWjR48G0dEWs/nChQuzZ80CAIvF8v6kScnJyQqlsnWbNl7e3jk5OX6+vtV4UM82VGgxW2NAFU3LiX6MnGXEiX1p5aVytQDG+J06kXOiW7Ry8dQxrIKiEMDdeSUBgKEoL43uCe4/8UBUSmVMTMzgIUOGvPwyxnj9unXy9OVLlyKE3hk7VqVWHzt2rNRojIyKiouLq1uvHgCwLNu0WbMWLVoEBQWVlZWlpaWp1er+/fqJknTq5ElJ+jOBvCAIcXFxTZo2bdq0qfxuUmZnZxcVFdXzhRe8vLwAIL+gAACOHzsmimJMTIy3r29hYeGF8+dtdxVR1PFjx8xm86WLF40lJRR15w+ZWq2ObtiwQYMGMTEx7h7kWYUgHosaHUECgCRJ3bp3X7ZiReKhQ66urqIoXrt+XZ6FMR40ePDR48d/XrhQp9OVlpZu27rVumBskybfzJ27fPlymqZFQTCZ/nyJolar58+bBwDLli2jbIb6FUUxOysLAIYOG7Z8xYrjJ08ePHTI0WD4219B4sGZTKaN8fEnTp3q27evVqOZPHlyRUWFPEsQhClTp27avLnDc88BxplZWQCwfdu2I0eOSJK05rff1qxde+z48Z27d0c1aEAuRLVACAosVdY/VTTjqFBWCvzbSYlXK8rkftZKmv6kfuz40AbdPP2WNGm3vXW3xU3aTwhr0MHdx6BQ2l44EUsRdo6tXTyr4UiI+0IIAHr16sXzfGFBgdFoxBivW7dOp9M1jo3t1q2bUqlcvHjxvZbOycnJzMxUq9Uvv/qqJEn79++3WCzWuRqNZvRbb709Zsw7b799JjnZOj335s3ff//9ww8+SEtLUygUdevWxRgnJCRIkhQcEjJi+HBRFFcsX24tL4piw4YN09PTEw8eTExMLCsri2rQwDZUpSgqLS2tY4cOPbp1e33ECGujJoIgHq2aHkECAM/zAJCbm2ssKWEYxtXF5Y5ZzZs3DwgIkCTp6tWr1nDw0qVL8+fNGzVyJM/zzs7Ofn5+1qX0dnaLFy0CgB+++87Ozs7a5lquLQWAWTNnvjhgQEZ6ure3N9TecbQ5Sfw9K0283Zjp8ZFPH8Mwb40ZQ9G0JEnyK2EAoGl62dKl77z9dvzGjbwgyBXce/bsoWm6ZcuW0dHRclVXaGgo1MYLUWwxnykpvG9uxeqFAIps2kEqKdpNpRmXfCix4CaNEMZYwnhoQPgA35AqgRcw5iTJWamOcXQZHhjxXUyr/e16zW/YsqO7j5KitQwz2C90c8vnSV7fxyqjsnxvXvZ//l4zLAsYUxRlMpmSk5M5jgsICFAqFM2bN2cYxvpu8m6HDh2qrKgIj4jw8vIyODmVl5cf2L//z9UyzNYtW7Zu2bJt61brHQAAzp07N37s2N/WrjWZTE2bNm3WrJkoikmnT2OMoxs27NChgyRJf/zxh7U8z3FdunTRarWLFi1K2LfPYDDExMTYRpAAIEnStbS09PT07OzsO2Y9vSQJb8q5Th6kiZqjRreDBACKotavX79v796zZ89qtdrGjRvXj4y8owxN00qVCiGUm5trDTJOnzp1/NgxlmUlSWrUuDFnsdAMAwAIoWbNmu3etWvON9+UV1T4+PiYzebyigoAYFl2yscfT5s6NSUlJfHgwXZt277Qq9fMzz83ODo+4aN+MhCg5emXv7qUNDwoYoh/qIKi77/Mw2FZVqfT8TxfUlIiT2EYZu6cOQghhULhYG+vVCgAIOn0aYRQWHg41Mao0Va5wLfZt7GfT/Co4LpRDs4182BFjEt4ix2jAACMMUtR7yQl7rqZpaRpjDGNqDeD604IbcBj6e79xwAsRT3v6feCd2Apb1HTDIMo0z+2myQeicFHdrdz8xoZUj/O2f3fLnsoMZFmGLlp46pVq0RRvHbtWkR4OE3TLMsajcb069f9AwLuXnD50qUKpTJh3z5vLy8KIbVaPW/evM5dushzTVVVy1etAowBIKpBA+tS/gEBcXFxSxYvNpvNs7/8kqbp3NzcEydOKJXKV19+mWFZOzs7s9m8auXKlwYNAgBASBCEXr17r1+3juO4+d9+e+zYMdtPlCRJQUFBu3bvLq+ooChKq60lfbYoCvVO3N7fN3h0SP0mTq40yatKVLeaHkEihMpKS8tKS+3s7JydnTfEx8NdjatEUbSYzRhjDw8P66wePXt+/PHHhw4fnvbxx7+uWtW4ceOeL7wgz2ratGniwYPz5s41mUyjRo1atnSpdVWNY2M3btp0KDFx/PjxRYWFm+LjQ0NDx40f/293+0q5cVVG6n1HfqteEoabpqqsqvKPUo79cPX8j41bN3R0faxbtJjNpaWler3eGpRzHBe/aZPezm7ZkiXLly9/e8yYxMOHQ0JCLl++nJmZ+TDb+ujsMeVdraNqmjKeV1L05pzr8TnXhweGT6sXy9a8fa7g+dsdrEECrKDos8ai+g63eunWszeMDK7HY0l+W/y3axAw5gWepWgBY156isNHjPHP1y7kmavuX7RalfEcS1F78rJ352X39PL/PKq5o0L5IAsigPTr1+fNncswTLt27TDGJ0+cwBj7BwT4+fkJgpCclFRcXHzu3Lm7I8icnJxz586pVKoOHTsKgiCJ4sGDB1POnDEajXIBXhA6depkLW+9VwcHB8+cNSslJSU5KenVV1/dtWvXxg0bJElSazSt27QRRTEjPT0jI2PxokW3IkgAAGjXvv0fmzfr9frnOnU6eOCA7Z5gjFmWpWja3t7+Ac/YhylHVfRjf4R+SAjAjmX35WUn5Of09ApYENOqhv/EPEkURSWfPj118vsv9O0zYuQojueqe4+eCTU9ghRFceSoUQNfeqllXFx6evrmTZu639UhN/HgwWvXrtE0HRYWZr0radRqdw+PPn36LPj225KSkozMTGtr69jYWEeDIS83V6vV9u7TZ9GiRfJ0jPGJEye0Wm2btm1PJyX179v36NGjmzdt+g8RpJ5h69k71fBfSglL+/NzAMBeoWzv5u3Iqv7zqjDGR48c8fL29vX1vX79OkLIwc7ujjJVVVXz588HAJVK5eXtbZ3u6ubm7+8/aMiQ1atXWyyW/Pz8tu3abdmy5Y/Nm69cuRISEmKxWE6eOBHXosW/2qVIeyem5kVjdyi2mOWs3XXtHCMdnKia93lBAKUCZ90vGlEpxkL5Twqhl/zrfFIvVs4N/s+fdttcP49xdx+/YJ29s1J9/3LVqshiohFCCAXr7GMNbooH+yLcuHGjTatW6enpejs7Ly+v9yZPLi8vP3rkiCiKvyxcKLck6denz9GjR/fu2dOte/c7FpezMDZq1GjpsmXylPr16pUajTu3b2/YqNF9tz512rQB/fpdv3r18KFDC3/5RZKkKVOmyCFjQkLCkEGDiouLrcEoALRt29bd3T08PNxgMNyxKoamz507FxEWhjEWBOHFgQM//eyzf956lINzDb9dIAAEIGKMAero7Bs6ujzdXyQbCKEbOTkxoWE6vV6ewrLs9gP7fWzant1ryV1bt65evmLW1195eHmVl5efPHYsKrpBjb6QtUtNjyABgOf5wMDA199446cffpgyZUqLli3lRLIIofiNG/fs2ZN+/brJZHJ1de3QsaN1qZSUlBmffXbgwIH069cxxuFhYaIoyrOUKtXw4cPfnzx50qRJCoUCbv/+FRQUDH31VUmSAgMDHRwcTp8+LYpii38ZuMjc1do+PjU9hYQgSb9mXOns4Tu9XqyvVg8A//Aa6Z/Fx8dPnDDBYDA4u7icP3dOkqQ2bdta5ypVqqGvvXbu7Nn8/Hye56d/8olWq5UvB03T386fbzKZ9ickSJKkUqmcnJz69O37808/ZWRk9O7Vq3Xr1qmpqamXL0+cOPGtMWMecH8wxv19g//DgTxhWVUVMy+c+jSyST+fILkVwX++BI8JAlRuE0ECgPxDiwAG+4dODm8oYlzT9vnxQQi1c/O+f7nqlllZ/vmFpI8iGg0NDFPRDDzY50oUxaLi4qbNmrVp02bkqFH2Dg4b1q+vrKxkWVYOHwGgXfv2hw4dio+P/3rOHNom5MIYb9u2jabp526/ZRRFsVFMzN69ew8cPNgwJgb+rvWeNdEPADSMjq5bt25KSsqMGTNyc3NFUezUubO8Zh8fHwcHh6KioqtXr9IUBRjTNO3g4LBl2za5FTvNMBhjOQEkRVEYgGEYOZewIAjWO/+9PC23CwCgj++dWrfxa//msj4VEEJKlcrRySk8IkKSJJph5PSf94IxliSJZdnTJ0/t3bXLZDbLaZ5ompZfFUmiSP31pbIkSXf32b+7GPGvoJrcLPe1V17ZtWvXiNdfnzptWl5ubvt27crLyydNnjxy1KiXBg48fOiQXFvh4OAQGBT0zZw53t7ekiSNGztWrgQRBEGj0RgMhsaxsXPmzpUkqU+vXhcuXNiwcWNoWNjHU6ZMfO89SZK6d+tmMZu3bNuGMZ40cWJycrLcU1ipVIaFhf1h07+7lhEkKamksLGTKzzcnQhjXJCfP3To0EsXLnA8T9N0dHT0+o0bASAlJaVt69Z6vV4QBHt7e0dHxxcHDhw9Zoz8/Xd2dLR3cOA4DiHk4ODg7Ow8/ZNP5NBzf0LC9OnTr1+7xnEcwzBubm7vjBs3yKYOq3ao4HmLJDopVf/5/FcJfMeEzVtbdX1MPyQUoNQKY7u98WqatqaCV1D00KDwiaENeOlv2j5Wlwqe75AQf+H5l6p7R6pfvrmqQhACdXb/+XOF//pe2fozYTvlXmv+51nwd++h77XIg2zFtsx/WM9TRML4UllJhL3h4Q9nVUbqhdLiyeENa0InI4TQzRs3mtSL7Nz1+V9//VWO90+lJI99Y2RUdPTMr7/a/scfsz/5dMSoUQMGD+4QF9fthZ7ZWdlbN8b3Gfji7u3bLRaLwWAYMmxoVMOGA3r07PT88/l5eSePHXtj9OhJH09hGObk0WOfTZ166vjxpnFx02fNCq9Xd8qkSUcTD705ZvSEMW9PnjJlzITxgiBU81kaQxiTAAAgAElEQVS4C4PQgMM7X/AKGBYUUd37ck81OoLMyMgoKytzdnb28PCQJOn69eumqiqdXu/n55eenl5ZUQEADMvqdTp3Dw+apuVjycnONpaWAgBgzLCsnKoaISRJ0vVr10xmc2BgoHXsE57n09LSsCQFBQcrFAqO4zIyMm7cuGEymfz8/IKCghQKRe24+zxuRqMxJzs7KyvL3cOjTp068hk2mUypqanyuwGNWm1vb+/k7CyfT4xxSkqKPAtjrNFoXF1d9Xq99WwbjcbMjIys7Gw3NzdfX19XV1dyIe72uCNIuetMh4RN1yvK5E1oaObr6Li2rl417cZBIsg7kO9LbfK3EfN/VgMjyGaRUX4BAc/36M5ZuPoNokJCQwe+0KtRk9gfly7d+Ntvk8aNHztx4uuj36rnH6DWaNzc3fsPGuTm7rZgztyM69dfGT6sbceOAPBy334qjWbAoJfWrlyl1mh2HDxQUlQ0sFdvhmHemfjunNlfuLi5bdi+7a3hw/fv2etoMHTs0nnAoEFhERE1MBB6KiLIGl2LbZuCh6Io26EFAv6uJ6D8vfL28fH28bl7LkVRQcF3VlWwLBsWFmb9U6FQBAcHh4SEyH/e64mZuJuDg4ODg4OcZNj6bVSr1VE2qYNl1qqru2eBzdmWVyhnHiYXoroghASMv4hqNuJEglkU27p6TQqP9tHoatTbR+JvkQtUm9SalsT3hJCxpGTDb7/zPCcIQnjdugihW7ngEKIQuv1PBABb9u7R6fQUwNb4zVcuX35l+HB//4CEvXs5jhv82quzZ846feLkqePHy0tLr165UlxUVCcsrLiwyMnVJfnUKQBAgDDGYyZMePmVVyyiIIlirT2rj1mNjiCrhe0niXyq/ptHft7IhahGGOMYR9eFsW0lDPXtDRRCJHwkCOLRwpJUt379/82fZ7FYdHpd7s1cABAEQR7v6s9iAMF16ihVKgtnYVkWYwkBmM1mCUtgc09yNBgAQBRFgRcoiqqqqCguLmoWF9coNlZemySKdevXswg8eVX/MEgESRDEP0EI8ViqZ38rfQ8mAT1BEI8aBlCp1f7+ARY5EQ9CCKGrqalb4uN/nL+Atunvgm/nDsMY2zs6Miy7ad26l4cNs70vYcAAgDH29ffTaLW8IHTu1q2yokKn+3M81VqTar4akW7vBEEQBEFUM9sQ0MvDq2WbNkWFhZPHjY9sEMUqFHeXF0Xxhb597ezsFnwz99OPpliHlwMA6na69br160+YPFkQhL7Pdx068KUdtbdrbLWo0T1pCIL4Z4+7J81ThPSkIYgHVKN60sgvFCmaBozF200SMcYURRXk5zs6OjIsK79xlCcCgGjTclEUhKysLE8vL4VCQdE0liRJkiiaRghJomitpM7JynJxc1MqlRhjRFEIQJL+aRyEakd60hAEQRAEQdzTrfGuRBH+2mFIkiQXV1c59Zt1ulz1bJu/iWaYwKAgORy0rgRLErZZOQB4+/nJweXfziX+G1KLTRAEQRBEzSK/d4R/jPNuRZ9/DSv/Fn6AMsS/Rd5BEgRxC2fhTh4/di0tTaFQ+vn7NWzcmGVZhNDmDRtomgEAlmXs7B1CQuvIXR0xxldTU6+mpkqS1DQuzsnZGQAEQTh66FB5Wbm8Tp1e5+Pr5xcYgAB4nj+SmFhVWdUgpqG7h0dJScnxw4dFUWzUpImrm1s1HjhBEATxb5EIkiAIAABRFEcNHXrqxAl5lCCWZUPDwxf9ukqv148aOszewQEwRhTFsiyiqI3bt/n6+4ui+NOCBds2/8Hz/LhJ740ZP57nec5i+WrW51dTUwFuZfVXqlSvDh8+8p23TVVVsz/9NDsz68t5c93c3b+fO+/XZct8/PxW/P5bdR89QRAE8e+QWmyCIAAATh47duLYMYRQx86du/fq5eTi7B8YaGdvDwCAkCiKL/Tr26pdW4OTk9lkGtirN0Mzoigm7j+g1WoVCsVvq35lwNqGCRBCzVq06Pvii77+/lWVlSuXLs29eRMAECCEEM0wxw4f/nnBdxRFTfroI2cXl2o8cIIgaqDKysqkkyePJCbm5+U97m1hjC9fvLRjyxZ5OHXiAZF3kARBAACUFJfwPK/V6UaNfSeiXr3cmzcNBoPA83ImNp7nh735Zmho2P6EfS/3H8BZLIWFBefOns3JyhoybNjlC+dPnzh55mxKaHi4vDaEUNeePfsM6H/pwoVBvfuUGo3lZWVqtVqem3rx0uKffmIY+uMZn3Xo3Inn+Wo7bIIgHgJCKDsrKzFhf+fu3ezlB85HgWGY14e8nJKcXFVVNfeH73v07v3I0zeaTKZ9u3Zptbo2HdrzHPf59OnHjx6d88P3HTt3frQbqsXIO0iCIG6l3tXpdOVlZe2bNX9//ISS4mL46/hMFeUVRSXFZ5KSKYqiaNrJ2XnVkiU0Tbd/rmNss2YKpXLDb79ZE/9ijEuNxpzs7COJh8xms8HJoNfr5VmiKM758kuz2dwgJqZnnz4cx1XDARME8SgwDLt31+7RI0bk5ebaTr9zLJm/yxt4RxnbYmlXr54/e9bFxWX5b2tbt2snd7W+V/LBOzZk/fPu8rZT8vPyPp78/oI5c2iGUSiVb40b9/rot+Jatfq3+/wsI+8gCYIAhFBEvXrTZs386vPZAPD76tVb4uM7d+s286v/yQUUCsWHEyfyPJ+Zni5J0ivDh1VVVR7Yl2Bnbx/TOFYQBJZlExP2V9nUAf0wb95PC77Nz8tnWbZj5y7Orq6VFRXytuSsbOfPnk3Ys6dz166CIFTLURME8TAwxkmnTm5ev97g5LRqyZKQOqEt27Vds2Jlt549Ew/sz0zP+GD6tG2bNh8/epTjuP4vDWzesqXJZJr3v68aN21SXla2b/fugMCgUWPfYRgmLTV16cJFpUZjyzZtevTpvfG33+VI7fDBxPbt2mfdyFm+aFFmekZUTMMBgwap1eozSUl7tu/o1K3rzq3bTKaqdh07Hjt8uHX79vt27bp548bQN94QBGHF4sVOzs4j33lHp9NlZWRs3rDx8qWLLq5uLw8d6uPn+9O3C0RBKCosnDVt2uhx42/m5ChYtjC/wNffLzMjY82KFTeys+Nate478EWapo8eOrRnx84Ro0b98v33+Xl5L73ySkxs4+o+/dWPZBQniKfYI8woLifXRQitXLJ0zhdfmM1miqLWbIqPqF/fy97B3t5erVbTNM0qFEPfeOP1UaO2/fHH2JEjMcZqjUYSRY7j9HZ2qzasd3V1Hdy379XUK2q1GmNcWVnpHxBwMDGxjOfKSktf6tU7Iz2doukmzZodTEigKOrstTQ5S/BDIhnFCeIBPaqM4hjj5YsW/W/mLIyxUqmsExY2ZsL4If36e3p5GUtK4lq18vL2Wr54iaPBYDGb8/PzT5w762AwRAYGMSyrUqkoiqqsrPzmu+969ukT4esLFKXVagsLClau+33W9E+upqZSFMXzfEra1bZNmpYUFas06ory8s7dun69YMGqpUtnTJ3m6upWWlYa06hR81atZn/6KUJIp9NxHGcxm7VaLSBUXl4+5/vv23d67sWeL2RmZChYtqKiIjK6wZLVqyP8/DUaDUKI47iDp05OmTTp0P4D3y9eFN2oUauYRhzHKVWqstLSzl27/rRo0fx5c2d/+hkCsLO3t1gs9aIil6xerVQqH831+DtPRUZxUotNEAQAgKmqav7XX2dlZr42fPjWhH0sy2KMr6WlyXM5i2XeTz9u25+w/8TxEaNGmS3m/fv2AoCDwdHTy9PHz49VKEqKi88mJyOKAgCE0MczPlu1Yb0kSlcuX16zYb01TEQIjRk37vvFi3z9/QHgsykfYzJALUE8nQa9+mr9qChTVdX8n3/69pefaZqmKIqzWL5fsnjarJmTPv74y3nzps+aWS8qSqfTrV21imEYiqJcXF2W/bZ2+Mg3FQrF4YMHCwvyjUajk5PTp1/MXrd1S9O4uAnvv09RVET9ejsOHli/9re8mzfjWrfauH27f0DAutVr0q9do2mGpmm1Rr1w5YpZc76RH4C7vdBzzaZ4Xz8/VqGY8MEHn8z+XBCE3Tu2O+rtpn8+63/z5737wQcIoZPHjvMcN+f77yiKCg0P33/iuL2DAwJE0bRarf70oylFhYXDR45c/ttaTy+vI4mJl66kIoREQej94oA1m+KVSmVOVrZco/KMIxEkQRCAEFq9YsUPc+d1ad2m/ws9e3ToyFksFEUFBgXLBTDGLq6ujgYDQojnOZ7nL549hxB6e8KEzbt379i7N7phQwDYuW27dXRajuNCQkN79u2jUCp//nZBRfmtGy5FUb7+fhRFDXvjdZqm/4iPz/1rCyqCIJ4KCCGdSq1SqSSMnV1dnV1cACGMcduOHVu0bu3u6ZmXmztl4nvvjnk7JSmJpunCwiK5dbWTs0tkgwYhYWE0TZcajS4urqMnjM/LzX3njTc/fPddnucNBgMAKBRKTy+v1EuXRFHs0KlTaGBg6/btEUIpyck0QwNA07i4RrGx7u7ucvPE8Lp1AwIDPb29McaNYmPjWrcWBaGwoAABHExIeGvY8E8++gghJPA8LwjOLq4IIVah8PTyoihKro1lWPbgvn2IovoMGFA/KsrR0dFkMhUXFgGAIAjN4uKiI+rSNF1VVUXa3gCJIAmCkEXUq9+0RZxSqTx1/ER5ebm3n+/7U6dGRkYCxgCA/zqcQ3Fh4dmUFABo1KSJJEkclnoPGCAIwq6tW01VVQhRAIAAYYyHvPaaWq2+dPHinp07aZq+1TsHIYRQ7wEDAoOCSo3GqZMmK1lF9Rw2QRAPAQOoNGqE0LXUKxaLxWYyqGimf7fuvMD/sGTx+1M/FgThzqY2t9vQiZLYum2737f80bZDh+TTSatXrGAVt24IkiSFhNahGWb3jh1XMzIS9uxGCNWPihIF8V77Y/OPW3/dKMhftnCRvYPDtoR9nl5ecrduhqERQqaqqqLCQuviAs+3atcWADatX3/h3HmjsUSr0xqcnR7qHNVepCcNQRAgSVLTuOaNYhtbLJbr1645Ohpc3FwVCoWF5wDgUmYGAFKpVdZm024eHqcvXQSE5AQ9oiB07tb1TIdUAFCp1SvXr5MkSaFQAEBE/XoHTp3EGDMMw7Ls739sljBWKhTylDWbN4miCADyhgiCeLqIgFu2brNz67b3J0wIWxgxZsIE6ywKwN7BoaysbOzIUQLPM8zfxxsMw2zeuHHCW6P1dnaiKDI0XS8yUr4tAIAkSa3btfP09jp88GDX9u0tZvPzPXoGBAcfPXTowXeSZRitVltUWNircxe5hTcAuHl4qDWatCtXmkZGHThxgkIIAMxm8+SPP/5jY/wP8+f/8sMPFrO5SbNmdULq7N6+/b+fo9qLnjZtWnXvA0EQ/xEvScvTUwf51XnInjTy4hRNKxQKdw8PO3s7iqKs62QVClbBor8m92EVCnnMQ3kKRVGsQsEqFAghOVikbjeIZFmWZVmaphFCDMvKs+QF5ZLWiu+HwUnSsvTLb4XUf/hVEUTtdra0qMBiauHi8fAdaUVRbNAwRpREjVYTFBIcGd2gqrIqJrZx/chIXpKee75rdmamr5/fa6+PcHB0jIxuEFG3bmZ6Rv2oqGYtWphMVSaTqV5UVK9+fe3s7Gia9g8MGD1+XNsOHcymquLi4vqRkTGxjR0dHTt07lxUUOju6dG8ZYupM2cwDFNRVi5JUnSjRhH16gKA0WhEAE3j4vwDAgry8w1Ozq3bt9NqtTlZ2dGNYtq27+Dg6CgKQuOmTV4eNgxj3LlrV3cPDw9PT85i8fX3axATg7FkMBiat2zp6ePTsXOXivIyV3e3bj1fmPX1VxiwsaQEY9yqbRtvX99r168HhYS0btdOqVI9/LW4Fwqh37PSwuwcGxpq7oALpC82QTzFHmFf7Kcd6YtNEA/oUfXFBpscDqIoyg+NFEVhjOWaYut9SS4jt1aUGx1KoogQkgvLK5EXoShKkiTbWbabYBhGzg0pP4VaN2T901rYOl0uQ1GUKIpywlp5rrwS635al7LdHE3T1n22zpVX8sgznN/hqeiLTWqxCaKWM5aUXE+7ZtNACJRKVWh4WPLp0wDg4+fn5u4OAKdPnpQkycvb28nZ+UxSEm2TYUelUtcJD5Pvmxaz+fq1azdzbiiUCk8vb/+AAAz41PETCKH6DaLkRI/X09KMRqOHp6erm1vSqVP0Hcl6EAoMCrJ3cOAsluvXruXevIkxuLu7+wcGqNRqAMAYX7l8We7qiAFUKpXBycnN3Z1EyQRR08jfSjnGkqfcETvKbBNx2xawxmFy3Gb7D+ss201Yl7Wu0DrXNpH43du6FbnKrbpvL2jdlu1StpuzLWn99+OOHZ8iJIIkiFru6KHDE8eMkWxqG3z9/PYfPjx18vuply7Vb9Bg+65dJ5OT+j7fTZLEXYcPURTVv1t3tUZjLR8YFLR6U7xOp8vLzRs3auTF8xeqKispinJ0dBwydOgrI4b36tyZZdmDp0/5+PqKovi/WbP2794zatzYl4cN6925i97OznZ/aJqe9+OPLdu2mTRuXGLC/vLycgDQaLX1IyPn/fyTg4ODIAiffPhR0qlTAAAYK1UqeweH/i8NfGvcOHLvJgiCqCFIBEkQtR6maJqh6fCICJphAMDN3V0AmD1nTt9uXS9fvHjqTPLXn8+mGbpf/4FREXXTMjPk141169dTqdQYwNPLk6YoURTHjBhxLiVFr9eHxcRQDHM1NdXD0xNRFE3T1O3xDAGAQoiiaYQQRdONmzbR6fRpV66UlpaqVKqI+vUQIBc31x+/XbBp3Xo7e/voRjEMzVy+ePH4kSPvvf3OwpUreJ5HFKIoKiQ01M/fLyX5TEF+wdJfFnbp3t0/MLDaziJBEARhg0SQBPFM8PHxWbZ2jVajBQAMYOEs4fXqtuvY8cC+hOkffnjp/AVnF5fR48dVCbxcvqK8fM2meLVChQEwAMdzSadOXTx/XqVWz/jfl127dRcBpyQn142MrKqq+tstYoy1Gs2WXbtpgK+++mreV195+fhsWL/BAlBUXPzdNz0Yln39rbfGvfsuAli7du3748cnnTx58fyFgKBbYWL3Xi+MfGv0tfTrA7r3KCsry8/LIxEkQTyzysvL1Wr1vfp0E08euRIE8UzIz8//5KMprEJhMZvHvz/ZyckJYzz4taEHE/afO5MCAI1iY719fKzl1RrNtMkfsgpWwrhz165N45pnXE/ned7D07NFmzZmngOA8Lp1JVG0bWF5NzPP0TQtZ9/FkmQBsPBc+rVriKIQQq8MH2biLADQpHkzN3e3osKivJs3rRGk2WQ2lpVeTU2tqKhQq9WOBsPjOz8EQTw8Lavo3bNnRnr64eSkR5tzu6iwsOdznSiEDpw6eccsjDHP8xhjhUJh2/5SEARREBRK5X2bUPMcJ2H8aEcpxBhbLBaGYWpxyFtrD4wgCFuVFRW/rVqFASrKy98cM9rJyQkAYmIbA4A8SEPXnj2sLccBQKFQrFm1EgCwJAWHBMe1apmZni6n49FoNNZsbQBw971ZXsU/3LWLigoBwM7eXk4wLucGUiiUgiCUlhqtxVYsWfL7mtXpadeUKlXLtm38/P0fwYkgCOKxoQDSr6enXbnyyPu90QwTUa+e3k7Pc5w137hMFMUpE98rLi6e+8P3Gq1Wnsiyih/mf3tg796vv1vg7uHxD2tWsYop703KvXlz0a+rxEcU9SKEiguLhvTrP2zkG31ffLG2NuAmESRBPBPcPTxmfPmlWqMRRdHF3V2OFNetWSPPZVh27apfO3TubH1crqioWBMfr1arMcYeXp6iKHp4eWKMBZ43maoUij8f1vU6O4wBIVRRXg4IYUniOR4hpLp3sjRXNzcAKC0xCjyPNBoA4Mxms9nMsqyj05/DP5SXlZWXlak1Gk8vr18WLa4iWccJ4gmy9lzGGFMIgU1GHtt0OX/2hpYkiQWKpmia+stKMJafFW3z6cgrF0WRpqg71mxNoAMAoiBQDIMA7O3tf1y6xHbHREGQG3abTaaLFy6UGo2iKFq7V9MAxw8fvnzpEmfhbFdrXbN1t2mAxIQEQEg+COsmbHMGWVdrPYRbh4+Q7Xhd1sOhKCorK/PCubOFBQW2i9cyZFRDgngmqNTq+tENGsQ0jIltrFQoAMBkMq1csgRj/OLgwQqF4tTx42fPnPnzVigIUQ2jo2IaNohp6OLqKklScJ06CoWioKBg785dcrE9O3YaS0oAICgkiGGYhD17sCTl5+dnZmQghGzrxO8QEBSkUqkAwTdffCl3uNmxdVtebq5Krfby9rYWe/eD99dv2yoIQtqVK+s3bqBtOusQBPG4iaL4/vjxrWIaBbq6tWnSdPuWLUpW8cv33zepW/+b2V80qVc/3Nfv+3nzJEkqKS4eP+qtMF+/xg2jKysq7qhK7vFcpx4dnxv4Qq8wb5+OcS2OHjrEsmxmevrwQYNDPDwb1a335WczGYZJOnmyaf3I2Z980qV1m04tW11NTe3zfNcQT69G4eFffjajvLy8R4eOz7dtR1HU3p272jdrHuzh2bpx7O7tO0YNG5Z+7ZqxpKRNbJM9O3bKj8FfffHF8aNHLWbzgB493n79DZPJNGXSpIahYRG+fq8NHJh29aocR1osloEDBuTl5eXl5sY1aBi/bt3+vXv7dese4ecf6u3z1rDhnMVSVVnVpF7998ePH9y3bx1Prxs5OW+++lodL+9wH99WjWPbN2uekpSUduXqKwMGBLt7xNaPnPvl/y5fvPj262/o9PqfF3zXqWWrvNzcaruKjxOJIAnimZCTlfVKv/79u/fo373HW8OGS5K44bffr1y6DACfzJwZXKeOxWKZOnmymr71DlKn1w/u07d/t+79u/cY9tKg8vLy+lFRAUGBPMdNGjvu5X79e3R8bsLo0bM//YzjuR69+9A0/f28+f26de/eoeON7GyGYZo0b36vndHr9RM/+lCSpHWrV3du3bp35y5fzZoFGLds0yYkpI61GGfhwuvWfXHIEJVa/c3sL4wlxnutkCCIRw4hRNPMW+PGfTB9Wllp6c8Lvqs0VZnN5vKK8lVLlw565RWlUrlnx05RFNevWbtj69aw8PA3R48pLyuDv75vKystvXbliqub26uvj8jJzn7vnbEY4yF9+x0/enT+zz/17N37p+8W7N21W8LYaDQuW7Q4IDhoymef/TB/ftLJkwtXrfxw+vQ2HdpzFktpaWmp0ahi2AmjRxuNxk17dg9+7dX6DaK69uyp1WrVGs3Y9ybWCQ8TJQkAWrVtq9PpFErlsDff7D/opekffLDxt99jYmOHvfnm4YOJg3v3YWgGABiGeenllwVBsHdwmPD+5Kjo6KDgYJVa/eOSxS1atdq1ffuG33+nGdpoNP6xMb7UWPrVgm9/W/Xrji1bvl7w7Zfz593MudG6fbuI+vUH9+mTdPLUj0uXtOvY8ft589KuXo1r1YrnuOatWr41btwdGc1qDRJBEsQzgef58+fOpSQnpyQnX7pwoTC/4LMpUyRJmjZrJgB8PudrjPH5lHPLV66Qn+Bpmj5/9qxc/tzZs6IgsArFdwsXRsfEMAxz8tixq6mpBien8Lp1RVEc+PKQ2GbNsCRdOHeO5zhnF5cPP5nm6uL655BXf63A4Thu4JAhQ19/3c7OLjM94+L582qNpkuPHjO/+h/HcwCA5AUQEgTh5aGv6fT6nKys7X/8UStrggiiZqIoatCrr9zIztm/Z6+EcWVFhdlkAoCqqqr/fTt/6pQpAJCdmYkl6dSJExzHjZs86fWhQz29vfFdzf6cXV2mz541duJETy8vzmI5cuhQcXGxvYPDoQMHs7OyWJZNSU6mKAowjoqOXrFiZZdOneJatdLp9R+9+96N7Bwvb2/rvUQA3Llbt6qKig/HT9DqdGqNpmPnLmqNRqVS9R80yD8gQBJFjHGjxo1VajXDMN16vdA0Li7tylWlUvnOxHc/+uSTiHr1SkpKrqReBgCGYZ7v2pXneI1GM/DFgUF16ri4ub04eNAf8fE52dkMw1xNvYIQAgwajWbz7l0vvzRIpVYBgCiIJpNJksSIevUL8vONRqPByenAvoS8mzcZmi7Iz2/cpAkvCJHR0X3799febp1Zy5B2kARRy7Vq125rwj7bKTRN6+3tdyYeBIw9vLzMPBcQGLT36BGMsVKpMhgM+44fs43VKIqys7eXRNHTx2fhqpWFBQVZGZl2dnoXNzcnZ2cAcPfw+H7Rwvz8/JysLHsHB08vL0eDwcxZrC2HXh42rFffvgzLWm63ZRRFcdLHU4aPGpmXm8dznLevj8HJSR5DjKbprxZ8y3Gcnb09xtg/MHDTzp2CIGi0GkxGYSWIJwIhlJWR2ef5rs7OztNmzZwyabJkHZcFwGKxyMXkSYhCCEDgebjHcH9YwmaTmad5i8WMEFKr1fLNITAoqE5YaNO45tGNGomCgAECgoKqREGSpF79+oWEhv4wb/6333yzdtWqHQf2y6symao+/WJ2l+7d5sz+4pMPP0o6efLjz2bIe2IxW5S3e16LWJKbZnIcd2t8QgCz2Wy2mEVRQAhZW8UIWAIEoihygJUU/erQYQf27n1n4rv1IiNnTpvOcZzcRNLb15eiqHKeG/7mm4t//OnTKVMwxh27dOk38MXszCz5dhkUEhxcJ6R1+3aNYmPPnUlBAJzZwouPsk96jUIiSIKo5TQajcZmgBkr23aKGGNPL6+/nfUXGDMM4+Hh4eXtbR2RVqZSq339/PwDAuTpd7Qct7Ozs7Opx5FnURTl7OIi96qRR8K1znJxdbUWpijKzcP93x41QRAPg6bp69fSTFVVUTENC/LyLWazTqezmW99lkOSJEU2aJCwe893c+elX0+/mZODqDurN8vKytatXp2VmVmQl183MrJRTCMvH58rly9LWPIPCDybcqZJ02aJt2JELG995ZIlqfIPGrcAACAASURBVJcuj31v4sVz527evCnczv+AEDX9gw8Cg4M//WJ213btb+TcUGvUKpUqLy/vt19Xde7WzdvHB2PMIsrewSH35s3439c1a9EivG7ds2fOfDdnbmh4+IWz5/wDAwODguUaDzWi7OzsMtLT161e3bR5XEpSkt7ePjQiIv73degvtR4YABiW/WPjxvz8/Hk//uDu6eni6lqQnx8QFOjh5ZV+/TpClI+v78ULFxo0iC4qLASAHVu31IuKiolt/Lc34acdqcUmiKcYQsgiifcv90i3CAjdHSPK7jX9H9b2qDoqilhSUuSRmCAeDUEQYps2jW3WbN+u3auWL9Pr9feqAZBE6cUhQxo3bXrpwoVlCxe2f67j3TlxKJpe+svCTes3xMTG/rJiOS+JK9f93rFL57lf/u+NV19NTNhfXFJ8R+vJqsrKDb/91r1Dx8rKym8WLJBfcAIAz/EIwReffda3W/egkJCpMz5TKpWDXn2lsqJixtRp69esUTAsAFhEYeZXXzk6On43d+60Dz748JPprwwfdvzIkeWLFsW1avXrxg28wMs3HxOWZn3ztaOj4+Sx49atWfPxjBkWs3n08BESlu4O+0RRbNi4McZ48rjxrw95uXeX518bOPDShQurNmxo067dlzNmjBo69NihQ8XGkpZt2nTq2vXyxUtjRoxIv3bt0VyVGgaRWiGCeHrxkvT8/j9mN2hWR2dfOxOOPbCbpsoxpw7ubNujuneEIGq6VRmpF0qLJ4c3vO9Ng2XZ/Lw8Tzd3+W1TJc8pWIUSwIQlnuPslCoMUMFzAKBhFcXlZXZ6OwpAAVB6uxELz/MdmsdhjDfv3oUxdnd1kwcRQAipWUVZZYUoSY56OxPP0TStoWgeQC7AsiyNqNzcm27uHqIkSpKkZVgAqOA5BasQJLGkuNjd2cUs8JIksQqFJIrFRUWurm4WziI/mlIUxTBsfl6us4uLKIosy3IcZzJVOds7VvGc9cEVY8wwjCAIpcYSg7OLXEfPsKyKYRkAHsDCc3pWIQJUchaNQvnG8OHHDh/+/Juv1Wr19A8/zMrInPfTj23at9ewCmN5OVDITqsz8xzGWKVQFhUXUTSt1+ttc+g+CAahAYd3vuAVMCwo4t9e3yeGPLITxFOMQaiuveFA/o0IO0euliatfRA0QudLiyPsyaA1BPHIyGO9GJycTDbxFsdzcltmRFHlNilaKzmLSqWS64XNdw0ogDGuqqry9PIy85x1VhVnYRUKFsDEcwAgimK5KMLtZQVBEAAMzs7c7a1X3N6cvBUHBwfrjvEchxByNBi422XkjJU8zxmcnOQ8kRzHIYS0Wl3l7RDWWlIQBISQo8FJ7oXDMAxgbLodBAOAfKQIITlDZnl52aIff1Iqlfm5eXXCwho2aiSfAYVKCQDm24UtPKe3s8MYy+t/tFenJiARJEE8xRBCEfaOG7KvjQ6pz8GzG0GyiNqdl93Uya26d4Qgag/rKzr4xyGmbAvfjabp+T//hDE2ODndUef5IEHVP2zddtbf7urfTvzbNjP/sPgdTJzl219+XrZw0fEjR3R6/dSZM17o10+uYb/vTtY+JIIkiKdbRzefLy8l7y+40djgev/StRECOFtatC8/Z0xIZG0d+4EgnlIURUXHxADAv63GrbEEQXhl+LChw4djABEkgeef2dsO6UlDEE8xjHGAzm5SWPQn505YassN+t/iJGlsUmJPr4Bwe8dn8z5OEDWZKIq1Jny0Nu4085yF5wRegNr7ivG+SARJEE8xucH48KAIL41uafqlZ/A2RiG0LitNxyg+j2pG+gUSBEE8MSSCJIinm/z4+2VU8x+unp+QdIirLc/6D2jWhVNfXU6e17BFde8IQRDEIyNizOOa3rSdRJAEURt4abTHOvYt4blXju9JMRbyTzZJ5JPHS1KKsWjY8b0Xy41bWnUL0TvAM1yXRBAPDmPc0sXjVEkBhUgAUHOVC3yuqaqLp19178g/IT1pCKI2wBg7KVVrmj+3P//GGyf2Kygq0sGpocG1kcElyt5ZgtpQvUsBulBWfLqk4GRJwcmifIskzqjftIeXP0NRz2xLdoL4txBCSooptJhpBEJtuDHUTqU8x1KUp7pGD6hNIkiCqA1upbQAaOvmldSp//6CGztzs7bcSF9y/WIZz9ea2ErLsA6sooWLx6f1m7R19dKxbO1OlkEQj4OWYTDADVOVs1JFYsiaqZSzBGjt7l+uWpExaQiiVrnjbVw5z1UKQu0IrzCAmqbtFco/p5BXjwTxn4w+eaCIM/8S21YkMUDNo6To4cf3tnDxfCO4bnXvyz8h7yAJola5I6LSswo9q6iunXncSPhIEP/NZ5FNGu5Yuycvu7WLJ/ke1SgUoC03M5KMhd80bFHDH5JJQ1qCIAiCeIZgjB0UyqVN2n9y/oRJFKp7d4i/yKoqf//MkWVNOzgr1TU5fAQSQRIEQRDEM0XOIxvn4hHj6Drm9EGzKJD2bDUBxrjYYh6ffPgV/7CGji41/6KQdpAEQRAE8SyqEoTvrp5bn532sn/oS751oEa/8Kr9Fl27+HPahdeDIv7f3n2HR1Xl/wP/nHPLzGTSOymkAAGRppTQEUFRRLCtq6ysrvKTXXWx7Ip1V9HVRWVxV13XXlhdQUFhV1GQKr2GIJ0kQBJILzMpk7ntfP8YGCNKmZ+aOJn36+Hhydy53Jw7PFzeOeVz7srpw4Nhlg4SJAAAQOgRRIyEEIcaXNdtWBohK1enZY1MSDkvMgaxoM1IjO2sq95cUzGv+JDCpYXDLkt2hLV3o84VEiQAAEBIazGNdw8f2OOuPdLortW9DkkOk7DQ9ifnMY1GQ4+3OTLCwvvHJk7JyJGCqrotEiQAAEDo8kcWIYQhhG5ZhrCQDNoAI1I4lxmXGfP/FQRLfCQkSICOymuaFS3NFV6PYf3cN1c9d5yxTo6weNURJqODBACgPeEpDNDRLCsvmXt4/4GG+iZDj1JsCR1l2wlG5DK0qhaPU1YynBGTUrOmZHZv70YBAIQo9EECdBzlnuZn9+9YWlYyPiVjUGziqISUBLvDEqKDLLEUxBmrbPFsrCnfVlf1xfGj3SNjZvUdkh4W3t4tAwAIOUiQAB2CoA+KD/5tf/4vOne5KSMnVrWbQlgkgmtWzVn5bocTkxhz6dq84kNzjxyY1qXn1C7nSx3oNgEAfv6QIAE6glUVx27dsvLl/iMvSkzVLLMjpcbTEULYJGm/u/6GDUtfHTh6VGJKe7cIACCEIEECBD2PYYxd/d9n+g4ZEJvgNUMiPvoIIWTOd9ZV/27bmqUXTewUPHXUAACCHXY1BAh6f969pXtEdK+oWM2yQic+EhFjzBSib3RcblzSzZuXt3dzAABCCBIkQHD7qvL4wpLCJ3sP4qGUHVuziJ7rN7TB0FdWlLZ3WwAAQgUSJEBw21ZXOTQ+Ocke1sYzUpoMfcLaz27a9GVVi6d9o6tvQuRVKVl7XLXt2hAAgBCCBAkQxCwh9rnqxiSl6W04fq1Z5v+OHRmx4pMD7rqN1eVjVi/eVFPRjiGSMWZY1tD45Ly66vZrBQBAaEGCBAhihmXtcdVOTM3SRRttPMOINlaX/2Hn+iZDlxhXuOQ1zSf2bHXpWts04HtZQqSEOQsaXe3YBgCAkIIECRDEDCEYY3IbzoAUREvLS4jI3+XJGNvrqnvv6EGFtdvzhDFm51KZp6m9GgAAEGqQIAGCmmjj4WO3rm2vrTpl1Y7K+d8P5G+rq2rHBwojpnA80AAA2ggeuADBrY0Lurp1rdTTeMpBX3/knP07W0yzbZsDAADtAwkSAM4VIzrS1NBk6P4j3SOiY1VbRlhEpjOi3Nv8QUnBWXcXNIW1o67q49LCGm/LT9xeAAD4qcjt3QAACBoy4/n11b4hbCGEXZaf6J07eeOyKq/Ht8HVa4V7pmTknDLGbQrRbBhey9hSU7n42OFPy47KxDhjEmf3d7/gjm69TOyMBQAQbJAgAeBcyZxtr6vixIjIEGJgTOKw+OTxKRkflxQqnBNjZZ6m27eufnXgKIlxS4h97rrNNRV5ddVFTe6iRpfHNGTGwyWZncygT+7dNjg+qV90PCIkAEBwQYIEgHMlMZ5fX+PrYjSFNTg+SbesmecP/KrqeL3XyxhTubS5tmLOgfwjTQ2HGuplziXGODGZsZyI6O+9ZhXGsgEAghASJACcqxpvS5mnKVJRiciwxJC4ZN2ywmX1wR4XPvr1Zt9gtGaaLx/azRk76y6LmmU+eN6Fk1IzG3U9pLbzBgDoALCSBgDOCSeWX1/tK/oohFAlfn5UrEXCFNa45PREu8O3rSJjzCZJCucSY2f4xYkujEm4Neu8JsNAfAQACDrogwSAc8IZ7ayvlviJtNft5Kg0YyxCUZ/vN/zGjcssIfxx0BIiwebIDo/sEh45KC5Jt6y7d6y1SzIRCSGcsvJ0n8E2SWqXewEAgB8ICRIAzo4Rbaut+rC4wL/xTJYzwv+uKcTA2MSbMru/XbRPIoq12S9P7jwpLSvbGWmTJJVLR5sabti4zMYlIhJCeIW1ZNhl3cKjTNF223kDAMCPCAkSAL6fxzTKPM3HPU3Lyou/qio74K6LUFT/7MZMZ2TrkzXLvKtbrzSHc1h8pz7RcRYJzbQsEoyoxttyx/avqr0eiXEikhif3Wdw1/BIxEcAgOCFBAkA38KJ5buqvywv+aK8pF7zunXNFBYnFqXa/OcwxnLCv7W2mjEWpdh+k9VDEDWbhv+4S9d+tenLgkaXLz62mMYjPQdcn94V8REAIKghQQLACY2GvrS8+JWCPXtctXYuKZz7Qp50cuRaCBEmK/GqPScyun9swnevYJ3yUoiZe7YWNLr9V5ia3XN6Tp9mA4uvAQCCGxIkABAR5ddXP7F729euGiLy1etpzRRicFzSuOT0AbGJqQ5nnM2uW9aZy4A7JOmuHWsXlBQ6JJmIDMu6Lr3Lwz37Iz4CAHQASJAAoY4T2+OuHb1yUYLN4Z/mKITgjIXJcpikTM7odm1adqYzssU0faPPmmWd+ZpCiFcK9vy39LAvPlpCDEvo9PKAUS7Ni/gIANABIEEChDqV89cK98aodtYqPkaptkuT0idndusZEatKXLesJkNnjJ1L/lMYX3S86Jl9O/xXOz8q9sULR7h1DfERAKBjQEVxACDfomlfSXAiYoy5NO+84kN/zNvwWfnRFtM0WxV6PCuJsd2uOo9p+l4Kom4RUeGy8pM0HQAA2gMSJECoa7HM5/oOuSY9u/U+hIwxibFDjfW/27rm4lWLH9u9ZX11mWaZ53JBj2nc171vVviJcj+csc/LinfWV6P7EQCgw0CCBAh1QgiVS0/2yh0an2yKbyY4MsYkxsNkuczT9FFx4a2bV05a+/nCkiL9bJMgiUjhfGp2T3+3ZYtp/Ktgt6+iOAAAdABIkAChzpfz7JL0rwGj/tpnyKDYRBuX/CPaJ05gxBgraHTdk7d20JcLzpwFfRe8Oi0rzeG0hCAiifHPjh9ddOywhHmQAAAdAhIkABARWUKoXLouvcubgy5eMGzcqMTU1iHSR2IsTJIjFEXQmSv5kBAiQlb+1GuA52R18QhZmb0/r07z/iStBwCAtoUECQBEJzsOiUhmLCciet7QS/8z5NLLOnWOUW1WqygpiLKdkd9bCtK0vjUCrlvWJUnp4zp19iVRxthxT/O6qjJ0QgIAdABIkADwLYwxi8ileQfEJszpN2zR8Mtz45L8/ZGCqEt41Cl9kDJj7xzeP/6rz2bt21HQUM9PXkezrNuze9qkE0PehrBmH9jJCBkSACDoIUECwKkYEWNMEHHGkuxhEzpl+AOjECLLGWF9e4B7WXnJM/t2HG1ueK1wz6iVi/ot/fDpvdu31FSUNjf2jo7rEx3vW6DDGStocP39YL6M2ZAAAEEOFcUB4EwY0TFPU+sjSfYwf36UGVt07PCD+RstIRhjKpdULjUZ+ltF+94o2ptgc6Q4nNGKrXNYhL9U0IrK0gkpGdnhUW17HwAA8GNCggSAs6j2tvi/Vrnk3zVbCLG2uvwPeetP2auGMebrZXTpmkvXrG8vyZEYO9ToRoIEAAhqSJAAcCaMqNLr8b+0S1KMaiMiibG11WV3bV/rf8vGpbQw53cWcJ/KImo5uUAbAACCFBIkAJwJY6za6/F3MNolKUpRiSi/vvrXm1bInDPGhBCGsD4aOq5fTPyCksIFpUU766pMIazvi5MSYw4JTx4AgOCG5zgAnEVVq1FsuyTHq/bdrtppW9fI/MRSvBjV9s/+o/rHJmimeX161ymZ3Y80ufPrazbVlO+sq9njqvGX+dEt69Lk9EuT09v8JgAA4MeEBAkAZ8KIqlr3QXKpVvPeumVFvaZxYsRIt6w5FwwfEJvgNU3GmC4sTTfjbY4xSWkXJ6YKojrNu7S8eFN1RbNlTEjJuCo1uz3vBwAAfgxIkABwJqYQLl3zzX30rYn5Y/4GIhar2nwnPNNv6MWJqR7T8C+m+dYXQsTb7Ldk9Zia3ZOIdMvSLJOhmg8AQJBDggSA02JELl3zF+JhREebG4qbG/wnvHjhiFEJKa3j46lXYEwQaZalkeU/8lM3GwAAfmqoKA4AZ9Kga/7ExxizhDCFMCxLCJrZa9CElEwhBEIhAECoQR8kAJwWI+Y2tO/GQ5nzR3sOuD69a5OhIz4CAIQgJEgAOC1BIka1y5z7OxqFEGGy8nSfwdemZSM+AgCELIxiA8BpWUKkOZwXxMT7XprC6h+b+MGQS65MyUB8BAAIZUiQAHBajDHdsl4fODrTGREhK1MyeiwYNq5HZIyBuY8AAKENo9gAcCaMsUSbY/noSW5di1ZsmmVh6QwAAKAPEgDOQhCZQjhlRRcWoRwPAAAgQQIAAABAoJAgAQAAACAwSJAAAAAAEBgkSAD4ltqamoaGhrOfBwAAIQwJEgBO+GrlqktHjOzXLad3dpcrLr74q5WrTNOUJCk1Krp7alpOSmq/bjlXjRv3xr9e8Xg8RCSEmDXziZyU1My4hC8+/UySJCJqbmq6cszY7qlp3VPTclLSLhqU+9iDDxUVFhKR2+UaN2LEeemdl362RAjxxaef9szI7J3dZfuWrYzjWQQAEEzw1AYAIqKD+/f/cfr046Wl2d26dc3JKSoofPapp2RFJiJJkhRVPb93b0VVCw4eeu4vf7l72m/tikpE69eutdntznDnf+bOtXHJdylFVVSbrXNmZrfuOVUVFQs++OC+O+501dcTkayoqs3GOaurrf3TjAcYY9N+f1f/QQMt02zHewcAgEAhQQIAEVFRQWGj2x2fkPDJ50s2bNny1Ozn5i36hAQRETHm9Xr/+dYbBw4evGfG/Yzz/Lw8V4O7sqJix9atFwwYEB0Ts3r58roGt/9qnPN7H5ixZt26d+bPt9ntpcXFdbV1RMSIGGOWENNuvsXtdvfq2+f+GQ8QKgQBAAQbJEgAICKKjolWbbb6urrbf33z/PnzJ15zTXhEhGVZvncZUXOzp1lYGdnZwrIYIy5J777xhqIo4ydeOfLi0XaHY8G8eb6BbB+v19uoa6ZpMMYkWZKkE08by7L+9vRf9+7endWly2tz53p0TQjRDjcMAAA/ABIkABAR9R84KKdHD2LswL5990+ffsXoMaXFxf6uQUmWl3625OnHZ874/XRFVaOjY+LCnB++/x9ZlkeNHn1B//6Konz84Uf+qwkh5s197zc33jj56ms0TeuckRkTF+d/t7ysjHNeU1WlaxqhAxIAIAghQQIAEZFqU+ct/mTWnDmjx46VZLm0pHjCmLFVlZW+dxVFeXH27LdefdWyrOGjRs1bvGjdls1erzciMvLt11/fsHYd57yutub4sWO+84UQu7/etXnDxjCnMzYu7vl/vRwREeH/XpZlpaanN7jd02+/XVGUdrhbAAD4YZAgAYCIyLKslhbvuCvGv/TmGx9/vkRVVSHE9i1biDEi0jTtoccff3f+vBUb1r82993omJhVK1YQkWma8997f9Xy5YyxirLyo4cP+67GOf/Lc88++sRMzes9VlLicDj8Q9VCiHHjx7+3cIGm6zt35C2cNx99kAAAQQcJEgCIiDauWzekT98/z3hg4bz5K5YtIyLGmH/o2TTNUWMuHj5qVEJSkmmauq7n78hjjE24atKsvz//t3++1Dkz0xJi+dJl8jd9iuzXt912fp8+dofjsQcfMk+utuacX37lhISEhN/+/i5Zll9+4QVXvasdbhgAAH4AJEgAIM75rp35QohFCxc+9uCDr7zwotfr7dOv34iRo0gIImJEpmEYhiGEYIx5W1p2bt9ORNdcf/3lEyZcPmHCxKuvtixr8YIFqqqevKowDOP/3XmHJEnr1qwpOHjQd5QxZlmWEGLKrbfa7LYjRUUL58/DWDYAQHCR27sBAND+TNO8+777BgwauOzzLyorym02e/9BA2+6+Rav5pUk6Ze/mizLSlR0tG8kmjHmcrkuufwym91+fu/eRGQYxmVXTigqLDANs7qqauxll/fuW5aZlU1Eo8eOvWHKlMaGhkMHDmR37XrJ+MvrampS09OFEJkZmX986OFdO3ce3H+gne8fAAACxFBHAyB4NRv6Jav/t2TkFT98KqE4WazRNAzGOedcCOF7PnDOicj3kjHm+933Hf3ntD7S+q1vjhNRq7d8v3P+TYkf+mGLsht1fezqxXvHT/4hHwIAAJwj9EECABGRL7sJIbgk0cmQ1zrw+V9+78HWR055q/WPqa2/9oXRUy4CAABBAfMgAQAAACAw6IME6OCaGhvLy8paH1EUJTU9/XBhIRHFJyTExMYKIYoKCizLiouPj4yMLCos4vybTkFVVVPT030jzrqmVZRXVJSXqaotMSkpMTlJCFF48BAxyszOVhRFCFFRVtbU1BQTGxsdE1N4qKD1pYiIGEvu1MnpdOq6XlFeXl1ZKQTFJyYkJSf7VuEIIY4fO9bi8ZxorapGRERGx0SjnxIA4OcDCRKgg1u7es0f7rpLnNyfkIg6Z2au3rjxzltvKywsHDR48OJPP925++vLRo4yTXPJqpW6rl86fLjD4fCfn9216/z//Tc8PLy6suqR++/ftnmz2+3mnCclJ982bdovfjX5otxcWZbX5+WlZ3Q2TfOJRx9dtXzFnffec8vUqRcPHty6ljgRSZL8wuuvjhw9eubDj3z5xee1NbWMWGRUZO6QIbP+/nxEVJRpGDOm371j2zZGJIjCnM7ExMQbbrrpN9Nut1rdBQAAtCMkSIAOT8iyTMQysjJ9+1Z3SkkRRM/84x+/nDRx186du/buef6ZZyVZnnTtNQMv7F9YfFSWZcZ5Rmam3eEQQqSlp3POhBDTfzstf0eezWbv3qOHrMglR4sdYQ7GuSzLsqLQyS5Czrksy5xzxvl5558fHhFRUlzc4vGoqpqZncUYj42Le/eNtz74978jIiLO69mTS1LxkSMrly9/+L4/vPLuO4auc4nLspyRlZWSkvL1rvxjpaWvvPTSRZeMzcjMbM8PEgAATkKCBAgJGVmZi7743Gl3EJEg8mjePhf0u2jMmA1r1z356J/y8/JiY+PueeCBZkP3nd/Y0PD5mtUOWRFEgqhF1/Lz8nbn71JttieemXX9ddfpRBvWr8sdMqSpufl7v6MQwul0rtu8WSKaPWfOP2bPTs/I+PLL5TpRrds1+eprFFmecuttj/75TxLRO3PnPv7ww5s3bty/d19G1omYeN0Nv7zjjjsPHS765ZUTGxoaKsvLkSABAH4mkCABQkJVZeVfn3hSVRSvpt157z2xsbFCiJtu+c3Gdeu3b91KRIOH9U/v3Nl/vt3heOrPjymqKoQYM+7Sgbm5R4uOaJqWnpw85tJLmnSNiAYMGmSapq9Qz+k065okSbquE5GwLC+RV9eKCgo45ybRtN/f2ax5iWjk6IsSk5LqamvLy477E6SmaY3elmMlpU3NzXaHIzIq6qf7fAAAICBIkAAhobGh4d9vvimIGhsabpl6W2xsLBENGTGciBhjhq5f/YvrJEnyl9ex2Wxz33xTEAkhUtPSBg8deuRIEWNMUVVneLh/i0Ii+u7yFnHi+GkXvlRXVhFRVHS0b3UOY0yx2Ww2m65p9bV1/tPee/udj+d/eGDfvjCnM3fIYHRAAgD8fCBBAoSEpOTkx55+2uFwmKaZkJTkS4qffPih711JlhfOnz923DhJPvFMaGpsfPfDDx0OhxDCtz4mObkTCWEYhsfjabV1IYWHR/oqhzc2NhJjwrJ0TWeM2R320zUmISmRiFz19YZh+FZYa15vS0uLoqoxcbH+0+pqa+tqa53h4Z1SUt56/z+arv0EHwwAAPz/QD1IgJDgCAsbkDsod+iQoSOG22w2ImppaXn/3blCiEnXXqvabFs3bd63d6+/Yo5hGLlDBucOHZI7dEhyp06WZXXN6SYrSk1V1fo1a3ynbd6wocHtZkSZ2dmyJG1cu5aEqKmpOVZayhhLSUs7XWMys7NVm00I8eqLL3HGOedfrVxVUV5ut9s7paT6T7v/kUfmLV5k6HrhoUNfLv3CtwwIAAB+DtAHCRASjpeW/vbmW2RZIqKY2NhnX3jhvx9/sm/PHlVVZz0/58C+fV/n5z/20EOfL13mO98ZHj51yq9lSSISYU7nX+fM6dW3b2Z29uHCwvun3z0gN9dVV3fwwIEJV02aOWvWxGuufvu111+c8/zq5SsKCwrcLpesKIMGDz5dY6Kiou+dMWPmI4+89/bbm9avdzqdu3ftIiGGjhiR06O752QlSM3rveDC/tfdeOMnH330zBNP9h84KDwivA0+KwAAOCv0QQKEBE3Ttm3ZsmnDxk0bNuZt3+Gqq5v58MOWZT302GMysadmP2dZVt627R8tXCjLMhHJsrxt8+ZNGzZs2rBx6+YthmHY7fYXX3+ta04307K+WrVqV35+mNPZKSXVNM0bpkzp1aePoevbtmxxrTqnrAAABC9JREFUu1yRUVH3PjCjU3Kn1nsYfrsx3pun3nb95MmOsLD9e/du37pVkuXho0fPen6Ob83NiTmUjHRDv3nqbU6ns/jo0eVLv0BRcQCAnwl2ukc8APz8NRv6Jav/t2TkFWeIVm6X61jpsdYrphVFSUxOLi0uFpbo3vM8X+3GfXv2WJbldIanpqXt37fXt8bFR5KkLl27ckninLe0tBw9fLi0pCQ8PCIlNSU9I8M3CbLB7T56+PDx48ejoqLSMzJS09JM0/S3qqqysrqyyma3denWTQjh+yOWZRUfPVp27Lih62kZndM7d5Zl2fdEKj5ypLm5OTEpKS4+3jKtw0WFmqZFRkWlnn5kvFHXx65evHf85B/hYwUAgLNBggQIYueSIH9cQghGxCXJEkJY1infl3MuhLC+c/wMV/NF1XP/I6eDBAkA0JYwDxIgiHHGrLb9IdCX83y7C343853u+Bmu5vsh9ocnYE2YKsdSGwCANoJ5kABBTGY81eHMr6+RQn6CYLNhJNgcZz8PAAB+DEiQAEFM5rxbZPTaquMyC+l/yxJjO+qqe0fHnv1UAAD4MYT0/zoAHcB5kTFfVpTKod0HKTO+paZicFwyJnYDALQNJEiA4DYxJbPK6/mg+FB7N6TdcKIN1WVLK0py45JQ7gcAoG0gQQIEMSFEtGp7f/AlM3dvbTb09m5O+6jXtft2rn+y16AMZ0R7twUAIFQgQQIEMd9a5n4x8RNTs57au91rmu3doramW9bLBbsHxCZe37krhrABANoM6kECBD0hRL2uTVy7ROH8nxeOSA0Lla3/6jXv77avcev6gmHjkuxh7d0cAIAQggQJ0HE8vnvr3MP7n+k3pF90fKSsOuSOWfC1xTQbdO1QY/3DuzaPSEj5x4XD27tFAAAhBwkSoCPw7RNIRAfc9Q/t2ljQ4IpS1Uxn5KiEFEEd59+4JSivrmqvu86ta9Gq7aX+I3tFxVKr2wcAgLaBBAnQcfiDVJOhr68u31BdvuT4kY4UrSTGRiakDI/vNDop1SkrhOwIANBOkCABOpoQCVUhcpsAAD9PSJAAAAAAEBhU8wEAAACAwCBBAgAAAEBgkCABAAAAIDBIkAAAAAAQGCRIAAAAAAgMEiQAAAAABAYJEgAAAAACgwQJAAAAAIFBggQAAACAwCBBAgAAAEBgkCABAAAAIDBIkAAAAAAQGCRIAAAAAAgMEiQAAAAABAYJEgAAAAACgwQJAAAAAIFBggQAAACAwCBBAgAAAEBgkCABAAAAIDBIkAAAAAAQGCRIAAAAAAgMEiQAAAAABAYJEgAAAAACgwQJAAAAAIFBggQAAACAwCBBAgAAAEBgkCABAAAAIDBIkAAAAAAQGCRIAAAAAAgMEiQAAAAABAYJEgAAAAACgwQJAAAAAIFBggQAAACAwCBBAgAAAEBg/g8FQKfV0O/muQAAAABJRU5ErkJggg==)"
      ],
      "metadata": {
        "id": "huDSxybA_B1e"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "1. Crear un DataFrame procedente de una base de datos relacional"
      ],
      "metadata": {
        "id": "QnAX_bGxuYFT"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "La base de datos Rfam es una colección de familias de secuencias de ARN estructurales que incluyen genes de ARN no codificantes, así como elementos cis-reguladores. Cada familia está representada por un alineamiento múltiple de secuencias y un modelo de covarianza (MC).\n",
        "\n",
        "[Enlace](https://docs.rfam.org/en/latest/about-rfam.html)"
      ],
      "metadata": {
        "id": "uNfiR2dd-xHu"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df = (spark.read.format(\"jdbc\")\n",
        ".option(\"url\",\"jdbc:mysql://mysql-rfam-public.ebi.ac.uk:4497/Rfam\")\n",
        ".option(\"dbtable\",\"family\")\n",
        ".option(\"user\",\"rfamro\")\n",
        ".option(\"password\",\"\")\n",
        ".load())"
      ],
      "metadata": {
        "id": "sSu1pJm0tyY8"
      },
      "execution_count": 18,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "df.show()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jm_IvJCm9cu3",
        "outputId": "c5d87ba4-249c-4633-f8ae-f1f04fdab316"
      },
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "+--------+-------------+---------+--------------------+--------------------+--------------------+----------------+--------------+------------+--------------------+--------------------+------------------+--------------------+--------------------+--------+--------+--------------+----------+--------------------+--------------------+-----------------+--------------------+---------------+--------+------------+---------+------------+--------------+----+----+---------------+-------+----------+-------------------+-------------------+\n",
            "|rfam_acc|      rfam_id|auto_wiki|         description|              author|         seed_source|gathering_cutoff|trusted_cutoff|noise_cutoff|             comment|         previous_id|           cmbuild|         cmcalibrate|            cmsearch|num_seed|num_full|num_genome_seq|num_refseq|                type|    structure_source|number_of_species|number_3d_structures|num_pseudonokts|tax_seed|ecmli_lambda| ecmli_mu|ecmli_cal_db|ecmli_cal_hits|maxl|clen|match_pair_node|hmm_tau|hmm_lambda|            created|            updated|\n",
            "+--------+-------------+---------+--------------------+--------------------+--------------------+----------------+--------------+------------+--------------------+--------------------+------------------+--------------------+--------------------+--------+--------+--------------+----------+--------------------+--------------------+-----------------+--------------------+---------------+--------+------------+---------+------------+--------------+----+----+---------------+-------+----------+-------------------+-------------------+\n",
            "| RF00001|      5S_rRNA|     1302|    5S ribosomal RNA|Griffiths-Jones S...|Szymanski et al, ...|            38.0|          38.0|        37.9|5S ribosomal RNA ...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     712|  139932|             0|         0|         Gene; rRNA;|Published; PMID:1...|             8022|                   0|           NULL|        |     0.59496| -5.32219|     1600000|        213632| 305| 119|           true|-3.7812|   0.71822|2013-10-03 20:41:44|2019-01-04 15:01:52|\n",
            "| RF00002|    5_8S_rRNA|     1303|  5.8S ribosomal RNA|Griffiths-Jones S...|Wuyts et al, Euro...|            42.0|          42.0|        41.9|5.8S ribosomal RN...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      61|    4879|             0|         0|         Gene; rRNA;|Published; PMID:1...|              598|                   0|           NULL|        |      0.6615| -9.18781|     1600000|        407997| 265| 154|           true|-3.4977|   0.71788|2013-10-03 20:47:00|2022-10-29 17:55:16|\n",
            "| RF00003|           U1|     1304| U1 spliceosomal RNA|Griffiths-Jones S...|Zwieb C, The uRNA...|            66.0|          66.0|        65.9|U1 is a small nuc...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     100|   11596|             0|         0|Gene; snRNA; spli...|Published; PMID:2...|              536|                   0|           NULL|        |     0.66474| -8.65259|     1600000|        410919| 262| 166|           true|-3.7311|   0.71618|2013-10-03 20:57:11|2022-10-29 17:55:16|\n",
            "| RF00004|           U2|     1305| U2 spliceosomal RNA|Griffiths-Jones S...|The uRNA database...|            46.0|          46.0|        45.9|U2 is a small nuc...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     208|   16562|             0|         0|Gene; snRNA; spli...|Published; PMID:2...|             1102|                   0|           NULL|        |     0.55222| -9.81359|     1600000|        403693| 301| 193|           true|-3.5144|   0.71292|2013-10-03 20:58:30|2021-07-05 14:43:57|\n",
            "| RF00005|         tRNA|     1306|                tRNA|Eddy SR, Griffith...|             Eddy SR|            29.0|          29.0|        28.9|Transfer RNA (tRN...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     954| 1429129|             0|         0|         Gene; tRNA;|Published; PMID:8...|             9951|                   0|           NULL|        |     0.64375| -4.21424|     1600000|        283681| 253|  71|           true|-2.6167|   0.73401|2013-10-03 21:00:26|2021-07-05 14:43:57|\n",
            "| RF00006|        Vault|     1307|           Vault RNA|Bateman A, Gardne...|Published; PMID:1...|            34.0|          34.1|        33.9|This family of RN...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      73|     564|             0|         0|               Gene;|Published; PMID:1...|               94|                   0|           NULL|        |     0.63669|  -4.8243|     1600000|        279629| 406| 101|           true|-3.5531|   0.71855|2013-10-03 22:04:04|2019-01-04 15:01:52|\n",
            "| RF00007|          U12|     1308|U12 minor spliceo...|Griffiths-Jones S...|Shukla GC and Pad...|            53.0|          53.0|        52.9|The U12 small nuc...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      62|     531|             0|         0|Gene; snRNA; spli...|Predicted; Griffi...|              336|                   0|           NULL|        |     0.55844| -9.95163|     1600000|        493455| 520| 155|           true|-3.1678|   0.71782|2013-10-03 22:04:07|2021-07-05 14:43:57|\n",
            "| RF00008| Hammerhead_3|     1309|Hammerhead ribozy...|Bateman A, Ontive...|           Bateman A|            41.8|          41.8|        41.7|The hammerhead ri...|          Hammerhead|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      85|     240|             0|         0|     Gene; ribozyme;|Published; PMID:7...|               39|                   0|           NULL|        |     0.65191| -3.95526|     1600000|        280797| 370|  54|           true|-4.4157|   0.71958|2013-10-03 22:04:11|2022-10-29 17:55:16|\n",
            "| RF00009|   RNaseP_nuc|     1310|     Nuclear RNase P|Griffiths-Jones S...|Brown JW, The Rib...|            28.0|          28.0|        27.9|Ribonuclease P (R...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     116|    1237|             0|         0|     Gene; ribozyme;|Published; PMID:9...|              763|                   0|           NULL|        |      0.7641| -8.04053|     1600000|        274636|1082| 303|           true|-4.3673|   0.70576|2013-10-03 22:04:14|2021-07-05 14:43:57|\n",
            "| RF00010|RNaseP_bact_a|     2441|Bacterial RNase P...|Griffiths-Jones S...|Brown JW, The Rib...|           100.0|         100.5|        99.6|Ribonuclease P (R...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     458|    6023|             0|         0|     Gene; ribozyme;|Published; PMID:9...|             6324|                   0|           NULL|        |     0.76804| -8.48988|     1600000|        366265| 873| 367|           true|-4.3726|   0.70355|2013-10-03 22:04:21|2021-07-05 14:43:57|\n",
            "| RF00011|RNaseP_bact_b|     2441|Bacterial RNase P...|Griffiths-Jones S...|Brown JW, The Rib...|            97.0|          97.1|        96.6|Ribonuclease P (R...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     114|     676|             0|         0|     Gene; ribozyme;|Published; PMID:9...|              767|                   0|           NULL|        |     0.69906|  -8.4903|     1600000|        418092| 675| 366|           true|-4.0357|   0.70361|2013-10-03 22:04:51|2021-07-05 14:43:57|\n",
            "| RF00012|           U3|     1312|Small nucleolar R...|  Gardner PP, Marz M|Published; PMID:1...|            34.0|          34.0|        33.9|Small nucleolar R...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      87|    3924|             0|         0|Gene; snRNA; snoR...|Published; PMID:1...|              416|                   0|           NULL|        |     0.59795| -9.77278|     1600000|        400072| 326| 218|           true|-3.8301|   0.71077|2013-10-03 22:04:54|2019-01-04 15:01:52|\n",
            "| RF00013|           6S|     2461|       6S / SsrS RNA|Bateman A, Barric...|          Barrick JE|            44.2|          44.2|        44.1|E. coli 6S RNA wa...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     150|    3636|             0|         0|               Gene;|Published; PMID:1...|             3414|                   0|           NULL|        |     0.54245|-10.21771|     1600000|        317741| 287| 187|           true|-3.6268|   0.71364|2013-10-03 22:05:06|2021-07-05 14:43:57|\n",
            "| RF00014|         DsrA|     1237|            DsrA RNA|           Bateman A|           Bateman A|            60.0|          61.5|        57.6|DsrA RNA regulate...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|       5|      35|             0|         0|         Gene; sRNA;|Published; PMID:9...|               39|                   0|           NULL|        |     0.53383| -8.38474|     1600000|        350673| 177|  85|           true|-3.3562|   0.71888|2013-02-01 11:56:19|2019-01-04 15:01:52|\n",
            "| RF00015|           U4|     1314| U4 spliceosomal RNA|Griffiths-Jones S...|Zwieb C, The uRNA...|            46.0|          46.0|        45.9|U4 small nuclear ...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     177|    8285|             0|         0|Gene; snRNA; spli...|Published; PMID:2...|             1028|                   0|           NULL|        |     0.60966| -8.50576|     1600000|        296541| 361| 138|           true|-3.5407|   0.71795|2013-10-03 22:05:22|2022-10-29 17:55:16|\n",
            "| RF00016|      SNORD14|     1242|Small nucleolar R...|Griffiths-Jones S...|  Griffiths-Jones SR|            64.0|          64.1|        63.9|U14 small nucleol...|                 U14|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      18|    1182|             0|         0|Gene; snRNA; snoR...|    Predicted; PFOLD|              221|                   0|           NULL|        |     0.63073| -3.65386|     1600000|        232910| 229| 116|           true| -3.128|   0.71819|2013-02-01 11:56:23|2019-01-04 15:01:52|\n",
            "| RF00017|  Metazoa_SRP|     1315|Metazoan signal r...|          Gardner PP|Published; PMID:1...|            70.0|          70.0|        69.9|The signal recogn...|SRP_euk_arch; 7SL...|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      91|   42386|             0|         0|               Gene;|Published; PMID:1...|              402|                   0|           NULL|        |     0.64536| -9.85267|     1600000|        488632| 514| 301|           true|-4.0177|   0.70604|2013-10-03 22:07:53|2021-07-05 14:43:57|\n",
            "| RF00018|         CsrB|     2460|CsrB/RsmB RNA family|Bateman A, Gardne...|           Bateman A|            71.0|          71.4|        70.9|The CsrB RNA bind...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      38|     254|             0|         0|         Gene; sRNA;|Predicted; PFOLD;...|              196|                   0|           NULL|        |     0.69326| -9.81172|     1600000|        546392| 555| 356|           true|-4.0652|   0.70388|2013-10-03 23:07:27|2021-07-05 14:43:57|\n",
            "| RF00019|        Y_RNA|     1317|               Y RNA|Griffiths-Jones S...|Griffiths-Jones S...|            38.0|          38.0|        37.9|Y RNAs are compon...|      Y1; Y2; Y3; Y5|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     104|    8521|             0|         0|               Gene;|Published; PMID:1...|              123|                   0|           NULL|        |     0.59183| -5.14312|     1600000|        189478| 249|  98|           true|-2.8418|    0.7187|2013-10-03 23:07:38|2019-01-04 15:01:52|\n",
            "| RF00020|           U5|     1318| U5 spliceosomal RNA|Griffiths-Jones S...|Zwieb C, The uRNA...|            40.0|          40.0|        39.9|U5 RNA is a compo...|                NULL|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     180|    7524|             0|         0|Gene; snRNA; spli...|Published; PMID:2...|             1001|                   0|           NULL|        |     0.50732| -5.54774|     1600000|        339349| 331| 116|           true|-4.1327|    0.7182|2013-10-03 23:08:43|2019-01-04 15:01:52|\n",
            "+--------+-------------+---------+--------------------+--------------------+--------------------+----------------+--------------+------------+--------------------+--------------------+------------------+--------------------+--------------------+--------+--------+--------------+----------+--------------------+--------------------+-----------------+--------------------+---------------+--------+------------+---------+------------+--------------+----+----+---------------+-------+----------+-------------------+-------------------+\n",
            "only showing top 20 rows\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Spark también permite definir la división o partición para que los datos se extraigan en paralelo de diferentes tareas generadas por los ejecutores de Spark.\n",
        "* PartitionColumn es un equivalente de la opción split-by en Sqoop.\n",
        "\n",
        "* NumPartitions también define el número máximo de conexiones JDBC \"concurrentes\" realizadas a las bases de datos. La conexión JDBC concurrente real podría ser inferior a este número en función del número de ejecutores de Spark disponibles para el trabajo.\n",
        "\n",
        "* LowerBound y UpperBound definen el rango mínimo y máximo de la clave primaria, que utiliza junto con numPartitions para permitir a Spark paralelizar la extracción de datos dividiendo el rango en múltiples tareas.\n",
        "\n"
      ],
      "metadata": {
        "id": "Yl2aM2kT_VWw"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df2 = (spark.read.format(\"jdbc\")\n",
        ".option(\"url\",\"jdbc:mysql://mysql-rfam-public.ebi.ac.uk:4497/Rfam\")\n",
        ".option(\"dbtable\",\"family\")\n",
        ".option(\"user\",\"rfamro\")\n",
        ".option(\"password\",\"\")\n",
        ".option(\"lowerBound\", 0)\n",
        ".option(\"upperBound\",10000)\n",
        ".option(\"numPartitions\", 4)\n",
        ".option(\"partitionColumn\", \"auto_wiki\")\n",
        ".load())\n"
      ],
      "metadata": {
        "id": "-kIY6ngr_jFp"
      },
      "execution_count": 20,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "df2.show()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "hpL9dxN6AURA",
        "outputId": "072826e1-fba8-42eb-ab68-38a61f8c2dbf"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "+--------+-------------+---------+--------------------+--------------------+--------------------+----------------+--------------+------------+--------------------+--------------------+------------------+--------------------+--------------------+--------+--------+--------------+----------+--------------------+--------------------+-----------------+--------------------+---------------+--------+------------+---------+------------+--------------+----+----+---------------+-------+----------+-------------------+-------------------+\n",
            "|rfam_acc|      rfam_id|auto_wiki|         description|              author|         seed_source|gathering_cutoff|trusted_cutoff|noise_cutoff|             comment|         previous_id|           cmbuild|         cmcalibrate|            cmsearch|num_seed|num_full|num_genome_seq|num_refseq|                type|    structure_source|number_of_species|number_3d_structures|num_pseudonokts|tax_seed|ecmli_lambda| ecmli_mu|ecmli_cal_db|ecmli_cal_hits|maxl|clen|match_pair_node|hmm_tau|hmm_lambda|            created|            updated|\n",
            "+--------+-------------+---------+--------------------+--------------------+--------------------+----------------+--------------+------------+--------------------+--------------------+------------------+--------------------+--------------------+--------+--------+--------------+----------+--------------------+--------------------+-----------------+--------------------+---------------+--------+------------+---------+------------+--------------+----+----+---------------+-------+----------+-------------------+-------------------+\n",
            "| RF00001|      5S_rRNA|     1302|    5S ribosomal RNA|Griffiths-Jones S...|Szymanski et al, ...|            38.0|          38.0|        37.9|5S ribosomal RNA ...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     712|  139932|             0|         0|         Gene; rRNA;|Published; PMID:1...|             8022|                   0|           null|        |     0.59496| -5.32219|     1600000|        213632| 305| 119|           true|-3.7812|   0.71822|2013-10-03 20:41:44|2019-01-04 15:01:52|\n",
            "| RF00002|    5_8S_rRNA|     1303|  5.8S ribosomal RNA|Griffiths-Jones S...|Wuyts et al, Euro...|            42.0|          42.0|        41.9|5.8S ribosomal RN...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      61|    4716|             0|         0|         Gene; rRNA;|Published; PMID:1...|              587|                   0|           null|        |     0.65546| -9.33442|     1600000|        410495| 277| 154|           true|-3.5135|   0.71791|2013-10-03 20:47:00|2019-01-04 15:01:52|\n",
            "| RF00003|           U1|     1304| U1 spliceosomal RNA|Griffiths-Jones S...|Zwieb C, The uRNA...|            67.8|          67.8|        67.7|U1 is a small nuc...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     100|   11452|             0|         0|Gene; snRNA; spli...|Published; PMID:2...|              526|                   0|           null|        |     0.67253| -8.56427|     1600000|        409669| 267| 166|           true|-3.7747|   0.71617|2013-10-03 20:57:11|2021-07-05 14:43:57|\n",
            "| RF00004|           U2|     1305| U2 spliceosomal RNA|Griffiths-Jones S...|The uRNA database...|            46.0|          46.0|        45.9|U2 is a small nuc...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     208|   16562|             0|         0|Gene; snRNA; spli...|Published; PMID:2...|             1102|                   0|           null|        |     0.55222| -9.81359|     1600000|        403693| 301| 193|           true|-3.5144|   0.71292|2013-10-03 20:58:30|2021-07-05 14:43:57|\n",
            "| RF00005|         tRNA|     1306|                tRNA|Eddy SR, Griffith...|             Eddy SR|            29.0|          29.0|        28.9|Transfer RNA (tRN...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     954| 1429129|             0|         0|         Gene; tRNA;|Published; PMID:8...|             9951|                   0|           null|        |     0.64375| -4.21424|     1600000|        283681| 253|  71|           true|-2.6167|   0.73401|2013-10-03 21:00:26|2021-07-05 14:43:57|\n",
            "| RF00006|        Vault|     1307|           Vault RNA|Bateman A, Gardne...|Published; PMID:1...|            34.0|          34.1|        33.9|This family of RN...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      73|     564|             0|         0|               Gene;|Published; PMID:1...|               94|                   0|           null|        |     0.63669|  -4.8243|     1600000|        279629| 406| 101|           true|-3.5531|   0.71855|2013-10-03 22:04:04|2019-01-04 15:01:52|\n",
            "| RF00007|          U12|     1308|U12 minor spliceo...|Griffiths-Jones S...|Shukla GC and Pad...|            53.0|          53.0|        52.9|The U12 small nuc...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      62|     531|             0|         0|Gene; snRNA; spli...|Predicted; Griffi...|              336|                   0|           null|        |     0.55844| -9.95163|     1600000|        493455| 520| 155|           true|-3.1678|   0.71782|2013-10-03 22:04:07|2021-07-05 14:43:57|\n",
            "| RF00008| Hammerhead_3|     1309|Hammerhead ribozy...|           Bateman A|           Bateman A|            32.9|          32.9|        32.8|The hammerhead ri...|          Hammerhead|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      86|    1504|             0|         0|     Gene; ribozyme;|Published; PMID:7...|              116|                   0|           null|        |     0.61508| -4.35009|     1600000|        264501| 375|  58|           true| -4.203|   0.71914|2013-10-03 22:04:11|2021-07-05 14:43:57|\n",
            "| RF00009|   RNaseP_nuc|     1310|     Nuclear RNase P|Griffiths-Jones S...|Brown JW, The Rib...|            28.0|          28.0|        27.9|Ribonuclease P (R...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     116|    1237|             0|         0|     Gene; ribozyme;|Published; PMID:9...|              763|                   0|           null|        |      0.7641| -8.04053|     1600000|        274636|1082| 303|           true|-4.3673|   0.70576|2013-10-03 22:04:14|2021-07-05 14:43:57|\n",
            "| RF00010|RNaseP_bact_a|     2441|Bacterial RNase P...|Griffiths-Jones S...|Brown JW, The Rib...|           100.0|         100.5|        99.6|Ribonuclease P (R...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     458|    6023|             0|         0|     Gene; ribozyme;|Published; PMID:9...|             6324|                   0|           null|        |     0.76804| -8.48988|     1600000|        366265| 873| 367|           true|-4.3726|   0.70355|2013-10-03 22:04:21|2021-07-05 14:43:57|\n",
            "| RF00011|RNaseP_bact_b|     2441|Bacterial RNase P...|Griffiths-Jones S...|Brown JW, The Rib...|            97.0|          97.1|        96.6|Ribonuclease P (R...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     114|     676|             0|         0|     Gene; ribozyme;|Published; PMID:9...|              767|                   0|           null|        |     0.69906|  -8.4903|     1600000|        418092| 675| 366|           true|-4.0357|   0.70361|2013-10-03 22:04:51|2021-07-05 14:43:57|\n",
            "| RF00012|           U3|     1312|Small nucleolar R...|  Gardner PP, Marz M|Published; PMID:1...|            34.0|          34.0|        33.9|Small nucleolar R...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      87|    3924|             0|         0|Gene; snRNA; snoR...|Published; PMID:1...|              416|                   0|           null|        |     0.59795| -9.77278|     1600000|        400072| 326| 218|           true|-3.8301|   0.71077|2013-10-03 22:04:54|2019-01-04 15:01:52|\n",
            "| RF00013|           6S|     2461|       6S / SsrS RNA|Bateman A, Barric...|          Barrick JE|            44.2|          44.2|        44.1|E. coli 6S RNA wa...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     150|    3636|             0|         0|               Gene;|Published; PMID:1...|             3414|                   0|           null|        |     0.54245|-10.21771|     1600000|        317741| 287| 187|           true|-3.6268|   0.71364|2013-10-03 22:05:06|2021-07-05 14:43:57|\n",
            "| RF00014|         DsrA|     1237|            DsrA RNA|           Bateman A|           Bateman A|            60.0|          61.5|        57.6|DsrA RNA regulate...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|       5|      35|             0|         0|         Gene; sRNA;|Published; PMID:9...|               39|                   0|           null|        |     0.53383| -8.38474|     1600000|        350673| 177|  85|           true|-3.3562|   0.71888|2013-02-01 11:56:19|2019-01-04 15:01:52|\n",
            "| RF00015|           U4|     1314| U4 spliceosomal RNA|  Griffiths-Jones SR|Zwieb C, The uRNA...|            46.0|          46.0|        45.9|U4 small nuclear ...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     176|    7812|             0|         0|Gene; snRNA; spli...|Published; PMID:2...|             1033|                   0|           null|        |     0.59583| -8.71387|     1600000|        283644| 530| 139|           true|-3.5999|   0.71794|2013-10-03 22:05:22|2021-07-05 14:43:57|\n",
            "| RF00016|      SNORD14|     1242|Small nucleolar R...|Griffiths-Jones S...|  Griffiths-Jones SR|            64.0|          64.1|        63.9|U14 small nucleol...|                 U14|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      18|    1182|             0|         0|Gene; snRNA; snoR...|    Predicted; PFOLD|              221|                   0|           null|        |     0.63073| -3.65386|     1600000|        232910| 229| 116|           true| -3.128|   0.71819|2013-02-01 11:56:23|2019-01-04 15:01:52|\n",
            "| RF00017|  Metazoa_SRP|     1315|Metazoan signal r...|          Gardner PP|Published; PMID:1...|            70.0|          70.0|        69.9|The signal recogn...|SRP_euk_arch; 7SL...|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      91|   42386|             0|         0|               Gene;|Published; PMID:1...|              402|                   0|           null|        |     0.64536| -9.85267|     1600000|        488632| 514| 301|           true|-4.0177|   0.70604|2013-10-03 22:07:53|2021-07-05 14:43:57|\n",
            "| RF00018|         CsrB|     2460|CsrB/RsmB RNA family|Bateman A, Gardne...|           Bateman A|            71.0|          71.4|        70.9|The CsrB RNA bind...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|      38|     254|             0|         0|         Gene; sRNA;|Predicted; PFOLD;...|              196|                   0|           null|        |     0.69326| -9.81172|     1600000|        546392| 555| 356|           true|-4.0652|   0.70388|2013-10-03 23:07:27|2021-07-05 14:43:57|\n",
            "| RF00019|        Y_RNA|     1317|               Y RNA|Griffiths-Jones S...|Griffiths-Jones S...|            38.0|          38.0|        37.9|Y RNAs are compon...|      Y1; Y2; Y3; Y5|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     104|    8521|             0|         0|               Gene;|Published; PMID:1...|              123|                   0|           null|        |     0.59183| -5.14312|     1600000|        189478| 249|  98|           true|-2.8418|    0.7187|2013-10-03 23:07:38|2019-01-04 15:01:52|\n",
            "| RF00020|           U5|     1318| U5 spliceosomal RNA|Griffiths-Jones S...|Zwieb C, The uRNA...|            40.0|          40.0|        39.9|U5 RNA is a compo...|                null|cmbuild -F CM SEED|cmcalibrate --mpi CM|cmsearch --cpu 4 ...|     180|    7524|             0|         0|Gene; snRNA; spli...|Published; PMID:2...|             1001|                   0|           null|        |     0.50732| -5.54774|     1600000|        339349| 331| 116|           true|-4.1327|    0.7182|2013-10-03 23:08:43|2019-01-04 15:01:52|\n",
            "+--------+-------------+---------+--------------------+--------------------+--------------------+----------------+--------------+------------+--------------------+--------------------+------------------+--------------------+--------------------+--------+--------+--------------+----------+--------------------+--------------------+-----------------+--------------------+---------------+--------+------------+---------+------------+--------------+----+----+---------------+-------+----------+-------------------+-------------------+\n",
            "only showing top 20 rows\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "db_name = \"rfam\"\n",
        "spark.sql(f\"CREATE DATABASE IF NOT EXISTS {db_name}\")\n",
        "df2.write.saveAsTable(\"rfam.family\")"
      ],
      "metadata": {
        "id": "nT6gMX6nAXtt"
      },
      "execution_count": 21,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "df2.write.csv(\"prueba.csv\")"
      ],
      "metadata": {
        "id": "xZXAclVQCUNP"
      },
      "execution_count": 22,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!ls prueba.csv"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "2c8tiztACm6F",
        "outputId": "6cd58180-585d-48ff-d1b7-eb218c8963dd"
      },
      "execution_count": 23,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "part-00000-59ca1e2d-7a8e-45ef-9dfa-6ce080d07dec-c000.csv  _SUCCESS\n",
            "part-00001-59ca1e2d-7a8e-45ef-9dfa-6ce080d07dec-c000.csv\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!cat prueba.csv/*"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "63vIX2J3BN9H",
        "outputId": "4214f0ea-7f32-404d-de8e-fd0abd5f6bcc"
      },
      "execution_count": 24,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "RF00001,5S_rRNA,1302,5S ribosomal RNA,\"Griffiths-Jones SR, Mifsud W, Gardner PP\",\"Szymanski et al, 5S ribosomal database, PMID:11752286\",38.0,38.0,37.9,\"5S ribosomal RNA (5S rRNA) is a component of the large ribosomal subunit in both prokaryotes and eukaryotes. In eukaryotes, it is synthesised by RNA polymerase III (the other eukaryotic rRNAs are cleaved from a 45S precursor synthesised by RNA polymerase I). In Xenopus oocytes, it has been shown that fingers 4-7 of the nine-zinc finger transcription factor TFIIIA can bind to the central region of 5S RNA. Thus, in addition to positively regulating 5S rRNA transcription, TFIIIA also stabilises 5S rRNA until it is required for transcription.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.99 -Z 549862.597050 CM SEQDB,712,139932,0,0,Gene; rRNA;,Published; PMID:11283358,8022,0,,\"\",0.59496,-5.32219,1600000,213632,305,119,true,-3.7812,0.71822,2013-10-03T20:41:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00002,5_8S_rRNA,1303,5.8S ribosomal RNA,\"Griffiths-Jones SR, Mifsud W\",\"Wuyts et al, European LSU rRNA database, PMID:11125083\",42.0,42.0,41.9,\"5.8S ribosomal RNA (5.8S rRNA) is a component of the large subunit of the eukaryotic ribosome. It is transcribed by RNA polymerase I as part of the 45S precursor that also contains 18S and 28S rRNA. Functionally, it is thought that 5.8S rRNA may be involved in ribosome translocation [2]. It is also known to form covalent linkage to the p53 tumour suppressor protein [3]. 5.8S rRNA is also found in archaea.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.62 -Z 742849.287494 CM SEQDB,61,4879,0,0,Gene; rRNA;,Published; PMID:11125083,598,0,,\"\",0.6615,-9.18781,1600000,407997,265,154,true,-3.4977,0.71788,2013-10-03T20:47:00.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00003,U1,1304,U1 spliceosomal RNA,\"Griffiths-Jones SR, Mifsud W, Moxon SJ, Ontiveros-Palacios N\",\"Zwieb C, The uRNA database, PMID:9016512\",66.0,66.0,65.9,\"U1 is a small nuclear RNA (snRNA) component of the spliceosome (involved in pre-mRNA splicing). Its 5' end forms complementary base pairs with the 5' splice junction, thus defining the 5' donor site of an intron. There are significant differences in sequence and secondary structure between metazoan and yeast U1 snRNAs, the latter being much longer (568 nucleotides as compared to 164 nucleotides in human). Nevertheless, secondary structure predictions suggest that all U1 snRNAs share a 'common core' consisting of helices I, II, the proximal region of III, and IV [1]. This family does not contain the larger yeast sequences. The structure of U1 spliceosomal RNA has been reported in [5,6]. It present 4 Stem loops (SL1, SL2, SL3, and SL4) and a region call Helix H. SL1, SL2, and SL3 are join for the Helix H, forming a four-helix junction that are separated of SL4. U1 snRNA is important in the precatalytic spliceosome, where the 5' splice site (5'SS) of the pre-mRNA is recognized by pairing with 5'-U1 snRNA. Where spliceosome activation is initiated by a disruption of the 5â€²SSâ€“U1 snRNP interaction by the DEAD-box helicase Prp28 [6]. The structure of U1 small nucleolar RNA was reported in PDB:6QX9\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,100,11596,0,0,Gene; snRNA; splicing;,Published; PMID:2405391,536,0,,\"\",0.66474,-8.65259,1600000,410919,262,166,true,-3.7311,0.71618,2013-10-03T20:57:11.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00004,U2,1305,U2 spliceosomal RNA,\"Griffiths-Jones SR, Mifsud W, Gardner PP\",\"The uRNA database, PMID:9016512; Griffiths-Jones SR; PMID:18390578\",46.0,46.0,45.9,\"U2 is a small nuclear RNA (snRNA) component of the spliceosome (involved in pre-mRNA splicing). Complementary binding between U2 snRNA (in an area lying towards the 5' end but 3' to hairpin I) and the branchpoint sequence (BPS) of the intron results the bulging out of an unpaired adenosine, on the BPS, which initiates a nucleophilic attack at the intronic 5' splice site, thus starting the first of two transesterification reactions that mediate splicing.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,208,16562,0,0,Gene; snRNA; splicing;,Published; PMID:2339054; Griffiths-Jones SR,1102,0,,\"\",0.55222,-9.81359,1600000,403693,301,193,true,-3.5144,0.71292,2013-10-03T20:58:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00005,tRNA,1306,tRNA,\"Eddy SR, Griffiths-Jones SR, Mifsud W\",Eddy SR,29.0,29.0,28.9,\"Transfer RNA (tRNA) molecules are approximately 80 nucleotides in length. Their secondary structure includes four short double-helical elements and three loops (D, anti-codon, and T loops). Further hydrogen bonds mediate the characteristic L-shaped molecular structure. tRNAs have two regions of fundamental functional importance: the anti-codon, which is responsible for specific mRNA codon recognition, and the 3' end, to which the tRNAs corresponding amino acid is attached (by aminoacyl-tRNA synthetases). tRNAs cope with the degeneracy of the genetic code in two manners: having more than one tRNA (with a specific anti-codon) for a particular amino acid; and 'wobble' base-pairing, i.e. permitting non-standard base-pairing at the 3rd anti-codon position.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.00 -Z 549862.597050 CM SEQDB,954,1429129,0,0,Gene; tRNA;,Published; PMID:8256282,9951,0,,\"\",0.64375,-4.21424,1600000,283681,253,71,true,-2.6167,0.73401,2013-10-03T21:00:26.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00006,Vault,1307,Vault RNA,\"Bateman A, Gardner PP\",Published; PMID:19491402,34.0,34.1,33.9,\"This family of RNAs are found as part of the enigmatic vault ribonucleoprotein complex. The complex consists of a major vault protein (MVP), two minor vault proteins (VPARP and TEP1), and several small untranslated RNA molecules. It has been suggested that the vault complex is involved in drug resistance. We have identified a putative novel vault RNA on chromosome 5 EMBL:AC005219.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,73,564,0,0,Gene;,Published; PMID:19491402,94,0,,\"\",0.63669,-4.8243,1600000,279629,406,101,true,-3.5531,0.71855,2013-10-03T22:04:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00007,U12,1308,U12 minor spliceosomal RNA,\"Griffiths-Jones S, Mifsud W, Gardner PP\",\"Shukla GC and Padgett RA, PMID:10199569; PMID:18390578\",53.0,53.0,52.9,\"The U12 small nuclear (snRNA), together with U4atac/U6atac, U5, and U11 snRNAs and associated proteins, forms a spliceosome that cleaves a divergent class of low-abundance pre-mRNA introns. Although the U12 sequence is very divergent from that of U2 (Rfam:RF00004), the two are functionally analogous [2]. The predicted secondary structure of U12 RNA is published (see for example, PMID:10199569), but the alternative single hairpin in the 3' end shown here seems to better match the alignment of divergent Drosophila and Arabidopsis sequences.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,62,531,0,0,Gene; snRNA; splicing;,Predicted; Griffiths-Jones S,336,0,,\"\",0.55844,-9.95163,1600000,493455,520,155,true,-3.1678,0.71782,2013-10-03T22:04:07.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00008,Hammerhead_3,1309,Hammerhead ribozyme (type III),\"Bateman A, Ontiveros-Palacios N\",Bateman A,41.8,41.8,41.7,\"The hammerhead ribozyme is one of the smallest catalytic RNAs. These RNAs have an endonuclease function, and most often are autocatalytic. Structurally it is composed of three base paired helices, separated by short linkers of conserved sequence. These helices are called I, II and III. We have classified hammerhead ribozymes into three types based on which helix the 5' and 3' ends of the sequence join. This family are the type III hammerheads. The conserved uridine-turn links helix I to helix II and usually has the sequence CUGA. Helix II and III are linked by a sequence CGAAA. The cleavage reaction occurs between helix III and I, and is usually a C. Hammerhead ribozymes are found in plant viroids and other small replicating satellite RNA species. Hammerhead ribozymes have been found in animals as well as plants. The structure of Hammerhead III ribozyme was reported in PDBs 2QUS, 2QUW [4], 5DI2, 5DI4, 5DQK [5], 5EAO, and 5EAQ [6].\",Hammerhead,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,85,240,0,0,Gene; ribozyme;,Published; PMID:7969422,39,0,,\"\",0.65191,-3.95526,1600000,280797,370,54,true,-4.4157,0.71958,2013-10-03T22:04:11.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00009,RNaseP_nuc,1310,Nuclear RNase P,\"Griffiths-Jones SR, Mifsud W\",\"Brown JW, The Ribonuclease P Database, PMID:9847214\",28.0,28.0,27.9,\"Ribonuclease P (RNase P) is a ubiquitous endoribonuclease, found in archaea, bacteria and eukarya as well as chloroplasts and mitochondria. Its best characterised activity is the generation of mature 5'-ends of tRNAs by cleaving the 5'-leader elements of precursor-tRNAs. Cellular RNase Ps are ribonucleoproteins. RNA from bacterial RNase Ps retains its catalytic activity in the absence of the protein subunit, i.e. it is a ribozyme. Isolated eukaryotic and archaeal RNase P RNA has not been shown to retain its catalytic function, but is still essential for the catalytic activity of the holoenzyme. Although the archaeal and eukaryotic holoenzyme s have a much greater protein content than the bacterial ones, the RNA cores from all the three lineages are homologous -- helices corresponding to P1, P2, P3, P4, and P10/11 are common to all cellular RNase P RNAs. Yet, there is considerable sequence variation, particularly among the eukaryotic RNAs.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 16.14 -Z 549862.597050 CM SEQDB,116,1237,0,0,Gene; ribozyme;,Published; PMID:9847214,763,0,,\"\",0.7641,-8.04053,1600000,274636,1082,303,true,-4.3673,0.70576,2013-10-03T22:04:14.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00010,RNaseP_bact_a,2441,Bacterial RNase P class A,\"Griffiths-Jones SR, Mifsud W\",\"Brown JW, The Ribonuclease P Database, PMID:9847214\",100.0,100.5,99.6,\"Ribonuclease P (RNase P) is a ubiquitous endoribonuclease, found in archaea, bacteria and eukarya as well as chloroplasts and mitochondria. Its best characterised activity is the generation of mature 5'-ends of tRNAs by cleaving the 5'-leader elements of precursor-tRNAs. Cellular RNase Ps are ribonucleoproteins. RNA from bacterial RNase Ps retains its catalytic activity in the absence of the protein subunit, i.e. it is a ribozyme. Isolated eukaryotic and archaeal RNase P RNA has not been shown to retain its catalytic function, but is still essential for the catalytic activity of the holoenzyme. Although the archaeal and eukaryotic holoenzyme s have a much greater protein content than the bacterial ones, the RNA cores from all the three lineages are homologous -- helices corresponding to P1, P2, P3, P4, and P10/11 are common to all cellular RNase P RNAs. Yet, there is considerable sequence variation, particularly among the eukaryotic RNAs. Identified in Yersinia as Ysr151/RnpB.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,458,6023,0,0,Gene; ribozyme;,Published; PMID:9847214,6324,0,,\"\",0.76804,-8.48988,1600000,366265,873,367,true,-4.3726,0.70355,2013-10-03T22:04:21.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00011,RNaseP_bact_b,2441,Bacterial RNase P class B,\"Griffiths-Jones SR, Mifsud W, Gardner PP\",\"Brown JW, The Ribonuclease P Database, 9847214\",97.0,97.1,96.6,\"Ribonuclease P (RNase P) is a ubiquitous endoribonuclease, found in archaea, bacteria and eukarya as well as chloroplasts and mitochondria. Its best characterised activity is the generation of mature 5'-ends of tRNAs by cleaving the 5'-leader elements of precursor-tRNAs. Cellular RNase Ps are ribonucleoproteins. RNA from bacterial RNase Ps retains its catalytic activity in the absence of the protein subunit, i.e. it is a ribozyme. Isolated eukaryotic and archaeal RNase P RNA has not been shown to retain its catalytic function, but is still essential for the catalytic activity of the holoenzyme. Although the archaeal and eukaryotic holoenzyme s have a much greater protein content than the bacterial ones, the RNA cores from all the three lineages are homologous -- helices corresponding to P1, P2, P3, P4, and P10/11 are common to all cellular RNase P RNAs. Yet, there is considerable sequence variation, particularly among the eukaryotic RNAs.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,114,676,0,0,Gene; ribozyme;,Published; PMID:9847214,767,0,,\"\",0.69906,-8.4903,1600000,418092,675,366,true,-4.0357,0.70361,2013-10-03T22:04:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00012,U3,1312,Small nucleolar RNA U3,\"Gardner PP, Marz M\",Published; PMID:19875933; Gardner PP,34.0,34.0,33.9,\"Small nucleolar RNAs (snoRNAs) are involved in the processing and modification of rRNA in the nucleolus. There are two main classes of snoRNAs: the box C/D class, and the box H/ACA class. U3 snoRNA is a member of the box C/D class. Indeed, the box C/D element is a subset of the six short sequence elements found in all U3 snoRNAs, namely boxes A, A', B, C, C', and D. The U3 snoRNA secondary structure is characterised by a small 5' domain (with boxes A and A'), and a larger 3' domain (with boxes B, C, C', and D), the two domains being linked by a single-stranded hinge. Boxes B and C form the B/C motif, which appears to be exclusive to U3 snoRNAs, and boxes C' and D form the C'/D motif. The latter is functionally similar to the C/D motifs found in other snoRNAs. The 5' domain and the hinge region act as a pre-rRNA-binding domain. The 3' domain has conserved protein-binding sites. Both the box B/C and box C'/D motifs are sufficient for nuclear retention of U3 snoRNA. The box C'/D motif is also necessary for nucleolar localisation, stability and hyper-methylation of U3 snoRNA. Both box B/C and C'/D motifs are involved in specific protein interactions and are necessary for the rRNA processing functions of U3 snoRNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,87,3924,0,0,Gene; snRNA; snoRNA; CD-box;,Published; PMID:19875933; Gardner PP,416,0,,\"\",0.59795,-9.77278,1600000,400072,326,218,true,-3.8301,0.71077,2013-10-03T22:04:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00013,6S,2461,6S / SsrS RNA,\"Bateman A, Barrick JE\",Barrick JE,44.2,44.2,44.1,\"E. coli 6S RNA was the first non coding RNA to be sequenced, but its function was unknown until recently. It consists of 184 nucleotides that fold into an extended hairpin structure with a large single-stranded internal bulge. The 6S RNA specifically associates with RNA polymerase holoenzyme containing the sigma70 specificity factor. This interaction represses expression from a sigma70-dependent promoter during stationary phase [1]. 6S RNA homologs have recently been identified in most bacterial genomes [2,3]. Many Gram-positive species have two copies of 6S RNA. In Bacillus subtilis, both copies appear to interact with RNA polymerase holoenzyme containing the housekeeping sigma factor and be expressed during different stages of growth. In many proteobacteria, 6S RNA may be processed from a transcript encoding homologs of the E. coli YgfA protein which is a putative 5-formyltetrahydrofolate cycloligase activity. Structure of 6S-RNAP regulator has been reported in 5VT0 PDB [5].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,150,3636,0,0,Gene;,Published; PMID:15811922,3414,0,,\"\",0.54245,-10.21771,1600000,317741,287,187,true,-3.6268,0.71364,2013-10-03T22:05:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00014,DsrA,1237,DsrA RNA,Bateman A,Bateman A,60.0,61.5,57.6,\"DsrA RNA regulates both transcription, by overcoming transcriptional silencing by the nucleoid-associated H-NS protein, and translation, by promoting efficient translation of the stress sigma factor, RpoS. These two activities of DsrA can be separated by mutation: the first of three stem-loops of the 85 nucleotide RNA is necessary for RpoS translation but not for anti-H-NS action, while the second stem-loop is essential for antisilencing and less critical for RpoS translation. The third stem-loop, which behaves as a transcription terminator, can be substituted by the trp transcription terminator without loss of either DsrA function. The sequence of the first stem-loop of DsrA is complementary with the upstream leader portion of RpoS messenger RNA, suggesting that pairing of DsrA with the RpoS message might be important for translational regulation.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,35,0,0,Gene; sRNA;,Published; PMID:9770508,39,0,,\"\",0.53383,-8.38474,1600000,350673,177,85,true,-3.3562,0.71888,2013-02-01T11:56:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00015,U4,1314,U4 spliceosomal RNA,\"Griffiths-Jones SR, Ontiveros-Palacios N\",\"Zwieb C, The uRNA database, PMID:9016512\",46.0,46.0,45.9,\"U4 small nuclear RNA (U4 snRNA) is a component of the major U2-dependent spliceosome. It forms a duplex with U6, and with each splicing round, it is displaced from U6 (and the spliceosome) in an ATP-dependent manner, allowing U6 to re-fold and create the active site for splicing catalysis. A recycling process involving protein Prp24 re-anneals U4 and U6. U4/U6 di-snRNA is conserved in eukaryotes and is part of the large ribonucleoprotein complex: U4 / U6.U5 tri-snRNP [5,8]. The structure of U4 snRNA from yeast[5,6,8,9,11] and human[2,7,10,12,13] has been reported. U4 snRNA present two regions where U4 and U6 form helixes (stem II and stem I share helixes) and two stem loops (5p-SL and 3p-SL), in human, is also present an extra SL in the 3p end. U4 snRNA presents a k-turn RNA motif in the 5p-SL related to recognition of Snu13 protein [8]. The structure of U4 spliceosomal RNA has been reported in PDBs: 2N7M, 3JCM, 3JCR, 5GAN, 5GAP, 5NRL, 5O9Z, 5ZWM, 5ZWO, 6AH0, 6AHD, AQW6, 6QX9 [5-13]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,177,8285,0,0,Gene; snRNA; splicing;,Published; PMID:2339054; Griffiths-Jones SR,1028,0,,\"\",0.60966,-8.50576,1600000,296541,361,138,true,-3.5407,0.71795,2013-10-03T22:05:22.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00016,SNORD14,1242,Small nucleolar RNA SNORD14,\"Griffiths-Jones SR, Mifsud W\",Griffiths-Jones SR,64.0,64.1,63.9,\"U14 small nucleolar RNA (U14 snoRNA) is required for early cleavages of eukaryotic precursor rRNAs. In yeasts, this molecule possess a stem-loop region (known as the Y-domain) which is essential for function. A similar structure, but with a different consensus sequence, is found in plants, but is absent in vertebrates [1].\",U14,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,18,1182,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,221,0,,\"\",0.63073,-3.65386,1600000,232910,229,116,true,-3.128,0.71819,2013-02-01T11:56:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00017,Metazoa_SRP,1315,Metazoan signal recognition particle RNA,Gardner PP,Published; PMID:19838050,70.0,70.0,69.9,\"The signal recognition particle (SRP) is a universally conserved ribonucleoprotein. It is involved in the co-translational targeting of proteins to membranes. The eukaryotic SRP consists of a 300-nucleotide 7S RNA and six proteins: SRPs 72, 68, 54, 19, 14, and 9. Archaeal SRP consists of a 7S RNA and homologues of the eukaryotic SRP19 and SRP54 proteins. Eukaryotic and archaeal 7S RNAs have very similar secondary structures, as represented by this family. Eight helical elements fold into the Alu and S domains, separated by a long linker region [1,2]. The Alu domain is thought to mediate the peptide chain elongation retardation function of the SRP [1]. The universally conserved helix which interacts with the SRP54 M domain mediates signal sequence recognition [2,3]. In eukaryotes and archaea, the SRP19-helix 6 complex is thought to be involved in SRP assembly and stabilises helix 8 for SRP54 binding [1]. The Signal Recognition Particle Database (SRPDB) [4] provides compilations of SRP components, with phylogenetic data and structural illustrations. Many sequences below the threshold for this family are clearly closely related, but are most likely to be SRP RNA derived pseudogenes. The human genome in particular is known to contain a large amount of SRP RNA related sequence, including Alu repeats. While the threshold of this family is set high so as to minimise the number of hits to such regions, it is likely that some remain.\",SRP_euk_arch; 7SL; CeN107,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.49 -Z 549862.597050 CM SEQDB,91,42386,0,0,Gene;,Published; PMID:19838050,402,0,,\"\",0.64536,-9.85267,1600000,488632,514,301,true,-4.0177,0.70604,2013-10-03T22:07:53.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00018,CsrB,2460,CsrB/RsmB RNA family,\"Bateman A, Gardner PP\",Bateman A,71.0,71.4,70.9,\"The CsrB RNA binds to approximately 18 copies of the CsrA protein Pfam:PF02599. CsrA proteins are found in a wide variety of bacteria. The CsrB RNAs contain a conserved motif CAGGXXG that is found in up to 18 copies and has been suggested to bind CsrA. The Csr regulatory system has a strong negative regulatory effect on glycogen biosynthesis, glyconeogenesis and glycogen catabolism and a positive regulatory effect on glycolysis [1]. In other bacteria such as Erwinia caratovara the RsmA protein has been shown to regulate the production of virulence determinants, such extracellular enzymes [2]. RsmA binds to RsmB regulatory RNA which is also a member of this family [2]. RsmB RNA was shown to be upregulated by GacS/A system, and increase downstream T3SS gene expression.[6] FlhDC, the master regulator of flagellar genes, also activates rsmB RNA production.[7] A regulatory network have been revealed connecting rsmB, FlhDC and T3SS.[8] Plays role in the biocontrol activity of Rahnella aquatilis HX2 (a biocontrol agent producing antibacterial substance).[5]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,38,254,0,0,Gene; sRNA;,Predicted; PFOLD; Bateman A; Gardner PP;,196,0,,\"\",0.69326,-9.81172,1600000,546392,555,356,true,-4.0652,0.70388,2013-10-03T23:07:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00019,Y_RNA,1317,Y RNA,\"Griffiths-Jones SR, Gardner PP\",\"Griffiths-Jones SR, Published; PMID:18087752, Gardner PP\",38.0,38.0,37.9,\"Y RNAs are components of the Ro ribonucleoprotein particle (Ro RNP), in association with Ro60 and La proteins. The Y RNAs and Ro60 and La proteins are well conserved, but the function of the Ro RNP is not known. In humans the RNA component can be one of four small RNAs -- hY1, hY3, hY4 and hY5. These small RNAs are predicted to fold into a conserved secondary structure containing three stem structures. The largest of the four, hY1, contains an additional hairpin. This family includes a significant number of annotated pseudogenes and the model is expected to discriminate poorly between real and Y RNA related pseudogenes.\",Y1; Y2; Y3; Y5,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.00 -Z 549862.597050 CM SEQDB,104,8521,0,0,Gene;,Published; PMID:10606662,123,0,,\"\",0.59183,-5.14312,1600000,189478,249,98,true,-2.8418,0.7187,2013-10-03T23:07:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00020,U5,1318,U5 spliceosomal RNA,\"Griffiths-Jones SR, Gardner PP\",\"Zwieb C, The uRNA database, PMID:9016512; PMID:18390578\",40.0,40.0,39.9,\"U5 RNA is a component of both types of known spliceosome. The precise function of this molecule is unknown, though it is known that the 5' loop is required for splice site selection and p220 binding, and that both the 3' stem-loop and the Sm site are important for Sm protein binding and cap methylation.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,180,7524,0,0,Gene; snRNA; splicing;,Published; PMID:2339054; Griffiths-Jones SR,1001,0,,\"\",0.50732,-5.54774,1600000,339349,331,116,true,-4.1327,0.7182,2013-10-03T23:08:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00021,Spot_42,1319,Spot 42 RNA,Bateman A,Bateman A,80.0,81.0,79.4,\"The physiological role of Escherichia coli Spot 42 (spf) RNA has remained obscure, even though the 109-nucleotide RNA was discovered almost three decades ago. Spot 42 seems to have a regulatory role on the galactose operon [1]. It is proposed that Spot 42 acts by an antisense mechanism where the RNA binds to the galK translation initiator region [1]. Changes in Spot 42 levels are implicated in affecting the adjacent DNA polymerase I levels [2]. The Hfq protein has been shown to interact with several small regulatory RNAs in E. coli, and the protein is required for OxyS, DsrA, RprA and Spot 42 RNA regulation [3].\",Spot42,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,189,0,0,Gene; sRNA;,Published; PMID:12101127,196,0,,\"\",0.55657,-7.408,1600000,307081,220,115,true,-3.178,0.71853,2013-10-03T23:09:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00022,GcvB,1320,GcvB RNA,\"Bateman A, Daub J\",\"Bateman A, PMID:17974919, Daub J\",42.0,45.0,41.6,\"The gcvB gene encodes a small untranslated RNA involved in expression of the dipeptide and oligopeptide transport systems in Escherichia coli. Included several unknown homologues in alignment (Bateman A pers obs). It base-pairs with SorC. This interaction triggers the degradation of GcvB by RNase E, alleviating the GcvB-mediated mRNA repression of other amino acid-related transport and metabolic genes [5].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,27,207,0,0,Gene;,Published; PMID:17974919,219,0,,\"\",0.60302,-8.70917,1600000,345337,406,198,true,-3.9895,0.71245,2013-10-03T23:09:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00023,tmRNA,1321,transfer-messenger RNA,Gardner PP,Published; PMID:19617710,47.0,47.0,46.8,\"Bacterial tmRNA, also known as 10Sa RNA or SsrA, is named for its dual tRNA and mRNA-like properties. Its role is to liberate the mRNA from a stalled ribosome. To accomplish this part of the tmRNA is used as a reading frame that ends in a translation stop signal. The broken mRNA is replaced in the ribosome by the tmRNA and translation of the tmRNA leads to addition of a proteolysis tag to the incomplete protein enabling recognition by a protease. Recently a number of permuted tmRNA genes have been found encoded in two parts. tmRNAs have been identified in eubacteria and some chloroplasts but are absent from archaeal and eukaryote nuclear genomes.\",10Sa_RNA; SsrA,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 16.62 -Z 549862.597050 CM SEQDB,477,6216,0,0,Gene;,Published; PMID:19617710,6400,0,,\"\",0.7041,-10.32425,1600000,541479,1336,354,true,-4.195,0.70396,2013-10-03T23:09:12.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00024,Telomerase-vert,1322,Vertebrate telomerase RNA,Machado Lima A,Machado Lima A,70.0,70.3,68.2,\"Telomerase is a ribonucleoprotein reverse transcriptase that synthesises telomeric DNA. Telomerase RNAs differ greatly in sequence and structure between vertebrates, ciliates and yeasts, but they share a 5' pseudoknot structure close to the template sequence. Vertebrate telomerase RNAs have a 3' H/ACA snoRNA-like domain. This family also contains viral telomerase RNAs from the Gallid herpes viruses as described in [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.69 -Z 549862.597050 CM SEQDB,37,77,0,0,Gene;,Published; PMID:10721988; Machado Lima A,85,0,,\"\",0.61989,-11.15725,1600000,585785,762,436,true,-4.7564,0.70196,2013-10-03T23:10:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00025,Telomerase-cil,1322,Ciliate telomerase RNA,\"Griffiths-Jones SR, Moxon SJ, Ontiveros-Palacios N\",\"McCormick-Graham and Romero, PMID:7739888; Griffiths-Jones SR, Moxon SJ\",52.0,52.2,42.2,\"Telomerase is a ribonucleoprotein reverse transcriptase that synthesises telomeric DNA. Telomerase RNAs differ greatly in sequence and structure between vertebrates, ciliates and yeasts, but they share a 5' pseudoknot structure close to the template sequence [1,2]. The structure of Tetrahymena telomerase has been reported in 6D6V, 7LMA and 7LMB [3,4].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,21,3,0,0,Gene;,\"Published; PMID:7958872; PMID:7739888, Moxon SJ\",21,0,,\"\",0.48263,-8.79918,1600000,384182,456,165,true,-3.1554,0.71625,2013-10-03T23:10:27.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00026,U6,1323,U6 spliceosomal RNA,\"Griffiths-Jones SR, Gardner PP\",Griffiths-Jones SR; ; PMID:18390578,44.0,44.0,43.9,\"U6 snRNA is a component of the spliceosome which is involved in splicing pre-mRNA. The putative secondary structure consensus base pairing is confined to a short 5' stem loop, but U6 snRNA is thought to form extensive base-pair interactions with U4 snRNA [2]. This model hits a large number of sequences in vertebrate genomes, which appear to be U6 derived pseudogenes or repeats. This family has a high significance threshold to minimise this problem, but many annotated pseudogenes remain.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,188,47401,0,0,Gene; snRNA; splicing;,Published; PMID:2339054; Griffiths-Jones SR,1087,0,,\"\",0.66804,-5.60435,1600000,269485,820,104,true,-4.0987,0.71824,2013-10-03T23:10:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00027,let-7,1324,let-7 microRNA precursor,\"Griffiths-Jones SR, Daub J\",Griffiths-Jones SR,52.6,52.6,52.3,\"This family represents the microRNAs (miRNA) let-7 and mir-98. let-7 was identified from a study of developmental timing in C. elegans [1], and was later shown to be part of a much larger class of non-coding RNAs termed miRNAs [2]. Both miRNAs let-7 and mir-98 have now been predicted or experimentally confirmed in a wide range of species (MIPF:MIPF000002). miRNAs are transcribed as ~70nt precursors (modelled here) and subsequently processed by the Dicer enzyme to give a ~22nt product. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. This family also includes the miR-98 precursor from human, which may be a let-7 homologue. The pre-miRNA structure of let-7 has been reported in complex with Dicer-TRBP [3]. Family represents miRNA precursor let-7 imported from miRBase [4].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,64,1415,0,0,Gene; miRNA;,Predicted; RNAalifold,192,0,,\"\",0.61407,-3.97151,1600000,191950,161,84,true,-3.7442,0.71891,2013-10-03T23:14:55.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00028,Intron_gpI,1325,Group I catalytic intron,Griffiths-Jones SR,Eddy SR,50.0,50.0,49.9,\"Group I catalytic introns are large self-splicing ribozymes. They catalyse their own excision from mRNA, tRNA and rRNA precursors in a wide range of organisms. The core secondary structure consists of 9 paired regions (P1-P9). These fold to essentially two domains - the P4-P6 domain (formed from the stacking of P5, P4, P6 and P6a helices) and the P3-P9 domain (formed from the P8, P3, P7 and P9 helices). The secondary structure mark-up for this family represents only this conserved core. Group I catalytic introns often have long ORFs inserted in loop regions. The window size for this search is chosen to minimise search time and catch the small examples. A larger window size will find more matches.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,2599,0,0,Intron;,Published; PMID:2197983,1143,0,,\"\",0.66222,-7.41638,1600000,324432,1563,251,true,-3.8811,0.7083,2013-10-03T23:15:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00029,Intron_gpII,1326,Group II catalytic intron,Griffiths-Jones SR,PMID:11726522,40.0,40.0,39.9,\"Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI. This model and alignment represents only domains V and VI. A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron-intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.00 -Z 549862.597050 CM SEQDB,92,15637,0,0,Intron;,Predicted; PFOLD,1847,0,,\"\",0.5652,-7.49312,1600000,262242,581,93,true,-3.5766,0.71847,2013-10-03T23:17:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00030,RNase_MRP,1327,RNase MRP,Griffiths-Jones SR,\"Published; PMID:12088147, Griffiths-Jones SR, Daub J\",50.0,50.0,49.9,\"RNase MRP is an enzymatically active ribonucleoprotein with two distinct roles in eukaryotes. In mitochondria it plays a direct role in the initiation of mitochondrial DNA replication. In the nucleus it is involved in precursor rRNA processing, where it cleaves the internal transcribed spacer 1 between 18S and 5.8S rRNAs. Despite distinct functions, RNase MRP has been shown to be evolutionarily related to RNase P. Like eukaryotic RNase P, RNase MRP is not catalytically active without associated protein subunits.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,66,1062,0,0,Gene; ribozyme;,Published; PMID:7678563,882,0,,\"\",0.68784,-9.60404,1600000,385002,2236,266,true,-4.5376,0.70748,2013-10-03T23:18:26.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00031,SECIS_1,1328,Selenocysteine insertion sequence 1,Griffiths-Jones SR,\"Gautheret D, PMID:12458087\",35.0,35.0,34.9,The incorporation of selenocysteine into a protein sequence is directed by an in-frame UGA codon (usually a stop codon) within the coding region of the mRNA. Selenoprotein mRNAs contain a conserved secondary structure in the 3' UTR that is required for the distinction of UGA stop from UGA selenocysteine. The selenocysteine insertion sequence (SECIS) is around 60 nt in length and adopts a hairpin structure which is sufficiently well-defined and conserved to act as a computational screen for selenoprotein genes [2].,SECIS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,61,611,0,0,Cis-reg;,Published; PMID:12458087,167,0,,\"\",0.61389,-3.14575,1600000,235124,123,64,true,-2.4006,0.74522,2013-10-03T23:18:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00032,Histone3,1329,Histone 3' UTR stem-loop,Gardner PP,Published; PMID:17998288,25.0,25.0,24.9,\"The mRNAs of metazoan histone genes lack a poly-A tail. 3' end processing occurs at a site between a highly conserved stem-loop (modelled by this family) and a purine rich region around 20 nts downstream (the histone downstream element, or HDE). The stem-loop is bound by a 31 kDa stem-loop binding protein (SLBP - also termed the histone hairpin binding protein, or HBP). Together with U7 snRNA binding of the HDE, SLBP binding nucleate s the formation of the processing complex. The histone 3' UTR stem-loop is also involved in nucleocytoplasmic transport of the mRNA, and in stability regulation and translation efficiency in the cytoplasm.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.68 -Z 549862.597050 CM SEQDB,46,30457,0,0,Cis-reg;,Published; PMID:17998288,813,0,,\"\",0.86412,-1.32523,1600000,239617,102,46,true,-3.2029,0.74543,2013-10-03T23:18:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00033,MicF,1330,MicF RNA,Bateman A,Noncoding RNAs Database,79.0,79.2,70.1,\"The micF gene is a stress response gene found in Escherichia coli and related bacteria that post-transcriptionally controls expression of the outer membrane porin gene ompF. The micF gene encodes a non-translated 93 nt antisense RNA that binds its target ompF mRNA and regulates ompF expression by inhibiting translation and inducing degradation of the message. In addition, other factors, such as the RNA chaperone protein StpA also play a role in this regulatory system. Expression of micF is controlled by both environmental and internal stress factors. Four transcriptional regulators are known to bind the micF promoter region and activate micF expression.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,38,0,0,Gene; antisense;,Predicted; Bateman A,42,0,,\"\",0.45941,-8.02453,1600000,321412,189,94,true,-3.3438,0.71847,2013-10-03T23:19:41.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00034,RprA,1331,RprA RNA,\"Bateman A, Moxon SJ\",Noncoding RNAs Database,70.0,70.2,69.5,\"Translational regulation of the stationary phase sigma factor RpoS is mediated by the formation of a double-stranded RNA stem-loop structure in the upstream region of the rpoS messenger RNA, occluding the translation initiation site. Clones carrying rprA (RpoS regulator RNA A) increased the translation of RpoS. The rprA gene encodes a 106 nucleotide regulatory RNA. As with DsrA Rfam:RF00014, RprA is predicted to form three stem-loops. Thus, at least two small RNAs, DsrA and RprA, participate in the positive regulation of RpoS translation. RprA also appears to bind to the RpoS leader [2]. RprA is non-essential [1]. The Y. pestis homologue was detected by similarity to the E. coli gene (pers obs. Bateman A).\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 36 -Z 549862.597050 CM SEQDB,13,85,0,0,Gene; sRNA;,Predicted; PFOLD; Moxon SJ,93,0,,\"\",0.44789,-5.03711,1600000,323894,201,108,true,-3.2161,0.71836,2013-10-03T23:19:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00035,OxyS,1332,OxyS RNA,Bateman A,Noncoding RNAs Database,75.0,75.1,74.9,\"OxyS is a small untranslated RNA which is induced in response to oxidative stress in Escherichia coli. This novel RNA acts as a global regulator to activate or repress the expression of as many as 40 genes, including the fhlA-encoded transcriptional activator and the rpoS-encoded sigma(s) subunit of RNA polymerase [1]. OxyS is bound by the Hfq protein, that increases the OxyS RNA interaction with its target messages [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,12,0,0,Gene; sRNA;,Published; PMID:11804582,15,0,,\"\",0.5664,-9.09368,1600000,391780,215,110,true,-3.5383,0.71852,2013-10-03T23:19:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00036,RRE,1333,HIV Rev response element,Griffiths-Jones SR,Griffiths-Jones SR,104.0,104.0,103.9,\"The Rev response element (RRE) is encoded within the HIV-env gene. Rev is an essential regulatory protein of HIV that binds an internal loop of the RRE leading, encouraging further Rev-RRE binding. This RNP complex is critical for mRNA export and hence for expression of the HIV structural proteins.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,65,20,0,0,Cis-reg;,Published; PMID:12177299,13,0,,\"\",0.733,-8.78817,1600000,427253,500,337,true,-3.7413,0.70454,2013-10-03T23:19:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00037,IRE_I,1334,Iron response element I,\"Stevens S, Gardner P, Brown C\",\"Leipuviene, R. and Theil, E.C.\",34.0,34.0,33.9,Iron respond element I 3D structure was reported in PDB 3SNP [6],IRE I; IRE; ferritin,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,62,1847,0,0,Cis-reg;,Published; PMID:17849083,107,0,,\"\",0.68512,-3.34985,1600000,315020,88,36,true,-4.2931,0.72684,2013-10-03T23:20:45.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00038,PrfA,1335,PrfA thermoregulator UTR,Bateman A,Bateman A,88.0,117.4,47.4,\"In Listeria monocytogenes, virulence genes are maximally expressed at 37 degrees C, almost silent at 30 degrees C and controlled by PrfA, a transcriptional activator whose expression is thermoregulated. It has been shown that the untranslated mRNA (UTR) preceding prfA, forms a secondary structure, which masks the ribosome binding region [1]. This family also contains UTRs from other closely related bacteria. The secondary structure of this motif forms a hairpin. The Shine-Dalgarno (SD) sequence is partly masked by the most conserved stem of the secondary structure. It is thought that at 37 degrees C the SD sequence is unmasked [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,1,0,0,Cis-reg; thermoregulator;,Predicted; PFOLD,4,0,,\"\",0.47515,-6.50079,1600000,361291,242,132,true,-2.6097,0.71815,2013-10-03T23:21:00.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00039,DicF,1336,DicF RNA,Bateman A,INFERNAL,48.0,49.5,46.0,\"DicF RNA is an antisense inhibitor of cell division gene ftsZ [1]. DicF and Hfq are necessary to reduce FtsZ protein levels, leading to cell filamentation under anaerobic conditions.[4]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,5,3,0,0,Gene; antisense;,Predicted; PFOLD,6,0,,\"\",0.63101,-6.70909,1600000,454072,127,52,true,-3.4051,0.72304,2013-10-03T23:21:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00040,rne5,1337,RNase E 5' UTR element,Griffiths-Jones SR,\"Griffiths-Jones SR, Moxon SJ\",180.0,182.8,170.7,This family represents a conserved element found in the 5' UTR of RNase E mRNA. RNase E is a key regulatory enzyme in the pathway of mRNA degradation in E. coli that is able to auto-regulate the degradation of its own mRNA in response to changes in RNase E activity. This rne 5' UTR element acts as a sensor of cellular RNase E concentration enabling tight regulation of RNase E concentration and synthesis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,105,0,0,Cis-reg;,Published; PMID:10817759,108,0,,\"\",0.74731,-8.28297,1600000,488498,1416,334,true,-4.2339,0.70468,2013-10-03T23:21:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00041,Entero_OriR,1338,Enteroviral 3' UTR element,\"Bateman A, Moxon SJ\",PMID:9862969,80.0,83.2,78.4,This RNA structure is found in the 3' UTR of various enteroviruses. The overall structure forms the origin of replication (OriR) for the initiation of (-) strand RNA synthesis [1]. Pseudoknots have also been predicted in this structure [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,60,0,0,0,Cis-reg;,Predicted; PFOLD,43,0,,\"\",0.55807,-8.24713,1600000,333049,227,121,true,-2.9029,0.7184,2013-10-03T23:21:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00042,CopA,1339,CopA-like RNA,Bateman A,Bateman A,60.0,66.3,57.5,\"In several groups of bacterial plasmids, antisense RNAs regulate copy number through inhibition of replication initiator protein synthesis. These RNAs are characterised by a long hairpin structure interrupted by several unpaired residues or bulged loops. In plasmid R1, the inhibitory complex between the antisense RNA (CopA) and its target mRNA (CopT) is characterised by a four-way junction structure and a side-by-side helical alignment [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.46 -Z 549862.597050 CM SEQDB,37,26,0,0,Gene; antisense;,Predicted; PFOLD; Moxon SJ,41,0,,\"\",0.54677,-5.96158,1600000,251815,183,92,true,-3.6385,0.71884,2013-10-03T23:21:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00043,Plasmid_R1162,1340,R1162-like plasmid antisense RNA,Bateman A,\"Bateman A, INFERNAL\",55.0,65.1,52.4,\"The copy-number of the broad host-range plasmid R1162 is controlled by the amounts of two proteins, encoded by cotranscribed genes comprising a region of the plasmid called RepI. RepI is negatively regulated by a 75 base RNA that is complementary to a segment of the RepI message [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,3,0,0,Gene; antisense;,Predicted; PFOLD,10,0,,\"\",0.50825,-8.28196,1600000,299438,159,75,true,-3.5644,0.71909,2013-10-03T23:21:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00044,Phage_pRNA,1341,Bacteriophage pRNA,Finn RD,Finn RD,40.0,61.9,35.9,\"During replication of linear dsDNA viruses, the viral genome is packaged into the pre-formed viral procapsid. The packaging of DNA into the procapsid requires a molecular motor which using ATP as energy to accomplish the energetically unfavourable motion. In some bacteriophage, an RNA (pRNA) molecule is a vital component of this motor [1]. Structural analyses of the packaging motor have demonstrated that the pRNA molecule has fivefold symmetry when attached to the prohead [2-4]. The pRNA is thought to be bound by the capsid connector protein (PFAM:PF05352) [4]. Only the first 120 bases of the pRNA are essential for packing the viral DNA [5,6]. The pRNA is proposed to be comprised of two domains, one corresponding to the first 120 bases and the second to the remaining 50 bases [6]. Nuclear cleavage occurs in the single strand region linking these two domains [6].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene;,Predicted; ILM,3,0,,\"\",0.67619,-8.8502,1600000,385188,387,244,true,-3.7859,0.70874,2013-10-03T23:21:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00045,SNORA73,1342,Small nucleolar RNA SNORA73 family,Griffiths-Jones SR,\"Griffiths-Jones SR; Cervelli et al, PMID:11821910\",74.0,74.2,73.9,\"U17 (SNORA73A and SNORA73B) also known as E1 belongs to the H/ACA class of small nucleolar RNAs (snoRNAs). Vertebrate U17 is intron-encoded and ranges in length from 200-230 nucleotides, longer than most snoRNAs. It is one of the most abundant snoRNAs in human cells and is essential for the cleavage of pre-rRNA within the 5' external transcribed spacer (ETS) [1]. This cleavage leads to the formation of 18S rRNA. Regions of the U17 RNA are complementary to rRNA and act as guides for RNA/RNA interactions, although these regions do not seem to be well conserved between organisms [2].\",U17,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,66,513,0,0,Gene; snRNA; snoRNA; HACA-box;,Published; PMID:11821910,134,0,,\"\",0.64967,-8.63062,1600000,295629,339,219,true,-3.9221,0.71063,2013-10-03T23:21:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00046,snoR30,2372,Small nucleolar RNA R30/Z108,Griffiths-Jones SR,Griffiths-Jones SR,60.0,61.3,58.9,snoR30 is a C/D box small nucleolar RNA which acts as a methylation guide for 18S ribosomal RNA in plants.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,219,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,75,0,,\"\",0.64613,-4.23625,1600000,237238,193,96,true,-3.5271,0.71839,2013-10-03T23:21:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00047,mir-2,1344,mir-2 microRNA precursor,\"Griffiths-Jones SR, Daub J\",Griffiths-Jones SR,45.5,45.5,45.4,\"This family represents the microRNA (miRNA) precursors mir-2 and mir-13 (MIPF:MIPF0000049). mir-2 has now been predicted or experimentally confirmed in a wide range of species. The miRNAs are transcribed as ~70nt precursors (modelled here) and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 3' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the mature sequence. This family also contains the mir-13 precursor which, despite giving rise to a distinct mature sequence appears to be related in sequence.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,56,434,0,0,Gene; miRNA;,Predicted; PFOLD,134,0,,\"\",0.58241,-4.47985,1600000,172753,145,72,true,-3.8374,0.71926,2013-10-03T23:21:33.000Z,2021-12-03T12:41:14.000Z\n",
            "RF00048,Entero_CRE,1238,Enterovirus cis-acting replication element,Bateman A,Bateman A,55.0,55.0,49.7,\"The first step in the replication of the plus-stranded poliovirus RNA is the synthesis of a complementary minus strand. This process is initiated by the covalent attachment of UMP to the terminal protein VPg, yielding VPgpU and VPgpUpU. A small RNA hairpin in the coding region of protein 2C as the site in PV1(M) RNA that is used as the primary template for the in vitro uridylylation [1-2].\",CRE,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,56,5,0,0,Cis-reg;,Published; PMID:11044080,32,0,,\"\",0.61542,-2.56685,1600000,312886,139,61,true,-3.9834,0.71915,2013-02-01T11:56:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00049,SNORD36,1243,Small nucleolar RNA SNORD36,Griffiths-Jones SR,Griffiths-Jones SR,37.0,37.0,36.9,\"U36 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U36 is encoded within the intron of rpL7a, and has two regions of complementarity to 18S and 28S ribosomal RNA. This complementarity suggests that U36 acts as a 2'-O-ribose methylation guide [1,2]. This family also contains snoR47, which has been shown to be homologous to U36 [2], and plant Z100 snoRNA.\",U36,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,59,839,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,293,0,,\"\",0.51485,-5.04282,1600000,247599,148,77,true,-3.4119,0.71863,2013-02-01T11:56:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00050,FMN,1345,FMN riboswitch (RFN element),\"Vitreshchak A, Gelfand M, Bateman A, Ontiveros-Palacios N\",Vitreshchak A,66.2,66.3,66.1,\"The RFN element is a highly conserved domain that is found frequently in the 5'-untranslated regions of prokaryotic mRNAs that encode for flavin mononucleotide (FMN) biosynthesis and transport proteins. This element is a metabolite-dependent riboswitch that directly binds FMN in the absence of proteins. In Bacillus subtilis, the riboswitch most likely controls gene expression by causing premature transcription termination within the 5' untranslated region of the ribDEAHT operon and precluding access to the ribosome-binding site of ypaA mRNA [3].The FMN riboswitch structure, reported in [4] present an FMN-bound six-stem junction, adopts a butterfly-like scaffold, stapled together by opposingly directed but nearly identically folded peripheral domains. The FMN riboswitch structure was reported in PDB: 3F2Q, 3F2T, 3F2W, 3F2X, 3F2Y, 3F30 [4] 6WJR, 6WJS [5]\",RFN,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,146,4077,0,0,Cis-reg; riboswitch;,Published; PMID:12456892,3376,0,,\"\",0.62775,-7.51726,1600000,330721,273,138,true,-3.3535,0.71809,2013-10-03T23:21:58.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00051,mir-17,1346,mir-17 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,48.3,48.4,48.1,\"This family represents the microRNA (miRNA) precursor mir-17, family, which includes mir-20, mir-91, and mir-103. miRNAs are transcribed as ~70nt precursors (modelled here) and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 3' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA. This family represents the microRNA (miRNA) precursor mir-17 imported from miRBase.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,233,770,0,0,Gene; miRNA;,Predicted; RNAalifold,124,0,,\"\",0.63775,-4.57068,1600000,192081,154,80,true,-4.0397,0.71875,2013-10-03T23:22:06.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00052,lin-4,1347,lin-4 microRNA precursor,Griffiths-Jones SR,Griffiths-Jones SR,40.0,41.4,37.9,\"This family represents the microRNA (miRNA) lin-4 (MIPF:MIPF0000303). lin-4 was identified from a study of developmental timing in C. elegans [1], and was later shown to be part of a much larger class of non-coding RNAs termed miRNAs [2]. miRNAs are transcribed as ~70nt precursors (modelled here) and subsequently processed by the Dicer enzyme to give a ~22nt product. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,24,0,0,Gene; miRNA;,Predicted; RNAalifold,22,0,,\"\",0.5799,-5.97038,1600000,267917,166,80,true,-3.2334,0.71887,2013-10-03T23:22:10.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00053,mir-7,1348,mir-7 microRNA precursor,\"Griffiths-Jones SR, Daub J\",Griffiths-Jones SR,51.8,52.2,51.4,This family represents the microRNA (miRNA) precursor mir-7. This miRNA has been predicted or experimentally confirmed in a wide range of species (MIPF:MIPF0000022). miRNAs are transcribed as ~70nt precursors (modelled here) and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 5' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,57,468,0,0,Gene; miRNA;,Predicted; PFOLD,224,0,,\"\",0.64723,-2.62721,1600000,145416,170,88,true,-4.0428,0.71876,2013-10-03T23:22:15.000Z,2022-05-06T13:10:02.000Z\n",
            "RF00054,SNORD25,1349,Small nucleolar RNA SNORD25,Griffiths-Jones SR,Griffiths-Jones SR,39.0,40.0,38.9,\"U25 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U25 is found in gene clusters in plants [1], and within the U22 snoRNA host gene (UHG) in mammals [2]. U25 is thought to as a 2'-O-ribose methylation guide for ribosomal RNA. This RNA has also been named snoRNA R73 in some plants.\",U25,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,374,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,82,0,,\"\",0.63487,-3.9721,1600000,200378,172,87,true,-3.5679,0.71854,2013-10-03T23:22:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00055,SNORD96,1350,Small nucleolar RNA SNORD96 family,\"Griffiths-Jones SR, Daub J\",\"Griffiths-Jones SR, Daub J\",43.0,43.5,42.5,\"snoRNA Z37 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Z37 acts as a methylation guide for 5.8S ribosomal RNA. This family contains a putative snoRNA found in the intron of the receptor for activated C kinase (RACK1) gene in mammals (S. Griffiths-Jones, pers. obs.). This family also includes human snoRNAs U96a (SNORD96A) and U96b (SNORD96B) and the apicomplexan snoRNA snr39b.\",snoZ37,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,9,388,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,111,0,,\"\",0.60444,-3.62034,1600000,251816,246,85,true,-3.3588,0.7185,2013-10-03T23:22:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00056,SNORA71,1351,Small nucleolar RNA SNORA71,\"Griffiths-Jones SR, Daub J\",\"Griffiths-Jones SR, INFERNAL\",60.0,60.4,59.8,\"The members of this family belong to the H/ACA class of snoRNAs which bind a number of proteins (including dyskerin, Gar1p and Nop10p) to form snoRNP complexes. This class of snoRNAs are thought to guide the sites of modification of uridines to pseudouridines by forming direct base pairing interactions with substrate RNAs [2]. This family includes U71 (SNORA71), ACA39 (SNORA39) and ACA60 (SNORA60) [1,3]. U71 and ACA60 are predicted to guide the pseudouridylation of U406 and U1004 in 18S rRNA respectively [4]. There is currently no target RNA identified for ACA39 [3].\",U71,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,40,384,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,99,0,,\"\",0.54703,-7.95765,1600000,238816,238,134,true,-3.1072,0.71833,2013-10-03T23:22:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00058,HgcF,1353,HgcF RNA (Pab35),Bateman A,Bateman A,70.0,120.3,34.7,\"The HgcF ncRNA gene was identified computationally in AT-rich hyperthermophiles. The genes were named hgcA through hgcG (\\\"high GC\\\"). It was also identified as Pab35 H/ACA snoRNA with rRNA targets.[2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,11,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,14,0,,\"\",0.6077,-9.37117,1600000,345853,292,168,true,-3.178,0.71597,2013-10-03T23:22:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00059,TPP,1354,TPP riboswitch (THI element),\"Rodionov DA, Vitreshchak AG, Mironov AA, Gelfand MS, Bateman A, Moxon SJ, Ontiveros-Palacios N\",Vitreshchak A,54.3,54.3,54.2,\"Vitamin B(1) in its active form thiamin pyrophosphate (TPP) is an essential coenzyme that is synthesised by coupling of pyrimidine and thiazole moieties in bacteria. The previously detected thiamin-regulatory element, thi box [1] was extended, resulting in a new, highly conserved RNA secondary structure, the THI element, which is widely distributed in eubacteria and also occurs in some archaea. Analysis of operon structures identified a large number of new candidate thiamin-regulated genes, mostly transporters, in various prokaryotic organisms [2]. The THI element is a riboswitch [3] that directly binds to TPP to regulate gene expression through a variety of mechanisms in archaea, bacteria and eukaryotes [4,5]. through a variety of mechanisms in archaea, bacteria and eukaryotes [4,5]. Tandem TPP riboswitches have been reported [6] where both TPP riboswitches respond independently to TPP. The structure of TPP riboswitch has been reported for Escherichia coli and Arabidopsis thaliana [7-12].\",THI,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,115,10091,0,0,Cis-reg; riboswitch;,Published; PMID:12376536;,5552,0,,\"\",0.56597,-6.92772,1600000,241882,264,105,true,-4.1217,0.71844,2013-10-03T23:22:43.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00060,HgcE,1355,HgcE RNA (Pab105),Bateman A,Bateman A,50.0,66.2,34.0,\"The HgcE ncRNA gene was identified computationally in AT-rich hyperthermophiles. The genes were named hgcA through hgcG (\\\"high GC\\\").It was also identified as a part of Pab105 H/ACA snoRNA with rRNA targets. [2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.16 -Z 549862.597050 CM SEQDB,4,7,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,10,0,,\"\",0.58901,-8.04642,1600000,320239,240,130,true,-2.9658,0.7182,2013-10-03T23:23:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00061,IRES_HCV,1356,Hepatitis C virus internal ribosome entry site,\"Lamkiewicz K, Triebel S, Marz M\",\"Kevin Lamkiewicz, Sandra Triebel, Manja Marz\",290.0,298.2,78.3,Genetic modification of RNA structures in Hepatitis C virus (HCV) genome affects replication and infectivity [1] Long distant RNA-RNA interactions in HCV RNAs are involved in translation to virion budding [2] It has been proposed that the 5'-3' UTR base-pairing in the HCV genome might play an important role in viral RNA replication [3],HCV_IRES,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,51,4,0,0,Cis-reg; IRES;,\"Predicted; BM VeGETA v.0.4, LocARNA 2.0.0RC8, RNAfold ViennaRNA v.2.4.18\",22,0,,\"\",0.74552,-8.2937,1600000,368657,514,352,true,-4.0744,0.70409,2013-10-03T23:23:19.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00062,HgcC,1357,HgcC family RNA,Bateman A,Bateman A,50.0,77.5,48.6,\"The HgcC ncRNA gene was identified computationally in AT-rich hyperthermophiles. The genes were named hgcA through hgcG (\\\"high GC\\\"). HgcC is of unknown function. Two other homologues were detected called HhcA and HhcB after \\\"homologue of hgcC\\\" [1]. This family contains a large number of putative archaeal matches to the 3' end of the known ncRNAs. Many of these partly overlap coding regions.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,13,0,0,Gene;,Predicted; RNAalifold,7,0,,\"\",0.62167,-8.06393,1600000,285117,619,130,true,-3.0923,0.71835,2013-10-03T23:23:31.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00063,SscA,1358,SscA RNA,Bateman A,Bateman A,60.0,64.8,37.1,The SscA ncRNA gene was identified computationally in AT-rich hyperthermophiles using QRNA [1]. SscA stands for Secondary Structure Conserved A. SscA is 97 nucleotides in length and is of unknown function. This RNA is currently restricted to pyrococcal species.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,11,0,0,Gene;,Predicted; PFOLD,15,0,,\"\",0.5262,-5.77538,1600000,234277,191,96,true,-3.7645,0.71866,2013-10-03T23:23:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00064,HgcG,1359,HgcG RNA (Pab40),Bateman A,Bateman A,77.0,77.9,66.4,\"The HgcG ncRNA gene was identified computationally in AT-rich hyperthermophiles. The genes were named hgcA through hgcG (\\\"high GC\\\") [1]. HgcG is of unknown function. hgcG is significantly similar to a region of the Archaeoglobus fulgidus genome. In a recent experimental screen for ncRNAs in A. fulgidus, this locus was identified as the second-most abundant transcript (Afu-4) (T.-H. Tang, J.-P. Bachellerie, H. Huber, M. Drungowski, T. Elge, J. Brosius, and A. Hüttenhofer, personal communication in [1]).It was also identified as Pab40 H/ACA snoRNA with rRNA targets.[2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.62 -Z 742849.287494 --mxsize 128 CM SEQDB,5,14,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,19,0,,\"\",0.71022,-8.81822,1600000,416758,457,276,true,-3.8246,0.70707,2013-10-03T23:23:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00065,snoR9,1360,Small nucleolar RNA snoR9,\"Bateman A, Daub J\",Bateman A,90.0,98.0,52.6,snoRNA R9 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. R9 was identified in a computational screen in AT-rich hyperthermophiles [1]. R9 was found to overlap with the smaller snoRNA R19 which is currently a member of Pyrococcus C/D box snoRNA family Rfam:RF00095.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 CM SEQDB,5,6,0,0,Gene; snRNA; snoRNA; CD-box;,Published; PMID:12032319,11,0,,\"\",0.6257,-8.28269,1600000,349210,238,128,true,-3.7822,0.71819,2013-10-03T23:23:49.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00066,U7,1361,U7 small nuclear RNA,\"Griffiths-Jones SR, Marz M\",Griffiths-Jones SR; Published; PMID:18267300,34.0,34.0,33.9,\"Animal histone pre-mRNAs are spliced by a different mechanism than nuclear pre-mRNAs and self-splicing introns. This processing involves the U7 snRNP, which contains only one RNA species: U7. The 5' end of U7 RNA is thought to base-pair to a conserved spacer element downstream of the cleavage site in histone pre-mRNA. This family contains a number of annotated U7 pseudogenes.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.34 -Z 549862.597050 CM SEQDB,49,2931,0,0,Gene; snRNA;,Predicted; PFOLD; Griffiths-Jones SR,115,0,,\"\",0.6122,-7.02458,1600000,280347,123,63,true,-4.1737,0.71894,2013-10-03T23:23:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00067,SNORD15,1362,Small nucleolar RNA SNORD15,Griffiths-Jones SR,\"Griffiths-Jones SR; Plant snoRNA database, PMID:12520043\",75.0,75.0,74.7,\"U15 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U15 is encoded within the introns of ribosomal protein S1 in Xenopus and S3 in human and Fugu. snoR75 from Arabidopsis [3] and homologues in rice and other plants are alternatively known as U15, and, despite a significantly shorter sequence length, appear to be related [4].\",U15,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.42 -Z 549862.597050 CM SEQDB,11,276,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Griffiths-Jones SR, Daub J\",106,0,,\"\",0.58151,-10.37617,1600000,560148,254,145,true,-2.9131,0.71789,2013-10-03T23:24:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00068,SNORD21,1363,Small nucleolar RNA SNORD21,Griffiths-Jones SR,Griffiths-Jones SR,63.0,64.0,62.5,\"U21 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U21 is encoded within an intron of the gene for ribosomal protein L5 in mammals [1], but within introns of the ADP ribosylation factor gene in Drosophila [2]. U21 snoRNA has a 13 nt region of complementarity with an invariant region of eukaryotic 28S ribosomal RNA.\",U21,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,98,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Griffiths-Jones SR, Daub J\",93,0,,\"\",0.57011,-4.12754,1600000,229533,187,95,true,-3.3965,0.71843,2013-10-03T23:24:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00069,SNORD24,1364,Small nucleolar RNA SNORD24,Griffiths-Jones SR,Griffiths-Jones SR,41.0,41.2,40.8,\"U24 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs [1]. C/D box snoRNAs have been shown to act as methylation guides for a number of RNA targets [2,3]. U24 is encoded within an intron of the gene for ribosomal protein L7a in mammals, chicken and Fugu [4]. The U76 snoRNA is found in an intron of the uRNA host gene (UHG) growth arrest specific 5 (GAS5) transcript gene [5]. snoRNAs Z20 and U76 snoRNAs show clear similarity to U24.\",U24,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,781,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Griffiths-Jones SR, Daub J\",369,0,,\"\",0.5542,-4.28849,1600000,254857,190,76,true,-3.3062,0.71866,2013-10-03T23:24:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00070,SNORD29,1365,Small nucleolar RNA SNORD29,\"Moxon SJ, Griffiths-Jones SR\",Moxon SJ; Griffiths-Jones SR,48.0,49.1,47.7,U29 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U29,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,10,114,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",72,0,,\"\",0.60287,-4.78751,1600000,264626,224,65,true,-3.7584,0.71889,2013-10-03T23:24:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00071,SNORD73,1366,Small nucleolar RNA SNORD73,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",42.0,43.5,41.1,U73 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U73 is encoded within the introns of ribosomal protein S3a and contains a 12nt region of complementarity to a conserved sequence in 28S ribosomal RNA. U73 is therefore thought to function as a 2'-O-ribose methylation guide for pre-rRNA.,U73,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,153,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Griffiths-Jones SR,110,0,,\"\",0.56324,-4.06014,1600000,282081,189,72,true,-3.4847,0.71874,2013-10-03T23:24:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00072,SNORA75,1367,Small nucleolar RNA SNORA75,\"Bachellerie JP, Griffiths-Jones SR\",Griffiths-Jones SR,80.0,80.1,78.0,\"U23 belongs to the H/ACA class of snoRNAs. snoRNAs bind a number of proteins (including dyskerin, Gar1p and Nop10p in the case of the H/ACA class) to form snoRNP complexes. This class are thought to guide the sites of modification of uridines to pseudouridines by forming direct base pairing interactions with substrate RNAs [3]. Targets include ribosomal and spliceosomal RNAs as well as the Trypanosoma spliced leader RNA (SL RNA) as possibly other, still unknown cellular RNAs [3]. U23 can direct the pseudouridylation of U97 in human 18S rRNA (Bachellerie JP, Ganot, P unpublished data). U23 is encoded within intron 12 of the nucleolin gene in human, mouse, rat chicken, and Xenopus laevis.\",U23,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,100,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Griffiths-Jones SR,76,0,,\"\",0.4775,-7.89248,1600000,283090,247,134,true,-3.0275,0.71824,2013-10-03T23:25:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00073,mir-156,1368,mir-156 microRNA precursor,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.1,61.9,\"This family represents the plant microRNA (miRNA) precursor mir-156. This miRNA has now been predicted or experimentally confirmed in a range of plant species (MIPF:MIPF0000008). Animal miRNAs are transcribed as ~70nt precursors and subsequently processed by the Dicer enzyme to give a ~22nt product. In plants the precursor sequences may be longer, and the carpel factory (caf) enzyme appears to be involved in processing. In this case the mature sequence comes from the 5' arm of the precursor, and both Arabidopsis and rice genomes contain a number of related miRNA precursors which give rise to almost identical mature sequences. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,476,0,0,Gene; miRNA;,Predicted; PFOLD,76,0,,\"\",0.62553,-4.17853,1600000,244917,190,88,true,-3.2776,0.71887,2013-10-03T23:25:08.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00074,mir-29,1369,mir-29 microRNA precursor,\"Griffiths-Jones SR, Daub J\",INFERNAL,46.7,46.7,46.6,This family represents the microRNA (miRNA) precursors mir-29. This miRNA has been predicted or confirmed in a wide range of species (MIPF:MIPF0000009). Animal miRNAs are transcribed as ~70nt precursors and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 3' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,10,523,0,0,Gene; miRNA;,Predicted; PFOLD,132,0,,\"\",0.62708,-3.8812,1600000,221064,151,71,true,-3.7753,0.71913,2013-10-03T23:25:24.000Z,2022-05-06T12:43:50.000Z\n",
            "RF00075,mir-166,1370,mir-166 microRNA precursor,\"Griffiths-Jones SR, Daub J\",\"Griffiths-Jones SR, Daub J\",60.0,60.4,59.8,\"This family represents the plant microRNA (miRNA) precursor mir-166. This miRNA has now been predicted or experimentally confirmed in a wide range of plant species (MIPF:MIPF0000004). Animal miRNAs are transcribed as ~70nt precursors and subsequently processed by the Dicer enzyme to give a ~22nt product. In plants the precursor sequences may be longer, and the carpel factory (caf) enzyme appears to be involved in processing. In this case the mature sequence comes from the 3' arm of the precursor, and both Arabidopsis and rice genomes contain a number of related miRNA precursors which give rise to almost identical mature sequences. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,31,887,0,0,Gene; miRNA;,Predicted; PFOLD,80,0,,\"\",0.68706,-4.31951,1600000,216169,686,126,true,-3.0799,0.71834,2013-10-03T23:25:29.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00076,mir-181,1371,mir-181 microRNA precursor,\"Griffiths-Jones SR, Daub J\",\"Griffiths-Jones SR, INFERNAL\",45.5,45.5,45.4,This family represents the microRNA (miRNA) mir-181 which has been predicted or experimentally confirmed in a wide number of vertebrate species (MIPF:MIPF0000007). miRNAs are transcribed as ~70nt precursors (modelled here) and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 5' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,19,327,0,0,Gene; miRNA;,Predicted; PFOLD,117,0,,\"\",0.68756,-2.4599,1600000,162672,164,76,true,-3.9529,0.71901,2013-10-03T23:25:43.000Z,2022-05-06T13:10:02.000Z\n",
            "RF00077,SraB,1372,SraB RNA,Bateman A,Bateman A,130.0,133.9,127.8,This RNA was discovered in E. coli during a large scale screen [1]. This ncRNA was found to be expressed only in stationary phase [1]. The function of this RNA is unknown.,sraB,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,9,0,0,Gene; sRNA;,Predicted; PFOLD,13,0,,\"\",0.53773,-9.57529,1600000,352566,292,169,true,-3.3408,0.71567,2013-10-03T23:25:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00078,MicA,1373,MicA sRNA,Barquist LE,Predicted; Barquist LE,41.0,42.3,40.5,,SraD; MicA,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,17,113,0,0,Gene; sRNA;,Predicted; Udekwu KI; WAR,128,0,,\"\",0.47393,-7.94632,1600000,250275,164,75,true,-3.288,0.71902,2013-10-03T23:25:50.000Z,2021-11-19T11:53:18.000Z\n",
            "RF00079,OmrA-B,1374,OmrA-B family,Barquist LE,Predicted; WAR; Infernal,40.0,42.9,37.9,,SraE_OmrA_OmrB; OmrA-B; SraE_RygA_RygB; SraE,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,146,0,0,Gene; sRNA;,Predicted; WAR,108,0,,\"\",0.55338,-6.85941,1600000,301359,187,87,true,-3.1662,0.71872,2013-10-03T23:25:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00080,yybP-ykoY,1375,yybP-ykoY manganese riboswitch,Moxon SJ,\"Barrick JE, Breaker RR\",45.0,45.0,44.9,\"This RNA was originally discovered in E. coli during a large scale screen and was named SraF [1]. The family was later found to exist upstream of related families of protein genes in many bacteria, including the the yybP and ykoK genes in B. subtilis. The specific functions of these proteins are unknown, but this structured RNA element may be involved in their genetic regulation as a riboswitch [2].\",SraF,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.53 -Z 549862.597050 --mxsize 128 CM SEQDB,29,803,0,0,Cis-reg; riboswitch;,\"Predicted; Barrick JE, Breaker RR\",736,0,,\"\",0.66591,-9.00901,1600000,249099,551,175,true,-3.647,0.71513,2013-10-03T23:25:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00081,ArcZ,1376,ArcZ RNA,Bateman A,Bateman A,66.0,67.0,65.7,\"This RNA was discovered in E. coli during a large scale screen [1]. SraH expression is highly abundant in stationary phase, but low levels could be detected in exponentially growing cells as well [1]. The function of this RNA is unknown.\",ArcZ_RNA; SraH,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,61,0,0,Gene; sRNA;,Predicted; PFOLD,67,0,,\"\",0.61927,-6.02763,1600000,230162,203,106,true,-3.3788,0.71848,2013-10-03T23:26:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00082,SraG,1239,SraG RNA,Bateman A,\"Bateman A, INFERNAL\",50.0,54.4,47.6,This RNA was discovered in E. coli during a large scale screen [1]. Evidence was found for specific cleavage of this RNA [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,120,0,0,Gene; sRNA;,Predicted; PFOLD,123,0,,\"\",0.49228,-9.38018,1600000,322345,297,177,true,-3.1795,0.71479,2013-02-01T11:56:41.000Z,2022-05-06T13:09:54.000Z\n",
            "RF00083,GlmZ_SraJ,1377,GlmZ RNA activator of glmS mRNA,\"Bateman A, Gardner P\",Published; PMID:18351803,40.0,40.4,37.7,\"This RNA was discovered in E. coli during a large scale screen [1]. The sraJ gene was expressed in early logarithmic phase, but its level decreased with cell growth [1]. Specific cleavage products of this RNA were found [1]. The function of this RNA is unknown. Identified in Yersinia pseudotuberculosis as Ysr248/153/sR016 [4].\",SraJ,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,100,0,0,Gene; sRNA;,Published; PMID:18351803,119,0,,\"\",0.63239,-8.90795,1600000,339949,409,212,true,-3.6118,0.71132,2013-10-03T23:26:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00084,CsrC,1378,CsrC RNA family,Bateman A,Bateman A,61.0,64.7,58.4,\"The 245 nucleotide sRNA of Escherichia coli, CsrC, was discovered using a genetic screen for factors that regulate glycogen biosynthesis. CsrC RNA binds multiple copies of CsrA Pfam:PF02599, a protein that post-transcriptionally regulates central carbon flux, biofilm formation and motility in E. coli. CsrC antagonises the regulatory effects of CsrA, presumably by sequestering this protein. The discovery of CsrC is intriguing, in that a similar sRNA, CsrB, performs essentially the same function. Both sRNAs possess similar imperfect repeat sequences (18 in CsrB, nine in CsrC), primarily localised in the loops of predicted hairpins, which may serve as CsrA binding elements. Transcription of csrC increases as the culture approaches the stationary phase of growth and is indirectly activated by CsrA via the response regulator UvrY [1]. This RNA was also discovered in E. coli during a large scale screen [2]. The gene called SraK, was highly abundant in stationary phase, but low levels could be detected in exponentially growing cells as well [2].\",SraK,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 16.55 -Z 549862.597050 CM SEQDB,4,38,0,0,Gene; sRNA;,Predicted; PFOLD,42,0,,\"\",0.76653,-8.06896,1600000,456319,397,255,true,-3.3165,0.70821,2013-10-03T23:26:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00085,SNORD28,1379,Small nucleolar RNA SNORD28,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",50.0,53.8,49.8,U28 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U28 is encoded within the U22 snoRNA host gene (UHG) in mammals [1] and is thought to act as a 2'-O-ribose methylation guide for ribosomal RNA.,U28,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,68,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Griffiths-Jones SR,45,0,,\"\",0.55624,-3.8246,1600000,254489,168,77,true,-3.3063,0.71866,2013-10-03T23:26:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00086,SNORD27,1380,Small nucleolar RNA SNORD27,\"Griffiths-Jones SR, Moxon SJ\",\"Griffiths-Jones SR, Moxon SJ, Daub J\",52.0,53.2,51.5,U27 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U27 is encoded within the U22 snoRNA host gene (UHG) in mammals [1] and is thought to act as a 2'-O-ribose methylation guide for ribosomal RNA. This family also contains several related snoRNAs from yeast and plants.,U27,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.44 -Z 549862.597050 CM SEQDB,8,71,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; Moxon SJ, Daub J\",55,0,,\"\",0.57043,-3.454,1600000,231351,153,71,true,-3.4497,0.71875,2013-10-03T23:26:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00087,SNORD26,1381,Small nucleolar RNA SNORD26,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",44.0,45.5,43.3,U26 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U26 is encoded within the U22 snoRNA host gene (UHG) in mammals [1] and is thought to act as a 2'-O-ribose methylation guide for ribosomal RNA.,U26,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,70,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Griffiths-Jones SR,63,0,,\"\",0.66029,-3.56949,1600000,269170,160,76,true,-3.5872,0.7187,2013-10-03T23:26:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00088,SNORD30,1382,Small nucleolar RNA SNORD30,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",40.0,40.3,39.7,U30 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U30 is encoded within the U22 snoRNA host gene (UHG) in mammals [1] and is thought to act as a 2'-O-ribose methylation guide for ribosomal RNA.,U30,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.95 -Z 549862.597050 CM SEQDB,20,78,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,60,0,,\"\",0.47604,-6.47956,1600000,256219,152,70,true,-3.9478,0.71896,2013-10-03T23:26:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00089,SNORD31,1383,Small nucleolar RNA SNORD31,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",40.0,40.0,39.5,U31 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U31 is encoded within the U22 snoRNA host gene (UHG) in mammals [1] and is thought to act as a 2'-O-ribose methylation guide for ribosomal RNA.,U31,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,222,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,114,0,,\"\",0.66233,-4.16658,1600000,216879,145,69,true,-3.9957,0.71875,2013-10-03T23:26:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00090,SNORA74,1384,Small nucleolar RNA SNORA74,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",71.0,72.4,70.6,\"U19 belongs to the H/ACA class of snoRNAs. snoRNAs bind a number of proteins (including dyskerin, Gar1p and Nop10p in the case of the H/ACA class) to form snoRNP complexes. This class are thought to guide the sites of modification of uridines to pseudouridines by forming direct base pairing interactions with substrate RNAs [1]. Targets may include ribosomal and spliceosomal RNAs but the exact functions of many snoRNAs, including U19, are not confirmed. Co-precipitation of U19 snoRNA with RNase MRP RNA suggests that U19 may be involved in pre-rRNA processing [2].\",U19,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.50 -Z 549862.597050 CM SEQDB,24,250,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Griffiths-Jones SR,110,0,,\"\",0.66878,-8.48468,1600000,392067,322,201,true,-3.4388,0.71229,2013-10-03T23:26:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00091,SNORA62,1385,Small nucleolar RNA SNORA62/SNORA6 family,\"Griffiths-Jones SR, Moxon SJ, Daub J\",Griffiths-Jones SR,80.0,80.1,79.9,\"The members of this family belong to the H/ACA class of snoRNAs which are predicted to guide the sites of modification of uridines to pseudouridines [3]. This family includes human snoRNAs E2 (SNORA62), ACA6 (SNORA6) and M2 and mouse MBI-136. In humans both E2 and ACA6 are encoded in the introns on the same host gene (LAMR1)[1,3]. E2 is involved in the processing of eukaryotic pre-rRNA and has regions of complementarity to 28S rRNA [2]. ACA6 is predicted to guide the pseudouridylation of residue U3616 of 28S rRNA [5].\",E2,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.00 -Z 549862.597050 CM SEQDB,18,380,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,105,0,,\"\",0.54094,-8.96921,1600000,301969,270,156,true,-3.1401,0.71764,2013-10-03T23:27:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00092,SNORA63,1386,Small nucleolar RNA SNORA63,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.0,89.9,E3 belongs to the H/ACA class of snoRNAs and is involved in the processing of eukaryotic pre-rRNA and has regions of complementarity to 18S rRNA. E3 is encoded in introns in the gene for protein synthesis initiation factor 4AII [1].,E3,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,233,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Griffiths-Jones SR,103,0,,\"\",0.48205,-8.87185,1600000,289376,237,130,true,-3.3759,0.71812,2013-10-03T23:27:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00093,SNORD18,1387,Small nucleolar RNA SNORD18,Griffiths-Jones SR,Griffiths-Jones SR,41.0,41.0,40.9,\"U18 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. It is encoded in introns of ribosomal protein L1 in Xenopus and human, and in introns of elongation factor 1 beta in yeast. Related snoRNAs in Arabidopsis and rice have been named R63 and Z106 respectively. Factors involved in the processing of the intronic snoRNA have recently been elucidated [2].\",U18,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,16,655,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Griffiths-Jones SR,308,0,,\"\",0.5556,-5.73799,1600000,264009,251,73,true,-3.718,0.71868,2013-10-03T23:27:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00094,HDV_ribozyme,1388,Hepatitis delta virus ribozyme,\"Griffiths-Jones SR, Daub J\",\"Griffiths-Jones SR, Daub J\",50.0,55.0,49.0,The hepatitis delta virus (HDV) ribozyme is necessary for viral replication and is thought to be the only catalytic RNA known to be required for viability of a human pathogen. The ribozyme acts to process the RNA transcripts to unit lengths in a self cleavage reaction. The ribozyme is found to be active in vivo in the absence of any protein factors and is the fastest known naturally occurring self-cleaving RNA [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,33,7,0,0,Gene; ribozyme;,Published; PMID:9783582,4,0,,\"\",0.40947,-10.31077,1600000,237676,185,91,true,-3.0531,0.71882,2013-10-03T23:27:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00095,snoPyro_CD,1389,Pyrococcus C/D box small nucleolar RNA,Griffiths-Jones SR,Griffiths-Jones SR,45.0,45.1,44.9,\"Computational screens have identified C/D box snoRNAs in a number of archaeal genomes [1,2]. This family represents several related snoRNAs from Pyrococcus. These RNAs are thought to act as guides for the 2'-O-ribose methylation of ribosomal RNA.\",CD_sno_Pyro,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,25,24,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,17,0,,\"\",0.6009,-6.72604,1600000,342045,118,58,true,-4.3363,0.72079,2013-10-03T23:27:49.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00096,U8,1390,U8 small nucleolar RNA,\"Griffiths-Jones SR, Gardner PP\",\"Griffiths-Jones SR, Gardner PP\",41.0,41.4,40.9,\"U8 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U8 is the RNA component of a small complex (the U8 snoRNP) which is required for accumulation of mature large subunit ribosomal RNAs, 5.8S and 28S rRNAs. U8 RNA has been identified in human, mouse, rat and Xenopus.\",CeN21; CeR-2,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,55,375,0,0,Gene; snRNA; snoRNA; CD-box;,Published; PMID:11675000; Gardner PP,95,0,,\"\",0.51924,-9.7571,1600000,421749,231,135,true,-3.6257,0.718,2013-10-03T23:28:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00097,snoR71,1391,Plant small nucleolar RNA R71,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",40.0,40.0,39.9,\"R71 is a member of the C/D box family of snoRNAs containing the C (UGAUGA) and D (CUGA) box motifs. Multiple nearly identical copies of this snoRNA have been identified in Arabidopsis, and it is thought to function as a 2'-O-ribose methylation guide for 18S ribosomal RNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,25,60585,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,69,0,,\"\",0.6011,-4.97338,1600000,227896,201,107,true,-3.4854,0.71827,2013-10-03T23:29:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00099,SNORD22,1392,Small nucleolar RNA SNORD22,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",50.0,51.2,49.2,U22 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U22 is encoded within the U22 snoRNA host gene (UHG) in mammals [1] and has also been identified in Xenopus [2]. U22 is thought to as a 2'-O-ribose methylation guide for ribosomal RNA.,U22,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,49,206,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,70,0,,\"\",0.6321,-4.97552,1600000,274308,230,126,true,-2.9721,0.71799,2013-10-03T23:30:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00100,7SK,1393,7SK RNA,\"Griffiths-Jones SR, Gardner PP\",Published; PMID:18193315,35.0,35.0,34.9,7SK was discovered as an abundant small nuclear RNA in the mid-70s but a possible function has only recently been suggested [1]. Two independent studies have found that 7SK RNA binds the CDK9/cyclin T complex (known as elongation factor P-TEFb). P-TEFb activates transcription by phosphorylating the C-terminal domain of RNA polymerase II. This process is negatively regulated by the 7SK RNP. The secondary structure of 7SK RNA shown here was proposed by Wasserman and Steitz on the basis of chemical modification and enzymatic cleavage experiments [2]. This family contains a number of annotated pseudogenes and 7SK derived repeats. The threshold is artificially high to minimise the pseudogene count.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,45,15232,0,0,Gene;,Published; PMID:1646389,136,0,,\"\",0.6781,-8.68824,1600000,312385,544,330,true,-4.439,0.70488,2013-10-03T23:30:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00101,SraC_RyeA,2373,SraC/RyeA RNA,Bateman A,Bateman A,37.0,37.2,36.8,This RNA was discovered in E. coli during a large scale screens [1-2]. The function of this RNA is unknown. This RNA overlaps RFAM:RF00111 on the opposite strand suggesting that the two may act in a concerted manner.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,185,0,0,Gene; sRNA;,Predicted; PFOLD,197,0,,\"\",0.61149,-4.33069,1600000,286924,261,145,true,-3.0355,0.718,2013-10-03T23:32:07.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00102,VA,1395,VA RNA,\"Bateman A, Moxon SJ\",\"Bateman A, Moxon SJ\",37.0,37.0,36.9,The VA (viral associated) RNA is found in adenovirus. It plays a role in regulating translation.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,54,81,0,0,Gene;,\"Published; PMID:8809020, Moxon SJ\",68,0,,\"\",0.48769,-9.33544,1600000,248227,300,159,true,-2.9768,0.71751,2013-10-03T23:32:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00103,mir-1,1396,mir-1 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.2,53.2,52.8,\"This family represents the microRNA (miRNA) mir-1 family. miRNAs are transcribed as ~70nt precursors (modelled here) and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 3' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA. This family also contains the mir-206 from human, which appears to be related to mir-1. Family represents the microRNA (miRNA) precursor mir-1 imported from miRBase [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,66,495,0,0,Gene; miRNA;,Predicted; RNAalifold,254,0,,\"\",0.60863,-3.53175,1600000,118466,154,77,true,-4.1776,0.71905,2013-10-03T23:32:16.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00104,mir-10,1397,mir-10 microRNA precursor family,\"Griffiths-Jones SR, Daub J\",Griffiths-Jones SR,47.5,47.5,47.4,\"This family represents the microRNA (miRNA) mir-10 family, which contains mir-10, mir-51, mir-57, mir-99 and mir-100. mir-10, mir-99 and mir-100 have now been predicted or experimentally confirmed in a wide range of species (MIPF:MIPF0000033, MIPF:MIPF0000025). mir-51 and mir-57 have currently only been identified in the nematode Caenorhabditis elegans (MIPF:MIPF0000268, MIPF:MIPF0000271). miRNAs are transcribed as ~70nt precursors (modelled here) and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 5' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,36,727,0,0,Gene; miRNA;,Predicted; PFOLD,216,0,,\"\",0.73695,-2.2581,1600000,172454,146,75,true,-3.9623,0.71887,2013-10-03T23:32:23.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00105,SNORD115,1398,Small nucleolar RNA SNORD115,Griffiths-Jones SR,Griffiths-Jones SR,39.0,39.0,38.9,\"HBII-52 is a member of the C/D class of snoRNAs, which contain the C (UGAUGA) and D (CUGA) box motifs. HBII-52 is encoded in a tandemly repeated array with another C/D snoRNA, HBII-85, in the Prader-Willi syndrome (PWS) region of chromosome 15. HBII-52 is exclusively expressed in the brain but is absent in PWS patients. HBII-52 lacks any significant complementarity with ribosomal RNAs, but does have an 18-nt region of conserved complementarity to serotonin 2C receptor mRNA.\",snoHBII-52; HBII-52,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,32,1184,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,38,0,,\"\",0.5973,-4.0654,1600000,280574,171,82,true,-3.2435,0.71863,2013-10-03T23:32:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00106,RNAI,1399,RNAI,Griffiths-Jones SR,Griffiths-Jones SR,47.0,47.0,46.9,\"RNAI is an antisense repressor of the replication of some E. coli plasmids, including ColE1. Plasmid replication is usually initiated by RNAII, which acts as a primer by binding to its template DNA. The complementary RNAI binds RNAII prohibiting it from its initiation role. The rate of degradation of RNAI is therefore a major factor in control of plasmid replication. This rate of degradation is aided by the pcnB (plasmid copy number B) gene product, which polyadenylates the 3' end of RNAI targeting it for degradation by PNPase. Please note: This family contains subsequences annotated as human (Z96734), Arabidopsis (AB003142), Xenopus (X91243) and S. pombe (L25927, L25928), which match with highly significant scores. These eukaryotic matches are almost certainly the result of vector contamination.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,141,0,0,Gene; antisense;,Published; PMID:7535193,89,0,,\"\",0.53317,-8.04217,1600000,248994,198,102,true,-2.9669,0.71874,2013-10-03T23:33:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00107,FinP,1400,FinP,Griffiths-Jones SR,Griffiths-Jones SR,45.0,50.8,43.1,\"The FinOP system regulates the transfer of F-like plasmids. FinP encodes an antisense RNA product that is complementary to part of the 5' UTR of the traJ mRNA. The traJ gene encodes a protein required for transcription from the major transfer promoter, pY. The FinO protein is essential for effective repression, acting by binding to FinP and protecting it from RNase E degradation.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,11,0,0,Gene;,Published; PMID:9917389,12,0,,\"\",0.56773,-6.16528,1600000,218265,170,79,true,-3.6484,0.71894,2013-10-03T23:33:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00108,SNORD116,1401,Small nucleolar RNA SNORD116,Griffiths-Jones SR,Griffiths-Jones SR,35.0,35.2,34.9,\"HBII-85 is a member of the C/D class of snoRNAs, which contain the C (UGAUGA) and D (CUGA) box motifs. HBII-85 is encoded in a tandemly repeated array with another C/D snoRNA, HBII-52, in the Prader-Willi syndrome (PWS) region of chromosome 15. Unlike most other snoRNAs, HBII-85 is expressed prevalently in the brain (but is absent in PWS patients) and lacks any significant complementarity with ribosomal RNA.\",snoHBII-85; HBII-85,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,48,809,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,52,0,,\"\",0.60297,-2.89502,1600000,240853,191,92,true,-3.4804,0.71852,2013-10-03T23:33:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00109,Vimentin3,1402,Vimentin 3' UTR protein-binding region,Griffiths-Jones SR,Griffiths-Jones SR,36.0,36.2,35.7,\"The 3' UTR of the vimentin mRNA contains a Y shaped structure which has been shown to have protein binding activity [1]. The same region has been implicated in the control of mRNA localisation to the perinuclear region of the cytoplasm, possibly at sites of intermediate filament assembly [2]. The identity of the proteins involved and the localisation mechanism are not known.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29 -Z 549862.597050 CM SEQDB,19,160,0,0,Cis-reg;,Published; PMID:9241253,103,0,,\"\",0.53445,-4.42432,1600000,230937,174,67,true,-3.7125,0.7191,2013-10-03T23:33:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00110,RybB,1403,RybB RNA,Bateman A,\"Gottesman S, Storz G\",40.0,41.6,38.1,This ncRNA was identified in a large scale screen of E. coli [1]. The function of this short RNA is unknown. The exact 5' and 3' ends of this RNA are uncertain.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,115,0,0,Gene; sRNA;,Predicted; PFOLD,117,0,,\"\",0.56616,-7.92658,1600000,339298,167,79,true,-3.3408,0.71866,2013-10-03T23:33:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00111,SdsR_RyeB,1404,SdsR_RyeB RNA,Bateman A,Gottesman S,38.0,38.9,36.9,\"This ncRNA was identified in a large scale screen of E. coli [1]. The exact 5' and 3' ends of this RNA are uncertain. This RNA overlaps RFAM:RF00101 on the opposite strand suggesting that the two may act in a concerted manner. The homologous sRNA in S. enterica was shown to regulate synthesis of major porin OmpD. A study using Salmonella identified 20 targets of this sRNA including transcriptional regulator, CRP, global DNA-binding factors, StpA and HupB, the antibiotic transporter protein, TolC, and the RtsA/B two-component system (TCS), and validated their post-transcriptional control by SdsR/RyeB RNA. SdsR (sigma-S-dependent sRNA).\",RyeB;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,222,0,0,Gene; sRNA;,Predicted; PFOLD,236,0,,\"\",0.5468,-6.18833,1600000,265426,192,100,true,-3.8347,0.71854,2013-10-03T23:33:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00112,CyaR_RyeE,1405,CyaR/Rye RNA,\"Bateman A, Daub J\",\"Gottesman S, Storz G, INFERNAL\",47.0,49.0,45.8,\"This sRNA was originally identified in a large scale screen of E. coli. It was shown to bind Hfq and called RyeE [1]. This sRNA has since been characterised as a post-transcriptional regulator of the 18 kDa OmpX porin in Salmonella [2]. Phylogenetic and mutational analyses suggest that a conserved RNA hairpin of this sRNA which features a C-rich apical loop, acts to sequester the Shine-Dalgarno sequence of ompX mRNA and inhibits translational initiation [2]. The expression of this sRNA has been shown to be tightly controlled by the cyclic AMP receptor protein, CRP and has been renamed CyaR (cyclic AMP-activated RNA)[2]. CyaR appears to be highly conserved amongst enterobacteria and the cyaR gene is frequently located downstream of the yegQ gene.\",RyeE,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,88,0,0,Gene; sRNA;,Predicted; PFOLD,99,0,,\"\",0.57024,-4.99537,1600000,259329,177,85,true,-3.2056,0.71869,2013-10-03T23:33:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00113,SIB_RNA,1244,Short Intergenic Abundant RNA,Bateman A,\"Gottesman S, Storz G\",56.0,59.3,55.7,\"This family contains multiple ncRNA from E. coli that were identified in a large scale screen of E. coli, were they were called candidates 43,55 and 61 [1]. These RNAs have also been identified by other groups and called QUAD RNAs.\",QUAD; QUAD_RNA,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.02 -Z 549862.597050 CM SEQDB,17,37,0,0,Gene; antitoxin;,Predicted; PFOLD,31,0,,\"\",0.57298,-7.33823,1600000,327560,258,147,true,-3.1671,0.71808,2013-02-01T11:56:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00114,S15,1406,Ribosomal S15 leader,Bateman A,Ji Y,39.5,39.5,39.4,\"The E. coli Ribosomal S15 gene 5' UTR can form two alternative structures. One that is a series of three hairpins, The other includes a pseudoknot. This structure causes translational regulation of the S15 protein. The alignment of this family contains only the final two hairpins which are conserved in other species, and also includes the complete region that forms the alternate structures.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,78,554,0,0,Cis-reg; leader;,Published; PMID:8955900,603,0,,\"\",0.63777,-7.79167,1600000,396848,209,117,true,-3.4321,0.71817,2013-10-03T23:34:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00115,McaS,1407,McaS/IsrA RNA,Argasinska J,Argasinska J,42.0,42.1,38.7,\"This family consists of several bacterial RNA genes which are found between the abgR and ydaL genes in Escherichia coli and Shigella flexneri.[1] It was discovered using a computational screen of the E. coli genome.[1] Subsequent characterisation of ISO61 region has revealed that the reverse strand is actually a CsrA binding ncRNA called McaS and that it has a role in biofilm formation control.[2] Furthermore, it has been shown that McaS(IsrA) exists as a ribonucleoprotein particles (sRNPs), which involve a defined set of proteins including Hfq, S1, CsrA, ProQ and PNPase.[4]\",IS061;,cmbuild -n -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,13,0,0,Gene; sRNA;,Predicted; 22289118,17,0,,\"\",0.52501,-7.54463,1600000,289422,194,96,true,-3.2838,0.71867,2013-10-03T23:34:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00116,C0465,1408,C0465 RNA,Moxon SJ,Hershberg R,40.0,42.5,35.3,This family consists of a number of bacterial RNA genes of 78 bases in length which are found between the tar and cheW genes in E. coli and Shigella flexneri. The function of this RNA is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,1,0,0,Gene; sRNA;,Predicted; PFOLD,9,0,,\"\",0.54001,-7.31484,1600000,389877,171,78,true,-3.3844,0.71889,2013-10-03T23:34:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00117,C0719,1409,C0719 RNA,Moxon SJ,Hershberg R,50.0,69.3,34.9,\"This family consists of a number of bacterial RNA genes, of 222 bases in length, which are found between the yghK and glcB genes in both Shigella flexneri and Escherichia coli. The function of this RNA is unknown.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.27 -Z 549862.597050 CM SEQDB,9,1,0,0,Gene; sRNA;,Predicted; ILM,10,0,,\"\",0.63554,-9.24939,1600000,408212,360,222,true,-3.6201,0.71034,2013-10-03T23:34:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00118,rydB,1410,rydB RNA,Barquist L,mafft; Infernal,50.0,59.2,37.1,\"This family consists of several bacterial sequences from E. coli, Shigella flexneri and Salmonella species. This RNA gene was discovered by an RNA screen [1] between the ydiC and ydiH in E. coli. This gene is only 67 bases in length and is composed of a hairpin like structure. The function of this RNA is unknown.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.5 -Z 549862.597050 CM SEQDB,7,49,0,0,Gene; sRNA;,Predicted; RNAalifold,45,0,,\"\",0.66075,-4.03185,1600000,202687,142,62,true,-3.8323,0.71915,2013-10-03T23:34:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00119,C0299,1411,C0299 RNA,Moxon SJ,Hershberg R,40.0,52.2,32.4,This family consists of a group of Shigella flexneri and Escherichia coli RNA genes which are 78 bases in length and are found between the hlyE and umuD genes. The function of this RNA is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,2,0,0,Gene; sRNA;,Predicted; PFOLD,6,0,,\"\",0.55514,-7.56688,1600000,386091,172,79,true,-3.4902,0.71873,2013-10-03T23:34:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00121,MicC,1412,MicC RNA,Moxon SJ,Hershberg R,90.0,94.7,81.3,MicC (previously known as IS063 [1]) is located between the ompN and ydbK genes in E. coli. This family of Hfq-associated sRNA is thought to be a regulator of OmpC expression. MicC inhibits the 30S ribosome binding through a conserved 22 bp RNA duplex near the start codon of the ompC mRNA. Along with Rfam:RF00033 it may act in conjunction with EnvZ-OmpR two-component system to control the OmpF/OmpC protein ratio in response to a variety of environmental stimuli [2]. The expression of micC was shown to be increased in the presence of beta-lactam antibiotics. The metabolic activity of E.coli expressing either OmpC or OmpF was strongly inhibited upon exposure to beta-lactams [3].,IS063,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,14,0,0,Gene; sRNA;,Predicted; PFOLD,21,0,,\"\",0.45925,-8.15303,1600000,273700,227,123,true,-2.9103,0.71829,2013-10-03T23:34:43.000Z,2022-05-06T13:10:02.000Z\n",
            "RF00122,GadY,1413,GadY,Moxon SJ,\"Hershberg R, INFERNAL\",45.0,55.4,33.5,\"This family represents a GadY RNA (previous named IS183 in [1]). GadY can form base pairs with the 3' UTR of its target mRNA gadX, this pairing is thought to confer increased stability to the transcript, allowing accumulation of gadX (a transcriptional regulator of the acid response) and therefore increased expression of downstream acid resistance genes [2].\",IS183,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,2,0,0,Gene; sRNA;,Predicted; ILM,6,0,,\"\",0.57306,-6.84715,1600000,261412,221,114,true,-3.5214,0.71844,2013-10-03T23:34:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00124,IS102,1414,IS102 RNA,Moxon SJ,Hershberg R,46.0,47.7,43.0,This family consists of a group of Shigella flexneri and Escherichia coli RNA genes which are 208 bases in length and are found between the yeeP and flu genes. The function of this RNA is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,5,4,0,0,Gene; sRNA;,Predicted; PFOLD,7,0,,\"\",0.48784,-10.0088,1600000,312993,338,206,true,-3.3862,0.7118,2013-10-03T23:34:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00125,IS128,1415,IS128 RNA,Moxon SJ,Hershberg R,80.0,94.3,43.8,This family consists of a number of bacterial RNA genes of 209 bases in length. The IS128 RNA gene is present in Escherichia coli and Shigella flexneri and where it is found between the sseA and sseB genes. The function of this RNA is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,2,0,0,Gene; sRNA;,Predicted; RNAalifold,6,0,,\"\",0.56241,-8.91338,1600000,357700,342,209,true,-3.613,0.71145,2013-10-03T23:34:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00126,ryfA,1416,ryfA RNA,Moxon SJ,Hershberg R,80.0,80.5,77.9,\"This family consists of a number of bacterial RNA genes of 305 bases in length. The ryfA RNA gene is present in E. coli, Shigella flexneri and Salmonella species where it is found between the ydaN and dbpA genes. The function of this RNA is unknown.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,23,0,0,Gene; sRNA;,Predicted; RNAalifold,27,0,,\"\",0.67263,-8.26974,1600000,343253,447,296,true,-4.0194,0.7062,2013-10-03T23:35:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00127,t44,1417,t44 RNA,Barquist L,Infernal; LocaRNA,44.0,44.1,43.8,This family consists of a number of bacterial RNA genes of between 135 and 170 bases in length. The t44 gene has been identified in several species of enteric bacteria but homologs have also been identified in Pseudomonas and Coxiella species. The t44 gene is found between the map and rpsB genes in all species in the full alignment apart from Shigella flexneri. The function of this RNA is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,78,723,0,0,Gene; sRNA;,Predicted; LocaRNA; Barquist L,745,0,,\"\",0.65858,-2.72914,1600000,126641,251,97,true,-3.504,0.7185,2013-10-03T23:35:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00128,GlmY_tke1,1418,Glm Y RNA activator of glmS mRNA,\"Moxon SJ, Gardner P\",Published; PMID:17854828,33.0,33.3,32.1,\"This family consists of a number of bacterial RNA genes of around 167 bases in length. The tke1 RNA gene is present in Escherichia coli, Shigella flexneri, Yersinia pestis and Salmonella species where it is found between the yfhK and purL genes. The function of this RNA is unknown.\",tke1,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,113,0,0,Gene; sRNA;,Published; PMID:17854828,127,0,,\"\",0.59097,-8.41452,1600000,275031,279,163,true,-3.4574,0.71674,2013-10-03T23:35:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00129,mir-103,2374,mir-103/107 microRNA precursor,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",59.6,59.6,46.5,This family consists of the mammalian microRNA (miRNA) precursors mir-103 and mir107. Animal miRNAs are transcribed as ~70nt precursors and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 3' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,93,327,0,0,Gene; miRNA;,Predicted; RNAalifold,123,0,,\"\",0.68395,-3.29177,1600000,144377,161,78,true,-3.7993,0.71905,2013-10-03T23:35:35.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00130,mir-192,2375,mir-192/215 microRNA precursor,Moxon SJ,Griffiths-Jones SR,54.0,63.5,52.0,This family represents the human microRNA (miRNA) precursors mir-192 and mir-215. Animal miRNAs are transcribed as ~70nt precursors and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 5' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,48,154,0,0,Gene; miRNA;,Predicted; PFOLD,118,0,,\"\",0.70242,-2.49662,1600000,183696,332,75,true,-4.0336,0.7191,2013-10-03T23:35:38.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00131,mir-30,1421,mir-30 microRNA precursor,\"Moxon SJ, Boursnell C\",Griffiths-Jones SR,50.2,50.3,50.0,This family consists of the mammalian microRNA (miRNA) precursor mir-30a. Animal miRNAs are transcribed as ~70nt precursors and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from both the 3' (miR-30) [1] and 5' (mir-97-6) [2] arms of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,49,575,0,0,Gene; miRNA;,Predicted; PFOLD,118,0,,\"\",0.62139,-4.20745,1600000,322571,135,60,true,-3.5026,0.71958,2013-10-03T23:35:53.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00132,snoR24,1422,Small nucleolar RNA R24,Moxon SJ,Griffiths-Jones SR,38.0,38.0,37.4,snoR24 is a C/D box small nucleolar RNA which acts as a methylation guide for 18S ribosomal RNA in plants [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,12,509,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,77,0,,\"\",0.57933,-4.55975,1600000,232217,192,97,true,-3.3913,0.71838,2013-10-03T23:35:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00133,SNORD33,2376,Small nucleolar RNA Z195/SNORD33/SNORD32 family,\"Moxon SJ, Daub J\",\"Griffiths-Jones SR, Daub J, INFERNAL\",44.0,44.0,43.9,\"Members of this family belong to the C/D box class of small nucleolar RNA which contain C (UGAUGA) and D (CUGA) box motifs. This family includes plant snoRNA Z195, human snoRNAs SNORD33 (U33) and SNORD32 (U32). snoRNA U33 is found in intron 4 of the ribosomal protein L13a gene in humans and is also present in mice. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs. Although the selection of the target nucleotide requires the antisense element and the conserved box D or D' of the snoRNA, the methyltransferase activity is supposed to reside in one of the protein components [3].\",snoZ195,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.26 -Z 549862.597050 CM SEQDB,72,487,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,181,0,,\"\",0.5584,-4.51249,1600000,258115,160,85,true,-3.4461,0.71852,2013-10-03T23:36:52.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00134,snoZ196,2377,Small nucleolar RNA Z196/R39/R59 family,\"Moxon SJ, Daub J\",Griffiths-Jones SR,55.0,56.5,54.9,\"This family includes plant snoRNA snoZ196 and yeast snoRNAs R39 and R59. Members of this family belong to the C/D box class of snoRNA which contain C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs. snoZ196 acts as a methylation guide for 18S ribosomal RNA in plants [1].Although the selection of the target nucleotide requires the antisense element and the conserved box D or D' of the snoRNA, the methyltransferase activity is supposed to reside in one of the protein components [4].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.25 -Z 549862.597050 CM SEQDB,22,302,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,90,0,,\"\",0.52539,-3.37293,1600000,231713,169,85,true,-3.5853,0.7185,2013-10-03T23:37:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00135,snoZ223,1425,Small nucleolar RNA Z223,\"Griffiths-Jones SR, Moxon SJ\",Griffiths-Jones SR,45.0,46.7,44.7,snoRNA Z223 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs identified in rice. C/D box small nucleolar RNAs are known to act as a methylation guides for 18S ribosomal RNA in plants [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,5,389,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,79,0,,\"\",0.6237,-2.84419,1600000,214116,190,94,true,-3.4858,0.71838,2013-10-03T23:37:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00136,SNORD81,1426,Small nucleolar RNA SNORD81,Griffiths-Jones SR,Griffiths-Jones SR,50.0,50.6,49.7,U81 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U81 acts as a guanine methylation guide and is found in intron 11 of the gas5 gene in mammals [1]. This RNA has also been called Z23 snoRNA.,U81,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,85,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,52,0,,\"\",0.61866,-2.3968,1600000,216395,197,77,true,-3.8124,0.71866,2013-10-03T23:37:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00137,SNORD83,1427,Small nucleolar RNA SNORD83,\"Griffiths-Jones SR, Daub J\",Griffiths-Jones SR,35.0,35.3,34.4,U83 (SNORD83) and U84 (SNORD84) are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. They act as guanine methylation guides but lack complementarity to ribosomal RNA [1]. U83 and U84 are spliced from introns 5 and 4 of the BAT1 gene in mammals.,U83,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,252,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,111,0,,\"\",0.61333,-3.77929,1600000,279518,167,78,true,-3.5201,0.71872,2013-10-03T23:37:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00138,SNORD16,1428,Small nucleolar RNA SNORD16,Griffiths-Jones SR,Griffiths-Jones SR,45.0,45.5,41.7,U16 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U81 acts as a guanine methylation guide and is found in an intron of ribosomal proteins L1 gene in animals [1]. This snoRNA was independently named MBII-98 in mouse [2].,U16,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,219,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,115,0,,\"\",0.61114,-2.89802,1600000,213180,198,100,true,-3.3012,0.71855,2013-10-03T23:37:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00139,SNORA72,1245,Small nucleolar RNA SNORA72,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",43.0,43.6,41.4,\"U72 is a member of the H/ACA class of snoRNAs. The snoRNA gene is located in intron 4 of the ribosomal L30 protein gene in human, mouse and chicken.\",U72,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29 -Z 549862.597050 CM SEQDB,29,334,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,107,0,,\"\",0.48657,-6.87906,1600000,252365,237,132,true,-3.0517,0.71818,2013-02-01T11:56:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00140,Alpha_RBS,1429,Alpha operon ribosome binding site,\"Griffiths-Jones SR, Moxon SJ\",Griffiths-Jones SR,38.0,38.0,37.8,\"The bacterial alpha operon ribosome binding site is surrounded by this complex pseudoknotted structure. Translation of the mRNA produces 4 ribosomal protein products, one of which (S4) acts as a translational repressor by binding to the nested pseudoknot region. The mechanism of repression is thought to involve a conformational switch in the pseudoknot region and ribosome entrapment [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.00 -Z 549862.597050 CM SEQDB,39,523,0,0,Cis-reg;,Published; PMID:11504736,537,0,,\"\",0.62317,-7.92672,1600000,252615,199,110,true,-3.4227,0.71817,2013-10-03T23:37:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00142,snoZ118,2378,Small nucleolar RNA Z118/Z121/Z120,Moxon SJ,Griffiths-Jones SR,48.0,48.5,46.5,\"This family consists of several snoRNA Z118, Z121 and Z120 sequences which are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family seems to be found exclusively in rice. C/D box small nucleolar RNAs are known to act as a methylation guides for 18S ribosomal RNA in plants [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,244,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,20,0,,\"\",0.578,-3.41655,1600000,183178,209,108,true,-2.9247,0.71825,2013-10-03T23:37:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00143,mir-6,1431,mir-6 microRNA precursor,Moxon SJ,Griffiths-Jones SR,58.0,59.2,56.5,This family consists of the microRNA (miRNA) precursor mir-6 which appears to be Drosophila specific. Animal miRNAs are transcribed as ~70nt precursors and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 3' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,36,41,0,0,Gene; miRNA;,Predicted; PFOLD,18,0,,\"\",0.5674,-4.50299,1600000,232755,196,74,true,-3.5119,0.71906,2013-10-03T23:38:04.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00144,mir-199,1432,mir-199 microRNA precursor,Griffiths-Jones SR,Griffiths-Jones SR,60.0,61.3,57.1,This family consists of the microRNA (miRNA) precursor mir-199 which appears to be specific to humans. Animal miRNAs are transcribed as ~70nt precursors and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 3' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,83,284,0,0,Gene; miRNA;,Predicted; RNAalifold,124,0,,\"\",0.65219,-4.19462,1600000,148677,167,80,true,-3.7861,0.71892,2013-10-03T23:38:07.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00145,snoZ105,1433,Small nucleolar RNA Z105,\"Moxon SJ, Wilkinson AC\",Predicted; CMfinder,45.0,45.4,43.9,This family consists of several snoRNA Z105 sequences which are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family seems to be found exclusively in plants. C/D box small nucleolar RNAs are known to act as a methylation guides for 18S ribosomal RNA [1].,snoR7,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,236,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder,72,0,,\"\",0.50486,-6.99999,1600000,263841,148,72,true,-3.0049,0.71886,2013-10-03T23:38:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00147,SNORD34,1434,Small nucleolar RNA SNORD34,Moxon SJ,Griffiths-Jones SR,38.0,38.0,37.8,\"snoRNA U34 is found in intron 5 of the ribosomal protein L13a gene in humans. U34 is a member of the C/D class of snoRNA which contain C (UGAUGA) and D (CUGA) box motifs, and functions in directing site-specific 2'-O-methylation of 25S rRNAs. Although the selection of the target nucleotide requires the antisense element and the conserved box D or D' of the snoRNA, the methyltransferase activity is thought to reside in one of the protein components [1]. U34 snoRNA has homologues in mouse, Arabidopsis (annotated as snoR4) and in several copies in rice (alternatively named snoZ181) [2]. This family also contains a putative homologue in Drosophila (Griffiths-Jones SR, pers. obs.).\",U34,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.84 -Z 549862.597050 CM SEQDB,14,443,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD, Griffiths-Jones SR\",155,0,,\"\",0.58673,-3.87878,1600000,196099,165,80,true,-3.6939,0.71864,2013-10-03T23:38:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00149,snoZ103,1435,Small nucleolar RNA Z103,Moxon SJ,Griffiths-Jones SR,44.0,44.2,43.8,This family consists of several snoRNA Z103 sequences which are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family seems to be found exclusively in plants. C/D box small nucleolar RNAs are known to act as a methylation guides for 18S ribosomal RNA [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.21 -Z 549862.597050 CM SEQDB,12,1457,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,83,0,,\"\",0.59742,-4.43391,1600000,260331,194,100,true,-3.4546,0.71833,2013-10-03T23:38:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00150,SNORD42,1436,Small nucleolar RNA SNORD42,Moxon SJ,Griffiths-Jones SR,39.0,39.2,38.6,\"snoRNA U42 is found in intron 3 of the ribosomal protein L23a (RPL23a) gene in humans and is also present in mice. U42 is a member of the C/D class of snoRNA which contain C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs. Although the selection of the target nucleotide requires the antisense element and the conserved box D or D' of the snoRNA, the methyltransferase activity is supposed to reside in one of the protein components [1].\",U42,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,7,172,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,102,0,,\"\",0.51828,-4.87439,1600000,223587,151,68,true,-3.31,0.71882,2013-10-03T23:38:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00151,SNORD58,1437,Small nucleolar RNA SNORD58,\"Moxon SJ, Daub J\",Griffiths-Jones SR,42.0,42.0,41.7,\"snoRNA U58 is found in introns 1, 3 and 6 of the ribosomal protein RPL17 gene in humans. U58 is a member of the C/D class of snoRNA which contain C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs. U58A and U58B are predicted to guide 2'O-ribose methylation of the large 28S rRNA subunit on position G4198 [2]. Although the selection of the target nucleotide requires the antisense element and the conserved box D or D' of the snoRNA, the methyltransferase activity is supposed to reside in one of the protein components [1].\",U58,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,33,422,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,135,0,,\"\",0.63874,-4.26863,1600000,268132,167,65,true,-3.6581,0.71887,2013-10-03T23:38:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00152,SNORD79,1246,Small nucleolar RNA SNORD79,Moxon SJ,Griffiths-Jones SR,47.0,47.1,45.1,\"snoRNA U79 or Z22 is found in intron 7 of the gas5 gene in humans and is also present in mice. U79 is a member of the C/D class of snoRNA which contain C (UGAUGA) and D (CUGA) box motifs. Most of the members of the C/D box family function in directing site-specific 2'-O-methylation of substrate RNAs. Although the selection of the target nucleotide requires the antisense element and the conserved box D or D' of the snoRNA, the methyltransferase activity is supposed to reside in one of the protein components [1].\",U79,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,123,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,116,0,,\"\",0.53321,-4.14611,1600000,235152,167,83,true,-3.5205,0.71858,2013-02-01T11:56:49.000Z,2021-07-05T14:43:56.000Z\n",
            "RF00153,SNORD62,1438,Small nucleolar RNA SNORD62,Moxon SJ,Griffiths-Jones SR,39.0,40.3,37.2,\"snoRNA U62 is found in both humans and mice. U62 is a member of the C/D class of snoRNA which contain C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs. Although the selection of the target nucleotide requires the antisense element and the conserved box D or D' of the snoRNA, the methyltransferase activity is supposed to reside in one of the protein components [1].\",U62,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,152,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,83,0,,\"\",0.62448,-3.37184,1600000,246585,177,86,true,-3.5727,0.71845,2013-10-03T23:38:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00154,SNORD63,1439,Small nucleolar RNA SNORD63,Moxon SJ,\"Griffiths-Jones SR, INFERNAL\",43.0,43.2,40.2,\"snoRNA U63 is found in both humans and mice. U63 is a member of the C/D class of snoRNA which contain C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs. Although the selection of the target nucleotide requires the antisense element and the conserved box D or D' of the snoRNA, the methyltransferase activity is supposed to reside in one of the protein components [1].\",U63,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,23,127,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,103,0,,\"\",0.57863,-5.08101,1600000,335152,154,70,true,-3.4158,0.71881,2013-10-03T23:38:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00155,SNORA66,1440,Small nucleolar RNA SNORA66,Moxon SJ,Griffiths-Jones SR,45.0,45.8,43.4,\"snoRNA U66 is found in both humans and mice. U66 is a member of the H/ACA class of snoRNA which contain H (ANANNA) and ACA box motifs. H/ACA box snoRNAs are involved in the conversion of uridines to pseudouridines (5-ribosyluracil) a constituent of structural RNAs (transfer, ribosomal, small nuclear, and small nucleolar) [1,2].\",U66,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,25,227,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,103,0,,\"\",0.50721,-8.16797,1600000,278152,242,133,true,-3.0623,0.71815,2013-10-03T23:39:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00156,SNORA70,1441,Small nucleolar RNA SNORA70,Moxon SJ,\"Griffiths-Jones SR, INFERNAL\",40.0,40.0,39.9,\"snoRNA U70 seems to be exclusive to mammals and is a member of the H/ACA class of snoRNA which contain H (ANANNA) and ACA box motifs. H/ACA box snoRNAs are involved in the conversion of uridines to pseudouridines (5-ribosyluracil) a constituent of structural RNAs (transfer, ribosomal, small nuclear, and small nucleolar) [1,2].\",U70,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,43,1290,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,73,0,,\"\",0.54823,-8.68325,1600000,300916,240,135,true,-3.4974,0.71807,2013-10-03T23:39:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00157,SNORD39,1442,Small nucleolar RNA SNORD55/SNORD39,Moxon SJ,\"Griffiths-Jones SR, INFERNAL\",35.0,35.0,34.9,\"snoRNA U39 (SNORD55), also described as U55 in [2], is a member of the C/D class of snoRNA which contain C (UGAUGA) and D (CUGA) box motifs [1]. U39 guides the methylation of 2'-O-ribose residues in 28S ribosomal RNA [1].\",SNORD55; U39; U55,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,182,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,103,0,,\"\",0.6686,-3.79524,1600000,241245,167,80,true,-3.4008,0.71858,2013-10-03T23:39:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00158,SNORD82,1443,Small nucleolar RNA SNORD82,Moxon SJ,Griffiths-Jones SR,43.0,43.6,42.8,\"snoRNA U82 or Z25 is found in both humans and mice. U82/Z25 is a member of the C/D class of snoRNA which contain C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs. Although the selection of the target nucleotide requires the antisense element and the conserved box D or D' of the snoRNA, the methyltransferase activity is supposed to reside in one of the protein components [1].\",U82,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,27,114,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,113,0,,\"\",0.60762,-4.91414,1600000,339544,151,70,true,-3.6848,0.71883,2013-10-03T23:39:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00159,snoZ168,2379,Small nucleolar RNA Z168/Z174,Moxon SJ,Griffiths-Jones SR,36.0,36.6,34.8,This family consists of several snoRNA Z168 and Z174 sequences which are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family seems to be found exclusively in rice. C/D box small nucleolar RNAs are known to act as a methylation guides for 18S ribosomal RNA in plants [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,53,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,11,0,,\"\",0.64081,-4.08611,1600000,286910,173,82,true,-3.5603,0.71858,2013-10-03T23:40:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00160,snoZ159,1727,Small nucleolar RNA Z159/U59,Moxon SJ,Griffiths-Jones SR,39.0,39.0,38.4,This family consists of several snoRNA Z159 or U59 sequences which are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family seems to be found exclusively in rice. C/D box small nucleolar RNAs are known to act as a methylation guides for 18S ribosomal RNA in plants [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,545,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,92,0,,\"\",0.58638,-4.10856,1600000,253281,238,87,true,-3.2814,0.7185,2013-10-03T23:40:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00161,nos_TCE,1446,Nanos 3' UTR translation control element,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",60.0,63.8,39.2,\"The Nanos protein in Drosophila is required for correct anterior/posterior patterning in the Drosophila embryo. Translation of the nanos mRNA is repressed in the bulk cytoplasm and activated in the posterior region. Repression is mediated by a translation control element (TCE) in the 3' UTR of the mRNA. This element forms a Y-shaped structure, part of which is recognised by the Smaug protein leading to translational repression [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.28 -Z 549862.597050 CM SEQDB,9,6,0,0,Cis-reg;,Published; PMID:10882131,11,0,,\"\",0.5683,-6.04495,1600000,276765,144,64,true,-4.0454,0.71926,2013-10-03T23:40:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00162,SAM,1447,SAM riboswitch (S box leader),\"Griffiths-Jones SR, Gardner PP, Ontiveros-Palacios N\",\"Grundy F, Henkin T, PMID:10094622\",44.4,44.4,44.3,\"The SAM riboswitch is found upstream of a number of genes which code for proteins involved in methionine or cysteine biosynthesis in Gram-positive bacteria. The SAM riboswitch acts at the level of transcription termination control. The predicted structure consists of a complex stem-loop region followed by a single stem loop terminator region. An alternative and mutually exclusive form involves bases in the 3' segment of helix 1 with those in the 5' region of helix 5 to form a structure termed the anti-terminator form [1]. This entry represents the conserved core of the complex structure involving helices 1-4. The structure of SAM riboswitch has been reported in PDBs: 2GIS, 2YDH, 2YGH, 3GX2, 3GX3, 3GX5, 3GX6, 3GX7, 3IQN, 3IQP, 3IQR, 3V7E_C, 3V7E_D, 4AOB, 4B5R, 4KQY, 5FJC, 5FK1, 5FK2, 5FK3, 5FK4, 5FK5, 5FK6, 5FKD, 5FKE, 5FKF, 5FKG, 5FKH, 7EAF [5-13]\",S_box,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,457,5704,0,0,Cis-reg; riboswitch;,Published; PMID:10094622,2540,0,,\"\",0.66961,-3.72866,1600000,213859,270,108,true,-3.7003,0.71848,2013-10-03T23:40:30.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00163,Hammerhead_1,1309,Hammerhead ribozyme (type I),Bateman A,Bateman A,34.0,34.0,33.9,\"The hammerhead ribozyme is one of the smallest catalytic RNAs. These RNAs have an endonuclease function, and most often are autocatalytic. Structurally it is composed of three base paired helices, separated by short linkers of conserved sequence. These helices are called I, II and III. We have classified hammerhead ribozymes into three types based on which helix the 5' and 3' ends of the sequence join. This family are the type I hammerheads. The conserved uridine-turn links helix I to helix II and usually has the sequence CUGA. Helix II and III are linked by a sequence CGAAA. The cleavage reaction occurs between helix III and I, and is usually a C. Hammerhead ribozymes are found in plant viroids and other small replicating satellite RNA species. Hammerhead ribozymes have been found in animals as well as plants.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.88 -Z 549862.597050 CM SEQDB,29,190650,0,0,Gene; ribozyme;,Published; PMID:9632772,24,0,,\"\",0.72644,-2.44555,1600000,284406,1024,46,true,-3.9333,0.7242,2013-10-03T23:40:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00164,s2m,1448,Coronavirus 3' stem-loop II-like motif (s2m),\"Griffiths-Jones SR, Nawrocki E\",\"PMID:9568965; Griffiths-Jones SR, Moxon SJ\",50.0,50.0,47.8,\"The s2m motif is found in the 3' UTRs of astrovirus, coronavirus and equine rhinovirus genomes [1]. Its function is unknown, but various viral 3' UTR regions have been found to play roles in viral replication and packaging. The seed alignment was expanded in 2020 by Eric Nawrocki.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,39,6,0,0,Cis-reg;,Published; PMID:15630477,22,0,,\"\",0.69273,-2.87863,1600000,269837,103,43,true,-4.275,0.7195,2013-10-03T23:42:02.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00165,Corona_pk3,1449,Coronavirus 3' UTR pseudoknot,\"Griffiths-Jones SR, Nawrocki E\",Griffiths-Jones SR,35.6,35.6,35.1,\"Coronaviruses are plus-strand RNA viruses with ~30kb genomes. The 3' UTR region of coronavirus genomes contains a conserved ~55nt pseudoknot structure which has been shown to be necessary for viral genome replication [1]. The mechanism of cis-regulation is unclear, but this element is postulated to function in the plus-strand. The seed alignment was expanded in 2020 by Eric Nawrocki.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,52,3,0,0,Cis-reg;,Published; PMID:10482585,55,0,,\"\",0.54986,-8.92456,1600000,538932,119,61,true,-3.3394,0.7229,2013-10-03T23:42:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00166,PrrB_RsmZ,2380,PrrB/RsmZ RNA family,\"Filiatrault M, Gardner P\",Filiatrault M,38.0,39.4,37.3,\"PrrB RNA is able to phenotypically complement gacS and gacA mutants and is itself regulated by the GacS-GacA two-component signal transduction system [1]. Inactivation of the prrB gene in Pseudomonas fluorescens F113 resulted in a significant reduction of 2, 4-diacetylphloroglucinol (Phl) and hydrogen cyanide (HCN) production, while increased metabolite production was observed when prrB was overexpressed [1]. The prrB gene sequence contains a number of imperfect repeats of the consensus sequence 5'-AGGA-3', and sequence analysis predicted a complex secondary structure featuring multiple putative stem-loops with the consensus sequences predominantly positioned at the single-stranded regions at the ends of the stem-loops. This structure is similar to the CsrB and RsmB regulatory RNAs Rfam:RF00018 [1], suggesting this RNA also interacts with a CsrA-like protein.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,37,102,0,0,Gene; sRNA;,\"Predicted; Filiatrault M, Gardner P\",94,0,,\"\",0.60821,-9.62275,1600000,474393,235,133,true,-3.1475,0.71824,2013-10-03T23:42:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00167,Purine,1451,Purine riboswitch,\"Boese B, Barrick JE, Breaker RR\",\"Boese B, Barrick JE\",37.0,37.0,36.9,\"Riboswitches are metabolite binding domains within certain messenger RNAs that serve as precision sensors for their corresponding targets. Allosteric rearrangement of mRNA structure is mediated by ligand binding, and this results in modulation of gene expression. A class of riboswitches that selectively recognises guanine and becomes saturated at concentrations as low as 5 nM. In Bacillus subtilis, this mRNA motif is located on at least five separate transcriptional units that together encode 17 genes that are mostly involved in purine transport and purine nucleotide biosynthesis. It has also been shown that some members of this family are specific for adenine instead. Hence this family are termed purine riboswitches.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,133,2528,0,0,Cis-reg; riboswitch;,Published; PMID:12787499,985,0,,\"\",0.56762,-2.93487,1600000,383852,179,102,true,-3.8568,0.71837,2013-10-03T23:42:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00168,Lysine,1452,Lysine riboswitch,\"Wickiser JK, Barrick JE, Breaker RR\",\"Wickiser JK, Barrick JE\",30.0,30.1,29.7,\"Riboswitches are metabolite binding domains within certain messenger RNAs that serve as precision sensors for their corresponding targets. Allosteric rearrangement of mRNA structure is mediated by ligand binding, and this results in modulation of gene expression. This family includes riboswitches that sense lysine [1] in a number of genes involved in lysine metabolism, including lysC [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,47,2214,0,0,Cis-reg; riboswitch;,Published; PMID:12787499,1379,0,,\"\",0.67764,-7.73091,1600000,337029,503,183,true,-3.4099,0.71403,2013-10-03T23:42:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00169,Bacteria_small_SRP,1315,Bacterial small signal recognition particle RNA,Gardner PP,Published; PMID:19838050,40.0,40.0,39.9,\"The signal recognition particle (SRP) is a universally conserved ribonucleoprotein involved in the co-translational targeting of proteins to membranes. The eukaryotic SRP consists of a 300-nucleotide 7S RNA (RFAM:RF00017) and six proteins: SRPs 72, 68, 54, 19, 14, and 9. Archaeal SRP consists of a 7S RNA and homologues of the eukaryotic SRP19 and SRP54 proteins. In most bacteria, the SRP consists of a 4.5S RNA and the Ffh protein (a homologue of the eukaryotic SRP54 protein). Some Gram-positive bacteria (e.g. Bacillus subtilis) have a longer eukaryote-like SRP RNA that includes an Alu domain [2]. The Alu domain of the longer SRP RNAs is not included in this model and therefore the reported bounds of some members are incorrect. The Signal Recognition Particle Database (SRPDB) [1] provides compilations of SRP components, with phylogenetic data and structural illustrations.\",SRP_bact_l; SRP_bact; 4.5S; 7SK; 7S; SCR,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.03 -Z 549862.597050 CM SEQDB,261,4458,0,0,Gene;,Published; PMID:19838050,4640,0,,\"\",0.66404,-2.40397,1600000,125609,149,97,true,-4.1441,0.71864,2013-10-03T23:42:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00171,Tombus_5,1454,Tombusvirus 5' UTR,\"Griffiths-Jones SR, Moxon SJ\",Griffiths-Jones SR,43.0,43.2,39.0,\"Tomato bushy stunt virus is the prototype member of the Tombusviridae family. In addition to viral proteins p33 and the RNA-dependent RNA polymerase p92, and unknown host factors, RNA signals within the UTRs are necessary for genome replication. 2 RNA domains in the 5' UTR have been reported [1,2], a 5' T-shaped domain (TSD) followed by a stem-loop (SL5) and a downstream domain (DSD). These domains are combined into a single family here. Reference [2] proposes that TSD-DSD interactions are involved in the mediation of viral RNA replication. The full alignment of this family contains a number of defective interfering (DI) deletion mutants, used to study RNA virus replication.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,8,0,0,Cis-reg;,Published; PMID:11162089,14,0,,\"\",0.53232,-9.48028,1600000,326108,331,166,true,-3.154,0.71612,2013-10-03T23:42:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00172,CAESAR,2381,ctgf/hcs24 CAESAR,Griffiths-Jones SR,Griffiths-Jones SR,45.0,48.8,38.4,\"The 3' UTR of human connective tissue growth factor/hypertrophic chondrocyte specific 24 (ctgf/hcs24) mRNA has been shown to repress ctgf gene expression [1]. Reference [2] identified a minimal RNA element, predicted to form a stable secondary structure, which acts as a post-transcriptional cis-acting element of structure-anchored repression (CAESAR), probably through direct interactions with as yet unknown nuclear factors.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,89,0,0,Cis-reg;,Published; PMID:11032028,89,0,,\"\",0.59258,-4.25998,1600000,256598,167,80,true,-3.151,0.71873,2013-10-03T23:42:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00173,Hairpin,1240,Hairpin ribozyme,Ditzler M,Ditzler M,65.0,73.3,38.1,\"The hairpin ribozyme, like the hammerhead (RFAM:RF00008 and RFAM:RF00163), is found in RNA satellites of plant viruses. It was first identified in the minus strand of the tobacco ringspot virus (TRSV) satellite RNA where it catalyses a self-cleavage reaction to process the products of rolling circle virus replication to unit-length satellite RNA. The structure consists of 2 domains, A and B. The 3' arm of domain A, which closes helices 1 and 2, contains the cleavage site and is linked to helix 3 of domain B by a linker of variable length and sequence. This region is thus not included in this family. The hairpin ribozyme has been identified in only 3 plant satellite RNAs -- from TRSV, chicory yellow mottle virus (CYMV) and arabis mosaic virus (ARMV).\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.92 -Z 549862.597050 --mxsize 128 CM SEQDB,5,1,0,0,Gene; ribozyme;,Published; PMID:7966321,4,0,,\"\",0.55144,-8.43568,1600000,492714,128,52,true,-3.7369,0.71926,2013-02-01T11:56:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00174,Cobalamin,1456,Cobalamin riboswitch,\"Barrick JE, Breaker RR, Moxon SJ, Gardner PP, Ontiveros-Palacios N\",\"Barrick JE, Breaker RR\",31.5,31.5,31.4,\"Riboswitches are metabolite binding domains within certain messenger RNAs that serve as precision sensors for their corresponding targets. Allosteric rearrangement of mRNA structure is mediated by ligand binding, and this results in modulation of gene expression [1]. This family represents a cobalamin riboswitch (B12-element) which is widely distributed in 5' UTRs of vitamin B(12)-related genes in bacteria. Cobalamin in the form of adenosylcobalamin (Ado-CBL) is known to repress expression of genes for vitamin B(12) biosynthesis and be transported by a post-transcriptional regulatory mechanism, which involves direct binding of Ado-CBL to 5' UTRs [2]. Cobalamin riboswitch structure was reported in PDB:4GMA [5], 4GXY [6] and 6VMY [7]. Cobalamin riboswitch forms a junctional structure that form a pocket that recognise adenosylcobalamin and is flanked by structural elements that connects by long-range interactions.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,433,13589,0,0,Cis-reg; riboswitch;,Predicted; PFOLD,5123,0,,\"\",0.69219,-7.73906,1600000,336073,446,189,true,-3.882,0.71341,2013-10-03T23:44:44.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00175,HIV-1_DIS,1457,Human immunodeficiency virus type 1 dimerisation initiation site,\"Bateman A, Daub J\",Bateman A,35.0,35.0,34.9,\"This cis element is composed of four hairpins called SL1 to SL4 (SL is for Stem Loop). This element is involved in encapsidation of the retroviral RNA an essential step in replication [1-3]. It is found in HIV, SIV and CIV. It has been shown that SL1 may provide a secondary binding site for Rev [4]. SL1 [5], SL2 [6], SL3 [7] and SL4 [8] have been independently expressed and structurally characterised.\",Retroviral_psi,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,70,17,0,0,Cis-reg;,Published; PMID:8627772,11,0,,\"\",0.54231,-8.11538,1600000,681074,101,40,true,-3.9272,0.71973,2013-10-03T23:45:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00176,Tombus_3_IV,1458,Tombusvirus 3' UTR region IV,Bateman A,Bateman A,33.0,33.5,31.7,\"Tomato bushy stunt virus is the prototype member of the Tombusviridae family. In addition to viral proteins p33 and the RNA-dependent RNA polymerase p92, and unknown host factors, RNA signals within the UTRs are necessary for genome replication. This element contains a pseudoknot [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,18,3,0,0,Cis-reg;,Published; PMID:12954222,10,0,,\"\",0.57335,-8.28423,1600000,313578,189,92,true,-3.5218,0.71865,2013-10-03T23:45:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00177,SSU_rRNA_bacteria,1459,Bacterial small subunit ribosomal RNA,\"Gutell RR, Nawrocki E\",Published; PMID:11869452,50.0,50.0,49.9,,SSU_rRNA_5,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.38 -Z 742849.287494 CM SEQDB,99,22081,0,0,Gene; rRNA;,Published; PMID:11869452,6977,0,,\"\",0.73917,-7.80744,1600000,269430,1851,1533,true,-5.9525,0.69565,2013-10-03T23:45:40.000Z,2022-10-29T17:55:16.000Z\n",
            "RF00178,mir-24,1460,mir-24 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,50.4,49.9,This family represents the microRNA (miRNA) mir-24 family. miRNAs are transcribed as ~70nt precursors (modelled here) and subsequently processed by the Dicer enzyme to give a ~22nt product. In this case the mature sequence comes from the 3' arm of the precursor. The extents of the hairpin precursors are not generally known and are estimated based on hairpin prediction. The products are thought to have regulatory roles through complementarity to mRNA. The involvement of Dicer in miRNA processing suggests a relationship with the phenomenon of RNA interference. miRNAs are numbered based on the sequence of the mature RNA. miR-24 has been identified in human and mouse [1]. Family represents the microRNA (miRNA) precursor mir-24 imported from miRBase [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,63,189,0,0,Gene; miRNA;,Predicted; RNAalifold,90,0,,\"\",0.67604,-3.36006,1600000,194197,152,69,true,-2.9226,0.7194,2013-10-04T00:09:45.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00179,GAIT,1461,GAIT element,Moxon SJ,See reference [1],37.0,37.9,35.2,\"This family represents a cis-acting element in the 3'-UTR of the ceruloplasmin (Cp) gene called gamma interferon inhibitor of translation (GAIT). The GAIT element is involved in selective translational silencing of the Cp transcript. Ceruloplasmin (Cp) is a multifunctional, copper-containing glycoprotein produced by the liver and secreted into the plasma. As an acute-phase protein, its plasma concentration can double during multiple inflammatory conditions. Plasma Cp has been reported to be an independent risk factor for cardiovascular disease, including atherosclerosis, carotid restenosis after endarterectomy, and myocardial infarction. Translational silencing of Cp, and possibly other transcripts, mediated by the GAIT element may contribute to the resolution of the local inflammatory response following cytokine activation of macrophages [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,32,0,0,Cis-reg;,Published; PMID:12588972,27,0,,\"\",0.62086,-3.07584,1600000,277816,160,71,true,-3.4317,0.71895,2013-10-04T00:09:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00180,REN-SRE,1462,Renin stability regulatory element (REN-SRE),Moxon SJ,PMID:12933794,41.0,41.0,39.6,\"This family represents a cis-acting element in the 3'-UTR of the renin (REN) gene called renin stability regulatory element (REN-SRE) which acts to regulate renin production. REN-SRE is thought to destabilise the mRNA, promoting degradation of the sequence. It has been found that a number of proteins bind to the REN-SRE to stabilise the mRNA and promote translation. Renin is secreted by renal juxtaglomerular cells and catalyses the conversion of angiotensinogen into angiotensin I which is the rate-limiting step in the production of angiotensin II. Angiotensin II induces hypertension and REN therefore requires tight control to ensure expression appropriate to need [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,36,0,0,Cis-reg;,Published; PMID:12933794,33,0,,\"\",0.67193,-7.73983,1600000,794799,102,37,true,-3.7515,0.72007,2013-10-04T00:10:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00181,SNORD113,1463,Small nucleolar RNA SNORD113/SNORD114 family,\"Griffiths-Jones SR, Daub J\",PMID:12045206,36.0,36.0,35.9,\"The imprinted human domain at 14q32 contains two clusters of tandemly repeated small nucleolar RNAs named 14q(I) and 14q(II) snoRNAs. These two clusters are named SNORD113 and SNORD114 respectively in the HGNC approved gene symbol nomenclature. The snoRNAs found in both clusters are clearly related and are represented here by a single family. These snoRNAs are intron encoded and are processed from the tissue-specific non-coding human MEG8 RNA, which lies downstream of the imprinted genes DLK1 and GTL2. snoRNAs 14q(I) and 14q(II) differ from previously identified C/D box snoRNAs in their expression profiles and lack of complementarity to rRNA and snRNA. Homologues of these snoRNAs are found in the imprinted non-coding mouse transcript Rian, and in the rat non-coding Bsr (brain-specific repetitive) RNA.\",sno_14q_I_II; sno_14q_I,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,59,1367,0,0,Gene; snRNA; snoRNA; CD-box;,Published; PMID:12045206,40,0,,\"\",0.60854,-4.005,1600000,251093,228,75,true,-3.5659,0.71868,2013-10-04T00:10:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00182,Corona_package,1464,Coronavirus packaging signal,\"Moxon SJ, Petrov AI\",PMID:17428856,80.0,101.7,38.2,\"This family represents a predicted coronavirus packaging signal. Within a virally infected cell, viral genome and viral proteins specifically and selectively unite into progeny viruses. This process is called packaging and is vital for viral replication. Viral RNA containing a packaging signal allows the sequence to be efficiently packaged into the virion. The family was reviewed in 2020 to ensure that all 4 AGC/GUAAU elements were present in SS_cons and to include more sequences in the seed. Previously this family was based on PMID:12791173 which incorrectly claimed that this structure is found in SARS-CoV.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,6,15,0,0,Cis-reg;,Published; PMID:17428856,13,0,,\"\",0.6041,-3.68708,1600000,252546,209,107,true,-3.4173,0.71848,2013-10-04T00:10:18.000Z,2020-04-23T12:35:20.000Z\n",
            "RF00183,G-CSF_SLDE,1465,G-CSF factor stem-loop destabilising element (SLDE),Moxon SJ,PMID:11865046,45.0,46.6,40.8,\"This family represents the granulocyte colony-stimulating factor stem-loop destabilising element (G-CSF SLDE). G-CSF is secreted by fibroblasts and endothelial cells in response to the inflammatory mediators interleukin-1 (IL-1) and tumour necrosis factor alpha and by activated macrophages. The synthesis of G-CSF is regulated both transcriptionally and through control of mRNA stability. In unstimulated cells G-CSF mRNA is unstable but becomes stabilised in response to IL-1 or tumour necrosis factor alpha, and also in the case of monocytes and macrophages, in response to lipopolysaccharide. It is likely that the presence of the SLDE in the G-CSF mRNA contributes to the specificity of regulation of G-CSF mRNA and enhances the rate of shortening of the poly(A) tail [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,37,0,0,Cis-reg;,Published; PMID:11865046,40,0,,\"\",0.54365,-7.72831,1600000,282580,189,96,true,-3.2169,0.71864,2013-10-04T00:10:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00184,PVX_3,1466,Potato virus X cis-acting regulatory element,Moxon SJ,PMID:12581634,42.0,44.0,38.1,This family represents a cis-acting regulatory element which is found in the 3' UTR of the Potato virus X genome. This element has been found to be required for minus strand RNA accumulation and is therefore essential for efficient viral replication [1].,PVX_cis-reg,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,4,1,0,0,Cis-reg;,Published; PMID:12581634,2,0,,\"\",0.60707,-8.10606,1600000,575263,134,54,true,-3.8797,0.7192,2013-10-04T00:10:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00185,Flavi_CRE,1467,Flavivirus 3' UTR cis-acting replication element (CRE),\"Bateman A, Moxon SJ, Weinberg Z\",\"Bateman A, Moxon SJ, Weinberg Z\",42.0,42.5,41.9,\"This family represents the Flavivirus 3'-SL cis-acting replication element (CRE). The family is thought to play a crucial role in replication and is thought to be needed for the formation of a \\\"replication complex\\\" [1]. Although evidence has been presented for an existence of a pseudoknot structure in this RNA [2], it does not appear to be well conserved across flaviviruses. Deletions of the 3' UTR of flaviviruses have been shown to be lethal for infectious clones.\",Flavi_pk3,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,84,4,0,0,Cis-reg;,\"Published; PMID:9696848; Bateman A, Moxon SJ, Weinberg Z\",19,0,,\"\",0.55812,-5.53421,1600000,265985,180,96,true,-3.5876,0.71863,2013-10-04T00:10:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00186,SNORD101,1468,Small nucleolar RNA SNORD101,Moxon SJ,\"PMID:14602913, INFERNAL\",47.0,48.1,41.6,snoRNA U101 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs and guide the 2'-O-methylation of rRNA and snRNAs. The precise target of this family is unknown [1].,U101,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,26,114,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,111,0,,\"\",0.61265,-4.113,1600000,306392,157,73,true,-3.6021,0.71876,2013-10-04T00:11:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00187,SNORD102,1469,Small nucleolar RNA SNORD102,\"Moxon SJ, Daub J\",\"PMID:14602913, INFERNAL\",47.0,48.4,44.4,snoRNA U102 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. It is found within intron 3 of the L21 ribosomal protein gene. The H/ACA box snoRNA ACA27 is found in the same host gene within a different intron [1]. This family is predicted to direct a highly conserved 2'-O-methylation in 28S rRNA [2].,U102,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,88,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,88,0,,\"\",0.54594,-3.76082,1600000,269432,158,72,true,-3.1618,0.71873,2013-10-04T00:11:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00188,SNORD103,1470,Small nucleolar RNA SNORD103/SNORD85,\"Moxon SJ, Daub J\",\"Moxon SJ, Daub J, INFERNAL\",45.0,45.2,43.6,\"This family includes the snoRNAs SNORD103 (U103) and SNORD85 (HBII-251) . The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. In the human genome, there are two identical copies of SNORD103 (SNORD103A and SNORD103B) which share the same host gene with SNORD85. All three snoRNAs are predicted to guide the 2'O-ribose methylation of 18S rRNA G601 [3].\",U103,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,41,276,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Moxon SJ,109,0,,\"\",0.63641,-4.06792,1600000,268688,171,83,true,-3.428,0.71858,2013-10-04T00:11:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00189,SNORD95,1471,Small nucleolar RNA SNORD95,Moxon SJ,PMID:14602913,55.0,56.5,54.9,\"snoRNA U95 (also referred to as Z38 ) is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family is predicted to target two neighbouring 2'-O-methylations, Cm2801 and Am2792, in 28S rRNA [1].\",U95,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.00 -Z 549862.597050 CM SEQDB,7,58,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,38,0,,\"\",0.67033,-3.73888,1600000,209151,154,68,true,-3.9351,0.71886,2013-10-04T00:11:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00190,SNORA16,1472,Small nucleolar RNA SNORA16B/SNORA16A family,Moxon SJ,PMID:14602913,35.0,35.0,34.9,\"The members of this family belong to the H/ACA class of snoRNAs which are predicted to guide the sites of modification of uridines to pseudouridines [2]. This family includes human snoRNAs U98b (SNORA16B) and ACA16 (SNORA16A) [1,2]. There are currently no predicted targets RNAs for either of these snoRNAs [1,2].\",U98,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,29,288,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,122,0,,\"\",0.55904,-8.53466,1600000,288072,237,135,true,-3.3118,0.71801,2013-10-04T00:12:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00191,SNORA57,1473,Small nucleolar RNA SNORA57,Moxon SJ,PMID:14602913,40.0,40.4,38.7,\"U99 belongs to the H/ACA class of snoRNAs which are thought to guide the sites of modification of uridines to pseudouridines, the target for this family is unknown [1].\",U99,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,520,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,83,0,,\"\",0.63468,-7.38569,1600000,254360,259,148,true,-3.2105,0.71807,2013-10-04T00:12:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00192,BLV_package,1474,Bovine leukaemia virus RNA packaging signal,Moxon SJ,PMID:12359429,70.0,85.4,38.5,This family represents the bovine leukaemia virus RNA encapsidation (packaging) signal which is essential for efficient viral replication [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,5,1,0,0,Cis-reg;,Published; PMID:12359429,1,0,,\"\",0.57604,-10.03886,1600000,751814,136,55,true,-3.1795,0.71929,2013-10-04T00:12:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00193,CTV_rep_sig,1475,Citrus tristeza virus replication signal,Moxon SJ,PMID:12202214,100.0,259.1,40.0,This family represents a predicted replication signal which is highly conserved in citrus tristeza viruses. Replication signals are required for viral replication and are usually found near the 5' and 3' termini. This family is predicted to form ten stem loop structures some of which are essential for functions that provide for efficient viral replication [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,1,0,0,Cis-reg;,Published; PMID:12202214,1,0,,\"\",0.81427,-6.75936,1600000,361039,423,273,true,-3.6835,0.7072,2013-10-04T00:12:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00194,Rubella_3,1476,Rubella virus 3' cis-acting element,Moxon SJ,PMID:10074193,35.0,35.2,34.6,\"This family represents a cis-acting element found at the 3' UTR in the rubella virus. This family contains three conserved step loop structures. Pfam:PF00262 (CAL) which is known to bind calcium in most eukaryotic cells, is able to specifically bind to the first stem loop. CAL binding is thought to be related to viral pathogenesis and in particular arthritis which is a occurs frequently in rubella infections in adults and is independent of viral viability. All stem loop structures are thought to be important for efficient viral replication and deletion of stem loop three is known to be lethal [1].\",Rubella_cis_act,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,3,0,0,Cis-reg;,Published; PMID:10074193,3,0,,\"\",0.55241,-8.00253,1600000,354525,155,68,true,-3.5083,0.71912,2013-10-04T00:12:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00195,RsmY,1477,RsmY RNA family,Griffiths-Jones SR,Griffiths-Jones SR,57.0,59.8,49.0,\"The rsmY gene, like rsmZ (Rfam:RF00166), is regulated by the GacS/GacA signal transduction system in the plant-beneficial soil bacterium and biocontrol model organism Pseudomonas fluorescens CHA0. GacA/GacS target genes are translationally repressed by the small RNA binding protein RsmA. RsmY and RsmZ RNAs bind RsmA to relieve this repression and so enhance secondary metabolism and biocontrol traits [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,71,0,0,Gene; sRNA;,Predicted; PFOLD,76,0,,\"\",0.63481,-9.35788,1600000,493225,225,121,true,-2.7516,0.71828,2013-10-04T00:12:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00196,AMV_RNA1_SL,1478,Alfalfa mosaic virus RNA 1 5' UTR stem-loop,Moxon SJ,PMID:14512577,60.0,62.7,36.7,This family represents a putative stem-loop structure found in the 5' UTR in RNA 1 of alfalfa mosaic virus. RNA 1 is responsible for encoding the viral replicase protein P1. This family is required for negative strand RNA synthesis in in the alfalfa mosaic virus and may also be involved in positive strand RNA synthesis [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,1,0,0,Cis-reg;,Published; PMID:14512577,1,0,,\"\",0.56039,-5.54949,1600000,609942,103,35,true,-3.5889,0.72178,2013-10-04T00:13:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00198,SL1,1479,SL1 RNA,Moxon SJ,\"Moxon SJ, Daub J\",75.0,75.1,74.1,This family represents the SL1 RNA. SL1 is commonly located in the spacer region between 5S-rRNA genes where it is involved in trans-splicing. Trans-splicing is a form of RNA processing. The acquisition of a spliced leader from a SL RNA is a intra-molecular reaction which precisely joins exons derived from separately transcribed RNAs. This mechanism of mRNA maturation has been shown to occur in a number of lower eukaryotes and is commonplace in nematodes [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 549862.597050 CM SEQDB,28,943,0,0,Gene;,Published; PMID:12949121; Moxon SJ,21,0,,\"\",0.51107,-8.60737,1600000,314581,205,103,true,-3.3581,0.71855,2013-10-04T00:13:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00199,SL2,1480,SL2 RNA,Moxon SJ,Moxon SJ,41.0,41.0,40.6,This family represents SL2 RNA. SL2 RNA is involved in trans splicing in lower eukaryotes. Trans-splicing is a form of RNA processing. The acquisition of a spliced leader from a SL RNA is a intra-molecular reaction which precisely joins exons derived from separately transcribed RNAs [2]. Approximately 25% of C. elegans genes are organised into polycistronic transcription units and the presence of SL2 on an mRNA is an indication the the gene is in an operon [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30 -Z 549862.597050 CM SEQDB,30,345,0,0,Gene;,Published; PMID:10958663; Moxon SJ,21,0,,\"\",0.44299,-10.06104,1600000,383742,212,110,true,-3.3545,0.71846,2013-10-04T00:13:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00200,snoZ199,1481,Small nucleolar RNA Z199,Griffiths-Jones SR,Griffiths-Jones SR,47.0,47.1,46.8,\"snoZ199 is a C/D box small nucleolar RNA, which acts as a methylation guide RNA for sites on 18S and 25S rRNA. Rice snoZ199 is reported to be homologous to Arabidopsis snoR13.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,17,406,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,75,0,,\"\",0.53679,-4.08008,1600000,200201,245,84,true,-3.2018,0.71865,2013-10-04T00:13:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00201,snoZ278,1482,Small nucleolar RNA Z278,Griffiths-Jones SR,Griffiths-Jones SR,36.0,36.1,35.9,snoZ278 is a C/D box small nucleolar RNA. Its methylation guide targets have not been reported.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,375,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,70,0,,\"\",0.60014,-5.52264,1600000,253347,222,115,true,-3.3018,0.71821,2013-10-04T00:13:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00202,snoR66,1483,Small nucleolar RNA R66,Griffiths-Jones SR,Griffiths-Jones SR,43.0,43.0,38.2,\"snoR66 is a C/D box small nucleolar RNA, which acts as a methylation guide for 18S ribosomal RNA in plants [1]. snoR66 has been alternatively named snoZ269 in rice.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,232,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,74,0,,\"\",0.62752,-4.48462,1600000,262101,165,76,true,-3.3936,0.7187,2013-10-04T00:13:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00203,snoR160,1484,Small nucleolar RNA R160,Griffiths-Jones SR,Griffiths-Jones SR,42.0,42.9,41.9,\"snoR160 is a C/D box small nucleolar RNA, which acts as a methylation guide for 25S ribosomal RNA in plants [1]. snoR160 has been alternatively named snoZ270 in rice.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,321,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones SR,72,0,,\"\",0.54165,-2.92108,1600000,239093,167,80,true,-3.2832,0.71859,2013-10-04T00:13:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00204,snoR12,1247,Small nucleolar RNA R12,Griffiths-Jones SR,Griffiths-Jones SR,39.0,43.7,37.2,\"snoR12 is a C/D box small nucleolar RNA, which acts as a methylation guide for 25S ribosomal RNA in plants [1]. snoR12 has been alternatively named snoZ44 in Arabidopsis and snoZ131 in rice.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,392,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD, Griffiths-Jones SR\",74,0,,\"\",0.58399,-4.32508,1600000,250785,174,83,true,-3.4659,0.71849,2013-02-01T11:56:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00205,snoR41,1485,Small nucleolar RNA R41,Griffiths-Jones SR,Griffiths-Jones SR,40.0,41.2,39.4,\"snoR41 is a C/D box small nucleolar RNA, which acts as a methylation guide for 18S and 25S ribosomal RNA in plants [1]. snoR41 has been alternatively named snoZ154 and snoZ231 in rice. The 18S methylation site is reported to be homologous to that targeted by U62 in humans (Rfam:RF00153) [1], but the snoRNA sequence similarity is not significant.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,7,171,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones SR,76,0,,\"\",0.56773,-3.68022,1600000,206526,202,97,true,-3.3851,0.71842,2013-10-04T00:17:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00206,U54,1486,Small nucleolar RNA U54,Griffiths-Jones SR,Griffiths-Jones SR,40.0,41.6,39.8,\"U54 is a C/D box small nucleolar RNA, which acts as a methylation guide for 18S ribosomal RNA in plants [1].\",SNORD54; U54,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,263,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD, Griffiths-Jones SR\",72,0,,\"\",0.56841,-4.91593,1600000,308921,167,81,true,-3.5213,0.71862,2013-10-04T00:17:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00207,K10_TLS,2382,K10 transport/localisation element (TLS),\"Holmes I, Moxon SJ\",PMID:8582290,43.0,45.2,41.7,\"This family represents the 44 nucleotide K10 transport/localisation element (TLS) from Drosophila melanogaster. K10 TLS is responsible for the transport and anterior localisation of K10 mRNA and acts to establish dorsoventral polarity in the oocyte [1]. K10 TLS structure was reported in the PDBs 2KE6, 2KUR, 2KUU, 2KUV, and 2KUW [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,7,0,0,Cis-reg;,Published; PMID:8582290,7,0,,\"\",0.70231,-2.50553,1600000,333042,121,48,true,-3.632,0.71954,2013-10-04T00:17:16.000Z,2021-06-14T15:02:32.000Z\n",
            "RF00208,snoR72,1488,Small nucleolar RNA R72,Griffiths-Jones SR,Griffiths-Jones SR,43.0,43.0,42.4,\"snoR72 is a C/D box small nucleolar RNA, which acts as a methylation guide for 25S ribosomal RNA in Arabidopsis [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,27,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD, Griffiths-Jones SR\",15,0,,\"\",0.51401,-3.68233,1600000,233856,184,87,true,-3.0644,0.71847,2013-10-04T00:17:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00209,IRES_Pesti,1489,Pestivirus internal ribosome entry site (IRES),Moxon SJ,PMID:11142379,45.0,47.2,43.2,This family represents the internal ribosome entry site (IRES) of the the pestiviruses. The pestivirus IRES allows cap and end-independent translation of mRNA in the host cell. The IRES achieves this by mediating the internal initiation of translation by recruiting a ribosomal 43S pre-initiation complex directly to the initiation codon and eliminates the requirement for eIF4F [1]. The classical swine virus UTR described in [1] appears to be longer at the 5' end than other pestivirus UTRs. This family represents the conserved core.,Pesti_IRES,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,1,0,0,Cis-reg; IRES;,Published; PMID:11142379,10,0,,\"\",0.76013,-8.23616,1600000,426310,420,274,true,-3.6033,0.70714,2013-10-04T00:17:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00210,IRES_Picorna_2,1490,Aphthovirus internal ribosome entry site (IRES),Moxon SJ,PMID:14561883,50.0,54.2,32.0,This family represents the internal ribosome entry site (IRES) of the the aphthoviruses. IRES elements allow cap and end-independent translation of mRNA in the host cell. The IRES achieves this by mediating the internal initiation of translation by recruiting a ribosomal 43S pre-initiation complex directly to the initiation codon and eliminates the requirement for eIF4F [2].,Aptho_IRES; IRES_Aptho,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 16.93 -Z 742849.287494 --mxsize 128 CM SEQDB,92,10,0,0,Cis-reg; IRES;,Published; PMID:14561883,27,0,,\"\",0.75096,-8.44153,1600000,406550,744,462,true,-4.8064,0.70145,2013-10-04T00:17:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00211,SNORD35,1491,Small nucleolar RNA SNORD35,Griffiths-Jones SR,Griffiths-Jones SR,36.0,36.2,35.9,U35 is a C/D box snoRNA located in intron 6 of ribosomal protein L13A and intron 3 of ribosomal protein S11 in humans [1] and at homologous positions in mouse and chicken ribosomal protein genes. U35 guides the methylation of 2'-O-ribose sites in 28S rRNA.,U35,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,313,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD, Griffiths-Jones SR\",108,0,,\"\",0.67763,-3.71023,1600000,257237,179,88,true,-3.566,0.71845,2013-10-04T00:17:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00212,SNORD38,1492,Small nucleolar RNA SNORD38,Griffiths-Jones SR,Griffiths-Jones SR,44.0,44.4,43.8,U38 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U38 is located in introns 4 and 5 of ribosomal protein S8 in human and in homologous genes in mouse and cow [1]. U38 guides the methylation of 2'-O-ribose residues in 28S rRNA. Mouse MBII-329 snoRNA is homologous to U38 [2].,U38,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,270,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD, Griffiths-Jones SR\",111,0,,\"\",0.61882,-4.00898,1600000,282583,154,70,true,-3.5616,0.71891,2013-10-04T00:17:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00213,snoR38,1493,Small nucleolar RNA R38,Griffiths-Jones SR,INFERNAL,45.0,45.0,44.9,\"snoR38 is a C/D box small nucleolar RNA conserved across yeast [1], human, Arabidopsis [2] and rice [3]. It is encoded in the intron of elongation factor-1 gamma in both pombe and cerevisiae. snoR38 guides the methylation of 2'-O-ribose sites in 28S rRNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,715,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD, Griffiths-Jones SR\",406,0,,\"\",0.51865,-4.52442,1600000,220268,233,91,true,-3.7104,0.71841,2013-10-04T00:17:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00214,Retro_dr1,1494,Retrovirus direct repeat 1 (dr1),Moxon SJ,PMID:10438832,42.0,43.7,35.4,\"This family represents the direct repeat 1 (dr1) element commonly found in the 3' UTR of Avian sarcoma, Rous sarcoma and Avian leukosis viruses (Alpharetroviruses and Avian type C retroviruses). dr1 is required for efficient viral replication and is thought to be involved in genomic RNA packaging although this may not be its only role [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,26,94,0,0,Cis-reg;,Published; PMID:10438832,11,0,,\"\",0.61354,-5.59833,1600000,303488,185,90,true,-3.3296,0.71874,2013-10-04T00:17:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00215,Tombus_3_III,1495,Tombus virus defective interfering (DI) RNA region 3,\"Moxon SJ, Daub J\",PMID:12477830,35.0,37.8,34.7,This family represents region 3 of the defective interfering (DI) RNA element of the Tombus viruses and is found in the 3' UTR. This family has been found to enhance DI RNA accumulation by approximately 10-fold as well as mediating viral replication [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,1,0,0,Cis-reg;,Published; PMID:12477830,10,0,,\"\",0.55243,-9.19415,1600000,505153,148,70,true,-3.8207,0.71889,2013-10-04T00:17:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00216,IRES_c-myc,1496,c-myc internal ribosome entry site (IRES),Moxon SJ,PMID:11419940,40.0,41.4,38.5,\"This family represents the c-myc internal ribosome entry site (IRES). The mammalian c-myc gene is a proto-oncogene which is required for cell proliferation, transformation and death. c-myc mRNA has an alternative method of translation via internal ribosome entry where ribosomes are recruited to the IRES located in the 5' UTR thus bypassing the typical eukaryotic cap-dependent translation pathway [1].\",c-myc_IRES,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,35,0,0,Cis-reg; IRES;,Published; PMID:11419940,36,0,,\"\",0.68763,-9.70545,1600000,423241,457,302,true,-3.7305,0.70582,2013-10-04T00:18:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00217,SNORD20,1497,Small nucleolar RNA SNORD20,Griffiths-Jones SR,\"Griffiths-Jones SR, INFERNAL\",43.0,45.4,42.8,\"U20 is a C/D box small nucleolar RNA that is encoded in intron 11 of the nucleolin gene in human, mouse and rat [1].\",U20,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,101,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones SR,97,0,,\"\",0.55741,-4.12863,1600000,232802,169,80,true,-2.9055,0.71865,2013-10-04T00:18:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00218,SNORD46,1498,Small nucleolar RNA SNORD46,\"Griffiths-Jones SR, Eberhardt R\",Griffiths-Jones SR,40.0,43.1,39.3,\"U40 is a C/D box small nucleolar RNA, also known as snoZ153 in plants. The human sequence has also been annotated in the sequence databases as U46. U40 is encoded in intron 2 of the ribosomal protein S8 gene in human, and is hypothesised to guide methylation of 2'-O-ribose residues on 28S ribosomal RNA [1]. This family represents vertebrate U40.\",U40,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.00 -Z 549862.597050 CM SEQDB,5,131,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones SR,106,0,,\"\",0.66504,-3.76092,1600000,246631,206,104,true,-3.1881,0.71827,2013-10-04T00:18:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00220,Rhino_CRE,1499,Human rhinovirus internal cis-acting regulatory element (CRE),Moxon SJ,PMID:9848654,45.0,47.4,40.3,This family represents a cis-acting regulatory element (CRE) from the human rhinoviruses. The CRE is located within the genome segment encoding the capsid proteins so is found in a protein coding region. The family has been found to be essential for efficient viral replication and it has been suggested that the CRE is required for initiation of minus-strand RNA synthesis [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,0,0,0,Cis-reg;,Published; PMID:9848654,10,0,,\"\",0.60763,-3.93195,1600000,193600,178,86,true,-3.736,0.71869,2013-10-04T00:18:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00221,SNORD43,1500,Small nucleolar RNA SNORD43,\"Griffiths-Jones SR, Eberhardt R\",Griffiths-Jones SR,45.0,45.4,44.8,\"U43 is a C/D box small nucleolar RNA that is encoded in intron 1 of the ribosomal protein L3 gene in human and cow [2]. Three other snoRNAs ( U82, U83a and U83b) are also encoded in the same host gene but from different introns [2]. The rice homologue is expressed from a cluster also containing snoR16 [1]. The Arabidopsis homologue is annotated as snoR41 in the sequence databases. U43 is hypothesised to guide methylation of 2'-O-ribose residues on 18S ribosomal RNA [1]. This family represents mammalian U43.\",U43,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.00 -Z 549862.597050 CM SEQDB,4,79,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones SR,38,0,,\"\",0.65705,-3.67693,1600000,258316,147,64,true,-3.9027,0.71894,2013-10-04T00:18:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00222,IRES_Bag1,1501,Bag-1 internal ribosome entry site (IRES),Moxon SJ,\"http://ifr31w3.toulouse.inserm.fr/IRESdatabase/, INFERNAL\",37.0,37.0,36.9,This family represents the bag-1 internal ribosome entry site (IRES). When expressed the BAG-1 protein is known to enhance the anti-apoptotic properties of Bcl-2. Although bag-1 translation usually occurs via a cap-dependent mechanism it has been found to contain a IRES in its 5' UTR. Translation via the IRES has been found to be common following heat shock when cap-dependent scanning is compromised [1].,Bag1_IRES,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,55,0,0,Cis-reg; IRES;,Predicted; PFOLD,42,0,,\"\",0.58978,-11.79201,1600000,700053,529,370,true,-3.9969,0.70348,2013-10-04T00:20:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00223,IRES_Bip,1502,bip internal ribosome entry site (IRES),Moxon SJ,\"http://ifr31w3.toulouse.inserm.fr/IRESdatabase/, INFERNAL\",40.0,43.2,39.2,This family represents the bip internal ribosome entry site (IRES). BiP protein expression has been found to be significantly enhanced by the heat shock response due to IRES-dependent translation. It is thought that this translational mechanism is essential for the survival of cells under stress [1].,Bip_IRES,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,67,0,0,Cis-reg; IRES;,Predicted; PFOLD,37,0,,\"\",0.71371,-10.5913,1600000,638004,415,263,true,-3.9985,0.70771,2013-10-04T00:20:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00224,IRES_FGF2,1503,FGF-2 internal ribosome entry site (IRES),Moxon SJ,\"http://ifr31w3.toulouse.inserm.fr/IRESdatabase/, INFERNAL\",35.0,35.2,34.9,\"This family represents the fibroblast growth factor 2 (FGF-2) internal ribosome entry site (IRES). When expressed the FGF-2 protein plays a pivotal role in cell proliferation, differentiation and survival as well as being involved in wound-healing [1,2]. It has been found that FGF-2 IRES activity is strictly controlled and highly tissue specific. It is thought that translational IRES dependent activation of FGF-2 plays a vital role in embryogenesis and in the adult brain [1].\",IRES_FGF; FGF_IRES,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,17,0,0,Cis-reg; IRES;,Predicted; PFOLD,18,0,,\"\",0.54334,-12.15671,1600000,483878,686,498,true,-4.2029,0.70087,2013-10-04T00:20:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00225,IRES_Tobamo,1504,Tobamovirus internal ribosome entry site (IRES),Moxon SJ,http://ifr31w3.toulouse.inserm.fr/IRESdatabase/,50.0,57.1,39.0,This family represents the Tobamovirus internal ribosome entry site (IRES). IRES elements allow cap and end-independent translation of mRNA in the host cell. The IRES achieves this by mediating the internal initiation of translation by recruiting a ribosomal 43S pre-initiation complex directly to the initiation codon and eliminates the requirement for eIF4F [2].,Tobamo_IRES,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,1,0,0,Cis-reg; IRES;,Published; PMID:9185586,5,0,,\"\",0.44596,-10.94851,1600000,438124,270,151,true,-3.1168,0.71779,2013-10-04T00:20:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00226,IRES_n-myc,1505,n-myc internal ribosome entry site (IRES),Moxon SJ,PMID:11419940,40.0,45.0,32.1,\"This family represents the n-myc internal ribosome entry site (IRES). The myc family of genes when expressed are known to be involved in the control of cell growth, differentiation and apoptosis. n-myc mRNA has an alternative method of translation via internal ribosome entry where ribosomes are recruited to the IRES located in the 5' UTR thus bypassing the typical eukaryotic cap-dependent translation pathway [1].\",n-myc_IRES,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,62,0,0,Cis-reg; IRES;,Predicted; PFOLD,33,0,,\"\",0.80616,-8.56659,1600000,483111,475,322,true,-4.138,0.70497,2013-10-04T00:20:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00227,FIE3,1506,FIE3 (ftz instability element 3') element,\"Bateman A, Griffiths-Jones SR\",PMID:8649416,40.0,40.3,39.4,\"The fushi tarazu gene is essential for the establishment of the Drosophila embryonic body plan. When first expressed in early embryogenesis, fushi tarazu mRNA is uniformly distributed over most of the embryo. Subsequently, fushi tarazu mRNA expression rapidly evolves into a pattern of seven stripes that encircle the embryo. The instability of fushi tarazu mRNA is probably crucial for attaining this localised pattern of expression [1]. Experiments provide evidence for at least two destabilising elements in the fushi tarazu mRNA, one located within the 5' one-third of the mRNA and the other near the 3' end (termed FIE3 for ftz instability element 3'). The FIE3 lies within a 201-nucleotide sequence just upstream of the polyadenylation signal and can act autonomously to destabilise a heterologous mRNA. Further deletion constructs identified an essential 68-nucleotide element within the FIE3 [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,8,0,0,Cis-reg;,\"Predicted; stemloc, Bateman A\",11,0,,\"\",0.46445,-7.35881,1600000,385132,253,65,true,-3.4566,0.71953,2013-10-04T00:20:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00228,IRES_HepA,1507,Hepatitis A virus internal ribosome entry site (IRES),Moxon SJ,http://ifr31w3.toulouse.inserm.fr/IRESdatabase/,55.0,58.2,36.8,This family represents the internal ribosome entry site (IRES) of the the Hepatitis A virus. IRES elements allow cap and end-independent translation of mRNA in the host cell. The IRES achieves this by mediating the internal initiation of translation by recruiting a ribosomal 43S pre-initiation complex directly to the initiation codon and eliminates the requirement for eIF4F [2].,HepA_IRES,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,1,0,0,Cis-reg; IRES;,Predicted; PFOLD,1,0,,\"\",0.73986,-8.40112,1600000,391319,774,572,true,-4.6982,0.6998,2013-10-04T00:20:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00229,IRES_Picorna,1508,Picornavirus internal ribosome entry site (IRES),Moxon SJ,http://ifr31w3.toulouse.inserm.fr/IRESdatabase/,33.0,33.0,32.9,\"This family represents the Picornavirus internal ribosome entry site (IRES). IRES elements allow cap and end-independent translation of mRNA in the host cell. It has been found that La autoantigen (La) is required for Coxsackievirus B3 (CVB3) IRES-mediated translation, and it has been suggested that La may be required for the efficient translation of the viral RNA in the pancreas [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,92,9,0,0,Cis-reg; IRES;,Predicted; PFOLD,58,0,,\"\",0.67968,-9.28524,1600000,405001,390,252,true,-4.18,0.70835,2013-10-04T00:20:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00230,T-box,1509,T-box leader,\"Barrick JE, Bateman A, Vitreschak AG, Gelfand MS, Gardner PP\",\"Barrick JE, predicted; Vitreschak A, PMID:12770710\",93.0,93.0,92.9,\"Uncharged tRNA acts as the effector for transcription antitermination of genes in the T-box leader family [1-3]. T-boxes are found upstream of many aminoacyl-tRNA synthetase genes and some amino acid biosynthetic genes usually in Gram-positive bacteria. The anticodon of a specific tRNA base pairs to a specifier sequence within the T-box motif, and the NCCA acceptor tail of the tRNA base pairs to a conserved bulge in the T-box antiterminator hairpin [4].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.50 -Z 549862.597050 CM SEQDB,48,12267,0,0,Cis-reg; leader;,Published; PMID:12770710,1421,0,,\"\",0.59513,-9.83089,1600000,407865,1204,224,true,-3.4659,0.71018,2013-10-04T00:20:52.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00231,SCARNA13,1248,Small Cajal body specific RNA 13,\"Moxon SJ, Daub J\",\"PMID:12409454, INFERNAL\",49.0,49.2,41.6,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. U93 is composed of two tandemly arranged box H/ACA RNA motifs and belongs to the H/ACA class of guide RNAs. U93 is predicted to guide pseudouridylation of the U2 spliceosomal snRNA [2].\",U93,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,24,136,0,0,Gene; snRNA; snoRNA; scaRNA;,Published; PMID:12409454,112,0,,\"\",0.66227,-8.79757,1600000,323344,412,275,true,-4.1725,0.70712,2013-02-01T11:57:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00232,Spi-1,1510,Spi-1 (PU.1) 5' UTR regulatory element,Moxon SJ,PMID:9207037,40.0,61.0,37.8,Spi-1 regulates myeloid gene expression during haemopoietic development. This family represents a regulatory element found in the 5' UTR of Spi-1 mRNA which is able to inhibit the translation Spi-1 transcripts by 8-fold. Mutations in this regulatory region of the 5' UTR can lead to overexpression of Spi-1 which has been linked to erythroleukemic transformation [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,5,37,0,0,Cis-reg;,Published; PMID:9207037,36,0,,\"\",0.52976,-9.55199,1600000,319555,292,168,true,-3.0075,0.71615,2013-10-04T00:21:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00234,glmS,1512,glmS glucosamine-6-phosphate activated ribozyme,\"Winkler WC, Nahvi A, Roth A, Collins JA, Barrick JE, Breaker RR\",\"Barrick JE, Breaker RR\",37.0,38.5,35.6,This entry contains a glucosamine-6-phosphate activated ribozyme that is found in the mRNA for GlmS. GlmS is an enzyme that uses fructose-6-phosphate and glutamine to generate glucosamine-6-phosphate. The ribozyme catalyses self cleavage of the glmS mRNA that is thought to regulate GlmS activity [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,925,0,0,Cis-reg; riboswitch;,Published; PMID:15029187,920,0,,\"\",0.63246,-8.96959,1600000,427034,517,171,true,-3.257,0.71554,2013-10-04T00:21:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00235,Plasmid_RNAIII,1513,Plasmid RNAIII,Bateman A,Pubmed:14583190,60.0,143.7,48.8,This RNA is found in bacterial plasmids including pIP501. RNAIII acts by transcriptional attenuation of the essential repR-mRNA. RNAIII is composed of four stem loops with loops L3 and L4 that interact with the RNA target [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,7,2,0,0,Gene;,Predicted; ILM,5,0,,\"\",0.47112,-9.40435,1600000,372469,243,131,true,-2.9824,0.71825,2013-10-04T00:21:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00236,ctRNA_pGA1,1514,ctRNA,Bateman A,PMID:12670963,45.0,45.2,42.3,Rolling circle replication plasmids contain a number of mechanisms to maintain a low copy number. One of these methods is to transcribe a small RNA called ctRNA (for counter-transcribed RNA) than is transcribed on the opposite strand to RepB the replication initiation protein. ctRNA binds to the mRNA of repB and causes translational inhibition.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,16,0,0,Gene; antisense;,Published; PMID:12670963,21,0,,\"\",0.57565,-7.1622,1600000,310044,163,79,true,-3.2259,0.71905,2013-10-04T00:22:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00237,mir-9,2383,mir-9/mir-79 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.6,58.8,58.5,\"miR-9 has been identified in Drosophila (MIR:MI0000129) [1], mouse (MIR:MI0000720) and human (MIR:MI0000466) [2], and the related miR-79 in C. elegans (MIR:MI0000050) [3] and worm (MIR:MI0000374) [4]. The mature ~21nt miRNAs are processed from hairpin precursor sequences represented here. miR-9 is processed from the 5' arm of its precursor, and miR-79 from the 3' arm. The bounds of the precursors are predicted based on conservation and base pairing and are not generally known.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,43,572,0,0,Gene; miRNA;,Predicted; RNAalifold,244,0,,\"\",0.62716,-3.60358,1600000,229452,136,62,true,-4.1157,0.71924,2013-10-04T00:22:15.000Z,2023-11-06T17:21:47.000Z\n",
            "RF00238,ctRNA_pND324,1514,ctRNA,Bateman A,PMID:9682491,45.0,46.5,44.7,Rolling circle replication plasmids contain a number of mechanisms to maintain a low copy number. One of these methods is to transcribe a small RNA called ctRNA (for counter-transcribed RNA) than is transcribed on the opposite strand to RepB the replication initiation protein. ctRNA binds to the mRNA of repB and causes translational inhibition.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,48,30,0,0,Gene; antisense;,Predicted; ILM,20,0,,\"\",0.49044,-5.53958,1600000,267249,171,86,true,-2.8606,0.71878,2013-10-04T00:22:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00239,mir-124,1516,mir-124 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,49.0,49.0,48.2,\"miR-124 has been identified in Drosophila (MIR:MI0000373) [1], C. elegans (MIR:MI0000302) [2], mouse (MIR:MI0000150) and human (MIR:MI0000443) [3]. The mature ~21nt miRNAs are processed from hairpin precursor sequences represented here, in this case originating from the 3' arm. The bounds of the precursors are predicted based on conservation and base pairing and are not generally known. Family represents the microRNA (miRNA) precursor mir-124 imported from miRBase [4].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,56,403,0,0,Gene; miRNA;,Predicted; PFOLD,240,0,,\"\",0.66825,-2.87532,1600000,136285,161,81,true,-3.673,0.71886,2013-10-04T00:22:48.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00240,RNA-OUT,1517,RNA-OUT,Bateman A,\"PMID:2482367, INFERNAL\",41.0,41.5,40.9,\"Transposition of insertion sequence IS10 is regulated by an anti-sense RNA which inhibits transposase expression when IS10 is present in multiple copies per cell. The anti-sense RNA (RNA-OUT) consists of a stem domain topped by a flexibly paired loop; the 5' end of the target molecule, RNA-IN, is complementary to the top of the loop, and complementarity extends for 35 base-pairs down one side of RNA-OUT [1]. This bacterial RNA is also found in the genomic sequence of eukaryotic organisms. This is probably due to vector contamination.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,76,0,0,Gene; antisense;,Predicted; Pfold,47,0,,\"\",0.58785,-4.55704,1600000,271205,168,70,true,-3.7852,0.71905,2013-10-04T00:22:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00241,mir-8,2384,mir-8/mir-141/mir-200 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,47.0,47.0,46.9,\"miR-8 from Drosophila [1] (MIR:MI0000128) and human and mouse mir-141 (MIR:MI0000166, MIR:MI0000457), mir-429 (MIR:MI0001642) and mir-200 (MIR:MI0000243, MIR:MI0000342) [2] are expressed from the 3' arm of related precursor hairpins (represented here). Members of this precursor family have now been predicted or experimentally confirmed in a wide range of species (MIPF:MIPF0000019). The bounds of the precursors are predicted based on conservation and base pairing and are not generally known.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.00 -Z 742849.287494 CM SEQDB,164,538,0,0,Gene; miRNA;,Predicted; RNAalifold,238,0,,\"\",0.64886,-3.57064,1600000,189396,144,72,true,-3.6874,0.71935,2013-10-04T00:23:03.000Z,2023-11-06T17:21:47.000Z\n",
            "RF00242,ctRNA_pT181,1514,ctRNA,Bateman A,PMID:2478296,56.0,58.9,52.3,Rolling circle replication plasmids contain a number of mechanisms to maintain a low copy number. One of these methods is to transcribe a small RNA called ctRNA (for counter-transcribed RNA) than is transcribed on the opposite strand to RepB the replication initiation protein. ctRNA binds to the mRNA of repB and causes translational inhibition.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,16,17,0,0,Gene; antisense;,Published; PMID:2478296,15,0,,\"\",0.48014,-4.2136,1600000,219357,189,94,true,-3.0682,0.71869,2013-10-04T00:23:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00243,traJ_5,1519,traJ 5' UTR,Bateman A,PMID:14633993,50.0,51.4,47.5,\"The FinOP system regulates the transfer of F-like plasmids. FinP Rfam:RF00107 encodes an antisense RNA product that is complementary to part of the 5' UTR of the traJ mRNA. The traJ gene encodes a protein required for transcription from the major transfer promoter, pY. The FinO protein is essential for effective repression, acting by binding to FinP and protecting it from RNase E degradation.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,6,8,0,0,Cis-reg;,Predicted; ILM,10,0,,\"\",0.49546,-7.23829,1600000,276057,212,109,true,-2.9953,0.7184,2013-10-04T00:23:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00244,mir-26,1520,mir-26 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,57.0,57.0,56.8,\"miR-26 is processed from multiple predicted precursor hairpins in both mouse (MIR:MI0000573, MIR:MI0000575, MIR:MI0000706) and human (MIR:MI0000083, MIR:MI0000084, and MIR:MI0000750) [1,2]. miR-26 appears to be a vertebrate specific miRNA and has now been predicted or experimentally validated in many vertebrate species (MIPF:MIPF0000043). The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 5' arm of the hairpin.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,82,343,0,0,Gene; miRNA;,Predicted; RNAalifold,122,0,,\"\",0.71076,-2.74758,1600000,177774,163,81,true,-3.8572,0.71902,2013-10-04T00:23:23.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00245,mir-19,1521,mir-19 microRNA precursor family,\"Griffiths-Jones SR, Daub J\",\"Griffiths-Jones SR, INFERNAL\",50.0,50.0,49.9,\"miR-19 is processed from multiple predicted precursor hairpins in both mouse (MIR:MI0000546, MIR:MI0000688, MIR:MI0000718) and human (MIR:MI0000073, MIR:MI0000074, and MIR:MI000075) [1,2,3]. miR-19 has now been predicted or experimentally confirmed in a wide range of vertebrate species (MIPF:MIPF0000011). The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 3' arm of the hairpin. mir-19 microRNA (miRNA) precursor was imported from miRBase [4].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 45.00 -Z 742849.287494 CM SEQDB,106,379,0,0,Gene; miRNA;,Predicted; RNAalifold,124,0,,\"\",0.58422,-3.81383,1600000,138644,161,79,true,-3.9844,0.7192,2013-10-04T00:23:26.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00246,mir-135,1522,mir-135 microRNA precursor family,\"Griffiths-Jones SR, Daub J\",\"Griffiths-Jones SR, INFERNAL\",59.2,60.2,57.7,\"miR-135 has been shown to be expressed in human (MIR:MI0000452, MIR:MI0000453), mouse (MIR:MI0000161, MIR:MI0000646, MIR:MI0000715) and rat (MIR:MI0000645) [1,2]. miR-135 has now been predicted or experimentally confirmed in a wide range of vertebrate species (MIPF:MIPF0000028). The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 5' arm of the hairpin.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,32,335,0,0,Gene; miRNA;,Predicted; PFOLD,121,0,,\"\",0.64974,-2.95919,1600000,134925,179,90,true,-3.799,0.71883,2013-10-04T00:23:38.000Z,2022-05-06T13:10:02.000Z\n",
            "RF00247,mir-160,1523,mir-160 microRNA precursor family,\"Griffiths-Jones SR, Daub J\",Griffiths-Jones SR,48.0,48.0,47.9,miR-160 has been predicted or experimentally confirmed in a range of plant species including Arabidopsis and rice (MIPF:MIPF0000032). miR-160 is predicted to bind complementary sites in the untranslated regions of auxin response factor genes to regulate their expression [1].The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 5' arm of the hairpin.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,511,0,0,Gene; miRNA;,Predicted; PFOLD,82,0,,\"\",0.64544,-3.88075,1600000,203017,583,90,true,-3.5578,0.71883,2013-10-04T00:23:41.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00248,mir-148,2385,mir-148/mir-152 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,53.0,53.9,51.8,\"The expression of miR-148 has been demonstrated in human (MIR:MI0000253), mouse (MIR:MI0000550), rat (MIR:MI0000616) and zebrafish (MIR:MI0002015) [1,2,3]. miR-148 has also been predicted in chicken (MIR:MI0001189). These predicted hairpin precursor sequence are related to those of mir-152, which has been expressed in mouse (MIR:MI0000174) and is predicted in human (MIR:MI0000462) [1]. The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 3' arm of the hairpin.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,70,191,0,0,Gene; miRNA;,Predicted; RNAalifold,89,0,,\"\",0.62917,-3.71283,1600000,149769,156,74,true,-3.6066,0.71909,2013-10-04T00:23:50.000Z,2022-05-06T12:43:53.000Z\n",
            "RF00249,mir-46,2386,mir-46/mir-47/mir-281 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,44.7,44.7,44.4,\"miR-46 (MIR:MI0000017) and miR-47 (MIR:MI0000018) are expressed in C. elegans from related hairpin precursor sequences [1]. The predicted hairpin precursor sequences for Drosophila miR-281 (MIR:MI0000366, MIR:MI0000370) [2,3] are also related and so are also present in this family. The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequences are expressed from the 3' arms of the hairpin precursors.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,57,156,0,0,Gene; miRNA;,Predicted; RNAalifold,137,0,,\"\",0.61081,-3.39515,1600000,124345,169,85,true,-4.0378,0.71886,2013-10-04T00:23:53.000Z,2023-11-06T16:44:07.000Z\n",
            "RF00250,mir-TAR,1526,Trans-activation response element (TAR),Moxon SJ,PMID:12882959,40.0,45.9,36.1,\"This family represents the HIV trans-activation response (TAR) element which is known to be required for the trans-activation of the viral promoter and for virus replication. The TAR hairpin acts as a binding site for the Tat protein and this interaction stimulates the activity of the long terminal repeat promoter [1]. Further experiments have shown that TAR acts as a miRNA that protects against apoptosis [2,3].\",TAR,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,45,28,0,0,Gene; miRNA;,Published; PMID:12882959,13,0,,\"\",0.64556,-3.8597,1600000,279931,139,61,true,-3.8843,0.71935,2013-10-04T00:23:56.000Z,2021-11-19T11:53:19.000Z\n",
            "RF00251,mir-219,1527,mir-219 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,54.5,54.5,54.4,\"miR-219 was predicted in vertebrates by conservation between human, mouse and Fugu and cloned in Fugu [1], and later predicted and experimentally confirmed in Drosophila [2]. miR-219 homologs have since been predicted or experimentally confirmed in a wide range of species which include the platyhelminth Schmidtea mediterranea, several arthropod species and a wide range of vertebrates (MIPF:MIPF0000044). The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 5' arm of the hairpin. Family represents the microRNA (miRNA) precursor mir-219 imported from miRBase [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,92,264,0,0,Gene; miRNA;,Predicted; RNAalifold,199,0,,\"\",0.66457,-3.45345,1600000,167346,189,79,true,-3.8228,0.71899,2013-10-04T00:24:12.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00252,Alfamo_CPB,1528,Alfalfa mosaic virus coat protein binding (CPB) RNA,Moxon SJ,PMID:14718638,70.0,72.0,68.1,This family represents the Alfalfa mosaic virus (AMV) coat protein (CP) binding RNA which is found in the 3' UTR. AMV CP can stimulate the translation of AMV RNA by between 50 and 100-fold. This family contains at least two CP binding sites which are thought to be essential for efficient RNA translation [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,4,0,0,Cis-reg;,Published; PMID:14718638,2,0,,\"\",0.70023,-7.84051,1600000,389337,308,182,true,-3.3408,0.71429,2013-10-04T00:24:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00253,mir-101,1529,mir-101 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.2,48.9,\"Expression of miR-101 has been validated in both human (MIR:MI0000103, MIR:MI0000739) [1] and mouse (MIR:MI0000148) [2].This mirna appears to be specific to the vertebrates and has now been predicted or confirmed in a wide range of vertebrate species (MIPF:MIPF0000046). The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 3' arm of the hairpin. This family also contains a predicted homologue of mir-101 from Tetraodon fluviatilis. Imported from miRBase.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,59,218,0,0,Gene; miRNA;,Predicted; RNAalifold,120,0,,\"\",0.67185,-3.79911,1600000,202071,162,75,true,-3.5881,0.719,2013-10-04T00:24:28.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00254,mir-16,1530,mir-16 microRNA precursor family,\"Griffiths-Jones SR, Daub J\",Griffiths-Jones SR,44.0,44.1,43.9,\"miR-16, miR-15, mir-195 and miR-457 are related miRNA precursor sequences (MIPF:MIPF000006). This miRNA family appears to be vertebrate specific and members of this family have been predicted or experimentally validated in a wide range of vertebrate species (MIPF:MIPF0000006). miR-16 and miR-15a are clustered within 0.5 kb at 13q14 in human, a region shown to be deleted or down-regulated in more than half of B cell chronic lymphocytic leukaemias (CLL) [1]. The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 5' arm of the hairpin.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,446,0,0,Gene; miRNA;,Predicted; PFOLD,116,0,,\"\",0.69466,-2.91088,1600000,151251,167,81,true,-3.5851,0.71895,2013-10-04T00:24:31.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00256,mir-196,1532,mir-196 microRNA precursor family,\"Griffiths-Jones SR, Daub J\",Griffiths-Jones SR,55.4,55.4,55.3,\"miR-196 has been shown to be expressed in human (MIR:MI0000238, MIR:MI0000279) and mouse (MI0000552, MI0000553) [1,2]. miR-196 appears to be a vertebrate specific microRNA and has now been predicted or experimentally confirmed in a wide range of vertebrate species (MIPF:MIPF0000031). In many species the miRNA appears to be expressed from intergenic regions in HOX gene clusters. The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 5' arm of the hairpin.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,14,319,0,0,Gene; miRNA;,Predicted; PFOLD,113,0,,\"\",0.68721,-1.95833,1600000,132100,182,92,true,-3.5172,0.71879,2013-10-04T00:24:42.000Z,2022-05-06T12:43:50.000Z\n",
            "RF00257,mir-194,1533,mir-194 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,52.0,52.8,51.6,\"Expression of miR-194 has been verified in mouse (MIR:MI0000236, MIR:MI0000733) [1] and human (MIR:MI0000488, MIR:MI0000732) [2]. miR-194 appears to be a vertebrate specific miRNA and has now been predicted or experimentally confirmed in a range of vertebrate species (MIPF:MIPF0000055). The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 5' arm of the hairpin. Family represents the microRNA (miRNA) precursor mir-412 imported from miRBase [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,46,172,0,0,Gene; miRNA;,Predicted; RNAalifold,117,0,,\"\",0.67747,-3.16945,1600000,135790,166,82,true,-4.1089,0.71889,2013-10-04T00:24:46.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00258,mir-130,1534,mir-130 microRNA precursor family,\"Griffiths-Jones SR, Daub J\",\"Griffiths-Jones SR, INFERNAL\",40.6,40.6,40.5,\"miR-130 has been identified in mouse (MIR:MI0000156, MIR:MI0000408) [1], and in human (MIR:MI0000448, MIR:MI0000748) [2]. miR-130 appears to be vertebrate specific miRNA and has now been predicted or experimentally confrimed in a range of vertebrate species (MIPF:MIPF0000034). The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 3' arm of the hairpin.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,439,0,0,Gene; miRNA;,Predicted; PFOLD,117,0,,\"\",0.61499,-3.81332,1600000,214432,175,83,true,-3.5316,0.71889,2013-10-04T00:24:53.000Z,2022-05-06T12:43:50.000Z\n",
            "RF00259,IFN_gamma,1535,Interferon gamma 5' UTR regulatory element,Moxon SJ,PMID:11832212,40.0,41.6,39.5,This family represents a pseudoknot containing stem-loop structure found in the 5' UTR of interferon (IFN) gamma mRNA. This structure is thought to be involved in translational regulation and the pseudoknot has been found to activate PKR which is known to be a translational inhibitor. Mutations in the pseudoknot structure have been found to reduce PKR activation and increase the translation of IFN-gamma [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,5,35,0,0,Cis-reg;,Published; PMID:11832212,34,0,,\"\",0.49807,-9.4308,1600000,316968,292,169,true,-3.5575,0.71599,2013-10-04T00:24:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00260,HepC_CRE,1536,Hepatitis C virus (HCV) cis-acting replication element (CRE),\"Lamkiewicz K, Triebel S, Marz M\",Kevin Lamkiewicz; Sandra Triebel; Manja Marz;,140.0,142.2,43.2,Genetic modification of RNA structures in Hepatitis C virus (HCV) genome affects replication and infectivity [1] Long distant RNA-RNA interactions in HCV RNAs are involved in translation to virion budding [2] It has been proposed that the 5'-3' UTR base-pairing in the HCV genome might play an important role in viral RNA replication [3],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,56,1,0,0,Cis-reg;,\"Predicted; BM VeGETA v. 0.4, LocARNA 2.0.0RC8, RNAfold\",21,0,,\"\",0.67698,-8.35053,1600000,375227,294,176,true,-3.3368,0.71506,2013-10-04T00:25:03.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00261,IRES_L-myc,1537,L-myc internal ribosome entry site (IRES),Moxon SJ,\"PMID:14730027, INFERNAL\",60.0,61.4,52.7,\"This family represents the L-myc internal ribosome entry site (IRES). The myc family of genes when expressed are known to be involved in the control of cell growth, differentiation and apoptosis. L-myc undergoes translation via internal ribosome entry and bypasses the typical eukaryotic cap-dependent translation pathway [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.13 -Z 549862.597050 CM SEQDB,11,31,0,0,Cis-reg; IRES;,Published; PMID:14730027,33,0,,\"\",0.54571,-10.39988,1600000,444084,360,221,true,-3.2295,0.71062,2013-10-04T00:25:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00262,sar,1538,sar RNA,Bateman A,\"Pubmed:9042942, INFERNAL\",43.0,44.8,40.9,Sar RNA is an antisense RNA that is partly responsible for the negative regulation of antirepressor synthesis during development of bacteriophage P22 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,4,9,0,0,Gene; antisense;,Predicted; RNAalifold; Daub J,11,0,,\"\",0.6061,-5.23381,1600000,323300,152,68,true,-3.3684,0.71911,2013-10-04T00:25:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00263,SNORA68,1539,Small nucleolar RNA SNORA68,Moxon SJ,\"Moxon SJ, INFERNAL\",43.0,43.3,42.6,U68 is a member of the H/ACA class of small nucleolar RNAs (snoRNAs) which act as guides in the the modification of uridines to pseudouridines [2].,U68,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.5 -Z 549862.597050 CM SEQDB,26,247,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,117,0,,\"\",0.54228,-6.85769,1600000,211996,234,133,true,-3.2545,0.71822,2013-10-04T00:25:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00264,SNORA64,2387,Small nucleolar RNA SNORA64/SNORA10 family,\"Moxon SJ, Daub J\",\"PMID:10487763, Daub J\",35.0,35.3,34.9,This family belongs to the H/ACA class of small nucleolar RNAs (snoRNAs) which act as guides in the the modification of uridines to pseudouridines [2]. This family includes the human snoRNAs U64 and ACA10 and mouse MBI-29.,U64,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,9,167,0,0,Gene; snRNA; snoRNA; HACA-box;,Published; PMID:10487763,55,0,,\"\",0.59523,-7.32381,1600000,208566,242,134,true,-3.2567,0.71832,2013-10-04T00:25:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00265,SNORA69,1249,Small nucleolar RNA SNORA69,Moxon SJ,\"http://ribosome.miyazaki-med.ac.jp/, INFERNAL\",47.0,48.2,44.4,U69 is a member of the H/ACA class of small nucleolar RNAs (snoRNAs) which act as guides in the the modification of uridines to pseudouridines [1].,U69,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,16,152,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,92,0,,\"\",0.51863,-7.13379,1600000,260530,242,132,true,-3.17,0.71822,2013-02-01T11:57:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00266,snoZ17,1541,Small nucleolar RNA Z17,Moxon SJ,\"http://ribosome.miyazaki-med.ac.jp/, INFERNAL\",44.0,44.7,42.6,Z17 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,26,133,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,107,0,,\"\",0.56826,-5.7512,1600000,367190,152,71,true,-3.6487,0.71882,2013-10-04T00:26:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00267,snoR64,2388,Small nucleolar RNA R64/Z200 family,\"Moxon SJ, Daub J\",INFERNAL,45.0,45.9,43.1,This family of plant small nucleolar RNAs belong to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family includes Arabidopsis thaliana snoRNA R64 and Oryza sativa snoRNAs Z200 and J16. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,314,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,78,0,,\"\",0.63074,-1.45922,1600000,167526,185,94,true,-3.5673,0.71843,2013-10-04T00:26:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00268,snoZ7,2389,Small nucleolar RNA snoZ7/snoR77,Moxon SJ,http://bioinf.scri.sari.ac.uk/cgi-bin/plant_snorna/home,43.0,43.4,42.3,snoZ7 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,snoR77,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.81 -Z 549862.597050 CM SEQDB,5,34,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,12,0,,\"\",0.54874,-5.92483,1600000,317927,168,78,true,-3.0088,0.71863,2013-02-01T11:57:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00270,SNORD61,1543,Small nucleolar RNA SNORD61,Moxon SJ,\"Moxon SJ, INFERNAL\",43.0,43.2,42.7,U61 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U61,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,36,165,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,125,0,,\"\",0.49324,-6.11346,1600000,290129,148,72,true,-3.1799,0.71871,2013-10-04T00:26:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00271,SNORD60,1544,Small nucleolar RNA SNORD60,Moxon SJ,\"Moxon SJ, INFERNAL\",42.0,42.5,41.9,U60 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U60,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,131,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,68,0,,\"\",0.61479,-3.10163,1600000,203624,172,83,true,-3.501,0.71866,2013-10-04T00:26:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00272,SNORA67,1545,Small nucleolar RNA SNORA67,Moxon SJ,Moxon SJ,45.0,46.5,44.3,U67 belongs to the H/ACA class of snoRNAs which are thought to guide the sites of modification of uridines to pseudouridines.,U67,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.00 -Z 549862.597050 CM SEQDB,12,164,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Daub J,36,0,,\"\",0.5493,-8.66797,1600000,298665,262,143,true,-2.8897,0.71814,2013-10-04T00:26:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00273,SNORD59,1546,Small nucleolar RNA SNORD59,Moxon SJ,\"Moxon SJ, INFERNAL\",45.0,45.4,43.4,U59 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U59,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,24,219,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; ILM; Moxon SJ, Daub J\",103,0,,\"\",0.64428,-3.27041,1600000,259222,155,73,true,-3.9512,0.71872,2013-10-04T00:26:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00274,SNORD57,1547,Small nucleolar RNA SNORD57,Moxon SJ,\"Moxon SJ, INFERNAL\",47.0,47.1,46.8,U57 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U57,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,100,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,88,0,,\"\",0.64701,-4.04623,1600000,304171,156,72,true,-3.629,0.71873,2013-10-04T00:26:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00275,SNORD56,1548,Small nucleolar RNA SNORD56,Moxon SJ,Moxon SJ,36.0,36.3,35.0,U56 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U56,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,154,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,54,0,,\"\",0.63622,-3.91797,1600000,272954,155,71,true,-3.4419,0.71879,2013-10-04T00:27:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00276,SNORD52,1549,Small nucleolar RNA SNORD52,Moxon SJ,\"Moxon SJ, INFERNAL\",45.0,46.3,44.1,U52 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U52,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,20,66,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",51,0,,\"\",0.63129,-5.31554,1600000,348514,145,67,true,-3.3708,0.71883,2013-10-04T00:27:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00277,SNORD49,1550,Small nucleolar RNA SNORD49,Moxon SJ,\"Moxon SJ, INFERNAL\",37.0,37.1,36.9,U49 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U49,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,201,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,120,0,,\"\",0.61926,-8.20932,1600000,314669,151,71,false,-4.0751,0.71871,2013-10-04T00:27:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00278,SNORD50,1551,Small nucleolar RNA SNORD50,Moxon SJ,Moxon SJ,42.0,42.3,41.6,\"U50 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. U50 is an intronic snoRNA gene located on chromosome 6q15, at the breakpoint of chromosomal translocation t(3;6)(q27;q15). The U50HG gene is composed of six exons, whose spliced transcripts have little potential for coding a protein, and its introns produce both U50 and U50-like (U50') snoRNAs that are localised in nucleoli [3].\",U50,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,295,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Moxon SJ,111,0,,\"\",0.63428,-4.05653,1600000,289073,155,72,true,-3.4774,0.71875,2013-10-04T00:27:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00279,SNORD45,1552,Small nucleolar RNA SNORD45,Moxon SJ,Moxon SJ,42.0,42.8,41.7,U45 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U45,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,11,336,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,96,0,,\"\",0.56036,-4.5723,1600000,230913,168,81,true,-3.1627,0.71861,2013-10-04T00:27:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00280,SNORD51,1553,Small nucleolar RNA SNORD51,Moxon SJ,\"Moxon SJ, INFERNAL\",50.0,50.1,49.5,U51 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U51,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,107,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",95,0,,\"\",0.55944,-3.86685,1600000,200774,167,79,true,-3.7292,0.71861,2013-10-04T00:27:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00281,SNORD47,1251,Small nucleolar RNA SNORD47,Moxon SJ,\"Moxon SJ, INFERNAL\",50.0,50.3,49.4,U47 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U47,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,129,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; ILM; Moxon SJ, Daub J\",115,0,,\"\",0.59525,-4.13258,1600000,269301,162,77,true,-3.596,0.71863,2013-02-01T11:57:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00282,SNORD48,1554,Small nucleolar RNA SNORD48,Moxon SJ,\"Moxon SJ, INFERNAL\",36.0,36.2,34.7,U48 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U48,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,50,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,39,0,,\"\",0.69348,-5.0284,1600000,336196,146,63,true,-4.0873,0.71888,2013-10-04T00:27:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00283,SCARNA18,1555,small Cajal body-specific RNA 18,\"Moxon SJ, Daub J\",\"PMID:12032087, INFERNAL\",40.0,42.1,38.7,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. This C/D box domain U91 (also known as U4-8) was identified by Darzacq and is predicted to guide the 2'O-ribose methylation of U4 snRNA C8 [1]. This C/D domain was later found associated in tandem with another C/D box domain U12-22 (RFAM:RF00492). Both the doublet (U12-22/U4-8) and singlet (U4-8) forms of this snRNA have been purified from Hela cells [1]. The doublet form U12-22/U4-8 has been shown to localise to the nucleoplasm and is proposed to reside in the Cajal bodies whereas the U4-8 single domain appears to accumulate in the nucleolus [3]. In humans the genomic location of U12-88/U4-8 is intergenic and the purified transcript has been shown to possess a methylated guanosine cap suggesting it is independently transcribed by RNA pol II [3].\",U91,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,104,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; PFOLD; Moxon SJ,51,0,,\"\",0.68308,-3.36121,1600000,213120,173,83,true,-3.6427,0.71859,2013-10-04T00:27:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00284,SNORD74,1556,Small nucleolar RNA SNORD74,\"Moxon SJ, Daub J\",\"Moxon SJ, INFERNAL\",41.0,41.7,40.4,\"Z18 also identified as U74 (SNORD74) belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. U74 is predicted to guide the 2'O-ribose methylation of 28S rRNA residue C3820 [3]. In humans U74 shares the same non-protein coding host gene (gas5) with 9 other snoRNAs (U44, U47, U75, U76, U77, U78, U79, U80 and U81)[2].\",snoZ18; Z18,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,184,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,110,0,,\"\",0.67884,-3.41532,1600000,186458,165,80,true,-4.2492,0.71856,2013-10-04T00:27:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00285,snoZ6,1557,Z6 small nucleolar RNA,Moxon SJ,Moxon SJ,35.0,35.1,34.4,Z6 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,Z6,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,7,52,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",32,0,,\"\",0.71009,-3.28091,1600000,260774,189,91,true,-3.4043,0.71844,2013-10-04T00:27:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00286,SCARNA8,1558,Small Cajal body specific RNA 8,Moxon SJ,\"Moxon SJ, INFERNAL\",53.0,54.1,52.9,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. U92 belong to the H/ACA box class of guide RNAs as it has the predicted hairpin-hinge-hairpin-tail structure, conserved H/ACA-box motifs and is found associated with GAR1 [1]. Its is predicted to guide the pseudouridylation of residues U44 in U2 snRNA [1]. This family also includes the mouse H/ACA box snoRNA MBI-57 identified in [2].\",U92,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,190,0,0,Gene; snRNA; snoRNA; scaRNA;,Published; PMID:12032087,143,0,,\"\",0.55402,-8.06922,1600000,302832,238,131,true,-3.2739,0.71822,2013-10-04T00:29:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00287,SNORD44,1559,Small nucleolar RNA SNORD44,Moxon SJ,\"Moxon SJ, INFERNAL\",40.0,40.7,38.6,\"U44 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",U44,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,12,60,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; Moxon SJ, Daub J\",49,0,,\"\",0.59207,-6.79194,1600000,376631,140,61,true,-3.6737,0.71891,2013-10-04T00:29:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00288,snoZ30,1560,Z30 small nucleolar RNA,Moxon SJ,\"Moxon SJ, INFERNAL\",50.0,52.6,43.8,Z30 belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,Z30,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,127,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,112,0,,\"\",0.61415,-4.05438,1600000,204103,192,97,true,-3.2765,0.7184,2013-10-04T00:29:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00289,SNORND104,1561,Small nucleolar RNA SNORD104,\"Moxon SJ, Daub J\",\"Moxon SJ, INFERNAL\",40.0,42.7,36.9,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. U104 (SNORD104) also known as Z12 is predicted to guide the 2'O-ribose methylation of C1327 of 28S rRNA [3].,snoZ12; Z12,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.43 -Z 549862.597050 CM SEQDB,18,68,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",62,0,,\"\",0.71245,-4.80025,1600000,379449,154,70,true,-3.5562,0.71876,2013-10-04T00:29:29.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00290,BaMV_CRE,1562,Bamboo mosaic potexvirus (BaMV) cis-regulatory element,Moxon SJ,PMID:14585345,100.0,181.3,35.7,This family represents a cloverleaf-like cis-regulatory element found in the 3' UTR of the bamboo mosaic potexvirus. This family is thought to play an important role in the initiation of minus-strand RNA synthesis and may also be involved in the regulation of viral replication [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,5,1,0,0,Cis-reg;,Published; PMID:14585345,1,0,,\"\",0.62779,-6.35309,1600000,277695,257,140,true,-3.3715,0.71798,2013-10-04T00:29:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00291,snoR639,1563,Small nucleolar RNA snoR639/H1,Moxon SJ,\"Moxon SJ, INFERNAL\",42.0,43.5,41.5,snoR639 or snoH1 belongs to the H/ACA class of snoRNAs which are thought to guide the sites of modification of uridines to pseudouridines. snoH1 is found in an intron of the mfl gene in Drosophila [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,9,38,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold_manual; Daub J,36,0,,\"\",0.58032,-7.76235,1600000,213817,252,140,true,-3.3305,0.71805,2013-10-04T00:29:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00292,snoTBR5,1564,Small nucleolar RNA TBR5,Moxon SJ,\"Moxon SJ, INFERNAL\",44.0,44.4,36.0,\"TBR5 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,78,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",20,0,,\"\",0.69073,-4.7767,1600000,290541,190,94,true,-2.92,0.71836,2013-10-04T00:29:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00293,snoM1,1565,Small nucleolar RNA snoM1,Moxon SJ,\"Moxon SJ, INFERNAL\",40.0,41.2,37.2,M1 is a member of the H/ACA class of snoRNAs which are thought to guide the sites of modification of uridines to pseudouridines. snoM1 seems to be found exclusively in Drosophila species.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; ILM: Moxon SJ,8,0,,\"\",0.57636,-3.29617,1600000,244561,199,99,true,-3.5645,0.71841,2013-10-04T00:29:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00294,snoTBR17,1566,Small nucleolar RNA TBR17,Moxon SJ,Moxon SJ,40.0,72.3,30.4,\"TBR17 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.79 -Z 549862.597050 CM SEQDB,6,69,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,16,0,,\"\",0.65481,-6.93584,1600000,302495,228,119,true,-3.6341,0.718,2013-10-04T00:29:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00295,snoTBR7,1567,Small nucleolar RNA TBR7,Moxon SJ,Moxon SJ,45.0,47.6,34.5,\"TBR7 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.87 -Z 549862.597050 CM SEQDB,10,80,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,21,0,,\"\",0.55544,-8.75573,1600000,378056,161,76,true,-3.3635,0.71863,2013-10-04T00:29:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00296,snoR16,1568,Small nucleolar RNA R16,Moxon SJ,Moxon SJ,45.0,45.9,42.9,snoR16 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,305,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",75,0,,\"\",0.55096,-3.56882,1600000,213247,174,86,true,-2.9656,0.71856,2013-10-04T00:30:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00300,snoZ221_snoR21b,1569,Small nucleolar RNA Z221/R21b,\"Moxon SJ, Wilkison A, Daub J\",\"Moxon SJ, Wilkison A, Daub J\",44.0,44.0,43.3,Z221 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,snoZ221,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,237,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,77,0,,\"\",0.63944,-4.41449,1600000,262504,307,91,true,-3.6794,0.7184,2013-10-04T00:30:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00301,snoZ256,1570,Small nucleolar RNA Z256,Moxon SJ,Moxon SJ,50.0,54.6,43.3,Z256 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,62,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Moxon SJ,10,0,,\"\",0.65825,-3.65457,1600000,233363,191,94,true,-3.6247,0.71829,2013-10-04T00:30:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00302,SNORA65,1571,Small nucleolar RNA SNORA65,Moxon SJ,Moxon SJ,70.0,70.1,69.8,\"U65 belongs to the H/ACA class of snoRNAs which are thought to guide the sites of modification of uridines to pseudouridines [1,2].\",U65,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.00 -Z 549862.597050 CM SEQDB,14,184,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,102,0,,\"\",0.52156,-7.72571,1600000,223830,229,127,true,-3.1319,0.71842,2013-10-04T00:30:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00303,snoR86,1252,Small nucleolar RNA snoR86,Moxon SJ,\"Moxon SJ, INFERNAL\",40.0,40.2,37.0,snoR86 is a member of the H/ACA class of small nucleolar RNAs (snoRNAs) which act as guides in the the modification of uridines to pseudouridines [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,278,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,74,0,,\"\",0.52996,-8.02309,1600000,305834,230,123,true,-3.3599,0.71828,2013-02-01T11:57:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00304,snoZ279_R105_R108,1572,Small nucleolar RNA Z279/snoR105/snoR108,\"Moxon SJ, Daub J\",\"Moxon SJ, Daub J, INFERNAL\",54.0,55.3,49.5,\"This family includes the plant snoRNAs known as Z279, snoR105 and snoR108. All three are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].\",snoZ279,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,276,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,76,0,,\"\",0.60692,-4.34335,1600000,232458,200,106,true,-3.247,0.71859,2013-10-04T00:30:21.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00305,snoZ248,1573,Small nucleolar RNA Z248,Moxon SJ,Moxon SJ,120.0,150.8,115.1,Z248 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,16,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,10,0,,\"\",0.57641,-5.45784,1600000,226020,253,137,true,-2.9565,0.71801,2013-10-04T00:30:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00306,snoZ178,1574,Small nucleolar RNA Z178,Moxon SJ,Moxon SJ,100.0,195.3,45.9,Z178 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,8,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,9,0,,\"\",0.60165,-6.44116,1600000,339318,273,152,true,-2.6083,0.71801,2013-10-04T00:30:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00307,snoR98,1575,Small nucleolar RNA snoR98,Moxon SJ,Moxon SJ,44.0,44.9,43.9,snoR98 belongs to the H/ACA class of snoRNAs which are thought to guide the sites of modification of uridines to pseudouridines [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,88,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; Griffiths-Jones S,31,0,,\"\",0.46326,-8.3061,1600000,371195,222,117,true,-3.1922,0.71826,2013-10-04T00:30:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00309,snosnR60_Z15,1576,Small nucleolar RNA snR60/Z15/Z230/Z193/J17,Moxon SJ,Griffiths-Jones S,41.0,41.0,40.9,\"The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. The family includes the yeast snoRNA snR60 (also known as Z15) and the corresponding human guide snoRNA U80 (SNORD80) [2,3]. This family also includes the Arabidopsis orthologue of Z15 and a large number of related plant snoRNAs Z230, Z193 and J17 [4,5].\",snoZ193,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,1026,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones S,633,0,,\"\",0.51992,-5.10385,1600000,234124,193,98,true,-3.2254,0.71833,2013-10-04T00:30:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00310,snoZ165,1577,Small nucleolar RNA Z165,Moxon SJ,Moxon SJ,90.0,103.2,36.9,Z165 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,14,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Moxon SJ,10,0,,\"\",0.58808,-3.76584,1600000,232950,190,92,true,-3.6824,0.71834,2013-10-04T00:30:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00311,snoZ188,1253,Small nucleolar RNA Z188,Moxon SJ,Moxon SJ,50.0,51.3,41.5,Z188 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,79,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,19,0,,\"\",0.58954,-6.15443,1600000,300309,226,123,true,-3.0094,0.71806,2013-02-01T11:57:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00312,snoZ206,1578,Small nucleolar RNA Z206,Moxon SJ,\"Moxon SJ, INFERNAL\",50.0,58.1,39.3,Z206 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,43,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,16,0,,\"\",0.53727,-7.03993,1600000,266986,242,129,true,-2.8348,0.71801,2013-10-04T00:30:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00313,snoZ173,1579,Small nucleolar RNA Z173,Moxon SJ,Moxon SJ,70.0,73.1,65.0,Z173 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,4,17,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,11,0,,\"\",0.57055,-4.77277,1600000,236359,212,107,true,-2.9566,0.71819,2013-10-04T00:30:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00314,snoZ182,1580,Small nucleolar RNA Z182,Moxon SJ,Moxon SJ,80.0,86.2,39.8,Z182 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,21,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Moxon SJ,11,0,,\"\",0.62123,-2.73232,1600000,202163,189,92,true,-3.4856,0.7184,2013-10-04T00:31:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00315,snoJ33,1581,Small nucleolar RNA J33,Moxon SJ,\"Moxon SJ, INFERNAL\",45.0,45.4,44.8,J33 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,262,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Moxon SJ,73,0,,\"\",0.63393,-4.04911,1600000,252197,171,81,true,-3.6424,0.7186,2013-10-04T00:31:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00316,snoR43,1582,Small nucleolar RNA R43,Moxon SJ,Moxon SJ,50.0,56.1,43.2,R43 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,16,11,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; ILM; Moxon SJ,1,0,,\"\",0.55934,-4.55752,1600000,258938,160,73,true,-3.3007,0.71875,2013-10-04T00:31:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00317,snoZ163,2390,Small nucleolar RNA Z163/Z177 family,\"Moxon SJ, Daub J\",Griffiths-Jones S,39.0,39.3,37.7,This family of plant snoRNAs belong to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family includes rice snoRNA Z163 and Z177. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,45,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones S,11,0,,\"\",0.68095,-3.36693,1600000,259298,186,90,true,-3.185,0.71847,2013-10-04T00:31:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00318,snoZ175,1584,Small nucleolar RNA Z175,Moxon SJ,Moxon SJ,43.0,49.4,39.5,Z175 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,13,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,9,0,,\"\",0.50982,-4.9864,1600000,234217,174,81,true,-3.4523,0.71862,2013-10-04T00:31:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00319,SNORA23,1585,Small nucleolar RNA SNORA23,Moxon SJ,Moxon SJ,45.0,46.5,43.8,\"The members of this family belong to the H/ACA class of snoRNAs which are thought to guide the sites of modification of uridines to pseudouridines [1]. This family includes human snoRNA ACA23 (SNORA23) and mouse snoMBI-1 [2,3]. ACA23 is predicted to guide the pseudouridylation of residue U3737 and U4331 of 28S rRNA [4].\",snoMBI-1,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.11 -Z 549862.597050 CM SEQDB,9,177,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Griffiths-Jones S,109,0,,\"\",0.5622,-8.83118,1600000,320482,307,184,true,-3.6856,0.71397,2013-10-04T00:31:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00320,snoZ185,1586,Small nucleolar RNA Z185,Moxon SJ,Moxon SJ,50.0,57.4,39.6,Z185 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,26,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,11,0,,\"\",0.70342,-2.21043,1600000,219833,181,86,true,-3.6651,0.71848,2013-10-04T00:31:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00321,snoZ247,1587,Small nucleolar RNA Z247,Moxon SJ,Moxon SJ,38.0,38.0,37.9,Z247 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,2804,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,58,0,,\"\",0.56442,-6.7818,1600000,257180,249,139,true,-3.2713,0.71791,2013-10-04T00:31:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00322,SNORA31,1588,Small nucleolar RNA SNORA31,\"Moxon SJ, Daub J\",Griffiths-Jones S,40.0,40.0,39.9,ACA31 (SNORA31) belongs to the H/ACA class of snoRNAs which are thought to guide the sites of modification of uridines to pseudouridines [1]. ACA31 is predicted to guide the pseudouridylation of U3713 in 28S rRNA and U218 in 18S rRNA [4]. This family includes the mouse homologue snoMBI-161 [3].,snoMBI-161,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,5,444,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,106,0,,\"\",0.53348,-7.3366,1600000,252576,240,130,true,-3.0438,0.7185,2013-10-04T00:31:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00323,snoR79,1589,Small nucleolar RNA R79,Moxon SJ,Moxon SJ,80.0,93.1,55.4,R79 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,4,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,2,0,,\"\",0.63713,-3.99876,1600000,304334,152,65,true,-3.6591,0.71885,2013-10-04T00:31:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00324,snoMBII-202,1590,Small nucleolar RNA MBII-202,Moxon SJ,Moxon SJ,40.0,40.3,39.1,MBII-202 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,30,170,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,99,0,,\"\",0.63428,-4.26898,1600000,213622,171,84,true,-4.1065,0.71851,2013-10-04T00:31:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00325,SNORD53_SNORD92,1591,Small nucleolar RNA SNORD53/SNORD92,\"Moxon SJ, Daub J\",\"Moxon SJ, INFERNAL\",46.0,46.5,45.8,U53 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].This family includes mouse MBII-316 identified in [3]. The human orthologue HBII-316 (SNORD92) is predicted to guide the 2'O-ribose methylation of 28S rRNA A3846 [4].,SNORD53; U53,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.47 -Z 549862.597050 CM SEQDB,37,396,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Moxon SJ,127,0,,\"\",0.6045,-4.80277,1600000,286655,163,78,true,-3.6164,0.71862,2013-10-04T00:31:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00326,snoZ155,1592,Small nucleolar RNA Z155,Moxon SJ,Griffiths-Jones S,43.0,43.2,42.1,Z155 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,150,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones S,70,0,,\"\",0.51758,-4.04912,1600000,231850,168,81,true,-3.1093,0.7186,2013-10-04T00:32:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00327,snoZ194,1593,Small nucleolar RNA Z194,Moxon SJ,Moxon SJ,50.0,59.3,39.7,Z194 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,5,27,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,16,0,,\"\",0.56899,-3.04892,1600000,212269,213,108,true,-3.2235,0.71837,2013-10-04T00:32:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00328,snoZ161_228,2391,Small nucleolar RNA Z161/Z228,\"Moxon SJ, Wilkinson AC\",Moxon SJ,45.0,47.6,43.4,\"Z161 and Z228 are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",snoZ161,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,11,219,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,70,0,,\"\",0.60477,-3.45548,1600000,234756,181,93,true,-3.2371,0.7184,2013-10-04T00:32:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00329,snoZ162,1595,Small nucleolar RNA Z162,Moxon SJ,Moxon SJ,45.0,46.3,44.2,Z162 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.28 -Z 549862.597050 CM SEQDB,9,60,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,33,0,,\"\",0.60011,-3.60249,1600000,178696,182,88,true,-3.3987,0.71867,2013-10-04T00:32:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00330,snoZ43,1596,Small nucleolar RNA Z43,Moxon SJ,\"Moxon SJ, INFERNAL\",43.0,44.5,42.3,Z43 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,322,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,81,0,,\"\",0.63618,-3.51018,1600000,222759,199,100,true,-3.2733,0.71836,2013-10-04T00:32:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00331,snoZ169,1597,Small nucleolar RNA Z169,Moxon SJ,Moxon SJ,55.0,59.3,38.7,Z169 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.60 -Z 549862.597050 CM SEQDB,3,14,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,9,0,,\"\",0.55175,-4.44952,1600000,242104,156,69,true,-3.4937,0.7188,2013-10-04T00:32:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00332,snoZ266,1598,Small nucleolar RNA Z266,Moxon SJ,\"Moxon SJ, Daub J\",43.0,44.2,42.1,Z266 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,272,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,73,0,,\"\",0.58959,-3.65348,1600000,222325,203,90,true,-3.4216,0.71845,2013-10-04T00:32:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00333,snoZ157,1599,Small nucleolar RNA Z157/R69/R10,Moxon SJ,Moxon SJ,55.0,55.5,54.1,\"Z157, R69 and R10 are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,295,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,80,0,,\"\",0.64949,-2.40723,1600000,197026,195,100,true,-3.3256,0.71843,2013-10-04T00:32:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00334,SNORA3,1600,Small nucleolar RNA SNORA3/SNORA45 family,\"Moxon SJ, Daub J\",INFERNAL,51.0,51.2,50.7,\"The members of this family belong to the H/ACA class of snoRNAs which are thought to guide the sites of modification of uridines to pseudouridines [1]. This family includes mouse snoMBI-28 and human homologues ACA3 (SNORA3) and ACA3-2 (SNORA45). Both ACA3 and ACA3-2 share the same host gene [3]. MBI-28, ACA3 and ACA3-2 are all predicted to guide the pseudouridylation of residues in 28S rRNA [1,2,3].\",snoMBI-28,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,6,390,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,108,0,,\"\",0.55107,-7.48725,1600000,206146,239,131,true,-2.7949,0.71846,2013-10-04T00:32:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00335,snoZ13_snr52,2392,Small nucleolar RNA Z13/snr52,Moxon SJ,\"Moxon SJ, INFERNAL\",47.0,51.0,41.8,Z13 or snr52 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.72 -Z 549862.597050 CM SEQDB,42,545,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,378,0,,\"\",0.60058,-3.52105,1600000,224787,261,101,true,-3.3913,0.71833,2013-10-04T00:32:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00336,snoJ26,1602,Small nucleolar RNA J26,Moxon SJ,Moxon SJ,50.0,62.0,43.0,J26 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,41,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; ILM; Moxon SJ, Daub J\",10,0,,\"\",0.50102,-7.15726,1600000,270841,268,147,true,-2.9406,0.71795,2013-10-04T00:32:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00337,snoZ112,1603,Small nucleolar RNA Z112,Moxon SJ,\"Moxon SJ, INFERNAL\",40.0,40.0,38.6,\"Z112 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19 -Z 549862.597050 CM SEQDB,6,187,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,36,0,,\"\",0.53362,-9.04911,1600000,349723,313,184,true,-3.5281,0.71383,2013-10-04T00:32:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00338,snoR53,1604,Small nucleolar RNA snR53,Moxon SJ,\"Moxon SJ, INFERNAL\",70.0,97.2,39.6,\"snR53 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",snR53,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Daub J,12,0,,\"\",0.46984,-4.76875,1600000,217229,189,91,true,-3.1769,0.71848,2013-10-04T00:33:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00339,snoR60,1605,Small nucleolar RNA snoR60,Moxon SJ,\"Moxon SJ, INFERNAL\",40.0,41.0,39.2,\"snoR60 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,366,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ; Daub J,77,0,,\"\",0.63646,-3.79659,1600000,243773,158,72,true,-3.6878,0.71875,2013-10-04T00:33:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00340,SNORA36,1606,Small nucleolar RNA SNORA36 family,\"Moxon SJ, Daub J\",Moxon SJ,43.0,43.5,41.9,ACA36 (SNORA36A) belongs to the H/ACA class of snoRNAs which are predicted to guide the sites of modification of uridines to pseudouridines [1]. ACA36 is predicted to guide the pseudouridylation of residue U105 and U1244 of 18S rRNA [4]. This family includes the mouse homologue snoMBI-87 [3] and also identifies the human paralog ACA36B (SNORA36B) [2].,snoMBI-87,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,276,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,100,0,,\"\",0.52253,-7.62034,1600000,308489,243,132,true,-2.9904,0.71827,2013-10-04T00:33:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00341,snoZ39,1607,Small nucleolar RNA Z39,Moxon SJ,Moxon SJ,39.0,39.9,36.8,Z39 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,159,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,96,0,,\"\",0.60847,-3.44768,1600000,254761,146,63,true,-3.3564,0.71918,2013-10-04T00:33:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00342,snoZ40,1608,Small nucleolar RNA Z40,Moxon SJ,Moxon SJ,50.0,51.7,44.7,Z40 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,119,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",98,0,,\"\",0.64146,-3.26255,1600000,232881,163,73,true,-3.6914,0.71868,2013-10-04T00:33:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00343,snoZ122,1609,Small nucleolar RNA Z122,Moxon SJ,\"Moxon SJ, INFERNAL\",38.0,39.3,37.3,Z122 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,278,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,102,0,,\"\",0.47863,-5.14828,1600000,248487,218,94,true,-2.655,0.71844,2013-10-04T00:33:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00344,snoZ267,1610,Small nucleolar RNA Z267,Moxon SJ,Moxon SJ,43.0,43.0,42.2,Z267 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,176,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,72,0,,\"\",0.60746,-3.75103,1600000,225315,161,73,true,-3.5316,0.71881,2013-10-04T00:33:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00345,snoR1,1611,Small nucleolar RNA snoR1,\"Moxon SJ, Wilkinson AC\",Moxon SJ,49.0,49.8,47.8,\"snoR1 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,232,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,83,0,,\"\",0.61808,-5.50956,1600000,290718,197,93,true,-3.5717,0.71837,2013-10-04T00:33:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00348,snoR9_plant,1360,Small nucleolar RNA snoR9,Moxon SJ,Moxon SJ,43.0,44.6,40.7,\"snoR9 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2]. This family is not related to Rfam:RF00065.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,7,108,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",2,0,,\"\",0.51508,-5.81898,1600000,282724,239,129,true,-2.8325,0.71789,2013-10-04T00:33:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00349,snoR11,2393,Small nucleolar RNA R11/Z151,Moxon SJ,Moxon SJ,50.0,52.0,43.9,\"snoR11/Z151 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,283,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,74,0,,\"\",0.58112,-3.27941,1600000,231780,186,90,true,-3.3162,0.71847,2013-10-04T00:34:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00350,snoZ152,1613,Small nucleolar RNA Z152/R70/R12/,Moxon SJ,\"Moxon SJ, INFERNAL\",50.0,51.9,43.3,\"Z152 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2]. This family also contains snos R70 and R12 from Arabidopsis thaliana.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,222,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,74,0,,\"\",0.5718,-4.71089,1600000,268116,246,109,true,-2.9025,0.71818,2013-10-04T00:34:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00351,snoR20,1614,Small nucleolar RNA R20,Moxon SJ,\"Moxon SJ, INFERNAL\",43.0,43.2,40.2,\"snoR20 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33 -Z 549862.597050 CM SEQDB,4,53,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",9,0,,\"\",0.46698,-5.6823,1600000,255158,189,90,true,-2.8384,0.71839,2013-10-04T00:34:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00352,snoR21,1615,Small nucleolar RNA R21,Moxon SJ,\"Moxon SJ, INFERNAL\",45.0,45.2,44.2,\"snoR21 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33 -Z 549862.597050 CM SEQDB,6,140,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,46,0,,\"\",0.4785,-4.81862,1600000,227819,172,82,true,-3.3698,0.71865,2013-10-04T00:34:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00353,snoR31_Z110_Z27,1616,Small nucleolar RNA snoR31/Z110/Z27,Moxon SJ,Moxon SJ,45.0,45.3,43.9,\"This family contains several related snoRNAs including R31, Z27 and Z110. Family members belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2,3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,230,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",67,0,,\"\",0.49593,-3.49187,1600000,187417,188,97,true,-3.1267,0.71836,2013-10-04T00:34:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00355,snoR28,1617,Small nucleolar RNA snoR28,Moxon SJ,\"Moxon SJ, INFERNAL\",45.0,48.0,39.3,\"R28 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,109,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",73,0,,\"\",0.57781,-4.65736,1600000,275411,166,78,true,-3.643,0.71866,2013-10-04T00:34:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00356,snoR32_R81,2394,Small nucleolar RNA R32/R81/Z41,Moxon SJ,Moxon SJ,45.0,46.4,40.1,\"This family contains the related snoRNAs R32, R81 and Z41 which are members of the C/D class of snoRNA containing the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,81,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",44,0,,\"\",0.48339,-4.25023,1600000,219974,206,98,true,-2.9554,0.71838,2013-10-04T00:34:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00357,snoR44_J54,2395,Small nucleolar RNA R44/J54/Z268 family,Moxon SJ,Moxon SJ,48.0,48.7,46.6,\"R44, J54 and Z268 are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,29,435,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,78,0,,\"\",0.57856,-4.99586,1600000,287473,194,97,true,-3.0524,0.71837,2013-10-04T00:34:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00358,snoZ101,1620,Small nucleolar RNA Z101,Moxon SJ,\"Moxon SJ, INFERNAL\",44.0,44.8,41.4,\"Z101 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,351,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",79,0,,\"\",0.59261,-2.8641,1600000,185709,190,96,true,-3.4476,0.71848,2013-10-04T00:34:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00359,snoZ102_R77,1621,Small nucleolar RNA Z102/R77,Moxon SJ,\"Moxon SJ, INFERNAL\",40.0,41.3,39.6,\"Z102 and R77 and are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2,3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,435,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,77,0,,\"\",0.53365,-3.40143,1600000,231163,181,88,true,-3.2663,0.71847,2013-10-04T00:34:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00361,snoZ119,1623,Small nucleolar RNA Z119,Moxon SJ,Moxon SJ,70.0,77.2,39.7,Z119 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,62,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,20,0,,\"\",0.59799,-3.14534,1600000,216335,198,98,true,-3.8701,0.71857,2013-10-04T00:35:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00362,Pospi_RY,1624,Pospiviroid RY motif stem loop,Moxon SJ,PMID:14500815,40.0,42.5,37.1,\"This family consists of several Pospiviroid sequences which are related to the potato spindle tuber viroid (PSTVd) RY motif stem-loop. The RY motif (5'-ACAGG and CUCUUCC-5') in PSTVd, is thought to bind with the tomato protein Virp1. The exact function of this motif and the significance of Virp1 binding is unknown. It is however thought that RY motifs are essential for establishing a viroid infection [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,0,0,0,Cis-reg;,Published; PMID:14500815,4,0,,\"\",0.58988,-3.72587,1600000,173303,170,79,true,-3.5459,0.71894,2013-10-04T00:35:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00363,mir-BART1,1625,mir-BART1 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,97.7,44.7,This family represents the mir-BART1 microRNA precursor found in Human herpesvirus 4 (Epstein-Barr virus) and Cercopithicine herpesvirus 15. mir-BART1 is found in all stages of infection but expression is significantly elevated in the lytic stage. In Epstein-Barr virus mir-BART1 is found in the intronic regions of the BART (Bam HI-A region rightward transcript) gene whose function is unknown. The mature sequence is excised from the 5' arm of the hairpin [1]. Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.67032,-3.02688,1600000,214676,160,70,true,-3.3414,0.72466,2013-10-04T00:35:36.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00364,mir-BART2,1626,mir-BART2 microRNA precursor family,Moxon SJ,PMID:15118162,60.0,62.8,38.5,This family represents the mir-BART2 microRNA precursor found in Human herpesvirus 4 (Epstein-Barr virus) and Cercopithicine herpesvirus 15. mir-BART2 is expressed in all stages of infection but expression is significantly elevated in the lytic stage. In Epstein-Barr virus mir-BART2 is found in the intronic regions of the BART (Bam HI-A region rightward transcript) gene whose function is unknown. mir-BART2 is thought to target the virally encoded polymerase BALF5 for degradation. The mature sequence is excised from the 5' arm of the hairpin [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,3,0,0,Gene; miRNA;,Published; PMID:15118162,1,0,,\"\",0.64206,-1.52194,1600000,226584,146,62,true,-4.1506,0.71922,2013-10-04T00:35:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00365,mir-BHRF1-1,1627,mir-BHRF1-1 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.9,41.1,This family represents the mir-BHRF1-1 microRNA precursor found in Human herpesvirus 4 (Epstein-Barr virus) and Cercopithicine herpesvirus 15. In Epstein-Barr virus mir-BHRF1-1 is found in the 5' UTR of the BHRF1 (Bam HI fragment H rightward open reading frame 1) gene which is known to encode a distant Bcl-2 homologue. The mature sequence is excised from the 5' arm of the hairpin [1]. Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.64091,-2.8827,1600000,231628,155,66,true,-3.1708,0.72439,2013-10-04T00:35:41.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00366,mir-BHRF1-2,1628,mir-BHRF1-2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.6,56.6,\"This family represents the mir-BHRF1-2 microRNA precursor found in human herpesvirus 4 (Epstein-Barr virus), cercopithicine herpesvirus 15 and herpesvirus papio. In Epstein-Barr virus mir-BHRF1-2 is found in the 3' UTR of the BHRF1 (Bam HI fragment H rightward open reading frame 1) gene which is known to encode a distant Bcl-2 homologue. The mature sequence is excised from the 3' arm of the hairpin [1]. Imported from miRBase.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.60012,-3.57494,1600000,290348,153,65,true,-3.2768,0.7295,2013-10-04T00:35:47.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00367,mir-BHRF1-3,1629,mir-BHRF1-3 microRNA precursor family,Moxon SJ,PMID:15118162,80.0,100.1,39.6,This family represents the mir-BHRF1-3 microRNA precursor found in Human herpesvirus 4 (Epstein-Barr virus). In Epstein-Barr virus mir-BHRF1-3 is found in the 3' UTR of the BHRF1 (Bam HI fragment H rightward open reading frame 1) gene which is known to encode a distant Bcl-2 homologue. The mature sequence is excised from the 5' arm of the hairpin [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,3,0,0,Gene; miRNA;,Published; PMID:15118162,1,0,,\"\",0.68725,-2.51453,1600000,255882,152,65,true,-3.5128,0.71922,2013-10-04T00:35:53.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00368,sroB,1630,sroB RNA,Barquist LE,\"Storz, G; LocARNA; Infernal\",44.0,44.8,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,70,0,0,Gene; sRNA;,Predicted; LocARNA,79,0,,\"\",0.46286,-5.11492,1600000,272661,166,83,true,-3.5923,0.71868,2013-10-04T00:35:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00369,sroC,1631,sroC RNA,Moxon SJ,PMID:14602901,45.0,46.9,42.9,\"This family consists of a number of bacterial sroC RNA genes of around 160 bases in length. sroC is found in several Enterobacterial species [1]. It base-pairs with GcvB. This interaction triggers the degradation of GcvB by RNase E, alleviating the GcvB-mediated mRNA repression of other amino acid-related transport and metabolic genes [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,56,0,0,Gene; sRNA;,Predicted; PFOLD,62,0,,\"\",0.5635,-5.77845,1600000,288117,280,163,true,-3.3463,0.71672,2013-10-04T00:35:58.000Z,2021-12-03T12:41:14.000Z\n",
            "RF00370,sroD,1632,sroD RNA,Moxon SJ,PMID:14602901,50.0,51.4,44.4,This family consists of a number of bacterial sroD RNA genes of around 90 bases in length. sroD is found in several Enterobacterial species but its function is unknown [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,38,0,0,Gene; sRNA;,Predicted; PFOLD,40,0,,\"\",0.56163,-7.42654,1600000,328807,181,86,true,-3.144,0.71865,2013-10-04T00:36:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00371,sroE,1633,sroE RNA,Moxon SJ,PMID:14602901,51.0,52.1,49.9,This family consists of a number of bacterial sroE RNA genes of around 90 bases in length. sroE is found in several Enterobacterial species but its function is unknown [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,34,0,0,Gene; sRNA;,Predicted; PFOLD,37,0,,\"\",0.67822,-4.24008,1600000,210811,188,92,true,-3.2661,0.71867,2013-10-04T00:36:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00372,sroH,1634,sroH RNA,Barquist L,\"PMID:14602901, INFERNAL\",80.0,88.7,72.9,This family consists of a number of bacterial sroH RNA genes of around 160 bases in length. The function of this family is unknown [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,6,0,0,Gene; sRNA;,Predicted; Barquist L,8,0,,\"\",0.61811,-10.14425,1600000,528031,281,161,true,-2.5152,0.71683,2013-10-04T00:36:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00373,RNaseP_arch,2441,Archaeal RNase P,Griffiths-Jones SR,\"Brown JW, The Ribonuclease P Database, PMID:9847214\",84.0,84.2,83.8,\"Ribonuclease P (RNase P) is a ubiquitous endoribonuclease, found in archaea, bacteria and eukarya as well as chloroplasts and mitochondria. Its best characterised activity is the generation of mature 5'-ends of tRNAs by cleaving the 5'-leader elements of precursor-tRNAs. Cellular RNase Ps are ribonucleoproteins. RNA from bacterial RNase Ps retains its catalytic activity in the absence of the protein subunit, i.e. it is a ribozyme. Isolated eukaryotic and archaeal RNase P RNA has not been shown to retain its catalytic function, but is still essential for the catalytic activity of the holoenzyme. Although the archaeal and eukaryotic holoenzymes have a much greater protein content than the bacterial ones, the RNA cores from all the three lineages are homologous -- helices corresponding to P1, P2, P3, P4, and P10/11 are common to all cellular RNase P RNAs. Yet, there is considerable sequence variation, particularly among the eukaryotic RNAs.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,70,303,0,0,Gene; ribozyme;,Published; PMID:9847214,340,0,,\"\",0.69973,-9.74148,1600000,423094,968,303,true,-4.3544,0.70576,2013-10-04T00:36:13.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00374,Gammaretro_CES,1635,Gammaretrovirus core encapsidation signal,\"Moxon SJ, Daub J\",PMID:15003457,51.0,51.0,50.9,This family represents the Gammaretrovirus (mammalian type C retrovirus) core encapsidation signal and related sequences. The Moloney murine leukaemia virus (MMLV) core encapsidation signal is know to be essential for stable dimerisation and efficient genome packaging during virus assembly [1]. Dimerisation of the viral RNA genomes is proposed to act as an RNA conformational switch which exposes conserved UCUG elements and enables efficient genome encapsidation [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.50 -Z 549862.597050 CM SEQDB,22,143,0,0,Cis-reg;,Published; PMID:15003457,19,0,,\"\",0.50921,-8.73624,1600000,300894,201,101,true,-3.3498,0.7187,2013-10-04T00:36:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00375,HIV_PBS,1636,HIV primer binding site (PBS),Moxon SJ,PMID:14757051,80.0,83.1,63.9,\"This family consists of several, structured HIV primer binding sites (PBSs). tRNA(3)(Lys) binds to the PBS to initiate reverse transcription [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,130,11,0,0,Cis-reg;,Predicted; PFOLD,8,0,,\"\",0.64335,-3.38771,1600000,246232,198,99,true,-3.0543,0.71844,2013-10-04T00:36:31.000Z,2022-05-06T13:10:03.000Z\n",
            "RF00376,HIV_GSL3,1637,HIV gag stem loop 3 (GSL3),Moxon SJ,PMID:14757051,42.0,42.0,41.9,This family consists of several HIV gag stem loop 3 (GSL3) sequences. GSL3 is known to direct specific packaging of HIV-1 genomic RNA. Deletion of GSL3 leads to decreases in both viral RNA packaging and dimerisation [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,72,23,0,0,Cis-reg;,Predicted; PFOLD,16,0,,\"\",0.58943,-3.91234,1600000,252386,172,84,true,-3.2877,0.7186,2013-10-04T00:36:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00377,snoU6-53,2396,Small nucleolar RNA U6-53/MBII-28,Moxon SJ,\"Moxon SJ, INFERNAL\",36.0,36.4,35.8,U6-53 or MBII-28 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. This family possesses sequence complementarity to U6 spliceosomal RNA and is likely to direct its 2'-O-methylation [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.39 -Z 549862.597050 CM SEQDB,25,168,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,103,0,,\"\",0.63709,-5.20786,1600000,237759,206,110,true,-3.3726,0.71821,2013-10-04T00:37:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00378,Qrr,1639,Qrr RNA,Moxon SJ,PMID:15242645,50.0,59.2,42.1,\"This family represents a set of small RNAs (the Qrr RNAs) that are thought to be involved in the regulation of quorum sensing in Vibrio species. It is believed that these RNAs, guided by a protein, Hfq, can mediate the destabilisation of the quorum-sensing master regulators LuxR/HapR mRNAs [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.99 -Z 549862.597050 CM SEQDB,30,170,0,0,Gene; sRNA;,Published; PMID:15242645; Moxon SJ,72,0,,\"\",0.54241,-6.89765,1600000,278985,203,108,true,-3.4444,0.71844,2013-10-04T00:37:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00379,ydaO-yuaA,2397,ydaO/yuaA leader,Moxon SJ,\"Barrick JE, Breaker RR\",38.0,38.0,36.2,\"This family represents a conserved RNA structure found upstream of the ydaO and yuaA genes in Bacillus subtilis and related genes in other bacteria. The ydaO/yuaA element is thought to act as a genetic \\\"off\\\" switch for the ydaO and yuaA genes in its native state. It is believed that the element may be triggered during osmotic shock leading to activation of ydaO, a predicted amino acid transporter gene, and members of the yuaA-yubG operon which code for KtrA and KtrB K+ transporters [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,106,3811,0,0,Cis-reg; riboswitch;,\"Predicted; Barrick JE, Breaker RR\",1285,0,,\"\",0.6399,-6.69945,1600000,194495,521,136,true,-3.6017,0.71816,2013-10-04T00:37:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00380,M-box,1641,M-box riboswitch (ykoK leader),\"Moxon SJ, Ontiveros-Palacios N\",\"Barrick JE, Breaker RR\",44.0,44.1,36.9,\"M-box riboswitch, also know as ykoK leader is described as a metaloregulatory RNA that sense divalent metals and function as a genetic control element of genes related to divalent metal transport in bacteria. It was first reported upstream of the B.subtilis ykoK gene and genes with related functions in other bacteria [1]. The structure of the M-box was first reported in PDB 2QBZ [2], as a structure of three parallel helices, where P3 and P4 helixes form a coaxial stack that folds alongside P2 and P5. Six Mg2+ recognition sites were reported in 2QBZ, while in a second structure report in PDB 3PDR, the authors report another three sites of Mn2+ bound that been predicted from phosphorothioate substitution experiments and six more additional Mn2+ proposed from the analysis of the anomalous density in the sequence [3]. Phylogenetic analysis of the distribution of M-box RNAs across bacterial genomes revealed that M-box riboswitches are most often employed for the regulation of Mg2+ transporter genes [4]\",ykoK;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,159,913,0,0,Cis-reg; riboswitch;,\"Predicted; Barrick JE, Breaker RR\",726,0,,\"\",0.60556,-7.84028,1600000,349348,312,169,true,-3.6506,0.71579,2013-10-04T00:37:33.000Z,2022-10-29T17:55:17.000Z\n",
            "RF00381,Antizyme_FSE,1642,Antizyme RNA frameshifting stimulation element,Moxon SJ,PMID:15147837,40.0,42.1,39.2,\"This family represents a structural element which is found in antizyme mRNA and is known to promote frameshifting. Antizyme genes have have two partially overlapping open reading frames, the second, which encodes the functional protein requires +1 translational frameshifting. This frameshift is stimulated by a pseudoknot present 3' of the frameshift site in the antizyme mRNA. The frameshifting efficiency is dependent on the level of polyamines in the cell, when the polyamine concentration is high frameshifting is more likely to occur which leads to an increase in the quantity of functional antizyme produced. The functional antizyme acts to reduce ornithine decarboxylase (ODC) activity which leads to a drop in polyamines present in the cell. Therefore, this family can be thought of as a biosensor for intracellular free polyamines [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,13,280,0,0,Cis-reg; frameshift_element;,Published; PMID:15147837,114,0,,\"\",0.66203,-4.23694,1600000,362460,137,59,true,-3.3804,0.71923,2013-10-04T00:37:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00382,DnaX,1643,DnaX ribosomal frameshifting element,Moxon SJ,PMID:9300054,35.0,35.0,34.9,\"This family represents a ribosomal frameshifting element found in the mRNA of the dnaX gene in E. coli. The dnaX gene has two encoded products tau and gamma which are produced in a 1:1 ratio. The gamma protein is synthesised due to programmed frameshifting and is shorter than tau. The two products of the dnaX gene are DNA polymerase III subunits [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,76,0,0,Cis-reg; frameshift_element;,Published; PMID:9300054,79,0,,\"\",0.59904,-8.07336,1600000,337282,149,65,true,-3.4234,0.71903,2013-10-04T00:37:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00383,IS1222_FSE,1644,Insertion sequence IS1222 ribosomal frameshifting element,Moxon SJ,PMID:15126494,45.0,45.3,39.0,This family represents the insertion sequence IS1222 ribosomal frameshifting element. The element stimulates frameshifting which is known to be required for transposition [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,95,0,0,Cis-reg; frameshift_element;,Published; PMID:15126494,51,0,,\"\",0.55669,-8.31821,1600000,317794,226,118,true,-3.3585,0.71813,2013-10-04T00:37:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00384,Pox_AX_element,1645,Poxvirus AX element late mRNA cis-regulatory element,Moxon SJ,PMID:10049834,60.0,62.6,41.8,This family represents a cis-regulatory element present at the 3' ends of poxvirus late mRNAs and is known as the AX element.The AX element is involved in directing the efficient production and orientation-dependent formation of of late RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.52 -Z 549862.597050 CM SEQDB,10,7,0,0,Cis-reg;,Published; PMID:10049834,13,0,,\"\",0.50632,-7.14669,1600000,336633,146,62,true,-3.6369,0.71926,2013-10-04T00:37:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00385,IBV_D-RNA,1646,Infectious bronchitis virus D-RNA,Moxon SJ,PMID:11119581,35.0,40.9,34.6,This family represents a cis-regulatory element known as defective RNA or D-RNA. This region is thought to be essential for viral replication and efficient packaging [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,10,8,0,0,Cis-reg;,Published; PMID:11119581,9,0,,\"\",0.70171,-3.9603,1600000,386392,113,42,true,-3.9979,0.71978,2013-10-04T00:38:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00386,Entero_5_CRE,1647,Enterovirus 5' cloverleaf cis-acting replication element,Moxon SJ,PMID:11250909,46.0,46.0,45.8,\"This family represents a cis-regulatory element found at the 5' end of Enterovirus genomes. The element has a cloverleaf like secondary structure and is known to be a multifunctional cis-acting replication element (CRE), required for the initiation of negative strand RNA synthesis [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,160,2,0,0,Cis-reg;,Published; PMID:11250909,87,0,,\"\",0.68121,-3.37059,1600000,207109,309,90,true,-3.2423,0.71888,2013-10-04T00:38:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00387,IRES_FGF1,1648,FGF-1 internal ribosome entry site (IRES),Moxon SJ,PMID:15314170,38.0,39.1,36.2,This family represents the FGF-1 internal ribosome entry site (IRES). The FGF-1 IRES is present in the 5' UTR of the mRNA and allows cap-independent translation. It is thought that FGF-1 IRES activity is strictly controlled and highly tissue specific [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,6,34,0,0,Cis-reg; IRES;,Published; PMID:15314170,34,0,,\"\",0.55902,-8.30086,1600000,248588,291,168,true,-3.182,0.71618,2013-10-04T00:38:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00388,Anti-Q_RNA,1649,Anti-Q RNA,Moxon SJ,PMID:14651627,45.0,46.3,35.6,Qa RNA is a cis-regulatory element from Enterococcus faecalis which is known to negatively regulate prgQ transcription via an antisense mechanism [1].,QaRNA,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,6,0,0,Gene; antisense;,Published; PMID:14651627,5,0,,\"\",0.47665,-7.87188,1600000,312793,231,103,true,-3.1627,0.71875,2013-10-04T00:38:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00389,satBaMV_CRE,1650,Bamboo mosaic virus satellite RNA cis-regulatory element,Moxon SJ,PMID:12832220,100.0,127.7,24.9,This family represents a cis-regulatory element found in the 5' UTR of bamboo mosaic virus satellite RNA (satBaMV). The family is thought to be essential for efficient replication [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,42,1,0,0,Cis-reg;,Published; PMID:12832220,1,0,,\"\",0.6424,-7.02054,1600000,253961,280,159,true,-3.2857,0.7173,2013-10-04T00:38:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00390,UPSK,1651,UPSK RNA,Moxon SJ,\"PMID:9223489, INFERNAL\",38.0,38.6,37.1,This family represents a pseudoknot structure from turnip yellow mosaic virus which is thought to be needed for efficient transcription as disruption of the pseudoknot structure gives rise to a 50% drop in transcription efficiency [1]. This family acts in conjunction with Rfam:RF00233 to enhance translation [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 --mxsize 128 CM SEQDB,6,1,0,0,Cis-reg;,Published; PMID:9223489,1,0,,\"\",0.55034,-8.27878,1600000,710521,87,23,true,-2.9244,0.73793,2013-10-04T00:38:26.000Z,2020-04-23T12:31:10.000Z\n",
            "RF00391,RtT,1652,RtT RNA,\"Moxon SJ, Gardner PP\",PMID:1840671,45.0,45.0,44.9,This family represents a bacterial cis-regulatory element known as RtT RNA. The exact function of RtT is unknown although it is thought that it may be involved in changing the cellular response in relation to amino acid starvation [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,57,507,0,0,Cis-reg;,Published; PMID:1840671; Moxon SJ,157,0,,\"\",0.59237,-5.76937,1600000,253181,230,130,true,-3.3646,0.71829,2013-10-04T00:38:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00392,SNORA5,1653,Small nucleolar RNA SNORA5,Moxon SJ,Moxon SJ,30.0,30.1,29.7,ACA5 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA5,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,6,413,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,106,0,,\"\",0.5916,-7.07534,1600000,224629,242,134,true,-3.1686,0.71824,2013-10-04T00:38:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00393,SNORA8,1654,Small nucleolar RNA SNORA8,Moxon SJ,Moxon SJ,38.0,38.0,37.8,ACA8 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA8,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,5,289,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,109,0,,\"\",0.52411,-7.9894,1600000,291191,254,139,true,-3.1654,0.71823,2013-10-04T00:38:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00394,SNORA4,1655,Small nucleolar RNA SNORA4,Moxon SJ,Moxon SJ,56.0,56.2,55.0,ACA4 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA4,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,95,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,38,0,,\"\",0.48339,-6.98421,1600000,313578,253,141,true,-3.056,0.71816,2013-10-04T00:38:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00396,SNORA13,1254,Small nucleolar RNA SNORA13,Moxon SJ,\"PMID:15199136, INFERNAL\",53.0,53.3,52.6,ACA13 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA13,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,50,171,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; CMfinder,110,0,,\"\",0.50914,-7.48825,1600000,263062,235,134,true,-3.3644,0.71827,2013-02-01T11:57:33.000Z,2021-07-05T14:43:56.000Z\n",
            "RF00397,SNORA14,1656,Small nucleolar RNA SNORA14,Moxon SJ,\"PMID:15199136, INFERNAL\",36.0,36.5,35.3,ACA14 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA14,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,151,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,116,0,,\"\",0.54698,-6.97819,1600000,209380,242,135,true,-3.0778,0.71825,2013-10-04T00:39:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00398,SNORA15,1657,Small nucleolar RNA SNORA15,Moxon SJ,\"PMID:15199136, INFERNAL\",47.0,47.0,46.7,ACA15 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1]. This family also includes the mouse MBI-79 sequence as described in [2].,snoACA15,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,196,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,111,0,,\"\",0.53472,-7.31263,1600000,274740,243,137,true,-2.9023,0.71821,2013-10-04T00:39:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00399,SNORA24,1658,Small nucleolar RNA SNORA24,Moxon SJ,PMID:15199136,40.0,41.3,39.7,ACA24 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA24,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,5,204,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,84,0,,\"\",0.53249,-7.58614,1600000,255742,243,131,true,-3.1839,0.71839,2013-10-04T00:39:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00400,SNORA28,1659,Small nucleolar RNA SNORA28,Moxon SJ,\"PMID:15199136, INFERNAL\",65.0,66.9,64.7,ACA28 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA28,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,217,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD,99,0,,\"\",0.52811,-6.68786,1600000,231888,229,126,true,-3.2519,0.71825,2013-10-04T00:39:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00401,SNORA20,1660,Small nucleolar RNA SNORA20,Moxon SJ,PMID:15199136,60.0,62.0,59.7,ACA20 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA20,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,120,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; ILM; Moxon SJ,48,0,,\"\",0.49786,-6.63717,1600000,216353,236,128,true,-2.9793,0.71835,2013-10-04T00:39:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00402,SNORA25,1661,Small nucleolar RNA SNORA25,Moxon SJ,\"PMID:15199136, INFERNAL\",85.0,85.2,84.7,ACA25 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA25,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,30,266,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,87,0,,\"\",0.51324,-7.74877,1600000,331383,231,128,true,-3.3074,0.71825,2013-10-04T00:39:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00403,SNORA41,1662,Small nucleolar RNA SNORA41,Moxon SJ,\"PMID:15199136, INFERNAL\",80.0,85.0,76.6,ACA41 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1]. The family also includes the mouse sequence MBI-83 described in [2].,snoACA41,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,31,141,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,99,0,,\"\",0.4925,-8.26464,1600000,304740,235,133,true,-3.0538,0.71814,2013-10-04T00:39:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00404,SNORA46,1663,Small nucleolar RNA SNORA46,Moxon SJ,\"Moxon SJ, INFERNAL\",80.0,81.2,76.1,ACA46 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA46,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,105,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,92,0,,\"\",0.50925,-8.2415,1600000,273164,242,135,true,-3.3031,0.71829,2013-10-04T00:40:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00405,SNORA44,1664,Small nucleolar RNA SNORA44,Moxon SJ,\"Moxon SJ, INFERNAL\",55.0,56.2,52.5,ACA44 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1]. The family also contains the mouse snoRNA sequence MBI-64 described in [2].,snoACA44,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,27,314,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,111,0,,\"\",0.59507,-6.94769,1600000,210557,236,131,true,-3.0615,0.71821,2013-10-04T00:40:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00406,SNORA42,1665,Small nucleolar RNA SNORA42/SNORA80 family,\"Moxon SJ, Daub J\",PMID:1637349,70.0,70.0,69.9,\"The members of this family belong to the H/ACA class of snoRNAs that guide the sites of modification of uridines to pseudouridines [1]. Human ACA42 (SNORA42) was cloned from HeLa cells by immunoprecipitation with anti-GAR1 antibody [1]. Its is predicted to guide the pseudouridylation of residues U109 and U572 of 18S rRNA [3]. ACA42 shares the same host gene with ACA26, ACA46, ACA50 [1]. This family also includes mammalian snoRNA ACA67 (SNORA80) which was identified by computational screening and its expression in mouse experimentally verified by Northern blot and primer extension analysis [2]. ACA67 is proposed to guide the pseudouridylation of the same 18S rRNA residues as ACA42 [3].\",snoACA42,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,21,422,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,49,0,,\"\",0.58218,-7.83368,1600000,277855,244,136,true,-3.3149,0.71821,2013-10-04T00:40:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00407,SNORA50,1666,Small nucleolar RNA SNORA50,Moxon SJ,\"Moxon SJ, INFERNAL\",90.0,90.2,89.7,ACA50 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA50,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,120,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold_manual; Daub J,106,0,,\"\",0.51135,-8.54811,1600000,261610,241,136,true,-3.2818,0.71806,2013-10-04T00:40:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00408,SNORA1,1667,Small nucleolar RNA SNORA1,Moxon SJ,\"Moxon SJ, INFERNAL\",70.0,70.1,69.5,ACA1 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA1,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,29,299,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,103,0,,\"\",0.51223,-6.88192,1600000,267400,239,135,true,-3.105,0.71811,2013-10-04T00:40:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00409,SNORA7,1668,Small nucleolar RNA SNORA7,Moxon SJ,\"Moxon SJ, INFERNAL\",100.0,100.0,99.9,ACA7 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1]. The family also contains the mouse sequence MBI-141 as mentioned in [2].,snoACA7,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,31,1698,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,61,0,,\"\",0.58723,-7.38584,1600000,249758,252,140,true,-3.1494,0.71825,2013-10-04T00:40:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00410,SNORA2,1669,Small nucleolar RNA SNORA2/SNORA34 family,\"Moxon SJ, Daub J\",Moxon SJ,70.0,70.0,69.9,\"The members of this family belong to the H/ACA class of snoRNAs which are predicted to guide the sites of modification of uridines to pseudouridines [1]. This family includes human snoRNAs ACA2a (SNORA2A), ACA2b (SNORA2B) and ACA34 (SNORA34). ACA2a and ACA2a are predicted to guide pseudouridylation of U4263 and U4282 in 28S rRNA and ACA34 is predicted to guide the pseudouridylation of residue U4269 and U4282 of 28S rRNA [2]. In humans all three of these snoRNAs share the same host gene [2].\",snoACA2,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,1217,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,96,0,,\"\",0.51553,-7.76598,1600000,290634,243,137,true,-3.2766,0.7181,2013-10-04T00:41:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00411,SNORA9,1670,Small nucleolar RNA SNORA9,Moxon SJ,\"Moxon SJ, INFERNAL\",85.0,85.6,84.0,ACA9 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA9,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,36,186,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,74,0,,\"\",0.58057,-7.5157,1600000,262660,236,133,true,-3.2468,0.7182,2013-10-04T00:41:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00412,SNORA21,1671,Small nucleolar RNA SNORA21,Moxon SJ,Moxon SJ,80.0,80.2,78.6,ACA21 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1]. This family also contains the mouse sequence MBI-3 as shown in [2].,snoACA21,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,191,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,78,0,,\"\",0.56587,-8.24876,1600000,315368,249,136,true,-2.948,0.71806,2013-10-04T00:41:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00413,SNORA19,1672,Small nucleolar RNA SNORA19,Moxon SJ,\"Moxon SJ, INFERNAL\",80.0,80.8,79.9,ACA19 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1]. The family also includes the mouse sequence MBI-51 as shown in [2].,snoACA19,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,34,262,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,113,0,,\"\",0.507,-6.67941,1600000,235158,235,131,true,-3.046,0.71818,2013-10-04T00:41:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00414,SNORA22,1673,Small nucleolar RNA SNORA22,Moxon SJ,\"Moxon SJ, INFERNAL\",80.0,80.0,79.2,ACA22 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA22,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,31,171,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,110,0,,\"\",0.50731,-8.55702,1600000,293173,239,134,true,-3.1794,0.71821,2013-10-04T00:41:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00415,SNORA30,1674,Small nucleolar RNA SNORA30/SNORA37 family,Moxon SJ,Moxon SJ,90.0,91.2,89.9,\"The members of this family belong to the H/ACA class of snoRNAs that guide the sites of modification of uridines to pseudouridines [1]. This family includes human ACA30 (SNORA30), ACA37 (SNORA37) and the mouse homolog MBI-26 [1,2]. Both ACA37 and ACA30 are predicted to guide the pseudouridylation of residue U4643 of 28S rRNA [3].\",snoACA30,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,109,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,38,0,,\"\",0.54712,-7.69391,1600000,251918,235,129,true,-3.3434,0.71836,2013-10-04T00:41:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00416,SNORA43,1675,Small nucleolar RNA SNORA43,Moxon SJ,Moxon SJ,80.0,80.4,79.6,ACA43 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA43,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,120,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,43,0,,\"\",0.56522,-7.40167,1600000,216109,245,138,true,-3.5381,0.71812,2013-10-04T00:41:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00417,SNORA56,1676,Small nucleolar RNA SNORA56,Moxon SJ,\"Moxon SJ, INFERNAL\",80.0,92.5,77.6,ACA56 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA56,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,34,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold_manual; Daub J,34,0,,\"\",0.55116,-7.91923,1600000,280939,235,129,true,-3.3254,0.71835,2013-10-04T00:41:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00418,SNORA58,1677,Small nucleolar RNA SNORA58,Moxon SJ,\"Moxon SJ, INFERNAL\",75.0,75.0,73.8,ACA52 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1]. This family also includes the mouse sequence MBI-12 as reported in [2].,snoACA58,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,124,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,76,0,,\"\",0.55541,-7.19106,1600000,246137,245,136,true,-3.0375,0.71828,2013-10-04T00:41:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00419,SNORA52,1678,Small nucleolar RNA SNORA52,Moxon SJ,\"Moxon SJ, INFERNAL\",71.0,71.2,67.4,ACA52 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA52,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,21,114,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,94,0,,\"\",0.56883,-8.24069,1600000,246627,236,134,true,-3.3214,0.71828,2013-10-04T00:41:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00420,SNORA61,1679,Small nucleolar RNA SNORA61,Moxon SJ,\"Moxon SJ, INFERNAL\",74.0,74.0,73.9,ACA61 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA61,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,202,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,83,0,,\"\",0.54611,-7.86108,1600000,267124,234,132,true,-3.3904,0.71819,2013-10-04T00:41:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00421,SNORA32,1680,Small nucleolar RNA SNORA32,Moxon SJ,Moxon SJ,60.0,60.3,59.5,ACA32 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA32,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,181,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,61,0,,\"\",0.49413,-7.10581,1600000,308657,227,121,true,-3.0914,0.71837,2013-10-04T00:42:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00422,SCARNA24,1681,Small Cajal body specific RNA 24,\"Moxon SJ, Daub J\",\"Moxon SJ, INFERNAL\",70.0,73.6,68.5,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. ACA12 belongs to the H/ACA box class of guide RNAs as it has the predicted hairpin-hinge-hairpin-tail structure,the conserved H/ACA-box motifs and is found associated with GAR1 [2]. ACA12 is predicted to guide the pseudouridylation of residue U40 of the spliceosomal U6 snRNA [2].\",snoACA12,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,166,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAFOLD; Moxon SJ,97,0,,\"\",0.47843,-8.13539,1600000,297410,239,130,true,-3.0561,0.71824,2013-10-04T00:42:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00423,SCARNA4,1682,Small Cajal body specific RNA 4,\"Moxon SJ, Daub J\",\"Moxon SJ, INFERNAL\",60.0,61.2,57.0,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. ACA26 belongs to the H/ACA box class of guide RNAs as it has the predicted hairpin-hinge-hairpin-tail structure, conserved H/ACA-box motifs and is found associated with GAR1 [2]. Its is predicted to guide the pseudouridylation of residues U39 and U41 in U2 snRNA [2].\",snoACA26,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,24,137,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAFOLD; Moxon SJ,96,0,,\"\",0.54504,-7.721,1600000,245670,235,129,true,-3.271,0.71827,2013-10-04T00:42:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00424,SCARNA16,1683,Small Cajal body specific RNA 16,\"Moxon SJ, Daub J\",\"Moxon SJ, INFERNAL\",65.0,66.1,63.9,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. ACA47 belongs to the H/ACA box class of guide RNAs as it has the predicted hairpin-hinge-hairpin-tail structure, the conserved H/ACA-box motifs and is found associated with GAR1 [2]. ACA47 is predicted to guide the pseudouridylation of residue U5 of the U1 spliceosomal snRNA [2].\",snoACA47,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,37,176,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAFOLD; Moxon SJ,93,0,,\"\",0.58323,-8.75938,1600000,289818,301,187,true,-3.2529,0.71374,2013-10-04T00:42:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00425,SNORA18,1684,Small nucleolar RNA SNORA18,Moxon SJ,\"Moxon SJ, INFERNAL\",70.0,70.1,69.8,ACA18 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA18,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.04 -Z 549862.597050 CM SEQDB,29,328,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,116,0,,\"\",0.50987,-7.63096,1600000,230850,235,132,true,-3.2035,0.71823,2013-10-04T00:42:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00426,SCARNA15,1685,Small Cajal body specific RNA 15,\"Moxon SJ, Daub J\",\"Moxon SJ, INFERNAL\",65.0,65.4,64.2,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. ACA45 belongs to the H/ACA box class of guide RNAs as it has the predicted hairpin-hinge-hairpin-tail structure, the conserved H/ACA-box motifs and is found associated with GAR1 [2]. ACA45 is predicted to guide the pseudouridylation of residue U37 of the U2 spliceosomal snRNA [2].\",snoACA45,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.00 -Z 549862.597050 CM SEQDB,22,126,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAFOLD; Moxon SJ,100,0,,\"\",0.53238,-7.69663,1600000,225204,226,128,true,-3.3909,0.7183,2013-10-04T00:42:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00427,SCARNA23,1686,Small Cajal body specific RNA 23,\"Moxon SJ, Daub J\",\"Moxon SJ, INFERNAL\",60.0,61.6,58.6,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. ACA11 belongs to the H/ACA box class of guide RNAs as it has the predicted hairpin-hinge-hairpin-tail structure,the conserved H/ACA-box motifs and is found associated with GAR1 [2]. H/ACA snRNAs are predicted to guide the modification of uridines to pseudouridines in substrate RNAs however, no target RNA has been identified for scaRNA ACA11.\",snoACA11,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,87,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAFOLD; Moxon SJ,77,0,,\"\",0.5735,-7.30926,1600000,263275,229,125,true,-3.2287,0.7184,2013-10-04T00:42:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00428,SNORA38,1687,Small nucleolar RNA SNORA38,Moxon SJ,\"Moxon SJ, INFERNAL\",60.0,60.0,58.0,ACA38 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA38,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,188,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,45,0,,\"\",0.56898,-8.28887,1600000,333273,240,131,true,-3.2124,0.71823,2013-10-04T00:42:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00429,SNORA29,1688,Small nucleolar RNA SNORA29,Moxon SJ,\"Moxon SJ, INFERNAL\",50.0,50.0,49.5,ACA29 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1]. The family also contains the mouse sequence MBI-39 as described in [2].,snoACA29,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,220,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,112,0,,\"\",0.51065,-8.15622,1600000,357249,248,140,true,-3.1003,0.71799,2013-10-04T00:42:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00430,SNORA54,1689,Small nucleolar RNA SNORA54,Moxon SJ,Moxon SJ,60.0,60.1,59.5,ACA54 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA54,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,214,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,108,0,,\"\",0.54603,-8.08148,1600000,255408,234,129,true,-3.1822,0.71826,2013-10-04T00:42:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00431,SNORA55,1690,Small nucleolar RNA SNORA55,Moxon SJ,\"Moxon SJ, INFERNAL\",45.0,47.2,44.0,ACA55 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA55,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,29,240,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,107,0,,\"\",0.58771,-7.0594,1600000,226057,242,135,true,-3.3528,0.71825,2013-10-04T00:42:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00432,SNORA51,1691,Small nucleolar RNA SNORA51,Moxon SJ,Moxon SJ,50.0,50.2,49.9,ACA51 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA51,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,591,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,51,0,,\"\",0.54413,-7.66052,1600000,282968,242,133,true,-3.1913,0.71816,2013-10-04T00:43:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00433,Hsp90_CRE,1692,Hsp90 cis-regulatory element,Moxon SJ,\"PMID:15347681, INFERNAL\",50.0,133.7,47.3,The hsp90 gene product is involved in the Drosophila heat shock response and translation of the hsp90 mRNA is very inefficient under normal growth temperatures. When heat shock occurs translation becomes more efficient. This family represents a section of the Drosophila hsp90 mRNA 5' UTR that is required for increased translational efficiency during the heat shock response. The exact mechanism of action unknown [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,8,0,0,Cis-reg; thermoregulator;,Published; PMID:15347681,6,0,,\"\",0.56434,-5.4155,1600000,207469,272,152,true,-3.3161,0.71803,2013-10-04T00:43:04.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00434,BTE,2491,Luteovirus cap-independent translation element (BTE),Moxon SJ,PMID:15351207,45.0,46.7,32.3,\"This family represents the Luteovirus cap-independent translation element (BTE) as found in barley yellow dwarf virus (BYDV), bean leafroll virus and soybean dwarf virus. This family mediates translation of genomic (gRNA) and subgenomic RNA1 (sgRNA1) [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19 -Z 549862.597050 CM SEQDB,17,11,0,0,Cis-reg;,Published; PMID:15351207,17,0,,\"\",0.70503,-2.55861,1600000,176814,262,100,true,-3.6889,0.71858,2013-10-04T00:43:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00435,ROSE,1694,Repression of heat shock gene expression (ROSE) element,\"Moxon SJ, Bateman A\",PMID:11726689,45.0,45.3,44.5,\"This family represents a 5' UTR cis-regulatory element found in heat shock mRNAs and is known as the ROSE (repression of heat shock gene expression) element. The ROSE element is a negative regulator of heat shock gene expression. The secondary structure is thought to be regulated by temperature. This regulation blocks access to the ribosome binding site under normal conditions. During heat shock however, the structure changes freeing the ribosome binding site and allowing expression to occur [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,238,0,0,Cis-reg; thermoregulator;,Published; PMID:11726689; Griffiths-Jones S,166,0,,\"\",0.45672,-9.34908,1600000,295494,180,94,true,-3.0234,0.71891,2013-10-04T00:43:15.000Z,2022-10-29T17:55:17.000Z\n",
            "RF00436,UnaL2,1695,UnaL2 LINE 3' element,Bateman A,Bateman A,55.0,55.0,54.9,\"The eel long interspersed element (LINE) UnaL2 and its partner short interspersed element (SINE) share a conserved 3' tail containing a stem-loop that is critical for their retrotransposition. Presumably, the first step of retrotransposition is the recognition of their 3' tails by UnaL2-encoded reverse transcriptase. The solution structure of a 17-nucleotide RNA derived from the 3' tail of UnaL2 was determined by NMR [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,30,53249,0,0,Cis-reg;,Published; PMID:15273327,19,0,,\"\",0.59732,-3.43387,1600000,187283,125,54,true,-4.2883,0.71929,2013-10-04T00:43:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00437,HLE,1696,Hairy RNA localisation element (HLE),Moxon SJ,PMID:12743042,60.0,69.2,55.6,This family represents the hairy localisation element (HLE) found in Drosophila species. HLE is located in the 3'-UTR of the hairy gene and contains two stem-loops. HLE is essential for the mediation of apical localisation and the two stem-loop structures act to allow the recognition of hairy mRNA by the localisation machinery [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,5,0,0,Cis-reg;,Published; PMID:12743042,7,0,,\"\",0.51859,-9.8399,1600000,335227,239,130,true,-3.5045,0.71806,2013-10-04T00:45:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00438,SNORA33,1697,Small nucleolar RNA SNORA33,Moxon SJ,\"Moxon SJ, INFERNAL\",74.0,74.4,73.9,ACA33 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA33,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,235,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,96,0,,\"\",0.49483,-6.72171,1600000,249012,235,130,true,-3.1705,0.71828,2013-10-04T00:45:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00439,SNORD87,1698,Small nucleolar RNA SNORD87,Moxon SJ,\"Moxon SJ, INFERNAL\",50.0,51.1,49.9,U87 (HBII-276) is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U87,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,101,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,82,0,,\"\",0.61796,-3.41533,1600000,233464,180,85,true,-3.3882,0.7186,2013-10-04T00:45:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00440,SNORD37,1699,Small nucleolar RNA SNORD37,Moxon SJ,Moxon SJ,45.0,45.0,44.8,U37 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,U37,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 549862.597050 CM SEQDB,6,117,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Moxon SJ,94,0,,\"\",0.57711,-5.75517,1600000,265016,146,65,true,-3.5693,0.71882,2013-10-04T00:45:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00441,snoZ242,1700,Small nucleolar RNA Z242,Moxon SJ,\"Moxon SJ, INFERNAL\",90.0,100.3,33.7,Z242 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,23,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD; Moxon SJ,8,0,,\"\",0.5779,-11.07695,1600000,467998,329,198,true,-2.9783,0.71233,2013-10-04T00:45:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00442,ykkC-yxkD,1701,Guanidine-I riboswitch,\"Moxon SJ, Ontiveros-Palacios N\",\"Barrick JE, Breaker RR\",63.0,63.0,62.9,\"This family represents a conserved RNA structure found upstream of the ykkC and yxkD genes in Bacillus subtilis and related genes in other bacteria. The function of this family is unclear although it has been suggested that it may function to switch on efflux pumps and detoxification systems in response to harmful environmental molecules [1]. The Thermoanaerobacter tengcongensis sequence EMBL:AE013027 overlaps with Rfam:RF00167 suggesting that the two riboswitches may work in conjunction to regulate the upstream gene which codes for Swiss:Q8RC62, a member of Pfam:PF00860 (Moxon SJ, pers. obs.). Nelson [2] showed that this riboswitch senses and responds to guanidine and it was renamed Guanidine-I riboswitch. Furthermore, they demonstrated that bacteria are capable of endogenously producing guanidine and the riboswitch controls genes whose products are involved in modification or pumping out guanidine as a toxic compound from bacteria. Crystal structures of the riboswitch bound to the ligand have also been determined in PDB 5T83, 5U3G and 7MLW [3][4].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,100,831,0,0,Cis-reg; riboswitch;,\"Predicted; Barrick JE, Breaker RR\",747,0,,\"\",0.51445,-9.18196,1600000,313439,215,107,true,-3.6173,0.7184,2013-10-04T00:45:33.000Z,2022-10-29T17:55:17.000Z\n",
            "RF00443,SNORA27,1702,Small nucleolar RNA SNORA27,Moxon SJ,\"Moxon SJ, INFERNAL\",64.0,65.1,63.8,ACA27 is a member of the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].,snoACA27,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,24,175,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold_manual; Daub J,100,0,,\"\",0.51392,-8.33485,1600000,312222,234,127,true,-2.9854,0.71823,2013-10-04T00:45:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00444,PrrF,1703,PrrF RNA,Moxon SJ,PMID:15210934,90.0,95.7,81.1,\"PrrF RNA is a small RNA involved in iron homeostasis and is found exclusively in Pseudomonas species. PrrF sequences are the functional homologues of Rfam:RF00057 and are found in tandem. Their expression is induced under iron limitation by the ferric uptake regulator (Fur) and they act to negatively regulate several genes including sodB and sdh. Both homologues inhibit translation of antR mRNA, which affects quorum sensing and virulence factor production.[2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,18,117,0,0,Gene; sRNA;,Published; PMID:15210934,79,0,,\"\",0.50244,-8.78756,1600000,362519,264,150,true,-3.0554,0.71812,2013-10-04T00:46:07.000Z,2022-05-06T13:10:03.000Z\n",
            "RF00445,mir-399,1704,mir-399 microRNA precursor family,Moxon SJ,Moxon SJ,57.0,57.0,56.9,mir-399 was identified in both Arabidopsis thaliana and Oryza sativa computationally and was later experimentally verified. mir-399 is thought to target mRNAs coding for a phosphatase transporter [1]. The mature sequence is excised from the 3' arm of the hairpin.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,820,0,0,Gene; miRNA;,Predicted; PFOLD,71,0,,\"\",0.72833,-2.87269,1600000,192870,286,118,true,-3.266,0.71846,2013-10-04T00:46:10.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00446,mir-133,1705,mir-133 microRNA precursor family,Moxon SJ,Moxon SJ,55.5,55.6,55.4,mir-133 was first experimentally characterised in mouse [1] and homologues have since been discovered in several other species. The mature sequence is excised from the 3' arm of the hairpin.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,46,621,0,0,Gene; miRNA;,Predicted; PFOLD,202,0,,\"\",0.6949,-2.95966,1600000,106739,170,87,true,-3.9924,0.71878,2013-10-04T00:46:22.000Z,2022-05-06T13:10:03.000Z\n",
            "RF00447,IRES_Kv1_4,1706,Voltage-gated potassium-channel Kv1.4 IRES,Moxon SJ,Published; PMID:15339906,100.0,121.3,99.8,\"This family represents the Kv1.4 voltage-gated potassium-channel internal ribosome entry site (IRES). This region has been shown to mediate internal ribosome entry in cells derived from brain, heart, and skeletal muscle, tissues known to express Kv1.4 mRNA species [1].\",IRES_Kv1.4,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,37,0,0,Cis-reg; IRES;,Predicted; RNAalifold,39,0,,\"\",0.57617,-10.29278,1600000,399083,403,260,true,-3.7451,0.70785,2013-10-04T00:46:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00448,IRES_EBNA,2472,Epstein-Barr virus nuclear antigen (EBNA) IRES,Moxon SJ,Published; 12555070,100.0,220.3,32.0,This family represents an Epstein-Barr virus internal ribosome entry site (IRES) which is found in the U leader exon in the 5' UTR of the Epstein-Barr nuclear antigen 1 (EBNA1) gene. The EBNA IRES allows EBNA1 translation to occur under situations where initiation from the 5' cap structure and ribosome scanning is reduced. It is thought that the EBNA IRES is necessary for the regulation of latent gene expression [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,6,4,0,0,Cis-reg; IRES;,Predicted; PFOLD,3,0,,\"\",0.64802,-9.46433,1600000,404017,350,217,true,-3.7457,0.71077,2013-10-04T00:46:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00449,IRES_HIF1,1708,HIF-1 alpha IRES,Moxon SJ,\"http://ifr31w3.toulouse.inserm.fr/IRESdatabase/, INFERNAL\",110.0,121.4,105.1,\"This family represents the hypoxia-inducible factor 1-alpha internal ribosome entry site. HIF-1a is a subunit of the HIF-1 transcription factor, which induces transcription of several genes involved in the cellular response to hypoxia. The HIF-1a IRES allows translation to be maintained under hypoxic cell conditions that inhibit cap-dependent translation [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17 -Z 549862.597050 CM SEQDB,17,27,0,0,Cis-reg; IRES;,Predicted; RNAFOLD; Moxon SJ,31,0,,\"\",0.6472,-9.55089,1600000,363695,421,279,true,-3.7244,0.70682,2013-10-04T00:46:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00451,mir-395,1255,mir-395 microRNA precursor family,Moxon SJ,Moxon SJ,56.0,56.0,55.9,mir-395 was identified in both Arabidopsis thaliana and Oryza sativa computationally and was later experimentally verified. mir-395 is thought to target mRNAs coding for ATP sulphurylases [1]. The mature sequence is excised from the 3' arm of the hairpin.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,25,1065,0,0,Gene; miRNA;,Predicted; PFOLD,74,0,,\"\",0.63467,-3.63138,1600000,166221,392,108,true,-2.9819,0.71867,2013-02-01T11:57:36.000Z,2021-11-19T10:36:02.000Z\n",
            "RF00452,mir-172,1709,mir-172 microRNA precursor family,Moxon SJ,Moxon SJ,70.0,70.0,69.9,\"mir-172 has been experimentally verified in Arabidopsis thaliana [2,3]. This family is thought to target mRNAs coding for APETALA2-like transcription factors [1]. The mature sequence is excised from the 3' arm of the hairpin.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,644,0,0,Gene; miRNA;,Predicted; PFOLD; Moxon SJ,73,0,,\"\",0.63606,-4.49623,1600000,161611,621,119,true,-3.4363,0.71844,2013-10-04T00:47:10.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00453,Cardiovirus_CRE,1710,Cardiovirus cis-acting replication element (CRE),Moxon SJ,PMID:10500216,37.0,37.3,36.7,This family represents a Cardiovirus cis-acting replication element (CRE) which is located within the region encoding the capsid protein VP2 and is required for viral replication [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Cis-reg;,Published; PMID:10500216,3,0,,\"\",0.57589,-5.14218,1600000,395590,104,33,true,-2.6612,0.73696,2013-10-04T00:47:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00454,p27_CRE,1711,p27 cis-regulatory element,Moxon SJ,\"PMID:12837699, INFERNAL\",42.0,42.7,35.3,This family represents a structured G/C rich element located in the 5' UTR of p27 mRNA. This element contributes to cell-cycle regulated translation and when deleted leads to a 50% drop in regulation. The element is thought to be able to form several different structures with similar free energy. It has been suggested that proteins may only bind to a specific structure thus conformational changes could regulate translation [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,25,0,0,Cis-reg;,Predicted; PFOLD; Moxon SJ,28,0,,\"\",0.6259,-5.23041,1600000,376597,140,59,true,-3.3263,0.71937,2013-10-04T00:47:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00455,mir-15,1712,mir-15 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,57.3,57.3,57.1,This family represents the related mir-15a and mir-15b sequences. In humans miR-15a and miR-16 are clustered within 0.5 kb at 13q14. This region has been shown to be deleted in more than half of B cell chronic lymphocytic leukaemias (CLL). Both miR-15a and miR-16 are deleted or down-regulated in more than two thirds of CLL cases [2]. The mature sequence is excised from the 5' arm of the hairpin. The family represents the microRNA (miRNA) precursor mir-125 imported from miRBase [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,164,296,0,0,Gene; miRNA;,Predicted; RNAalifold,129,0,,\"\",0.73838,-2.54442,1600000,177245,157,88,true,-3.8994,0.71878,2013-10-04T00:47:23.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00456,mir-34,1713,mir-34 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.4,53.5,53.3,Human mir-34a was discovered computationally [1] and later experimentally verified [2]. The mature sequence is excised from the 5' arm of the hairpin. Precursor mir-34 imported from miRBase [4],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,113,421,0,0,Gene; miRNA;,Predicted; RNAalifold,216,0,,\"\",0.65832,-3.74783,1600000,186089,170,72,true,-3.7576,0.71925,2013-10-04T00:47:26.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00457,IRES_mnt,1714,Mnt IRES,Moxon SJ,\"Published; 11314024, INFERNAL\",100.0,118.3,90.1,This family represents the Mnt internal ribosome entry site (IRES). Mnt is a transcriptional repressor related to the Myc/Mad family of transcription factors. It is thought that this family allows efficient Mnt synthesis when cap-dependent translation initiation is reduced [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,33,0,0,Cis-reg; IRES;,Predicted; PFOLD; Moxon SJ,38,0,,\"\",0.66516,-8.67748,1600000,395728,332,203,true,-3.2251,0.71199,2013-10-04T00:47:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00458,IRES_Cripavirus,1715,Cripavirus internal ribosome entry site (IRES),Moxon SJ,Published; 11233983,80.0,97.6,52.9,This family represents a Cripavirus internal ribosome entry site (IRES) which is required for the production of capsid proteins through IRES-mediated translation [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,10,0,0,Cis-reg; IRES;,Published; PMID:11233983,16,0,,\"\",0.5412,-9.06514,1600000,431797,314,201,true,-3.514,0.71214,2013-10-04T00:47:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00459,MPMV_package,1716,Mason-Pfizer monkey virus packaging signal,Moxon SJ,\"PMID:14687944, INFERNAL\",54.0,54.0,53.9,This family represents a group of sequences that are related to the Mason-Pfizer monkey virus (MPMV) packaging signal. The packaging signal is located in the 5' UTR and is required for specific RNA encapsidation [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,614,0,0,Cis-reg;,Predicted; PFOLD; Moxon SJ,23,0,,\"\",0.62806,-8.35516,1600000,421648,419,279,true,-3.7729,0.70689,2013-10-04T00:47:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00460,U1A_PIE,1717,U1A polyadenylation inhibition element (PIE),\"Moxon SJ, Weinberg Z\",PMID:15491147,50.0,53.3,47.7,\"This family represents the U1A polyadenylation inhibition element (PIE). PIE is a regulatory RNA which is responsible for the regulation of the length of the polyA tail of the U1A pre-mRNA. The PIE is located in the U1A mRNA 3' UTR. PIE adopts a U-shaped structure, with binding sites for a single U1A protein at each bend and when complexed with the two proteins it blocks activity of poly(A) polymerase (PAP), and inhibits its activity [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,39,94,0,0,Cis-reg;,Published; PMID:15491147,63,0,,\"\",0.66571,-1.88582,1600000,171269,151,74,true,-4.2185,0.7189,2013-10-04T00:47:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00461,IRES_VEGF_A,1718,Vascular endothelial growth factor (VEGF) IRES A,Moxon SJ,Published; PMID:9774635,160.0,162.0,158.5,\"This family represents the vascular endothelial growth factor (VEGF) internal ribosome entry site (IRES) A. VEGF is an endothelial cell mitogen with many crucial functions such as embryogenic development and wound healing. The 5' UTR of VEGF mRNA contains two IRES elements which are able to promote efficient translation at the AUG start codon [1], this family represents IRES A.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,24,0,0,Cis-reg; IRES;,Published; PMID:9774635,26,0,,\"\",0.57286,-11.36968,1600000,501802,462,308,true,-3.6311,0.70566,2013-10-04T00:47:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00462,IRES_APC,1719,APC internal ribosome entry site (IRES),Moxon SJ,Published; 12034871,50.0,59.9,40.3,This family represents an APC internal ribosome entry site (IRES) which is located in the coding sequence of the gene. APC is a tumour suppressor gene which is associated with the inherited disease adenomatous polyposis coli (APC).It is thought that IRES-mediated translation of APC is important for an apoptotic cascade [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,6,0,0,0,Cis-reg; IRES;,Published; PMID:12034871,4,0,,\"\",0.63265,-4.03843,1600000,301276,130,53,true,-3.7429,0.71927,2013-10-04T00:47:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00463,ApoB_5_CRE,1720,Apolipoprotein B (apoB) 5' UTR cis-regulatory element,Moxon SJ,PMID:15157107,55.0,68.1,51.2,This family represents an apolipoprotein B (apoB) cis regulatory element located in the 5' UTR of apoB mRNA. The element is important for optimal translation [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,5,9,0,0,Cis-reg;,Published; PMID:15157107,9,0,,\"\",0.43296,-10.27235,1600000,310891,238,127,true,-2.913,0.71832,2013-10-04T00:48:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00464,mir-92,1721,mir-92 microRNA precursor family,Moxon SJ,Moxon SJ,47.0,49.3,46.8,This family represents the microRNA precursor mir-92. Human miR-92 has two predicted hairpin precursor sequences: mir-92-1 on chromosome 13 and mir-92-2 on chromosome X. miR-92 has also been cloned from mouse embryonic stem cells [2]. The mature sequence is excised from the 3' arm of the hairpin.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,23,220,0,0,Gene; miRNA;,Predicted; PFOLD,121,0,,\"\",0.75783,-2.39259,1600000,181452,166,80,true,-3.955,0.71896,2013-10-04T00:48:06.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00467,RSV_PBS,1256,Rous sarcoma virus (RSV) primer binding site (PBS),Moxon SJ,PMID:15153244,47.0,47.3,46.5,This family represents a structured region around the Rous sarcoma virus primer binding site (PBS). This region is known to be required for the efficient initiation of reverse transcription [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,31,0,0,Cis-reg;,Published; PMID:15153244,18,0,,\"\",0.63653,-3.30337,1600000,238434,165,75,true,-3.6742,0.71883,2013-02-01T11:57:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00468,HCV_SLVII,1722,Hepatitis C virus stem-loop VII,\"Lamkiewicz K, Triebel S, Marz M\",Kevin Lamkiewicz; Sandra Triebel; Manja Marz;,63.0,63.2,36.3,Genetic modification of RNA structures in Hepatitis C virus (HCV) genome affects replication and infectivity [1] Long distant RNA-RNA interactions in HCV RNAs are involved in translation to virion budding [2] It has been proposed that the 5'-3' UTR base-pairing in the HCV genome might play an important role in viral RNA replication [3],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,57,1,0,0,Cis-reg;,\"Predicted; BM VeGETA v. 0.4, LocARNA 2.0.0RC8, RNAfold\",21,0,,\"\",0.69225,-3.25197,1600000,203003,150,70,true,-3.7943,0.71905,2013-10-04T00:48:23.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00470,Toga_5_CRE,1724,Togavirus 5' plus strand cis-regulatory element,Moxon SJ,PMID:11720292,120.0,140.6,115.0,This family represents a Togavirus virus 5' plus strand cis-regulatory element which is thought to be essential for both plus and minus strand RNA synthesis [1].Genus Alphaviruse belongs to the Togaviridae family. Alpha viruses contain secondary structural motifs in the 5' UTR that allow them to avoid detection by IFIT1[2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,33,3,0,0,Cis-reg;,Predicted; PFOLD,9,0,,\"\",0.67006,-9.32141,1600000,409597,415,277,true,-3.6546,0.70696,2013-10-04T00:48:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00471,snosnR48,1725,Small nucleolar RNA snR48,Moxon SJ,Moxon SJ,60.0,67.2,57.2,snR48 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 36.46 -Z 549862.597050 CM SEQDB,6,54,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; ILM,59,0,,\"\",0.40773,-9.17395,1600000,351052,213,113,true,-2.9066,0.71809,2013-10-04T00:48:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00472,snosnR55,1726,Small nucleolar RNA snR55/Z10,Moxon SJ,Moxon SJ,50.0,51.6,48.6,snR55 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. This family does not appear to have the C/D box snoRNA terminal stem structure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,53,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones S,57,0,,\"\",0.48376,-9.1425,1600000,287207,181,92,false,-3.3492,0.71837,2013-10-04T00:48:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00473,snosnR54,1727,Small nucleolar RNA snR54,Moxon SJ,Moxon SJ,50.0,61.8,46.2,snR54 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.20 -Z 549862.597050 CM SEQDB,8,65,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones S,71,0,,\"\",0.44195,-8.49368,1600000,292989,309,87,false,-3.4564,0.71843,2013-10-04T00:48:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00474,snosnR57,1728,Small nucleolar RNA snR57,Moxon SJ,Moxon SJ,50.0,52.8,46.7,snR57 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,59,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones S,59,0,,\"\",0.5527,-2.1777,1600000,198884,189,95,true,-3.1903,0.71834,2013-10-04T00:48:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00475,snosnR69,1729,Small nucleolar RNA snR69,Moxon SJ,Moxon SJ,50.0,50.0,49.9,snR69 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,73,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PFOLD,73,0,,\"\",0.46515,-9.0334,1600000,297705,243,101,true,-3.3829,0.71827,2013-10-04T00:48:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00476,snosnR61,1730,Small nucleolar RNA snR61/Z1/Z11,\"Moxon SJ, Daub J\",Moxon SJ,45.0,45.1,44.7,\"This family of small nucleolar RNAs belongs to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family includes Schizosaccharomyces pombe snR61, Drosophila melanogaster Z1 and Saccharomyces cerevisiae snR61 and Z11 snoRNAs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,56,578,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,530,0,,\"\",0.44138,-10.76428,1600000,364100,164,85,false,-3.7473,0.71848,2013-10-04T00:48:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00477,snosnR66,1731,Small nucleolar RNA snR66,Moxon SJ,Moxon SJ,65.0,69.1,63.2,snR66 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.00 -Z 549862.597050 CM SEQDB,4,29,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Griffiths-Jones S,30,0,,\"\",0.55148,-2.73028,1600000,193028,210,104,true,-3.2086,0.71825,2013-10-04T00:49:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00478,SCARNA6,1732,Small Cajal body specific RNA 6,\"Moxon SJ, Marantidis E, Daub J\",\"Moxon SJ, Daub J, INFERNAL\",68.0,68.0,67.7,\"U88 is small Cajal body specific RNA (scaRNA). scaRNAs are a novel class of small nuclear RNAs which localise to the Cajal bodies [1]. U88 is found associated with both fibrillarin and Gar1p and co-localises with coilin in Cajal bodies [1]. U88 scaRNA is composed of both an H/ACA box and a C/D box conserved domains domains and is predicted to guide 2'-O-methylation of residue U41 of the U5 snRNA [1]. This family also includes human scaRNA U87 and a mouse homologue MBI-46 described in [2]. In humans both U88 and U87 scaRNAs share the same host gene and both are predicted to guide modification of the same residue (U41 of U5 snRNA)[1,3].\",U88,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,214,0,0,Gene; snRNA; snoRNA; scaRNA;,\"Published; PMID:12032087, Daub J\",111,0,,\"\",0.69028,-9.90054,1600000,443601,400,265,true,-3.7226,0.70759,2013-10-04T00:49:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00479,snosnR71,1733,Small nucleolar RNA snR71,Moxon SJ,Moxon SJ,60.0,60.0,59.4,snR71 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,58,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Moxon SJ,63,0,,\"\",0.51237,-5.19071,1600000,204950,188,97,true,-3.3347,0.71831,2013-10-04T00:49:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00480,HIV_FE,1734,HIV Ribosomal frameshift signal,Moxon SJ,PMID:14747573,42.0,42.0,41.9,\"This family represents the HIV Ribosomal frameshift signal. HIV uses a system called -1 ribosomal frameshifting to produce the required ratio of Gag and Gag-Pol polyproteins. The stem-loop structure of the frameshift signal is thought to impede the ribosome and cause slippage in the 5' direction, this causes the -1 frameshift and translation then continues in the new frame [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,145,16,0,0,Cis-reg; frameshift_element;,Published; PMID:14747573,9,0,,\"\",0.44781,-9.87817,1600000,609843,122,52,true,-3.9181,0.7193,2013-10-04T00:49:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00481,HCV_X3,1735,Hepatitis C virus 3' X element,\"Lamkiewicz K, Triebel S, Marz M\",Kevin Lamkiewicz; Sandra Triebel; Manja Marz;,65.0,67.0,37.9,Genetic modification of RNA structures in Hepatitis C virus (HCV) genome affects replication and infectivity [1] Long distant RNA-RNA interactions in HCV RNAs are involved in translation to virion budding [2] It has been proposed that the 5'-3' UTR base-pairing in the HCV genome might play an important role in viral RNA replication [3],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,1,0,0,Cis-reg;,\"Predicted; BM VeGETA v. 0.4, LocARNA 2.0.0RC8, RNAfold\",10,0,,\"\",0.51961,-9.22544,1600000,336142,208,105,true,-3.5497,0.71871,2013-10-04T00:49:48.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00482,snoF1_F2,2398,Small nucleolar RNA F1/F2/snoR5a,Moxon SJ,Moxon SJ,60.0,60.0,59.8,\"This family contains F1, F2 and snoR5a sequences, all of which are members of the the H/ACA class of snoRNA that guide the sites of modification of uridines to pseudouridines [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,132,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD,34,0,,\"\",0.56275,-7.02959,1600000,298328,254,140,true,-3.4802,0.71816,2013-10-04T00:49:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00483,IRES_IGF2,1737,Insulin-like growth factor II IRES,Moxon SJ,Moxon SJ,70.0,80.4,48.7,This family represents the insulin-like growth factor II (IGF-II) internal ribosome entry site (IRES) which is found in the 5' UTR of IGF-II leader 2 mRNA. This family allows cap-independent translation of the mRNA and it is thought that this family may facilitate a continuous IGF-II production in rapidly dividing cells during development [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,9,27,0,0,Cis-reg; IRES;,Published; PMID:11903044,30,0,,\"\",0.5735,-9.35093,1600000,352065,230,121,true,-3.1911,0.71834,2013-10-04T00:49:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00484,IRES_Cx32,2499,Connexin-32 internal ribosome entry site (IRES),Moxon SJ,Moxon SJ,80.0,83.3,34.7,This family represents the connexin-32 internal ribosome entry site (IRES) which allows cap independent translation during conditions such as heat shock and stress [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,6,35,0,0,Cis-reg; IRES;,Published; PMID:10931843,35,0,,\"\",0.66593,-5.50956,1600000,241927,268,149,true,-3.0771,0.71815,2013-10-04T00:50:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00485,K_chan_RES,1739,Potassium channel RNA editing signal,Moxon SJ,PMID:15361858,51.0,51.0,50.9,This family represents a group of sequences related to the human potassium channel Kv1.1 hairpin element which directs the efficient modification of an adenosine to inosine by an adenosine deaminase acting on RNA (ADAR). The ADAR modification causes an isoleucine/valine recoding event which lies in the ion-conducting pore of the potassium channel. It is thought that this editing event targets the process of fast inactivation and allows a more rapid recovery from inactivation at negative potentials [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,85,2986,0,0,Cis-reg;,Published; PMID:15361858; Moxon SJ,311,0,,\"\",0.59736,-4.93438,1600000,266350,211,114,true,-3.4065,0.71829,2013-10-04T00:50:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00486,mir-129,1740,mir-129 microRNA precursor family,Moxon SJ,Moxon SJ,50.0,60.4,49.6,mir-129 was first experimentally characterised in mouse [1] and homologues have since been discovered in several other species. The mature sequence is excised from the 5' arm of the hairpin.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,171,0,0,Gene; miRNA;,Predicted; PFOLD,76,0,,\"\",0.64994,-3.74144,1600000,157439,187,92,true,-3.7606,0.71877,2013-10-04T00:50:21.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00487,IRES_Cx43,2498,Connexin-43 internal ribosome entry site (IRES),Moxon SJ,Moxon SJ,70.0,70.3,65.4,This family represents the connexin-43 internal ribosome entry site (IRES) which allows cap independent translation during conditions such as heat shock and stress [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,127,0,0,Cis-reg; IRES;,Published; PMID:10618489,88,0,,\"\",0.63004,-8.42785,1600000,398376,334,204,true,-3.4775,0.71206,2013-10-04T00:50:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00488,U1_yeast,1742,Yeast U1 spliceosomal RNA,Griffiths-Jones SR,\"Zwieb C, The uRNA database, PMID:9016512\",80.0,87.6,70.7,\"U1 is a small nuclear RNA (snRNA) component of the spliceosome (involved in pre-mRNA splicing). Its 5' end forms complementary base pairs with the 5' splice junction, thus defining the 5' donor site of an intron. There are significant differences in sequence and secondary structure between metazoan and yeast U1 snRNAs, the latter being much longer (568 nucleotides as compared to 164 nucleotides in human). Nevertheless, secondary structure predictions suggest that all U1 snRNAs share a 'common core' consisting of helices I, II, the proximal region of III, and IV [1]. This family represents the larger yeast sequences. Family Rfam:RF00003 contains the metazoan U1 sequences.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,35,0,0,Gene; snRNA; splicing;,Published; PMID:2405391,38,0,,\"\",0.55089,-10.42187,1600000,476340,838,569,true,-4.6039,0.69985,2013-10-04T00:50:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00489,ctRNA_p42d,1514,ctRNA,Moxon SJ,PMID:15554980,50.0,50.9,49.7,\"This family represents a group of ctRNAs (or counter-transcribed RNAs) from p42d and related plasmids. ctRNAs are small, highly structured, antisense sequences which are involved in negative regulation. Members of this family are known to bind to the mRNA of repC causing translational repression [1]. This model contains the predicted stem-loop structure followed by a U-rich tract which is proposed to act as a Rho-independent transcriptional terminator.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,212,0,0,Gene; antisense;,Published; PMID:15554980,103,0,,\"\",0.67674,-3.71757,1600000,365440,117,46,true,-3.6299,0.71965,2013-10-04T00:50:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00490,S-element,1743,S-element,Moxon SJ,PMID:15554980,47.0,47.4,45.1,\"This family represents a group of S-elements from p42d and related plasmids. The S-element has multiple functions and is believed to act as a negative regulator of repC transcription, be required for efficient replication and act as a translational enhancer of repC [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,29,205,0,0,Cis-reg;,Published; PMID:15554980,91,0,,\"\",0.52682,-7.18725,1600000,427953,143,67,true,-3.8706,0.7191,2013-10-04T00:50:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00492,SCARNA17,1744,small Cajal body-specific RNA 17,\"Moxon SJ, Daub J\",Moxon SJ,75.0,76.3,74.7,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. The complete human U12-22/U4-8 scaRNA is composed of two tandem C/D box domains (termed U12-22 and U4-8). The 5' and 3' C/D domains are predicted to guide the 2'O-ribose methylation of residue U22 in U12 (RFAM:RF00007) and residue C8 in U4 (RFAM:RF00015) snRNAs respectively [1]. This family only includes only the 5' C/D box domain (U12-22) as the 3' C/D box is represented by RFAM:RF00283. The 3' C/D domain (U4-8) was also cloned previously by Darzacq [2] and called U91. Both the doublet (U12-22/U4-8) and singlet (U4-8) forms of this snRNA have been purified from Hela cells [1]. The doublet form U12-22/U4-8 has been shown to localise to the nucleoplasm and is proposed to reside in the Cajal bodies whereas the U4-8 single domain appears to accumulate in the nucleolus [1]. In humans the genomic location of U12-88/U4-8 is intergenic and the purified transcript has been shown to possess a methylated guanosine cap suggesting it is independently transcribed by RNA pol II [1].\",snoU12-22,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.32 -Z 742849.287494 CM SEQDB,6,70,0,0,Gene; snRNA; snoRNA; scaRNA;,Published;PMID:15556860,36,0,,\"\",0.57819,-5.81125,1600000,243131,257,144,true,-3.4068,0.71818,2013-10-04T00:50:42.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00493,snoU2-30,1745,Small nucleolar RNA U2-30,Moxon SJ,\"Moxon SJ, INFERNAL\",60.0,60.7,57.4,U2-30 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs the 2'-O-methylation of A30 in Rfam:RF00004 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,130,0,0,Gene; snRNA; snoRNA; CD-box;,Published;PMID:15556860,44,0,,\"\",0.62415,-3.65951,1600000,289981,153,70,true,-3.1377,0.71891,2013-10-04T00:50:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00494,snoU2_19,1746,Small nucleolar RNA U2-19,Moxon SJ,\"Moxon SJ, INFERNAL\",39.0,39.1,38.5,U2-19 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs the 2'-O-methylation of G19 in Rfam:RF00004 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.62 -Z 549862.597050 CM SEQDB,29,223,0,0,Gene; snRNA; snoRNA; CD-box;,Published;PMID:15556860,105,0,,\"\",0.44744,-8.44765,1600000,261960,163,80,true,-3.7439,0.71869,2013-10-04T00:50:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00495,IRES_Hsp70,1747,Heat shock protein 70 (Hsp70) internal ribosome entry site (IRES),Moxon SJ,Published,80.0,91.8,70.6,This family represents heat shock protein 70 (Hsp70) internal ribosome entry site (IRES) which allows cap independent translation during conditions such as heat shock and stress.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,30,0,0,Cis-reg; IRES;,Predicted; PFOLD; Moxon SJ,11,0,,\"\",0.6564,-9.4568,1600000,435496,346,216,true,-3.662,0.71097,2013-10-04T00:50:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00498,EAV_LTH,1749,Equine arteritis virus leader TRS hairpin (LTH),Moxon SJ,PMID:14970388,70.0,76.9,40.4,This family represents the equine arteritis virus leader transcription-regulating sequence hairpin (LTH). The LTH is thought to be a key structural element in discontinuous subgenomic RNA synthesis and is critical for leader TRS function [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,6,1,0,0,Cis-reg; leader;,Published; PMID:14970388,2,0,,\"\",0.73532,-3.43877,1600000,380649,120,45,true,-3.8035,0.71954,2013-10-04T00:51:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00499,Parecho_CRE,1750,Human parechovirus 1 (HPeV1) cis regulatory element (CRE),Moxon SJ,PMID:12438643,55.0,59.9,52.5,This family represents a Parechovirus cis regulatory element which is located in the 5'-terminal 112 nucleotides of the genome of human parechovirus 1 (HPeV1). The family consists of two stem-loop structures (SL-A and SL-B) together with a pseudoknot. Disruption of any of these elements impairs both viral replication and growth [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19 -Z 549862.597050 CM SEQDB,5,2,0,0,Cis-reg;,Published; PMID:12438643,4,0,,\"\",0.5884,-6.01832,1600000,210447,222,116,true,-2.949,0.7186,2013-10-04T00:51:13.000Z,2019-01-10T15:16:02.000Z\n",
            "RF00500,TCV_H5,1751,Turnip crinkle virus (TCV) repressor of minus strand synthesis H5,Moxon SJ,PMID:15220455,70.0,72.9,37.0,\"This family represents the turnip crinkle virus (TCV) hairpin 5 (H5) which contains a large symmetrical internal loop (LSL). H5 can repress minus-strand synthesis when the 3' side of the LSL pairs with the 3'-terminal 4 bases of the RNA(GCCC-OH) [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,1,0,0,Cis-reg;,Published; PMID:15220455,2,0,,\"\",0.60626,-5.48504,1600000,430441,120,45,true,-3.8268,0.71957,2013-10-04T00:51:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00501,Rota_CRE,1752,Rotavirus cis-acting replication element (CRE),Moxon SJ,PMID:,40.0,40.2,39.4,This family represents a Rotavirus cis-acting replication element (CRE) found at the 3'-end of rotavirus mRNAs. The family is thought to promote the synthesis of minus strand RNA to form viral dsRNA [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,1,0,0,Cis-reg;,Predicted; Pfold; Moxon SJ,7,0,,\"\",0.63429,-4.05967,1600000,338093,153,68,true,-3.3555,0.71906,2013-10-04T00:51:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00502,TCV_Pr,1753,Turnip crinkle virus (TCV) core promoter hairpin (Pr),Moxon SJ,PMID:15220455,40.0,46.3,37.7,This family represents the turnip crinkle virus (TCV) core promoter hairpin (Pr). Pr is located in the 3' UTR and is required for minus strand RNA synthesis [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Cis-reg;,Published; PMID:15220455,2,0,,\"\",0.6137,-5.34,1600000,457057,105,34,true,-2.8611,0.73062,2013-10-04T00:51:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00503,RNAIII,1754,RNAIII,Argasinska J,Gardner P,47.0,47.6,34.6,\"This family represents RNAIII which is known to regulate the expression of many Staphylococcus aureus genes encoding exoproteins and cell wall associated proteins. In S.aureus, RNAIII acts as the effector of the agr system and is transcribed from the P3 operon. The RNAIII transcript also contains the 26 amino acid delta-haemolysin gene (hld) [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,10,20,0,0,Gene;,Published; PMID:26504242,26,0,,\"\",0.638,-8.15301,1600000,339240,834,511,true,-4.0305,0.70068,2013-10-04T00:51:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00504,Glycine,1755,Glycine riboswitch,Moxon SJ,\"Barrick JE, Breaker RR\",32.0,32.0,31.9,\"This family represents the bacterial glycine riboswitch. Glycine riboswitches usually consist of two metabolite-binding aptamers domains with similar structures in tandem. The aptamers cooperatively bind glycine to regulate the expression of downstream genes. In Bacillus subtilis, this riboswitch is found upstream of the gcvT operon which controls glycine degradation. It is thought that when glycine is in excess it will bind to both aptamers to activate these genes and facilitate glycine degradation [1].Â In vivo experiments demonstrated that glycine does not need to bind both aptamers for regulation. Mutation to the first aptamer caused greatest reduction in downstream gene expression, while mutation to the second one had varying effects. Glycine-induced expression of the gcvT operon is needed for B. subtilise growth, swarming motility and biofilm formation (in high glycine enviroment).[2]\",gcvT,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,44,4556,0,0,Cis-reg; riboswitch;,\"Predicted; Barrick JE, Breaker RR\",3019,0,,\"\",0.60968,-3.97297,1600000,143931,359,94,true,-4.0151,0.71854,2013-10-04T00:51:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00505,RydC,1756,RydC RNA,Moxon SJ,PMID:15618228,45.0,45.8,30.9,\"This family represents a bacterial non-coding RNA called RydC. RydC is thought to regulate an mRNA, yejABEF which encodes an ABC transporter protein. RydC is known to bind the protein Hfq which causes a conformational change in the RNA molecule. The Hfq/RydC complex is then thought to bind to the target mRNA and induce its degradation [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,16,0,0,Gene; sRNA;,Published; PMID:15618228,21,0,,\"\",0.54152,-6.73153,1600000,323325,148,64,true,-3.7338,0.71905,2013-10-04T00:51:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00506,Thr_leader,1757,Threonine operon leader,Bateman A,Bateman A,55.0,56.5,54.8,At least 6 amino acid operons are known to be regulated by attenuation [1]. In each a leader sequence of 150-200 bp is found upstream of the first gene in the operon. This leader sequence can assume two different secondary structures known as the terminator and the anti-terminator structure. In each case the leader also codes for very short peptide sequence that is rich in the end product amino acid of the operon. The terminator structure is recognised as a termination signal for RNA polymerase and the operon is not transcribed. This structure forms when the cell has an excess of the regulatory amino acid and ribosome movement over the leader transcript is not impeded [1]. When there is a deficiency of the charged tRNA of the regulatory amino acid the ribosome translating the leader peptide stalls and the antiterminator structure can form. This allows RNA polymerase to transcribe the operon [1]. In the Rfam alignment the consensus structure is of the terminator. The alternative consensus structure is the antiterminator structure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,174,0,0,Cis-reg; leader;,Published; PMID:6186194,180,0,,\"\",0.53939,-7.63489,1600000,287172,228,118,true,-3.1373,0.71842,2013-10-04T00:51:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00507,Corona_FSE,1758,Coronavirus frameshifting stimulation element,\"Moxon SJ, Nawrocki E, Ontiveros-Palacios N\",PMID:15680415,46.0,46.4,33.1,\"This family represents a conserved stem-loop found in Coronaviruses which can promote ribosomal frameshifting. This family interacts with a downstream region to form a pseudoknot structure, the region varies according to the virus but pseudoknot formation is known to stimulate frameshifting. In the classical situation a sequence 32nt downstream of the stem is complementary to part of the loop. In other Coronaviruses however, there is another stem loop structure around 150nt downstream that can interact with this family to form kissing stem loops and stimulate frameshifting [1]. The structure of coronavirus frameshifting has been reported in PDB 6XRZ [2]. Revised in 2020-2021 to expand the seed alignment.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,50,156,0,0,Cis-reg; frameshift_element;,Published; PMID:15680415,87,0,,\"\",0.65017,-4.75871,1600000,291300,155,78,true,-3.3721,0.71875,2013-10-04T00:52:14.000Z,2022-10-29T17:55:17.000Z\n",
            "RF00509,snosnR64,1759,Small nucleolar RNA snR64,Moxon SJ,\"Moxon SJ, INFERNAL\",38.0,43.2,37.0,snR64 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,42,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,47,0,,\"\",0.47007,-10.8106,1600000,289243,202,106,false,-3.3183,0.71815,2013-10-04T00:52:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00510,Tombus_IRE,1760,Tombusvirus internal replication element (IRE),Moxon SJ,PMID:15795270,45.0,64.8,33.1,This family represents the Tombusvirus internal replication element (IRE) which is located within the region coding for p92 polymerase. The element is essential for viral replication and is thought to be required at an early stage of this process such as template recruitment and/or replicase complex assembly [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,2,0,0,Cis-reg;,Predicted; PFOLD,10,0,,\"\",0.64184,-2.97537,1600000,224392,188,94,true,-3.3858,0.71875,2013-10-04T00:52:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00512,Leu_leader,1762,Leucine operon leader,Bateman A,Bateman A,50.0,51.2,47.4,At least 6 amino acid operons are known to be regulated by attenuation [1]. In each a leader sequence of 150-200 bp is found upstream of the first gene in the operon. This leader sequence can assume two different secondary structures known as the terminator and the anti-terminator structure. In each case the leader also codes for very short peptide sequence that is rich in the end product amino acid of the operon. The terminator structure is recognised as a termination signal for RNA polymerase and the operon is not transcribed. This structure forms when the cell has an excess of the regulatory amino acid and ribosome movement over the leader transcript is not impeded [1]. When there is a deficiency of the charged tRNA of the regulatory amino acid the ribosome translating the leader peptide stalls and the antiterminator structure can form. This allows RNA polymerase to transcribe the operon [1]. In the Rfam alignment the consensus structure is of the terminator. The alternative consensus structure is the antiterminator structure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,112,0,0,Cis-reg; leader;,Published; PMID:6186194,112,0,,\"\",0.59617,-6.36098,1600000,266061,258,148,true,-3.6764,0.71825,2013-10-04T00:52:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00513,Trp_leader,1763,Tryptophan operon leader,Bateman A,Bateman A,41.0,41.4,40.8,At least 6 amino acid operons are known to be regulated by attenuation [1]. In each a leader sequence of 150-200 bp is found upstream of the first gene in the operon. This leader sequence can assume two different secondary structures known as the terminator and the anti-terminator structure. In each case the leader also codes for very short peptide sequence that is rich in the end product amino acid of the operon. The terminator structure is recognised as a termination signal for RNA polymerase and the operon is not transcribed. This structure forms when the cell has an excess of the regulatory amino acid and ribosome movement over the leader transcript is not impeded [1]. When there is a deficiency of the charged tRNA of the regulatory amino acid the ribosome translating the leader peptide stalls and the antiterminator structure can form. This allows RNA polymerase to transcribe the operon [1]. In the Rfam alignment the consensus structure is of the terminator structure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,165,0,0,Cis-reg; leader;,Published; PMID:6186194,176,0,,\"\",0.60851,-4.92012,1600000,239255,218,100,true,-2.9411,0.71855,2013-10-04T00:52:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00514,His_leader,1257,Histidine operon leader,Bateman A,Bateman A,49.0,49.1,48.9,At least 6 amino acid operons are known to be regulated by attenuation [1]. In each a leader sequence of 150-200 bp is found upstream of the first gene in the operon. This leader sequence can assume two different secondary structures known as the terminator and the anti-terminator structure. In each case the leader also codes for very short peptide sequence that is rich in the end product amino acid of the operon. The terminator structure is recognised as a termination signal for RNA polymerase and the operon is not transcribed. This structure forms when the cell has an excess of the regulatory amino acid and ribosome movement over the leader transcript is not impeded [1]. When there is a deficiency of the charged tRNA of the regulatory amino acid the ribosome translating the leader peptide stalls and the antiterminator structure can form. This allows RNA polymerase to transcribe the operon [1]. In the Rfam alignment the consensus structure is of the terminator structure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,33,257,0,0,Cis-reg; leader;,Published; PMID:6186194,262,0,,\"\",0.47851,-10.18713,1600000,361806,234,127,true,-2.6825,0.71851,2013-02-01T11:57:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00515,PyrR,1764,PyrR binding site,Bateman A,Bateman A,54.0,54.0,53.9,This family represents the PyrR binding site and terminator structure. This RNA structure is found upstream of a variety of genes involved in pyrimidine biosynthesis and transport. The alignment also includes a portion of the antiterminator structure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.78 -Z 549862.597050 CM SEQDB,41,1160,0,0,Cis-reg;,Published; PMID:11726695,734,0,,\"\",0.40827,-11.16586,1600000,343240,429,126,true,-2.9947,0.71834,2013-10-04T00:52:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00516,ylbH,1765,ylbH leader,\"Barrick JE, Breaker RR\",Barrick JE,50.0,55.0,49.4,This family is a putative regulatory RNA structure that is found upstream of the ylbH gene in B. subtilis and related low GC Gram-positive bacteria.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,233,0,0,Cis-reg; leader;,Predicted; Barrick JE,234,0,,\"\",0.58346,-8.48169,1600000,319010,330,122,true,-3.1572,0.71827,2013-10-04T00:53:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00517,serC,1766,serC leader,Barrick JE,Barrick JE,35.0,35.9,34.4,This family is a putative regulatory RNA structure that is found upstream of the serC-serA operon in some alpha-proteobacteria. The final stem of the structure overlaps the ribosome binding site of the serC reading frame.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,70,557,0,0,Cis-reg; leader;,Predicted; Barrick JE,563,0,,\"\",0.60336,-8.68544,1600000,595667,117,52,true,-4.3748,0.7193,2013-10-04T00:53:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00518,speF,1767,speF leader,\"Barrick JE, Eberhardt R\",Predicted; ClustalW,37.0,37.8,33.6,This family is a putative regulatory RNA structure that is found upstream of the speF gene encoding ornithine decarboxylase in Agrobacterium tumefaciens and other alpha-proteobacteria.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,222,0,0,Cis-reg; leader;,Predicted; RNAalifold,227,0,,\"\",0.65202,-9.20724,1600000,416399,263,153,true,-3.3431,0.71798,2013-10-04T00:53:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00519,suhB,1768,Makes More Granules Regulator RNA (mmgR),Barrick JE,Barrick JE,41.0,41.0,40.7,\"SuhB later renamed mmgR (makes more granules regulator) was a putative non-coding RNA found multiple times in the Agrobacterium tumefaciens genome and related alpha-proteobacteria. [1] Other non-coding RNAs uncovered in the same analysis include: speF, ybhL, metA, and serC. [1]Several studies inSinorhizobium meliloti showed that suhB element is indeed a non-coding RNA:firstly it was detected on Northern blot and called Sm8RNA[2], secondly in an RNAseq study referred to as SmelC689.[3]The mutant phenotype's (strain lacking the small RNA) cytoplasm contains higher content of polyhydroxybutyrate (PBH)storage granules than wild type strain. The sRNA is required to limit the PBH intracellular accumulation when the nitrogen-fixing Sinorhizobium meliloti is under condition of surplus carbon in relation to nitrogen. It has been renamed to mmgR. Further study confirmed that mmgR fine-tunes the regulation of PBH storage.[4]Northern blot confirmed the expression of the sRNA in other rhizobia species. MmgR homologues were found in most alpha-proteobacteria. The Rho-independent terminator and a single stranded region 10-mer (UUUCCUCCCU)are completely conserved. Hence, it was proposed to define a new family of alpha-proteobacterial sRNA: alpha-r8 of which mmgR is a member.[5] RNA binding protein Hfq is binding and stabilising mmgR.[6][7] Expression of the mmgR gene was shown to be controlled by nitrogen (N).[8] Further study[9] has shown that the regulatory proteins NtrC may be required for expression of mmgR gene upon N limitation. Under carbon (C) limitation, mmgR transcription is repressed by AntiA.[9]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.00 -Z 549862.597050 CM SEQDB,87,940,0,0,Gene; sRNA;,Predicted; Barrick JE,501,0,,\"\",0.59829,-6.72824,1600000,268107,140,77,true,-3.3927,0.71873,2013-10-04T00:53:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00520,ybhL,1258,ybhL leader,Barrick JE,Barrick JE,50.0,52.0,35.3,This family is a putative regulatory RNA structure that is found upstream of an uncharacterized membrane protein in alpha-proteobacteria.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,81,0,0,Cis-reg; leader;,Predicted; Barrick JE,82,0,,\"\",0.49026,-7.51231,1600000,239783,172,77,true,-3.206,0.71889,2013-02-01T11:57:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00521,SAM_alpha,1769,SAM riboswitch (alpha-proteobacteria),\"Barrick JE, Corbino KA, Breaker RR\",Barrick JE,36.0,36.0,35.9,\"This family is a riboswitch found predominantly in alpha-proteobacteria that binds S-adenosyl methionine (SAM). Its structure and sequence appear to be unrelated to the SAM riboswitch Rfam:RF00162 found in Gram-positive bacteria. This SAM riboswitch is located upstream of the metA and metC genes in Agrobacterium tumefaciens, and other methionine and SAM biosynthesis genes in other alpha-proteobacteria. Like the other SAM riboswitch, it probably functions to turn off expression of these genes in response to elevated SAM levels.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,40,800,0,0,Cis-reg; riboswitch;,Predicted; Barrick JE,631,0,,\"\",0.61852,-6.39179,1600000,166258,161,79,true,-4.0002,0.71867,2013-10-04T00:55:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00522,PreQ1,1770,PreQ1 riboswitch,\"Barrick JE, Breaker RR, Ontiveros-Palacios N\",Barrick JE,45.5,45.5,45.4,\"This family is a bacterial riboswitch that binds to the metabolite preQ1, an intermediate in the biosynthesis of the modified nucleoside queuosine from GTP. One example of this riboswitch is located in the leader of the Bacillus subtilis ykvJKLM (queCDEF) operon which encodes four genes necessary for queuosine production [1]. PreQ1 binding to the riboswitch aptamer is thought to induce premature transcription termination within the leader to down-regulate expression of these genes. The 3D structure of PreQ1 riboswitch was reported in PDB 2L1V [2]. Methylation activity was reported for PreQ1 riboswitch [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,36,362,0,0,Cis-reg; riboswitch;,Predicted; Barrick JE,349,0,,\"\",0.61009,-6.42193,1600000,265666,200,45,true,-3.9367,0.72389,2013-10-04T00:55:22.000Z,2022-10-29T17:55:17.000Z\n",
            "RF00523,Prion_pknot,1259,Prion pseudoknot,Moxon SJ,PMID:11160898,39.0,39.0,38.9,This family represents a predicted pseudoknot structure found in prion protein mRNA. It has been suggested that the family has a possible effect in prion protein translation [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,148,93,0,0,Cis-reg;,Published; PMID:11160898,106,0,,\"\",0.52113,-10.23017,1600000,763647,103,40,true,-3.9384,0.71963,2013-02-01T11:57:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00524,R2_retro_el,1771,R2 RNA element,Moxon SJ,PMID:15146081,38.0,38.1,37.9,\"This family represents the 3' UTR region of the R2 retrotransposable element. This family is known to interact with the R2-encoded reverse transcriptase in a reaction known as target primed reverse transcription (TPRT) [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,1564,0,0,Cis-reg;,Published; PMID:15146081,53,0,,\"\",0.55937,-9.30927,1600000,258422,373,212,true,-3.5865,0.71123,2013-10-04T00:55:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00525,Flavivirus_DB,1467,Flavivirus DB element,Moxon SJ,PMID:15956576,36.0,36.2,35.6,This family represents the flavivirus 3' UTR elements DB1 and DB2. These DB elements are thought to be required for efficient translation [1].,Flavivirus_DB1,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,111,5,0,0,Cis-reg;,Published; PMID:15956576,14,0,,\"\",0.61955,-5.20821,1600000,219131,148,72,true,-4.0576,0.71894,2013-10-04T00:55:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00526,snoMe28S-U3344,1772,Small nucleolar RNA Me28S-U3344,Moxon SJ,PMID:15987805,50.0,51.6,38.3,Me28S-Um3344 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S U-3344 [1].,snoMe28S-U3344a; sno28S-U3344a,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,37,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",19,0,,\"\",0.56862,-3.23509,1600000,175089,174,83,true,-3.5799,0.71852,2013-10-04T00:56:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00527,snoMe28S-G3255,1773,Small nucleolar RNA Me28S-Gm3255,Moxon SJ,PMID:15987805,44.0,44.6,43.2,Me28S-Gm3255 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S G-3255 [1].,snoMe28S-G3255a,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,19,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",16,0,,\"\",0.45545,-4.49702,1600000,215777,173,84,true,-3.452,0.7186,2013-10-04T00:56:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00528,snoMe28S-Gm1083,1774,Small nucleolar RNA Me28S-Gm1083,Moxon SJ,\"PMID:15987805, INFERNAL\",48.0,67.4,37.8,Me28S-Gm1083 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S G-1083 [1].,snoMe28S-Gm1083a,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,68,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,16,0,,\"\",0.58862,-3.66208,1600000,195565,184,89,true,-3.7337,0.71846,2013-10-04T00:56:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00529,snoMe28S-Am2589,1775,Small nucleolar RNA Me28S-Am2589,Moxon SJ,PMID:15987805,37.0,37.4,36.6,Me28S-Am2589 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S A-2589 [1].,snoMe28S-Am2589a,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,94,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",48,0,,\"\",0.52203,-4.64279,1600000,268986,204,109,true,-3.3967,0.71816,2013-10-04T00:56:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00530,snoMe28S-Cm2645,1776,Small nucleolar RNA Me28S-Cm2645,Moxon SJ,PMID:15987805,48.0,48.6,46.7,Me28S-Cm2645 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S C-2645 [1].,snoMe28S-Cm2645a,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,56,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Moxon SJ, Daub J\",24,0,,\"\",0.626,-3.39711,1600000,210712,160,75,true,-3.7996,0.71866,2013-10-04T00:56:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00531,snoMe28S-Gm3113,1777,Small nucleolar RNA Me28S-Gm3113,Moxon SJ,PMID:15987805,70.0,72.5,38.4,Me28S-Gm3113 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S G-3113 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,15,23,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; ILM; Moxon SJ, Daub J\",15,0,,\"\",0.57011,-3.61655,1600000,208470,171,85,true,-3.9045,0.71848,2013-10-04T00:56:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00532,snoMe18S-Um1356,1778,Small nucleolar RNA Me18S-Um1356,Moxon SJ,PMID:15987805,50.0,53.5,46.9,Me18S-Um1356 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 18S U-1356 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.74 -Z 549862.597050 CM SEQDB,21,41,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; ILM; Moxon SJ,19,0,,\"\",0.50108,-4.27967,1600000,200158,172,86,true,-3.571,0.71846,2013-10-04T00:56:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00533,snoMe18S-Gm1358,1779,Small nucleolar RNA Me18S-Gm1358,Moxon SJ,PMID:15987805,50.0,50.5,44.8,Me18S-Gm1358 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 18S G-1358 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.64 -Z 549862.597050 CM SEQDB,12,24,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,16,0,,\"\",0.58746,-4.13067,1600000,206153,167,79,true,-3.7906,0.71856,2013-10-04T00:56:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00534,SgrS,1780,SgrS RNA,\"Moxon SJ, Daub J\",Published;PMID:17383224,37.0,38.4,36.8,\"In sugar transport-related sRNA (SgrS) is induced under conditions of glucose phosphate stress in E.coli. This small RNA acts to negatively regulate the translation and stability of the ptsG mRNA, which encodes the major glucose transporter in E. coli [2]. SgrS forms a specific ribonucleoprotein complex with RNAse E and the chaperone Hfq and acts by base pairing with the ptsG mRNA. The crucial base-pairs for action of SgrS are confined to the 6 nt region overlapping the Shine-Dalgarno sequence of the ptsG mRNA [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,49,0,0,Gene; antisense;,Predicted; RNAalifold,52,0,,\"\",0.69937,-8.67908,1600000,353128,460,232,true,-3.6061,0.70984,2013-10-04T00:56:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00535,snoMe28S-Am982,1781,Small nucleolar RNA Me28S-Am982,Moxon SJ,PMID:15987805,49.0,49.1,47.9,Me28S-Am982 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S A-982 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,47,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,42,0,,\"\",0.52867,-5.19394,1600000,202298,165,80,true,-3.7126,0.71859,2013-10-04T00:56:37.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00536,snoMe28S-Cm3227,1782,Small nucleolar RNA Me28S-Cm3227,Moxon SJ,PMID:15987805; INFERNAL,43.0,43.3,42.6,Me28S-Cm3227 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S C-3227 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,32,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,18,0,,\"\",0.50327,-4.78965,1600000,221910,175,82,true,-3.5891,0.71855,2013-10-04T00:56:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00537,snoMe28S-Am2634,1783,Small nucleolar RNA Me28S-Am2634,Moxon SJ,PMID:15987805,50.0,58.1,41.6,Me28S-Am2634 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S A-2634 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,30,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; ILM; Daub J,15,0,,\"\",0.53781,-4.36785,1600000,221203,188,97,true,-3.2929,0.71831,2013-10-04T00:56:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00538,snoCD11,1784,Small nucleolar RNA CD11,Moxon SJ,PMID:15987805,60.0,67.6,38.1,\"CD11 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,33,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; ILM; Daub J,17,0,,\"\",0.60908,-2.94118,1600000,183695,158,75,true,-4.065,0.71873,2013-10-04T00:56:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00539,snoMe28S-Cm788,1785,Small nucleolar RNA Me28S-Cm788,Moxon SJ,PMID:15987805,45.0,46.0,39.0,Me28S-Cm788 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family directs 2'-O-methylation of 28S C-788 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32 -Z 549862.597050 CM SEQDB,5,19,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; ILM; Daub J,13,0,,\"\",0.47208,-4.7741,1600000,224285,162,74,true,-3.6649,0.71863,2013-10-04T00:56:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00540,snopsi18S-1854,1786,Small nucleolar RNA psi18S-1854,Moxon SJ,\"Moxon SJ, INFERNAL\",40.0,43.0,36.2,psi18S-1854 is a member of the H/ACA class of snoRNA. This family is responsible for guiding the modification of uridines 1854 and 1937 to pseudouridine in Drosophila 18S rRNA [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,16,45,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; ILM; Moxon SJ,23,0,,\"\",0.46287,-8.85171,1600000,285724,293,156,true,-2.9873,0.71768,2013-10-04T00:56:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00541,snopsi28S-2876,1787,Small nucleolar RNA psi28S-2876,Moxon SJ,Moxon SJ,100.0,127.8,38.0,psi28S-2876 is a member of the H/ACA class of snoRNA. This family is responsible for guiding the modification of uridines 2876 and 2956 to pseudouridine in Drosophila 28S rRNA [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,8,13,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; PFOLD; Moxon SJ,12,0,,\"\",0.5038,-9.44766,1600000,350832,283,143,true,-3.1958,0.71812,2013-10-04T00:57:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00542,snopsi28S-1192,1788,Small nucleolar RNA psi28S-1192,Moxon SJ,\"Moxon SJ, INFERNAL\",44.0,44.5,41.0,psi28S-1192 is a member of the H/ACA class of snoRNA. This family is responsible for guiding the modification of uridine 1192 to pseudouridine in Drosophila 28S rRNA [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,15,65,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; ILM; Moxon SJ,29,0,,\"\",0.52385,-7.24163,1600000,277148,255,138,true,-3.4537,0.71819,2013-10-04T00:57:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00543,snopsi18S-1377,1789,Small nucleolar RNA psi18S-1377,Moxon SJ,\"Moxon SJ, INFERNAL\",48.0,48.1,44.2,psi18S-1377 is a member of the H/ACA class of snoRNA. This family is responsible for guiding the modification of uridines 1377 and 1279 to pseudouridine in Drosophila 18S rRNA [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,78,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,26,0,,\"\",0.52526,-7.71382,1600000,208229,250,140,true,-3.0338,0.71811,2013-10-04T00:57:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00544,snopsi28S-3327,1790,Small nucleolar RNA psi28S-3327,Moxon SJ,\"Moxon SJ, INFERNAL\",70.0,78.8,48.0,psi28S-3327 is a member of the H/ACA class of snoRNA. This family is responsible for guiding the modification of uridine 3327 in Drosophila 28S and U1920 in Drosophila 18S rRNA to pseudouridine [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,39,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,21,0,,\"\",0.52375,-8.02917,1600000,283321,257,143,true,-3.1204,0.71811,2013-10-04T00:57:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00545,snopsi18S-841,2399,Small nucleolar RNA psi18S-841/snoR66,Moxon SJ,Moxon SJ,50.0,54.7,41.9,psi18S-841 is a member of the H/ACA class of snoRNA. This family is responsible for guiding the modification of uridine 841 in Drosophila 18S rRNA to pseudouridine [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,24,59,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,28,0,,\"\",0.52276,-7.66619,1600000,236227,252,145,true,-3.2028,0.71824,2013-10-04T00:57:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00546,snopsi28S-3316,1792,Small nucleolar RNA psi28S-3316,Moxon SJ,Moxon SJ,65.0,67.3,48.5,psi28S-3316 is a member of the H/ACA class of snoRNA. This family is responsible for guiding the modification of uridine 3316 in Drosophila 28S rRNA to pseudouridine [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,33,50,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAFOLD; Moxon SJ,14,0,,\"\",0.51571,-7.57536,1600000,254471,236,133,true,-3.0386,0.71817,2013-10-04T00:57:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00547,IRES_TrkB,1793,TrkB IRES,Moxon SJ,PMID:15908588; INFERNAL,40.0,43.6,38.2,This family represents the TrkB internal ribosome entry site (IRES) which is present in the 5' leader sequence of the mRNA. TrkB is a neurotrophin receptor which is essential for the development and maintenance of the nervous system. The IRES element allows cap-independent translation of TrkB which may be needed for efficient translation in neuronal dendrites [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,42,0,0,Cis-reg; IRES;,Predicted; RNAFOLD,39,0,,\"\",0.6728,-8.86678,1600000,386093,498,352,true,-3.73,0.70414,2013-10-04T00:57:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00548,U11,1794,U11 spliceosomal RNA,\"Griffiths-Jones S, Gardner PP, Marz M\",Griffiths-Jones S; PMID:18390578,39.0,39.1,38.9,\"The U11 small nuclear (snRNA), together with U4atac/U6atac, U5, and U12 snRNAs and associated proteins, forms a spliceosome that cleaves a divergent class of low-abundance pre-mRNA introns.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,72,682,0,0,Gene; snRNA; splicing;,Published; PMID:15210936,343,0,,\"\",0.50524,-9.70921,1600000,325233,1090,137,true,-3.5762,0.71812,2013-10-04T00:57:35.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00549,IRES_c-sis,1795,c-sis internal ribosome entry site (IRES),Moxon SJ,Published; 10409733,31.0,31.8,22.9,\"This family represents the c-sis 5'-UTR which contains three modules that can individually mediate internal ribosome entry. However, the full length sequence is required for maximal IRES activity. It is thought that the three IRES elements are somehow responsive to cellular changes and act to regulate the level of translation [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,35,0,0,Cis-reg; IRES;,Predicted; ILM; Moxon SJ,33,0,,\"\",0.63775,-9.40826,1600000,328524,830,621,true,-4.5591,0.69933,2013-10-04T00:57:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00550,HepE_CRE,1796,Hepatitis E virus cis-reactive element,Moxon SJ,PMID:15890906,45.0,45.7,39.6,\"This family represents a cis-reactive element from Hepatitis E virus which is thought to be essential for \\\"some step in gene expression\\\" [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,46,1,0,0,Cis-reg;,Published; PMID:15890906,3,0,,\"\",0.51468,-9.10578,1600000,329248,265,150,true,-3.3411,0.71802,2013-10-04T00:57:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00551,bicoid_3,1797,Bicoid 3'-UTR regulatory element,Moxon SJ,PMID:15016447,91.0,95.5,48.6,\"This family represents the core structure of the bicoid 3'-UTR and consists of domains II, III, IV and V which are known to be essential for correct transport and localisation of bicoid mRNA during oocyte and embryo differentiation [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,10,0,0,Cis-reg;,Published; PMID:15016447,16,0,,\"\",0.59787,-9.81822,1600000,339252,753,547,true,-4.5166,0.70016,2013-10-04T00:57:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00552,rncO,1798,rncO,Moxon SJ,PMID:8972772,43.0,43.5,42.0,\"This family represents a bacterial regulatory element (rncO) found in the rnc leader sequence. The rnc operon is negatively auto-regulated by transcript stability. rnc, the first gene in the operon codes for RNase III which cleaves the long rncO stem II leading to transcript degradation and a reduction in translation [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,119,0,0,Cis-reg;,Published; PMID:8972772,131,0,,\"\",0.58649,-8.81493,1600000,277225,340,214,true,-3.67,0.71102,2013-10-04T00:57:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00553,SCARNA1,1799,Small Cajal body specific RNA 1,\"Marantidis E, Daub J\",\"Marantidis E, INFERNAL\",45.0,45.9,40.6,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. ACA35 belong to the H/ACA box class of guide RNAs as it has the predicted hairpin-hinge-hairpin-tail structure, conserved H/ACA-box motifs and is found associated with GAR1 [2]. It is predicted to guide pseudouridylation of the U2 spliceosomal snRNA at residue U89 [2].\",snoACA35,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,29,131,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAalifold,110,0,,\"\",0.48872,-8.7621,1600000,335772,281,166,true,-3.0976,0.71621,2013-10-04T00:57:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00554,SNORA48,1800,Small nucleolar RNA SNORA48,Marantidis E,\"Marantidis E, INFERNAL\",50.0,50.0,49.9,The snoRNA ACA 48 has been identified to guide pseudouridylation of the residue U3797 in the 28S rRNA subunit. Its host gene EIF4A1 (eukaryotic translation initiation factor 4A1) is shared with U67 and snoRNA mgU6-77,snoACA48,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,24,1003,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,44,0,,\"\",0.56419,-8.23452,1600000,287244,253,135,true,-3.3812,0.71816,2013-10-04T00:58:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00555,L13_leader,1801,Ribosomal protein L13 leader,Barrick JE,\"Predicted; Barrick JE, CMFinder\",44.0,44.3,43.4,This family is a putative r-protein leader autoregulatory structure [1] found in B. subtilis and other low-GC Gram-positive bacteria. It is located in the 5' untranslated regions of mRNAs encoding the L13-S9 (rplM-rpsI) ribosomal protein operon.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,26,267,0,0,Cis-reg; leader;,\"Predicted; Barrick JE, CMFinder\",279,0,,\"\",0.51535,-5.43747,1600000,392620,351,60,true,-3.6512,0.71919,2013-10-04T00:58:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00556,L19_leader,1802,Ribosomal protein L19 leader,Barrick JE,\"Barrick JE, CMFinder\",36.0,36.3,35.8,This family is a putative r-protein leader autoregulatory structure [1] found in B. subtilis and other low-GC Gram-positive bacteria. It is located in the 5' untranslated regions of mRNAs encoding ribosomal protein L19 (rplS).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.00 -Z 549862.597050 CM SEQDB,10,166,0,0,Cis-reg; leader;,\"Predicted; Barrick JE, CMFinder\",168,0,,\"\",0.65613,-4.50849,1600000,394306,108,42,true,-3.174,0.73071,2013-10-04T00:58:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00557,L10_leader,1803,Ribosomal protein L10 leader,Barrick JE,\"Predicted; CMFinder, Barrick JE, Moy RH\",44.0,44.1,43.9,This family is a putative r-protein leader autoregulatory structure [1] found in B. subtilis and other low-GC Gram-positive bacteria. It is located in the 5' untranslated regions of mRNAs encoding ribosomal proteins L10 and L12 (rplJ-rplL). A Rho-independent transcription terminator structure that is probably involved in regulation is included at the 3' end.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,97,1615,0,0,Cis-reg; leader;,\"Predicted; CMFinder, Barrick JE, Moy RH\",1652,0,,\"\",0.44622,-11.49083,1600000,341928,440,139,true,-3.3734,0.71804,2013-10-04T00:58:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00558,L20_leader,1804,Ribosomal protein L20 leader,Barrick JE,\"Predicted; CMFinder, Barrick JE, Moy RH\",58.7,58.7,58.6,This family is a putative r-protein leader autoregulatory structure [1] found in B. subtilis and other low-GC Gram-positive bacteria. It is located in the 5' untranslated regions of mRNAs encoding initiation factor 3 followed by ribosomal proteins L35 and L20 (infC-rpmI-rplT). A Rho-independent transcription terminator structure that is probably involved in regulation is included at the 3' end.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 49.00 -Z 742849.287494 CM SEQDB,43,484,0,0,Cis-reg; leader;,\"Predicted; CMFinder, Barrick JE, Moy RH\",526,0,,\"\",0.56744,-6.80688,1600000,308304,221,127,true,-2.9525,0.71828,2013-10-04T00:58:38.000Z,2022-10-29T17:55:17.000Z\n",
            "RF00559,L21_leader,1805,Ribosomal protein L21 leader,\"Barrick JE, Moy RH\",\"Predicted; CMFinder, Barrick JE, Moy RH\",36.0,36.1,35.7,\"This family is a putative r-protein leader autoregulatory structure [1] found in B. subtilis and other low-GC Gram-positive bacteria. It is located in the 5' untranslated regions of mRNAs encoding ribosomal protein L21, a protein of unknown function, and ribosomal protein L27 (rplU-ysxB-rpmA).\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,38,635,0,0,Cis-reg; leader;,\"Predicted; CMFinder, Barrick JE, Moy RH\",648,0,,\"\",0.58853,-3.89232,1600000,178102,371,78,true,-3.2765,0.71882,2013-10-04T00:58:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00560,SNORA17,1806,Small nucleolar RNA SNORA17,Marantidis E,\"Marantidis E, INFERNAL\",29.0,29.1,28.9,\"snoRNA ACA 17 belongs to the H/ACA family of snoRNAs and is predicted to guide pseudouridylation of the 28S rRNA at positions U4659 and U4598. It shares the same host gene together with ACA43. NB This family has a high threshold value to exclude a large number of closely related sequences that do not appear to have conserved H and ACA boxes, and may be pseudogenes.\",snoACA17,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,53,1690,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,139,0,,\"\",0.54019,-8.27273,1600000,261711,245,132,true,-3.4068,0.7183,2013-10-04T00:58:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00561,SNORA40,1807,Small nucleolar RNA SNORA40,Marantidis E,\"Marantidis E, INFERNAL\",51.0,51.0,50.8,\"The snoRNA ACA 40 belongs to the H/ACA family of snoRNAs and guides the pseudouridylation of 28S rRNA subunit at position U4565. NB This family has a high threshold value to exclude a large number of closely related sequences that do not appear to have conserved H and ACA boxes, and may be pseudogenes.\",snoACA40,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,823,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,104,0,,\"\",0.52694,-7.24203,1600000,293213,233,128,true,-3.0454,0.71827,2013-10-04T00:59:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00562,SNORA49,1808,Small nucleolar RNA SNORA49,Marantidis E,\"Marantidis E, INFERNAL\",35.0,35.9,33.8,The snoRNA ACA49 has no identified RNA target. NB genomic sequence 1-nt different from published.,snoACA49,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,21,132,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,101,0,,\"\",0.55245,-8.25018,1600000,264764,241,133,true,-3.2973,0.71827,2013-10-04T00:59:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00563,SNORA53,1809,Small nucleolar RNA SNORA53,Marantidis E,\"Marantidis E, INFERNAL\",38.0,38.4,37.9,The snoRNA ACA53 belongs to the H/ACA family of pseudouridylation guide snoRNAs. RNA residues targeted for pseudouridylation by this molecule have not been identified. n.b: The GenBank accession number for 'Homo sapiens snoRNA ACA53 gene' (AJ609478) corresponds to a portion of snoRNA ACA24.,snoACA53,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,189,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,159,0,,\"\",0.62809,-8.48589,1600000,329811,375,248,true,-3.7468,0.70869,2013-10-04T00:59:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00564,SCARNA11,1810,Small Cajal body specific RNA 11,\"Marantidis E, Daub J\",\"Marantidis E, INFERNAL\",37.0,37.2,35.0,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. ACA57 belongs to the H/ACA box class of guide RNAs as it has the predicted hairpin-hinge-hairpin-tail structure, conserved H/ACA-box motifs and is found associated with GAR1 [1]. ACA57 is predicted to guide the pseudouridylation of the U5 spliceosomal RNA at position U43 [2].\",snoACA57,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19 -Z 549862.597050 CM SEQDB,24,184,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAalifold,103,0,,\"\",0.55768,-7.38885,1600000,210505,247,140,true,-3.3345,0.71826,2013-10-04T00:59:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00565,SCARNA3,1811,Small Cajal body specific RNA 3,\"Marantidis E, Daub J\",\"Marantidis E, INFERNAL\",47.0,48.1,45.3,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. This family includes the mouse H/ACA-box snoRNAs MBI-100 and MBI-114 identified in [2]. Both MBI-100 and MBI-114 are predicted to guide the pseudouridylation of position U40 in the U6 snRNA [2].\",SCARNA25; snoHBI-100,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,23,136,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAalifold,100,0,,\"\",0.5625,-6.72005,1600000,202698,247,141,true,-3.5351,0.71819,2013-10-04T00:59:37.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00566,SNORA35,1812,Small nucleolar RNA SNORA35,Marantidis E,\"Marantidis E, INFERNAL\",39.0,39.2,38.8,\"The snoRNA HBI-36 snoRNA belongs to the H/ACA family of snoRNAs and is specifically expressed in the brain, specifically in the choroids plexus. No RNA target has been identified.\",snoHBI-36,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19 -Z 549862.597050 CM SEQDB,29,162,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,115,0,,\"\",0.545,-8.11877,1600000,317598,224,129,true,-3.2898,0.71828,2013-10-04T00:59:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00567,SNORD17,1813,Small nucleolar RNA SNORD17,Marantidis E,\"Marantidis E, INFERNAL\",45.0,51.2,40.6,\"The snoRNA HBI-43 belongs to the C/D family of snoRNAs, guiding 2'O-ribose methylation at position U3797 of the 28S rRNA. Pseudouridylation in position U3797 is predicted to be guided by the H/ACA snoRNA ACA48.\",snoHBI-43,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,152,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold,96,0,,\"\",0.6141,-10.1407,1600000,508432,368,238,true,-3.5311,0.70938,2013-10-04T00:59:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00568,SNORA26,1814,Small nucleolar RNA SNORA26,Marantidis E,\"Marantidis E, INFERNAL\",40.0,40.6,39.9,\"The snoRNA HBI-6 belong to the H/ACA family of snoRNAs, guiding the pseudouridylation of position U4522 if the 28S rRNA. HBI-6 is the human orthologue of mouse MBI-6 snoRNA.\",snoHBI-6,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,76,408,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,118,0,,\"\",0.52757,-6.77375,1600000,246558,247,122,true,-3.0696,0.71829,2013-10-04T01:00:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00569,SNORD19,1815,Small nucleolar RNA SNORD19,Marantidis E,\"Marantidis E, INFERNAL\",35.0,35.0,34.8,Homo sapiens C/D box snoRNA HBII-108: This snoRNA is the human orthologue of the mouse MBII-108 and is predicted to guide 2'O-ribose methylation of the small 18S rRNA on position G683. snoRNA HBII-108 is encoded in a nucleostemin (NS) gene intron like the HBII-210,snoHBII-108,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,22,227,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,109,0,,\"\",0.57658,-3.74824,1600000,180921,160,76,true,-3.5266,0.71865,2013-10-04T01:00:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00570,SNORD64,1816,Small Nucleolar RNA SNORD64,Marantidis E,\"Marantidis E, INFERNAL\",40.0,41.3,39.2,\"The human snoRNA HBII-13 belongs to the C/D family of snoRNAs. It is expressed mainly in the brain, but is also in the lungs, the kidneys and muscle. HBII-13 snoRNA has no identified RNA target and is encoded together with HBII-436, HBII-437, HBII-438A/B and the clusters of HBII-85, HBII-52 snoRNA in a 460 kb intron of the large, paternally expressed, transcription unit (SNURF-SNRNP-UBE3A AS). This is antisense to maternally expressed UBE3A gene (Runt et al., 2001).\",snoHBII-13,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,40,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold,35,0,,\"\",0.62264,-3.27653,1600000,285486,145,67,true,-3.4504,0.719,2013-10-04T01:00:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00571,SNORD65,1817,Small nucleolar RNA SNORD65,Marantidis E,\"Marantidis E, INFERNAL\",41.0,41.7,40.6,snoRNA HBII-135 belongs to the CD family of snoRNAs. It is the human orthologue of the mouse MBII-135 snoRNA and is predicted to guide 2'O-ribose methylation of the small 18S rRNA at residue U627.,snoHBII-135,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,200,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold,113,0,,\"\",0.58178,-8.21319,1600000,378071,153,73,false,-3.826,0.71869,2013-10-04T01:00:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00572,SNORD66,1818,Small nucleolar RNA SNORD66,Marantidis E,\"Marantidis E, INFERNAL\",32.0,32.0,31.9,snoRNA HBII-142 is part of the C/D family of snoRNAs. It is the human orthologue of the mouse MBII-142 snoRNA and is predicted to guide 2'O-ribose methylation of small 18S rRNA at residue C1272.,snoHBII-142,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,219,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold,109,0,,\"\",0.71977,-2.71802,1600000,250203,162,76,true,-3.8843,0.71866,2013-10-04T01:00:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00573,SNORD67,1819,Small nucleolar RNA SNORD67,Marantidis E,\"Marantidis E, INFERNAL\",35.0,35.0,34.6,snoRNA HBII-166 belongs to the C/D family of snoRNAs. It is the human orthologue of the mouse MBII-166 and is predicted to guide 2'O-ribose methylation of snRNA U6 at residue C60.,snoHBII-166,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,139,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,109,0,,\"\",0.64727,-4.55162,1600000,241227,211,111,true,-2.8685,0.71829,2013-10-04T01:00:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00574,SNORD69,1820,Small nucleolar RNA SNORD69,Marantidis E,\"marantidis E, INFERNAL\",37.0,38.3,35.8,snoRNA HBII-210 belongs to the C/D family of snoRNAs. It is the human orthologue of the mouse MBII-210 and is predicted to guide the 2'O-ribose methylation of large 28S rRNA on residue G4464.,snoHBII-210,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,11,93,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,90,0,,\"\",0.57881,-3.91464,1600000,225222,167,77,true,-3.6984,0.71872,2013-10-04T01:00:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00575,SNORD70,1821,Small nucleolar RNA SNORD70,Marantidis E,\"Marantidis E, INFERNAL\",50.0,50.1,48.9,\"snoRNA HBII-234 belongs to the C/D family of snoRNAs. It is the human orthologue of the mouse MBII-234 and is predicted to guide 2'O-ribose methylation of the small 18S rRNA on position A512. It is hosted, together with HBII-95, by the core C/D box snoRNA protein encoding gene NOP5/NOP58.\",snoHBII-234,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,21,202,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,112,0,,\"\",0.50727,-4.46705,1600000,204694,174,89,true,-3.382,0.71859,2013-10-04T01:00:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00576,SNORD71,1822,Small nucleolar RNA SNORD71,Marantidis E,\"Marantidis E, INFERNAL\",34.0,34.0,33.8,snoRNA HBII-239 belongs to the family of C/D snoRNAs.It is the human orthologue of the mouse MBII-239 described and is predicted to guide 2'O-ribose methylation of 5.8S rRNA on residue U14.,snoHBII-239,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,104,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,100,0,,\"\",0.64742,-3.58318,1600000,228533,177,86,true,-3.1808,0.71879,2013-10-04T01:00:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00577,SNORD72,1823,Small nucleolar RNA SNORD72,Marantidis E,\"Marantidis E, INFERNAL\",43.0,44.7,39.9,snoRNA HBII-240 belongs to the CD family of snoRNAs. It is the human orthologue of the mouse MBII-240 and is predicted to guide 2'O-ribose methylation of the large 28S rRNA at residue U4590.,snoHBII-240,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,113,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,87,0,,\"\",0.50827,-3.87659,1600000,199949,168,80,true,-3.5775,0.7186,2013-10-04T01:00:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00578,SNORD89,1824,Small nucleolar RNA SNORD89,Marantidis E,marantidis E.,41.0,42.0,39.7,snoRNA HBII-289 belongs to the family of C/D snoRNAs. It is the human orthologue of the mouse MBII-289 and has no identified RNA target.,snoHBII-289,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,113,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold,105,0,,\"\",0.58829,-5.1789,1600000,236614,218,114,true,-3.0484,0.71846,2013-10-04T01:00:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00579,SNORD90,1825,Small nucleolar RNA SNORD90,Marantidis E,\"Marantidis E, INFERNAL\",41.0,41.9,39.0,snoRNA HBII-295 belongs to the family of C/D snoRNAs. It is the human orthologue of the mouse MBII-295 and has no identified RNA target.,snoHBII-295,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,97,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold,100,0,,\"\",0.54989,-3.5008,1600000,202798,213,111,true,-3.1977,0.71836,2013-10-04T01:00:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00580,SNORD91,1264,Small nucleolar RNA SNORD91 family,\"Marantidis E, Daub J\",\"Marantidis E, INFERNAL\",51.0,53.3,50.9,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. This family includes HBII-296A (SNORD91A) and HBII-296B (SNORD91B). They are close paralogues and share the same host gene (FLJ10534). Both snoRNAs and are predicted to guide 2'O-ribose methylation of the large 28S rRNA at position G4588 [3].,snoHBII-296A,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,24,168,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,93,0,,\"\",0.56842,-3.53605,1600000,186879,186,95,true,-3.6465,0.71843,2013-10-04T01:00:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00581,SNORD12,2400,Small nucleolar SNORD12/SNORD106,\"Marantidis E, Daub J\",Marantidis E.,43.0,43.2,42.7,\"This family belong to the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. This family includes the human ortholog of mouse snoRNA HBII-99 and human snoRNA U106. In humans both HB11-99 and U106 snoRNAs share the same host gene [1]. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs. HBII-99 is predicted to guide the 2'O-ribose methylation of 28S rRNA G3878 [2]. U106 contains conserved antisense elements which would predict U106 methylates residues G1536 and U1602 of 18S rRNA. However, these targets do not appear to be methylated and U106 might function as an RNA chaperone during rRNA folding [1]. A similar role has been suggested for the H/ACA snoRNAs U17a and U17b Rfam:RF00045 [1].\",SNORD106; snoHBII-99,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,360,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; RNAalifold, Daub J\",111,0,,\"\",0.63381,-3.39784,1600000,233709,178,90,true,-3.8383,0.71847,2013-10-04T01:00:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00582,SCARNA14,1828,Small Cajal body specific RNA 14,\"Marantidis E, Daub J\",\"Marantidis E, INFERNAL\",40.0,41.0,39.2,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. U100 belongs to the H/ACA box class of guide RNAs as it has the predicted hairpin-hinge-hairpin-tail structure and the conserved H/ACA-box motifs [2]. U100 is the human orthologue of mouse H/ACA snoRNA MBII-201 identified in [3] and also included in this family. U100 is predicted to guide the pseudouridylation of U2 snRNA at residue U7 [4].\",snoU100,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,22,146,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAalifold,116,0,,\"\",0.51018,-6.96175,1600000,240504,243,136,true,-3.3278,0.7182,2013-10-04T01:00:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00584,SNORD105,1829,Small nucleolar RNA SNORD105,Marantidis E,\"Marantidis E, INFERNAL\",45.0,52.5,41.6,U105 belongs to the C/D family of snoRNAs. It is encoded in an intron of the Suppressor of SWI4 1 homolog (Ssf-1) (Peter Pan homolog) gene. and is predicted to guide 2'O-ribose methylation of residue U799 of the small 18S rRNA subunit.,snoU105,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,38,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,41,0,,\"\",0.66115,-3.01796,1600000,184158,177,87,true,-3.855,0.71853,2013-10-04T01:01:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00586,SNORA12,1830,Small nucleolar RNA SNORA12,Marantidis E,\"Marantidis E, INFERNAL\",45.0,45.0,41.6,U108 belongs to the H/ACA family of snoRNAs. The sequence is predicted to guide the pseudouridylation of the U372 residue in the 28S rRNA subunit. However it has not been reported as a pseudouridylation site.,snoU108,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,140,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,92,0,,\"\",0.57713,-8.11743,1600000,358987,271,148,true,-3.2456,0.71814,2013-10-04T01:01:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00588,SNORD41,1831,Small nucleolar RNA SNORD41,Marantidis E,Marantidis E.,41.0,41.7,40.7,snoRNA U41 belongs to the C/D family of snoRNAs. It is predicted to guide 2'O-ribose methylation of the large 28S rRNA on residue U4276.,snoU41,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,20,111,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold,78,0,,\"\",0.60278,-4.58471,1600000,285638,151,70,true,-3.509,0.71877,2013-10-04T01:01:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00591,SNORD77,1260,Small nucleolar RNA SNORD77,Marantidis E,\"Marantidis E, INFERNAL\",35.0,35.0,34.8,\"U77 belongs to the C/D family of snoRNAs. It is predicted to guide 2'O-ribose methylation of large 28S rRNA subunit at position A1521. The C/D snoRNAs U44, U47, U74, U75, U76, U78, U79, U80 and U81 share the same host gene as U77 (non-coding).\",snoU77,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,21,105,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold,41,0,,\"\",0.58101,-4.1074,1600000,318181,149,68,true,-3.7475,0.71901,2013-02-01T11:57:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00592,SNORD78,1261,Small nucleolar RNA SNORD78,Marantidis E,\"Marantidis E, INFERNAL\",38.0,38.2,37.1,\"U78 belongs to the CD family of snoRNAs. It is predicted to guide 2'O-ribose methylation of the large 28S rRNA subunit at position G4593. The snoRNAs U44, U47, U74, U75, U76, U77, U79, U80 and U81, also of the C/D family, share the same host gene with U78 (protein non-coding).\",snoU78,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,19,97,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,68,0,,\"\",0.48553,-10.04407,1600000,431275,142,65,false,-3.6358,0.71882,2013-02-01T11:57:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00593,snoU83B,1832,Small nucleolar RNA U83B,Marantidis E,\"Marantidis E, INFERNAL\",35.0,35.3,34.7,U83B belongs to the C/D family of snoRNAs. U83B like U83A has no documented RNA target and they share the same host gene with the C/D box snoRNA U43 (RPL3). N.B. U83A/B/C have no sequence similarity with the U83 snoRNA that was cloned by Jady and Kiss (2000).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,177,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold_manual; Daub J,114,0,,\"\",0.64575,-4.20468,1600000,268657,187,94,true,-3.4055,0.71833,2013-10-04T01:02:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00594,SNORD86,1833,Small nucleolar RNA SNORD86,Marantidis E,Marantidis E.,35.0,36.3,34.8,\"U86 belongs to the C/D family of snoRNAs. It is the human orthologue of Xenopus laevis U86 and has no identified RNA target. The snoRNAs U86, U56, U57 and HBII-55,of the C/D family, and the H/ACA box snoRNA ACA51 share the same host gene (NOL5A) with U86.\",snoU86,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,63,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold,38,0,,\"\",0.68614,-1.95265,1600000,208758,177,86,true,-3.6927,0.71875,2013-10-04T01:02:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00598,SNORA76,1834,Small nucleolar RNA SNORA76,Daub J,PMID:16373490. INFERNAL,37.0,37.0,36.7,ACA62 (SNORA76) is a member of the H/ACA class of snoRNAs that guide the sites of modification of uridines to pseudouridines [1]. This snoRNA was identified by computational screening and its expression in mouse experimentally verified by Northern blot and primer extension analysis [2]. ACA62 is proposed to guide the pseudouridylation of 18S rRNA U34 and U105 [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,22,221,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,76,0,,\"\",0.49406,-9.30276,1600000,280316,241,132,true,-2.9071,0.71835,2013-10-04T01:02:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00599,SNORA77,1835,Small nucleolar RNA SNORA77,Daub J,PMID:16373490; INFERNAL,42.0,42.0,41.4,ACA63 (SNORA77) is a member of the H/ACA class of snoRNAs that guide the sites of modification of uridines to pseudouridines [1]. This snoRNA was identified by computational screening and its expression in mouse experimentally verified by Northern blot and primer extension analysis [2]. ACA63 is proposed to guide the pseudouridylation of 18S rRNA U814 [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,186,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,111,0,,\"\",0.56858,-7.53745,1600000,211415,227,125,true,-3.2717,0.7184,2013-10-04T01:02:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00600,SNORA79,1836,Small nucleolar RNA SNORA79,Daub J,PMID:16373490; INFERNAL,55.0,56.1,54.3,ACA65 (SNORA79) is a member of the H/ACA class of snoRNAs that guide the sites of modification of uridines to pseudouridines [1]. This snoRNA was identified by computational screening and its expression in mouse experimentally verified by Northern blot and primer extension analysis [2]. ACA65 is proposed to guide the pseudouridylation of residue U31 in U6 snRNA (Rfam:RF00026)[2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,190,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,88,0,,\"\",0.50368,-6.95666,1600000,199741,252,143,true,-3.2893,0.71815,2013-10-04T01:02:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00601,SCARNA20,1837,Small Cajal body specific RNA 20,Daub J,PMID:16373490,44.0,44.0,42.9,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. ACA66 (SCARNA20) is a member of the H/ACA class of snoRNAs that guide the sites of modification of uridines to pseudouridines. This snoRNA was identified by computational screening and its expression in mouse experimentally verified by Northern blot and primer extension analysis [2]. It is predicted to guide the pseudouridylation of residue U28 in U12 snRNA (Rfam:RF00007) [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,5,191,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAalifold,49,0,,\"\",0.5477,-8.13942,1600000,236637,239,130,true,-3.206,0.71832,2013-10-04T01:02:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00602,SCARNA21,1838,Small Cajal body specific RNA 21,Daub J,PMID:16373490,60.0,65.9,56.6,\"scaRNAs are a specific class of small nuclear RNAs which localise to the Cajal bodies and guide the modification of RNA polymerase II transcribed spliceosomal RNAs U1, U2, U4, U5 and U12 [1]. ACA68 (SCARNA21) is a member of the H/ACA class of snoRNAs that guide the sites of modification of uridines to pseudouridines. This snoRNA was identified by computational screening and its expression in mouse experimentally verified by Northern blot and primer extension analysis [2]. ACA68 is proposed to guide the pseudouridylation of residue U19 in U12 snRNA (Rfam:RF00007)[2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,24,94,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAalifold; Daub J,43,0,,\"\",0.59511,-7.67772,1600000,242514,307,140,true,-2.871,0.71822,2013-10-04T01:02:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00603,SNORD23,1839,Small nucleolar RNA SNORD23,Daub J,INFERNAL,64.0,65.6,63.5,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. This family includes mouse MBII-115 identified in [2] and the human orthologue HBII-115 (SNORD23). There is currently no predicted target RNA for HBII-115 [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,44,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,45,0,,\"\",0.66123,-4.33403,1600000,285604,211,110,true,-3.1519,0.71816,2013-10-04T01:02:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00604,SNORD88,1840,Small nucleolar RNA SNORD88,Daub J,INFERNAL,46.0,46.8,45.0,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. This family includes mouse MBII-211 and MBII-180 identified in [2] and the human orthologue HBII-180 (SNORD88). HBII-180 is predicted to guide the 2'O-ribose methylation of 28S rRNA C3680 [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,35,222,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Daub J,67,0,,\"\",0.65423,-3.96943,1600000,258883,177,91,true,-3.5066,0.71844,2013-10-04T01:02:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00606,SNORD93,1841,Small nucleolar RNA SNORD93,Daub J,INFERNAL,60.0,62.7,54.6,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. This family includes mouse MBII-336 identified in [2]. The human orthologue HBII-336 (SNORD93) is predicted to guide the 2'O-ribose methylation of 18S rRNA A576 [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.52 -Z 549862.597050 CM SEQDB,17,102,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Daub J,90,0,,\"\",0.56106,-5.31647,1600000,291111,217,74,true,-3.4177,0.71874,2013-10-04T01:02:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00607,SNORD98,1842,Small nucleolar RNA SNORD98,Daub J,INFERNAL,50.0,54.1,43.0,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. HBII-419 (SNORD98) is predicted to guide the 2'0-ribose methylation of 18S rRNA G867 [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,90,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,92,0,,\"\",0.6127,-4.51942,1600000,319087,145,65,true,-3.6452,0.71883,2013-10-04T01:02:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00608,SNORD99,1288,Small nucleolar RNA SNORD99,Daub J,INFERNAL,60.0,60.1,59.9,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. HBII-420 (SNORD99) is predicted to guide the 2'O-ribose methylation of 28S rRNA A2774 [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,111,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Daub J,106,0,,\"\",0.63356,-3.44311,1600000,238513,156,74,true,-3.6639,0.71876,2013-02-01T12:06:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00609,SNORD100,1843,Small nucleolar RNA SNORD100,Daub J,INFERNAL,58.0,60.5,57.4,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. HBII-429 (SNORD100) is predicted to guide the 2'O-ribose methylation of 18S rRNA G436 [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.09 -Z 549862.597050 CM SEQDB,26,150,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Daub J,106,0,,\"\",0.6408,-3.28241,1600000,209927,167,82,true,-3.6502,0.71857,2013-10-04T01:02:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00610,SNORD110,1844,Small nucleolar RNA SNORD110,Daub J,INFERNAL,64.0,64.2,51.8,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. HBII-55 (SNORD110) is predicted to guide the 2'O-ribose methylation of 18S rRNA U1288 [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,35,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Daub J,41,0,,\"\",0.64912,-3.55136,1600000,294448,156,71,true,-3.609,0.7188,2013-10-04T01:02:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00611,SNORD111,1845,Small nucleolar RNA SNORD111,Daub J,INFERNAL,46.0,46.0,45.2,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. HBII-82 (SNORD111) is predicted to guide the 2'O-ribose methylation of 28S rRNA G3923 [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,5,144,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Daub J,88,0,,\"\",0.55831,-4.23118,1600000,217859,183,89,true,-3.2863,0.71846,2013-10-04T01:02:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00612,SNORD75,1846,Small nucleolar RNA SNORD75,Daub J,INFERNAL,50.0,51.1,48.0,\"U75 (SNORD75) belongs to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. This family includes the mouse orthologue Z19. U75 is predicted to guide the 2'O-ribose methylation of 28S rRNA residue C4032 [3]. In humans U75 shares the same non-protein coding host gene (gas5) with 9 other snoRNAs (U44, U47, U74, U76, U77, U78, U79, U80 and U81)[2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,5,126,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Daub J,108,0,,\"\",0.82146,-1.71394,1600000,250897,200,62,true,-3.924,0.71893,2013-10-04T01:03:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00613,SNORD94,1847,Small nucleolar RNA SNORD94,Daub J,INFERNAL,60.0,60.0,47.8,The members of this family belong to the C/D class of snoRNAs which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNAs [1]. U94 (SNORD94) is predicted to guide the 2'O-ribose methylation of C62 of the snRNA U6 (Rfam:RF00026)[2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,128,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Daub J,109,0,,\"\",0.63171,-7.73471,1600000,357600,247,137,true,-3.2894,0.71785,2013-10-04T01:04:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00614,SNORA11,1289,Small nucleolar RNA SNORA11,Daub J,INFERNAL,36.0,36.2,35.8,U107 (SNORA11) is a member of the H/ACA class of snoRNAs that guide the sites of modification of uridines to pseudouridines [1]. This snoRNA was identified by RT-PCR from blood cells and its expression confirmed by Northern blot analysis [2]. There is no predicted target for this guide snRNA [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,35,612,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,54,0,,\"\",0.58748,-7.591,1600000,346034,235,129,true,-3.3306,0.71834,2013-02-01T12:06:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00615,LhrA,1848,Listeria Hfq binding LhrA,Daub J,Daub J,100.0,150.8,35.2,This RNA was identifed by screening for RNA molecules which co-immunoprecipitated with the RNA chaperone Hfq [1]. It appears to be transcribed from a region overlapping with predicted protein of unknown function (Lmo2257) and is located between a putative intracellular protease/amidase and a putative protein of the ribulose-phosphate 3 epimerase family. Its is highly expressed in stationary growth phase but its function is unkown. It is proposed to be a regulatory RNA which controls gene expression at the post transcriptional level by binding its target mRNA in an Hfq dependent fashion [1]. This RNA molecule appears to be conserved amongst Listeria species but has not been identified in other bacterial species.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,7,0,0,Gene; sRNA;,Published; PMID:16682563,17,0,,\"\",0.66846,-8.36707,1600000,400033,418,269,true,-3.7553,0.70743,2013-10-04T01:04:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00616,LhrC,1849,Listeria LhrC,Daub J,Daub J,110.0,115.3,107.2,\"This RNA was identifed by screening for RNA molecules which co-immunoprecipitated with the RNA chaperone Hfq [1]. However, neither the stability nor the activity of LhrC is depending on the presence of Hfq[2][3]. LhrC is transcribed from the intergenic region between the protein coding genes cysK, a putative cysteine synthase and sul, a putative dihydropteroate synthase [1]. In Listeria monocytogenes four additional copies of lhrC have been identifed in the genome, three of which are located in tandem repeat upstream of the originally characterised lhrC [1]. This RNA molecule appears to be conserved amongst Listeria species but has not been identified in other bacterial species. The direct mRNA targets of LhrC are the virulence adhesion LapB, and the oligo-peptide binding protein OppA. The 3 conserved UCCC motifs common to all copies of LhrC are involved in the mRNA binding and post-transcriptional repression of the target genes. Interestingly, 2 other Listerina monocytogenes sRNAs Rli22 and Rli33 contain two UCCC motifs and use them to repress oppA mRNA expression.[4]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,16,0,0,Gene; sRNA;,Published; PMID:16682563,12,0,,\"\",0.4778,-8.02974,1600000,382984,214,111,true,-3.1069,0.71833,2013-10-04T01:04:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00617,cHP,1262,Flavivirus capsid hairpin cHP,Daub J,PMID:16474125,40.0,40.2,39.9,\"cHP is a conserved RNA hairpin structure identified within the capsid coding region of several Flavivirus genomes. These positive strand RNA genomes are translated as a single polypeptide and subsequently cleaved into constituent proteins, the first of which is the capsid protein. The cHP hairpin is located within the capsid coding region between two AUG start codons. The cHP cis element has been shown to direct translation start from the thermodynamically unstable first start codon [1]. The ability of cHP to direct initiation from the first start codon is proportional to its thermodynamic stability, is position dependent, and is sequence independent [1]. It has been demonstrated that both AUGs and the conserved cHP are necessary for efficient viral replication in human and mosquito cells [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,59,92,0,0,Cis-reg;,Published; PMID:16474125,34,0,,\"\",0.5864,-8.40679,1600000,743570,104,45,true,-3.6407,0.71938,2013-02-01T11:57:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00618,U4atac,1850,U4atac minor spliceosomal RNA,\"Daub J, Gardner PP, Marz M\",PMID:19030770; Gardner PP,53.0,53.2,52.9,\"U4atac is an essential component of the minor U12-type spliceosome complex which is required for removal of the rarer class of eukaryotic introns (AT-AC, U12-type) [1]. U4atac snRNA is proposed to form a base-paired complex with another spliceosomal RNA U6atac via two stem loop regions. These interacting stem loops have been shown to be required for in vivo splicing [2]. U4atac also contains an 3' Sm protein binding site which has been shown to be essential for splicing activity [2]. U4atac is the functional analog of U4 (Rfam:RF00015) in the major U2-type spliceosomal complex [2]. The drosophila U4atac snRNA has an additional predicted 3' stem loop terminal to the Sm binding site [3] which is not included in this model.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,61,388,0,0,Gene; snRNA; splicing;,Published; PMID:11911359,190,0,,\"\",0.48629,-10.68633,1600000,408947,324,133,true,-3.2445,0.71823,2013-10-04T01:04:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00619,U6atac,1851,U6atac minor spliceosomal RNA,\"Daub J, Gardner PP\",\"PMID:10199569; Daub J, INFERNAL; PMID:18390578\",33.0,33.0,32.9,\"U6atac is an essential component of the minor U12-type spliceosome complex which is required for removal of the rarer class of eukaryotic introns (AT-AC, U12-type) [1]. U6atac snRNA is proposed to form a base-paired complex with another spliceosomal RNA U4atac via two stem loop regions. These interacting stem loops have been shown to be required for in vivo splicing [2]. U6atac is the functional analog of U6 (Rfam:RF00026) in the major U2-type spliceosomal complex [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,62,2521,0,0,Gene; snRNA; splicing;,Published; PMID:11911359,375,0,,\"\",0.65236,-8.2378,1600000,287519,218,126,true,-3.4737,0.71803,2013-10-04T01:04:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00620,HCV_ARF_SL,1852,Hepatitis C alternative reading frame stem-loop,\"Lamkiewicz K, Triebel S, Marz M\",\"Kevin Lamkiewicz, Sandra Triebel, Manja Marz\",42.0,42.5,40.5,Genetic modification of RNA structures in Hepatitis C virus (HCV) genome affects replication and infectivity [1] Long distant RNA-RNA interactions in HCV RNAs are involved in translation to virion budding [2] It has been proposed that the 5'-3' UTR base-pairing in the HCV genome might play an important role in viral RNA replication [3],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,57,2,0,0,Cis-reg;,\"Predicted; BM VeGETA v. 0.4, LocARNA 2.0.0RC8, RNAfold\",21,0,,\"\",0.51136,-9.25908,1600000,308469,249,136,true,-2.9724,0.71825,2013-10-04T01:05:07.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00621,CoTC_ribozyme,1853,Beta-globin co-transcriptional cleavage ribozyme,Daub J,Daub J,58.0,59.9,57.9,\"Transcription termination of RNA polymerase II transcripts is proposed to occur by a two stage process [1]. The first stage involves pre-termination cleavage (PTC) of the nascent transcript downstream of the poly(A) site. This process is also referred to as co-transcriptional cleavage (CoTC). The CoTC process in the human beta-globin gene has been shown to involve an RNA self-cleaving activity located in the 3' flanking region of the beta-globin gene. This Rfam model represents the catalytic core of the beta-globin CoTC ribozyme. The CoTC core is highly conserved in the 3' flanking regions of other primate beta-globin genes [2]. Functionally, this CoTC ribozyme resembles the 3' processive, self-cleaving ribozymes described for the protein-encoding genes from the myxomycetes Didymium iridis and Physarum polycephalum [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,20,0,0,Gene; ribozyme;,Published; PMID:15565159,22,0,,\"\",0.53359,-9.27125,1600000,347041,319,191,true,-3.2219,0.71321,2013-10-04T01:05:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00622,CPEB3_ribozyme,1854,Mammalian CPEB3 ribozyme,Daub J,Published; PMID:16990549; INFERNAL,60.0,60.3,59.8,\"The mammalian CPEB3 ribozyme is a self cleaving RNA located in the second intron of the CPEB3 gene which belongs to a family of genes regulating messenger RNA polyadenylation [1]. This ribozyme is highly conserved and found only in mammals [1]. The CPEB3 ribozyme is structurally and biochemically related to the human hepatitis delta virus (HDV) ribozyme (Rfam:RF00094) and the HDV ribozyme is proposed to have arisen from the human transcriptome [1]. There is currently only one other known self cleaving ribozyme in mammals, the CoTC motif (Rfam:RF00621)[2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,98,0,0,Gene; ribozyme;,Published; PMID:16990549,40,0,,\"\",0.76967,-3.79125,1600000,238214,165,78,true,-3.6615,0.71889,2013-10-04T01:05:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00623,P1,1855,Pseudomonas sRNA P1,Daub J,INFERNAL,100.0,110.7,56.1,This small RNA (sRNA) was computationally identified in the genome of the opportunistic pathogen Pseudomonas aeruginosa and its expression verified by northern blot analysis [1]. There appears to be two related copies of P1 sRNA in the P.aeruginosa PA01 genome and both copies appear to be located upstream of predicted glutamine synthetase genes. This sRNA appears to be conserved amongst several Pseudomonas species. P1 has a predicted Rho independent terminator at the 3' end but the function of P1 is unknown [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,55,0,0,Gene;,Predicted; RNAalifold,64,0,,\"\",0.5227,-11.22143,1600000,435063,297,180,true,-3.4449,0.71445,2013-10-04T01:05:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00624,P9,1856,Pseudomonas sRNA P9,Daub J,INFERNAL,43.0,43.0,42.8,This small RNA (sRNA) was computationally identified in the genome of the opportunistic pathogen Pseudomonas aeruginosa and its expression verified by northern blot analysis [1]. P9 appears to be conserved in several Pseudomonas species in addition to Bordetella spp. In both Pseudomonas and Bordetella spp P9 appears to be located upstream of a predicted threonine dehydratase gene. P9 has a predicted Rho independent terminator at the 3' end but the function of P9 is unknown [1]. This sRNA also appears to be present in the Betaproteobacteria (Burkholderia and Ralstonia species).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,123,0,0,Gene;,Predicted; RNAalifold,124,0,,\"\",0.59589,-7.4398,1600000,333973,199,74,true,-3.5521,0.71898,2013-10-04T01:05:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00625,P11,1857,Pseudomonas sRNA P11,Daub J,INFERNAL,70.0,77.4,49.4,This small RNA (sRNA) was computationally identified in the genome of the opportunistic pathogen Pseudomonas aeruginosa and its expression verified by northern blot analysis [1]. P11 is located between a putative threonine protein kinase and putative nitrate reductase and appears to be conserved in several Pseudomonas species. P11 has a predicted Rho independent terminator at the 3'end but the function of P11 is unknown [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,41,0,0,Gene;,Predicted; RNAalifold,50,0,,\"\",0.58335,-9.65986,1600000,394302,244,139,true,-3.4427,0.71827,2013-10-04T01:05:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00626,Gurken,1858,Gurken localisation signal,Bateman A,\"Bateman A, INFERNAL\",90.0,95.3,53.6,\"Drosophila gurken mRNA is localized by dynein-mediated transport to a crescent near the oocyte nucleus, thus targeting the TGFalpha signal and forming the primary embryonic axes [1]. Gurken contains an RNA stem loop within the coding region that forms a signal for dynein-mediated RNA transport to the oocyte nucleus [1]. This hairpin is conserved in other drosophila species.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Cis-reg;,Published; PMID:15992540,2,0,,\"\",0.64543,-2.4203,1600000,201853,157,69,true,-3.695,0.7191,2013-10-04T01:05:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00627,P15,1859,Pseudomonas sRNA P15,Daub J,INFERNAL,70.0,78.1,68.6,This small RNA (sRNA) was computationally identified in the genome of the opportunistic pathogen Pseudomonas aeruginosa and its expression verified by northern blot analysis [1]. P15 is conserved across a wide range of Gammaproteobacteria. It appears to be consistently located upstream of a 3-deoxy-7-phosphoheptulonate synthase gene. P15 has a predicted Rho independent terminator at the 3'end but the function of P15 is unknown [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,66,0,0,Gene;,Predicted; RNAalifold,78,0,,\"\",0.56177,-9.26563,1600000,294569,223,119,true,-3.1679,0.71859,2013-10-04T01:05:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00628,RgsA,1860,RgsA sRNA,Daub J,INFERNAL,88.4,88.4,77.2,\"This small RNA (sRNA) was computationally identified in the genome of the opportunistic pathogen Pseudomonas aeruginosa and its expression verified by northern blot analysis [1]. P16 sRNA appears to be conserved across several Pseudomonas species and Azotobacter vinelandii. In these species it is consistently located downstream of a predicted TatD deoxyribonuclease gene. P16 has a predicted Rho independent terminator at the 3' end but the function of P16 is unknown [1]. It has been shown that this sRNA is transcribed from an RpoS-dependent promoter under positive, probably indirect GacA control in two Pseudomonas species. It was renamed RgsA (for regulation by GacA and stress).[2]RpoS mRNA expression is repressed by RgsA during the exponential phase. The Hfq RNA chaperone is required for the repression.[5]\",P16;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,27,62,0,0,Gene; sRNA;,Predicted; RNAalifold,85,0,,\"\",0.61498,-10.69795,1600000,574064,314,194,true,-3.6332,0.71282,2013-10-04T01:05:39.000Z,2022-05-06T13:10:04.000Z\n",
            "RF00629,P24,1861,Pseudomonas sRNA P24,Daub J,INFERNAL,100.0,109.1,94.9,This small RNA (sRNA) was computationally identified in the genome of the opportunistic pathogen Pseudomonas aeruginosa and its expression verified by Northern blot analysis [1]. P24 is conserved across several Pseudomonas species and in Azotobacter vinelandii. It appears to be consistently located between a hypothetical protein gene and a transcriptional regulator gene (AsnC family) in the genomes of all these species. P24 has a predicted Rho independent terminator at the 3' end but the function of P24 is unknown [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.84 -Z 742849.287494 CM SEQDB,14,57,0,0,Gene;,Predicted; RNAalifold,70,0,,\"\",0.6986,-8.22578,1600000,351270,392,255,true,-3.7119,0.7082,2013-10-04T01:05:45.000Z,2022-10-29T17:55:17.000Z\n",
            "RF00630,P26,1862,Pseudomonas sRNA P26,Daub J,INFERNAL,36.0,36.0,35.9,This small RNA (sRNA) was computationally identified in the genome of the opportunistic pathogen Pseudomonas aeruginosa and its expression verified by northern blot analysis [1]. P26 is conserved across many Gammaproteobacteria species and appears to be consistently located between the DNA directed RNA polymerase (beta subunit) and 50S ribosomal protein L7/L12 genes. There are two eukaryotic sequences (AAGJ02000785.1 and AAAB01036679.1) included in this alignment which appear to be contamination.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,27,330,0,0,Gene;,Predicted; RNAalifold,339,0,,\"\",0.58282,-6.84215,1600000,207374,136,66,true,-3.7329,0.7192,2013-10-04T01:05:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00632,sxy,1863,sxy 5' UTR element,Bateman A,Bateman A,70.0,71.3,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Published; PMID:17981840,3,0,,\"\",0.56452,-3.95741,1600000,212798,208,104,true,-2.9955,0.7185,2013-10-04T01:05:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00634,SAM-IV,1864,\"S-adenosyl methionine (SAM) riboswitch,\",\"Weinberg Z, Barrick JE\",Published; PMID:18369181,38.0,38.6,37.1,\"SAM-IV is a mRNA element that regulates gene expression via direct sensing of S-adenosylmethionine (SAM or AdoMet). SAM-IV riboswitches share conserved nucleotide positions with the previously described SAM-I riboswitches sequence analysis and molecular recognition experiments suggest that SAM-I and SAM-IV riboswitches share similar ligand binding sites, but have different scaffolds.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,40,1204,0,0,Cis-reg; riboswitch;,Published; PMID:18369181,654,0,,\"\",0.51212,-8.06531,1600000,277467,224,116,true,-3.1506,0.71835,2013-10-04T01:05:53.000Z,2021-07-05T14:43:57.000Z\n",
            "RF00635,HAR1A,1865,Highly accelerated region 1A/1B,Eberhardt R,Published; PMID:16915236; Eberhardt R,40.0,44.3,37.7,,HAR1F; HAR1R; HAR1B,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 37.49 -Z 549862.597050 CM SEQDB,13,88,0,0,Gene; lncRNA;,Published; PMID:16915236;,90,0,,\"\",0.38534,-10.31463,1600000,290641,221,118,false,-2.8123,0.7181,2013-10-04T01:05:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00636,NRON,1866,ncRNA Repressor of NFAT (nuclear factor of activated T cells),Gardner PP,Predicted; Gardner PP,50.0,100.9,32.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,24,93,0,0,Gene;,Predicted; Gardner PP,100,0,,\"\",0.69821,-9.54325,1600000,497841,532,370,true,-4.0719,0.70345,2013-10-04T01:06:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00637,mir-276,1287,microRNA mir-276,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.9,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,42,126,0,0,Gene; miRNA;,Predicted; RNAalifold,92,0,,\"\",0.69558,-3.26922,1600000,102728,179,91,true,-3.9813,0.71884,2013-10-04T01:06:05.000Z,2021-12-03T12:04:12.000Z\n",
            "RF00638,MIR159,1287,microRNA MIR159,Wilkinson A,miRBase; Wilkinson A,74.0,74.0,73.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,66,1174,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,94,0,,\"\",0.57322,-9.85352,1600000,364519,721,196,true,-3.7613,0.7127,2013-10-04T01:06:14.000Z,2021-11-19T11:53:18.000Z\n",
            "RF00639,mir-515,1287,mir-515 microRNA precursor family,\"Griffiths-Jones SR, Wilkinson A\",Griffiths-Jones SR,68.0,68.0,67.7,Family represents the microRNA (miRNA) precursor mir-515 imported from miRBase.,mir-373; mir-512,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.00 -Z 742849.287494 CM SEQDB,181,393,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.63031,-3.2753,1600000,241907,174,87,true,-3.5628,0.71894,2013-10-04T01:06:18.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00640,MIR167,1287,MIR167 microRNA precursor,Griffiths-Jones SR,Griffiths-Jones SR,100.0,110.6,98.3,This family represents the plant microRNA (miRNA) precursor MIR167 imported from miRBase.,MIR167_2; MIR167_1,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62326,-3.09266,1600000,191563,187,89,true,-3.03,0.71888,2013-10-04T01:06:27.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00641,mir-154,1867,mir-154 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,52.0,52.0,51.8,Family represents the microRNA (miRNA) precursor mir-154 imported from miRBase [5].,mir-368; mir-329; mir-379; mir-485; mir-654; mir-889,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,121,638,0,0,Gene; miRNA;,Predicted; RNAalifold,44,0,,\"\",0.62662,-4.04822,1600000,293655,143,80,true,-3.6612,0.71919,2013-10-04T01:06:32.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00642,mir-23,1868,microRNA mir-23,Wilkinson A,miRBase; Wilkinson A,47.9,56.3,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,22,214,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,116,0,,\"\",0.69069,-3.72821,1600000,199164,158,75,true,-3.8566,0.71891,2013-10-04T01:06:41.000Z,2022-05-06T13:10:04.000Z\n",
            "RF00643,MIR171_1,1287,microRNA MIR171_1,Wilkinson A,miRBase; Wilkinson A,48.0,48.0,47.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,57,1277,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,79,0,,\"\",0.571,-4.07969,1600000,131358,487,101,true,-3.8096,0.71859,2013-10-04T01:06:44.000Z,2021-11-19T11:53:18.000Z\n",
            "RF00644,mir-27,1290,microRNA mir-27,Griffiths-Jones SR,Griffiths-Jones SR,48.5,48.5,47.3,This family represents the microRNA (miRNA) precursor mir-27 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,82,225,0,0,Gene; miRNA;,Predicted; RNAalifold,123,0,,\"\",0.64679,-4.60356,1600000,144059,162,80,true,-3.6788,0.71891,2013-02-01T12:06:40.000Z,2022-05-06T12:43:49.000Z\n",
            "RF00645,MIR169_2,1287,microRNA MIR169_2,Wilkinson A,miRBase; Wilkinson A,50.0,50.0,49.9,,MIR169_1,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.55 -Z 549862.597050 CM SEQDB,100,1767,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,80,0,,\"\",0.79206,-2.09162,1600000,178244,1089,134,true,-3.3685,0.7184,2013-10-04T01:06:57.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00646,mir-204,1287,microRNA mir-204,Griffiths-Jones SR,Griffiths-Jones SR,48.4,48.4,39.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,60,267,0,0,Gene; miRNA;,Predicted; RNAalifold,121,0,,\"\",0.69169,-3.93304,1600000,155346,177,89,true,-3.6767,0.71871,2013-10-04T01:07:01.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00647,MIR164,1287,MIR164 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.0,63.8,This family represents the microRNA (miRNA) precursor MIR164 imported from miRBase [4].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,68,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.62973,-4.114,1600000,251810,157,68,true,-2.7806,0.727,2013-10-04T01:07:07.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00648,MIR396,2401,microRNA MIR396,Wilkinson A,miRBase; Wilkinson A,80.0,80.1,79.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,26,200,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,67,0,,\"\",0.63888,-3.20566,1600000,196496,675,118,true,-3.456,0.71835,2013-10-04T01:07:13.000Z,2021-11-19T11:53:19.000Z\n",
            "RF00649,mir-128,1287,microRNA mir-128,Griffiths-Jones SR,Griffiths-Jones SR,49.0,49.7,40.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,57,217,0,0,Gene; miRNA;,Predicted; RNAalifold,121,0,,\"\",0.72934,-2.53898,1600000,142562,178,83,true,-3.8541,0.7189,2013-10-04T01:07:16.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00650,mir-153,1870,microRNA mir-153,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.0,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,63,254,0,0,Gene; miRNA;,Predicted; RNAalifold,158,0,,\"\",0.5768,-3.94875,1600000,161298,152,73,true,-3.6932,0.71937,2013-10-04T01:07:24.000Z,2022-05-06T12:43:51.000Z\n",
            "RF00651,mir-221,1871,microRNA mir-221,Griffiths-Jones SR,Griffiths-Jones SR,46.6,46.6,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,72,289,0,0,Gene; miRNA;,Predicted; RNAalifold,120,0,,\"\",0.63508,-4.54833,1600000,125771,167,85,true,-4.0164,0.71888,2013-10-04T01:07:28.000Z,2022-05-06T12:43:51.000Z\n",
            "RF00652,MIR478,1287,microRNA MIR478,Wilkinson A,\"miRBase; Wilkinson A, Daub J\",50.0,50.5,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,17,523,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.43967,-9.37106,1600000,207015,181,91,false,-2.996,0.7184,2013-10-04T01:07:30.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00653,mir-22,1872,microRNA mir-22,Griffiths-Jones SR,Griffiths-Jones SR,48.0,48.4,42.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,48,128,0,0,Gene; miRNA;,Predicted; RNAalifold,118,0,,\"\",0.70678,-2.45559,1600000,139777,171,86,true,-3.8396,0.71881,2013-10-04T01:07:33.000Z,2022-05-06T12:43:51.000Z\n",
            "RF00654,mir-216,1287,microRNA mir-216,Wilkinson A,miRBase; Wilkinson A,53.5,53.5,53.4,Family represents the microRNA (miRNA) precursor mir-216 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 45.00 -Z 742849.287494 CM SEQDB,57,272,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,124,0,,\"\",0.60993,-3.13316,1600000,174537,148,77,true,-3.6848,0.7189,2013-10-04T01:07:36.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00655,mir-28,1874,microRNA mir-28,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.4,63.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,34,72,0,0,Gene; miRNA;,Predicted; RNAalifold,46,0,,\"\",0.5882,-4.40746,1600000,258654,166,78,true,-3.3197,0.71919,2013-10-04T01:07:43.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00656,mir-205,1875,microRNA mir-205,Griffiths-Jones SR,Griffiths-Jones SR,55.0,55.0,53.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,44,177,0,0,Gene; miRNA;,Predicted; RNAalifold,118,0,,\"\",0.70302,-3.29906,1600000,155696,158,75,true,-3.9842,0.719,2013-10-04T01:07:52.000Z,2021-12-03T12:04:12.000Z\n",
            "RF00657,mir-184,1876,microRNA mir-184,Wilkinson A,miRBase; Wilkinson A,54.1,54.6,53.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,32,193,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,185,0,,\"\",0.61979,-4.02401,1600000,117673,181,84,true,-3.896,0.71896,2013-10-04T01:07:55.000Z,2022-05-06T13:10:04.000Z\n",
            "RF00658,mir-21,1877,microRNA mir-21,Griffiths-Jones SR,Griffiths-Jones SR,52.0,53.8,51.5,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,41,121,0,0,Gene; miRNA;,Predicted; RNAalifold,119,0,,\"\",0.67331,-3.83703,1600000,193396,157,72,true,-3.4257,0.71906,2013-10-04T01:08:02.000Z,2022-05-06T12:43:51.000Z\n",
            "RF00659,mir-365,1878,microRNA mir-365,Griffiths-Jones SR,Griffiths-Jones SR,62.0,64.1,58.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,46,215,0,0,Gene; miRNA;,Predicted; RNAalifold,117,0,,\"\",0.62335,-3.46334,1600000,158520,179,86,true,-3.39,0.71877,2013-10-04T01:08:05.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00660,mir-214,1879,mir-214 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,66.7,66.7,66.6,Family represents the microRNA (miRNA) precursor mir-214 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,39,225,0,0,Gene; miRNA;,Predicted; RNAalifold,119,0,,\"\",0.69921,-2.89164,1600000,118850,197,101,true,-3.6588,0.71861,2013-10-04T01:08:08.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00661,mir-31,1880,mir-31 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,55.4,55.7,55.2,Family represents the microRNA (miRNA) precursor mir-31 imported from miRBase [4]. mir-31 is part of clan CL00133: RF00661_mir-31 and RF04091_mir-72,mir-72,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,28,170,0,0,Gene; miRNA;,Predicted; RNAalifold,154,0,,\"\",0.62299,-3.65517,1600000,123982,321,81,true,-3.953,0.7189,2013-10-04T01:08:10.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00663,mir-183,1287,microRNA mir-183,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.9,48.8,This family represents the microRNA (miRNA) precursor mir-183 imported from miRBase [4].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 45.00 -Z 742849.287494 CM SEQDB,39,110,0,0,Gene; miRNA;,Predicted; RNAalifold,105,0,,\"\",0.67447,-2.99995,1600000,170394,157,74,true,-3.4659,0.71914,2013-10-04T01:08:24.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00664,mir-223,1882,microRNA mir-223,Griffiths-Jones SR,Griffiths-Jones SR,57.0,57.5,54.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,27,107,0,0,Gene; miRNA;,Predicted; RNAalifold,115,0,,\"\",0.69271,-3.52166,1600000,107450,204,106,true,-3.7704,0.71863,2013-10-04T01:08:27.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00665,mir-290,1287,mir-290 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.0,50.9,This family represents the microRNA (miRNA) precursor mir-290 imported from miRBase [4].,mir-72; mir-373; mir-512,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,31,273,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.60214,-4.54232,1600000,218749,138,70,true,-3.6633,0.71919,2013-02-01T12:06:42.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00666,mir-32,1883,mir-32 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,74.3,41.9,This family represents the microRNA (miRNA) precursor mir-32 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,22,95,0,0,Gene; miRNA;,Predicted; RNAalifold,99,0,,\"\",0.68419,-3.22942,1600000,250719,158,70,true,-3.543,0.71912,2013-10-04T01:08:46.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00667,mir-33,1884,microRNA mir-33,Wilkinson A,miRBase; Wilkinson A,50.0,50.5,49.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,50,250,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,203,0,,\"\",0.65328,-4.58536,1600000,223503,143,69,true,-3.8087,0.71909,2013-10-04T01:08:49.000Z,2022-05-06T13:10:04.000Z\n",
            "RF00668,mir-302,1287,microRNA mir-302,Griffiths-Jones SR,Griffiths-Jones SR,55.0,55.0,54.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,49,382,0,0,Gene; miRNA;,Predicted; RNAalifold,97,0,,\"\",0.62439,-3.57005,1600000,282482,146,69,true,-3.6708,0.71948,2013-10-04T01:08:56.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00669,mir-96,1885,microRNA mir-96,Griffiths-Jones SR,Griffiths-Jones SR,65.0,68.0,64.9,,MIR96,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,29,104,0,0,Gene; miRNA;,Predicted; RNAalifold,101,0,,\"\",0.64158,-3.13109,1600000,163008,163,78,true,-3.8616,0.71902,2013-10-04T01:09:00.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00670,mir-105,1287,mir-105 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.9,66.0,This family represents the microRNA (miRNA) precursor mir-105 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,23,66,0,0,Gene; miRNA;,Predicted; RNAalifold,42,0,,\"\",0.6504,-2.96226,1600000,180483,174,81,true,-3.4778,0.71902,2013-10-04T01:09:03.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00671,mir-138,1886,microRNA mir-138,Griffiths-Jones SR,Griffiths-Jones SR,50.0,50.3,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,44,212,0,0,Gene; miRNA;,Predicted; RNAalifold,115,0,,\"\",0.63755,-4.23916,1600000,224562,163,78,true,-3.8566,0.71905,2013-10-04T01:09:14.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00672,mir-190,1887,microRNA mir-190,Griffiths-Jones SR,Griffiths-Jones SR,48.0,48.1,47.9,This family represents the microRNA (miRNA) precursor mir-190 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,29,462,0,0,Gene; miRNA;,Predicted; RNAalifold,218,0,,\"\",0.63259,-2.4657,1600000,150994,174,89,true,-3.7303,0.71884,2013-10-04T01:09:17.000Z,2022-10-29T17:55:17.000Z\n",
            "RF00673,mir-217,1287,microRNA mir-217,Griffiths-Jones SR,Griffiths-Jones SR,70.0,79.6,69.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,28,113,0,0,Gene; miRNA;,Predicted; RNAalifold,118,0,,\"\",0.56842,-3.42405,1600000,137424,186,93,true,-3.7811,0.71861,2013-10-04T01:09:25.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00674,mir-187,1888,microRNA mir-187,Griffiths-Jones SR,Griffiths-Jones SR,58.0,59.6,54.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,27,105,0,0,Gene; miRNA;,Predicted; RNAalifold,112,0,,\"\",0.63856,-3.81551,1600000,139036,171,81,true,-3.4417,0.71894,2013-10-04T01:09:29.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00675,mir-145,1889,microRNA mir-145,Griffiths-Jones SR,Griffiths-Jones SR,50.0,53.1,49.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,26,100,0,0,Gene; miRNA;,Predicted; RNAalifold,105,0,,\"\",0.64266,-4.49523,1600000,157996,176,86,true,-3.698,0.71877,2013-10-04T01:09:31.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00676,mir-127,1890,microRNA mir-127,Griffiths-Jones SR,Griffiths-Jones SR,60.0,64.5,40.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,19,38,0,0,Gene; miRNA;,Predicted; RNAalifold,45,0,,\"\",0.70209,-2.40737,1600000,134727,197,97,true,-3.7016,0.71857,2013-10-04T01:09:34.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00677,MIR168,1287,microRNA MIR168,Wilkinson A,miRBase; Wilkinson A,66.0,66.3,64.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,171,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,69,0,,\"\",0.68172,-4.94611,1600000,197897,384,133,true,-3.5905,0.71832,2013-10-04T01:09:36.000Z,2021-11-19T11:53:18.000Z\n",
            "RF00678,mir-140,1287,microRNA mir-140,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.8,55.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,32,95,0,0,Gene; miRNA;,Predicted; RNAalifold,99,0,,\"\",0.6705,-3.81403,1600000,173242,171,79,true,-3.6376,0.71907,2013-10-04T01:09:48.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00679,mir-210,1891,microRNA mir-210,Wilkinson A,miRBase; Wilkinson A,50.0,50.0,48.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,26,160,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,150,0,,\"\",0.71523,-1.98313,1600000,118105,181,96,true,-4.2882,0.71877,2013-10-04T01:09:51.000Z,2022-05-06T13:10:04.000Z\n",
            "RF00680,mir-224,1892,mir-224 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,75.0,53.0,This family represents the microRNA (miRNA) precursor mir-224 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,36,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.63545,-3.10638,1600000,235082,174,81,true,-3.4938,0.71909,2013-10-04T01:09:53.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00681,mir-198,1893,mir-198 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,84.5,47.3,This family represents the microRNA (miRNA) precursor mir-198 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,9,0,0,Gene; miRNA;,Predicted; RNAalifold,16,0,,\"\",0.65357,-3.01616,1600000,299354,146,62,true,-3.513,0.71919,2013-10-04T01:09:55.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00682,mir-144,1894,microRNA mir-144,Griffiths-Jones SR,Griffiths-Jones SR,60.0,65.1,47.7,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,32,78,0,0,Gene; miRNA;,Predicted; RNAalifold,88,0,,\"\",0.66752,-3.61716,1600000,162025,165,78,true,-3.6592,0.71899,2013-10-04T01:10:05.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00683,mir-143,1895,microRNA mir-143,Griffiths-Jones SR,Griffiths-Jones SR,60.0,63.4,50.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,30,107,0,0,Gene; miRNA;,Predicted; RNAalifold,115,0,,\"\",0.68871,-3.26424,1600000,126865,175,84,true,-3.533,0.71882,2013-10-04T01:10:07.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00684,mir-122,1896,mir-122 microRNA precursor,Griffiths-Jones SR,Griffiths-Jones SR,70.0,75.3,69.7,\"miR-122 is miRNA precursor family MIPF:MIPF0000095 that appears to be vertebrate specific. In mammals, miR-122 is enriched in liver cells where it has been implicated as a regulator of fatty-acid metabolism in mouse studies. The hairpin precursors (represented here) are predicted based on base pairing and cross-species conservation -- their extents are not known. In this case the mature sequence is excised from the 5' arm of the hairpin.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,32,114,0,0,Gene; miRNA;,Predicted; RNAalifold,114,0,,\"\",0.62283,-4.50128,1600000,211040,152,73,true,-3.8701,0.71905,2013-10-04T01:10:15.000Z,2021-12-03T12:04:12.000Z\n",
            "RF00685,mir-36,1287,microRNA mir-36,Wilkinson A,miRBase; Wilkinson A,48.0,48.1,47.0,,mir-35; mir-39,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,46,71,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,9,0,,\"\",0.64784,-3.14655,1600000,106961,186,103,true,-4.414,0.7186,2013-10-04T01:10:17.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00686,mir-338,1897,microRNA mir-338,Griffiths-Jones SR,Griffiths-Jones SR,57.0,57.5,55.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,36,158,0,0,Gene; miRNA;,Predicted; RNAalifold,107,0,,\"\",0.60619,-3.71279,1600000,176155,151,69,true,-3.6491,0.71912,2013-10-04T01:10:21.000Z,2021-12-03T12:04:12.000Z\n",
            "RF00687,mir-136,1287,microRNA mir-136,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.2,59.5,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,37,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.61304,-3.68384,1600000,218505,176,82,true,-3.8147,0.71906,2013-10-04T01:10:25.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00688,MIR394,2402,microRNA MIR394,Wilkinson A,miRBase; Wilkinson A,60.0,60.2,59.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,19,213,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,73,0,,\"\",0.607,-4.3563,1600000,210197,272,111,true,-3.1429,0.71848,2013-10-04T01:10:28.000Z,2021-12-03T12:41:14.000Z\n",
            "RF00689,MIR390,2403,microRNA MIR390,Wilkinson A,miRBase; Wilkinson A,77.0,77.1,75.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,209,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,67,0,,\"\",0.64832,-2.93404,1600000,198381,269,123,true,-3.2166,0.71836,2013-10-04T01:10:33.000Z,2021-12-03T12:41:14.000Z\n",
            "RF00690,MIR408,2404,microRNA MIR408,Wilkinson A,miRBase; Wilkinson A,64.0,65.5,63.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,99,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,62,0,,\"\",0.70608,-5.21761,1600000,267429,842,136,true,-3.4212,0.71821,2013-10-04T01:10:35.000Z,2021-12-03T12:41:14.000Z\n",
            "RF00691,mir-146,1901,microRNA mir-146,Wilkinson A,miRBase; Wilkinson A,50.0,51.2,48.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,33,258,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,117,0,,\"\",0.6416,-3.78764,1600000,148243,173,90,true,-4.111,0.7187,2013-10-04T01:10:52.000Z,2021-12-03T12:41:14.000Z\n",
            "RF00692,MIR171_2,1287,microRNA MIR171_2,Wilkinson A,miRBase; Wilkinson A,65.0,65.1,64.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,108,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,53,0,,\"\",0.67842,-3.14494,1600000,193176,218,112,true,-3.3116,0.71845,2013-10-04T01:10:56.000Z,2021-12-03T12:41:13.000Z\n",
            "RF00693,mir-147,1287,mir-147 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,57.0,57.2,56.6,Family represents the microRNA (miRNA) precursor mir-147 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,26,147,0,0,Gene; miRNA;,Predicted; RNAalifold,111,0,,\"\",0.68595,-1.91371,1600000,204463,163,78,true,-3.8755,0.71913,2013-10-04T01:11:00.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00694,mir-137,1902,microRNA mir-137,Griffiths-Jones SR,Griffiths-Jones SR,56.7,56.7,52.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,62,226,0,0,Gene; miRNA;,Predicted; RNAalifold,202,0,,\"\",0.62615,-3.32165,1600000,179140,154,72,true,-3.8219,0.71926,2013-10-04T01:11:08.000Z,2021-12-03T12:04:12.000Z\n",
            "RF00695,MIR398,1903,microRNA MIR398,Wilkinson A,miRBase; Wilkinson A,55.0,55.0,54.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,205,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,75,0,,\"\",0.65046,-4.43872,1600000,198637,365,109,true,-3.2291,0.71847,2013-10-04T01:11:12.000Z,2021-12-03T12:41:14.000Z\n",
            "RF00696,mir-203,1904,microRNA mir-203,Griffiths-Jones SR,Griffiths-Jones SR,65.0,65.0,64.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,32,122,0,0,Gene; miRNA;,Predicted; RNAalifold,62,0,,\"\",0.64716,-4.12656,1600000,149778,166,80,true,-3.822,0.7189,2013-10-04T01:11:14.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00697,mir-186,1905,microRNA mir-186,Griffiths-Jones SR,Griffiths-Jones SR,70.0,71.1,56.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,17,39,0,0,Gene; miRNA;,Predicted; RNAalifold,44,0,,\"\",0.6024,-3.41081,1600000,166687,174,82,true,-3.4554,0.71898,2013-10-04T01:11:22.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00698,mir-489,1906,microRNA mir-489,Griffiths-Jones SR,Griffiths-Jones SR,61.0,61.8,60.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,21,102,0,0,Gene; miRNA;,Predicted; RNAalifold,102,0,,\"\",0.63586,-3.34874,1600000,222966,175,84,true,-3.373,0.71885,2013-10-04T01:11:44.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00699,mir-134,1907,microRNA mir-134,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.1,61.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,31,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.61049,-3.83255,1600000,224240,163,73,true,-3.2775,0.71907,2013-10-04T01:11:47.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00700,mir-375,1908,microRNA mir-375,Griffiths-Jones SR,Griffiths-Jones SR,46.7,46.7,46.5,This family represents the microRNA (miRNA) precursor mir-375 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.00 -Z 742849.287494 CM SEQDB,47,157,0,0,Gene; miRNA;,Predicted; RNAalifold,160,0,,\"\",0.68943,-3.55309,1600000,229186,151,67,true,-4.1595,0.7192,2013-10-04T01:11:49.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00701,mir-126,1909,microRNA mir-126,Griffiths-Jones SR,Griffiths-Jones SR,47.2,47.2,46.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,37,225,0,0,Gene; miRNA;,Predicted; RNAalifold,111,0,,\"\",0.58622,-4.0454,1600000,140594,156,74,true,-4.1473,0.71908,2013-10-04T01:11:53.000Z,2022-05-06T12:43:51.000Z\n",
            "RF00702,mir-182,1287,mir-182 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,72.5,64.4,Family represents the microRNA (miRNA) precursor mir-182 imported from miRBase [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 CM SEQDB,28,91,0,0,Gene; miRNA;,Predicted; RNAalifold,84,0,,\"\",0.67521,-2.90509,1600000,174862,300,77,true,-3.2135,0.71899,2013-10-04T01:11:56.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00703,mir-139,1287,microRNA mir-139,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.5,41.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,29,81,0,0,Gene; miRNA;,Predicted; RNAalifold,86,0,,\"\",0.66583,-2.96644,1600000,193541,151,68,true,-3.6171,0.71926,2013-10-04T01:11:58.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00704,MIR397,2405,microRNA MIR397,Wilkinson A,miRBase; Wilkinson A,55.0,55.0,54.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,176,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,69,0,,\"\",0.59348,-3.89576,1600000,216731,215,117,true,-3.2191,0.71843,2013-10-04T01:12:01.000Z,2021-12-03T12:04:12.000Z\n",
            "RF00705,mir-202,1911,microRNA mir-202,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.1,49.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,23,104,0,0,Gene; miRNA;,Predicted; RNAalifold,105,0,,\"\",0.65999,-3.96944,1600000,179176,163,76,true,-3.4282,0.71901,2013-10-04T01:12:14.000Z,2022-05-06T12:43:51.000Z\n",
            "RF00706,mir-263,1287,mir-263 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,51.3,51.3,51.0,This family represents the microRNA (miRNA) precursor mir-263 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,52,180,0,0,Gene; miRNA;,Predicted; RNAalifold,113,0,,\"\",0.72662,-2.61839,1600000,138604,157,86,true,-3.621,0.71865,2013-10-04T01:12:17.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00707,mir-197,1287,microRNA mir-197,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.6,57.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,26,0,0,Gene; miRNA;,Predicted; RNAalifold,32,0,,\"\",0.55574,-4.51309,1600000,274371,166,75,true,-3.2811,0.71908,2013-10-04T01:12:20.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00708,mir-450,1287,microRNA mir-450,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.3,58.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,36,95,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.57168,-2.86195,1600000,179196,165,78,true,-3.4738,0.71897,2013-10-04T01:12:22.000Z,2021-12-03T12:04:12.000Z\n",
            "RF00709,mir-455,1912,mir-455 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,73.9,67.0,This family represents the microRNA (miRNA) precursor mir-455 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,29,121,0,0,Gene; miRNA;,Predicted; RNAalifold,114,0,,\"\",0.66065,-3.79559,1600000,164274,178,84,true,-3.7191,0.7188,2013-10-04T01:12:39.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00710,mir-44,1287,microRNA mir-44,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.2,53.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,13,48,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.67705,-3.19106,1600000,136036,174,88,true,-3.8837,0.71884,2013-02-01T12:06:45.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00711,mir-449,1913,mir-449 microRNA precursor family,\"Griffiths-Jones SR, Wilkinson A\",Griffiths-Jones SR; Wilkinson A,60.4,60.4,60.3,Family represents the microRNA (miRNA) precursor mir-449 imported from miRBase [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 55.00 -Z 742849.287494 CM SEQDB,41,79,0,0,Gene; miRNA;,Predicted; RNAalifold,52,0,,\"\",0.69379,-2.28187,1600000,180636,167,86,true,-3.6645,0.71919,2013-10-04T01:12:49.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00712,mir-460,1287,microRNA mir-460,Griffiths-Jones SR,Griffiths-Jones SR,52.0,52.5,49.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,21,144,0,0,Gene; miRNA;,Predicted; RNAalifold,74,0,,\"\",0.6824,-3.85528,1600000,160674,159,77,true,-4.063,0.71891,2013-10-04T01:13:07.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00713,mir-239,1287,microRNA mir-239,Griffiths-Jones SR,Griffiths-Jones SR,49.0,49.3,48.7,This family represents the microRNA (miRNA) precursor mir-239 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,31,0,0,Gene; miRNA;,Predicted; RNAalifold,21,0,,\"\",0.65915,-3.22865,1600000,186383,143,74,true,-3.5382,0.71979,2013-10-04T01:14:11.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00714,MIR535,1291,MIR535 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,64.0,64.0,63.9,Family represents the microRNA (miRNA) precursor MIR535 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,20,133,0,0,Gene; miRNA;,Predicted; RNAalifold,30,0,,\"\",0.61025,-3.84302,1600000,151737,187,95,true,-3.7041,0.71867,2013-02-01T12:06:47.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00715,mir-383,1914,microRNA mir-383,Griffiths-Jones SR,Griffiths-Jones SR,70.0,75.2,43.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,21,98,0,0,Gene; miRNA;,Predicted; RNAalifold,100,0,,\"\",0.66907,-3.52204,1600000,212078,162,73,true,-3.617,0.71909,2013-10-04T01:14:35.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00716,mir-3,1915,microRNA mir-3,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.7,53.6,family represents the microRNA (miRNA) precursor mir-3 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 CM SEQDB,28,67,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.60852,-3.87702,1600000,293312,151,74,true,-3.5526,0.71892,2013-10-04T01:14:53.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00717,mir-315,1287,microRNA mir-315,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.7,61.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,27,71,0,0,Gene; miRNA;,Predicted; RNAalifold,80,0,,\"\",0.57987,-2.93953,1600000,174780,162,83,true,-4.1274,0.71885,2013-10-04T01:15:00.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00718,mir-431,1916,mir-431 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,102.0,53.1,This family represents the microRNA (miRNA) precursor mir-431 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,35,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.54804,-4.55855,1600000,217322,182,86,true,-3.081,0.71877,2013-10-04T01:15:15.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00719,mir-326,1917,microRNA mir-326,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.0,50.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,35,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.6044,-3.48486,1600000,160046,194,95,true,-3.4957,0.7188,2013-10-04T01:15:17.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00720,mir-317,1287,microRNA mir-317,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.4,54.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,25,81,0,0,Gene; miRNA;,Predicted; RNAalifold,82,0,,\"\",0.68638,-4.05576,1600000,201575,178,93,true,-3.6142,0.71864,2013-10-04T01:15:21.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00721,MIR475,1287,microRNA MIR475,Griffiths-Jones SR,Griffiths-Jones SR,70.0,76.3,68.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,9,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58006,-3.72438,1600000,165306,234,128,true,-2.9736,0.71834,2013-10-04T01:15:24.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00722,mir-451,1918,microRNA mir-451,Griffiths-Jones SR,Griffiths-Jones SR,66.5,66.5,65.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,22,99,0,0,Gene; miRNA;,Predicted; RNAalifold,71,0,,\"\",0.65582,-3.40269,1600000,171620,153,71,true,-3.998,0.71913,2013-10-04T01:15:32.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00723,mir-448,1919,mir-448 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,101.3,87.2,This family represents the microRNA (miRNA) precursor mir-448 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.00 -Z 742849.287494 CM SEQDB,10,33,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.56936,-4.78686,1600000,154992,216,111,true,-3.1871,0.71852,2013-10-04T01:15:36.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00724,mir-282,1920,mir-282 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,55.0,55.1,54.3,Family represents the microRNA (miRNA) precursor mir-282 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,21,82,0,0,Gene; miRNA;,Predicted; RNAalifold,78,0,,\"\",0.59095,-3.12719,1600000,118683,181,92,true,-3.946,0.71874,2013-10-04T01:15:45.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00725,mir-iab-4,1921,mir-iab-4 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.6,72.0,This family represents the microRNA (miRNA) precursor mir-iab-4 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,82,0,0,Gene; miRNA;,Predicted; RNAalifold,81,0,,\"\",0.65937,-2.95095,1600000,205293,159,71,true,-3.275,0.71919,2013-10-04T01:15:48.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00726,mir-87,1287,mir-87 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,49.0,49.2,48.8,This family represents the microRNA (miRNA) precursor mir-87 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,44,206,0,0,Gene; miRNA;,Predicted; RNAalifold,156,0,,\"\",0.63491,-4.04116,1600000,159332,207,79,true,-3.9808,0.71886,2013-10-04T01:15:52.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00727,bantam,1922,bantam microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,61.0,61.8,60.5,Family represents the microRNA (miRNA) precursor bantam imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,28,90,0,0,Gene; miRNA;,Predicted; RNAalifold,95,0,,\"\",0.62113,-2.55503,1600000,163989,163,82,true,-3.7804,0.71887,2013-10-04T01:16:07.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00728,mir-81,1287,mir-81 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,54.5,54.6,54.3,family represents the microRNA (miRNA) precursor mir-81 imported from miRBase,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,10,55,0,0,Gene; miRNA;,Predicted; RNAalifold,44,0,,\"\",0.67615,-2.58164,1600000,113289,176,89,true,-3.8117,0.71881,2013-10-04T01:16:10.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00729,mir-278,1923,microRNA mir-278,Griffiths-Jones SR,Griffiths-Jones SR,55.0,55.2,54.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,26,67,0,0,Gene; miRNA;,Predicted; RNAalifold,77,0,,\"\",0.68846,-3.56275,1600000,190994,152,75,true,-3.6485,0.71901,2013-10-04T01:16:17.000Z,2022-05-06T12:43:51.000Z\n",
            "RF00730,mir-277,1924,microRNA mir-277,Griffiths-Jones SR,Griffiths-Jones SR,43.0,43.6,41.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,25,79,0,0,Gene; miRNA;,Predicted; RNAalifold,86,0,,\"\",0.69196,-2.93217,1600000,115216,206,90,true,-3.7552,0.71878,2013-10-04T01:16:20.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00731,mir-155,2482,microRNA mir-155,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.6,55.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,25,107,0,0,Gene; miRNA;,Predicted; RNAalifold,110,0,,\"\",0.64253,-2.52697,1600000,170122,146,65,true,-3.9624,0.71921,2013-10-04T01:16:26.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00732,mir-305,1926,microRNA mir-305,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.8,64.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23000 -Z 742849.287494 CM SEQDB,23,80,0,0,Gene; miRNA;,Predicted; RNAalifold,85,0,,\"\",0.70589,-2.15176,1600000,255521,158,73,true,-3.7068,0.71903,2013-10-04T01:16:34.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00733,mir-296,1927,microRNA mir-296,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.7,55.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,32,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.63565,-3.48341,1600000,178024,170,79,true,-3.5916,0.71903,2013-10-04T01:16:39.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00734,mir-52,1928,microRNA mir-52,Griffiths-Jones SR,Griffiths-Jones SR,50.0,67.4,38.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,9,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.66789,-4.02949,1600000,169408,186,92,true,-3.3666,0.71866,2013-10-04T01:16:44.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00735,mir-367,1929,microRNA mir-367,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.3,51.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,97,0,0,Gene; miRNA;,Predicted; RNAalifold,95,0,,\"\",0.62921,-4.38237,1600000,229494,153,68,true,-3.6367,0.7192,2013-10-04T01:16:46.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00736,mir-320,1930,mir-320 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,55.5,55.8,55.2,This family represents the microRNA (miRNA) precursor mir-320 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,38,176,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.66815,-3.69601,1600000,146156,160,76,true,-3.9507,0.71892,2013-10-04T01:16:48.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00737,mir-322,1931,microRNA mir-322,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.7,61.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,15,33,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.61869,-3.89005,1600000,212493,177,84,true,-3.6616,0.71894,2013-10-04T01:18:48.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00739,MIR476,1287,MIR476 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,111.6,73.9,This family represents the microRNA (miRNA) precursor MIR476 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59293,-2.85149,1600000,159388,190,92,true,-3.0518,0.71877,2013-10-04T01:18:51.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00740,mir-370,1932,mir-370 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,87.7,40.0,This family represents the microRNA (miRNA) precursor mir-370 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,36,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.69837,-2.5649,1600000,248505,166,75,true,-3.6467,0.7191,2013-10-04T01:19:00.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00741,mir-378,1287,microRNA mir-378,\"Wilkinson A, Daub J\",\"miRBase; Wilkinson A, Daub J\",80.0,80.3,48.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,36,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.62642,-3.94041,1600000,245658,153,66,true,-3.6487,0.71931,2013-10-04T01:19:04.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00742,MIR162_2,1287,microRNA MIR162_2,Wilkinson A,miRBase; Wilkinson A,60.0,60.0,59.8,,MIR162_1,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,217,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,64,0,,\"\",0.61869,-3.72947,1600000,175470,419,92,true,-3.21,0.7188,2013-10-04T01:19:08.000Z,2022-05-06T13:09:57.000Z\n",
            "RF00743,mir-308,1287,mir-308 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,60.0,60.3,58.5,Family represents the microRNA (miRNA) precursor mir-308 imported from miRBase [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,18,40,0,0,Gene; miRNA;,Predicted; RNAalifold,47,0,,\"\",0.65976,-3.48104,1600000,260506,141,63,true,-3.1853,0.71926,2013-10-04T01:19:11.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00744,mir-361,1933,microRNA mir-361,Griffiths-Jones SR,Griffiths-Jones SR,70.0,73.1,56.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,34,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.66462,-3.13751,1600000,280458,159,71,true,-3.0109,0.71911,2013-10-04T01:19:21.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00745,mir-499,1287,microRNA mir-499,Griffiths-Jones SR,Griffiths-Jones SR,60.0,62.7,54.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,31,113,0,0,Gene; miRNA;,Predicted; RNAalifold,113,0,,\"\",0.59161,-5.77418,1600000,207892,168,80,true,-3.9255,0.71887,2013-10-04T01:19:23.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00746,mir-454,1934,microRNA mir-454,Griffiths-Jones SR,Griffiths-Jones SR,60.0,64.6,49.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,20,107,0,0,Gene; miRNA;,Predicted; RNAalifold,100,0,,\"\",0.65353,-2.8916,1600000,103455,198,103,true,-3.9047,0.71873,2013-10-04T01:19:29.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00747,mir-283,1287,microRNA mir-283,Wilkinson A,miRBase; Wilkinson A,50.0,51.1,49.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,58,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,63,0,,\"\",0.6043,-2.27385,1600000,109724,190,99,true,-3.5599,0.71866,2013-10-04T01:19:32.000Z,2022-05-06T13:09:57.000Z\n",
            "RF00748,mir-433,1935,mir-433 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,81.9,44.8,This family represents the microRNA (miRNA) precursor mir-433 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,38,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.59336,-3.79057,1600000,220979,192,93,true,-3.7778,0.71878,2013-10-04T01:19:38.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00749,mir-208,1936,microRNA mir-208,Griffiths-Jones SR,Griffiths-Jones SR,56.4,56.4,56.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,29,136,0,0,Gene; miRNA;,Predicted; RNAalifold,58,0,,\"\",0.69143,-2.77138,1600000,171787,159,77,true,-4.0767,0.71898,2013-10-04T01:19:39.000Z,2022-05-06T12:43:52.000Z\n",
            "RF00750,mir-458,1287,microRNA mir-458,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.8,54.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,146,0,0,Gene; miRNA;,Predicted; RNAalifold,72,0,,\"\",0.64728,-3.26091,1600000,209550,164,78,true,-3.5865,0.71912,2013-10-04T01:19:56.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00751,mir-12,1287,microRNA mir-12,Griffiths-Jones SR,Griffiths-Jones SR,55.0,55.6,54.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,21,58,0,0,Gene; miRNA;,Predicted; RNAalifold,66,0,,\"\",0.64262,-2.94446,1600000,180791,642,74,true,-4.1671,0.71903,2013-10-04T01:19:59.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00752,mir-14,1937,microRNA mir-14,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.7,51.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,23,79,0,0,Gene; miRNA;,Predicted; RNAalifold,84,0,,\"\",0.64002,-4.38137,1600000,273286,146,63,true,-3.764,0.71927,2013-10-04T01:20:02.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00753,mir-503,1938,microRNA mir-503,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.7,50.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,32,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.67687,-3.26051,1600000,243401,158,71,true,-3.5545,0.71919,2013-10-04T01:21:30.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00754,mir-279,1939,mir-279 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,56.0,56.0,55.9,Family represents the microRNA (miRNA) precursor mir-279 imported from miRBase [4].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,48,161,0,0,Gene; miRNA;,Predicted; RNAalifold,98,0,,\"\",0.61945,-2.67399,1600000,122755,157,87,true,-4.5204,0.71879,2013-10-04T01:21:32.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00755,mir-542,1940,microRNA mir-542,Griffiths-Jones SR,Griffiths-Jones SR,70.0,71.5,44.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,13,32,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.66297,-2.79489,1600000,189081,173,80,true,-3.6258,0.71886,2013-10-04T01:21:39.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00756,mir-299,1941,microRNA mir-299,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.5,65.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,69,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.6657,-3.04026,1600000,260200,147,63,true,-4.1042,0.71947,2013-10-04T01:21:41.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00757,mir-275,1942,microRNA mir-275,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.3,55.5,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,25,77,0,0,Gene; miRNA;,Predicted; RNAalifold,83,0,,\"\",0.60314,-4.4073,1600000,107715,176,89,true,-3.5985,0.71874,2013-10-04T01:21:55.000Z,2022-05-06T12:43:52.000Z\n",
            "RF00758,mir-346,1943,mir-346 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,89.0,69.0,This family represents the microRNA (miRNA) precursor mir-346 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,34,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.554,-4.24262,1600000,180105,195,95,true,-3.3097,0.7188,2013-10-04T01:21:57.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00760,mir-342,1287,mir-342 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.2,43.0,This family represents the microRNA (miRNA) precursor mir-342 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,35,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.64558,-3.67805,1600000,131147,198,99,true,-3.2354,0.71865,2013-10-04T01:22:00.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00761,mir-340,1944,microRNA mir-340,Griffiths-Jones SR,Griffiths-Jones SR,70.0,73.3,59.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,14,37,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.65465,-2.22289,1600000,139135,194,95,true,-3.2355,0.71863,2013-10-04T01:22:05.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00762,mir-412,1287,mir-412 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.7,74.9,This family represents the microRNA (miRNA) precursor mir-412 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,32,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.56619,-3.78861,1600000,164728,182,87,true,-3.4168,0.71891,2013-10-04T01:22:20.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00763,mir-339,1945,microRNA mir-339,Griffiths-Jones SR,Griffiths-Jones SR,60.0,61.4,58.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,31,0,0,Gene; miRNA;,Predicted; RNAalifold,33,0,,\"\",0.53569,-4.58117,1600000,167222,186,92,true,-3.3538,0.71879,2013-10-04T01:22:22.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00764,mir-191,1946,microRNA mir-191,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.7,53.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,24,103,0,0,Gene; miRNA;,Predicted; RNAalifold,63,0,,\"\",0.64721,-3.72198,1600000,150540,183,91,true,-3.5851,0.71879,2013-10-04T01:22:26.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00765,mir-337,1947,mir-337 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.6,50.0,This family represents the microRNA (miRNA) precursor mir-337 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,29,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.62979,-4.03172,1600000,156967,191,93,true,-3.5411,0.71885,2013-10-04T01:22:32.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00766,mir-335,1287,microRNA mir-335,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.9,63.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,14,37,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.67105,-3.05754,1600000,161683,190,92,true,-3.5651,0.7189,2013-10-04T01:22:35.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00767,mir-150,1948,microRNA mir-150,Griffiths-Jones SR,Griffiths-Jones SR,55.0,56.4,53.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,22,47,0,0,Gene; miRNA;,Predicted; RNAalifold,53,0,,\"\",0.59638,-4.68848,1600000,157369,167,82,true,-3.9753,0.71898,2013-10-04T01:22:37.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00768,MIR405,1287,MIR405 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,102.0,102.1,101.3,This family represents the microRNA (miRNA) precursor MIR405 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.5989,-1.70633,1600000,144942,176,83,true,-3.72,0.71898,2013-10-04T01:22:52.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00769,mir-331,1949,microRNA mir-331,Griffiths-Jones SR,Griffiths-Jones SR,80.0,91.4,54.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,15,38,0,0,Gene; miRNA;,Predicted; RNAalifold,42,0,,\"\",0.60216,-3.95567,1600000,157479,190,92,true,-3.2359,0.71873,2013-10-04T01:22:56.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00770,mir-330,1950,microRNA mir-330,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.2,47.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,33,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.61983,-3.10511,1600000,141002,192,94,true,-3.2725,0.7188,2013-10-04T01:23:08.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00771,mir-185,1951,microRNA mir-185,Griffiths-Jones SR,Griffiths-Jones SR,60.0,67.2,48.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,14,35,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.58308,-4.20101,1600000,195704,171,79,true,-3.4485,0.71919,2013-10-04T01:23:11.000Z,2022-05-06T12:43:52.000Z\n",
            "RF00772,mir-328,1287,microRNA mir-328,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.6,53.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,35,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.53789,-5.1611,1600000,216044,165,75,true,-3.3008,0.71906,2013-10-04T01:23:58.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00773,mir-298,1287,microRNA mir-298,Griffiths-Jones SR,Griffiths-Jones SR,80.0,84.6,72.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,13,0,0,Gene; miRNA;,Predicted; RNAalifold,14,0,,\"\",0.64645,-3.41914,1600000,241550,178,85,true,-3.4595,0.71892,2013-10-04T01:26:47.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00774,mir-360,1287,microRNA mir-360,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.6,52.7,This family represents the microRNA (miRNA) precursor mir-360 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.65086,-3.57147,1600000,256732,144,62,true,-3.59,0.71928,2013-10-04T01:26:50.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00775,mir-432,1952,mir-432 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,70.0,78.3,51.0,Family represents the microRNA (miRNA) precursor mir-432 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,12,34,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.6237,-3.47177,1600000,150031,183,87,true,-3.5526,0.71882,2013-10-04T01:26:57.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00776,mir-540,1287,microRNA mir-540,Griffiths-Jones SR,Griffiths-Jones SR,80.0,92.3,50.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.581,-4.29145,1600000,274455,156,67,true,-3.1182,0.72165,2013-10-04T01:26:59.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00777,mir-541,1953,mir-541 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,44.0,44.3,38.7,This family represents the microRNA (miRNA) precursor mir-541 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,31,0,0,Gene; miRNA;,Predicted; RNAalifold,32,0,,\"\",0.67109,-3.10341,1600000,236369,177,84,true,-3.5956,0.71894,2013-10-04T01:27:05.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00778,MIR473,1287,microRNA MIR473,Wilkinson A,miRBase; Wilkinson A,52.0,52.0,51.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,48,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,27,0,,\"\",0.6157,-4.81229,1600000,197378,215,95,true,-2.9037,0.71879,2013-10-04T01:27:08.000Z,2022-05-06T13:09:57.000Z\n",
            "RF00779,MIR474,2406,microRNA MIR474,Wilkinson A,miRBase; Wilkinson A,48.0,48.1,47.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,34,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.65959,-2.07697,1600000,168872,293,107,true,-3.7826,0.7187,2013-10-04T01:27:12.000Z,2022-05-06T12:43:54.000Z\n",
            "RF00780,MIR477,1287,microRNA MIR477,Wilkinson A,miRBase; Wilkinson A,65.0,65.2,63.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,13,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,9,0,,\"\",0.66077,-3.26665,1600000,203252,200,99,true,-2.9761,0.7187,2013-10-04T01:27:18.000Z,2022-05-06T13:09:57.000Z\n",
            "RF00781,mir-505,1955,microRNA mir-505,Griffiths-Jones SR,Griffiths-Jones SR,60.0,69.6,50.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,35,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.62738,-3.80293,1600000,169754,178,83,true,-3.2703,0.71898,2013-10-04T01:27:23.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00782,MIR480,1287,microRNA MIR480,Wilkinson A,miRBase; Wilkinson A,35.0,35.0,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33 -Z 549862.597050 CM SEQDB,69,1131,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,37,0,,\"\",0.50448,-3.25373,1600000,276711,247,140,true,-3.4041,0.71796,2013-10-04T01:27:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00783,mir-484,1956,microRNA mir-484,Griffiths-Jones SR,Griffiths-Jones SR,40.0,42.2,36.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,35,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.63,-4.0294,1600000,226527,152,67,true,-3.2032,0.7192,2013-10-04T01:27:29.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00784,mir-486,1287,microRNA mir-486,Griffiths-Jones SR,Griffiths-Jones SR,69.0,73.5,68.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,56,0,0,Gene; miRNA;,Predicted; RNAalifold,34,0,,\"\",0.65856,-4.42968,1600000,206314,152,68,true,-3.2983,0.71913,2013-10-04T01:27:31.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00785,mir-90,1287,microRNA mir-90,Griffiths-Jones SR,Griffiths-Jones SR,68.0,69.3,48.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,7,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.64584,-3.02472,1600000,178544,174,84,true,-3.8603,0.71891,2013-10-04T01:27:33.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00786,mir-289,1287,mir-289 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,83.6,69.8,This family represents the microRNA (miRNA) precursor mir-289 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,7,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.66508,-2.28079,1600000,176653,201,96,true,-3.2493,0.71873,2013-10-04T01:27:37.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00787,mir-288,1287,mir-288 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,68.3,36.7,This family represents the microRNA (miRNA) precursor mir-288 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,12,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.61756,-3.04339,1600000,204600,195,96,true,-3.0665,0.71873,2013-10-04T01:27:39.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00788,mir-287,1287,microRNA mir-287,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.5,43.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,11,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.65031,-3.39497,1600000,172682,161,73,true,-3.6389,0.71903,2013-10-04T01:27:41.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00789,mir-286,1287,microRNA mir-286,Griffiths-Jones SR,Griffiths-Jones SR,54.0,56.3,52.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,15,64,0,0,Gene; miRNA;,Predicted; RNAalifold,47,0,,\"\",0.65384,-2.22728,1600000,221601,197,101,true,-3.324,0.71855,2013-10-04T01:27:52.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00790,mir-358,1287,mir-358 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,65.0,65.6,58.9,This family represents the microRNA (miRNA) precursor mir-358 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.67789,-2.53245,1600000,170595,214,111,true,-3.1441,0.71867,2013-10-04T01:27:55.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00791,mir-357,1287,microRNA mir-357,Griffiths-Jones SR,Griffiths-Jones SR,49.4,49.4,40.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.61541,-4.34212,1600000,207765,157,72,true,-3.3911,0.71902,2013-02-01T12:06:49.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00792,mir-490,1287,microRNA mir-490,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.3,41.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,19,94,0,0,Gene; miRNA;,Predicted; RNAalifold,98,0,,\"\",0.7021,-2.44323,1600000,154483,193,95,true,-3.4521,0.71867,2013-10-04T01:28:05.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00793,mir-497,1957,microRNA mir-497,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.5,60.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,18,73,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.60347,-4.02475,1600000,162708,177,85,true,-3.7984,0.71886,2013-10-04T01:28:07.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00794,mir-42,1958,microRNA mir-42,Griffiths-Jones SR,Griffiths-Jones SR,70.0,73.4,57.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.67577,-2.76354,1600000,151885,202,102,true,-3.2149,0.71882,2013-10-04T01:28:09.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00795,mir-43,1287,microRNA mir-43,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.8,48.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.60415,-3.3335,1600000,153259,179,87,true,-3.2755,0.71894,2013-10-04T01:28:14.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00796,mir-48,1959,microRNA mir-48,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,69.0,69.6,43.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.64446,-3.59549,1600000,213687,173,84,true,-3.3231,0.71884,2013-10-04T01:28:17.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00797,mir-355,1287,microRNA mir-355,Griffiths-Jones SR,Griffiths-Jones SR,70.3,70.3,51.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.61847,-3.07455,1600000,179311,164,75,true,-3.9505,0.71907,2013-10-04T01:28:20.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00798,mir-49,1287,microRNA mir-49,Griffiths-Jones SR,Griffiths-Jones SR,60.0,64.9,47.7,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.61281,-3.88831,1600000,185569,162,74,true,-3.437,0.71917,2013-10-04T01:28:25.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00799,mir-354,1287,mir-354 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,55.3,45.4,This family represents the microRNA (miRNA) precursor mir-354 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.69648,-2.21865,1600000,204750,214,109,true,-3.3328,0.71859,2013-10-04T01:28:27.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00801,mir-280,1287,microRNA mir-280,Griffiths-Jones SR,Griffiths-Jones SR,70.0,83.5,49.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,11,0,0,Gene; miRNA;,Predicted; RNAalifold,14,0,,\"\",0.59326,-3.16536,1600000,193572,176,83,true,-3.5062,0.71885,2013-10-04T01:28:32.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00802,mir-207,1287,microRNA mir-207,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.8,87.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.57994,-4.61125,1600000,224996,171,78,true,-3.6867,0.71908,2013-10-04T01:29:32.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00803,mir-425,1960,microRNA mir-425,Griffiths-Jones SR,Griffiths-Jones SR,60.0,70.7,48.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,19,69,0,0,Gene; miRNA;,Predicted; RNAalifold,70,0,,\"\",0.69413,-2.14575,1600000,147488,177,85,true,-3.4656,0.71885,2013-10-04T01:29:52.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00804,mir-240,1287,mir-240 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,67.5,67.5,66.6,Family represents the microRNA (miRNA) precursor mir-240 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.65619,-1.31167,1600000,169491,198,99,true,-3.4134,0.71864,2013-10-04T01:29:56.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00805,mir-351,1961,microRNA mir-351,Griffiths-Jones SR,Griffiths-Jones SR,70.0,103.0,59.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.68087,-3.09181,1600000,267770,166,75,true,-3.5456,0.7197,2013-10-04T01:29:58.000Z,2022-05-06T12:43:52.000Z\n",
            "RF00806,mir-350,1962,microRNA mir-350,Griffiths-Jones SR,Griffiths-Jones SR,60.0,62.6,46.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,36,0,0,Gene; miRNA;,Predicted; RNAalifold,34,0,,\"\",0.62981,-2.96107,1600000,185413,182,87,true,-3.5837,0.71895,2013-10-04T01:30:00.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00807,mir-314,1287,microRNA mir-314,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.2,62.1,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,13,0,0,Gene; miRNA;,Predicted; RNAalifold,17,0,,\"\",0.62196,-3.62227,1600000,221098,186,90,true,-3.3922,0.71891,2013-10-04T01:30:02.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00808,mir-86,1287,microRNA mir-86,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.0,57.5,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,38,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.68897,-2.9575,1600000,158216,164,77,true,-3.5812,0.71896,2013-10-04T01:30:06.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00809,mir-241,1963,microRNA mir-241,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.3,42.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.64466,-4.41492,1600000,257986,162,74,true,-3.2854,0.71898,2013-10-04T01:30:09.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00810,mir-85,1287,microRNA mir-85,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.5,59.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.6073,-3.28135,1600000,158315,172,82,true,-3.3058,0.71904,2013-10-04T01:30:10.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00811,mir-84,1964,microRNA mir-84,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.6,49.1,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,29,0,0,Gene; miRNA;,Predicted; RNAalifold,23,0,,\"\",0.70869,-2.88683,1600000,171954,150,75,true,-3.4127,0.71893,2013-10-04T01:30:13.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00812,mir-83,1287,microRNA mir-83,Griffiths-Jones SR,Griffiths-Jones SR,44.0,44.3,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,47,0,0,Gene; miRNA;,Predicted; RNAalifold,44,0,,\"\",0.56989,-3.56857,1600000,132334,172,84,true,-3.5306,0.71888,2013-10-04T01:30:16.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00813,mir-11,1965,mir-11 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,45.0,45.0,44.9,This family represents the microRNA (miRNA) precursor mir-11 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,20,68,0,0,Gene; miRNA;,Predicted; RNAalifold,73,0,,\"\",0.62975,-3.93331,1600000,160076,167,78,true,-3.8684,0.71895,2013-10-04T01:30:23.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00814,mir-316,1287,microRNA mir-316,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.4,79.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,20,11,0,0,Gene; miRNA;,Predicted; RNAalifold,24,0,,\"\",0.67272,-1.45457,1600000,147070,175,91,true,-4.1341,0.71875,2013-10-04T01:30:40.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00815,mir-244,1287,mir-244 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,62.5,62.5,52.4,Family represents the microRNA (miRNA) precursor mir-244 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.66481,-2.38656,1600000,175161,191,94,true,-3.9452,0.71889,2013-10-04T01:30:44.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00816,mir-245,1287,microRNA mir-245,Griffiths-Jones SR,Griffiths-Jones SR,70.0,72.8,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.6472,-2.87091,1600000,162053,180,86,true,-3.2474,0.71875,2013-10-04T01:30:47.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00817,mir-80,1287,microRNA mir-80,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.9,48.7,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.67762,-2.4137,1600000,171672,184,89,true,-3.5795,0.71883,2013-10-04T01:30:53.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00818,mir-318,1287,microRNA mir-318,Griffiths-Jones SR,Griffiths-Jones SR,70.0,74.2,48.0,family represents the microRNA (miRNA) precursor mir-318 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,15,0,0,Gene; miRNA;,Predicted; RNAalifold,18,0,,\"\",0.61912,-3.73731,1600000,251210,155,69,true,-3.6581,0.71915,2013-10-04T01:30:56.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00819,mir-246,1287,microRNA mir-246,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.3,46.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,7,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.62985,-4.3168,1600000,250187,157,72,true,-3.4759,0.71901,2013-02-01T12:06:50.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00820,mir-248,1287,mir-248 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.7,86.5,This family represents the microRNA (miRNA) precursor mir-248 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.73561,-1.57763,1600000,182721,188,93,true,-3.6458,0.71882,2013-10-04T01:31:09.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00821,mir-249,1287,mir-249 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,96.7,57.9,This family represents the microRNA (miRNA) precursor mir-249 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.68544,-2.31474,1600000,138563,194,97,true,-3.5099,0.71877,2013-10-04T01:31:13.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00822,mir-274,1287,microRNA mir-274,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.3,53.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,24,0,0,Gene; miRNA;,Predicted; RNAalifold,27,0,,\"\",0.64963,-2.03397,1600000,141428,211,97,true,-3.4519,0.71881,2013-10-04T01:31:17.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00823,lsy-6,1966,lsy-6 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,61.9,56.6,This family represents the microRNA (miRNA) precursor lsy-6 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.66709,-3.1357,1600000,226455,165,74,true,-3.2067,0.72652,2013-10-04T01:31:22.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00824,mir-50,1287,mir-50 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,43.0,43.1,39.7,Family represents the microRNA (miRNA) precursor mir-50 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,43,0,0,Gene; miRNA;,Predicted; RNAalifold,43,0,,\"\",0.60778,-4.51352,1600000,156357,160,75,true,-3.8183,0.71907,2013-10-04T01:31:27.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00825,mir-344,1967,mir-344 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,52.5,52.5,52.4,Family represents the microRNA (miRNA) precursor mir-344 imported from miRBase [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,18,173,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.6417,-3.87239,1600000,204776,152,71,true,-3.4348,0.71911,2013-10-04T01:31:30.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00826,mir-55,1287,microRNA mir-55,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.2,47.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.61823,-4.22875,1600000,220445,161,74,true,-3.7216,0.71906,2013-10-04T01:31:37.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00827,mir-77,1287,microRNA mir-77,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,83.0,83.1,69.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,7,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.67469,-2.94222,1600000,155800,188,89,true,-3.9132,0.71891,2013-02-01T12:06:52.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00828,mir-75,1287,microRNA mir-75,Griffiths-Jones SR,Griffiths-Jones SR,60.0,62.4,56.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.65791,-2.70502,1600000,180228,185,92,true,-3.7081,0.71885,2013-10-04T01:31:47.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00829,mir-149,1287,microRNA mir-149,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.7,48.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,35,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.53275,-4.65229,1600000,188650,181,86,true,-3.6112,0.71885,2013-10-04T01:31:49.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00830,mir-74,1287,mir-74 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,57.0,57.0,53.2,Family represents the microRNA (miRNA) precursor mir-74 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,11,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.70759,-1.86706,1600000,159306,166,80,true,-3.5843,0.71905,2013-10-04T01:31:55.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00831,mir-73,1287,mir-73 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,86.0,92.2,85.4,Family represents the microRNA (miRNA) precursor mir-73 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,6,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.69248,-3.1343,1600000,142077,187,94,true,-3.5032,0.71872,2013-10-04T01:31:57.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00832,mir-71,1968,microRNA mir-71,Griffiths-Jones SR,Griffiths-Jones SR,50.0,50.2,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,36,149,0,0,Gene; miRNA;,Predicted; RNAalifold,136,0,,\"\",0.61006,-3.83795,1600000,134837,174,73,true,-3.7542,0.71906,2013-10-04T01:31:59.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00833,mir-70,1287,mir-70 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,90.0,91.4,57.7,Family represents the microRNA (miRNA) precursor mir-70 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63452,-2.26951,1600000,163679,172,79,true,-3.0381,0.7294,2013-10-04T01:32:14.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00834,mir-268,1287,microRNA mir-268,Griffiths-Jones SR,Griffiths-Jones SR,47.0,62.6,46.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.62861,-1.48292,1600000,157556,180,89,true,-3.6401,0.71981,2013-10-04T01:32:16.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00835,mir-58,1969,microRNA mir-58,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.4,69.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,7,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.6313,-3.58688,1600000,245975,181,77,true,-3.3278,0.71909,2013-10-04T01:32:32.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00836,mir-250,1287,microRNA mir-250,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.2,56.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,14,0,0,Gene; miRNA;,Predicted; RNAalifold,13,0,,\"\",0.72003,-2.29011,1600000,189677,167,81,true,-3.7608,0.71896,2013-10-04T01:32:35.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00837,mir-251,1287,microRNA mir-251 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,45.0,45.6,44.9,Family represents the microRNA (miRNA) precursor mir-251 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.66371,-2.96063,1600000,254541,138,61,true,-3.6071,0.72006,2013-10-04T01:32:37.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00838,mir-252,1287,mir-252 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,45.0,45.0,44.9,Family represents the microRNA (miRNA) precursor mir-252 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,44,203,0,0,Gene; miRNA;,Predicted; RNAalifold,167,0,,\"\",0.61257,-3.40931,1600000,174036,181,77,true,-3.9578,0.71884,2013-10-04T01:32:39.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00839,mir-452,1970,microRNA mir-452,Griffiths-Jones SR,Griffiths-Jones SR,70.0,74.4,61.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,35,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.61831,-3.24878,1600000,214546,180,85,true,-3.2646,0.71894,2013-10-04T01:32:46.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00840,mir-374,1971,microRNA mir-374,Wilkinson A,miRBase; Wilkinson A,69.0,69.1,68.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,56,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,37,0,,\"\",0.6638,-2.13194,1600000,222357,156,71,true,-3.8583,0.71921,2013-10-04T01:32:55.000Z,2022-05-06T13:10:04.000Z\n",
            "RF00841,mir-384,1972,mir-384 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,72.0,72.1,52.2,Family represents the microRNA (miRNA) precursor mir-384 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,30,0,0,Gene; miRNA;,Predicted; RNAalifold,29,0,,\"\",0.57362,-2.95668,1600000,170884,184,88,true,-3.3124,0.71872,2013-10-04T01:32:58.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00842,MIR403,2407,microRNA MIR403,Wilkinson A,miRBase; Wilkinson A,70.0,70.0,69.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,56,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,27,0,,\"\",0.63074,-2.8758,1600000,145509,324,115,true,-3.1224,0.71877,2013-10-04T01:33:11.000Z,2021-11-19T11:53:19.000Z\n",
            "RF00843,mir-228,1287,microRNA mir-228,Griffiths-Jones SR,Griffiths-Jones SR,45.0,46.5,44.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,23,0,0,Gene; miRNA;,Predicted; RNAalifold,19,0,,\"\",0.68518,-2.52208,1600000,123147,176,85,true,-3.7448,0.71894,2013-02-01T12:06:54.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00844,mir-67,1974,microRNA mir-67,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.5,51.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,33,102,0,0,Gene; miRNA;,Predicted; RNAalifold,101,0,,\"\",0.65778,-3.15319,1600000,103901,169,84,true,-4.3257,0.71884,2013-10-04T01:33:16.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00845,MIR158,1287,MIR158 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.2,66.3,This family represents the microRNA (miRNA) precursor MIR158 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,14,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.61693,-2.65402,1600000,142726,198,100,true,-3.5561,0.71879,2013-10-04T01:33:19.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00846,mir-64,1287,microRNA mir-64,Wilkinson A,miRBase; Wilkinson A,45.0,48.9,44.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,27,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,5,0,,\"\",0.64413,-4.96603,1600000,192388,205,110,true,-3.4984,0.71842,2013-10-04T01:33:23.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00847,mir-62,1975,mir-62 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,85.3,41.0,This family represents the microRNA (miRNA) precursor mir-62 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.59107,-4.86895,1600000,321328,140,58,true,-3.9803,0.7194,2013-10-04T01:33:25.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00848,mir-61,1976,microRNA mir-61,Griffiths-Jones SR,Griffiths-Jones SR,77.0,77.7,60.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,5,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.58602,-4.02581,1600000,163463,175,85,true,-3.6783,0.71906,2013-10-04T01:33:35.000Z,2022-10-29T17:42:07.000Z\n",
            "RF00849,mir-60,1287,microRNA mir-60,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.1,65.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,10,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.64278,-3.02496,1600000,212553,159,72,true,-3.2296,0.71925,2013-10-04T01:33:38.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00850,mir-259,1287,microRNA mir-259,Griffiths-Jones SR,Griffiths-Jones SR,50.0,68.6,47.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.65669,-3.0075,1600000,202037,167,78,true,-3.4946,0.71903,2013-10-04T01:33:43.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00851,mir-230,1287,mir-230 precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,72.7,92.6,72.6,Family represents the microRNA (miRNA) precursor mir-230 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.68855,-2.27187,1600000,191070,173,82,true,-3.5052,0.719,2013-10-04T01:33:46.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00852,mir-231,1287,microRNA mir-231,Griffiths-Jones SR,Griffiths-Jones SR,50.0,69.5,49.1,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.61419,-3.46501,1600000,176913,170,80,true,-3.5337,0.71903,2013-10-04T01:33:51.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00853,mir-304,1287,microRNA mir-304,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.2,87.7,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,8,0,0,Gene; miRNA;,Predicted; RNAalifold,13,0,,\"\",0.66177,-2.81345,1600000,226234,184,88,true,-3.3195,0.71884,2013-10-04T01:33:56.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00854,mir-5,1977,microRNA mir-5,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.5,71.0,Imported from miRBase.,mir-4,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,15,0,0,Gene; miRNA;,Predicted; RNAalifold,18,0,,\"\",0.6168,-3.62068,1600000,221055,157,70,true,-3.5449,0.71916,2013-10-04T01:34:03.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00855,mir-254,1287,mir-254 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,70.0,78.8,66.6,Family represents the microRNA (miRNA) precursor mir-254 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.66213,-2.82254,1600000,233747,145,63,true,-3.9462,0.71926,2013-02-01T12:06:55.000Z,2023-11-06T16:44:05.000Z\n",
            "RF00856,mir-232,1287,microRNA mir-232,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.2,50.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.65889,-2.91659,1600000,202865,171,81,true,-3.6344,0.71895,2013-10-04T01:34:11.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00857,mir-233,1287,microRNA mir-233,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.1,48.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.60728,-3.60913,1600000,189760,168,78,true,-3.4914,0.71904,2013-10-04T01:34:15.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00858,mir-306,1287,microRNA mir-306,Griffiths-Jones SR,Griffiths-Jones SR,52.0,52.0,51.7,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,13,0,0,Gene; miRNA;,Predicted; RNAalifold,18,0,,\"\",0.6922,-2.8759,1600000,193646,172,81,true,-3.8899,0.71901,2013-10-04T01:34:18.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00859,mir-234,1287,microRNA mir-234,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.2,57.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,30,0,0,Gene; miRNA;,Predicted; RNAalifold,29,0,,\"\",0.55943,-4.69079,1600000,167459,150,70,true,-3.1185,0.71911,2013-10-04T01:34:21.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00861,mir-488,1978,microRNA mir-488,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.6,82.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,34,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.70274,-2.13211,1600000,213152,177,83,true,-3.121,0.71892,2013-10-04T01:34:27.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00862,mir-491,1979,microRNA mir-491,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.9,57.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,34,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.61986,-3.41461,1600000,200722,179,84,true,-3.4398,0.71898,2013-10-04T01:34:30.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00863,mir-BART17,1287,microRNA mir-BART17,Wilkinson A,miRBase; Wilkinson A,100.0,112.8,50.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.63254,-3.98328,1600000,184564,205,104,true,-2.8296,0.71858,2013-10-04T01:34:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00864,mir-BART20,1287,mir-BART20 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,99.5,65.1,This family represents the microRNA (miRNA) precursor mir-BART20 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62129,-3.56229,1600000,206672,174,81,true,-3.2599,0.72208,2013-10-04T01:34:41.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00865,MIR169_5,1287,microRNA MIR169_5,Wilkinson A,miRBase; Wilkinson A,44.0,44.0,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,906,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,73,0,,\"\",0.57817,-5.56469,1600000,193162,273,129,true,-2.7591,0.71824,2013-10-04T01:34:44.000Z,2022-05-06T13:09:57.000Z\n",
            "RF00866,mir-BART3,1287,mir-BART3 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,74.4,59.8,This family represents the microRNA (miRNA) precursor mir-BART3 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.57294,-4.44978,1600000,258727,166,77,true,-3.2151,0.71912,2013-10-04T01:34:50.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00867,mir-BART5,1287,mir-BART5 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,85.1,47.6,This family represents the microRNA (miRNA) precursor mir-BART5 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.57708,-3.78934,1600000,192347,185,89,true,-3.4677,0.71915,2013-10-04T01:34:55.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00868,mir-BART15,1287,mir-BART15 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,81.9,45.8,This family represents the microRNA (miRNA) precursor mir-BART15 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63417,-3.17497,1600000,225996,172,79,true,-3.248,0.72276,2013-10-04T01:34:58.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00869,mir-BART7,1287,microRNA mir-BART7,Griffiths-Jones SR,Griffiths-Jones SR,70.0,92.6,61.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62839,-3.14489,1600000,231327,178,84,true,-3.1423,0.72012,2013-10-04T01:35:00.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00870,mir-423,1980,microRNA mir-423,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.7,49.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,36,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.6248,-3.59924,1600000,188355,175,81,true,-3.1863,0.71899,2013-10-04T01:35:03.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00872,mir-652,1981,mir-652 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,73.0,73.0,62.2,Family represents the microRNA (miRNA) precursor mir-652 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,14,34,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.6375,-3.395,1600000,182775,176,82,true,-3.6981,0.71902,2013-10-04T01:35:18.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00873,mir-550,1982,microRNA mir-550,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.8,79.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,26,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.65252,-3.05776,1600000,216833,177,85,true,-3.0647,0.71899,2013-10-04T01:35:27.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00874,mir-BART12,1287,microRNA mir-BART12,Griffiths-Jones SR,Griffiths-Jones SR,70.0,88.8,45.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63486,-2.95177,1600000,211122,178,84,true,-2.9427,0.72322,2013-10-04T01:35:29.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00875,mir-692,1287,microRNA mir-692,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.2,97.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,26,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.60083,-3.83929,1600000,217973,215,109,true,-2.9722,0.71869,2013-10-04T01:35:32.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00876,mir-684,1287,microRNA mir-684,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.8,99.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,26,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.66261,-2.35342,1600000,228421,181,86,true,-3.1487,0.71887,2013-10-04T01:35:58.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00877,mir-592,1983,microRNA mir-592,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.4,61.1,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,37,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.67806,-2.20473,1600000,178697,185,88,true,-3.4066,0.71899,2013-10-04T01:36:07.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00878,mir-456,1287,microRNA mir-456,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.2,35.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,69,0,0,Gene; miRNA;,Predicted; RNAalifold,68,0,,\"\",0.63874,-3.80597,1600000,181652,161,73,true,-3.566,0.71904,2013-10-04T01:36:23.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00879,mir-615,1984,microRNA mir-615,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.7,67.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,30,0,0,Gene; miRNA;,Predicted; RNAalifold,30,0,,\"\",0.52402,-5.27509,1600000,218188,181,85,true,-3.1478,0.71905,2013-10-04T01:36:31.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00883,MIR820,1287,MIR820 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,196.0,196.6,195.2,Family represents the microRNA (miRNA) precursor MIR820 imported from miRBase,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,26,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.49055,-8.49096,1600000,258959,322,191,true,-3.1509,0.71342,2013-10-04T01:36:43.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00884,MIR815,1287,microRNA MIR815,Wilkinson A,miRBase; Wilkinson A,63.0,64.2,61.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,11,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,7,0,,\"\",0.61784,-3.51319,1600000,231452,173,81,true,-3.3176,0.71898,2013-10-04T01:36:49.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00885,MIR821,1287,microRNA MIR821,Griffiths-Jones SR,Griffiths-Jones SR,249.0,249.0,248.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,879,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.51254,-10.383,1600000,375993,419,275,true,-3.6742,0.7072,2013-10-04T01:37:02.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00886,MIR807,1287,microRNA MIR807,Wilkinson A,miRBase; Wilkinson A,113.0,113.2,112.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,30,1637,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,9,0,,\"\",0.56129,-7.42697,1600000,284639,301,176,true,-3.5319,0.71505,2013-10-04T01:37:17.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00887,mir-802,1985,mir-802 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,86.0,86.4,84.3,Family represents the microRNA (miRNA) precursor mir-802 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,37,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.6528,-1.96053,1600000,140618,191,94,true,-3.4085,0.71874,2013-10-04T01:37:23.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00888,mir-770,1287,mir-770 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,83.0,83.3,80.0,Family represents the microRNA (miRNA) precursor mir-770 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,19,0,0,Gene; miRNA;,Predicted; RNAalifold,19,0,,\"\",0.54939,-4.31894,1600000,170525,176,83,true,-2.8331,0.71906,2013-02-01T12:06:57.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00890,mir-668,1287,mir-668 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,87.0,87.9,73.5,Family represents the microRNA (miRNA) precursor mir-668 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,19,0,0,Gene; miRNA;,Predicted; RNAalifold,22,0,,\"\",0.57799,-4.03781,1600000,250916,151,66,true,-3.4884,0.71928,2013-10-04T01:37:40.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00891,mir-671,1292,microRNA mir-671,Griffiths-Jones SR,Griffiths-Jones SR,65.0,65.9,64.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,37,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.5476,-4.68473,1600000,247762,179,84,true,-3.4092,0.71899,2013-02-01T12:06:58.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00892,mir-551,1287,microRNA mir-551,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.3,49.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,29,183,0,0,Gene; miRNA;,Predicted; RNAalifold,112,0,,\"\",0.67828,-3.25227,1600000,218969,157,74,true,-3.6716,0.71913,2013-10-04T01:37:59.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00893,MIR854,1986,microRNA MIR854,Wilkinson A,miRBase; Wilkinson A,50.0,50.1,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,167,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.59871,-11.1194,1600000,482496,360,221,true,-3.1334,0.71044,2013-10-04T01:38:02.000Z,2022-05-06T12:43:52.000Z\n",
            "RF00894,mir-790,1287,microRNA mir-790,Griffiths-Jones SR,Griffiths-Jones SR,63.0,69.2,59.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,8,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.60994,-4.2262,1600000,235410,155,70,true,-3.8892,0.71913,2013-10-04T01:38:08.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00895,mir-786,1987,mir-786 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,93.0,93.9,92.9,Family represents the microRNA (miRNA) precursor mir-786 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.68151,-2.45211,1600000,219906,182,90,true,-3.2289,0.71887,2013-10-04T01:38:10.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00896,mir-787,1287,mir-787 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,57.0,57.7,51.2,Family represents the microRNA (miRNA) precursor mir-787 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62025,-3.71613,1600000,230385,162,72,true,-3.2676,0.7286,2013-10-04T01:38:15.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00897,mir-675,1988,microRNA mir-675,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.6,80.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,25,0,0,Gene; miRNA;,Predicted; RNAalifold,26,0,,\"\",0.51655,-5.46657,1600000,290067,163,73,true,-3.6043,0.71911,2013-10-04T01:38:17.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00898,mir-242,1287,microRNA mir-242,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.8,48.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.64473,-2.59779,1600000,176476,166,77,true,-3.4851,0.71897,2013-10-04T01:38:22.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00899,mir-235,1287,mir-235 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,75.0,75.9,67.0,Family represents the microRNA (miRNA) precursor mir-235 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.58327,-4.0402,1600000,180157,167,77,true,-3.0729,0.71905,2013-10-04T01:38:27.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00900,mir-255,1287,microRNA mir-255,Griffiths-Jones SR,Griffiths-Jones SR,50.0,50.1,48.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,18,0,0,Gene; miRNA;,Predicted; RNAalifold,16,0,,\"\",0.58189,-4.00061,1600000,266991,139,59,true,-3.4143,0.71947,2013-10-04T01:38:32.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00901,MIR845_1,1287,microRNA MIR845_1,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.0,75.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,5,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.68349,-2.38383,1600000,221918,179,86,true,-3.3303,0.71892,2013-10-04T01:38:34.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00902,mir-791,1287,microRNA mir-791,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.8,51.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.62153,-3.80283,1600000,244434,181,86,true,-3.1435,0.724,2013-10-04T01:38:39.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00903,mir-359,1287,mir-359 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,68.0,68.1,63.2,Family represents the microRNA (miRNA) precursor mir-359 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.64298,-2.2744,1600000,180414,177,83,true,-3.2215,0.72368,2013-10-04T01:38:41.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00904,mir-392,1287,microRNA mir-392,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.8,56.7,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62124,-3.1526,1600000,230955,167,75,true,-3.2908,0.72869,2013-10-04T01:38:44.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00905,mir-789,1287,microRNA mir-789,Griffiths-Jones SR,Griffiths-Jones SR,100.0,104.4,67.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61342,-2.93874,1600000,195122,192,97,true,-3.4155,0.71886,2013-10-04T01:38:46.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00906,MIR1122,1287,microRNA MIR1122,Wilkinson A,miRBase; Wilkinson A,77.0,77.0,76.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.99 -Z 549862.597050 CM SEQDB,12,100454,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,57,0,,\"\",0.59392,-4.21307,1600000,152011,241,135,true,-3.1934,0.71832,2013-10-04T01:38:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00907,mir-941,1287,microRNA mir-941,Griffiths-Jones SR,Griffiths-Jones SR,60.0,84.9,55.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.54815,-4.84226,1600000,253248,163,72,true,-3.2529,0.71979,2013-10-04T01:39:15.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00908,MIR529,1989,microRNA MIR529,Wilkinson A,miRBase; Wilkinson A,75.0,75.8,73.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,32,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,19,0,,\"\",0.6761,-3.69337,1600000,188354,242,131,true,-3.4239,0.71818,2013-10-04T01:39:17.000Z,2022-05-06T12:43:52.000Z\n",
            "RF00909,mir-883,1287,mir-883 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.5,83.3,Family represents the microRNA (miRNA) precursor mir-883 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.63407,-3.15694,1600000,193189,167,77,true,-3.3914,0.71904,2013-10-04T01:39:24.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00910,mir-873,1990,microRNA mir-873,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.7,94.5,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,38,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.69133,-2.69472,1600000,234971,166,77,true,-3.5549,0.71908,2013-10-04T01:39:29.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00911,mir-672,1991,microRNA mir-672,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.6,96.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,12,0,0,Gene; miRNA;,Predicted; RNAalifold,13,0,,\"\",0.63605,-3.27609,1600000,211602,175,81,true,-3.6195,0.71907,2013-10-04T01:39:32.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00912,mir-877,1287,microRNA mir-877,Griffiths-Jones SR,Griffiths-Jones SR,47.0,47.2,46.7,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,36,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.60412,-3.93321,1600000,179275,180,86,true,-3.2643,0.7189,2013-10-04T01:40:17.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00914,mir-674,1287,microRNA mir-674,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.0,59.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.57418,-4.18431,1600000,221760,170,77,true,-2.9904,0.71902,2013-10-04T01:40:22.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00915,mir-760,1287,mir-760 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.2,88.1,Family represents the microRNA (miRNA) precursor mir-760 imported from miRBase [4].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,22,0,0,Gene; miRNA;,Predicted; RNAalifold,24,0,,\"\",0.66286,-3.21552,1600000,211090,174,80,true,-3.0121,0.71896,2013-10-04T01:40:24.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00917,mir-708,1992,mir-708 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,71.0,71.3,56.9,Family represents the microRNA (miRNA) precursor mir-708 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,13,35,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.59434,-4.0173,1600000,187989,184,88,true,-3.4818,0.71881,2013-10-04T01:40:26.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00918,mir-872,1993,mir-872 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,75.0,75.3,58.2,Family represents the microRNA (miRNA) precursor mir-872 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,15,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.63956,-3.12748,1600000,200793,172,81,true,-3.3135,0.71905,2013-10-04T01:40:32.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00919,mir-874,1994,microRNA mir-874,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.6,46.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,36,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.58586,-3.50556,1600000,201172,170,78,true,-3.0728,0.71898,2013-10-04T01:40:35.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00920,MIR444,2408,microRNA MIR444,Griffiths-Jones SR,Griffiths-Jones SR,110.0,111.6,107.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,19,0,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.58528,-5.76699,1600000,195762,217,116,true,-2.8382,0.71838,2013-10-04T01:40:40.000Z,2023-11-06T16:44:07.000Z\n",
            "RF00921,mir-665,1287,microRNA mir-665,Griffiths-Jones SR,Griffiths-Jones SR,52.0,52.2,45.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,33,0,0,Gene; miRNA;,Predicted; RNAalifold,33,0,,\"\",0.60735,-4.5185,1600000,304558,160,72,true,-3.123,0.71924,2013-10-04T01:40:44.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00922,mir-673,1287,microRNA mir-673,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.0,39.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.57263,-4.38598,1600000,219532,178,83,true,-3.2773,0.7189,2013-10-04T01:40:48.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00925,MIR1027,1287,microRNA MIR1027,\"Wilkinson A, Daub J\",miRBase; Daub J,150.0,172.1,80.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold; Daub J,2,0,,\"\",0.62608,-4.50806,1600000,227890,260,142,true,-3.3419,0.71802,2013-10-04T01:40:50.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00926,MIR1151,1287,microRNA MIR1151,Wilkinson A,miRBase; Wilkinson A,90.0,113.9,87.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.49937,-5.26831,1600000,208615,216,110,true,-2.8777,0.71876,2013-10-04T01:41:00.000Z,2022-05-06T12:43:45.000Z\n",
            "RF00927,mir-582,1287,microRNA mir-582,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.7,68.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,35,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.61933,-3.31168,1600000,191397,177,83,true,-3.2383,0.71885,2013-10-04T01:41:05.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00928,mir-590,1996,microRNA mir-590,Griffiths-Jones SR,Griffiths-Jones SR,65.0,65.0,62.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,28,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.61187,-3.40057,1600000,198450,180,86,true,-3.2186,0.71888,2013-10-04T01:41:08.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00929,mir-574,1287,mir-574 microRNA precursor family,\"Griffiths-Jones SR, Wilkinson A\",Griffiths-Jones SR,80.0,80.0,79.9,Family represents the microRNA (miRNA) precursor mir-574 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,2138,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.60322,-4.8533,1600000,175642,173,81,true,-3.7107,0.71895,2013-10-04T01:41:17.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00931,mir-879,1287,microRNA mir-879,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.1,51.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.65398,-3.31843,1600000,265012,165,74,true,-3.6755,0.72028,2013-10-04T01:51:12.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00932,mir-471,1287,microRNA mir-471,Griffiths-Jones SR,Griffiths-Jones SR,80.0,99.4,63.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65721,-3.02653,1600000,251372,155,66,true,-3.4662,0.72332,2013-10-04T01:51:15.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00933,mir-875,1287,microRNA mir-875,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.1,83.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,12,55,0,0,Gene; miRNA;,Predicted; RNAalifold,55,0,,\"\",0.69181,-2.3956,1600000,200961,160,73,true,-3.4384,0.71908,2013-10-04T01:51:17.000Z,2022-05-06T12:43:46.000Z\n",
            "RF00934,mir-463,1287,microRNA mir-463,Griffiths-Jones SR,Griffiths-Jones SR,70.0,93.6,53.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62919,-3.51025,1600000,261319,155,66,true,-3.2852,0.72634,2013-02-01T12:07:00.000Z,2022-05-06T12:43:46.000Z\n",
            "RF00935,mir-876,1287,microRNA mir-876,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.1,66.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,37,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.62611,-3.48151,1600000,203708,174,81,true,-3.7427,0.71907,2013-10-04T01:51:32.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00936,mir-744,1997,microRNA mir-744,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.1,42.7,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,36,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.59469,-3.57238,1600000,163029,196,96,true,-3.2449,0.7188,2013-10-04T01:51:43.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00937,mir-653,1287,microRNA mir-653,Griffiths-Jones SR,Griffiths-Jones SR,77.0,77.0,56.5,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,32,0,0,Gene; miRNA;,Predicted; RNAalifold,33,0,,\"\",0.63539,-3.20711,1600000,212505,168,78,true,-3.1768,0.71891,2013-10-04T01:51:46.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00939,mir-504,1998,microRNA mir-504,Griffiths-Jones SR,Griffiths-Jones SR,57.0,57.8,49.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,35,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.63503,-3.39455,1600000,186154,169,78,true,-3.664,0.71911,2013-10-04T01:51:49.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00940,mir-327,1287,microRNA mir-327,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.0,70.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6659,-3.30785,1600000,286315,149,62,true,-2.7907,0.7285,2013-10-04T01:51:51.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00941,mir-434,1999,microRNA mir-434,Griffiths-Jones SR,Griffiths-Jones SR,100.0,105.3,58.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60467,-3.68187,1600000,222695,175,81,true,-3.4876,0.71952,2013-10-04T01:51:54.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00942,mir-1224,1287,microRNA mir-1224,Griffiths-Jones SR,Griffiths-Jones SR,105.0,105.2,104.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,16,0,0,Gene; miRNA;,Predicted; RNAalifold,22,0,,\"\",0.57553,-4.28218,1600000,250620,180,86,true,-3.1582,0.719,2013-10-04T01:52:01.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00943,MIR824,2000,microRNA MIR824,Wilkinson A,miRBase; Wilkinson A,100.0,152.8,81.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,18,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,7,0,,\"\",0.50764,-13.88763,1600000,751910,851,654,true,-4.6536,0.69896,2013-10-04T01:52:22.000Z,2022-05-06T12:43:52.000Z\n",
            "RF00945,mir-1226,1287,mir-1226 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,95.0,95.6,54.8,Family represents the microRNA (miRNA) precursor mir-1226 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.58885,-3.88896,1600000,189609,167,75,true,-3.1226,0.71915,2013-10-04T01:52:24.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00946,mir-1225,1287,microRNA mir-1225,Griffiths-Jones SR,Griffiths-Jones SR,91.5,91.5,90.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,33,0,0,Gene; miRNA;,Predicted; RNAalifold,34,0,,\"\",0.50827,-4.81628,1600000,203656,190,90,true,-2.8555,0.71892,2013-10-04T01:52:27.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00947,mir-929,1287,microRNA mir-929,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.5,49.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,18,76,0,0,Gene; miRNA;,Predicted; RNAalifold,78,0,,\"\",0.68904,-2.41421,1600000,165750,158,74,true,-3.825,0.71899,2013-10-04T01:52:29.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00948,mir-996,1287,mir-996 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,40.3,40.4,40.0,Family represents the microRNA (miRNA) precursor mir-996 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,53,0,0,Gene; miRNA;,Predicted; RNAalifold,52,0,,\"\",0.62904,-4.08627,1600000,222947,157,70,true,-3.7176,0.71938,2013-10-04T01:52:39.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00949,mir-983,1287,microRNA mir-983,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.1,56.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.52648,-3.0831,1600000,165085,186,90,true,-3.1606,0.71871,2013-10-04T01:52:44.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00950,mir-927,1287,microRNA mir-927,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.4,59.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,94,0,0,Gene; miRNA;,Predicted; RNAalifold,79,0,,\"\",0.56725,-3.02893,1600000,112020,172,86,true,-3.962,0.71895,2013-10-04T01:52:47.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00951,mir-1302,1287,microRNA mir-1302,Wilkinson A,miRBase; Wilkinson A,62.0,62.0,61.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,24,3572,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,45,0,,\"\",0.58066,-2.48156,1600000,323012,122,52,true,-3.7848,0.71933,2013-10-04T01:52:53.000Z,2022-05-06T12:43:46.000Z\n",
            "RF00952,mir-650,2001,microRNA mir-650,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.7,99.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,47,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.64305,-2.77205,1600000,177910,188,91,true,-2.7408,0.71874,2013-10-04T01:53:10.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00953,mir-1497,1287,microRNA mir-1497,Wilkinson A,miRBase; Wilkinson A,80.0,99.5,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.66231,-3.85155,1600000,246557,165,74,true,-3.0701,0.7239,2013-10-04T01:53:16.000Z,2021-11-19T11:53:18.000Z\n",
            "RF00954,MIR1446,1287,MIR1446 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,70.0,75.8,68.5,Family represents the microRNA (miRNA) precursor MIR1446 imported from miRBase,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,13,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.67771,-2.5407,1600000,200263,209,105,true,-3.3252,0.71871,2013-10-04T01:53:18.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00955,mir-1829,1287,microRNA mir-1829,Griffiths-Jones SR,Griffiths-Jones SR,60.0,62.7,43.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63365,-3.12595,1600000,277444,138,58,true,-3.6989,0.71932,2013-10-04T01:53:27.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00956,MIR1444,1287,microRNA MIR1444,Wilkinson A,miRBase; Wilkinson A,80.0,90.5,74.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.62285,-2.94758,1600000,211172,178,85,true,-3.579,0.71886,2013-10-04T01:53:30.000Z,2022-05-06T12:43:46.000Z\n",
            "RF00957,mir-663,2002,mir-663 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,75.0,75.3,72.0,Family represents the microRNA (miRNA) precursor mir-663 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,42,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.55068,-4.1174,1600000,195942,187,90,true,-3.0814,0.71886,2013-10-04T01:53:32.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00958,mir-498,2003,microRNA mir-498,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.4,75.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.60716,-3.93475,1600000,239620,179,85,true,-3.4319,0.71902,2013-10-04T01:53:38.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00959,mir-612,2004,mir-612 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,100.0,101.0,93.3,Family represents the microRNA (miRNA) precursor mir-612 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.60129,-3.39496,1600000,210179,182,86,true,-3.3767,0.71891,2013-10-04T01:53:42.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00960,mir-661,2005,microRNA mir-661,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.8,63.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.53481,-4.73877,1600000,253311,215,84,true,-3.198,0.71891,2013-10-04T01:53:46.000Z,2022-05-06T12:43:52.000Z\n",
            "RF00961,mir-581,1287,microRNA mir-581,Griffiths-Jones SR,Griffiths-Jones SR,100.0,106.5,69.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.6258,-3.41902,1600000,235576,186,89,true,-3.2717,0.71887,2013-10-04T01:53:49.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00962,mir-586,1287,microRNA mir-586,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.4,71.7,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.67189,-2.44829,1600000,192990,183,87,true,-3.4335,0.71888,2013-10-04T01:53:53.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00963,mir-642,1287,microRNA mir-642,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.0,87.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.54617,-4.92503,1600000,207811,176,83,true,-3.3645,0.7191,2013-10-04T01:53:56.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00964,mir-938,2006,microRNA mir-938,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.2,50.6,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.62905,-3.6559,1600000,225307,174,81,true,-3.4398,0.71899,2013-10-04T01:54:02.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00965,mir-549,2007,microRNA mir-549,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.9,86.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.6499,-2.60295,1600000,241165,161,72,true,-3.5692,0.71925,2013-10-04T01:54:04.000Z,2022-05-06T12:43:52.000Z\n",
            "RF00966,mir-676,1287,mir-676 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,61.0,69.3,60.5,This family represents the microRNA (miRNA) precursor mir-676 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,30,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.68412,-2.3324,1600000,204977,161,73,true,-3.712,0.71904,2013-10-04T01:54:10.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00967,mir-281,2008,mir-281 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,96.7,50.4,This family represents the microRNA (miRNA) precursor mir-281 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6154,-3.45982,1600000,164537,181,86,true,-3.1394,0.72139,2013-10-04T01:54:12.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00968,mir-626,1287,mir-626 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,91.6,55.0,This family represents the microRNA (miRNA) precursor mir-626 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66223,-2.55052,1600000,223532,178,84,true,-3.7594,0.71891,2013-10-04T01:54:15.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00969,mir-556,1287,microRNA mir-556,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.2,83.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.58824,-3.67682,1600000,191532,184,88,true,-3.6326,0.7189,2013-02-01T12:07:02.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00970,mir-648,1287,mir-648 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.2,34.7,This family represents the microRNA (miRNA) precursor mir-648 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.64415,-3.13813,1600000,197672,179,84,true,-3.0669,0.71895,2013-10-04T01:54:22.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00971,mir-578,1287,mir-578 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.8,88.2,This family represents the microRNA (miRNA) precursor mir-578 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.59231,-3.55002,1600000,239755,179,84,true,-3.611,0.71892,2013-10-04T01:54:25.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00972,mir-651,1287,mir-651 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,106.0,106.6,104.7,Family represents the microRNA (miRNA) precursor mir-651 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.63664,-2.89821,1600000,202599,182,86,true,-3.1911,0.71886,2013-10-04T01:54:30.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00973,mir-597,1287,mir-597 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,100.0,103.5,84.7,Family represents the microRNA (miRNA) precursor mir-597 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61897,-3.24161,1600000,187314,198,97,true,-3.3894,0.71867,2013-10-04T01:54:39.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00974,mir-607,1287,microRNA mir-607,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.2,90.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.57537,-4.509,1600000,230120,189,91,true,-3.3797,0.71882,2013-10-04T01:54:43.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00975,MIR845_2,1287,microRNA MIR845_2,Wilkinson A,miRBase; Wilkinson A,80.0,80.5,78.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,86,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.58222,-4.60246,1600000,284502,244,131,true,-3.0819,0.71819,2013-10-04T01:54:45.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00976,mir-583,1287,microRNA mir-583,Griffiths-Jones SR,Griffiths-Jones SR,80.0,86.6,77.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.63862,-3.68227,1600000,280579,147,62,true,-3.4895,0.71947,2013-10-04T01:54:48.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00977,mir-600,1287,microRNA mir-600,Griffiths-Jones SR,Griffiths-Jones SR,80.0,85.7,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.61439,-3.63221,1600000,232029,173,80,true,-3.1944,0.71887,2013-10-04T01:54:51.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00978,mir-638,2009,microRNA mir-638,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.7,82.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.49075,-4.96236,1600000,228138,193,93,true,-3.0268,0.71883,2013-10-04T01:54:58.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00979,mir-553,1287,microRNA mir-553,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.4,51.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61855,-3.31122,1600000,270003,146,62,true,-3.7316,0.71927,2013-10-04T01:55:00.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00980,mir-643,1287,microRNA mir-643,Griffiths-Jones SR,Griffiths-Jones SR,65.0,65.0,60.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.67674,-2.23941,1600000,198457,180,85,true,-3.5953,0.71901,2013-10-04T01:55:03.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00981,mir-939,2010,microRNA mir-939,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.2,55.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.56053,-4.27982,1600000,214001,176,82,true,-3.1234,0.71914,2013-10-04T01:55:06.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00983,mir-662,1287,microRNA mir-662,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.1,79.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.58443,-3.88652,1600000,222398,177,82,true,-3.2709,0.71899,2013-10-04T01:55:18.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00984,mir-576,1287,microRNA mir-576,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.7,86.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66817,-2.43087,1600000,165431,188,90,true,-3.334,0.71881,2013-10-04T01:55:21.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00985,mir-640,1287,microRNA mir-640,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.0,68.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.64523,-2.98464,1600000,208285,186,89,true,-3.4583,0.71889,2013-10-04T01:55:27.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00986,mir-920,1287,mir-920 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,92.5,61.1,This family represents the microRNA (miRNA) precursor mir-920 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.65405,-3.18545,1600000,223868,164,74,true,-3.0369,0.71897,2013-10-04T01:55:31.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00987,mir-589,2011,microRNA mir-589,Griffiths-Jones SR,Griffiths-Jones SR,54.8,54.8,54.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,12,0,0,Gene; miRNA;,Predicted; RNAalifold,14,0,,\"\",0.59343,-3.27376,1600000,179316,195,96,true,-3.3112,0.71869,2013-10-04T01:55:33.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00988,mir-657,2012,microRNA mir-657,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.2,45.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.6095,-3.41958,1600000,173914,182,86,true,-3.3748,0.71896,2013-10-04T01:55:41.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00989,mir-492,2013,microRNA mir-492,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.2,74.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,47,0,0,Gene; miRNA;,Predicted; RNAalifold,16,0,,\"\",0.71354,-2.81082,1600000,171042,165,75,true,-3.691,0.71898,2013-10-04T01:55:44.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00990,mir-552,2014,microRNA mir-552,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.1,73.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.62512,-3.54857,1600000,238768,177,83,true,-3.316,0.71902,2013-10-04T01:55:56.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00991,mir-599,1287,microRNA mir-599,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.8,93.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,34,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.67316,-2.20053,1600000,145516,183,88,true,-3.717,0.71887,2013-10-04T01:56:02.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00992,mir-593,1287,microRNA mir-593,Griffiths-Jones SR,Griffiths-Jones SR,100.0,116.5,81.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.6684,-2.70657,1600000,161089,202,100,true,-3.1687,0.71871,2013-10-04T01:56:05.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00993,mir-1473,1287,microRNA mir-1473,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.2,48.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63211,-3.90683,1600000,246216,173,80,true,-3.0839,0.72331,2013-10-04T01:56:09.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00994,mir-1255,1287,mir-1255 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.0,62.9,This family represents the microRNA (miRNA) precursor mir-1255 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,24,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.65186,-3.32463,1600000,254271,391,67,true,-3.8725,0.7194,2013-10-04T01:56:11.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00995,mir-616,2015,microRNA mir-616,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.9,86.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66658,-2.6961,1600000,195648,189,91,true,-3.4466,0.7189,2013-10-04T01:56:33.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00996,mir-631,1287,microRNA mir-631,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.2,53.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,23,0,0,Gene; miRNA;,Predicted; RNAalifold,23,0,,\"\",0.58413,-3.62051,1600000,260309,162,73,true,-3.6277,0.71924,2013-10-04T01:56:38.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00997,mir-942,1287,microRNA mir-942,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.4,77.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.6137,-3.59758,1600000,226680,182,86,true,-3.5101,0.71888,2013-10-04T01:56:45.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00998,mir-562,2016,mir-562 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.7,61.2,This family represents the microRNA (miRNA) precursor mir-562 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.68116,-2.19892,1600000,211948,184,89,true,-3.2421,0.71894,2013-10-04T01:56:48.000Z,2023-11-06T16:44:06.000Z\n",
            "RF00999,mir-924,1287,microRNA mir-924,Griffiths-Jones SR,Griffiths-Jones SR,70.0,71.7,49.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.65849,-3.53325,1600000,288784,130,51,true,-3.7916,0.71955,2013-10-04T01:59:07.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01000,mir-580,2017,microRNA mir-580,Griffiths-Jones SR,Griffiths-Jones SR,83.0,100.8,82.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.647,-2.64324,1600000,199134,192,93,true,-3.354,0.71883,2013-10-04T01:59:10.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01001,mir-609,1287,microRNA mir-609,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.3,75.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.5918,-4.14436,1600000,200065,190,92,true,-3.3115,0.71885,2013-10-04T01:59:14.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01002,mir-936,1287,microRNA mir-936,Griffiths-Jones SR,Griffiths-Jones SR,110.0,116.3,102.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.59818,-3.78039,1600000,172710,199,98,true,-3.2762,0.71866,2013-10-04T01:59:18.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01003,mir-563,1287,microRNA mir-563,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.9,67.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,11,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.63557,-3.61913,1600000,285962,157,69,true,-3.6815,0.71924,2013-10-04T01:59:21.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01004,mir-557,1287,microRNA mir-557,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.3,87.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.716,-2.33325,1600000,226308,157,69,true,-3.7447,0.71912,2013-10-04T01:59:26.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01005,MIR530,1287,microRNA MIR530,Wilkinson A,miRBase; Wilkinson A,85.0,85.2,84.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,29,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,19,0,,\"\",0.55827,-6.46343,1600000,262511,295,172,true,-3.0321,0.71548,2013-10-04T01:59:29.000Z,2022-05-06T13:09:57.000Z\n",
            "RF01006,mir-601,2018,microRNA mir-601,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.0,96.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.70574,-2.56893,1600000,236002,160,71,true,-3.8186,0.71914,2013-10-04T01:59:48.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01007,mir-624,2019,microRNA mir-624,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.4,86.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.62592,-3.5469,1600000,226458,182,86,true,-3.6164,0.71908,2013-10-04T01:59:50.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01008,mir-636,2020,microRNA mir-636,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.3,68.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.48485,-5.32322,1600000,220949,197,96,true,-3.0834,0.7188,2013-10-04T01:59:53.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01009,mir-M7,2021,microRNA mir-M7,Griffiths-Jones SR,Griffiths-Jones SR,91.0,91.1,44.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6624,-3.12041,1600000,186057,156,67,true,-2.9234,0.72911,2013-10-04T01:59:55.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01010,mir-632,2022,microRNA mir-632,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.2,52.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,36,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.54612,-4.21535,1600000,191036,177,84,true,-3.438,0.71904,2013-10-04T02:00:01.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01011,mir-605,2023,microRNA mir-605,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.1,77.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.60145,-4.31787,1600000,242261,176,82,true,-3.7051,0.71923,2013-10-04T02:00:06.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01012,mir-628,1287,microRNA mir-628,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.9,66.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,31,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.67726,-2.85195,1600000,213302,168,77,true,-3.0984,0.71905,2013-10-04T02:00:11.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01013,mir-577,1287,microRNA mir-577,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.5,90.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.57496,-4.92468,1600000,245060,182,86,true,-3.5451,0.71905,2013-10-04T02:00:19.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01014,mir-1306,1287,microRNA mir-1306,Griffiths-Jones SR,Griffiths-Jones SR,70.0,75.7,49.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,19,113,0,0,Gene; miRNA;,Predicted; RNAalifold,109,0,,\"\",0.63347,-4.18033,1600000,180119,161,72,true,-3.5846,0.71905,2013-10-04T02:00:23.000Z,2021-12-03T12:04:12.000Z\n",
            "RF01015,mir-885,2024,microRNA mir-885,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.8,63.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,28,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.64931,-3.30012,1600000,215935,162,74,true,-3.868,0.71913,2013-10-04T02:00:29.000Z,2022-05-06T12:43:52.000Z\n",
            "RF01016,mir-584,2025,mir-584 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.5,79.2,Family represents the microRNA (miRNA) precursor mir-584 imported from miRBase [4].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,9,0,0,Gene; miRNA;,Predicted; RNAalifold,21,0,,\"\",0.62345,-2.87292,1600000,119071,249,97,true,-3.3714,0.71877,2013-10-04T02:00:37.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01018,mir-569,2027,mir-569 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,110.0,113.1,102.4,Family represents the microRNA (miRNA) precursor mir-569 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,10,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.62109,-2.70689,1600000,193695,195,95,true,-3.4623,0.71871,2013-10-04T02:01:02.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01019,mir-922,1287,microRNA mir-922,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.6,91.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.66101,-2.68081,1600000,181247,175,81,true,-3.721,0.71899,2013-10-04T02:01:05.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01020,mir-572,1293,microRNA mir-572,Griffiths-Jones SR,Griffiths-Jones SR,80.0,87.5,74.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,11,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.50763,-4.66614,1600000,205841,188,90,true,-3.1279,0.71887,2013-02-01T12:07:03.000Z,2022-05-06T12:43:49.000Z\n",
            "RF01021,mir-558,2028,microRNA mir-558,Griffiths-Jones SR,Griffiths-Jones SR,90.0,91.5,59.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.69476,-2.47664,1600000,276226,159,70,true,-3.6134,0.71918,2013-10-04T02:01:19.000Z,2022-05-06T12:43:52.000Z\n",
            "RF01022,mir-611,1287,microRNA mir-611,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.3,77.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66807,-3.65692,1600000,278484,152,65,true,-3.4445,0.7192,2013-10-04T02:02:52.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01023,mir-940,1287,microRNA mir-940,Griffiths-Jones SR,Griffiths-Jones SR,106.0,106.1,104.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.50914,-4.9145,1600000,226677,194,94,true,-3.0098,0.71877,2013-10-04T02:03:01.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01024,mir-944,1287,microRNA mir-944,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.6,85.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.56431,-4.29947,1600000,221672,185,88,true,-3.9001,0.71895,2013-10-04T02:03:04.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01025,mir-934,1287,microRNA mir-934,Griffiths-Jones SR,Griffiths-Jones SR,80.0,89.5,70.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.61528,-3.82697,1600000,246330,178,83,true,-3.2701,0.71901,2013-10-04T02:03:11.000Z,2021-12-03T12:04:12.000Z\n",
            "RF01026,MIR828,2029,microRNA MIR828,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.2,68.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,70,0,0,Gene; miRNA;,Predicted; RNAalifold,49,0,,\"\",0.57511,-4.28639,1600000,196545,360,121,true,-2.704,0.71841,2013-10-04T02:03:13.000Z,2022-05-06T12:43:52.000Z\n",
            "RF01027,mir-765,2030,microRNA mir-765,Griffiths-Jones SR,Griffiths-Jones SR,130.0,131.6,107.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.59871,-4.62082,1600000,219556,221,114,true,-3.48,0.71842,2013-10-04T02:03:24.000Z,2022-05-06T12:43:52.000Z\n",
            "RF01028,mir-633,2031,microRNA mir-633,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.1,73.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.66268,-2.2906,1600000,215459,180,85,true,-3.1522,0.71891,2013-10-04T02:03:27.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01029,mir-649,1287,microRNA mir-649,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.4,56.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.62139,-2.90085,1600000,195806,191,93,true,-3.1665,0.71876,2013-10-04T02:03:30.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01030,mir-422,1287,microRNA mir-422,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.5,57.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.59184,-3.97605,1600000,237903,181,85,true,-3.449,0.71895,2013-10-04T02:03:34.000Z,2022-05-06T12:43:46.000Z\n",
            "RF01031,mir-639,1287,microRNA mir-639,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.9,82.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.50924,-4.48327,1600000,185271,199,98,true,-2.973,0.71875,2013-10-04T02:03:37.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01032,mir-554,1287,microRNA mir-554,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.6,57.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66301,-2.44326,1600000,190531,189,91,true,-3.2831,0.71882,2013-10-04T02:03:46.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01033,mir-767,2032,microRNA mir-767,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.6,88.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,24,0,0,Gene; miRNA;,Predicted; RNAalifold,27,0,,\"\",0.63058,-3.21616,1600000,185750,181,86,true,-3.6078,0.71894,2013-10-04T02:03:48.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01034,mir-618,2033,microRNA mir-618,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,96.0,96.7,64.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61968,-3.21727,1600000,182214,194,95,true,-3.1569,0.71876,2013-10-04T02:03:51.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01035,mir-887,1287,microRNA mir-887,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.9,47.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,9,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.58668,-4.18278,1600000,243428,171,78,true,-3.3465,0.71918,2013-10-04T02:03:53.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01036,mir-567,1287,microRNA mir-567,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.6,64.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.63956,-2.78942,1600000,239338,159,70,true,-3.3739,0.71914,2013-10-04T02:04:01.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01037,mir-644,1287,microRNA mir-644,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.9,84.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.68335,-2.70879,1600000,265602,164,73,true,-3.3864,0.72152,2013-10-04T02:04:16.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01038,mir-1307,1287,microRNA mir-1307,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.1,61.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,24,0,0,Gene; miRNA;,Predicted; RNAalifold,27,0,,\"\",0.59929,-3.24351,1600000,178686,187,90,true,-3.579,0.7189,2013-10-04T02:04:22.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01039,mir-937,1287,microRNA mir-937,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.9,81.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.54397,-4.12113,1600000,213627,182,86,true,-3.3076,0.71883,2013-10-04T02:04:24.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01040,mir-573,1287,microRNA mir-573,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.3,68.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.5978,-3.96409,1600000,202280,173,80,true,-3.3269,0.71897,2013-10-04T02:04:36.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01041,mir-604,1287,microRNA mir-604,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.0,56.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,11,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.64194,-2.92993,1600000,208860,182,86,true,-3.3501,0.71882,2013-10-04T02:04:44.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01042,mir-891,1287,microRNA mir-891,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.8,84.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,14,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.62304,-3.41403,1600000,229634,171,79,true,-3.2456,0.71892,2013-10-04T02:04:48.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01043,MIR1023,1287,microRNA MIR1023,\"Wilkinson A, Daub J\",\"miRBase; Wilkinson A, Daub J\",90.0,117.4,89.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59884,-6.37664,1600000,207826,280,165,true,-3.2764,0.71649,2013-10-04T02:04:52.000Z,2022-05-06T12:43:46.000Z\n",
            "RF01044,mir-345,2034,microRNA mir-345,Griffiths-Jones SR,Griffiths-Jones SR,65.0,66.6,63.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,34,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.65512,-3.33923,1600000,157770,187,91,true,-3.4215,0.71872,2013-10-04T02:05:34.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01045,mir-544,2035,microRNA mir-544,Griffiths-Jones SR,Griffiths-Jones SR,66.5,66.5,66.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,16,2424,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.61327,-2.8495,1600000,169684,224,84,true,-3.5152,0.71913,2013-10-04T02:05:40.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01046,WLE3,2036,Wingless localisation element 3 (WLE3),Brown C,Predicted,50.0,55.0,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19 -Z 549862.597050 CM SEQDB,10,9,0,0,Cis-reg;,Predicted; RNAalifold,14,0,,\"\",0.70276,-3.65537,1600000,429166,112,44,true,-3.9367,0.71965,2013-10-04T02:06:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01047,HBV_epsilon,2037,HBV RNA encapsidation signal epsilon,\"Chen A, Brown C\",\"Chen A, Brown C\",45.0,83.7,38.5,The HBV RNA encapsidation signal epsilon (HBV_epsilon) is an element essential for HBV virus replication. Its structure was reported in [4],HBV,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,33,3,0,0,Cis-reg;,Published; PMID:12409471,6,0,,\"\",0.62294,-3.96223,1600000,296380,145,61,true,-3.9047,0.71937,2013-10-04T02:06:20.000Z,2022-05-06T13:10:04.000Z\n",
            "RF01050,Sacc_telomerase,1322,Saccharomyces telomerase,Gardner PP,Published; PMID:15242611,80.0,102.9,37.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,9,0,0,Gene;,Published; PMID:15242611,19,0,,\"\",0.5795,-9.79073,1600000,352561,1537,1208,true,-5.5799,0.69632,2013-10-04T02:06:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01051,c-di-GMP-I,2038,Cyclic di-GMP-I riboswitch,Weinberg Z,Weinberg Z,35.0,35.0,34.9,\"GEMM was predicted in a large-scale screen for bacterial structured RNA regulatory elements. Genes associated with GEMM are involved in a diverse array of functions, including chemotaxis regulation, signal transduction, secretion and chitin.\",GEMM_RNA_motif,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,155,4058,0,0,Cis-reg;,Published; PMID:17621584,1390,0,,\"\",0.54602,-7.72484,1600000,250881,300,87,true,-3.6804,0.71863,2013-10-04T02:06:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01052,Arthropod_7SK,1393,Arthropod 7SK RNA,\"Gruber A, Stadler PF, Gardner PP\",Published; PMID:18566019,44.0,44.4,42.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,102,0,0,Gene;,Published; PMID:18566019,73,0,,\"\",0.69961,-10.65851,1600000,464304,645,445,true,-4.3207,0.70174,2013-10-04T02:06:38.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01053,Deinococcus_Y_RNA,1317,Deinococcus radiodurans Y RNA,Gardner PP,Gardner PP,80.0,163.4,45.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene;,Published; PMID:10766734,2,0,,\"\",0.54013,-6.15672,1600000,241640,238,126,true,-2.9921,0.71824,2013-10-04T02:06:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01054,preQ1-II,2039,preQ1-II (pre queuosine) riboswitch,\"Weinberg Z, Gardner PP, Ontiveros-Palacios N\",\"Published; PMID:18305186; Weinberg Z, Meyer MM, Gardner PP\",42.0,42.7,41.2,\"preQ1-II riboswitch was first reported for Weinberg et al. 2007 [7] it was recognised in Streptococcaceae and Lactococcus families in 5' untranslated regions of genes of a membrane protein classified as COG4708 or DUF988. Meyers et al. 2008 [1] confirm that preQ1-II recognise the queuosine (Q) precursor pre-queuosine 1 (preQ1). Q is a hyper modified nucleoside present in some anticodons in tRNAs (Try, Asn, Asp and His). The structure of preQ1-II riboswitch has been reported by Kang et al. 2014 [3], as preQ1 riboswitch (RF00522) preQ1-II structure present an unusual H-type pseudoknot featuring and differs in the interactions that recognise preQ1. The structure of preQ1-II riboswitch was reported in PDB:2MIY [3]\",COG4708,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,57,0,0,Cis-reg; riboswitch;,Published; PMID:18305186; Gardner PP,67,0,,\"\",0.54846,-4.69978,1600000,257762,275,99,true,-3.405,0.71842,2013-10-04T02:06:42.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01055,MOCO_RNA_motif,2040,Moco (molybdenum cofactor) riboswitch,Weinberg Z,\"Barrick JE, Weinberg Z\",36.0,36.0,35.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,160,1230,0,0,Cis-reg; riboswitch;,Published; PMID:18363797,787,0,,\"\",0.68702,-4.88098,1600000,263850,477,135,true,-3.5156,0.71817,2013-10-04T02:06:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01056,Mg_sensor,2041,Magnesium Sensor,Gardner PP,Gardner PP,40.0,41.4,36.6,\"The Magnesium sensor RNA element is a cis-regulatory element that regulates the expression of the magnesium transporter protein MgtA. It is located in the 5' UTR of this gene. The element adopts different structures depending on the presence or absence of Mg2+, this has the affect of regulating whether transcription reads through the mgtA coding region or stops within the 5' UTR.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,38,0,0,Cis-reg; riboswitch;,Published; PMID:17381304,40,0,,\"\",0.55564,-7.88813,1600000,261884,218,115,true,-3.7414,0.71843,2013-10-04T02:06:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01057,SAH_riboswitch,2042,S-adenosyl-L-homocysteine riboswitch,Wang J,Published; PMID:18374645,37.0,37.1,36.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,52,809,0,0,Cis-reg; riboswitch;,Published; PMID:18374645,687,0,,\"\",0.51598,-8.31866,1600000,235441,440,78,true,-3.6642,0.71872,2013-10-04T02:06:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01059,mir-598,2043,microRNA mir-598,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.0,62.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,30,0,0,Gene; miRNA;,Predicted; RNAalifold,32,0,,\"\",0.64007,-3.43903,1600000,191620,173,80,true,-3.6696,0.71903,2013-10-04T02:06:54.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01061,mir-548,2044,microRNA mir-548,Wilkinson A,miRBase; Wilkinson A,90.0,90.0,89.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,951,0,0,Gene; miRNA;,Predicted; WAR; Wilkinson A,11,0,,\"\",0.60566,-3.62097,1600000,214506,175,84,true,-3.374,0.71903,2013-10-04T02:07:24.000Z,2021-12-03T12:04:12.000Z\n",
            "RF01064,mir-253,1287,microRNA mir-253,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.0,68.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.63113,-3.15897,1600000,178468,197,98,true,-3.4748,0.71878,2013-10-04T02:09:14.000Z,2021-11-19T10:36:03.000Z\n",
            "RF01065,23S-methyl,2045,23S methyl RNA motif,\"Weinberg Z, Gardner PP\",Published; PMID:17621584; Weinberg Z,50.0,50.9,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,190,0,0,Cis-reg;,Published; PMID:17621584; Weinberg Z,200,0,,\"\",0.49256,-7.01491,1600000,260086,193,102,true,-3.4275,0.71864,2013-10-04T02:09:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01066,6C,2046,6C RNA,\"Weinberg Z, Gardner PP\",Published; PMID:17621584; Weinberg Z,47.0,47.0,46.9,,390,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.35 -Z 549862.597050 CM SEQDB,18,394,0,0,Cis-reg;,Published; PMID:17621584; Weinberg Z,397,0,,\"\",0.39794,-11.30154,1600000,336561,161,75,true,-3.4646,0.71892,2013-10-04T02:09:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01067,ATPC,2047,ATPC RNA motif,\"Weinberg Z, Gardner PP\",Published; PMID:17621584; Weinberg Z,45.0,45.5,41.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.00 -Z 549862.597050 CM SEQDB,90,15,0,0,Gene;,Published; PMID:17621584; Weinberg Z,26,0,,\"\",0.56152,-6.57273,1600000,232173,175,92,true,-3.8212,0.71871,2013-10-04T02:09:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01068,mini-ykkC,1300,Guanidine-II riboswitch,\"Weinberg Z, Gardner PP\",Published; PMID:17621584; Weinberg Z,32.0,32.0,31.9,\"Simpler in structure and has fewer highly conserved nucleotide positions than the ykkC-yxkD leader.[1] Despite this each of its two stem-loop structures directly bind free guanidine. Therefore, mini-ykkC RNA motif represents a distinct class of guanidine-sensing RNAs.[2] Its crystal structure was also determined.[3]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.00 -Z 549862.597050 CM SEQDB,72,929,0,0,Cis-reg;,Published; PMID:17621584; Weinberg Z,788,0,,\"\",0.53606,-11.16592,1600000,878218,103,48,true,-3.5583,0.73256,2013-02-01T12:14:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01069,purD,2048,purD RNA motif,\"Weinberg Z, Gardner PP\",Published; PMID:17621584; Weinberg Z,41.0,42.1,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,21,80,0,0,Cis-reg;,Published; PMID:17621584; Weinberg Z,87,0,,\"\",0.5452,-7.20408,1600000,272302,220,125,true,-3.3034,0.71821,2013-10-04T02:09:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01070,sucA,2049,SucA RNA motif,\"Weinberg Z, Gardner PP\",Published; PMID:17621584; Weinberg Z,40.0,41.2,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,35,249,0,0,Cis-reg;,Published; PMID:17621584; Weinberg Z,257,0,,\"\",0.54747,-6.41143,1600000,225470,177,83,true,-3.5953,0.71891,2013-10-04T02:09:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01071,OLE,2050,Ornate Large Extremophilic RNA,\"Weinberg Z, Gardner PP\",Published; PMID:17164334; Weinberg Z,50.0,57.2,45.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,274,0,0,Gene;,Published; PMID:17164334; Weinberg Z,276,0,,\"\",0.69483,-8.98988,1600000,365507,989,616,true,-5.1487,0.69935,2013-10-04T02:09:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01072,TMV_UPD-PK3,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,34.0,34.1,33.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.00 -Z 549862.597050 CM SEQDB,25,15,0,0,Cis-reg;,Pseudobase,32,0,,\"\",0.67904,-7.21806,1600000,543635,88,31,true,-3.7753,0.72742,2013-10-04T02:09:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01073,GP_knot1,2409,Gag/pol translational readthrough site,Wilkinson A,Pseudobase,67.7,67.7,67.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,757,0,0,Cis-reg;,Pseudobase,36,0,,\"\",0.62551,-5.47554,1600000,411274,142,62,true,-3.307,0.71908,2013-10-04T02:10:03.000Z,2022-10-29T17:42:08.000Z\n",
            "RF01074,RF_site1,1295,Putative RNA-dependent RNA polymerase ribosomal frameshift site,Wilkinson A,Pseudobase,40.0,42.0,38.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.93 -Z 742849.287494 --mxsize 128 CM SEQDB,4,8,0,0,Cis-reg; frameshift_element;,Pseudobase,11,0,,\"\",0.51915,-10.4759,1600000,985059,113,40,true,-3.0435,0.72896,2013-02-01T12:07:07.000Z,2020-04-23T12:35:10.000Z\n",
            "RF01075,TLS-PK1,2052,Pseudoknot of tRNA-like structure,Wilkinson A,Pseudobase,80.0,109.2,37.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.56939,-7.04447,1600000,247342,197,97,true,-3.3743,0.71854,2013-10-04T02:10:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01076,RF_site2,1295,Polymerase ribosomal frameshift site,Wilkinson A,Pseudobase,70.0,84.0,32.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Cis-reg; frameshift_element;,Pseudobase,4,0,,\"\",0.58976,-3.68548,1600000,227714,163,72,true,-3.2485,0.72186,2013-10-04T02:10:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01077,TLS-PK2,2052,Pseudoknot of tRNA-like structure,Wilkinson A,Pseudobase,42.0,42.6,38.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,2,0,0,Cis-reg;,Pseudobase,6,0,,\"\",0.56589,-6.2593,1600000,305491,150,66,true,-3.4596,0.71912,2013-10-04T02:10:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01078,PK-PYVV,1296,3'-terminal pseudoknot in PYVV,Wilkinson A,Pseudobase,65.0,74.6,37.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.97 -Z 742849.287494 --mxsize 128 CM SEQDB,3,1,0,0,Cis-reg;,Pseudobase,1,0,,\"\",0.52333,-8.23124,1600000,363420,139,58,true,-3.3712,0.72156,2013-02-01T12:07:08.000Z,2020-04-23T12:35:10.000Z\n",
            "RF01079,RF_site3,1295,Putative RNA-dependent RNA polymerase ribosomal frameshift site,Wilkinson A,Pseudobase,50.0,60.1,33.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg; frameshift_element;,Pseudobase,2,0,,\"\",0.57146,-6.28882,1600000,410860,117,39,true,-2.3146,0.74314,2013-10-04T02:10:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01080,UPD-PKib,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,39.0,39.9,34.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,2,0,0,Cis-reg;,Pseudobase,4,0,,\"\",0.53859,-7.74439,1600000,507814,102,32,true,-3.1901,0.73905,2013-10-04T02:10:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01081,SBWMV1_UPD-PKe,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,38.0,39.0,32.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.31 -Z 549862.597050 CM SEQDB,3,3,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.55367,-6.81126,1600000,466680,96,26,true,-2.7808,0.75549,2013-10-04T02:10:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01082,SBWMV1_UPD-PKh,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,32.0,32.3,31.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.33 -Z 549862.597050 CM SEQDB,2,3,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.64708,-3.41455,1600000,348281,97,25,true,-2.3661,0.7827,2013-10-04T02:10:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01083,UPD-PK2,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,32.0,32.0,31.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,7,0,0,Cis-reg;,Pseudobase,7,0,,\"\",0.57464,-6.42484,1600000,574590,86,21,true,-2.9358,0.7503,2013-10-04T02:10:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01084,TLS-PK3,2052,Pseudoknot of tRNA-like structure,Wilkinson A,Pseudobase,45.0,53.6,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,9,0,0,Cis-reg;,Pseudobase,11,0,,\"\",0.59313,-7.58435,1600000,249798,243,130,true,-3.3996,0.71823,2013-10-04T02:10:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01085,TLS-PK4,2052,Pseudoknot of tRNA-like structure,Wilkinson A,Pseudobase,80.0,136.9,58.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,2,0,0,0,Cis-reg;,Pseudobase,1,0,,\"\",0.61049,-6.82034,1600000,254880,226,118,true,-3.1415,0.7184,2013-10-04T02:10:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01087,PK-repZ,2053,Pseudoknot of the regulatory region of the repZ gene,Wilkinson A,Pseudobase,44.0,46.3,42.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,0,0,0,Cis-reg;,Pseudobase,6,0,,\"\",0.58748,-7.90469,1600000,216002,268,150,true,-2.7904,0.71791,2013-10-04T02:10:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01088,TLS-PK5,2052,Pseudoknot of tRNA-like structure,Wilkinson A,Pseudobase,70.0,79.5,33.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,0,0,0,Cis-reg;,Pseudobase,3,0,,\"\",0.57169,-4.65998,1600000,247065,153,68,true,-3.7759,0.71906,2013-10-04T02:10:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01089,PK-repBA,2054,Pseudoknot of the regulatory region of the repBA gene,Wilkinson A,Pseudobase,45.0,45.8,44.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.00 -Z 742849.287494 --mxsize 128 CM SEQDB,7,38,0,0,Cis-reg;,Pseudobase,36,0,,\"\",0.62249,-6.74766,1600000,293665,226,124,true,-3.0784,0.71812,2013-10-04T02:10:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01090,RF_site4,1295,Edr gene ribosomal frameshift signal,Wilkinson A,Pseudobase,45.0,50.3,37.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,37,0,0,Cis-reg; frameshift_element;,Pseudobase,38,0,,\"\",0.53573,-7.70037,1600000,419055,151,66,true,-3.4062,0.71902,2013-10-04T02:10:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01091,PK-SPCSV,2055,3'-terminal pseudoknot in SPCSV,Wilkinson A,Pseudobase,36.0,36.4,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.45 -Z 549862.597050 CM SEQDB,2,6,0,0,Cis-reg;,Pseudobase,4,0,,\"\",0.54901,-6.4112,1600000,345088,148,61,true,-3.1768,0.72491,2013-10-04T02:11:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01092,GP_knot2,2409,Gag/pol translational readthrough site,Wilkinson A,Pseudobase,80.0,86.0,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,0,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.56983,-6.7561,1600000,367971,148,61,true,-2.86,0.72408,2013-10-04T02:11:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01093,RF_site5,1295,Ma3 gene ribosomal frameshift signal,Wilkinson A,Pseudobase,35.0,36.0,34.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,103,0,0,Cis-reg; frameshift_element;,Pseudobase,43,0,,\"\",0.51187,-9.73219,1600000,698201,137,60,true,-3.5524,0.71913,2013-10-04T02:11:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01094,RF_site6,1295,Polymerase ribosomal frameshift site,Wilkinson A,Pseudobase,90.0,154.0,38.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg; frameshift_element;,Pseudobase,2,0,,\"\",0.51495,-5.06211,1600000,249610,226,118,true,-3.2218,0.71823,2013-10-04T02:11:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01095,PK-CuYV_BPYV,2056,3'-terminal pseudoknot of CuYV/BPYV,Wilkinson A,Pseudobase,60.0,89.9,34.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.56516,-6.36783,1600000,411202,141,56,true,-3.13,0.72723,2013-10-04T02:11:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01096,PK-HAV,2057,HepA virus 3'-terminal pseudoknot,Wilkinson A,Pseudobase,50.0,54.0,45.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.45476,-6.0697,1600000,394283,141,56,true,-2.6114,0.73329,2013-10-04T02:11:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01097,RF_site8,1295,Gag/pro ribosomal frameshift site,Wilkinson A,Pseudobase,43.0,43.0,42.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,883,0,0,Cis-reg; frameshift_element;,Pseudobase,36,0,,\"\",0.51554,-9.82953,1600000,678662,130,52,true,-3.5977,0.71927,2013-10-04T02:11:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01098,RF_site9,1295,Gag/pro ribosomal frameshift site,Wilkinson A,Pseudobase,70.0,70.4,63.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,2,4,0,0,Cis-reg; frameshift_element;,Pseudobase,4,0,,\"\",0.5467,-7.23948,1600000,471970,133,50,true,-2.8685,0.73334,2013-10-04T02:11:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01099,PK-IAV,2058,Pseudoknot of influenza A virus gene,\"Wilkinson A, Moore B\",Pseudobase,47.0,47.3,44.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,32,0,0,0,Cis-reg;,Pseudobase,32,0,,\"\",0.62776,-5.83839,1600000,438263,121,48,true,-3.7572,0.71931,2013-10-04T02:11:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01100,PK-BYV,2059,3'-terminal pseudoknot in BYV,Wilkinson A,Pseudobase,50.0,59.1,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.62293,-6.22579,1600000,378819,119,40,true,-2.5086,0.74,2013-10-04T02:11:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01101,TLS-PK6,2052,Pseudoknot of tRNA-like structure,Wilkinson A,Pseudobase,44.0,47.7,31.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Cis-reg;,Pseudobase,4,0,,\"\",0.56949,-7.4766,1600000,408727,116,40,true,-3.2065,0.73181,2013-10-04T02:12:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01102,PK1-TEV_CVMV,1297,5'-leader pseudoknot of TEV/CVMV,Wilkinson A,Pseudobase,40.0,42.7,38.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg; leader;,Pseudobase,3,0,,\"\",0.54819,-4.47842,1600000,311926,114,37,true,-2.3718,0.75727,2013-02-01T12:07:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01103,UPD-PKc,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,40.0,44.3,34.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.51549,-7.14038,1600000,467179,103,29,true,-2.5908,0.76215,2013-10-04T02:12:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01104,SBWMV2_UPD-PKb,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,39.0,42.2,31.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Cis-reg;,Pseudobase,3,0,,\"\",0.60831,-6.36082,1600000,632755,100,29,true,-2.936,0.74482,2013-10-04T02:12:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01105,SBWMV2_UPD-PKl,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,39.0,41.7,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.50 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Pseudobase,1,0,,\"\",0.50846,-5.32709,1600000,394175,100,27,true,-2.4566,0.76565,2013-10-04T02:12:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01106,SBWMV1_UPD-PKb,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,38.0,39.5,35.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.51555,-6.51222,1600000,392050,97,25,true,-2.3569,0.77506,2013-10-04T02:12:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01107,SBRMV1_UPD-PKf,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,39.0,42.1,32.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Cis-reg;,Pseudobase,1,0,,\"\",0.59948,-5.25916,1600000,424928,100,27,true,-2.5426,0.76321,2013-10-04T02:12:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01108,BMV3_UPD-PK1,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,38.0,40.8,35.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Pseudobase,1,0,,\"\",0.65415,-4.40576,1600000,426316,98,26,true,-2.2978,0.76646,2013-10-04T02:12:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01109,SBRMV1_UPD-PKd,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,33.0,35.5,30.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,2,2,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.5842,-5.46766,1600000,431766,90,21,true,-2.2481,0.78265,2013-10-04T02:12:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01111,SBWMV2_UPD-PKk,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,35.0,36.3,33.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.26 -Z 549862.597050 CM SEQDB,4,1,0,0,Cis-reg;,Pseudobase,3,0,,\"\",0.56733,-7.68732,1600000,673678,86,21,true,-2.8507,0.74995,2013-10-04T02:12:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01113,BMV3_UPD-PK3,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,36.0,37.0,33.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 549862.597050 CM SEQDB,2,2,0,0,Cis-reg;,Pseudobase,2,0,,\"\",0.53492,-4.47632,1600000,356879,93,23,true,-1.862,0.78125,2013-10-04T02:12:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01114,TMV_UPD-PK1,1651,Pseudoknot of upstream pseudoknot domain (UPD) of the 3'UTR,Wilkinson A,Pseudobase,33.0,33.9,31.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Cis-reg;,Pseudobase,4,0,,\"\",0.6637,-3.20997,1600000,352659,92,22,true,-2.6238,0.7808,2013-10-04T02:12:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01116,Yfr1,2060,Cyanobacterial functional RNA 1,\"Voss B, Gardner PP\",Published; PMID:17941988,38.0,38.3,37.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.00 -Z 549862.597050 CM SEQDB,37,120,0,0,Gene; sRNA;,Published; PMID:17941988,129,0,,\"\",0.56468,-8.04994,1600000,390460,129,59,true,-3.5873,0.71952,2013-10-04T02:12:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01117,ciona-mir-92,1287,Ciona microRNA mir-92,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.4,44.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,5,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62579,-3.80134,1600000,219276,153,71,true,-3.7206,0.72377,2013-10-04T02:12:46.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01119,sR32,1264,Small nucleolar RNA sR32,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,50.7,35.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,5,0,,\"\",0.70912,-7.82785,1600000,492612,136,57,false,-3.613,0.71973,2013-10-04T02:13:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01120,sR33,1264,Small nucleolar RNA sR33,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.6,35.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.37 -Z 549862.597050 CM SEQDB,3,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,10,0,,\"\",0.55329,-9.02327,1600000,307023,133,55,false,-3.8104,0.72604,2013-10-04T02:13:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01121,sR38,1264,Small nucleolar RNA sR38,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,69.6,35.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,5,0,,\"\",0.61747,-9.33987,1600000,552058,140,59,false,-3.7292,0.71897,2013-10-04T02:13:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01122,sR39,1264,Small nucleolar RNA sR39,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,65.4,33.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,10,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.65153,-9.29859,1600000,785278,135,56,false,-3.7436,0.71903,2013-10-04T02:13:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01123,sR35,1264,Small nucleolar RNA sR35,Wilkinson A,Predicted; WAR; Wilkinson A,60.0,74.6,37.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.00 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.66854,-7.40004,1600000,365648,136,57,false,-4.0223,0.72176,2013-10-04T02:13:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01124,sR36,1264,Small nucleolar RNA sR36,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,72.5,32.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.16 -Z 549862.597050 CM SEQDB,5,10,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,13,0,,\"\",0.65858,-8.84998,1600000,575616,133,55,false,-3.8917,0.71905,2013-10-04T02:13:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01125,sR4,1264,Small nucleolar RNA sR4,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,78.3,34.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.10 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,3,0,,\"\",0.61696,-8.26981,1600000,415348,135,56,false,-3.6391,0.72125,2013-10-04T02:13:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01126,sR41,1264,Small nucleolar RNA sR41,Wilkinson A,Predicted; WAR; Wilkinson A,54.0,57.6,53.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.10 -Z 549862.597050 CM SEQDB,7,16,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,18,0,,\"\",0.62731,-9.55919,1600000,653998,123,53,false,-4.034,0.7191,2013-10-04T02:13:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01127,sR42,1264,Small nucleolar RNA sR42,Wilkinson A,Predicted; WAR; Wilkinson A,38.0,38.5,34.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.61744,-8.61975,1600000,351280,144,63,false,-3.847,0.71984,2013-10-04T02:14:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01128,sR43,1264,Small nucleolar RNA sR42,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,58.5,35.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.19 -Z 549862.597050 CM SEQDB,6,11,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,13,0,,\"\",0.65168,-8.97019,1600000,521623,161,57,false,-3.7747,0.71899,2013-10-04T02:14:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01129,sR44,1264,Small nucleolar RNA sR44,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,59.8,36.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.65388,-8.68722,1600000,646264,133,55,false,-3.3056,0.72215,2013-10-04T02:14:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01130,sR46,1264,Small nucleolar RNA sR46,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,69.8,33.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,10,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.64192,-8.0824,1600000,519651,141,56,true,-3.7884,0.71912,2013-10-04T02:14:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01131,sR47,1264,Small nucleolar RNA sR47,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,43.6,39.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,11,0,,\"\",0.62165,-9.16525,1600000,541940,124,55,false,-4.0526,0.71905,2013-10-04T02:14:08.000Z,2022-10-29T17:55:16.000Z\n",
            "RF01132,sR48,1264,Small nucleolar RNA sR48,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,47.5,33.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.49847,-8.44662,1600000,227909,142,59,false,-2.9537,0.73418,2013-10-04T02:14:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01133,sR3,1264,Small nucleolar RNA sR3,Wilkinson A,Predicted; WAR; Wilkinson A,41.0,42.1,40.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.87 -Z 549862.597050 CM SEQDB,19,54,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,56,0,,\"\",0.66691,-8.68288,1600000,439985,226,55,false,-4.1412,0.71905,2013-10-04T02:14:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01134,sR30,1264,Small nucleolar RNA sR30,Wilkinson A,Predicted; WAR; Wilkinson A,38.0,41.3,33.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,5,0,,\"\",0.65977,-8.56314,1600000,517101,136,57,false,-3.8083,0.71928,2013-10-04T02:14:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01135,sR24,1264,Small nucleolar RNA sR24,Wilkinson A,Predicted; WAR; Wilkinson A,36.0,37.8,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.67058,-5.63199,1600000,422306,136,55,true,-3.579,0.71907,2013-10-04T02:14:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01136,sR28,1264,Small nucleolar RNA sR28,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,40.4,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,12,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,16,0,,\"\",0.68177,-7.96953,1600000,430485,127,55,false,-3.9712,0.72206,2013-10-04T02:14:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01137,sR21,1264,Small nucleolar RNA sR21,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,63.6,31.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.53219,-7.12044,1600000,246880,153,67,false,-2.2683,0.73793,2013-10-04T02:14:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01138,sR23,1264,Small nucleolar RNA sR23,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,57.9,36.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.00 -Z 549862.597050 CM SEQDB,6,10,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.65997,-8.67515,1600000,554887,127,54,false,-4.2138,0.71908,2013-10-04T02:14:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01139,sR2,1264,Small nucleolar RNA sR2,Wilkinson A,Predicted; WAR; Wilkinson A,42.0,48.9,41.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.00 -Z 549862.597050 CM SEQDB,8,15,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,16,0,,\"\",0.59639,-8.78876,1600000,382307,127,57,false,-3.8829,0.72261,2013-10-04T02:15:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01140,sR20,1264,Small nucleolar RNA sR20,Wilkinson A,Predicted; WAR; Wilkinson A,44.0,44.6,41.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,11,0,,\"\",0.61201,-6.34545,1600000,359917,130,56,true,-3.9743,0.72193,2013-10-04T02:20:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01141,sR18,1264,Small nucleolar RNA sR18,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,53.7,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,3,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.6004,-9.32957,1600000,546902,129,56,false,-3.9149,0.72078,2013-10-04T02:20:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01142,sR19,1264,Small nucleolar RNA sR19,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,59.5,33.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.62815,-8.67119,1600000,390886,145,63,false,-3.7147,0.71885,2013-10-04T02:20:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01143,sR16,1264,Small nucleolar RNA sR16,Wilkinson A,Predicted; WAR; Wilkinson A,70.0,79.1,33.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.83 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.54049,-8.71524,1600000,354393,134,53,false,-3.0358,0.72843,2013-10-04T02:20:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01144,sR17,1264,Small nucleolar RNA sR17,Wilkinson A,Predicted; WAR; Wilkinson A,36.0,36.5,35.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.38 -Z 549862.597050 CM SEQDB,5,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,10,0,,\"\",0.63175,-8.50811,1600000,497441,128,55,false,-3.5386,0.72232,2013-10-04T02:20:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01145,sR14,1264,Small nucleolar RNA sR14,Wilkinson A,Predicted; WAR; Wilkinson A,38.0,40.4,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,14,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,17,0,,\"\",0.69831,-7.94333,1600000,533490,123,54,false,-3.9556,0.72099,2013-10-04T02:20:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01146,sR15,1264,Small nucleolar RNA sR15,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,47.9,29.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.59 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,3,0,,\"\",0.55886,-5.59377,1600000,188331,150,65,false,-2.6368,0.74215,2013-10-04T02:20:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01147,sR12,1264,Small nucleolar RNA sR12,Wilkinson A,Predicted; WAR; Wilkinson A,38.0,39.6,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,5,5,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,8,0,,\"\",0.55834,-9.86894,1600000,547807,155,55,false,-3.472,0.72414,2013-10-04T02:20:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01149,sR10,1264,Small nucleolar RNA sR10,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,69.0,34.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,3,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,5,0,,\"\",0.66771,-7.9857,1600000,405847,130,54,false,-3.9408,0.71909,2013-10-04T02:20:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01150,sR11,1264,Small nucleolar RNA sR11,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,40.2,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.87 -Z 549862.597050 CM SEQDB,8,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,8,0,,\"\",0.52766,-9.68743,1600000,410507,143,57,false,-3.2406,0.72886,2013-10-04T02:20:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01151,snoU82P,1264,Small nucleolar RNA U82P,Wilkinson A,Predicted; WAR; Wilkinson A,42.0,45.7,39.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,42,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,45,0,,\"\",0.66432,-8.65542,1600000,569031,130,55,false,-3.8822,0.71906,2013-10-04T02:20:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01152,sR1,1264,Small nucleolar RNA sR1,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,46.7,43.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19 -Z 549862.597050 CM SEQDB,12,22,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,24,0,,\"\",0.66863,-8.9894,1600000,598719,121,57,false,-3.889,0.7202,2013-10-04T02:20:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01153,sn3060,1264,Small nucleolar RNA sn3060,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,48.9,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,12,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,10,0,,\"\",0.65558,-3.76017,1600000,296658,155,67,true,-3.5411,0.71886,2013-10-04T02:20:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01155,snoR101,1264,Small nucleolar RNA snoR101,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,40.7,36.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,189,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,74,0,,\"\",0.53464,-9.81742,1600000,409340,147,66,false,-3.4112,0.71882,2013-10-04T02:20:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01156,SNORD123,1264,Small nucleolar RNA SNORD123,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,45.8,44.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,12,92,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,94,0,,\"\",0.62287,-3.90935,1600000,309070,155,70,true,-3.6293,0.71881,2013-10-04T02:20:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01157,sn1185,1264,Small nucleolar RNA sn1185,Wilkinson A,Predicted; WAR; Wilkinson A,60.0,67.2,52.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,7,0,,\"\",0.62145,-4.6126,1600000,302161,152,66,true,-3.6537,0.71882,2013-10-04T02:21:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01158,SNORD108,1264,Small nucleolar RNA SNORD108,Wilkinson A,Predicted; WAR; Wilkinson A,46.0,46.5,44.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.24 -Z 549862.597050 CM SEQDB,4,15,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,15,0,,\"\",0.5883,-4.97642,1600000,275647,156,71,true,-3.856,0.71874,2013-10-04T02:21:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01159,snoU18,1264,Small nucleolar RNA U18,Wilkinson A,Predicted; WAR; Wilkinson A,47.0,47.2,46.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.54 -Z 549862.597050 CM SEQDB,16,91,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,57,0,,\"\",0.44148,-10.05077,1600000,273722,149,71,false,-3.5818,0.71871,2013-02-01T11:58:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01160,sn2841,1264,Small nucleolar RNA sn2841,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,60.7,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,14,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,13,0,,\"\",0.65593,-3.53246,1600000,286199,163,72,true,-3.1354,0.72101,2013-10-04T02:21:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01161,SNORD5,1264,Small nucleolar RNA SNORD5,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,55.7,54.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,20,117,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,100,0,,\"\",0.46599,-9.46928,1600000,305795,151,72,false,-3.5735,0.7187,2013-10-04T02:21:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01162,sn668,1264,Small nucleolar RNA sn668,\"Wilkinson A, Eberhardt R\",Predicted; WAR; Wilkinson A,40.0,40.6,38.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.80 -Z 549862.597050 CM SEQDB,6,21,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,20,0,,\"\",0.65733,-3.16694,1600000,280760,151,65,true,-3.6599,0.7189,2013-10-04T02:21:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01163,snoR64a,1264,Small nucleolar RNA snoR64a,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,45.0,43.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.69 -Z 549862.597050 CM SEQDB,4,109,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,44,0,,\"\",0.47296,-9.36012,1600000,305149,163,77,false,-3.5665,0.7186,2013-10-04T02:21:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01164,SNORD107,1264,Small nucleolar RNA SNORD107,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,55.9,46.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,40,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,39,0,,\"\",0.59437,-4.33876,1600000,300373,160,74,true,-3.4439,0.71873,2013-10-04T02:21:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01165,sn2317,1264,Small nucleolar RNA sn2317,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,73.9,40.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.59218,-4.03889,1600000,249042,160,70,true,-3.436,0.71881,2013-02-01T11:58:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01166,sn3071,1264,Small nucleolar RNA sn3071,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,40.6,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,14,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,13,0,,\"\",0.65219,-3.6547,1600000,281772,169,77,true,-3.3246,0.71945,2013-10-04T02:21:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01167,sn2429,1264,Small nucleolar RNA sn2429,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,74.7,36.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.62913,-3.78273,1600000,251751,168,77,true,-3.2946,0.71868,2013-10-04T02:21:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01168,SNORD126,1264,Small nucleolar RNA SNORD126,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,66.1,52.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.08 -Z 549862.597050 CM SEQDB,6,45,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,43,0,,\"\",0.63563,-3.55733,1600000,233334,166,77,true,-3.839,0.7187,2013-10-04T02:21:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01169,SNORD112,1264,Small nucleolar RNA SNORD112,Wilkinson A,Predicted; WAR; Wilkinson A,58.0,58.1,57.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,51,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,35,0,,\"\",0.63202,-3.20873,1600000,241776,162,77,true,-3.751,0.71868,2013-10-04T02:21:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01170,snoU61,1264,Small nucleolar RNA U61,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,55.9,52.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.70 -Z 549862.597050 CM SEQDB,8,130,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,67,0,,\"\",0.67768,-3.32738,1600000,262278,191,84,true,-3.2583,0.7185,2013-10-04T02:21:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01171,snoR4,1264,Small nucleolar RNA snoR4,Wilkinson A,Predicted; WAR; Wilkinson A,60.0,81.9,37.8,This model is based on the Medicago trunculata homolog of Arabidposis thaliana snoR4 which is included in Rfam model RF00147 (along with mouse and other species). This sequence was obtained from the plant snoRNA database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.44578,-7.1475,1600000,203927,170,80,false,-2.6256,0.72861,2013-10-04T02:21:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01172,sn2343,1264,Small nucleolar RNA sn2343,Wilkinson A,Predicted; WAR; Wilkinson A,60.0,89.9,38.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,7,0,,\"\",0.71694,-3.22081,1600000,285998,170,78,true,-3.6359,0.71864,2013-10-04T02:21:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01173,snoU105B,1264,Small nucleolar RNA U105B,\"Wilkinson A, Eberhardt R\",Predicted; WAR,49.0,49.9,46.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.01 -Z 549862.597050 CM SEQDB,13,34,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR,37,0,,\"\",0.65102,-3.81555,1600000,214602,167,80,true,-3.7607,0.71878,2013-10-04T02:21:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01174,snoU43,1264,Small nucleolar RNA U43,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,45.3,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.42 -Z 549862.597050 CM SEQDB,11,51,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,46,0,,\"\",0.45861,-9.07596,1600000,338249,162,79,false,-3.2219,0.71859,2013-10-04T02:21:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01175,snoU83D,1264,Small nucleolar RNA U83,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,80.0,39.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,12,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,13,0,,\"\",0.56308,-5.73495,1600000,252568,171,81,true,-3.462,0.71856,2013-10-04T02:21:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01176,snR78,1264,Small nucleolar RNA snR78,Wilkinson A,Predicted; WAR; Wilkinson A,48.0,48.2,44.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.52 -Z 549862.597050 CM SEQDB,8,37,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,43,0,,\"\",0.47833,-6.97919,1600000,290111,195,82,true,-3.4035,0.7186,2013-10-04T02:21:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01177,snR67,1264,Small nucleolar RNA snR67,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,62.7,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,64,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,73,0,,\"\",0.54362,-9.32818,1600000,343675,300,83,false,-3.2782,0.71852,2013-10-04T02:21:52.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01178,snoR77Y,1264,Small nucleolar RNA snoR77Y,Wilkinson A,Predicted; WAR; Wilkinson A,52.0,53.5,50.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,217,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,77,0,,\"\",0.59068,-4.03959,1600000,217132,170,85,true,-3.6125,0.71858,2013-10-04T02:21:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01179,snoU83,1264,Small nucleolar RNA U83,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,50.7,48.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.33 -Z 549862.597050 CM SEQDB,7,104,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,56,0,,\"\",0.52659,-5.60611,1600000,284303,168,84,true,-3.1247,0.71854,2013-02-01T11:58:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01180,snoR110,1264,Small nucleolar RNA snoR110,Wilkinson A,Predicted; WAR; Wilkinson A,44.0,44.6,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 549862.597050 CM SEQDB,2,25,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,8,0,,\"\",0.56315,-3.4773,1600000,198381,184,89,true,-3.2706,0.71863,2013-10-04T02:21:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01181,snR77,1264,Small nucleolar RNA snR77,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,54.2,42.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,68,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,74,0,,\"\",0.43306,-9.88275,1600000,282445,170,83,false,-3.5525,0.71854,2013-10-04T02:22:00.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01182,SNORD11,1264,Small nucleolar RNA SNORD11,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,57.4,52.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.85 -Z 549862.597050 CM SEQDB,19,109,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,101,0,,\"\",0.59781,-3.51665,1600000,222538,174,84,true,-3.7356,0.71853,2013-10-04T02:22:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01183,SNORD19B,1264,Small nucleolar RNA SNORD19B,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,51.6,48.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,17,43,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,46,0,,\"\",0.5261,-4.37444,1600000,222476,170,84,true,-3.4085,0.71856,2013-10-04T02:22:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01184,snR79,1264,Small nucleolar RNA snR79,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,58.2,50.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.66 -Z 549862.597050 CM SEQDB,18,62,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,66,0,,\"\",0.47897,-8.85132,1600000,298535,178,89,false,-2.9717,0.7184,2013-10-04T02:22:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01185,snR75,1264,Small nucleolar RNA snR75,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,43.0,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.82 -Z 549862.597050 CM SEQDB,61,459,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,468,0,,\"\",0.54588,-9.23022,1600000,343233,200,87,false,-3.4987,0.71846,2013-10-04T02:22:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01186,SNORD127,1264,Small nucleolar RNA SNORD127,Wilkinson A,\"Predicted; WAR; Wilkinson A, Daub J\",45.0,47.5,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.99 -Z 549862.597050 CM SEQDB,13,95,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,94,0,,\"\",0.53205,-4.26279,1600000,238659,177,86,true,-3.3069,0.71858,2013-10-04T02:22:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01188,snR56,1264,Small nucleolar RNA snR56,Wilkinson A,Predicted; WAR; Wilkinson A,54.0,54.1,53.6,Checked in using file,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.31 -Z 549862.597050 CM SEQDB,16,94,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,100,0,,\"\",0.41883,-11.10112,1600000,348926,176,85,false,-3.414,0.7185,2013-10-04T02:22:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01189,sn1502,1264,Small nucleolar RNA sn1502,Wilkinson A,Predicted; WAR; Wilkinson A,60.0,88.7,40.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.66641,-2.87711,1600000,227761,183,87,true,-3.4008,0.71856,2013-10-04T02:22:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01190,snR50,1264,Small nucleolar RNA snR50,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,49.6,41.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.43 -Z 549862.597050 CM SEQDB,3,11,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,13,0,,\"\",0.50828,-5.71144,1600000,276043,186,89,true,-3.0196,0.71837,2013-10-04T02:22:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01191,SNORD121A,1264,Small nucleolar RNA SNORD121A,Wilkinson A,Predicted; WAR; Wilkinson A,44.0,44.1,43.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.80 -Z 549862.597050 CM SEQDB,26,195,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,94,0,,\"\",0.59776,-3.6,1600000,226662,168,82,true,-3.5222,0.71862,2013-10-04T02:22:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01192,SNORD11B,1264,Small nucleolar RNA SNORD11B,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,51.5,49.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.48 -Z 549862.597050 CM SEQDB,37,109,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,119,0,,\"\",0.56109,-3.72753,1600000,205392,171,87,true,-3.7266,0.71851,2013-10-04T02:22:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01193,snoR20a,1264,Small nucleolar RNA snoR20a,Wilkinson A,Predicted; WAR; Wilkinson A,42.0,43.8,41.7,This family is based on the Oryza sativum homolog of Arabidposis thaliana snoR20 (which is included in model RF00351). The O.sativum sequence was obtained from the plant snoRNA database. The name originally given to the O.sativum homolog in PMID:12736310 is Z160a.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 549862.597050 CM SEQDB,5,145,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,61,0,,\"\",0.52689,-4.54398,1600000,269372,179,90,true,-2.9789,0.71848,2013-10-04T02:22:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01194,sn2903,1264,Small nucleolar RNA sn2903,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,52.5,43.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.72 -Z 549862.597050 CM SEQDB,4,22,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,21,0,,\"\",0.70615,-3.2117,1600000,260523,190,91,true,-3.2028,0.71846,2013-10-04T02:22:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01195,snR52,1264,Small nucleolar RNA snR52,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,45.7,43.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.00 -Z 549862.597050 CM SEQDB,5,33,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,36,0,,\"\",0.39629,-10.27296,1600000,276258,187,92,false,-2.9845,0.7184,2013-10-04T02:22:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01196,snoZ30a,1264,Small nucleolar RNA Z30a,Wilkinson A,Predicted; WAR; Wilkinson A,80.0,83.1,34.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,10,11,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,13,0,,\"\",0.71932,-3.01272,1600000,235338,185,92,true,-3.3447,0.71833,2013-10-04T02:22:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01197,snR39,1264,Small nucleolar RNA snR39,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,42.4,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.69 -Z 549862.597050 CM SEQDB,4,43,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,41,0,,\"\",0.55687,-5.22405,1600000,265513,195,95,true,-3.2936,0.71825,2013-10-04T02:23:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01198,snoR69Y,1264,Small nucleolar RNA snoR69Y,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,46.2,43.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.37 -Z 549862.597050 CM SEQDB,7,282,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,54,0,,\"\",0.56565,-4.99325,1600000,259001,194,99,true,-3.4938,0.71827,2013-10-04T02:23:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01199,snR58,1264,Small nucleolar RNA snR58,Wilkinson A,Predicted; WAR; Wilkinson A,70.0,76.9,64.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.85 -Z 549862.597050 CM SEQDB,3,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,12,0,,\"\",0.45787,-8.92805,1600000,236690,192,97,false,-3.5661,0.71834,2013-10-04T02:23:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01200,SNORD125,1264,Small nucleolar RNA SNORD125,Wilkinson A,Predicted; WAR; Wilkinson A,56.0,62.3,53.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,83,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,87,0,,\"\",0.57734,-4.71482,1600000,192577,192,97,true,-3.8237,0.71843,2013-10-04T02:23:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01201,snR40,1264,Small nucleolar RNA snR40,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,50.1,42.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,58,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,68,0,,\"\",0.52186,-9.37791,1600000,287964,181,93,false,-3.3496,0.71837,2013-10-04T02:23:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01202,sn2991,1264,Small nucleolar RNA sn2991,Wilkinson A,Predicted; WAR; Wilkinson A,60.0,66.3,39.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,8,0,,\"\",0.5908,-3.58384,1600000,214025,198,98,true,-3.4327,0.71837,2013-10-04T02:23:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01203,snR47,1264,Small nucleolar RNA snR47,Wilkinson A,Predicted; WAR; Wilkinson A,61.0,66.8,54.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,67,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,75,0,,\"\",0.38901,-8.54093,1600000,286477,188,96,true,-2.9004,0.71837,2013-10-04T02:23:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01204,snR65,1264,Small nucleolar RNA snR65,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,40.9,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 36 -Z 549862.597050 CM SEQDB,5,30,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,33,0,,\"\",0.39588,-7.87657,1600000,265121,197,100,false,-3.2111,0.71825,2013-10-04T02:23:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01205,snR62,1264,Small nucleolar RNA snR62,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,61.4,44.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34 -Z 549862.597050 CM SEQDB,2,10,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,12,0,,\"\",0.42024,-4.4209,1600000,223003,202,100,true,-2.8794,0.71821,2013-10-04T02:23:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01206,snoR109,1264,Small nucleolar RNA snoR109,Wilkinson A,Predicted; WAR; Wilkinson A,34.0,34.6,33.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,194,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,72,0,,\"\",0.52989,-5.08674,1600000,191713,256,92,true,-3.3176,0.71858,2013-10-04T02:23:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01207,snR73,1264,Small nucleolar RNA snR73,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.3,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,5,244,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,246,0,,\"\",0.47864,-9.58685,1600000,293826,199,103,false,-3.2274,0.71822,2013-10-04T02:23:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01208,snoR99,1264,Small nucleolar RNA snoR99,Wilkinson A,Predicted; WAR; Wilkinson A,41.0,41.1,40.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,251,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,74,0,,\"\",0.48207,-7.65993,1600000,269262,200,106,true,-3.1404,0.71838,2013-10-04T02:23:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01209,snR76,1264,Small nucleolar RNA snR76,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,59.0,51.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,69,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,78,0,,\"\",0.65135,-3.77483,1600000,253907,210,101,true,-3.2682,0.71832,2013-10-04T02:23:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01210,snoU13,1264,Small nucleolar RNA U13,Wilkinson A,Predicted; WAR; Wilkinson A,46.0,46.0,45.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,34,5445,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,97,0,,\"\",0.57262,-8.44604,1600000,283759,192,104,false,-3.8121,0.71821,2013-10-04T02:23:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01211,SNORD124,1264,Small nucleolar RNA SNORD124,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,41.9,38.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,51,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,56,0,,\"\",0.53184,-5.94046,1600000,218808,202,104,true,-3.5123,0.7183,2013-10-04T02:23:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01212,sn2524,1264,Small nucleolar RNA sn2524,Wilkinson A,Predicted; WAR; Wilkinson A,44.0,44.7,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.05 -Z 549862.597050 CM SEQDB,2,46,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.51618,-4.80731,1600000,189493,211,109,true,-3.2181,0.71811,2013-10-04T02:23:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01213,snoR103,1264,Small nucleolar RNA snoR103,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,51.2,44.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,295,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,76,0,,\"\",0.46272,-8.02122,1600000,217593,200,106,true,-3.3782,0.71843,2013-10-04T02:23:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01214,snR51,1264,Small nucleolar RNA snR51,\"Burge SW, Wilkinson A\",Predicted; WAR; Wilkinson A,44.0,44.3,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,127,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,131,0,,\"\",0.38726,-9.01288,1600000,257629,194,100,true,-2.9717,0.71838,2013-10-04T02:23:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01215,snoR97,1264,Small nucleolar RNA snoR97,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,40.4,39.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,263,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,74,0,,\"\",0.46438,-7.64345,1600000,219728,203,106,true,-3.4036,0.71839,2013-02-01T11:58:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01216,snR87,1264,Small nucleolar RNA snR87,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.7,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,40,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,42,0,,\"\",0.56749,-3.98699,1600000,205289,213,109,true,-3.1328,0.71817,2013-10-04T02:23:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01218,snR41,1264,Small nucleolar RNA snR41,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,68.8,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,63,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,72,0,,\"\",0.58362,-2.97757,1600000,250046,209,111,true,-3.031,0.71814,2013-02-01T12:07:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01219,snoR100,1264,Small nucleolar RNA snoR100,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,52.8,48.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,226,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,75,0,,\"\",0.5879,-5.73144,1600000,281495,216,116,true,-3.3538,0.71829,2013-10-04T02:23:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01220,snoR104,1264,Small nucleolar RNA snoR104,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,41.9,38.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,220,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,77,0,,\"\",0.49568,-8.34518,1600000,276722,216,118,true,-3.3011,0.71828,2013-10-04T02:23:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01221,snoU6-47,1264,Small nucleolar RNA U6-47,Wilkinson A,Predicted; WAR; Wilkinson A,54.0,54.7,51.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,5,13,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.49869,-7.34414,1600000,325598,271,132,true,-2.5015,0.718,2013-10-04T02:23:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01222,sn2417,1264,Small nucleolar RNA sn2417,Wilkinson A,Predicted; WAR; Wilkinson A,80.0,111.9,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.59411,-6.95469,1600000,291561,233,123,true,-3.0348,0.718,2013-10-04T02:23:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01223,snR13,1264,Small nucleolar RNA snR13,Wilkinson A,Predicted; WAR; Wilkinson A,47.0,48.6,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,53,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,54,0,,\"\",0.53896,-6.06367,1600000,279804,235,124,true,-2.8631,0.71815,2013-10-04T02:23:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01224,snoR80,1264,Small nucleolar RNA snoR80,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,46.7,42.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,248,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,72,0,,\"\",0.49611,-7.74811,1600000,270127,244,129,true,-2.6152,0.71824,2013-10-04T02:23:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01225,ACA64,1264,Small nucleolar RNA ACA64,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.0,34.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,30,224,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,101,0,,\"\",0.58541,-7.78587,1600000,238908,229,128,true,-3.4307,0.71818,2013-10-04T02:23:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01226,snoZ5,1264,Small nucleolar RNA Z5,Wilkinson A,Predicted; WAR; Wilkinson A,60.0,65.4,51.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Wilkinson A,13,0,,\"\",0.49993,-7.40848,1600000,276257,218,113,true,-3.1832,0.71816,2013-02-01T11:58:28.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01227,snoR83,1264,Small nucleolar RNA snoR83,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,40.3,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,301,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,78,0,,\"\",0.55414,-5.6952,1600000,178287,270,129,true,-3.5099,0.71825,2013-10-04T02:24:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01228,snoR111,1264,Small nucleolar RNA snoR111,Wilkinson A,Predicted; WAR; Wilkinson A,38.0,39.5,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,163,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,75,0,,\"\",0.54017,-5.93374,1600000,216090,235,128,true,-3.3987,0.71847,2013-10-04T02:24:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01229,SNORA84,1264,Small nucleolar RNA SNORA84,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,46.0,41.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,139,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,111,0,,\"\",0.59954,-6.74802,1600000,211808,237,134,true,-3.0578,0.71833,2013-10-04T02:24:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01230,snoR77,1264,Small nucleolar RNA snoR77,Wilkinson A,Predicted; WAR; Wilkinson A,47.0,47.5,45.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,226,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,74,0,,\"\",0.50342,-6.96834,1600000,221693,273,131,true,-3.3779,0.7182,2013-10-04T02:24:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01231,snoR74,1264,Small nucleolar RNA snoR74,Wilkinson A,Predicted; WAR; Wilkinson A,44.0,44.4,42.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,9,457,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,75,0,,\"\",0.4625,-8.93826,1600000,311309,239,134,true,-3.2002,0.7182,2013-10-04T02:24:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01232,snoR442,1264,Small nucleolar RNA R442,\"Wilkinson A, Eberhardt R\",Predicted; WAR; Eberhardt R,50.0,64.3,33.5,,snoU40,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,18,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR;,19,0,,\"\",0.68465,-4.04581,1600000,358219,119,46,true,-3.6412,0.71943,2013-10-04T02:24:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01233,snoU109,1264,Small nucleolar RNA U109,\"Burge SW, Wilkinson A\",Predicted; WAR; Wilkinson A,43.0,43.2,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,322,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,113,0,,\"\",0.49398,-7.11637,1600000,265265,247,134,true,-3.135,0.71817,2013-10-04T02:24:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01234,SNORA47,1264,Small nucleolar RNA SNORA47,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,46.3,44.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.10 -Z 549862.597050 CM SEQDB,20,118,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,108,0,,\"\",0.56765,-8.12392,1600000,256049,246,138,true,-3.2879,0.71825,2013-10-04T02:24:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01235,snR68,1264,Small nucleolar RNA snR68,\"Wilkinson A, Eberhardt R\",Predicted; ClustalW; Eberhardt R,80.0,128.5,45.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold;,12,0,,\"\",0.54063,-5.73129,1600000,245354,252,137,true,-3.1235,0.7183,2013-10-04T02:24:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01236,snoU19,1264,Small nucleolar RNA U19,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,45.3,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,3,118,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,70,0,,\"\",0.47455,-9.37043,1600000,346000,258,144,true,-2.7245,0.71817,2013-10-04T02:24:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01237,snR161,1264,Small nucleolar RNA snR161,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,54.9,45.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,30,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,38,0,,\"\",0.51549,-7.97648,1600000,264794,280,165,true,-3.51,0.71633,2013-10-04T02:24:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01238,snR70,1264,Small nucleolar RNA snR70,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,55.0,55.4,53.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,27,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,32,0,,\"\",0.65502,-4.0374,1600000,268677,284,165,true,-3.2458,0.7164,2013-10-04T02:24:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01239,snR49,1264,Small nucleolar RNA snR49,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,45.8,42.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,49,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,52,0,,\"\",0.55553,-8.14454,1600000,326064,307,170,true,-2.9828,0.71568,2013-02-01T12:07:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01240,snR85,1264,Small nucleolar RNA snR85,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.9,36.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,29,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,33,0,,\"\",0.53901,-8.28828,1600000,273676,305,180,true,-3.3567,0.71436,2013-10-04T02:24:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01241,SNORA81,1264,Small nucleolar RNA SNORA81,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,46.4,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,226,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,116,0,,\"\",0.51468,-9.83244,1600000,362955,297,179,true,-3.2168,0.71445,2013-10-04T02:24:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01242,snR36,1264,Small nucleolar RNA snR36,Wilkinson A,Predicted; WAR; Wilkinson A,43.0,43.7,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,303,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,304,0,,\"\",0.53453,-8.63941,1600000,303341,308,180,true,-3.291,0.71448,2013-10-04T02:24:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01243,snR33,1264,Small nucleolar RNA snR33,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,47.5,42.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,5,14,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,18,0,,\"\",0.5398,-8.29152,1600000,336002,311,183,true,-3.4098,0.71402,2013-10-04T02:24:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01244,snR4,1264,Small nucleolar RNA snR4,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,37.0,38.3,35.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,35,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,38,0,,\"\",0.56298,-7.86434,1600000,292137,312,187,true,-3.1276,0.71374,2013-10-04T02:24:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01245,snR9,1264,Small nucleolar RNA snR9,Wilkinson A,Predicted; WAR; Wilkinson A,58.0,60.2,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,11,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,15,0,,\"\",0.46255,-10.12748,1600000,378717,315,187,true,-3.4085,0.71355,2013-10-04T02:24:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01246,snR81,1264,Small nucleolar RNA snR81,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,52.5,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,9,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,11,0,,\"\",0.49426,-10.18211,1600000,331327,317,188,true,-3.3917,0.71355,2013-10-04T02:24:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01247,snR32,1264,Small nucleolar RNA snR32,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,45.6,44.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,11,95,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,101,0,,\"\",0.48349,-9.32044,1600000,309760,317,191,true,-3.4611,0.71315,2013-10-04T02:24:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01248,snR8,1264,Small nucleolar RNA snR8,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,60.1,36.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,64,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,73,0,,\"\",0.53451,-8.52118,1600000,318014,530,184,true,-3.6428,0.71394,2013-10-04T02:24:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01249,snR190,1264,Small nucleolar RNA snR190,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,40.0,42.4,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,42,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,48,0,,\"\",0.62927,-7.93247,1600000,308035,321,198,true,-3.365,0.71265,2013-10-04T02:24:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01250,snR189,1264,Small nucleolar RNA snR189,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,53.7,37.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.00 -Z 549862.597050 CM SEQDB,4,26,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,29,0,,\"\",0.47987,-9.1491,1600000,334328,323,193,true,-3.0804,0.71305,2013-10-04T02:24:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01251,snR3,1264,Small nucleolar RNA snR3,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,40.3,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,4,57,0,0,Gene; snRNA; snoRNA; HACA-box;,Published; PMID:9106664; Wilkinson A,59,0,,\"\",0.50847,-8.73737,1600000,337216,321,193,true,-3.1164,0.71297,2013-10-04T02:24:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01252,snR5,1264,Small nucleolar RNA snR5,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,46.4,40.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,58,0,0,Gene; snRNA; snoRNA; HACA-box;,Published; PMID:15075370; Wilkinson A,66,0,,\"\",0.50953,-9.24895,1600000,353026,314,196,true,-3.4593,0.71273,2013-10-04T02:24:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01253,snR46,1264,Small nucleolar RNA snR46,Wilkinson A,Predicted; WAR; Wilkinson A,43.0,43.7,42.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,55,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,59,0,,\"\",0.53499,-8.78309,1600000,283970,327,197,true,-3.3158,0.71264,2013-10-04T02:24:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01254,snR34,1264,Small nucleolar RNA snR34,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,40.7,36.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,42,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,46,0,,\"\",0.42041,-10.39566,1600000,316685,343,206,true,-3.3501,0.71168,2013-10-04T02:24:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01255,snR35,1264,Small nucleolar RNA snR35,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,41.8,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,59,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,66,0,,\"\",0.57634,-8.99732,1600000,339603,330,204,true,-3.5828,0.71194,2013-10-04T02:24:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01256,snR43,1264,Small nucleolar RNA snR43,Wilkinson A,Predicted; WAR; Wilkinson A,60.0,60.1,55.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,66,0,0,Gene; snRNA; snoRNA; HACA-box;,Published; PMID:18332121; Wilkinson A,74,0,,\"\",0.56742,-9.16013,1600000,377940,328,207,true,-3.8735,0.7116,2013-10-04T02:24:58.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01257,snR31,1264,Small nucleolar RNA snR31,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,48.2,43.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,37,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,43,0,,\"\",0.53414,-9.73461,1600000,328721,366,223,true,-3.7186,0.71034,2013-10-04T02:25:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01258,snR10,1264,Small nucleolar RNA snR10,Wilkinson A,Predicted; WAR; Wilkinson A,70.0,107.9,64.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,65,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,73,0,,\"\",0.56709,-10.17946,1600000,312451,414,245,true,-4.0166,0.70884,2013-10-04T02:25:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01259,snR63,1264,Small nucleolar RNA snR63,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,50.0,53.0,44.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,29,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,33,0,,\"\",0.70145,-7.95178,1600000,400641,399,255,true,-3.5644,0.7082,2013-10-04T02:25:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01260,snR11,1264,Small nucleolar RNA snR11,Wilkinson A,Predicted; WAR; Wilkinson A,42.0,42.3,40.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,6,31,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,35,0,,\"\",0.65572,-8.02774,1600000,364338,492,260,true,-3.5036,0.7079,2013-10-04T02:25:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01261,snR82,1264,Small nucleolar RNA snR82,Wilkinson A,Predicted; WAR; Wilkinson A,60.0,63.5,50.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,30,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,33,0,,\"\",0.57693,-10.38629,1600000,434112,422,270,true,-3.8154,0.70734,2013-10-04T02:25:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01262,snR44,1264,Small nucleolar RNA snR44,Wilkinson A,Predicted; WAR; Wilkinson A,33.0,33.6,32.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,250,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,247,0,,\"\",0.55027,-8.68136,1600000,285309,341,211,true,-3.3938,0.71135,2013-10-04T02:25:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01263,snR191,1264,Small nucleolar RNA snR191,Wilkinson A,Predicted; WAR; Wilkinson A,46.0,46.9,45.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,234,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,239,0,,\"\",0.61783,-9.05349,1600000,458239,409,273,true,-3.9076,0.7072,2013-10-04T02:25:13.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01264,snR83,1264,Small nucleolar RNA snR83,Wilkinson A,Predicted; WAR; Wilkinson A,75.0,77.4,73.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.42 -Z 742849.287494 --mxsize 128 CM SEQDB,5,17,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,21,0,,\"\",0.59366,-9.59562,1600000,385881,463,308,true,-4.0242,0.70557,2013-10-04T02:25:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01265,snR42,1264,Small nucleolar RNA snR42,Wilkinson A,Predicted; WAR; Wilkinson A,61.0,62.3,60.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,39,0,0,Gene; snRNA; snoRNA; HACA-box;,Published; PMID:9106664; Wilkinson A,42,0,,\"\",0.67488,-8.63223,1600000,436785,509,349,true,-4.0783,0.70422,2013-10-04T02:25:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01266,snR45,1264,Small nucleolar RNA snR45,Wilkinson A,Predicted; WAR; Wilkinson A,80.0,120.5,71.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34 -Z 549862.597050 CM SEQDB,8,31,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,35,0,,\"\",0.35878,-13.13822,1600000,391964,312,180,false,-3.4439,0.71418,2013-10-04T02:25:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01267,snR37,1264,Small nucleolar RNA snR37,Wilkinson A,Predicted; WAR; Wilkinson A,47.0,48.5,44.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,50,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,56,0,,\"\",0.66044,-8.6193,1600000,416151,588,385,true,-4.4265,0.70315,2013-10-04T02:25:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01268,SCARNA2,1264,Small Cajal body-specific RNA 2,Wilkinson A,Predicted; WAR; Wilkinson A,48.0,48.6,46.5,,snoU2-25,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,83,0,0,Gene; snRNA; snoRNA; scaRNA;,Published; PMID:15556860; Wilkinson A,68,0,,\"\",0.64279,-9.55235,1600000,374250,666,421,true,-4.4197,0.70224,2013-10-04T02:25:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01269,snR80,1264,Small nucleolar RNA snR80,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,40.2,39.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,6,198,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,175,0,,\"\",0.46275,-8.80459,1600000,336609,299,176,true,-3.0438,0.71493,2013-10-04T02:25:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01270,snR84,1264,Small nucleolar RNA snR84,Wilkinson A,Predicted; WAR; Wilkinson A,109.0,129.6,62.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,9,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.71726,-8.41602,1600000,438398,777,568,true,-4.5212,0.69989,2013-10-04T02:25:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01271,snR30,1264,Small nucleolar RNA snR30,Wilkinson A,Predicted; WAR; Wilkinson A,80.0,96.6,74.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.51 -Z 742849.287494 --mxsize 128 CM SEQDB,5,30,0,0,Gene; snRNA; snoRNA; HACA-box;,Published; PMID:14749391; Wilkinson A,34,0,,\"\",0.72756,-8.34058,1600000,316984,819,612,true,-5.0357,0.69939,2013-10-04T02:25:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01272,snR86,1264,Small nucleolar RNA snR86,Wilkinson A,Predicted; WAR; Wilkinson A,120.0,126.7,104.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,9,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,13,0,,\"\",0.63548,-10.65111,1600000,441461,1253,1008,true,-5.4068,0.69697,2013-10-04T02:25:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01273,sR34,1264,Small nucleolar RNA sR34,\"Wilkinson A, Eberhardt R\",Predicted; WAR; Eberhardt R,40.0,63.0,35.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,12,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR;,13,0,,\"\",0.54366,-8.30156,1600000,446646,134,54,true,-3.5246,0.71923,2013-10-04T02:25:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01274,sR45,1264,Small nucleolar RNA sR45,\"Wilkinson A, Eberhardt R\",Predicted; WAR; Eberhardt R,40.0,43.0,33.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,11,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR;,15,0,,\"\",0.69187,-3.59192,1600000,198811,147,55,true,-3.9055,0.71935,2013-10-04T02:25:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01275,sR22,1264,Small nucleolar RNA sR22,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,40.0,74.3,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,11,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,14,0,,\"\",0.64125,-5.04349,1600000,380079,140,58,true,-3.5013,0.71924,2013-10-04T02:25:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01276,sR53,1264,Small nucleolar RNA sR53,\"Wilkinson A, Eberhardt R\",Predicted; WAR; Eberhardt R,49.0,49.7,48.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.77 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR;,4,0,,\"\",0.60587,-9.36075,1600000,451158,140,60,false,-3.6098,0.71892,2013-10-04T02:25:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01277,snoU54,1264,Small nucleolar RNA U54,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.6,34.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,127,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,115,0,,\"\",0.63043,-4.41481,1600000,283595,144,67,true,-3.5182,0.71886,2013-10-04T02:25:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01278,SNORD109A,1264,Small nucleolar RNA SNORD109A,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.4,34.6,\"Humans have two copies of the SNORD109 gene, SNORD109A and SNORD109B.\",SNORD109B,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,142,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,37,0,,\"\",0.65054,-3.90758,1600000,322235,168,67,true,-3.6832,0.71884,2013-10-04T02:25:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01279,snoR53Y,1264,Small nucleolar RNA snoR53Y,Wilkinson A,Predicted; WAR; Wilkinson A,43.0,44.4,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,176,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,78,0,,\"\",0.57349,-4.3851,1600000,238590,193,79,true,-3.2848,0.71866,2013-10-04T02:25:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01280,snoR14,1264,Small nucleolar RNA snoR14,Wilkinson A,Predicted; Wilkinson A,40.0,43.0,37.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,185,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Wilkinson A,75,0,,\"\",0.61575,-4.36874,1600000,292783,190,84,true,-3.4277,0.71852,2013-10-04T02:25:48.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01281,snoR35,1264,Small nucleolar RNA snoR35,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,40.0,44.5,35.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,199,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,76,0,,\"\",0.56423,-8.82413,1600000,320488,165,81,true,-3.7698,0.71861,2013-10-04T02:25:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01283,snoU30,1264,Small nucleolar RNA U30,Wilkinson A,Predicted; Wilkinson A,45.0,48.3,41.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,103,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Wilkinson A,54,0,,\"\",0.55931,-5.7754,1600000,320015,177,86,true,-3.1058,0.7185,2013-10-04T02:25:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01284,snoR8a,1264,Small nucleolar RNA snoR8a,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.0,36.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,383,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,81,0,,\"\",0.57058,-3.28715,1600000,233587,182,92,true,-3.1052,0.71848,2013-10-04T02:25:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01285,snoU31b,1264,Small nucleolar RNA U31b,Wilkinson A,Predicted; Wilkinson A,37.0,37.8,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,446,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Wilkinson A,84,0,,\"\",0.58513,-5.53985,1600000,313090,174,86,true,-3.4302,0.71852,2013-10-04T02:25:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01286,snoR26,1264,Small nucleolar RNA snoR26,Wilkinson A,Predicted; WAR; Wilkinson A,47.0,49.0,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,203,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,73,0,,\"\",0.58772,-3.80297,1600000,239513,186,94,true,-3.1122,0.71848,2013-10-04T02:25:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01287,snoR27,1264,Small nucleolar RNA snoR27,Wilkinson A,Predicted; WAR; Wilkinson A,41.0,42.0,40.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,175,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,61,0,,\"\",0.5035,-4.48121,1600000,238170,186,95,true,-2.9129,0.71844,2013-10-04T02:25:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01288,snoR31,1264,Small nucleolar RNA snoR31,Wilkinson A,Predicted; WAR; Wilkinson A,44.0,44.9,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,193,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,72,0,,\"\",0.51644,-5.46869,1600000,289943,193,100,true,-3.3506,0.71833,2013-10-04T02:25:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01289,snoR17,1264,Small nucleolar RNA snoR17,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,60.0,66.7,42.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,20,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,8,0,,\"\",0.45986,-8.23066,1600000,251922,211,110,true,-3.1961,0.71826,2013-10-04T02:26:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01290,SNORD10,1264,Small nucleolar RNA SNORD10,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.6,34.7,,snoU6-77,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,90,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,69,0,,\"\",0.59381,-7.52219,1600000,191418,246,139,true,-3.2292,0.71795,2013-10-04T02:26:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01291,SNORD97,1264,Small nucleolar RNA SNORD97,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,47.5,41.8,,snoU97,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,109,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,108,0,,\"\",0.51126,-7.86006,1600000,280174,602,142,true,-3.1928,0.71788,2013-10-04T02:26:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01292,snoR2,1264,Small nucleolar RNA snoR2/U65,Wilkinson A,Predicted; WAR; Wilkinson A,44.0,45.1,42.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,293,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,79,0,,\"\",0.53849,-7.49153,1600000,221291,275,149,true,-3.4311,0.71814,2013-10-04T02:26:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01293,ACA59,1264,Small nucleolar RNA ACA59,Wilkinson A,Predicted; WAR; Wilkinson A,43.0,44.9,40.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,16,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.56887,-8.65453,1600000,323479,357,151,true,-2.9891,0.71797,2013-10-04T02:26:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01294,snoU89,1264,Small nucleolar RNA U89,Wilkinson A,Predicted; Wilkinson A,44.0,44.1,43.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,51,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; Wilkinson A,41,0,,\"\",0.71081,-9.26683,1600000,385744,429,270,true,-3.9631,0.70736,2013-10-04T02:26:12.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01295,SCARNA7,1264,Small Cajal body-specific RNA 7,Wilkinson A,Predicted; Wilkinson A,65.0,67.9,51.2,,snoU90,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,101,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; Wilkinson A,100,0,,\"\",0.5771,-13.35157,1600000,707504,518,330,true,-3.7672,0.70463,2013-10-04T02:26:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01296,snoU85,1264,Small nucleolar RNA U85,Wilkinson A,Predicted; Locarna; Wilkinson A,41.0,42.0,40.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,182,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; Locarna; Wilkinson A,140,0,,\"\",0.65982,-10.32471,1600000,443986,467,329,true,-3.9566,0.70472,2013-10-04T02:28:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01297,sR40,1264,Small nucleolar RNA sR40,\"Wilkinson A, Eberhardt R\",Predicted; WAR; Eberhardt,50.0,97.4,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR;,4,0,,\"\",0.55095,-9.67225,1600000,560758,148,61,true,-3.545,0.7242,2013-10-04T02:28:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01298,snoU25,1264,Small nucleolar RNA U25,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,98.4,35.6,,snoU35,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,5,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.66142,-3.45541,1600000,254859,158,69,true,-3.055,0.7189,2013-10-04T02:28:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01299,SNORD2,1264,Small nucleolar RNA SNORD2,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.3,36.8,,snR39B,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,213,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,130,0,,\"\",0.64193,-3.24459,1600000,244327,183,70,true,-3.4951,0.71879,2013-10-04T02:28:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01300,snoU49,1264,Small nucleolar RNA U49,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,35.0,37.1,33.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,93,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,46,0,,\"\",0.60871,-5.36027,1600000,320871,147,65,true,-3.5274,0.71895,2013-10-04T02:28:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01301,snoR4a,1264,Small nucleolar RNA snoR4a,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,58.5,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,36,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,8,0,,\"\",0.57847,-3.91129,1600000,260374,169,77,true,-3.6553,0.71903,2013-10-04T02:28:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01302,snoU36a,1264,SnoRNA U36a,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,46.2,43.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.59 -Z 549862.597050 CM SEQDB,15,355,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,90,0,,\"\",0.61869,-4.17906,1600000,290423,179,90,true,-3.235,0.71843,2013-10-04T02:28:13.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01303,sR49,1264,Small nucleolar RNA sR49,\"Wilkinson A, Eberhardt R\",Predicted; MAFFT; Eberhardt R,50.0,61.6,34.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold;,3,0,,\"\",0.65946,-4.66965,1600000,423132,137,56,true,-3.939,0.72057,2013-10-04T02:28:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01304,sR5,1264,Small nucleolar RNA sR5,\"Wilkinson A, Eberhardt R\",Predicted; ClustalW; Eberhardt R,40.0,42.3,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,15,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold;,18,0,,\"\",0.62599,-8.71987,1600000,535411,122,53,true,-3.8583,0.72508,2013-10-04T02:28:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01305,sR51,1264,Small nucleolar RNA sR51,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,38.0,39.2,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,9,0,,\"\",0.64904,-4.35262,1600000,281749,134,56,true,-3.9724,0.71971,2013-10-04T02:28:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01306,sR52,1264,Small nucleolar RNA sR52,\"Wilkinson A, Eberhardt R\",Predicted; WAR; Eberhardt R,50.0,53.2,41.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR;,2,0,,\"\",0.57321,-10.00812,1600000,679560,145,59,true,-2.969,0.72792,2013-10-04T02:28:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01307,sR55,1264,Small nucleolar RNA sR55,\"Wilkinson A, Eberhardt R\",Predicted; WAR; Eberhardt R,35.0,44.5,33.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.20 -Z 549862.597050 CM SEQDB,4,11,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR;,13,0,,\"\",0.71441,-4.90913,1600000,367031,147,64,true,-3.8878,0.71888,2013-10-04T02:28:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01308,sR58,1264,Small nucleolar RNA sR58,Wilkinson A,Predicted; WAR; Wilkinson A,55.0,71.1,31.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,11,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.65768,-5.93647,1600000,426194,134,55,true,-3.6965,0.71911,2013-10-04T02:28:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01309,sR60,1264,Small nucleolar RNA sR60,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,62.8,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Wilkinson A,13,0,,\"\",0.69196,-5.56953,1600000,519307,135,55,true,-3.8167,0.71906,2013-10-04T02:28:27.000Z,2022-10-29T17:55:16.000Z\n",
            "RF01310,sR7,1264,Small nucleolar RNA sR7,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,37.0,38.2,33.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.26 -Z 549862.597050 CM SEQDB,5,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,10,0,,\"\",0.60193,-8.83414,1600000,391860,132,58,false,-3.5182,0.72348,2013-10-04T02:28:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01312,sR9,1264,Small nucleolar RNA sR9,\"Wilkinson A, Eberhardt R\",Predicted; CMfinder; Eberhardt R,36.0,36.1,33.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.54 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; CMfinder;,6,0,,\"\",0.62313,-4.556,1600000,217403,138,55,true,-2.4216,0.74986,2013-10-04T02:28:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01313,AHBV_epsilon,2062,Avian HBV RNA encapsidation signal epsilon,\"Chen A, Brown C, Daub J\",\"Chen A, Brown C\",45.0,46.5,36.1,,HHBV_epsilon,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,4,0,0,Cis-reg;,Published; PMID:17430968,6,0,,\"\",0.64045,-4.17371,1600000,330670,138,57,true,-3.8012,0.71932,2013-10-04T02:28:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01314,mir-1227,1287,microRNA mir-1227,Griffiths-Jones SR,Griffiths-Jones SR,70.0,81.9,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.55983,-3.8613,1600000,195002,185,88,true,-3.1842,0.71888,2013-10-04T02:28:34.000Z,2021-12-03T12:04:12.000Z\n",
            "RF01315,CRISPR-DR2,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.0,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 549862.597050 CM SEQDB,19,9051,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,124,0,,\"\",0.61123,-4.2153,1600000,476169,88,30,true,-3.1224,0.73164,2013-02-01T11:58:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01316,CRISPR-DR3,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,41.0,41.2,40.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29 -Z 549862.597050 CM SEQDB,20,1307,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,43,0,,\"\",0.42211,-10.31268,1600000,793254,88,30,true,-3.5899,0.72622,2013-10-04T02:29:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01317,CRISPR-DR4,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.0,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.77 -Z 549862.597050 CM SEQDB,33,10236,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,264,0,,\"\",0.70862,-3.0975,1600000,570316,83,28,true,-3.4739,0.72986,2013-10-04T02:29:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01318,CRISPR-DR5,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,40.0,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,12,4678,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,52,0,,\"\",0.5194,-6.40185,1600000,661721,100,37,true,-3.5224,0.72484,2013-10-04T02:29:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01319,CRISPR-DR6,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,39.4,38.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,615,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.64843,-4.79057,1600000,548447,95,30,true,-3.2603,0.72657,2013-10-04T02:29:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01320,CRISPR-DR7,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.0,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,2400,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,68,0,,\"\",0.62017,-4.11298,1600000,318921,102,37,true,-3.5051,0.72537,2013-10-04T02:29:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01321,CRISPR-DR8,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,39.5,36.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,72,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,5,0,,\"\",0.67522,-3.54433,1600000,312220,107,37,true,-2.8265,0.73894,2013-10-04T02:29:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01322,CRISPR-DR9,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,45.4,44.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,1362,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,58,0,,\"\",0.68789,-3.8889,1600000,403954,104,37,true,-3.3037,0.72772,2013-10-04T02:29:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01323,CRISPR-DR10,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,39.8,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,294,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,7,0,,\"\",0.6827,-3.54794,1600000,241430,107,37,true,-3.4377,0.73625,2013-10-04T02:29:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01324,CRISPR-DR11,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,56.0,35.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,38,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.59969,-5.42995,1600000,521981,106,36,true,-3.2516,0.72591,2013-10-04T02:29:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01325,CRISPR-DR12,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.3,36.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,257,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,10,0,,\"\",0.65714,-4.29933,1600000,463675,98,30,true,-3.3221,0.73286,2013-10-04T02:29:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01326,CRISPR-DR13,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.1,36.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,4,130,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.49839,-7.81157,1600000,575318,99,30,true,-3.1027,0.73344,2013-10-04T02:29:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01327,CRISPR-DR14,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,45.3,44.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.00 -Z 549862.597050 CM SEQDB,5,184,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,7,0,,\"\",0.42824,-9.37412,1600000,608214,94,29,false,-2.998,0.73428,2013-10-04T02:29:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01328,CRISPR-DR17,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,42.0,42.3,40.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,45,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.6145,-4.88118,1600000,693903,94,25,true,-2.2914,0.75385,2013-10-04T02:29:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01329,CRISPR-DR15,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,36.0,36.0,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.87 -Z 549862.597050 CM SEQDB,3,104,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,21,0,,\"\",0.65066,-3.75687,1600000,356421,111,37,true,-2.8565,0.74162,2013-10-04T02:29:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01330,CRISPR-DR16,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.0,34.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,589,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,27,0,,\"\",0.72145,-3.43505,1600000,462708,106,37,true,-3.2736,0.72358,2013-10-04T02:29:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01331,CRISPR-DR18,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,56.5,39.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,28,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.50916,-8.62235,1600000,487603,106,37,true,-3.2779,0.72662,2013-10-04T02:29:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01332,CRISPR-DR19,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,41.2,38.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,405,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,5,0,,\"\",0.55922,-5.45642,1600000,378375,109,37,true,-3.4891,0.73439,2013-10-04T02:29:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01333,CRISPR-DR20,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,42.0,44.1,35.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,38,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.64599,-4.45633,1600000,434399,110,36,true,-2.4088,0.74379,2013-10-04T02:29:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01334,CRISPR-DR21,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,48.0,50.0,44.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,276,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.57762,-4.25613,1600000,344087,114,37,true,-2.6487,0.74811,2013-10-04T02:29:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01335,CRISPR-DR22,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,49.0,49.2,48.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,478,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,40,0,,\"\",0.57723,-9.16572,1600000,931621,101,36,true,-3.6937,0.726,2013-10-04T02:29:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01336,CRISPR-DR23,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,48.9,39.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,3,148,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.5785,-5.78584,1600000,369317,110,36,true,-3.0919,0.73379,2013-10-04T02:29:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01337,CRISPR-DR24,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.9,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,4,1217,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,35,0,,\"\",0.54045,-8.61739,1600000,618257,99,30,true,-3.221,0.73871,2013-02-01T11:58:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01338,CRISPR-DR25,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,36.0,36.4,35.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,303,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.64216,-4.72177,1600000,751638,91,25,true,-3.0301,0.73965,2013-10-04T02:29:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01339,CRISPR-DR27,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.1,34.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,3,130,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.53977,-5.32285,1600000,455495,94,25,true,-2.6435,0.75383,2013-10-04T02:30:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01340,CRISPR-DR29,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,39.7,38.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,129,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,12,0,,\"\",0.66173,-3.67097,1600000,294775,114,37,true,-2.4313,0.74426,2013-10-04T02:30:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01341,CRISPR-DR30,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,54.3,34.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.8 -Z 549862.597050 CM SEQDB,3,19,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.55276,-6.41814,1600000,273270,113,38,true,-3.0553,0.73776,2013-10-04T02:30:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01342,CRISPR-DR32,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,48.0,49.6,42.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,34,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.58447,-3.75746,1600000,278213,114,37,true,-2.7109,0.75334,2013-10-04T02:30:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01343,CRISPR-DR33,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,39.3,38.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,178,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.69029,-2.54233,1600000,245739,111,37,true,-2.8423,0.75164,2013-10-04T02:30:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01344,CRISPR-DR34,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,36.0,34.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 549862.597050 CM SEQDB,5,377,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,31,0,,\"\",0.64415,-3.43253,1600000,391656,106,36,true,-3.6656,0.72664,2013-10-04T02:30:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01345,CRISPR-DR35,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.3,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,129,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,5,0,,\"\",0.62356,-4.11747,1600000,323652,113,36,true,-2.6599,0.74919,2013-10-04T02:30:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01346,CRISPR-DR36,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,38.0,38.4,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,67,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.60889,-8.38716,1600000,670594,110,36,true,-3.4385,0.7325,2013-10-04T02:30:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01347,CRISPR-DR37,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,41.6,35.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,3,182,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,9,0,,\"\",0.60982,-3.25037,1600000,310255,109,35,true,-2.8873,0.7451,2013-10-04T02:30:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01348,CRISPR-DR38,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,44.0,45.8,42.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,109,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,5,0,,\"\",0.49252,-8.63807,1600000,446276,104,35,true,-3.3458,0.7318,2013-10-04T02:30:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01349,CRISPR-DR40,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,42.0,42.6,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.5 -Z 549862.597050 CM SEQDB,3,0,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.5461,-3.21799,1600000,328456,101,30,true,-2.6034,0.74459,2013-10-04T02:30:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01350,CRISPR-DR41,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,36.0,36.2,35.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,36,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.66621,-2.8938,1600000,359337,103,29,true,-2.5082,0.76609,2013-10-04T02:30:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01351,CRISPR-DR42,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,41.6,33.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.5 -Z 549862.597050 CM SEQDB,3,5,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.47496,-7.16751,1600000,409137,98,29,false,-2.774,0.74524,2013-10-04T02:30:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01352,CRISPR-DR43,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.0,34.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,3,985,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,28,0,,\"\",0.5281,-5.93493,1600000,419708,98,29,false,-2.7599,0.74982,2013-10-04T02:30:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01353,CRISPR-DR44,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.8,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.8 -Z 549862.597050 CM SEQDB,3,33,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.56487,-4.18823,1600000,473658,98,28,true,-2.4702,0.75518,2013-10-04T02:30:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01354,CRISPR-DR45,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.1,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,323,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.57977,-4.66301,1600000,491519,93,24,true,-2.5722,0.75617,2013-10-04T02:30:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01355,CRISPR-DR26,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,39.4,38.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.02 -Z 742849.287494 CM SEQDB,4,45,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,3,0,,\"\",0.53297,-7.80611,1600000,720616,92,25,true,-2.8905,0.74446,2013-10-04T02:30:46.000Z,2022-10-29T17:55:16.000Z\n",
            "RF01356,CRISPR-DR28,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.0,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,614,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,18,0,,\"\",0.56629,-5.76719,1600000,448432,93,24,true,-2.5769,0.7547,2013-10-04T02:30:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01357,CRISPR-DR31,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,35.6,34.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,19,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.67115,-3.7464,1600000,321571,111,37,true,-3.4557,0.73453,2013-10-04T02:30:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01358,CRISPR-DR39,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,39.2,38.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,3,136,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,3,0,,\"\",0.5235,-4.64152,1600000,394790,103,31,true,-2.6372,0.74956,2013-10-04T02:30:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01359,CRISPR-DR46,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,52.9,43.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.51716,-5.69497,1600000,365902,116,38,true,-2.062,0.75154,2013-10-04T02:30:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01360,CRISPR-DR47,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,40.1,36.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,2,43,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.54716,-7.80444,1600000,554725,114,37,true,-2.8245,0.74368,2013-10-04T02:30:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01361,CRISPR-DR48,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,60.6,37.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,9,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.52842,-6.95039,1600000,429615,114,37,true,-2.904,0.74676,2013-10-04T02:31:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01362,CRISPR-DR49,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,41.0,41.0,40.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,214,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,23,0,,\"\",0.5075,-8.35611,1600000,556993,125,47,true,-3.221,0.72618,2013-10-04T02:31:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01363,CRISPR-DR50,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,47.0,48.1,44.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,386,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,12,0,,\"\",0.53509,-3.63376,1600000,237922,127,46,true,-2.1599,0.74273,2013-10-04T02:31:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01364,CRISPR-DR51,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,51.9,33.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,23,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.55188,-3.68422,1600000,303050,116,38,true,-2.6055,0.75479,2013-10-04T02:31:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01365,CRISPR-DR52,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,50.0,61.7,35.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,50,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.57714,-6.02921,1600000,461984,114,37,true,-2.5895,0.74639,2013-10-04T02:31:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01366,CRISPR-DR53,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,51.0,39.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,14,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.60669,-3.91305,1600000,278609,114,37,true,-2.8577,0.74784,2013-10-04T02:31:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01367,CRISPR-DR54,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,39.2,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,21,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.64677,-3.46642,1600000,316219,114,37,true,-2.3478,0.74694,2013-10-04T02:31:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01368,CRISPR-DR55,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,41.0,42.4,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29 -Z 549862.597050 CM SEQDB,2,45,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.50647,-4.92552,1600000,369785,112,37,false,-2.1318,0.75237,2013-10-04T02:31:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01369,CRISPR-DR56,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,35.0,36.0,34.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,100,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.62957,-3.91246,1600000,336976,114,37,true,-2.5066,0.74545,2013-10-04T02:31:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01370,CRISPR-DR57,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,39.0,39.0,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,439,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,14,0,,\"\",0.68028,-3.76131,1600000,318048,111,37,true,-3.2177,0.7399,2013-10-04T02:31:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01371,CRISPR-DR58,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,38.0,39.0,34.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,100,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.6225,-4.14656,1600000,333214,114,37,true,-2.7896,0.74719,2013-10-04T02:31:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01373,CRISPR-DR60,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,50.1,39.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,2,169,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.60988,-3.3929,1600000,312839,114,37,true,-2.2626,0.75502,2013-10-04T02:31:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01374,CRISPR-DR61,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,38.0,38.6,36.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,4,0,,\"\",0.56158,-4.58849,1600000,368759,114,37,true,-2.4454,0.74798,2013-10-04T02:31:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01375,CRISPR-DR62,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,58.9,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.10 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.46483,-6.12998,1600000,454382,114,37,true,-2.0687,0.74824,2013-10-04T02:31:42.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01376,CRISPR-DR63,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,37.0,37.6,36.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,38,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,6,0,,\"\",0.49894,-7.08395,1600000,424873,114,37,true,-2.3823,0.74677,2013-10-04T02:31:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01377,CRISPR-DR64,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,40.0,61.1,37.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,0,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,1,0,,\"\",0.52647,-7.06847,1600000,384280,114,37,true,-2.6667,0.7453,2013-10-04T02:31:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01378,CRISPR-DR65,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,43.0,55.4,38.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.58512,-2.4654,1600000,280622,114,37,true,-2.4401,0.75526,2013-10-04T02:31:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01379,CRISPR-DR66,1265,CRISPR RNA direct repeat element,Wilkinson A,Predicted; WAR; Wilkinson A,45.0,50.6,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,126,0,0,Gene; CRISPR;,Predicted; WAR; Wilkinson A,2,0,,\"\",0.56964,-5.28671,1600000,417776,109,37,true,-3.3933,0.72668,2013-10-04T02:31:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01380,HIV-1_SD,1457,Human immunodeficiency virus type 1 major splice donor,\"Chen A, Brown C, Daub J\",Chen A,30.0,30.4,29.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.67 -Z 742849.287494 --mxsize 128 CM SEQDB,22,19,0,0,Cis-reg;,Published; PMID:18160437,19,0,,\"\",0.67105,-3.66922,1600000,528027,74,19,true,-3.6215,0.73887,2013-10-04T02:31:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01381,HIV-1_SL3,1457,HIV-1 stem-loop 3 Psi packaging signal,\"Chen A, Brown C, Daub J\",Chen A,33.0,33.0,32.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.00 -Z 549862.597050 CM SEQDB,19,14,0,0,Cis-reg;,Published; PMID:18713870,10,0,,\"\",0.76425,-3.92123,1600000,871521,79,23,true,-3.7355,0.73254,2013-10-04T02:31:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01382,HIV-1_SL4,1457,HIV-1 stem-loop 4 packaging signal,\"Chen A, Brown C, Daub J\",Chen A,31.0,31.1,30.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.20 -Z 549862.597050 CM SEQDB,16,28,0,0,Cis-reg;,Published; PMID:18713870,20,0,,\"\",0.64216,-5.49278,1600000,545209,77,20,true,-3.3051,0.73925,2013-10-04T02:32:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01383,GRIK4_3p_UTR,2063,GRIK4 3 prime UTR element,Gardner PP,Predicted; CMfinder,38.0,39.2,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.72 -Z 549862.597050 CM SEQDB,8,18,0,0,Gene;,Predicted; CMfinder,19,0,,\"\",0.60605,-5.57229,1600000,272717,195,94,true,-3.1848,0.71851,2013-10-04T02:32:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01384,InvR,2064,Invasion gene-associated RNA,Daub J,PMID 17971080,60.0,101.8,35.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,5,0,0,Gene;,Predicted; RNAalifold,9,0,,\"\",0.65078,-2.93355,1600000,273531,188,91,true,-3.3382,0.71861,2013-10-04T02:32:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01385,isrA,1266,isrA Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,51.0,53.0,49.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,38,0,0,Gene; sRNA;,Predicted; WAR,31,0,,\"\",0.60409,-8.13116,1600000,352219,239,129,true,-2.9806,0.71827,2013-02-01T11:58:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01386,isrB,1266,isrB Hfq binding RNA,Barquist L,Predicted; WAR; Infernal,35.0,38.5,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; sRNA;,Predicted; WAR,7,0,,\"\",0.51817,-8.81311,1600000,318638,181,87,true,-3.423,0.71885,2013-02-01T11:58:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01387,isrC,1266,isrC Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,39.0,39.7,38.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,16,15,0,0,Gene; sRNA;,Predicted; WAR,25,0,,\"\",0.65447,-3.80164,1600000,256500,231,123,true,-3.2926,0.71837,2013-10-04T02:32:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01388,isrD,1266,isrD Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,37.0,38.3,36.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.24 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; WAR,7,0,,\"\",0.57653,-7.56495,1600000,476418,134,51,true,-2.9923,0.73219,2013-10-04T02:32:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01389,isrF,1266,isrF Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,40.0,45.1,29.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; WAR,7,0,,\"\",0.66399,-10.97292,1600000,569196,442,292,true,-3.8857,0.70625,2013-10-04T02:32:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01390,isrG,1266,isrG Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; ClustalW; Infernal,100.0,100.6,99.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,129,0,0,Gene; sRNA;,Predicted; RNAalifold,14,0,,\"\",0.67348,-9.65561,1600000,431341,428,279,true,-3.8517,0.7069,2013-10-04T02:32:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01391,isrH,1266,isrH Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; ClustalW; Infernal,45.0,49.2,38.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,5,0,0,Gene; sRNA;,Predicted; RNAalifold,10,0,,\"\",0.72676,-7.7193,1600000,346609,431,280,true,-3.5616,0.70691,2013-10-04T02:32:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01392,isrI,1266,isrI Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,45.0,46.6,43.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,30,9,0,0,Gene; sRNA;,Predicted; WAR,27,0,,\"\",0.6394,-2.05088,1600000,304914,194,98,true,-3.4625,0.71841,2013-10-04T02:32:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01393,isrJ,1266,isrJ Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; ClustalW; Infernal,37.0,37.4,36.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,5,7,0,0,Gene; sRNA;,Predicted; RNAalifold,11,0,,\"\",0.5342,-7.89245,1600000,292475,162,75,true,-3.325,0.71889,2013-10-04T02:32:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01394,isrK,1266,isrK Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; CMfinder; Infernal,50.0,50.0,49.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.00 -Z 549862.597050 CM SEQDB,52,385,0,0,Gene; sRNA;,Predicted; CMfinder,258,0,,\"\",0.58895,-8.90962,1600000,444744,148,77,true,-3.5651,0.71882,2013-10-04T02:32:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01395,isrL,1266,isrL Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; MAFFT; Infernal,35.0,36.4,34.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,16,0,0,Gene; sRNA;,Predicted; RNAalifold,20,0,,\"\",0.73969,-7.71588,1600000,313726,515,345,true,-4.2757,0.70429,2013-10-04T02:32:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01396,isrN,1266,isrN Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,44.0,44.6,43.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,123,0,0,Gene; sRNA;,Predicted; WAR,87,0,,\"\",0.59852,-8.74901,1600000,302885,238,143,true,-3.0945,0.71805,2013-10-04T02:32:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01397,isrO,1266,isrO Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; ClustalW; Infernal,42.0,42.8,38.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,16,0,0,Gene; sRNA;,Predicted; RNAalifold,20,0,,\"\",0.56729,-9.52941,1600000,344778,328,201,true,-3.5211,0.71235,2013-10-04T02:32:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01398,isrP,1266,isrP Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; ClustalW; Infernal,45.0,45.1,42.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,4,0,0,Gene; sRNA;,Predicted; RNAalifold,11,0,,\"\",0.54799,-8.23973,1600000,314797,267,148,true,-3.376,0.71818,2013-10-04T02:32:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01399,isrQ,1266,isrQ Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; ClustalW; Infernal,50.0,58.2,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,5,0,0,Gene; sRNA;,Predicted; RNAalifold,10,0,,\"\",0.551,-8.5757,1600000,347624,286,162,true,-3.1038,0.71679,2013-10-04T02:32:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01400,istR,2065,istR Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; CMfinder; Infernal,45.0,47.1,38.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,32,0,0,Gene; antitoxin;,Predicted; CMfinder,36,0,,\"\",0.56313,-8.42909,1600000,309063,240,130,true,-3.2929,0.71833,2013-10-04T02:32:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01401,rseX,1266,rseX Hfq binding RNA,Barquist L,Predicted; mafft; Infernal,50.0,51.4,45.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,7,0,0,Gene; sRNA;,Predicted; RNAalifold,17,0,,\"\",0.42808,-7.45525,1600000,272652,185,94,true,-3.2023,0.71862,2013-10-04T02:32:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01402,STnc150,1266,STnc150 Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,60.0,60.8,51.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.98 -Z 549862.597050 CM SEQDB,9,6,0,0,Gene; sRNA;,Predicted; WAR,15,0,,\"\",0.56357,-10.40654,1600000,431055,444,267,true,-3.4329,0.70737,2013-10-04T02:33:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01403,STnc290,1266,STnc290 Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; ClustalW; Infernal,60.0,67.3,41.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,5,0,0,Gene; sRNA;,Predicted; RNAalifold,11,0,,\"\",0.53264,-7.1683,1600000,386135,152,65,true,-3.5639,0.71917,2013-02-01T11:58:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01404,PinT,2428,PinT (STnc440) Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; CMfinder; Infernal,60.0,72.2,33.2,,STnc440;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,5,0,0,Gene; sRNA;,Predicted; CMfinder,9,0,,\"\",0.58849,-6.89067,1600000,323183,168,76,true,-3.3933,0.71886,2013-10-04T02:33:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01405,STnc490,1266,STnc490 Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,56.0,56.8,55.7,,STnc490k;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,76,184,0,0,Gene; sRNA;,Predicted; WAR,97,0,,\"\",0.54705,-9.26061,1600000,395939,231,124,true,-3.0674,0.71846,2013-10-04T02:33:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01406,STnc500,1266,STnc500 Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,44.0,44.9,25.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,5,0,0,Gene; sRNA;,Predicted; WAR,12,0,,\"\",0.68856,-9.10975,1600000,407500,438,285,true,-4.0161,0.70649,2013-10-04T02:33:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01407,STnc560,1266,STnc560 Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,55.0,56.5,54.0,Original sequence was AE014075.1/1805202-1804553. Based upon sequence conservation this was trimmed to AE014075.1/1805101-1804887.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,18,0,0,Gene; sRNA;,Predicted; WAR,29,0,,\"\",0.56487,-9.49705,1600000,352339,349,214,true,-3.5287,0.7111,2013-10-04T02:33:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01409,STnc250,1266,STnc250 Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; WAR; Infernal,42.0,42.2,39.9,,ypfM_ORF,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,10,0,0,Gene; sRNA;,Predicted; WAR,13,0,,\"\",0.56074,-7.65728,1600000,320483,194,94,true,-3.3006,0.71864,2013-10-04T02:33:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01410,BsrC,2066,BsrC,Gardner PP,Predicted; CMfinder; Infernal,38.0,38.3,36.8,The 301 nucleotide sequence from Kouji Nakamura has been trimmed to a highly conserved ~85 nucleotide long core to form this family.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,29,324,0,0,Gene; sRNA;,Predicted; CMfinder,301,0,,\"\",0.51461,-8.08259,1600000,267100,170,87,true,-3.5723,0.71872,2013-10-04T02:33:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01411,BsrF,2066,BsrF,Gardner PP,Predicted; WAR; Infernal,50.0,54.7,31.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,112,0,0,Gene; sRNA;,Predicted; WAR,116,0,,\"\",0.53152,-7.77261,1600000,296435,211,113,true,-3.2767,0.71839,2013-10-04T02:33:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01412,BsrG,2066,BsrG,Gardner PP,Predicted; WAR; Infernal,55.0,55.4,54.8,\"bsrE, bsrG and bsrH pair through intermolecular interactions with antisense sRNAs. They form type I toxin/antitoxin system that includes an mRNA encoding for a short, toxic peptide (bsrE, bsrG and bsrH) and an antitoxin that is comprised of an antisense RNA.[7] bsrG encodes a 39-amino-acid toxin, and the 180 nucleotide antisense sRNA called SR4 acts as the antitoxin (they overlap by 123 nucleotides). SR4 interaction with the 3'UTR of bsrG RNA promotes bsrG degradation and inhibits its translation.[2][4] BsrG interferes with cell envelope biosynthesis, causes membrane invaginations and delocalisation of the cell wall synthesis and initiates autolysis.[3] bsrE RNA encodes 30 amino-acid toxin peptide. Its antitoxin gene, SR5 overlaps by 112 nucleotides at the 3' end of bsrE.[5] SR5 also overlaps bsrG from nucleotide 163 to 52 (Rfam overlap check). The antitoxin SR5 promotes bsrE degradation but unlike SR4 it does not directly inhibits toxin mRNA translation.[5][6]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,295,0,0,Gene; sRNA;,Predicted; WAR,166,0,,\"\",0.62771,-8.93185,1600000,368026,479,259,true,-3.9971,0.70797,2013-10-04T02:33:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01413,miR-430,2067,miR-430 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,47.3,47.3,47.2,This family represents the microRNA (miRNA) precursor mir-430 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.00 -Z 742849.287494 CM SEQDB,32,611,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.57188,-4.42658,1600000,197967,142,70,true,-3.7807,0.71921,2013-10-04T02:33:21.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01414,Dicty_Class_I_RNA,2068,Class I RNA from Dictyostelium,Jonas K,Jonas K; PMID:33479022;,41.5,42.0,41.1,\"Dictyostelium Class I RNA was first reported for Aspegren et al. 2004 in Dictyostelium discoideum [1]. Sucgang et al. 2011 [2] identify putative homologs of Class I RNA in D. discoideum and D. purpureum. Avesson et al. 2011 [3] report that Class I RNA present a down regulation during the development of social amoeba Dictyostelium discoideum, they also found that deletion of one of the Class I RNAs, DdR-21, results in aberrant development. Kjellin et al. 2020 [4] report an extended number of expressed Class I RNA genes in class Dictyostelia, in their analysis they remark that Class I RNA appear to be unique to dictyostelid social amoebas, and they hypothesise that Class I RNAs were involved in evolution of multicellularity in Dictyostelia.\",class_I_RNA;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,112,40,0,0,Gene;,Published; PMID:33479022;,8,0,,\"\",0.57737,-5.84391,1600000,506393,510,61,true,-2.7501,0.72543,2013-10-04T02:33:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01415,Flavivirus_SLIV,1467,Flavivirus 3'UTR stem loop IV,Osuch I,INFERNAL,38.6,38.6,38.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,20,97,0,0,Cis-reg;,Published; PMID:19064258,17,0,,\"\",0.4666,-9.79371,1600000,366811,159,75,true,-3.3243,0.71898,2013-10-04T02:33:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01416,NrrF,2069,NrrF RNA,Osuch I,INFERNAL,60.0,67.4,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; antisense;,Published; PMID:17351036,8,0,,\"\",0.5493,-9.02327,1600000,354035,282,160,true,-3.2598,0.71715,2013-10-04T02:33:35.000Z,2019-01-10T15:16:02.000Z\n",
            "RF01417,RSV_RNA,2070,Retroviral 3'UTR stability element,Osuch I,INFERNAL,105.0,106.0,104.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,71,0,0,Cis-reg;,Published; PMID:19091866,12,0,,\"\",0.73787,-8.23018,1600000,384427,438,289,true,-3.7074,0.70639,2013-10-04T02:33:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01418,HIV_POL-1_SL,2071,HIV pol-1 stem loop,Osuch I,INFERNAL,54.0,54.0,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,29,19,0,0,Cis-reg;,Published; PMID:18974280,12,0,,\"\",0.59873,-4.66215,1600000,228486,212,113,true,-3.3079,0.71842,2013-10-04T02:34:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01419,IsrR,2072,Antisense RNA which regulates isiA expression,Osuch I,Predicted; MAFFT,52.0,52.0,51.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,308,1126,0,0,Gene; antisense;,Predicted; PFOLD; Osuch I,408,0,,\"\",0.54041,-11.41194,1600000,687312,328,187,true,-3.2305,0.71345,2013-10-04T02:35:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01420,snoR113,1264,Small nucleolar RNA snoR113,Osuch I,INFERNAL,47.0,47.7,44.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,152,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Osuch I,69,0,,\"\",0.63475,-4.93652,1600000,271172,227,103,true,-3.3651,0.71827,2013-10-04T02:35:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01421,snoR114,1264,small nucleolar RNA snoR114,Osuch I,INFERNAL,45.0,45.0,44.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,24,482,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Osuch I,63,0,,\"\",0.58882,-4.9907,1600000,300056,169,87,true,-3.672,0.7185,2013-10-04T02:35:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01422,snoR116,1264,small nucleolar RNA snoR116,Osuch I,INFERNAL,47.0,48.3,39.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,90,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Osuch I,48,0,,\"\",0.60707,-4.17577,1600000,249368,180,86,true,-3.3735,0.71857,2013-10-04T02:35:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01423,snoR117,1264,small nucleolar RNA snoR117,Osuch I,INFERNAL,42.0,43.6,41.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,152,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Osuch I,73,0,,\"\",0.64619,-4.48884,1600000,314522,168,80,true,-3.4892,0.71863,2013-10-04T02:35:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01424,snoR118,1264,small nucleolar RNA snoR118,Osuch I,INFERNAL,44.0,44.3,41.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,227,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,75,0,,\"\",0.51656,-9.75958,1600000,302247,180,92,false,-3.4175,0.71838,2013-10-04T02:35:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01425,snoR121,1264,small nucleolar RNA snoR121,Osuch I,INFERNAL,50.0,54.3,42.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,14,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Osuch I,8,0,,\"\",0.6073,-4.21683,1600000,264232,184,88,true,-3.167,0.71836,2013-10-04T02:35:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01426,snoR126,1264,small nucleolar RNA snoR126,Osuch I,INFERNAL,39.0,39.2,37.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,169,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Osuch I,73,0,,\"\",0.56911,-5.20811,1600000,282741,199,101,true,-2.963,0.71835,2013-10-04T02:35:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01427,snoR127,1264,small nucleolar RNA snoR127,Osuch I,INFFERNAL,37.0,37.4,36.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,92,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Osuch I,57,0,,\"\",0.56138,-6.55238,1600000,281685,254,139,true,-2.9933,0.7179,2013-10-04T02:35:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01428,snoR128,1264,small nucleolar RNA snoR128,Osuch I,INFERNAL,40.0,40.9,38.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,5,215,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Osuch I,69,0,,\"\",0.51029,-3.89844,1600000,230867,186,91,true,-2.9379,0.71844,2013-10-04T02:35:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01429,snoR130,1264,small nucleolar RNA snoR130,Osuch I,INFERNAL,40.0,41.8,38.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,32,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAalifold; Osuch I,16,0,,\"\",0.55247,-5.77217,1600000,258113,229,120,true,-2.8745,0.71824,2013-10-04T02:35:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01430,snoR134,1264,small nucleolar RNA snoR134,Osuch I,INFERNAL,40.0,40.5,35.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,278,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,76,0,,\"\",0.5809,-7.4766,1600000,241199,261,149,true,-3.3505,0.71807,2013-10-04T02:35:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01431,snoR135,1264,small nucleolar RNA snoR135,Osuch I,INFERNAL,45.0,45.4,44.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,278,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,75,0,,\"\",0.5561,-8.16228,1600000,261686,257,143,true,-3.0947,0.71818,2013-10-04T02:35:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01432,snoR143,1264,small nucleolar RNA snoR143,Osuch I,INFERNAL,50.0,50.5,49.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,217,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,75,0,,\"\",0.53525,-7.30984,1600000,228841,288,169,true,-3.198,0.71582,2013-10-04T02:35:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01433,snoR137,1264,small nucleolar RNA snoR137,Osuch I,INFERNAL,44.0,44.1,42.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,252,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,73,0,,\"\",0.51305,-7.97321,1600000,291054,268,146,true,-2.8835,0.71799,2013-10-04T02:35:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01434,S_pombe_snR3,1264,small nucleolar RNA snR3,Osuch I,INFERNAL,100.0,128.3,33.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.56115,-7.80586,1600000,283365,288,167,true,-3.1296,0.71611,2013-10-04T02:35:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01435,S_pombe_snR5,1264,small nucleolar RNA snR5,Osuch I,INFERNAL,80.0,97.9,39.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.51368,-7.84215,1600000,247292,258,143,true,-3.0288,0.71815,2013-10-04T02:36:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01436,S_pombe_snR33,1264,small nucleolar RNA snR33,Osuch I,INFERNAL,80.0,91.2,41.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.49547,-7.02663,1600000,215975,259,145,true,-3.0113,0.71805,2013-02-01T11:58:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01437,S_pombe_snR10,1264,small nucleolar RNA snR10,Osuch I,INFERNAL,50.0,58.4,40.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.45208,-8.18738,1600000,302646,292,167,true,-2.7537,0.71612,2013-10-04T02:36:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01438,S_pombe_snR35,1264,small nucleolar RNA snR35,Osuch I,INFERNAL,35.0,35.0,34.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,27,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,24,0,,\"\",0.59998,-8.29095,1600000,319194,295,172,true,-3.2338,0.71534,2013-10-04T02:36:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01439,S_pombe_snR36,1264,small nucleolar RNA snR36,Osuch I,INFERNAL,50.0,58.4,43.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.53557,-8.51895,1600000,348476,289,164,true,-3.1243,0.71645,2013-10-04T02:36:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01440,S_pombe_snR42,1264,small nucleolar RNA snR42,Osuch I,INFERNAL,90.0,142.7,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.69 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.49052,-9.75361,1600000,275460,338,208,true,-3.2944,0.71147,2013-10-04T02:36:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01441,S_pombe_snR46,1264,small nulceolar RNA snR46,Osuch I,INFERNAL,60.0,67.3,43.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,2,0,,\"\",0.49515,-8.52303,1600000,302608,294,168,true,-2.9757,0.71592,2013-10-04T02:36:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01442,S_pombe_snR90,1264,small nucleolar RNA snR90,Osuch I,INFERNAL,80.0,102.4,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.45834,-9.00434,1600000,348195,277,158,true,-3.0845,0.71734,2013-10-04T02:36:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01443,S_pombe_snR91,1264,small nucleolar RNA snR91,Osuch I,INFERNAL,50.0,71.2,41.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.64 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,2,0,,\"\",0.5222,-7.58671,1600000,283411,263,144,true,-2.8919,0.71807,2013-10-04T02:36:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01444,S_pombe_snR92,1264,small nucleolar RNA snR92,Osuch I,INFERNAL,80.0,84.3,38.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.5714,-6.99058,1600000,227604,257,144,true,-3.281,0.71807,2013-10-04T02:36:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01445,S_pombe_snR94,1264,small nucleolar RNA snR94,Osuch I,INFERNAL,100.0,159.2,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.47896,-10.49396,1600000,413793,341,210,true,-3.4101,0.71147,2013-10-04T02:36:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01446,S_pombe_snR95,1264,small nucleolar RNA snR95,Osuch I,INFERNAL,60.0,80.4,34.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,2,0,,\"\",0.53511,-9.42746,1600000,307730,361,222,true,-3.2884,0.71045,2013-10-04T02:36:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01447,S_pombe_snR98,1264,small nucleolar RNA snR98,Osuch I,INFERNAL,40.0,41.3,33.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.61151,-8.70464,1600000,308959,349,212,true,-3.5864,0.71137,2013-10-04T02:36:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01448,S_pombe_snR93,1264,small nucleolar RNA snR93,Osuch I,INFERNAL,45.0,46.0,40.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,2,0,,\"\",0.49209,-7.53249,1600000,265460,261,143,true,-3.1565,0.71799,2013-10-04T02:36:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01449,S_pombe_snR100,1264,small nucleolar RNA snR100,Osuch I,INFERNAL,80.0,99.5,55.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,3,0,,\"\",0.63782,-8.00032,1600000,308202,312,187,true,-3.4719,0.71393,2013-10-04T02:36:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01450,S_pombe_snR96,1264,small nucleolar RNA snR96,Osuch I,INFERNAL,100.0,274.3,33.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,1,0,,\"\",0.56953,-9.22527,1600000,352110,348,211,true,-3.0559,0.71129,2013-10-04T02:36:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01451,S_pombe_snR97,1264,small nucleolar RNA snR97,Osuch I,INFERNL,100.0,192.8,37.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold; Osuch I,1,0,,\"\",0.50051,-7.51381,1600000,345187,268,148,true,-2.825,0.71804,2013-10-04T02:36:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01452,S_pombe_snR99,1264,small nucleolar RNA snR99,Osuch I,INFERNAL,90.0,167.6,46.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,6,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,3,0,,\"\",0.60789,-8.22005,1600000,323167,313,185,true,-3.3864,0.71384,2013-10-04T02:36:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01453,RCNMV_TE_DR1,2073,3'TE-DR1 translation enhancer element,Osuch I,INFERNAL,80.0,160.3,60.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Cis-reg;,Published; PMID:19577782,2,0,,\"\",0.64425,-6.3765,1600000,240061,254,138,true,-3.1616,0.7182,2013-10-04T02:36:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01454,RCNMV_5UTR,2073,5'UTR enhancer element,Osuch I,INFERNAL,75.0,136.0,40.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Cis-reg;,Published; PMID:19577782,2,0,,\"\",0.51929,-8.8778,1600000,385289,231,123,true,-3.3424,0.71827,2013-10-04T02:36:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01455,DPB,2074,Stem loopII regulatory element in POLB,Osuch I,INFERNAL,40.0,41.0,39.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24 -Z 549862.597050 CM SEQDB,19,56,0,0,Cis-reg;,Published; PMID:17704138,41,0,,\"\",0.54799,-2.9594,1600000,155374,176,86,true,-3.7869,0.71865,2013-10-04T02:36:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01456,VrrA,2075,Vibrio regulatory RNA of OmpA,Osuch I,INFERNAL,50.0,54.8,42.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,32,0,0,Gene; antisense;,Published; PMID:18681937,43,0,,\"\",0.49808,-8.78335,1600000,321677,251,140,true,-3.197,0.71818,2013-10-04T02:36:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01457,rli22,1267,Listeria sRNA rli22,Osuch I,INFERNAL,80.0,116.9,41.1,The UCCC motifs prevent ribosomal binding to oppA mRNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,1,0,0,Gene; sRNA;,Predicted; RNAalifold,7,0,,\"\",0.48838,-8.35479,1600000,322973,217,111,true,-2.9954,0.71835,2013-10-04T02:36:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01458,rli23,1267,Listeria snRNA rli23,Osuch I,INFERNAL,45.0,46.7,42.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,7,45,0,0,Gene; antisense;,Predicted; RNAalifold,30,0,,\"\",0.51969,-6.23236,1600000,242171,194,98,true,-3.3134,0.71863,2013-10-04T02:36:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01459,rliE,1267,Listeria sRNA rliE,Osuch I,INFERNAL,90.0,119.5,51.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.48 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; WAR,5,0,,\"\",0.45877,-11.00955,1600000,340322,356,222,true,-3.2087,0.71033,2013-10-04T02:36:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01460,rliH,1267,Listeria sRNA rliH,Osuch I,INFERNAL,100.0,151.7,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.56 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; WAR,4,0,,\"\",0.7275,-8.13179,1600000,382156,607,429,true,-4.3111,0.70212,2013-10-04T02:36:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01461,rli24,1267,Listeria sRNA rli22,Osuch I,INFERNAL,100.0,143.7,48.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.73 -Z 549862.597050 CM SEQDB,6,1,0,0,Gene; sRNA;,Predicted; RNAalifold,7,0,,\"\",0.5291,-8.06906,1600000,287925,280,158,true,-3.1598,0.71735,2013-10-04T02:36:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01462,rli26,1267,Listeria sRNA rli26,Osuch I,INFERNAL,100.0,197.3,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.42 -Z 549862.597050 CM SEQDB,5,1,0,0,Gene; sRNA;,Predicted; RNAalifold,6,0,,\"\",0.47425,-9.76855,1600000,342983,317,188,true,-2.9692,0.71346,2013-10-04T02:36:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01463,rli27,1267,Listeria sRNA rli27,Osuch I,INFERNAL,55.0,56.5,41.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 38.00 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAalifold,4,0,,\"\",0.41475,-5.19686,1600000,300096,200,99,true,-3.0212,0.71836,2013-10-04T02:36:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01464,rliA,1267,Listeria sRNA rliA,Osuch I,INFERNAL,50.0,50.1,47.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,3,0,0,Gene; sRNA;,Predicted; RNAalifold,6,0,,\"\",0.56244,-8.62877,1600000,353765,362,224,true,-3.6101,0.71025,2013-10-04T02:36:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01465,rli31,1267,Listeria sRNA rli31,Osuch I,INFERNAL,80.0,81.8,62.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAalifold,4,0,,\"\",0.53945,-4.18613,1600000,245307,221,115,true,-2.8713,0.71836,2013-10-04T02:36:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01466,rli34,1267,Listeria sRNA rli34,Osuch I,INFERNAL,48.0,48.3,46.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,3,0,0,Gene; sRNA;,Predicted;RNAalifold,7,0,,\"\",0.43538,-5.68078,1600000,328847,176,84,true,-3.1288,0.71861,2013-02-01T11:58:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01467,rli36,1267,Listeria sRNA rli36,Osuch I,INFERNAL,50.0,52.0,40.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,6,1,0,0,Gene; sRNA;,Predicted; RNAalifold,7,0,,\"\",0.50842,-7.8716,1600000,376288,178,84,true,-3.3039,0.71878,2013-10-04T02:37:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01468,rli32,1267,Listeria sRNA rli32,Osuch I,INFERNAL,100.0,109.5,48.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,1,0,0,Gene; sRNA;,Predicted;RNAalifold,6,0,,\"\",0.56709,-9.01479,1600000,399297,264,147,true,-3.2039,0.71817,2013-10-04T02:37:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01469,rli33,1267,Listeria sRNA rli33,Osuch I,INFERNAL,46.0,47.8,44.5,The UCCC motifs prevent ribosomal binding to oppA mRNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 15 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; RNAalifold,5,0,,\"\",0.71539,-8.29328,1600000,398465,728,532,true,-4.4631,0.70036,2013-10-04T02:37:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01470,rli38,1267,Listeria sRNA rli38,Osuch I,INFERNAL,49.0,49.2,48.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,20,80,0,0,Gene; sRNA;,Predicted; RNAalifold,84,0,,\"\",0.43958,-8.64504,1600000,313772,251,150,true,-3.0146,0.71822,2013-10-04T02:37:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01471,rliB,1267,Listeria sRNA rliB,Osuch I,INFERNAL,40.0,41.5,37.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,5,0,0,Gene; sRNA;,Predicted; RNAalifold,7,0,,\"\",0.54272,-10.87738,1600000,577502,517,360,true,-3.6057,0.70372,2013-10-04T02:38:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01472,rli40,1267,Listeria sRNA rli40,Osuch I,INFERNAL,100.0,207.1,44.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; RNAalifold,5,0,,\"\",0.65202,-8.73657,1600000,415662,408,263,true,-3.8298,0.70783,2013-10-04T02:38:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01473,rli41,1267,Listeria sRNA rli41,Osuch I,INFERNAL,55.0,58.3,35.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,5,0,0,Gene; sRNA;,Predicted; RNAalifold,11,0,,\"\",0.56644,-9.15474,1600000,337883,683,494,true,-4.181,0.70092,2013-10-04T02:38:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01474,rli42,1267,Listeia sRNA rli42,Osuch I,INFERNAL,80.0,147.9,49.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; RNAalifold,5,0,,\"\",0.60595,-8.40878,1600000,400553,296,171,true,-3.26,0.71559,2013-10-04T02:38:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01475,rli45,1267,Listeria snRNA rli45,Osuch I,INFERNAL,60.0,89.8,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,1,0,0,Gene; antisense;,Predicted;RNAalifold,7,0,,\"\",0.59206,-3.20074,1600000,286263,168,78,true,-3.3066,0.71875,2013-10-04T02:38:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01476,rliF,1267,Listeria sRNA rliF,Osuch I,INFERNAL,60.0,63.2,52.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,4,0,0,Gene; sRNA;,Predicted; WAR,8,0,,\"\",0.4869,-9.77496,1600000,363690,350,220,true,-3.3157,0.71064,2013-10-04T02:38:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01477,rli43,1267,Listeria sRNA rli43,Osuch I,INFERNAL,100.0,191.8,98.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; RNAalifold,5,0,,\"\",0.63721,-8.83283,1600000,426339,400,255,true,-3.5461,0.70825,2013-10-04T02:38:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01478,rli47,1267,Listeria sRNA rli47,Osuch I,INFERNAL,200.0,413.0,36.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,1,0,0,Gene; sRNA;,Predicted;RNAalifold,10,0,,\"\",0.69519,-8.64467,1600000,384036,701,508,true,-4.5544,0.70071,2013-10-04T02:38:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01479,rli48,1267,Listeria sRNA rli48,Osuch I,INFERNAL,100.0,169.5,52.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted;RNaalfold,7,0,,\"\",0.57825,-3.99794,1600000,208451,267,149,true,-3.2418,0.71811,2013-10-04T02:38:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01480,rli52,1267,Listeria sRNA rli52,Osuch I,INFERNAL,70.0,72.9,49.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,1,0,0,Cis-reg;,Predicted; RNAalifold,7,0,,\"\",0.4811,-7.32348,1600000,335856,194,95,true,-3.4207,0.71865,2013-10-04T02:38:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01481,rli53,1267,Listeria sRNA rli53,Osuch I,INFERNAL,70.0,74.8,37.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,1,0,0,Cis-reg;,Predicted; RNAalifold,6,0,,\"\",0.58155,-6.77653,1600000,275080,299,173,true,-3.3395,0.71528,2013-10-04T02:38:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01482,AdoCbl_riboswitch,1456,AdoCbl riboswitch,Osuch I,INFERNAL,50.0,50.0,49.9,,rli55,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,168,0,0,Cis-reg; riboswitch;,Published; PMID:19246383,164,0,,\"\",0.53664,-7.92363,1600000,324226,258,149,true,-3.5305,0.71814,2013-10-04T02:38:19.000Z,2022-10-29T17:55:16.000Z\n",
            "RF01483,rli56,1267,Listeria sRNA rli56,Osuch I,INFERNAL,47.0,49.2,41.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,1,0,0,Cis-reg;,Predicted;RNAalifold,7,0,,\"\",0.5398,-4.99305,1600000,254650,202,100,true,-3.3573,0.71864,2013-02-01T11:58:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01484,rli59,1267,Listeria sRNA rli59,Osuch I,INFERNAL,100.0,150.8,44.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,1,0,0,Cis-reg;,Predicted; WAR; cmfinder,6,0,,\"\",0.49511,-8.09811,1600000,330398,309,181,true,-2.9219,0.71428,2013-10-04T02:38:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01485,rli61,1267,Listeria sRNA rli61,Osuch I,INFERNAL,50.0,50.4,49.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,9,0,0,Cis-reg;,Predicted; RNAalifold,13,0,,\"\",0.53665,-6.53851,1600000,231340,210,106,true,-3.3887,0.71848,2013-10-04T02:38:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01486,rli62,1267,Listeria sRNA rli62,Osuch I,INFERNAL,83.0,102.4,82.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.97 -Z 549862.597050 CM SEQDB,2,5,0,0,Cis-reg;,Predicted;RNAalifold,7,0,,\"\",0.53208,-8.41206,1600000,336669,244,134,true,-3.1417,0.71816,2013-10-04T02:38:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01487,rliI,1267,Listeria sRNA rliI,Osuch I,INFERNAL,45.0,48.5,38.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,7,0,0,Gene; sRNA;,Predicted;RNAalifold,12,0,,\"\",0.69165,-7.61417,1600000,361126,382,239,true,-3.6263,0.70929,2013-10-04T02:38:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01488,rli49,1267,Listeria sRNA rli49,Osuch I,INFERNAL,40.0,46.2,39.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,1,0,0,Gene; sRNA;,Predicted; RNAalifold,6,0,,\"\",0.50915,-9.87066,1600000,442202,286,163,true,-3.1574,0.71653,2013-10-04T02:38:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01489,sbrA,1267,Listeria sRNA sbrA,Osuch I,INFERNAL,45.0,50.7,37.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,30,0,0,Gene; sRNA;,Predicted; WAR,35,0,,\"\",0.61236,-5.55983,1600000,308863,158,71,true,-3.3909,0.71899,2013-10-04T02:38:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01490,rli51,1267,Listeria snRNA rli51,Osuch I,INFERNAL,48.0,48.5,46.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,3,0,0,Cis-reg;,Predicted; RNAalifold,5,0,,\"\",0.54994,-5.35749,1600000,284407,228,121,true,-3.0685,0.7183,2013-10-04T02:38:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01491,rli54,1267,Listeria sRNA rli54,Osuch I,INFERNAL,80.0,103.0,36.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,4,0,0,Cis-reg;,Predicted;RNAalifold,9,0,,\"\",0.68389,-8.0754,1600000,337037,724,528,true,-4.2278,0.7004,2013-10-04T02:38:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01492,rli28,1267,Listeria sRNA rli28,Osuch I,INFERNAL,45.0,45.0,44.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,397,0,0,Gene; sRNA;,Predicted; RNAalifold,186,0,,\"\",0.37855,-13.72784,1600000,463710,292,186,true,-3.395,0.71366,2013-10-04T02:38:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01493,rli37,1267,Listeria sRNA rli37,Osuch I,INFERNAL,90.0,156.3,44.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; RNAalifold,5,0,,\"\",0.53843,-10.42392,1600000,468119,390,247,true,-3.3855,0.70862,2013-10-04T02:38:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01494,rliD,1267,Listeria sRNA rliD,\"Osuch I, Daub J\",INFERNAL,50.0,50.1,49.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,88,0,0,Gene; antisense;,Predicted; RNAalifold,97,0,,\"\",0.71751,-7.89356,1600000,376884,487,327,true,-4.1191,0.70481,2013-10-04T02:38:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01495,ACAT,2077,Human acetyl-coA cholesterol acyltransferase mRNA,\"Osuch I, Daub J\",INFERNAL,37.0,37.2,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,8,30,0,0,Cis-reg;,Published; PMID:18542101,24,0,,\"\",0.5729,-8.63033,1600000,338253,272,147,true,-3.059,0.71808,2013-10-04T02:38:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01496,Afu_182,2078,A. fumigatus sRNA Afu_182,\"Osuch I, Daub J\",INFERNAL,65.0,69.4,63.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,280,0,0,Gene; sRNA;,Predicted; clustal_alifold;,269,0,,\"\",0.50627,-10.38837,1600000,344648,322,196,true,-3.0855,0.71269,2013-10-04T02:38:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01497,ALIL,2079,ALIL pseudoknot,Osuch I,INFERNAL,50.0,50.0,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,1389,0,0,Cis-reg;,Published; PMID:18474594,668,0,,\"\",0.68353,-8.3013,1600000,438520,214,117,true,-3.3465,0.71834,2013-10-04T02:38:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01498,Afu_190,1264,A. fumigatus snoRNA Afu_190,Osuch I,INFERNAL,35.0,35.0,34.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,3,273,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,263,0,,\"\",0.54837,-9.9219,1600000,405780,192,100,false,-3.5636,0.71826,2013-10-04T02:38:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01499,Afu_191,1264,A. fumigatus snoRNA Afu_191,Osuch I,INFERNAL,60.0,70.4,47.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,86,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,87,0,,\"\",0.46749,-9.52981,1600000,263410,187,93,false,-3.0961,0.71839,2013-10-04T02:38:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01500,Afu_198,1264,A. fumigatus snoRNA Afu_198,Osuch I,INFERNAL,40.0,40.8,37.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,117,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,116,0,,\"\",0.504,-10.33255,1600000,220983,253,149,false,-3.4369,0.71776,2013-10-04T02:38:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01501,Afu_199,1264,A. fimigatus snoRNA Afu_199,Osuch I,INFERNAL,45.0,50.4,39.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,98,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,99,0,,\"\",0.51424,-8.72618,1600000,294651,162,77,false,-3.5214,0.71859,2013-10-04T02:38:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01502,Fungi_SRP,1315,Fungal signal recognition particle RNA,Gardner PP,Published; PMID:19838050,65.0,65.6,64.8,,Afu_202; SRP_euk_arch; SCR1; Afu_202,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.99 -Z 549862.597050 CM SEQDB,50,330,0,0,Gene;,Published; PMID:19838050,344,0,,\"\",0.58808,-11.31913,1600000,518713,1499,314,true,-4.3373,0.70558,2013-10-04T02:38:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01503,Afu_203,2078,A. fumigatus sRNA Afu_203,Osuch I,INFERNAL,37.0,37.1,33.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,8,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,9,0,,\"\",0.54164,-6.64895,1600000,222941,217,111,true,-3.3146,0.71847,2013-10-04T02:38:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01504,Afu_254,2078,A. fumigatus sRNA Afu_254,Osuch I,INFERNAL,60.0,64.7,57.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,105,0,0,Gene; sRNA;,Predicted; WAR; Mafft_alifold;,77,0,,\"\",0.49181,-9.95702,1600000,539257,268,158,true,-3.4196,0.71723,2013-10-04T02:39:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01505,Afu_264,1264,A. fumigatus snoRNA Afu_264,Osuch I,INFERNAL,40.0,41.5,39.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,247,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,244,0,,\"\",0.55041,-8.07566,1600000,298995,195,103,false,-3.371,0.71824,2013-10-04T02:39:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01506,Afu_294,1264,A. fumigatus snoRNA Afu_294,Osuch I,INFERNAL,43.0,43.8,42.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,181,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I;,178,0,,\"\",0.54814,-8.32685,1600000,291661,165,83,false,-3.4292,0.71852,2013-10-04T02:39:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01507,Afu_298,2078,A. fimigatus snoRNA Afu_298,Osuch I,INFERNAL,40.0,40.8,38.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,163,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,148,0,,\"\",0.41845,-10.33361,1600000,293837,187,93,false,-3.2477,0.7184,2013-10-04T02:39:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01508,BYDV_5_UTR,2080,Barley yellow dwarf virus 5'UTR,Osuch I,INFERNAL,75.0,99.9,32.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,4,0,0,Cis-reg;,Published; PMID:11389856,6,0,,\"\",0.61278,-8.48203,1600000,353402,265,149,true,-3.12,0.71811,2013-10-04T02:39:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01509,Afu_300,1264,A. fumigatus snoRNA Afu_300,Osuch I,INFERNAL,40.0,44.7,39.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,260,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,257,0,,\"\",0.54732,-7.913,1600000,307121,178,92,false,-3.4683,0.71837,2013-10-04T02:39:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01510,MFR,2081,M. florum riboswitch,\"Osuch I, Eberhardt R\",INFERNAL,82.7,82.7,81.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,2,0,0,Cis-reg; riboswitch;,Published; PMID:17911257,1,0,,\"\",0.61205,-3.49701,1600000,266521,146,63,true,-3.1658,0.72275,2013-10-04T02:39:11.000Z,2020-09-14T15:04:54.000Z\n",
            "RF01511,Afu_304,1264,A. fumigatus snoRNA Afu_304,Osuch I,INFERNAL,35.0,35.2,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,255,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,241,0,,\"\",0.48673,-10.12195,1600000,271232,203,107,false,-3.1911,0.7182,2013-10-04T02:39:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01512,Afu_309,2078,A. fumigatus sRNA Afu_309,Osuch I,INFERNAL,48.0,49.2,45.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,273,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,266,0,,\"\",0.70376,-8.49498,1600000,373878,549,322,true,-4.2003,0.70511,2013-10-04T02:39:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01513,Afu_335,1264,A. fumigatus snoRNA Afu_335,Osuch I,INFERNAL,50.0,56.3,39.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,5,281,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,267,0,,\"\",0.4817,-10.05979,1600000,263554,188,96,false,-3.4961,0.71829,2013-10-04T02:39:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01514,Afu_513,1264,A. fumigatus snoRNA Afu_513,Osuch I,INFERNAL,48.0,49.8,42.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,277,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,272,0,,\"\",0.57292,-9.56449,1600000,308904,197,103,false,-3.3657,0.71823,2013-10-04T02:39:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01515,Afu_514,1264,A. fumigatus snoRNA Afu_514,Osuch I,INFERNAL,39.0,39.7,32.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,268,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,266,0,,\"\",0.5206,-9.81935,1600000,336282,201,106,false,-3.2374,0.71822,2013-10-04T02:39:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01516,v-snoRNA-1,2476,Human herpesvirus 1 small nucleolar RNA,Osuch I,INFERNAL,75.0,85.1,33.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,3,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,2,0,,\"\",0.54393,-9.04729,1600000,329549,151,65,false,-3.1373,0.72681,2013-10-04T02:39:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01517,iscRS,2083,iscR stability element,Osuch I,INFERNAL,40.0,43.4,35.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,36,0,0,Cis-reg;,Published; PMID:19407815,39,0,,\"\",0.70017,-3.63988,1600000,218269,214,110,true,-3.5013,0.71849,2013-10-04T02:39:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01518,pRNA,2084,\"NoRC associated RNA, pRNA\",Osuch I,INFERNAL,34.0,34.0,33.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,92,0,0,Gene;,Published; PMID:18600236,42,0,,\"\",0.62441,-4.18099,1600000,230577,175,90,true,-2.9852,0.71889,2013-10-04T02:39:24.000Z,2019-01-10T15:16:02.000Z\n",
            "RF01519,CC0196,2085,Caulobacter sRNA CC0196,Osuch I,INFERNAL,35.0,35.4,34.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,14,0,0,Gene; sRNA;,Predicted; RNAalifold;,15,0,,\"\",0.6106,-8.32844,1600000,446857,144,58,true,-3.2471,0.72658,2013-10-04T02:39:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01520,CC0734,2085,caulobacter sRNA CC0734,Osuch I,INFERNAL,45.0,53.0,40.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,6,0,0,Gene; sRNA;,Predicted;Mafft_pfold;,3,0,,\"\",0.55243,-5.28599,1600000,209351,194,95,true,-3.3223,0.71855,2013-10-04T02:39:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01521,CC1840,2085,caulobacter sRNA CC1840,Osuch I,INFERNAL,50.0,60.4,40.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,10,0,0,Gene; sRNA;,Predicted;Clustal_alifold;,11,0,,\"\",0.40161,-11.86634,1600000,316439,198,95,true,-3.1158,0.71873,2013-10-04T02:39:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01522,TB10Cs1H1,2410,Trypanosomatid snoRNA TB10Cs1H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,45.0,48.1,38.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,72,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,15,0,,\"\",0.62697,-4.48715,1600000,276962,158,71,true,-3.5322,0.71895,2013-10-04T02:39:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01523,TB10Cs1H2,2411,Trypanosomatid snoRNA TB10Cs1H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,69.6,35.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,34,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,15,0,,\"\",0.62603,-4.84762,1600000,333423,145,63,true,-3.3509,0.71923,2013-10-04T02:39:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01524,TB10Cs1H3,2411,Trypanosomatid snoRNA TB10cs1H3,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,70.3,47.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,34,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,15,0,,\"\",0.63234,-4.59942,1600000,203927,160,73,true,-3.3945,0.719,2013-10-04T02:39:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01525,TB10Cs2H1,2411,Trypanosomatid snoRNA TB10Cs2H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,35.0,43.1,31.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,33,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,12,0,,\"\",0.59077,-5.85751,1600000,255213,149,66,true,-3.2918,0.71932,2013-10-04T02:39:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01526,ceN106,1264,C. elegans snoRNA ceN106,Osuch I,INFERNAL,60.0,78.8,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.75 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,6,0,,\"\",0.62097,-4.64609,1600000,246719,169,77,true,-3.6566,0.71864,2013-10-04T02:39:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01527,CrfA,1236,Caulobacter sRNA CrfA,Osuch I,INFERNAL,38.0,38.4,36.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,33,0,0,Gene; sRNA;,Predicted; Clustal_alifold;,32,0,,\"\",0.50233,-10.45067,1600000,359981,251,136,true,-3.0244,0.71834,2013-10-04T02:39:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01528,CC3513,2085,caulobacter sRNA CC3513,Osuch I,INFERNAL,40.0,42.2,33.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,9,0,0,Gene; sRNA;,Predicted; Mafft_alifold;,10,0,,\"\",0.49181,-10.7011,1600000,332139,299,173,true,-2.8406,0.71538,2013-10-04T02:39:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01529,CC3552,2085,Cauldobacter sRNA CC3552,Osuch I,INFERNAL,50.0,71.7,40.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,26,0,0,Gene; sRNA;,Predicted;RNAalifold;,27,0,,\"\",0.47141,-11.26164,1600000,465206,208,106,true,-3.3804,0.71864,2013-10-04T02:39:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01530,CC3664,2085,Caulobacter sRNA CC3664,Osuch I,INFERNAL,60.0,63.5,46.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,2,14,0,0,Gene; sRNA;,Predicted; Clustal_alifold;,14,0,,\"\",0.47004,-11.71801,1600000,465380,252,136,true,-2.96,0.71848,2013-10-04T02:39:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01531,TB10Cs3H1,2411,Trypanosomatid snoRNA TB10Cs3H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,40.0,44.1,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,48,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,14,0,,\"\",0.58569,-5.7253,1600000,338074,147,62,true,-3.5773,0.71923,2013-10-04T02:39:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01532,TB10Cs3H2,2411,Trypanosomatid snoRNA TB10Cs3H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,45.0,50.2,34.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,48,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,13,0,,\"\",0.55699,-5.25538,1600000,200688,153,69,true,-3.0415,0.71908,2013-10-04T02:39:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01533,TB10Cs4H3,2411,Trypanosomatid snoRNA TB10Cs4H3,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,45.0,47.7,37.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,25,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,11,0,,\"\",0.59884,-5.41726,1600000,296559,152,67,true,-3.207,0.72168,2013-10-04T02:39:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01534,TB10Cs4H4,2411,Trypanosomatid snoRNA TB10Cs4H4,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,68.5,45.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,16,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,7,0,,\"\",0.61963,-4.81361,1600000,326086,154,66,true,-3.0227,0.72858,2013-10-04T02:39:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01535,TB10Cs5H2,2411,Trypanosomatid snoRNA TB10Cs5H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,40.0,40.9,34.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,9,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,11,0,,\"\",0.64117,-5.05837,1600000,309934,163,78,true,-3.397,0.71875,2013-10-04T02:39:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01536,TB10Cs5H3,2411,Trypanosomatid snoRNA TB10Cs5H3,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,45.0,46.0,40.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,7,17,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,15,0,,\"\",0.60637,-4.90872,1600000,290833,151,70,true,-3.3807,0.71892,2013-10-04T02:39:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01537,TB11Cs2H1,2411,Trypanosomatid snoRNA TB11Cs2H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,40.0,43.9,37.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,66,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,14,0,,\"\",0.6294,-3.60657,1600000,168436,152,70,true,-3.9618,0.71899,2013-10-04T02:39:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01538,TB11Cs3H1,2411,Trypanosomatid snoRNA TB11Cs3H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,58.2,49.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.48 -Z 549862.597050 CM SEQDB,4,27,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,15,0,,\"\",0.64,-3.8582,1600000,218627,157,72,true,-3.5717,0.71895,2013-10-04T02:40:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01539,TB11Cs4H1,2411,Trypanosomatid snoRNA TB11Cs4H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,45.0,46.2,43.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,157,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,15,0,,\"\",0.59945,-4.66991,1600000,251364,157,72,true,-3.1026,0.71894,2013-10-04T02:40:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01540,TB11Cs4H2,2411,Trypanosomatid snoRNA TB11Cs4H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,55.3,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,6,155,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,13,0,,\"\",0.62282,-4.52723,1600000,300184,144,64,true,-3.6542,0.71912,2013-10-04T02:40:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01541,TB11Cs4H3,2411,Trypanosomatid snoRNA TB11Cs4H3,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,40.0,40.6,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,130,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,9,0,,\"\",0.57494,-5.94883,1600000,302974,162,74,true,-2.9783,0.71885,2013-10-04T02:40:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01542,TB11Cs5H1,2411,Trypanosomatid snoRNA TB11Cs5H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,45.0,48.3,32.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,14,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,7,0,,\"\",0.67172,-3.49198,1600000,227602,161,71,true,-3.0887,0.72757,2013-10-04T02:40:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01543,TB11Cs5H2,2411,Trypanosomatid snoRNA TB11Cs5H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,47.0,47.1,46.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,30,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,18,0,,\"\",0.66339,-4.41714,1600000,310740,150,68,true,-3.4696,0.71909,2013-02-01T11:58:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01544,TB11Cs5H3,2411,Trypanosomatid snoRNA TB11Cs5H3,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,52.1,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,5,11,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,9,0,,\"\",0.69365,-3.35569,1600000,272411,147,65,true,-3.8794,0.71925,2013-10-04T02:40:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01545,TB3Cs2H1,2411,Trypanosomatid snoRNA TB3Cs2H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,60.0,62.5,33.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,6,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,7,0,,\"\",0.58497,-5.35089,1600000,273986,158,69,true,-2.8863,0.72735,2013-10-04T02:40:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01546,TB6Cs1H1,2411,Trypanosomatid snoRNA TB6Cs1H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,40.0,63.2,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,99,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,19,0,,\"\",0.66514,-4.19309,1600000,273107,149,66,true,-3.6631,0.71911,2013-10-04T02:40:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01547,TB6Cs1H3,2411,Trypanosomatid snoRNA TB6Cs1H3,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,40.0,44.1,32.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,87,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,18,0,,\"\",0.62117,-5.18926,1600000,389886,146,64,true,-3.5354,0.71911,2013-10-04T02:40:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01548,TB6Cs1H4,2411,Trypanosomatid snoRNA TB6Cs1H4,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,60.0,70.9,34.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,69,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,16,0,,\"\",0.62334,-3.90497,1600000,249069,161,73,true,-3.7087,0.71887,2013-10-04T02:40:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01549,TB8Cs2H1,2411,Trypanosomatid snoRNA TB8Cs2H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,40.0,41.2,33.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,48,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,15,0,,\"\",0.60881,-5.90714,1600000,385127,141,65,true,-3.4752,0.71912,2013-10-04T02:40:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01550,TB8Cs3H1,2411,Trypanosomatid snoRNA TB8Cs3H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,45.0,56.0,31.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,44,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,17,0,,\"\",0.65631,-4.56504,1600000,289650,144,63,true,-3.6287,0.71915,2013-10-04T02:40:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01551,TB8Cs4H2,2411,Trypanosomatid snoRNA TB8Cs4H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,56.3,34.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,6,0,,\"\",0.62281,-4.66393,1600000,259887,160,70,true,-2.8848,0.72944,2013-10-04T02:40:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01552,TB9Cs1H2,2411,Trypanosomatid snoRNA TB9Cs1H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,35.0,42.0,32.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,52,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,17,0,,\"\",0.54347,-6.27438,1600000,224031,177,86,true,-3.0097,0.71856,2013-10-04T02:40:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01553,TB9Cs1H3,2411,Trypanosomatid snoRNA TB9Cs1H3,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,35.0,35.8,32.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,39,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,14,0,,\"\",0.65331,-4.67723,1600000,264584,149,65,true,-3.3926,0.7222,2013-10-04T02:40:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01554,TB9Cs2H1,2411,Trypanosomatid snoRNA TB9Cs2H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,45.0,70.4,33.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,57,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,19,0,,\"\",0.60191,-5.01636,1600000,281816,153,68,true,-3.2503,0.71909,2013-10-04T02:40:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01555,TB9Cs3H1,2411,Trypanosomatid snoRNA TB9Cs3H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,45.0,53.1,33.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,36,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,16,0,,\"\",0.60338,-4.39214,1600000,260986,148,68,true,-3.7661,0.719,2013-10-04T02:40:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01556,TB9Cs3H2,2411,Trypanosomatid snoRNA TB9Cs3H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,51.1,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,30,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,14,0,,\"\",0.64684,-4.61645,1600000,317962,146,63,true,-3.3874,0.72305,2013-10-04T02:40:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01557,TB9Cs4H1,2411,Trypanosomatid snoRNA TB9Cs4H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,40.0,47.9,33.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,37,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,13,0,,\"\",0.60634,-4.80876,1600000,287917,144,62,true,-3.4839,0.71986,2013-10-04T02:40:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01558,TB9Cs4H2,2411,Trypanosomatid snoRNA TB9Cs4H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,51.1,38.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,46,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,15,0,,\"\",0.56884,-5.30142,1600000,244011,158,71,true,-3.0042,0.71894,2013-10-04T02:40:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01559,DdR1,1264,Dictyostelium snoRNA DdR1,Osuch I,INFERNAL,70.0,80.8,43.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.91 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,3,0,,\"\",0.49615,-9.6656,1600000,364016,197,66,false,-3.1562,0.71882,2013-10-04T02:40:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01560,DdR10,1264,Dictyostelium snoRNA DdR10,Osuch I,INFERNAL,48.0,49.9,43.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.08 -Z 549862.597050 CM SEQDB,4,5,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,7,0,,\"\",0.36998,-9.93336,1600000,315778,169,81,false,-2.9498,0.71857,2013-10-04T02:40:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01561,DdR11,1264,Dictyostelium snoRNA DdR11,Osuch I,INFERNAL,65.0,80.7,40.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.03 -Z 549862.597050 CM SEQDB,3,5,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,3,0,,\"\",0.53684,-8.56883,1600000,339197,163,76,false,-3.3149,0.71861,2013-10-04T02:40:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01562,DdR12,1264,D. discoideum snoRNA DdR12,Osuch I,INFERNAL,70.0,110.1,37.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,2,0,,\"\",0.40356,-9.95489,1600000,292918,164,75,false,-3.283,0.71938,2013-10-04T02:40:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01563,DdR13,1264,Dictyostelium snoRNA DdR13,Osuch I,INFERNAL,65.0,88.9,32.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.91 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,3,0,,\"\",0.54407,-9.26477,1600000,345648,168,80,false,-3.5497,0.71857,2013-10-04T02:40:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01564,DdR14,1264,D. discoideum snoRNA DdR14,Osuch I,INFERNAL,80.0,95.1,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,2,0,,\"\",0.42292,-8.94398,1600000,338292,151,65,false,-3.1034,0.72377,2013-10-04T02:40:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01565,DdR15,1264,Dictyostelium snoRNA DdR15,Osuch I,INFERNAL,70.0,118.0,40.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.57 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,3,0,,\"\",0.40025,-10.30812,1600000,274109,215,113,false,-3.0816,0.71816,2013-10-04T02:40:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01566,DdR16,1264,Dictyostelium snoRNA DdR16,Osuch I,INFERNAL,70.0,73.8,40.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.02 -Z 549862.597050 CM SEQDB,4,5,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,5,0,,\"\",0.47461,-9.15904,1600000,346007,164,78,false,-3.2746,0.71862,2013-10-04T02:40:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01567,DdR17,1264,D. discoideum snoRNA DdR17,Osuch I,INFERNAL,80.0,101.2,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,2,0,,\"\",0.40191,-7.60294,1600000,300516,157,70,false,-2.8829,0.72161,2013-10-04T02:40:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01568,DdR18,1264,Dictyostelium snoRNA DdR18,Osuch I,INFERNAL,80.0,125.7,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.82 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Published; PMID:15333696,3,0,,\"\",0.48715,-8.00266,1600000,291756,264,147,true,-3.029,0.71808,2013-10-04T02:40:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01569,DdR2,1264,Dictyostelium snoRNA DdR2,Osuch I,INFERNAL,80.0,101.5,42.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.74 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted;,2,0,,\"\",0.43006,-10.07634,1600000,289364,173,83,false,-3.2054,0.71852,2013-10-04T02:40:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01570,Dictyostelium_SRP,1315,Dictyostelium signal recognition particle,Osuch I,INFERNAL,90.0,249.1,63.6,,DdR20,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.33 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; snRNA;,Published; PMID:15333696,3,0,,\"\",0.72024,-8.73335,1600000,414900,429,283,true,-3.5665,0.70675,2013-10-04T02:40:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01572,DdR4,1264,Dictyostelium snoRNA DdR4,Osuch I,INFERNAL,70.0,122.6,38.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.68 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,3,0,,\"\",0.45593,-8.2161,1600000,230961,205,106,false,-3.2632,0.71822,2013-10-04T02:40:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01573,DdR5,1264,Dictyostelium snoRNA DdR5,Osuch I,INFERNAL,70.0,107.3,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.08 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted;,3,0,,\"\",0.4172,-10.00938,1600000,283896,171,81,false,-3.4008,0.71857,2013-10-04T02:40:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01574,DdR6,1264,Dictyostelium snoRNA DdR6,Osuch I,INFERNAL,90.0,103.0,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.47 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,3,0,,\"\",0.48672,-8.71098,1600000,342591,178,86,false,-3.3131,0.71848,2013-10-04T02:40:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01575,DdR7,1264,Dictyostelium snoRNA Dd7,Osuch I,INFERNAL,80.0,109.9,34.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.31 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,3,0,,\"\",0.49151,-10.12289,1600000,284619,196,98,false,-3.4985,0.71832,2013-10-04T02:41:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01576,DdR8,1264,D. discoideum snoRNA DdR8,Osuch I,INFERNAL,80.0,123.8,33.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,2,0,,\"\",0.49211,-9.50691,1600000,270928,182,88,false,-3.5,0.71838,2013-10-04T02:41:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01577,RNase_P,2441,Plasmodium RNase_P,Osuch I,INFERNAL,100.0,155.9,91.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,22,0,0,Gene; ribozyme;,Published; PMID:17901154,24,0,,\"\",0.46729,-11.01908,1600000,438291,837,626,true,-4.86,0.69922,2013-10-04T02:41:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01578,RUF1,2087,Plasmodium RNA of unkown function RUF1,Osuch I,INFERNAL,70.0,87.6,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.00 -Z 549862.597050 CM SEQDB,3,20,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,22,0,,\"\",0.52948,-9.77269,1600000,434035,410,264,true,-3.4553,0.70761,2013-10-04T02:41:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01579,RUF2,2088,Plasmodium RNA of unknown function RUF2,Osuch I,INFERNAL,90.0,110.7,52.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,22,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,22,0,,\"\",0.48727,-6.54011,1600000,320486,285,154,true,-3.0009,0.71808,2013-10-04T02:41:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01580,RUF3,2089,Plasmodium RNA of unkown function RU6-F3,Osuch I,INFERNAL,80.0,117.8,64.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,10,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,13,0,,\"\",0.40739,-9.93153,1600000,378997,316,187,true,-3.3021,0.71365,2013-10-04T02:41:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01581,RUF6-5,2089,Plasmodium RNA of unknown function RUF6-5,Osuch I,INFERNAL,70.0,71.8,67.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,125,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,20,0,,\"\",0.58935,-5.76987,1600000,300950,272,156,true,-3.1561,0.71783,2013-10-04T02:41:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01582,RUF4,2090,RNA of unknown function RUF4,Osuch I,INFERNAL,55.0,55.4,53.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,15,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,16,0,,\"\",0.53454,-9.52905,1600000,357727,427,277,true,-3.6357,0.707,2013-10-04T02:41:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01583,snoR01,1264,small nucleolar RNA snoR01,Osuch I,INFERNAL,55.0,63.1,46.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,22,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR; Clustal_alifold;,23,0,,\"\",0.41397,-5.07327,1600000,209140,174,83,true,-3.2168,0.71856,2013-10-04T02:41:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01584,snoR03,1264,small nucleolar RNA snoR03,Osuch I,INFERNAL,80.0,116.2,52.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34 -Z 549862.597050 CM SEQDB,2,9,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,10,0,,\"\",0.42986,-7.72088,1600000,298118,280,160,true,-2.8617,0.71711,2013-10-04T02:41:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01585,snoR07,1264,small nucleolar RNA snoR07,Osuch I,INFERNAL,44.0,46.9,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,4,24,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,24,0,,\"\",0.54907,-3.62667,1600000,279855,183,88,true,-2.7324,0.71849,2013-10-04T02:41:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01586,snoR09,1264,small nucleolar RNA snoR09,Osuch I,INFERNAL,50.0,68.1,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,17,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,18,0,,\"\",0.50873,-4.17264,1600000,252694,182,84,true,-3.3477,0.71851,2013-10-04T02:41:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01587,snoR10,1264,small nucleolar RNA snoR10,Osuch I,INFERNAL,65.0,83.1,41.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,24,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,25,0,,\"\",0.47865,-3.58172,1600000,244042,165,76,true,-3.3749,0.71867,2013-10-04T02:41:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01588,snoR13,1264,small nucleolar RNA snoR13,Osuch I,INFERNAL,60.0,77.9,38.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,48,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,23,0,,\"\",0.50886,-4.24067,1600000,206754,206,105,true,-3.0107,0.71822,2013-02-01T11:58:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01589,plasmodium_snoR11,1264,small nucleolar RNA snoR11,Osuch I,INFERNAL,80.0,161.3,58.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,9,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,10,0,,\"\",0.46704,-8.88405,1600000,362871,341,207,true,-3.3124,0.71166,2013-10-04T02:41:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01590,plasmodium_snoR14,1264,small nucleolar RNA snoR14,Osuch I,INFERNAL,90.0,114.6,46.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.15 -Z 549862.597050 CM SEQDB,4,12,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,13,0,,\"\",0.44107,-6.52468,1600000,279979,238,127,true,-2.7794,0.71797,2013-10-04T02:41:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01591,snoR15,1264,small nucleolar RNA snoR15,Osuch I,INFERNAL,50.0,50.7,45.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 CM SEQDB,4,25,0,0,Gene; snRNA; snoRNA; CD-box;,Published; PMID:17901154,26,0,,\"\",0.47707,-4.14293,1600000,260040,240,131,true,-2.7177,0.71804,2013-10-04T02:41:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01592,plasmodium_snoR17,1264,small nucleolar RNA snoR17,Osuch I,INFERNAL,51.0,51.2,50.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 36.87 -Z 549862.597050 CM SEQDB,5,17,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,19,0,,\"\",0.43931,-4.88316,1600000,269167,170,80,true,-3.1782,0.71855,2013-10-04T02:41:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01593,plasmodium_snoR16,1264,small nucleolar RNA snoR16,Osuch I,INFERNAL,70.0,78.7,40.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.70 -Z 549862.597050 CM SEQDB,3,26,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,25,0,,\"\",0.53144,-5.27359,1600000,343828,235,126,true,-2.488,0.71801,2013-10-04T02:41:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01594,snoR18,1264,small nucleolar RNA snoR18,Osuch I,INFERNAL,55.0,61.3,41.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.32 -Z 549862.597050 CM SEQDB,3,24,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,26,0,,\"\",0.45881,-4.44892,1600000,243886,187,90,true,-2.8635,0.71842,2013-10-04T02:41:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01595,snoR19,1264,small nucleolar RNA snoR19,Osuch I,INFERNAL,60.0,92.0,43.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,24,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,26,0,,\"\",0.46797,-2.52729,1600000,231022,195,96,true,-3.1611,0.71841,2013-10-04T02:41:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01596,plasmodium_snoR20,1264,small nucleolar RNA snoR20,Osuch I,INFERNAL,45.0,48.7,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.00 -Z 549862.597050 CM SEQDB,2,30,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,30,0,,\"\",0.53861,-7.80754,1600000,256206,173,82,false,-3.3373,0.71986,2013-10-04T02:41:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01597,snoR22,1264,small nucleolar RNA snoR22,Osuch I,INFERNAL,50.0,50.4,42.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27 -Z 549862.597050 CM SEQDB,3,33,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,25,0,,\"\",0.51005,-3.0768,1600000,249856,176,83,true,-3.0776,0.71852,2013-02-01T11:58:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01598,snoR23,1264,small nucleolar RNA snoR23,Osuch I,INFERNAL,70.0,70.5,57.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 37.05 -Z 549862.597050 CM SEQDB,3,23,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,23,0,,\"\",0.42623,-5.78206,1600000,246837,177,84,true,-3.3399,0.71853,2013-10-04T02:42:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01599,plasmodium_snoR24,1264,small nucleolar RNA snoR24,Osuch I,INFERNAL,55.0,63.3,45.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.00 -Z 549862.597050 CM SEQDB,4,17,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,16,0,,\"\",0.48037,-3.52248,1600000,219968,173,82,true,-3.2491,0.71855,2013-10-04T02:42:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01600,snoR25,1264,small nucleolar RNA snoR25,Osuch I,INFERNAL,50.0,61.9,45.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,24,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,24,0,,\"\",0.53756,-3.45155,1600000,255244,157,70,true,-3.2247,0.7188,2013-10-04T02:42:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01601,plasmodium_snoR26,1264,small nucleolar RNA snoR26,Osuch I,INFERNAL,50.0,52.7,44.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33 -Z 549862.597050 CM SEQDB,3,27,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,28,0,,\"\",0.3953,-10.10031,1600000,452442,282,160,true,-2.7287,0.71692,2013-10-04T02:42:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01602,plasmodium_snoR27,1264,small nucleolar RNA snoR27,Osuch I,INFERNAL,50.0,59.3,42.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,24,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,24,0,,\"\",0.47715,-6.53494,1600000,218384,259,143,true,-2.9317,0.71808,2013-10-04T02:42:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01603,snoR29,1264,small nucleolar RNA snoR29,Osuch I,INFERNAL,50.0,51.5,44.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,22,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,24,0,,\"\",0.51608,-2.78609,1600000,225668,177,84,true,-3.0579,0.71852,2013-10-04T02:42:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01604,plasmodium_snoR28,1264,small nucleolar RNA snoR28,Osuch I,INFERNAL,63.0,74.0,41.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,23,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,23,0,,\"\",0.55928,-2.56153,1600000,222693,169,78,true,-3.2903,0.71865,2013-10-04T02:42:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01605,plasmodium_snoR30,1264,small nucleolar RNA snoR30,Osuch I,INFERNAL,43.0,43.4,40.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,26,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,27,0,,\"\",0.54105,-3.83154,1600000,300292,165,74,true,-3.2464,0.71865,2013-10-04T02:42:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01606,plasmodium_snoR31,1264,small nucleolar RNA snoR31,Osuch I,INFERNAL,60.0,61.6,53.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,23,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,25,0,,\"\",0.45643,-7.34812,1600000,370770,260,145,true,-2.8414,0.7181,2013-10-04T02:42:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01607,ceN100,1264,C. elegans snoRNA ceN100,Osuch I,INFERNAL,80.0,105.5,66.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,15,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.54886,-6.74448,1600000,226476,242,132,true,-3.2938,0.71831,2013-10-04T02:42:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01608,ceN102,1264,Caenorhabditis snoRNA ceN80,Osuch I,INFERNAL,50.0,52.4,39.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,8,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.46646,-8.14641,1600000,259171,258,142,true,-3.0852,0.71803,2013-10-04T02:42:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01609,ceN103,1264,Caenorhabditis snoRNA ceN103,Osuch I,INFERNAL,55.0,60.5,42.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,7,0,,\"\",0.48417,-6.14296,1600000,257160,214,109,true,-2.9936,0.71817,2013-10-04T02:42:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01610,ceN104,1264,Caenorhabditis snoRNA ceN104,Osuch I,INFERNAL,65.0,80.1,37.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,14,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.4981,-7.7146,1600000,282238,648,163,true,-3.2322,0.71668,2013-10-04T02:43:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01611,ceN108,1264,Caenorhabditis snoRNA ceN108,Osuch I,INFERNAL,50.0,53.1,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,14,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,6,0,,\"\",0.67817,-3.47524,1600000,308567,159,70,true,-3.5805,0.71875,2013-10-04T02:43:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01612,ceN105,1264,Nematode snoRNA ceN105,Osuch I,INFERNAL,55.0,56.5,53.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,7,0,,\"\",0.50996,-7.40018,1600000,260933,288,140,true,-2.8518,0.71825,2013-10-04T02:43:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01613,ceN109,1264,Nematode snoRNA ceN109,Osuch I,INFERNAL,52.0,53.6,48.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,17,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,14,0,,\"\",0.69713,-3.52076,1600000,282226,154,67,true,-3.5974,0.71888,2013-10-04T02:43:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01614,ceN110,1264,Caenorhabditis snoRNA ceN110,Osuch I,INFERNAL,50.0,51.2,44.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.51833,-6.57048,1600000,207688,236,126,true,-3.0313,0.71831,2013-10-04T02:43:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01615,ceN111,1264,Nematode snoRNA ceN111,Osuch I,INFERNAL,43.0,43.6,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,29,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,26,0,,\"\",0.59235,-3.80638,1600000,278118,157,70,true,-3.739,0.71881,2013-10-04T02:43:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01617,ceN113,1264,Caenorhabditis snoRNA ceN113,Osuch I,INFERNAL,40.0,40.1,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,6,0,,\"\",0.50622,-6.65548,1600000,272237,241,131,true,-2.7685,0.71788,2013-10-04T02:43:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01618,ceN114,1264,Nematode snoRNA ceN114,Osuch I,INFERNAL,44.0,44.3,39.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,17,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,16,0,,\"\",0.60556,-4.70524,1600000,298956,148,63,true,-3.6347,0.719,2013-10-04T02:43:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01619,ceN115,2091,C. elegans sRNA ceN115,Osuch I,INFERNAL,44.0,45.9,42.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,6,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,2,0,,\"\",0.58769,-7.74551,1600000,334618,282,159,true,-3.0081,0.71726,2013-10-04T02:43:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01620,ceN125,1264,Nematode snoRNA ceN125,Osuch I,INFERNAL,60.0,65.9,35.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,10,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,8,0,,\"\",0.58013,-7.99743,1600000,275323,247,135,true,-3.2369,0.71832,2013-10-04T02:43:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01621,ceN126,1264,Caenorhabditis snoRNA ceN126,Osuch I,INFERNAL,80.0,150.4,48.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.53462,-6.79502,1600000,248016,243,131,true,-2.7505,0.71826,2013-10-04T02:43:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01622,ceN22,1264,Nematode snoRNA ceN22,Osuch I,INFERNAL,45.0,47.9,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,17,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,15,0,,\"\",0.60714,-4.26233,1600000,209213,168,76,true,-3.2493,0.71926,2013-10-04T02:43:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01623,ceN23-1,2091,Caenorhabditis sRNA ceN23-1,Osuch I,INFERNAL,40.0,72.5,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,14,0,0,Gene; sRNA;,Predicted; WAR; cmfinder;,7,0,,\"\",0.58092,-4.06043,1600000,182622,201,100,true,-3.2622,0.71851,2013-10-04T02:43:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01624,ceN27,1264,Nematode snoRNA ceN27,Osuch I,INFERNAL,38.0,38.1,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,2,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,7,0,,\"\",0.57403,-3.79839,1600000,192373,173,80,true,-3.5869,0.71944,2013-10-04T02:43:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01625,ceN28,1264,Caenorhabditis snoRNA ceN28,Osuch I,INFERNAL,70.0,104.5,47.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,7,0,,\"\",0.653,-3.6176,1600000,226403,184,88,true,-3.3081,0.71853,2013-10-04T02:43:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01626,ceN30,1264,Caenorhabditis snoRNA ceN30,Osuch I,INFERNAL,65.0,90.7,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,6,0,,\"\",0.66248,-3.38872,1600000,234518,162,73,true,-3.4956,0.71877,2013-10-04T02:43:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01627,ceN36-1,1264,Nematode snoRNA ceN36-1,Osuch I,INFERNAL,48.0,49.2,42.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,15,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,13,0,,\"\",0.5247,-7.71201,1600000,261683,241,130,true,-3.1793,0.71812,2013-10-04T02:43:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01628,ceN38,1264,Caenorhabditis snoRNA ceN38,Osuch I,INFERNAL,50.0,50.7,35.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,3,5,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,5,0,,\"\",0.52697,-7.02448,1600000,217364,251,137,true,-3.0866,0.71824,2013-10-04T02:43:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01629,ceN39,1264,Nematode snoRNA ceN39,Osuch I,INFERNAL,35.0,36.6,33.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,56,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,22,0,,\"\",0.59878,-7.17334,1600000,291805,244,131,true,-3.4666,0.71837,2013-10-04T02:43:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01630,ceN40,1264,Nematode snoRNA ceN40,Osuch I,INFERNAL,35.0,35.1,32.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.75 -Z 549862.597050 CM SEQDB,4,22,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,17,0,,\"\",0.65982,-6.1554,1600000,288198,231,123,true,-3.3271,0.71804,2013-10-04T02:43:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01631,ceN41,1264,Caenorhabditis snoRNA ceN41,Osuch I,INFERNAL,50.0,50.3,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,8,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.5365,-7.42452,1600000,270880,249,137,true,-3.3535,0.71833,2013-10-04T02:43:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01632,ceN42,1264,Caenorhabditis snoRNA ceN42,Osuch I,INFERNAL,42.0,44.2,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,7,0,,\"\",0.52189,-6.71813,1600000,269009,241,130,true,-3.0015,0.71829,2013-10-04T02:43:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01633,ceN43,1264,Caenorhabditis snoRNA ceN43,Osuch I,INFERNAL,50.0,71.7,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,6,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.55756,-7.12777,1600000,239698,240,129,true,-3.196,0.71827,2013-10-04T02:43:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01634,ceN44,1264,Caenorhabditis snoRNA ceN44,Osuch I,INFERNAL,70.0,88.8,35.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,7,0,,\"\",0.65348,-3.32673,1600000,223846,164,74,true,-3.6228,0.71876,2013-10-04T02:43:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01635,ceN45,1264,Caenorhabditis snoRNA ceN45,Osuch I,INFERNAL,60.0,90.2,47.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,10,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,8,0,,\"\",0.54386,-7.73428,1600000,293170,237,128,true,-3.2841,0.71843,2013-10-04T02:43:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01636,ceN46,1264,Caenorhabditis snoRNA ceN46,Osuch I,INFERNAL,50.0,90.9,31.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.59848,-6.93857,1600000,252666,236,125,true,-3.1476,0.71828,2013-10-04T02:43:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01637,ceN47,1264,Nematode snoRNA ceN47,Osuch I,INFERNAL,35.0,35.7,34.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,6,36,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted;Osuch I;,27,0,,\"\",0.57988,-8.96472,1600000,388420,270,155,true,-2.8072,0.71758,2013-10-04T02:43:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01638,ceN48,1264,Caenorhabditis snoRNA ceN48,Osuch I,INFERNAL,70.0,112.2,68.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,7,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,6,0,,\"\",0.55903,-7.30601,1600000,309183,235,125,true,-3.0649,0.71837,2013-10-04T02:44:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01639,ceN49,1264,Caenorhabditis snoRNA ceN49,Osuch I,INFERNAL,65.0,66.9,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,8,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.58712,-7.63614,1600000,342661,247,134,true,-3.0338,0.71822,2013-10-04T02:44:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01640,ceN51,1264,Caenorhabditis snoRNA ceN51,Osuch I,INFERNAL,65.0,65.9,31.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.60991,-6.91356,1600000,263923,257,141,true,-3.2962,0.71836,2013-10-04T02:44:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01641,ceN53,1264,Caenorhabditis snoRNA ceN53,Osuch I,INFERNAL,50.0,50.9,46.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,6,0,,\"\",0.64137,-3.65325,1600000,265711,164,74,true,-3.7278,0.71869,2013-10-04T02:44:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01642,ceN54,1264,Caenorhabditis snoRNA ceN54,Osuch I,INFERNAL,60.0,62.0,39.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,6,0,,\"\",0.65002,-5.03763,1600000,342800,155,68,true,-3.9796,0.7188,2013-10-04T02:44:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01643,ceN56,2091,Caenorhabditis sRNA ceN56,Osuch I,INFERNAL,60.0,69.5,59.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,4,6,0,0,Gene; sRNA;,Predicted; WAR; cmfinder;,6,0,,\"\",0.63098,-3.90743,1600000,188603,235,125,true,-3.1239,0.71831,2013-10-04T02:44:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01644,ceN59,1264,Caenorhabditis snoRNA ceN59,Osuch I,INFERNAL,50.0,76.3,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,11,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR; Clustal_alifold;,8,0,,\"\",0.50362,-7.1581,1600000,261715,250,137,true,-2.8035,0.71817,2013-10-04T02:44:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01645,ceN58,1264,Nematode snoRNA ceN58,Osuch I,INFERNAL,60.0,61.1,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,14,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,13,0,,\"\",0.55193,-8.06567,1600000,279639,254,139,true,-3.301,0.71818,2013-10-04T02:44:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01646,ceN63,1264,Caenorhabditis snoRNA ceN63,Osuch I,INFERNAL,65.0,85.5,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,7,0,,\"\",0.63192,-4.05344,1600000,265784,157,69,true,-3.4402,0.71878,2013-10-04T02:44:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01647,ceN61,1264,Caenorhabditis snoRNA ceN61,Osuch I,INFNERAL,60.0,63.7,50.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,6,0,,\"\",0.65921,-3.38083,1600000,212504,167,76,true,-3.6295,0.71873,2013-10-04T02:44:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01648,ceN65,1264,Caenorhabditis snoRNA ceN65,Osuch I,INFERNAL,48.0,48.2,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,6,0,,\"\",0.68157,-3.90346,1600000,294984,151,65,true,-3.7692,0.71894,2013-02-01T11:58:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01649,ceN67,1264,Caenorhabditis snoRNA ceN67,Osuch I,INFERNAL,70.0,73.4,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.50969,-7.39869,1600000,231393,237,130,true,-3.2308,0.7183,2013-10-04T02:44:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01650,ceN68,1264,Caenorhabditis snoRNA ceN48,Osuch I,INFERNAL,80.0,157.9,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold,1,0,,\"\",0.52304,-7.38649,1600000,273945,231,121,true,-2.9055,0.71827,2013-10-04T02:44:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01651,ceN69,1264,Caenorhabditis snoRNA ceN69,Osuch I,INFERNAL,45.0,47.2,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,6,0,,\"\",0.60458,-4.37206,1600000,246515,211,107,true,-2.95,0.71821,2013-10-04T02:44:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01652,ceN70,1264,Caenorhabditis snoRNA ceN70,Osuch I,INFERNAL,65.0,70.7,46.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,12,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,10,0,,\"\",0.66859,-3.89179,1600000,286545,159,71,true,-3.5198,0.71885,2013-10-04T02:44:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01653,ceN80,1264,Caenorhabditis snoRNA ceN80,Osuch I,INFERNAL,80.0,111.9,63.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,13,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,7,0,,\"\",0.57185,-6.83476,1600000,244718,242,131,true,-3.4224,0.71839,2013-10-04T02:44:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01654,ceN82,1264,Caenorhabditis snoRNA ceN82,Osuch I,INFERNAL,80.0,98.7,47.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,13,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.51994,-6.2094,1600000,273936,248,135,true,-3.1921,0.71816,2013-10-04T02:44:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01655,ceN84,1264,Caenorhabditis snoRNA ceN84,Osuch I,INFERNAL,80.0,81.9,55.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22 -Z 549862.597050 CM SEQDB,5,13,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.49161,-7.31509,1600000,236882,236,130,true,-2.9461,0.71819,2013-10-04T02:44:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01656,ceN72-3_ceN74-2,2091,Nematode sRNA ceN72-3_ceN74-2,\"Osuch I, Eberhardt R, Daub J\",INFERNAL,38.0,38.2,37.8,,ceN72-3,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,265,0,0,Gene; sRNA;,Predicted; WAR;,22,0,,\"\",0.59508,-5.07236,1600000,200424,280,131,true,-3.1181,0.71839,2013-10-04T02:44:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01658,ceN81,1264,Caenorhabditis snoRNA ceN81,Osuch I,INFERNAL,50.0,56.7,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.35 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.52859,-7.22487,1600000,251990,237,126,true,-3.391,0.71841,2013-10-04T02:44:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01659,ceN86,1264,Caenorhabditis snoRNA ceN86,Osuch I,INFERNAL,40.0,41.0,34.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,5,0,,\"\",0.5492,-6.68379,1600000,266774,233,124,true,-3.1658,0.71839,2013-10-04T02:44:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01660,ceN88,1264,Nematode snoRNA ceN88,Osuch I,INFERNAL,40.0,41.3,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,54,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,33,0,,\"\",0.55269,-9.19798,1600000,371913,255,140,true,-3.0876,0.7182,2013-10-04T02:44:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01661,ceN92,1264,Caenorhabditis snoRNA ceN92,Osuch I,INFERNAL,50.0,53.0,43.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,4,13,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.50447,-6.35774,1600000,273756,237,126,true,-3.0209,0.71821,2013-10-04T02:44:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01662,ceN89,1264,Caenorhabditis snoRNA ceN89,Osuch I,INFERNAL,48.0,49.0,36.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,7,0,,\"\",0.65758,-3.69324,1600000,283762,168,78,true,-3.1771,0.71867,2013-10-04T02:44:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01663,ceN93,2091,Caenorhabditis sRNA ceN93,Osuch I,INFERNAL,60.0,62.4,46.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,6,0,0,Gene; sRNA;,Predicted; WAR;,6,0,,\"\",0.5325,-6.60853,1600000,270631,242,131,true,-2.9957,0.71831,2013-10-04T02:44:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01664,ceN101,1264,Caenorhabditis snoRNA ceN101,Osuch I,INFERNAL,70.0,87.5,43.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,13,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAalifold;,6,0,,\"\",0.52748,-7.56683,1600000,283114,239,128,true,-3.2416,0.7183,2013-10-04T02:44:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01665,P13,1269,Pseudomonas sRNA P13,Osuch I,INFERNAL,70.0,99.2,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,2,0,,\"\",0.44585,-10.61303,1600000,359259,168,76,true,-3.1783,0.72145,2013-02-01T11:58:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01666,rox2,2092,Drosophila rox2 ncRNA,Osuch I,INFERNAL,55.0,56.9,47.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,8,0,0,Cis-reg;,Published; PMID:18039876,8,0,,\"\",0.56419,-4.29149,1600000,309331,158,71,true,-3.3205,0.71897,2013-10-04T02:44:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01667,rox1,2092,Drosophila rox1 ncRNA,Osuch I,INFERNAL,58.0,59.2,52.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Cis-reg;,Published; PMID:18039876,4,0,,\"\",0.63834,-4.12484,1600000,315230,142,59,true,-3.4346,0.71935,2013-10-04T02:44:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01668,P10,1269,Pseudomonas sRNA P10,Osuch I,INFERNAL,37.0,37.5,36.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17 -Z 549862.597050 CM SEQDB,8,74,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,44,0,,\"\",0.72541,-3.89071,1600000,268292,153,72,true,-3.4172,0.71896,2013-10-04T02:44:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01669,P14,1269,Pseudomonas sRNA P14,Osuch I,INFERNAL,45.0,46.6,38.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,64,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,66,0,,\"\",0.54499,-8.91773,1600000,344092,259,146,true,-2.9594,0.7182,2013-10-04T02:45:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01670,P17,1269,Pseudomonas sRNA P17,Osuch I,INFERNAL,45.0,46.9,39.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19 -Z 549862.597050 CM SEQDB,3,17,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,19,0,,\"\",0.62606,-9.84034,1600000,367125,433,282,true,-3.6411,0.70682,2013-10-04T02:45:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01671,P18,1269,Pseudomonas sRNA P18,Osuch I,INFERNAL,60.0,73.2,52.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26 -Z 549862.597050 CM SEQDB,4,28,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,29,0,,\"\",0.47467,-10.14726,1600000,312888,246,135,true,-3.1282,0.71821,2013-10-04T02:45:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01672,P2,1269,Psudomonas sRNA P2,Osuch I,INFERNAL,80.0,163.3,38.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,1,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,7,0,,\"\",0.50785,-8.07018,1600000,292924,255,138,true,-2.7908,0.71845,2013-10-04T02:45:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01673,PhrS,2093,PhrS,Burge SW,Burge SW,37.0,38.1,35.5,,P20,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,101,0,0,Gene; sRNA;,Predicted; WAR; cmfinder;,106,0,,\"\",0.63779,-10.10521,1600000,538719,333,213,true,-3.6066,0.71104,2013-10-04T02:45:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01674,P27,1269,Pseudomonas sRNA P27,Osuch I,INFERNAL,50.0,107.8,49.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,77,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,78,0,,\"\",0.60796,-8.06533,1600000,283038,321,192,true,-3.4053,0.71325,2013-10-04T02:45:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01675,CrcZ,2094,Pseudomonas sRNA CrcZ,\"Burge S, Osuch I\",INFERNAL,95.0,107.8,88.5,,P29,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,149,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,88,0,,\"\",0.67617,-9.83728,1600000,482118,564,398,true,-4.5799,0.70274,2013-10-04T02:45:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01676,P31,1269,Pseudomonas sRNA P31,Osuch I,INFERNAL,40.0,43.0,39.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,4,74,0,0,Gene; sRNA;,Predicted; WAR; cmfinder;,76,0,,\"\",0.53049,-8.7448,1600000,375943,174,82,true,-3.1868,0.71881,2013-10-04T02:45:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01677,P35,1269,Pseudomonas sRNA P35,Osuch I,INFERNAL,70.0,89.4,41.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAalifold;,2,0,,\"\",0.47555,-9.17356,1600000,514128,149,62,true,-3.0737,0.72425,2013-10-04T02:45:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01678,P37,1269,Pseudomonas sRNA P37,Osuch I,INFERNAL,50.0,91.1,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAalifold;,2,0,,\"\",0.55968,-8.01957,1600000,483075,149,62,true,-2.9811,0.72593,2013-10-04T02:45:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01679,P36,1269,Pseudomonas sRNA P36,Osuch I,INFERNAL,36.0,37.1,34.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,37,0,0,Gene; sRNA;,Predicted; RNAalifold;,37,0,,\"\",0.55892,-8.73518,1600000,464643,148,61,true,-3.3169,0.725,2013-10-04T02:45:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01680,P5,1269,Pseudomonas sRNA P5,Osuch I,INFERNAL,60.0,101.4,34.5,Identified as IgRs 491 in [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; WAR; Mafta_alifold;,2,0,,\"\",0.48848,-10.26885,1600000,432401,161,71,true,-3.0398,0.72232,2013-10-04T02:45:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01681,P4,1269,Pseudomonas sRNA P4,Osuch I,INFERNAL,37.0,37.3,35.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,6,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,7,0,,\"\",0.67092,-8.60794,1600000,387175,352,215,true,-3.2709,0.71101,2013-10-04T02:45:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01682,P8,1269,Pseudomonas sRNA P8,Osuch I,INFERNAL,60.0,105.4,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,2,0,,\"\",0.53651,-7.49394,1600000,350970,171,78,true,-3.5368,0.71888,2013-10-04T02:45:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01683,P6,1269,Pseudomonas sRNA P6,Osuch I,INFERNAL,50.0,51.6,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; RNAalifold;,6,0,,\"\",0.7061,-7.86658,1600000,297632,703,510,true,-4.5641,0.70072,2013-10-04T02:45:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01684,mascRNA-menRNA,1301,MALAT1-associated small cytoplasmic RNA/MEN beta RNA,\"Marz M, Stadler PF, Gardner PP\",\"Marz M, Stadler PF\",42.0,42.2,41.7,\"The seed alignment for this family was obtained from the following publication: Marz M, Stadler PF. Homology Search for Small Structured ncRNAs. Handbook of RNA biochemistry. 2010;1:1-1. There is no PMID available for this book chapter so only the authors have been referenced above.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,53,260,0,0,Gene;,Published; PMID:19041754;,75,0,,\"\",0.60803,-5.32505,1600000,290428,127,56,true,-3.1983,0.7193,2013-02-01T12:14:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01685,6S-Flavo,2461,6S-Flavo RNA,Weinberg Z,Published; PMID:20230605;,85.0,85.3,84.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.40 -Z 549862.597050 CM SEQDB,82,148,0,0,Gene; sRNA;,Published; PMID:20230605;,148,0,,\"\",0.51543,-3.24261,1600000,237828,212,109,true,-2.9325,0.71827,2013-10-04T02:45:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01686,Acido-1,1271,Acido-1 RNA,Weinberg Z,Published; PMID:20230605;,50.0,59.4,48.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,4,0,0,Gene; sRNA;,Published; PMID:20230605;,2,0,,\"\",0.52746,-10.39364,1600000,589014,144,64,true,-3.2209,0.72267,2013-02-01T11:59:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01687,Acido-Lenti-1,2095,Acido-Lenti-1 RNA,Weinberg Z,Published; PMID:20230605;,37.0,37.0,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,87,1891,0,0,Gene; sRNA;,Published; PMID:20230605;,151,0,,\"\",0.62586,-8.8728,1600000,779441,99,46,true,-3.6231,0.72842,2013-10-04T02:45:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01688,Actino-pnp,2096,Actino-pnp RNA,Weinberg Z,Published; PMID:20230605;,47.7,47.7,47.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,77,674,0,0,Cis-reg;,Published; PMID:20230605;,632,0,,\"\",0.645,-3.91924,1600000,236963,137,63,true,-3.6642,0.71913,2013-10-04T02:45:50.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01689,AdoCbl-variant,1456,AdoCbl variant RNA,\"Weinberg Z, Ontiveros-Palacios N\",Published; PMID:20230605;,36.5,36.5,36.4,AdoCbl riboswitches (clan CL00101) are cis-regulatory elements that recognise coenzyme B12 and modulate the expression of genes relate to Cobalamin obtention. AdoCbl riboswitch variant was first reported for Weinberg [1] and its structure was reported in 4FRN and 4FRG PDBs [2],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,146,278,0,0,Cis-reg; riboswitch;,Published; PMID:20230605;,131,0,,\"\",0.71206,-2.57318,1600000,203824,167,81,true,-3.8799,0.71876,2013-10-04T02:45:53.000Z,2022-10-29T17:55:16.000Z\n",
            "RF01690,Bacillaceae-1,2097,Bacillaceae-1 RNA,Weinberg Z,Published; PMID:20230605;,35.0,35.3,34.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,95,2235,0,0,Gene; sRNA;,Published; PMID:20230605;,120,0,,\"\",0.54042,-7.483,1600000,276917,129,67,true,-3.9973,0.719,2013-10-04T02:45:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01691,Bacillus-plasmid,2098,Bacillus-plasmid RNA,Weinberg Z,Published; PMID:20230605;,31.0,31.3,30.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,31,96,0,0,Gene; sRNA;,Published; PMID:20230605;,82,0,,\"\",0.65209,-4.18092,1600000,197960,201,61,true,-3.8137,0.71907,2013-10-04T02:45:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01692,Bacteroid-trp,2099,Bacteroidete tryptophan peptide leader RNA,Weinberg Z,Published; PMID:20230605;,49.0,49.1,48.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 37.90 -Z 549862.597050 CM SEQDB,35,180,0,0,Cis-reg; leader;,Published; PMID:20230605;,188,0,,\"\",0.42121,-6.10998,1600000,326727,241,131,true,-2.5237,0.71817,2013-10-04T02:45:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01693,Bacteroidales-1,2100,Bacteroidales-1 6S-Bacteroidales RNA suggested,Weinberg Z,Published; PMID:20230605;,60.0,61.9,59.3,\"Bacteroidales-1 was first reported in a comparative genomics-based approach of genome and metagenome sequences from Weinberg et al. 2010 [1] In a high-resolution transcriptome map study of Bacteroides thetaiotaomicron, Ryan et al. 2020 [2] report this sRNA as 6S RNA homologue of Bacteroides, in the study they obtain evidence of small product RNAs (pRNAs) related to 6S function. This sRNA presents a different secondary structure than the one of 6S (RF00013) and 6S-Flavo (RF01685) families.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,41,189,0,0,Gene; sRNA;,Published; PMID:20230605;,169,0,,\"\",0.47734,-11.73952,1600000,382879,294,179,true,-3.5556,0.71441,2013-10-04T02:46:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01694,Bacteroides-1,2101,Bacteroides-1 RNA,Weinberg Z,Published; PMID:20230605;,43.0,43.5,40.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,40,41,0,0,Gene; sRNA;,Published; PMID:20230605;,37,0,,\"\",0.5382,-6.14924,1600000,317350,168,83,true,-3.6928,0.71887,2013-10-04T02:46:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01695,C4,2102,C4 antisense RNA,Weinberg Z,Published; PMID:20230605;,45.0,45.0,44.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,456,2158,0,0,Gene; antisense;,Published; PMID:20230605;,632,0,,\"\",0.55593,-5.8084,1600000,228368,158,87,true,-3.7075,0.71862,2013-10-04T02:46:07.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01696,Chlorobi-1,2103,Chlorobi-1 RNA,Weinberg Z,Published; PMID:20230605;,50.0,52.5,38.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,20,0,0,Gene; sRNA;,Published; PMID:20230605;,9,0,,\"\",0.7642,-4.16137,1600000,245342,144,67,true,-3.9041,0.71902,2013-10-04T02:46:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01697,Chlorobi-RRM,1272,Chlorobi-RRM RNA,Weinberg Z,Published; PMID:20230605;,45.0,46.6,43.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.33 -Z 549862.597050 CM SEQDB,18,20,0,0,Cis-reg;,Published; PMID:20230605;,13,0,,\"\",0.63371,-4.99931,1600000,342545,153,73,true,-3.8076,0.71915,2013-02-01T11:59:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01698,Chloroflexi-1,2104,Chloroflexi-1 RNA,Weinberg Z,Published; PMID:20230605;,80.0,96.2,37.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,0,0,0,Gene; sRNA;,Published; PMID:20230605;,1,0,,\"\",0.59976,-5.45794,1600000,225035,177,84,true,-3.2411,0.71875,2013-10-04T02:46:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01699,Clostridiales-1,2105,Clostridiales-1 RNA,Weinberg Z,Published; PMID:20230605;,48.0,48.1,47.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,194,913,0,0,Gene; sRNA;,Published; PMID:20230605;,119,0,,\"\",0.60845,-6.66194,1600000,251184,264,166,true,-3.5948,0.71626,2013-10-04T02:46:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01700,Collinsella-1,2106,Collinsella-1 RNA,Weinberg Z,Published; PMID:20230605;,35.0,37.9,32.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,12,0,0,Gene; sRNA;,Published; PMID:20230605;,12,0,,\"\",0.66984,-8.33834,1600000,318497,565,196,true,-3.2998,0.7128,2013-10-04T02:46:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01701,Cyano-1,2107,Cyano-1 RNA,Weinberg Z,Published; PMID:20230605;,41.0,41.1,40.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.40 -Z 549862.597050 CM SEQDB,265,258,0,0,Gene; sRNA;,Published; PMID:20230605;,136,0,,\"\",0.54409,-6.50388,1600000,298824,153,69,true,-4.0098,0.71881,2013-10-04T02:46:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01702,Cyano-2,2108,Cyano-2 RNA,Weinberg Z,Published; PMID:20230605;,40.0,42.4,35.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,57,9,0,0,Gene; sRNA;,Published; PMID:20230605;,5,0,,\"\",0.62394,-10.04232,1600000,476553,325,204,true,-3.5276,0.71196,2013-10-04T02:46:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01703,Dictyoglomi-1,2109,Dictyoglomi-1 RNA,\"Yang J, Weinberg Z\",Published; PMID:20230605;,150.0,220.9,68.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,2,0,0,Gene; sRNA;,Published; PMID:20230605;,2,0,,\"\",0.57635,-10.07535,1600000,361502,374,236,true,-3.272,0.70941,2013-10-04T02:46:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01704,Glutamine-II,2110,Glutamine-II riboswitch (downstream peptide RNA),\"Weinberg Z, Ontiveros-Palacios N\",Published; PMID:20230605;,33.0,33.1,32.4,\"Glutamine-II riboswich also known as downstream-peptide was first reported in a metagenomic analysis of conserved ncRNAs [1] The structure of Glutamine-II riboswitch has been reported in PDB:6QN3 [2] The structure comprises three coaxial helical domains, the central one of which is a pseudoknot (PK). The major groove of the PK provides the binding site for L-glutamine\",Downstream-peptide;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,628,419,0,0,Cis-reg; riboswitch;,Published; PMID:20230605;,49,0,,\"\",0.65944,-8.21676,1600000,529470,111,60,true,-4.0693,0.71903,2013-10-04T02:46:28.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01705,Flavo-1,2111,Flavo-1 RNA,Weinberg Z,Published; PMID:20230605;,42.0,42.0,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.05 -Z 549862.597050 CM SEQDB,201,5164,0,0,Gene; sRNA;,Published; PMID:20230605;,230,0,,\"\",0.63631,-3.58292,1600000,137358,288,76,true,-3.7985,0.71871,2013-10-04T02:46:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01706,Gut-1,2112,Gut-1 RNA,Weinberg Z,Published; PMID:20230605;,50.0,50.2,49.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,38,72,0,0,Gene; sRNA;,Published; PMID:20230605;,19,0,,\"\",0.59142,-5.59355,1600000,267452,206,77,true,-3.7584,0.71874,2013-10-04T02:46:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01707,JUMPstart,2113,JUMPstart RNA,Weinberg Z,Published; PMID:20230605;,35.0,35.2,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,128,271,0,0,Cis-reg;,Published; PMID:20230605;,198,0,,\"\",0.66534,-5.62254,1600000,718063,88,38,true,-4.0614,0.72298,2013-10-04T02:46:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01708,L17DE,2114,L17 ribosomal protein downstream element,Weinberg Z,Published; PMID:20230605;,37.0,37.2,36.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.00 -Z 549862.597050 CM SEQDB,52,153,0,0,Cis-reg;,Published; PMID:20230605;,195,0,,\"\",0.54724,-6.8515,1600000,308375,446,76,true,-4.1987,0.71883,2013-10-04T02:46:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01709,Lacto-rpoB,2115,Lacto-rpoB RNA,Weinberg Z,Published; PMID:20230605;,40.0,40.7,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.00 -Z 549862.597050 CM SEQDB,31,76,0,0,Cis-reg;,Published; PMID:20230605;,96,0,,\"\",0.54803,-3.83682,1600000,391052,118,58,true,-4.0734,0.72489,2013-10-04T02:46:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01710,Lacto-usp,2116,Lacto-usp RNA,Weinberg Z,Published; PMID:20230605;,50.0,53.3,34.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,20,0,0,Gene; sRNA;,Published; PMID:20230605;,22,0,,\"\",0.52306,-4.69661,1600000,185344,177,90,true,-3.5947,0.71877,2013-10-04T02:46:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01711,Lnt,2117,Lnt RNA,Weinberg Z,Published; PMID:20230605;,55.0,61.1,47.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,11,0,0,Cis-reg;,Published; PMID:20230605;,12,0,,\"\",0.45064,-9.86761,1600000,456131,162,63,true,-3.173,0.71898,2013-10-04T02:46:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01712,Methylobacterium-1,2118,Methylobacterium-1 RNA,Weinberg Z,Published; PMID:20230605;,40.0,43.3,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,34,13,0,0,Gene; sRNA;,Published; PMID:20230605;,12,0,,\"\",0.51851,-8.00632,1600000,268833,186,104,true,-3.3866,0.71855,2013-10-04T02:46:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01713,Moco-II,2119,Moco-II RNA,Weinberg Z,Published; PMID:20230605;,35.0,42.2,30.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,18,0,0,Cis-reg;,Published; PMID:20230605;,6,0,,\"\",0.53535,-7.52229,1600000,403378,116,46,true,-3.2783,0.72856,2013-10-04T02:46:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01714,Ocean-V,2120,Ocean-V RNA,Weinberg Z,Published; PMID:20230605;,75.0,86.3,53.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,0,0,0,Gene; sRNA;,Published; PMID:20230605;,1,0,,\"\",0.64849,-3.75386,1600000,275088,159,72,true,-3.3649,0.71887,2013-10-04T02:46:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01715,Pedo-repair,2121,Pedo-repair RNA,Weinberg Z,Published; PMID:20230605;,150.0,192.5,46.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,6,0,0,Cis-reg;,Published; PMID:20230605;,1,0,,\"\",0.60378,-8.23951,1600000,292274,341,211,true,-3.4675,0.71139,2013-10-04T02:46:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01716,PhotoRC-I,1273,PhotoRC-I RNA,Weinberg Z,Published; PMID:20230605;,45.0,52.0,39.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,65,12,0,0,Cis-reg;,Published; PMID:20230605;,12,0,,\"\",0.662,-8.55655,1600000,884279,109,44,true,-4.0809,0.7199,2013-10-04T02:47:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01717,PhotoRC-II,1273,PhotoRC-II RNA,Weinberg Z,Published; PMID:20230605;,39.0,39.6,37.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,338,45,0,0,Cis-reg;,Published; PMID:20230605;,46,0,,\"\",0.50501,-5.79677,1600000,226143,299,94,true,-3.7433,0.71843,2013-02-01T11:59:04.000Z,2021-11-19T11:53:18.000Z\n",
            "RF01718,Polynucleobacter-1,2122,Polynucleobacter-1 RNA,Weinberg Z,Published; PMID:20230605;,37.0,38.8,31.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.00 -Z 549862.597050 CM SEQDB,16,2,0,0,Gene; sRNA;,Published; PMID:20230605;,3,0,,\"\",0.59845,-8.58965,1600000,259324,237,142,true,-3.5591,0.7181,2013-10-04T02:47:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01719,Pseudomon-1,2123,Pseudomon-1/ErsA RNA,Weinberg Z,Published; PMID:20230605;,50.0,59.3,49.5,\"Functions to regulate the AlgC enzyme.[2] Based on this information, the new name ErsA for the RNA was adopted. Negatively regulates major porin oprD expression responsible for uptake of carbapenem antibiotics, by base pairing with oprD 5'UTR (leading to increased bacterial resistance to meropenem)[4].Positively regulates amrZ mRNA and contributes to biofilm formation and motility.[5]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,19,94,0,0,Gene; sRNA;,Published; PMID:20230605;,91,0,,\"\",0.55811,-7.33901,1600000,233397,220,116,true,-3.2085,0.71832,2013-10-04T02:47:12.000Z,2022-05-06T13:10:05.000Z\n",
            "RF01720,Pseudomon-Rho,2124,Pseudomon-Rho RNA,Weinberg Z,Published; PMID:20230605;,45.0,48.0,40.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,16,74,0,0,Cis-reg;,Published; PMID:20230605;,80,0,,\"\",0.43488,-11.07245,1600000,390860,197,102,true,-2.8168,0.7186,2013-10-04T02:47:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01721,Pseudomon-groES,2125,Pseudomon-groES RNA,Weinberg Z,Published; PMID:20230605;,50.0,50.7,35.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.91 -Z 549862.597050 CM SEQDB,18,70,0,0,Cis-reg;,Published; PMID:20230605;,79,0,,\"\",0.49353,-11.06356,1600000,400309,209,111,true,-3.1767,0.71857,2013-10-04T02:47:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01722,Pyrobac-1,2126,Pyrobac-1 RNA,Weinberg Z,Published; PMID:20230605;,40.0,41.2,35.6,The motif has been shown by [2] to be part of sRNA202 and sRNA203 canonical and noncanonical pseudouridine guide RNA (H/ACA RNA).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,5,0,0,Gene; sRNA;,Published; PMID:20230605;,5,0,,\"\",0.55287,-5.97328,1600000,240541,133,60,true,-3.8731,0.72011,2013-10-04T02:47:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01723,Rhizobiales-2,2127,Rhizobiales-2 RNA,\"Corbino K, Weinberg Z\",Published; PMID:20230605;,40.0,43.6,37.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33 -Z 549862.597050 CM SEQDB,15,48,0,0,Gene; sRNA;,Published; PMID:20230605;,38,0,,\"\",0.39105,-12.05057,1600000,379268,197,104,true,-3.2459,0.71884,2013-10-04T02:47:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01724,SAM-Chlorobi,2128,SAM-Chlorobi RNA,Weinberg Z,Published; PMID:20230605;,35.0,47.8,32.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,17,0,0,Cis-reg;,Published; PMID:20230605;,18,0,,\"\",0.53687,-8.05654,1600000,290660,176,87,true,-2.904,0.71886,2013-10-04T02:47:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01725,SAM-I-IV-variant,1447,SAM-I/IV variant riboswitch,\"Weinberg Z, Ontiveros-Palacios N\",Published; PMID:20230605;,41.1,41.1,41.0,\"SAM-I-IV variant riboswitch is part of the clan CL00012 (SAM-I, SAM-IV and SAM-I-IV) which share a common SAM binding core. It is found upstream of a number of genes which code for proteins involved in methionine or cysteine biosynthesis. It has been found mostly in the order Actinomycetales [1]. PDB 4L81 and 4OQU has been reported for SAM-I-IV aptamer, which shows the SAM binding core and an alternative PK-2 subdomain shared with SAM-IV [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,417,348,0,0,Cis-reg; riboswitch;,Published; PMID:20230605;,325,0,,\"\",0.50816,-5.68665,1600000,198805,212,99,true,-3.8082,0.71843,2013-10-04T02:47:21.000Z,2022-10-29T17:55:16.000Z\n",
            "RF01726,SAM-II_long_loops,1769,SAM-II long loop,Weinberg Z,Published; PMID:20230605;,44.0,44.7,41.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,126,457,0,0,Cis-reg;,Published; PMID:20230605;,462,0,,\"\",0.57089,-5.54402,1600000,212783,222,60,true,-4.1579,0.7191,2013-10-04T02:47:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01727,SAM-SAH,2129,SAM/SAH riboswitch,\"Yang J, Weinberg Z, Ontiveros-Palacios N\",Published; PMID:20230605;,40.0,40.1,35.9,The structure of SAM/SAH riboswitch has been reported in PDB 6HAG [2],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,54,245,0,0,Cis-reg; riboswitch;,Published; PMID:20230605;,213,0,,\"\",0.60557,-8.45977,1600000,461961,104,49,true,-3.9696,0.72079,2013-10-04T02:47:28.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01728,STAXI,2130,STAXI RNA,Weinberg Z,Published; PMID:20230605;,40.5,40.5,40.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,43,163,0,0,Gene; sRNA;,Published; PMID:20230605;,66,0,,\"\",0.5694,-9.90459,1600000,373550,360,153,true,-3.2275,0.71802,2013-10-04T02:47:30.000Z,2021-11-19T11:53:19.000Z\n",
            "RF01729,Termite-flg,2131,Termite-flg RNA,Weinberg Z,Published; PMID:20230605;,40.0,43.9,34.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23 -Z 549862.597050 CM SEQDB,13,2,0,0,Cis-reg;,Published; PMID:20230605;,3,0,,\"\",0.51842,-7.48978,1600000,239526,198,104,true,-3.3651,0.71855,2013-10-04T02:47:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01730,Termite-leu,2132,Termite-leu RNA,Weinberg Z,Published; PMID:20230605;,51.0,51.0,50.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,0,0,0,Cis-reg;,Published; PMID:20230605;,1,0,,\"\",0.55138,-6.23364,1600000,214199,217,98,true,-2.9373,0.71877,2013-10-04T02:47:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01731,TwoAYGGAY,2133,TwoAYGGAY RNA,Weinberg Z,Published; PMID:20230605;,43.0,43.0,42.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,210,1693,0,0,Cis-reg;,Published; PMID:20230605;,185,0,,\"\",0.52306,-4.29938,1600000,232402,250,168,true,-4.0102,0.71602,2013-10-04T02:47:37.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01732,asd,2134,MarS sRNA,Weinberg Z,Published; PMID:20230605;,84.0,84.8,57.2,\"Interacts with the 5'UTR of the mga transcript (the multiple virulence gene regulator gene) and was renamed MarS for mag-activating regulatory sRNA. In MarS deletion strains expression of mga and several Mga-activated genes is reduced. This down-regulation of virulence factors leads to increased susceptibility of the deletion strain to phagocytosis, reduced adherence to human keratinocytes. However, the lack of MarS increased bacterial dissemination and tolerance towards oxidative stress.[2] Called SR914400 in [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,31,75,0,0,Gene; sRNA;,Published; PMID:20230605;,85,0,,\"\",0.54436,-9.0929,1600000,338303,295,143,true,-3.602,0.71807,2013-10-04T02:47:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01733,atoC,2135,atoC RNA,Weinberg Z,Published; PMID:20230605;,50.0,58.2,40.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,18,0,0,Cis-reg;,Published; PMID:20230605;,17,0,,\"\",0.5503,-5.58735,1600000,238069,176,93,true,-3.7062,0.71852,2013-10-04T02:47:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01734,Fluoride,2136,Fluoride riboswitch (crcB),\"Weinberg Z, Ontiveros-Palacios N\",Published; PMID:20230605;,32.0,32.0,31.9,\"Previously known as crcB RNA. The structure of Fluoride riboswitch has been reported in PDB:3VRS, 4EN5, 4ENA, 4ENB, 4ENC, 5KH8 [3,4]\",crcB;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,289,2093,0,0,Cis-reg; riboswitch;,Published; PMID:20230605;,1643,0,,\"\",0.62355,-4.99843,1600000,189666,198,64,true,-4.1293,0.72074,2013-10-04T02:47:43.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01735,epsC,2137,epsC RNA,\"Yang J, Weinberg Z\",Published; PMID:20230605;,42.0,42.8,33.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,30,216,0,0,Cis-reg;,Published; PMID:20230605;,172,0,,\"\",0.49214,-9.70648,1600000,310387,221,117,true,-3.0316,0.71845,2013-10-04T02:47:46.000Z,2021-11-19T11:53:19.000Z\n",
            "RF01736,flg-Rhizobiales,2138,flg-Rhizobiales RNA,\"Corbino K, Weinberg Z\",Published; PMID:20230605;,55.0,71.4,33.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,36,0,0,Cis-reg;,Published; PMID:20230605;,41,0,,\"\",0.54527,-10.72458,1600000,477085,250,143,true,-2.9737,0.7182,2013-10-04T02:47:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01737,flpD,2139,flpD RNA,Weinberg Z,Published; PMID:20230605;,40.0,45.1,33.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,5,0,0,Cis-reg;,Published; PMID:20230605;,9,0,,\"\",0.71108,-3.27441,1600000,372118,116,47,true,-3.0461,0.72917,2013-10-04T02:47:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01738,gabT,2140,gabT RNA,Weinberg Z,Published; PMID:20230605;,45.0,49.3,42.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,56,0,0,Cis-reg;,Published; PMID:20230605;,62,0,,\"\",0.58581,-3.18755,1600000,333133,131,61,true,-3.7428,0.71919,2013-10-04T02:47:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01739,glnA,2141,Glutamine riboswitch,\"Weinberg Z, Ontiveros-Palacios N\",Published; PMID:20230605;,41.0,41.0,40.9,\"Glutamine riboswitch was first reported in a metagenomic comparative analysis [1], it was later reported that this ncRNA selectively bind glutamine [2] and its structure was later reported in PDB: 5DDP, 5DDQ, 5DDR and 5DDO [3] Previously known as glnA RNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,940,772,0,0,Cis-reg; riboswitch;,Published; PMID:20230605;,87,0,,\"\",0.637,-4.90381,1600000,256113,145,53,true,-4.0127,0.71917,2013-10-04T02:47:52.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01740,gyrA,2142,gyrA RNA,Weinberg Z,Published; PMID:20230605;,45.0,51.1,33.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,86,0,0,Cis-reg;,Published; PMID:20230605;,67,0,,\"\",0.73987,-3.01884,1600000,225809,169,88,true,-3.0832,0.71885,2013-10-04T02:47:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01742,lactis-plasmid,2144,lactis-plasmid RNA,Weinberg Z,Published; PMID:20230605;,70.0,73.3,53.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,7,0,0,Gene; sRNA;,Published; PMID:20230605;,10,0,,\"\",0.50533,-5.86327,1600000,229019,205,106,true,-3.0719,0.71859,2013-10-04T02:48:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01743,leu-phe_leader,1274,leu/phe leader RNA from Lactococcus,Weinberg Z,Published; PMID:20230605;,70.0,74.3,45.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,7,0,0,Cis-reg; leader;,Published; PMID:20230605;,5,0,,\"\",0.4893,-8.86004,1600000,388133,280,162,true,-2.9385,0.71668,2013-02-01T11:59:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01744,livK,2145,livK RNA,Weinberg Z,Published; PMID:20230605;,65.0,78.0,34.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.99 -Z 549862.597050 CM SEQDB,8,18,0,0,Cis-reg;,Published; PMID:20230605;,9,0,,\"\",0.46413,-8.61555,1600000,280469,171,84,true,-3.3913,0.719,2013-10-04T02:48:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01745,manA,2146,manA RNA,Weinberg Z,Published; PMID:20230605;,55.2,55.2,50.9,manA RNA has been located in 5'UTR regions of genes of mannose or fructose metabolism. Has been proposed as a cis-regulatory element and as a riboswitch candidate for his complex secondary structure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,165,18,0,0,Cis-reg;,Published; PMID:20230605;,20,0,,\"\",0.65409,-7.8337,1600000,317872,509,193,true,-3.2899,0.71297,2013-10-04T02:48:06.000Z,2020-09-14T15:04:59.000Z\n",
            "RF01746,mraW,2147,mraW RNA,Weinberg Z,Published; PMID:20230605;,50.0,50.2,49.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.08 -Z 549862.597050 CM SEQDB,45,457,0,0,Cis-reg;,Published; PMID:20230605;,466,0,,\"\",0.6049,-3.67424,1600000,104210,180,108,true,-3.8662,0.71854,2013-10-04T02:48:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01747,msiK,2148,msiK RNA,Weinberg Z,Published; PMID:20230605;,33.0,34.6,29.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,120,652,0,0,Cis-reg;,Published; PMID:20230605;,648,0,,\"\",0.68407,-4.88406,1600000,172325,125,60,true,-4.4525,0.71926,2013-10-04T02:48:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01748,nuoG,2149,nuoG RNA,Weinberg Z,Published; PMID:20230605;,33.0,33.2,32.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.40 -Z 549862.597050 CM SEQDB,31,68,0,0,Cis-reg;,Published; PMID:20230605;,82,0,,\"\",0.58109,-8.31215,1600000,523537,440,44,true,-3.5315,0.72167,2013-10-04T02:48:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01749,pan,2150,pan motif,Weinberg Z,Published; PMID:20230605;,36.2,36.3,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.77 -Z 549862.597050 CM SEQDB,74,480,0,0,Cis-reg;,Published; PMID:20230605;,478,0,,\"\",0.52352,-6.52858,1600000,232138,338,93,true,-3.3228,0.71855,2013-10-04T02:48:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01750,ZMP-ZTP,2151,ZMP/ZTP riboswitch,\"Weinberg Z, Ontiveros-Palacios N\",Published; PMID:20230605;,41.0,41.0,40.9,\"Previously known as pfl RNA. ZMP is also known as AICAR. ZMP/ZTP riboswitch (pfl riboswitch)was first recognise by a comparative analysis [1] ZMP/ZTP riboswitch selectively binds ZMP and regulates genes associated with purine biosynthesis and one-carbon metabolism [3] The structure of ZMP/ZTP has been reported in [2,3,4,5,6,7]\",pfl;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,185,1737,0,0,Cis-reg; riboswitch;,Published; PMID:20230605;,1308,0,,\"\",0.47726,-10.45036,1600000,367261,478,86,true,-3.6919,0.71867,2013-10-04T02:48:18.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01751,potC,2152,potC RNA,Weinberg Z,Published; PMID:20230605;,44.0,45.3,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25 -Z 549862.597050 CM SEQDB,136,1,0,0,Cis-reg;,Published; PMID:20230605;,2,0,,\"\",0.62325,-3.37609,1600000,218428,147,70,true,-3.805,0.71915,2013-10-04T02:48:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01752,psaA,2153,psaA RNA,Weinberg Z,Published; PMID:20230605;,50.0,52.1,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18 -Z 549862.597050 CM SEQDB,31,14,0,0,Cis-reg;,Published; PMID:20230605;,20,0,,\"\",0.61082,-8.81153,1600000,321041,317,195,true,-3.0043,0.71289,2013-10-04T02:48:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01753,psbNH,2154,psbNH RNA,Weinberg Z,Published; PMID:20230605;,45.0,47.0,31.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,39,8,0,0,Cis-reg;,Published; PMID:20230605;,15,0,,\"\",0.63518,-8.17083,1600000,525932,131,60,true,-3.6145,0.71912,2013-10-04T02:48:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01754,radC,2155,radC RNA,Weinberg Z,Published; PMID:20230605;,40.0,40.1,39.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,148,424,0,0,Cis-reg;,Published; PMID:20230605;,128,0,,\"\",0.47972,-12.15207,1600000,462046,278,148,true,-3.3452,0.71812,2013-10-04T02:48:28.000Z,2021-11-19T11:53:19.000Z\n",
            "RF01755,rmf,2156,rmf RNA,Weinberg Z,Published; PMID:20230605;,45.0,48.3,32.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,72,0,0,Cis-reg;,Published; PMID:20230605;,79,0,,\"\",0.58588,-5.93678,1600000,284839,236,132,true,-3.0745,0.71828,2013-10-04T02:48:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01756,rne-II,2157,rne-II RNA,Weinberg Z,Published; PMID:20230605;,50.0,58.3,40.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,74,0,0,Cis-reg;,Published; PMID:20230605;,82,0,,\"\",0.6994,-8.27941,1600000,323910,1010,296,true,-3.9046,0.70613,2013-10-04T02:48:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01757,sbcD,2158,sbcD RNA,Weinberg Z,Published; PMID:20230605;,40.0,41.9,39.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,18,0,0,Gene; sRNA;,Published; PMID:20230605;,20,0,,\"\",0.52799,-10.32536,1600000,354524,271,155,true,-2.8207,0.71808,2013-10-04T02:48:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01758,sucA-II,2159,sucA-II RNA,Weinberg Z,Published; PMID:20230605;,70.0,132.9,63.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,73,0,0,Cis-reg;,Published; PMID:20230605;,82,0,,\"\",0.6003,-9.49777,1600000,347519,344,218,true,-3.4075,0.71088,2013-10-04T02:48:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01759,sucC,2160,sucC RNA,Weinberg Z,Published; PMID:20230605;,50.0,61.9,41.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,63,0,0,Cis-reg;,Published; PMID:20230605;,74,0,,\"\",0.65156,-3.98935,1600000,224940,152,70,true,-3.2455,0.7189,2013-10-04T02:48:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01760,traJ-II,2161,traJ-II RNA,Weinberg Z,Published; PMID:20230605;,40.0,40.6,38.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,47,153,0,0,Cis-reg;,Published; PMID:20230605;,146,0,,\"\",0.55811,-5.61202,1600000,217658,184,104,true,-3.1966,0.71854,2013-10-04T02:48:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01761,wcaG,2162,wcaG RNA,Weinberg Z,Published; PMID:20230605;,42.0,42.4,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,108,22,0,0,Cis-reg;,Published; PMID:20230605;,21,0,,\"\",0.53363,-8.12131,1600000,289184,183,98,true,-3.6473,0.7185,2013-10-04T02:48:42.000Z,2021-11-19T11:53:19.000Z\n",
            "RF01762,whalefall-1,2163,Whalefall-1 RNA,Weinberg Z,Published; PMID:20230605;,37.0,37.6,35.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,41,0,0,Gene; sRNA;,Published; PMID:20230605;,11,0,,\"\",0.73713,-4.57358,1600000,316002,125,56,true,-3.6951,0.71925,2013-10-04T02:48:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01763,ykkC-III,1701,Guanidine-III riboswitch,\"Weinberg Z, Ontiveros-Palacios N\",Published; PMID:20230605;,36.0,36.0,35.8,\"Although the structure of ykkC-III RNAs does not resemble ykkC/yxkD RNAs, both have a structure complexity that led to the proposal that they represent riboswitches. The ykkC-III motif has a rigidly conserved ACGA sequence within it that resembles a less rigidly conserved ACGA or ACGG sequence found in mini-ykkC RNAs, but it is unknown whether this observation relates to a biological relationship. Biochemical validation has been presented to show that this motif is a third class of guanidine riboswitches [2]. The structure of Guanidine-III riboswitch has been reported in [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,41,662,0,0,Cis-reg;,Published; PMID:20230605;,626,0,,\"\",0.57837,-7.71323,1600000,204017,132,67,true,-4.3901,0.71901,2013-10-04T02:48:55.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01764,yjdF,2164,yjdF RNA,Weinberg Z,Published; PMID:20230605;,45.0,45.2,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.00 -Z 549862.597050 CM SEQDB,62,308,0,0,Cis-reg;,Published; PMID:20230605;,327,0,,\"\",0.59015,-2.43965,1600000,189239,447,101,true,-3.6714,0.71847,2013-10-04T02:48:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01765,srg1,2165,srg1,\"Gardner PP, Moore B\",Gardner PP,70.0,144.0,36.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,9,0,0,Gene;,Predicted; Gardner PP,11,0,,\"\",0.60899,-9.00673,1600000,324689,754,556,true,-4.697,0.70002,2013-10-04T02:48:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01766,cspA,2166,cspA thermoregulator,Moore B,Published; PMID:20129052,78.0,78.0,77.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,1597,0,0,Cis-reg; thermoregulator;,Published; PMID:20129052,923,0,,\"\",0.67165,-8.82464,1600000,406903,591,428,true,-4.2929,0.70213,2013-10-04T02:49:01.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01767,SMK_box_riboswitch,2167,SMK box translational riboswitch (SAM-III),Gardner PP,Gardner PP,45.0,45.1,44.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,25,167,0,0,Cis-reg; riboswitch;,Published; PMID:18806797; Gardner PP,189,0,,\"\",0.72442,-2.20069,1600000,216651,183,82,true,-3.5789,0.71884,2013-10-04T02:49:09.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01768,flavi_FSE,2488,ribosomal frameshift site,Boursnell C,Published; PMID:19196463,43.0,43.0,37.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,34,0,0,Cis-reg; frameshift_element;,Published; PMID:19196463,9,0,,\"\",0.61034,-8.35325,1600000,405951,145,62,true,-3.792,0.71911,2013-10-04T02:49:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01769,greA,2169,Enterobacteria greA leader,Moore B,Published; PMID:20920266,60.0,61.4,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,99,0,0,Cis-reg; leader;,Published; PMID:20920266,111,0,,\"\",0.50759,-9.07611,1600000,304669,225,124,true,-3.3123,0.71822,2013-10-04T02:49:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01770,rimP,2170,Gammaprotebacteria rimP leader,Moore B,Published; PMID:20920266,47.0,47.5,46.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,46,240,0,0,Cis-reg; leader;,Published; PMID:20920266,266,0,,\"\",0.47074,-8.55631,1600000,286659,353,111,true,-3.3306,0.71841,2013-10-04T02:49:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01771,rnk_leader,2171,Enterobacteria rnk leader,Moore B,Published; PMID:20920266,50.0,53.8,47.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,13,36,0,0,Cis-reg; leader;,Published; PMID:20920266,46,0,,\"\",0.53205,-5.5832,1600000,216709,207,109,true,-3.217,0.71848,2013-10-04T02:49:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01772,rnk_pseudo,2172,Pseudomonas rnk leader,Moore B,Published; PMID:20920266,55.0,59.4,47.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,38,0,0,Cis-reg; leader;,Published; PMID:20920266,47,0,,\"\",0.51371,-8.0183,1600000,324031,194,108,true,-3.3963,0.71845,2013-10-04T02:49:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01773,rpsL_pseudo,2173,Pseudomonas rpsL leader,Moore B,Published; PMID:20920266,70.0,77.9,67.7,,rpsL_psuedo;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,72,0,0,Cis-reg; leader;,Published; PMID:20920266,76,0,,\"\",0.55123,-9.79953,1600000,441659,256,128,true,-3.4115,0.71836,2013-10-04T02:49:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01774,rpsL_ricks,2174,Rickettsia rpsL leader,Moore B,Published; PMID:20920266,60.0,64.3,48.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 37 -Z 549862.597050 CM SEQDB,7,4,0,0,Cis-reg; leader;,Published; PMID:20920266,11,0,,\"\",0.40222,-7.73115,1600000,262501,272,131,true,-2.9818,0.71808,2013-10-04T02:49:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01775,RsaOG,2175,RNA S.aureus Orsay G,Moore B,Published; PMID:20200491;,50.0,66.7,43.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,21,0,0,Gene; sRNA;,Published; PMID:20200491;,25,0,,\"\",0.47434,-8.34638,1600000,282575,274,148,true,-3.1015,0.71796,2013-10-04T02:49:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01776,RatA,2176,RNA anti-toxin A,Moore B,Predicted; Clustal;,49.0,49.1,48.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33 -Z 549862.597050 CM SEQDB,16,216,0,0,Gene; antitoxin;,Predicted; RNAalifold;,121,0,,\"\",0.49559,-4.80235,1600000,398379,175,90,true,-3.5662,0.71855,2013-10-04T02:49:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01778,Evf-2_5p,2497,Embryonic ventral forebrain-2 long ncRNA,\"Moore B, Gardner PP\",Published;,40.0,45.1,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,94,0,0,Gene; lncRNA;,Predicted; CMfinder;,94,0,,\"\",0.46392,-11.20754,1600000,306377,251,142,false,-3.1128,0.71785,2013-10-04T02:49:27.000Z,2022-10-29T17:42:08.000Z\n",
            "RF01779,AS1726,2178,AS1726 sRNA,Moore B,Published; PMID:19555452,50.0,98.9,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene;,Published; PMID:19555452,3,0,,\"\",0.4824,-8.69706,1600000,333544,171,78,true,-2.9901,0.71893,2013-10-04T02:49:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01780,AS1890,2178,AS1890 sRNA,Moore B,Published; PMID:19555452,39.0,39.8,33.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20 -Z 549862.597050 CM SEQDB,2,13,0,0,Gene;,Published; PMID:19555452,15,0,,\"\",0.52183,-7.67637,1600000,264462,215,109,true,-2.8891,0.7186,2013-10-04T02:49:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01781,ASdes,2178,ASdes TB sRNA,Moore B,Published; PMID:19555452,40.0,40.1,38.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,26,729,0,0,Gene; sRNA;,Published; PMID:19555452,431,0,,\"\",0.67624,-4.23859,1600000,279454,149,68,true,-3.1933,0.71906,2013-10-04T02:49:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01782,ASpks,2178,ASpks TB sRNA,Moore B,Published; PMID:19555452,52.0,52.0,51.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,77,4571,0,0,Gene; antisense;,Published; PMID:19555452,375,0,,\"\",0.53853,-8.21802,1600000,370018,157,75,true,-3.4299,0.71888,2013-10-04T02:49:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01783,b55,2178,Mycobacterium B11,Moore B,Published; PMID:19555452,70.0,85.3,44.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Published; PMID:19555452,3,0,,\"\",0.50021,-9.1931,1600000,489200,148,61,true,-3.3236,0.7242,2013-10-04T02:49:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01784,bablM,2179,bablM sRNA,Moore B,Published; PMID:19531245,38.0,38.5,37.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33 -Z 549862.597050 CM SEQDB,12,10,0,0,Gene; sRNA;,Published; PMID:19531245,13,0,,\"\",0.48411,-4.63299,1600000,310067,156,70,true,-3.5687,0.7189,2013-10-04T02:49:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01785,blv_FSE,2180,ribosomal frameshift site,Boursnell C,Published; PMID:15629717,34.0,34.4,33.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Cis-reg; frameshift_element;,Published; PMID:15629717,2,0,,\"\",0.60356,-5.30735,1600000,359089,100,27,true,-2.2939,0.76956,2013-10-04T02:49:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01786,c-di-GMP-II,2181,Cyclic di-GMP-II riboswitch,\"Weinberg Z, Moore B\",Predicted; Weinberg Z;,52.0,52.0,51.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,54,532,0,0,Cis-reg; riboswitch;,Predicted; Weinberg Z;,292,0,,\"\",0.65097,-3.0205,1600000,154699,251,85,true,-4.0245,0.71855,2013-10-04T02:49:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01787,drz-agam-1,2182,drz-agam-1 ribozyme,\"Moore B, Burge S\",Published; PMID:19965505,37.0,37.0,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,9217,0,0,Gene; ribozyme;,Published; PMID:19965505,38,0,,\"\",0.57699,-6.39504,1600000,279775,195,82,true,-3.6003,0.71882,2013-10-04T02:49:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01788,drz-agam-2-2,2182,drz-agam-2-2 ribozyme,Moore B,Published; PMID:19965505,38.0,38.3,37.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,2137,0,0,Gene; ribozyme;,Published; PMID:19965505,24,0,,\"\",0.67931,-8.27774,1600000,370417,320,191,true,-3.6591,0.71318,2013-10-04T02:49:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01789,EBER1,2479,Epstein-Barr virus EBER1,Moore B,Published; PMID:17381320,50.0,51.5,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; sRNA;,Published; PMID:17381320,4,0,,\"\",0.56489,-9.17216,1600000,297361,486,166,true,-2.9901,0.71648,2013-10-04T02:49:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01790,htlv_FSE,2184,ribosomal frameshift site,Boursnell C,Published; PMID:8416368,45.0,48.4,38.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,8,0,0,Cis-reg; frameshift_element;,Published; PMID:8416368,9,0,,\"\",0.59348,-6.32751,1600000,580741,112,43,true,-3.833,0.71969,2013-10-04T02:49:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01791,F6,2178,F6 TB sRNA,Moore B,Published; PMID:19555452,46.0,47.3,45.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,187,0,0,Gene; sRNA;,Published; PMID:19555452,187,0,,\"\",0.55198,-4.90347,1600000,192527,129,58,true,-3.8453,0.7193,2013-10-04T02:49:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01792,eiav_FSE,2185,ribosomal frameshift site,Boursnell C,Published; PMID:3003905,40.0,42.3,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,0,0,0,Cis-reg; frameshift_element;,Published; PMID:3003905,1,0,,\"\",0.63789,-8.76575,1600000,806758,104,35,true,-3.5834,0.72454,2013-10-04T02:49:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01793,ffh,2179,ffh sRNA,\"Moore B, Meyer MM\",Published; PMID:19531245,35.0,35.0,34.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,89,539,0,0,Gene; sRNA;,Published; PMID:19531245,562,0,,\"\",0.55841,-6.68728,1600000,258761,106,52,true,-4.188,0.72034,2013-10-04T02:49:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01794,sok,2412,sok antitoxin (CssrC),\"Gardner PP, Moore B\",Published; PMID:17462020,59.0,59.7,57.5,Also identified in Shigella as CssrC [3].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,91,202,0,0,Gene; antitoxin;,Published; PMID:17462020,149,0,,\"\",0.61139,-8.33308,1600000,319401,355,157,true,-3.1011,0.71761,2013-10-04T02:50:03.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01795,FourU,2187,FourU thermometer RNA element,\"Bateman A, Moore B\",Published; PMID:17630972,60.0,61.9,36.8,FourU is a small thermoregulator element found upstream of the agsA gene in Salmonella species. The four Us occlude the Shine-Dalgarno (SD) sequence by base pairing. It has been suggested that this element is related to that found in Yersina pestis LcrF. Two further potential FourU regulators were found in S. aureus groES and B. melitensis dnaJ [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,16,0,0,Cis-reg; thermoregulator;,Published; PMID:17630972; Bateman A,19,0,,\"\",0.56371,-7.2577,1600000,353448,151,65,true,-3.5099,0.71917,2013-10-04T02:50:06.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01796,frnS,1275,Fumarate/nitrate reductase regulator sRNA,Moore B,Published; PMID:20070527,60.0,65.8,56.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,108,0,0,Gene; sRNA;,Published; PMID:20070527,116,0,,\"\",0.50563,-8.31447,1600000,332501,221,119,true,-3.0537,0.71839,2013-02-01T11:59:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01797,fstAT,2188,Fst antitoxin sRNA,Moore B,Predicted; Mafft;,72.0,72.2,71.9,It is located on the opposite strand of SprA1 sRNA (PMID: 22198463).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.77 -Z 549862.597050 CM SEQDB,22,35,0,0,Gene; antitoxin;,Predicted; RNAalifold;,33,0,,\"\",0.55843,-4.97814,1600000,214542,187,94,true,-3.5011,0.71852,2013-10-04T02:50:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01798,g2,2178,G2,Moore B,Published; PMID:19555452,70.0,102.4,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene;,Published; PMID:19555452,3,0,,\"\",0.5366,-5.73445,1600000,235162,156,67,true,-3.3203,0.7225,2013-10-04T02:50:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01800,gadd7,2189,growth arrested DNA-damage inducible gene 7,Moore B,Published; PMID:2658040,70.0,181.3,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; lncRNA;,Predicted;,3,0,,\"\",0.66986,-8.65412,1600000,289987,986,754,true,-5.0436,0.69823,2013-10-04T02:50:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01802,HSUR,2190,Herpesvirus saimiri U RNA1/RNA2,\"Moore B, Daub J\",Published; PMID:17381320,50.0,53.4,43.8,,HSUR1,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,2,0,0,Gene; snRNA;,Published; PMID:17381320,2,0,,\"\",0.39036,-9.11482,1600000,272300,256,123,true,-2.7972,0.71811,2013-10-04T02:50:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01803,GABA3,2191,Editing element of GABA-3 exon 9,\"Gardner PP, Moore B\",Gardner PP,57.0,58.4,55.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,52,689,0,0,Cis-reg;,Published; PMID:17369310,132,0,,\"\",0.71737,-2.67387,1600000,346645,131,54,true,-4.1435,0.71946,2013-10-04T02:50:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01804,Lambda_thermo,2192,Lambda phage CIII thermoregulator element,\"Bateman A, Moore B\",Published; PMID:2532257,50.0,83.1,38.7,There is an evidence supporting presence of a riboswitch upstream of the thermometer.[2],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,20,0,0,Cis-reg; thermoregulator;,Published; PMID:2532257; Bateman A,24,0,,\"\",0.68327,-4.24266,1600000,298236,141,59,true,-3.7148,0.71928,2013-10-04T02:50:22.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01807,GIR1,2193,GIR1 branching ribozyme,\"Gardner PP, Moore B\",Predicted; Infernal;,100.0,115.9,66.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,0,0,0,Gene; ribozyme;,Published; PMID:16141078,12,0,,\"\",0.57192,-9.0115,1600000,309842,370,199,true,-3.6148,0.71232,2013-10-04T02:50:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01808,MicX,2194,MicX Vibrio cholerae sRNA,\"Gardner PP, Moore B\",Gardner PP,75.0,96.6,71.5,,A10,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,30,0,0,Gene; sRNA;,Predicted; CMfinder,38,0,,\"\",0.60974,-7.85346,1600000,320960,317,194,true,-3.3118,0.71287,2013-10-04T02:50:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01809,symR,2195,SymR antitoxin,Moore B,Published; PMID:17462020,61.0,62.2,60.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,17,0,0,Gene; antitoxin;,Published; PMID:17462020,34,0,,\"\",0.55663,-6.89869,1600000,282915,163,77,true,-3.8147,0.71906,2013-10-04T02:50:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01810,pntA,2179,pntA sRNA,Moore B,Published; PMID:19531245,54.0,54.0,53.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,6,0,0,Gene; sRNA;,Published; PMID:19531245,8,0,,\"\",0.48918,-8.44395,1600000,626799,145,62,true,-3.3711,0.71925,2013-10-04T02:50:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01811,PtaRNA1,2196,Plasmid transferred anti-sense RNA,Moore B,Published; PMID:20220307,47.0,48.2,46.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,30,0,0,Gene; antitoxin;,Published; PMID:20220307,37,0,,\"\",0.57572,-6.74747,1600000,278772,155,72,true,-3.2409,0.71903,2013-10-04T02:50:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01812,Pxr,2197,Pxr regulatory sRNA,Moore B,Published; PMID:20489016,80.0,81.6,79.7,\"Downregulates genes responsible for the formation of fruiting bodies in Myxococcus xanthus, inhibiting the transition from growth to development in the presence of abundant nutrients.[1] Two-component system operon (histidine kinase gene and a Ïƒ54 response regulator) is associated with production and processing of Pxr sRNA.[2] Folds into 3 steam loops. SL1 and SL 2 are highly conserved across mycobacteria and SL1 is necessary for the regulatory function. However, a conserved eight-base-pair segment of the variable SL3 is necessary for PxR accumulation and multicellular development.[3]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; sRNA;,Published; PMID:20489016,6,0,,\"\",0.50262,-9.04178,1600000,263608,214,109,true,-2.8493,0.7187,2013-10-04T02:50:34.000Z,2022-05-06T13:10:05.000Z\n",
            "RF01813,rdlD,2217,rdlD antitoxin,\"Gardner PP, Moore B\",Predicted; Gardner PP;,50.0,51.2,48.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,52,25,0,0,Gene; antitoxin;,Predicted; Gardner PP;,49,0,,\"\",0.60494,-5.47102,1600000,275368,147,66,true,-3.6258,0.71917,2013-10-04T02:50:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01814,rhtB,2179,rhtB sRNA,Moore B,Published; PMID:19531245,55.0,57.0,52.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,9,0,0,Gene; sRNA;,Published; PMID:19531245,12,0,,\"\",0.60786,-1.95137,1600000,199900,170,83,true,-4.0146,0.71883,2013-10-04T02:50:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01815,rpsB,2179,rpsB sRNA,Moore B,Published; PMID:19531245,40.0,40.0,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,150,0,0,Gene; sRNA;,Published; PMID:19531245,150,0,,\"\",0.5506,-9.58507,1600000,345611,344,144,true,-3.0871,0.71854,2013-10-04T02:50:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01816,RsaA,2199,RNA Staph. aureus A,Moore B,Published; PMID:19786493,50.0,61.6,49.8,\"Represses the translation of the transcriptional regulator MgrA by binding to its mRNA, enhances biofilm formation and decreases bacterial virulence.[2] Other mRNAs, including SsaA-like enzymes involved in peptidoglycan metabolism and the secreted anti-inflammatory FLIPr protein were validated as direct targets of RsaA.[3]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,11,0,0,Gene; sRNA;,Published; PMID:19786493,18,0,,\"\",0.49766,-8.41173,1600000,324439,240,134,true,-3.2497,0.71828,2013-10-04T02:50:43.000Z,2022-05-06T13:10:05.000Z\n",
            "RF01817,RsaB,2199,RNA Staph. aureus A,Moore B,Published; PMID:19786493,55.0,58.5,38.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Published; PMID:19786493,3,0,,\"\",0.47536,-8.45213,1600000,542076,146,60,true,-2.7333,0.72798,2013-10-04T02:50:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01818,RsaC,2199,RNA Staph. aureus C,Moore B,Published; PMID:19786493,100.0,215.3,31.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Published; PMID:19786493,4,0,,\"\",0.66897,-8.00835,1600000,317893,622,440,true,-4.2981,0.70186,2013-10-04T02:50:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01819,RsaD,2199,RNA Staph. aureus D,Moore B,Published; PMID:19786493,50.0,90.4,46.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,17,0,0,Gene; sRNA;,Published; PMID:19786493,21,0,,\"\",0.50521,-9.36924,1600000,321184,324,189,true,-3.4755,0.71346,2013-10-04T02:50:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01820,RsaE,2199,RNA Staph. aureus E (RoxS),Moore B,Published; PMID:19786493,79.0,79.3,78.7,It was shown to be regulated by the presence of nitric oxide (NO). In Bacillus subtilis it controls expression of genes with functions related to oxidative stress and oxidation-reduction reactions and it was renamed RoxS (for related to oxidative stress).[2],ncr22; ncr629,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,117,0,0,Gene; sRNA;,Published; PMID:19786493,131,0,,\"\",0.48595,-8.88537,1600000,304409,214,112,true,-3.8642,0.71846,2013-10-04T02:50:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01821,RsaH,2199,RNA Staph. aureus H,Moore B,Published; PMID:19786493,50.0,65.7,39.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,14,0,0,Gene; sRNA;,Published; PMID:19786493,16,0,,\"\",0.49292,-7.61446,1600000,280348,232,123,true,-3.2038,0.71836,2013-10-04T02:50:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01822,RsaJ,2199,RNA Staph. aureus A,Moore B,Published; PMID:19786493,100.0,287.1,89.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Published; PMID:19786493,4,0,,\"\",0.62991,-8.67128,1600000,388496,439,287,true,-3.7402,0.70646,2013-10-04T02:50:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01823,rspL,2179,rpsL sRNA,Moore B,Published; PMID:19531245,37.0,37.5,36.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,14,0,0,Gene; sRNA;,Published; PMID:19531245,16,0,,\"\",0.57521,-3.58596,1600000,230130,181,88,true,-3.5378,0.7187,2013-10-04T02:50:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01824,RUF20,2200,RNA of unknown function 20,Moore B,Predicted;,60.0,134.7,27.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,9,0,0,Gene;,Predicted; WAR,15,0,,\"\",0.65263,-8.2927,1600000,364642,677,425,true,-3.8636,0.70211,2013-10-04T02:50:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01825,RUF21,2200,RNA of unknown function 21,Moore B,Predicted;,60.0,139.1,50.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,9,0,0,Gene;,Predicted; WAR,13,0,,\"\",0.48145,-11.53101,1600000,381232,1087,690,true,-4.594,0.69866,2013-10-04T02:50:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01826,SAM_V,2201,SAM-V riboswitch,\"Moore B, Ontiveros-Palacios N\",Published; PMID:19531245,41.0,46.2,39.8,\"SAM-V riboswitch is a cis-regulatory element that was recognised in marine bacteria [1,2] it recognise S-adenosylmethione (SAM). The binding of SAM leads to the association of the triplex third strand in the aptamer of SAM-V riboswitch that stabilizes a short helix and occludes the ribosome binding site. SAM-V riboswitch structure was reported in PDB:6FZ0 [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,12,0,0,Cis-reg; riboswitch;,Published; PMID:19531245,12,0,,\"\",0.495,-7.72177,1600000,322256,148,66,true,-3.577,0.71897,2013-10-04T02:51:02.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01827,SAR11_0636,2179,SAR11_0636 sRNA,Moore B,Published; PMID:19531245,67.0,68.1,66.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,8,0,0,Gene; sRNA;,Published; PMID:19531245,15,0,,\"\",0.54701,-3.38605,1600000,199251,185,90,true,-3.3145,0.71868,2013-10-04T02:51:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01828,SprD,2202,Small pathogenicity island RNA D,Moore B,Published; PMID:20532214,100.0,160.0,42.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,8,0,0,Gene; sRNA;,Published; PMID:20532214,10,0,,\"\",0.57173,-8.91268,1600000,388451,260,142,true,-3.2278,0.71825,2013-10-04T02:51:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01829,sR6,1264,sR6 snoRNA,Moore B,Published; PMID:10775111,60.0,89.2,35.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Mfold; Moore B,2,0,,\"\",0.47971,-9.00598,1600000,551609,137,53,true,-2.9357,0.72863,2013-10-04T02:51:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01830,StyR-44,1236,Salmonella enterica Typhi npcRNA 44,Moore B,Published; PMID:20460466,54.0,54.0,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.57 -Z 549862.597050 CM SEQDB,34,421,0,0,Gene;,Predicted; CMfinder,124,0,,\"\",0.46968,-7.97956,1600000,286424,204,107,true,-3.1309,0.71838,2013-10-04T02:51:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01831,THF,2203,THF riboswitch,\"Weinberg Z, Ames T, Ontiveros-Palacios N\",Published; PMID:20659680,36.0,36.7,34.8,\"Tetrahydrofolate (THF) riboswitch was first reported for Ames et al. [1] as an RNA motif in 5' untranslated (5'-UTR)regions of folate transport and biosynthesis genes in Firmicutes genomes. The structure of THF riboswitch has been reported in [2,3,4], it presents an inverted three-way junctional architecture. THF riboswitch presents a pseudoknot that joins P1-P2 and the loop of P3 and two THF binding. Tetrahydrofolate riboswitch structure is reported in PDB: 4LVV, 4LVW, 4LVX, 4LVY, 4LVZ, 4LW0 [4], 3SD3 [3], 3SUH, 3SUX, 3SUY [2], 6Q57 and 7KD1.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,101,595,0,0,Cis-reg; riboswitch;,Published; PMID:20659680,554,0,,\"\",0.66579,-2.95694,1600000,230244,245,99,true,-3.6001,0.71866,2013-10-04T02:51:16.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01832,ROSE_2,1694,Repression of heat shock gene expression (ROSE) element,\"Bateman A, Moore B\",Published; PMID:11726689,65.0,65.0,64.4,\"This family represents a 5' UTR cis-regulatory element found in heat shock mRNAs and is known as the ROSE (repression of heat shock gene expression) element. The ROSE element is a negative regulator of heat shock gene expression. The secondary structure is thought to be regulated by temperature. This regulation blocks access to the ribosome binding site under normal conditions. During heat shock however, the structure changes freeing the ribosome binding site and allowing expression to occur [1,2]. This family includes predicted members of the family from gamma-proteobacteria.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,28,0,0,Cis-reg; thermoregulator;,Published; PMID:16438677; Bateman A,42,0,,\"\",0.63404,-4.50046,1600000,222684,158,73,true,-3.7619,0.71895,2013-10-04T02:51:18.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01833,astro_FSE,2204,ribosomal frameshift site,Boursnell C,Published; PMID:8248142,70.0,76.3,44.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Cis-reg; frameshift_element;,Published; PMID:8248142,4,0,,\"\",0.67768,-7.01888,1600000,362744,135,55,true,-3.8401,0.71922,2013-10-04T02:51:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01834,fiv_FSE,2205,ribosomal frameshift site,Boursnell C,Published; PMID:1310175,53.0,55.8,38.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,0,0,0,Cis-reg; frameshift_element;,Published; PMID:1310175,1,0,,\"\",0.57774,-10.1497,1600000,899341,104,35,true,-3.3697,0.72451,2013-10-04T02:51:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01835,HIV_FS2,2206,ribosomal frameshift site,Boursnell C,Published; PMID:1575083,44.0,47.2,43.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,3,0,0,Cis-reg; frameshift_element;,Published; PMID:1575083,3,0,,\"\",0.66205,-3.59412,1600000,299159,115,45,true,-3.8975,0.71968,2013-10-04T02:51:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01836,weev_FSE,2207,ribosomal frameshift site,Boursnell C,Published; PMID:19895695,40.0,40.6,39.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,3,0,0,Cis-reg; frameshift_element;,Published; PMID:19895695,6,0,,\"\",0.52696,-6.67043,1600000,731982,92,27,true,-3.2323,0.73647,2013-10-04T02:51:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01837,toga_FSE,2208,togavirus ribosomal frameshift element,Boursnell C,Published; PMID:18822126,40.0,50.0,37.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,0,0,0,Cis-reg; frameshift_element;,Published; PMID:18822126,2,0,,\"\",0.62528,-4.61666,1600000,434192,100,29,true,-2.9141,0.74324,2013-10-04T02:51:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01838,sobemo_FSE,2209,sobemovirus ribosomal frameshift elemental,Boursnell C,Published; PMID:10848968,55.0,57.4,35.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,4,0,0,Cis-reg; frameshift_element;,Published; PMID:10848968,5,0,,\"\",0.56167,-7.01963,1600000,197854,164,79,true,-3.7923,0.71878,2013-10-04T02:51:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01839,eeev_FSE,2210,eastern equine encephalitis ribosomal frameshift element,Boursnell C,Published; PMID:18822126,80.0,87.5,54.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,0,0,0,Cis-reg; frameshift_element;,Published; PMID:18822126,1,0,,\"\",0.55615,-7.66895,1600000,316665,162,73,true,-3.4691,0.71886,2013-10-04T02:51:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01840,ovine_lenti_FSE,2211,ribosomal frameshift element,Boursnell C,Published; PMID:15629717,45.0,51.1,38.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,1,0,0,Cis-reg; frameshift_element;,Published; PMID:15629717,4,0,,\"\",0.55334,-9.27798,1600000,586622,132,54,true,-3.7593,0.71925,2013-10-04T02:51:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01841,veev_FSE,2212,venezuelan equine encephalitis virus ribosomal frameshift element,Boursnell C,Published; PMID:18822126,55.0,78.0,52.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,0,0,0,Cis-reg; frameshift_element;,Published; PMID:18822126,1,0,,\"\",0.56242,-6.8422,1600000,308525,158,71,true,-3.4851,0.71892,2013-10-04T02:51:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01842,mycoplasma_FSE,2213,mycoplasma ribosomal frameshift element,Boursnell C,Published; PMID:7476162,88.0,88.5,46.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,24,0,0,Cis-reg; frameshift_element;,Published; PMID:7476162,4,0,,\"\",0.44736,-4.75166,1600000,310123,183,87,true,-3.0652,0.71855,2013-10-04T02:51:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01843,neisseria_FSE,2214,neiserria ribosomal frameshift element,Boursnell C,Published; PMID:11943474,50.0,57.6,34.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,2,0,0,Cis-reg; frameshift_element;,Published; PMID:11943474,6,0,,\"\",0.69108,-3.79442,1600000,391899,113,40,true,-3.595,0.72378,2013-10-04T02:51:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01844,SmY,2215,SmY spliceosomal RNA,\"Jones T, Moore B, Daub J, Marz M\",Published; PMID:19106623,39.0,39.8,37.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,61,261,0,0,Gene; snRNA; splicing;,Published; PMID:19106623,42,0,,\"\",0.5843,-8.04167,1600000,256117,149,80,true,-4.2033,0.71873,2013-10-04T02:51:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01845,enod40,2216,Early nodulin 40,Gardner PP,Published; PMID:17452360,45.0,50.0,44.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,35,125,0,0,Gene;,Published; PMID:17452360,72,0,,\"\",0.52897,-11.54489,1600000,453993,500,269,true,-3.4795,0.70727,2013-10-04T02:51:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01846,Fungi_U3,1312,Fungal small nucleolar RNA U3,\"Gardner PP, Marz M\",Published; PMID:19875933,95.0,96.1,91.8,\"Small nucleolar RNAs (snoRNAs) are involved in the processing and modification of rRNA in the nucleolus. There are two main classes of snoRNAs: the box C/D class, and the box H/ACA class. U3 snoRNA is a member of the box C/D class. Indeed, the box C/D element is a subset of the six short sequence elements found in all U3 snoRNAs, namely boxes A, A', B, C, C', and D. The U3 snoRNA secondary structure is characterised by a small 5' domain (with boxes A and A'), and a larger 3' domain (with boxes B, C, C', and D), the two domains being linked by a single-stranded hinge. Boxes B and C form the B/C motif, which appears to be exclusive to U3 snoRNAs, and boxes C' and D form the C'/D motif. The latter is functionally similar to the C/D motifs found in other snoRNAs. The 5' domain and the hinge region act as a pre-rRNA-binding domain. The 3' domain has conserved protein-binding sites. Both the box B/C and box C'/D motifs are sufficient for nuclear retention of U3 snoRNA. The box C'/D motif is also necessary for nucleolar localisation, stability and hyper-methylation of U3 snoRNA. Both box B/C and C'/D motifs are involved in specific protein interactions and are necessary for the rRNA processing functions of U3 snoRNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.23 -Z 549862.597050 CM SEQDB,66,577,0,0,Gene; snRNA; snoRNA; CD-box;,Published; PMID:19875933,360,0,,\"\",0.64339,-9.13412,1600000,467174,724,276,true,-4.3456,0.70709,2013-10-04T02:51:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01847,Plant_U3,1312,Plant small nucleolar RNA U3,Gardner PP,Published; PMID:19875933; Gardner PP,80.0,80.3,79.8,\"Small nucleolar RNAs (snoRNAs) are involved in the processing and modification of rRNA in the nucleolus. There are two main classes of snoRNAs: the box C/D class, and the box H/ACA class. U3 snoRNA is a member of the box C/D class. Indeed, the box C/D element is a subset of the six short sequence elements found in all U3 snoRNAs, namely boxes A, A', B, C, C', and D. The U3 snoRNA secondary structure is characterised by a small 5' domain (with boxes A and A'), and a larger 3' domain (with boxes B, C, C', and D), the two domains being linked by a single-stranded hinge. Boxes B and C form the B/C motif, which appears to be exclusive to U3 snoRNAs, and boxes C' and D form the C'/D motif. The latter is functionally similar to the C/D motifs found in other snoRNAs. The 5' domain and the hinge region act as a pre-rRNA-binding domain. The 3' domain has conserved protein-binding sites. Both the box B/C and box C'/D motifs are sufficient for nuclear retention of U3 snoRNA. The box C'/D motif is also necessary for nucleolar localisation, stability and hyper-methylation of U3 snoRNA. Both box B/C and C'/D motifs are involved in specific protein interactions and are necessary for the rRNA processing functions of U3 snoRNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.66 -Z 549862.597050 CM SEQDB,24,1185,0,0,Gene; snRNA; snoRNA; CD-box;,Published; PMID:19875933; Gardner PP,92,0,,\"\",0.69853,-8.2173,1600000,415983,334,221,true,-3.5267,0.71053,2013-10-04T02:51:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01848,ACEA_U3,1312,ACEA small nucleolar RNA U3,Gardner PP,Published; PMID:19875933,55.0,55.0,54.9,\"Small nucleolar RNAs (snoRNAs) are involved in the processing and modification of rRNA in the nucleolus. There are two main classes of snoRNAs: the box C/D class, and the box H/ACA class. U3 snoRNA is a member of the box C/D class. Indeed, the box C/D element is a subset of the six short sequence elements found in all U3 snoRNAs, namely boxes A, A', B, C, C', and D. The U3 snoRNA secondary structure is characterised by a small 5' domain (with boxes A and A'), and a larger 3' domain (with boxes B, C, C', and D), the two domains being linked by a single-stranded hinge. Boxes B and C form the B/C motif, which appears to be exclusive to U3 snoRNAs, and boxes C' and D form the C'/D motif. The latter is functionally similar to the C/D motifs found in other snoRNAs. The 5' domain and the hinge region act as a pre-rRNA-binding domain. The 3' domain has conserved protein-binding sites. Both the box B/C and box C'/D motifs are sufficient for nuclear retention of U3 snoRNA. The box C'/D motif is also necessary for nucleolar localisation, stability and hyper-methylation of U3 snoRNA. Both box B/C and C'/D motifs are involved in specific protein interactions and are necessary for the rRNA processing functions of U3 snoRNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,28,123,0,0,Gene; snRNA; snoRNA; CD-box;,Published; PMID:19875933,94,0,,\"\",0.55628,-9.19641,1600000,345800,496,220,true,-3.7669,0.71055,2013-10-04T02:51:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01849,alpha_tmRNA,1321,Alphaproteobacteria transfer-messenger RNA,Gardner PP,Published; PMID:19617710,50.0,50.0,49.8,\"Bacterial tmRNA, also known as 10Sa RNA or SsrA, is named for its dual tRNA and mRNA-like properties. Its role is to liberate the mRNA from a stalled ribosome. To accomplish this part of the tmRNA is used as a reading frame that ends in a translation stop signal. The broken mRNA is replaced in the ribosome by the tmRNA and translation of the tmRNA leads to addition of a proteolysis tag to the incomplete protein enabling recognition by a protease. Recently a number of permuted tmRNA genes have been found encoded in two parts. tmRNAs have been identified in eubacteria and some chloroplasts but are absent from archaeal and eukaryote nuclear genomes.\",10Sa_RNA; SsrA; tmRNA,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 16.48 -Z 549862.597050 CM SEQDB,111,831,0,0,Gene;,Published; PMID:19617710,890,0,,\"\",0.68481,-11.25656,1600000,516306,2019,338,true,-3.9626,0.70444,2013-10-04T02:51:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01850,beta_tmRNA,1321,Betaproteobacteria transfer-messenger RNA,Gardner PP,Published; PMID:19617710,84.0,84.9,83.1,\"Bacterial tmRNA, also known as 10Sa RNA or SsrA, is named for its dual tRNA and mRNA-like properties. Its role is to liberate the mRNA from a stalled ribosome. To accomplish this part of the tmRNA is used as a reading frame that ends in a translation stop signal. The broken mRNA is replaced in the ribosome by the tmRNA and translation of the tmRNA leads to addition of a proteolysis tag to the incomplete protein enabling recognition by a protease. Recently a number of permuted tmRNA genes have been found encoded in two parts. tmRNAs have been identified in eubacteria and some chloroplasts but are absent from archaeal and eukaryote nuclear genomes.\",10Sa_RNA; SsrA; tmRNA,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,34,0,0,Gene;,Published; PMID:19617710,39,0,,\"\",0.77142,-8.64831,1600000,424064,673,331,true,-4.3475,0.70468,2013-10-04T02:51:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01851,cyano_tmRNA,1321,Cyanobacteria transfer-messenger RNA,Gardner PP,Published; PMID:19617710,70.0,70.0,67.5,\"Bacterial tmRNA, also known as 10Sa RNA or SsrA, is named for its dual tRNA and mRNA-like properties. Its role is to liberate the mRNA from a stalled ribosome. To accomplish this part of the tmRNA is used as a reading frame that ends in a translation stop signal. The broken mRNA is replaced in the ribosome by the tmRNA and translation of the tmRNA leads to addition of a proteolysis tag to the incomplete protein enabling recognition by a protease. Recently a number of permuted tmRNA genes have been found encoded in two parts. tmRNAs have been identified in eubacteria and some chloroplasts but are absent from archaeal and eukaryote nuclear genomes.\",10Sa_RNA; SsrA; tmRNA,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,27,21,0,0,Gene;,Published; PMID:19617710,42,0,,\"\",0.703,-8.8492,1600000,402190,491,288,true,-4.2205,0.7065,2013-10-04T02:52:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01852,tRNA-Sec,1306,Selenocysteine transfer RNA,Gardner PP,Gardner PP,47.0,47.0,46.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.39 -Z 549862.597050 CM SEQDB,109,2250,0,0,Gene; tRNA;,Published; PMID:10682860,2039,0,,\"\",0.59769,-5.29381,1600000,292574,164,91,true,-3.8558,0.71862,2013-10-04T02:52:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01853,mtDNA_ssA,2454,Mitochondrial DNA control region secondary structure A,\"Gardner PP, Moore B\",Gardner PP;,40.0,40.0,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,53,195,0,0,Gene; antisense;,Published; PMID:18845547,39,0,,\"\",0.49628,-4.51282,1600000,256561,192,106,true,-3.1728,0.71857,2013-10-04T02:52:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01854,Bacteria_large_SRP,1315,Bacterial large signal recognition particle RNA,Gardner PP,Published; PMID:19838050,60.0,60.0,59.9,\"The signal recognition particle (SRP) is a universally conserved ribonucleoprotein involved in the co-translational targeting of proteins to membranes. The eukaryotic SRP consists of a 300-nucleotide 7S RNA (RFAM:RF00017) and six proteins: SRPs 72, 68, 54, 19, 14, and 9. Archaeal SRP consists of a 7S RNA and homologues of the eukaryotic SRP19 and SRP54 proteins. In most bacteria, the SRP consists of a 4.5S RNA and the Ffh protein (a homologue of the eukaryotic SRP54 protein). Some Gram-positive bacteria (e.g. Bacillus subtilis) have a longer eukaryote-like SRP RNA that includes an Alu domain [2]. The Alu domain of the longer SRP RNAs is not included in this model and therefore the reported bounds of some members are incorrect. The Signal Recognition Particle Database (SRPDB) [1] provides compilations of SRP components, with phylogenetic data and structural illustrations.\",SRP_bact_l; SRP_bact,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 549862.597050 CM SEQDB,92,1051,0,0,Gene;,Published; PMID:19838050,1089,0,,\"\",0.4595,-10.0278,1600000,257958,693,255,true,-3.7382,0.70833,2013-10-04T02:52:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01855,Plant_SRP,1315,Plant signal recognition particle RNA,Gardner PP,Published; PMID:19838050,105.0,105.1,104.9,,SRP_euk_arch,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,63,1335,0,0,Gene;,Published; PMID:19838050,100,0,,\"\",0.69593,-9.36073,1600000,526674,434,305,true,-3.9574,0.70581,2013-10-04T02:52:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01856,Protozoa_SRP,1315,Protozoan signal recognition particle RNA,Gardner PP,Published; PMID:19838050,65.0,65.0,64.9,,SRP_euk_arch; 7SL,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.96 -Z 549862.597050 CM SEQDB,18,68,0,0,Gene;,Published; PMID:19838050,66,0,,\"\",0.55362,-8.71569,1600000,362783,452,241,true,-3.9986,0.70914,2013-10-04T02:53:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01857,Archaea_SRP,1315,Archaeal signal recognition particle RNA,Gardner PP,Published; PMID:19838050,89.0,89.0,88.9,,SRP_euk_arch,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,53,291,0,0,Gene;,Published; PMID:19838050,334,0,,\"\",0.60192,-10.82988,1600000,560582,435,318,true,-4.0529,0.70531,2013-10-04T02:53:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01858,RsaF,2199,RNA Staph. aureus F,Moore B,Published; PMID:19786493,60.0,65.2,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; sRNA;,Published; PMID:19786493,5,0,,\"\",0.59362,-5.27547,1600000,288846,228,119,true,-2.7014,0.7183,2013-10-04T02:53:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01859,Phe_leader,2218,Phenylalanine leader peptide,Gardner PP,Predicted; CMfinder,55.0,55.0,54.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.86 -Z 549862.597050 CM SEQDB,71,296,0,0,Cis-reg; leader;,Predicted; CMfinder,232,0,,\"\",0.51068,-6.82533,1600000,238396,244,125,true,-3.1906,0.71855,2013-10-04T02:53:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01860,Afu_455,1264,A. fumigatus snoRNA Afu_455,Osuch I,INFERNAL,48.0,48.5,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,273,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,268,0,,\"\",0.48124,-9.45651,1600000,261929,182,92,false,-3.6274,0.7184,2013-10-04T02:53:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01861,TB9Cs1H1,2411,Trypanosomatid snoRNA TB9Cs1H1,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,55.0,59.1,43.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.00 -Z 549862.597050 CM SEQDB,4,67,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,14,0,,\"\",0.60837,-5.72226,1600000,368575,144,63,true,-3.5843,0.72052,2013-10-04T02:53:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01862,TB10Cs4H2,2411,Trypanosomatid snoRNA TB10Cs4H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,55.1,35.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.49 -Z 549862.597050 CM SEQDB,4,32,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,11,0,,\"\",0.58994,-5.46416,1600000,247470,149,67,true,-3.3085,0.72129,2013-10-04T02:53:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01863,TB10Cs2H2,2411,Trypanosomatid snoRNA TB10Cs2H2,\"Doniger T, Michaeli S, Unger R, Osuch I\",Predicted; WAR;,50.0,66.7,48.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR;,2,0,,\"\",0.63155,-4.27875,1600000,248617,155,67,true,-3.3878,0.71904,2013-10-04T02:53:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01864,plasmodium_snoR21,1264,small nucleolar RNA snoR21,Osuch I,INFERNAL,90.0,126.5,70.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,24,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,26,0,,\"\",0.44309,-7.20927,1600000,437832,261,145,true,-2.8071,0.71804,2013-10-04T02:53:47.000Z,2022-10-29T17:55:16.000Z\n",
            "RF01865,Vg1_ribozyme,2219,Manganese dependent ribozyme in Vg1 mRNA,Osuch I,INFERNAL,90.0,108.4,39.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; ribozyme;,Published; PMID:18753150,3,0,,\"\",0.44358,-8.63319,1600000,331896,252,142,true,-3.2221,0.71814,2013-10-04T02:53:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01866,ceN33,1264,C. elegans snoRNA ceN33,Osuch I,INFERNAL,75.0,89.3,41.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.72 -Z 549862.597050 CM SEQDB,5,7,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; Osuch I,7,0,,\"\",0.64402,-4.64627,1600000,249964,224,118,true,-3.2565,0.71809,2013-10-04T02:53:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01867,CC2171,2085,caulobacter sRNA CC2171,Osuch I,INFERNAL,75.0,75.2,74.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,55,0,0,Gene; sRNA;,Predicted;RNAalifold;,67,0,,\"\",0.56524,-10.25774,1600000,381547,387,159,true,-3.2128,0.71738,2013-10-04T02:53:52.000Z,2022-10-29T17:55:17.000Z\n",
            "RF01871,MALAT1,2222,Metastasis associated lung adenocarcinoma transcript 1,Eberhardt R,Eberhardt R,70.0,72.4,68.9,\"MALAT 1 (metastasis associated lung adenocarcinoma transcript 1) is also known as NEAT2 (noncoding nuclear-enriched abundant transcript 2). It is a large, infrequently spliced RNA, which is highly conserved amongst mammals. It is highly expressed in the nucleus [1]. It is associated with metastasis, and positively regulates cell motility via the transcriptional and/or post-transcriptional regulation of motility-related genes [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 38.00 -Z 549862.597050 CM SEQDB,17,39,0,0,Gene; lncRNA;,Predicted; RNAz; Eberhardt R,41,0,,\"\",0.41259,-9.4686,1600000,286978,193,95,false,-3.0921,0.71836,2013-10-04T02:53:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01872,MEG3_2,2221,Maternally expressed 3 conserved region 2,Eberhardt R,Eberhardt R,60.0,63.7,53.6,\"MEG3 (maternally expressed 3) is a maternally expressed, imprinted noncoding RNA gene. At least 12 different isoforms of MEG3 are generated by alternative splicing [1]. Expression of MEG3 is lost in cancer cells [1,2]. It acts as a growth suppressor in tumour cells, and activates p53 [2,3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.82 -Z 549862.597050 CM SEQDB,16,37,0,0,Gene; lncRNA;,Predicted; RNAz; Eberhardt R,40,0,,\"\",0.50998,-10.29545,1600000,286395,202,105,false,-3.1251,0.71821,2013-10-04T02:54:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01873,PISRT1,2223,Polled intersex syndrome regulated transcript 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,70.0,70.6,67.3,\"PISRT1 (polled intersex syndrome regulated transcript 1) is a noncoding RNA. It has been implicated in the polled intersex (PIS) mutation in goats [1]. It has been shown not to be involved in sex determination in the rodent Ellobius lutescens [2] and in dogs [3], and is unlikely to be involved in sex-specific gonad differentiation in mice [4].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.00 -Z 549862.597050 CM SEQDB,18,38,0,0,Gene; lncRNA;,Predicted; CMfinder;,42,0,,\"\",0.43061,-8.79476,1600000,303849,204,107,false,-2.8939,0.71819,2013-10-04T02:54:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01874,MIAT_exon1,2224,Myocardial infarction associated transcript exon 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,110.0,117.8,61.8,\"MIAT (myocardial infarction associated transcript), also known as RNCR2 (retinal non-coding RNA 2) or Gomafu, is a noncoding RNA. Single nucleotide polymorphisms (SNPs) in MIAT are associated with a risk of myocardial infarction [1]. It is expressed in neurons, and located in the nucleus [2]. It plays a role in the regulation of retinal cell fate specification [3].\",RNCR2; Gomafu,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 45.83 -Z 549862.597050 CM SEQDB,10,33,0,0,Gene; lncRNA;,Predicted; CMfinder;,36,0,,\"\",0.28162,-21.75017,1600000,535705,244,135,false,-3.1485,0.71785,2013-10-04T02:54:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01875,MIAT_exon5_1,2224,Myocardial infarction associated transcript exon 5 conserved region 1,\"Eberhardt R, Burge S\",Predicted; WAR; Eberhardt R,100.0,131.2,67.0,\"MIAT (myocardial infarction associated transcript), also known as RNCR2 (retinal non-coding RNA 2) or Gomafu, is a noncoding RNA. Single nucleotide polymorphisms (SNPs) in MIAT are associated with a risk of myocardial infarction [1]. It is expressed in neurons, and located in the nucleus [2]. It plays a role in the regulation of retinal cell fate specification [3].\",RNCR2; Gomafu; MIAT,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.64 -Z 549862.597050 CM SEQDB,9,36,0,0,Gene; lncRNA;,Predicted; WAR;,40,0,,\"\",0.4089,-13.51438,1600000,304001,261,146,false,-2.9322,0.71773,2013-10-04T02:54:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01876,MIAT_exon5_2,2224,Myocardial infarction associated transcript exon 5 conserved region 2,Eberhardt R,Predicted; WAR; Eberhardt R,75.0,81.2,48.7,\"MIAT (myocardial infarction associated transcript), also known as RNCR2 (retinal non-coding RNA 2) or Gomafu, is a noncoding RNA. Single nucleotide polymorphisms (SNPs) in MIAT are associated with a risk of myocardial infarction [1]. It is expressed in neurons, and located in the nucleus [2]. It plays a role in the regulation of retinal cell fate specification [3].\",RNCR2; Gomafu,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.44 -Z 549862.597050 CM SEQDB,10,37,0,0,Gene; lncRNA;,Predicted; WAR;,40,0,,\"\",0.588,-9.41498,1600000,396366,171,83,false,-3.3256,0.71847,2013-10-04T02:54:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01877,MIAT_exon5_3,2224,Myocardial infarction associated transcript exon 5 conserved region 3,Eberhardt R,Predicted; WAR; Eberhardt R,90.0,93.3,62.5,\"MIAT (myocardial infarction associated transcript), also known as RNCR2 (retinal non-coding RNA 2) or Gomafu, is a noncoding RNA. Single nucleotide polymorphisms (SNPs) in MIAT are associated with a risk of myocardial infarction [1]. It is expressed in neurons, and located in the nucleus [2]. It plays a role in the regulation of retinal cell fate specification [3].\",RNCR2; Gomafu,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 16.26 -Z 549862.597050 CM SEQDB,16,37,0,0,Gene; lncRNA;,Predicted; WAR;,42,0,,\"\",0.63421,-14.55412,1600000,895293,536,382,false,-4.0318,0.70305,2013-10-04T02:54:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01879,TUSC7,1276,Non-coding RNA LOC285194,Eberhardt R,Eberhardt R,65.0,91.5,47.2,LOC285194 is a non-coding RNA which is often deleted in osteosarcoma. Deletion of this gene is associated with poor survival of osteosarcoma patients. It may act as a growth suppressor in osteoblasts [1].,LOC285194;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.49 -Z 742849.287494 --mxsize 128 CM SEQDB,25,37,0,0,Gene; lncRNA;,Predicted; RNAz; Eberhardt R,47,0,,\"\",0.39335,-14.35634,1600000,338639,311,165,false,-3.0607,0.71609,2013-10-04T02:54:13.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01880,Xist_exon1,2469,X-chromosome inactivation gene exon 1,\"Gardner PP, Eberhardt R\",Gardner PP,50.0,56.1,39.7,\"Xist is an RNA gene on the X chromosome of the placental mammals that acts as major effector of the X inactivation process. The Xist RNA, a large (17 kb in humans) [1] transcript, is expressed on the inactive chromosome and not on the active one. It is processed similarly to mRNAs, through splicing and polyadenylation, however, it remains untranslated. It has been suggested that this RNA gene evolved at least partly from a protein coding gene that became a pseudogene [2]. The inactive X is coated with this transcript, which is essential for the inactivation. X lacking Xist will not be inactivated, while duplication of the Xist gene on another chromosome causes inactivation of that chromosome. This Xist exon was identified in a screen for conserved and structured elements in the human genome [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.72 -Z 549862.597050 CM SEQDB,24,31,0,0,Gene; lncRNA;,Predicted; RNAz; Gardner PP,44,0,,\"\",0.72951,-6.85076,1600000,368137,166,85,false,-3.6235,0.71849,2013-10-04T02:54:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01881,Xist_exon4,2469,X-chromosome inactivation gene exon 4,\"Gardner PP, Eberhardt R\",Gardner PP,100.0,107.6,47.4,\"Xist is an RNA gene on the X chromosome of the placental mammals that acts as major effector of the X inactivation process. The Xist RNA, a large (17 kb in humans) [1] transcript, is expressed on the inactive chromosome and not on the active one. It is processed similarly to mRNAs, through splicing and polyadenylation, however, it remains untranslated. It has been suggested that this RNA gene evolved at least partly from a protein coding gene that became a pseudogene [2]. The inactive X is coated with this transcript, which is essential for the inactivation. X lacking Xist will not be inactivated, while duplication of the Xist gene on another chromosome causes inactivation of that chromosome. This Xist exon was identified in a screen for conserved and structured elements in the human genome [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.43 -Z 549862.597050 CM SEQDB,25,35,0,0,Gene; lncRNA;,Predicted; RNAz; Gardner PP,45,0,,\"\",0.54281,-10.54189,1600000,296089,231,126,false,-3.241,0.71803,2013-10-04T02:54:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01882,TUG1_1,2227,Taurine upregulated gene 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,84.2,68.7,TUG1 (taurine upregulated gene 1) is a long non-coding RNA expressed in retina and brain. It was first identified in a screen for genes upregulated by in developing retinal cells in response to taurine. It is required for the normal development of photoreceptors in the retina [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.27 -Z 549862.597050 CM SEQDB,21,45,0,0,Gene; lncRNA;,Predicted; CMfinder;,45,0,,\"\",0.40181,-10.29173,1600000,259927,266,145,false,-3.0824,0.71782,2013-10-04T02:54:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01883,TUG1_2,2227,Taurine upregulated gene 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,90.0,91.0,81.3,TUG1 (taurine upregulated gene 1) is a long non-coding RNA expressed in retina and brain. It was first identified in a screen for genes upregulated by in developing retinal cells in response to taurine. It is required for the normal development of photoreceptors in the retina [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.61 -Z 549862.597050 CM SEQDB,17,37,0,0,Gene; lncRNA;,Predicted; CMfinder;,34,0,,\"\",0.45969,-10.27846,1600000,335473,180,89,false,-3.6536,0.71844,2013-10-04T02:54:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01884,LINC00901,1276,Non-coding RNA BC040587,Eberhardt R,Eberhardt R,55.0,55.1,54.9,BC040587 is a long non-coding RNA which is often deleted in osteosarcoma. Deletion of this gene is associated with poor survival of osteosarcoma patients [1].,BC040587;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.91 -Z 549862.597050 --mxsize 128 CM SEQDB,30,179,0,0,Gene; lncRNA;,Predicted; RNAz; Eberhardt R,103,0,,\"\",0.4893,-12.0675,1600000,374423,282,152,false,-3.1045,0.71776,2013-02-01T11:59:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01885,HSR-omega_1,2228,Heat shock gene hsromega conserved region 1,Eberhardt R,Predicted; WAR; Eberhardt R,80.0,92.2,33.7,\"The heat shock gene hsromega is a long non-coding RNA which produces a number of transcripts. These transcripts include a nuclear transcript, hsromega-n, which is detected as small speckles in the interchromatin space and co-localises with heterogeneous nuclear RNA-binding proteins (hnRNPs) [1]. Hsromega-n interacts directly and indirectly with a number of regulatory proteins including hnRNPs, cAMP response element binding binding protein (CBP), proteasome, and Drosophila inhibitor of apoptosis protein 1 (DIAP1). Through these interactions, it has roles in the regulation of apoptosis [2] and the suppression of poly(Q) toxicity [3].\",HSR-omega_1; ; DLX6-AS1_2,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.27 -Z 549862.597050 CM SEQDB,13,11,0,0,Gene; lncRNA;,Predicted; WAR;,14,0,,\"\",0.46204,-9.6113,1600000,264737,166,78,false,-3.5454,0.71859,2013-10-04T02:54:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01886,HSR-omega_2,2228,Heat shock gene hsromega conserved region 2,Eberhardt R,Predicted; WAR; Eberhardt R,55.0,56.9,32.1,\"The heat shock gene hsromega is a long non-coding RNA which produces a number of transcripts. These transcripts include a nuclear transcript, hsromega-n, which is detected as small speckles in the interchromatin space and co-localises with heterogeneous nuclear RNA-binding proteins (hnRNPs) [1]. Hsromega-n interacts directly and indirectly with a number of regulatory proteins including hnRNPs, cAMP response element binding binding protein (CBP), proteasome, and Drosophila inhibitor of apoptosis protein 1 (DIAP1). Through these interactions, it has roles in the regulation of apoptosis [2] and the suppression of poly(Q) toxicity [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.15 -Z 549862.597050 CM SEQDB,10,12,0,0,Gene; lncRNA;,Predicted; WAR;,15,0,,\"\",0.54101,-8.96075,1600000,240401,177,87,false,-3.9176,0.71843,2013-10-04T02:54:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01887,DLX6-AS1_1,2497,Embryonic ventral forebrain RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,75.5,45.1,\"Evf-1 (Embryonic ventral forebrain-1) is a developmentally-regulated long non-coding RNA. In rats, it is expressed in neurons in the subventricular zone of the developing forebrain. Its expression is linked to that of the Shh (sonic hedgehog) and Dlx families of genes, which are important in ventral forebrain and craniofacial development [1]. An alternatively spliced form of Evf-1, Evf-2, forms a stable complex with the Dlx-2 protein. This complex activates transcription of Dlx-5 and Dlx-6 [2].\",Evf1_1;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.06 -Z 549862.597050 --mxsize 128 CM SEQDB,15,36,0,0,Gene; lncRNA;,Predicted; CMfinder;,41,0,,\"\",0.35862,-12.04956,1600000,380393,301,180,false,-3.3854,0.71418,2013-10-04T02:54:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01888,DLX6-AS1_2,2497,Embryonic ventral forebrain RNA 1 conserved region 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,150.0,152.6,148.9,\"Evf-1 (Embryonic ventral forebrain-1) is a developmentally-regulated long non-coding RNA. In rats, it is expressed in neurons in the subventricular zone of the developing forebrain. Its expression is linked to that of the Shh (sonic hedgehog) and Dlx families of genes, which are important in ventral forebrain and craniofacial development [1]. An alternatively spliced form of Evf-1, Evf-2, forms a stable complex with the Dlx-2 protein. This complex activates transcription of Dlx-5 and Dlx-6 [2].\",Evf1_2;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 44.87 -Z 549862.597050 --mxsize 128 CM SEQDB,19,36,0,0,Gene; lncRNA;,Predicted; Pfold;,43,0,,\"\",0.30761,-16.82632,1600000,508363,332,207,false,-3.3435,0.7115,2013-10-04T02:54:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01889,lincRNA-p21_1,1276,LincRNA-p21 conserved region 1,Eberhardt R,Predicted; WAR; Eberhardt R,75.0,79.6,33.4,LincRNA-p21 is an large intergenic non-coding RNA. Transcription of lincRNA-p21 is induced by p53. It interacts with heterogeneous nuclear riboprotein K (hnRNP-K) and has an important role in the p53-dependent apoptotic responses to DNA damage [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.61 -Z 549862.597050 CM SEQDB,2,6,0,0,Gene; lncRNA;,Predicted; WAR;,5,0,,\"\",0.45569,-12.93998,1600000,307576,296,174,false,-3.3421,0.71495,2013-10-04T02:54:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01890,lincRNA-p21_2,1276,LincRNA-p21 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,70.0,73.9,65.8,LincRNA-p21 is an large intergenic non-coding RNA. Transcription of lincRNA-p21 is induced by p53. It interacts with heterogeneous nuclear riboprotein K (hnRNP-K) and has an important role in the p53-dependent apoptotic responses to DNA damage [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.30 -Z 549862.597050 CM SEQDB,2,3,0,0,Gene; lncRNA;,Predicted; CMfinder;,2,0,,\"\",0.52707,-9.18119,1600000,385167,133,52,false,-3.0668,0.7323,2013-10-04T02:54:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01891,TUG1_3,2227,Taurine upregulated gene 1 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,90.0,90.4,65.4,TUG1 (taurine upregulated gene 1) is a long non-coding RNA expressed in retina and brain. It was first identified in a screen for genes upregulated by in developing retinal cells in response to taurine. It is required for the normal development of photoreceptors in the retina [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.91 -Z 549862.597050 CM SEQDB,24,43,0,0,Gene; lncRNA;,Predicted; CMfinder;,43,0,,\"\",0.48557,-15.0206,1600000,472068,365,236,false,-3.4033,0.70923,2013-10-04T02:54:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01892,TUG1_4,2227,Taurine upregulated gene 1 conserved region 4,Eberhardt R,Predicted; CMfinder; Eberhardt R,100.0,118.4,76.6,TUG1 (taurine upregulated gene 1) is a long non-coding RNA expressed in retina and brain. It was first identified in a screen for genes upregulated by in developing retinal cells in response to taurine. It is required for the normal development of photoreceptors in the retina [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.75 -Z 549862.597050 CM SEQDB,25,44,0,0,Gene; lncRNA;,Predicted; CMfinder;,45,0,,\"\",0.38,-12.09175,1600000,228581,298,180,false,-3.2943,0.7142,2013-10-04T02:54:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01894,PCA3_2,2229,Prostate cancer antigen RNA 3 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,100.0,111.1,63.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.44 -Z 549862.597050 CM SEQDB,20,92,0,0,Gene; lncRNA;,Predicted; CMfinder;,99,0,,\"\",0.54085,-15.68454,1600000,517801,574,409,false,-4.2927,0.70238,2013-10-04T02:54:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01895,mir-193,2230,microRNA mir-193,Griffiths-Jones SR,Griffiths-Jones SR,55.0,55.4,54.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,54,187,0,0,Gene; miRNA;,Predicted; RNAalifold,131,0,,\"\",0.66073,-3.48763,1600000,162386,257,76,true,-3.9327,0.71885,2013-10-04T02:54:39.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01896,mir-142,1287,microRNA mir-142,Griffiths-Jones SR,Griffiths-Jones SR,57.0,57.0,54.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,37,246,0,0,Gene; miRNA;,Predicted; RNAalifold,109,0,,\"\",0.60701,-4.29524,1600000,174770,176,85,true,-3.1593,0.71876,2013-10-04T02:54:41.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01897,mir-188,2231,microRNA mir-188,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.7,62.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,16,34,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.63233,-3.11532,1600000,216937,179,84,true,-3.6808,0.71894,2013-10-04T02:54:43.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01898,mir-363,2232,microRNA mir-363,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.1,48.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,22,100,0,0,Gene; miRNA;,Predicted; RNAalifold,103,0,,\"\",0.66016,-3.96603,1600000,196640,171,80,true,-3.4026,0.7189,2013-10-04T02:54:45.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01899,mir-2241,1287,mir-2241 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,104.9,59.1,This family represents the microRNA (miRNA) precursor mir-2241 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,10,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66598,-2.64306,1600000,200276,186,89,true,-3.4613,0.71887,2013-10-04T02:54:47.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01900,mir-2024,1287,microRNA mir-2024,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.2,68.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,16,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61927,-3.76697,1600000,182193,166,77,true,-3.4761,0.71909,2013-10-04T02:54:48.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01901,mir-284,1287,mir-284 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,63.5,46.8,This family represents the microRNA (miRNA) precursor mir-284 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,21,0,0,Gene; miRNA;,Predicted; RNAalifold,24,0,,\"\",0.58426,-3.99802,1600000,194045,198,98,true,-3.4769,0.71867,2013-10-04T02:54:50.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01902,MIR439,1287,microRNA MIR439,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.2,88.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,51,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.60004,-3.27148,1600000,153722,191,93,true,-3.33,0.71904,2013-10-04T02:54:51.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01903,mir-500,1299,microRNA mir-500,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.6,76.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,38,108,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.65669,-4.29775,1600000,196339,172,84,true,-3.5126,0.71875,2013-02-01T12:07:17.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01904,HOTAIR_1,2233,HOTAIR conserved region 1,Eberhardt R,Predicted; WAR; Eberhardt R,44.0,48.3,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,34,0,0,Gene; lncRNA;,Predicted; WAR;,39,0,,\"\",0.54369,-9.4125,1600000,723519,101,37,false,-3.9601,0.71959,2013-10-04T02:54:56.000Z,2022-05-06T13:10:05.000Z\n",
            "RF01905,HOTAIR_2,2233,HOTAIR conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,66.8,50.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,34,0,0,Gene; lncRNA;,Predicted; CMfinder;,39,0,,\"\",0.64786,-8.20632,1600000,511507,152,69,false,-3.4898,0.71873,2013-10-04T02:54:58.000Z,2021-12-03T12:41:14.000Z\n",
            "RF01906,HOTAIR_3,2233,HOTAIR conserved region 3,Eberhardt R,Predicted; WAR; Eberhardt R,80.0,81.0,68.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,17,30,0,0,Gene; lncRNA;,Predicted; WAR;,38,0,,\"\",0.58055,-9.22651,1600000,322873,192,98,false,-3.39,0.71828,2013-10-04T02:54:59.000Z,2021-12-03T12:41:14.000Z\n",
            "RF01909,CDKN2B-AS,2234,CDKN2B antisense RNA 1 intronic convserved region,Eberhardt R,Eberhardt R,80.0,83.4,42.4,\"CDKN2B-AS, also known as ANRIL (antisense non-coding RNA in the INK4 locus) is a long non-coding RNA consisting of 19 exons, spanning 126.3kb in the genome, and spliced product is a 3834bp RNA. It is located within the p15/CDKN2B-p16/CDKN2A-p14/ARF gene cluster, in the antisense direction. Single nucleotide polymorphisms (SNPs) which alter the expression of CDKN2B-AS are associated with many diseases, including coronary artery disease, diabetes and many cancers [1]. It binds to chromobox 7 (CBX7) within the polycomb repressive complex 1 and to SUZ12, a component of polycomb repression complex 2 and through these interactions is involved in transciptional repression [2,3].\",ANRIL,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,25,34,0,0,Gene; lncRNA;,Predicted; RNAz; Eberhardt R,46,0,,\"\",0.45812,-12.33774,1600000,289421,256,150,false,-3.0517,0.7178,2013-10-04T02:55:06.000Z,2022-05-06T13:10:05.000Z\n",
            "RF01910,mir-506,2235,microRNA mir-506,Boursnell C,Predicted; ClustalW2,78.0,84.8,66.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,18,0,0,Gene; miRNA;,Predicted; RNAalifold,18,0,,\"\",0.66956,-2.3049,1600000,200763,182,87,true,-3.4972,0.71887,2013-10-04T02:55:10.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01911,MIR2118,1287,microRNA MIR2118,Boursnell C,Predicted; ClustalW2,85.0,85.0,84.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,1241,0,0,Gene; miRNA;,Predicted; RNAalifold,20,0,,\"\",0.56146,-6.55566,1600000,191449,294,170,true,-3.1566,0.71579,2013-10-04T02:55:12.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01912,mir-2807,1287,microRNA mir-2807,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.0,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,4739,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.53014,-3.51336,1600000,216121,172,84,true,-3.6632,0.71892,2013-10-04T02:55:17.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01913,mir-2778,1287,microRNA mir-2778,Griffiths-Jones SR,Griffiths-Jones SR,86.0,89.4,85.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,20,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62906,-2.89439,1600000,208166,168,77,true,-3.4229,0.71896,2013-10-04T02:55:25.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01914,mir-932,1287,microRNA mir-932,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.8,42.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,75,0,0,Gene; miRNA;,Predicted; RNAalifold,78,0,,\"\",0.66874,-3.23982,1600000,155708,162,76,true,-3.6641,0.71899,2013-10-04T02:55:26.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01915,mir-2238,1287,microRNA mir-2238,Griffiths-Jones SR,Griffiths-Jones SR,52.2,52.2,52.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,19,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.68738,-2.02797,1600000,188971,175,86,true,-3.7022,0.71887,2013-10-04T02:55:28.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01916,mir-988,1287,microRNA mir-988,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.1,71.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,30,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.6234,-3.65474,1600000,199508,169,80,true,-3.6788,0.71897,2013-10-04T02:55:29.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01917,MIR2587,1287,microRNA MIR2587,Griffiths-Jones SR,Griffiths-Jones SR,75.5,75.5,75.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,239,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62879,-3.01315,1600000,196136,195,94,true,-3.5603,0.71876,2013-10-04T02:55:31.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01918,mir-1249,1287,microRNA mir-1249,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.7,42.9,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,36,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.61001,-4.71702,1600000,186936,165,75,true,-3.747,0.71904,2013-10-04T02:55:33.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01919,mir-1419,1287,microRNA mir-1419,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.5,51.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,13,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66866,-2.6113,1600000,212398,177,87,true,-3.4369,0.71886,2013-10-04T02:55:35.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01920,mir-764,2236,microRNA mir-764,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.5,56.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,34,0,0,Gene; miRNA;,Predicted; RNAalifold,34,0,,\"\",0.6558,-2.79356,1600000,224844,179,85,true,-3.416,0.71896,2013-10-04T02:55:36.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01921,mir-1296,1287,microRNA mir-1296,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.6,60.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,30,0,0,Gene; miRNA;,Predicted; RNAalifold,33,0,,\"\",0.59723,-3.8076,1600000,186155,184,87,true,-3.5429,0.7188,2013-10-04T02:55:38.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01922,mir-654,1287,microRNA mir-654,Griffiths-Jones SR,Griffiths-Jones SR,63.5,63.5,62.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,30,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.69083,-2.95491,1600000,230332,173,81,true,-3.6257,0.71896,2013-10-04T02:55:40.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01923,mir-711,2237,microRNA mir-711,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.3,58.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,32,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.67949,-2.52226,1600000,202623,156,69,true,-3.6949,0.71915,2013-10-04T02:55:42.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01924,mir-2774,1287,microRNA mir-2774,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.0,50.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,3412,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.78298,-1.83627,1600000,226535,152,67,true,-3.454,0.71906,2013-10-04T02:55:43.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01925,MIR1428,1287,microRNA MIR1428,Boursnell C,Predicted; ClustalW2,79.0,79.4,76.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,103,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.56937,-4.22582,1600000,169021,214,110,true,-3.5082,0.71856,2013-10-04T02:55:48.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01926,mir-981,1287,microRNA mir-981,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.7,50.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,17,87,0,0,Gene; miRNA;,Predicted; RNAalifold,88,0,,\"\",0.87105,-1.37266,1600000,180661,167,86,true,-3.9349,0.71877,2013-10-04T02:55:50.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01927,MIR1222,1287,microRNA MIR1222,Boursnell C,Predicted; ClustalW2,90.0,101.1,68.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,5,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63225,-4.12749,1600000,191422,238,134,true,-3.6017,0.71837,2013-10-04T02:55:51.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01928,Mico1,1276,Maternal intergenic circadian oscillating RNA 1,Eberhardt R,Predicted; ClustalW; Eberhardt R,60.0,82.1,49.8,Maternal intergenic circadian oscillating 1 (Mico1) and Mico1 opposite strand (Mico1os) are lon non-coding RNAs. Their expression shows circadian oscillations [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.32 -Z 549862.597050 CM SEQDB,3,39,0,0,Gene; lncRNA;,Predicted; Pfold;,38,0,,\"\",0.43817,-16.73405,1600000,702492,333,205,false,-3.4646,0.71164,2013-02-01T11:59:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01929,bxd_1,2238,Bithoraxoid conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,75.0,86.9,40.1,\"Bithoraxoid (bxd) is a long non-coding RNA found in Drosophila. It silences the expression of the Ultrabithorax (Ubx) gene by transcriptional interference [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,6,0,0,Gene; lncRNA;,Predicted; CMfinder;,8,0,,\"\",0.5831,-8.75128,1600000,274079,179,86,false,-3.56,0.71846,2013-10-04T02:55:55.000Z,2021-12-03T12:41:14.000Z\n",
            "RF01930,bxd_2,2238,Bithoraxoid conserved region 2,Eberhardt R,Predicted; WAR; Eberhardt R,80.0,91.0,35.0,\"Bithoraxoid (bxd) is a long non-coding RNA found in Drosophila. It silences the expression of the Ultrabithorax (Ubx) gene by transcriptional interference [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,5,0,0,Gene; lncRNA;,Predicted; WAR;,8,0,,\"\",0.58545,-9.518,1600000,357803,183,91,false,-3.2479,0.71835,2013-10-04T02:55:57.000Z,2021-12-03T12:41:14.000Z\n",
            "RF01931,bxd_3,2238,Bithoraxoid conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,70.0,102.1,47.3,\"Bithoraxoid (bxd) is a long non-coding RNA found in Drosophila. It silences the expression of the Ultrabithorax (Ubx) gene by transcriptional interference [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; lncRNA;,Predicted; CMfinder;,10,0,,\"\",0.40226,-9.4583,1600000,202559,246,137,false,-3.0069,0.71793,2013-10-04T02:55:58.000Z,2021-11-19T11:53:19.000Z\n",
            "RF01932,bxd_4,2238,Bithoraxoid conserved region 4,Eberhardt R,Predicted; ClustalW; Eberhardt R,40.0,75.1,36.9,\"Bithoraxoid (bxd) is a long non-coding RNA found in Drosophila. It silences the expression of the Ultrabithorax (Ubx) gene by transcriptional interference [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; lncRNA;,Predicted; RNAalifold;,11,0,,\"\",0.47302,-11.46268,1600000,355350,203,106,false,-2.6627,0.71826,2013-10-04T02:56:00.000Z,2021-11-19T11:53:19.000Z\n",
            "RF01933,bxd_5,2238,Bithoraxoid conserved region 5,Eberhardt R,Predicted; WAR; Eberhardt R,50.0,67.4,34.2,\"Bithoraxoid (bxd) is a long non-coding RNA found in Drosophila. It silences the expression of the Ultrabithorax (Ubx) gene by transcriptional interference [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,9,0,0,Gene; lncRNA;,Predicted; WAR;,11,0,,\"\",0.64428,-8.06106,1600000,408944,136,57,false,-3.9303,0.71898,2013-10-04T02:56:01.000Z,2021-12-03T12:41:14.000Z\n",
            "RF01934,bxd_6,2238,Bithoraxoid conserved region 6,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,95.8,37.5,\"Bithoraxoid (bxd) is a long non-coding RNA found in Drosophila. It silences the expression of the Ultrabithorax (Ubx) gene by transcriptional interference [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,9,0,0,Gene; lncRNA;,Predicted; CMfinder;,11,0,,\"\",0.48371,-10.35026,1600000,247616,187,93,false,-3.7104,0.71839,2013-10-04T02:56:03.000Z,2021-12-03T12:41:14.000Z\n",
            "RF01935,bxd_7,2238,Bithoraxoid conserved region 7,Eberhardt R,Predicted; MAFFT; Eberhardt R,60.0,91.4,43.7,\"Bithoraxoid (bxd) is a long non-coding RNA found in Drosophila. It silences the expression of the Ultrabithorax (Ubx) gene by transcriptional interference [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; lncRNA;,Predicted; RNAalifold;,4,0,,\"\",0.53058,-8.88818,1600000,397603,150,66,false,-3.7042,0.71881,2013-10-04T02:56:04.000Z,2021-12-03T12:41:14.000Z\n",
            "RF01936,mir-63,1287,microRNA mir-63,Boursnell C,Predicted; ClustalW2,55.0,55.5,51.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,6,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63038,-2.80146,1600000,175716,221,116,true,-3.2868,0.71872,2013-10-04T02:56:06.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01937,mir-2780,1287,microRNA mir-2780,Griffiths-Jones SR,Griffiths-Jones SR,95.5,95.6,95.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,1169,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.71939,-2.1644,1600000,206390,185,88,true,-3.3203,0.71879,2013-10-04T02:56:07.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01938,mir-1251,1287,microRNA mir-1251,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.6,69.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,38,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.69346,-3.00234,1600000,218474,152,66,true,-3.7069,0.71915,2013-10-04T02:56:10.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01939,mir-761,1287,microRNA mir-761,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.2,55.5,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,37,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.62023,-4.53572,1600000,257676,151,65,true,-3.423,0.71933,2013-10-04T02:56:12.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01940,hvt-mir-H,1287,microRNA hvt-mir-H9,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.0,42.2,This family represents the microRNA (miRNA) precursor of Herpesvirus of turkeys hvt-mir-H imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.69898,-3.04164,1600000,294853,152,64,true,-3.4607,0.72507,2013-10-04T02:56:14.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01941,MIR1223,1287,microRNA MIR1223,Griffiths-Jones SR,Griffiths-Jones SR,77.0,77.0,68.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,10,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6824,-3.08982,1600000,192383,227,102,true,-3.6061,0.71859,2013-10-04T02:56:15.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01943,mir-999,1287,microRNA mir-999,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.1,57.4,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,45,0,0,Gene; miRNA;,Predicted; RNAalifold,43,0,,\"\",0.67232,-2.85042,1600000,174696,175,84,true,-3.791,0.71897,2013-10-04T02:56:37.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01944,mir-2518,1287,microRNA mir-2518,Griffiths-Jones SR,Griffiths-Jones SR,61.0,61.4,41.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,57,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62809,-4.02235,1600000,252188,154,67,true,-3.5428,0.71922,2013-10-04T02:56:38.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01945,mir-1388,1287,microRNA mir-1388,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.4,56.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,78,0,0,Gene; miRNA;,Predicted; RNAalifold,79,0,,\"\",0.6396,-2.85395,1600000,128245,185,93,true,-3.779,0.71884,2013-10-04T02:56:40.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01946,KCNQ1OT1_1,1277,KCNQ1 overlapping transcript 1 conserved region 1,Eberhardt R,Predicted; ClustalW; Eberhardt R,70.0,92.3,39.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,34,0,0,Gene; lncRNA;,Predicted; RNAalifold;,36,0,,\"\",0.66383,-13.37086,1600000,363265,635,464,false,-4.7094,0.70132,2013-10-04T02:56:41.000Z,2021-12-03T12:04:12.000Z\n",
            "RF01947,KCNQ1OT1_2,1277,KCNQ1 overlapping transcript 1 conserved region 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,80.0,86.2,46.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,18,31,0,0,Gene; lncRNA;,Predicted; RNAalifold;,39,0,,\"\",0.37709,-18.13004,1600000,470463,318,196,false,-3.3521,0.71251,2013-02-01T11:59:14.000Z,2022-05-06T13:09:56.000Z\n",
            "RF01948,KCNQ1OT1_3,1277,KCNQ1 overlapping transcript 1 conserved region 3,Eberhardt R,Predicted; ClustalW; Eberhardt R,60.0,69.8,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,31,0,0,Gene; lncRNA;,Predicted; RNAalifold;,32,0,,\"\",0.44036,-9.91389,1600000,274287,217,116,false,-3.1248,0.71813,2013-10-04T02:56:46.000Z,2021-12-03T12:04:12.000Z\n",
            "RF01950,KCNQ1OT1_5,1277,KCNQ1 overlapping transcript 1 conserved region 5,Eberhardt R,Predicted; CMfinder; Eberhardt R,120.0,156.7,37.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,32,0,0,Gene; lncRNA;,Predicted; CMfinder;,36,0,,\"\",0.36962,-16.39133,1600000,453641,356,226,false,-3.505,0.7099,2013-10-04T02:56:47.000Z,2021-12-03T12:41:13.000Z\n",
            "RF01951,SOX2OT_exon1,1278,SOX2 overlapping transcript exon 1,Eberhardt R,Predicted; RNAz; Eberhardt R,65.0,73.4,45.4,\"SOX2 overlapping transcript (SOX2OT) is a long non-coding RNA, containing at least 5 exons. The SOX2 gene, an important regulator of neurogenesis, lies within one of the introns of SOX2OT [1]. SOX2OT and SOX2DOT (an isoform of SOXOT transcribed from a distal highly conserved element) are expressed in zones of neurogenesis within the brain. It is associated with central nervous system structures in zebrafish and chicken embryonic brains, and is dynamically regulated during embryogenesis [2]. SOX2OT may play a role in vertebrate development [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,29,98,0,0,Gene; lncRNA;,Predicted; RNAz;,107,0,,\"\",0.48925,-15.36204,1600000,524643,356,230,false,-3.5066,0.70964,2013-10-04T02:56:49.000Z,2021-12-03T12:41:13.000Z\n",
            "RF01952,SOX2OT_exon2,1278,SOX2 overlapping transcript exon 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,45.0,57.7,37.0,\"SOX2 overlapping transcript (SOX2OT) is a long non-coding RNA, containing at least 5 exons. The SOX2 gene, an important regulator of neurogenesis, lies within one of the introns of SOX2OT [1]. SOX2OT and SOX2DOT (an isoform of SOXOT transcribed from a distal highly conserved element) are expressed in zones of neurogenesis within the brain. It is associated with central nervous system structures in zebrafish and chicken embryonic brains, and is dynamically regulated during embryogenesis [2]. SOX2OT may play a role in vertebrate development [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,37,0,0,Gene; lncRNA;,Predicted; Pfold;,44,0,,\"\",0.60604,-8.59587,1600000,350048,150,69,false,-4.0226,0.71875,2013-10-04T02:56:51.000Z,2021-12-03T12:41:13.000Z\n",
            "RF01953,SOX2OT_exon3,1278,SOX2 overlapping transcript exon 3,Eberhardt R,Predicted; MAFFT; Eberhardt R,70.0,72.3,55.0,\"SOX2 overlapping transcript (SOX2OT) is a long non-coding RNA, containing at least 5 exons. The SOX2 gene, an important regulator of neurogenesis, lies within one of the introns of SOX2OT [1]. SOX2OT and SOX2DOT (an isoform of SOXOT transcribed from a distal highly conserved element) are expressed in zones of neurogenesis within the brain. It is associated with central nervous system structures in zebrafish and chicken embryonic brains, and is dynamically regulated during embryogenesis [2]. SOX2OT may play a role in vertebrate development [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,84,0,0,Gene; lncRNA;,Predicted; RNAalifold;,86,0,,\"\",0.57365,-14.26366,1600000,509563,413,276,false,-4.1941,0.7069,2013-10-04T02:56:53.000Z,2021-12-03T12:41:13.000Z\n",
            "RF01954,SOX2OT_exon4,1278,SOX2 overlapping transcript exon 4,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,40.1,32.7,\"SOX2 overlapping transcript (SOX2OT) is a long non-coding RNA, containing at least 5 exons. The SOX2 gene, an important regulator of neurogenesis, lies within one of the introns of SOX2OT [1]. SOX2OT and SOX2DOT (an isoform of SOXOT transcribed from a distal highly conserved element) are expressed in zones of neurogenesis within the brain. It is associated with central nervous system structures in zebrafish and chicken embryonic brains, and is dynamically regulated during embryogenesis [2]. SOX2OT may play a role in vertebrate development [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,60,0,0,Gene; lncRNA;,Predicted; CMfinder;,62,0,,\"\",0.51296,-11.24919,1600000,409002,204,109,false,-3.3482,0.71821,2013-02-01T11:59:16.000Z,2021-12-03T12:41:13.000Z\n",
            "RF01955,NEAT1_1,1279,Nuclear enriched abundant transcript 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,50.1,39.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.34 -Z 549862.597050 CM SEQDB,17,46,0,0,Gene; lncRNA;,Predicted; CMfinder;,42,0,,\"\",0.49232,-11.04892,1600000,287506,223,122,false,-3.2698,0.71805,2013-10-04T02:56:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01956,NEAT1_2,1279,Nuclear enriched abundant transcript 1 conserved region 2,Eberhardt R,Predicted; MAFFT; Eberhardt R,60.0,74.7,50.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.89 -Z 549862.597050 CM SEQDB,13,44,0,0,Gene; lncRNA;,Predicted; RNAalifold;,40,0,,\"\",0.54406,-10.77626,1600000,452376,214,116,false,-3.2651,0.71807,2013-10-04T02:56:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01957,NEAT1_3,1279,Nuclear enriched abundant transcript 1 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,55.0,57.8,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.61 -Z 549862.597050 CM SEQDB,18,53,0,0,Gene; lncRNA;,Predicted; CMfinder;,44,0,,\"\",0.41338,-14.16122,1600000,316830,275,159,false,-3.2412,0.71703,2013-02-01T11:59:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01959,SSU_rRNA_archaea,1459,Archaeal small subunit ribosomal RNA,\"Gutell RR, Nawrocki E, Gardner PP\",Published; PMID:11869452,50.0,50.0,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.25 -Z 549862.597050 CM SEQDB,86,546,0,0,Gene; rRNA;,Published; PMID:11869452,437,0,,\"\",0.68847,-8.52203,1600000,277036,2777,1477,true,-5.9371,0.69575,2013-10-04T02:57:24.000Z,2022-10-29T17:55:16.000Z\n",
            "RF01960,SSU_rRNA_eukarya,1459,Eukaryotic small subunit ribosomal RNA,\"Gutell RR, Gardner PP, Nawrocki E\",Published; PMID:11869452,50.0,50.0,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.38 -Z 742849.287494 CM SEQDB,90,21572,0,0,Gene; rRNA;,Published; PMID:11869452,1038,0,,\"\",0.70379,-8.00813,1600000,296590,2762,1831,true,-6.3316,0.69523,2013-10-04T03:08:45.000Z,2022-10-29T17:55:16.000Z\n",
            "RF01962,RMST_1,2241,Rhabdomyosarcoma 2 associated transcript conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,46.8,35.3,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.76 -Z 549862.597050 CM SEQDB,14,37,0,0,Gene; lncRNA;,Predicted; CMfinder;,41,0,,\"\",0.61864,-8.57604,1600000,412475,136,61,false,-3.6274,0.71891,2013-10-04T03:17:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01963,RMST_2,2241,Rhabdomyosarcoma 2 associated transcript conserved region 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,45.0,47.7,43.3,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.94 -Z 549862.597050 CM SEQDB,16,42,0,0,Gene; lncRNA;,Predicted; RNAalifold;,50,0,,\"\",0.46141,-12.60598,1600000,388728,240,135,false,-3.2242,0.71795,2013-10-04T03:17:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01964,RMST_3,2241,Rhabdomyosarcoma 2 associated transcript conserved region 3,Eberhardt R,Predicted; MAFFT; Eberhardt R,55.0,59.2,53.7,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.21 -Z 549862.597050 CM SEQDB,16,38,0,0,Gene; lncRNA;,Predicted; RNAalifold;,44,0,,\"\",0.48655,-11.49241,1600000,437502,208,109,false,-3.1321,0.71812,2013-10-04T03:17:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01965,RMST_4,2241,Rhabdomyosarcoma 2 associated transcript conserved region 4,Eberhardt R,Predicted; MAFFT; Eberhardt R,40.0,44.4,38.2,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.37 -Z 549862.597050 CM SEQDB,17,39,0,0,Gene; lncRNA;,Predicted; Pfold:,47,0,,\"\",0.48249,-11.41273,1600000,390215,205,108,false,-3.2135,0.71822,2013-10-04T03:17:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01966,RMST_5,2241,Rhabdomyosarcoma 2 associated transcript conserved region 5,Eberhardt R,Predicted; CMfinder; Eberhardt R,58.0,67.2,34.3,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.46 -Z 549862.597050 CM SEQDB,21,39,0,0,Gene; lncRNA;,Predicted; CMfinder;,47,0,,\"\",0.51201,-10.65849,1600000,313502,232,128,false,-3.4174,0.71797,2013-10-04T03:17:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01967,RMST_6,2241,Rhabdomyosarcoma 2 associated transcript conserved region 6,Eberhardt R,Predicted; CMfinder; Eberhardt R,90.0,90.7,67.2,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.26 -Z 549862.597050 CM SEQDB,18,94,0,0,Gene; lncRNA;,Predicted; CMfinder;,102,0,,\"\",0.38154,-16.16296,1600000,450927,329,203,false,-3.2369,0.71182,2013-10-04T03:17:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01968,RMST_7,2241,Rhabdomyosarcoma 2 associated transcript conserved region 7,Eberhardt R,Predicted; CMfinder; Eberhardt R,100.0,126.5,61.0,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 44.55 -Z 549862.597050 CM SEQDB,22,91,0,0,Gene; lncRNA;,Predicted; CMfinder;,101,0,,\"\",0.29725,-19.61884,1600000,558785,405,268,false,-3.2434,0.70728,2013-10-04T03:17:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01969,RMST_8,2241,Rhabdomyosarcoma 2 associated transcript conserved region 8,Eberhardt R,Predicted; ClustalW; Eberhardt R,80.0,95.6,61.4,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.22 -Z 549862.597050 CM SEQDB,22,94,0,0,Gene; lncRNA;,Predicted; RNAalifold;,99,0,,\"\",0.45871,-12.92897,1600000,361162,250,143,false,-3.2107,0.71784,2013-10-04T03:17:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01970,RMST_9,2241,Rhabdomyosarcoma 2 associated transcript conserved region 9,Eberhardt R,Predicted; CMfinder; Eberhardt R,65.0,69.2,63.8,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.38 -Z 549862.597050 CM SEQDB,28,102,0,0,Gene; lncRNA;,Predicted; CMfinder;,111,0,,\"\",0.44831,-14.64387,1600000,441848,315,193,false,-3.6216,0.71283,2013-10-04T03:17:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01971,RMST_10,2241,Rhabdomyosarcoma 2 associated transcript conserved region 10,Eberhardt R,Predicted; CMfinder; Eberhardt R,55.0,57.8,35.0,,NCRMS,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.13 -Z 549862.597050 CM SEQDB,21,39,0,0,Gene; lncRNA;,Predicted; CMfinder;,49,0,,\"\",0.43391,-12.03428,1600000,257291,265,154,false,-3.1851,0.71775,2013-10-04T03:17:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01972,H19_1,2242,H19 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,53.3,39.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 37.53 -Z 549862.597050 CM SEQDB,35,29,0,0,Gene; lncRNA;,Predicted; CMfinder;,37,0,,\"\",0.36176,-14.61231,1600000,453570,154,71,false,-3.0989,0.71871,2013-10-04T03:17:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01973,H19_2,2242,H19 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,51.0,34.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.23 -Z 549862.597050 CM SEQDB,23,30,0,0,Gene; lncRNA;,Predicted; CMfinder;,38,0,,\"\",0.65995,-7.79566,1600000,313529,151,69,false,-3.621,0.71876,2013-10-04T03:17:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01974,H19_3,2242,H19 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,46.0,47.4,44.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.46 -Z 549862.597050 CM SEQDB,5,29,0,0,Gene; lncRNA;,Predicted; CMfinder;,20,0,,\"\",0.48468,-11.95781,1600000,355220,220,116,false,-3.1527,0.71813,2013-10-04T03:17:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01975,HOTAIRM1_1,2243,HOX antisense intergenic RNA myeloid 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,70.0,75.0,50.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.23 -Z 549862.597050 CM SEQDB,14,33,0,0,Gene; lncRNA;,Predicted; CMfinder;,37,0,,\"\",0.33255,-15.64121,1600000,340697,230,125,false,-2.7119,0.71797,2013-10-04T03:17:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01976,HOTAIRM1_2,2243,HOX antisense intergenic RNA myeloid 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,37.0,44.8,34.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.21 -Z 549862.597050 CM SEQDB,7,112,0,0,Gene; lncRNA;,Predicted; CMfinder;,106,0,,\"\",0.39412,-14.61925,1600000,301591,350,220,false,-3.6078,0.71033,2013-10-04T03:17:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01977,HOTAIRM1_3,2243,HOX antisense intergenic RNA myeloid 1 conserved region 3,Eberhardt R,Predicted; MAFFT; Eberhardt R,37.0,43.9,32.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.76 -Z 549862.597050 CM SEQDB,11,38,0,0,Gene; lncRNA;,Predicted; Pfold;,44,0,,\"\",0.65371,-8.4628,1600000,577703,135,59,false,-3.8828,0.71895,2013-10-04T03:17:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01978,HOTAIRM1_4,2243,HOX antisense intergenic RNA myeloid 1 conserved region 4,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,45.3,35.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.87 -Z 549862.597050 CM SEQDB,10,41,0,0,Gene; lncRNA;,Predicted; CMfinder;,43,0,,\"\",0.41661,-11.65029,1600000,330639,198,102,false,-3.127,0.71824,2013-10-04T03:17:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01979,HOTAIRM1_5,2243,HOX antisense intergenic RNA myeloid 1 conserved region 5,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,67.1,52.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.29 -Z 549862.597050 CM SEQDB,10,37,0,0,Gene; lncRNA;,Predicted; CMfinder;,39,0,,\"\",0.4164,-11.76128,1600000,269444,256,145,false,-3.1741,0.71776,2013-10-04T03:17:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01981,PCGEM1,2245,Prostate-specific transcript 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,75.0,80.8,35.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.34 -Z 549862.597050 CM SEQDB,26,36,0,0,Gene; lncRNA;,Predicted; CMfinder;,46,0,,\"\",0.40528,-11.07873,1600000,287261,236,132,false,-3.3103,0.7179,2013-10-04T03:17:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01982,PYLIS_1,2246,Pyrrolysine insertion sequence 1,Eberhardt R,Predicted; WAR,70.0,78.3,59.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,39,0,0,Cis-reg;,Predicted; WAR,23,0,,\"\",0.64203,-4.48547,1600000,225056,193,100,true,-3.2124,0.71839,2013-10-04T03:17:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01983,Pinc,2495,Pregnancy induced noncoding RNA,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,76.5,50.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.82 -Z 549862.597050 CM SEQDB,16,34,0,0,Gene; lncRNA;,Predicted; CMfinder;,40,0,,\"\",0.45475,-11.71454,1600000,294950,265,154,false,-3.2875,0.71768,2013-10-04T03:17:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01984,ZEB2_AS1_1,2248,ZEB2 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,107.9,66.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.79 -Z 549862.597050 CM SEQDB,11,88,0,0,Gene; lncRNA;,Predicted; CMfinder;,94,0,,\"\",0.46684,-11.30651,1600000,391506,282,128,false,-3.2367,0.71797,2013-10-04T03:17:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01985,ZEB2_AS1_2,2248,ZEB2 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,70.0,72.1,49.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.58 -Z 549862.597050 CM SEQDB,11,36,0,0,Gene; lncRNA;,Predicted; CMfinder;,45,0,,\"\",0.63666,-8.03997,1600000,450019,139,61,false,-3.5567,0.71891,2013-10-04T03:17:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01986,ZEB2_AS1_3,2248,ZEB2 antisense RNA 1 conserved region 3,Eberhardt R,Predicted; ClustalW; Eberhardt R,55.0,59.6,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.03 -Z 549862.597050 CM SEQDB,11,39,0,0,Gene; lncRNA;,Predicted; RNAalifold;,43,0,,\"\",0.53511,-11.29661,1600000,472730,201,100,false,-3.3836,0.71824,2013-10-04T03:17:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01987,ZEB2_AS1_4,2248,ZEB2 antisense RNA 1 conserved region 4,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,52.0,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.07 -Z 549862.597050 CM SEQDB,13,36,0,0,Gene; lncRNA;,Predicted; CMfinder;,43,0,,\"\",0.35986,-11.56316,1600000,341818,166,78,false,-3.2185,0.71857,2013-10-04T03:17:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01988,SECIS_2,1328,Selenocysteine insertion sequence 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,70.0,73.6,64.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,56,0,0,Cis-reg;,Predicted; RNAalifold;,55,0,,\"\",0.62731,-3.96811,1600000,355330,134,55,true,-3.6761,0.7194,2013-10-04T03:17:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01989,SECIS_3,1328,Selenocysteine insertion sequence 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,63.0,63.2,62.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.41 -Z 549862.597050 CM SEQDB,24,454,0,0,Cis-reg;,Predicted; CMfinder;,408,0,,\"\",0.67429,-3.98821,1600000,307291,137,59,true,-3.4385,0.7192,2013-10-04T03:17:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01990,SECIS_4,1328,Selenocysteine insertion sequence 4,Eberhardt R,Predicted; ClustalW; Eberhardt R,38.0,38.2,37.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,24,317,0,0,Cis-reg;,Predicted; Pfold;,200,0,,\"\",0.69155,-4.53012,1600000,685866,102,38,true,-4.196,0.71973,2013-10-04T03:17:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01991,SECIS_5,1328,Selenocysteine insertion sequence 5,Eberhardt R,Predicted; WAR; Eberhardt R,50.0,61.5,35.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,25,0,0,Cis-reg;,Predicted; WAR;,26,0,,\"\",0.61202,-2.95839,1600000,198069,153,66,true,-3.3481,0.71914,2013-10-04T03:17:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01992,VIS1,2486,Viral integration site 1,Eberhardt R,Predicted; ClustalW; Eberhardt R,100.0,122.4,64.2,,HIS-1,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.59 -Z 549862.597050 CM SEQDB,27,37,0,0,Gene; lncRNA;,Predicted; Pfold;,47,0,,\"\",0.40822,-15.33785,1600000,403380,323,202,false,-3.4573,0.71192,2013-10-04T03:17:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01996,mir-995,1287,microRNA mir-995,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.1,71.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.6157,-3.4607,1600000,163357,182,88,true,-3.6249,0.71892,2013-10-04T03:17:56.000Z,2023-11-06T16:44:06.000Z\n",
            "RF01997,mir-969,1287,microRNA mir-969,Griffiths-Jones SR,Griffiths-Jones SR,61.0,61.6,53.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,10,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.60641,-3.85911,1600000,236880,180,86,true,-3.4127,0.7188,2013-02-01T12:04:57.000Z,2022-10-29T17:42:07.000Z\n",
            "RF01998,group-II-D1D4-1,1326,Group II catalytic intron D1-D4-1,\"Weinberg Z, Gardner PP\",Weinberg Z,35.0,35.0,34.9,\"Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI. This model and alignment represents only domains V and VI. A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron-intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,237,2034,0,0,Intron;,Predicted; Weinberg Z,872,0,,\"\",0.52566,-9.55999,1600000,435829,229,84,true,-4.2364,0.7186,2013-10-04T03:18:00.000Z,2021-07-05T14:43:57.000Z\n",
            "RF01999,group-II-D1D4-2,1326,Group II catalytic intron D1-D4-2,\"Weinberg Z, Gardner PP\",Weinberg Z,36.0,36.5,35.3,\"Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI. This model and alignment represents only domains V and VI. A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron-intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,144,678,0,0,Intron;,Predicted; Weinberg Z,429,0,,\"\",0.55763,-9.4284,1600000,414385,215,118,true,-3.5664,0.71841,2013-10-04T03:18:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02000,MIR1846,1287,MIR1846 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,77.5,77.6,77.2,Family represents the microRNA (miRNA) precursor MIR1846 imported from miRBase [3].,C1674,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,71,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.6095,-2.71371,1600000,116958,212,110,true,-3.2894,0.71871,2013-10-04T03:18:07.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02001,group-II-D1D4-3,1326,Group II catalytic intron D1-D4-3,\"Weinberg Z, Gardner PP\",Weinberg Z,32.0,32.0,31.9,\"Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI. This model and alignment represents only domains V and VI. A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron-intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.96 -Z 549862.597050 CM SEQDB,406,3139,0,0,Intron;,Predicted; Weinberg Z,1157,0,,\"\",0.56506,-9.25076,1600000,411885,498,171,true,-2.9924,0.71558,2013-10-04T03:18:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02003,group-II-D1D4-4,1326,Group II catalytic intron D1-D4-4,\"Weinberg Z, Gardner PP\",Weinberg Z,49.0,49.0,48.9,\"Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI. This model and alignment represents only domains V and VI. A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron-intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.36 -Z 549862.597050 CM SEQDB,90,530,0,0,Intron;,Predicted; Weinberg Z,336,0,,\"\",0.57628,-9.92348,1600000,449125,506,138,true,-2.7697,0.71806,2013-10-04T03:18:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02004,group-II-D1D4-5,1326,Group II catalytic intron D1-D4-5,\"Weinberg Z, Gardner PP\",Weinberg Z,34.5,34.6,34.3,\"Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI. This model and alignment represents only domains V and VI. A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron-intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,139,447,0,0,Intron;,Predicted; Weinberg Z,290,0,,\"\",0.58957,-8.90674,1600000,367737,375,201,true,-3.3376,0.71217,2013-10-04T03:18:19.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02005,group-II-D1D4-6,1326,Group II catalytic intron D1-D4-6,\"Weinberg Z, Gardner PP\",Weinberg Z,49.0,49.0,48.8,\"Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI. This model and alignment represents only domains V and VI. A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron-intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,174,1189,0,0,Intron;,Predicted; Weinberg Z,524,0,,\"\",0.73383,-8.37008,1600000,459785,413,225,true,-3.6647,0.71022,2013-10-04T03:18:22.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02006,mir-1253,1287,microRNA mir-1253,Griffiths-Jones SR,Griffiths-Jones SR,103.0,109.3,102.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.63753,-2.62123,1600000,168738,194,94,true,-3.301,0.7188,2013-10-04T03:18:25.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02007,mir-1237,1287,mir-1237 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,65.0,65.9,48.3,Family represents the microRNA (miRNA) precursor mir-1237 imported from miRBase [4].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.48971,-5.0585,1600000,203597,205,102,true,-2.8772,0.71866,2013-10-04T03:18:27.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02008,mir-621,1287,microRNA mir-621,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.9,51.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,13,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61771,-3.89237,1600000,242372,178,83,true,-3.4247,0.71889,2013-10-04T03:18:28.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02009,mir-987,1287,microRNA mir-987,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.5,73.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,12,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.61751,-2.7913,1600000,156268,190,92,true,-3.4477,0.71879,2013-10-04T03:18:30.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02010,mir-3180,2251,microRNA mir-3180,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.9,70.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.52468,-4.34954,1600000,165356,191,92,true,-2.8148,0.71899,2013-10-04T03:18:32.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02011,mir-575,2252,microRNA mir-575,Griffiths-Jones SR,Griffiths-Jones SR,100.0,107.8,90.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.63772,-3.40266,1600000,195125,172,79,true,-3.1943,0.7191,2013-10-04T03:18:33.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02012,group-II-D1D4-7,1326,Group II catalytic intron D1-D4-7,\"Weinberg Z, Gardner PP\",Weinberg Z,34.0,34.2,33.8,\"Group II introns are found in rRNA, tRNA and mRNA of organelles in fungi, plants and protists, and also in mRNA in bacteria. They are large self-splicing ribozymes and have 6 structural domains (usually designated dI to dVI. This model and alignment represents only domains V and VI. A subset of group II introns also encode essential splicing proteins in intronic ORFs. The length of these introns can therefore be up to 3kb. Splicing occurs in almost identical fashion to nuclear pre-mRNA splicing with two transesterification steps. The 2' hydroxyl of a bulged adenosine in domain VI attacks the 5' splice site, followed by nucleophilic attack on the 3' splice site by the 3' OH of the upstream exon. Protein machinery is required for splicing in vivo, and long range intron-intron and intron-exon interactions are important for splice site positioning. Group II introns are further sub-classified into groups IIA and IIB which differ in splice site consensus, distance of bulged A from 3' splice site, some tertiary interactions, and intronic ORF phylogeny [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,244,804,0,0,Intron;,Predicted; Weinberg Z,507,0,,\"\",0.53016,-10.36704,1600000,400355,317,148,true,-3.4234,0.71797,2013-10-04T03:18:35.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02014,mir-1178,1287,mir-1178 microRNA precursor family,\"Griffiths-Jones SR, Griffiths-Jones SR\",Griffiths-Jones SR,75.0,96.9,73.9,Family represents the microRNA (miRNA) precursor mir-1178 imported from miRBase [4].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.55581,-4.67644,1600000,187186,190,91,true,-3.4109,0.71873,2013-10-04T03:18:40.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02015,mir-1287,1287,microRNA mir-1287,Griffiths-Jones SR,Griffiths-Jones SR,93.3,103.6,93.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.6236,-3.364,1600000,238941,184,88,true,-3.3047,0.71889,2013-10-04T03:18:42.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02016,mir-1183,1287,microRNA mir-1183,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.0,34.2,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.69007,-2.54488,1600000,196450,187,89,true,-3.2423,0.71875,2013-10-04T03:18:44.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02017,mir-1912,1287,microRNA mir-1912,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.5,60.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,33,0,0,Gene; miRNA;,Predicted; RNAalifold,33,0,,\"\",0.72921,-1.99415,1600000,237614,168,77,true,-3.4847,0.71897,2013-10-04T03:18:45.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02018,mir-1207,1287,microRNA mir-1207,Griffiths-Jones SR,Griffiths-Jones SR,90.0,94.5,43.1,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.61808,-3.07205,1600000,206878,184,87,true,-3.4934,0.71889,2013-10-04T03:18:47.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02019,mir-1265,1287,microRNA mir-1265,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.0,68.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.63214,-3.45926,1600000,218764,175,82,true,-3.5886,0.71905,2013-10-04T03:18:49.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02020,mir-25,2253,mir-25 microRNA precursor family,\"Griffiths-Jones SR, Boursnell C\",Griffiths-Jones SR,40.5,45.9,39.1,Family represents the microRNA (miRNA) precursor mir-412 imported from miRBase [6].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,55,0,0,Gene; miRNA;,Predicted; RNAalifold,52,0,,\"\",0.57758,-3.71436,1600000,225361,179,84,true,-3.2557,0.71884,2013-10-04T03:18:51.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02021,mir-3179,1287,microRNA mir-3179,Boursnell C,Predicted; ClustalW2,90.0,108.9,83.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,11,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.59427,-4.22718,1600000,242203,180,84,true,-3.389,0.71904,2013-10-04T03:18:52.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02022,mir-1275,1287,microRNA mir-1275,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.8,64.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.6648,-2.88355,1600000,244785,171,78,true,-3.4851,0.71882,2013-10-04T03:18:55.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02023,mir-1208,1287,microRNA mir-1208,Griffiths-Jones SR,Griffiths-Jones SR,90.0,95.5,66.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61485,-3.82153,1600000,265963,164,73,true,-3.4851,0.71916,2013-02-01T12:07:19.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02024,mir-1180,1287,microRNA mir-1180,Griffiths-Jones SR,Griffiths-Jones SR,49.0,49.3,41.8,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,15,0,0,Gene; miRNA;,Predicted; RNAalifold,17,0,,\"\",0.54409,-4.79091,1600000,286093,155,68,true,-3.4152,0.71925,2013-10-04T03:18:59.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02025,mir-3017,1287,microRNA mir-3017,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.5,50.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,34,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59196,-4.98962,1600000,312707,160,70,true,-3.1245,0.72331,2013-10-04T03:19:01.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02026,mir-2833,1287,microRNA mir-2833,Griffiths-Jones SR,Griffiths-Jones SR,65.0,65.3,54.0,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,15,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64178,-3.08239,1600000,277411,158,69,true,-3.4905,0.71914,2013-10-04T03:19:03.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02027,MIR2907,1287,microRNA MIR2907,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.0,59.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,141,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61879,-3.7227,1600000,236789,149,65,true,-3.4526,0.71915,2013-10-04T03:19:04.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02028,mir-1827,1287,microRNA mir-1827,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.4,73.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,14,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.68575,-2.75324,1600000,225261,153,66,true,-3.4361,0.71915,2013-10-04T03:19:07.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02029,sraA,2254,sraA,Barquist LE,Predicted; FSA; RNAalifold; Infernal,56.0,58.3,55.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.00 -Z 549862.597050 CM SEQDB,20,100,0,0,Gene; sRNA;,Predicted; FSA; RNAalifold,110,0,,\"\",0.4846,-6.654,1600000,244085,184,95,true,-3.698,0.71869,2013-10-04T03:19:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02030,tp2,2254,tp2,Barquist LE,Predicted; FSA; RNAalifold; Infernal,50.0,51.5,45.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.70 -Z 549862.597050 CM SEQDB,24,104,0,0,Gene; sRNA;,Predicted; FSA; RNAalifold,116,0,,\"\",0.52844,-4.03307,1600000,160760,324,114,true,-3.4977,0.71845,2013-10-04T03:19:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02031,tpke11,2254,tpke11,Barquist LE,Predicted; WAR; Infernal,43.0,44.7,41.5,In Yersinia located in ICR upstream dnaJ (chaperone)gene. Has temperature responsive RNA structure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.24 -Z 549862.597050 CM SEQDB,28,124,0,0,Gene; sRNA;,Predicted; WAR; T-coffee consensus,137,0,,\"\",0.62148,-5.08835,1600000,240341,152,76,true,-3.9753,0.71903,2013-10-04T03:19:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02032,GOLLD,2255,\"Giant, ornate, lake- and Lactobacillales-derived (GOLLD) RNA\",\"Weinberg Z, Eberhardt R\",Published; PMID:12376536; Weinberg Z,150.0,128.1,40.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,35,23,0,0,Gene;,Published; PMID:12376536; Weinberg Z,26,0,,\"\",0.6775,-9.08673,1600000,471314,831,436,true,-4.2087,0.7019,2013-10-04T03:19:14.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02033,HEARO,2256,HNH endonuclease-associated RNA and ORF (HEARO) RNA,\"Weinberg Z, Eberhardt R\",Published; PMID:12376536; Weinberg Z,75.9,75.9,75.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,139,843,0,0,Gene;,Published; PMID:12376536; Weinberg Z,193,0,,\"\",0.55999,-9.64518,1600000,317127,435,211,true,-3.5791,0.71133,2013-10-04T03:19:17.000Z,2023-11-06T17:21:47.000Z\n",
            "RF02034,IMES-1,2257,IMES-1 RNA motif,\"Weinberg Z, Eberhardt R\",Published; PMID:12376536; Weinberg Z,81.0,81.9,40.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,172,4,0,0,Gene;,Published; PMID:12376536; Weinberg Z,4,0,,\"\",0.55775,-9.44459,1600000,415850,308,191,true,-2.8079,0.7131,2013-10-04T03:19:19.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02035,IMES-2,2258,IMES-2 RNA motif,\"Weinberg Z, Eberhardt R\",Published; PMID:12376536; Weinberg Z,54.7,54.7,45.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,320,7,0,0,Gene;,Published; PMID:12376536; Weinberg Z,8,0,,\"\",0.67378,-8.00665,1600000,420710,244,141,true,-2.7814,0.71817,2013-10-04T03:19:22.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02036,IMES-3,2259,IMES-3 RNA motif,\"Weinberg Z, Eberhardt R\",Published; PMID:12376536; Weinberg Z,55.0,57.0,38.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,32,0,0,0,Gene;,Published; PMID:12376536; Weinberg Z,1,0,,\"\",0.57127,-6.75322,1600000,244970,263,149,true,-2.8412,0.71818,2013-10-04T03:19:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02037,IMES-4,2260,IMES-4 RNA motif,\"Weinberg Z, Eberhardt R\",Published; PMID:12376536; Weinberg Z,48.0,50.0,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,30,0,0,0,Gene;,Published; PMID:12376536; Weinberg Z,1,0,,\"\",0.48648,-9.36626,1600000,477556,168,80,true,-3.2563,0.71886,2013-10-04T03:19:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02038,SPRY4-IT1_1,2261,SPRY4-IT1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,72.8,44.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.35 -Z 549862.597050 CM SEQDB,23,34,0,0,Gene; lncRNA;,Predicted; CMfinder;,41,0,,\"\",0.5125,-11.31609,1600000,420629,178,91,false,-3.0767,0.7184,2013-10-04T03:19:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02039,SPRY4-IT1_2,2261,SPRY4-IT1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,43.0,31.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.44 -Z 549862.597050 CM SEQDB,24,36,0,0,Gene; lncRNA;,Predicted; CMfinder;,44,0,,\"\",0.57288,-10.08972,1600000,361278,212,121,false,-3.2998,0.71802,2013-10-04T03:19:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02040,HOTTIP_1,2262,\"HOXA transcript at the distal tip, conserved region 1\",Eberhardt R,Predicted; ClustalW; Eberhardt R,50.0,51.8,41.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 549862.597050 CM SEQDB,7,28,0,0,Gene; lncRNA;,Predicted; RNAalifold;,30,0,,\"\",0.56743,-10.74059,1600000,796598,133,57,false,-3.6114,0.719,2013-10-04T03:19:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02041,HOTTIP_2,2262,\"HOXA transcript at the distal tip, conserved region 2\",Eberhardt R,Predicted; MAFFT; Eberhardt R,80.0,87.7,71.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 15.68 -Z 549862.597050 CM SEQDB,25,74,0,0,Gene; lncRNA;,Predicted; Pfold;,83,0,,\"\",0.6349,-14.69987,1600000,694452,551,374,false,-4.183,0.70327,2013-10-04T03:19:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02042,HOTTIP_3,2262,\"HOXA transcript at the distal tip, conserved region 3\",Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,99.7,39.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 16.13 -Z 549862.597050 CM SEQDB,23,36,0,0,Gene; lncRNA;,Predicted; CMfinder;,45,0,,\"\",0.62587,-14.53449,1600000,627333,501,353,false,-4.2006,0.70389,2013-10-04T03:19:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02043,HOTTIP_4,2262,\"HOXA transcript at the distal tip, conserved region 4\",Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,45.8,33.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.08 -Z 549862.597050 CM SEQDB,19,37,0,0,Gene; lncRNA;,Predicted; CMfinder;,46,0,,\"\",0.43624,-12.95224,1600000,267502,279,166,false,-3.6457,0.71601,2013-10-04T03:19:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02044,CDKN2B-AS_2,2234,CDKN2B antisense RNA 1 convserved region 2,Eberhardt R,Predicted; WAR; Eberhardt R,50.0,53.6,42.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,17,0,0,Gene; lncRNA;,Predicted; WAR;,21,0,,\"\",0.56019,-10.01476,1600000,547192,115,46,false,-3.5513,0.72193,2013-10-04T03:19:42.000Z,2022-05-06T12:43:53.000Z\n",
            "RF02045,CDKN2B-AS_3,2234,CDKN2B antisense RNA 1 convserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,72.0,48.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,18,29,0,0,Gene; lncRNA;,Predicted; CMfinder;,37,0,,\"\",0.40266,-11.73395,1600000,454738,243,144,false,-3.1389,0.71786,2013-10-04T03:19:44.000Z,2021-12-03T12:41:14.000Z\n",
            "RF02046,Sphinx_1,2263,Sphinx_ conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,34.0,34.0,33.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.67 -Z 549862.597050 CM SEQDB,3,155,0,0,Gene; lncRNA;,Predicted; CMfinder;,75,0,,\"\",0.49053,-10.98755,1600000,306889,198,100,false,-3.3457,0.71824,2013-10-04T03:19:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02047,Sphinx_2,2263,Sphinx conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,34.0,34.0,31.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.64 -Z 549862.597050 CM SEQDB,37,168,0,0,Gene; lncRNA;,Predicted; CMfinder;,83,0,,\"\",0.48319,-12.14497,1600000,401326,321,158,false,-3.3461,0.71715,2013-10-04T03:19:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02048,STnc30,1236,Salmonella enterica conserved region STnc30,Burge SW,Burge SW,120.0,159.8,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,4,0,0,Gene; sRNA;,\"Predicted; WAR; Burge, SW\",5,0,,\"\",0.4833,-8.60095,1600000,296176,246,132,true,-3.1743,0.71831,2013-10-04T03:19:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02049,STnc460,1236,Salmonella enterica sRNA STnc460,\"Eberhardt R, Barquist LE\",Predicted; FSA,70.0,76.0,38.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; sRNA;,Predicted; RNIE; RNAalifold,2,0,,\"\",0.55774,-8.15339,1600000,344538,262,146,true,-2.8979,0.71822,2013-10-04T03:19:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02050,STnc470,1236,Salmonella enterica sRNA STnc470,Burge SW,Burge SW,80.0,84.5,58.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; WAR; Burge SW,5,0,,\"\",0.62967,-9.57358,1600000,396370,359,223,true,-3.9196,0.71031,2013-10-04T03:19:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02051,STnc450,1236,Enterobacterial sRNA STnc450,Eberhardt R,Predicted; CMfinder; Eberhardt R,37.0,37.4,34.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,33,0,0,Gene; sRNA;,Predicted; CMfinder;,42,0,,\"\",0.63447,-5.72784,1600000,330627,131,57,true,-3.2842,0.71935,2013-10-04T03:19:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02052,STnc630,1236,Enterobacterial sRNA STnc630,Eberhardt R,Predicted; CMfinder; Eberhardt R,55.0,60.1,49.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,10,0,0,Gene; sRNA;,Predicted; CMfinder;,18,0,,\"\",0.53933,-8.04658,1600000,341221,285,166,true,-3.1681,0.7162,2013-10-04T03:19:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02053,STnc430,1236,Enterobacterial sRNA STnc430,Eberhardt R,Predicted; CMfinder; Eberhardt R,47.0,47.1,45.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,22,0,0,Gene; sRNA;,Predicted; CMfinder;,24,0,,\"\",0.4844,-8.12652,1600000,327705,276,162,true,-2.9587,0.7168,2013-10-04T03:19:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02054,STnc420,1236,Salmonella enterica sRNA STnc420,Eberhardt R,Predicted; CMfinder; Eberhardt R,70.0,78.0,59.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; CMfinder;,5,0,,\"\",0.61694,-4.04259,1600000,292822,141,58,true,-3.7452,0.7192,2013-10-04T03:20:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02055,STnc380,1236,Enterobacterial sRNA STnc380,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,92.3,41.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,2,0,0,Gene; sRNA;,Predicted; CMfinder;,7,0,,\"\",0.55523,-8.09807,1600000,331582,210,108,true,-3.1343,0.71843,2013-10-04T03:20:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02056,STnc390,1236,Salmonella sRNA STnc390,Eberhardt R,Predicted; WAR; Eberhardt R,50.0,52.9,46.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.72 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; WAR;,7,0,,\"\",0.65328,-4.82488,1600000,365177,146,60,true,-3.3198,0.72733,2013-10-04T03:20:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02057,STnc40,1236,Salmonella enterica sRNA STnc40,Barquist LE,Predicted; LocARNA; Infernal,45.0,46.0,43.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,49,0,0,Gene; sRNA;,Predicted; RNIE; LocARNA,59,0,,\"\",0.54727,-5.5797,1600000,285820,147,70,true,-3.0312,0.71906,2013-10-04T03:20:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02058,STnc400,1236,Gammaproteobacterial sRNA STnc400,Eberhardt R,Predicted; ClustalW; Eberhardt R,80.0,102.3,71.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene;,Predicted; RNAalifold;,3,0,,\"\",0.54736,-6.6413,1600000,224370,231,124,true,-3.3638,0.71834,2013-10-04T03:20:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02059,STnc50,1236,Salmonella enterica sRNA STnc40,Barquist LE,Predicted; FSA; Infernal,55.0,55.0,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; RNIE; RNAalifold,5,0,,\"\",0.5399,-5.62965,1600000,306661,178,84,true,-3.6376,0.71887,2013-10-04T03:20:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02060,STnc410,1236,Enterobacterial sRNA STnc410,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,52.1,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,10,0,0,Gene; sRNA;,Predicted; CMfinder;,19,0,,\"\",0.57815,-8.97565,1600000,353268,269,158,true,-3.2351,0.71719,2013-10-04T03:20:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02061,mir-301,2264,microRNA mir-301,Burge SW,Burge SW; ENSDART00000115588;,56.0,63.7,50.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,228,0,0,Gene; miRNA;,Predicted; WAR; Burge SW;,113,0,,\"\",0.64141,-3.42913,1600000,181028,173,84,true,-3.8174,0.71881,2013-10-04T03:20:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02062,STnc361,1236,Salmonella enterica sRNA STnc361,Eberhardt R,Predicted; ClustalW; Eberhardt R,80.0,107.9,37.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; RNAalifold;,5,0,,\"\",0.47939,-7.41841,1600000,290883,269,150,true,-2.9397,0.71811,2013-10-04T03:20:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02063,STnc350,1236,Salmonella enterica sRNA STnc350,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,87.5,55.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; CMfinder;,6,0,,\"\",0.55315,-6.83778,1600000,333548,157,68,true,-3.2349,0.72479,2013-10-04T03:20:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02064,STnc370,1236,Enterobacterial sRNA STnc370,Eberhardt R,Predicted; WAR; Eberhardt R,54.0,54.2,53.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.31 -Z 549862.597050 CM SEQDB,10,16,0,0,Gene; sRNA;,Predicted; WAR;,25,0,,\"\",0.48861,-4.20181,1600000,264804,152,71,true,-3.5524,0.71911,2013-10-04T03:20:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02065,STnc340,1236,Enterobacterial sRNA STnc340,Eberhardt R,Predicted; WAR; Eberhardt R,80.0,112.5,52.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,5,0,0,Gene; sRNA;,Predicted; WAR;,7,0,,\"\",0.42952,-11.03727,1600000,548423,239,130,true,-2.9172,0.71811,2013-10-04T03:20:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02066,STnc320,1236,Salmonella enterica sRNA STnc320,Eberhardt R,Predicted; WAR; Eberhardt R,50.0,51.2,46.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,5,0,0,Gene; sRNA;,Predicted; WAR;,7,0,,\"\",0.549,-7.58833,1600000,400879,136,55,true,-3.8862,0.71978,2013-10-04T03:20:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02067,STnc310,1236,Salmonella enterica sRNA STnc310,Eberhardt R,Predicted; ClustalW; Eberhardt R,40.0,45.7,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.60 -Z 549862.597050 CM SEQDB,8,4,0,0,Gene; sRNA;,Predicted; RNAalifold;,10,0,,\"\",0.54501,-7.48839,1600000,341199,142,61,true,-3.6334,0.7193,2013-10-04T03:20:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02068,STnc480,1266,Enterobacterial sRNA STnc480,Burge SW,Burge SW,72.0,72.0,70.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,4,0,0,Gene; sRNA;,Predicted;WAR ;Burge SW,9,0,,\"\",0.4974,-5.19705,1600000,251603,173,72,true,-3.6679,0.71884,2013-10-04T03:20:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02069,STnc70,2265,Enterobacterial sRNA STnc70,\"Burge SW, Barquist LE\",Burge SW,40.0,40.2,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,32,0,0,Gene; sRNA;,Predicted; Pfold; Burge SW,32,0,,\"\",0.56497,-8.216,1600000,278270,269,149,true,-3.0971,0.71796,2013-10-04T03:20:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02070,STnc300,1236,Salmonella enterica sRNA STnc300,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,83.3,71.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,4,0,0,Gene; sRNA;,Predicted; CMfinder;,7,0,,\"\",0.42565,-9.49326,1600000,390440,279,159,true,-2.9043,0.71724,2013-10-04T03:20:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02071,STnc280,1236,Salmonella enterica sRNA STnc280,Eberhardt R,Predicted; WAR; Eberhardt R,80.0,95.0,53.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,5,0,0,Gene; sRNA;,Predicted; WAR;,6,0,,\"\",0.54483,-6.18048,1600000,300879,175,83,true,-3.6636,0.71868,2013-10-04T03:20:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02072,STnc590,1236,Salmonella enterica sRNA STnc590,Eberhardt R,Predicted; CMfinder; Eberhardt R,100.0,142.8,60.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; CMfinder;,5,0,,\"\",0.43657,-7.95292,1600000,406000,283,160,true,-2.5844,0.71694,2013-10-04T03:20:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02073,STnc260,1236,Salmonella enterica sRNA STnc260,Eberhardt R,Predicted; CMfinder; Eberhardt R,100.0,137.0,78.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; CMfinder;,7,0,,\"\",0.46023,-8.42719,1600000,295189,302,176,true,-3.0637,0.71481,2013-10-04T03:20:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02074,STnc240,1236,Enterobacterial sRNA STnc240,Eberhardt R,Predicted; WAR; Eberhardt R,55.0,55.6,53.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.65 -Z 549862.597050 CM SEQDB,15,66,0,0,Gene; sRNA;,Predicted; WAR;,76,0,,\"\",0.4883,-5.94189,1600000,272368,160,77,true,-3.3158,0.71886,2013-10-04T03:20:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02075,STnc230,1236,Enterobacterial sRNA STnc230,Eberhardt R,Predicted; WAR; Eberhardt R,50.0,63.8,44.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,15,0,0,Gene; sRNA;,Predicted; WAR;,22,0,,\"\",0.47773,-5.85326,1600000,177898,167,83,true,-3.6974,0.71877,2013-10-04T03:20:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02076,STnc100,1236,Gammaproteobacterial sRNA STnc100,Burge SW,Predicted; CMfinder; Burge SW,47.0,47.1,46.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.97 -Z 549862.597050 CM SEQDB,24,111,0,0,Gene; sRNA;,Predicted; CMfinder,71,0,,\"\",0.57598,-8.96429,1600000,282551,325,206,true,-3.9323,0.71186,2013-10-04T03:20:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02077,STnc220,1236,Salmonella enterica sRNA STnc220,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,69.0,49.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,5,0,0,Gene; sRNA;,Predicted; CMfinder;,7,0,,\"\",0.5622,-4.16271,1600000,247186,173,81,true,-3.5845,0.71873,2013-10-04T03:20:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02078,STnc210,1236,Salmonella enterica sRNA STnc210,Eberhardt R,Predicted; WAR; Eberhardt R,100.0,148.2,45.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; WAR;,5,0,,\"\",0.5431,-7.99336,1600000,348386,303,177,true,-3.3354,0.71482,2013-10-04T03:20:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02079,STnc180,1236,Enterobacterial sRNA STnc180,Eberhardt R,Predicted; ClustalW; Eberhardt R,50.0,55.3,40.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,21,0,0,Gene; sRNA;,Predicted; RNAalifold;,27,0,,\"\",0.57215,-9.63732,1600000,393146,345,217,true,-3.6517,0.71083,2013-10-04T03:20:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02080,STnc170,1236,Salmonella enterica sRNA STnc170,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,97.9,28.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.58 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; CMfinder;,7,0,,\"\",0.6112,-9.02804,1600000,388218,495,333,true,-3.7148,0.70466,2013-10-04T03:20:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02081,STnc550,1236,Enterobacterial sRNA STnc550,Eberhardt R,Predicted; CMfinder; Eberhardt R,48.0,49.2,46.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,64,0,0,Gene; sRNA;,Predicted; CMfinder;,70,0,,\"\",0.62513,-9.89753,1600000,466091,548,388,true,-4.0106,0.70299,2013-02-01T11:59:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02082,STnc540,1236,Enterobacterial sRNA STnc540,Eberhardt R,Predicted; WAR; Eberhardt R,38.0,38.3,36.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,18,0,0,Gene; sRNA;,Predicted; WAR;,19,0,,\"\",0.61653,-6.78943,1600000,275849,333,202,true,-3.6614,0.7121,2013-10-04T03:21:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02083,OrzO-P,2266,OrzO-P RNA antitoxin family,Barquist LE,Predicted; Barquist LE; FSA; Infernal,60.0,66.3,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,2,0,0,Gene; antitoxin;,Predicted; RNAalifold,9,0,,\"\",0.48085,-8.34788,1600000,290663,164,76,true,-3.4312,0.71886,2013-10-04T03:21:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02084,STnc130,1236,Enterobacteria sRNA STnc130,Burge SW,Predicted; WAR;Burge SW,55.0,57.3,48.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.17 -Z 549862.597050 CM SEQDB,13,29,0,0,Gene; sRNA;,Predicted; WAR;,38,0,,\"\",0.47403,-8.08195,1600000,350010,239,135,true,-2.9642,0.71805,2013-10-04T03:21:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02085,Yar_1,2267,Yellow-achaete intergenic RNA conserved region 1,Eberhardt R,Predicted; WAR; Eberhardt R,50.0,77.9,35.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.62 -Z 549862.597050 CM SEQDB,10,8,0,0,Gene; lncRNA;,Predicted; WAR;,13,0,,\"\",0.57712,-7.86739,1600000,227429,164,74,false,-3.9797,0.71867,2013-10-04T03:21:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02086,Yar_2,2267,Yellow-achaete intergenic RNA conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,86.4,33.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.77 -Z 549862.597050 CM SEQDB,7,10,0,0,Gene; lncRNA;,Predicted; CMfinder;,11,0,,\"\",0.556,-9.58687,1600000,330883,160,74,false,-3.4453,0.71865,2013-10-04T03:21:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02087,Yar_3,2267,Yellow-achaete intergenic RNA conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,59.4,38.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.89 -Z 549862.597050 CM SEQDB,4,6,0,0,Gene; lncRNA;,Predicted; CMfinder;,8,0,,\"\",0.66597,-7.92923,1600000,323234,150,67,false,-3.7333,0.7188,2013-10-04T03:21:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02088,STnc510,1266,Enterobacterial sRNA STnc510,Burge SW,Burge SW,80.0,88.3,27.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,6,0,0,Gene; sRNA;,Predicted; WAR; Burge SW,8,0,,\"\",0.63286,-8.62241,1600000,314101,1175,807,true,-4.8539,0.69789,2013-10-04T03:21:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02089,CLRN1-AS1,2268,CLRN1 antisense RNA 1 conserved region,Eberhardt R,Predicted; WAR; Eberhardt R,48.0,49.2,43.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.77 -Z 549862.597050 CM SEQDB,19,34,0,0,Gene; lncRNA;,Predicted; WAR;,40,0,,\"\",0.67091,-7.62399,1600000,279988,157,79,false,-3.628,0.71859,2013-10-04T03:21:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02090,DAOA-AS1_1,2269,DAOA antisense RNA 1 conserved region 1,Eberhardt R,Predicted; WAR; Eberhardt R,50.0,54.0,44.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 36.49 -Z 549862.597050 CM SEQDB,24,37,0,0,Gene; lncRNA;,Predicted; WAR;,47,0,,\"\",0.36849,-15.25529,1600000,556140,320,201,false,-3.4869,0.71199,2013-10-04T03:21:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02091,DAOA-AS1_2,2269,DAOA antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,43.0,43.6,40.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.43 -Z 549862.597050 CM SEQDB,24,64,0,0,Gene; lncRNA;,Predicted; CMfinder;,49,0,,\"\",0.53995,-13.07091,1600000,359236,315,205,false,-3.6857,0.71165,2013-10-04T03:21:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02092,mir-2970,1287,mir-2970 microRNA precursor,Griffiths-Jones SR,Griffiths-Jones SR,72.0,77.0,71.5,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 77.00 -Z 742849.287494 CM SEQDB,4,28,0,0,Gene; miRNA;,Predicted; RNAalifold,28,0,,\"\",0.61567,-3.87526,1600000,179472,171,80,true,-3.55,0.71896,2013-10-04T03:21:19.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02093,mir-2968,1287,mir-2968 microRNA precursor,Moore B,Published; PMID:21627805,70.0,83.5,51.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,55,0,0,Gene; miRNA;,Published; PMID:21627805,56,0,,\"\",0.58523,-3.77609,1600000,216074,207,106,true,-3.1369,0.71868,2013-10-04T03:21:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02094,mir-1803,1287,mir-1803 microRNA precursor,Griffiths-Jones SR,Griffiths-Jones SR,95.0,98.8,94.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,44,0,0,Gene; miRNA;,Predicted; RNAalifold,44,0,,\"\",0.60225,-3.8762,1600000,246996,168,76,true,-3.6416,0.71934,2013-10-04T03:21:22.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02095,mir-2985,1287,mir-2985-2 microRNA precursor,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.2,66.7,\"RNAs can be post-transcriptional modified, in micro RNAs (miRNA) editing of pre-mRNA is a common process. The structure of the mir-2985 has been reported in complex with ADAR protein [3] where they call it R/G site, in the structure, reported in PDB 2L3J, R/G site is reported as a regular A-form helix structure, composed of a stem-loop containing three base-pair mismatches, 2 C:A and 1 G:G. Based on the sequences presented in the SEED alignment, one of the C:A mismatches reported in [3] was updated to a a:u base pair. This family represents the microRNA (miRNA) precursor mir-2985 imported from miRBase [4].\",mir-2985-2;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,337,0,0,Gene; miRNA;,Predicted; RNAalifold,112,0,,\"\",0.65761,-2.44376,1600000,167077,158,71,true,-3.5587,0.71927,2013-10-04T03:21:27.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02097,mir-1662,1287,mir-1662 microRNA precursor,Griffiths-Jones SR,Griffiths-Jones SR,85.0,90.5,84.5,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 90.00 -Z 742849.287494 CM SEQDB,3,50,0,0,Gene; miRNA;,Predicted; RNAalifold,50,0,,\"\",0.61955,-4.2978,1600000,313423,153,66,true,-3.3824,0.71921,2013-10-04T03:21:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02099,rivX,2271,rivX sRNA,Moore B,Published; PMID:18005100,100.0,301.7,51.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; WAR,3,0,,\"\",0.6618,-8.87808,1600000,412635,392,248,true,-3.7266,0.70857,2013-10-04T03:21:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02100,tfoR,1863,Translational regulator of tfoXVC,Moore B,Published; PMID:21317321,70.0,80.1,52.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,42,0,0,Gene; sRNA;,Predicted; WAR,48,0,,\"\",0.70566,-5.13427,1600000,292380,199,101,true,-3.0457,0.71859,2013-10-04T03:21:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02101,HULC,2272,Highly up-regulated in liver cancer conserved region,Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,47.4,44.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 549862.597050 CM SEQDB,19,32,0,0,Gene; lncRNA;,Predicted; CMfinder;,39,0,,\"\",0.29943,-17.02155,1600000,393172,355,241,false,-3.268,0.70889,2013-10-04T03:21:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02102,DISC2,2489,Disrupted in schizophrenia 2 conserved region,Eberhardt R,Predicted; WAR; Eberhardt R,100.0,100.8,83.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 45.21 -Z 549862.597050 CM SEQDB,5,12,0,0,Gene; lncRNA;,Predicted; WAR;,13,0,,\"\",0.3193,-13.66738,1600000,424608,333,205,false,-3.2196,0.7116,2013-10-04T03:21:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02103,DLEU1_1,2464,Deleted in lymphocytic leukemia 1 conserved region 1,Eberhardt R,Predicted; MAFFT; Eberhardt R,30.0,30.1,29.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.65 -Z 549862.597050 CM SEQDB,26,57,0,0,Gene; lncRNA;,Predicted; Pfold;,54,0,,\"\",0.46634,-12.93614,1600000,303335,298,190,false,-3.5022,0.71308,2013-10-04T03:21:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02104,DLEU1_2,2464,Deleted in lymphocytic leukemia 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,100.0,100.0,83.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.10 -Z 549862.597050 CM SEQDB,16,22,0,0,Gene; lncRNA;,Predicted; CMfinder;,27,0,,\"\",0.35834,-18.68736,1600000,476997,484,344,false,-4.1361,0.70416,2013-10-04T03:21:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02105,DLEU2_1,1280,Deleted in lymphocytic leukemia 2 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,52.0,54.0,51.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 41.46 -Z 549862.597050 CM SEQDB,27,50,0,0,Gene; lncRNA;,Predicted; CMfinder;,39,0,,\"\",0.3665,-8.97675,1600000,310022,214,121,false,-2.9079,0.71804,2013-10-04T03:21:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02106,DLEU2_2,1280,Deleted in lymphocytic leukemia 2 conserved region 2,Eberhardt R,Predicted; MAFFT; Eberhardt R,45.0,46.5,42.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.17 -Z 549862.597050 CM SEQDB,18,33,0,0,Gene; lncRNA;,Predicted; Pfold;,34,0,,\"\",0.41953,-8.57013,1600000,270956,175,90,false,-2.8495,0.71843,2013-10-04T03:21:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02107,DLEU2_3,1280,Deleted in lymphocytic leukemia 2 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,60.9,56.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.92 -Z 549862.597050 CM SEQDB,24,37,0,0,Gene; lncRNA;,Predicted; CMfinder;,40,0,,\"\",0.44499,-10.34314,1600000,427311,156,76,false,-3.3664,0.71862,2013-10-04T03:21:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02108,DLEU2_4,1280,Deleted in lymphocytic leukemia 2 conserved region 4,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,40.9,39.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.82 -Z 549862.597050 CM SEQDB,21,47,0,0,Gene; lncRNA;,Predicted; CMfinder;,39,0,,\"\",0.56428,-8.53311,1600000,435172,129,58,false,-3.7329,0.71898,2013-10-04T03:21:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02109,DLEU2_5,1280,Deleted in lymphocytic leukemia 2 conserved region 5,Eberhardt R,Predicted; MAFFT; Eberhardt R,40.0,41.9,38.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.88 -Z 549862.597050 CM SEQDB,30,112,0,0,Gene; lncRNA;,Predicted; RNAalifold;,98,0,,\"\",0.49016,-8.99824,1600000,336788,167,84,false,-3.354,0.71849,2013-02-01T11:59:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02110,DLEU2_6,1280,Deleted in lymphocytic leukemia 2 conserved region 6,Eberhardt R,Predicted; ClustalW; Eberhardt R,60.0,62.6,58.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.89 -Z 549862.597050 CM SEQDB,32,65,0,0,Gene; lncRNA;,Predicted; Pfold;,49,0,,\"\",0.50077,-10.09832,1600000,321471,233,135,false,-2.8787,0.71789,2013-10-04T03:22:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02111,IS009,2254,IS009,Burge SW,Burge SW,117.0,117.2,114.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,12,0,0,Gene;,Predicted; CMFinder; Burge SW,20,0,,\"\",0.61573,-9.79151,1600000,471227,323,198,true,-3.231,0.71243,2013-10-04T03:22:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02112,DLG2-AS1_1,2275,DLG2 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,96.4,75.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 43.12 -Z 549862.597050 CM SEQDB,21,40,0,0,Gene; lncRNA;,Predicted; CMfinder;,46,0,,\"\",0.32556,-14.30722,1600000,382765,336,215,false,-3.0187,0.71077,2013-10-04T03:22:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02113,DLG2-AS1_2,2275,DLG2 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,70.0,86.3,64.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.03 -Z 549862.597050 CM SEQDB,23,36,0,0,Gene; lncRNA;,Predicted; RNAalifold;,46,0,,\"\",0.55247,-13.32678,1600000,510999,395,215,false,-3.5659,0.71076,2013-10-04T03:22:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02114,FAM13A-AS1_1,1276,FAM13A antisense RNA 1 conserved region 1,Eberhardt R,Predicted; WAR; Eberhardt R,70.0,77.9,63.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.29 -Z 549862.597050 CM SEQDB,10,24,0,0,Gene; lncRNA;,Predicted; WAR;,28,0,,\"\",0.47456,-10.3247,1600000,265026,211,118,false,-3.1206,0.71809,2013-10-04T03:22:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02115,FAM13A-AS1_2,1276,FAM13A antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,38.0,39.9,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.96 -Z 549862.597050 CM SEQDB,10,29,0,0,Gene; lncRNA;,Predicted; CMfinder;,32,0,,\"\",0.52171,-11.08543,1600000,428037,208,114,false,-3.137,0.71809,2013-10-04T03:22:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02116,FAS-AS1,2485,FAS antisense RNA 1 conserved region,Eberhardt R,Predicted; ClustalW; Eberhardt R,55.0,59.0,46.0,,SAF,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.20 -Z 549862.597050 CM SEQDB,19,34,0,0,Gene; lncRNA;,Predicted; Pfold;,40,0,,\"\",0.38734,-12.80814,1600000,346766,275,171,false,-3.2045,0.71531,2013-10-04T03:22:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02117,FMR1-AS1_1,2277,FMR1 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,41.6,33.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.58 -Z 549862.597050 CM SEQDB,20,31,0,0,Gene; lncRNA;,Predicted; CMfinder;,35,0,,\"\",0.63532,-8.40418,1600000,545460,130,56,false,-3.8799,0.71902,2013-10-04T03:22:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02118,FMR1-AS1_2,2277,FMR1 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,88.8,61.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 44.76 -Z 549862.597050 CM SEQDB,25,28,0,0,Gene; lncRNA;,Predicted; CMfinder;,34,0,,\"\",0.29519,-20.42184,1600000,660113,232,127,false,-2.8181,0.718,2013-10-04T03:22:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02119,FTX_1,2278,\"FTX transcript, XIST regulator conserved region 1\",Eberhardt R,Predicted; WAR; Eberhardt R,50.0,52.1,36.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.63 -Z 549862.597050 CM SEQDB,11,33,0,0,Gene; lncRNA;,Predicted; WAR;,30,0,,\"\",0.53109,-9.62124,1600000,393060,151,71,false,-3.5358,0.71871,2013-10-04T03:22:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02120,FTX_2,2278,\"FTX transcript, XIST regulator conserved region 2\",Eberhardt R,Predicted; CMfinder; Eberhardt R,48.0,49.8,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.02 -Z 549862.597050 CM SEQDB,18,28,0,0,Gene; lncRNA;,Predicted; CMfinder;,32,0,,\"\",0.51829,-9.85247,1600000,345807,182,97,false,-3.3009,0.71832,2013-10-04T03:22:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02121,FTX_3,2278,\"FTX transcript, XIST regulator conserved region 3\",Eberhardt R,Predicted; CMfinder; Eberhardt R,55.0,56.4,48.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.82 -Z 549862.597050 CM SEQDB,14,28,0,0,Gene; lncRNA;,Predicted; CMfinder;,34,0,,\"\",0.44306,-11.85076,1600000,303135,260,147,false,-2.8504,0.71782,2013-10-04T03:22:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02122,FTX_5,2278,\"FTX transcript, XIST regulator conserved region 5\",Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,48.1,36.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.75 -Z 549862.597050 CM SEQDB,14,36,0,0,Gene; lncRNA;,Predicted; CMfinder;,34,0,,\"\",0.36201,-18.87608,1600000,547217,329,208,false,-3.2439,0.71138,2013-10-04T03:22:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02123,FTX_4,2278,\"FTX transcript, XIST regulator conserved region 4\",Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,96.9,44.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.19 -Z 549862.597050 CM SEQDB,13,25,0,0,Gene; lncRNA;,Predicted; CMfinder;,29,0,,\"\",0.45933,-9.85321,1600000,283609,251,141,false,-3.2946,0.71786,2013-10-04T03:22:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02124,JPX_1,1281,\"JPX transcript, XIST activator conserved region 1\",Eberhardt R,Predicted; ClustalW; Eberhardt R,55.0,59.9,49.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.42 -Z 549862.597050 CM SEQDB,14,34,0,0,Gene; lncRNA;,Predicted; Pfold;,30,0,,\"\",0.46719,-12.91807,1600000,467920,236,135,false,-2.965,0.71789,2013-10-04T03:22:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02125,JPX_2,1281,\"JPX transcript, XIST activator conserved region 2\",Eberhardt R,Predicted; MAFFT; Eberhardt R,48.0,49.1,44.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.63 -Z 549862.597050 CM SEQDB,11,23,0,0,Gene; lncRNA;,Predicted; Pfold;,27,0,,\"\",0.56267,-8.73893,1600000,414246,146,69,false,-3.9155,0.71876,2013-02-01T11:59:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02126,GHRLOS,2279,Ghrelin opposite strand RNA conserved region,Eberhardt R,Predicted; WAR; Eberhardt R,48.0,48.5,40.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.38 -Z 549862.597050 CM SEQDB,16,21,0,0,Gene; lncRNA;,Predicted; WAR;,23,0,,\"\",0.65927,-8.35049,1600000,489057,151,70,false,-3.4835,0.71871,2013-10-04T03:22:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02127,GNAS-AS1_1,1282,GNAS antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,46.2,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.08 -Z 549862.597050 CM SEQDB,21,37,0,0,Gene; lncRNA;,Predicted; CMfinder;,42,0,,\"\",0.44231,-12.28666,1600000,400180,240,105,false,-3.1715,0.71823,2013-10-04T03:22:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02128,GNAS-AS1_2,1282,GNAS antisense RNA 1 conserved region 2,Eberhardt R,Predicted; WAR; Eberhardt R,47.0,48.5,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.25 -Z 549862.597050 CM SEQDB,16,29,0,0,Gene; lncRNA;,Predicted; WAR;,35,0,,\"\",0.41459,-13.3675,1600000,313880,216,119,false,-2.9226,0.71804,2013-10-04T03:22:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02129,GNAS-AS1_3,1282,GNAS antisense RNA 1 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,42.0,43.7,41.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.77 -Z 549862.597050 CM SEQDB,9,24,0,0,Gene; lncRNA;,Predicted; CMfinder;,24,0,,\"\",0.64157,-8.50991,1600000,794926,102,38,false,-3.9084,0.72207,2013-10-04T03:22:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02130,GNAS-AS1_4,1282,GNAS antisense RNA 1 conserved region 4,Eberhardt R,Predicted; ClustalW; Eberhardt R,50.0,51.5,43.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.70 -Z 549862.597050 CM SEQDB,18,42,0,0,Gene; lncRNA;,Predicted; RNAalifold;,42,0,,\"\",0.37787,-15.15187,1600000,441668,220,122,false,-2.9371,0.718,2013-10-04T03:22:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02131,GNAS-AS1_5,1282,GNAS antisense RNA 1 conserved region 5,Eberhardt R,Predicted; MAFFT; Eberhardt R,70.0,77.5,43.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.12 -Z 549862.597050 CM SEQDB,22,38,0,0,Gene; lncRNA;,Predicted; RNAalifold;,45,0,,\"\",0.33464,-16.16846,1600000,441186,226,124,false,-2.7775,0.71799,2013-02-01T11:59:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02132,HOXB13-AS1_1,2496,HOXB13 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,65.3,57.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.73 -Z 549862.597050 CM SEQDB,12,26,0,0,Gene; lncRNA;,Predicted; CMfinder;,32,0,,\"\",0.53291,-11.17499,1600000,348540,301,136,false,-3.4125,0.7179,2013-10-04T03:22:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02133,HOXB13-AS1_2,2496,HOXB13 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,40.0,43.0,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.77 -Z 549862.597050 CM SEQDB,14,29,0,0,Gene; lncRNA;,Predicted; RNAalifold;,35,0,,\"\",0.6146,-8.95746,1600000,462019,161,81,false,-3.6075,0.71855,2013-10-04T03:22:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02134,HTT-AS1_1,2492,HTT antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,200.0,241.1,57.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.26 -Z 549862.597050 CM SEQDB,3,9,0,0,Gene; lncRNA;,Predicted; CMfinder;,8,0,,\"\",0.40132,-17.93047,1600000,488095,366,230,false,-3.4753,0.7096,2013-10-04T03:22:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02135,HTT-AS1_2,2492,HTT antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,100.0,126.7,37.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.30 -Z 549862.597050 CM SEQDB,3,8,0,0,Gene; lncRNA;,Predicted; CMfinder;,8,0,,\"\",0.39673,-14.6161,1600000,352178,303,175,false,-3.3963,0.71483,2013-10-04T03:22:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02136,HTT-AS1_3,2492,HTT antisense RNA 1 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,70.0,76.7,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.50 -Z 549862.597050 CM SEQDB,4,7,0,0,Gene; lncRNA;,Predicted; CMfinder;,8,0,,\"\",0.52708,-10.64027,1600000,544149,138,60,false,-3.9688,0.71892,2013-10-04T03:22:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02137,HOXA11-AS1_1,2483,HOXA11 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,50.9,32.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.28 -Z 549862.597050 CM SEQDB,15,37,0,0,Gene; lncRNA;,Predicted; CMfinder;,44,0,,\"\",0.48644,-10.43061,1600000,269104,189,96,false,-3.3494,0.71834,2013-10-04T03:22:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02138,HOXA11-AS1_2,2483,HOXA11 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,66.6,41.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.35 -Z 549862.597050 CM SEQDB,21,88,0,0,Gene; lncRNA;,Predicted; CMfinder;,94,0,,\"\",0.3817,-16.61947,1600000,382228,304,187,false,-3.465,0.7134,2013-10-04T03:22:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02139,HOXA11-AS1_3,2483,HOXA11 antisense RNA 1 conserved region 3,Eberhardt R,Predicted; MAFFT; Eberhardt R,50.0,62.1,34.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.69 -Z 549862.597050 CM SEQDB,24,36,0,0,Gene; lncRNA;,Predicted; RNAalifold;,46,0,,\"\",0.46412,-11.63887,1600000,391344,189,98,false,-3.2851,0.71831,2013-10-04T03:22:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02140,HOXA11-AS1_4,2483,HOXA11 antisense RNA 1 conserved region 4,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,87.9,66.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 45.48 -Z 549862.597050 CM SEQDB,20,85,0,0,Gene; lncRNA;,Predicted; CMfinder;,94,0,,\"\",0.31145,-14.60412,1600000,389554,361,233,false,-3.3812,0.70946,2013-10-04T03:22:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02141,HOXA11-AS1_5,2483,HOXA11 antisense RNA 1 conserved region 5,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,53.2,49.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.96 -Z 549862.597050 CM SEQDB,28,86,0,0,Gene; lncRNA;,Predicted; CMfinder;,95,0,,\"\",0.42522,-16.49823,1600000,472263,445,203,false,-3.5513,0.7118,2013-10-04T03:22:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02142,HOXA11-AS1_6,2483,HOXA11 antisense RNA 1 conserved region 6,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,57.2,38.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.78 -Z 549862.597050 CM SEQDB,21,37,0,0,Gene; lncRNA;,Predicted; CMfinder;,46,0,,\"\",0.50509,-12.88637,1600000,321322,368,179,false,-3.8307,0.71429,2013-10-04T03:23:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02143,HYMAI,2283,Hydatidiform mole associated and imprinted conserved region 1,Eberhardt R,Predicted; ClustalW; Eberhardt R,80.0,112.6,52.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.46 -Z 549862.597050 CM SEQDB,16,36,0,0,Gene; lncRNA;,Predicted; RNAalifold;,39,0,,\"\",0.51994,-11.81493,1600000,452975,394,242,false,-3.8378,0.70879,2013-10-04T03:23:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02144,rsmX,2284,rsmX,\"Moll S, Schneider DJ, Stodghill P, Myers CR, Cartinhour SW, Filiatrault M\",\"Moll S, Schneider DJ, Stodghill P, Myers CR, Cartinhour SW, Filiatrault M\",56.0,66.5,52.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,96,0,0,Gene;,Predicted;,53,0,,\"\",0.51911,-9.86352,1600000,341381,215,114,true,-3.0959,0.71868,2013-10-04T03:23:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02145,MEG8_1,2470,Maternally expressed 8 exon 1,Eberhardt R,Predicted; ClustalW; Eberhardt R,70.0,76.8,35.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.80 -Z 549862.597050 CM SEQDB,23,37,0,0,Gene; lncRNA;,Predicted; RNAalifold;,45,0,,\"\",0.65514,-7.32135,1600000,291727,186,97,false,-3.6286,0.71832,2013-10-04T03:23:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02146,MEG8_2,2470,Maternally expressed 8 exon 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,46.3,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.10 -Z 549862.597050 CM SEQDB,12,26,0,0,Gene; lncRNA;,Predicted; CMfinder;,28,0,,\"\",0.42391,-13.76496,1600000,529730,218,111,false,-3.3536,0.71815,2013-10-04T03:23:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02147,MEG8_3,2470,Maternally expressed 8 exon 3,Eberhardt R,Predicted; ClustalW; Eberhardt R,40.0,41.2,37.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.26 -Z 549862.597050 CM SEQDB,10,30,0,0,Gene; lncRNA;,Predicted; RNAalifold;,32,0,,\"\",0.66007,-8.54879,1600000,527412,143,66,false,-3.653,0.71879,2013-10-04T03:23:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02148,MESTIT1_1,2286,\"MEST intronic transcript 1, antisense RNA conserved region 1\",Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,54.6,47.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.38 -Z 549862.597050 CM SEQDB,16,33,0,0,Gene; lncRNA;,Predicted; CMfinder;,41,0,,\"\",0.39518,-12.53958,1600000,328188,296,144,false,-3.6087,0.71782,2013-10-04T03:23:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02149,MESTIT1_2,2286,\"MEST intronic transcript 1, antisense RNA conserved region 2\",Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,133.9,55.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.92 -Z 549862.597050 CM SEQDB,16,37,0,0,Gene; lncRNA;,Predicted; CMfinder;,42,0,,\"\",0.40444,-15.9659,1600000,333596,371,235,false,-3.817,0.70927,2013-10-04T03:23:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02150,MESTIT1_3,2286,\"MEST intronic transcript 1, antisense RNA conserved region 3\",Eberhardt R,Predicted; MAFFT; Eberhardt R,60.0,66.1,36.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.85 -Z 549862.597050 CM SEQDB,18,34,0,0,Gene; lncRNA;,Predicted; RNAalifold;,40,0,,\"\",0.55355,-9.60152,1600000,319862,212,99,false,-3.4903,0.71827,2013-10-04T03:23:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02151,MIMT1_1,2465,MER1 repeat containing imprinted transcript 1 conserved region 1,Eberhardt R,Eberhardt R,45.0,47.8,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.86 -Z 549862.597050 CM SEQDB,21,26,0,0,Gene; lncRNA;,Predicted; RNAz; Eberhardt R,35,0,,\"\",0.50084,-11.0283,1600000,307724,195,88,false,-3.5356,0.71845,2013-10-04T03:23:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02153,NBR2,2288,Neighbour of BRCA1 gene 2 converved region,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,40.5,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.04 -Z 549862.597050 CM SEQDB,26,111,0,0,Gene; lncRNA;,Predicted; CMfinder;,104,0,,\"\",0.44428,-10.09073,1600000,391580,143,63,false,-3.5071,0.71888,2013-10-04T03:23:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02159,PART1_1,2473,Prostate androgen-regulated transcript 1 conserved region 1,Eberhardt R,Predicted; ClustalW; Eberhardt R,65.0,70.5,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.48 -Z 549862.597050 CM SEQDB,22,38,0,0,Gene; lncRNA;,Predicted; RNAalifold;,46,0,,\"\",0.38467,-14.19069,1600000,393479,410,218,false,-3.5551,0.71054,2013-10-04T03:23:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02160,PART1_2,2473,Prostate androgen-regulated transcript 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,55.0,57.1,48.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.90 -Z 549862.597050 CM SEQDB,29,92,0,0,Gene; lncRNA;,Predicted; CMfinder;,102,0,,\"\",0.50012,-15.21822,1600000,335382,509,249,false,-3.7161,0.70835,2013-10-04T03:23:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02161,PART1_3,2473,Prostate androgen-regulated transcript 1 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,43.5,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.59 -Z 549862.597050 CM SEQDB,3,24,0,0,Gene; lncRNA;,Predicted; CMfinder;,24,0,,\"\",0.57968,-9.31604,1600000,313098,350,131,false,-3.1729,0.71793,2013-10-04T03:23:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02162,XIST_A_REPEAT,2469,XIST RNA A repeat,Burge SW,\"Duszczyk MM, Wutz A, Rybin V, Sattler M, Burge SW;\",33.0,33.1,32.9,,XIST,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,54,240,0,0,Gene; sRNA;,Predicted; CMFinder;Burge SW,55,0,,\"\",0.80466,-2.77402,1600000,570602,90,37,true,-4.1939,0.72046,2013-10-04T03:23:41.000Z,2022-05-06T13:10:07.000Z\n",
            "RF02163,sR-tMet,2293,Small nucleolar RNA sR-tMet,Eberhardt R,Predicted; PMID:21654217; Eberhardt R,50.0,53.5,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.18 -Z 549862.597050 CM SEQDB,2,70,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; PMID:21654217;,68,0,,\"\",0.58403,-4.45193,1600000,307160,162,72,true,-2.9069,0.72424,2013-10-04T03:23:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02164,PVT1_1,1283,Pvt1 oncogene conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,85.8,79.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 61.36 -Z 549862.597050 CM SEQDB,17,29,0,0,Gene; lncRNA;,Predicted; CMfinder;,32,0,,\"\",0.22976,-26.19468,1600000,611351,291,178,false,-2.9522,0.71447,2013-10-04T03:23:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02165,PVT1_2,1283,Pvt1 oncogene conserved region 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,47.0,48.4,40.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.62 -Z 549862.597050 CM SEQDB,17,36,0,0,Gene; lncRNA;,Predicted; RNAalifold;,41,0,,\"\",0.40016,-16.58049,1600000,464627,338,173,false,-3.1358,0.71506,2013-10-04T03:23:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02166,PVT1_3,1283,Pvt1 oncogene conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,46.0,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.64 -Z 549862.597050 CM SEQDB,25,53,0,0,Gene; lncRNA;,Predicted; CMfinder;,47,0,,\"\",0.64656,-9.29503,1600000,388106,174,95,false,-3.765,0.71834,2013-10-04T03:23:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02167,PVT1_4,1283,Pvt1 oncogene conserved region 4,Eberhardt R,Predicted; WAR; Eberhardt R,60.0,64.1,49.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.42 -Z 549862.597050 CM SEQDB,10,21,0,0,Gene; lncRNA;,Predicted; WAR;,24,0,,\"\",0.42822,-14.52079,1600000,432665,292,173,false,-3.1521,0.71504,2013-10-04T03:23:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02168,PVT1_5,1283,Pvt1 oncogene conserved region 5,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,63.4,56.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.08 -Z 549862.597050 CM SEQDB,23,33,0,0,Gene; lncRNA;,Predicted; CMfinder;,41,0,,\"\",0.4574,-11.75465,1600000,376551,226,133,false,-3.5262,0.71791,2013-02-01T11:59:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02169,PVT1_6,1283,Pvt1 oncogene conserved region 6,Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,47.4,39.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.58 -Z 549862.597050 CM SEQDB,7,27,0,0,Gene; lncRNA;,Predicted; CMfinder;,30,0,,\"\",0.47652,-11.06521,1600000,289926,208,113,false,-3.31,0.71815,2013-02-01T11:59:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02170,PVT1_7,1283,Pvt1 oncogene conserved region 7,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,93.3,77.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.77 -Z 549862.597050 CM SEQDB,8,16,0,0,Gene; lncRNA;,Predicted; CMfinder;,16,0,,\"\",0.48789,-12.28708,1600000,337415,238,137,false,-3.1977,0.71787,2013-10-04T03:23:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02171,RFPL3-AS1_1,2474,RFPL3 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,70.0,109.7,53.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.66 -Z 549862.597050 CM SEQDB,4,14,0,0,Gene; lncRNA;,Predicted; CMfinder;,7,0,,\"\",0.64602,-7.95432,1600000,311212,181,88,false,-3.5612,0.71844,2013-10-04T03:23:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02172,RFPL3-AS1_2,2474,RFPL3 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,41.2,39.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.84 -Z 549862.597050 CM SEQDB,3,14,0,0,Gene; lncRNA;,Predicted; CMfinder;,10,0,,\"\",0.67452,-7.88425,1600000,384885,155,71,false,-3.6569,0.71873,2013-10-04T03:24:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02173,SMAD5-AS1_1,2478,SMAD5 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; ClustalW; Eberhardt R,45.0,47.8,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.96 -Z 549862.597050 CM SEQDB,18,36,0,0,Gene; lncRNA;,Predicted; RNAalifold;,42,0,,\"\",0.52259,-14.15157,1600000,456385,389,263,false,-3.7172,0.70757,2013-10-04T03:24:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02174,SMAD5-AS1_2,2478,SMAD5 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,50.0,51.4,47.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.89 -Z 549862.597050 CM SEQDB,20,35,0,0,Gene; lncRNA;,Predicted; RNAalifold;,46,0,,\"\",0.58902,-11.17248,1600000,461837,226,129,false,-3.546,0.71795,2013-10-04T03:24:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02175,SMAD5-AS1_3,2478,SMAD5 antisense RNA 1 conserved region 3,Eberhardt R,Predicted; ClustalW; Eberhardt R,60.0,69.0,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.42 -Z 549862.597050 CM SEQDB,18,34,0,0,Gene; lncRNA;,Predicted; RNAalifold;,44,0,,\"\",0.57491,-9.97352,1600000,356532,225,125,false,-3.0535,0.71798,2013-10-04T03:24:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02176,SMAD5-AS1_4,2478,SMAD5 antisense RNA 1 conserved region 4,Eberhardt R,Predicted; ClustalW; Eberhardt R,50.0,64.5,34.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.36 -Z 549862.597050 CM SEQDB,19,34,0,0,Gene; lncRNA;,Predicted; RNAalifold;,43,0,,\"\",0.40557,-14.63199,1600000,366472,288,173,false,-3.3924,0.71506,2013-10-04T03:24:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02177,SMCR2_1,2493,\"Smith-Magenis syndrome chromosome region, candidate 2 conserved region 1\",Eberhardt R,Predicted; ClustalW; Eberhardt R,50.0,51.6,47.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.51 -Z 549862.597050 CM SEQDB,6,22,0,0,Gene; lncRNA;,Predicted; RNAalifold;,21,0,,\"\",0.45122,-14.35178,1600000,463982,301,159,false,-2.9892,0.71698,2013-10-04T03:24:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02178,SMCR2_2,2493,\"Smith-Magenis syndrome chromosome region, candidate 2 conserved region 2\",Eberhardt R,Predicted; ClustalW; Eberhardt R,55.0,56.5,50.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.01 -Z 549862.597050 CM SEQDB,5,18,0,0,Gene; lncRNA;,Predicted; RNAalifold;,19,0,,\"\",0.45984,-12.74123,1600000,253017,279,165,false,-3.0855,0.71615,2013-10-04T03:24:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02179,ST7-AS1_1,2490,ST7 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,46.1,42.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.60 -Z 549862.597050 CM SEQDB,25,31,0,0,Gene; lncRNA;,Predicted; CMfinder;,45,0,,\"\",0.40115,-10.06612,1600000,262811,254,117,false,-3.0441,0.71808,2013-10-04T03:24:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02180,ST7-AS1_2,2490,ST7 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,55.7,40.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.67 -Z 549862.597050 CM SEQDB,24,30,0,0,Gene; lncRNA;,Predicted; CMfinder;,41,0,,\"\",0.58187,-9.27442,1600000,344332,202,110,false,-3.2818,0.71815,2013-10-04T03:24:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02181,ST7-AS2_1,2471,ST7 antisense RNA 2 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,94.1,61.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.74 -Z 549862.597050 CM SEQDB,4,9,0,0,Gene; lncRNA;,Predicted; CMfinder;,11,0,,\"\",0.54393,-9.38599,1600000,334990,157,72,false,-3.5324,0.7187,2013-10-04T03:24:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02182,ST7-AS2_2,2471,ST7 antisense RNA 2 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,50.8,43.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.34 -Z 549862.597050 CM SEQDB,8,13,0,0,Gene; lncRNA;,Predicted; CMfinder;,17,0,,\"\",0.63919,-8.68297,1600000,629601,115,47,false,-4.0171,0.71927,2013-10-04T03:24:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02183,ST7-OT3_1,2475,ST7 overlapping transcript 3 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,54.6,40.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.63 -Z 549862.597050 CM SEQDB,28,36,0,0,Gene; lncRNA;,Predicted; CMfinder;,49,0,,\"\",0.47415,-9.66229,1600000,358358,160,79,false,-3.303,0.71858,2013-10-04T03:24:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02184,ST7-OT3_2,2475,ST7 overlapping transcript 3 conserved region 2,Eberhardt R,Predicted; WAR; Eberhardt R,100.0,139.9,54.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.52 -Z 549862.597050 CM SEQDB,5,93,0,0,Gene; lncRNA;,Predicted; WAR;,92,0,,\"\",0.45074,-12.85599,1600000,365187,250,138,false,-3.0815,0.71789,2013-10-04T03:24:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02185,ST7-OT3_3,2475,ST7 overlapping transcript 3 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,58.0,40.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.33 -Z 549862.597050 CM SEQDB,21,35,0,0,Gene; lncRNA;,Predicted; CMfinder;,46,0,,\"\",0.42804,-10.63669,1600000,282807,208,116,false,-3.2457,0.71809,2013-10-04T03:24:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02186,ST7-OT3_4,2475,ST7 overlapping transcript 3 conserved region 4,Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,92.9,41.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.08 -Z 549862.597050 CM SEQDB,8,9,0,0,Gene; lncRNA;,Predicted; CMfinder;,14,0,,\"\",0.6218,-8.89947,1600000,363661,158,73,false,-3.7961,0.71871,2013-02-01T11:59:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02187,ST7-OT4_1,2477,ST7 overlapping transcript 4 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,53.2,49.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.67 -Z 549862.597050 CM SEQDB,24,31,0,0,Gene; lncRNA;,Predicted; CMfinder;,44,0,,\"\",0.42178,-13.52759,1600000,237812,321,201,false,-3.3871,0.71201,2013-10-04T03:24:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02188,ST7-OT4_2,2477,ST7 overlapping transcript 4 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,55.0,62.8,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.30 -Z 549862.597050 CM SEQDB,29,35,0,0,Gene; lncRNA;,Predicted; CMfinder;,49,0,,\"\",0.44962,-12.75432,1600000,301958,277,172,false,-3.4429,0.71519,2013-10-04T03:24:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02189,ST7-OT4_3,2477,ST7 overlapping transcript 4 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,51.0,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.91 -Z 549862.597050 CM SEQDB,20,36,0,0,Gene; lncRNA;,Predicted; CMfinder;,45,0,,\"\",0.56152,-12.24799,1600000,621026,506,302,false,-4.0564,0.70571,2013-10-04T03:24:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02190,ST7-OT4_4,2477,ST7 overlapping transcript 4 conserved region 4,Eberhardt R,Predicted; CMfinder; Eberhardt R,70.0,76.2,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.00 -Z 549862.597050 CM SEQDB,26,31,0,0,Gene; lncRNA;,Predicted; CMfinder;,41,0,,\"\",0.32685,-14.94272,1600000,485002,356,211,false,-3.4734,0.71109,2013-10-04T03:24:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02191,TCL6_1,2466,T-cell leukemia/lymphoma 6 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,70.1,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.23 -Z 549862.597050 CM SEQDB,9,21,0,0,Gene; lncRNA;,Predicted; CMfinder;,22,0,,\"\",0.55382,-10.24761,1600000,328267,211,113,false,-3.4278,0.71809,2013-10-04T03:24:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02192,TCL6_2,2466,T-cell leukemia/lymphoma 6 conserved region 2,Eberhardt R,Predicted; ClustalW; Eberhardt R,45.0,47.0,37.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.00 -Z 549862.597050 CM SEQDB,12,27,0,0,Gene; lncRNA;,Predicted; RNAalifold;,29,0,,\"\",0.48119,-11.09205,1600000,370119,222,126,false,-3.4551,0.71797,2013-10-04T03:24:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02193,TCL6_3,2466,T-cell leukemia/lymphoma 6 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,40.0,43.9,38.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.67 -Z 549862.597050 CM SEQDB,26,29,0,0,Gene; lncRNA;,Predicted; CMfinder;,33,0,,\"\",0.52807,-11.9398,1600000,426161,244,139,false,-3.6069,0.71787,2013-10-04T03:24:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02194,HPnc0260,2301,Bacterial antisense RNA HPnc0260,Burge SW,Burge SW,59.0,59.0,58.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,31,147,0,0,Gene; antisense;,Predicted; CMFinder,166,0,,\"\",0.42624,-4.60077,1600000,310719,254,156,true,-2.5915,0.71767,2013-10-04T03:24:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02195,TP53TG1_1,2302,TP53 target 1 conserved region 1,Eberhardt R,Predicted; ClustalW,40.0,44.9,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.18 -Z 549862.597050 CM SEQDB,16,30,0,0,Gene; lncRNA;,Predicted; RNAalifold,34,0,,\"\",0.48126,-12.7035,1600000,390725,257,153,false,-3.0113,0.71772,2013-10-04T03:24:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02196,TP53TG1_2,2302,TP53 target 1 conserved region 2,Eberhardt R,Predicted; ClustalW,75.0,78.9,67.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 48.22 -Z 549862.597050 CM SEQDB,12,19,0,0,Gene; lncRNA;,Predicted; RNAalifold,22,0,,\"\",0.26873,-23.16176,1600000,623501,317,186,false,-2.9384,0.71354,2013-10-04T03:24:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02198,TTC28-AS1_1,1276,TTC28 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder,70.0,73.7,67.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 54.33 -Z 549862.597050 CM SEQDB,14,23,0,0,Gene; lncRNA;,Predicted; CMfinder,28,0,,\"\",0.24131,-23.81874,1600000,450728,267,162,false,-3.0533,0.71651,2013-10-04T03:24:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02199,TTC28-AS1_2,1276,TTC28 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder,50.0,52.7,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 39.12 -Z 549862.597050 CM SEQDB,8,21,0,0,Gene; lncRNA;,Predicted; CMfinder,26,0,,\"\",0.35333,-13.89345,1600000,397119,264,156,false,-2.9548,0.71748,2013-10-04T03:24:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02200,TTC28-AS1_3,1276,TTC28 antisense RNA 1 conserved region 3,Eberhardt R,Predicted; ClustalW,55.0,56.5,51.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.02 -Z 549862.597050 CM SEQDB,6,19,0,0,Gene; lncRNA;,Predicted; Pfold,21,0,,\"\",0.3623,-18.94697,1600000,627612,301,184,false,-2.941,0.71376,2013-10-04T03:24:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02201,TTC28-AS1_4,1276,TTC28 antisense RNA 1 conserved region 4,Eberhardt R,Predicted; MAFFT,45.0,48.8,40.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.96 -Z 549862.597050 CM SEQDB,18,34,0,0,Gene; lncRNA;,Predicted; RNAalifold,41,0,,\"\",0.45089,-11.59669,1600000,254922,206,113,false,-3.2967,0.7181,2013-10-04T03:25:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02203,WT1-AS_1,2494,WT1 antisense RNA conserved region 1,Eberhardt R,Predicted; ClustalW,40.0,43.4,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.85 -Z 549862.597050 CM SEQDB,10,29,0,0,Gene; lncRNA;,Predicted; RNAalifold,29,0,,\"\",0.38113,-14.08789,1600000,366336,207,109,false,-3.0664,0.7182,2013-10-04T03:25:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02204,WT1-AS_2,2494,WT1 antisense RNA conserved region 2,Eberhardt R,Predicted; CMfinder,45.0,49.8,37.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.87 -Z 549862.597050 CM SEQDB,18,28,0,0,Gene; lncRNA;,Predicted; CMfinder,32,0,,\"\",0.48607,-12.50045,1600000,366693,238,137,false,-3.0697,0.71788,2013-10-04T03:25:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02205,WT1-AS_3,2494,WT1 antisense RNA conserved region 3,Eberhardt R,Predicted; CMfinder,55.0,66.7,49.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.78 -Z 549862.597050 CM SEQDB,19,36,0,0,Gene; lncRNA;,Predicted; CMfinder,42,0,,\"\",0.33828,-20.10321,1600000,472204,367,245,false,-3.5306,0.70864,2013-10-04T03:25:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02206,WT1-AS_4,2494,WT1 antisense RNA conserved region 4,Eberhardt R,Predicted; MAFFT,60.0,62.7,51.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.67 -Z 549862.597050 CM SEQDB,12,26,0,0,Gene; lncRNA;,Predicted; RNAalifold,28,0,,\"\",0.50346,-11.16612,1600000,330236,233,134,false,-3.2321,0.7179,2013-10-04T03:25:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02207,WT1-AS_5,2494,WT1 antisense RNA conserved region 5,Eberhardt R,Predicted; CMfinder,80.0,89.0,47.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.44 -Z 549862.597050 CM SEQDB,4,9,0,0,Gene; lncRNA;,Predicted; CMfinder,9,0,,\"\",0.61471,-8.73601,1600000,331352,171,79,false,-3.5046,0.71862,2013-10-04T03:25:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02208,WT1-AS_6,2494,WT1 antisense RNA conserved region 6,Eberhardt R,Predicted; MAFFT,80.0,83.9,58.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 41.00 -Z 549862.597050 CM SEQDB,19,33,0,0,Gene; lncRNA;,Predicted; RNAalifold,40,0,,\"\",0.32994,-15.99776,1600000,428161,220,121,false,-3.25,0.71803,2013-10-04T03:25:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02209,WT1-AS_7,2494,WT1 antisense RNA conserved region 7,Eberhardt R,Predicted; ClustalW,50.0,58.2,45.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.18 -Z 549862.597050 CM SEQDB,21,65,0,0,Gene; lncRNA;,Predicted; RNAalifold,66,0,,\"\",0.57386,-13.61699,1600000,434036,433,294,false,-4.5104,0.70604,2013-10-04T03:25:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02210,WT1-AS_8,2494,WT1 antisense RNA conserved region 8,Eberhardt R,Predicted; CMfinder,70.0,80.4,58.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 48.44 -Z 549862.597050 CM SEQDB,24,39,0,0,Gene; lncRNA;,Predicted; CMfinder,49,0,,\"\",0.27447,-19.95755,1600000,413523,531,281,false,-3.5984,0.70662,2013-10-04T03:25:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02211,ZFAT-AS1_1,2480,ZFAT antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder,80.0,83.1,37.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.30 -Z 549862.597050 CM SEQDB,3,9,0,0,Gene; lncRNA;,Predicted; CMfinder,9,0,,\"\",0.54377,-9.88493,1600000,343828,169,79,false,-3.2409,0.71854,2013-10-04T03:25:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02212,ZFAT-AS1_2,2480,ZFAT antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder,70.0,72.3,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.81 -Z 549862.597050 CM SEQDB,11,23,0,0,Gene; lncRNA;,Predicted; CMfinder,27,0,,\"\",0.41975,-14.99021,1600000,280860,509,205,false,-3.5842,0.71162,2013-10-04T03:25:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02213,ZFAT-AS1_3,2480,ZFAT antisense RNA 1 conserved region 3,Eberhardt R,Predicted; ClustalW,70.0,77.3,57.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.82 -Z 549862.597050 CM SEQDB,5,9,0,0,Gene; lncRNA;,Predicted; RNAalifold,11,0,,\"\",0.58505,-9.08473,1600000,372758,168,80,false,-3.5322,0.71857,2013-10-04T03:25:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02214,mir-56,1287,microRNA mir-56,Griffiths-Jones SR,Griffiths-Jones SR,80.0,88.9,70.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60821,-4.43056,1600000,261515,153,65,true,-2.8661,0.72801,2013-10-04T03:25:28.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02215,ZNFX1-AS1_1,2484,ZNFX1 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; ClustalW,70.0,78.0,62.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.54 -Z 549862.597050 CM SEQDB,15,33,0,0,Gene; lncRNA;,Predicted; RNAalifold,37,0,,\"\",0.31266,-20.51619,1600000,568016,317,198,false,-3.1789,0.7123,2013-10-04T03:25:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02216,ZNFX1-AS1_2,2484,ZNFX1 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder,45.0,48.9,42.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.93 -Z 549862.597050 CM SEQDB,18,59,0,0,Gene; lncRNA;,Predicted; CMfinder,39,0,,\"\",0.61669,-9.34536,1600000,374517,173,93,false,-3.8752,0.71838,2013-10-04T03:25:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02217,ZNFX1-AS1_3,2484,ZNFX1 antisense RNA 1 conserved region 3,Eberhardt R,Predicted; CMfinder,60.0,71.7,44.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.64 -Z 549862.597050 CM SEQDB,3,9,0,0,Gene; lncRNA;,Predicted; CMfinder,9,0,,\"\",0.45703,-9.64962,1600000,288818,180,87,false,-3.2161,0.71848,2013-10-04T03:25:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02218,ZNRD1-AS1_1,2468,ZNRD1 antisense RNA 1 conserved region 1,Eberhardt R,Predicted; ClustalW,50.0,50.0,43.8,,HTEX4,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.32 -Z 549862.597050 CM SEQDB,8,36,0,0,Gene; lncRNA;,Predicted; RNAalifold,13,0,,\"\",0.57311,-8.6466,1600000,466169,148,68,false,-3.6282,0.71877,2013-10-04T03:25:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02219,ZNRD1-AS1_2,2468,ZNRD1 antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder,60.0,62.8,53.5,,HTEX4,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.89 -Z 549862.597050 CM SEQDB,34,93,0,0,Gene; lncRNA;,Predicted; CMfinder,50,0,,\"\",0.6536,-7.98545,1600000,457398,155,76,false,-3.7479,0.71861,2013-10-04T03:25:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02220,ZNRD1-AS1_3,2468,ZNRD1 antisense RNA 1 conserved region 3,Eberhardt R,Predicted; ClustalW,60.0,74.7,54.3,,HTEX4,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.33 -Z 549862.597050 CM SEQDB,9,15,0,0,Gene; lncRNA;,Predicted; RNAalifold,14,0,,\"\",0.5781,-8.7197,1600000,324485,202,104,false,-2.8945,0.71817,2013-10-04T03:25:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02221,sRNA-Xcc1,1285,sRNA-Xcc1,\"Gardner PP, Daub J\",Published; PMID:21941121,53.0,53.0,52.8,,sX1; sX10,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,74,337,0,0,Gene; sRNA;,Predicted; RNAalifold,243,0,,\"\",0.54673,-4.89956,1600000,209664,168,87,true,-3.6231,0.71876,2013-10-04T03:25:42.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02222,sX2,1285,Xanthomonas sRNA sX2,Eberhardt R,Predicted; CMfinder,40.0,40.4,31.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,5,0,0,Gene; sRNA;,Predicted; CMfinder,12,0,,\"\",0.59307,-8.02656,1600000,355777,213,110,true,-3.0439,0.71854,2013-10-04T03:25:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02223,sX4,1285,Proteobacterial sRNA sX4,Eberhardt R,Predicted; ClustalW,70.0,70.2,65.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,30,0,0,Gene; sRNA;,Predicted; RNAalifold,34,0,,\"\",0.56835,-9.71048,1600000,382066,289,167,true,-3.2186,0.71632,2013-02-01T11:59:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02224,sX5,1285,Xanthomonadaceae sRNA sX5,Eberhardt R,Predicted; MAFFT,50.0,61.6,45.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,19,0,0,Gene; sRNA;,Predicted; RNAalifold,27,0,,\"\",0.68125,-4.22199,1600000,279256,125,49,true,-3.9498,0.71948,2013-02-01T11:59:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02225,sX6,1285,Proteobacterial sRNA sX6,Eberhardt R,Predicted; MAFFT,40.0,42.0,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,21,0,0,Gene; sRNA;,Predicted; RNAalifold,26,0,,\"\",0.65279,-9.68291,1600000,373421,500,342,true,-3.787,0.70436,2013-10-04T03:26:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02226,sX7,1285,Xanthomonas sRNA sX7,Eberhardt R,Predicted; MAFFT,55.0,57.2,47.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,9,0,0,Gene; sRNA;,Predicted; RNAalifold,15,0,,\"\",0.55957,-7.64532,1600000,269555,195,95,true,-3.1025,0.71873,2013-10-04T03:26:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02227,sX8,1285,Proteobacterial sRNA sX8,Eberhardt R,Predicted; CMfinder,50.0,72.5,33.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,35,0,0,Gene; sRNA;,Predicted; CMfinder,36,0,,\"\",0.57443,-7.17871,1600000,329765,171,82,true,-3.6048,0.71889,2013-10-04T03:26:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02228,sX9,1285,Xanthomonas sRNA sX9,Eberhardt R,Predicted; ClustalW,44.0,44.1,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,378,0,0,Gene; sRNA;,Predicted; RNAalifold,45,0,,\"\",0.51485,-9.79002,1600000,409423,157,76,true,-3.3607,0.71885,2013-10-04T03:26:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02230,sX11,1285,Proteobacterial sRNA sX11,Eberhardt R,Predicted; CMfinder,70.0,71.5,38.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,15,0,0,Gene; sRNA;,Predicted; CMfinder,23,0,,\"\",0.52295,-9.65346,1600000,343960,253,138,true,-2.9274,0.71815,2013-10-04T03:26:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02231,sX12,1285,Xanthomonas sRNA sX12,Eberhardt R,Predicted; CMfinder,106.0,106.0,38.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 742849.287494 CM SEQDB,8,0,0,0,Gene; sRNA;,Predicted; CMfinder,8,0,,\"\",0.51296,-8.36483,1600000,265966,170,78,true,-3.5252,0.71893,2013-10-04T03:26:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02232,sX13,1285,Xanthomonadaceae sRNA sX13,Eberhardt R,Predicted; CMfinder,50.0,66.7,41.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,49,0,0,Gene; sRNA;,Predicted; CMfinder,57,0,,\"\",0.53196,-9.3539,1600000,316938,219,115,true,-2.8757,0.71849,2013-02-01T11:59:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02233,sX14,1285,Xanthomonas sRNA sX14/Xoo3,Eberhardt R,Predicted; CMfinder,80.0,106.1,53.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,3,0,0,Gene; sRNA;,Predicted; CMfinder,9,0,,\"\",0.52319,-9.57593,1600000,360200,190,92,true,-3.4036,0.71882,2013-10-04T03:26:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02234,sX15,1285,Xanthomonadaceae sRNA sX15,Eberhardt R,Predicted; CMfinder,55.0,56.0,51.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,24,0,0,Gene; sRNA;,Predicted; CMfinder,14,0,,\"\",0.48207,-9.0108,1600000,220071,277,159,true,-2.9046,0.71747,2013-10-04T03:26:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02235,asX1,1285,Xanthomonadaceae sRNA asX1,Eberhardt R,Published; PMID:22080557,70.0,81.4,49.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,49,0,0,Gene; sRNA;,Predicted; CMfinder,58,0,,\"\",0.43863,-10.43658,1600000,258151,218,115,true,-2.9581,0.71855,2013-10-04T03:26:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02236,asX2,1285,Xanthomonas sRNA asX2,Eberhardt R,Predicted; CMfinder,70.0,96.9,46.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,0,0,0,Gene; sRNA;,Predicted; CMfinder,2,0,,\"\",0.54762,-8.21544,1600000,297481,182,87,true,-3.3485,0.71894,2013-10-04T03:26:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02237,asX3,1285,Xanthomonas sRNA asX3,Eberhardt R,Predicted; CMfinder,40.0,44.0,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; CMfinder,4,0,,\"\",0.54416,-10.02554,1600000,332747,309,182,true,-3.2666,0.71425,2013-10-04T03:26:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02238,asX4,1285,Xanthomonas sRNA asX4/Xoo4,Eberhardt R,Predicted; CMfinder,100.0,155.7,27.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,3,0,0,Gene; sRNA;,Predicted; CMfinder,9,0,,\"\",0.71787,-8.22108,1600000,345617,466,309,true,-3.9426,0.70557,2013-10-04T03:26:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02239,asX6,1285,Xanthomonas sRNA asX6,Eberhardt R,Predicted; CMfinder,50.0,58.5,36.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,0,0,0,Gene; sRNA;,Predicted; CMfinder,3,0,,\"\",0.51647,-7.57387,1600000,263746,191,93,true,-3.5865,0.7189,2013-10-04T03:26:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02240,Xoo1,1285,Xanthomonadaceae sRNA Xoo1,Eberhardt R,Predicted; CMfinder,70.0,80.5,58.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,4,0,0,Gene; sRNA;,Predicted; CMfinder,7,0,,\"\",0.4845,-9.70368,1600000,363798,192,95,true,-3.5707,0.71861,2013-10-04T03:26:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02241,Xoo2,1285,Xanthomonadaceae sRNA Xoo2,Eberhardt R,Predicted; CMfinder,45.0,46.4,39.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; sRNA;,Predicted; CMfinder,10,0,,\"\",0.54559,-8.78822,1600000,360949,170,78,true,-3.3694,0.71892,2013-10-04T03:26:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02242,Xoo5,1285,Xanthomonadaceae sRNA Xoo5,Eberhardt R,Predicted; ClustalW,50.0,65.9,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,27,0,0,Gene; sRNA;,Predicted; RNAalifold,29,0,,\"\",0.59222,-6.73591,1600000,257257,233,124,true,-3.208,0.71841,2013-10-04T03:26:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02243,Xoo8,1285,Proteobacterial sRNA Xoo8,Eberhardt R,Predicted; CMfinder,90.0,99.4,88.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,8,0,0,Gene; sRNA;,Predicted; CMfinder,11,0,,\"\",0.73398,-7.8913,1600000,359343,479,323,true,-4.0597,0.70504,2013-10-04T03:26:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02244,mir-785,1287,microRNA mir-785,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.5,56.3,Imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.65498,-2.89505,1600000,180770,186,90,true,-3.4605,0.71891,2013-10-04T03:26:33.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02245,mir-788,1287,microRNA mir-788,Griffiths-Jones SR,Griffiths-Jones SR,90.0,101.2,82.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.69897,-1.7552,1600000,204910,173,80,true,-3.4658,0.72083,2013-10-04T03:26:35.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02246,Six3os1_1,2309,Six3os1 conserved region 1,Eberhardt R,Predicted; ClustalW,50.0,50.1,42.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.64 -Z 549862.597050 CM SEQDB,9,34,0,0,Gene; lncRNA;,Predicted; RNAalifold,35,0,,\"\",0.57428,-9.5008,1600000,301250,174,87,false,-3.6114,0.71844,2013-10-04T03:26:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02247,Six3os1_2,2309,Six3os1 conserved region 2,Eberhardt R,Predicted; CMfinder,80.0,119.4,59.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.26 -Z 549862.597050 CM SEQDB,15,36,0,0,Gene; lncRNA;,Predicted; CMfinder,39,0,,\"\",0.40849,-19.23184,1600000,774134,350,217,false,-3.2219,0.71063,2013-10-04T03:26:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02248,Six3os1_3,2309,Six3os1 conserved region 3,Eberhardt R,Predicted; ClustalW,40.0,43.7,32.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.57 -Z 549862.597050 CM SEQDB,10,36,0,0,Gene; lncRNA;,Predicted; RNAalifold,35,0,,\"\",0.68284,-7.77156,1600000,372631,159,75,false,-3.6691,0.71862,2013-10-04T03:27:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02249,Six3os1_4,2309,Six3os1 conserved region 4,Eberhardt R,Predicted; ClustalW,70.0,79.8,59.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.13 -Z 549862.597050 CM SEQDB,7,36,0,0,Gene; lncRNA;,Predicted; RNAalifold,37,0,,\"\",0.35295,-22.50565,1600000,689596,477,326,false,-3.6037,0.7048,2013-10-04T03:27:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02250,Six3os1_5,2309,Six3os1 conserved region 5,Eberhardt R,Predicted; CMfinder,50.0,52.0,43.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.81 -Z 549862.597050 CM SEQDB,15,104,0,0,Gene; lncRNA;,Predicted; CMfinder,105,0,,\"\",0.57289,-13.41652,1600000,303123,409,272,false,-3.7015,0.70707,2013-10-04T03:27:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02251,Six3os1_6,2309,Six3os1 conserved region 6,Eberhardt R,Predicted; CMfinder,80.0,85.7,76.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 57.23 -Z 549862.597050 CM SEQDB,4,15,0,0,Gene; lncRNA;,Predicted; CMfinder,15,0,,\"\",0.22129,-26.82505,1600000,348516,324,200,false,-3.2719,0.71215,2013-10-04T03:27:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02252,Six3os1_7,2309,Six3os1 conserved region 7,Eberhardt R,Predicted; CMfinder,50.0,54.9,37.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.99 -Z 549862.597050 CM SEQDB,5,110,0,0,Gene; lncRNA;,Predicted; CMfinder,103,0,,\"\",0.38637,-15.00005,1600000,328848,340,194,false,-3.4769,0.71275,2013-10-04T03:27:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02253,IRE_II,1334,Iron response element II,\"Stevens S, Gardner P, Brown C\",\"Leipuviene, R. and Theil, E.C.\",30.0,30.0,29.9,\"IRE, TFRC, SLC40A1, SDHB, SLC11A2, CDC42BPA, ACO2, ALAS2_H.SAPIENS, CDC14A, EPAS1, ALAS2, qoxD, Hao1, NDUFS1, transferrin, Aconitate. The 3D structure of IRE_II was reported in PDB 3SN2 [14]\",Model1_new,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,30,914,0,0,Cis-reg;,Published; PMID:17849083,140,0,,\"\",0.6416,-2.08294,1600000,284257,87,31,true,-2.8638,0.75436,2013-10-04T03:27:30.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02254,mir-35,2265,microRNA mir-35,Griffiths-Jones SR,Griffiths-Jones SR,70.2,70.3,69.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60261,-3.79882,1600000,249954,158,72,true,-3.1996,0.72176,2013-10-04T03:27:34.000Z,2023-11-06T17:21:47.000Z\n",
            "RF02255,adapt33_1,2310,Adaptive response 33 conserved region 1,Eberhardt R,Predicted; CMfinder,60.0,65.3,52.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 37.93 -Z 549862.597050 CM SEQDB,3,7,0,0,Gene; lncRNA;,Predicted; CMfinder,5,0,,\"\",0.38902,-10.03822,1600000,369996,161,76,false,-3.2994,0.71861,2013-10-04T03:27:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02256,adapt33_2,2310,Adaptive response 33 conserved region 2,Eberhardt R,Predicted; CMfinder,60.0,79.8,53.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.02 -Z 549862.597050 CM SEQDB,2,4,0,0,Gene; lncRNA;,Predicted; CMfinder,3,0,,\"\",0.46742,-8.1889,1600000,264965,158,71,false,-2.6778,0.72659,2013-10-04T03:27:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02257,adapt33_3,2310,Adaptive response 33 conserved region 3,Eberhardt R,Predicted; CMfinder,70.0,82.5,54.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.25 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; lncRNA;,Predicted; CMfinder,5,0,,\"\",0.55925,-9.65465,1600000,285224,249,140,false,-3.2596,0.71788,2013-10-04T03:27:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02258,adapt33_4,2310,Adaptive response 33 conserved region 4,Eberhardt R,Predicted; CMfinder,80.0,119.4,38.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 37.29 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; lncRNA;,Predicted; CMfinder,5,0,,\"\",0.34113,-18.20713,1600000,485709,300,179,false,-3.2545,0.7143,2013-10-04T03:27:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02259,Vax2os1_1,2311,Vax2os1 conserved region 1,Eberhardt R,Predicted; CMfinder,100.0,178.0,81.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.04 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; lncRNA;,Predicted; CMfinder,2,0,,\"\",0.36338,-16.72891,1600000,298687,336,187,false,-3.3457,0.71337,2013-10-04T03:27:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02260,MAT2A_A,2481,MAT2A 3'UTR stem loop A,Daub J,Published; PMID:21994249,40.0,41.0,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,20,95,0,0,Cis-reg;,Published; PMID:21994249,75,0,,\"\",0.69443,-3.05142,1600000,243170,148,65,true,-3.9512,0.71911,2013-10-04T03:27:45.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02261,MAT2A_B,2481,MAT2A 3'UTR stem loop B,Daub J,Published; PMID:21994249,45.0,47.1,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,28,66,0,0,Cis-reg;,Published; PMID:21994249,49,0,,\"\",0.63915,-3.54928,1600000,271790,147,64,true,-3.4541,0.71927,2013-02-01T11:59:39.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02262,MAT2A_C,2481,MAT2A 3'UTR stem loop C,Daub J,Published; PMID:21994249,47.0,49.9,43.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,18,76,0,0,Cis-reg;,Published; PMID:21994249,47,0,,\"\",0.6326,-4.396,1600000,328444,128,52,true,-3.9121,0.71954,2013-10-04T03:27:52.000Z,2022-10-29T17:42:08.000Z\n",
            "RF02263,MAT2A_D,2481,MAT2A 3'UTR stem loop D,Daub J,Published; PMID:21994249,32.0,32.0,31.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,35,107,0,0,Cis-reg;,Published; PMID:21994249,61,0,,\"\",0.67509,-4.33754,1600000,411906,97,41,true,-4.1576,0.72256,2013-10-04T03:27:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02264,MAT2A_E,2481,MAT2A 3'UTR stem loop E,Daub J,Published; PMID:21994249,46.0,46.9,41.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.33 -Z 742849.287494 CM SEQDB,28,83,0,0,Cis-reg;,Published; PMID:21994249,55,0,,\"\",0.65052,-3.45986,1600000,292679,149,67,true,-3.5649,0.71917,2013-10-04T03:27:55.000Z,2022-10-29T17:42:08.000Z\n",
            "RF02265,MAT2A_F,2481,MAT2A 3'UTR stem loop F,Daub J,Published; PMID:21994249,51.0,52.0,50.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.17 -Z 549862.597050 CM SEQDB,26,92,0,0,Cis-reg;,Published; PMID:21994249,55,0,,\"\",0.64808,-3.21549,1600000,283739,150,66,true,-3.8368,0.71918,2013-10-04T03:27:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02266,XIST_intron,2469,XIST 3' intron conserved motif,Daub J,Published; PMID:21994249,70.0,80.1,58.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,35,0,0,Gene; lncRNA;,Published; PMID:21994249,43,0,,\"\",0.58434,-4.15754,1600000,267709,153,68,true,-3.6638,0.71909,2013-10-04T03:27:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02267,Vax2os1_2,2311,Vax2os1 conserved region 2,Eberhardt R,Predicted; ClustalW,55.0,57.6,49.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 13.33 -Z 549862.597050 CM SEQDB,3,33,0,0,Gene; lncRNA;,Predicted; RNAalifold,33,0,,\"\",0.71632,-13.57512,1600000,679766,645,466,false,-4.4868,0.70129,2013-10-04T03:28:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02268,MtlS,2312,MtlS RNA,Burge SW,PMID 22101846,80.0,91.6,52.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.34 -Z 549862.597050 CM SEQDB,6,25,0,0,Gene; sRNA;,Predicted; WAR; Burge SW,30,0,,\"\",0.58678,-4.57635,1600000,219734,226,121,true,-3.0183,0.71853,2013-10-04T03:28:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02269,HPnc0580,1236,Heliobacter pylori small RNA HPnc0580,Burge SW,Predicted; MAFFT,37.0,37.9,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,27,0,0,Gene; sRNA;,Predicted; RNAalifold,15,0,,\"\",0.52816,-6.18686,1600000,319095,206,105,true,-3.4039,0.7184,2013-02-01T11:59:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02270,nse_sRNA,2313,Neisseria sigma-E sRNA,Gardner PP,Gardner PP,75.0,79.3,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,3,0,0,Gene;,Published; PMID:22194974; Gardner PP,8,0,,\"\",0.55459,-5.39992,1600000,316894,163,73,true,-3.4629,0.71898,2013-10-04T03:28:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02271,uc_338,2314,TUC338,Burge SW,Burge SW,120.0,120.0,119.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.50 -Z 549862.597050 CM SEQDB,34,75130,0,0,Cis-reg;,Predicted; WAR; Burge SW,105,0,,\"\",0.65828,-7.38245,1600000,272233,296,180,true,-3.2765,0.71449,2013-10-04T03:28:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02272,Vax2os1_3,2311,Vax2os1 conserved region 3,Eberhardt R,Predicted; MAFFT,50.0,57.2,41.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.19 -Z 549862.597050 CM SEQDB,5,31,0,0,Gene; lncRNA;,Predicted; RNAalifold,32,0,,\"\",0.46799,-15.76158,1600000,612086,322,199,false,-3.5299,0.71218,2013-10-04T03:30:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02273,FsrA,2315,FsrA,Burge SW,Burge SW,80.0,108.5,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,7,0,0,Gene;,Predicted; WAR; mafft_alifold,10,0,,\"\",0.55741,-9.28476,1600000,410745,177,83,true,-3.6034,0.71878,2013-10-04T03:30:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02274,AniS,2265,AniS,Burge SW,Burge SW,50.0,50.1,46.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,9,0,0,Gene;,Predicted; WAR; CMFinder,14,0,,\"\",0.5236,-7.06649,1600000,286239,209,106,true,-3.6059,0.71849,2013-10-04T03:30:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02275,Hammerhead_HH9,1309,Hammerhead ribozyme HH9,\"de la Pena M, Burge SW\",Burge SW,52.0,52.4,51.9,,Hammerhead_RECK,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,33,235,0,0,Gene; ribozyme;,\"Published; PMID:20651741; Burge, SW\",104,0,,\"\",0.73853,-2.62455,1600000,165569,166,77,true,-4.0471,0.7189,2013-10-04T03:30:14.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02276,Hammerhead_II,1309,Hammerhead ribozyme (type II),\"de la Pena M, Burge SW\",de la Pena M; Burge SW,37.0,37.2,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,24,203,0,0,Gene; ribozyme;,Predicted; WAR,151,0,,\"\",0.59885,-5.46981,1600000,261397,149,67,true,-3.3776,0.71896,2013-10-04T03:30:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02277,Hammerhead_HH10,1309,Hammerhead ribozyme HH10,\"de la Pena M, Burge SW\",de la Pena M; Burge SW,75.0,83.0,68.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,18,33,0,0,Gene; ribozyme;,Predicted; WAR; mafft_alifold,42,0,,\"\",0.60414,-4.80295,1600000,220830,234,125,true,-3.3883,0.71834,2013-10-04T03:30:20.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02278,Betaproteobacteria_toxic_sRNA,2316,Betaproteobacteria toxic small RNA,\"Levy A, Burge SW\",Levy A; Burge SW,50.0,50.1,49.6,\"Overexpression in amino aid rich medium attenuates growth, suggesting role in repressing amino acid intake and metabolism.[2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.71 -Z 549862.597050 CM SEQDB,51,579,0,0,Gene; sRNA;,Predicted; WAR; Burge SW,302,0,,\"\",0.58138,-7.78701,1600000,465449,135,64,true,-3.4886,0.71924,2013-10-04T03:30:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02279,TtnuCD1,1264,Tetrahymena snoRNA TtnuCD1,Eberhardt R,Eberhardt R,70.0,84.0,38.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.62 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.43414,-7.62096,1600000,268134,148,63,false,-3.0575,0.72584,2013-10-04T03:30:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02280,TtnuCD3,1264,Tetrahymena snoRNA TtnuCD3,Eberhardt R,Eberhardt R,70.0,104.0,41.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.20 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,4,0,,\"\",0.43621,-8.83947,1600000,268218,158,72,false,-3.7323,0.71871,2013-10-04T03:30:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02281,TtnuCD4,1264,Tetrahymena snoRNA TtnuCD4,Eberhardt R,Eberhardt R,60.0,74.8,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.54 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,5,0,,\"\",0.50078,-10.27843,1600000,488552,139,60,false,-3.265,0.72154,2013-10-04T03:30:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02282,TtnuCD5,1264,Tetrahymena snoRNA TtnuCD5,Eberhardt R,Eberhardt R,70.0,104.9,43.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.74 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.41086,-10.28594,1600000,315040,166,77,false,-2.7426,0.71953,2013-10-04T03:30:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02283,TtnuCD6,1264,Tetrahymena snoRNA TtnuCD6,Eberhardt R,Eberhardt R,50.0,63.9,33.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.11 -Z 549862.597050 CM SEQDB,2,3,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,4,0,,\"\",0.47366,-8.21037,1600000,221723,160,72,false,-3.4115,0.72611,2013-10-04T03:30:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02284,TtnuCD7,1264,Tetrahymena snoRNA TtnuCD7,Eberhardt R,Eberhardt R,90.0,127.7,42.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 39.82 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.38007,-8.76688,1600000,304194,209,108,false,-2.9463,0.71824,2013-10-04T03:30:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02285,TtnuCD8,1264,Tetrahymena snoRNA TtnuCD8,Eberhardt R,Eberhardt R,70.0,75.7,37.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.18 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,4,0,,\"\",0.47651,-7.13843,1600000,247792,142,59,false,-2.9256,0.73037,2013-10-04T03:30:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02286,TtnuCD9,1264,Tetrahymena snoRNA TtnuCD9,Eberhardt R,Eberhardt R,70.0,78.6,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.17 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.46027,-6.93803,1600000,302307,137,55,false,-2.8425,0.72986,2013-10-04T03:30:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02287,TtnuCD10,1264,Tetrahymena snoRNA TtnuCD10,Eberhardt R,Eberhardt R,85.0,108.9,42.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.49 -Z 549862.597050 CM SEQDB,4,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,6,0,,\"\",0.41601,-11.0377,1600000,322319,219,117,false,-3.0935,0.71804,2013-10-04T03:30:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02288,TtnuCD11,1264,Tetrahymena snoRNA TtnuCD11,Eberhardt R,Eberhardt R,70.0,78.9,32.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.39 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,5,0,,\"\",0.61309,-7.88088,1600000,333959,158,74,false,-3.685,0.71868,2013-10-04T03:30:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02289,TtnuCD13,1264,Tetrahymena snoRNA TtnuCD13,Eberhardt R,Eberhardt R,75.0,88.6,40.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.53 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.46913,-8.9251,1600000,318057,148,63,false,-3.0817,0.72463,2013-10-04T03:30:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02290,TtnuCD14,1264,Tetrahymena snoRNA TtnuCD14,Eberhardt R,Eberhardt R,75.0,95.5,36.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.64 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.44744,-8.50612,1600000,287822,162,74,false,-3.2897,0.72161,2013-10-04T03:30:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02291,TtnuCD15,1264,Tetrahymena snoRNA TtnuCD15,Eberhardt R,Eberhardt R,75.0,79.3,48.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.76 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,4,0,,\"\",0.44525,-7.61193,1600000,290628,150,65,false,-2.9079,0.72795,2013-10-04T03:30:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02292,TtnuCD16,1264,Tetrahymena snoRNA TtnuCD16,Eberhardt R,Eberhardt R,75.0,83.4,37.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.66 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.56338,-8.05584,1600000,294654,141,58,false,-3.1484,0.72778,2013-10-04T03:30:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02293,TtnuCD17,1264,Tetrahymena snoRNA TtnuCD17,Eberhardt R,Eberhardt R,75.0,91.4,38.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.90 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.45658,-8.89701,1600000,357343,154,68,false,-3.2174,0.72274,2013-10-04T03:30:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02294,TtnuCD18,1264,Tetrahymena snoRNA TtnuCD18,Eberhardt R,Eberhardt R,55.0,64.8,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.32 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,4,0,,\"\",0.45794,-6.73288,1600000,269402,141,58,false,-3.1346,0.73153,2013-10-04T03:30:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02295,TtnuCD19,1264,Tetrahymena snoRNA TtnuCD19,Eberhardt R,Eberhardt R,70.0,80.5,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 36.27 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,4,0,,\"\",0.42244,-6.39028,1600000,194987,157,70,false,-2.9142,0.72617,2013-10-04T03:30:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02296,TtnuCD20,1264,Tetrahymena snoRNA TtnuCD20,Eberhardt R,Eberhardt R,80.0,92.6,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.13 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.47934,-8.66832,1600000,347063,152,66,false,-3.1548,0.72303,2013-10-04T03:30:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02297,TtnuCD21,1264,Tetrahymena snoRNA TtnuCD21,Eberhardt R,Eberhardt R,70.0,78.2,35.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.38 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,4,0,,\"\",0.47159,-6.33116,1600000,247172,150,65,false,-2.9616,0.72896,2013-10-04T03:30:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02298,TtnuCD22,1264,Tetrahymena snoRNA TtnuCD22,Eberhardt R,Eberhardt R,85.0,108.6,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.48 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.50868,-8.6824,1600000,283373,166,77,false,-3.3547,0.71881,2013-10-04T03:30:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02299,TtnuCD23,1264,Tetrahymena snoRNA TtnuCD23,Eberhardt R,Eberhardt R,70.0,91.2,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.45 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.4331,-9.38094,1600000,331142,160,72,false,-3.1936,0.72142,2013-10-04T03:30:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02300,TtnuCD24,1264,Tetrahymena snoRNA TtnuCD24,Eberhardt R,Eberhardt R,75.0,84.2,36.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.13 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.48915,-9.15688,1600000,395966,141,58,false,-3.1324,0.72734,2013-10-04T03:31:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02301,TtnuCD25,1264,Tetrahymena snoRNA TtnuCD25,Eberhardt R,Eberhardt R,85.0,98.0,32.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.94 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,5,0,,\"\",0.59389,-7.19706,1600000,313371,181,89,false,-3.5835,0.71843,2013-10-04T03:31:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02302,TtnuCD26,1264,Tetrahymena snoRNA TtnuCD26,Eberhardt R,Eberhardt R,90.0,108.2,36.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.26 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,4,0,,\"\",0.42388,-10.0101,1600000,268562,208,108,false,-3.1265,0.71817,2013-10-04T03:31:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02303,TtnuCD28,1264,Tetrahymena snoRNA TtnuCD28,Eberhardt R,Eberhardt R,70.0,83.5,46.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.37 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.46086,-7.94593,1600000,340706,149,64,false,-3.1187,0.72525,2013-10-04T03:31:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02304,TtnuCD29,1264,Tetrahymena snoRNA TtnuCD29,Eberhardt R,Eberhardt R,90.0,110.3,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.64 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.42302,-9.90676,1600000,290775,168,78,false,-3.3609,0.71863,2013-10-04T03:31:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02305,TtnuCD31,1264,Tetrahymena snoRNA TtnuCD31,Eberhardt R,Eberhardt R,75.0,86.4,32.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.59 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.48451,-8.36448,1600000,282263,153,67,false,-3.2716,0.72341,2013-10-04T03:31:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02306,TtnuCD32,1264,Tetrahymena snoRNA TtnuCD32,Eberhardt R,Eberhardt R,100.0,128.1,34.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.39 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,4,0,,\"\",0.5245,-8.8627,1600000,311309,196,98,false,-3.5261,0.71831,2013-10-04T03:31:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02307,TtnuCD33,1264,Tetrahymena snoRNA TtnuCD33,Eberhardt R,Eberhardt R,80.0,103.7,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.73 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.44901,-7.83216,1600000,236888,163,74,false,-3.3139,0.72111,2013-10-04T03:31:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02308,TtnuCD34,1264,Tetrahymena snoRNA TtnuCD34,Eberhardt R,Eberhardt R,80.0,89.4,32.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.18 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold;,3,0,,\"\",0.51364,-8.48522,1600000,263159,149,64,false,-3.3694,0.72559,2013-10-04T03:31:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02309,TtnuHACA1,1264,Tetrahymena snoRNA TtnuHACA1,Eberhardt R,Eberhardt R,100.0,137.4,50.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.51785,-7.25973,1600000,310453,220,113,true,-3.3306,0.71849,2013-10-04T03:31:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02310,TtnuHACA2,1264,Tetrahymena snoRNA TtnuHACA2,Eberhardt R,Eberhardt R,100.0,139.0,38.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,4,0,,\"\",0.50322,-6.06654,1600000,255514,230,121,true,-2.7617,0.71827,2013-10-04T03:31:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02311,TtnuHACA3,1264,Tetrahymena snoRNA TtnuHACA3,Eberhardt R,Eberhardt R,100.0,177.4,41.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,2,0,,\"\",0.46532,-8.32194,1600000,295242,272,152,true,-3.0565,0.71814,2013-10-04T03:31:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02312,TtnuHACA5,1264,Tetrahymena snoRNA TtnuHACA5,Eberhardt R,Eberhardt R,100.0,150.7,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.5425,-6.96348,1600000,269080,245,132,true,-3.1116,0.71829,2013-10-04T03:31:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02313,TtnuHACA6,1264,Tetrahymena snoRNA TtnuHACA6,Eberhardt R,Eberhardt R,100.0,150.2,45.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.46043,-9.72904,1600000,343161,244,131,true,-3.0589,0.71816,2013-10-04T03:31:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02314,TtnuHACA7,1264,Tetrahymena snoRNA TtnuHACA7,Eberhardt R,Eberhardt R,100.0,129.7,32.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.60096,-4.90176,1600000,236147,239,129,true,-3.4742,0.71832,2013-10-04T03:31:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02315,TtnuHACA8,1264,Tetrahymena snoRNA TtnuHACA8,Eberhardt R,Eberhardt R,50.0,63.1,36.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,4,0,,\"\",0.47327,-8.25001,1600000,303038,242,130,true,-3.1326,0.71835,2013-10-04T03:31:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02316,TtnuHACA9,1264,Tetrahymena snoRNA TtnuHACA9,Eberhardt R,Eberhardt R,100.0,127.3,47.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,5,0,,\"\",0.5018,-6.70974,1600000,242915,247,135,true,-2.7834,0.71818,2013-10-04T03:31:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02317,TtnuHACA10,1264,Tetrahymena snoRNA TtnuHACA10,Eberhardt R,Eberhardt R,100.0,126.3,42.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,4,0,,\"\",0.47094,-6.16668,1600000,274889,206,104,true,-3.0198,0.71843,2013-10-04T03:31:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02318,TtnuHACA11a,1264,Tetrahymena snoRNA TtnuHACA11a,Eberhardt R,Eberhardt R,100.0,160.3,48.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.52563,-7.08281,1600000,264804,258,141,true,-3.0541,0.71813,2013-10-04T03:31:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02319,TtnuHACA12,1264,Tetrahymena snoRNA TtnuHACA12,Eberhardt R,Eberhardt R,80.0,89.5,51.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.03 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.47834,-6.43676,1600000,285736,241,131,true,-2.8637,0.71824,2013-10-04T03:31:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02320,TtnuHACA13,1264,Tetrahymena snoRNA TtnuHACA13,Eberhardt R,Eberhardt R,100.0,156.9,45.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.55512,-5.61559,1600000,285368,248,134,true,-2.6628,0.71813,2013-10-04T03:31:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02321,TtnuHACA14,1264,Tetrahymena snoRNA TtnuHACA14,Eberhardt R,Eberhardt R,100.0,159.4,49.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,4,0,,\"\",0.50011,-7.48574,1600000,247130,246,133,true,-2.8678,0.71811,2013-10-04T03:31:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02322,TtnuHACA15,1264,Tetrahymena snoRNA TtnuHACA15,Eberhardt R,Eberhardt R,100.0,148.0,47.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.52252,-6.80304,1600000,248612,225,117,true,-3.0377,0.71832,2013-10-04T03:31:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02323,TtnuHACA16,1264,Tetrahymena snoRNA TtnuHACA16,Eberhardt R,Eberhardt R,100.0,148.6,38.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.46598,-7.28637,1600000,325646,225,117,true,-3.0901,0.71836,2013-10-04T03:31:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02324,TtnuHACA17,1264,Tetrahymena snoRNA TtnuHACA17,Eberhardt R,Eberhardt R,100.0,127.4,41.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,4,0,,\"\",0.46351,-6.60264,1600000,289423,225,118,true,-3.2549,0.71826,2013-10-04T03:31:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02325,TtnuHACA18,1264,Tetrahymena snoRNA TtnuHACA18,Eberhardt R,Eberhardt R,100.0,143.9,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,4,0,,\"\",0.49055,-6.94903,1600000,321953,241,130,true,-2.9333,0.71817,2013-10-04T03:31:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02326,TtnuHACA19,1264,Tetrahymena snoRNA TtnuHACA19,Eberhardt R,Eberhardt R,100.0,154.9,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.47992,-7.55107,1600000,262833,231,121,true,-3.1351,0.71822,2013-10-04T03:31:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02327,TtnuHACA20,1264,Tetrahymena snoRNA TtnuHACA20,Eberhardt R,Eberhardt R,100.0,143.8,39.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.48095,-8.85213,1600000,290895,234,123,true,-3.1873,0.71843,2013-10-04T03:31:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02328,TtnuHACA21,1264,Tetrahymena snoRNA TtnuHACA21,Eberhardt R,Eberhardt R,100.0,164.7,43.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.50129,-7.68163,1600000,289818,243,130,true,-2.986,0.71821,2013-10-04T03:31:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02329,TtnuHACA22,1264,Tetrahymena snoRNA TtnuHACA22,Eberhardt R,Eberhardt R,90.0,115.1,40.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.49093,-5.2172,1600000,211242,205,103,true,-2.917,0.71868,2013-10-04T03:31:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02330,TtnuHACA23,1264,Tetrahymena snoRNA TtnuHACA23,Eberhardt R,Eberhardt R,100.0,130.0,41.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,4,0,,\"\",0.46702,-7.24942,1600000,305101,246,135,true,-2.6933,0.71827,2013-10-04T03:31:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02331,TtnuHACA24,1264,Tetrahymena snoRNA TtnuHACA24,Eberhardt R,Eberhardt R,100.0,154.1,40.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.5305,-6.58106,1600000,221458,239,127,true,-3.1652,0.71835,2013-10-04T03:31:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02332,TtnuHACA25,1264,Tetrahymena snoRNA TtnuHACA25,Eberhardt R,Eberhardt R,100.0,135.6,44.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.54127,-4.31051,1600000,217207,217,111,true,-2.9902,0.71833,2013-10-04T03:31:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02333,TtnuHACA27,1264,Tetrahymena snoRNA TtnuHACA27,Eberhardt R,Eberhardt R,80.0,106.0,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.45264,-5.34904,1600000,290430,182,86,true,-2.9598,0.71861,2013-10-04T03:31:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02334,TtnuHACA28,1264,Tetrahymena snoRNA TtnuHACA28,Eberhardt R,Eberhardt R,130.0,170.6,55.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold;,3,0,,\"\",0.48729,-8.08668,1600000,298397,249,134,true,-3.3676,0.71819,2013-10-04T03:31:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02335,GlsR2_miR4,2467,Giardia snoRNA GlsR2,Eberhardt R,Published; PMID:22028939,100.0,120.3,33.3,\"Reported as a snoRNA related to miRNA [1], updated to only snoRNA after reviewed with mirBase data (2021)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30 -Z 742849.287494 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA;,Predicted; PMID:22028939,2,0,,\"\",0.52079,-7.57949,1600000,250822,210,106,true,-3.0757,0.71851,2013-10-04T03:31:57.000Z,2021-11-19T10:36:03.000Z\n",
            "RF02336,GlsR1_miR6,2467,Giardia snoRNA GlsR1,Eberhardt R,Published; PMID:22568619,100.0,110.4,33.7,\"Reported as a snoRNA related to miRNA [1], been updated to snoRNA after reviewed with mirBase data (2021)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA;,Predicted; PMID:22568619,3,0,,\"\",0.57059,-5.20145,1600000,185510,180,85,true,-3.4022,0.71871,2013-10-04T03:31:59.000Z,2021-11-19T10:36:03.000Z\n",
            "RF02337,GlsR8_miR10,2467,Giardia snoRNA GlsR8,Eberhardt R,Published; PMID:22568619,100.0,103.3,34.4,\"Reported as a snoRNA related to miRNA [1], updated to only snoRNA after reviewed with mirBase data (2021)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA;,Predicted; PMID:22568619,3,0,,\"\",0.67036,-3.3949,1600000,217072,159,70,true,-3.8438,0.71903,2013-10-21T09:34:28.000Z,2021-11-19T10:36:03.000Z\n",
            "RF02338,GlsR16_miR3,2467,Giardia snoRNA GlsR16,Eberhardt R,Published; PMID:19043559,100.0,103.6,34.2,\"Reported as a snoRNA related to miRNA [1], updated to only snoRNA after reviewed with mirBase data (2021)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA;,Predicted; PMID:19043559,4,0,,\"\",0.74532,-1.70431,1600000,219210,169,77,true,-3.2928,0.71891,2013-10-04T03:32:02.000Z,2021-11-19T10:36:03.000Z\n",
            "RF02339,GlsR17_miR2,2467,Giardia snoRNA GlsR17,Eberhardt R,Published; PMID:19043559,84.0,84.8,36.9,\"Reported as a snoRNA related to miRNA [1], updated to only snoRNA after reviewed with mirBase data (2021)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,2,0,0,Gene; snRNA; snoRNA;,Predicted; PMID:19043559,3,0,,\"\",0.6996,-2.88524,1600000,250749,147,63,true,-3.0644,0.71936,2013-10-04T03:32:04.000Z,2021-11-19T10:36:03.000Z\n",
            "RF02340,DENV_SLA,1467,Dengue virus SLA,\"Eberhardt R, Choi K\",Published; PMID:19004935,35.4,35.4,35.1,\"Flavivirus stem-loop A (SLA) is a secondary structure present in the 5 prime untranslated (5'-UTR) region of viruses of the genus Flavivirus. SLA structure was first proposed from enzymatic and chemical probing methods [1,2] afterwards the 3D structure of Dengue (PDB:7LYF) and Zika (PDB:5U0B) SLA viruses were reported [3]. SLA interacts with the nonstructural protein 5 (NS5), where SLA functions as a promoter for viral RNA synthesis by NS5 polymerase. The SLA secondary structure includes a bottom stem-loop (previously reported as SL1 and SL2), a top stem-loop (previously reported as SL3 and TL) and a side-loop (previously proposed as side SL). The side SL variate in length between flaviviruses (Tick-borne flaviviruses present a side SL >40nt), despite the lack of sequence conservation, deletion of the side SL abolish replication in Dengue virus. It is proposed that the side SL is essential to maintain the overall fold of Flavivirus SLA [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,83,0,0,Cis-reg;,Published; PMID:19004935,27,0,,\"\",0.5712,-5.60824,1600000,348354,147,70,true,-3.2712,0.71896,2013-10-04T03:32:05.000Z,2021-07-05T13:06:19.000Z\n",
            "RF02341,ncrMT1302,2178,Mycobacterium sRNA ncrMT1302,Eberhardt R,Predicted; WAR,45.0,46.3,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,57,0,0,Gene; sRNA;,Predicted; WAR,63,0,,\"\",0.55226,-7.21478,1600000,280346,209,109,true,-2.8945,0.71841,2013-10-04T03:32:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02342,ar7,2318,Alphaproteobacterial sRNA ar7,Eberhardt R,Published; PMID:22418845,60.0,77.4,47.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.83 -Z 549862.597050 CM SEQDB,29,101,0,0,Gene; sRNA;,Predicted; PMID:22418845,120,0,,\"\",0.5501,-10.67055,1600000,509553,251,146,true,-3.2163,0.71813,2013-10-04T03:32:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02343,ar9,2319,Alphaproteobacterial sRNA ar9,Eberhardt R,Published; PMID:22418845,45.0,46.9,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.71 -Z 549862.597050 CM SEQDB,28,123,0,0,Gene; sRNA;,Predicted; PMID:22418845,140,0,,\"\",0.50336,-7.53513,1600000,243354,262,149,true,-3.281,0.71823,2013-10-04T03:32:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02344,ar14,2320,Alphaproteobacterial ar14,Eberhardt R,Published; PMID:22418845,58.8,58.8,58.7,Smr14C2 was renamed NfeR1 (Nodule Formation Efficiency RNA) highly expressed in salt stress and during the symbiotic interaction on legume roots.[2],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,118,498,0,0,Gene; sRNA;,Predicted; PMID:22418845,198,0,,\"\",0.51197,-11.60424,1600000,527493,210,123,true,-3.6329,0.71826,2013-10-04T03:32:14.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02345,ar15,2365,Alphaproteobacterial ar15,Eberhardt R,Published; PMID:22418845,40.0,40.6,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,61,241,0,0,Gene; sRNA;,Predicted; PMID:22418845,126,0,,\"\",0.57102,-7.66858,1600000,276788,211,113,true,-3.4913,0.71856,2013-10-04T03:32:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02346,ar35,2322,Alphaproteobacterial sRNA ar35,Eberhardt R,Published; PMID:22418845,45.0,45.2,44.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,181,0,0,Gene; sRNA;,Predicted; PMID:22418845,99,0,,\"\",0.52547,-9.96073,1600000,341920,252,144,true,-3.1392,0.71834,2013-10-04T03:32:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02347,ar45,2323,Alphaproteobacterial sRNA ar45,Eberhardt R,Published; PMID:22418845,40.0,44.7,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,36,211,0,0,Gene; sRNA;,Predicted; PMID:22418845,231,0,,\"\",0.60959,-7.08808,1600000,256403,263,148,true,-2.6085,0.71829,2013-10-04T03:32:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02348,tracrRNA,2324,Trans-activating crRNA,Eberhardt R,Published; PMID:22745249,74.0,76.8,62.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,35,45,0,0,Gene;,Predicted; PMID:22745249,76,0,,\"\",0.48525,-8.18665,1600000,259902,180,91,true,-3.3937,0.71851,2013-10-04T03:32:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02349,psRNA2,1236,Proteobacterial sRNA psRNA2,Eberhardt R,Published; PMID:20840601,65.0,70.0,62.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,77,0,0,Gene; sRNA;,Predicted; PMID:20840601,16,0,,\"\",0.64998,-8.82099,1600000,408784,391,201,true,-3.7213,0.71238,2013-10-04T03:32:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02350,psRNA6,1236,Nitrosomonas sRNA psRNA6,Eberhardt R,Published; PMID:20840601,60.0,63.6,35.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; PMID:20840601,3,0,,\"\",0.73879,-7.74868,1600000,364936,478,324,true,-4.0004,0.70506,2013-10-04T03:32:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02351,psRNA14,1236,Proteobacteria sRNA psRNA14,Eberhardt R,Published; PMID:20840601,60.0,79.5,54.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; PMID:20840601,3,0,,\"\",0.52951,-5.50498,1600000,226453,173,84,true,-2.7841,0.72374,2013-10-04T03:32:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02353,BjrC68,1236,Bradyrhizobiaceae sRNA BjrC68,Eberhardt R,Published; PMID:22258152,39.0,39.4,37.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,38,0,0,Gene; sRNA;,Predicted; PMID:22258152,46,0,,\"\",0.71617,-5.37333,1600000,243950,170,82,true,-3.5806,0.71882,2013-10-04T03:32:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02354,BjrC80,1236,Bradyrhizobiaceae sRNA BjrC80,Eberhardt R,Published; PMID:22258152,80.0,94.9,69.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,29,0,0,Gene; sRNA;,Predicted; PMID:22258152,36,0,,\"\",0.51491,-11.7306,1600000,545948,272,162,true,-3.0716,0.71693,2013-10-04T03:32:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02355,BjrC174,1236,Bradyrhizobiaceae sRNA BjrC174,Eberhardt R,Published; PMID:22258152,45.0,50.6,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,59,0,0,Gene; sRNA;,Predicted; PMID:22258152,65,0,,\"\",0.51967,-9.05922,1600000,300922,323,201,true,-3.6472,0.71226,2013-10-04T03:32:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02356,BjrC1505,1236,Alphaproteobacterial sRNA BjrC1505,Eberhardt R,Published; PMID:22258152,50.0,54.2,46.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,165,0,0,Gene; sRNA;,Predicted; PMID:22258152,174,0,,\"\",0.54165,-8.77869,1600000,374815,247,140,true,-3.3185,0.71815,2013-10-04T03:32:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02357,RNaseP-T,2441,RNaseP truncated form,Eberhardt R,Published; PMID:21135215,60.0,76.7,47.5,\"Pyrobaculum and Caldivirga RNase P RNAs are reported as the smallest naturally occurring form of RNase P. They function as trans-acting precursor tRNA-processing ribozymes. This variant of RNase P, have a loss of the specificity domain, which suggests altered substrate specificity [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,9,0,0,Gene; ribozyme;,Predicted; PMID:21135215,18,0,,\"\",0.57874,-10.43148,1600000,493955,336,208,true,-2.9423,0.71165,2013-10-04T03:32:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02358,hsp17,2325,Hsp17 thermometer,Eberhardt R,Published; PMID:21131278,45.0,52.8,32.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.49 -Z 549862.597050 CM SEQDB,3,2,0,0,Cis-reg; thermoregulator;,Published; PMID:21131278,3,0,,\"\",0.55474,-5.78446,1600000,301526,127,49,true,-2.4698,0.73589,2013-10-04T03:32:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02359,MS2,2326,Bacteriophage MS2 operator hairpin,Eberhardt R,Eberhardt R,40.0,44.3,35.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.56 -Z 549862.597050 CM SEQDB,5,1,0,0,Cis-reg;,Published; PMID:11095669,4,0,,\"\",0.65228,-7.58001,1600000,524019,106,36,true,-3.6055,0.72645,2013-10-04T03:32:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02360,Yfr8,2265,Cyanobacterial functional RNA 8/9,Eberhardt R,Eberhardt R,40.0,43.2,39.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.29 -Z 549862.597050 CM SEQDB,5,1,0,0,Gene; sRNA;,Predicted; mfold,4,0,,\"\",0.60227,-8.93929,1600000,429424,363,226,true,-3.6924,0.71011,2013-10-04T03:32:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02362,Yfr10,2265,Cyanobacterial functional RNA 10,Eberhardt R,Eberhardt R,43.0,43.2,42.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.41 -Z 549862.597050 CM SEQDB,6,3,0,0,Gene; sRNA;,Predicted; mfold,6,0,,\"\",0.4815,-6.50907,1600000,246954,162,78,true,-3.2336,0.71884,2013-10-04T03:32:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02363,Yfr11,2265,Cyanobacterial functional RNA 11,Eberhardt R,Eberhardt R,44.0,44.7,42.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.84 -Z 549862.597050 CM SEQDB,4,2,0,0,Gene; sRNA;,Predicted; mfold,3,0,,\"\",0.49388,-7.79668,1600000,344468,259,144,true,-2.6802,0.71822,2013-10-04T03:32:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02364,Yfr13,2265,Cyanobacterial functional RNA 13,Eberhardt R,Eberhardt R,39.0,39.0,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.27 -Z 549862.597050 CM SEQDB,7,2,0,0,Gene; sRNA;,Predicted; mfold,8,0,,\"\",0.4738,-8.98396,1600000,347179,195,98,true,-3.1474,0.71851,2013-10-04T03:32:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02365,Yfr17,2265,Cyanobacterial functional RNA 17,Eberhardt R,Eberhardt R,50.0,52.0,43.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.53 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; sRNA;,Predicted; mfold,4,0,,\"\",0.52064,-9.58722,1600000,425957,409,264,true,-3.2652,0.70764,2013-10-04T03:32:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02366,Yfr19,2265,Cyanobacterial functional RNA 19,Eberhardt R,Eberhardt R,70.0,72.5,46.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.05 -Z 549862.597050 CM SEQDB,6,1,0,0,Gene; sRNA;,Predicted; mfold,6,0,,\"\",0.49909,-5.88099,1600000,294939,174,82,true,-3.1106,0.71894,2013-10-04T03:32:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02367,Yfr20,2265,Cyanobacterial functional RNA 20,Eberhardt R,Eberhardt R,44.0,44.0,38.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.13 -Z 549862.597050 CM SEQDB,7,1,0,0,Gene; sRNA;,Predicted; mfold,7,0,,\"\",0.51953,-9.1271,1600000,441473,184,89,true,-3.5226,0.71873,2013-10-04T03:33:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02368,Yfr21,2265,Cyanobacterial functional RNA 21,Eberhardt R,Eberhardt R,100.0,222.9,39.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.28 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; mfold,2,0,,\"\",0.45789,-10.2964,1600000,341845,338,205,true,-3.2085,0.71173,2013-10-04T03:33:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02369,h2cR,2327,h2cR sRNA,Eberhardt R,Eberhardt R,70.0,82.3,48.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.73 -Z 549862.597050 CM SEQDB,7,7,0,0,Gene; sRNA;,Predicted; PMID:23082228,10,0,,\"\",0.59537,-4.64875,1600000,208238,251,137,true,-3.3721,0.7183,2013-10-04T03:33:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02370,Trp_leader_2,1763,Bacillus tryptophan operon leader,Eberhardt R,Eberhardt R,100.0,119.9,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,7,0,0,Cis-reg; leader;,Published; PMID:14712717,16,0,,\"\",0.42649,-9.78202,1600000,348869,280,147,true,-2.6891,0.71795,2013-10-04T03:33:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02371,PyrG_leader,2328,PyrG leader,Eberhardt R,Eberhardt R,65.0,71.4,50.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.69 -Z 549862.597050 CM SEQDB,10,14,0,0,Cis-reg; leader;,Published; PMID:17302819,23,0,,\"\",0.44831,-7.63323,1600000,322480,163,78,true,-3.3406,0.71879,2013-10-04T03:33:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02372,PyrC_leader,2329,PyrC leader,Eberhardt R,Eberhardt R,50.0,60.9,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,12,0,0,Cis-reg; leader;,Published; PMID:18535147,18,0,,\"\",0.49001,-8.70348,1600000,579378,123,50,true,-3.8697,0.7193,2013-10-04T03:33:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02373,PyrD_leader,2330,PyrD leader,Eberhardt R,Eberhardt R,47.0,49.0,46.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,27,59,0,0,Cis-reg; leader;,Published; PMID:7510398,77,0,,\"\",0.5009,-7.58138,1600000,395114,158,74,true,-3.7416,0.71879,2013-10-04T03:33:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02374,YenS,2331,Yersinia YenS sRNA,Eberhardt R,Eberhardt R,80.0,102.3,36.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,2,0,0,Gene; sRNA;,Predicted; mfold,8,0,,\"\",0.60881,-8.39396,1600000,407650,290,169,true,-3.238,0.71584,2013-10-04T03:33:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02375,Aar,2332,Aar sRNA,Eberhardt R,Eberhardt R,60.0,69.7,41.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,30,0,0,Gene; sRNA;,Predicted; mfold,39,0,,\"\",0.45563,-9.15955,1600000,317803,343,191,true,-3.3394,0.71325,2013-10-04T03:33:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02376,antisense_SR1,2333,SR1 sRNA,Eberhardt R,Eberhardt R,45.0,45.0,44.8,,SR1;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,109,0,0,Gene; sRNA;,Published; PMID:17576690,105,0,,\"\",0.52441,-10.50525,1600000,352754,370,210,true,-3.4366,0.71135,2013-10-04T03:33:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02377,SurA,1236,SurA sRNA,Eberhardt R,Eberhardt R,100.0,118.8,55.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,5,0,0,Gene; sRNA;,Predicted; mfold,8,0,,\"\",0.69719,-8.20154,1600000,394695,468,278,true,-4.0875,0.70705,2013-10-04T03:33:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02378,SurC,1236,SurC sRNA,Eberhardt R,Eberhardt R,80.0,120.1,56.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.69 -Z 549862.597050 CM SEQDB,4,4,0,0,Gene; sRNA;,Predicted; mfold,8,0,,\"\",0.54966,-7.52989,1600000,247275,282,161,true,-3.2598,0.71696,2013-10-04T03:33:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02379,csRNA,2334,Cia-dependent small RNA csRNA1,Eberhardt R,Eberhardt R,57.0,57.9,55.3,,csRNA1;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,48,74,0,0,Gene; sRNA;,Predicted; PMID:17725562,60,0,,\"\",0.43161,-8.24756,1600000,304667,185,93,true,-3.4311,0.71859,2013-10-04T03:33:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02384,FasX,2335,FasX small RNA,Eberhardt R,Eberhardt R,100.0,155.4,65.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.24 -Z 549862.597050 CM SEQDB,8,11,0,0,Gene; sRNA;,Predicted; PMID:21143309,15,0,,\"\",0.48582,-10.27429,1600000,389224,333,207,true,-3.9704,0.71171,2013-10-04T03:33:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02385,sau-13,1236,Staphylococcus sRNA sau-13,Eberhardt R,Eberhardt R,55.0,58.7,51.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,15,0,0,Gene; sRNA;,Predicted; mfold,19,0,,\"\",0.50934,-5.68304,1600000,256632,203,104,true,-3.3865,0.71846,2013-10-04T03:33:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02386,sau-19,1236,Staphylococcus sRNA sau-19,Eberhardt R,Eberhardt R,80.0,97.5,52.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; mfold,5,0,,\"\",0.60287,-2.88438,1600000,205007,165,75,true,-3.3359,0.71903,2013-10-04T03:33:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02387,sau-27,1236,Staphylococcus sRNA sau-27,Eberhardt R,Eberhardt R,80.0,98.7,39.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; mfold,4,0,,\"\",0.5043,-5.98657,1600000,272646,189,92,true,-3.3822,0.71864,2013-10-04T03:33:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02388,sau-30,1236,Staphylococcus sRNA sau-30,Eberhardt R,Eberhardt R,70.0,88.6,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; sRNA;,Predicted; mfold,3,0,,\"\",0.54511,-4.96948,1600000,386873,152,64,true,-3.3768,0.72722,2013-10-04T03:33:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02389,sau-31,1236,Staphylococcus sRNA sau-31,Eberhardt R,Eberhardt R,50.0,54.8,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; mfold,4,0,,\"\",0.53176,-4.07536,1600000,259626,153,66,true,-3.4312,0.71897,2013-10-04T03:33:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02390,sau-41,1236,Staphylococcus sRNA sau-41,Eberhardt R,Eberhardt R,80.0,101.2,38.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; mfold,3,0,,\"\",0.45965,-10.01578,1600000,397384,278,156,true,-2.6984,0.71763,2013-10-04T03:33:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02392,sau-53,1236,Staphylococcus sRNA sau-53,Eberhardt R,Eberhardt R,100.0,211.2,37.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; mfold,5,0,,\"\",0.54412,-9.07633,1600000,345928,322,192,true,-3.4266,0.7132,2013-10-04T03:33:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02393,sau-59,1236,Staphylococcus sRNA sau-59,Eberhardt R,Eberhardt R,80.0,82.8,65.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.90 -Z 549862.597050 CM SEQDB,3,8,0,0,Gene; sRNA;,Predicted; mfold,10,0,,\"\",0.43572,-9.69604,1600000,517559,239,128,true,-3.0281,0.71819,2013-10-04T03:33:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02394,sau-63,1236,Staphylococcus sRNA sau-63,Eberhardt R,Eberhardt R,100.0,124.9,51.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; mfold,4,0,,\"\",0.53938,-7.6927,1600000,290063,199,98,true,-3.4884,0.71857,2013-10-04T03:33:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02395,sau-66,1236,Staphylococcus sRNA sau-66,Eberhardt R,Eberhardt R,80.0,110.4,46.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; sRNA;,Predicted; mfold,5,0,,\"\",0.55212,-3.50385,1600000,239736,174,80,true,-2.947,0.71883,2013-10-04T03:33:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02396,sau-5949,1236,Staphylococcus sRNA sau-5949,Eberhardt R,Eberhardt R,56.0,57.4,52.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,59,0,0,Gene; sRNA;,Predicted; mfold,13,0,,\"\",0.48227,-8.27878,1600000,436020,138,59,true,-3.3498,0.7192,2013-10-04T03:33:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02397,sau-5971,1236,Staphylococcus sRNA sau-5971,Eberhardt R,Eberhardt R,90.0,108.2,52.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; mfold,4,0,,\"\",0.50716,-8.75044,1600000,332085,196,96,true,-3.0544,0.71878,2013-10-04T03:34:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02398,sau-6072,1236,Staphylococcus sRNA sau-6072,Eberhardt R,Eberhardt R,50.0,82.4,38.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; mfold,3,0,,\"\",0.58405,-7.23022,1600000,487503,137,53,true,-3.1241,0.73273,2013-10-04T03:34:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02399,NsiR1,1236,Nitrogen stress-induced RNA 1,Eberhardt R,Eberhardt R,39.0,39.5,38.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.20 -Z 549862.597050 CM SEQDB,17,490,0,0,Gene; sRNA;,Predicted; PMID:20227418,81,0,,\"\",0.57096,-8.05942,1600000,513043,131,60,true,-3.1745,0.71922,2013-10-04T03:34:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02400,Hsp83_3_UTR,2336,Heat shock protein 83 3' UTR,Eberhardt R,Eberhardt R,90.0,121.6,78.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,23,0,0,Cis-reg;,Predicted; mfold,7,0,,\"\",0.72595,-8.39217,1600000,312712,729,470,true,-4.1962,0.7014,2013-10-04T03:34:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02401,ClpQY_promoter,2337,ClpQY promoter,Eberhardt R,Eberhardt R,42.0,42.1,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,39,65,0,0,Cis-reg;,Predicted; PMID:19440251,91,0,,\"\",0.56177,-10.60931,1600000,675114,165,58,true,-3.761,0.71931,2013-10-04T03:34:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02402,Afu_263,1264,A. fumigatus snoRNA Afu_263,\"Osuch I, Eberhardt R\",INFERNAL,60.0,60.3,55.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,90,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted;,95,0,,\"\",0.48267,-5.75576,1600000,233554,188,96,true,-3.3716,0.7185,2013-10-21T09:37:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02403,Afu_328,1264,A. fumigatus snoRNA Afu_328,\"Osuch I, Eberhardt R\",INFERNAL,70.0,84.2,60.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.85 -Z 549862.597050 CM SEQDB,4,14,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted;,14,0,,\"\",0.44067,-9.81446,1600000,274158,169,73,false,-3.4897,0.7187,2013-10-21T09:37:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02404,P33,1269,Pseudomonas sRNA P33,\"Osuch I, Eberhardt R\",INFERNAL,45.0,46.7,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; sRNA;,Predicted; WAR; Clustal_alifold;,3,0,,\"\",0.54696,-9.29669,1600000,329197,282,160,true,-2.9056,0.71714,2013-10-04T03:34:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02405,P34,1269,Pseudomonas sRNA P34,\"Osuch I, Eberhardt R\",INFERNAL,44.0,46.3,37.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,58,0,0,Gene; sRNA;,Predicted; WAR; cmfinder;,61,0,,\"\",0.65802,-8.37085,1600000,293580,542,383,true,-4.2484,0.70322,2013-10-04T03:34:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02406,snoR120,1264,Small nucleolar RNA snoR120,Eberhardt R,Predicted,55.0,79.7,41.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.19 -Z 549862.597050 CM SEQDB,4,20,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold,10,0,,\"\",0.42328,-10.02687,1600000,390441,177,85,false,-3.1717,0.7185,2013-10-04T03:34:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02407,snoR122,1264,Small nucleolar RNA snoR122,Eberhardt R,Predicted,36.0,36.2,35.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.69 -Z 549862.597050 CM SEQDB,4,94,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold,65,0,,\"\",0.47175,-9.58169,1600000,323241,160,74,false,-3.374,0.7187,2013-10-04T03:34:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02408,snoR124,1264,Small nucleolar RNA snoR124,Eberhardt R,Predicted,70.0,92.6,38.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.41 -Z 549862.597050 CM SEQDB,5,15,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold,10,0,,\"\",0.52801,-4.14427,1600000,244653,167,77,true,-3.4757,0.71855,2013-10-04T03:34:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02409,snoR125,1264,Small nucleolar RNA snoR125,Eberhardt R,Predicted,100.0,183.5,66.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,11,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; mfold,10,0,,\"\",0.62219,-7.92271,1600000,317721,330,200,true,-3.5541,0.71232,2013-10-04T03:34:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02410,snoR136,1264,Small nucleolar RNA snoR136,Eberhardt R,Predicted,43.0,43.8,42.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,167,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold,73,0,,\"\",0.51306,-7.26335,1600000,336574,259,152,true,-2.9415,0.71819,2013-10-04T03:34:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02411,snoR138,1264,Small nucleolar RNA snoR138,Eberhardt R,Predicted,44.0,44.3,43.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,92,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold,61,0,,\"\",0.54442,-7.86976,1600000,242316,270,154,true,-3.1521,0.71812,2013-10-04T03:34:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02412,snoR144,1264,Small nucleolar RNA snoR144,Eberhardt R,Predicted,100.0,140.8,43.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,15,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold,10,0,,\"\",0.55579,-7.2966,1600000,290013,260,144,true,-3.4015,0.71804,2013-10-04T03:34:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02413,snoR145,1264,Small nucleolar RNA snoR145,Eberhardt R,Predicted,57.0,60.1,52.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,19,91,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; mfold,72,0,,\"\",0.48327,-9.0235,1600000,300813,256,147,true,-3.132,0.71818,2013-10-04T03:34:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02414,rli60,1267,Listeria sRNA rli60,\"Osuch I, Eberhardt R\",INFERNAL,60.0,66.9,56.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,23,0,0,Gene; sRNA;,Predicted;RNAalifold,31,0,,\"\",0.45324,-7.41873,1600000,375056,514,191,true,-3.1031,0.71316,2013-10-04T03:34:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02415,rliG,1267,Listeria sRNA rliG,\"Osuch I, Eberhardt R\",Predicted; ClustalW,55.0,55.6,54.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.42 -Z 549862.597050 CM SEQDB,5,8,0,0,Gene; sRNA;,Predicted; RNAalifold,13,0,,\"\",0.56481,-9.68397,1600000,382973,436,267,true,-3.2857,0.70739,2013-10-04T03:34:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02416,TCV_H4,2339,Turnip crinkle virus 3'UTR,Eberhardt R,Eberhardt R,50.0,57.1,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Published; PMID:19656866,2,0,,\"\",0.56934,-5.27275,1600000,376946,134,51,true,-2.8342,0.73778,2013-10-04T03:34:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02417,VR-RNA,2340,VR-RNA,Eberhardt R,Eberhardt R,100.0,396.2,46.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; mfold,4,0,,\"\",0.59255,-8.87429,1600000,418960,570,397,true,-4.0369,0.70277,2013-10-04T03:34:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02418,Spd-sr07,2341,Streptococcus sRNA Spd-sr07,Eberhardt R,Predicted; WAR,55.0,61.6,42.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; WAR,6,0,,\"\",0.46614,-8.91192,1600000,375413,149,63,true,-3.3649,0.71913,2013-10-04T03:34:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02419,Spd-sr37,2341,Streptococcus sRNA Spd-sr37,Eberhardt R,Predicted; WAR,45.0,46.3,36.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,25,44,0,0,Gene; sRNA;,Predicted; WAR,66,0,,\"\",0.50578,-8.36537,1600000,357351,148,67,true,-3.7284,0.71905,2013-10-04T03:34:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02420,Bp1_162,1236,Burkholderia sRNA Bp1_Cand612_SIPHT,Eberhardt R,Predicted; ClustalW,50.0,50.3,36.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,11,54,0,0,Gene; sRNA;,Predicted; RNAalifold,59,0,,\"\",0.53758,-6.07281,1600000,328963,199,100,true,-3.3906,0.71852,2013-10-04T03:34:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02421,Bp1_684,1236,Burkholderia sRNA Bp1_Cand684_SIPHT,Eberhardt R,Predicted; ClustalW,44.0,44.3,43.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,158,0,0,Gene; sRNA;,Predicted; RNAalifold,40,0,,\"\",0.6171,-7.71076,1600000,419419,217,113,true,-2.6889,0.71839,2013-10-04T03:34:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02422,Bp1_738,1236,Burkholderia sRNA Bp1_Cand738_SIPHT,Eberhardt R,Predicted; ClustalW,45.0,47.1,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,54,0,0,Gene; sRNA;,Predicted; RNAalifold,66,0,,\"\",0.55749,-10.09354,1600000,448284,333,208,true,-3.313,0.71153,2013-10-04T03:34:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02423,Bp1_781,2451,Burkholderia sRNA Bp1_Cand871_SIPHT,Eberhardt R,Predicted; CMfinder,80.0,170.4,34.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,15,59,0,0,Gene; sRNA;,Predicted; CMfinder,67,0,,\"\",0.65675,-9.27574,1600000,371483,360,230,true,-3.4625,0.70989,2013-10-04T03:34:47.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02424,Bp2_287,1236,Burkholderia sRNA Bp2_Cand287_SIPHT,Eberhardt R,Predicted; ClustalW,70.0,146.3,51.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,39,0,0,Gene; sRNA;,Predicted; RNAalifold,47,0,,\"\",0.59735,-9.41224,1600000,351056,348,174,true,-3.1861,0.71532,2013-10-04T03:34:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02425,SpF01_sRNA,2341,Streptococcus sRNA SpF01,Eberhardt R,Eberhardt R,55.0,57.1,51.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,7,0,0,Gene; sRNA;,Predicted; mfold,19,0,,\"\",0.47957,-5.30137,1600000,295283,167,80,true,-3.6677,0.71882,2013-10-04T03:34:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02426,SpF03_sRNA,2341,Streptococcus sRNA SpF03,Eberhardt R,Eberhardt R,50.0,71.4,39.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,6,0,0,Gene; sRNA;,Predicted; mfold,7,0,,\"\",0.51103,-4.53427,1600000,242695,210,106,true,-3.2346,0.71852,2013-10-04T03:34:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02427,SpF10_sRNA,2341,Streptococcus sRNA SpF10,Eberhardt R,Eberhardt R,60.0,81.4,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,4,0,0,Gene; sRNA;,Predicted; mfold,6,0,,\"\",0.61589,-3.88945,1600000,290614,142,57,true,-2.8514,0.7279,2013-10-04T03:34:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02428,SpF11_sRNA,2341,Streptococcus sRNA SpF11,Eberhardt R,Eberhardt R,54.0,54.0,52.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; mfold,7,0,,\"\",0.50442,-5.63305,1600000,265086,194,95,true,-3.2082,0.71857,2013-10-04T03:34:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02429,SpF14_sRNA,2341,Streptococcus sRNA SpF14,Eberhardt R,Eberhardt R,100.0,107.8,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,3,0,0,Gene; sRNA;,Predicted; mfold,7,0,,\"\",0.53724,-3.16859,1600000,241652,226,120,true,-3.0138,0.71834,2013-10-04T03:34:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02430,SpF19_sRNA,2341,Streptococcus sRNA SpF19,Eberhardt R,Eberhardt R,60.0,89.1,53.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; mfold,7,0,,\"\",0.61507,-2.29815,1600000,197360,197,99,true,-3.142,0.71873,2013-10-04T03:34:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02431,SpF22_sRNA,2341,Streptococcus sRNA SpF22,Eberhardt R,Eberhardt R,55.0,59.6,43.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,22,9,0,0,Gene; sRNA;,Predicted; mfold,29,0,,\"\",0.55712,-8.02332,1600000,322375,338,140,true,-3.0457,0.71813,2013-10-04T03:35:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02432,SpF25_sRNA,2341,Streptococcus sRNA SpF25,Eberhardt R,Eberhardt R,48.0,48.9,43.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,15,0,0,Gene; sRNA;,Predicted; mfold,27,0,,\"\",0.50569,-6.69313,1600000,241555,197,102,true,-3.0299,0.71859,2013-10-04T03:35:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02433,SpF36_sRNA,2341,Streptococcus sRNA SpF36,Eberhardt R,Eberhardt R,45.0,46.6,44.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,5,0,0,Gene; sRNA;,Predicted; mfold,9,0,,\"\",0.52413,-3.64452,1600000,326190,128,51,true,-3.8759,0.71954,2013-10-04T03:35:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02434,SpF39_sRNA,2341,Streptococcus sRNA SpF39,Eberhardt R,Eberhardt R,60.0,64.5,53.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,13,0,0,Gene; sRNA;,Predicted; mfold,22,0,,\"\",0.45116,-6.06342,1600000,305065,250,109,true,-2.8469,0.71843,2013-10-04T03:35:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02435,SpF41_sRNA,2341,Streptococcus sRNA SpF41,Eberhardt R,Eberhardt R,52.0,53.6,50.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,9,0,0,Gene; sRNA;,Predicted; mfold,15,0,,\"\",0.49509,-6.98277,1600000,296809,201,105,true,-3.6292,0.71851,2013-10-04T03:35:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02436,SpF43_sRNA,2341,Streptococcus sRNA SpF43,Eberhardt R,Eberhardt R,70.0,76.4,54.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,11,0,0,Gene; sRNA;,Predicted; mfold,16,0,,\"\",0.6516,-2.24043,1600000,153308,197,99,true,-3.2599,0.71869,2013-10-04T03:35:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02437,SpF44_sRNA,2341,Streptococcus sRNA SpF44,Eberhardt R,Eberhardt R,70.0,85.2,45.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,12,8,0,0,Gene; sRNA;,Predicted; mfold,19,0,,\"\",0.53057,-7.36847,1600000,311636,200,102,true,-2.5266,0.71856,2013-10-04T03:35:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02438,SpF51_sRNA,2341,Streptococcus sRNA SpF51,Eberhardt R,Eberhardt R,50.0,51.2,37.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,8,0,0,Gene; sRNA;,Predicted; mfold,14,0,,\"\",0.5549,-7.66751,1600000,627805,118,45,true,-3.5236,0.71955,2013-10-04T03:35:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02439,SpF56_sRNA,2341,Streptococcus sRNA SpF56,Eberhardt R,Eberhardt R,80.0,107.3,37.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; mfold,3,0,,\"\",0.54432,-3.95339,1600000,242474,201,99,true,-2.8505,0.71862,2013-10-04T03:35:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02440,SpF59_sRNA,2341,Streptococcus sRNA SpF59 (ldcC RNA),Eberhardt R,Eberhardt R,37.2,37.2,36.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.01 -Z 742849.287494 --mxsize 128 CM SEQDB,410,140,0,0,Gene; sRNA;,Predicted; mfold,194,0,,\"\",0.53524,-9.06415,1600000,523518,188,57,true,-3.6646,0.71908,2013-10-04T03:35:16.000Z,2019-01-21T11:11:38.000Z\n",
            "RF02441,SpF61_sRNA,2341,Streptococcus sRNA SpF61,Eberhardt R,Eberhardt R,60.0,82.1,49.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; mfold,7,0,,\"\",0.60137,-2.74399,1600000,461639,140,58,true,-3.6353,0.71964,2013-10-04T03:35:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02442,SpF66_sRNA,2341,Streptococcus sRNA SpF66,Eberhardt R,Eberhardt R,42.0,42.1,41.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,45,0,0,Gene; sRNA;,Predicted; mfold,43,0,,\"\",0.63072,-8.49896,1600000,319331,375,239,true,-3.8414,0.70924,2013-10-04T03:35:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02443,SpR08_sRNA,2341,Streptococcus sRNA SpR08,Eberhardt R,Eberhardt R,80.0,104.4,39.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; sRNA;,Predicted; mfold,5,0,,\"\",0.5496,-6.22524,1600000,475842,164,73,true,-3.3788,0.71891,2013-10-04T03:35:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02444,SpR10_sRNA,2341,Streptococcus sRNA SpR10,Eberhardt R,Eberhardt R,50.0,66.3,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,4,0,0,Gene; sRNA;,Predicted; mfold,9,0,,\"\",0.68982,-4.43871,1600000,453338,122,47,true,-3.7713,0.7195,2013-10-04T03:35:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02445,SpR14_sRNA,2341,Streptococcus sRNA SpR14,Eberhardt R,Eberhardt R,50.0,80.7,44.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,3,0,0,Gene; sRNA;,Predicted; mfold,8,0,,\"\",0.50173,-3.80228,1600000,226558,171,80,true,-3.2464,0.71883,2013-10-04T03:35:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02446,SpR18_sRNA,2341,Streptococcus sRNA SpR18,Eberhardt R,Eberhardt R,60.0,63.3,51.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,75,0,0,Gene; sRNA;,Predicted; mfold,51,0,,\"\",0.47298,-6.00886,1600000,301307,237,132,true,-2.7194,0.71812,2013-10-04T03:35:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02447,SpR19_sRNA,2341,Streptococcus sRNA SpR19,Eberhardt R,Eberhardt R,79.0,79.1,77.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.53 -Z 549862.597050 CM SEQDB,23,23,0,0,Gene; sRNA;,Predicted; mfold,43,0,,\"\",0.5243,-7.65969,1600000,299550,369,123,true,-2.5399,0.71832,2013-10-04T03:35:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02448,SpR20_sRNA,2341,Streptococcus sRNA SpR20,Eberhardt R,Eberhardt R,45.0,47.7,38.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,7,3,0,0,Gene; sRNA;,Predicted; mfold,10,0,,\"\",0.61734,-6.96467,1600000,460366,118,45,true,-3.9715,0.71957,2013-10-04T03:35:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02449,ncr1015,1236,Bacillus sRNA ncr1015,Eberhardt R,Eberhardt R,60.0,67.9,51.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.56 -Z 549862.597050 CM SEQDB,16,48,0,0,Gene; sRNA;,Published; PMID:20525796,56,0,,\"\",0.46887,-7.01132,1600000,333143,224,121,true,-3.1848,0.71837,2013-10-04T03:35:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02450,ncr1175,1236,Bacillus sRNA ncr1175,Eberhardt R,Eberhardt R,60.0,61.4,48.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,4,0,0,Gene; sRNA;,Published; PMID:20525796,7,0,,\"\",0.51811,-7.9704,1600000,273666,224,110,true,-3.3454,0.71849,2013-10-04T03:35:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02451,ncr1241,1236,Bacillus sRNA ncr1241,Eberhardt R,Eberhardt R,60.0,64.7,39.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,16,0,0,Gene; sRNA;,Published; PMID:20525796,20,0,,\"\",0.49979,-8.39982,1600000,395094,140,63,true,-3.7872,0.71922,2013-10-04T03:35:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02452,ncr1575,1236,Bacillus sRNA ncr1575,Eberhardt R,Eberhardt R,60.0,69.7,50.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,29,79,0,0,Gene; sRNA;,Published; PMID:20525796,95,0,,\"\",0.47683,-7.50622,1600000,327930,340,215,true,-3.1252,0.71098,2013-10-04T03:35:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02453,ncr952,1236,Bacillus sRNA ncr952,Eberhardt R,Eberhardt R,100.0,140.3,37.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,1,0,0,Gene; sRNA;,Published; PMID:20525796,6,0,,\"\",0.61067,-8.1473,1600000,369104,258,143,true,-2.9474,0.71805,2013-10-04T03:35:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02454,ncr982,1236,Bacillus sRNA ncr982,Eberhardt R,Eberhardt R,50.0,53.3,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,6,0,0,Gene; sRNA;,Published; PMID:20525796,11,0,,\"\",0.49019,-5.58302,1600000,252942,167,79,true,-3.3629,0.71884,2013-10-04T03:35:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02455,Virus_CITE_1,2342,Dianthovirus RNA2 cap-independent translation element,Eberhardt R,Eberhardt R,80.0,111.4,32.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,0,0,0,Cis-reg;,Predicted; PMID:20561661,2,0,,\"\",0.60738,-4.8636,1600000,233137,191,93,true,-3.4081,0.71866,2013-10-04T03:35:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02456,Diantho_3_UTR,2073,Dianthovirus RNA2 3'UTR stem loops,Eberhardt R,Eberhardt R,100.0,205.0,33.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,0,0,0,Cis-reg;,Predicted; PMID:20561661,2,0,,\"\",0.61482,-10.19857,1600000,507798,337,205,true,-3.1652,0.71179,2013-10-04T03:35:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02457,Virus_CITE_2,2342,Tombusvirus 3' cap-independent translation element,Eberhardt R,Eberhardt R,100.0,116.7,29.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,1,0,0,Cis-reg;,Predicted; PMID:23192876,9,0,,\"\",0.62969,-6.0966,1600000,247784,284,167,true,-3.3033,0.71614,2013-10-04T03:35:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02458,Virus_CITE_3,2342,Aureusvirus cap-independent translation element,Eberhardt R,Eberhardt R,100.0,117.3,34.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,3,0,0,Cis-reg;,Predicted; PMID:23192876,4,0,,\"\",0.6625,-4.16775,1600000,231137,228,122,true,-3.2508,0.71831,2013-10-04T03:35:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02459,Virus_CITE_4,2342,Necrovirus cap-independent translation element,Eberhardt R,Eberhardt R,40.0,44.5,34.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,6,0,0,Cis-reg;,Predicted; PMID:23192876,7,0,,\"\",0.63947,-3.41665,1600000,211753,174,85,true,-3.4701,0.71886,2013-10-04T03:35:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02460,Virus_CITE_5,2342,Satellite tobacco necrosis virus cap-independent translation element,Eberhardt R,Eberhardt R,90.0,108.3,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,0,0,0,Cis-reg;,Predicted; PMID:18031280,1,0,,\"\",0.63999,-2.84367,1600000,202855,192,94,true,-3.0961,0.71877,2013-10-04T03:35:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02461,Virus_CITE_6,2342,Blackcurrant reversion virus cap-independent translation element,Eberhardt R,Eberhardt R,41.0,41.5,37.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.47 -Z 549862.597050 CM SEQDB,3,0,0,0,Cis-reg;,Predicted; PMID:18031280,1,0,,\"\",0.5337,-8.16635,1600000,310877,220,115,true,-3.3968,0.71855,2013-10-04T03:35:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02462,Telomerase_Asco,2265,Ascomycota telomerase RNA,Eberhardt R,Eberhardt R,30.0,31.3,29.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,186,0,0,Gene;,Predicted; PMID:23093598,181,0,,\"\",0.70977,-11.59524,1600000,496782,2352,1859,true,-6.61,0.6952,2013-10-04T03:35:53.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02463,Ms_AS-1,1236,Mycobacterium sRNA Ms_AS-1,Eberhardt R,Eberhardt R,37.0,37.0,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,104,0,0,Gene; sRNA;,Predicted; PMID:23169799,98,0,,\"\",0.5445,-8.85093,1600000,329509,195,98,true,-3.452,0.71852,2013-10-04T03:35:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02464,Ms_AS-4,1236,Actinobacteria sRNA Ms_AS-4,Eberhardt R,Eberhardt R,50.0,51.0,48.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,91,0,0,Gene; sRNA;,Predicted; PMID:23169799,57,0,,\"\",0.45695,-10.38708,1600000,347965,170,82,true,-3.3005,0.71877,2013-10-04T03:35:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02465,Ms_AS-5,1236,Mycobacterium sRNA Ms_AS-5,Eberhardt R,Eberhardt R,46.0,46.3,45.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,72,0,0,Gene; sRNA;,Predicted; PMID:23169799,76,0,,\"\",0.5374,-8.61729,1600000,497935,116,45,true,-3.6733,0.71946,2013-10-04T03:35:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02466,Ms_AS-8,1236,Actinobacteria sRNA Ms_AS-8,Eberhardt R,Eberhardt R,48.0,49.4,43.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,91,0,0,Gene; sRNA;,Predicted; PMID:23169799,48,0,,\"\",0.58245,-6.58251,1600000,257803,159,74,true,-3.4819,0.71901,2013-10-04T03:36:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02467,Ms_IGR-2,1236,Mycobacterium sRNA Ms_IGR-2,Eberhardt R,Eberhardt R,45.0,55.3,32.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,32,0,0,Gene; sRNA;,Predicted; PMID:23169799,34,0,,\"\",0.62587,-6.76302,1600000,312622,175,82,true,-3.3531,0.71862,2013-10-04T03:36:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02468,Ms_IGR-4,1236,Mycobacterium sRNA Ms_IGR-4,Eberhardt R,Eberhardt R,55.0,61.0,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,6,0,0,Gene; sRNA;,Predicted; PMID:23169799,7,0,,\"\",0.57836,-4.94968,1600000,359898,116,38,true,-2.4304,0.74547,2013-10-04T03:36:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02469,Ms_IGR-7,1236,Actinobacteria sRNA Ms_IGR-7,Eberhardt R,Eberhardt R,45.0,45.4,44.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,236,0,0,Gene; sRNA;,Predicted; PMID:23169799,111,0,,\"\",0.57934,-7.37686,1600000,312103,215,118,true,-3.5138,0.71846,2013-10-04T03:36:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02470,Ms_IGR-8,1236,Mycobacterium sRNA Ms_IGR-8,Eberhardt R,Eberhardt R,44.0,44.0,43.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,39,0,0,Gene; sRNA;,Predicted; PMID:23169799,17,0,,\"\",0.59226,-7.47749,1600000,241057,147,63,true,-3.261,0.72782,2013-10-04T03:36:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02471,Ms_IGR-5,1236,Actinobacteria sRNA Ms_IGR-5,Eberhardt R,Eberhardt R,45.0,45.1,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,543,0,0,Gene; sRNA;,Predicted; PMID:23169799,552,0,,\"\",0.59086,-9.86186,1600000,358519,444,184,true,-3.2162,0.71403,2013-10-04T03:36:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02472,Gl_RNase_MRP,1327,Giardia RNase MRP,\"Eberhardt R, Nawrocki EP\",Published; PMID: 22053856,60.0,239.5,47.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.52 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; ribozyme;,Published; PMID: 22053856,2,0,,\"\",0.64461,-7.95677,1600000,316041,320,189,true,-3.1179,0.71354,2013-10-04T03:36:11.000Z,2021-06-14T15:02:31.000Z\n",
            "RF02473,GlsR4,1264,Small nucleolar RNA GlsR4,Eberhardt R,Predicted,70.0,76.1,36.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.55 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR,3,0,,\"\",0.54177,-8.48379,1600000,295667,145,61,false,-2.804,0.72765,2013-10-04T03:36:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02474,GlsR5,1264,Small nucleolar RNA GlsR5,Eberhardt R,Predicted,100.0,114.1,31.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.23 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR,3,0,,\"\",0.60713,-8.28784,1600000,324914,184,89,false,-3.4919,0.71844,2013-10-04T03:36:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02475,GlsR6,1264,Small nucleolar RNA GlsR6,Eberhardt R,Predicted,75.0,87.9,34.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.97 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR,4,0,,\"\",0.61502,-9.07931,1600000,570178,139,59,false,-3.8635,0.71896,2013-10-04T03:36:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02476,GlsR7,1264,Small nucleolar RNA GlsR7,Eberhardt R,Predicted,75.0,88.9,33.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.15 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR,4,0,,\"\",0.73465,-3.34316,1600000,395247,141,59,true,-3.2166,0.71904,2013-10-04T03:36:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02477,GlsR9,1264,Small nucleolar RNA GlsR9,Eberhardt R,Predicted,95.0,107.7,33.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.10 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR,3,0,,\"\",0.69923,-2.7043,1600000,199951,172,79,true,-3.4261,0.71943,2013-10-04T03:36:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02478,GlsR10,1264,Small nucleolar RNA GlsR10,Eberhardt R,Predicted,75.0,86.3,33.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.93 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR,3,0,,\"\",0.68209,-2.88114,1600000,254030,157,68,true,-3.5846,0.72232,2013-10-04T03:36:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02479,GlsR13,1264,Small nucleolar RNA GlsR13,Eberhardt R,Predicted,100.0,128.5,32.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.82 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR,4,0,,\"\",0.56103,-9.16493,1600000,316175,198,100,false,-3.3673,0.71826,2013-10-04T03:36:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02480,GlsR14,1264,Small nucleolar RNA GlsR14,Eberhardt R,Predicted,90.0,99.7,34.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.64 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; WAR,4,0,,\"\",0.63497,-8.23872,1600000,503884,152,68,false,-3.6637,0.71878,2013-10-04T03:36:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02481,GlsR18,1264,Small nucleolar RNA GlsR18,Eberhardt R,Predicted,100.0,126.9,31.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,4,0,,\"\",0.64319,-5.37622,1600000,232454,265,147,true,-3.0429,0.71821,2013-10-04T03:36:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02482,GlsR19,1264,Small nucleolar RNA GlsR19,Eberhardt R,Predicted,100.0,180.4,30.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,4,0,,\"\",0.62274,-8.51048,1600000,393015,269,150,true,-3.1606,0.71793,2013-10-04T03:36:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02483,GlsR20,1264,Small nucleolar RNA GlsR20,Eberhardt R,Predicted,100.0,128.2,31.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,4,0,,\"\",0.63231,-5.37538,1600000,252442,211,107,true,-3.3049,0.7185,2013-10-04T03:36:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02484,GlsR21,1264,Small nucleolar RNA GlsR21,Eberhardt R,Predicted,100.0,119.4,35.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,4,0,,\"\",0.56396,-6.76528,1600000,269443,217,112,true,-3.1723,0.71843,2013-10-04T03:36:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02485,GlsR22,1264,Small nucleolar RNA GlsR22,Eberhardt R,Predicted,100.0,179.0,37.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,4,0,,\"\",0.5764,-5.65995,1600000,264223,269,150,true,-3.1467,0.71806,2013-10-04T03:36:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02486,GlsR23,1264,Small nucleolar RNA GlsR23,Eberhardt R,Predicted,100.0,121.4,38.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,3,0,,\"\",0.59935,-7.16214,1600000,317504,268,149,true,-2.9949,0.71792,2013-10-04T03:36:35.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02487,GlsR24,1264,Small nucleolar RNA GlsR24,Eberhardt R,Predicted,100.0,135.1,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,4,0,,\"\",0.67761,-6.7813,1600000,269389,269,147,true,-2.6021,0.71825,2013-10-04T03:36:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02488,GlsR25,1264,Small nucleolar RNA GlsR25,Eberhardt R,Predicted,100.0,143.8,33.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,4,0,,\"\",0.55086,-8.13368,1600000,333136,264,146,true,-2.9316,0.71797,2013-10-04T03:36:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02489,GlsR26,1264,Small nucleolar RNA GlsR26,Eberhardt R,Predicted,100.0,125.8,31.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,4,0,,\"\",0.53616,-8.54066,1600000,277318,219,114,true,-3.1583,0.71851,2013-10-04T03:36:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02490,GlsR27,1264,Small nucleolar RNA GlsR27,Eberhardt R,Predicted,100.0,139.0,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; WAR,3,0,,\"\",0.57074,-7.90855,1600000,290093,220,113,true,-2.8953,0.71847,2013-10-04T03:36:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02491,Gl_U1,1304,Giardia U1 spliceosomal RNA,Eberhardt R,Eberhardt R,100.0,141.6,32.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; splicing;,Predicted; PMID:23019220,3,0,,\"\",0.73991,-3.42785,1600000,208226,230,121,true,-3.0134,0.71846,2013-10-04T03:36:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02492,Gl_U2,1305,Giardia U2 spliceosomal RNA,Eberhardt R,Eberhardt R,100.0,172.0,30.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; splicing;,Predicted; PMID:23019220,3,0,,\"\",0.61369,-8.22899,1600000,323665,252,136,true,-3.1256,0.71807,2013-10-04T03:36:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02493,Gl_U4,1314,Giardia U4 spliceosomal RNA,Eberhardt R,Eberhardt R,100.0,126.2,34.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; splicing;,Predicted; PMID:23019220,3,0,,\"\",0.59992,-5.5653,1600000,250162,195,95,true,-3.1276,0.71846,2013-10-04T03:36:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02494,Gl_U6,1323,Giardia U6 spliceosomal RNA,Eberhardt R,Eberhardt R,100.0,133.4,33.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; snRNA; splicing;,Predicted; PMID:23019220,3,0,,\"\",0.5891,-7.38354,1600000,345311,205,102,true,-3.3337,0.71854,2013-10-04T03:36:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02495,ohsC_RNA,2266,Oppression of Hydrophobic ORF by sRNA,Gardner P,Gardner P,50.0,51.6,45.1,,RyfC,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,34,3,0,0,Gene; antitoxin;,Published; PMID:19052321,37,0,,\"\",0.48435,-7.87897,1600000,404841,163,77,true,-3.2595,0.71906,2013-10-04T03:36:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02496,Atu_At1,1236,Rhizobiales sRNA Atu_At1,Eberhardt R,Predicted; CMfinder,60.0,70.4,55.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,33,135,0,0,Gene; sRNA;,Predicted; CMfinder,126,0,,\"\",0.49566,-7.02632,1600000,238773,198,108,true,-3.7146,0.71844,2013-10-04T03:36:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02497,Atu_C10,1236,Rhizobiales sRNA Atu_C10,Eberhardt R,Predicted; CMfinder,60.0,120.0,42.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,4,0,0,Gene; sRNA;,Predicted; CMfinder,5,0,,\"\",0.56911,-6.80236,1600000,258841,187,89,true,-3.2927,0.71874,2013-10-04T03:36:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02498,Atu_C3,1236,Rhizobiales sRNA Atu_C3,Eberhardt R,Predicted; CMfinder,100.0,169.4,41.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; sRNA;,Predicted; CMfinder,3,0,,\"\",0.57241,-9.5583,1600000,375926,278,156,true,-3.0375,0.71798,2013-10-04T03:36:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02499,Atu_C4,1236,Rhizobiales sRNA Atu_C4,Eberhardt R,Predicted; CMfinder,80.0,99.9,36.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,2,0,0,Gene; sRNA;,Predicted; CMfinder,3,0,,\"\",0.5564,-8.42862,1600000,525067,160,70,true,-3.1263,0.72196,2013-10-04T03:36:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02501,Atu_C7,1236,Rhizobiales sRNA Atu_C7,Eberhardt R,Predicted; WAR,45.0,50.8,33.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,47,0,0,Gene; sRNA;,Predicted; WAR,49,0,,\"\",0.60879,-9.97264,1600000,465600,279,157,true,-2.9255,0.71741,2013-10-04T03:37:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02502,Atu_C8,1236,Rhizobiales sRNA Atu_C8,Eberhardt R,Predicted; CMfinder,50.0,54.4,48.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,27,167,0,0,Gene; sRNA;,Predicted; CMfinder,158,0,,\"\",0.62758,-3.92433,1600000,223409,223,122,true,-3.4439,0.71852,2013-10-04T03:37:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02503,Atu_C9,1236,Rhizobiales sRNA Atu_C9,Eberhardt R,Predicted; WAR,40.0,41.3,36.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,9,66,0,0,Gene; sRNA;,Predicted; WAR,71,0,,\"\",0.65429,-10.19448,1600000,484483,318,192,true,-3.3018,0.71295,2013-10-04T03:37:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02504,Atu_L1,1236,Rhizobiales sRNA Atu_L1,Eberhardt R,Predicted; CMfinder,60.0,70.5,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,11,0,0,Gene; sRNA;,Predicted; CMfinder,13,0,,\"\",0.62488,-8.1369,1600000,315905,300,174,true,-3.1859,0.71508,2013-10-04T03:37:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02505,Atu_L6,1236,Rhizobiales sRNA Atu_L6,Eberhardt R,Predicted; WAR,100.0,122.5,39.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; WAR,4,0,,\"\",0.60897,-7.85171,1600000,292571,233,124,true,-3.0477,0.71842,2013-10-04T03:37:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02506,Atu_Ti1,1236,Rhizobiales sRNA Atu_Ti1,Eberhardt R,Predicted; WAR,55.0,60.1,41.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,14,45,0,0,Gene; sRNA;,Predicted; WAR,41,0,,\"\",0.45181,-10.29561,1600000,320719,182,92,true,-2.9851,0.71874,2013-10-04T03:37:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02507,Atu_Ti3,1236,Rhizobiales sRNA Atu_Ti3,Eberhardt R,Predicted; WAR,50.0,51.3,43.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,4,12,0,0,Gene; sRNA;,Predicted; WAR,14,0,,\"\",0.67559,-3.16128,1600000,231514,183,89,true,-3.6934,0.71851,2013-10-04T03:37:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02508,Atu_Ti4,1236,Rhizobiales sRNA Atu_Ti4,Eberhardt R,Predicted; WAR,40.0,42.5,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; WAR,3,0,,\"\",0.62269,-4.71436,1600000,232144,211,107,true,-3.3452,0.71835,2013-10-04T03:37:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02509,PYLIS_2,2246,Pyrrolysine insertion sequence mtbB,Eberhardt R,Predicted; WAR,36.0,39.3,31.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,70,0,0,Cis-reg;,Predicted; WAR,47,0,,\"\",0.67654,-5.09762,1600000,251102,194,100,true,-3.4381,0.7185,2013-10-04T03:37:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02510,PYLIS_3,2246,Pyrrolysine insertion sequence mttB,Eberhardt R,Predicted; CMfinder,38.0,38.0,37.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,23,367,0,0,Cis-reg;,Predicted; CMfinder,262,0,,\"\",0.57183,-9.07526,1600000,394844,188,100,true,-3.4282,0.71831,2013-10-04T03:37:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02511,PYLIS_4,2246,Pyrrolysine insertion sequence TetR,Eberhardt R,Predicted; WAR,50.0,56.1,38.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,3,0,0,Cis-reg;,Predicted; WAR,2,0,,\"\",0.54419,-4.46025,1600000,260890,201,100,true,-2.8748,0.71839,2013-10-04T03:37:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02512,PYLIS_5,2246,Pyrrolysine insertion sequence transposase 1,Eberhardt R,Predicted; WAR,60.0,78.5,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,17,17,0,0,Cis-reg;,Predicted; WAR,3,0,,\"\",0.50043,-7.35393,1600000,280751,201,100,true,-3.4297,0.71847,2013-10-04T03:37:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02513,PYLIS_6,2246,Pyrrolysine insertion sequence transposase 2,Eberhardt R,Predicted; WAR,80.0,136.5,42.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,4,0,0,Cis-reg;,Predicted; WAR,2,0,,\"\",0.53774,-4.64359,1600000,278121,202,100,true,-3.0471,0.71848,2013-10-04T03:37:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02514,5_ureB_sRNA,2343,5' ureB small RNA,Eberhardt R,Eberhardt R,40.0,40.0,39.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,2642,0,0,Gene; sRNA;,Predicted; mfold;,2457,0,,\"\",0.69232,-8.35712,1600000,368631,434,292,true,-4.0883,0.70632,2013-10-04T03:37:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02515,AfaR,2344,AfaR small RNA,Eberhardt R,Published; PMID:23563153,100.0,295.6,58.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,0,0,0,Gene; sRNA;,Predicted; PMID:23563153,2,0,,\"\",0.49936,-10.18895,1600000,434508,375,234,true,-3.8255,0.70948,2013-10-04T03:37:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02516,mir-393,1287,mir-393 microRNA precursor family,Eberhardt R,miRBase,60.0,60.0,59.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,27,362,0,0,Gene; miRNA;,Predicted; miRBase,75,0,,\"\",0.64362,-4.28519,1600000,204066,240,129,true,-2.9398,0.71855,2013-10-04T03:37:32.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02517,mir-1017,1287,mir-1017 microRNA precursor family,Eberhardt R,miRBase,50.0,54.3,38.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,11,0,0,Gene; miRNA;,Predicted; miRBase,15,0,,\"\",0.62202,-4.47104,1600000,254456,261,71,true,-3.3508,0.71904,2013-10-04T03:37:34.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02518,mir-2494,1287,mir-2494 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,140.0,143.8,68.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58666,-4.23156,1600000,237016,220,113,true,-3.1197,0.7184,2013-10-04T03:37:36.000Z,2023-11-06T16:44:06.000Z\n",
            "RF02519,ToxI,2266,ToxI antitoxin,Eberhardt R,Published; PMID:23267117,50.0,54.7,35.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,2,0,0,Gene; antitoxin;,Published; PMID:23267117,7,0,,\"\",0.52336,-9.54935,1600000,792362,103,34,true,-3.6394,0.72867,2013-10-04T03:37:38.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02520,mir-965,1287,mir-965 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,43.0,43.1,42.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,87,0,0,Gene; miRNA;,Predicted; RNAalifold,83,0,,\"\",0.62754,-3.623,1600000,183454,156,77,true,-3.987,0.71898,2013-10-04T03:37:40.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02521,Virus_CITE_7,2342,Pea enation mosaic virus-2 cap-independent translation element,Eberhardt R,Eberhardt R,80.0,105.8,33.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Predicted; PMID:23682606,1,0,,\"\",0.67588,-2.438,1600000,205966,180,85,true,-3.6705,0.71881,2013-10-04T03:37:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02522,Virus_CITE_8,2342,Pea enation mosaic virus-2 cap-independent translation element,Eberhardt R,Eberhardt R,80.0,101.3,35.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Predicted; PMID:23682606,1,0,,\"\",0.62264,-3.98498,1600000,311130,161,71,true,-3.3136,0.72111,2013-10-04T03:37:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02523,ROSE_3,1694,Repression of heat shock gene expression (ROSE) element,Eberhardt R,Published; PMID:23741480,43.0,43.3,41.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,73,0,0,Cis-reg; thermoregulator;,Predicted; PMID:23741480,77,0,,\"\",0.58548,-7.20641,1600000,288588,146,69,true,-3.5626,0.71911,2013-10-04T03:37:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02524,sagA,1236,Streptococcus sRNA sagA,Eberhardt R,Predicted; CMfinder,60.0,76.9,47.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,6,0,0,Gene; sRNA;,Predicted; CMfinder,12,0,,\"\",0.455,-6.97764,1600000,267574,277,161,true,-3.0748,0.71701,2013-10-04T03:37:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02525,SSRC30,1236,Streptococcus sRNA SSRC30,Eberhardt R,Predicted; CMfinder,100.0,182.7,37.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; CMfinder,4,0,,\"\",0.52954,-6.69019,1600000,282811,272,151,true,-3.0266,0.71804,2013-10-04T03:37:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02526,SSRC34_1,1236,Streptococcus sRNA SSRC34,Eberhardt R,Predicted; CMfinder,64.0,65.7,60.9,It is based on computationally predicted coordinates (RNAz).,SSRC34;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,16,22,0,0,Gene; sRNA;,Predicted; CMfinder,32,0,,\"\",0.46931,-6.90243,1600000,301724,243,128,true,-2.8628,0.71815,2013-10-04T03:37:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02527,SSRC38,1236,Streptococcus sRNA SSRC38,Eberhardt R,Predicted; CMfinder,80.0,123.3,58.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; CMfinder,6,0,,\"\",0.49843,-9.61363,1600000,325372,382,166,true,-3.4138,0.71618,2013-10-04T03:37:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02528,SSRC41,1236,Streptococcus sRNA SSRC41,Eberhardt R,Predicted; CMfinder,60.0,60.4,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,8,10,0,0,Gene; sRNA;,Predicted; CMfinder,15,0,,\"\",0.44479,-6.6276,1600000,295645,212,111,true,-2.874,0.71832,2013-10-04T03:37:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02529,SSRC8_1,1236,Streptococcus sRNA SSRC8,Eberhardt R,Predicted; CMfinder,80.0,89.0,43.1,It is based on computationally predicted coordinates (RNAz).,SSRC8;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; CMfinder,4,0,,\"\",0.55594,-6.5278,1600000,313821,270,150,true,-2.9083,0.718,2013-10-04T03:37:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02530,URE2_IRES,2345,Ure2 internal ribosome entry site (IRES),Eberhardt R,Published; PMID:15314170,70.0,100.5,44.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,9,0,0,Cis-reg; IRES;,Published; PMID:15314170,13,0,,\"\",0.55922,-3.15443,1600000,277954,207,105,true,-3.3348,0.71837,2013-10-04T03:37:57.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02531,NRF2_IRES,2346,Nrf2 internal ribosome entry site (IRES),Eberhardt R,Published; PMID:19934254,50.0,52.6,44.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,20,25,0,0,Cis-reg; IRES;,Published; PMID:19934254,34,0,,\"\",0.43501,-11.7318,1600000,360768,300,128,true,-3.185,0.7182,2013-10-04T03:37:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02532,MNV_3UTR,2347,Murine norovirus 3'UTR,Eberhardt R,Published; PMID:20053745,70.0,73.3,56.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,10,0,0,0,Cis-reg;,Published; PMID:20053745,10,0,,\"\",0.59209,-8.89352,1600000,299252,365,222,true,-3.5337,0.71054,2013-10-04T03:38:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02533,HAV_CRE,2348,Hepatitis A virus (HAV) cis-acting replication element (CRE),Eberhardt R,PMID:18684812,55.0,55.4,44.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Published; PMID:18684812,2,0,,\"\",0.63657,-2.53398,1600000,216023,221,114,true,-3.2785,0.71856,2013-10-04T03:38:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02534,Noro_CRE,2349,Norovirus cis-acting replication element (CRE),Eberhardt R,PMID:19735574,35.0,36.6,34.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,21,1,0,0,Cis-reg;,Published; PMID:19735574,22,0,,\"\",0.71462,-3.27128,1600000,306258,127,52,true,-3.7997,0.71932,2013-10-04T03:38:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02535,ODC_IRES,2350,ODC internal ribosome entry site (IRES),Eberhardt R,Eberhardt R,44.0,44.3,42.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,13,94,0,0,Cis-reg; IRES;,Predicted; mfold,35,0,,\"\",0.48262,-7.96888,1600000,298931,264,142,true,-3.2728,0.71819,2013-10-04T03:38:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02536,AEV_CRE,2351,Avian encephalitis virus (AEV) cis-acting replication element (CRE),Eberhardt R,PMID:18684812,100.0,164.9,54.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Cis-reg;,Published; PMID:18684812,1,0,,\"\",0.60178,-4.81064,1600000,224021,250,135,true,-3.4766,0.71821,2013-10-04T03:38:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02537,TarA,2352,Vibrio ToxT activated RNA TarA,Eberhardt R,PMID:22546941,80.0,121.5,37.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Published; PMID:22546941,3,0,,\"\",0.50739,-8.45333,1600000,401041,200,99,true,-3.2226,0.71847,2013-10-04T03:38:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02538,TarB,2352,Vibrio ToxT activated RNA TarB,Eberhardt R,PMID:22546941,80.0,111.9,36.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,2,1,0,0,Gene; sRNA;,Published; PMID:22546941,3,0,,\"\",0.54093,-8.4363,1600000,434239,172,79,true,-3.816,0.71921,2013-10-04T03:38:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02539,SNOR75,1264,Plant small nucleolar RNA SNOR75,Griffiths-Jones SR,\"Griffiths-Jones SR; Plant snoRNA database, PMID:12520043\",50.0,50.2,46.3,\"U15 is a member of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. U15 is encoded within the introns of ribosomal protein S1 in Xenopus and S3 in human and Fugu. snoR75 from Arabidopsis [3] and homologues in rice and other plants are alternatively known as U15, and, despite a significantly shorter sequence length, appear to be related [4].\",U15,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.42 -Z 549862.597050 CM SEQDB,6,398,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; PFOLD; Griffiths-Jones SR, Daub J\",75,0,,\"\",0.63346,-3.18918,1600000,244518,182,88,true,-3.4663,0.71849,2014-05-01T10:08:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02540,LSU_rRNA_archaea,1459,Archaeal large subunit ribosomal RNA,\"Gutell RR, Nawrocki E\",Published; PMID:11869452,50.0,50.0,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.38 -Z 742849.287494 CM SEQDB,91,1370,0,0,Gene; rRNA;,Published; PMID:11869452,667,0,,\"\",0.7349,-8.17894,1600000,265300,3811,2987,true,-6.9503,0.69443,2014-06-11T16:01:24.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02541,LSU_rRNA_bacteria,1459,Bacterial large subunit ribosomal RNA,\"Gutell RR, Nawrocki E\",Published; PMID:11869452,50.0,50.0,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.38 -Z 549862.597050 --mxsize 512--noali CM SEQDB,102,25344,0,0,Gene; rRNA;,Published; PMID:11869452,7040,0,,\"\",0.77134,-7.79686,1600000,263856,5738,2925,true,-6.6859,0.69446,2014-06-11T16:04:37.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02542,SSU_rRNA_microsporidia,1459,Microsporidia small subunit ribosomal RNA,\"Gutell RR, Nawrocki E\",Published; PMID:11869452,50.0,50.0,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.38 -Z 742849.287494 CM SEQDB,46,353,0,0,Gene; rRNA;,Published; PMID:11869452,179,0,,\"\",0.71589,-8.31927,1600000,314746,1933,1311,true,-5.605,0.69607,2014-06-24T19:54:58.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02543,LSU_rRNA_eukarya,1459,Eukaryotic large subunit ribosomal RNA,\"Gutell RR, Nawrocki E\",Published; PMID:11869452,50.0,50.0,49.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.38 -Z 742849.287494 CM SEQDB,88,33233,0,0,Gene; rRNA;,Published; PMID:11869452,1077,0,,\"\",0.63537,-7.9863,1600000,240749,5746,3401,true,-7.2519,0.69427,2014-07-14T02:57:59.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02544,mt-tmRNA,1321,Mitochondrion-encoded tmRNA,Eberhardt R,Hafez M,65.0,72.7,57.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,11,28,0,0,Gene;,Hafez M,16,0,,\"\",0.57649,-3.08994,1600000,209693,231,77,true,-3.5553,0.71887,2014-08-07T09:51:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02545,SSU_trypano_mito,1459,Trypanosomatid mitochondrial small subunit ribosomal RNA,Eberhardt R,Predicted; ClustalW,60.0,72.6,51.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,12,0,0,Gene; rRNA;,Predicted; RNAalifold,12,0,,\"\",0.59111,-8.39876,1600000,401705,828,623,true,-4.6234,0.69928,2015-03-19T08:53:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02546,LSU_trypano_mito,1459,Trypanosomatid mitochondrial large subunit ribosomal RNA,Eberhardt R,Predicted; ClustalW,60.0,62.9,46.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,6,13,0,0,Gene; rRNA;,Predicted; RNAalifold,13,0,,\"\",0.5933,-9.66079,1600000,504993,750,561,true,-4.1584,0.69995,2015-03-19T13:53:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02547,mtPerm-5S,1302,Permuted mitochondrial genome encoded 5S rRNA,\"Valach M, Eberhardt R\",Valach M,39.0,39.7,36.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,6,1,0,0,Gene; rRNA;,Valach M,6,0,,\"\",0.50684,-8.25849,1600000,290946,207,106,true,-3.5532,0.71855,2015-04-22T13:14:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02548,Oskar_OES,2353,Oocyte entry signal (OES) from oskar mRNA,Argasinska J,\"Jambor et al, PMID:24572808\",46.0,46.0,43.9,\"In Drosophila, oskar mRNA harbours a stem-loop structure in the 3â€™UTR, called the oocyte entry signal (OES), that promotes dynein-based mRNA accumulation in the oocyte [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,14,10,0,0,Cis-reg;,Published; PMID:24572808,15,0,,\"\",0.594,-2.95856,1600000,234753,145,67,true,-3.9661,0.71922,2016-01-13T12:48:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02550,RnaG,2355,RnaG sRNA,Argasinska J,Argasinska J,153.0,153.2,38.7,\"Small regulatory RNA encoded by the virulence plasmid of S. flexneri, a Gram-negative pathogenic bacterium that causes human bacillary dysentery. It is 450 nt long and it contains a region with specific secondary structure that interacts with icsA mRNA and forms a transcription terminator. Acting as antisense, RnaG is transcribed from the complementary strand of its target, icsA mRNA [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,1,0,0,Gene; sRNA;,Published; PMID:20129941,4,0,,\"\",0.54363,-5.30948,1600000,232433,230,120,true,-2.9943,0.71825,2016-01-19T15:50:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02551,DapZ,1236,ABC transporter regulator,Argasinska J,Argasinska J,56.0,56.3,55.9,\"DapZ sRNA, encoded in the 3â€²-UTR of the conserved dapB gene, as a translational repressor with a GcvB-like seed domain. This is the first example of a widely conserved bacterial gene whose 3â€²-UTR has dual function. [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,6,58,0,0,Cis-reg;,Published; PMID:22922465,61,0,,\"\",0.52303,-7.03269,1600000,257887,165,78,true,-3.4851,0.71895,2016-01-20T11:01:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02552,RcsR1,2413,RcsR1 sRNA,Argasinska J,\"Baumgardt et al, PMID:26588798\",60.0,60.0,56.8,\"RcsR1 (rhizobial cold and salinity stress riboregulator 1) trans-acting sRNA, formerly known as SmelC587, is a stress-related riboregulator, conserved in Sinorhizobium, Rhizobium and Agrobacterium. It contains highly conserved stem-loops involved in the interaction with several target mRNAs (PhoR, MotE, anti-ÏƒE1, GntR, FgA, TrpC). In S. meliloti RcsR1 less conserved central region is responsible for the species-specific interaction with the 5â€™UTR of autoinducer synthase encoding mRNA sinI. The interaction negatively influences sinI translation. [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,50,0,0,Gene; sRNA;,Published; PMID:26588798,60,0,,\"\",0.58342,-6.72388,1600000,243098,193,96,true,-3.1636,0.71878,2016-02-01T09:34:14.000Z,2022-05-06T13:10:06.000Z\n",
            "RF02553,YrlA,1317,Y RNA-like,\"Argasinska J, Ontiveros-Palacios N\",Argasinska J,62.0,62.0,61.9,\"In Deinococcus radiodurans Rsr is tethered via Y RNA to the exoribonuclease PNPase and channels single-stranded RNA into the PNPase cavity. Rsr and Y RNA enhance degradation of structured RNAs by PNPase. This role could be conserved, as Rsr and ncRNAs called YrlA and YrlB (Y RNA like) also associate with PNPase in an evolutionary distant bacterium Salmonella Typhimurium [1]. The 3D structure of Y RNA-like was reported in PDB:6CU1 [3]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,118,0,0,Gene; sRNA;,Published; PMID:23540697,81,0,,\"\",0.66553,-2.9907,1600000,191633,215,114,true,-3.5191,0.71844,2016-02-03T12:37:55.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02554,ppoRNA,1302,mt-5S-like P. polycephalum,Argasinska J,Argasinska J,99.0,105.8,42.2,mtDNA-encoded RNA with 5S rRNA-like structures that is associated with the mitoribosome.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,0,0,0,Gene; rRNA;,Published; PMID:20304999,1,0,,\"\",0.59068,-3.67821,1600000,202831,199,98,true,-2.9986,0.71866,2016-02-09T13:13:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02555,hveRNA,1302,mt-5S-like H. vermiformis,Argasinska J,\"Argasinska, J\",90.0,99.1,40.6,mtDNA-encoded RNA with 5S rRNA-like structures that is associated with the mitoribosome.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,7,0,0,0,Gene; rRNA;,Published; PMID:20304999,1,0,,\"\",0.51317,-6.45767,1600000,277977,234,124,true,-3.2478,0.7182,2016-02-09T13:18:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02556,snaR-A,2358,snaR-A sRNA,Argasinska J,Argasinska J,98.0,98.0,97.3,Small NF90/ILF3-associated RNAs (snaR) are ~120 nucleotides long and are known to interact with NF90 double-stranded RNA-binding motifs. snaR-A is abundant in human testis and has been shown to associate with ribosomes in HeLa cells. snaR-A is present in human and gorilla but not in chimpanzee. Other snaR RNAs are found in African Great Apes (including chimpanzee and bonobo).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,40,0,0,Gene; sRNA;,Predicted; PMID:17855395,4,0,,\"\",0.65684,-3.67794,1600000,213733,223,115,true,-3.0444,0.7184,2016-02-24T12:15:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02557,CbSR1,2359,Coxiella burnetii sRNA 1,Argasinska J,Argasinska J,146.0,146.5,38.8,\"In bacteria small regulatory RNAs are activated during stress and virulence conditions. Coxiella burnetii small RNAs: CbSRs 1, 11, 12, and 14 encoded within intergenic region IGR, and CbSRs 2, 3, 4 and 9 located antisense to identified ORFs, are up-regulated during intracellular growth in host cells.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; CMfinder,2,0,,\"\",0.57514,-7.51254,1600000,285724,219,112,true,-3.0837,0.71838,2016-02-29T12:44:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02558,CbSR2,2359,Coxiella burnetii sRNA 2,Argasinska J,Argasinska J,300.0,303.5,39.0,\"In bacteria small regulatory RNAs are activated during stress and virulence conditions. Coxiella burnetii small RNAs: CbSRs 1, 11, 12, and 14 encoded within intergenic region IGR, and CbSRs 2, 3, 4 and 9 located antisense to identified ORFs, are up-regulated during intracellular growth in host cells.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; antisense;,Predicted; CMfinder,3,0,,\"\",0.67252,-8.18885,1600000,349633,387,243,true,-3.9298,0.70906,2016-02-29T14:58:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02559,CbSR4,2359,Coxiella burnetii sRNA 4,Argasinska J,Argasinska J,243.0,243.6,36.3,\"In bacteria small regulatory RNAs are activated during stress and virulence conditions. Coxiella burnetii small RNAs: CbSRs 1, 11, 12, and 14 encoded within intergenic region IGR, and CbSRs 2, 3, 4 and 9 located antisense to identified ORFs, are up-regulated during intracellular growth in host cells.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; antisense;,Predicted; CMfinder,2,0,,\"\",0.55536,-9.44775,1600000,323917,329,196,true,-3.3136,0.71278,2016-02-29T15:42:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02560,CbSR9,2359,Coxiella burnetii sRNA 9,Argasinska J,Argasinska J,279.0,279.1,38.1,\"In bacteria small regulatory RNAs are activated during stress and virulence conditions. Coxiella burnetii small RNAs: CbSRs 1, 11, 12, and 14 encoded within intergenic region IGR, and CbSRs 2, 3, 4 and 9 located antisense to identified ORFs, are up-regulated during intracellular growth in host cells.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; antisense;,Predicted; CMfinder,3,0,,\"\",0.52452,-9.66941,1600000,379500,350,213,true,-3.5455,0.71112,2016-02-29T16:32:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02561,CbSR12,2359,Coxiella burnetii sRNA 12,Argasinska J,Argasinska J,195.0,196.5,33.7,\"In bacteria small regulatory RNAs are activated during stress and virulence conditions. Coxiella burnetii small RNAs: CbSRs 1, 11, 12, and 14 encoded within intergenic region IGR, and CbSRs 2, 3, 4 and 9 located antisense to identified ORFs, are up-regulated during intracellular growth in host cells.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; CMfinder,2,0,,\"\",0.55248,-8.6221,1600000,338960,268,148,true,-3.1669,0.71805,2016-03-01T09:51:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02562,CbSR14,2359,Coxiella burnetii sRNA 14,Argasinska J,Argasinska J,172.0,172.7,39.6,\"In bacteria small regulatory RNAs are activated during stress and virulence conditions. Coxiella burnetii small RNAs (CbSRs 1, 11, 12, and 14) are encoded within intergenic region IGR. CbSRs 2, 3, 4 and 9 are located antisense to identified ORFs and are up-regulated during intracellular growth in host cells.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; CMfinder,3,0,,\"\",0.57596,-9.02811,1600000,336922,318,189,true,-3.3783,0.71341,2016-03-01T10:55:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02563,CbSR3,2359,Coxiella burnetii sRNA 3,Argasinska J,Argasinska J,101.0,101.1,100.9,\"In bacteria small regulatory RNAs are activated during stress and virulence conditions. Coxiella burnetii small RNAs (CbSRs 1, 11, 12, and 14) are encoded within intergenic region IGR. CbSRs 2, 3, 4 and 9 are located antisense to identified ORFs. CbSRs are up-regulated during intracellular growth in host cells.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,19,0,0,Gene; antisense;,Predicted; CMfinder,2,0,,\"\",0.6117,-8.38243,1600000,369106,331,198,true,-3.3099,0.71267,2016-03-01T15:37:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02564,naRNA4,2360,Nucleoid-associated noncoding RNA 4 (CssrE),Argasinska J,Argasinska J,59.5,59.5,59.4,In E. coli small RNA transcribed from repeated extragenic palindromic element (REP325) called nucleoid-associated ncRNA 4 (nRNA4) collaborates HU protein in condensing the DNA. The secondary structure but not the sequence of the RNA is important in nucleoid condensation. Also identified in Shigella as CssrE [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,167,0,0,Gene; sRNA;,Published; PMID:26307168,19,0,,\"\",0.51549,-9.62341,1600000,479374,170,77,true,-3.4522,0.71898,2016-03-02T12:21:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02565,YrlB,1317,Y RNA-like,Argasinska J,Argasinska J,97.0,97.0,36.3,\"In Deinococcus radiodurans Rsr is tethered via Y RNA to the exoribonuclease PNPase and channels single-stranded RNA into the PNPase cavity. Rsr and Y RNA enhance degradation of structured RNAs by PNPase. This role could be conserved, as Rsr and ncRNAs called YrlA and YrlB (Y RNA like) also associate with PNPase in an evolutionary distant bacterium Salmonella Typhimurium.[1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,2,0,0,Gene; sRNA;,Published; PMID:23540697,4,0,,\"\",0.63224,-3.45017,1600000,232921,171,78,true,-3.0835,0.72436,2016-03-07T11:16:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02566,Ms1,2361,Mycobacterium smegmatis small RNA 1,Argasinska J,Argasinska J,162.0,162.4,151.0,\"Mycobacterium smegmatis small RNA 1 is highly expressed during stationary phase of growth. Ms1 RNA directly interacts with RNA polymerase (RNAP), but in a different way than 6S, which is present in other bacteria. Ms1 does not require the presence of the main sigma factor for RNAP interaction. There is evidence that Ms1 RNA may function similar to 6S RNA in M. smegmatis which does not have 6S RNA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,111,0,0,Gene; sRNA;,Predicted; CMfinder,114,0,,\"\",0.65197,-9.5693,1600000,375488,472,304,true,-3.6817,0.70582,2016-03-08T11:35:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02567,Va-907,1236,Vibrio alginolyticus sRNA 907,Argasinska J,Argasinska J,45.0,60.0,32.4,Negative control of the adhesion process and negative control of key chemotaxis regulators translation. The number 907 comes from the Huang et al. name candidate 907.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; CMfinder,6,0,,\"\",0.70931,-8.12521,1600000,345208,469,311,true,-4.1169,0.70551,2016-03-09T11:14:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02568,UptR,2254,Escherichia coli small RNA (uptR) gene,Argasinska J,Argasinska J,100.0,101.1,34.5,UptR scRNA transcribed from the uptR gene is implicated in suppressing extracytoplasmic toxicity by reducing the amount of membrane-bound toxic hybrid protein.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,0,0,0,Gene; sRNA;,Predicted; PMID:11157926,2,0,,\"\",0.48246,-8.95754,1600000,324072,190,92,true,-3.2205,0.71872,2016-03-22T09:24:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02569,IhtA,2370,IhtA sRNA,Argasinska J,\"Tattersall et al, PMID:23071807\",76.0,76.4,38.8,\"A histone like protein HctA and HctB play role in controlling the differentiation between the two cell types. The expression of HctA is tightly regulated and repressed by small non-coding RNA, IhtA until the late RB to EB re-differentiation. The IhtA RNA is conserved across Chlamydia species.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,4,0,0,Gene; sRNA;,Predicted; RNAfold,7,0,,\"\",0.51285,-9.25764,1600000,324229,209,107,true,-3.2339,0.71864,2016-04-11T14:22:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02570,BM-sr0117,2371,Brucella melitensis small RNA 0117,Argasinska J,Argasinska J,112.0,112.2,31.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; PMID:26778206,4,0,,\"\",0.68782,-2.91155,1600000,209621,178,83,true,-3.3428,0.71886,2016-04-11T14:27:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02571,Mcr7,2178,mcr7 sRNA,Argasinska J,Argasinska J,380.0,385.0,35.2,Mcr7 sRNA encoded by the mcr7 gene modulates translation of the tatC mRNA and impacts the activity of the Twin Arginine Translocation (Tat) protein secretion apparatus.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold;,2,0,,\"\",0.62705,-9.89542,1600000,403318,517,351,true,-3.819,0.7042,2016-04-11T14:32:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02572,babR_5UTR,2371,Brucella babR 5'UTR,Argasinska J,Argasinska J,93.0,94.2,44.0,Hfq protein regulates virB operon which is required for full virulence of the bacteria. It can bind to 5' untranslated region of virB transcriptional regulator BabR and mediate its effects on babR expression.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Cis-reg;,Predicted; PMID:22020650,4,0,,\"\",0.53137,-5.89723,1600000,293857,249,134,true,-3.1585,0.71821,2016-04-11T14:38:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02573,LLnc147,2414,Lactococcus lactis non-coding RNA 147,Argasinska J,Argasinska J,127.0,127.6,37.4,Hundreds of novel RNAs were identified by Meulen et al. LLnc147 was shown to be involved in carbon uptake and metabolism.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,2,0,0,Gene; sRNA;,Predicted; RNAfold;,4,0,,\"\",0.51463,-8.14323,1600000,323134,204,102,true,-3.2409,0.71857,2016-05-03T12:32:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02574,Ricks_sRNA10,2415,Rickettsia sRNA 10,Argasinska J,Argasinska J,97.0,97.1,96.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,2,0,0,Gene; sRNA;,Predicted; RNAfold,5,0,,\"\",0.58137,-9.28564,1600000,402671,510,352,true,-3.724,0.704,2016-05-03T12:38:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02575,DM_SisR1,2416,Drosophila melanogaster stable intronic sequence RNA 1,Argasinska J,Argasinska J,62.0,62.3,46.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,2,0,0,Gene;,Predicted; RNAfold;,2,0,,\"\",0.60647,-8.62502,1600000,332760,618,389,true,-3.8728,0.70301,2016-05-03T13:46:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02576,tsr1,1236,tsr1 small RNA,Argasinska J,Argasinska J,235.0,235.9,185.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; RNAfold,4,0,,\"\",0.65334,-10.80549,1600000,547361,866,657,true,-4.8122,0.699,2016-05-03T13:49:38.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02577,tsr24,1236,S. aureus tsr24 small RNA,Argasinska J,Argasinska J,50.0,50.6,49.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,105,0,0,Gene; sRNA;,Predicted; RNAfold,22,0,,\"\",0.59662,-9.31174,1600000,365317,380,238,true,-3.3649,0.70926,2016-05-03T13:52:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02578,tsr25,1236,S. aureus tsr25 small RNA,Argasinska J,Argasinska J,63.0,63.9,50.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,16,0,0,Gene; sRNA;,Predicted; RNAfold,19,0,,\"\",0.58228,-9.00863,1600000,376405,310,181,true,-3.5996,0.71438,2016-05-03T13:52:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02579,tsr26,1236,S. aureus tsr26 small RNA,Argasinska J,Argasinska J,141.0,141.0,50.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,0,0,0,Gene; sRNA;,Predicted; RNAfold,2,0,,\"\",0.5193,-4.90989,1600000,249176,232,122,true,-2.9456,0.71815,2016-05-03T13:53:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02580,tsr31,1236,S. aureus tsr31 small RNA,Argasinska J,Argasinska J,50.0,60.5,49.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,17,0,0,Gene; sRNA;,Predicted; RNAfold,20,0,,\"\",0.52623,-6.3994,1600000,401826,158,70,true,-3.4582,0.71904,2016-05-03T13:53:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02581,tsr32,1236,S. aureus tsr32 small RNA,Argasinska J,Argasinska J,98.0,98.2,46.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; RNAfold,5,0,,\"\",0.59829,-5.77947,1600000,245760,271,151,true,-3.0795,0.71825,2016-05-03T13:53:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02582,tsr33,1236,S. aureus tsr33 small RNA,Argasinska J,Argasinska J,109.0,109.4,42.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.44838,-6.68761,1600000,281103,181,85,true,-3.331,0.71867,2016-05-03T13:54:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02583,Teg23,1236,S. aureus Teg23 small RNA,Argasinska J,\"Carroll et al, PMID:26861020\",166.0,166.1,41.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,6,1,0,0,Gene; sRNA;,Predicted; RNAalifold,6,0,,\"\",0.63386,-8.32596,1600000,364951,613,433,true,-4.0699,0.70199,2016-05-03T13:55:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02584,KRAS_3UTR,2417,KRAS 3' UTR cis-regulatory element,Argasinska J,Argasinska J,49.0,49.7,47.9,AU-rich 49 nt fragment in KRAS 3'UTR. Its sequence but not the secondary is necessary and sufficient for reporter repression in HeLa cells.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,29,0,0,Cis-reg;,Predicted; RNAfold;,21,0,,\"\",0.43949,-6.61279,1600000,475559,132,49,true,-2.6224,0.73334,2016-05-03T13:59:14.000Z,2022-10-29T17:42:08.000Z\n",
            "RF02589,Spy779816,1236,S. pyogenes small RNA 779816,Argasinska J,\"Rhun et al, PMID:26580233\",67.0,67.4,64.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,9,0,0,Gene; sRNA;,Predicted; RNAalifold,11,0,,\"\",0.52199,-5.71373,1600000,244365,181,88,true,-3.2587,0.71888,2016-05-17T14:01:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02590,Spy1186876,1236,S. pyogenes small RNA 1186876,Argasinska J,\"Rhun et al, PMID:26580233\",208.0,208.2,43.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; RNAalifold,6,0,,\"\",0.5949,-9.18691,1600000,446506,386,243,true,-3.308,0.70889,2016-05-17T14:02:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02591,Spy1786666,1236,S. pyogenes small RNA 1786666,Argasinska J,\"Rhun et al, PMID:26580233\",64.0,64.3,57.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,3,0,0,Gene; sRNA;,Predicted; RNAalifold,5,0,,\"\",0.54975,-7.38097,1600000,316504,183,87,true,-3.2035,0.71878,2016-05-17T14:04:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02592,Spy392987,1236,S. pyogenes antisense RNA 392987,Argasinska J,Argasinska J,47.0,47.5,46.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,68,0,0,Gene; antisense;,Predicted; RNAalifold,30,0,,\"\",0.58697,-3.85832,1600000,263588,148,63,true,-3.3159,0.71921,2016-05-17T14:44:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02593,NsiR8,1236,Nitrogen stress-induced RNA 8,Argasinska J,Argasinska J,61.0,61.6,43.3,Transcribed in developing heterocysts. Transcription requires HetR suggestingthat it could be related to the differentiation of heterocysts.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,10,0,0,Gene; sRNA;,Predicted; RNAalifold,11,0,,\"\",0.51133,-9.54202,1600000,331874,298,172,true,-3.1949,0.71545,2016-05-31T12:48:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02594,NsiR9,1236,Nitrogen stress-induced RNA 9,Argasinska J,Argasinska J,90.0,90.2,39.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,13,0,0,Gene; sRNA;,Predicted; RNAalifold,13,0,,\"\",0.5834,-8.33765,1600000,350635,232,123,true,-3.2227,0.71845,2016-05-31T13:51:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02595,EBv-sisRNA-1,2419,Epstein-Barr virus stable intronic sequence RNA 1,Argasinska J,Moss at al; PMID:23937650,58.0,58.0,44.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,8,26,0,0,Gene;,Predicted; PMID:23937650,8,0,,\"\",0.68987,-4.76905,1600000,313829,198,81,true,-3.5185,0.7187,2016-05-31T14:40:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02596,cyano_het_sRNA,1236,Cyanobacteria heterocyst sRNA,Argasinska J,\"Brendes-Alvarez et al, PMID:26870012\",35.7,35.7,35.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,26,0,0,Gene; sRNA;,Predicted; PMID:26870012,22,0,,\"\",0.55667,-8.64561,1600000,513952,136,56,true,-4.0955,0.71935,2016-06-01T16:31:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02597,shuA_thermo,2420,shuA/chuA 5' UTR thermoregulator,Argasinska J,\"Kouse et al, PMID:23704938\",58.0,58.6,39.3,Shingella ShuA the outer-membrane heme receptor and its E.coli orthologue ChuA expression are subject to temperature-dependent post-transcriptional regulation by 5' UTR regions containing FourURNA thermometer.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,0,0,0,Cis-reg; thermoregulator;,Published; PMID:23704938,5,0,,\"\",0.49911,-6.95886,1600000,689089,100,32,true,-3.7648,0.72567,2016-06-06T16:02:00.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02598,EBv-sisRNA-2,2419,Epstein-Barr virus stable intronic sequence RNA 2,Argasinska J,Argasinska J,41.8,41.8,41.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,71,0,0,Gene;,Predicted; PMID:24441309,7,0,,\"\",0.51396,-4.88216,1600000,269322,168,76,true,-3.0289,0.71952,2016-06-07T09:31:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02599,BASRCI408,2371,Brucella sRNA CI408,Argasinska J,Argasinska J,398.0,398.4,185.5,\"Brucella abortus sRNA chromosome I, candidate 408\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.67135,-8.63621,1600000,339115,615,435,true,-4.2922,0.702,2016-06-08T11:00:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02600,BASRCI27,2371,Brucella sRNA CI27,Argasinska J,Argasinska J,59.8,59.8,59.7,\"Brucella abortus sRNA chromosome I, candidate 27\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,23,0,0,Gene; sRNA;,Predicted; RNAfold,15,0,,\"\",0.5998,-8.00975,1600000,316505,293,167,true,-3.2396,0.71629,2016-06-08T11:26:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02601,BASRCI337,2371,Brucella sRNA CI337,Argasinska J,Argasinska J,229.0,229.5,125.8,\"Brucella abortus sRNA chromosome I, candidate 337\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; RNAfold,4,0,,\"\",0.64616,-8.45734,1600000,377408,322,191,true,-3.5489,0.71327,2016-06-08T11:44:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02602,BASRCI414,2371,Brucella sRNA CI414,Argasinska J,Argasinska J,270.0,270.0,37.1,\"Brucella abortus sRNA chromosome I, candidate 414\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.62122,-9.5587,1600000,414121,369,229,true,-3.6379,0.71004,2016-06-08T16:29:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02603,BASRCII26,2371,Brucella sRNA CII26,Argasinska J,Argasinska J,218.0,218.8,108.8,\"Brucella abortus sRNA chromosome II, candidate 26\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,2,0,,\"\",0.65942,-8.15735,1600000,347500,321,190,true,-3.1447,0.7134,2016-06-08T16:59:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02604,BASRCI153,2371,Brucella sRNA CI153,Argasinska J,Argasinska J,137.0,137.1,134.1,\"Brucella abortus sRNA chromosome I, candidate 408\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,2,0,0,Gene; sRNA;,Predicted; RNAfold,2,0,,\"\",0.62045,-9.05194,1600000,345093,350,214,true,-3.7146,0.71112,2016-06-13T11:01:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02605,scr5239,2421,Streptomyces sRNA scr5239,Argasinska J,Published; PMID:25785836,64.0,64.2,45.9,\"It inhibits agarase DagA expression by direct base pairing to the dagA coding region, and it represses translation of methionine synthase metE (SCO0985) at the 5â€™ end of its open reading frame.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,6,179,0,0,Gene; sRNA;,Published; PMID:25785836,179,0,,\"\",0.46822,-12.0824,1600000,459162,276,158,true,-2.7145,0.71757,2016-06-13T11:03:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02606,AbsR28,2422,Acinetobacter sRNA 28,Argasinska J,Argasinska J,50.0,50.2,35.1,\"Three sRNAs: AbsR11, AbsR25 and AbsR28 have been experimentally validated in the MTCC 1425 (ATCC15308) strain, which is an MDR strain showing resistance to 12 antibiotics. AbsR25 sRNA could play role in the efflux pump regulation and drug resistance.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,13,0,0,Gene; sRNA;,Predicted; RNAfold,15,0,,\"\",0.61875,-7.70776,1600000,359455,308,180,true,-3.3989,0.71459,2016-06-13T14:25:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02607,AbsR25,2422,Acinetobacter sRNA 25,Argasinska J,Argasinska J,47.0,47.8,44.7,\"Three sRNAs: AbsR11, AbsR25 and AbsR28 have been experimentally validated in the MTCC 1425 (ATCC15308) strain, which is an MDR (multiple drug resistance) strain showing resistance to 12 antibiotics. AbsR25 sRNA could play a role in the efflux pump regulation and drug resistance.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.49718,-9.29054,1600000,344826,288,164,true,-2.9993,0.71654,2016-06-13T14:38:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02608,AbsR11,2422,Acinetobacter sRNA 11,Argasinska J,Argasinska J,158.0,158.5,90.8,\"Three sRNAs: AbsR11, AbsR25 and AbsR28 have been experimentally validated in the MTCC 1425 (ATCC15308) strain, which is an MDR (multiple drug resistance) strain showing resistance to 12 antibiotics. AbsR25 sRNA could play a role in the efflux pump regulation and drug resistance.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,26,0,0,Gene; sRNA;,Predicted; RNAfold,28,0,,\"\",0.57977,-8.62209,1600000,322494,333,200,true,-3.5541,0.71238,2016-06-13T14:43:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02609,BSR0602,2371,Brucella sRNA 0602,Argasinska J,Argasinska J,191.0,191.4,31.6,Highly induced in stationary phase and modulate the intracellular survival.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.60791,-8.5357,1600000,352030,295,169,true,-2.9961,0.71592,2016-06-15T11:10:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02610,BSR0709,2371,Brucella sRNA 0709,Argasinska J,Argasinska J,382.0,382.5,38.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; RNAfold,5,0,,\"\",0.73118,-7.44709,1600000,338570,477,318,true,-3.9132,0.70521,2016-06-15T11:23:04.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02611,BSR0653,2371,Brucella sRNA 0653,Argasinska J,Argasinska J,627.0,627.8,31.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; RNAfold,5,0,,\"\",0.72353,-8.38527,1600000,322769,857,642,true,-5.1402,0.69913,2016-06-15T11:53:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02612,BSR1350,2371,Brucella sRNA 1350,Argasinska J,Argasinska J,216.0,216.5,76.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,4,0,,\"\",0.54082,-7.07866,1600000,266100,304,176,true,-3.4926,0.71503,2016-06-15T11:59:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02613,BSR0739,2371,Brucella sRNA 0739,Argasinska J,Argasinska J,191.0,192.2,91.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,1,0,0,Gene; sRNA;,Predicted; RNAfold,5,0,,\"\",0.60164,-7.21505,1600000,263049,284,160,true,-3.3019,0.71728,2016-06-15T12:06:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02614,BSR1073,2371,Brucella sRNA 1073,Argasinska J,Argasinska J,241.0,241.5,109.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.68002,-8.18302,1600000,444013,329,196,true,-3.5962,0.71281,2016-06-15T12:13:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02615,BSR0626,2371,Brucella sRNA 0626,Argasinska J,Argasinska J,155.0,155.1,58.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.52154,-8.58429,1600000,373719,241,128,true,-2.7358,0.71832,2016-06-15T12:17:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02616,SSRC8_2,1236,Streptococcus sRNA 8,Argasinska J,Argasinska J,72.0,72.0,71.1,Sequence is experimentally verified.,SSRC8;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,11,0,0,Gene; sRNA;,Predicted; RNAfold,5,0,,\"\",0.72958,-7.57589,1600000,342083,433,281,true,-3.8762,0.70677,2016-06-21T11:50:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02617,SSRC10,1236,Streptococcus sRNA 10,Argasinska J,Argasinska J,56.0,56.6,55.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,49,0,0,Gene; sRNA;,Predicted; RNAfold,50,0,,\"\",0.53548,-8.32812,1600000,299717,220,113,true,-3.2401,0.71841,2016-06-21T11:50:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02618,SSRC34_2,1236,Streptococcus sRNA 34,Argasinska J,Argasinska J,48.0,48.4,46.6,Sequence is experimentally verified.,SSRC34;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,6,0,0,Gene; sRNA;,Predicted; RNAfold,9,0,,\"\",0.5025,-7.15592,1600000,266158,237,125,true,-2.8297,0.71833,2016-06-21T11:51:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02619,BSnc115,2371,Brucella sRNA 115,Argasinska J,Argasinska J,115.0,115.6,35.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,4,0,,\"\",0.65642,-3.66726,1600000,256786,190,91,true,-2.984,0.71867,2016-06-21T15:10:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02620,BSnc119,2371,Brucella sRNA 119,Argasinska J,Argasinska J,115.0,115.2,66.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,4,0,,\"\",0.55707,-5.84882,1600000,242631,188,90,true,-3.0819,0.71866,2016-06-21T15:11:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02621,BSnc120,2371,Brucella sRNA 120,Argasinska J,Argasinska J,114.0,114.9,70.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,4,0,,\"\",0.72612,-2.28044,1600000,282225,185,88,true,-3.3711,0.71877,2016-06-21T15:12:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02622,BSnc121,2371,Brucella sRNA 121,Argasinska J,Argasinska J,114.0,138.7,53.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.51215,-9.14582,1600000,338802,225,116,true,-2.794,0.71831,2016-06-21T16:12:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02623,BSnc140,2371,Brucella sRNA 140,Argasinska J,Argasinska J,126.0,126.0,112.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.53318,-6.5367,1600000,311731,194,94,true,-2.9074,0.71857,2016-06-21T16:50:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02624,BSnc150,2371,Brucella sRNA 150,Argasinska J,Argasinska J,73.0,73.1,33.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.58391,-4.77415,1600000,276620,124,44,true,-2.6361,0.7369,2016-06-21T16:54:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02625,WsnRNA46,2423,Wolbachia sRNA 46,Argasinska J,Argasinska J,65.3,65.3,65.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,122,0,0,Gene; sRNA;,Predicted; PMID:25512495,17,0,,\"\",0.64215,-2.9192,1600000,249701,282,151,true,-3.2307,0.71808,2016-06-27T15:54:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02626,WsnRNA59,2423,Wolbachia sRNA 59,Argasinska J,Argasinska J,49.0,49.7,47.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,3,0,0,Gene; sRNA;,Predicted; PMID:25512495,4,0,,\"\",0.67254,-2.39457,1600000,182942,185,89,true,-3.313,0.72069,2016-06-27T15:56:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02627,Ssr1,2424,Shigella small RNA 1,Argasinska J,Argasinska J,820.0,820.4,365.2,\"Exists only in Shigella, may play role in resistance to acidic stress and regulation of virulence.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,0,0,0,Gene; sRNA;,Predicted; RNAfold;,2,0,,\"\",0.69854,-7.65446,1600000,293316,871,654,true,-5.2163,0.69905,2016-07-04T14:05:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02628,Hrs1,1266,Hfq-regulated sRNA 1,Argasinska J,Argasinska J,56.4,56.4,56.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,28,0,0,Gene; sRNA;,Predicted; RNAfold;,29,0,,\"\",0.56035,-8.15032,1600000,484213,165,74,true,-2.7858,0.72136,2016-07-12T09:33:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02629,RmaA,2425,Regulator of motility and amylovoran A,Argasinska J,Argasinska J,57.0,57.4,48.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,0,0,0,Gene; sRNA;,Predicted; RNAfold;,2,0,,\"\",0.5212,-7.0619,1600000,264283,220,113,true,-3.4327,0.71849,2016-07-12T09:34:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02630,Hrs12,1266,Hfq-regulated sRNA 12,Argasinska J,Argasinska J,70.0,70.7,50.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold;,3,0,,\"\",0.49636,-7.908,1600000,373213,182,87,true,-3.2168,0.71889,2016-07-12T09:35:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02631,Hrs13,1266,Hfq-regulated sRNA 13,Argasinska J,Argasinska J,52.0,52.9,39.0,,Hrs10;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,4,0,0,Gene; sRNA;,Predicted; RNAfold;,6,0,,\"\",0.6949,-5.575,1600000,309470,176,82,true,-2.9453,0.72043,2016-07-12T09:36:08.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02632,Hrs10,1266,Hfq-regulated sRNA 10,Argasinska J,Argasinska J,108.0,108.4,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold;,3,0,,\"\",0.58727,-8.63306,1600000,404357,268,148,true,-3.1142,0.71804,2016-07-12T11:46:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02633,Hrs21,1266,Hfq-regulated sRNA 21,Argasinska J,Argasinska J,102.0,102.9,31.1,Could play role in virulence regulation.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold;,3,0,,\"\",0.64385,-4.79747,1600000,256457,250,135,true,-3.2345,0.71823,2016-07-12T11:54:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02634,EF3314_EF3315,2426,Enterococcus sRNA EF3314_EF3315,Argasinska J,Argasinska J,100.0,177.6,69.0,It may be involved in the turnover of some abundant proteins. Involvement in stress response and virulence.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; sRNA;,Predicted; RNAfold;,4,0,,\"\",0.62065,-8.43568,1600000,418235,258,141,true,-3.2111,0.71823,2016-07-12T16:52:33.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02635,EF0820_EF0821,2426,Enterococcus sRNA EF0820_EF0821,Argasinska J,Argasinska J,442.0,442.6,116.0,Expressed during exponential phase. Seem not to be involved in virulence.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,3,0,0,Gene; sRNA;,Predicted; RNAfold;,4,0,,\"\",0.66243,-8.47813,1600000,367868,539,370,true,-3.927,0.70351,2016-07-13T11:26:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02636,EF1368_EF1369,2426,Enterococcus sRNA EF1368_EF1369,Argasinska J,Argasinska J,730.0,738.3,94.9,Expressed during exponential phase. Involvement in stress response and virulence.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,2,0,0,Gene; sRNA;,Predicted; RNAfold;,6,0,,\"\",0.7463,-7.48141,1600000,305589,838,626,true,-4.9871,0.69926,2016-07-18T10:33:24.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02637,EF0408_EF0409,2426,Enterococcus sRNA EF0408_EF0409,Argasinska J,Argasinska J,440.0,448.4,60.0,Expressed during exponential phase. Involvement in stress response and virulence.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,2,0,0,Gene; sRNA;,Predicted; RNAfold;,5,0,,\"\",0.68441,-8.25987,1600000,380232,589,412,true,-3.7262,0.70248,2016-07-18T10:41:20.000Z,2021-12-03T12:41:14.000Z\n",
            "RF02638,EF0605_EF0606,2426,Enterococcus sRNA EF0605_EF0606,Argasinska J,Argasinska J,219.0,219.9,43.3,Expressed during exponential phase. Involvement in stress response and virulence.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; sRNA;,Predicted; RNAfold;,1,0,,\"\",0.53479,-8.97527,1600000,352885,307,179,true,-3.3397,0.7145,2016-07-18T11:14:32.000Z,2021-12-03T12:41:14.000Z\n",
            "RF02639,EF0869_EF0870,2426,Enterococcus sRNA EF0869_EF0870,Argasinska J,Argasinska J,533.0,559.7,56.8,\"Expressed after 24 h of starvation[1]. Identified in E. faecium as sRNA_0160. Expressed in mid,late exponential and early stationary phases of growth. Transcript level decreased significantly under antibiotic exposure [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,3,0,0,Gene; sRNA;,Predicted; RNAfold;,6,0,,\"\",0.68646,-8.23494,1600000,331438,662,474,true,-4.5033,0.70122,2016-07-18T14:06:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02640,MOSES4,1236,S. pyogenes small RNA MOSES4,Argasinska J,Argasinska J,54.8,54.8,54.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,2,0,0,Gene; sRNA;,Predicted; RNAfold;,5,0,,\"\",0.53969,-9.30818,1600000,384795,303,176,true,-2.9769,0.71492,2016-07-20T14:37:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02641,Spy490483c,1236,S. pyogenes small RNA Spy490483c,Argasinska J,Argasinska J,65.0,65.8,37.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; RNAfold;,7,0,,\"\",0.53454,-6.35745,1600000,247317,199,98,true,-3.1718,0.71863,2016-07-20T15:09:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02642,Spy491311c,1236,S. pyogenes small RNA Spy491311c,Argasinska J,Argasinska J,178.0,178.8,61.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold;,3,0,,\"\",0.59151,-5.21783,1600000,302826,261,143,true,-3.1669,0.71815,2016-07-25T14:43:22.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02643,Spy491738,1236,S. pyogenes small RNA Spy491738,Argasinska J,Argasinska J,91.0,91.9,39.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,6,0,0,Gene; sRNA;,Predicted; RNAfold;,8,0,,\"\",0.46312,-9.62184,1600000,347362,180,85,true,-2.726,0.71874,2016-07-25T14:52:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02644,Spy490380c,1236,S. pyogenes small RNA Spy490380c,Argasinska J,Argasinska J,65.0,90.3,48.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,6,0,0,Gene; sRNA;,Predicted; RNAfold;,9,0,,\"\",0.4279,-7.35467,1600000,293284,195,96,true,-3.3054,0.71862,2016-07-25T16:34:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02645,Cis2_sRNA,2427,Ruegeria cis2 sRNA,Argasinska J,Argasinska J,50.0,50.4,48.4,Cis-encoded. Targets ABC transporter. Functions in transport.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,174,0,0,Gene; sRNA;,Predicted; RNAfold;,175,0,,\"\",0.56252,-9.09542,1600000,328720,273,153,true,-2.8725,0.71819,2016-07-26T15:29:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02646,Cis8_sRNA,2427,Ruegeria cis8 sRNA,Argasinska J,Argasinska J,58.0,58.0,57.4,Cis-encoded. Targets hypothetical protein. Higher sRNA expression during C-limited growth.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,90,0,0,Gene; sRNA;,Predicted; RNAfold;,91,0,,\"\",0.51871,-7.8676,1600000,285564,239,128,true,-3.1603,0.71824,2016-08-08T14:19:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02647,Cis52_sRNA,2427,Ruegeria cis52 sRNA,Argasinska J,Argasinska J,56.0,56.0,55.3,Cis-encoded. Targets oligo/dipeptide ABC transporter. Functions in transport. Higher sRNA expression during C-limited growth.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,147,0,0,Gene; sRNA;,Predicted; RNAfold;,146,0,,\"\",0.53116,-10.03023,1600000,367315,307,179,true,-3.0176,0.71464,2016-08-08T14:37:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02648,Cis90_sRNA,2427,Ruegeria cis90 sRNA,Argasinska J,Argasinska J,50.0,50.5,49.9,Cis-encoded. Targets ABC transporter. Functions in transport. Higher sRNA expression during C-limited growth.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,17,0,0,Gene; sRNA;,Predicted; RNAfold;,18,0,,\"\",0.62887,-9.06155,1600000,344628,371,234,true,-3.527,0.70959,2016-08-08T14:51:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02649,Trans44_sRNA,2427,Ruegeria trans44 sRNA,Argasinska J,Argasinska J,108.0,108.1,70.8,Trans-encoded. Higher sRNA expression during C-limited growth.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,15,0,0,Gene; sRNA;,Predicted; RNAfold;,13,0,,\"\",0.65974,-9.19025,1600000,379161,414,267,true,-3.7858,0.70755,2016-08-09T10:10:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02650,CncR1,2429,Cag non-coding RNA1,Argasinska J,Argasinska J,76.0,76.1,43.3,\"Encoded by the virulence-associated cag pathogenicity island (cag-PAI). It is an abundant and conserved sRNA which does not target cag-PAI genes, but down-regulates bacterial motility and have positive role on the adhesion to host cells.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; PMID:268638,4,0,,\"\",0.6206,-8.6778,1600000,426529,349,213,true,-3.5444,0.71116,2016-08-10T14:40:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02651,SorY,2430,Singlet oxygen resistance RNA Y,Argasinska J,Argasinska J,111.0,111.3,67.8,Singlet oxygen induced SorY is induced under several stress conditions and conferrs resistance against singlet oxygen by affecting a metabolite transporter.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold;,2,0,,\"\",0.5693,-8.33839,1600000,375982,178,83,true,-3.3955,0.71887,2016-08-10T14:48:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02652,StyR-3,2431,Salmonella enterica Typhi npcRNA 3,Argasinska J,Argasinska J,59.0,59.2,35.1,Promoter associated sRNA gene that is co-transcribed with the ramA gene (mediates multidrug resistance) and overlaps with the DNA binding site of the RamR repressor.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,5,0,0,Gene; sRNA;,Predicted; RNAfold;,10,0,,\"\",0.60547,-5.7155,1600000,240221,263,144,true,-2.8666,0.7182,2016-08-22T10:41:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02653,StyR-143,2431,Salmonella enterica Typhi npcRNA 143,Argasinska J,Argasinska J,176.0,176.4,57.4,Antisense to the 3'-end of ORF of the hypothetical protein-coding t4293 gene.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,0,0,0,Gene; sRNA;,Predicted; RNAfold;,2,0,,\"\",0.5688,-8.58707,1600000,348521,262,144,true,-3.1362,0.71813,2016-08-22T10:49:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02654,MicL,2432,MicL sRNA,Argasinska J,Argasinska J,56.0,56.0,55.9,mRNA-interfering complementary RNA regulator of Lpp. Transcription factor sigma E dependant sRNA. Together with MicA and RybB it allows sigma E to repress the synthesis of all abundant outer membrane proteins in response to stress.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,35,0,0,Gene; sRNA;,Predicted; RNAfold;,37,0,,\"\",0.72527,-7.89121,1600000,361581,466,308,true,-3.938,0.70567,2016-08-24T10:20:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02655,BSR0441,2371,Brucella BSR0441 sRNA,Argasinska J,Argasinska J,291.0,291.5,35.4,Plays important role in Brucella's intracellular survival.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,1,0,0,Gene; sRNA;,Predicted; RNAfold;,6,0,,\"\",0.66137,-8.39641,1600000,355070,383,240,true,-3.5444,0.70921,2016-08-24T10:51:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02656,sot0042,1236,Sense overlapping transcript RNA 0042 (sot),Argasinska J,Argasinska J,88.0,88.0,86.5,\"It is associated with IS1341-type transposase (tnpB) genes, at the 3'-end in H. salinarum NRC-1. The tetraloop structural motif may be functional.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,142,0,0,Gene; sRNA;,Predicted; RNAfold;,54,0,,\"\",0.6848,-8.19684,1600000,391387,301,174,true,-3.2246,0.71534,2016-08-24T14:12:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02657,sot2652,1236,Sense overlapping transcript RNA 2652 (sot),Argasinska J,Argasinska J,61.0,61.0,52.9,\"It is associated with IS1341-type transposase (tnpB) genes, at the 3'-end in H. salinarum NRC-1. The tetraloop structural motif may be functional.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,25,0,0,Gene; sRNA;,Predicted; RNAfold;,17,0,,\"\",0.5838,-9.54614,1600000,352917,316,187,true,-3.2044,0.71366,2016-08-24T14:12:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02658,IRES_RhPV,2487,Rhopalosiphum padi virus 5'UTR internal ribosome entry site,Argasinska J,Argasinska J,688.0,688.7,55.1,\"This IRES element can function efficiently in mammalian, plant and insect translation systems.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,0,0,0,Cis-reg; IRES;,Predicted; RNAfold;,1,0,,\"\",0.68521,-8.15971,1600000,356002,783,579,true,-4.7822,0.69976,2016-09-13T13:24:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02659,ncRv12659,2178,ncRv12659 sRNA,Argasinska J,Argasinska J,209.0,209.7,111.5,Original name MTS2048. It is induced during starvation and infection.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold;,2,0,,\"\",0.5944,-8.7073,1600000,316812,301,174,true,-3.2297,0.71518,2016-09-13T13:41:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02660,icaR_3p_UTR,2458,icaR 3'UTR,Argasinska J,Argasinska J,369.0,369.9,357.2,The 3'UTR of icaR mRNA (repressor of the main expolysaccharidic compound of the bacteria biofilm matrix) binding to the 5' UTR can interfere with the translation initiation complex and generate a double stranded substrate for RNaseIII. Mozos at al. showed that the interaction was between UCCCCUG motif in the 3'UTR and the Shine-Dalagarno region at the 5'UTR. Deletion of the motif resulted in IcaR repressor accumulation and inhibition of biofilm development.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Cis-reg;,Predicted; RNAfold;,4,0,,\"\",0.66583,-7.51952,1600000,378320,563,390,true,-3.9239,0.70298,2016-09-13T16:35:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02661,icaR_5p_UTR,2458,icaR 5' UTR,Argasinska J,Argasinska J,88.0,88.7,49.2,The 3'UTR of icaR mRNA (repressor of the main expolysaccharidic compound of the bacteria biofilm matrix) binding to the 5' UTR can interfere with the translation initiation complex and generate a double stranded substrate for RNaseIII. Mozos at al. showed that the interaction was between UCCCCUG motif in the 3'UTR and the Shine-Dalagarno region at the 5'UTR. Deletion of the motif resulted in IcaR repressor accumulation and inhibition of biofilm development.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,2,0,0,Cis-reg;,Predicted; RNAfold;,4,0,,\"\",0.57045,-3.19112,1600000,286408,167,75,true,-3.2812,0.71958,2016-09-13T16:35:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02662,BLi_r0872,2434,Bacillus asRNA 0872,Argasinska J,Argasinska J,61.0,61.1,53.0,\"Antisense RNA against 3' end of Subtilisin Carlsberg encoding mRNA, discovered in an RNA-seq based study. It may have a putative impact on protease production and serve as a target for strain improvement.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,2,0,0,Gene; antisense;,Predicted; RNAfold;,3,0,,\"\",0.56296,-5.74695,1600000,222586,263,144,true,-3.242,0.71818,2016-09-19T11:47:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02663,GmglnB1_5p_UTR,2435,GmglnB1 5'UTR,Argasinska J,Argasinska J,54.0,54.1,38.4,The 5'UTR of the Soybean cytosolic glutamine synthetase gene (GmglnÎ²1) contains prokaryotic translation initiation signals and acts as a translational enhancer.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,3,0,0,Cis-reg;,Predicted; RNAfold;,2,0,,\"\",0.56956,-5.2619,1600000,226090,181,85,true,-3.4064,0.71884,2016-09-21T11:55:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02664,IRES_cyp24a1,2436,Cyp24a1 internal ribosome entry site (IRES),Argasinska J,Argasinska J,191.0,191.0,144.8,\"Cyp24a1 (1,25-dihydroxyvitamin D3 24-hydroxylase) expression is translationally regulated via IRES within the 5'UTR, which is responsive to an inflammatory environment.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,9,0,0,Cis-reg; IRES;,Predicted; RNAfold;,9,0,,\"\",0.72578,-8.64817,1600000,354725,579,404,true,-4.0671,0.70269,2016-09-21T16:28:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02665,PsiU1-6,1799,Drosophila small cajal body-specific RNA PsiU1-6,Argasinska J,Argasinska J,72.0,72.4,57.8,Pseudouridylation guide activity for U1 snRNA and 28s rRNA,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,11,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold;,12,0,,\"\",0.57369,-8.35735,1600000,322421,282,159,true,-2.8169,0.71724,2016-09-26T15:23:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02666,PsiU2-35.45,1799,Drosophila small cajal body-specific RNA PsiU2-35.45,Argasinska J,Argasinska J,89.0,89.7,87.0,Pseudouridylation guide activity for U2 snRNA,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,5,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold;,6,0,,\"\",0.5203,-7.69901,1600000,285950,255,140,true,-3.0203,0.71814,2016-09-27T12:44:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02667,PsiU2-38.40.42,1799,Drosophila small cajal body-specific RNA PsiU2-38.40.4,Argasinska J,Argasinska J,88.0,88.4,52.9,Pseudouridylation guide activity for U2 snRNA,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,11,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold;,13,0,,\"\",0.52167,-7.73212,1600000,274665,250,136,true,-3.0806,0.71816,2016-09-27T14:28:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02668,PsiU2-55,1799,Drosophila small cajal body-specific RNA PsiU2-55,Argasinska J,Argasinska J,55.0,55.3,38.0,Pseudouridylation guide activity for U2 snRNA and 28s rRNA,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,10,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold;,10,0,,\"\",0.63337,-8.20206,1600000,348404,306,178,true,-3.4106,0.71475,2016-09-28T10:41:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02669,PsiU6-40,1799,Drosophila small cajal body-specific RNA PsiU6-40,Argasinska J,Argasinska J,48.0,48.9,33.2,Pseudouridylation guide activity for U6 snRNA,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,9,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold;,10,0,,\"\",0.57654,-7.24871,1600000,270548,259,142,true,-3.1334,0.71831,2016-09-28T10:59:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02670,scaDm46E3,1799,small Cajal body-specific RNA Dm46E3,Argasinska J,Argasinska J,61.0,61.1,31.9,\"Targets U1b snRNA, a developmentally regulated isoform of the spliceosomal U1 snRNA\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,4,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold;,4,0,,\"\",0.63092,-5.66025,1600000,236581,253,137,true,-2.9258,0.71805,2016-09-28T12:33:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02671,Ysr35,2437,Yersinia sRNA 35,Argasinska J,Argasinska J,428.0,428.3,167.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold;,3,0,,\"\",0.71682,-8.52837,1600000,406724,503,339,true,-4.4083,0.70453,2016-10-04T10:40:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02672,SprX,2438,Small pathogenicity island RNA X,Argasinska J,Published; PMID:24557948,80.0,179.7,55.5,\"SprX was shown to influence antibiotic resistance of the bacteria to Vancomycin and Teicoplanin glycopeptides, which are used to treat MRSA infections. In this study the authors identified a SprX target, stage V sporulation protein G (Spo VG). By reducing Spo VG expression levels, SprX affects S. aureus resistance to the glycopeptide antibiotics. It is involved in regulation of pathogenicity factors.[2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,15,0,0,Gene; sRNA;,Published; PMID:24557948,17,0,,\"\",0.58381,-8.86101,1600000,384107,271,151,true,-3.2989,0.71796,2016-10-05T16:56:56.000Z,2022-05-06T13:10:07.000Z\n",
            "RF02673,scr4677,2421,Streptomyces sRNA 4677,Argasinska J,Argasinska J,41.0,41.9,37.1,Located in the intergenic region between anti-sigma factor SCO4677 gene and a putative regulatory protein gene SCO4676. scr4677 expression requires the SCO4677 activity and scr4677 sRNA itself seem to affect the levels of the SCO4676-associated transcripts.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,12,282,0,0,Gene; sRNA;,Predicted; PMID:24465751,143,0,,\"\",0.68111,-9.53852,1600000,456311,400,247,true,-3.6562,0.70863,2016-11-08T13:46:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02674,AsdA,2431,antisense RNA of dnaA mRNA,Argasinska J,Argasinska J,52.0,52.2,51.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,208,0,0,Gene; antisense;,Predicted; RNAfold;,207,0,,\"\",0.58394,-7.2152,1600000,309186,174,81,true,-3.6257,0.7189,2016-11-09T10:34:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02675,Ysr141,2440,Yersinia sRNA 141,Argasinska J,Argasinska J,196.0,196.3,194.0,Shown to regulate the synthesis of the type III secretion system (T3SS) effector protein YopJ.The Yop-Ysc T3SS is a critical component of virulence for Yersinia species.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,2,0,0,Gene; sRNA;,Predicted; PMID:24532772;,5,0,,\"\",0.70574,-8.03713,1600000,352237,398,255,true,-3.794,0.70824,2016-11-16T17:00:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02676,Esr41,2254,Enterohemorrhagic E. coli sRNA 41,Argasinska J,Argasinska J,102.0,102.4,39.3,\"Enhances cell motility in both enterohemorrhagic E.coli O157:H7 and nonpathogenic E. coli K12. However, the sequence is specific to O157:H7 and not present in K12.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,0,0,0,Gene; sRNA;,Predicted; PMID:24646762;,3,0,,\"\",0.54259,-8.62523,1600000,437690,165,74,true,-3.5245,0.71895,2016-11-21T10:40:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02677,NsiR4,2442,Nitrogen stress-induced RNA 4,Argasinska J,Argasinska J,40.0,40.6,32.9,\"Former name SyR12, is a cyanobacterial non-coding RNA which plays role in the regulation of Glutamine synthetase (GS), a key enzyme in biological nitrogen assimilation. NsiR4 interacts with the 5'UTR of the mRNA of the GS inactivating factor IF7 (gifA mRNA) and reduces its expression. NsiR4 expression is under positive control of the nitrogen control transcription factor (NtcA). NsR4 is a first example of an sRNA controlling the assimilation of a micronutrient.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,25,0,0,Gene; sRNA;,Predicted; RNAalifold;,25,0,,\"\",0.61623,-8.17109,1600000,575635,149,65,true,-3.623,0.71905,2016-11-21T16:27:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02678,Hatchet,2443,Hatchet ribozyme,\"Weinberg Z, Ontiveros-Palacios N\",Published; PMID:26167874,51.0,54.5,39.6,Hatchet ribozyme was first report in [1] and biochemical analysis report it has self-cleaving activity [2]. The structure of Hatchet ribozyme was reported in PDB 6JQ5 and 6JQ6 [3],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,155,2,0,0,Gene; ribozyme;,Published; PMID:26167874,4,0,,\"\",0.59766,-5.50502,1600000,303381,372,133,true,-2.8143,0.71807,2016-11-28T12:52:20.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02679,Pistol,2444,Pistol ribozyme,Argasinska J,Weinberg Z,34.0,34.0,33.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,45,30,0,0,Gene; ribozyme;,Published; PMID:26167874,43,0,,\"\",0.56235,-8.60917,1600000,494459,149,70,true,-4.179,0.71969,2016-11-28T14:43:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02680,PreQ1-III,2445,PreQ1-III riboswitch,\"Argasinska J, Ontiveros-Palacios N\",\"Zasha Weinberg, Phillip McCown, Jonathan Liang\",63.0,63.4,35.9,\"A third preQ1-binding (preQ1-III riboswitch that differs structurally from preQ1 (RF00522) and preQ1-II (RF01054) was first reported for McCown et al. 2014 [1]. The structure of preQ1-III has been reported for Mathews et al. 2015 [2], preQ1-III riboswitch aptamer presents four pairing regions P1-P4, organised as a HL out-type pseudoknot in its structure. preQ1-II presents a bipartite architecture, wich permit a dynamic conformational switch involved in the expression platform accessibility. PreQ1-III riboswitch structure was reported in PDB: 4RZD_A [2] and 6XKN_A and 6XKO_A\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,31,85,0,0,Cis-reg; riboswitch;,Published; PMID:25036777,13,0,,\"\",0.56056,-8.12985,1600000,303833,199,102,true,-3.0371,0.71844,2016-11-30T12:50:08.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02681,Twister-sister,2446,Twister_sister_ribozyme,Weinberg Z,Weinberg Z; 0000-0002-6681-3624,37.8,37.8,36.0,\"Twister-sister is a self-cleaving ribozyme, it was first reported in [1], it resemble twister ribozyme in terms of the connectivity of helices and the sequence of a conserved loop [2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.00 -Z 742849.287494 CM SEQDB,112,1,0,0,Gene; ribozyme;,Published; PMID:26167874,3,0,,\"\",0.63394,-3.65928,1600000,185715,275,83,true,-3.5765,0.71867,2016-11-30T15:37:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02682,HDV-F-prausnitzii,1388,HDV ribozyme from F. prausnitzii,Argasinska J,Weinberg Z,44.0,44.1,43.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,48,3005,0,0,Gene; ribozyme;,Published; PMID:19965505,50,0,,\"\",0.53196,-8.58026,1600000,331261,178,74,true,-3.6734,0.71883,2016-11-30T16:09:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02683,NiCo,2447,NiCo riboswitch,\"Argasinska J, Ontiveros-Palacios N\",\"Weinberg Z, Vallery T\",34.0,34.0,33.9,\"NiCo riboswitch responds to transition metals, its structure was reported in [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,83,397,0,0,Cis-reg; riboswitch;,Published; PMID:25794617,198,0,,\"\",0.66115,-3.88778,1600000,183854,267,96,true,-3.6427,0.71864,2016-12-05T15:04:53.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02684,Twister-P5,2448,Type-P5 twister ribozyme,\"Weinberg Z, Ames TD\",Published; 24240507;,37.6,37.6,37.5,\"Twister ribozyme motif, present stem P5 variant (type P3) [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,215,14,0,0,Gene; ribozyme;,Published; 24240507;,16,0,,\"\",0.58648,-8.17274,1600000,440783,248,60,true,-2.9613,0.72662,2016-12-12T15:42:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02685,RAGATH-5,2449,RAGATH-5 RNA,Argasinska J,Weinberg Z,50.0,51.5,40.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,39,19,0,0,Gene; ribozyme;,Published; PMID:26167874,6,0,,\"\",0.49202,-9.69159,1600000,574970,122,53,true,-3.8113,0.71923,2016-12-12T16:53:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02686,RAGATH-6,2449,RAGATH-6 RNA,Argasinska J,Weinberg Z,170.0,177.4,40.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,0,0,0,Gene; ribozyme;,Published; PMID:26167874,1,0,,\"\",0.44216,-8.50227,1600000,322484,270,151,true,-2.8632,0.71792,2016-12-13T11:07:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02687,RAGATH-8,2449,RAGATH-8 RNA,Weinberg Z,Published; PMID:26167874,86.7,86.7,35.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,69,3,0,0,Gene; sRNA;,Published; PMID:26167874,4,0,,\"\",0.67366,-9.29324,1600000,420573,443,276,true,-3.065,0.707,2016-12-13T12:59:01.000Z,2020-09-14T15:06:17.000Z\n",
            "RF02688,RAGATH-13,2449,RAGATH-13 RNA,Argasinska J,Weinberg Z,80.0,80.7,34.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,20,51,0,0,Gene; ribozyme;,Published; PMID:26167874,32,0,,\"\",0.63227,-2.72964,1600000,181293,186,95,true,-3.4535,0.71876,2016-12-13T14:13:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02689,hilD_3p_UTR,2450,hilD 3'UTR,Argasinska J,Argasinska J,74.0,74.1,35.0,Target for hilD mRNA degradation by the degradosome and it may play an role in hilD and SPI-1 expression by serving as a target for the Hfq RNA chaperone.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,5,0,0,Cis-reg;,Predicted; RNAfold;,8,0,,\"\",0.56982,-9.32388,1600000,389662,464,310,true,-3.8294,0.70554,2017-01-10T11:32:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02690,BTH_s1,2452,Burkholderia sRNA 1,Argasinska J,Argasinska J,74.0,74.2,33.6,\"Trans-encoded, exhibits differential expression profiles dependent on growth phase and cell stimuli, such as antbiotics and serum.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,12,0,0,Gene; sRNA;,Predicted; RNAfold;,16,0,,\"\",0.62657,-8.85638,1600000,343730,337,205,true,-3.4109,0.71195,2017-01-11T10:47:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02691,BTH_s19,2452,Burkholderia sRNA 19,Argasinska J,Argasinska J,153.0,153.1,38.2,\"Cis-encoded, located in the 5'UTR of their cognate gene. Expression correlated with tight regulation of the gene transcript levels.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,7,11,0,0,Gene; sRNA;,Predicted; RNAfold;,17,0,,\"\",0.6565,-8.78082,1600000,359656,358,224,true,-3.6228,0.71031,2017-01-16T12:04:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02692,BTH_s39,2452,Burkholderia sRNA 39,Argasinska J,Argasinska J,77.0,77.7,66.4,\"Trans-encoded, exhibits differential expression profiles dependent on growth phase and cell stimuli, such as antbiotics and serum. May function in bacterial metabolism and adaptation to the host.[1] Shown to be expressed in biotin. Named ncS63 in[2]. Significantly up-regulated in response to low iron. Its 5'-adjacent gene BCAL2297 encoding hemin-uptake protein HemP, is up-regulated to the same fold change. Expression and target genes suggest a potential role in regulating iron homoeostasis[2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,7,56,0,0,Gene; sRNA;,Predicted; RNAfold;,61,0,,\"\",0.66135,-5.29808,1600000,225633,241,130,true,-3.1314,0.71834,2017-01-17T10:51:51.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02693,psm_mec_RNA,2453,psm_mec locus RNA,Argasinska J,Argasinska J,85.0,85.7,63.3,psm_mec locus strain dependent gene (immune evasion factor protein A) regulatory RNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,2,0,0,Gene; sRNA;,Predicted; RNAfold;,6,0,,\"\",0.43873,-8.93538,1600000,340945,293,143,true,-3.0995,0.71796,2017-01-17T15:54:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02694,RalA,2254,RalR antitoxin,Argasinska J,Argasinska J,170.0,170.3,47.3,\"Trans-encoded, has 16 nucleotides complimentary to coding region of toxin RalR mRNA. RalA functions as an antitoxin by preventing translation of RalR (a non-specific endonuclease that cleaves methylated and unmethylated DNA). Its activity requires RNA chaperone Hfq. RalR and RalA form a type I toxin-antitoxin (TA) system. RalR/RaLA TA locus is responsible for resistance to the antibiotic fosfomycin in E.coli.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,10,1,0,0,Gene; sRNA;,Predicted; PMID:24748661;,11,0,,\"\",0.60157,-8.5128,1600000,395549,306,179,true,-3.1719,0.71473,2017-01-24T11:41:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02695,NrsZ,2455,Nitrogen regulated small RNA,Argasinska J,Argasinska J,76.9,76.9,76.1,Found in the opportunistic pathogen Pseudomonas aeruginosa PAO1. Its transcription is induced during nitrogen limitation by the NtrB/C two-component system (an important regulator of nitrogen assimilation and swarming motility) together with the alternative sigma factor RpoN (a global regulator involved in nitrogen metabolism). By activating rhlA (a gene essential for rhamnolipids synthesis) it positively regulates the production of rhamnolipid surfactants needed for swarming motility.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,37,0,0,Gene; sRNA;,Predicted; PMID:24308329;,32,0,,\"\",0.6475,-9.84042,1600000,394148,372,236,true,-3.7363,0.70946,2017-01-25T11:03:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02696,Teg49,2456,Teg49 sRNA,Argasinska J,Argasinska J,186.0,186.4,56.1,Present in the extended promoter region of the staphylococcal accessory regulator sarA. It is modulated by sigB (sarA regulator) and cshA (an ATP-dependant DEAD box RNA helicase) and it most likely contributes to virulence of S. aureus by modulating SarA expression.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,1,0,0,Gene; sRNA;,Predicted; PMID: 25092913;,7,0,,\"\",0.46335,-9.84924,1600000,335274,327,196,true,-3.0463,0.71261,2017-01-25T12:07:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02697,LDH1_5p_UTR,2457,LDH1 5'UTR,Argasinska J,Argasinska J,113.0,113.5,35.4,the translational repression of lactate dehydrogenase 1 (LDH1) is mediated through its 5'UTR. A small 16 nucleotide long regulatory RNA hairpin is essential for the repression.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,2,0,0,Cis-reg;,Predicted; PMID:25223457;,6,0,,\"\",0.51525,-6.91296,1600000,269347,205,102,true,-3.659,0.71854,2017-01-25T16:26:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02700,HtrA_thermometer,2187,HtrA 5' UTR thermometer,Argasinska J,Argasinska J,52.0,53.7,49.0,FourU thermometer found in the 5'UTR of Salmonella htrAp3 and E. coli htrA genes (high temperature requirement A gene).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,12,0,0,Cis-reg; thermoregulator;,Published; PMID:28739676;,16,0,,\"\",0.62518,-4.12194,1600000,441778,121,46,true,-4.0317,0.71954,2017-02-06T11:17:05.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02701,PsrR1,2462,Photosynthesis regulatory RNA1,Argasinska J,\"Georg et al, PMID:25248550\",42.0,42.0,41.3,\"Formerly known as Syr1 discovered in Synechocystis sp PCC 6803, is a regulatory factor controlling photosynthetic functions. PsrR1 interacts with the ribosome binding regions of several photosynthesis-related mRNAs. In particular, psaL mRNA is processed by RNase E only in the presence of PsrR1.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,13,46,0,0,Gene; sRNA;,Predicted; CMfinder,39,0,,\"\",0.54036,-6.66673,1600000,195614,242,78,true,-3.3612,0.71878,2017-02-07T12:40:52.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02702,AgvB,2463,Anti GcvB sRNA,Argasinska J,Argasinska J,56.0,56.2,52.9,Inhibits GcvB sRNA repression.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; RNAfold;,6,0,,\"\",0.54702,-8.35557,1600000,452782,146,62,true,-3.4883,0.71925,2017-02-08T16:51:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02703,AsxR,2463,Anti stx2 sRNA,Argasinska J,Argasinska J,63.0,63.1,51.9,Increases expression of the ChuS heme oxygenase via destabilisation of FnrS sRNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,9,0,0,Gene; sRNA;,Predicted; RNAfold;,11,0,,\"\",0.48893,-7.30584,1600000,378377,137,53,true,-2.8836,0.73107,2017-02-08T17:04:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02712,EBER2,2479,Epstein-Barr virus EBER2,Quinones-Olvera N,Quinones-Olvera N,200.0,217.3,69.8,\"Highly structured, non-coding RNA highly abundant in EBV-infected cells during latency. Contributes to oncogenesis by promoting growth of tumor cells and by inducing apoptosis resistance. Known interactions include paired PAX5 and cellular protein La.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; sRNA;,Published; PMID:25662012,1,0,,\"\",0.63948,-7.15688,1600000,241356,300,173,true,-3.2804,0.71542,2017-03-07T11:18:33.000Z,2022-05-06T12:43:54.000Z\n",
            "RF02713,MCS4,2213,Mycoplasma sRNA MCS4,Quinones-Olvera N,Quinones-Olvera N,56.0,56.1,53.5,\"Abundant, small RNA of unknown function, found only in Mycoplasma. Extensive sequence similarity to eukaryotic U6 snRNAs.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,6,0,0,Gene; sRNA;,Published; PMID:9498566,7,0,,\"\",0.44061,-9.55082,1600000,339649,233,123,true,-3.1736,0.71834,2017-03-07T11:43:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02714,snopsi28S-3378,1264,Small nucleolar RNA psi28S-3378,Quinones-Olvera N,Quinones-Olvera N,136.0,136.4,37.8,psi28S-3378 is a member of the H/ACA class of snoRNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,8,9,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; CMfinder;,10,0,,\"\",0.5279,-8.19609,1600000,302446,265,145,true,-3.1046,0.71821,2017-03-07T11:51:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02723,sno_ZL1,1264,Small nucleolar RNA ZL1,Argasinska J,Argasinska J,64.0,64.3,63.9,Host gene TRRAP. Predicted target U2@U47. Only GU-rich region is homologous to other known snoRNAs. Conserved in vertebrates.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,68,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,67,0,,\"\",0.65153,-8.13619,1600000,369153,326,196,true,-3.3955,0.7127,2017-03-07T16:26:56.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02724,sno_ZL2,1264,Small nucleolar RNA ZL2,Argasinska J,Argasinska J,50.0,50.9,49.0,\"Host gene TNPO2. Predicted target 28S@U4276, U4571. Previously described as SNORD41B.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,35,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,35,0,,\"\",0.62297,-3.646,1600000,259561,166,77,true,-3.1191,0.71899,2017-03-07T16:27:48.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02725,sno_ZL8,1264,Small nucleolar RNA ZL8,Argasinska J,Argasinska J,52.0,60.4,47.2,Host gene C6orf48. No homologs outside of mammals.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,30,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,23,0,,\"\",0.66816,-2.31916,1600000,171314,176,83,true,-3.6859,0.71894,2017-03-07T16:28:23.000Z,2022-10-29T17:55:16.000Z\n",
            "RF02726,sno_ZL63,1264,Small nucleolar RNA ZL63,Argasinska J,Argasinska J,50.0,50.1,37.1,Host gene FZD6. The authors call it a mini-snoRNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,8,0,,\"\",0.64793,-4.09276,1600000,361776,121,43,true,-3.5276,0.72673,2017-03-08T15:14:33.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02727,sno_ZL116,1264,Small nucleolar RNA ZL116,Argasinska J,Argasinska J,82.0,88.4,44.7,Host gene SUPT5H.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,9,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,10,0,,\"\",0.67109,-3.06279,1600000,215514,161,72,true,-3.7564,0.71903,2017-03-08T15:14:59.000Z,2022-10-29T17:42:07.000Z\n",
            "RF02733,ToxT_thermometer,2187,ToxT 5' UTR thermometer,Argasinska J,Argasinska J,81.0,81.0,35.7,\"Controls temperature-dependant translation of toxT, facilitating V. cholerae virulance. At human body temperature, the thermometer structure opens and to allow transcriptional activator protein ToxT translation.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Cis-reg; thermoregulator;,Predicted; PMID:25228776;,4,0,,\"\",0.53822,-4.44054,1600000,233065,177,71,true,-3.6209,0.71897,2017-03-28T16:19:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02749,Ncrwmel02,2423,Wolbachia sRNA mel02,Argasinska J,Argasinska J,290.8,290.8,101.3,May play role in the interaction with the host.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; RNAfold,7,0,,\"\",0.67077,-7.70667,1600000,391539,396,251,true,-3.5605,0.70848,2017-04-25T14:09:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02750,ES003,2254,ES003 sRNA,Argasinska J,Argasinska J,37.0,37.1,35.2,Differentially expressed under several chemical stress and fermentation conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,12,0,0,Gene; sRNA;,Predicted; RNAfold,20,0,,\"\",0.55855,-8.96319,1600000,597219,132,53,true,-4.086,0.71943,2017-05-02T11:21:51.000Z,2022-05-06T12:43:53.000Z\n",
            "RF02751,ES036,2254,ES036 (CssrF) sRNA,Argasinska J,Argasinska J,37.6,37.6,37.3,Differentially expressed under several chemical stress conditions. Also identified in Shigella as CssrF [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,50,0,0,Gene; sRNA;,Predicted; RNAfold,10,0,,\"\",0.61216,-5.29433,1600000,469213,117,43,true,-3.7073,0.72278,2017-05-02T11:29:04.000Z,2022-05-06T12:43:53.000Z\n",
            "RF02752,ES056,2254,ES056 sRNA,Argasinska J,Argasinska J,43.0,43.6,40.2,Differentially expressed under several chemical stress and fermentation conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,11,0,0,Gene; sRNA;,Predicted; RNAfold,17,0,,\"\",0.52713,-6.01272,1600000,215332,190,92,true,-3.6227,0.71872,2017-05-02T11:32:57.000Z,2022-05-06T13:10:05.000Z\n",
            "RF02753,ES173,2254,ES173 sRNA,Argasinska J,Argasinska J,90.0,92.9,74.3,Differentially expressed under several chemical stress conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,2,0,0,Gene; sRNA;,Predicted; RNAfold,6,0,,\"\",0.5908,-5.91212,1600000,417719,149,64,true,-3.4109,0.71911,2017-05-02T11:34:44.000Z,2022-05-06T12:43:53.000Z\n",
            "RF02754,ES205,2254,ES205 sRNA,Argasinska J,Argasinska J,86.0,101.5,36.6,Differentially expressed under several chemical stress and fermentation conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,3,0,0,Gene; sRNA;,Predicted; RNAfold,8,0,,\"\",0.50417,-7.61888,1600000,273874,161,72,true,-3.3977,0.71896,2017-05-02T11:35:24.000Z,2022-05-06T12:43:53.000Z\n",
            "RF02755,ES222,2254,ES222 sRNA,Argasinska J,Argasinska J,62.0,63.8,58.5,Differentially expressed under several chemical stress and fermentation conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,24,0,0,Gene; sRNA;,Predicted; RNAfold,29,0,,\"\",0.54648,-5.83251,1600000,220712,213,108,true,-3.4951,0.71854,2017-05-02T11:36:48.000Z,2021-11-19T10:36:03.000Z\n",
            "RF02756,ES239,2254,ES239 sRNA,Argasinska J,Argasinska J,62.0,147.0,39.9,Differentially expressed under several chemical stress and fermentation conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,3,0,0,Gene; sRNA;,Predicted; RNAfold,8,0,,\"\",0.52674,-8.53739,1600000,315492,234,123,true,-3.105,0.71813,2017-05-02T11:37:13.000Z,2022-05-06T12:43:53.000Z\n",
            "RF02757,Erse,2254,Erse small RNA,Argasinska J,Argasinska J,138.0,138.7,137.5,May be essential for survival.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,6,29,0,0,Gene; sRNA;,Predicted; RNAfold,34,0,,\"\",0.73133,-8.20159,1600000,443208,395,252,true,-3.898,0.70845,2017-05-09T10:34:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02758,RhlA_thermometer,1694,RhlA 5' UTR ROSE like thermometer,Argasinska J,\"Grosso-Becerra et al, PMID:25313031\",85.0,85.7,75.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,1,0,0,Cis-reg; thermoregulator;,Predicted; PMID:25313031;,5,0,,\"\",0.64802,-3.41798,1600000,209846,216,110,true,-3.3429,0.71849,2017-05-09T10:41:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02759,LasI_thermometer,1694,LasI 5' UTR ROSE like thermometer,Argasinska J,\"Grosso-Becerra et al, PMID:25313031\",76.0,76.8,40.1,ROSE-like thermometer in the 5'UTR blocks the translation at lower temperature.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Cis-reg; thermoregulator;,Predicted; PMID:25313031;,3,0,,\"\",0.51698,-7.68687,1600000,233992,216,110,true,-3.3373,0.71843,2017-05-09T10:41:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02760,sR035,2440,sR035 sRNA,Argasinska J,Argasinska J,233.0,233.5,46.1,\"Temperature-dependent expression pattern. Potentially involved in Y. pestis pathogenesis. Predicted target is ymoA, a thermo-sensitive regulator.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,4,0,,\"\",0.69636,-8.18647,1600000,371916,341,206,true,-3.5397,0.71188,2017-05-17T10:26:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02761,sR084,2440,sR084 sRNA,Argasinska J,Argasinska J,95.0,99.1,38.3,\"Temperature-dependent expression pattern. Potentially involved in Y. pestis pathogenesis. Predicted target is fur, ferric uptake regulator.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.53995,-8.1011,1600000,314081,169,78,true,-3.5928,0.71901,2017-05-17T10:26:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02762,C1_109596F_thermometer,2420,C1_109596F  5' UTR thermometer,Argasinska J,Argasinska J,125.0,125.3,30.2,May regulate translation of the major cold shock protein cspA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,2,0,0,Cis-reg; thermoregulator;,Predicted; RNAfold,3,0,,\"\",0.71997,-8.24191,1600000,398081,367,227,true,-3.4717,0.71,2017-05-17T10:27:20.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02764,Ysr190,2437,Yersinia sRNA 190,Argasinska J,Argasinska J,66.0,66.2,62.2,The sRNA was identified by [2] and is confirmed to have a stable secondary structure by [1] using genome-wide structure probing at different temperatures. The function is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,9,0,0,Gene; sRNA;,Predicted; PMID:27298343;,13,0,,\"\",0.50457,-6.16955,1600000,288072,204,102,true,-3.2489,0.71843,2017-06-07T14:05:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02765,Ysr209,2437,Yersinia sRNA 209,Argasinska J,Argasinska J,59.0,59.2,52.3,The sRNA was identified by [2] and is confirmed to have a stable secondary structure by [1] using genome-wide structure probing at different temperatures. The function is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,4,0,0,Gene; sRNA;,Predicted; PMID:27298343;,8,0,,\"\",0.50537,-9.21413,1600000,523901,122,46,true,-3.5582,0.71946,2017-06-07T14:10:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02766,Ysr49,2437,Yersinia sRNA 49,Argasinska J,Argasinska J,95.0,95.2,61.3,The sRNA was identified by [2] and is confirmed to have a stable secondary structure by [1] using genome-wide structure probing at different temperatures. The function is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; PMID:27298343;,5,0,,\"\",0.54517,-8.65542,1600000,358589,235,124,true,-3.3166,0.71821,2017-06-07T14:15:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02767,Ysr186_sR026_CsrC,2437,Yersinia sRNA 186/sR026/CsrC,Argasinska J,Argasinska J,279.0,279.7,95.1,The sRNA was identified by [2] and is confirmed to have a stable secondary structure by [1] using genome-wide structure probing at different temperatures. The function is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,3,0,0,Gene; sRNA;,Predicted; PMID:27298343;,7,0,,\"\",0.67579,-8.97635,1600000,439635,482,325,true,-3.8659,0.70502,2017-06-07T14:35:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02768,Ysr155_RyfD,2437,Yersinia sRNA 155(RyfD),Argasinska J,Argasinska J,42.0,68.4,41.8,The sRNA was identified by [2] and is confirmed to have a stable secondary structure by [1] using genome-wide structure probing at different temperatures. The function is unknown.,Ysr155(RyfD);,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,103,0,0,Gene; sRNA;,Predicted; PMID:27298343;,111,0,,\"\",0.59625,-6.26985,1600000,229958,253,140,true,-2.9456,0.71821,2017-06-12T12:26:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02769,Ysr202,2437,Yersinia sRNA 202,Argasinska J,Argasinska J,315.0,315.6,34.7,The sRNA was identified by [2] and is confirmed to have a stable secondary structure by [1] using genome-wide structure probing at different temperatures. The function is unknown.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; PMID:27298343;,3,0,,\"\",0.70755,-8.66696,1600000,394291,433,282,true,-3.6231,0.7068,2017-06-12T15:15:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02770,Ysr224,2437,Yersinia sRNA 224,Argasinska J,Argasinska J,70.0,70.0,69.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,121,0,0,Gene; sRNA;,Predicted; PMID:27298343;,11,0,,\"\",0.65712,-4.56516,1600000,193747,209,107,true,-3.3589,0.71842,2017-06-12T15:16:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02771,CnfY_thermometer,2437,CnfY 5' UTR thermometer,Argasinska J,Argasinska J,130.0,130.7,38.8,CnfY gene is associated with Yersinia virulence. RNAT functionality is impaired by point mutating the 5'UTR.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,4,0,,\"\",0.49869,-3.47352,1600000,256419,218,112,true,-2.966,0.71832,2017-06-12T15:26:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02772,AilA_thermometer,2437,AilA 5' UTR thermometer,Argasinska J,Argasinska J,81.0,81.4,41.0,AilA gene is associated with Yersinia virulence. RNAT functionality is impaired by point mutating the 5'UTR. Temperature affects the secondary structure and ribosome binding efficiency of the 5'UTR.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,1,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,6,0,,\"\",0.52308,-5.24067,1600000,249899,184,89,true,-3.4932,0.71865,2017-06-12T15:27:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02773,TrxA_thermometer,2437,TrxA 5' UTR thermometer,Argasinska J,Argasinska J,58.0,58.2,57.4,Involved in protection against oxidative stress.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,8,62,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,69,0,,\"\",0.51007,-6.16167,1600000,245670,214,88,true,-3.0674,0.71872,2017-06-12T15:28:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02774,KatA_thermometer,2437,KatA 5' UTR thermometer,Argasinska J,Argasinska J,111.0,111.2,43.6,Involved in protection against oxidative stress.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,2,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,7,0,,\"\",0.47706,-9.72787,1600000,381187,273,154,true,-3.0832,0.71801,2017-06-12T15:40:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02775,SodB_thermometer,2437,SodB 5' UTR thermometer,Argasinska J,Argasinska J,98.0,98.3,65.6,Involved in protection against oxidative stress.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,2,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,7,0,,\"\",0.54953,-7.20103,1600000,350349,225,119,true,-3.3954,0.7182,2017-06-12T15:41:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02776,SodC_thermometer,2437,SodC 5' UTR thermometer,Argasinska J,Argasinska J,125.0,125.5,41.7,Involved in protection against oxidative stress.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,3,0,,\"\",0.52371,-3.65486,1600000,230485,192,93,true,-2.9671,0.71858,2017-06-12T15:41:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02777,OppA_thermometer,2437,OppA 5' UTR thermometer,Argasinska J,Argasinska J,290.0,291.6,54.5,Positioned upstream of gene for oligopeptide uptake.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,4,0,,\"\",0.67194,-9.12526,1600000,410299,461,304,true,-4.0255,0.70576,2017-06-12T15:42:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02778,FdoG1_thermometer,2437,FdoG-1 5' UTR thermometer,Argasinska J,Argasinska J,50.0,50.1,49.7,Located upstream of fdoG-1 gene which is involved in metabolic processes.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,14,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,17,0,,\"\",0.65995,-8.28243,1600000,371365,328,193,true,-3.2625,0.7129,2017-06-12T15:43:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02779,PepN_thermometer,2437,PepN 5' UTR thermometer,Argasinska J,Argasinska J,83.0,83.9,38.2,Position upstream of pepN gene responsible for amino acid processing.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,3,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,7,0,,\"\",0.51872,-7.39047,1600000,242165,293,136,true,-2.8916,0.71822,2017-06-12T15:50:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02780,PutA_thermometer,2437,PutA 5' UTR thermometer,Argasinska J,Argasinska J,51.0,51.8,40.2,Position upstream of pepN gene responsible for amino acid processing.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,2,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,6,0,,\"\",0.50484,-9.61254,1600000,526098,160,72,true,-3.1356,0.71893,2017-06-12T15:51:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02781,ManX_thermometer,2437,ManX 5' UTR thermometer,Argasinska J,Argasinska J,76.0,76.3,37.0,Short UTR positioned upstream of ManX gene responsible for sugar uptake.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,3,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,7,0,,\"\",0.62089,-3.03025,1600000,223695,223,117,true,-3.3979,0.71839,2017-06-12T15:52:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02782,CpoB_ybgF_thermometer,2437,CpoB/ybgF ICR thermometer,Argasinska J,Argasinska J,51.0,51.4,50.6,Imprinting control region from upstream of cpoB/ybgF (tol/pal system protein).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,8,137,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,142,0,,\"\",0.54752,-8.18257,1600000,351180,254,139,true,-3.0858,0.71823,2017-06-12T15:58:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02783,IscS1_thermometer,2437,IscS-1 ICR thermometer,Argasinska J,Argasinska J,58.0,58.2,54.0,Imprinting control region from upstream of iscS-1 (cysteine desulfurase).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,31,0,0,Cis-reg; thermoregulator;,Predicted; PMID:27298343;,35,0,,\"\",0.52474,-9.25252,1600000,367886,307,183,true,-3.1447,0.714,2017-06-12T16:26:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02784,SorX,2430,Singlet oxygen resistance RNA X,Argasinska J,Argasinska J,46.6,46.6,46.5,SorX (RSs2461)like SorY is a singlet oxygen induced sRNA that counteracts oxidative stress by targeting mRNA for a transporter in Rhodobacter. It also has an impact on resistance against organic hydroperoxides. It targets SD sequence of potA mRNA via an aSD sequence. The aSD sequence and some neighbouring nucleotides are identical to the one of SorY (targets takP mRNA).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,8,271,0,0,Gene; sRNA;,Predicted; 27420112;,172,0,,\"\",0.67702,-5.24454,1600000,252929,157,73,true,-3.5042,0.71891,2017-10-02T09:59:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02785,snoRNA2,1264,Trypanosomatid small nucleolar RNA 2,Argasinska J,Argasinska J,108.0,108.4,32.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,12,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,5,0,,\"\",0.60841,-4.90188,1600000,266274,177,83,true,-3.3041,0.71876,2017-10-02T10:23:52.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02791,CcsR1,2430,Conserved CCUCCUCCC motif stress-induced RNA 1,Argasinska J,Argasinska J,65.3,65.3,65.0,A cluster of four homologous sRNAs called CcsR 1-4 has been shown to play a role in photo-oxidative stress resistance.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 605271.01655 --mxsize 128 CM SEQDB,2,8,0,0,Gene; sRNA;,Predicted; 25777678;,7,0,,\"\",0.681,-5.88205,1600000,296150,177,83,true,-2.7544,0.72504,2017-10-11T16:27:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02797,PssrA,1236,Plasmid-encoded Shigella sRNA A,Argasinska J,Argasinska J,69.0,69.5,63.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,6,38,0,0,Gene; sRNA;,Predicted; RNAfold,42,0,,\"\",0.52433,-7.06334,1600000,294893,191,93,true,-2.9231,0.71864,2017-10-25T09:15:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02798,CssrA,1236,Chromosome-encoded Shigella sRNA A,Argasinska J,Argasinska J,93.0,93.3,60.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,7,0,0,Gene; sRNA;,Predicted; RNAfold,9,0,,\"\",0.55708,-9.38914,1600000,384065,217,111,true,-3.0384,0.71857,2017-10-25T09:15:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02799,CssrB,1236,Chromosome-encoded Shigella sRNA B,Argasinska J,Argasinska J,76.0,76.3,51.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,2,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.56097,-9.17535,1600000,359691,309,180,true,-3.1697,0.71445,2017-10-25T09:16:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02800,Rp_sR47,2415,Rickettsia sRNA47,Argasinska J,Argasinska J,60.0,60.0,59.5,Expressed in human microvascular endothelial cells (HMEC) infected with Rickettsia.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,27,0,0,Gene; sRNA;,Predicted; RNAfold,27,0,,\"\",0.67345,-8.60623,1600000,407121,474,316,true,-4.0532,0.70524,2017-10-31T09:57:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02813,PA5194_thermometer,1269,Pseudomonas PA5194 thermometer,Argasinska J,Argasinska J,60.0,60.3,39.1,\"Identified in genetic Tet-Tap screening, no obvious similarities to known RNATs.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,3,0,0,Cis-reg; thermoregulator;,Published; PMID:25336583,3,0,,\"\",0.5411,-9.23935,1600000,340308,205,105,true,-3.5368,0.71881,2017-11-20T10:58:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02822,Srn266,2341,Streptococcus RNA 266,Argasinska J,Argasinska J,63.0,63.8,62.7,SP_0836 target predicted.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,187,0,0,Gene; sRNA;,Predicted; 22274957,40,0,,\"\",0.47142,-9.52659,1600000,364112,216,112,true,-3.2331,0.71849,2017-12-04T14:19:40.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02826,Scr6925,2421,Streptomyces RNA 6925,Argasinska J,Argasinska J,50.0,50.5,43.3,Identified by deep sequencing of transcriptome at the end of exponential growth phase. Sequencing size consistent with northern blot size. Streptomyces specific. Small 12 nt possible coding region observed.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,38,0,0,Gene; sRNA;,Predicted; 21521948,37,0,,\"\",0.47077,-10.99946,1600000,364373,242,131,true,-2.8229,0.71843,2017-12-11T10:33:03.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02827,Scr6106,2421,Streptomyces RNA 6106,Argasinska J,Argasinska J,52.5,52.5,52.1,Identified by deep sequencing of transcriptome at the end of exponential growth phase. Appears smaller on northern blot than sequencing transcript. This indicates processing step. Streptomyces specific.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,63,0,0,Gene; sRNA;,Predicted; 21521948,67,0,,\"\",0.37277,-12.8025,1600000,375787,179,85,true,-2.9797,0.71896,2017-12-11T10:36:42.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02828,Scr3920,2421,Streptomyces RNA 3920,Argasinska J,Argasinska J,88.0,88.4,84.7,Identified by deep sequencing of transcriptome at the end of exponential growth phase. Sequencing size consistent with northern blot size. Streptomyces specific.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,130,0,0,Gene; sRNA;,Predicted; 21521948,136,0,,\"\",0.52547,-10.05059,1600000,368212,236,125,true,-3.0265,0.71843,2017-12-11T10:37:07.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02829,Scr4115,2421,Streptomyces RNA 4115,Argasinska J,Argasinska J,58.0,58.1,54.7,Identified by deep sequencing of transcriptome at the end of exponential growth phase. Sequencing size consistent with notrthern blot size. Streptomyces specific.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,173,0,0,Gene; sRNA;,Predicted; 21521948,178,0,,\"\",0.39125,-11.11934,1600000,243131,213,109,true,-3.0734,0.71862,2017-12-11T10:37:32.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02830,Scr1601,2421,Streptomyces RNA 1601,Argasinska J,Argasinska J,100.0,101.6,97.1,Identified by deep sequencing of transcriptome at the end of exponential growth phase. Appears longer on northern blot than sequencing transcript. Streptomyces specific.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,172,0,0,Gene; sRNA;,Predicted; 21521948,176,0,,\"\",0.54117,-10.36901,1600000,295839,341,208,true,-3.2343,0.71174,2017-12-11T10:37:59.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02831,Scr2736,2421,Streptomyces RNA 2736,Argasinska J,Argasinska J,43.0,43.0,42.9,Identified by deep sequencing of transcriptome at the end of exponential growth phase. Appears smaller on northern blot than sequencing transcript. This indicates processing step. Streptomyces specific.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,163,0,0,Gene; sRNA;,Predicted; 21521948,166,0,,\"\",0.53074,-6.44211,1600000,294246,150,64,true,-3.3448,0.7192,2017-12-11T10:38:30.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02832,Scr5676,2421,Streptomyces RNA 5676,Argasinska J,Argasinska J,56.0,56.9,48.4,Identified by deep sequencing of transcriptome at the end of exponential growth phase. Appears longer on northern blot than sequencing transcript. This indicates processing step. Streptomyces specific.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,159,0,0,Gene; sRNA;,Predicted; 21521948,164,0,,\"\",0.58248,-6.06262,1600000,225294,241,130,true,-2.5699,0.71834,2017-12-11T14:38:11.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02833,Scr3202,2421,Streptomyces RNA 3202,Argasinska J,Argasinska J,52.0,52.2,44.8,Identified by deep sequencing of transcriptome at the end of exponential growth phase. Appears smaller on northern blot than sequencing transcript. This indicates processing step. Streptomyces specific.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,76,0,0,Gene; sRNA;,Predicted; 21521948,80,0,,\"\",0.42083,-10.26252,1600000,281991,188,92,true,-2.7521,0.71896,2017-12-11T14:38:47.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02835,Bcj11,2451,Burkholderia RNA 11,Argasinska J,Argasinska J,49.0,49.4,48.6,\"Co-purified with Hfq, expression confirmed by Northern blot. Predicted target marZ. Overlaps coding sequence.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,152,0,0,Gene; sRNA;,Predicted; RNAfold,152,0,,\"\",0.5028,-8.37359,1600000,263227,229,122,true,-2.9743,0.7184,2017-12-13T16:17:51.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02836,Bcj14,2451,Burkholderia RNA 14,Argasinska J,Argasinska J,59.0,59.9,58.8,\"Co-purified with Hfq, expression confirmed by Northern blot. Putative functional homologue of PA-SraF. Overlaps coding sequence.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,91,0,0,Gene; sRNA;,Predicted; RNAfold,74,0,,\"\",0.49648,-10.67087,1600000,344833,296,171,true,-2.7785,0.71578,2017-12-13T16:19:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02838,Ref55,2426,Enterococcus sRNA 55,\"Argasinska J, Repoila F\",Argasinska J,43.0,43.8,42.2,Name â€˜Refâ€™ stands for RNA in E. faecalis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,9,0,0,Gene; sRNA;,Predicted; RNAfold;,9,0,,\"\",0.48598,-6.22813,1600000,398989,131,54,true,-3.1808,0.72888,2018-01-22T11:47:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02839,Ref66,2426,Enterococcus sRNA 66,\"Argasinska J, Repoila F\",Argasinska J,92.0,92.2,40.0,Name â€˜Refâ€™ stands for RNA in E. faecalis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB 128 CM SEQDB,2,8,0,0,Gene; antisense;,Predicted; RNAfold;,3,0,,\"\",0.54522,-7.78035,1600000,274678,224,118,true,-3.4782,0.7184,2018-01-22T11:48:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02840,Ref68,2426,Enterococcus sRNA 68 (Lacto-3 RNA),\"Argasinska J, Repoila F, Weinberg Z\",Weinberg Z,44.0,44.0,43.6,Name â€˜Refâ€™ stands for RNA in E. faecalis[1]. Also known as Lacto-3 RNA.[2] Lactobacillales (Firmicutes). Lineage (Lactobacillaceae)[2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.12 -Z 742849.287494 --mxsize 128 CM SEQDB,78,125,0,0,Gene; sRNA;,Published; PMID:28977401;,168,0,,\"\",0.58176,-9.9033,1600000,473899,301,124,true,-3.2175,0.71824,2018-01-22T11:49:16.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02841,Ref70,2426,Enterococcus sRNA 70,\"Argasinska J, Repoila F\",Argasinska J,115.0,115.7,49.5,Name â€˜Refâ€™ stands for RNA in E. faecalis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,10,0,0,Gene; sRNA;,Predicted; RNAfold;,10,0,,\"\",0.54347,-6.17703,1600000,315106,229,122,true,-2.7522,0.71829,2018-01-22T11:49:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02842,RefA1,2426,Enterococcus sRNA A1,\"Argasinska J, Repoila F\",Argasinska J,94.0,94.2,61.3,Name â€˜Refâ€™ stands for RNA in E. faecalis. Encoded by pTEF1 plasmid.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,8,0,0,Gene; sRNA;,Predicted; RNAfold;,7,0,,\"\",0.59887,-8.08229,1600000,351031,358,222,true,-3.186,0.71029,2018-01-22T11:51:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02843,RefA5,2426,Enterococcus sRNA A5,\"Argasinska J, Repoila F\",Argasinska J,91.0,91.8,39.6,Name â€˜Refâ€™ stands for RNA in E. faecalis. Encoded by pTEF1 plasmid.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,6,0,0,Gene; antisense;,Predicted; RNAfold;,2,0,,\"\",0.54877,-9.46977,1600000,581250,154,68,true,-3.4433,0.71911,2018-01-22T11:51:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02844,RefA9,2426,Enterococcus sRNA A9,\"Argasinska J, Repoila F\",Argasinska J,49.0,49.6,46.5,Name â€˜Refâ€™ stands for RNA in E. faecalis. Encoded by pTEF1 plasmid.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,13,0,0,Gene; antisense;,Predicted; RNAfold;,10,0,,\"\",0.47593,-8.95298,1600000,492889,145,64,true,-3.0083,0.7245,2018-01-22T11:52:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02845,Ref1C,2426,Enterococcus sRNA 1C,\"Argasinska J, Repoila F\",Argasinska J,90.0,90.4,48.1,Name â€˜Refâ€™ stands for RNA in E. faecalis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,6,0,0,Gene; sRNA;,Predicted; RNAfold;,5,0,,\"\",0.47239,-7.18296,1600000,282331,238,130,true,-2.769,0.7183,2018-01-22T12:16:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02846,Ref84,2426,Enterococcus sRNA 84,\"Argasinska J, Repoila F\",Argasinska J,52.0,52.5,48.2,Name â€˜Refâ€™ stands for RNA in E. faecalis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,45,0,0,Gene; antisense;,Predicted; RNAfold;,33,0,,\"\",0.42021,-8.58299,1600000,300737,178,87,true,-3.5322,0.71867,2018-01-22T12:29:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02847,Ref64,2426,Enterococcus sRNA 64,\"Argasinska J, Repoila F\",Argasinska J,44.3,44.3,44.0,Name â€˜Refâ€™ stands for RNA in E. faecalis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,20,0,0,Gene; sRNA;,Predicted; RNAfold;,13,0,,\"\",0.61823,-3.5911,1600000,274947,155,69,true,-3.8049,0.71908,2018-01-22T12:33:12.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02848,RefB11,2426,Enterococcus sRNA B11,\"Argasinska J, Repoila F\",Argasinska J,54.0,54.6,53.6,Name â€˜Refâ€™ stands for RNA in E. faecalis. Encoded by pTEF2 plasmid. 22 nucleotides removed from 3'end of original sequence as it overlapped with RF02844 (RefA9).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,39,0,0,Gene; sRNA;,Predicted; RNAfold;,30,0,,\"\",0.54796,-8.80476,1600000,286474,329,167,true,-3.5727,0.71621,2018-01-23T11:45:46.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02849,Ysr197,2437,Yersinia sRNA 197,Argasinska J,Argasinska J,167.0,167.7,155.9,\"5' located ORF: gmhB, 3' located ORF:16S rRNA. Repressed during phase.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,8,0,0,Gene; sRNA;,Predicted; RNAfold,9,0,,\"\",0.64074,-8.93734,1600000,363123,395,254,true,-3.5669,0.70827,2018-02-05T15:13:33.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02850,Ysr276,2437,Yersinia sRNA 276,Argasinska J,Argasinska J,100.0,100.1,96.9,Antisense to pepA gene.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,132,0,0,Gene; antisense;,Predicted; RNAfold,131,0,,\"\",0.75376,-7.34772,1600000,324329,324,195,true,-3.3423,0.71293,2018-02-05T15:34:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02851,Ysr283,2437,Yersinia sRNA 283,Argasinska J,Argasinska J,58.0,58.0,54.1,Identified in transcriptional profiling by RNA seq.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,9,0,0,Gene; antisense;,Predicted; RNAfold,9,0,,\"\",0.71482,-8.04965,1600000,335706,395,253,true,-3.7381,0.70831,2018-02-05T16:01:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02852,Ysr206,2437,Yersinia sRNA 206,Argasinska J,Argasinska J,124.0,124.9,41.9,\"5' located ORF: rssB. Exclusively detected in the crp mutant, suggesting CRP regulon is a regulator of the sRNA. CRP binding in a cAMP dependent manner was confirmed.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,3,0,0,Gene; sRNA;,Predicted; RNAfold,4,0,,\"\",0.6952,-7.33351,1600000,351196,463,310,true,-3.8311,0.70556,2018-02-05T16:17:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02853,Ysr201,2437,Yersinia sRNA 201,Argasinska J,Argasinska J,62.0,70.3,33.2,Increased expression in the crp mutant in the stationary phases at 25 degree suggesting CRP regulon is a regulator of the sRNA.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,3,3,0,0,Gene; sRNA;,Predicted; RNAfold,6,0,,\"\",0.6659,-2.96766,1600000,259466,158,70,true,-3.7273,0.71906,2018-02-05T16:26:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02854,Ysr100,2437,Yersinia sRNA 100,Argasinska J,Argasinska J,47.0,47.1,36.2,\"5' located ORF: qor, 3' located ORF: pspG. Decreased expression in the crp mutant (in the stationary phase), suggesting CRP regulon is a regulator of the sRNA. CRP binding in a cAMP dependent manner was confirmed.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; RNAfold,6,0,,\"\",0.54143,-7.67409,1600000,321912,228,120,true,-3.0689,0.71825,2018-02-05T16:31:42.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02855,Ysr251,2437,Yersinia sRNA 251,Argasinska J,Argasinska J,100.0,102.8,86.2,Antisense to phoH gene. Trimmed on 5'end from the original size.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,94,0,0,Gene; antisense;,Predicted; RNAfold,95,0,,\"\",0.6849,-8.26322,1600000,362765,321,194,true,-3.4559,0.71291,2018-02-05T16:38:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02865,ncS01,2451,Burkholderia sRNA 1,Argasinska J,Argasinska J,58.0,58.8,39.5,\"Identified by RNA-seq, RACE and Northern blot. Strongly expressed in biofilm.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,52,0,0,Gene; sRNA;,Predicted; RNAfold;,52,0,,\"\",0.50287,-9.02341,1600000,287415,214,110,true,-3.1676,0.71867,2018-02-27T15:34:49.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02866,ncS16,2451,Burkholderia sRNA 16 (Bc_KC_sr1),Argasinska J,Argasinska J,48.0,48.5,39.7,\"Identified by RNA-seq, RACE and Northern blot. Strongly expressed in biofilm [1]. Also know as Bc_KC_sr1 [2]. Upregulated in response to iron ion depletion. Predicted targets include cysteine desulfurase and isochorismatase desulfurase.[2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,55,0,0,Gene; sRNA;,Predicted; RNAfold;,55,0,,\"\",0.55311,-7.76638,1600000,290642,175,83,true,-3.5785,0.71881,2018-02-27T15:35:52.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02867,ncS011,2451,Burkholderia sRNA 11,Argasinska J,Argasinska J,48.0,48.1,43.6,\"Identified by RNA-seq, RACE and Northern blot. Strongly expressed in biofilm.[1] Identified in [2] as ncRNA13. It is complimentary by 40 nucleotides with a gene encoding a regulatory protein (BCAL1948). ncS11 is strongly up-regulated in the stationary phase and biofilms, while BCAL1948 is down-regulated under these conditions.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,51,0,0,Gene; sRNA;,Predicted; RNAfold;,51,0,,\"\",0.66705,-8.42402,1600000,347544,340,207,true,-3.1027,0.71189,2018-02-27T15:36:20.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02868,ncS037,2451,Burkholderia sRNA 37,Argasinska J,Argasinska J,112.0,112.5,46.1,\"Identified by RNA-seq, RACE and Northern blot. Strongly expressed in biofilm.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; RNAfold;,4,0,,\"\",0.46259,-11.04792,1600000,309216,269,152,true,-2.6134,0.71827,2018-02-27T15:37:19.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02869,ncS025,2451,Burkholderia sRNA 25,Argasinska J,Argasinska J,46.0,46.2,40.4,\"Identified by RNA-seq, RACE and Northern blot. Strongly expressed in biofilm.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,52,0,0,Gene; sRNA;,Predicted; RNAfold;,53,0,,\"\",0.51013,-9.47261,1600000,407950,180,85,true,-3.4532,0.71881,2018-02-27T15:37:55.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02870,ncS035,2451,Burkholderia sRNA 35,Argasinska J,Argasinska J,79.0,79.1,34.7,\"Identified by RNA-seq, RACE and Northern blot. Strongly expressed in biofilm.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,14,0,0,Gene; sRNA;,Predicted; RNAfold;,14,0,,\"\",0.61383,-8.45996,1600000,285259,290,168,true,-3.0153,0.71614,2018-02-27T15:38:26.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02871,ncS54,2451,Burkholderia sRNA 54,Argasinska J,Argasinska J,43.0,43.0,42.5,\"Identified by RNA-seq, RACE and Northern blot. Strongly expressed in biofilm.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,91,0,0,Gene; sRNA;,Predicted; RNAfold;,53,0,,\"\",0.65516,-5.11536,1600000,232863,172,79,true,-3.1355,0.71894,2018-03-05T12:09:16.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02883,BcKCsr2,2451,Burkholderia sRNA 2,Argasinska J,Argasinska J,40.0,40.5,37.8,Upregulated in response to iron ion depletion. Predicted targets include MerR family transcriptional regulator.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,46,0,0,Gene; sRNA;,Predicted; RNAfold,46,0,,\"\",0.59818,-8.2318,1600000,288935,345,130,true,-2.7542,0.71826,2018-03-06T15:23:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02884,BcKCsr7,2451,Burkholderia sRNA 7,Argasinska J,Argasinska J,59.0,59.0,46.9,\"Did not show any change in expression in iron-depletion and hydrogen peroxide (oxidative stress) treatment. Constitutively expressed. Possibly targets the GTPase, ObgE.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,51,0,0,Gene; sRNA;,Predicted; R-scape;,51,0,,\"\",0.57021,-8.00449,1600000,269734,290,157,true,-3.1081,0.71782,2018-03-06T15:26:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02886,npcTB_6715,2178,Mycobacterium sRNA 6715,Argasinska J,Argasinska J,72.0,72.0,66.2,First sRNA identified as a potential biomarker for the detection of MTB in patients. Overlaps partially the 5'leader and SDS of hypothetical protein LH_10405.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,38,0,0,Gene; sRNA;,Predicted; RNAfold;,38,0,,\"\",0.66816,-8.46904,1600000,377829,343,212,true,-3.1395,0.71131,2018-03-07T12:41:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02890,SprC,2458,Small pathogenicity island RNA C (srn_3610),Argasinska J,Argasinska J,100.0,147.0,79.6,Reduces autolysin ALT expression by binding to the mRNA and preventing ribosome loading. Reduces virulence and bacterial load in mouse model.[1] SarA TF represses its transcription by binding to an essential 22 bp region.[2],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; sRNA;,Predicted; RNAfold,2,0,,\"\",0.45798,-9.01351,1600000,317940,266,150,true,-2.8712,0.71797,2018-05-23T11:49:45.000Z,2022-05-06T13:10:07.000Z\n",
            "RF02891,AgrB,2254,ArsR-gov region gene B,Argasinska J,Argasinska J,40.0,40.5,39.4,arsR-gov region gene B is transcribed opposite of dinQ (translates into a toxic single transmembrane peptide) with 30 complimentary nucleotides. AgrB appears to repress accumulation of dinQ by RNA interference and counteracts its toxicity.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,4,22,0,0,Gene; antisense;,Predicted; RNAfold,17,0,,\"\",0.50669,-7.51056,1600000,288649,175,84,true,-3.1524,0.71878,2018-05-23T12:02:06.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02894,S35,2458,Staphylococcus sRNA 35 (srn_0335),Argasinska J,Argasinska J,69.0,69.6,67.0,\"Other name: Srn_0335 [2], SAOUHSCs258 [3]. Northern blot transcript confirmed in exponentially growing and stationary cells in three cultivation media. Trans-encoded.[1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,3,64,0,0,Gene; sRNA;,Predicted; RNAfold,9,0,,\"\",0.67895,-7.74624,1600000,350559,501,270,true,-3.9121,0.70745,2018-06-18T14:47:01.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02895,S414,2458,Staphylococcus sRNA 414,Argasinska J,Argasinska J,45.0,45.6,43.8,Antisense of an annotated CDS (SAOUHSC_1422) but was considered an trans-encoded sRNA rather than a typical antisense RNA [1]. Target prediction revealed enrichment of iron-coding genes and genes with function in cofator metabolism.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,5,38,0,0,Gene; sRNA;,Predicted; RNAfold,43,0,,\"\",0.47714,-9.27228,1600000,398008,295,175,true,-3.0715,0.71498,2018-06-18T14:50:45.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02896,S774,2458,Staphylococcus sRNA 774,Argasinska J,Argasinska J,48.0,48.4,47.0,Northern blot transcript confirmed in exponentially growing and stationary cells in three cultivation media. Trans-encoded.[1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,4,24,0,0,Gene; sRNA;,Predicted; R-scape;,28,0,,\"\",0.41917,-8.87465,1600000,329636,194,95,true,-2.7839,0.7184,2018-06-18T14:53:19.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02897,S808,2458,Staphylococcus sRNA 808,Argasinska J,Argasinska J,76.0,76.8,41.8,Other name: Srn9345[2]. Own promoter plus 3'UTR of SAOUHSC_02016.[2] Highly expressed under all conditions tested. Northern blot transcript confirmed in exponentially growing and stationary cells in three cultivation media. Trans-encoded.[1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,4,22,0,0,Gene; sRNA;,Predicted; RNAfold,26,0,,\"\",0.64857,-8.96388,1600000,480803,397,255,true,-3.4366,0.70814,2018-06-18T14:54:15.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02907,patAB,2341,patAB leader,Argasinska J,Argasinska J,92.0,92.7,41.9,Mutation in hairpin loop weakens it and seems to increase the transcription of the downstream CDS.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,2,2,0,0,Cis-reg; leader;,Predicted; RNAfold;,4,0,,\"\",0.55806,-4.33586,1600000,356793,143,57,true,-3.2628,0.72817,2018-06-18T16:32:04.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02908,rli30,1267,Listeria sRNA rli30,Argasinska J,Argasinska J,41.0,41.8,36.8,Coevolved with internalin gene lmo2445. Paired with CDS partners lmo0501 and lmo0508.[2] Antisense to lmo0506 [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,3,17,0,0,Gene; antisense;,Predicted; RNAfold;,19,0,,\"\",0.52667,-7.95698,1600000,283978,221,116,true,-2.9455,0.71838,2018-06-18T16:36:07.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02909,rli117,1267,Listeria sRNA rli117,Argasinska J,Argasinska J,87.0,87.6,84.5,\"Coevolved with internalin genes lmo0549, lmo0263, lmo02470. Internalins are important virulence factors.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,3,12,0,0,Gene; sRNA;,Predicted; RNAfold;,8,0,,\"\",0.66199,-7.98137,1600000,354385,415,272,true,-3.3428,0.70727,2018-06-18T16:38:39.000Z,2019-01-10T15:16:02.000Z\n",
            "RF02922,RAGATH-32,2449,RAGATH-32 RNA,Weinberg Z,Weinberg Z,41.8,41.8,36.6,Legionellales (_-Proteobacteria). Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.80 -Z 742849.287494 --mxsize 128 CM SEQDB,19,37,0,0,Cis-reg;,Published; PMID:28977401;,33,0,,\"\",0.61586,-8.07702,1600000,390097,166,78,true,-3.2295,0.71872,2018-07-03T14:52:21.000Z,2019-01-17T14:00:27.000Z\n",
            "RF02944,c4-2,2102,c4-2 RNA,Weinberg Z,Weinberg Z,41.0,41.1,40.5,\"Bacteroidetes, some Proteobacteria. Lineage (Flavobacteria)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,224,81,0,0,Gene; sRNA;,Published; PMID:28977401;,45,0,,\"\",0.65866,-3.20537,1600000,309837,130,79,true,-3.9001,0.72199,2018-12-12T16:40:04.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03047,RAGATH-16,2449,RAGATH-16 RNA,Weinberg Z,Weinberg Z,65.6,65.6,33.9,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.71 -Z 742849.287494 --mxsize 128 CM SEQDB,18,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.57532,-6.04733,1600000,330086,149,72,true,-4.116,0.71881,2018-12-21T10:24:30.000Z,2019-01-10T15:16:02.000Z\n",
            "RF03048,RAGATH-20,2449,RAGATH-20 RNA,Weinberg Z,Weinberg Z,101.4,101.4,40.5,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.70 -Z 742849.287494 --mxsize 128 CM SEQDB,9,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.46534,-9.65854,1600000,308794,201,102,true,-3.1549,0.71836,2018-12-21T10:31:32.000Z,2019-01-10T15:16:02.000Z\n",
            "RF03049,RAGATH-24,2449,RAGATH-24 RNA,Weinberg Z,Weinberg Z,45.3,45.3,31.0,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.26 -Z 742849.287494 --mxsize 128 CM SEQDB,22,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.6509,-2.98784,1600000,207546,611,103,true,-3.0824,0.71853,2018-12-21T10:38:32.000Z,2019-01-10T15:16:02.000Z\n",
            "RF03050,RAGATH-25,2449,RAGATH-25 RNA,Weinberg Z,Weinberg Z,49.0,49.0,36.4,Firmicutes. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.66 -Z 742849.287494 --mxsize 128 CM SEQDB,21,2,0,0,Gene; sRNA;,Published; PMID:28977401;,5,0,,\"\",0.67556,-7.06664,1600000,576644,112,50,true,-3.579,0.72039,2018-12-21T10:41:15.000Z,2019-01-10T15:16:02.000Z\n",
            "RF03051,RAGATH-26,2449,RAGATH-26 RNA,Weinberg Z,Weinberg Z,86.3,86.3,34.1,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.74 -Z 742849.287494 --mxsize 128 CM SEQDB,8,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.51888,-8.6302,1600000,338521,180,76,true,-3.5113,0.7187,2018-12-21T10:43:11.000Z,2019-01-10T15:16:02.000Z\n",
            "RF03052,RAGATH-28,2449,RAGATH-28 RNA,Weinberg Z,Weinberg Z,45.3,45.3,44.7,Clostridiales (Firmicutes). Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.14 -Z 742849.287494 --mxsize 128 CM SEQDB,85,10,0,0,Gene; sRNA;,Published; PMID:28977401;,15,0,,\"\",0.48611,-11.79311,1600000,579012,156,85,true,-2.9247,0.71854,2018-12-21T10:46:33.000Z,2019-01-10T15:16:02.000Z\n",
            "RF03055,RAGATH-30,2449,RAGATH-30 RNA,Weinberg Z,Weinberg Z,59.5,59.5,37.9,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.59 -Z 742849.287494 --mxsize 128 CM SEQDB,33,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.54913,-8.8188,1600000,600873,130,59,true,-3.5217,0.71908,2018-12-21T11:05:28.000Z,2019-01-10T15:16:02.000Z\n",
            "RF03056,RAGATH-35,2449,RAGATH-35 RNA,Weinberg Z,Weinberg Z,38.9,38.9,35.4,Clostridium (Firmicutes). Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.13 -Z 742849.287494 --mxsize 128 CM SEQDB,16,7,0,0,Gene; sRNA;,Published; PMID:28977401;,9,0,,\"\",0.62032,-3.53717,1600000,211319,175,87,true,-3.9438,0.71854,2018-12-21T11:10:57.000Z,2019-01-10T15:16:02.000Z\n",
            "RF03064,RAGATH-18,2449,RAGATH-18 RNA,Weinberg Z,Weinberg Z,44.9,45.0,44.3,\"Actinobacteria, Firmicutes, Fusobacteria, Thermotogae (1 example). Nearby to self-cleaving ribozyme genes\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.12 -Z 742849.287494 CM SEQDB,1420,355,0,0,Gene; sRNA;,Published; PMID:28977401;,317,0,,\"\",0.59597,-4.60276,1600000,352571,176,71,true,-3.3532,0.71876,2018-12-23T18:08:33.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03095,RAGATH-17,2449,RAGATH-17 RNA,Weinberg Z,Weinberg Z,102.3,102.3,40.8,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 35.91 -Z 742849.287494 --mxsize 128 CM SEQDB,7,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.42937,-7.75458,1600000,299304,234,96,true,-2.977,0.71841,2019-01-15T10:23:52.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03096,RAGATH-19,2449,RAGATH-19 RNA,Weinberg Z,Weinberg Z,85.6,85.6,41.1,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.43 -Z 742849.287494 --mxsize 128 CM SEQDB,11,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.45041,-10.38585,1600000,325492,245,141,true,-3.3185,0.71806,2019-01-15T10:25:22.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03097,RAGATH-21,2449,RAGATH-21 RNA,Weinberg Z,Weinberg Z,70.3,70.3,35.2,Clostridiales (Firmicutes). Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.46 -Z 742849.287494 --mxsize 128 CM SEQDB,38,0,0,0,Gene; sRNA;,Published; PMID:28977401;,5,0,,\"\",0.57887,-4.89727,1600000,293430,297,77,true,-3.7661,0.71883,2019-01-15T10:26:17.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03098,RAGATH-22,2449,RAGATH-22 RNA,Weinberg Z,Weinberg Z,120.3,120.3,44.2,Veillonella (Firmicutes). Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.67 -Z 742849.287494 --mxsize 128 CM SEQDB,18,1,0,0,Gene; sRNA;,Published; PMID:28977401;,2,0,,\"\",0.53118,-9.17092,1600000,398693,285,166,true,-3.0562,0.71618,2019-01-15T10:27:09.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03099,RAGATH-23,2449,RAGATH-23 RNA,Weinberg Z,Weinberg Z,92.6,92.6,47.0,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.15 -Z 742849.287494 --mxsize 128 CM SEQDB,11,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.58594,-5.36645,1600000,225782,194,99,true,-3.2355,0.71846,2019-01-15T10:28:04.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03100,RAGATH-27,2449,RAGATH-27 RNA,Weinberg Z,Weinberg Z,76.1,76.1,41.5,Clostridiales (Firmicutes). Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.82 -Z 742849.287494 --mxsize 128 CM SEQDB,15,1,0,0,Gene; sRNA;,Published; PMID:28977401;,3,0,,\"\",0.4249,-9.82026,1600000,371954,172,86,true,-3.2313,0.71851,2019-01-15T10:28:58.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03101,RAGATH-31,2449,RAGATH-31 RNA,Weinberg Z,Weinberg Z,71.8,71.8,37.2,Oscillibacter (Firmicutes). Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.54 -Z 742849.287494 --mxsize 128 CM SEQDB,26,1,0,0,Gene; sRNA;,Published; PMID:28977401;,3,0,,\"\",0.61166,-4.45115,1600000,368464,131,54,true,-3.63,0.71945,2019-01-15T10:31:00.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03102,RAGATH-33,2449,RAGATH-33 RNA,Weinberg Z,Weinberg Z,120.4,120.4,33.6,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.88 -Z 742849.287494 --mxsize 128 CM SEQDB,7,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.57212,-8.24498,1600000,366726,308,113,true,-2.4995,0.71829,2019-01-15T10:32:31.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03103,RAGATH-34,2449,RAGATH-34 RNA,Weinberg Z,Weinberg Z,60.7,60.7,37.8,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.91 -Z 742849.287494 --mxsize 128 CM SEQDB,24,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.48307,-6.55367,1600000,252327,202,106,true,-3.239,0.71837,2019-01-15T10:33:20.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03104,RAGATH-36,2449,RAGATH-36 RNA,Weinberg Z,Weinberg Z,80.7,80.7,36.1,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.51 -Z 742849.287494 --mxsize 128 CM SEQDB,14,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.58814,-8.83046,1600000,391436,359,129,true,-3.2636,0.71821,2019-01-15T10:34:18.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03105,RAGATH-7-assoc,2449,RAGATH-7-assoc RNA,Weinberg Z,Weinberg Z,88.5,88.5,38.1,Environmental. Nearby to self-cleaving ribozyme genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.58 -Z 742849.287494 --mxsize 128 CM SEQDB,26,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.55529,-5.84707,1600000,247967,235,131,true,-3.168,0.71821,2019-01-15T10:35:22.000Z,2019-01-17T14:00:27.000Z\n",
            "RF03126,L20-Deltaproteobacteria,1804,L20-Deltaproteobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,42.8,42.8,35.0,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,117,33,0,0,Cis-reg; leader;,Published; PMID:32448158;,41,0,,\"\",0.57083,-6.89704,1600000,325492,223,75,true,-4.0345,0.71888,2020-07-07T16:45:40.000Z,2020-09-14T15:04:32.000Z\n",
            "RF03127,L13-Bacteroidia,1801,L13-Bacteroidia ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,38.0,38.4,36.7,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,370,52,0,0,Cis-reg; leader;,Published; PMID:32448158;,109,0,,\"\",0.69062,-3.68595,1600000,302727,143,72,true,-4.0717,0.71883,2020-07-09T11:54:15.000Z,2020-09-14T15:04:31.000Z\n",
            "RF03130,L19-Flavobacteria,1802,L19-Flavobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,31.5,31.5,31.4,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,460,224,0,0,Cis-reg; leader;,Published; PMID:32448158;,330,0,,\"\",0.6186,-4.70764,1600000,430511,90,30,true,-3.4689,0.72919,2020-07-10T12:38:24.000Z,2020-09-14T15:04:31.000Z\n",
            "RF03137,S15-Methanomicrobia,1406,S15-Methanomicrobia ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,42.2,42.2,31.7,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,100,14,0,0,Cis-reg; leader;,Published; PMID:32448158;,21,0,,\"\",0.52787,-7.2157,1600000,251468,244,76,true,-3.6385,0.71898,2020-07-10T16:31:17.000Z,2020-09-14T15:04:01.000Z\n",
            "RF03143,S15-Halobacteria,1406,S15-Halobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,49.0,49.8,37.0,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,237,63,0,0,Cis-reg; leader;,Published; PMID:32448158;,126,0,,\"\",0.61824,-4.61215,1600000,222445,336,67,true,-3.7533,0.71907,2020-07-16T09:19:59.000Z,2020-09-14T15:04:01.000Z\n",
            "RF03145,RAGATH-14,2449,RAGATH-14 RNA,Weinberg Z,Published; PMID:26167874;,149.1,149.1,33.4,RNA motif Associated with Genes Associated with Twister and Hammerhead ribozymes (RAGATH).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,0,0,0,Gene; sRNA;,Published; PMID:26167874;,1,0,,\"\",0.49009,-10.58343,1600000,422549,729,189,true,-3.3407,0.71333,2020-07-16T14:45:51.000Z,2020-09-14T15:06:21.000Z\n",
            "RF03146,RAGATH-9,2449,RAGATH-9 RNA,Weinberg Z,Published; PMID:26167874;,75.2,75.2,36.7,RNA motif Associated with Genes Associated with Twister and Hammerhead ribozymes (RAGATH).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,53,0,0,0,Gene; sRNA;,Published; PMID:26167874;,1,0,,\"\",0.50355,-9.35546,1600000,396618,280,173,true,-2.9686,0.71519,2020-07-20T11:14:21.000Z,2020-09-14T15:06:21.000Z\n",
            "RF03147,RAGATH-10,2449,RAGATH-10 RNA,Weinberg Z,Published; PMID:26167874;,41.2,41.2,35.1,RNA motif Associated with Genes Associated with Twister and Hammerhead ribozymes (RAGATH).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,149,1,0,0,Gene; sRNA;,Published; PMID:26167874:,2,0,,\"\",0.55737,-6.37532,1600000,259108,166,82,true,-3.0508,0.71885,2020-07-20T11:20:01.000Z,2020-09-14T15:06:22.000Z\n",
            "RF03148,RAGATH-11,2449,RAGATH-11 RNA,Weinberg Z,Published; PMID:26167874;,79.4,79.4,44.9,RNA motif Associated with Genes Associated with Twister and Hammerhead ribozymes (RAGATH).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,0,0,0,Gene; sRNA;,Published; PMID:26167874;,1,0,,\"\",0.50675,-9.05915,1600000,310864,214,117,true,-3.5806,0.71833,2020-07-20T11:24:28.000Z,2020-09-14T15:06:22.000Z\n",
            "RF03149,RAGATH-12,2449,RAGATH-12 RNA,Weinberg Z,Published; PMID:26167874;,75.0,75.0,40.0,RNA motif Associated with Genes Associated with Twister and Hammerhead ribozymes (RAGATH).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,24,0,0,0,Gene; sRNA;,Published; PMID:26167874;,1,0,,\"\",0.5833,-7.3022,1600000,385492,167,78,true,-3.2712,0.71869,2020-07-20T11:27:46.000Z,2020-09-14T15:06:22.000Z\n",
            "RF03150,RAGATH-15,2449,RAGATH-15 RNA,Weinberg Z,Published; PMID:26167874;,199.5,199.5,47.7,RNA motif Associated with Genes Associated with Twister and Hammerhead ribozymes (RAGATH).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,0,0,0,Gene; sRNA;,Published; PMID:26167874;,1,0,,\"\",0.68988,-7.72389,1600000,364248,429,297,true,-3.5963,0.70608,2020-07-20T11:30:12.000Z,2020-09-14T15:06:22.000Z\n",
            "RF03152,RAGATH-1-hammerhead,1309,RAGATH-1 hammerhead ribozyme,Weinberg Z,Published; PMID:26167874;,56.3,56.3,37.0,RNA motif Associated with Genes Associated with Twister and Hammerhead ribozymes (RAGATH) [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,0,0,0,Gene; ribozyme;,Published; PMID:26167874;,1,0,,\"\",0.50788,-7.81691,1600000,373652,435,78,true,-3.9229,0.71875,2020-07-20T14:01:18.000Z,2020-09-14T15:03:52.000Z\n",
            "RF03153,RAGATH-4,2449,RAGATH-4 RNA,Weinberg Z,Published; PMID:26167874;,104.4,104.4,103.9,RNA motif Associated with Genes Associated with Twister and Hammerhead ribozymes (RAGATH).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,0,0,0,Gene; sRNA;,Published; PMID:26167874;,1,0,,\"\",0.62414,-4.39933,1600000,232694,236,131,true,-2.839,0.71822,2020-07-20T14:04:03.000Z,2020-09-14T15:06:23.000Z\n",
            "RF03154,twister-P3,2448,type-P3 twister ribozyme,\"Weinberg Z, Ames TD\",Published; PMID:24240507,39.9,53.6,38.2,\"Twister ribozyme motif, present stem P3 variant (type P3) [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 37.00 -Z 742849.287494 CM SEQDB,9,0,0,0,Gene; ribozyme;,Published; PMID:24240507,1,0,,\"\",0.64299,-3.11264,1600000,240176,148,73,true,-3.0743,0.72779,2020-07-20T15:09:09.000Z,2020-09-14T15:06:17.000Z\n",
            "RF03159,RAGATH-7,2449,RAGATH-7 RNA,Weinberg Z,Published; PMID:26167874;,111.8,111.8,41.7,RNA motif Associated with Genes Associated with Twister and Hammerhead ribozymes (RAGATH).,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,44,0,0,0,Gene; sRNA;,Published; PMID:26167874;,1,0,,\"\",0.58282,-9.73933,1600000,533701,700,211,true,-3.2982,0.71122,2020-07-30T14:42:50.000Z,2020-09-14T15:06:23.000Z\n",
            "RF03160,twister-P1,2448,type-P1 twister ribozyme,\"Weinberg Z, Ames TD\",Published; PMID:24240507;,67.2,67.2,67.1,\"The twister ribozyme motif, present stem P3 variant (type P3) [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,1613,1108,0,0,Gene; ribozyme;,Published; PMID:24240507;,46,0,,\"\",0.67679,-3.44765,1600000,310004,270,67,true,-3.7133,0.72353,2020-07-31T12:23:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03161,S15-Flavobacteria,1406,S15-Flavobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,46.5,46.5,45.7,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,935,237,0,0,Cis-reg; leader;,Published; PMID:32448158;,377,0,,\"\",0.43656,-8.60782,1600000,460426,223,63,true,-3.379,0.71888,2020-08-11T11:06:35.000Z,2021-06-14T15:02:32.000Z\n",
            "RF03163,c4-a1b1,2102,\"a1, b1 targets of C4 antisense RNA\",Weinberg Z,Published; PMID:20230605;,46.7,46.7,46.6,\"c4-a1b1 is a c4 antisense RNA [1], it is use by phagues to inhibit the synthesis of antirepressor genes. It is propose that c4 antisense RNAs are responsible for heteroimmunity in phagues [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,77,380,0,0,Gene; antisense;,Published; PMID:20230605;,140,0,,\"\",0.69227,-6.66129,1600000,447110,107,53,true,-3.8589,0.72525,2020-08-11T16:51:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03164,NiCo-AnGGAG,2447,NiCo-AnGGAG riboswitch,Weinberg Z,Published; PMID:28265071,64.5,64.5,63.7,\"Nico riboswitches are selective metalloregulatory riboswitches that recognice Ni2+ and Co2+ metals [2]. NiCo-AnGGAG is a variant of Nico riboswitch [2], with a change in position 87, from G to A [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,7,0,0,Cis-reg;,Published; PMID:28265071,9,0,,\"\",0.63967,-4.06043,1600000,211860,181,89,true,-3.0374,0.71867,2020-08-12T16:37:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03165,2dG-II,1451,2dG-II purine riboswitch,Weinberg Z,Published; PMID:28265071,60.0,60.0,59.4,Purine riboswitch variant that recognize 2'-deoxyguanosine,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,30,59,0,0,Cis-reg;,Published; PMID:28265071,26,0,,\"\",0.73099,-2.38861,1600000,235970,141,71,true,-3.6879,0.71884,2020-08-13T10:42:03.000Z,2021-06-14T15:02:32.000Z\n",
            "RF03166,glycine-GGAnGA,1755,glycine-GGAnGA riboswitch,Weinberg Z,Published; PMID:28265071,110.0,110.3,109.3,glycine-GGAnGA riboswitch is a variant of glycine riboswitch that have in common a U69A change. This riboswitch is associate with genes that encode either saccharopine dehydrogenase or amino acid transporters and other genes with not predicted function.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,26,22,0,0,Cis-reg;,Published; PMID:28265071,25,0,,\"\",0.58478,-9.08996,1600000,356260,362,180,true,-2.9236,0.71447,2020-08-14T17:53:26.000Z,2020-09-14T15:04:27.000Z\n",
            "RF03167,c-di-GMP-I-GGC,2038,c-di-GMP-I-GGC riboswitch,Weinberg Z,Published; PMID:28265071,55.0,55.0,54.9,\"c-di-MP-I-GGC is a variant of c-di-GMPI riboswitch RF01051, that includes CORESEQ GGC\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,38,805,0,0,Cis-reg;,Published; PMID:28265071,385,0,,\"\",0.6359,-6.3841,1600000,282791,553,78,true,-3.8485,0.71872,2020-08-14T18:19:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03168,c-di-GMP-I-UAU,2038,c-di-GMP-I-UAU riboswitch,Weinberg Z,Published; PMID:28265071,65.9,65.9,65.8,\"c-di-GMP-I-GGC is a variant of c-di-GMP-I riboswitch RF01051, that includes CORESEQ UAU\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,17,0,0,Cis-reg;,Published; PMID:28265071,18,0,,\"\",0.61281,-5.5186,1600000,232033,161,81,true,-3.4888,0.72382,2020-08-14T18:26:52.000Z,2021-06-14T15:02:32.000Z\n",
            "RF03169,c-di-GMP-II-GAG,2181,c-di-GMP-II-GAG riboswitch,Weinberg Z,Published; PMID:28265071,40.0,40.0,39.8,\"c-di-GMP-II-GAG is a variant of c-di-GMP-II riboswitch RF01786, that includes CORESEQ GAG\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,472,169,0,0,Cis-reg;,Published; PMID:28265071,80,0,,\"\",0.61664,-3.82365,1600000,160621,409,88,true,-3.7694,0.71852,2020-08-14T18:57:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03170,c-di-GMP-II-GCG,2181,c-di-GMP-II-GCG riboswitch,Weinberg Z,Published; PMID:28265071,47.9,47.9,47.8,\"c-di-GMP-II-GCG is a variant of c-di-GMP-II riboswitch RF01786, that includes CORESEQ GCG\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,33,382,0,0,Cis-reg;,Published; PMID:28265071,49,0,,\"\",0.55075,-4.36965,1600000,169330,306,86,true,-3.5709,0.7187,2020-08-14T19:10:00.000Z,2020-09-14T15:05:03.000Z\n",
            "RF03171,mir-1012,1287,mir-1012 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,88.4,48.5,This family represents the microRNA (miRNA) precursor mir-1012 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; PFOLD,2,0,,\"\",0.6058,-4.42807,1600000,346889,145,59,true,-2.8137,0.7273,2020-08-17T13:45:46.000Z,2020-09-14T14:54:33.000Z\n",
            "RF03172,mir-6012,1287,mir-6012 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.7,51.2,This family represents the microRNA (miRNA) precursor mir-6012 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,21,0,0,Gene; miRNA;,Predicted; RNAalifold,21,0,,\"\",0.59097,-4.14937,1600000,264376,180,85,true,-2.874,0.72207,2020-08-31T15:25:10.000Z,2020-09-14T14:54:33.000Z\n",
            "RF03173,mir-4504,1287,mir-4504 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,104.0,104.9,78.1,This family represents the microRNA (miRNA) precursor mir-4504 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.63448,-3.08162,1600000,191836,191,92,true,-3.3263,0.71886,2020-08-31T16:35:21.000Z,2020-09-14T14:54:33.000Z\n",
            "RF03174,mir-4803,1287,mir-4803 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,91.1,73.6,This family represents the microRNA (miRNA) precursor mir-4803 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.59393,-4.14315,1600000,253227,165,74,true,-3.4192,0.72114,2020-09-01T19:25:23.000Z,2020-11-05T15:43:16.000Z\n",
            "RF03175,MIR3454,1287,MIR3454 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,142.0,142.4,130.6,This family represents the microRNA (miRNA) precursor MIR3454 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,19,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61823,-4.00605,1600000,175130,264,147,true,-3.0873,0.71831,2020-09-01T19:58:07.000Z,2020-09-14T14:54:33.000Z\n",
            "RF03176,MIR7729,1287,MIR7729 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,92.0,92.7,91.6,This family represents the microRNA (miRNA) precursor MIR7729 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,64,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.69403,-2.76589,1600000,264973,183,86,true,-3.4631,0.7188,2020-09-01T19:58:59.000Z,2020-11-05T15:43:16.000Z\n",
            "RF03177,mir-2067,1287,mir-2067 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,105.3,53.0,This family represents the microRNA (miRNA) precursor mir-2067 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64131,-2.88319,1600000,209256,163,72,true,-3.3983,0.72084,2020-09-01T19:59:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03178,mir-4515,1287,mir-4515 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.7,50.9,This family represents the microRNA (miRNA) precursor mir-4515 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.57126,-3.87491,1600000,229930,176,81,true,-3.0299,0.71902,2020-09-01T19:59:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03179,mir-1182,1287,mir-1182 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,100.0,81.0,This family represents the microRNA (miRNA) precursor mir-1182 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.60927,-3.23884,1600000,204418,197,96,true,-3.2792,0.71887,2020-09-01T20:00:28.000Z,2020-11-05T15:43:16.000Z\n",
            "RF03180,mir-3126,1287,mir-3126 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.5,58.4,This family represents the microRNA (miRNA) precursor mir-3126 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.63144,-2.37444,1600000,231452,165,74,true,-3.712,0.71954,2020-09-01T20:02:05.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03181,mir-2755,1287,mir-2755 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.1,47.7,This family represents the microRNA (miRNA) precursor mir-2755 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.70185,-2.57577,1600000,181829,163,74,true,-3.6155,0.719,2020-09-01T20:03:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03182,MIR6032,1287,MIR6032 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,102.0,102.9,60.4,This family represents the microRNA (miRNA) precursor MIR6032 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.64587,-2.29703,1600000,188674,202,100,true,-3.4076,0.71868,2020-09-01T20:03:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03183,MIR5516,1287,MIR5516 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.4,86.4,This family represents the microRNA (miRNA) precursor MIR5516 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,15,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.62766,-3.80576,1600000,227584,159,69,true,-3.4954,0.72164,2020-09-01T20:04:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03184,mir-1250,1287,mir-1250 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,119.3,74.3,This family represents the microRNA (miRNA) precursor mir-1250 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,7,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61294,-3.25886,1600000,163519,207,104,true,-3.404,0.71875,2020-09-01T20:05:02.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03185,mir-3193,1287,mir-3193 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,55.0,55.2,48.3,This family represents the microRNA (miRNA) precursor mir-3193 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.6281,-3.94153,1600000,318587,140,55,true,-3.1032,0.72816,2020-09-01T20:05:34.000Z,2020-09-14T14:54:34.000Z\n",
            "RF03186,mir-3689,1287,mir-3689 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.1,51.6,This family represents the microRNA (miRNA) precursor mir-3689 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,76,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.6391,-3.76979,1600000,287740,158,72,true,-3.5594,0.71912,2020-09-01T20:06:28.000Z,2020-09-14T14:54:34.000Z\n",
            "RF03187,MIR1512,1287,MIR1512 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.3,64.3,This family represents the microRNA (miRNA) precursor MIR1512 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,12,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.6268,-2.65312,1600000,162684,237,94,true,-3.2455,0.71881,2020-09-01T20:06:59.000Z,2020-11-05T15:43:17.000Z\n",
            "RF03188,mir-2843,1287,mir-2843 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.2,80.5,This family represents the microRNA (miRNA) precursor mir-2843 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,31,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65245,-3.43105,1600000,282215,169,77,true,-3.7088,0.71962,2020-09-01T20:07:20.000Z,2020-09-14T14:54:34.000Z\n",
            "RF03189,mir-1175,1287,mir-1175 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.2,49.8,This family represents the microRNA (miRNA) precursor mir-1175 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,78,0,0,Gene; miRNA;,Predicted; RNAalifold,77,0,,\"\",0.70114,-2.62851,1600000,173822,147,72,true,-3.6787,0.71904,2020-09-01T20:08:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03190,mir-4857,1287,mir-4857 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,103.6,43.1,This family represents the microRNA (miRNA) precursor mir-4857 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.68186,-2.63077,1600000,226234,171,78,true,-3.358,0.72025,2020-09-01T20:08:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03191,mir-1913,1287,mir-1913 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,105.0,105.1,102.7,This family represents the microRNA (miRNA) precursor mir-1913 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.63291,-3.41446,1600000,252096,173,80,true,-3.3929,0.71894,2020-09-01T20:09:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03192,mir-6794,1287,mir-6794 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.9,48.1,This family represents the microRNA (miRNA) precursor mir-6794 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.57391,-5.42372,1600000,288011,149,62,true,-2.942,0.72561,2020-09-01T20:09:31.000Z,2020-11-05T15:43:18.000Z\n",
            "RF03193,mir-2941,1287,mir-2941 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,66.1,56.5,This family represents the microRNA (miRNA) precursor mir-2941 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.69813,-2.91319,1600000,186242,168,82,true,-4.1098,0.71898,2020-09-01T20:10:13.000Z,2020-11-05T15:43:18.000Z\n",
            "RF03194,mir-1272,1287,mir-1272 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,137.4,77.1,This family represents the microRNA (miRNA) precursor mir-1272 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.56908,-5.32027,1600000,235225,241,128,true,-3.1261,0.71846,2020-09-01T20:10:39.000Z,2020-09-14T14:54:35.000Z\n",
            "RF03195,mir-3934,1287,mir-3934 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.3,81.3,This family represents the microRNA (miRNA) precursor mir-3934 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.60938,-3.30037,1600000,193263,191,92,true,-2.9107,0.71882,2020-09-01T20:10:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03196,mir-6790,1287,mir-6790 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.3,45.8,This family represents the microRNA (miRNA) precursor mir-6790 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.63107,-4.07157,1600000,318375,151,63,true,-2.9334,0.72515,2020-09-01T20:12:23.000Z,2020-11-05T15:43:18.000Z\n",
            "RF03197,mir-4887,1287,mir-4887 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,101.5,54.4,This family represents the microRNA (miRNA) precursor mir-4887 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58016,-3.79395,1600000,182152,182,87,true,-3.8896,0.72002,2020-09-01T20:12:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03198,mir-7385,1287,mir-7385 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.6,66.1,This family represents the microRNA (miRNA) precursor mir-7385 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,8,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66212,-3.98703,1600000,286777,141,62,true,-3.6734,0.7192,2020-09-01T20:13:20.000Z,2020-11-05T15:43:18.000Z\n",
            "RF03199,MIR5271,1287,MIR5271 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.9,88.3,This family represents the microRNA (miRNA) precursor MIR5271 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,18,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57288,-4.33136,1600000,216355,169,78,true,-3.2408,0.71892,2020-09-01T20:14:05.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03200,mir-BART13,1287,mir-BART13 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,97.9,47.9,This family represents the microRNA (miRNA) precursor mir-BART13 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61629,-3.37576,1600000,233077,177,83,true,-3.3434,0.72167,2020-09-01T20:16:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03201,mir-617,1287,mir-617 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,87.5,77.7,This family represents the microRNA (miRNA) precursor mir-617 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.63903,-3.25687,1600000,222949,176,82,true,-3.6975,0.71895,2020-09-01T20:16:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03202,mir-555,1287,mir-555 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,92.0,94.1,91.2,This family represents the microRNA (miRNA) precursor mir-555 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.72657,-1.82871,1600000,200919,187,89,true,-3.3672,0.71881,2020-09-01T20:18:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03203,mir-3070,1287,mir-3070 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,91.0,66.3,This family represents the microRNA (miRNA) precursor mir-3070 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.67521,-2.99521,1600000,195615,186,89,true,-3.3153,0.71878,2020-09-01T20:19:05.000Z,2020-11-05T15:43:18.000Z\n",
            "RF03204,mir-4510,1287,mir-4510 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.3,46.1,This family represents the microRNA (miRNA) precursor mir-4510 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.68162,-3.10367,1600000,221527,162,72,true,-3.587,0.71907,2020-09-01T20:19:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03205,mir-1898,1287,mir-1898 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.0,77.7,This family represents the microRNA (miRNA) precursor mir-1898 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,39,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.58879,-4.37141,1600000,305493,148,61,true,-2.8878,0.72542,2020-09-01T20:20:20.000Z,2020-09-14T14:54:35.000Z\n",
            "RF03206,mir-3102,1287,mir-3102 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,61.0,61.3,60.6,This family represents the microRNA (miRNA) precursor mir-3102 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.55313,-4.13009,1600000,177455,208,104,true,-3.0222,0.71877,2020-09-01T20:21:04.000Z,2020-09-14T14:54:36.000Z\n",
            "RF03207,mir-3960,1287,mir-3960 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.1,70.1,This family represents the microRNA (miRNA) precursor mir-3960 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,11,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.49488,-5.65931,1600000,302253,163,73,true,-2.8548,0.7222,2020-09-01T20:21:49.000Z,2020-09-14T14:54:36.000Z\n",
            "RF03208,mir-1781,1287,mir-1781 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.1,65.4,This family represents the microRNA (miRNA) precursor mir-1781 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,45,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.60438,-3.74659,1600000,208470,181,86,true,-3.7485,0.71891,2020-09-01T20:22:14.000Z,2020-09-14T14:54:36.000Z\n",
            "RF03209,MIR9657,1287,MIR9657 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.4,52.6,This family represents the microRNA (miRNA) precursor MIR9657 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,28,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.54581,-3.86503,1600000,220984,180,85,true,-3.442,0.71895,2020-09-01T20:22:38.000Z,2020-11-05T15:43:19.000Z\n",
            "RF03210,mir-7359,1287,mir-7359 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.5,50.8,This family represents the microRNA (miRNA) precursor mir-7359 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61015,-3.15216,1600000,220435,141,56,true,-3.3172,0.73021,2020-09-01T20:23:03.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03211,mir-666,1287,mir-666 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,109.1,36.8,This family represents the microRNA (miRNA) precursor mir-666 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.58308,-3.89846,1600000,203248,183,86,true,-3.0767,0.71891,2020-09-01T20:23:25.000Z,2020-09-14T14:54:36.000Z\n",
            "RF03212,mir-2788,1287,mir-2788 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.3,55.1,This family represents the microRNA (miRNA) precursor mir-2788 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,27,0,0,Gene; miRNA;,Predicted; RNAalifold,29,0,,\"\",0.69523,-1.69884,1600000,120739,188,92,true,-3.8112,0.71887,2020-09-01T20:23:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03213,mir-5885,1287,mir-5885 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.0,37.6,This family represents the microRNA (miRNA) precursor mir-5885 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,34,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.66329,-3.49701,1600000,196691,149,65,true,-3.8496,0.7193,2020-09-01T20:24:27.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03214,mir-769,1287,mir-769 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,57.0,57.5,56.8,This family represents the microRNA (miRNA) precursor mir-769 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,34,0,0,Gene; miRNA;,Predicted; RNAalifold,27,0,,\"\",0.61157,-3.56588,1600000,164583,173,81,true,-3.6178,0.71903,2020-09-01T20:25:14.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03215,mir-5360,1287,mir-5360 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.1,65.1,This family represents the microRNA (miRNA) precursor mir-5360 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,16,0,0,Gene; miRNA;,Predicted; RNAalifold,16,0,,\"\",0.6453,-2.86912,1600000,172722,174,81,true,-3.3134,0.71976,2020-09-01T20:25:56.000Z,2020-11-05T15:43:19.000Z\n",
            "RF03216,mir-BART10,1287,mir-BART10 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,99.5,54.9,This family represents the microRNA (miRNA) precursor mir-BART10 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.5405,-4.8963,1600000,286487,173,80,true,-3.3881,0.72131,2020-09-01T20:26:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03217,mir-3167,1287,mir-3167 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.9,57.4,This family represents the microRNA (miRNA) precursor mir-3167 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.57259,-4.68659,1600000,313241,145,59,true,-3.0557,0.72797,2020-09-01T20:27:23.000Z,2020-11-05T15:43:19.000Z\n",
            "RF03218,mir-3015,1287,mir-3015 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.0,58.9,This family represents the microRNA (miRNA) precursor mir-3015 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,79,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60252,-4.5413,1600000,294826,135,57,true,-2.7026,0.72263,2020-09-01T20:29:21.000Z,2020-11-05T15:43:19.000Z\n",
            "RF03219,mir-602,1287,mir-602 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,95.6,81.0,This family represents the microRNA (miRNA) precursor mir-602 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.57948,-3.95131,1600000,195033,180,85,true,-2.9398,0.71902,2020-09-01T20:30:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03220,mir-2582,1287,mir-2582 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.7,52.3,This family represents the microRNA (miRNA) precursor mir-2582 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6347,-2.70192,1600000,247475,173,80,true,-3.1698,0.7223,2020-09-01T20:30:40.000Z,2020-11-05T15:43:20.000Z\n",
            "RF03221,mir-6715,1287,mir-6715 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.0,81.5,This family represents the microRNA (miRNA) precursor mir-6715 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,11,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.59095,-4.71399,1600000,318379,143,57,true,-3.0428,0.73047,2020-09-01T20:31:11.000Z,2020-09-14T14:54:37.000Z\n",
            "RF03222,mir-4900,1287,mir-4900 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,61.6,54.7,This family represents the microRNA (miRNA) precursor mir-4900 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,16,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63486,-3.39752,1600000,175045,166,75,true,-3.2894,0.72673,2020-09-01T20:32:37.000Z,2020-11-05T15:43:20.000Z\n",
            "RF03223,mir-2209,1287,mir-2209 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,89.5,47.0,This family represents the microRNA (miRNA) precursor mir-2209 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65549,-3.16749,1600000,223671,151,63,true,-3.2082,0.72982,2020-09-01T20:33:03.000Z,2020-11-05T15:43:20.000Z\n",
            "RF03224,MIR9560,1287,MIR9560 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.5,65.0,This family represents the microRNA (miRNA) precursor MIR9560 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.6152,-3.61891,1600000,210077,196,96,true,-2.8751,0.71873,2020-09-01T20:33:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03225,mir-2763,1287,mir-2763 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,50.9,49.9,This family represents the microRNA (miRNA) precursor mir-2763 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.65295,-2.48578,1600000,189565,176,83,true,-3.5891,0.71891,2020-09-01T20:33:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03226,mir-8196,1287,mir-8196 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.5,48.4,This family represents the microRNA (miRNA) precursor mir-8196 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.55693,-4.18436,1600000,279690,153,65,true,-3.1938,0.72499,2020-09-01T20:35:14.000Z,2020-11-05T15:43:20.000Z\n",
            "RF03227,mir-2528,1287,mir-2528 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,102.0,102.4,92.3,This family represents the microRNA (miRNA) precursor mir-2528 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,115,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.644,-2.82311,1600000,220703,227,118,true,-3.4658,0.71857,2020-09-01T20:35:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03228,mir-1471,1287,mir-1471 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.0,77.2,This family represents the microRNA (miRNA) precursor mir-1471 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.63476,-3.35983,1600000,328314,143,57,true,-2.9312,0.72696,2020-09-01T20:36:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03229,mir-1297,1287,mir-1297 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.5,83.9,This family represents the microRNA (miRNA) precursor mir-1297 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,24,0,0,Gene; miRNA;,Predicted; RNAalifold,25,0,,\"\",0.64267,-2.57462,1600000,207047,169,77,true,-3.3294,0.71891,2020-09-01T20:37:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03230,mir-967,1287,mir-967 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,116.0,51.3,This family represents the microRNA (miRNA) precursor mir-967 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59738,-3.91001,1600000,211295,188,90,true,-3.3357,0.71883,2020-09-01T20:37:26.000Z,2020-09-14T14:54:38.000Z\n",
            "RF03231,mir-4523,1287,mir-4523 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.1,95.6,This family represents the microRNA (miRNA) precursor mir-4523 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.50071,-5.38205,1600000,241038,159,69,true,-3.0251,0.72067,2020-09-01T20:37:48.000Z,2020-09-14T14:54:38.000Z\n",
            "RF03232,mir-6526,1287,mir-6526 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,57.0,57.3,56.5,This family represents the microRNA (miRNA) precursor mir-6526 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,19,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61995,-3.73926,1600000,233653,172,79,true,-3.0141,0.719,2020-09-01T20:38:09.000Z,2020-09-14T14:54:38.000Z\n",
            "RF03233,hvt-mir-H16,1287,hvt-mir-H16 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,94.8,46.0,This family represents the microRNA (miRNA) precursor hvt-mir-H16 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64369,-3.50816,1600000,265838,152,64,true,-3.3745,0.72495,2020-09-01T20:38:59.000Z,2020-09-14T14:54:39.000Z\n",
            "RF03234,mir-2813,1287,mir-2813 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,92.8,34.9,This family represents the microRNA (miRNA) precursor mir-2813 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60351,-3.13204,1600000,242876,156,67,true,-3.5673,0.72226,2020-09-01T20:39:22.000Z,2020-09-14T14:54:39.000Z\n",
            "RF03235,mir-2543,1287,mir-2543 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,125.7,59.8,This family represents the microRNA (miRNA) precursor mir-2543 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65844,-3.10752,1600000,225112,224,116,true,-3.4946,0.71854,2020-09-01T20:39:47.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03236,mir-1206,1287,mir-1206 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.1,59.3,This family represents the microRNA (miRNA) precursor mir-1206 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.63346,-4.31958,1600000,370405,144,58,true,-3.3318,0.7267,2020-09-01T20:40:15.000Z,2020-09-14T14:54:39.000Z\n",
            "RF03237,MIR6425,1287,MIR6425 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,142.0,142.3,69.6,This family represents the microRNA (miRNA) precursor MIR6425 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,11,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65731,-2.74583,1600000,157028,237,125,true,-3.349,0.71854,2020-09-01T20:40:36.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03238,mir-1542,1287,mir-1542 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,100.3,43.8,This family represents the microRNA (miRNA) precursor mir-1542 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64949,-2.94659,1600000,211148,163,72,true,-3.3282,0.72353,2020-09-01T20:40:57.000Z,2020-11-05T15:43:21.000Z\n",
            "RF03239,mir-963,1287,mir-963 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,109.4,75.6,This family represents the microRNA (miRNA) precursor mir-963 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61669,-3.07934,1600000,224578,176,82,true,-3.7764,0.71893,2020-09-01T20:41:42.000Z,2020-09-14T14:54:40.000Z\n",
            "RF03240,MIR5534,1287,MIR5534 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.4,78.9,This family represents the microRNA (miRNA) precursor MIR5534 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,64,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.58543,-4.42952,1600000,209541,204,102,true,-3.4989,0.71874,2020-09-01T20:42:02.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03241,mir-1789,1287,mir-1789 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.3,87.8,This family represents the microRNA (miRNA) precursor mir-1789 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,35,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.66107,-3.82261,1600000,329003,151,63,true,-3.2434,0.72529,2020-09-01T20:42:22.000Z,2020-09-14T14:54:40.000Z\n",
            "RF03242,mir-1397,1287,mir-1397 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,102.0,51.9,This family represents the microRNA (miRNA) precursor mir-1397 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,55,0,0,Gene; miRNA;,Predicted; RNAalifold,54,0,,\"\",0.65356,-2.81804,1600000,204060,179,84,true,-3.4019,0.71896,2020-09-01T20:42:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03243,mir-2731,1287,mir-2731 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,89.1,71.4,This family represents the microRNA (miRNA) precursor mir-2731 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66196,-2.46502,1600000,186003,190,91,true,-3.4824,0.71888,2020-09-01T20:43:38.000Z,2020-11-05T15:43:21.000Z\n",
            "RF03244,mir-4798,1287,mir-4798 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.3,73.3,This family represents the microRNA (miRNA) precursor mir-4798 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.64646,-2.81528,1600000,233400,167,75,true,-3.7768,0.72076,2020-09-01T20:43:58.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03246,MIR1515,1287,MIR1515 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,140.0,161.0,88.6,This family represents the microRNA (miRNA) precursor MIR1515 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60628,-4.94801,1600000,234947,252,137,true,-3.1476,0.7183,2020-09-01T20:44:49.000Z,2020-11-05T15:43:21.000Z\n",
            "RF03247,mir-1324,1287,mir-1324 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,77.8,61.6,This family represents the microRNA (miRNA) precursor mir-1324 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,59,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61334,-3.80831,1600000,220764,188,91,true,-3.2599,0.71874,2020-09-01T20:45:39.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03248,MIR9772,1287,MIR9772 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.3,61.6,This family represents the microRNA (miRNA) precursor MIR9772 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,19,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.56826,-4.26773,1600000,195474,190,92,true,-3.2383,0.71878,2020-09-01T20:47:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03249,mir-2114,1287,mir-2114 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.4,93.0,This family represents the microRNA (miRNA) precursor mir-2114 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,12,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.63695,-3.2031,1600000,197119,178,84,true,-3.4146,0.71889,2020-09-01T20:47:59.000Z,2020-09-14T14:54:41.000Z\n",
            "RF03251,mir-4446,1287,mir-4446 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.4,65.4,This family represents the microRNA (miRNA) precursor mir-4446 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,9,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.63921,-3.29586,1600000,200632,176,82,true,-3.2032,0.71891,2020-09-01T20:49:01.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03252,mir-678,1287,mir-678 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,106.0,64.9,This family represents the microRNA (miRNA) precursor mir-678 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.56484,-4.37684,1600000,256564,171,78,true,-3.2192,0.71958,2020-09-01T20:49:35.000Z,2020-09-14T14:54:42.000Z\n",
            "RF03253,mir-4654,1287,mir-4654 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.9,47.2,This family represents the microRNA (miRNA) precursor mir-4654 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.63406,-3.11116,1600000,238933,168,76,true,-3.8337,0.71907,2020-09-01T20:50:00.000Z,2020-11-05T15:43:22.000Z\n",
            "RF03254,mir-9460,1287,mir-9460 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.2,79.7,This family represents the microRNA (miRNA) precursor mir-9460 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,53,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67527,-3.20383,1600000,239233,156,68,true,-3.4139,0.71906,2020-09-01T20:50:25.000Z,2020-09-14T14:54:42.000Z\n",
            "RF03255,mir-2188,1287,mir-2188 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,62.6,56.9,This family represents the microRNA (miRNA) precursor mir-2188 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,75,0,0,Gene; miRNA;,Predicted; RNAalifold,73,0,,\"\",0.62719,-3.67894,1600000,209752,165,78,true,-3.5736,0.719,2020-09-01T20:51:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03256,mir-1286,1287,mir-1286 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,80.6,50.5,This family represents the microRNA (miRNA) precursor mir-1286 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.64533,-3.50617,1600000,209437,168,76,true,-3.906,0.71895,2020-09-01T20:52:48.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03257,mir-4427,1287,mir-4427 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.5,63.7,This family represents the microRNA (miRNA) precursor mir-4427 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.66571,-3.18174,1600000,273780,157,68,true,-3.5181,0.72243,2020-09-01T20:53:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03258,mir-4,1287,mir-4 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,104.1,62.8,This family represents the microRNA (miRNA) precursor mir-4 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,12,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.61147,-3.4431,1600000,210323,175,81,true,-3.4631,0.71886,2020-09-01T20:53:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03259,MIR1514,1287,MIR1514 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,118.8,75.9,This family represents the microRNA (miRNA) precursor MIR1514 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58763,-2.97146,1600000,170011,213,109,true,-3.2039,0.71837,2020-09-01T20:54:32.000Z,2020-11-05T15:43:22.000Z\n",
            "RF03260,mir-8229,1287,mir-8229 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,45.0,45.1,44.8,This family represents the microRNA (miRNA) precursor mir-8229 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62742,-4.68112,1600000,390644,145,59,true,-3.067,0.73055,2020-09-01T20:54:54.000Z,2020-11-05T15:43:22.000Z\n",
            "RF03261,mir-1759,1287,mir-1759 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.3,52.9,This family represents the microRNA (miRNA) precursor mir-1759 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,43,0,0,Gene; miRNA;,Predicted; RNAalifold,43,0,,\"\",0.60716,-3.54808,1600000,227925,166,75,true,-3.524,0.72191,2020-09-01T20:55:19.000Z,2020-11-05T15:43:22.000Z\n",
            "RF03262,mir-1181,1287,mir-1181 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,97.2,91.5,This family represents the microRNA (miRNA) precursor mir-1181 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.58091,-4.11704,1600000,208133,175,81,true,-3.0946,0.71905,2020-09-01T20:55:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03263,MIR861,1287,MIR861 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,114.2,60.0,This family represents the microRNA (miRNA) precursor MIR861 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.58821,-3.87261,1600000,205973,215,110,true,-3.1585,0.7185,2020-09-01T20:56:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03264,MIR1024,1287,MIR1024 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,112.5,41.5,This family represents the microRNA (miRNA) precursor MIR1024 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59199,-4.13073,1600000,185909,212,109,true,-2.9257,0.71863,2020-09-01T20:56:41.000Z,2020-09-14T14:54:42.000Z\n",
            "RF03265,mir-1460,1287,mir-1460 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,79.9,73.7,This family represents the microRNA (miRNA) precursor mir-1460 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,44,0,0,Gene; miRNA;,Predicted; RNAalifold,44,0,,\"\",0.53547,-4.90207,1600000,230823,190,91,true,-3.2356,0.71881,2020-09-01T20:57:11.000Z,2020-09-14T14:54:43.000Z\n",
            "RF03266,mir-2943,1287,mir-2943 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.8,47.6,This family represents the microRNA (miRNA) precursor mir-2943 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,26,0,0,Gene; miRNA;,Predicted; RNAalifold,21,0,,\"\",0.62566,-3.18573,1600000,268224,161,71,true,-2.8128,0.72519,2020-09-01T20:57:32.000Z,2020-09-14T14:54:43.000Z\n",
            "RF03267,mir-645,1287,mir-645 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,100.5,83.5,This family represents the microRNA (miRNA) precursor mir-645 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.6518,-2.91493,1600000,203906,169,77,true,-3.5617,0.71912,2020-09-01T20:58:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03268,mir-1911,1287,mir-1911 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.0,87.5,This family represents the microRNA (miRNA) precursor mir-1911 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.64014,-4.06439,1600000,282288,145,59,true,-3.0221,0.72689,2020-09-01T20:59:34.000Z,2020-09-14T14:54:43.000Z\n",
            "RF03269,mir-5697,1287,mir-5697 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,50.3,47.4,This family represents the microRNA (miRNA) precursor mir-5697 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.63263,-3.7794,1600000,310128,145,59,true,-3.1977,0.72871,2020-09-01T21:00:21.000Z,2020-11-05T15:43:23.000Z\n",
            "RF03270,MIR3712,1287,MIR3712 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.7,75.1,This family represents the microRNA (miRNA) precursor MIR3712 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.64227,-3.46598,1600000,316663,153,65,true,-3.0524,0.72595,2020-09-01T21:00:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03271,mir-1247,1287,mir-1247 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.6,49.8,This family represents the microRNA (miRNA) precursor mir-1247 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,24,0,0,Gene; miRNA;,Predicted; RNAalifold,25,0,,\"\",0.54569,-4.47636,1600000,209953,164,74,true,-3.5549,0.71914,2020-09-01T21:01:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03272,mir-2187,1287,mir-2187 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.0,41.1,This family represents the microRNA (miRNA) precursor mir-2187 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,23,0,0,Gene; miRNA;,Predicted; RNAalifold,14,0,,\"\",0.58395,-3.74094,1600000,267635,149,62,true,-3.6259,0.72542,2020-09-01T21:03:12.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03274,mir-1238,1287,mir-1238 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.5,38.7,This family represents the microRNA (miRNA) precursor mir-1238 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.60017,-3.8454,1600000,232782,177,83,true,-3.1603,0.71899,2020-09-01T21:04:26.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03275,mir-2542,1287,mir-2542 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,74.6,49.9,This family represents the microRNA (miRNA) precursor mir-2542 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65556,-2.46994,1600000,214689,181,85,true,-3.4616,0.71896,2020-09-01T21:04:51.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03276,mir-2232,1287,mir-2232 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,91.8,40.3,This family represents the microRNA (miRNA) precursor mir-2232 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63045,-3.40477,1600000,193298,186,90,true,-3.0271,0.72293,2020-09-01T21:05:36.000Z,2020-11-05T15:43:23.000Z\n",
            "RF03277,mir-5886,1287,mir-5886 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.0,84.6,This family represents the microRNA (miRNA) precursor mir-5886 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,39,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63369,-2.2884,1600000,197975,186,89,true,-3.5673,0.71885,2020-09-01T21:05:55.000Z,2020-11-05T15:43:23.000Z\n",
            "RF03278,mir-980,1287,mir-980 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,47.0,47.0,46.8,This family represents the microRNA (miRNA) precursor mir-980 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,73,0,0,Gene; miRNA;,Predicted; RNAalifold,64,0,,\"\",0.6871,-2.63853,1600000,193414,165,80,true,-3.7017,0.71889,2020-09-01T21:06:26.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03279,mir-1193,1287,mir-1193 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.7,47.9,This family represents the microRNA (miRNA) precursor mir-1193 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,35,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.56333,-4.21016,1600000,241891,174,81,true,-3.8304,0.71903,2020-09-01T21:07:14.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03280,mir-7583,1287,mir-7583 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,48.0,48.6,39.3,This family represents the microRNA (miRNA) precursor mir-7583 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,15,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61375,-4.07812,1600000,300581,138,58,true,-3.4273,0.7194,2020-09-01T21:07:49.000Z,2020-09-14T14:54:44.000Z\n",
            "RF03281,mir-596,1287,mir-596 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,96.0,72.6,This family represents the microRNA (miRNA) precursor mir-596 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.53333,-4.53846,1600000,256818,163,73,true,-3.249,0.71909,2020-09-01T21:08:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03283,mir-1010,1287,mir-1010 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,78.9,40.9,This family represents the microRNA (miRNA) precursor mir-1010 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,12,0,0,Gene; miRNA;,Predicted; RNAalifold,13,0,,\"\",0.68121,-2.54231,1600000,214573,191,93,true,-3.5545,0.71873,2020-09-01T21:09:26.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03284,mir-4637,1287,mir-4637 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.8,60.1,This family represents the microRNA (miRNA) precursor mir-4637 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.60779,-4.15544,1600000,281403,178,84,true,-3.4024,0.71885,2020-09-01T21:09:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03285,mir-564,1287,mir-564 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,87.0,50.8,This family represents the microRNA (miRNA) precursor mir-564 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.5231,-4.48092,1600000,195077,177,82,true,-3.1391,0.71893,2020-09-01T21:10:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03286,MIR6146,1287,MIR6146 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.7,53.1,This family represents the microRNA (miRNA) precursor MIR6146 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.66409,-2.95386,1600000,227599,167,75,true,-3.4566,0.72061,2020-09-01T21:10:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03287,mir-5904,1287,mir-5904 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.1,72.6,This family represents the microRNA (miRNA) precursor mir-5904 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,41,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65583,-2.79099,1600000,223556,178,84,true,-3.4814,0.71898,2020-09-01T21:11:07.000Z,2020-09-14T14:54:44.000Z\n",
            "RF03288,mir-3085,1287,mir-3085 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.6,80.6,42.8,This family represents the microRNA (miRNA) precursor mir-3085 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,35,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.51906,-4.7914,1600000,182647,187,89,true,-3.3277,0.71896,2020-09-01T21:11:48.000Z,2020-11-05T15:43:24.000Z\n",
            "RF03289,mir-1205,1287,mir-1205 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,85.8,71.2,This family represents the microRNA (miRNA) precursor mir-1205 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.67273,-3.49318,1600000,257952,149,63,true,-3.4313,0.71924,2020-09-01T21:12:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03290,mir-9428,1287,mir-9428 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.6,80.6,This family represents the microRNA (miRNA) precursor mir-9428 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,51,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63509,-4.59809,1600000,313215,154,68,true,-3.5585,0.71906,2020-09-01T21:13:13.000Z,2020-11-05T15:43:24.000Z\n",
            "RF03291,mir-591,1287,mir-591 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,85.5,42.6,This family represents the microRNA (miRNA) precursor mir-591 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.6332,-3.27846,1600000,213022,167,75,true,-3.2859,0.71906,2020-09-01T21:13:45.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03292,mir-1301,1287,mir-1301 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.0,56.1,This family represents the microRNA (miRNA) precursor mir-1301 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,25,0,0,Gene; miRNA;,Predicted; RNAalifold,27,0,,\"\",0.5732,-4.0582,1600000,226247,173,80,true,-3.6012,0.71912,2020-09-01T21:14:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03293,mir-4672,1287,mir-4672 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.4,78.9,This family represents the microRNA (miRNA) precursor mir-4672 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.633,-2.95354,1600000,226643,175,81,true,-3.4908,0.71906,2020-09-01T21:15:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03294,mir-H3,1287,mir-H3 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,94.4,54.8,This family represents the microRNA (miRNA) precursor mir-H3 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.48671,-5.60013,1600000,217932,175,81,true,-3.0584,0.72345,2020-09-01T21:16:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03295,mir-7594,1287,mir-7594 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,88.2,39.2,This family represents the microRNA (miRNA) precursor mir-7594 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6417,-3.7998,1600000,275421,151,63,true,-3.0294,0.72914,2020-09-01T21:17:10.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03296,MIR8788,1287,MIR8788 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,124.0,124.8,101.3,This family represents the microRNA (miRNA) precursor MIR8788 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.54107,-4.66106,1600000,180531,251,138,true,-3.0451,0.71834,2020-09-01T21:17:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03297,MIR1033,1287,MIR1033 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,90.9,64.6,This family represents the microRNA (miRNA) precursor MIR1033 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,5,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60425,-4.25511,1600000,193042,174,82,true,-3.8038,0.71904,2020-09-01T21:18:25.000Z,2020-09-14T14:54:45.000Z\n",
            "RF03298,mir-4488,1287,mir-4488 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.9,53.6,This family represents the microRNA (miRNA) precursor mir-4488 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.47396,-6.87254,1600000,337090,149,62,true,-2.8626,0.72417,2020-09-01T21:18:58.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03299,mir-1000,1287,mir-1000 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,52.9,48.2,This family represents the microRNA (miRNA) precursor mir-1000 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,76,0,0,Gene; miRNA;,Predicted; RNAalifold,76,0,,\"\",0.70492,-2.05357,1600000,170804,170,72,true,-3.7313,0.71905,2020-09-01T21:19:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03300,mir-3617,1287,mir-3617 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,107.0,107.3,97.0,This family represents the microRNA (miRNA) precursor mir-3617 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.63261,-3.37543,1600000,231803,172,79,true,-3.5264,0.71887,2020-09-01T21:20:48.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03301,mir-3154,1287,mir-3154 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.8,62.9,This family represents the microRNA (miRNA) precursor mir-3154 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,31,0,0,Gene; miRNA;,Predicted; RNAalifold,32,0,,\"\",0.6174,-3.08793,1600000,199697,181,86,true,-3.5765,0.71897,2020-09-01T21:22:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03302,mir-6133,1287,mir-6133 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.6,54.0,This family represents the microRNA (miRNA) precursor mir-6133 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.59586,-3.67284,1600000,207929,208,104,true,-3.5375,0.71868,2020-09-01T21:22:40.000Z,2020-09-14T14:54:45.000Z\n",
            "RF03303,mir-973,1287,mir-973 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.7,52.3,This family represents the microRNA (miRNA) precursor mir-973 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.61135,-3.04595,1600000,188950,178,84,true,-3.2402,0.72326,2020-09-01T21:23:17.000Z,2020-09-14T14:54:45.000Z\n",
            "RF03304,mir-1744,1287,mir-1744 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.1,71.1,This family represents the microRNA (miRNA) precursor mir-1744 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,28,0,0,Gene; miRNA;,Predicted; RNAalifold,28,0,,\"\",0.66384,-2.96017,1600000,192888,166,75,true,-3.3775,0.7227,2020-09-01T21:23:39.000Z,2020-11-05T15:43:25.000Z\n",
            "RF03305,mir-3338,1287,mir-3338 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.2,53.3,This family represents the microRNA (miRNA) precursor mir-3338 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.63058,-3.02883,1600000,194348,177,84,true,-3.1097,0.71892,2020-09-01T21:25:20.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03306,mir-3923,1287,mir-3923 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.8,88.3,This family represents the microRNA (miRNA) precursor mir-3923 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.58419,-4.54978,1600000,275869,178,83,true,-3.5656,0.71897,2020-09-01T21:25:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03307,mir-1825,1287,mir-1825 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,77.6,67.3,This family represents the microRNA (miRNA) precursor mir-1825 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.63981,-4.30203,1600000,381125,136,52,true,-2.8966,0.73065,2020-09-01T21:26:28.000Z,2020-09-14T14:54:46.000Z\n",
            "RF03308,MIR848,1287,MIR848 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,107.7,80.3,This family represents the microRNA (miRNA) precursor MIR848 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.68178,-1.85662,1600000,136682,197,97,true,-3.5969,0.71884,2020-09-01T21:26:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03309,mir-4791,1287,mir-4791 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.1,87.6,This family represents the microRNA (miRNA) precursor mir-4791 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61799,-2.76121,1600000,186823,180,84,true,-3.6649,0.71885,2020-09-01T21:27:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03310,mir-1544,1287,mir-1544 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,46.0,46.5,39.5,This family represents the microRNA (miRNA) precursor mir-1544 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,20,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61496,-5.2095,1600000,294868,140,59,true,-3.6194,0.71926,2020-09-01T21:27:47.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03311,MIR169,1287,MIR169 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,83.9,70.1,This family represents the microRNA (miRNA) precursor MIR169 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,20,0,0,Gene; miRNA;,Predicted; RNAalifold,17,0,,\"\",0.59991,-4.22391,1600000,183875,218,111,true,-3.1345,0.71856,2020-09-01T21:28:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03312,mir-880,1287,mir-880 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,105.4,73.5,This family represents the microRNA (miRNA) precursor mir-880 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.66141,-3.20894,1600000,235684,171,78,true,-3.3905,0.72054,2020-09-01T21:29:02.000Z,2020-09-14T14:54:46.000Z\n",
            "RF03313,mir-1468,1287,mir-1468 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,96.4,93.5,This family represents the microRNA (miRNA) precursor mir-1468 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,21,0,0,Gene; miRNA;,Predicted; RNAalifold,23,0,,\"\",0.60983,-3.78415,1600000,245993,181,86,true,-3.6023,0.71885,2020-09-01T21:29:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03314,mir-1908,1287,mir-1908 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.2,71.8,This family represents the microRNA (miRNA) precursor mir-1908 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.49393,-5.72162,1600000,260717,164,73,true,-2.8915,0.72043,2020-09-01T21:30:28.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03315,mir-723,1287,mir-723 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.0,62.8,This family represents the microRNA (miRNA) precursor mir-723 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,21,0,0,Gene; miRNA;,Predicted; RNAalifold,13,0,,\"\",0.60507,-4.13974,1600000,285202,142,60,true,-3.4468,0.72059,2020-09-01T21:30:52.000Z,2020-09-14T14:54:46.000Z\n",
            "RF03316,mir-1263,1287,mir-1263 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,101.5,88.0,This family represents the microRNA (miRNA) precursor mir-1263 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.58654,-4.26736,1600000,235549,179,84,true,-3.3261,0.71899,2020-09-01T21:31:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03317,mir-4874,1287,mir-4874 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,111.3,54.8,This family represents the microRNA (miRNA) precursor mir-4874 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.55464,-3.72385,1600000,146948,194,94,true,-3.2675,0.71888,2020-09-01T21:31:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03318,mir-2208,1287,mir-2208 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,91.0,44.2,This family represents the microRNA (miRNA) precursor mir-2208 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65415,-3.49869,1600000,287688,148,61,true,-3.3704,0.72885,2020-09-01T21:32:16.000Z,2020-09-14T14:54:46.000Z\n",
            "RF03319,mir-989,1287,mir-989 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.2,54.7,This family represents the microRNA (miRNA) precursor mir-989 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,49,0,0,Gene; miRNA;,Predicted; RNAalifold,49,0,,\"\",0.6801,-3.01819,1600000,167871,728,119,true,-3.2564,0.71866,2020-09-01T21:32:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03320,mir-1743,1287,mir-1743 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.3,77.2,This family represents the microRNA (miRNA) precursor mir-1743 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,52,0,0,Gene; miRNA;,Predicted; RNAalifold,52,0,,\"\",0.6173,-3.48915,1600000,230267,172,79,true,-3.1772,0.71892,2020-09-01T21:33:18.000Z,2020-09-14T14:54:46.000Z\n",
            "RF03321,mir-1006,1287,mir-1006 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.8,42.0,This family represents the microRNA (miRNA) precursor mir-1006 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,11,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.5992,-3.30301,1600000,183820,170,78,true,-3.3578,0.71898,2020-09-01T21:33:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03322,mir-1641,1287,mir-1641 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,104.0,104.7,98.9,This family represents the microRNA (miRNA) precursor mir-1641 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,49,0,0,Gene; miRNA;,Predicted; RNAalifold,48,0,,\"\",0.64847,-3.14486,1600000,211949,181,85,true,-3.1653,0.7189,2020-09-01T21:34:06.000Z,2020-11-05T15:43:27.000Z\n",
            "RF03323,mir-3613,1287,mir-3613 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,91.0,91.2,59.1,This family represents the microRNA (miRNA) precursor mir-3613 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,32,0,0,Gene; miRNA;,Predicted; RNAalifold,33,0,,\"\",0.64502,-3.37114,1600000,181146,182,86,true,-3.5708,0.71876,2020-09-01T21:34:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03324,mir-1288,1287,mir-1288 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,93.6,62.0,This family represents the microRNA (miRNA) precursor mir-1288 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.6494,-3.33886,1600000,245669,165,74,true,-3.5179,0.71982,2020-09-01T21:36:13.000Z,2020-11-05T15:43:27.000Z\n",
            "RF03325,mir-4872,1287,mir-4872 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.1,53.0,This family represents the microRNA (miRNA) precursor mir-4872 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,35,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64558,-3.51055,1600000,233459,164,76,true,-3.5681,0.71913,2020-09-01T21:36:42.000Z,2020-11-05T15:43:27.000Z\n",
            "RF03326,mir-312,1287,mir-312 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,55.0,60.5,46.5,This family represents the microRNA (miRNA) precursor mir-312 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,4,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.65139,-3.83813,1600000,251299,151,67,true,-3.5615,0.71916,2020-09-01T21:37:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03327,mir-1266,1287,mir-1266 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,86.0,58.8,This family represents the microRNA (miRNA) precursor mir-1266 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.58671,-4.34226,1600000,228018,177,82,true,-3.3288,0.71905,2020-09-01T21:38:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03328,mir-606,1287,mir-606 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,96.8,75.7,This family represents the microRNA (miRNA) precursor mir-606 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.65738,-2.76028,1600000,217171,179,84,true,-3.406,0.71898,2020-09-01T21:38:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03329,mir-4743,1287,mir-4743 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.0,65.7,This family represents the microRNA (miRNA) precursor mir-4743 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,11,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.66067,-2.50901,1600000,156038,203,101,true,-3.1055,0.71884,2020-09-01T21:39:37.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03330,mir-750,1287,mir-750 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,64.2,57.2,This family represents the microRNA (miRNA) precursor mir-750 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,6,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.67075,-3.37481,1600000,226192,170,83,true,-3.5242,0.71898,2020-09-01T21:40:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03331,mir-247,1287,mir-247 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,102.6,53.5,This family represents the microRNA (miRNA) precursor mir-247 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.70228,-2.27179,1600000,202810,185,89,true,-3.2346,0.72374,2020-09-01T21:41:05.000Z,2020-09-14T14:54:47.000Z\n",
            "RF03332,mir-2237,1287,mir-2237 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.7,50.4,This family represents the microRNA (miRNA) precursor mir-2237 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63131,-3.45721,1600000,232083,185,91,true,-2.9619,0.71885,2020-09-01T21:41:37.000Z,2020-09-14T14:54:47.000Z\n",
            "RF03333,mir-2820,1287,mir-2820 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.1,82.2,This family represents the microRNA (miRNA) precursor mir-2820 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,33,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64902,-3.54808,1600000,228899,178,83,true,-3.5744,0.7189,2020-09-01T21:42:32.000Z,2020-11-05T15:43:28.000Z\n",
            "RF03334,MIR391,1287,MIR391 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,86.5,64.3,This family represents the microRNA (miRNA) precursor MIR391 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.68675,-2.11889,1600000,211215,206,103,true,-3.3437,0.71871,2020-09-01T21:43:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03335,MIR2950,1287,MIR2950 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.6,71.9,This family represents the microRNA (miRNA) precursor MIR2950 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,16,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.64914,-3.51743,1600000,225401,211,108,true,-3.3529,0.71857,2020-09-01T21:43:43.000Z,2020-09-14T14:54:48.000Z\n",
            "RF03336,mir-1322,1287,mir-1322 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,87.5,64.2,This family represents the microRNA (miRNA) precursor mir-1322 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.63476,-3.40357,1600000,274492,157,69,true,-3.7733,0.719,2020-09-01T21:44:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03337,mir-6505,1287,mir-6505 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.9,65.1,This family represents the microRNA (miRNA) precursor mir-6505 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.65718,-3.32813,1600000,253431,157,68,true,-3.3067,0.72262,2020-09-01T21:47:02.000Z,2020-09-14T14:54:48.000Z\n",
            "RF03338,MIR4416,1287,MIR4416 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.7,57.9,This family represents the microRNA (miRNA) precursor MIR4416 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,12,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.60072,-6.14723,1600000,216528,320,147,true,-3.1214,0.71809,2020-09-01T21:47:45.000Z,2020-11-05T15:43:28.000Z\n",
            "RF03339,mir-3047,1287,mir-3047 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.5,49.5,This family represents the microRNA (miRNA) precursor mir-3047 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61673,-4.19847,1600000,303214,148,61,true,-3.1817,0.72687,2020-09-01T21:49:50.000Z,2020-09-14T14:54:48.000Z\n",
            "RF03340,mir-6511,1287,mir-6511 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.4,68.8,This family represents the microRNA (miRNA) precursor mir-6511 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,17,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.53929,-5.24365,1600000,264331,155,67,true,-3.387,0.71923,2020-09-01T21:50:45.000Z,2020-09-14T14:54:48.000Z\n",
            "RF03341,mir-943,1287,mir-943 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,104.4,85.9,This family represents the microRNA (miRNA) precursor mir-943 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.57854,-3.47959,1600000,204045,199,98,true,-3.3481,0.71891,2020-09-01T21:51:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03342,mir-1467,1287,mir-1467 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.3,65.5,This family represents the microRNA (miRNA) precursor mir-1467 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,47,0,0,Gene; miRNA;,Predicted; RNAalifold,47,0,,\"\",0.60314,-3.27469,1600000,161669,192,93,true,-3.3158,0.71884,2020-09-01T21:52:28.000Z,2020-09-14T14:54:48.000Z\n",
            "RF03343,mir-3347,1287,mir-3347 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,124.0,124.1,80.3,This family represents the microRNA (miRNA) precursor mir-3347 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67818,-2.47932,1600000,183302,210,106,true,-3.5025,0.7185,2020-09-01T21:53:06.000Z,2020-11-05T15:43:28.000Z\n",
            "RF03344,mir-5365,1287,mir-5365 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,103.5,58.6,This family represents the microRNA (miRNA) precursor mir-5365 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65758,-2.83894,1600000,232622,170,78,true,-3.4297,0.71915,2020-09-01T21:53:40.000Z,2020-11-05T15:43:28.000Z\n",
            "RF03345,MIR3444,1287,MIR3444 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,96.5,77.4,This family represents the microRNA (miRNA) precursor MIR3444 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60931,-3.77115,1600000,147495,203,103,true,-3.1701,0.71879,2020-09-01T21:54:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03346,mir-881,1287,mir-881 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,104.0,57.7,This family represents the microRNA (miRNA) precursor mir-881 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62045,-2.82918,1600000,209958,170,78,true,-3.3002,0.72045,2020-09-01T21:54:47.000Z,2020-11-05T15:43:29.000Z\n",
            "RF03347,mir-2816,1287,mir-2816 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,49.0,49.5,39.5,This family represents the microRNA (miRNA) precursor mir-2816 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,53,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67015,-2.52465,1600000,201284,165,74,true,-3.687,0.71897,2020-09-01T21:55:14.000Z,2020-11-05T15:43:29.000Z\n",
            "RF03348,mir-667,1287,mir-667 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,119.5,40.4,This family represents the microRNA (miRNA) precursor mir-667 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.59494,-3.43093,1600000,205153,191,92,true,-3.3482,0.71874,2020-09-01T21:55:41.000Z,2020-11-05T15:43:29.000Z\n",
            "RF03349,mir-3151,1287,mir-3151 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.4,69.0,This family represents the microRNA (miRNA) precursor mir-3151 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.52987,-5.20688,1600000,206499,180,85,true,-3.0462,0.71903,2020-09-01T21:56:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03350,mir-3851,1287,mir-3851 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,106.0,106.4,83.2,This family represents the microRNA (miRNA) precursor mir-3851 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,6,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62135,-1.37858,1600000,144341,192,94,true,-3.7836,0.71874,2020-09-01T21:57:20.000Z,2020-09-14T14:54:49.000Z\n",
            "RF03351,MIR6020,1287,MIR6020 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,118.0,118.5,71.9,This family represents the microRNA (miRNA) precursor MIR6020 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63799,-2.98384,1600000,217917,214,110,true,-3.2384,0.71858,2020-09-01T21:57:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03352,mir-4869,1287,mir-4869 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,106.4,44.9,This family represents the microRNA (miRNA) precursor mir-4869 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.66211,-2.91883,1600000,221309,171,78,true,-3.6742,0.71973,2020-09-01T21:58:13.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03353,MIR5291,1287,MIR5291 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.3,53.5,This family represents the microRNA (miRNA) precursor MIR5291 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,42,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58713,-3.34209,1600000,259003,151,65,true,-3.9893,0.71919,2020-09-01T21:58:44.000Z,2020-11-05T15:43:29.000Z\n",
            "RF03354,mir-459,1287,mir-459 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.6,42.1,This family represents the microRNA (miRNA) precursor mir-459 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,12,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.65394,-3.94149,1600000,176811,170,81,true,-3.6449,0.71898,2020-09-01T21:59:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03355,mir-2008,1287,mir-2008 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.0,63.2,This family represents the microRNA (miRNA) precursor mir-2008 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,1,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.64236,-3.13719,1600000,198382,178,85,true,-3.8199,0.71891,2020-09-01T21:59:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03356,mir-4782,1287,mir-4782 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,108.9,92.1,This family represents the microRNA (miRNA) precursor mir-4782 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.57988,-4.49495,1600000,254643,172,79,true,-3.841,0.71904,2020-09-01T22:00:26.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03357,mir-1842,1287,mir-1842 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,81.0,84.1,80.6,This family represents the microRNA (miRNA) precursor mir-1842 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,14,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.57477,-5.36951,1600000,297162,144,60,true,-3.7152,0.71931,2020-09-01T22:00:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03358,mir-634,1287,mir-634 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,105.6,39.2,This family represents the microRNA (miRNA) precursor mir-634 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.63248,-2.50491,1600000,155939,185,88,true,-3.2954,0.71884,2020-09-01T22:01:26.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03359,mir-1729,1287,mir-1729 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,92.1,79.8,This family represents the microRNA (miRNA) precursor mir-1729 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,48,0,0,Gene; miRNA;,Predicted; RNAalifold,47,0,,\"\",0.6634,-3.22803,1600000,268849,161,71,true,-3.3426,0.72304,2020-09-01T22:01:49.000Z,2020-09-14T14:54:49.000Z\n",
            "RF03360,mir-3156,1287,mir-3156 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.5,71.2,This family represents the microRNA (miRNA) precursor mir-3156 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,20,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.61746,-3.45434,1600000,198138,169,77,true,-3.6229,0.71912,2020-09-01T22:02:17.000Z,2020-09-14T14:54:49.000Z\n",
            "RF03361,mir-3136,1287,mir-3136 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.4,62.8,This family represents the microRNA (miRNA) precursor mir-3136 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.66114,-2.44801,1600000,225232,171,78,true,-3.7683,0.71912,2020-09-01T22:03:04.000Z,2020-09-14T14:54:49.000Z\n",
            "RF03362,mir-9436,1287,mir-9436 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.1,79.8,This family represents the microRNA (miRNA) precursor mir-9436 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,26,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65026,-2.99885,1600000,251009,155,66,true,-3.2341,0.72514,2020-09-01T22:03:41.000Z,2020-09-14T14:54:50.000Z\n",
            "RF03363,mir-6129,1287,mir-6129 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,114.0,114.0,96.5,This family represents the microRNA (miRNA) precursor mir-6129 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.64026,-2.74701,1600000,214773,191,92,true,-3.344,0.71875,2020-09-01T22:04:17.000Z,2020-11-05T15:43:30.000Z\n",
            "RF03364,mir-427,1287,mir-427 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,77.2,65.2,This family represents the microRNA (miRNA) precursor mir-427 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,64,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60664,-4.4887,1600000,301647,159,69,true,-3.6592,0.72159,2020-09-01T22:04:51.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03365,mir-1236,1287,mir-1236 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,115.0,120.7,113.3,This family represents the microRNA (miRNA) precursor mir-1236 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,10,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.57264,-4.58972,1600000,227404,205,102,true,-3.0922,0.71868,2020-09-01T22:05:20.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03366,mir-3122,1287,mir-3122 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,92.0,92.5,79.4,This family represents the microRNA (miRNA) precursor mir-3122 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.58107,-4.40711,1600000,280429,164,73,true,-3.348,0.72143,2020-09-01T22:06:51.000Z,2020-11-05T15:43:30.000Z\n",
            "RF03367,mir-975,1287,mir-975 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,109.0,70.2,This family represents the microRNA (miRNA) precursor mir-975 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61519,-2.61066,1600000,157417,189,91,true,-3.2596,0.71888,2020-09-01T22:08:38.000Z,2020-09-14T14:54:50.000Z\n",
            "RF03368,mir-1822,1287,mir-1822 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.0,56.8,This family represents the microRNA (miRNA) precursor mir-1822 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,6,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.65135,-3.90919,1600000,320847,141,61,true,-3.411,0.71931,2020-09-01T22:09:09.000Z,2020-11-05T15:43:30.000Z\n",
            "RF03369,mir-2217,1287,mir-2217 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,69.5,40.5,This family represents the microRNA (miRNA) precursor mir-2217 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,11,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62794,-4.35675,1600000,266722,146,62,true,-3.6727,0.71931,2020-09-01T22:09:44.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03370,mir-2566,1287,mir-2566 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.8,55.4,This family represents the microRNA (miRNA) precursor mir-2566 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,17,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.60887,-3.11957,1600000,232700,177,83,true,-3.681,0.71899,2020-09-01T22:10:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03371,mir-4788,1287,mir-4788 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,91.0,91.3,72.9,This family represents the microRNA (miRNA) precursor mir-4788 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.59016,-4.0254,1600000,239223,174,80,true,-3.7007,0.71908,2020-09-01T22:10:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03372,mir-5918,1287,mir-5918 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.3,57.2,This family represents the microRNA (miRNA) precursor mir-5918 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,11,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59133,-3.76701,1600000,196211,176,85,true,-3.4426,0.71903,2020-09-01T22:11:37.000Z,2020-09-14T14:54:50.000Z\n",
            "RF03373,mir-3170,1287,mir-3170 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.1,63.1,This family represents the microRNA (miRNA) precursor mir-3170 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.66252,-2.96681,1600000,260722,170,77,true,-3.4774,0.71904,2020-09-01T22:12:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03374,mir-2001,1287,mir-2001 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.6,54.4,This family represents the microRNA (miRNA) precursor mir-2001 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,2,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.67388,-3.20969,1600000,241710,155,72,true,-3.5968,0.71918,2020-09-01T22:12:35.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03375,MIR1094,1287,MIR1094 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,112.4,69.1,This family represents the microRNA (miRNA) precursor MIR1094 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63081,-3.44349,1600000,152193,207,104,true,-3.487,0.71867,2020-09-01T22:13:22.000Z,2020-09-14T14:54:51.000Z\n",
            "RF03376,mir-3075,1287,mir-3075 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,91.0,91.2,42.0,This family represents the microRNA (miRNA) precursor mir-3075 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.61616,-3.34152,1600000,204811,181,85,true,-3.3261,0.71892,2020-09-01T22:13:51.000Z,2020-11-05T15:43:31.000Z\n",
            "RF03377,mir-960,1287,mir-960 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,114.1,84.7,This family represents the microRNA (miRNA) precursor mir-960 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63715,-2.85645,1600000,209730,192,93,true,-3.7314,0.71874,2020-09-01T22:14:19.000Z,2020-11-05T15:43:31.000Z\n",
            "RF03378,mir-7398,1287,mir-7398 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,44.0,44.2,41.4,This family represents the microRNA (miRNA) precursor mir-7398 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,23,59,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60721,-4.81748,1600000,316863,129,60,true,-3.7681,0.71935,2020-09-01T22:15:22.000Z,2020-09-14T14:54:51.000Z\n",
            "RF03379,mir-4667,1287,mir-4667 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.0,66.4,This family represents the microRNA (miRNA) precursor mir-4667 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,15,0,0,Gene; miRNA;,Predicted; RNAalifold,17,0,,\"\",0.5601,-4.15752,1600000,161507,191,93,true,-3.3247,0.71883,2020-09-01T22:16:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03380,mir-9441,1287,mir-9441 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,91.0,91.2,56.4,This family represents the microRNA (miRNA) precursor mir-9441 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59963,-4.88458,1600000,258148,157,68,true,-3.2537,0.72273,2020-09-01T22:17:20.000Z,2020-11-05T15:43:31.000Z\n",
            "RF03381,mir-2796,1287,mir-2796 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.4,71.8,This family represents the microRNA (miRNA) precursor mir-2796 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,54,0,0,Gene; miRNA;,Predicted; RNAalifold,55,0,,\"\",0.66337,-2.88353,1600000,125315,164,77,true,-3.8176,0.71907,2020-09-01T22:18:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03382,mir-635,1287,mir-635 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,112.7,89.3,This family represents the microRNA (miRNA) precursor mir-635 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.67561,-2.48134,1600000,180084,191,92,true,-3.3761,0.71876,2020-09-01T22:19:02.000Z,2020-09-14T14:54:51.000Z\n",
            "RF03383,mir-1976,1287,mir-1976 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.4,53.1,This family represents the microRNA (miRNA) precursor mir-1976 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.62763,-4.54559,1600000,358585,136,52,true,-2.6983,0.73099,2020-09-01T22:19:37.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03384,mir-7180,1287,mir-7180 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.4,59.6,This family represents the microRNA (miRNA) precursor mir-7180 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,19,0,0,Gene; miRNA;,Predicted; RNAalifold,19,0,,\"\",0.66405,-3.813,1600000,313646,144,58,true,-2.9271,0.72853,2020-09-01T22:20:17.000Z,2020-09-14T14:54:51.000Z\n",
            "RF03385,mir-614,1287,mir-614 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.3,81.1,This family represents the microRNA (miRNA) precursor mir-614 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.54413,-4.67012,1600000,259508,174,80,true,-3.3978,0.71904,2020-09-01T22:20:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03386,mir-1994,1287,mir-1994 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,47.0,47.2,39.0,This family represents the microRNA (miRNA) precursor mir-1994 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.59666,-4.81,1600000,211446,183,66,true,-3.5718,0.72336,2020-09-01T22:21:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03387,mir-1902,1287,mir-1902 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,82.5,78.6,This family represents the microRNA (miRNA) precursor mir-1902 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,37,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.641,-3.10836,1600000,250861,172,79,true,-3.5473,0.7195,2020-09-01T22:22:02.000Z,2020-09-14T14:54:51.000Z\n",
            "RF03388,mir-9418,1287,mir-9418 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.2,37.8,This family represents the microRNA (miRNA) precursor mir-9418 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.55536,-5.28693,1600000,285243,148,61,true,-2.9992,0.7299,2020-09-01T22:22:27.000Z,2020-09-14T14:54:52.000Z\n",
            "RF03389,mir-4738,1287,mir-4738 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.0,66.0,This family represents the microRNA (miRNA) precursor mir-4738 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.6228,-3.16295,1600000,224626,184,87,true,-3.345,0.71902,2020-09-01T22:22:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03390,mir-2450,1287,mir-2450 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.1,43.9,This family represents the microRNA (miRNA) precursor mir-2450 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,13,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58712,-4.87969,1600000,268014,162,73,true,-3.3424,0.71915,2020-09-01T22:23:21.000Z,2020-09-14T14:54:52.000Z\n",
            "RF03391,mir-588,1287,mir-588 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,108.2,59.8,This family represents the microRNA (miRNA) precursor mir-588 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.61324,-3.48216,1600000,236366,175,81,true,-3.4873,0.71898,2020-09-01T22:23:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03392,mir-3943,1287,mir-3943 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.1,51.4,This family represents the microRNA (miRNA) precursor mir-3943 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,10,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.48889,-5.56693,1600000,241565,180,84,true,-2.9072,0.71896,2020-09-01T22:24:18.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03393,mir-2849,1287,mir-2849 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,117.3,96.4,This family represents the microRNA (miRNA) precursor mir-2849 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.69011,-2.29433,1600000,169181,198,97,true,-2.9972,0.71864,2020-09-01T22:24:50.000Z,2020-11-05T15:43:32.000Z\n",
            "RF03394,mir-2574,1287,mir-2574 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.0,86.9,This family represents the microRNA (miRNA) precursor mir-2574 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65704,-3.22932,1600000,278495,157,68,true,-3.2789,0.72547,2020-09-01T22:25:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03395,mir-718,1287,mir-718 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,55.0,55.0,43.5,This family represents the microRNA (miRNA) precursor mir-718 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,16,0,0,Gene; miRNA;,Predicted; RNAalifold,16,0,,\"\",0.50812,-5.058,1600000,269429,152,66,true,-3.2281,0.71937,2020-09-01T22:26:17.000Z,2020-09-14T14:54:52.000Z\n",
            "RF03396,mir-2814,1287,mir-2814 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.8,76.7,This family represents the microRNA (miRNA) precursor mir-2814 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,18,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64064,-2.66488,1600000,221323,182,87,true,-3.6967,0.71892,2020-09-01T22:26:43.000Z,2020-09-14T14:54:52.000Z\n",
            "RF03397,mir-1234,1287,mir-1234 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,74.0,58.6,This family represents the microRNA (miRNA) precursor mir-1234 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.52766,-5.87971,1600000,321685,143,57,true,-2.7177,0.7289,2020-09-01T22:27:16.000Z,2020-09-14T14:54:52.000Z\n",
            "RF03398,mir-3552,1287,mir-3552 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,120.0,124.8,117.8,This family represents the microRNA (miRNA) precursor mir-3552 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.66735,-2.29602,1600000,208751,209,105,true,-3.4938,0.71859,2020-09-01T22:28:44.000Z,2020-09-14T14:54:52.000Z\n",
            "RF03399,mir-986,1287,mir-986 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,94.9,63.9,This family represents the microRNA (miRNA) precursor mir-986 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.6983,-2.08458,1600000,194418,191,94,true,-3.1044,0.71877,2020-09-01T22:29:06.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03400,mir-728,1287,mir-728 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.8,69.1,This family represents the microRNA (miRNA) precursor mir-728 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,25,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.5822,-3.76658,1600000,240457,144,62,true,-3.6456,0.71926,2020-09-01T22:31:14.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03401,mir-1323,1287,mir-1323 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.1,78.0,This family represents the microRNA (miRNA) precursor mir-1323 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66197,-3.01847,1600000,197962,164,73,true,-3.545,0.71908,2020-09-01T22:31:52.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03402,mir-957,1287,mir-957 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.4,46.7,This family represents the microRNA (miRNA) precursor mir-957 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,43,0,0,Gene; miRNA;,Predicted; RNAalifold,43,0,,\"\",0.6383,-3.83299,1600000,172328,165,77,true,-3.8219,0.71898,2020-09-01T22:32:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03403,mir-5366,1287,mir-5366 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.6,67.1,This family represents the microRNA (miRNA) precursor mir-5366 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,15,0,0,Gene; miRNA;,Predicted; RNAalifold,14,0,,\"\",0.63786,-3.92461,1600000,216602,161,71,true,-3.2203,0.72509,2020-09-01T22:32:56.000Z,2020-11-05T15:43:33.000Z\n",
            "RF03404,mir-1298,1287,mir-1298 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,91.0,63.4,This family represents the microRNA (miRNA) precursor mir-1298 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,36,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.62907,-3.01019,1600000,165753,206,104,true,-3.1729,0.71864,2020-09-01T22:33:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03405,mir-3200,1287,mir-3200 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.3,71.0,This family represents the microRNA (miRNA) precursor mir-3200 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,15,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.56245,-3.66484,1600000,181244,191,92,true,-3.4057,0.71897,2020-09-01T22:34:33.000Z,2020-09-14T14:54:53.000Z\n",
            "RF03406,mir-3099,1287,mir-3099 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.9,67.5,This family represents the microRNA (miRNA) precursor mir-3099 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.6204,-3.7454,1600000,226077,179,84,true,-3.3964,0.71889,2020-09-01T22:35:35.000Z,2020-09-14T14:54:53.000Z\n",
            "RF03407,MIR531,1287,MIR531 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,73.8,68.6,This family represents the microRNA (miRNA) precursor MIR531 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,49,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.59377,-3.92664,1600000,244077,159,69,true,-3.3165,0.72143,2020-09-01T22:36:35.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03408,mir-5890,1287,mir-5890 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.1,53.9,This family represents the microRNA (miRNA) precursor mir-5890 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,123,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.66072,-3.44787,1600000,183509,169,78,true,-3.719,0.71899,2020-09-01T22:37:04.000Z,2020-09-14T14:54:53.000Z\n",
            "RF03409,mir-646,1287,mir-646 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,103.3,56.1,This family represents the microRNA (miRNA) precursor mir-646 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.58745,-3.84566,1600000,204728,182,86,true,-3.4762,0.71888,2020-09-01T22:38:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03410,mir-1891,1287,mir-1891 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,99.9,61.5,This family represents the microRNA (miRNA) precursor mir-1891 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,23,0,0,Gene; miRNA;,Predicted; RNAalifold,22,0,,\"\",0.65019,-2.90272,1600000,220966,183,87,true,-3.4363,0.71887,2020-09-01T22:38:39.000Z,2020-09-14T14:54:53.000Z\n",
            "RF03411,mir-2856,1287,mir-2856 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,67.9,59.4,This family represents the microRNA (miRNA) precursor mir-2856 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,26,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67364,-1.82597,1600000,187113,184,87,true,-3.6362,0.71884,2020-09-01T22:41:21.000Z,2020-09-14T14:54:53.000Z\n",
            "RF03412,mir-1915,1287,mir-1915 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.6,81.0,This family represents the microRNA (miRNA) precursor mir-1915 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.52132,-5.10371,1600000,261450,164,73,true,-3.153,0.71984,2020-09-01T22:41:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03413,MIR904,1287,MIR904 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,152.1,69.8,This family represents the microRNA (miRNA) precursor MIR904 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62377,-3.9243,1600000,203967,257,141,true,-3.0334,0.71836,2020-09-01T22:42:04.000Z,2020-11-05T15:43:33.000Z\n",
            "RF03414,mir-670,1287,mir-670 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.5,84.1,This family represents the microRNA (miRNA) precursor mir-670 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,35,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.59556,-3.68464,1600000,223318,187,90,true,-3.3704,0.71893,2020-09-01T22:43:28.000Z,2020-11-05T15:43:34.000Z\n",
            "RF03416,mir-7383,1287,mir-7383 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.8,38.9,This family represents the microRNA (miRNA) precursor mir-7383 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65263,-3.63867,1600000,279810,142,59,true,-3.8472,0.71926,2020-09-01T22:45:19.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03417,mir-343,1287,mir-343 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,94.5,47.0,This family represents the microRNA (miRNA) precursor mir-343 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.67088,-3.24655,1600000,234113,160,70,true,-3.038,0.72216,2020-09-01T22:45:40.000Z,2020-11-05T15:43:34.000Z\n",
            "RF03418,mir-3198,1287,mir-3198 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,91.0,91.5,67.6,This family represents the microRNA (miRNA) precursor mir-3198 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,17,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.59485,-4.1092,1600000,252090,174,80,true,-3.658,0.71899,2020-09-01T22:46:10.000Z,2020-09-14T14:54:54.000Z\n",
            "RF03419,mir-998,1287,mir-998 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,52.0,52.3,51.6,This family represents the microRNA (miRNA) precursor mir-998 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,46,0,0,Gene; miRNA;,Predicted; RNAalifold,48,0,,\"\",0.64729,-2.74616,1600000,134805,561,91,true,-3.9021,0.71877,2020-09-01T22:46:36.000Z,2022-10-29T17:42:07.000Z\n",
            "RF03420,mir-1200,1287,mir-1200 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.1,52.8,This family represents the microRNA (miRNA) precursor mir-1200 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.66419,-3.01553,1600000,235432,168,76,true,-3.2011,0.71905,2020-09-01T22:47:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03421,mir-2558,1287,mir-2558 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,92.8,40.6,This family represents the microRNA (miRNA) precursor mir-2558 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.60264,-4.23426,1600000,229441,179,84,true,-3.0621,0.71874,2020-09-01T22:48:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03422,mir-637,1287,mir-637 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,74.3,70.1,This family represents the microRNA (miRNA) precursor mir-637 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.62018,-3.33022,1600000,178546,188,90,true,-3.1147,0.71886,2020-09-01T22:48:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03423,mir-962,1287,mir-962 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,115.3,54.9,This family represents the microRNA (miRNA) precursor mir-962 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63267,-2.92465,1600000,170148,190,92,true,-3.643,0.71872,2020-09-01T22:48:43.000Z,2020-09-14T14:54:54.000Z\n",
            "RF03424,mir-4856,1287,mir-4856 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,108.0,50.1,This family represents the microRNA (miRNA) precursor mir-4856 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61891,-2.93096,1600000,184952,179,84,true,-3.302,0.71896,2020-09-01T22:49:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03425,mir-1292,1287,mir-1292 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.9,88.9,This family represents the microRNA (miRNA) precursor mir-1292 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.55146,-5.02029,1600000,312977,153,66,true,-3.5425,0.71933,2020-09-01T22:51:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03426,mir-1294,1287,mir-1294 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,114.4,75.4,This family represents the microRNA (miRNA) precursor mir-1294 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.69286,-2.10983,1600000,166901,227,118,true,-3.192,0.71866,2020-09-01T22:51:45.000Z,2020-11-05T15:43:34.000Z\n",
            "RF03427,mir-1005,1287,mir-1005 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,90.4,65.8,This family represents the microRNA (miRNA) precursor mir-1005 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64072,-3.8126,1600000,295319,148,61,true,-3.3444,0.72662,2020-09-01T22:52:09.000Z,2020-09-14T14:54:54.000Z\n",
            "RF03428,mir-6134,1287,mir-6134 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,109.0,109.9,97.5,This family represents the microRNA (miRNA) precursor mir-6134 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.63977,-3.03338,1600000,196380,183,87,true,-3.5507,0.71877,2020-09-01T22:52:32.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03429,mir-4451,1287,mir-4451 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.3,81.7,This family represents the microRNA (miRNA) precursor mir-4451 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.62038,-4.10242,1600000,247866,155,66,true,-3.5042,0.72325,2020-09-01T22:52:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03430,MIR5294,1287,MIR5294 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,115.0,115.1,114.9,This family represents the microRNA (miRNA) precursor MIR5294 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,31,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.685,-2.59877,1600000,201084,216,111,true,-3.4233,0.71842,2020-09-01T22:53:20.000Z,2020-09-14T14:54:54.000Z\n",
            "RF03431,mir-4657,1287,mir-4657 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.2,57.8,This family represents the microRNA (miRNA) precursor mir-4657 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,35,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.6699,-3.22901,1600000,354622,137,53,true,-3.096,0.72924,2020-09-01T22:53:47.000Z,2020-11-05T15:43:35.000Z\n",
            "RF03432,mir-7331,1287,mir-7331 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.2,55.4,This family represents the microRNA (miRNA) precursor mir-7331 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65547,-3.3186,1600000,241319,157,68,true,-3.4708,0.72404,2020-09-01T22:54:06.000Z,2020-11-05T15:43:35.000Z\n",
            "RF03433,mir-2576,1287,mir-2576 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.3,80.7,This family represents the microRNA (miRNA) precursor mir-2576 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,61,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.51001,-4.84265,1600000,196250,230,120,true,-2.8339,0.71857,2020-09-01T22:54:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03434,mir-1246,1287,mir-1246 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,98.1,61.1,This family represents the microRNA (miRNA) precursor mir-1246 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.65687,-2.63679,1600000,218157,163,73,true,-3.4283,0.71897,2020-09-01T22:54:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03435,mir-726,1287,mir-726 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.3,64.5,This family represents the microRNA (miRNA) precursor mir-726 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,16,0,0,Gene; miRNA;,Predicted; RNAalifold,16,0,,\"\",0.64249,-2.70881,1600000,166816,182,87,true,-3.5565,0.72269,2020-09-01T22:55:22.000Z,2020-09-14T14:54:55.000Z\n",
            "RF03436,mir-759,1287,mir-759 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.3,91.5,This family represents the microRNA (miRNA) precursor mir-759 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,38,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.60142,-2.82714,1600000,219490,186,89,true,-3.4392,0.71884,2020-09-01T22:56:00.000Z,2020-11-05T15:43:35.000Z\n",
            "RF03437,mir-2392,1287,mir-2392 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.3,42.3,This family represents the microRNA (miRNA) precursor mir-2392 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.59168,-3.65437,1600000,198786,180,84,true,-3.1116,0.71893,2020-09-01T22:56:41.000Z,2020-11-05T15:43:35.000Z\n",
            "RF03438,mir-1956,1287,mir-1956 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.6,81.3,This family represents the microRNA (miRNA) precursor mir-1956 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.5836,-4.55815,1600000,270979,153,65,true,-3.0492,0.72275,2020-09-01T22:57:04.000Z,2020-09-14T14:54:55.000Z\n",
            "RF03439,mir-1791,1287,mir-1791 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.3,53.9,This family represents the microRNA (miRNA) precursor mir-1791 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,50,0,0,Gene; miRNA;,Predicted; RNAalifold,50,0,,\"\",0.63687,-3.4405,1600000,219064,173,80,true,-3.4777,0.71907,2020-09-01T22:58:25.000Z,2020-11-05T15:43:35.000Z\n",
            "RF03440,MIR4414,1287,MIR4414 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.8,85.8,This family represents the microRNA (miRNA) precursor MIR4414 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,12,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.65047,-4.2608,1600000,222167,237,128,true,-3.2795,0.71836,2020-09-01T22:58:49.000Z,2020-09-14T14:54:55.000Z\n",
            "RF03441,mir-613,1287,mir-613 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,98.5,67.3,This family represents the microRNA (miRNA) precursor mir-613 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.67409,-2.45224,1600000,205937,184,87,true,-3.7008,0.71893,2020-09-01T23:00:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03442,mir-4003,1287,mir-4003 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,75.2,61.8,This family represents the microRNA (miRNA) precursor mir-4003 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,7,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58621,-4.56351,1600000,282174,139,58,true,-3.5536,0.71931,2020-09-01T23:01:12.000Z,2020-09-14T14:54:55.000Z\n",
            "RF03443,mir-2131,1287,mir-2131 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,92.0,92.4,72.8,This family represents the microRNA (miRNA) precursor mir-2131 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,43,0,0,Gene; miRNA;,Predicted; RNAalifold,42,0,,\"\",0.60356,-3.9296,1600000,223894,186,89,true,-3.5495,0.71885,2020-09-01T23:01:40.000Z,2020-09-14T14:54:55.000Z\n",
            "RF03444,MIR5635,1287,MIR5635 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,149.0,149.1,126.8,This family represents the microRNA (miRNA) precursor MIR5635 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,11,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.53891,-7.10992,1600000,241203,310,182,true,-3.3571,0.71429,2020-09-01T23:02:03.000Z,2020-11-05T15:43:36.000Z\n",
            "RF03445,mir-7552,1287,mir-7552 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.3,54.9,This family represents the microRNA (miRNA) precursor mir-7552 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,26,0,0,Gene; miRNA;,Predicted; RNAalifold,14,0,,\"\",0.53334,-4.84898,1600000,269791,146,63,true,-3.4467,0.71918,2020-09-01T23:02:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03446,mir-2954,1287,mir-2954 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.2,41.0,This family represents the microRNA (miRNA) precursor mir-2954 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,44,0,0,Gene; miRNA;,Predicted; RNAalifold,44,0,,\"\",0.60154,-4.51698,1600000,248824,151,63,true,-3.3534,0.72381,2020-09-01T23:03:03.000Z,2020-11-05T15:43:36.000Z\n",
            "RF03447,mir-1231,1287,mir-1231 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.0,68.3,This family represents the microRNA (miRNA) precursor mir-1231 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61679,-3.46838,1600000,204079,179,85,true,-3.1993,0.72117,2020-09-01T23:03:28.000Z,2020-09-14T14:54:56.000Z\n",
            "RF03448,mir-6131,1287,mir-6131 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.5,54.7,This family represents the microRNA (miRNA) precursor mir-6131 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.63084,-3.27446,1600000,214242,209,105,true,-3.3108,0.71866,2020-09-01T23:03:55.000Z,2020-09-14T14:54:56.000Z\n",
            "RF03449,mir-6827,1287,mir-6827 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.7,60.1,This family represents the microRNA (miRNA) precursor mir-6827 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.62584,-3.91357,1600000,282854,140,55,true,-3.3034,0.7304,2020-09-01T23:04:25.000Z,2020-09-14T14:54:56.000Z\n",
            "RF03450,MIR1513,1287,MIR1513 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,106.2,80.5,This family represents the microRNA (miRNA) precursor MIR1513 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61047,-3.54449,1600000,211197,189,91,true,-3.4182,0.71882,2020-09-01T23:05:29.000Z,2020-11-05T15:43:36.000Z\n",
            "RF03451,mir-3188,1287,mir-3188 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.4,51.6,This family represents the microRNA (miRNA) precursor mir-3188 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.53544,-4.36842,1600000,186056,182,86,true,-3.281,0.71899,2020-09-01T23:06:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03452,mir-3121,1287,mir-3121 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,108.0,108.4,72.5,This family represents the microRNA (miRNA) precursor mir-3121 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.73214,-1.3995,1600000,228232,169,77,true,-3.3285,0.71907,2020-09-01T23:07:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03453,mir-1276,1287,mir-1276 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,109.0,97.4,This family represents the microRNA (miRNA) precursor mir-1276 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.64886,-2.76678,1600000,211017,177,82,true,-3.5958,0.71893,2020-09-01T23:07:49.000Z,2020-11-05T15:43:36.000Z\n",
            "RF03454,mir-1267,1287,mir-1267 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,79.4,68.4,This family represents the microRNA (miRNA) precursor mir-1267 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.6338,-3.45017,1600000,224611,170,78,true,-3.3665,0.71906,2020-09-01T23:08:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03455,mir-9847,1287,mir-9847 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.9,53.3,This family represents the microRNA (miRNA) precursor mir-9847 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.69614,-2.90662,1600000,214193,167,75,true,-3.0981,0.71888,2020-09-01T23:08:49.000Z,2020-11-05T15:43:36.000Z\n",
            "RF03456,mir-724,1287,mir-724 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.7,41.9,This family represents the microRNA (miRNA) precursor mir-724 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,27,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.65786,-2.9932,1600000,324461,143,60,true,-3.7728,0.71929,2020-09-01T23:09:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03457,MIR173,1287,MIR173 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,87.4,64.2,This family represents the microRNA (miRNA) precursor MIR173 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.62702,-3.56382,1600000,208494,204,102,true,-3.2795,0.71869,2020-09-01T23:10:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03458,mir-766,1287,mir-766 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,110.0,125.4,106.4,This family represents the microRNA (miRNA) precursor mir-766 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.69527,-2.74802,1600000,205024,213,108,true,-3.1779,0.71852,2020-09-01T23:10:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03459,mir-1890,1287,mir-1890 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.8,75.3,This family represents the microRNA (miRNA) precursor mir-1890 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,16,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.61239,-3.62995,1600000,193274,169,78,true,-3.2125,0.71905,2020-09-01T23:11:07.000Z,2021-11-19T10:36:03.000Z\n",
            "RF03460,mir-561,1287,mir-561 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,104.9,78.2,This family represents the microRNA (miRNA) precursor mir-561 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.57903,-4.5294,1600000,239895,175,81,true,-3.0407,0.71895,2020-09-01T23:11:32.000Z,2020-09-14T14:54:57.000Z\n",
            "RF03461,MIR842,1287,MIR842 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,144.1,74.4,This family represents the microRNA (miRNA) precursor MIR842 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.60215,-4.27815,1600000,212911,239,127,true,-3.0285,0.71838,2020-09-01T23:12:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03462,mir-4677,1287,mir-4677 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.7,79.6,This family represents the microRNA (miRNA) precursor mir-4677 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,18,0,0,Gene; miRNA;,Predicted; RNAalifold,17,0,,\"\",0.65333,-3.31224,1600000,264606,145,59,true,-3.511,0.7271,2020-09-01T23:12:58.000Z,2020-09-14T14:54:57.000Z\n",
            "RF03463,mir-3036,1287,mir-3036 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,72.6,47.4,This family represents the microRNA (miRNA) precursor mir-3036 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58804,-3.41162,1600000,265607,141,58,true,-3.9338,0.71937,2020-09-01T23:13:18.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03464,mir-465,1287,mir-465 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,65.0,66.9,62.2,This family represents the microRNA (miRNA) precursor mir-465 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,10,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.63481,-3.43521,1600000,219015,170,79,true,-3.4131,0.71907,2020-09-01T23:13:47.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03465,MIR9666,1287,MIR9666 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.2,83.8,This family represents the microRNA (miRNA) precursor MIR9666 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,150,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65705,-2.61095,1600000,164706,199,98,true,-3.4453,0.71884,2020-09-01T23:14:14.000Z,2020-09-14T14:54:57.000Z\n",
            "RF03466,mir-1184,1287,mir-1184 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,77.8,63.3,This family represents the microRNA (miRNA) precursor mir-1184 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,57,0,0,Gene; miRNA;,Predicted; RNAalifold,32,0,,\"\",0.62344,-3.43324,1600000,170197,201,99,true,-3.181,0.71873,2020-09-01T23:15:15.000Z,2020-09-14T14:54:57.000Z\n",
            "RF03467,mir-2851,1287,mir-2851 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.2,58.7,This family represents the microRNA (miRNA) precursor mir-2851 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,15,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59713,-2.64159,1600000,179069,188,90,true,-3.8139,0.71881,2020-09-01T23:15:36.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03468,mir-2830,1287,mir-2830 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,99.0,61.5,This family represents the microRNA (miRNA) precursor mir-2830 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62261,-3.65646,1600000,213983,168,76,true,-3.4699,0.71917,2020-09-01T23:16:41.000Z,2020-11-05T15:43:37.000Z\n",
            "RF03469,mir-1179,1287,mir-1179 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.4,77.9,This family represents the microRNA (miRNA) precursor mir-1179 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,19,0,0,Gene; miRNA;,Predicted; RNAalifold,20,0,,\"\",0.63007,-3.31311,1600000,191040,176,82,true,-3.8472,0.71907,2020-09-01T23:17:13.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03470,mir-571,1287,mir-571 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,40.0,40.2,39.8,This family represents the microRNA (miRNA) precursor mir-571 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,71,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.62841,-3.3997,1600000,229724,180,85,true,-3.5304,0.71891,2020-09-01T23:17:57.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03471,mir-745,1287,mir-745 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.1,55.9,This family represents the microRNA (miRNA) precursor mir-745 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,10,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.65431,-2.81105,1600000,170608,159,77,true,-3.9363,0.71907,2020-09-01T23:18:26.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03472,mir-623,1287,mir-623 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,92.0,43.7,This family represents the microRNA (miRNA) precursor mir-623 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.65395,-2.64434,1600000,175830,184,87,true,-3.4257,0.71891,2020-09-01T23:18:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03473,mir-8813,1287,mir-8813 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.8,46.0,This family represents the microRNA (miRNA) precursor mir-8813 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,56,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.54864,-4.18598,1600000,192298,197,96,true,-2.9181,0.71873,2020-09-01T23:19:29.000Z,2020-09-14T14:54:58.000Z\n",
            "RF03474,mir-4766,1287,mir-4766 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.8,62.4,This family represents the microRNA (miRNA) precursor mir-4766 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61583,-4.29869,1600000,334824,141,56,true,-3.0549,0.7293,2020-09-01T23:19:55.000Z,2020-09-14T14:54:58.000Z\n",
            "RF03475,mir-2387,1287,mir-2387 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.4,82.9,This family represents the microRNA (miRNA) precursor mir-2387 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,11,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.55667,-4.46591,1600000,256874,171,78,true,-3.2691,0.71907,2020-09-01T23:20:23.000Z,2020-11-05T15:43:38.000Z\n",
            "RF03476,mir-7360,1287,mir-7360 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,47.0,47.3,43.0,This family represents the microRNA (miRNA) precursor mir-7360 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,34,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60885,-3.65571,1600000,293262,145,59,true,-3.0337,0.72734,2020-09-01T23:21:41.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03477,MIR862_2,1287,MIR862_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.7,63.8,This family represents the microRNA (miRNA) precursor MIR862_2 imported from miRBase.,MIR862;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,12,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.61262,-2.95332,1600000,151466,212,90,true,-3.3157,0.71881,2020-09-01T23:22:17.000Z,2021-06-14T15:02:18.000Z\n",
            "RF03478,mir-630,1287,mir-630 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.8,87.2,This family represents the microRNA (miRNA) precursor mir-630 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.62374,-3.20304,1600000,202840,168,76,true,-3.4482,0.71901,2020-09-01T23:22:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03479,mir-324,1287,mir-324 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,72.2,46.2,This family represents the microRNA (miRNA) precursor mir-324 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,15,37,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.65456,-3.54138,1600000,168630,178,83,true,-3.359,0.71889,2020-09-01T23:23:42.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03480,mir-1284,1287,mir-1284 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.6,79.9,This family represents the microRNA (miRNA) precursor mir-1284 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,29,0,0,Gene; miRNA;,Predicted; RNAalifold,30,0,,\"\",0.60184,-2.67855,1600000,173716,190,92,true,-3.4916,0.71861,2020-09-01T23:24:57.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03481,mir-9014,1287,mir-9014 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.6,62.1,This family represents the microRNA (miRNA) precursor mir-9014 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,252,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6411,-3.11812,1600000,194681,185,88,true,-3.5145,0.71882,2020-09-01T23:26:03.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03482,mir-4774,1287,mir-4774 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.3,54.6,This family represents the microRNA (miRNA) precursor mir-4774 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.60066,-2.90414,1600000,212729,168,76,true,-3.2907,0.71899,2020-09-01T23:26:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03483,MIR319,1287,MIR319 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.0,53.2,This family represents the microRNA (miRNA) precursor MIR319 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,11,90,0,0,Gene; miRNA;,Predicted; RNAalifold,26,0,,\"\",0.65496,-3.413,1600000,229603,223,92,true,-3.1119,0.71871,2020-09-01T23:27:21.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03484,mir-K12-10,1287,mir-K12-10 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,101.8,46.8,This family represents the microRNA (miRNA) precursor mir-K12-10 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60495,-4.18164,1600000,318662,160,70,true,-3.3882,0.72121,2020-09-01T23:28:10.000Z,2020-09-14T14:54:58.000Z\n",
            "RF03485,mir-4000,1287,mir-4000 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,68.2,38.3,This family represents the microRNA (miRNA) precursor mir-4000 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,6,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61693,-4.4764,1600000,280131,129,54,true,-3.4286,0.72329,2020-09-01T23:28:37.000Z,2020-09-14T14:54:59.000Z\n",
            "RF03486,mir-3940,1287,mir-3940 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.5,50.9,This family represents the microRNA (miRNA) precursor mir-3940 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.59238,-3.53977,1600000,205217,191,92,true,-3.5607,0.71894,2020-09-01T23:29:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03487,mir-4423,1287,mir-4423 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.1,50.6,This family represents the microRNA (miRNA) precursor mir-4423 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.65559,-2.65834,1600000,233423,174,80,true,-3.5931,0.71891,2020-09-01T23:29:29.000Z,2020-09-14T14:54:59.000Z\n",
            "RF03488,mir-2366,1287,mir-2366 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.6,63.7,This family represents the microRNA (miRNA) precursor mir-2366 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.57758,-3.89345,1600000,168418,178,83,true,-3.2894,0.71891,2020-09-01T23:29:57.000Z,2020-09-14T14:54:59.000Z\n",
            "RF03489,mir-4796,1287,mir-4796 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.6,70.9,This family represents the microRNA (miRNA) precursor mir-4796 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.63576,-3.04387,1600000,166925,192,93,true,-3.1581,0.71881,2020-09-01T23:30:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03490,mir-1426,1287,mir-1426 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.0,56.7,This family represents the microRNA (miRNA) precursor mir-1426 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,49,0,0,Gene; miRNA;,Predicted; RNAalifold,49,0,,\"\",0.60577,-3.34008,1600000,166920,185,89,true,-3.5401,0.72123,2020-09-01T23:31:24.000Z,2020-09-14T14:54:59.000Z\n",
            "RF03491,mir-3146,1287,mir-3146 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.5,85.3,This family represents the microRNA (miRNA) precursor mir-3146 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.60981,-3.83778,1600000,285893,172,79,true,-3.4116,0.71904,2020-09-01T23:31:51.000Z,2020-11-05T15:43:39.000Z\n",
            "RF03492,mir-878,1287,mir-878 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,93.4,77.8,This family represents the microRNA (miRNA) precursor mir-878 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.65607,-3.40714,1600000,241132,168,76,true,-3.1425,0.72003,2020-09-01T23:32:15.000Z,2020-09-14T14:54:59.000Z\n",
            "RF03493,mir-1281,1287,mir-1281 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.9,45.3,This family represents the microRNA (miRNA) precursor mir-1281 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,25,0,0,Gene; miRNA;,Predicted; RNAalifold,26,0,,\"\",0.52177,-6.59217,1600000,377080,135,54,true,-3.6355,0.71954,2020-09-01T23:32:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03494,mir-4680,1287,mir-4680 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.3,75.2,This family represents the microRNA (miRNA) precursor mir-4680 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,27,0,0,Gene; miRNA;,Predicted; RNAalifold,25,0,,\"\",0.60756,-3.50569,1600000,234939,157,68,true,-3.1163,0.72272,2020-09-01T23:33:30.000Z,2020-09-14T14:54:59.000Z\n",
            "RF03495,mir-968,1287,mir-968 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.1,78.0,This family represents the microRNA (miRNA) precursor mir-968 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.61186,-2.75203,1600000,191711,168,79,true,-3.5005,0.71899,2020-09-01T23:33:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03496,MIR2086,1287,MIR2086 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,50.2,49.3,This family represents the microRNA (miRNA) precursor MIR2086 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.62542,-2.23854,1600000,184684,181,87,true,-2.9821,0.71882,2020-09-01T23:35:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03497,MIR5038,1287,MIR5038 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.6,69.3,This family represents the microRNA (miRNA) precursor MIR5038 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61451,-3.68692,1600000,257232,167,75,true,-3.3824,0.72017,2020-09-01T23:35:29.000Z,2020-11-05T15:43:39.000Z\n",
            "RF03498,mir-3145,1287,mir-3145 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.8,82.8,This family represents the microRNA (miRNA) precursor mir-3145 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.62301,-3.02085,1600000,188161,178,84,true,-3.6096,0.71894,2020-09-01T23:36:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03499,mir-702,1287,mir-702 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,93.8,44.1,This family represents the microRNA (miRNA) precursor mir-702 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.53415,-4.43494,1600000,174260,204,102,true,-3.1768,0.71873,2020-09-01T23:36:52.000Z,2020-09-14T14:55:00.000Z\n",
            "RF03500,mir-1262,1287,mir-1262 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,95.3,59.1,This family represents the microRNA (miRNA) precursor mir-1262 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.65573,-2.11312,1600000,207039,190,91,true,-3.2663,0.71876,2020-09-01T23:37:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03501,mir-3140,1287,mir-3140 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.3,62.8,This family represents the microRNA (miRNA) precursor mir-3140 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.60945,-3.38651,1600000,196829,188,90,true,-3.167,0.71875,2020-09-01T23:38:09.000Z,2020-09-14T14:55:00.000Z\n",
            "RF03502,mir-1914,1287,mir-1914 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.4,53.1,This family represents the microRNA (miRNA) precursor mir-1914 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.54956,-4.74068,1600000,235132,173,80,true,-3.3438,0.71895,2020-09-01T23:38:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03503,mir-547,1287,mir-547 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,101.9,87.9,This family represents the microRNA (miRNA) precursor mir-547 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.69527,-2.62144,1600000,267466,171,78,true,-3.431,0.72102,2020-09-01T23:39:21.000Z,2020-11-05T15:43:40.000Z\n",
            "RF03504,mir-9461,1287,mir-9461 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.5,79.5,This family represents the microRNA (miRNA) precursor mir-9461 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,11,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6813,-2.70214,1600000,240934,173,80,true,-3.101,0.71892,2020-09-01T23:39:44.000Z,2020-09-14T14:55:00.000Z\n",
            "RF03505,mir-9128,1287,mir-9128 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.6,42.0,This family represents the microRNA (miRNA) precursor mir-9128 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,11,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.54614,-4.67931,1600000,220615,195,95,true,-3.0398,0.71888,2020-09-01T23:40:06.000Z,2020-09-14T14:55:00.000Z\n",
            "RF03506,mir-587,1287,mir-587 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,90.3,79.4,This family represents the microRNA (miRNA) precursor mir-587 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.65696,-3.28278,1600000,216474,165,74,true,-3.8108,0.71907,2020-09-01T23:40:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03507,mir-4429,1287,mir-4429 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.0,58.6,This family represents the microRNA (miRNA) precursor mir-4429 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.72335,-2.97323,1600000,294649,164,73,true,-3.4022,0.72033,2020-09-01T23:41:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03508,mir-970,1287,mir-970 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.0,40.6,This family represents the microRNA (miRNA) precursor mir-970 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,53,0,0,Gene; miRNA;,Predicted; RNAalifold,55,0,,\"\",0.59286,-3.96029,1600000,196003,159,74,true,-3.6441,0.71902,2020-09-01T23:41:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03509,mir-4660,1287,mir-4660 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.3,73.0,This family represents the microRNA (miRNA) precursor mir-4660 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.63531,-3.12204,1600000,194024,182,86,true,-3.4568,0.71888,2020-09-01T23:42:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03510,mir-3660,1287,mir-3660 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.7,66.7,This family represents the microRNA (miRNA) precursor mir-3660 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,33,0,0,Gene; miRNA;,Predicted; RNAalifold,34,0,,\"\",0.64838,-2.71674,1600000,218040,176,82,true,-3.5487,0.71903,2020-09-01T23:43:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03511,mir-5444,1287,mir-5444 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,101.5,64.1,This family represents the microRNA (miRNA) precursor mir-5444 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63211,-2.77375,1600000,151841,185,89,true,-2.8617,0.72216,2020-09-01T23:44:18.000Z,2020-11-05T15:43:41.000Z\n",
            "RF03512,mir-3479,1287,mir-3479 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.7,66.8,This family represents the microRNA (miRNA) precursor mir-3479 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.64101,-3.15149,1600000,245742,165,74,true,-3.3929,0.72129,2020-09-01T23:45:00.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03514,mir-6128,1287,mir-6128 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,92.0,92.7,54.2,This family represents the microRNA (miRNA) precursor mir-6128 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.5908,-2.6037,1600000,246216,150,64,true,-3.4403,0.71925,2020-09-01T23:45:43.000Z,2020-09-14T14:55:01.000Z\n",
            "RF03516,MIR5229,1287,MIR5229 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,111.3,62.0,This family represents the microRNA (miRNA) precursor MIR5229 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67121,-2.114,1600000,188629,184,87,true,-3.6076,0.71893,2020-09-01T23:46:47.000Z,2020-11-05T15:43:41.000Z\n",
            "RF03517,MIR5225,1287,MIR5225 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,72.0,73.0,71.0,This family represents the microRNA (miRNA) precursor MIR5225 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,14,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.62052,-5.0138,1600000,187516,232,126,true,-3.1038,0.71831,2020-09-01T23:47:12.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03518,mir-1893,1287,mir-1893 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.2,57.4,This family represents the microRNA (miRNA) precursor mir-1893 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,31,0,0,Gene; miRNA;,Predicted; RNAalifold,32,0,,\"\",0.58093,-3.6669,1600000,185483,178,83,true,-3.2186,0.71887,2020-09-01T23:48:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03519,mir-238,1287,mir-238 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,87.2,74.0,This family represents the microRNA (miRNA) precursor mir-238 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.61704,-3.28856,1600000,163483,176,85,true,-3.6686,0.71888,2020-09-01T23:48:34.000Z,2020-11-05T15:43:41.000Z\n",
            "RF03520,mir-3059,1287,mir-3059 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.1,53.9,This family represents the microRNA (miRNA) precursor mir-3059 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,36,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.60058,-5.15859,1600000,312982,145,59,true,-2.964,0.72614,2020-09-01T23:49:11.000Z,2020-11-05T15:43:41.000Z\n",
            "RF03521,mir-23c,1287,mir-23c microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.8,64.7,This family represents the microRNA (miRNA) precursor mir-23c imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.63028,-3.05294,1600000,196736,202,100,true,-3.0775,0.71873,2020-09-01T23:51:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03522,mir-4850,1287,mir-4850 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.7,81.8,This family represents the microRNA (miRNA) precursor mir-4850 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65643,-2.16311,1600000,197284,184,88,true,-3.6906,0.71887,2020-09-01T23:52:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03523,mir-4703,1287,mir-4703 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.1,70.3,This family represents the microRNA (miRNA) precursor mir-4703 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.64157,-3.45118,1600000,248872,148,61,true,-3.3111,0.72681,2020-09-01T23:53:48.000Z,2020-11-05T15:43:41.000Z\n",
            "RF03524,mir-3174,1287,mir-3174 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.1,79.8,This family represents the microRNA (miRNA) precursor mir-3174 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66221,-2.83815,1600000,232664,184,87,true,-3.4697,0.71888,2020-09-01T23:54:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03525,mir-1343,1287,mir-1343 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.4,50.7,This family represents the microRNA (miRNA) precursor mir-1343 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,60,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.54429,-4.04309,1600000,177321,176,82,true,-3.1427,0.7191,2020-09-01T23:56:02.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03526,mir-7177,1287,mir-7177 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.0,85.6,This family represents the microRNA (miRNA) precursor mir-7177 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,23,0,0,Gene; miRNA;,Predicted; RNAalifold,23,0,,\"\",0.65026,-3.68898,1600000,238521,161,71,true,-3.2954,0.72137,2020-09-01T23:56:40.000Z,2020-11-05T15:43:42.000Z\n",
            "RF03527,mir-3072,1287,mir-3072 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.2,50.5,This family represents the microRNA (miRNA) precursor mir-3072 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.54338,-4.36679,1600000,243365,178,83,true,-2.9379,0.71905,2020-09-01T23:57:04.000Z,2020-11-05T15:43:42.000Z\n",
            "RF03528,mir-1264,1287,mir-1264 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.0,47.7,This family represents the microRNA (miRNA) precursor mir-1264 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,35,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.6351,-3.5986,1600000,250906,154,67,true,-3.3634,0.71921,2020-09-02T19:41:46.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03548,mir-302_2,1287,mir-302_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,91.1,59.3,This family represents the microRNA (miRNA) precursor mir-302 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.65204,-2.66235,1600000,195227,156,68,true,-3.6596,0.71911,2020-09-18T14:08:35.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03549,mir-15_2,1287,mir-15_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,90.1,54.8,This family represents the microRNA (miRNA) precursor mir-15_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62956,-4.27373,1600000,291309,141,56,true,-3.3589,0.72836,2020-09-18T14:44:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03550,mir-31_2,1287,mir-31_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.5,79.4,This family represents the microRNA (miRNA) precursor mir-31_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.6256,-3.31882,1600000,209734,177,83,true,-3.3103,0.72167,2020-09-18T14:56:21.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03552,mir-43_2,1287,mir-43_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.5,60.9,This family represents the microRNA (miRNA) precursor mir-43_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.69298,-2.63526,1600000,187981,167,78,true,-3.6717,0.71899,2020-10-16T16:54:55.000Z,2020-11-05T15:43:42.000Z\n",
            "RF03553,mir-190_2,1287,mir-190_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,129.0,129.2,106.9,This family represents the microRNA (miRNA) precursor mir-190_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.66371,-3.02589,1600000,189865,233,123,true,-3.3486,0.71833,2020-10-16T17:07:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03554,mir-92_2,1721,mir-92_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.0,43.9,This family represents the microRNA (miRNA) precursor mir-92_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59169,-3.97051,1600000,208863,175,82,true,-3.0915,0.72872,2020-10-16T17:17:48.000Z,2020-11-05T15:44:19.000Z\n",
            "RF03555,mir-2_2,1287,mir-2_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.4,77.8,This family represents the microRNA (miRNA) precursor mir-2_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.63718,-3.48885,1600000,250171,171,78,true,-3.5045,0.71973,2020-10-16T17:42:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03556,mir-559,1287,mir-559 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.4,94.3,This family represents the microRNA (miRNA) precursor mir-559 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.6043,-4.04772,1600000,263700,189,91,true,-3.3015,0.71892,2020-11-26T11:07:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03557,MIR6019,1287,MIR6019 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,125.3,63.5,This family represents the microRNA (miRNA) precursor MIR6019 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62271,-3.44905,1600000,184391,217,111,true,-3.1951,0.71862,2020-11-26T11:08:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03558,mir-629,1287,mir-629 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.3,71.8,This family represents the microRNA (miRNA) precursor mir-629 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.51353,-5.60133,1600000,222332,178,83,true,-3.1549,0.71909,2020-11-26T11:09:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03559,mir-8303,1287,mir-8303 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,91.8,37.2,This family represents the microRNA (miRNA) precursor mir-8303 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64416,-4.20428,1600000,331632,144,58,true,-3.3303,0.72679,2020-11-26T11:09:40.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03560,mir-3926,1287,mir-3926 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.0,61.2,This family represents the microRNA (miRNA) precursor mir-3926 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,12,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.57637,-4.79181,1600000,284986,151,63,true,-2.8982,0.72702,2020-11-26T11:10:03.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03561,mir-3533,1287,mir-3533 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,104.0,104.1,103.5,This family represents the microRNA (miRNA) precursor mir-3533 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,56,0,0,Gene; miRNA;,Predicted; RNAalifold,44,0,,\"\",0.63801,-3.21219,1600000,240574,179,84,true,-3.7007,0.71893,2020-11-26T11:11:14.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03562,mir-2004,1287,mir-2004 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,115.7,59.0,This family represents the microRNA (miRNA) precursor mir-2004 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58891,-3.48744,1600000,175361,200,98,true,-3.3901,0.71875,2020-11-26T11:11:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03563,mir-BART16,1287,mir-BART16 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.4,54.3,This family represents the microRNA (miRNA) precursor mir-BART16 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63497,-2.63688,1600000,208934,182,87,true,-3.534,0.71964,2020-11-26T11:11:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03564,mir-9416,1287,mir-9416 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.4,94.4,This family represents the microRNA (miRNA) precursor mir-9416 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60935,-4.20377,1600000,285695,157,68,true,-3.3844,0.72226,2020-11-26T11:12:08.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03565,mir-1282,1287,mir-1282 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,107.0,107.2,106.5,This family represents the microRNA (miRNA) precursor mir-1282 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,45,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.61404,-3.73526,1600000,194227,203,101,true,-3.5001,0.71867,2020-11-26T11:13:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03566,mir-4875,1287,mir-4875 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,94.4,73.4,This family represents the microRNA (miRNA) precursor mir-4875 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64477,-3.0113,1600000,241756,164,73,true,-2.8553,0.72758,2020-11-26T11:13:44.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03567,MIR8024,1287,MIR8024 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,57.3,57.3,57.1,This family represents the microRNA (miRNA) precursor MIR8024 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,377,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.63012,-2.9131,1600000,212692,182,86,true,-3.3898,0.71873,2020-11-26T11:14:28.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03568,mir-3204,1287,mir-3204 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.1,61.0,This family represents the microRNA (miRNA) precursor mir-3204 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,16,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.647,-3.06699,1600000,244396,167,75,true,-3.5126,0.72039,2020-11-26T11:14:47.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03569,mir-7973,1287,mir-7973 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.9,77.3,This family represents the microRNA (miRNA) precursor mir-7973 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.62449,-4.20026,1600000,264714,165,74,true,-2.7641,0.72527,2020-11-26T11:15:05.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03570,mir-8286,1287,mir-8286 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,72.8,33.7,This family represents the microRNA (miRNA) precursor mir-8286 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,13,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64247,-4.21848,1600000,318724,141,56,true,-3.2836,0.72839,2020-11-26T11:15:22.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03571,mir-734,1287,mir-734 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,114.1,60.7,This family represents the microRNA (miRNA) precursor mir-734 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61284,-3.53874,1600000,228110,193,93,true,-3.5549,0.71887,2020-11-26T11:15:39.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03572,MIR2109,1287,MIR2109 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,104.8,51.4,This family represents the microRNA (miRNA) precursor MIR2109 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64227,-3.46927,1600000,262214,161,71,true,-3.2309,0.7203,2020-11-26T11:15:56.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03573,mir-1954,1287,mir-1954 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.5,84.4,This family represents the microRNA (miRNA) precursor mir-1954 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66675,-3.19041,1600000,211278,176,82,true,-3.2818,0.72267,2020-11-26T11:16:17.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03574,mir-4847,1287,mir-4847 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.8,50.6,This family represents the microRNA (miRNA) precursor mir-4847 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.70854,-1.89697,1600000,193509,172,79,true,-3.2468,0.72038,2020-11-26T11:16:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03575,mir-rL1-14,1287,mir-rL1-14 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,68.3,42.3,This family represents the microRNA (miRNA) precursor mir-rL1-14 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59529,-3.70701,1600000,204348,181,87,true,-3.5572,0.71899,2020-11-26T11:16:53.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03576,mir-4435,1287,mir-4435 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.9,53.7,This family represents the microRNA (miRNA) precursor mir-4435 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.67365,-3.11959,1600000,243318,165,74,true,-3.397,0.71897,2020-11-26T11:17:19.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03577,mir-9193,1287,mir-9193 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,55.0,55.0,54.9,This family represents the microRNA (miRNA) precursor mir-9193 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3294,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.56537,-4.40855,1600000,209124,165,74,true,-3.076,0.72291,2020-11-26T11:18:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03578,mir-3910,1287,mir-3910 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,97.3,94.4,This family represents the microRNA (miRNA) precursor mir-3910 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62319,-3.43301,1600000,212496,176,82,true,-3.3803,0.72015,2020-11-26T11:18:45.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03579,mir-2058,1287,mir-2058 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.0,46.8,This family represents the microRNA (miRNA) precursor mir-2058 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59217,-4.2606,1600000,258180,162,72,true,-3.2776,0.72287,2020-11-26T11:19:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03580,mir-928,1287,mir-928 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,108.0,96.8,This family represents the microRNA (miRNA) precursor mir-928 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.64913,-2.93982,1600000,208920,200,100,true,-3.235,0.71857,2020-11-26T11:22:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03581,mir-4878,1287,mir-4878 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,106.6,46.6,This family represents the microRNA (miRNA) precursor mir-4878 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63154,-3.7438,1600000,212486,164,73,true,-3.4107,0.72018,2020-11-26T11:23:02.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03582,mir-m22-1,1287,mir-m22-1 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,82.9,44.8,This family represents the microRNA (miRNA) precursor mir-m22-1 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.53001,-5.78902,1600000,301404,156,67,true,-2.2368,0.73437,2020-11-26T11:23:19.000Z,2020-12-09T15:48:42.000Z\n",
            "RF03583,MIR781,1287,MIR781 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.8,81.7,This family represents the microRNA (miRNA) precursor MIR781 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.60724,-2.61659,1600000,150122,191,94,true,-3.0138,0.71878,2020-11-26T11:23:39.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03584,mir-2290,1287,mir-2290 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,56.8,39.8,This family represents the microRNA (miRNA) precursor mir-2290 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.5641,-4.15053,1600000,269414,174,80,true,-3.3839,0.71895,2020-11-26T11:24:00.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03585,mir-3938,1287,mir-3938 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.8,93.2,This family represents the microRNA (miRNA) precursor mir-3938 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.63472,-3.25936,1600000,192655,184,88,true,-2.9533,0.719,2020-11-26T11:24:28.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03586,mir-5132,1287,mir-5132 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.9,67.8,This family represents the microRNA (miRNA) precursor mir-5132 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.53523,-4.59524,1600000,274287,161,71,true,-3.0759,0.72134,2020-11-26T11:24:51.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03587,mir-1202,1287,mir-1202 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.8,66.5,This family represents the microRNA (miRNA) precursor mir-1202 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.62172,-3.57484,1600000,206176,174,80,true,-2.829,0.71904,2020-11-26T11:25:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03588,mir-2319,1287,mir-2319 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.5,65.0,This family represents the microRNA (miRNA) precursor mir-2319 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63943,-3.36801,1600000,233770,161,71,true,-3.1023,0.7243,2020-11-26T11:26:02.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03589,mir-5583,1287,mir-5583 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.8,77.4,This family represents the microRNA (miRNA) precursor mir-5583 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,11,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.54312,-4.79352,1600000,273369,145,59,true,-3.0012,0.73288,2020-11-26T11:26:24.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03590,mir-2009,1287,mir-2009 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.4,59.3,This family represents the microRNA (miRNA) precursor mir-2009 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,1,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.68769,-2.38143,1600000,194008,176,83,true,-3.7502,0.71893,2020-11-26T11:26:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03591,mir-3117,1287,mir-3117 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.6,54.6,This family represents the microRNA (miRNA) precursor mir-3117 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.6307,-3.28195,1600000,235567,171,78,true,-3.183,0.71907,2020-11-26T11:27:10.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03592,mir-2567,1287,mir-2567 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.7,77.6,This family represents the microRNA (miRNA) precursor mir-2567 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,14,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63762,-1.97653,1600000,188352,176,83,true,-3.6989,0.71887,2020-11-26T11:27:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03593,mir-2331,1287,mir-2331 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,104.5,68.5,This family represents the microRNA (miRNA) precursor mir-2331 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.5636,-4.61509,1600000,266118,161,71,true,-3.5738,0.72066,2020-11-26T11:27:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03594,mir-3817,1287,mir-3817 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,95.2,68.0,This family represents the microRNA (miRNA) precursor mir-3817 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60259,-3.77246,1600000,186563,185,89,true,-3.1056,0.72106,2020-11-26T11:28:10.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03595,mir-3569,1287,mir-3569 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,52.3,42.2,This family represents the microRNA (miRNA) precursor mir-3569 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.60471,-4.77756,1600000,279115,144,58,true,-2.8806,0.72993,2020-11-26T11:28:30.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03596,mir-5880,1287,mir-5880 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.3,66.5,This family represents the microRNA (miRNA) precursor mir-5880 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63329,-3.44999,1600000,236250,172,79,true,-3.1289,0.72295,2020-11-26T11:28:47.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03597,MIR6438,1287,MIR6438 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.2,51.5,This family represents the microRNA (miRNA) precursor MIR6438 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57384,-3.66565,1600000,210377,199,100,true,-3.1559,0.71879,2020-11-26T11:29:04.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03598,MIR6024,1287,MIR6024 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.4,65.6,This family represents the microRNA (miRNA) precursor MIR6024 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.63954,-3.09709,1600000,183292,199,99,true,-3.0847,0.71876,2020-11-26T11:29:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03599,MIR1883,1287,MIR1883 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,128.0,128.6,127.0,This family represents the microRNA (miRNA) precursor MIR1883 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.55008,-4.7182,1600000,158433,226,118,true,-3.0746,0.71849,2020-11-26T11:29:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03600,MIR3627,1287,MIR3627 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.6,61.6,This family represents the microRNA (miRNA) precursor MIR3627 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.69059,-2.50161,1600000,188651,181,89,true,-3.4169,0.71889,2020-11-26T11:30:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03601,MIR7993,1287,MIR7993 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.0,99.6,This family represents the microRNA (miRNA) precursor MIR7993 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,19,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.57963,-3.95352,1600000,218072,253,138,true,-2.8733,0.71833,2020-11-26T11:31:20.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03602,mir-8250,1287,mir-8250 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,83.1,37.4,This family represents the microRNA (miRNA) precursor mir-8250 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59617,-4.55502,1600000,311231,140,60,true,-3.7266,0.71935,2020-11-26T11:31:42.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03603,mir-4899,1287,mir-4899 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.6,42.0,This family represents the microRNA (miRNA) precursor mir-4899 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61257,-3.01742,1600000,158818,182,87,true,-3.3009,0.71939,2020-11-26T11:32:02.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03604,mir-5728,1287,mir-5728 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,106.3,94.6,This family represents the microRNA (miRNA) precursor mir-5728 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61661,-2.89789,1600000,200096,179,84,true,-3.4517,0.71899,2020-11-26T11:32:20.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03605,MIR4221,1287,MIR4221 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,129.8,72.1,This family represents the microRNA (miRNA) precursor MIR4221 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62181,-4.02182,1600000,229793,243,132,true,-3.0651,0.71821,2020-11-26T11:32:38.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03606,mir-2071,1287,mir-2071 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,106.7,37.7,This family represents the microRNA (miRNA) precursor mir-2071 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.73439,-2.207,1600000,273707,165,74,true,-3.4605,0.71907,2020-11-26T11:32:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03607,mir-9447,1287,mir-9447 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.5,66.1,This family represents the microRNA (miRNA) precursor mir-9447 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60145,-4.42565,1600000,264121,157,68,true,-3.3889,0.72238,2020-11-26T11:33:31.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03608,mir-9440,1287,mir-9440 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.7,79.1,This family represents the microRNA (miRNA) precursor mir-9440 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,355,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67976,-2.66487,1600000,215982,156,68,true,-3.9256,0.71904,2020-11-26T11:33:54.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03609,mir-5470,1287,mir-5470 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.0,56.1,This family represents the microRNA (miRNA) precursor mir-5470 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64151,-3.13858,1600000,193464,174,81,true,-3.3542,0.72124,2020-11-26T11:34:20.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03610,mir-H7,1287,mir-H7 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.0,51.1,This family represents the microRNA (miRNA) precursor mir-H7 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.54728,-5.41723,1600000,293985,158,69,true,-3.4944,0.72356,2020-11-26T11:34:37.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03611,mir-7147,1287,mir-7147 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,93.3,75.4,This family represents the microRNA (miRNA) precursor mir-7147 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64897,-3.99412,1600000,284629,144,58,true,-3.3208,0.72649,2020-11-26T11:34:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03612,mir-608,1287,mir-608 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,96.2,78.7,This family represents the microRNA (miRNA) precursor mir-608 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.55606,-4.92865,1600000,254480,177,82,true,-3.4727,0.71885,2020-11-26T11:35:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03613,MIR6234,1287,MIR6234 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.0,89.2,This family represents the microRNA (miRNA) precursor MIR6234 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,11,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.68653,-3.51292,1600000,248213,265,146,true,-3.3181,0.71831,2020-11-26T11:35:42.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03614,mir-3431,1287,mir-3431 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,114.3,97.6,This family represents the microRNA (miRNA) precursor mir-3431 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6883,-2.08198,1600000,186729,183,86,true,-3.3877,0.71901,2020-11-26T11:36:00.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03615,mir-4877,1287,mir-4877 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,110.0,51.0,This family represents the microRNA (miRNA) precursor mir-4877 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59439,-3.84085,1600000,184124,183,87,true,-3.3935,0.7189,2020-11-26T11:36:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03616,MIR1888,1287,MIR1888 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,72.0,72.2,71.8,This family represents the microRNA (miRNA) precursor MIR1888 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,20,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62728,-2.77543,1600000,198811,202,101,true,-3.2189,0.71881,2020-11-26T11:36:46.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03617,MIR8757,1287,MIR8757 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.0,78.2,This family represents the microRNA (miRNA) precursor MIR8757 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,12,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65468,-3.26878,1600000,211619,163,72,true,-2.8743,0.72575,2020-11-26T11:37:08.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03618,mir-5592,1287,mir-5592 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.8,54.4,This family represents the microRNA (miRNA) precursor mir-5592 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.55682,-5.10227,1600000,292453,154,66,true,-3.2522,0.7262,2020-11-26T11:37:26.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03619,mir-5729,1287,mir-5729 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,41.0,41.9,40.2,This family represents the microRNA (miRNA) precursor mir-5729 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63599,-2.98453,1600000,191580,172,79,true,-3.3486,0.71912,2020-11-26T11:37:47.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03620,mir-1233,1287,mir-1233 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.5,86.7,This family represents the microRNA (miRNA) precursor mir-1233 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,111,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.54916,-4.59183,1600000,187694,177,82,true,-3.3035,0.71891,2020-11-26T11:38:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03621,MIR4245,1287,MIR4245 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.1,71.1,This family represents the microRNA (miRNA) precursor MIR4245 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.55127,-4.83614,1600000,211651,238,128,true,-2.9567,0.71838,2020-11-26T11:38:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03622,mir-731,1287,mir-731 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.1,69.9,This family represents the microRNA (miRNA) precursor mir-731 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.6037,-3.21674,1600000,211266,186,90,true,-3.2117,0.72121,2020-11-26T11:39:15.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03623,mir-1252,1287,mir-1252 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,78.6,68.3,This family represents the microRNA (miRNA) precursor mir-1252 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.54561,-4.90047,1600000,292485,151,65,true,-3.4004,0.71924,2020-11-26T11:39:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03624,mir-5345,1287,mir-5345 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.6,45.3,This family represents the microRNA (miRNA) precursor mir-5345 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65476,-3.49792,1600000,223331,168,76,true,-3.4027,0.72155,2020-11-26T11:40:08.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03625,MIR6152,1287,MIR6152 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,57.0,57.1,56.9,This family represents the microRNA (miRNA) precursor MIR6152 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4031,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.61205,-2.83478,1600000,180637,198,97,true,-3.1141,0.71854,2020-11-26T11:44:09.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03626,mir-9791,1287,mir-9791 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,74.3,49.3,This family represents the microRNA (miRNA) precursor mir-9791 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64441,-3.392,1600000,238030,172,79,true,-3.2908,0.71891,2020-11-26T11:44:40.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03627,mir-9168,1287,mir-9168 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,120.1,40.4,This family represents the microRNA (miRNA) precursor mir-9168 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64734,-2.52453,1600000,197466,195,95,true,-3.4668,0.71869,2020-11-26T11:45:01.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03628,mir-2355,1287,mir-2355 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.2,78.8,This family represents the microRNA (miRNA) precursor mir-2355 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,27,0,0,Gene; miRNA;,Predicted; RNAalifold,27,0,,\"\",0.66161,-2.811,1600000,214660,161,71,true,-3.4837,0.72218,2020-11-26T11:46:02.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03629,mir-641,1287,mir-641 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.5,74.1,This family represents the microRNA (miRNA) precursor mir-641 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.59078,-3.97599,1600000,184148,199,98,true,-3.4934,0.71871,2020-11-26T11:46:25.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03630,mir-2818,1287,mir-2818 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.1,89.9,This family represents the microRNA (miRNA) precursor mir-2818 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,947,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66247,-3.18803,1600000,285506,147,60,true,-3.1763,0.72556,2020-11-26T11:46:51.000Z,2020-12-09T15:48:43.000Z\n",
            "RF03631,MIR3440,1287,MIR3440 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,105.0,115.4,104.3,This family represents the microRNA (miRNA) precursor MIR3440 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65365,-3.49614,1600000,190212,238,127,true,-3.0892,0.71858,2020-11-26T11:47:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03632,MIR1882,1287,MIR1882 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,107.0,107.3,105.4,This family represents the microRNA (miRNA) precursor MIR1882 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,97,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.61786,-3.6985,1600000,173326,243,131,true,-2.9776,0.71857,2020-11-26T11:47:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03633,mir-5882,1287,mir-5882 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.8,90.6,This family represents the microRNA (miRNA) precursor mir-5882 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59695,-3.39822,1600000,234116,169,77,true,-3.0216,0.72637,2020-11-26T11:48:14.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03634,mir-3524,1287,mir-3524 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.6,87.9,This family represents the microRNA (miRNA) precursor mir-3524 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.68205,-2.4697,1600000,214579,179,84,true,-3.0927,0.7266,2020-11-26T11:48:33.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03635,mir-2066,1287,mir-2066 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,104.2,47.1,This family represents the microRNA (miRNA) precursor mir-2066 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60885,-4.05917,1600000,229394,163,72,true,-3.2971,0.72051,2020-11-26T11:48:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03636,MIR4376,1287,MIR4376 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.6,54.7,This family represents the microRNA (miRNA) precursor MIR4376 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.64029,-3.37037,1600000,192111,184,88,true,-3.4355,0.7188,2020-11-26T11:49:13.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03637,mir-5879,1287,mir-5879 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.7,71.4,This family represents the microRNA (miRNA) precursor mir-5879 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.68219,-2.48213,1600000,215394,168,76,true,-2.9907,0.72429,2020-11-26T11:49:32.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03638,mir-1537,1287,mir-1537 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.3,60.0,This family represents the microRNA (miRNA) precursor mir-1537 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.63594,-3.63695,1600000,288127,148,61,true,-3.3379,0.7265,2020-11-26T11:50:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03639,mir-619,1287,mir-619 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.9,50.7,This family represents the microRNA (miRNA) precursor mir-619 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66963,-3.41785,1600000,265316,159,70,true,-3.2956,0.71916,2020-11-26T11:50:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03640,mir-3747,1287,mir-3747 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.8,61.4,This family represents the microRNA (miRNA) precursor mir-3747 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62876,-3.81512,1600000,258387,156,67,true,-2.6158,0.72869,2020-11-26T11:50:56.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03641,mir-991,1287,mir-991 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,109.0,58.1,This family represents the microRNA (miRNA) precursor mir-991 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.64548,-2.87929,1600000,231471,184,88,true,-2.9526,0.71882,2020-11-26T11:51:14.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03642,MIR5743,1287,MIR5743 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,111.0,111.6,60.2,This family represents the microRNA (miRNA) precursor MIR5743 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61693,-2.97136,1600000,215153,175,81,true,-3.5016,0.71903,2020-11-26T11:51:34.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03643,mir-2056,1287,mir-2056 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.4,47.1,This family represents the microRNA (miRNA) precursor mir-2056 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61366,-3.8813,1600000,226506,165,74,true,-3.291,0.72093,2020-11-26T11:51:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03644,mir-2064,1287,mir-2064 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.9,51.1,This family represents the microRNA (miRNA) precursor mir-2064 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.67387,-3.44369,1600000,256545,160,70,true,-3.641,0.72179,2020-11-26T11:52:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03645,mir-9080,1287,mir-9080 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.5,80.1,This family represents the microRNA (miRNA) precursor mir-9080 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67494,-2.25398,1600000,191789,208,107,true,-3.2985,0.71862,2020-11-26T11:52:34.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03646,MIR9776,1287,MIR9776 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,110.3,75.7,This family represents the microRNA (miRNA) precursor MIR9776 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.53614,-4.91763,1600000,228538,180,84,true,-3.2283,0.719,2020-11-26T11:52:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03647,mir-7640,1287,mir-7640 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,94.0,42.8,This family represents the microRNA (miRNA) precursor mir-7640 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6948,-2.78588,1600000,290959,147,60,true,-3.3634,0.72625,2020-11-26T11:53:09.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03648,mir-9413,1287,mir-9413 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.4,62.6,This family represents the microRNA (miRNA) precursor mir-9413 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62514,-3.64074,1600000,277366,157,68,true,-3.2104,0.72253,2020-11-26T11:53:26.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03649,MIR5643,1287,MIR5643 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.4,93.9,This family represents the microRNA (miRNA) precursor MIR5643 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,32,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62133,-3.17042,1600000,220787,178,83,true,-3.2948,0.7188,2020-11-26T11:53:45.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03650,mir-1290,1287,mir-1290 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.0,81.8,This family represents the microRNA (miRNA) precursor mir-1290 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,21,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.66075,-2.8665,1600000,272974,168,76,true,-3.5522,0.71906,2020-11-26T11:54:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03651,mir-2070,1287,mir-2070 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.4,47.7,This family represents the microRNA (miRNA) precursor mir-2070 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64314,-4.04766,1600000,273958,161,71,true,-3.4201,0.7246,2020-11-26T11:54:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03652,mir-2329,1287,mir-2329 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.3,62.7,This family represents the microRNA (miRNA) precursor mir-2329 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62852,-3.4454,1600000,258687,155,66,true,-3.4236,0.72545,2020-11-26T11:55:15.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03653,mir-971,1287,mir-971 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.5,52.9,This family represents the microRNA (miRNA) precursor mir-971 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,39,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.6173,-3.65035,1600000,140562,161,77,true,-3.8282,0.71897,2020-11-26T11:55:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03654,MIR4239,1287,MIR4239 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,118.5,84.3,This family represents the microRNA (miRNA) precursor MIR4239 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63085,-3.5884,1600000,190715,228,120,true,-3.21,0.71858,2020-11-26T11:56:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03655,MIR1887,1287,MIR1887 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.0,55.9,This family represents the microRNA (miRNA) precursor MIR1887 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65608,-2.56544,1600000,235529,151,63,true,-3.1319,0.72485,2020-11-26T11:56:21.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03656,mir-2432,1287,mir-2432 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.9,52.5,This family represents the microRNA (miRNA) precursor mir-2432 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63069,-3.73124,1600000,250877,157,68,true,-3.4454,0.72211,2020-11-26T11:56:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03657,MIR6145,1287,MIR6145 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.4,53.4,53.2,This family represents the microRNA (miRNA) precursor MIR6145 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,27,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.62886,-3.66426,1600000,246574,171,78,true,-3.3115,0.72645,2020-11-26T11:57:03.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03658,mir-2300,1287,mir-2300 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,99.2,80.0,This family represents the microRNA (miRNA) precursor mir-2300 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65272,-3.14896,1600000,252635,165,74,true,-3.2808,0.72285,2020-11-26T11:57:20.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03659,MIR2871,1287,MIR2871 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.3,65.0,This family represents the microRNA (miRNA) precursor MIR2871 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,15,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.57442,-4.77726,1600000,218411,227,118,true,-3.2589,0.71855,2020-11-26T11:57:48.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03660,mir-4018,1287,mir-4018 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.6,44.0,This family represents the microRNA (miRNA) precursor mir-4018 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63854,-3.487,1600000,211959,138,54,true,-2.5538,0.74562,2020-11-26T11:58:05.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03661,mir-933,1287,mir-933 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.2,69.9,This family represents the microRNA (miRNA) precursor mir-933 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.6579,-2.97524,1600000,173248,169,77,true,-3.4344,0.71904,2020-11-26T11:58:35.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03662,mir-2060,1287,mir-2060 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,54.5,54.5,54.4,This family represents the microRNA (miRNA) precursor mir-2060 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,901,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.67325,-2.58381,1600000,153280,191,94,true,-3.5825,0.71876,2020-11-26T11:59:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03663,mir-5911,1287,mir-5911 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.0,83.9,This family represents the microRNA (miRNA) precursor mir-5911 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,17,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63517,-3.76556,1600000,258780,162,74,true,-3.5009,0.71911,2020-11-26T11:59:48.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03664,MIR6288,1287,MIR6288 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.0,73.8,This family represents the microRNA (miRNA) precursor MIR6288 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,43,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61976,-3.30418,1600000,182303,189,92,true,-3.209,0.71892,2020-11-26T12:00:11.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03665,MIR4359,1287,MIR4359 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.0,74.9,This family represents the microRNA (miRNA) precursor MIR4359 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,101,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62472,-2.76858,1600000,270246,156,67,true,-2.6669,0.72991,2020-11-26T12:00:40.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03666,MIR7724,1287,MIR7724 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,197.0,89.8,This family represents the microRNA (miRNA) precursor MIR7724 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59893,-6.17884,1600000,256533,294,169,true,-3.0183,0.71604,2020-11-26T12:00:59.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03667,MIR1031,1287,MIR1031 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,197.8,57.8,This family represents the microRNA (miRNA) precursor MIR1031 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59869,-9.0417,1600000,325240,308,183,true,-3.2237,0.71406,2020-11-26T12:01:15.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03668,mir-1788,1287,mir-1788 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,57.0,57.7,55.0,This family represents the microRNA (miRNA) precursor mir-1788 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,55,0,0,Gene; miRNA;,Predicted; RNAalifold,55,0,,\"\",0.65422,-3.4757,1600000,150515,160,77,true,-3.4848,0.71922,2020-11-26T12:02:05.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03669,mir-2767,1287,mir-2767 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.1,59.1,This family represents the microRNA (miRNA) precursor mir-2767 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,4,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.62272,-2.87879,1600000,224583,167,80,true,-3.5384,0.71903,2020-11-26T12:02:37.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03670,MIR3932,1287,MIR3932 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,150.0,153.1,71.7,This family represents the microRNA (miRNA) precursor MIR3932 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6043,-3.67648,1600000,189789,233,123,true,-3.4274,0.71838,2020-11-26T12:02:58.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03671,MIR3522,1287,MIR3522 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,107.2,54.9,This family represents the microRNA (miRNA) precursor MIR3522 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60414,-3.8845,1600000,204677,186,89,true,-3.1402,0.71876,2020-11-26T12:03:14.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03672,MIR840,1287,MIR840 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,150.0,158.9,140.1,This family represents the microRNA (miRNA) precursor MIR840 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.55763,-5.27737,1600000,207247,266,147,true,-3.3156,0.71821,2020-11-26T12:03:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03673,mir-2822,1287,mir-2822 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.1,83.9,This family represents the microRNA (miRNA) precursor mir-2822 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,554,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59368,-3.77369,1600000,259854,156,67,true,-3.3686,0.72396,2020-11-26T12:03:53.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03674,MIR5387,1287,MIR5387 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.0,68.9,This family represents the microRNA (miRNA) precursor MIR5387 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,968,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.61061,-3.81696,1600000,172935,194,96,true,-3.1199,0.71877,2020-11-26T12:05:35.000Z,2020-12-09T15:48:44.000Z\n",
            "RF03675,mir-1001,1287,mir-1001 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,117.1,52.4,This family represents the microRNA (miRNA) precursor mir-1001 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59699,-4.03698,1600000,210567,188,90,true,-2.9741,0.71875,2020-11-26T12:05:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03676,MIR5148,1287,MIR5148 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.5,93.6,93.4,This family represents the microRNA (miRNA) precursor MIR5148 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,306,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.62029,-3.40015,1600000,185915,170,78,true,-3.3125,0.71895,2020-11-26T12:07:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03677,mir-1304,1287,mir-1304 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.8,87.7,This family represents the microRNA (miRNA) precursor mir-1304 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.61935,-3.37456,1600000,193480,183,87,true,-3.4009,0.71903,2020-11-26T12:07:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03678,mir-3956,1287,mir-3956 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,119.6,90.5,This family represents the microRNA (miRNA) precursor mir-3956 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.57415,-3.96842,1600000,179381,199,98,true,-3.4636,0.71879,2020-11-26T12:07:49.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03679,MIR2084,1287,MIR2084 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,119.7,73.5,This family represents the microRNA (miRNA) precursor MIR2084 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6782,-1.82389,1600000,145241,209,107,true,-3.4732,0.71873,2020-11-26T12:08:06.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03680,mir-9465,1287,mir-9465 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.2,36.7,This family represents the microRNA (miRNA) precursor mir-9465 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62828,-4.08107,1600000,212007,165,74,true,-3.278,0.72065,2020-11-26T12:08:23.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03681,mir-1295,1287,mir-1295 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.8,73.0,This family represents the microRNA (miRNA) precursor mir-1295 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.5955,-4.14414,1600000,236619,171,78,true,-3.6879,0.71908,2020-11-26T12:08:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03682,mir-3912,1287,mir-3912 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.1,92.5,This family represents the microRNA (miRNA) precursor mir-3912 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.61081,-3.57602,1600000,201009,189,91,true,-3.3066,0.71892,2020-11-26T12:09:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03683,mir-462,1287,mir-462 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,98.8,47.3,This family represents the microRNA (miRNA) precursor mir-462 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63343,-3.94073,1600000,243203,152,64,true,-3.4806,0.72376,2020-11-26T12:09:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03684,mir-1990,1287,mir-1990 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.7,62.1,This family represents the microRNA (miRNA) precursor mir-1990 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,2,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.68437,-3.13395,1600000,142860,158,75,true,-3.2288,0.72003,2020-11-26T12:10:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03685,MIR9677,1287,MIR9677 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,94.0,63.1,This family represents the microRNA (miRNA) precursor MIR9677 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59348,-3.94107,1600000,225965,184,87,true,-3.1653,0.71889,2020-11-26T12:10:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03686,mir-9424,1287,mir-9424 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.1,60.1,This family represents the microRNA (miRNA) precursor mir-9424 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6625,-2.7855,1600000,220391,157,68,true,-3.3538,0.72582,2020-11-26T12:10:51.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03687,mir-2011,1287,mir-2011 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,83.5,59.4,This family represents the microRNA (miRNA) precursor mir-2011 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,1,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.61145,-3.50325,1600000,196824,174,83,true,-3.6997,0.71909,2020-11-26T12:11:29.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03688,mir-3045,1287,mir-3045 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.0,88.3,This family represents the microRNA (miRNA) precursor mir-3045 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.666,-3.09365,1600000,277725,166,75,true,-3.2159,0.72295,2020-11-26T12:11:50.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03689,mir-5892,1287,mir-5892 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.5,64.9,This family represents the microRNA (miRNA) precursor mir-5892 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62882,-3.28014,1600000,221514,188,90,true,-3.425,0.71895,2020-11-26T12:12:07.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03690,mir-9058,1287,mir-9058 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.0,79.9,This family represents the microRNA (miRNA) precursor mir-9058 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,895,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57942,-3.48973,1600000,179504,198,97,true,-3.2113,0.7188,2020-11-26T12:14:47.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03691,mir-2765,1287,mir-2765 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.8,55.9,This family represents the microRNA (miRNA) precursor mir-2765 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,53,0,0,Gene; miRNA;,Predicted; RNAalifold,53,0,,\"\",0.65451,-2.45379,1600000,150561,170,83,true,-3.7292,0.71895,2020-11-26T12:15:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03692,MIR827,1287,MIR827 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,65.0,65.1,63.9,This family represents the microRNA (miRNA) precursor MIR827 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,31,0,0,Gene; miRNA;,Predicted; RNAalifold,30,0,,\"\",0.58969,-3.84076,1600000,186227,227,125,true,-2.9045,0.71836,2020-11-26T12:15:52.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03693,MIR1919,1287,MIR1919 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.9,68.4,This family represents the microRNA (miRNA) precursor MIR1919 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,13,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.6266,-3.26607,1600000,168172,561,123,true,-3.2526,0.71829,2020-11-26T12:16:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03694,mir-266,1287,mir-266 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.0,79.9,This family represents the microRNA (miRNA) precursor mir-266 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,33,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59938,-4.3656,1600000,244776,177,83,true,-3.1437,0.72244,2020-11-26T12:16:47.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03695,MIR6151,1287,MIR6151 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.3,70.7,This family represents the microRNA (miRNA) precursor MIR6151 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,612,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.68458,-3.10399,1600000,204670,158,71,true,-4.0307,0.71917,2020-11-26T12:17:43.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03696,mir-4904,1287,mir-4904 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,109.2,43.8,This family represents the microRNA (miRNA) precursor mir-4904 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59138,-4.21691,1600000,205749,172,79,true,-3.6172,0.71895,2020-11-26T12:18:07.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03697,mir-722,1287,mir-722 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.0,55.3,This family represents the microRNA (miRNA) precursor mir-722 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,28,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.62768,-3.26947,1600000,285511,154,67,true,-3.501,0.71918,2020-11-26T12:18:38.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03698,mir-2861,1287,mir-2861 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.3,56.2,This family represents the microRNA (miRNA) precursor mir-2861 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,21,0,0,Gene; miRNA;,Predicted; RNAalifold,21,0,,\"\",0.56417,-3.79218,1600000,219619,176,82,true,-2.9225,0.71891,2020-11-26T12:19:24.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03699,mir-729,1287,mir-729 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,61.0,61.7,60.9,This family represents the microRNA (miRNA) precursor mir-729 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.62897,-2.74616,1600000,178212,181,87,true,-3.2206,0.72145,2020-11-26T12:21:20.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03700,mir-1923,1287,mir-1923 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.0,92.8,This family represents the microRNA (miRNA) precursor mir-1923 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,46,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.66446,-2.61188,1600000,215222,177,83,true,-3.0847,0.71876,2020-11-26T12:21:36.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03701,mir-4041,1287,mir-4041 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.8,51.7,This family represents the microRNA (miRNA) precursor mir-4041 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60807,-3.369,1600000,279678,138,54,true,-2.9487,0.73871,2020-11-26T12:21:53.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03702,MIR851,1287,MIR851 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.8,77.6,This family represents the microRNA (miRNA) precursor MIR851 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6079,-3.78423,1600000,210144,211,108,true,-3.0226,0.71864,2020-11-26T12:22:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03703,mir-3622,1287,mir-3622 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.5,72.1,This family represents the microRNA (miRNA) precursor mir-3622 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.56621,-4.60823,1600000,216220,178,84,true,-2.8472,0.71926,2020-11-26T12:22:31.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03704,MIR4380,1287,MIR4380 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.5,82.9,This family represents the microRNA (miRNA) precursor MIR4380 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61562,-3.39388,1600000,155433,173,82,true,-4.0399,0.71909,2020-11-26T12:22:51.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03705,mir-1278,1287,mir-1278 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,61.0,61.6,60.4,This family represents the microRNA (miRNA) precursor mir-1278 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,13,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.64235,-3.47909,1600000,252784,169,77,true,-3.8901,0.71906,2020-11-26T12:23:21.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03706,mir-2253,1287,mir-2253 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,47.0,47.7,46.9,This family represents the microRNA (miRNA) precursor mir-2253 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62614,-4.41206,1600000,297930,137,57,true,-3.9169,0.71944,2020-11-26T12:23:40.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03707,mir-1892,1287,mir-1892 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.7,82.4,This family represents the microRNA (miRNA) precursor mir-1892 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.67018,-3.16062,1600000,262459,173,80,true,-3.3871,0.72108,2020-11-26T12:24:02.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03709,MIR1878,1287,MIR1878 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,77.9,62.6,This family represents the microRNA (miRNA) precursor MIR1878 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,20,0,0,Gene; miRNA;,Predicted; RNAalifold,20,0,,\"\",0.60988,-3.09858,1600000,196608,200,100,true,-3.0573,0.71864,2020-11-26T12:24:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03710,mir-1451,1287,mir-1451 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,87.3,84.5,This family represents the microRNA (miRNA) precursor mir-1451 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,46,0,0,Gene; miRNA;,Predicted; RNAalifold,46,0,,\"\",0.50973,-5.21452,1600000,212436,190,92,true,-3.0851,0.71897,2020-11-26T12:25:08.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03711,mir-784,1287,mir-784 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.1,53.0,This family represents the microRNA (miRNA) precursor mir-784 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63434,-3.29102,1600000,196208,169,77,true,-3.3408,0.72417,2020-11-26T12:25:26.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03712,MIR6450,1287,MIR6450 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.2,69.7,This family represents the microRNA (miRNA) precursor MIR6450 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,41,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62775,-3.29271,1600000,207089,208,107,true,-3.0752,0.71869,2020-11-26T12:25:46.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03713,mir-1756,1287,mir-1756 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.5,97.1,This family represents the microRNA (miRNA) precursor mir-1756 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.60868,-3.8034,1600000,222391,186,90,true,-3.1668,0.72189,2020-11-26T12:26:06.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03714,MIR774,1287,MIR774 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.9,69.2,This family represents the microRNA (miRNA) precursor MIR774 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59704,-3.6016,1600000,181947,195,98,true,-3.7299,0.71879,2020-11-26T12:26:26.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03715,mir-977,1287,mir-977 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,109.0,109.8,59.6,This family represents the microRNA (miRNA) precursor mir-977 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61672,-3.22713,1600000,168556,187,90,true,-3.5227,0.71892,2020-11-26T12:26:46.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03716,mir-7131,1287,mir-7131 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.8,73.9,This family represents the microRNA (miRNA) precursor mir-7131 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,41,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64581,-2.86489,1600000,208512,178,84,true,-3.3255,0.71916,2020-11-26T12:27:05.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03717,mir-3119,1287,mir-3119 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,87.3,70.5,This family represents the microRNA (miRNA) precursor mir-3119 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,16,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.66555,-2.33936,1600000,214679,180,85,true,-2.9039,0.72041,2020-11-26T12:27:41.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03718,mir-2804,1287,mir-2804 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.5,84.3,This family represents the microRNA (miRNA) precursor mir-2804 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,118,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64953,-2.09129,1600000,231217,170,78,true,-3.5956,0.71909,2020-11-26T12:28:03.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03719,mir-3914,1287,mir-3914 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.6,91.6,This family represents the microRNA (miRNA) precursor mir-3914 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.68453,-1.84591,1600000,185579,193,95,true,-3.4099,0.71909,2020-11-26T12:28:26.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03720,MIR4228,1287,MIR4228 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,180.0,180.9,104.8,This family represents the microRNA (miRNA) precursor MIR4228 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.70953,-3.27369,1600000,219098,272,152,true,-3.1541,0.7182,2020-11-26T12:28:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03721,mir-3478,1287,mir-3478 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.1,43.6,This family represents the microRNA (miRNA) precursor mir-3478 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.6665,-3.36076,1600000,223967,170,78,true,-3.3742,0.72328,2020-11-26T12:29:05.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03722,MIR4378,1287,MIR4378 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,98.4,76.8,This family represents the microRNA (miRNA) precursor MIR4378 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.57676,-4.14956,1600000,207432,187,91,true,-3.0461,0.72196,2020-11-26T12:29:24.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03723,MIR7125,1287,MIR7125 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,115.9,73.0,This family represents the microRNA (miRNA) precursor MIR7125 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.68242,-2.21689,1600000,201497,210,107,true,-2.995,0.71876,2020-11-26T12:29:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03724,mir-1013,1287,mir-1013 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,66.8,36.7,This family represents the microRNA (miRNA) precursor mir-1013 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.60353,-4.06196,1600000,251833,148,61,true,-3.2567,0.725,2020-11-26T12:29:58.000Z,2020-12-09T15:48:45.000Z\n",
            "RF03725,mir-7094,1287,mir-7094 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,93.3,72.1,This family represents the microRNA (miRNA) precursor mir-7094 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64426,-4.0345,1600000,271348,147,60,true,-3.4905,0.72611,2020-11-26T12:30:17.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03726,mir-m21-1,1287,mir-m21-1 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.4,47.8,This family represents the microRNA (miRNA) precursor mir-m21-1 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60844,-3.92173,1600000,243649,165,74,true,-2.6869,0.73257,2020-11-26T12:30:33.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03727,mir-3477,1287,mir-3477 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,108.0,108.6,100.9,This family represents the microRNA (miRNA) precursor mir-3477 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.62117,-3.07611,1600000,206634,189,92,true,-3.2483,0.71874,2020-11-26T12:30:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03728,MIR1511,1287,MIR1511 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.2,59.7,This family represents the microRNA (miRNA) precursor MIR1511 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59856,-3.86578,1600000,217624,180,85,true,-3.4986,0.72193,2020-11-26T12:31:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03729,mir-2363,1287,mir-2363 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,99.3,69.0,This family represents the microRNA (miRNA) precursor mir-2363 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60065,-3.36669,1600000,241980,163,72,true,-2.7918,0.72454,2020-11-26T12:31:30.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03730,mir-7615,1287,mir-7615 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,98.4,45.2,This family represents the microRNA (miRNA) precursor mir-7615 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67957,-2.98348,1600000,270172,152,64,true,-3.2391,0.72389,2020-11-26T12:31:47.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03731,mir-7371,1287,mir-7371 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.6,62.6,57.3,This family represents the microRNA (miRNA) precursor mir-7371 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,10,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59638,-3.35586,1600000,263342,135,59,true,-3.6698,0.71938,2020-11-26T12:32:15.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03732,mir-2057,1287,mir-2057 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.8,50.7,This family represents the microRNA (miRNA) precursor mir-2057 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62627,-3.73174,1600000,248026,163,72,true,-3.4971,0.72168,2020-11-26T12:32:39.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03733,mir-BART14,1287,mir-BART14 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,95.0,57.8,This family represents the microRNA (miRNA) precursor mir-BART14 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6309,-3.10924,1600000,210736,170,78,true,-3.3086,0.72363,2020-11-26T12:32:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03734,mir-4773,1287,mir-4773 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.6,73.4,This family represents the microRNA (miRNA) precursor mir-4773 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.56061,-4.859,1600000,235993,171,78,true,-3.3065,0.71987,2020-11-26T12:33:34.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03735,mir-1502,1287,mir-1502 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.2,54.7,This family represents the microRNA (miRNA) precursor mir-1502 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.55109,-3.48155,1600000,232700,145,59,true,-2.0855,0.74162,2020-11-26T12:33:50.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03736,mir-3130,1287,mir-3130 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.0,83.1,This family represents the microRNA (miRNA) precursor mir-3130 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.61066,-3.80649,1600000,227729,167,75,true,-3.2811,0.72142,2020-11-26T12:34:11.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03737,mir-8315,1287,mir-8315 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,86.1,35.4,This family represents the microRNA (miRNA) precursor mir-8315 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6543,-3.68011,1600000,293574,146,60,true,-3.096,0.72912,2020-11-26T12:34:32.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03738,mir-2320,1287,mir-2320 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,95.8,68.6,This family represents the microRNA (miRNA) precursor mir-2320 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.5958,-4.14519,1600000,234636,166,75,true,-3.429,0.7196,2020-11-26T12:34:51.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03739,mir-4529,1287,mir-4529 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.0,74.6,This family represents the microRNA (miRNA) precursor mir-4529 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,10,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.58976,-4.20005,1600000,193188,176,83,true,-3.3504,0.71916,2020-11-26T12:35:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03740,mir-921,1287,mir-921 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,86.2,70.3,This family represents the microRNA (miRNA) precursor mir-921 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.6339,-4.09111,1600000,292201,141,56,true,-2.8547,0.72819,2020-11-26T12:36:05.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03741,mir-325,1287,mir-325 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.5,68.1,This family represents the microRNA (miRNA) precursor mir-325 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,28,0,0,Gene; miRNA;,Predicted; RNAalifold,28,0,,\"\",0.59202,-4.33076,1600000,169643,198,98,true,-3.2805,0.7186,2020-11-26T12:36:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03742,mir-2162,1287,mir-2162 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,65.0,68.1,60.6,This family represents the microRNA (miRNA) precursor mir-2162 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.61345,-4.12275,1600000,245741,168,76,true,-3.2225,0.72139,2020-11-26T12:37:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03743,MIR7696,1287,MIR7696 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.8,72.9,This family represents the microRNA (miRNA) precursor MIR7696 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59707,-3.85834,1600000,219622,184,88,true,-2.9992,0.71895,2020-11-26T12:37:37.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03744,MIR3704,1287,MIR3704 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,78.2,57.4,This family represents the microRNA (miRNA) precursor MIR3704 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.58726,-4.01108,1600000,206018,254,108,true,-3.0641,0.71858,2020-11-26T12:37:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03745,mir-2063,1287,mir-2063 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,107.6,44.8,This family represents the microRNA (miRNA) precursor mir-2063 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60444,-3.4278,1600000,246638,169,77,true,-3.5378,0.71913,2020-11-26T12:38:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03746,mir-2318,1287,mir-2318 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.7,78.1,This family represents the microRNA (miRNA) precursor mir-2318 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65715,-3.04532,1600000,223247,171,79,true,-3.2094,0.72076,2020-11-26T12:38:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03747,mir-4536,1287,mir-4536 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.3,71.2,This family represents the microRNA (miRNA) precursor mir-4536 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,14,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.57201,-4.44655,1600000,242525,182,88,true,-3.2055,0.71887,2020-11-26T12:39:16.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03748,mir-2772,1287,mir-2772 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,110.0,112.8,37.3,This family represents the microRNA (miRNA) precursor mir-2772 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67753,-2.41748,1600000,213630,182,86,true,-3.5794,0.7188,2020-11-26T12:39:42.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03749,MIR2617,1287,MIR2617 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,46.5,46.5,46.4,This family represents the microRNA (miRNA) precursor MIR2617 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,1170,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.54663,-4.38868,1600000,238240,181,86,true,-3.3883,0.71871,2020-11-26T12:40:57.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03750,mir-2489,1287,mir-2489 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,112.3,44.5,This family represents the microRNA (miRNA) precursor mir-2489 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6866,-1.97815,1600000,196687,195,95,true,-3.2061,0.71879,2020-11-26T12:41:56.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03751,mir-7445,1287,mir-7445 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.9,58.5,This family represents the microRNA (miRNA) precursor mir-7445 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59302,-2.99481,1600000,265569,147,60,true,-3.2167,0.73147,2020-11-26T12:42:15.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03752,MIR5292,1287,MIR5292 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,105.0,105.0,104.9,This family represents the microRNA (miRNA) precursor MIR5292 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,103,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60281,-2.61892,1600000,177074,193,94,true,-3.3163,0.71864,2020-11-26T12:42:42.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03753,mir-1986,1287,mir-1986 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,50.6,43.1,This family represents the microRNA (miRNA) precursor mir-1986 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.5809,-5.579,1600000,383440,147,60,true,-2.9452,0.72979,2020-11-26T12:43:38.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03754,mir-3116,1287,mir-3116 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,65.0,65.0,63.2,This family represents the microRNA (miRNA) precursor mir-3116 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.65427,-3.1581,1600000,244918,157,68,true,-2.9424,0.72603,2020-11-26T12:44:02.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03755,mir-4005,1287,mir-4005 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,86.8,48.3,This family represents the microRNA (miRNA) precursor mir-4005 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63504,-4.11013,1600000,275072,138,54,true,-3.1161,0.73096,2020-11-26T12:44:21.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03756,mir-4891,1287,mir-4891 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,109.9,52.2,This family represents the microRNA (miRNA) precursor mir-4891 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64242,-3.30735,1600000,193330,185,88,true,-3.3998,0.71886,2020-11-26T12:45:13.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03757,mir-6536,1287,mir-6536 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.7,91.6,This family represents the microRNA (miRNA) precursor mir-6536 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61784,-3.62289,1600000,216099,174,81,true,-3.5146,0.71967,2020-11-26T12:45:31.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03758,mir-2756,1287,mir-2756 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,61.0,61.3,60.3,This family represents the microRNA (miRNA) precursor mir-2756 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.67939,-2.48799,1600000,188992,170,78,true,-2.9081,0.7268,2020-11-26T12:45:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03759,MIR8703,1287,MIR8703 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.1,93.8,This family represents the microRNA (miRNA) precursor MIR8703 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,98,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59315,-2.13714,1600000,239163,177,83,true,-3.0633,0.71889,2020-11-26T12:46:33.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03760,mir-2041,1287,mir-2041 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.3,67.9,This family represents the microRNA (miRNA) precursor mir-2041 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,25,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59645,-4.44839,1600000,275518,154,68,true,-3.1204,0.7193,2020-11-26T12:46:54.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03761,mir-5897,1287,mir-5897 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,94.4,50.9,This family represents the microRNA (miRNA) precursor mir-5897 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.68243,-2.55835,1600000,246863,179,84,true,-3.7667,0.71931,2020-11-26T12:47:14.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03762,MIR859,1287,MIR859 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,130.0,138.4,84.9,This family represents the microRNA (miRNA) precursor MIR859 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.58364,-4.44516,1600000,212009,239,128,true,-3.168,0.71833,2020-11-26T12:47:31.000Z,2022-10-29T17:42:07.000Z\n",
            "RF03763,mir-2568,1287,mir-2568 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.4,58.5,This family represents the microRNA (miRNA) precursor mir-2568 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.66982,-2.60655,1600000,206188,184,88,true,-3.0846,0.72185,2020-11-26T12:47:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03764,mir-647,1287,mir-647 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.0,86.6,This family represents the microRNA (miRNA) precursor mir-647 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.54671,-4.43665,1600000,205409,183,86,true,-3.1539,0.71885,2020-11-26T12:48:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03765,mir-2059,1287,mir-2059 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.5,50.1,This family represents the microRNA (miRNA) precursor mir-2059 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61512,-4.2502,1600000,291232,157,68,true,-3.3757,0.72534,2020-11-26T12:48:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03766,mir-3155,1287,mir-3155 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,71.3,69.6,This family represents the microRNA (miRNA) precursor mir-3155 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.62159,-4.37657,1600000,230982,141,56,true,-2.4686,0.73611,2020-11-26T12:48:51.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03767,mir-3432,1287,mir-3432 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.2,74.0,This family represents the microRNA (miRNA) precursor mir-3432 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63578,-3.0734,1600000,210910,166,78,true,-3.554,0.71896,2020-11-26T12:49:12.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03768,mir-1329,1287,mir-1329 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,101.2,101.2,101.1,This family represents the microRNA (miRNA) precursor mir-1329 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,47,0,0,Gene; miRNA;,Predicted; RNAalifold,47,0,,\"\",0.61117,-3.55573,1600000,185270,179,85,true,-3.7103,0.71898,2020-11-26T12:49:49.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03769,mir-3913,1287,mir-3913 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.7,83.8,This family represents the microRNA (miRNA) precursor mir-3913 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6078,-3.8871,1600000,197684,202,100,true,-3.4027,0.71899,2020-11-26T12:50:14.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03770,mir-125,1287,mir-125 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.8,57.6,This family represents the microRNA (miRNA) precursor mir-125 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.6184,-3.64534,1600000,219504,178,84,true,-3.0168,0.72024,2020-11-26T12:50:33.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03771,mir-341,1287,mir-341 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.4,65.2,This family represents the microRNA (miRNA) precursor mir-341 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.66611,-2.32053,1600000,143522,197,96,true,-3.4326,0.71872,2020-11-26T12:51:19.000Z,2020-12-09T15:48:46.000Z\n",
            "RF03772,mir-4867,1287,mir-4867 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.7,36.5,This family represents the microRNA (miRNA) precursor mir-4867 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65032,-2.64893,1600000,199086,164,73,true,-3.3993,0.72359,2020-11-26T12:51:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03773,mir-978,1287,mir-978 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,112.8,71.4,This family represents the microRNA (miRNA) precursor mir-978 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61617,-3.92579,1600000,251042,182,86,true,-3.3418,0.71893,2020-11-26T12:52:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03774,mir-2110,1287,mir-2110 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.9,84.0,This family represents the microRNA (miRNA) precursor mir-2110 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.58721,-4.19122,1600000,266317,167,75,true,-3.6731,0.71939,2020-11-26T12:52:35.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03775,mir-7578,1287,mir-7578 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,61.5,43.6,This family represents the microRNA (miRNA) precursor mir-7578 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.67103,-3.09265,1600000,232813,176,82,true,-3.3955,0.71898,2020-11-26T12:52:57.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03776,mir-2068,1287,mir-2068 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.6,51.8,This family represents the microRNA (miRNA) precursor mir-2068 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.68106,-3.07004,1600000,264204,157,68,true,-3.5804,0.72311,2020-11-26T12:53:12.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03777,MIR533,1287,MIR533 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.8,76.5,This family represents the microRNA (miRNA) precursor MIR533 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,5,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63282,-4.46499,1600000,216344,296,125,true,-3.3916,0.71833,2020-11-26T12:53:35.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03778,mir-483,1287,mir-483 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,66.6,56.0,This family represents the microRNA (miRNA) precursor mir-483 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,28,0,0,Gene; miRNA;,Predicted; RNAalifold,30,0,,\"\",0.54847,-6.0363,1600000,222623,162,75,true,-3.8336,0.7192,2020-11-26T12:54:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03779,mir-1805,1287,mir-1805 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,64.0,64.3,58.0,This family represents the microRNA (miRNA) precursor mir-1805 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,61,0,0,Gene; miRNA;,Predicted; RNAalifold,59,0,,\"\",0.67715,-2.72485,1600000,217797,178,84,true,-3.5515,0.71884,2020-11-26T12:55:03.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03780,mir-3389,1287,mir-3389 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.1,74.8,This family represents the microRNA (miRNA) precursor mir-3389 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,147,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.58269,-3.9658,1600000,187212,158,69,true,-2.8688,0.72588,2020-11-26T12:55:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03781,mir-4524,1287,mir-4524 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.6,66.3,This family represents the microRNA (miRNA) precursor mir-4524 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,10,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.66116,-2.52642,1600000,146963,176,84,true,-3.4138,0.71901,2020-11-26T12:55:53.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03782,mir-3661,1287,mir-3661 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,77.0,77.2,76.2,This family represents the microRNA (miRNA) precursor mir-3661 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,48,0,0,Gene; miRNA;,Predicted; RNAalifold,32,0,,\"\",0.64028,-3.10715,1600000,191848,179,84,true,-3.4597,0.71899,2020-11-26T12:56:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03783,mir-4854,1287,mir-4854 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,110.6,56.9,This family represents the microRNA (miRNA) precursor mir-4854 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58131,-3.8875,1600000,223844,186,89,true,-3.4812,0.71885,2020-11-26T12:57:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03784,mir-2240,1287,mir-2240 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.5,93.5,75.9,This family represents the microRNA (miRNA) precursor mir-2240 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,1,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61175,-3.02544,1600000,190264,195,97,true,-3.3749,0.71874,2020-11-26T12:57:39.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03785,mir-974,1287,mir-974 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,96.6,66.4,This family represents the microRNA (miRNA) precursor mir-974 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60968,-3.52061,1600000,227758,187,91,true,-3.6072,0.71874,2020-11-26T12:57:59.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03786,mir-9449,1287,mir-9449 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.0,54.9,This family represents the microRNA (miRNA) precursor mir-9449 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6256,-3.3567,1600000,232506,157,68,true,-3.4978,0.72257,2020-11-26T12:58:23.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03787,mir-2010,1287,mir-2010 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.8,46.6,This family represents the microRNA (miRNA) precursor mir-2010 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.56563,-4.55365,1600000,243557,168,76,true,-3.2715,0.72095,2020-11-26T12:58:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03788,MIR5304,1287,MIR5304 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.9,72.3,This family represents the microRNA (miRNA) precursor MIR5304 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.67892,-2.69649,1600000,220754,173,80,true,-3.4092,0.71939,2020-11-26T12:58:57.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03789,MIR5269,1287,MIR5269 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.0,59.9,This family represents the microRNA (miRNA) precursor MIR5269 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,371,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59436,-3.46356,1600000,224328,173,80,true,-3.1182,0.72193,2020-11-26T12:59:45.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03790,MIR8008,1287,MIR8008 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.0,65.9,This family represents the microRNA (miRNA) precursor MIR8008 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,137,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63755,-4.06529,1600000,325742,140,55,true,-3.2004,0.73516,2020-11-26T13:00:07.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03791,mir-4526,1287,mir-4526 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.6,98.9,This family represents the microRNA (miRNA) precursor mir-4526 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.6036,-3.0435,1600000,191636,184,87,true,-3.1561,0.71886,2020-11-26T13:00:29.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03792,MIR6462,1287,MIR6462 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.2,72.0,This family represents the microRNA (miRNA) precursor MIR6462 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,14,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.54789,-5.07601,1600000,202137,226,119,true,-3.1774,0.71852,2020-11-26T13:00:53.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03793,MIR810,1287,MIR810 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,170.0,179.3,129.4,This family represents the microRNA (miRNA) precursor MIR810 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.5732,-6.25668,1600000,205243,360,223,true,-3.3699,0.7105,2020-11-26T13:01:14.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03794,MIR5298,1287,MIR5298 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.1,69.7,This family represents the microRNA (miRNA) precursor MIR5298 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,269,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.51436,-3.83245,1600000,182203,182,88,true,-3.4751,0.71869,2020-11-26T13:01:50.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03795,MIR6201,1287,MIR6201 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.8,65.5,This family represents the microRNA (miRNA) precursor MIR6201 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.64347,-3.69488,1600000,166065,213,109,true,-3.1701,0.71855,2020-11-26T13:02:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03796,mir-5984,1287,mir-5984 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.9,66.2,This family represents the microRNA (miRNA) precursor mir-5984 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63037,-2.49422,1600000,163104,165,74,true,-3.1882,0.72281,2020-11-26T13:02:28.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03797,mir-9235,1287,mir-9235 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.1,93.1,This family represents the microRNA (miRNA) precursor mir-9235 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.66102,-2.96187,1600000,260422,160,70,true,-3.1618,0.72601,2020-11-26T13:02:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03798,MIR4227,1287,MIR4227 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,121.4,75.6,This family represents the microRNA (miRNA) precursor MIR4227 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63294,-2.59854,1600000,157132,231,122,true,-3.4145,0.71836,2020-11-26T13:03:02.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03799,mir-3202,1287,mir-3202 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.0,72.0,This family represents the microRNA (miRNA) precursor mir-3202 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.66194,-2.47313,1600000,227449,172,79,true,-3.0381,0.72214,2020-11-26T13:03:32.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03800,mir-4477,1287,mir-4477 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.9,73.7,This family represents the microRNA (miRNA) precursor mir-4477 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,87,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.60279,-3.78101,1600000,199124,175,81,true,-3.1701,0.72171,2020-11-26T13:04:05.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03801,mir-7143,1287,mir-7143 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,93.7,88.7,This family represents the microRNA (miRNA) precursor mir-7143 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63171,-4.3689,1600000,300027,148,61,true,-3.2297,0.72654,2020-11-26T13:04:22.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03802,MIR5649,1287,MIR5649 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,108.9,85.1,This family represents the microRNA (miRNA) precursor MIR5649 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62291,-3.28522,1600000,190406,202,101,true,-3.0044,0.71878,2020-11-26T13:04:40.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03803,MIR1886,1287,MIR1886 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.0,61.9,This family represents the microRNA (miRNA) precursor MIR1886 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,628,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62724,-4.27788,1600000,296654,132,54,true,-3.934,0.71965,2020-11-26T13:05:30.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03804,mir-4989,1287,mir-4989 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,49.0,49.1,45.2,This family represents the microRNA (miRNA) precursor mir-4989 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.65785,-2.9088,1600000,176903,174,80,true,-3.2179,0.71896,2020-11-26T13:05:51.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03805,bpcv-mir-B1,1287,bpcv-mir-B1 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.9,42.9,This family represents the microRNA (miRNA) precursor bpcv-mir-B1 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.68816,-2.72787,1600000,241477,162,72,true,-3.3716,0.72206,2020-11-26T13:06:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03806,mir-2007,1287,mir-2007 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,105.8,50.0,This family represents the microRNA (miRNA) precursor mir-2007 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6416,-2.9552,1600000,212883,187,90,true,-3.7023,0.7189,2020-11-26T13:06:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03807,mir-4890,1287,mir-4890 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,111.9,42.2,This family represents the microRNA (miRNA) precursor mir-4890 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.67786,-2.42987,1600000,159972,190,91,true,-3.4083,0.71881,2020-11-26T13:06:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03808,mir-3957,1287,mir-3957 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.1,50.4,This family represents the microRNA (miRNA) precursor mir-3957 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.57894,-4.53555,1600000,229099,170,77,true,-3.2384,0.71902,2020-11-26T13:07:01.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03809,mir-1203,1287,mir-1203 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.9,99.0,This family represents the microRNA (miRNA) precursor mir-1203 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.54732,-4.4579,1600000,250740,180,84,true,-3.0732,0.71898,2020-11-26T13:07:21.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03810,mir-8932,1287,mir-8932 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,58.9,38.3,This family represents the microRNA (miRNA) precursor mir-8932 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.52651,-4.97763,1600000,263934,169,76,true,-3.1202,0.7191,2020-11-26T13:08:08.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03811,mir-1896,1287,mir-1896 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,108.4,75.0,This family represents the microRNA (miRNA) precursor mir-1896 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62687,-3.50151,1600000,199922,175,81,true,-3.5711,0.71894,2020-11-26T13:08:25.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03812,mir-BART19,1287,mir-BART19 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.5,63.3,This family represents the microRNA (miRNA) precursor mir-BART19 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62435,-3.43075,1600000,181056,190,93,true,-3.3644,0.7209,2020-11-26T13:08:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03813,MIR8041,1287,MIR8041 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.0,79.6,This family represents the microRNA (miRNA) precursor MIR8041 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,231,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.62798,-3.51573,1600000,237376,161,71,true,-3.3737,0.72114,2020-11-26T13:09:34.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03814,mir-2797,1287,mir-2797 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.0,61.5,This family represents the microRNA (miRNA) precursor mir-2797 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60309,-3.2823,1600000,249704,159,77,true,-3.7276,0.71904,2020-11-26T13:09:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03815,mir-3051,1287,mir-3051 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.3,48.8,This family represents the microRNA (miRNA) precursor mir-3051 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58092,-3.293,1600000,241179,169,77,true,-3.0832,0.72053,2020-11-26T13:10:14.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03816,MIR6297,1287,MIR6297 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,114.0,114.2,113.6,This family represents the microRNA (miRNA) precursor MIR6297 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,28,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57812,-4.53225,1600000,236067,210,107,true,-3.1072,0.71863,2020-11-26T13:10:33.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03817,mir-3836,1287,mir-3836 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,110.5,79.0,This family represents the microRNA (miRNA) precursor mir-3836 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58996,-3.04555,1600000,184229,189,91,true,-3.3178,0.71881,2020-11-26T13:10:49.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03818,MIR7994,1287,MIR7994 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.4,99.5,This family represents the microRNA (miRNA) precursor MIR7994 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59954,-3.90255,1600000,204261,187,90,true,-3.211,0.71892,2020-11-26T13:11:05.000Z,2020-12-09T15:48:47.000Z\n",
            "RF03819,mir-Ro6-3,1287,mir-Ro6-3 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.1,60.4,This family represents the microRNA (miRNA) precursor mir-Ro6-3 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.52741,-4.22438,1600000,182872,162,72,true,-3.0169,0.72572,2020-11-26T13:11:21.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03820,mir-5429,1287,mir-5429 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,89.2,72.5,This family represents the microRNA (miRNA) precursor mir-5429 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60775,-3.94917,1600000,239887,177,83,true,-2.83,0.72417,2020-11-26T13:11:40.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03821,mir-2002,1287,mir-2002 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,49.0,49.3,46.7,This family represents the microRNA (miRNA) precursor mir-2002 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6526,-2.48892,1600000,251115,157,68,true,-3.4965,0.7232,2020-11-26T13:11:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03822,mir-3931,1287,mir-3931 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,49.0,49.1,48.8,This family represents the microRNA (miRNA) precursor mir-3931 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6191,-3.62201,1600000,211621,168,76,true,-2.943,0.72578,2020-11-26T13:12:12.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03823,mir-1814,1287,mir-1814 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.4,73.2,This family represents the microRNA (miRNA) precursor mir-1814 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62293,-4.88542,1600000,255248,160,70,true,-3.3534,0.72233,2020-11-26T13:12:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03824,MIR5285,1287,MIR5285 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.5,88.7,88.3,This family represents the microRNA (miRNA) precursor MIR5285 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,196,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67611,-3.046,1600000,276904,151,63,true,-2.9873,0.72585,2020-11-26T13:12:58.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03825,mir-2335,1287,mir-2335 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,96.7,46.0,This family represents the microRNA (miRNA) precursor mir-2335 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.64167,-2.58164,1600000,239816,159,69,true,-3.0108,0.72259,2020-11-26T13:13:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03826,MIR825,1287,MIR825 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,160.0,168.6,138.9,This family represents the microRNA (miRNA) precursor MIR825 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.60337,-3.40773,1600000,262490,264,146,true,-3.0257,0.71809,2020-11-26T13:13:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03827,mir-2229,1287,mir-2229 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,65.0,65.6,38.3,This family represents the microRNA (miRNA) precursor mir-2229 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6626,-3.31969,1600000,264041,145,62,true,-3.457,0.72302,2020-11-26T13:13:49.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03828,MIR831,1287,MIR831 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,159.0,89.8,This family represents the microRNA (miRNA) precursor MIR831 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62082,-5.38978,1600000,228137,306,180,true,-3.5003,0.71459,2020-11-26T13:14:05.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03829,mir-1257,1287,mir-1257 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,53.0,40.8,This family represents the microRNA (miRNA) precursor mir-1257 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.56976,-4.67452,1600000,201209,225,117,true,-3.1177,0.71836,2020-11-26T13:14:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03830,mir-1003,1287,mir-1003 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.9,58.0,This family represents the microRNA (miRNA) precursor mir-1003 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65611,-4.0304,1600000,346274,141,56,true,-3.1623,0.72816,2020-11-26T13:14:42.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03831,mir-627,1287,mir-627 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.5,84.0,This family represents the microRNA (miRNA) precursor mir-627 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.70473,-1.68835,1600000,174784,183,87,true,-3.6567,0.7189,2020-11-26T13:15:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03832,mir-H4,1287,mir-H4 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,96.3,53.6,This family represents the microRNA (miRNA) precursor mir-H4 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.51703,-5.39105,1600000,254073,168,76,true,-2.621,0.72687,2020-11-26T13:15:21.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03833,mir-5928,1287,mir-5928 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,59.0,59.0,58.9,This family represents the microRNA (miRNA) precursor mir-5928 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,4139,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.63757,-3.43138,1600000,282208,161,75,true,-3.3944,0.7191,2020-11-26T13:16:47.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03834,mir-3173,1287,mir-3173 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.0,81.1,This family represents the microRNA (miRNA) precursor mir-3173 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.56911,-4.21486,1600000,206539,173,80,true,-3.2101,0.71913,2020-11-26T13:17:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03835,mir-955,1287,mir-955 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,61.3,47.7,This family represents the microRNA (miRNA) precursor mir-955 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.69615,-2.31126,1600000,202028,175,81,true,-3.8148,0.71896,2020-11-26T13:17:43.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03836,mir-3547,1287,mir-3547 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,77.0,77.9,48.3,This family represents the microRNA (miRNA) precursor mir-3547 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.56444,-4.37706,1600000,201487,185,88,true,-3.2945,0.71887,2020-11-26T13:18:00.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03837,mir-3138,1287,mir-3138 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,98.6,82.7,This family represents the microRNA (miRNA) precursor mir-3138 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.50956,-6.22899,1600000,273130,177,82,true,-3.2606,0.71893,2020-11-26T13:18:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03838,MIR8032,1287,MIR8032 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.0,77.6,This family represents the microRNA (miRNA) precursor MIR8032 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,59,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66001,-4.67606,1600000,413214,125,48,true,-3.7786,0.71957,2020-11-26T13:19:15.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03839,MIR2599,1287,MIR2599 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,120.0,120.0,119.3,This family represents the microRNA (miRNA) precursor MIR2599 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,86,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.5762,-5.06549,1600000,216133,243,132,true,-2.9435,0.71834,2020-11-26T13:19:42.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03840,mir-3064,1287,mir-3064 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.1,73.8,This family represents the microRNA (miRNA) precursor mir-3064 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,93,0,0,Gene; miRNA;,Predicted; RNAalifold,90,0,,\"\",0.67254,-3.61341,1600000,258130,154,67,true,-3.7784,0.71917,2020-11-26T13:20:31.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03841,mir-8941,1287,mir-8941 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,87.4,65.9,This family represents the microRNA (miRNA) precursor mir-8941 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.53091,-4.91173,1600000,239311,180,84,true,-3.4491,0.71899,2020-11-26T13:21:01.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03842,MIR2680,1287,MIR2680 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,155.0,155.6,151.8,This family represents the microRNA (miRNA) precursor MIR2680 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,243,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58716,-6.85889,1600000,257505,293,168,true,-3.2597,0.7159,2020-11-26T13:21:34.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03843,MIR2600,1287,MIR2600 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.2,97.7,This family represents the microRNA (miRNA) precursor MIR2600 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,54,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62163,-3.92989,1600000,267683,179,85,true,-3.321,0.71899,2020-11-26T13:22:03.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03844,mir-3190,1287,mir-3190 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.6,69.0,This family represents the microRNA (miRNA) precursor mir-3190 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,12,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.5257,-5.77169,1600000,295740,169,77,true,-2.6056,0.72425,2020-11-26T13:22:25.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03845,mir-3327,1287,mir-3327 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.2,57.8,This family represents the microRNA (miRNA) precursor mir-3327 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.59948,-2.53262,1600000,176793,185,89,true,-3.3221,0.71873,2020-11-26T13:23:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03846,mir-4861,1287,mir-4861 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,110.1,65.4,This family represents the microRNA (miRNA) precursor mir-4861 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64164,-2.99619,1600000,158809,193,94,true,-3.1211,0.71866,2020-11-26T13:23:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03847,mir-1538,1287,mir-1538 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.1,69.1,This family represents the microRNA (miRNA) precursor mir-1538 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.61452,-4.51029,1600000,304962,144,58,true,-2.8648,0.72865,2020-11-26T13:24:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03848,mir-1549,1287,mir-1549 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,91.0,91.7,43.7,This family represents the microRNA (miRNA) precursor mir-1549 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.57633,-5.23245,1600000,224328,151,63,true,-3.1932,0.72816,2020-11-26T13:24:31.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03849,MIR1219,1287,MIR1219 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.9,64.8,This family represents the microRNA (miRNA) precursor MIR1219 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,5,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6303,-3.61354,1600000,201056,178,82,true,-3.5391,0.71898,2020-11-26T13:24:57.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03850,mir-961,1287,mir-961 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,104.0,63.1,This family represents the microRNA (miRNA) precursor mir-961 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60247,-3.71467,1600000,216266,180,85,true,-3.7227,0.71906,2020-11-26T13:25:14.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03851,mir-1002,1287,mir-1002 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.8,58.5,This family represents the microRNA (miRNA) precursor mir-1002 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.61096,-1.20123,1600000,153966,180,89,true,-2.9917,0.71879,2020-11-26T13:25:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03852,mir-610,1287,mir-610 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,98.9,73.2,This family represents the microRNA (miRNA) precursor mir-610 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.67767,-2.82667,1600000,260551,167,75,true,-3.4022,0.71959,2020-11-26T13:26:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03853,mir-2013,1287,mir-2013 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,74.9,56.6,This family represents the microRNA (miRNA) precursor mir-2013 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,2,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.63767,-3.39051,1600000,202764,171,83,true,-3.5609,0.71894,2020-11-26T13:26:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03854,mir-1949,1287,mir-1949 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.0,97.0,This family represents the microRNA (miRNA) precursor mir-1949 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6666,-3.54032,1600000,258403,160,70,true,-3.2503,0.72166,2020-11-26T13:26:51.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03855,MIR5241,1287,MIR5241 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.1,78.4,This family represents the microRNA (miRNA) precursor MIR5241 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,13,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67403,-1.88461,1600000,196145,166,75,true,-3.7305,0.71901,2020-11-26T13:27:11.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03856,mir-1188,1287,mir-1188 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,106.0,106.5,103.0,This family represents the microRNA (miRNA) precursor mir-1188 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.66986,-2.68996,1600000,209821,177,82,true,-2.9457,0.71888,2020-11-26T13:27:34.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03857,mir-3129,1287,mir-3129 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.3,85.2,This family represents the microRNA (miRNA) precursor mir-3129 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.61638,-3.60768,1600000,173203,176,82,true,-3.4465,0.71899,2020-11-26T13:28:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03858,mir-1991,1287,mir-1991 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.7,42.1,This family represents the microRNA (miRNA) precursor mir-1991 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.61347,-3.20999,1600000,228224,155,66,true,-2.6713,0.73046,2020-11-26T13:28:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03859,mir-6132,1287,mir-6132 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,112.0,112.6,110.5,This family represents the microRNA (miRNA) precursor mir-6132 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,12,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.70908,-1.81925,1600000,185775,194,94,true,-3.1823,0.71868,2020-11-26T13:29:03.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03860,MIR1520,1287,MIR1520 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.0,52.9,This family represents the microRNA (miRNA) precursor MIR1520 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,18,1779,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.47778,-6.39457,1600000,342908,222,122,true,-2.8322,0.71819,2020-11-26T13:30:53.000Z,2023-11-06T17:21:46.000Z\n",
            "RF03861,mir-4484,1287,mir-4484 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.6,63.3,This family represents the microRNA (miRNA) precursor mir-4484 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.56753,-4.06551,1600000,194827,186,89,true,-3.4834,0.71896,2020-11-26T13:31:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03862,MIR5274,1287,MIR5274 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.4,71.7,This family represents the microRNA (miRNA) precursor MIR5274 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.651,-2.96204,1600000,216917,194,95,true,-3.2696,0.71894,2020-11-26T13:32:02.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03863,mir-1258,1287,mir-1258 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.2,61.1,This family represents the microRNA (miRNA) precursor mir-1258 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.57691,-4.73995,1600000,282228,163,72,true,-3.4451,0.7202,2020-11-26T13:32:28.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03864,mir-BART9,1287,mir-BART9 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,57.2,44.3,This family represents the microRNA (miRNA) precursor mir-BART9 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6149,-3.34129,1600000,193969,184,88,true,-3.2533,0.72258,2020-11-26T13:32:46.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03865,mir-7637,1287,mir-7637 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.9,43.5,This family represents the microRNA (miRNA) precursor mir-7637 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61806,-3.83299,1600000,266011,157,68,true,-3.0303,0.72349,2020-11-26T13:33:02.000Z,2020-12-09T15:48:48.000Z\n",
            "RF03866,mir-1889,1287,mir-1889 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,88.5,59.5,This family represents the microRNA (miRNA) precursor mir-1889 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62257,-4.7448,1600000,192200,237,128,true,-3.017,0.71829,2020-11-26T13:33:19.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03867,mir-4081,1287,mir-4081 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.1,64.0,This family represents the microRNA (miRNA) precursor mir-4081 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60783,-3.53754,1600000,264862,134,51,true,-2.9416,0.74058,2020-11-26T13:33:37.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03868,mir-H2,1287,mir-H2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,92.0,92.0,58.7,This family represents the microRNA (miRNA) precursor mir-H2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.53543,-4.97772,1600000,247330,169,77,true,-2.4543,0.72789,2020-11-26T13:33:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03869,mir-2834,1287,mir-2834 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.0,86.9,This family represents the microRNA (miRNA) precursor mir-2834 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,569,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63913,-3.99921,1600000,240395,151,63,true,-3.2976,0.72383,2020-11-26T13:34:18.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03870,mir-2040,1287,mir-2040 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,51.0,51.8,50.3,This family represents the microRNA (miRNA) precursor mir-2040 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60515,-3.58932,1600000,256046,175,81,true,-3.3689,0.71899,2020-11-26T13:34:35.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03871,mir-5681,1287,mir-5681 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.5,73.3,This family represents the microRNA (miRNA) precursor mir-5681 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.66967,-2.9782,1600000,271120,148,61,true,-3.0945,0.72931,2020-11-26T13:35:14.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03872,MIR6426,1287,MIR6426 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.9,95.8,This family represents the microRNA (miRNA) precursor MIR6426 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60698,-3.98949,1600000,203098,179,84,true,-3.5434,0.719,2020-11-26T13:36:01.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03873,mir-3192,1287,mir-3192 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,87.4,67.7,This family represents the microRNA (miRNA) precursor mir-3192 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.62827,-3.58059,1600000,257688,169,77,true,-3.4803,0.71905,2020-11-26T13:36:20.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03874,mir-8834,1287,mir-8834 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,39.0,39.0,38.9,This family represents the microRNA (miRNA) precursor mir-8834 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3831,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.54884,-4.54368,1600000,189673,210,105,true,-3.0297,0.71861,2020-11-26T13:42:50.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03875,MIR5272,1287,MIR5272 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,64.5,64.5,64.4,This family represents the microRNA (miRNA) precursor MIR5272 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,236,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.48783,-5.56752,1600000,322619,161,74,true,-3.4309,0.71891,2020-11-26T13:43:30.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03876,mir-7132,1287,mir-7132 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,77.6,64.5,This family represents the microRNA (miRNA) precursor mir-7132 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.611,-4.5457,1600000,387277,146,60,true,-2.9811,0.72924,2020-11-26T13:43:53.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03877,mir-3937,1287,mir-3937 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.5,78.7,This family represents the microRNA (miRNA) precursor mir-3937 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.61986,-3.25086,1600000,156516,205,102,true,-2.8514,0.71875,2020-11-26T13:44:15.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03878,mir-B6,1287,mir-B6 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.6,66.1,This family represents the microRNA (miRNA) precursor mir-B6 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.45552,-6.12697,1600000,237840,177,83,true,-2.7464,0.72254,2020-11-26T13:44:33.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03879,mir-2831,1287,mir-2831 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,98.5,88.3,This family represents the microRNA (miRNA) precursor mir-2831 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62278,-3.58336,1600000,228720,166,75,true,-3.5025,0.72159,2020-11-26T13:44:49.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03880,mir-1684,1287,mir-1684 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,61.1,59.7,This family represents the microRNA (miRNA) precursor mir-1684 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61888,-3.79037,1600000,210781,184,87,true,-3.4798,0.71893,2020-11-26T13:45:07.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03881,mir-59,1287,mir-59 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.3,55.2,This family represents the microRNA (miRNA) precursor mir-59 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,5,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.6265,-4.29313,1600000,277292,150,66,true,-3.5102,0.71921,2020-11-26T13:45:26.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03882,mir-4889,1287,mir-4889 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,91.9,59.1,This family represents the microRNA (miRNA) precursor mir-4889 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63523,-2.49537,1600000,170427,175,84,true,-3.657,0.71891,2020-11-26T13:45:48.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03883,mir-9256,1287,mir-9256 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.2,79.1,This family represents the microRNA (miRNA) precursor mir-9256 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,5,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62823,-3.50168,1600000,222086,152,67,true,-3.7633,0.71923,2020-11-26T13:46:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03884,mir-725,1287,mir-725 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.7,66.2,This family represents the microRNA (miRNA) precursor mir-725 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63721,-4.3916,1600000,303552,137,56,true,-3.2498,0.72485,2020-11-26T13:46:44.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03885,mir-4178,1287,mir-4178 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,93.4,57.3,This family represents the microRNA (miRNA) precursor mir-4178 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63078,-3.61098,1600000,293814,147,60,true,-2.9744,0.72797,2020-11-26T13:47:12.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03886,mir-1993,1287,mir-1993 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.5,51.3,This family represents the microRNA (miRNA) precursor mir-1993 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64,-3.27726,1600000,205427,157,68,true,-2.8598,0.72891,2020-11-26T13:47:35.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03887,mir-2511,1287,mir-2511 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.9,91.4,This family represents the microRNA (miRNA) precursor mir-2511 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63217,-3.16944,1600000,226576,181,85,true,-3.7617,0.71885,2020-11-26T13:48:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03888,mir-3160,1287,mir-3160 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,85.8,70.8,This family represents the microRNA (miRNA) precursor mir-3160 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.52177,-5.65559,1600000,253365,177,83,true,-3.1529,0.71915,2020-11-26T13:48:37.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03889,mir-4776,1287,mir-4776 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.8,74.4,This family represents the microRNA (miRNA) precursor mir-4776 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.60025,-3.98981,1600000,226716,173,80,true,-3.2497,0.71982,2020-11-26T13:49:16.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03890,mir-B8,1287,mir-B8 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,98.0,46.2,This family represents the microRNA (miRNA) precursor mir-B8 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.5282,-5.38916,1600000,348213,156,67,true,-3.2166,0.72315,2020-11-26T13:49:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03891,mir-2545,1287,mir-2545 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.2,90.7,This family represents the microRNA (miRNA) precursor mir-2545 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.57933,-4.38551,1600000,206653,227,118,true,-3.2207,0.71861,2020-11-26T13:50:12.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03892,mir-4860,1287,mir-4860 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,104.6,48.0,This family represents the microRNA (miRNA) precursor mir-4860 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63055,-3.73768,1600000,234903,165,74,true,-3.7498,0.71953,2020-11-26T13:50:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03893,mir-4879,1287,mir-4879 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,105.3,46.2,This family represents the microRNA (miRNA) precursor mir-4879 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.66383,-2.9524,1600000,197925,167,75,true,-3.5161,0.7202,2020-11-26T13:51:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03894,mir-5736,1287,mir-5736 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,96.2,58.3,This family represents the microRNA (miRNA) precursor mir-5736 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63765,-2.04968,1600000,182622,173,80,true,-3.256,0.72146,2020-11-26T13:51:55.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03895,MIR2679,1287,MIR2679 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.5,64.7,This family represents the microRNA (miRNA) precursor MIR2679 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,25,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62095,-3.50555,1600000,200379,228,119,true,-3.1889,0.71843,2020-11-26T13:52:26.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03896,MIR2275,1287,MIR2275 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.5,60.5,60.4,This family represents the microRNA (miRNA) precursor MIR2275 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,13,297,0,0,Gene; miRNA;,Predicted; RNAalifold,40,0,,\"\",0.58361,-3.47111,1600000,146014,209,98,true,-2.8818,0.71865,2020-11-26T13:53:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03897,mir-4077,1287,mir-4077 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.7,43.9,This family represents the microRNA (miRNA) precursor mir-4077 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57426,-4.15742,1600000,275628,137,56,true,-3.532,0.71999,2020-11-26T13:54:07.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03898,mir-7616,1287,mir-7616 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.1,40.5,This family represents the microRNA (miRNA) precursor mir-7616 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62099,-3.20797,1600000,222658,156,67,true,-3.4242,0.72253,2020-11-26T13:54:37.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03899,mir-5912,1287,mir-5912 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.4,38.3,This family represents the microRNA (miRNA) precursor mir-5912 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.68014,-3.08028,1600000,173529,184,88,true,-3.4191,0.71875,2020-11-26T13:55:01.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03900,mir-9242,1287,mir-9242 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.0,61.9,This family represents the microRNA (miRNA) precursor mir-9242 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,568,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.55812,-4.94558,1600000,330328,154,66,true,-2.8956,0.72696,2020-11-26T13:55:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03901,mir-BART18,1287,mir-BART18 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.0,49.7,This family represents the microRNA (miRNA) precursor mir-BART18 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61984,-3.33392,1600000,239119,177,83,true,-2.9885,0.72189,2020-11-26T13:55:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03902,MIR7784,1287,MIR7784 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,130.0,130.8,127.7,This family represents the microRNA (miRNA) precursor MIR7784 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61024,-3.80463,1600000,218366,234,123,true,-3.2884,0.71851,2020-11-26T13:56:09.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03903,mir-679,1287,mir-679 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,83.2,47.8,This family represents the microRNA (miRNA) precursor mir-679 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.67105,-3.02832,1600000,264812,165,74,true,-3.7774,0.72077,2020-11-26T13:56:40.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03904,MIR7991,1287,MIR7991 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.9,57.7,This family represents the microRNA (miRNA) precursor MIR7991 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,42,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65591,-3.62801,1600000,311711,129,47,true,-2.9204,0.73496,2020-11-26T13:57:00.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03905,MIR6149,1287,MIR6149 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,106.4,72.7,This family represents the microRNA (miRNA) precursor MIR6149 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6235,-2.82119,1600000,185642,199,99,true,-3.1814,0.71861,2020-11-26T13:57:20.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03906,mir-3199,1287,mir-3199 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.7,96.8,This family represents the microRNA (miRNA) precursor mir-3199 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.56176,-4.25104,1600000,224329,183,87,true,-2.9822,0.71911,2020-11-26T13:57:39.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03907,mir-4428,1287,mir-4428 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,91.2,89.8,This family represents the microRNA (miRNA) precursor mir-4428 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.56957,-4.60057,1600000,240281,164,73,true,-3.3214,0.7195,2020-11-26T13:58:15.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03908,MIR6459,1287,MIR6459 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.1,68.8,This family represents the microRNA (miRNA) precursor MIR6459 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6864,-2.29681,1600000,179304,182,87,true,-2.9875,0.72258,2020-11-26T13:59:35.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03909,mir-3804,1287,mir-3804 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,98.9,44.8,This family represents the microRNA (miRNA) precursor mir-3804 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62742,-3.04736,1600000,170464,176,82,true,-3.1901,0.72382,2020-11-26T13:59:51.000Z,2020-12-09T15:48:49.000Z\n",
            "RF03910,mir-H5,1287,mir-H5 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,94.3,46.9,This family represents the microRNA (miRNA) precursor mir-H5 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.46241,-6.37468,1600000,226319,173,80,true,-2.6804,0.72414,2020-11-26T14:00:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03911,mir-1910,1287,mir-1910 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,79.1,62.3,This family represents the microRNA (miRNA) precursor mir-1910 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66061,-3.27419,1600000,202492,174,80,true,-3.5515,0.71877,2020-11-26T14:00:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03912,mir-4659,1287,mir-4659 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,92.0,92.1,63.6,This family represents the microRNA (miRNA) precursor mir-4659 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,18,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.65071,-2.65946,1600000,221544,164,73,true,-2.9546,0.72479,2020-11-26T14:00:56.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03913,mir-992,1287,mir-992 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.3,56.5,This family represents the microRNA (miRNA) precursor mir-992 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63525,-3.13055,1600000,242724,180,86,true,-3.329,0.72124,2020-11-26T14:01:13.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03914,mir-BART21,1287,mir-BART21 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,77.8,49.4,This family represents the microRNA (miRNA) precursor mir-BART21 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.614,-4.1116,1600000,272932,168,76,true,-2.8958,0.72481,2020-11-26T14:01:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03915,MIR5286,1287,MIR5286 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.2,92.9,This family represents the microRNA (miRNA) precursor MIR5286 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,191,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64712,-3.48363,1600000,221182,171,78,true,-3.048,0.71946,2020-11-26T14:02:05.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03916,MIR400,1287,MIR400 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.9,73.0,This family represents the microRNA (miRNA) precursor MIR400 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.65136,-2.45448,1600000,157175,206,103,true,-3.298,0.71866,2020-11-26T14:02:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03917,mir-4449,1287,mir-4449 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.0,70.4,This family represents the microRNA (miRNA) precursor mir-4449 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.52463,-4.96434,1600000,237501,158,69,true,-2.7549,0.72241,2020-11-26T14:02:47.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03918,mir-4922,1287,mir-4922 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.4,47.8,This family represents the microRNA (miRNA) precursor mir-4922 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.56684,-4.29655,1600000,261757,153,65,true,-2.6395,0.72694,2020-11-26T14:03:15.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03919,mir-1540,1287,mir-1540 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,94.1,44.8,This family represents the microRNA (miRNA) precursor mir-1540 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64703,-3.78836,1600000,279195,145,59,true,-3.4313,0.72612,2020-11-26T14:03:37.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03920,mir-237,1287,mir-237 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.9,70.1,This family represents the microRNA (miRNA) precursor mir-237 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.66892,-2.95233,1600000,175995,167,79,true,-3.5219,0.71899,2020-11-26T14:04:24.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03921,MIR860,1287,MIR860 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.0,81.6,This family represents the microRNA (miRNA) precursor MIR860 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,11,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.62162,-3.74523,1600000,211831,221,115,true,-3.5864,0.71859,2020-11-26T14:04:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03922,MIR2119,1287,MIR2119 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.0,70.7,This family represents the microRNA (miRNA) precursor MIR2119 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,15,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.52171,-5.31885,1600000,246412,263,148,true,-2.9326,0.71801,2020-11-26T14:05:13.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03923,mir-964,1287,mir-964 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,114.2,62.1,This family represents the microRNA (miRNA) precursor mir-964 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6325,-2.99919,1600000,213674,188,90,true,-3.3524,0.71873,2020-11-26T14:05:32.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03924,mir-1228,1287,mir-1228 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.4,44.2,This family represents the microRNA (miRNA) precursor mir-1228 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.53199,-5.35904,1600000,257685,164,73,true,-3.2129,0.72026,2020-11-26T14:05:58.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03925,mir-1011,1287,mir-1011 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.0,58.6,This family represents the microRNA (miRNA) precursor mir-1011 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.60003,-4.59414,1600000,315862,138,54,true,-2.9539,0.72961,2020-11-26T14:06:16.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03926,MIR1435,1287,MIR1435 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.0,93.9,This family represents the microRNA (miRNA) precursor MIR1435 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,1062,0,0,Gene; miRNA;,Predicted; RNAalifold,20,0,,\"\",0.47855,-6.54351,1600000,244886,422,186,true,-3.1179,0.71376,2020-11-26T14:12:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03927,MIR538,1287,MIR538 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,122.4,82.3,This family represents the microRNA (miRNA) precursor MIR538 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62595,-4.42103,1600000,226475,292,160,true,-3.5894,0.7173,2020-11-26T14:12:29.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03929,mir-6579,1287,mir-6579 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,47.0,47.2,46.0,This family represents the microRNA (miRNA) precursor mir-6579 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58005,-4.70473,1600000,302060,157,68,true,-3.3257,0.72142,2020-11-26T14:13:15.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03930,mir-1909,1287,mir-1909 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,84.9,77.1,This family represents the microRNA (miRNA) precursor mir-1909 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.65412,-3.4094,1600000,270831,163,72,true,-3.2024,0.72062,2020-11-26T14:13:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03931,mir-2061,1287,mir-2061 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,104.4,36.9,This family represents the microRNA (miRNA) precursor mir-2061 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59903,-3.85666,1600000,197023,165,74,true,-3.3063,0.7201,2020-11-26T14:13:48.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03932,mir-1597,1287,mir-1597 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.0,92.8,This family represents the microRNA (miRNA) precursor mir-1597 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,40,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.62952,-2.63075,1600000,241953,164,73,true,-3.2204,0.72227,2020-11-26T14:14:13.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03933,MIR6445,1287,MIR6445 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.2,69.1,This family represents the microRNA (miRNA) precursor MIR6445 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61621,-3.85385,1600000,222548,223,117,true,-3.0596,0.71856,2020-11-26T14:14:31.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03934,mir-m107-1,1287,mir-m107-1 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.4,48.2,This family represents the microRNA (miRNA) precursor mir-m107-1 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.57118,-5.32579,1600000,307082,153,65,true,-2.6402,0.73457,2020-11-26T14:14:46.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03935,mir-763,1287,mir-763 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,112.0,98.3,This family represents the microRNA (miRNA) precursor mir-763 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,36,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.65313,-2.83079,1600000,192150,230,120,true,-2.8815,0.71851,2020-11-26T14:15:28.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03936,mir-4079,1287,mir-4079 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.4,51.7,This family represents the microRNA (miRNA) precursor mir-4079 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.69429,-2.42871,1600000,273846,134,51,true,-2.6066,0.7408,2020-11-26T14:15:46.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03937,mir-9456,1287,mir-9456 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.9,76.3,This family represents the microRNA (miRNA) precursor mir-9456 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60052,-4.50631,1600000,269882,157,68,true,-3.3272,0.72368,2020-11-26T14:16:03.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03938,mir-4662,1287,mir-4662 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.2,93.4,This family represents the microRNA (miRNA) precursor mir-4662 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.64541,-2.93096,1600000,265596,156,67,true,-2.9878,0.72599,2020-11-26T14:16:27.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03939,mir-4868,1287,mir-4868 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,72.7,54.3,This family represents the microRNA (miRNA) precursor mir-4868 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62831,-3.77262,1600000,179073,163,78,true,-3.8693,0.71893,2020-11-26T14:16:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03940,MIR9563,1287,MIR9563 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.0,89.9,This family represents the microRNA (miRNA) precursor MIR9563 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3495,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.60968,-3.86989,1600000,208709,234,125,true,-2.5145,0.71841,2020-11-26T14:19:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03941,mir-3158,1287,mir-3158 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.2,70.8,This family represents the microRNA (miRNA) precursor mir-3158 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.56545,-4.78102,1600000,231930,175,81,true,-3.3248,0.72009,2020-11-26T14:19:50.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03942,mir-5727,1287,mir-5727 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.3,90.8,This family represents the microRNA (miRNA) precursor mir-5727 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62345,-2.67842,1600000,194884,189,92,true,-3.242,0.72099,2020-11-26T14:20:06.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03943,mir-1824,1287,mir-1824 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.9,40.4,This family represents the microRNA (miRNA) precursor mir-1824 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65906,-3.38892,1600000,270167,151,63,true,-3.4577,0.72454,2020-11-26T14:20:23.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03944,mir-935,1287,mir-935 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.6,58.9,This family represents the microRNA (miRNA) precursor mir-935 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,21,0,0,Gene; miRNA;,Predicted; RNAalifold,21,0,,\"\",0.48328,-5.22356,1600000,200971,189,91,true,-3.0922,0.71887,2020-11-26T14:20:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03945,mir-4876,1287,mir-4876 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,112.5,49.1,This family represents the microRNA (miRNA) precursor mir-4876 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.56856,-3.86187,1600000,159652,190,92,true,-3.1176,0.71887,2020-11-26T14:21:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03946,mir-3132,1287,mir-3132 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.9,52.6,This family represents the microRNA (miRNA) precursor mir-3132 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.55033,-4.93045,1600000,267611,167,75,true,-3.5386,0.71929,2020-11-26T14:21:55.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03947,mir-658,1287,mir-658 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.5,81.9,This family represents the microRNA (miRNA) precursor mir-658 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,14,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.52364,-4.73485,1600000,226594,201,100,true,-2.8437,0.71882,2020-11-26T14:22:21.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03948,mir-5045,1287,mir-5045 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.1,47.6,This family represents the microRNA (miRNA) precursor mir-5045 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.63653,-4.09044,1600000,318926,142,57,true,-2.4823,0.73158,2020-11-26T14:22:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03949,mir-8830,1287,mir-8830 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.8,67.0,This family represents the microRNA (miRNA) precursor mir-8830 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.54655,-4.07752,1600000,162481,194,95,true,-3.2305,0.71883,2020-11-26T14:26:49.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03950,mir-4928,1287,mir-4928 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,107.9,57.3,This family represents the microRNA (miRNA) precursor mir-4928 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59832,-3.82835,1600000,209860,172,79,true,-3.4633,0.71905,2020-11-26T14:27:07.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03951,MIR7725,1287,MIR7725 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,77.0,77.7,76.2,This family represents the microRNA (miRNA) precursor MIR7725 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.55752,-5.35159,1600000,215877,260,142,true,-3.2708,0.71825,2020-11-26T14:27:25.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03952,mir-3718,1287,mir-3718 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.3,68.0,This family represents the microRNA (miRNA) precursor mir-3718 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.69598,-2.14562,1600000,202745,189,92,true,-3.3808,0.72166,2020-11-26T14:27:42.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03953,mir-5350,1287,mir-5350 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,91.8,69.7,This family represents the microRNA (miRNA) precursor mir-5350 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.62354,-3.82431,1600000,239865,165,74,true,-3.0819,0.72548,2020-11-26T14:27:59.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03954,mir-8993,1287,mir-8993 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.1,64.9,This family represents the microRNA (miRNA) precursor mir-8993 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.615,-3.57006,1600000,190790,195,95,true,-2.9602,0.71868,2020-11-26T14:28:19.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03955,MIR2934,1287,MIR2934 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,123.5,77.9,This family represents the microRNA (miRNA) precursor MIR2934 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58571,-3.98004,1600000,203718,216,113,true,-2.8633,0.71853,2020-11-26T14:28:35.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03956,mir-737,1287,mir-737 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.3,56.1,This family represents the microRNA (miRNA) precursor mir-737 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,37,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.54037,-4.16222,1600000,270137,145,63,true,-3.3637,0.71933,2020-11-26T14:29:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03957,mir-5856,1287,mir-5856 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,91.7,67.5,This family represents the microRNA (miRNA) precursor mir-5856 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65167,-3.26173,1600000,211554,160,70,true,-3.1177,0.72573,2020-11-26T14:29:25.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03958,mir-959,1287,mir-959 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.1,58.7,This family represents the microRNA (miRNA) precursor mir-959 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,11,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.61487,-2.68108,1600000,203075,188,91,true,-3.5365,0.71873,2020-11-26T14:29:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03959,mir-303,1287,mir-303 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,77.0,77.6,53.4,This family represents the microRNA (miRNA) precursor mir-303 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59569,-2.7749,1600000,225303,160,70,true,-2.8766,0.72758,2020-11-26T14:30:14.000Z,2020-12-09T15:48:50.000Z\n",
            "RF03960,mir-6644,1287,mir-6644 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.4,52.2,This family represents the microRNA (miRNA) precursor mir-6644 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66628,-3.04916,1600000,247528,156,67,true,-3.1463,0.72297,2020-11-26T14:30:30.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03961,mir-9033,1287,mir-9033 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,106.1,83.9,This family represents the microRNA (miRNA) precursor mir-9033 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63321,-3.47583,1600000,198495,163,72,true,-3.2118,0.71972,2020-11-26T14:30:47.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03962,mir-958,1287,mir-958 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.0,53.2,This family represents the microRNA (miRNA) precursor mir-958 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,10,0,0,Gene; miRNA;,Predicted; RNAalifold,11,0,,\"\",0.68311,-2.32694,1600000,179226,189,92,true,-3.5707,0.71881,2020-11-26T14:31:07.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03963,mir-2996,1287,mir-2996 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,107.0,74.9,This family represents the microRNA (miRNA) precursor mir-2996 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63971,-4.10025,1600000,245262,163,72,true,-3.1845,0.72057,2020-11-26T14:31:26.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03964,mir-H1,1287,mir-H1 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,97.0,97.1,60.2,This family represents the microRNA (miRNA) precursor mir-H1 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.51859,-5.35384,1600000,197963,180,85,true,-3.0249,0.72687,2020-11-26T14:32:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03965,mir-8927,1287,mir-8927 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,160.0,160.2,120.0,This family represents the microRNA (miRNA) precursor mir-8927 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60898,-5.28156,1600000,249149,264,145,true,-2.9707,0.71838,2020-11-26T14:32:38.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03966,mir-76,1287,mir-76 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.7,68.7,This family represents the microRNA (miRNA) precursor mir-76 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.63901,-3.0921,1600000,164362,174,80,true,-3.8426,0.71906,2020-11-26T14:32:59.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03967,mir-1270,1287,mir-1270 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.1,88.8,This family represents the microRNA (miRNA) precursor mir-1270 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,25,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.66028,-3.05162,1600000,215977,177,83,true,-3.6539,0.71891,2020-11-26T14:33:38.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03968,mir-8934,1287,mir-8934 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,110.5,48.8,This family represents the microRNA (miRNA) precursor mir-8934 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.49947,-5.22236,1600000,202540,186,88,true,-3.1779,0.71888,2020-11-26T14:33:55.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03969,mir-622,1287,mir-622 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,86.0,85.8,This family represents the microRNA (miRNA) precursor mir-622 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,157,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.67305,-3.14589,1600000,217565,162,72,true,-3.7643,0.71912,2020-11-26T14:37:48.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03970,mir-2766,1287,mir-2766 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,61.7,57.5,This family represents the microRNA (miRNA) precursor mir-2766 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.67594,-3.43689,1600000,269098,157,68,true,-3.177,0.7252,2020-11-26T14:38:12.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03971,mir-3572,1287,mir-3572 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,108.7,79.1,This family represents the microRNA (miRNA) precursor mir-3572 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65695,-3.22435,1600000,222536,182,86,true,-3.1392,0.71897,2020-11-26T14:38:30.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03972,mir-3150,1287,mir-3150 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.4,81.1,This family represents the microRNA (miRNA) precursor mir-3150 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.55856,-5.05174,1600000,231178,173,80,true,-3.0936,0.72257,2020-11-26T14:38:51.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03973,mir-5615,1287,mir-5615 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.9,62.7,This family represents the microRNA (miRNA) precursor mir-5615 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61345,-4.2423,1600000,264455,148,61,true,-2.8123,0.72629,2020-11-26T14:39:08.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03974,MIR838,1287,MIR838 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,190.0,196.8,183.9,This family represents the microRNA (miRNA) precursor MIR838 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.56257,-6.1513,1600000,214299,295,170,true,-3.4079,0.7159,2020-11-26T14:42:25.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03975,mir-5881,1287,mir-5881 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.9,73.6,This family represents the microRNA (miRNA) precursor mir-5881 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.74837,-1.37049,1600000,187800,156,67,true,-3.0716,0.72785,2020-11-26T14:42:53.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03976,MIR898,1287,MIR898 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.9,73.9,This family represents the microRNA (miRNA) precursor MIR898 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61684,-4.86838,1600000,219533,273,154,true,-3.0759,0.71823,2020-11-26T14:43:11.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03977,mir-976,1287,mir-976 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,112.4,55.5,This family represents the microRNA (miRNA) precursor mir-976 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58085,-3.96445,1600000,229375,181,85,true,-3.5403,0.71882,2020-11-26T14:43:33.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03978,mir-1269,1287,mir-1269 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,110.0,98.5,This family represents the microRNA (miRNA) precursor mir-1269 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.51784,-5.76743,1600000,236752,188,90,true,-2.8687,0.71884,2020-11-26T14:44:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03979,mir-2006,1287,mir-2006 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.0,55.8,This family represents the microRNA (miRNA) precursor mir-2006 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,1,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65743,-2.92059,1600000,189182,187,91,true,-3.3535,0.71878,2020-11-26T14:44:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03980,mir-2012,1287,mir-2012 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.5,70.1,This family represents the microRNA (miRNA) precursor mir-2012 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60785,-3.80621,1600000,210801,178,84,true,-3.182,0.72408,2020-11-26T14:44:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03981,mir-8799,1287,mir-8799 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,46.2,46.2,46.1,This family represents the microRNA (miRNA) precursor mir-8799 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,4760,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.52472,-4.02381,1600000,185310,185,90,true,-3.1005,0.7189,2020-11-26T14:53:18.000Z,2023-11-06T16:44:06.000Z\n",
            "RF03982,mir-4865,1287,mir-4865 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,110.2,46.8,This family represents the microRNA (miRNA) precursor mir-4865 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60775,-3.80976,1600000,216283,179,84,true,-3.3528,0.71891,2020-11-26T14:53:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03983,MIR5284,1287,MIR5284 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.6,92.4,This family represents the microRNA (miRNA) precursor MIR5284 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,143,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63677,-2.92993,1600000,142771,215,111,true,-3.3181,0.71869,2020-11-26T14:54:17.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03984,MIR2933,1287,MIR2933 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,78.8,62.3,This family represents the microRNA (miRNA) precursor MIR2933 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,15,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65677,-3.26,1600000,228193,219,112,true,-3.1883,0.71844,2020-11-26T14:54:41.000Z,2020-12-09T15:48:51.000Z\n",
            "RF03985,mir-9250,1287,mir-9250 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,95.8,74.7,This family represents the microRNA (miRNA) precursor mir-9250 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.56793,-4.36257,1600000,267671,151,63,true,-3.0783,0.72437,2020-12-04T11:17:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03986,mir-6095,1287,mir-6095 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,106.7,62.6,This family represents the microRNA (miRNA) precursor mir-6095 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61367,-2.28734,1600000,214554,177,83,true,-3.4816,0.71884,2020-12-04T11:18:02.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03987,MIR6136,1287,MIR6136 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.1,80.8,This family represents the microRNA (miRNA) precursor MIR6136 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.56374,-4.31126,1600000,180581,207,105,true,-3.0648,0.71885,2020-12-04T11:18:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03988,mir-6301,1287,mir-6301 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,99.1,55.5,This family represents the microRNA (miRNA) precursor mir-6301 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.70043,-2.33694,1600000,242565,170,78,true,-3.5561,0.72174,2020-12-04T11:18:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03989,mir-7950,1287,mir-7950 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.0,40.9,This family represents the microRNA (miRNA) precursor mir-7950 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63591,-3.68361,1600000,280360,145,59,true,-3.1608,0.72977,2020-12-04T11:19:28.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03990,mir-4888,1287,mir-4888 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,99.0,47.3,This family represents the microRNA (miRNA) precursor mir-4888 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.57235,-4.40677,1600000,250027,164,73,true,-3.5185,0.722,2020-12-04T11:19:53.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03991,mir-9209,1287,mir-9209 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.3,78.8,This family represents the microRNA (miRNA) precursor mir-9209 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64585,-3.75997,1600000,265312,161,72,true,-3.3885,0.71914,2020-12-04T11:20:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03992,mir-6302,1287,mir-6302 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.0,54.3,This family represents the microRNA (miRNA) precursor mir-6302 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66563,-2.93251,1600000,220491,172,79,true,-3.2678,0.72499,2020-12-04T11:20:48.000Z,2022-10-29T17:42:07.000Z\n",
            "RF03993,mir-8536,1287,mir-8536 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.3,63.3,This family represents the microRNA (miRNA) precursor mir-8536 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67225,-1.73698,1600000,134160,181,86,true,-3.0856,0.72434,2020-12-04T11:21:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03994,mir-7558,1287,mir-7558 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.8,73.6,This family represents the microRNA (miRNA) precursor mir-7558 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60583,-4.635,1600000,347160,134,51,true,-2.6567,0.73423,2020-12-04T11:21:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03995,mir-7957,1287,mir-7957 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.6,46.2,This family represents the microRNA (miRNA) precursor mir-7957 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61623,-3.46566,1600000,286698,140,59,true,-3.6708,0.71928,2020-12-04T11:22:13.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03996,mir-3012,1287,mir-3012 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,92.0,59.7,This family represents the microRNA (miRNA) precursor mir-3012 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61089,-3.3288,1600000,291197,157,68,true,-3.3087,0.72455,2020-12-04T11:22:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03997,mir-8510,1287,mir-8510 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,92.5,44.2,This family represents the microRNA (miRNA) precursor mir-8510 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64951,-3.33937,1600000,221738,155,67,true,-3.3345,0.71913,2020-12-04T11:23:14.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03998,mir-2160,1287,mir-2160 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,119.0,53.2,This family represents the microRNA (miRNA) precursor mir-2160 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59762,-1.96225,1600000,189755,192,93,true,-3.215,0.71863,2020-12-04T11:23:38.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03999,mir-7883,1287,mir-7883 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,50.0,52.9,49.4,This family represents the microRNA (miRNA) precursor mir-7883 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61063,-3.70233,1600000,284984,143,57,true,-3.3783,0.72793,2020-12-04T11:24:02.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04000,MIR8139,1287,MIR8139 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.0,37.6,This family represents the microRNA (miRNA) precursor MIR8139 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66993,-3.69034,1600000,317320,141,58,true,-3.7377,0.71941,2020-12-04T11:24:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04001,mir-7918,1287,mir-7918 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.7,41.3,This family represents the microRNA (miRNA) precursor mir-7918 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65973,-3.81712,1600000,329808,138,58,true,-3.5323,0.71933,2020-12-04T11:25:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04002,MIR6143,1287,MIR6143 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.5,69.1,This family represents the microRNA (miRNA) precursor MIR6143 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61053,-3.39202,1600000,204739,185,89,true,-3.3322,0.72156,2020-12-04T11:25:42.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04003,mir-4841,1287,mir-4841 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,109.6,58.1,This family represents the microRNA (miRNA) precursor mir-4841 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63464,-2.38316,1600000,172945,181,86,true,-3.3751,0.71945,2020-12-04T11:26:05.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04004,mir-7940,1287,mir-7940 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.2,36.4,This family represents the microRNA (miRNA) precursor mir-7940 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64627,-3.40817,1600000,279141,142,57,true,-2.8015,0.7344,2020-12-04T11:26:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04005,mir-4599,1287,mir-4599 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.1,44.4,This family represents the microRNA (miRNA) precursor mir-4599 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.7227,-2.16138,1600000,164709,176,82,true,-3.0695,0.72253,2020-12-04T11:26:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04006,mir-9338,1287,mir-9338 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.5,92.3,This family represents the microRNA (miRNA) precursor mir-9338 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67537,-3.02539,1600000,205790,154,67,true,-3.6571,0.71919,2020-12-04T11:27:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04007,mir-8489,1287,mir-8489 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.8,71.6,This family represents the microRNA (miRNA) precursor mir-8489 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64764,-2.50737,1600000,164758,186,90,true,-3.2202,0.72061,2020-12-04T11:27:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04008,mir-4880,1287,mir-4880 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.0,48.7,This family represents the microRNA (miRNA) precursor mir-4880 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63573,-3.78197,1600000,265816,166,75,true,-3.2929,0.72108,2020-12-04T11:28:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04009,mir-754,1287,mir-754 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.8,56.6,This family represents the microRNA (miRNA) precursor mir-754 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62109,-3.15601,1600000,235755,170,78,true,-3.6623,0.71905,2020-12-04T11:28:39.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04010,mir-3003,1287,mir-3003 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.9,61.3,This family represents the microRNA (miRNA) precursor mir-3003 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58989,-2.73345,1600000,191039,168,76,true,-2.9712,0.72149,2020-12-04T11:29:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04011,mir-8515,1287,mir-8515 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,106.0,73.3,This family represents the microRNA (miRNA) precursor mir-8515 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.5735,-4.41356,1600000,154492,189,92,true,-3.3696,0.71888,2020-12-04T11:29:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04012,MIR7532,1287,MIR7532 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,123.5,45.8,This family represents the microRNA (miRNA) precursor MIR7532 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.55487,-4.30449,1600000,210764,211,106,true,-3.2485,0.71866,2020-12-04T11:29:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04013,mir-4818,1287,mir-4818 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,92.0,92.5,53.7,This family represents the microRNA (miRNA) precursor mir-4818 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66877,-2.60055,1600000,195465,185,89,true,-3.4147,0.72497,2020-12-04T11:30:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04014,mir-8338,1287,mir-8338 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.1,52.3,This family represents the microRNA (miRNA) precursor mir-8338 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63971,-4.11586,1600000,283598,148,61,true,-2.8169,0.73121,2020-12-04T11:30:35.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04015,MIR946,1287,MIR946 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,190.0,193.7,154.1,This family represents the microRNA (miRNA) precursor MIR946 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57341,-7.78576,1600000,249314,316,189,true,-3.5701,0.71348,2020-12-04T11:30:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04016,mir-4873,1287,mir-4873 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.8,47.9,This family represents the microRNA (miRNA) precursor mir-4873 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59115,-4.12381,1600000,238430,171,78,true,-3.4762,0.71951,2020-12-04T11:31:21.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04017,mir-4864,1287,mir-4864 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,107.4,38.7,This family represents the microRNA (miRNA) precursor mir-4864 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6061,-4.13573,1600000,214946,175,81,true,-3.4026,0.71892,2020-12-04T11:31:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04018,mir-5316,1287,mir-5316 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,103.2,39.9,This family represents the microRNA (miRNA) precursor mir-5316 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.69045,-2.49404,1600000,196577,185,89,true,-3.2667,0.71882,2020-12-04T11:32:05.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04019,MIR950,1287,MIR950 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,97.5,89.8,This family represents the microRNA (miRNA) precursor MIR950 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60612,-3.74248,1600000,237694,190,92,true,-3.1356,0.71886,2020-12-04T11:32:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04020,mir-4859,1287,mir-4859 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,109.2,47.0,This family represents the microRNA (miRNA) precursor mir-4859 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60377,-3.54186,1600000,193252,174,80,true,-3.3987,0.71889,2020-12-04T11:32:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04021,mir-3492,1287,mir-3492 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,44.0,44.0,43.9,This family represents the microRNA (miRNA) precursor mir-3492 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,14778,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.64026,-3.48689,1600000,226783,172,79,true,-3.3846,0.72267,2020-12-04T11:37:00.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04022,mir-4863,1287,mir-4863 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,118.4,46.8,This family represents the microRNA (miRNA) precursor mir-4863 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61884,-3.52291,1600000,175936,192,93,true,-3.6119,0.71888,2020-12-04T11:37:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04023,MIR1314,1287,MIR1314 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,108.0,108.5,90.2,This family represents the microRNA (miRNA) precursor MIR1314 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6616,-2.85746,1600000,213849,206,104,true,-3.1442,0.71862,2020-12-04T11:37:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04024,mir-9571,1287,mir-9571 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,108.6,47.8,This family represents the microRNA (miRNA) precursor mir-9571 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60617,-3.67559,1600000,197062,182,86,true,-3.0927,0.71894,2020-12-04T11:38:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04025,mir-7964,1287,mir-7964 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,83.7,36.9,This family represents the microRNA (miRNA) precursor mir-7964 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62714,-4.35949,1600000,334147,134,51,true,-3.0953,0.7331,2020-12-04T11:38:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04026,mir-8499,1287,mir-8499 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,105.9,56.9,This family represents the microRNA (miRNA) precursor mir-8499 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61779,-3.71318,1600000,208189,169,77,true,-3.1093,0.71974,2020-12-04T11:38:53.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04027,mir-1422,1287,mir-1422 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.7,54.4,This family represents the microRNA (miRNA) precursor mir-1422 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,18,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.56998,-3.76576,1600000,187399,165,84,true,-3.4861,0.71889,2020-12-04T11:39:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04028,mir-7948,1287,mir-7948 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.4,38.6,This family represents the microRNA (miRNA) precursor mir-7948 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63916,-3.67196,1600000,232804,147,63,true,-3.9562,0.71927,2020-12-04T11:40:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04029,mir-8521,1287,mir-8521 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.8,45.8,This family represents the microRNA (miRNA) precursor mir-8521 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64812,-3.45673,1600000,194893,155,68,true,-3.4519,0.71926,2020-12-04T11:40:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04030,mir-6096,1287,mir-6096 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,111.7,51.7,This family represents the microRNA (miRNA) precursor mir-6096 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64616,-3.18084,1600000,199406,181,85,true,-3.1824,0.71897,2020-12-04T11:41:29.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04031,mir-2686,1287,mir-2686 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.3,40.9,This family represents the microRNA (miRNA) precursor mir-2686 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63983,-3.47541,1600000,255424,157,68,true,-3.319,0.73001,2020-12-04T11:42:00.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04032,MIR2912,1287,MIR2912 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.3,53.2,This family represents the microRNA (miRNA) precursor MIR2912 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3999,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58336,-6.36193,1600000,245995,277,156,true,-3.243,0.71779,2020-12-04T11:43:24.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04033,mir-8512,1287,mir-8512 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,95.9,37.3,This family represents the microRNA (miRNA) precursor mir-8512 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67931,-3.30973,1600000,264495,148,61,true,-3.3885,0.72547,2020-12-04T11:43:50.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04035,mir-7910,1287,mir-7910 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.6,50.6,This family represents the microRNA (miRNA) precursor mir-7910 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58703,-3.77441,1600000,322414,146,60,true,-2.9528,0.72992,2020-12-04T11:44:26.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04036,mir-2076,1287,mir-2076 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.9,56.6,This family represents the microRNA (miRNA) precursor mir-2076 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64081,-3.59136,1600000,271794,157,68,true,-3.4769,0.72341,2020-12-04T11:44:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04037,mir-7911,1287,mir-7911 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,81.5,38.3,This family represents the microRNA (miRNA) precursor mir-7911 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.56796,-5.28826,1600000,275572,146,63,true,-3.4813,0.72192,2020-12-04T11:45:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04038,mir-5985,1287,mir-5985 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.9,95.2,This family represents the microRNA (miRNA) precursor mir-5985 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60964,-3.62462,1600000,240113,165,74,true,-3.4575,0.72118,2020-12-04T11:45:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04039,mir-2242,1287,mir-2242 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,112.4,57.5,This family represents the microRNA (miRNA) precursor mir-2242 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.73468,-1.12144,1600000,167593,194,95,true,-3.849,0.71881,2020-12-04T11:45:58.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04040,mir-4866,1287,mir-4866 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,109.5,57.1,This family represents the microRNA (miRNA) precursor mir-4866 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63381,-3.24853,1600000,207453,177,82,true,-3.2738,0.71903,2020-12-04T11:46:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04041,MIR6154,1287,MIR6154 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,103.2,44.8,This family represents the microRNA (miRNA) precursor MIR6154 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67776,-2.61555,1600000,209018,169,77,true,-3.5374,0.71908,2020-12-04T11:46:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04042,mir-7961,1287,mir-7961 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,91.2,47.3,This family represents the microRNA (miRNA) precursor mir-7961 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66723,-2.61081,1600000,273639,141,56,true,-3.347,0.7288,2020-12-04T11:47:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04043,mir-2062,1287,mir-2062 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,97.9,48.4,This family represents the microRNA (miRNA) precursor mir-2062 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6533,-3.6665,1600000,299171,163,72,true,-3.0714,0.72178,2020-12-04T11:47:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04044,mir-7939,1287,mir-7939 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.8,42.8,This family represents the microRNA (miRNA) precursor mir-7939 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,10,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63303,-4.44107,1600000,294989,142,61,true,-3.7274,0.71924,2020-12-04T11:48:00.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04045,mir-2147,1287,mir-2147 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,88.7,44.2,This family represents the microRNA (miRNA) precursor mir-2147 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67133,-2.19876,1600000,205813,175,85,true,-3.4048,0.7189,2020-12-04T11:48:30.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04046,mir-2154,1287,mir-2154 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,115.4,50.7,This family represents the microRNA (miRNA) precursor mir-2154 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6208,-1.947,1600000,186812,198,97,true,-3.4415,0.71874,2020-12-04T11:48:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04047,mir-9228,1287,mir-9228 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,112.4,76.7,This family represents the microRNA (miRNA) precursor mir-9228 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6704,-2.85926,1600000,151523,181,85,true,-3.5825,0.71891,2020-12-04T11:49:29.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04048,mir-7927,1287,mir-7927 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.0,41.4,This family represents the microRNA (miRNA) precursor mir-7927 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62101,-4.26059,1600000,251087,135,56,true,-3.6684,0.71942,2020-12-04T11:50:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04049,MIR7127,1287,MIR7127 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,164.9,77.2,This family represents the microRNA (miRNA) precursor MIR7127 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58084,-4.93862,1600000,221780,246,132,true,-3.3842,0.71828,2020-12-04T11:50:37.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04050,mir-2072,1287,mir-2072 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,101.2,48.6,This family represents the microRNA (miRNA) precursor mir-2072 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.66698,-2.67103,1600000,245929,156,67,true,-3.2052,0.72238,2020-12-04T11:50:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04051,mir-8517,1287,mir-8517 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,78.0,58.2,This family represents the microRNA (miRNA) precursor mir-8517 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66712,-2.49218,1600000,205348,211,108,true,-3.3876,0.71859,2020-12-04T11:51:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04052,mir-466,1287,mir-466 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.6,79.7,This family represents the microRNA (miRNA) precursor mir-466 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,15,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61427,-3.48577,1600000,233386,173,80,true,-3.7467,0.71898,2021-05-19T14:27:04.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04053,mir-95,1287,mir-95 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,67.5,67.5,67.3,This family represents the microRNA (miRNA) precursor mir-95 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,34,75,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.61508,-3.44188,1600000,214161,167,79,true,-3.2291,0.71914,2021-05-19T14:33:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04054,mir-2944,1287,mir-2944 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.0,55.9,This family represents the microRNA (miRNA) precursor mir-2944 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,46,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.67186,-1.38978,1600000,137004,169,82,true,-3.8639,0.71907,2021-05-19T14:43:17.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04055,mir-1261,1287,mir-1261 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,91.0,91.3,90.2,This family represents the microRNA (miRNA) precursor mir-1261 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,6,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.59348,-4.02906,1600000,217037,176,82,true,-3.7566,0.71886,2021-05-19T14:43:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04056,mir-982,1287,mir-982 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,82.8,57.5,This family represents the microRNA (miRNA) precursor mir-982 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59754,-2.83015,1600000,201720,184,89,true,-3.0911,0.71883,2021-05-19T14:43:59.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04057,MIR1432,1287,MIR1432 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,100.5,76.1,This family represents the microRNA (miRNA) precursor MIR1432 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,2,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.58029,-4.14364,1600000,215147,186,89,true,-3.3812,0.71896,2021-05-19T14:44:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04059,MIR5020,1287,MIR5020 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.3,70.8,This family represents the microRNA (miRNA) precursor MIR5020 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63986,-2.9049,1600000,188268,305,109,true,-3.2984,0.71856,2021-05-19T14:45:44.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04060,mir-703,1287,mir-703 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.8,85.3,This family represents the microRNA (miRNA) precursor mir-703 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.64268,-2.28904,1600000,236393,154,67,true,-3.714,0.71922,2021-05-19T14:46:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04061,mir-1420,1287,mir-1420 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,79.1,57.3,This family represents the microRNA (miRNA) precursor mir-1420 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66107,-2.6428,1600000,162249,173,87,true,-3.7848,0.71881,2021-05-19T14:46:23.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04062,MIR949,1287,MIR949 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,121.4,61.5,This family represents the microRNA (miRNA) precursor MIR949 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.5805,-4.08754,1600000,157571,207,106,true,-3.2935,0.7187,2021-05-19T14:46:59.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04063,MIR2629,1287,MIR2629 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.0,59.9,This family represents the microRNA (miRNA) precursor MIR2629 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,653,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.72155,-3.16054,1600000,174997,146,67,true,-3.7345,0.71922,2021-05-19T14:47:22.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04064,MIR5267,1287,MIR5267 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.0,86.9,This family represents the microRNA (miRNA) precursor MIR5267 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,194,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58784,-2.86053,1600000,224062,174,81,true,-3.4671,0.71886,2021-05-19T14:47:54.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04065,mir-7563,1287,mir-7563 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,86.6,72.4,This family represents the microRNA (miRNA) precursor mir-7563 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60623,-4.23937,1600000,258659,181,76,true,-3.5597,0.71914,2021-05-19T14:48:22.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04066,mir-1256,1287,mir-1256 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,88.9,65.3,This family represents the microRNA (miRNA) precursor mir-1256 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,8,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.65886,-3.01118,1600000,245725,156,70,true,-3.2382,0.71934,2021-05-19T14:48:40.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04067,mir-4520,1287,mir-4520 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.1,89.1,This family represents the microRNA (miRNA) precursor mir-4520 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.66314,-3.59993,1600000,150281,156,70,true,-3.771,0.71921,2021-05-19T14:49:09.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04068,mir-3927,1287,mir-3927 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.5,88.3,This family represents the microRNA (miRNA) precursor mir-3927 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,10,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.56274,-4.5187,1600000,208085,179,85,true,-3.4748,0.71909,2021-05-19T14:49:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04069,MIR6135,1287,MIR6135 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.9,90.3,This family represents the microRNA (miRNA) precursor MIR6135 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,11,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58192,-2.256,1600000,186854,257,122,true,-2.8506,0.71862,2021-05-19T14:49:57.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04070,MIR6440,1287,MIR6440 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,95.1,94.6,This family represents the microRNA (miRNA) precursor MIR6440 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,13,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58587,-4.32746,1600000,196048,212,108,true,-3.4137,0.71871,2021-05-19T14:50:29.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04071,MIR2863,1287,MIR2863 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.1,93.4,This family represents the microRNA (miRNA) precursor MIR2863 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,23,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.634,-3.0884,1600000,194114,227,113,true,-3.0208,0.71872,2021-05-19T14:50:47.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04072,mir-7386,1287,mir-7386 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,44.0,44.2,43.7,This family represents the microRNA (miRNA) precursor mir-7386 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,14,38,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59266,-4.54272,1600000,227505,127,59,true,-3.9873,0.71962,2021-05-19T14:51:14.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04073,mir-727,1287,mir-727 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,104.0,104.0,103.0,This family represents the microRNA (miRNA) precursor mir-727 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,18,0,0,Gene; miRNA;,Predicted; RNAalifold,14,0,,\"\",0.62186,-3.10548,1600000,203258,187,91,true,-3.4985,0.71889,2021-05-19T14:51:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04074,mir-1199,1287,mir-1199 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,82.5,71.7,This family represents the microRNA (miRNA) precursor mir-1199 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,22,0,0,Gene; miRNA;,Predicted; RNAalifold,23,0,,\"\",0.48489,-5.18455,1600000,197671,200,100,true,-3.2457,0.71882,2021-05-19T14:52:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04075,mir-1545,1287,mir-1545 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.1,76.6,This family represents the microRNA (miRNA) precursor mir-1545 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,6,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.69669,-3.07562,1600000,273307,138,58,true,-3.8167,0.71934,2021-05-19T14:52:44.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04076,mir-H11,1287,mir-H11 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.8,68.4,This family represents the microRNA (miRNA) precursor mir-H11 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,50,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.59215,-3.55461,1600000,253973,159,72,true,-3.5948,0.71931,2021-05-19T14:53:05.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04077,mir-1245,1287,mir-1245 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.4,62.2,This family represents the microRNA (miRNA) precursor mir-1245 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,16,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.64063,-1.90024,1600000,236428,154,69,true,-3.637,0.71918,2021-05-19T14:53:28.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04078,mir-1905,1287,mir-1905 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.0,82.7,This family represents the microRNA (miRNA) precursor mir-1905 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,112,0,0,Gene; miRNA;,Predicted; RNAalifold,54,0,,\"\",0.59687,-3.97512,1600000,172559,161,73,true,-3.6909,0.71909,2021-05-19T14:53:57.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04079,MIR161,1287,MIR161 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,93.2,83.7,This family represents the microRNA (miRNA) precursor MIR161 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,10,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.65505,-3.578,1600000,225227,249,136,true,-3.298,0.71834,2021-05-19T14:54:20.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04080,mir-762,1287,mir-762 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.5,68.7,This family represents the microRNA (miRNA) precursor mir-762 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,27,0,0,Gene; miRNA;,Predicted; RNAalifold,19,0,,\"\",0.47409,-6.21165,1600000,293715,164,75,true,-3.3034,0.71901,2021-05-19T14:54:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04081,mir-1992,1287,mir-1992 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,82.7,65.6,This family represents the microRNA (miRNA) precursor mir-1992 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,1,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.70849,-2.2859,1600000,165714,161,75,true,-3.6276,0.72139,2021-05-19T14:55:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04082,mir-54,1287,mir-54 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.9,65.3,This family represents the microRNA (miRNA) precursor mir-54 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,8,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.60372,-4.17364,1600000,203468,176,88,true,-3.5262,0.71887,2021-05-19T14:55:42.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04083,MIR528,1287,MIR528 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.9,54.3,This family represents the microRNA (miRNA) precursor MIR528 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,23,0,0,Gene; miRNA;,Predicted; RNAalifold,22,0,,\"\",0.59685,-5.77545,1600000,292672,179,89,true,-2.9198,0.71905,2021-05-19T14:56:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04084,mir-595,1287,mir-595 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,90.0,53.4,This family represents the microRNA (miRNA) precursor mir-595 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.6827,-2.58172,1600000,172534,181,86,true,-3.4021,0.71886,2021-05-19T14:56:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04085,mir-1271,1287,mir-1271 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.5,80.5,80.2,This family represents the microRNA (miRNA) precursor mir-1271 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,13,29,0,0,Gene; miRNA;,Predicted; RNAalifold,33,0,,\"\",0.68831,-2.76357,1600000,197000,190,77,true,-3.5576,0.71909,2021-05-19T14:57:12.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04087,MIR5565,1287,MIR5565 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.0,79.9,This family represents the microRNA (miRNA) precursor MIR5565 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,1903,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6673,-2.99951,1600000,182834,210,109,true,-3.5222,0.71874,2021-05-19T14:58:08.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04088,MIR812,1287,MIR812 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.0,79.9,This family represents the microRNA (miRNA) precursor MIR812 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,22,58318,0,0,Gene; miRNA;,Predicted; RNAalifold,15,0,,\"\",0.41409,-9.321,1600000,568614,324,205,true,-3.3365,0.71176,2021-05-19T14:59:32.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04089,mir-753,1287,mir-753 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.4,56.5,This family represents the microRNA (miRNA) precursor mir-753 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.68867,-1.2692,1600000,231590,145,62,true,-3.9039,0.71933,2021-05-19T15:00:28.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04090,mir-6529,1287,mir-6529 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.4,51.5,This family represents the microRNA (miRNA) precursor mir-6529 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,32,0,0,Gene; miRNA;,Predicted; RNAalifold,32,0,,\"\",0.629,-3.25193,1600000,199285,171,79,true,-3.5637,0.71905,2021-05-19T15:00:50.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04091,mir-72,1287,mir-72 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,60.0,60.0,59.8,This family represents the microRNA (miRNA) precursor mir-72 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,45,0,0,Gene; miRNA;,Predicted; RNAalifold,45,0,,\"\",0.64796,-4.48928,1600000,191450,308,70,true,-3.4837,0.71907,2021-05-19T15:01:20.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04092,MIR1508,1287,MIR1508 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,89.7,69.5,This family represents the microRNA (miRNA) precursor MIR1508 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 88.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61327,-3.65409,1600000,174891,188,91,true,-3.4135,0.7189,2021-05-19T16:58:34.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04093,mir-1784,1287,mir-1784 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.5,97.1,This family represents the microRNA (miRNA) precursor mir-1784 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,49,0,0,Gene; miRNA;,Predicted; RNAalifold,49,0,,\"\",0.62097,-3.32752,1600000,166587,191,92,true,-3.3719,0.71879,2021-05-20T10:54:35.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04094,MIR537,1287,MIR537 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,123.3,82.9,This family represents the microRNA (miRNA) precursor MIR537 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66082,-4.46206,1600000,181062,436,151,true,-3.0988,0.71823,2021-05-20T10:54:51.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04095,MIR9746,1287,MIR9746 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.1,77.1,This family represents the microRNA (miRNA) precursor MIR9746 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,28,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61484,-3.08537,1600000,167082,216,110,true,-3.3852,0.71858,2021-05-20T10:55:07.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04096,mir-4433,1287,mir-4433 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,77.0,77.3,74.4,This family represents the microRNA (miRNA) precursor mir-4433 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,12,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.60593,-4.06154,1600000,199298,176,83,true,-3.1093,0.72146,2021-05-20T10:55:28.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04097,MIR2630,1287,MIR2630 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,144.0,144.0,142.0,This family represents the microRNA (miRNA) precursor MIR2630 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,23,288,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.55941,-7.69661,1600000,243129,364,229,true,-3.092,0.70994,2021-05-20T10:55:58.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04098,MIR9471,1287,MIR9471 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,150.0,156.7,139.9,This family represents the microRNA (miRNA) precursor MIR9471 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66383,-3.29112,1600000,190686,250,135,true,-3.0012,0.71835,2021-05-20T10:56:39.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04099,MIR837,1287,MIR837 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,117.0,117.6,93.5,This family represents the microRNA (miRNA) precursor MIR837 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62,-3.79285,1600000,214936,231,122,true,-3.2173,0.71861,2021-05-20T10:56:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04100,mir-493,1287,mir-493 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,68.0,68.0,57.4,This family represents the microRNA (miRNA) precursor mir-493 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,35,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.54655,-4.84854,1600000,151908,186,89,true,-3.4999,0.71877,2021-05-20T10:57:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04101,mir-8864,1287,mir-8864 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,120.0,128.2,113.5,This family represents the microRNA (miRNA) precursor mir-8864 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.55852,-4.3395,1600000,188805,216,110,true,-3.0027,0.71868,2021-05-20T10:57:49.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04102,MIR6457,1287,MIR6457 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,140.0,140.3,126.2,This family represents the microRNA (miRNA) precursor MIR6457 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57508,-5.74056,1600000,209700,304,177,true,-3.5079,0.71511,2021-05-20T10:58:04.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04103,mir-8356,1287,mir-8356 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,86.6,40.7,This family represents the microRNA (miRNA) precursor mir-8356 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65052,-2.33677,1600000,240278,153,65,true,-3.3523,0.72761,2021-05-20T10:58:19.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04104,MIR918,1287,MIR918 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,268.0,268.4,180.6,This family represents the microRNA (miRNA) precursor MIR918 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.49719,-8.95554,1600000,184207,412,265,true,-3.3483,0.70777,2021-05-20T10:58:34.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04105,mir-3165,1287,mir-3165 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.8,67.6,This family represents the microRNA (miRNA) precursor mir-3165 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.5708,-4.38659,1600000,276999,167,75,true,-3.2954,0.71959,2021-05-20T10:58:50.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04106,mir-1303,1287,mir-1303 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,107.0,107.9,101.2,This family represents the microRNA (miRNA) precursor mir-1303 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.5945,-4.26437,1600000,243329,180,85,true,-3.6498,0.71903,2021-05-20T10:59:08.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04107,mir-5391,1287,mir-5391 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.3,99.3,99.0,This family represents the microRNA (miRNA) precursor mir-5391 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.65139,-3.46967,1600000,222805,157,68,true,-3.0286,0.72295,2021-05-20T10:59:25.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04108,mir-2513,1287,mir-2513 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.7,88.2,This family represents the microRNA (miRNA) precursor mir-2513 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59414,-4.37011,1600000,224790,227,120,true,-2.9208,0.71853,2021-05-20T10:59:41.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04109,mir-1305,1287,mir-1305 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,63.0,63.5,59.2,This family represents the microRNA (miRNA) precursor mir-1305 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,18,0,0,Gene; miRNA;,Predicted; RNAalifold,19,0,,\"\",0.59603,-3.75391,1600000,185693,178,83,true,-3.3626,0.71884,2021-05-20T10:59:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04110,MIR5084,1287,MIR5084 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,110.0,110.2,109.8,This family represents the microRNA (miRNA) precursor MIR5084 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,75,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.5845,-4.59506,1600000,217449,230,122,true,-3.2544,0.71853,2021-05-20T11:00:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04111,MIR6274,1287,MIR6274 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,80.0,79.9,This family represents the microRNA (miRNA) precursor MIR6274 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,62,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64812,-1.48669,1600000,161552,209,107,true,-3.0146,0.71865,2021-05-20T11:00:27.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04112,MIR5512,1287,MIR5512 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,139.0,139.6,138.7,This family represents the microRNA (miRNA) precursor MIR5512 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.59565,-3.39543,1600000,180034,244,132,true,-2.8217,0.71831,2021-05-20T11:00:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04113,MIR1523,1287,MIR1523 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.0,67.0,This family represents the microRNA (miRNA) precursor MIR1523 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6448,-2.25619,1600000,178298,189,92,true,-2.9477,0.72142,2021-05-20T11:00:58.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04114,MIR4374,1287,MIR4374 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,105.0,105.0,104.7,This family represents the microRNA (miRNA) precursor MIR4374 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,62,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64282,-3.32906,1600000,151967,224,119,true,-2.9068,0.7186,2021-05-20T11:01:13.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04115,mir-6130,1287,mir-6130 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,117.0,117.0,78.5,This family represents the microRNA (miRNA) precursor mir-6130 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.63195,-2.64866,1600000,195853,206,103,true,-3.547,0.71864,2021-05-20T11:01:32.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04116,MIR9672,1287,MIR9672 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.4,72.9,This family represents the microRNA (miRNA) precursor MIR9672 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59101,-4.31375,1600000,226403,230,120,true,-3.0476,0.7185,2021-05-20T11:01:55.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04117,MIR5200,1287,MIR5200 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,71.4,69.4,This family represents the microRNA (miRNA) precursor MIR5200 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,39,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.61464,-3.37267,1600000,148943,186,92,true,-3.1734,0.71886,2021-05-20T11:02:11.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04118,MIR9481,1287,MIR9481 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,130.0,141.5,117.7,This family represents the microRNA (miRNA) precursor MIR9481 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63246,-3.86383,1600000,195707,248,135,true,-2.7014,0.71828,2021-05-20T11:02:29.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04119,MIR1858,1287,MIR1858 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,137.0,137.8,135.5,This family represents the microRNA (miRNA) precursor MIR1858 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,21,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.52087,-5.8535,1600000,177369,330,197,true,-3.2377,0.71298,2021-05-20T11:02:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04120,MIR5998,1287,MIR5998 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,180.0,188.1,93.0,This family represents the microRNA (miRNA) precursor MIR5998 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60019,-7.12507,1600000,263209,295,173,true,-3.0095,0.71554,2021-05-20T11:02:58.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04121,mir-1289,1287,mir-1289 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,66.0,66.7,64.0,This family represents the microRNA (miRNA) precursor mir-1289 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,15,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.65069,-2.33495,1600000,156347,189,91,true,-3.5207,0.71876,2021-05-20T11:03:15.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04122,mir-1204,1287,mir-1204 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.6,65.8,This family represents the microRNA (miRNA) precursor mir-1204 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,11,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.62897,-4.12747,1600000,251195,154,67,true,-3.5648,0.71922,2021-05-20T11:03:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04123,mir-3127,1287,mir-3127 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.0,82.9,65.0,This family represents the microRNA (miRNA) precursor mir-3127 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.54702,-4.36175,1600000,201305,184,87,true,-3.6572,0.71889,2021-05-20T11:03:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04124,MIR8565,1287,MIR8565 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,136.0,136.0,135.9,This family represents the microRNA (miRNA) precursor MIR8565 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,456,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.54256,-5.14464,1600000,173812,271,154,true,-3.1573,0.71836,2021-05-20T11:04:14.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04125,MIR9408,1287,MIR9408 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,120.9,79.2,This family represents the microRNA (miRNA) precursor MIR9408 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.60519,-3.75204,1600000,171532,224,117,true,-3.1546,0.71851,2021-05-20T11:04:36.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04126,mir-9198,1287,mir-9198 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,92.0,92.0,91.9,This family represents the microRNA (miRNA) precursor mir-9198 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,29,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.5649,-5.18843,1600000,246163,169,79,true,-3.443,0.71891,2021-05-20T11:04:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04127,MIR8007,1287,MIR8007 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,195.0,195.0,194.9,This family represents the microRNA (miRNA) precursor MIR8007 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,60,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.53295,-9.02955,1600000,321497,363,226,true,-3.4323,0.71023,2021-05-20T11:05:15.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04128,MIR8562,1287,MIR8562 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,211.0,211.2,210.9,This family represents the microRNA (miRNA) precursor MIR8562 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,36,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.51268,-10.07302,1600000,402243,381,242,true,-3.638,0.70898,2021-05-20T11:05:32.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04129,MIR4240,1287,MIR4240 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,200.0,235.0,181.9,This family represents the microRNA (miRNA) precursor MIR4240 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59527,-8.33286,1600000,334737,339,205,true,-3.4928,0.71206,2021-05-20T11:05:48.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04130,MIR7996,1287,MIR7996 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,119.0,119.0,113.9,This family represents the microRNA (miRNA) precursor MIR7996 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,114,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.513,-8.02454,1600000,244341,321,190,true,-3.0629,0.71346,2021-05-20T11:06:04.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04131,mir-2032,1287,mir-2032 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,108.7,53.8,This family represents the microRNA (miRNA) precursor mir-2032 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.6218,-3.43709,1600000,255258,169,77,true,-3.4556,0.71892,2021-05-20T11:06:20.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04132,mir-4436,1287,mir-4436 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.8,64.1,This family represents the microRNA (miRNA) precursor mir-4436 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,32,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.63739,-3.69529,1600000,194176,180,85,true,-2.9965,0.72368,2021-05-20T11:06:35.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04133,MIR2646,1287,MIR2646 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,245.0,245.1,231.1,This family represents the microRNA (miRNA) precursor MIR2646 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,9,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58867,-8.37397,1600000,279297,401,256,true,-4.0235,0.70819,2021-05-20T11:06:50.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04134,MIR7504,1287,MIR7504 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.0,82.9,This family represents the microRNA (miRNA) precursor MIR7504 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,1767,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.56954,-3.57902,1600000,192169,226,124,true,-2.9298,0.71843,2021-05-20T11:07:10.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04135,MIR5062,1287,MIR5062 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,140.0,141.4,105.3,This family represents the microRNA (miRNA) precursor MIR5062 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.57088,-6.2424,1600000,266391,281,158,true,-3.3848,0.71763,2021-05-20T11:07:37.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04136,MIR1862,1287,MIR1862 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.0,83.8,This family represents the microRNA (miRNA) precursor MIR1862 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,82,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.6006,-4.30272,1600000,205838,188,90,true,-3.6428,0.71899,2021-05-20T11:07:58.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04137,MIR834,1287,MIR834 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,200.0,216.6,81.4,This family represents the microRNA (miRNA) precursor MIR834 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6296,-6.31303,1600000,266701,308,180,true,-3.2792,0.71445,2021-05-20T11:08:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04138,mir-9243,1287,mir-9243 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.0,82.5,This family represents the microRNA (miRNA) precursor mir-9243 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.61952,-3.18349,1600000,180721,180,86,true,-3.2942,0.72196,2021-05-20T11:08:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04139,MIR841,1287,MIR841 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,200.0,207.0,197.6,This family represents the microRNA (miRNA) precursor MIR841 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,5,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59147,-7.00724,1600000,355916,347,212,true,-3.5929,0.71138,2021-05-20T11:08:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04140,mir-3118,1287,mir-3118 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.6,88.7,This family represents the microRNA (miRNA) precursor mir-3118 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,26,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.63936,-2.51162,1600000,223363,168,76,true,-3.2613,0.72032,2021-05-20T11:09:07.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04141,MIR8622,1287,MIR8622 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,197.0,197.1,196.9,This family represents the microRNA (miRNA) precursor MIR8622 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,77,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.50225,-9.8265,1600000,231098,346,212,true,-3.0986,0.71146,2021-05-20T11:09:23.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04142,mir-1299,1287,mir-1299 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,89.0,89.0,88.9,This family represents the microRNA (miRNA) precursor mir-1299 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,67,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.61038,-3.75647,1600000,212423,176,83,true,-3.4705,0.71894,2021-05-20T11:09:39.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04143,MIR869,1287,MIR869 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,330.0,336.5,186.4,This family represents the microRNA (miRNA) precursor MIR869 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6096,-9.42086,1600000,350222,449,296,true,-4.2213,0.70619,2021-05-20T11:09:56.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04144,MIR9783,1287,MIR9783 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.2,99.8,This family represents the microRNA (miRNA) precursor MIR9783 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.63992,-2.25806,1600000,151385,193,95,true,-3.5084,0.71877,2021-05-20T11:10:12.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04145,MIR8706,1287,MIR8706 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,97.7,94.6,This family represents the microRNA (miRNA) precursor MIR8706 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60801,-3.37578,1600000,179247,203,101,true,-3.0494,0.71871,2021-05-20T11:10:27.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04146,mir-4742,1287,mir-4742 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.1,63.0,This family represents the microRNA (miRNA) precursor mir-4742 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.58271,-3.69971,1600000,193455,181,85,true,-3.2341,0.71892,2021-05-20T11:10:44.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04147,MIR6224,1287,MIR6224 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,171.0,171.0,168.9,This family represents the microRNA (miRNA) precursor MIR6224 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,8,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.5505,-8.16902,1600000,269019,301,176,true,-3.3582,0.71509,2021-05-20T11:11:00.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04148,MIR844,1287,MIR844 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.8,67.1,This family represents the microRNA (miRNA) precursor MIR844 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6601,-2.56669,1600000,171472,220,114,true,-3.349,0.71856,2021-05-20T11:11:16.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04149,MIR447,1287,MIR447 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,200.0,225.8,125.0,This family represents the microRNA (miRNA) precursor MIR447 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.54709,-9.57088,1600000,340851,375,236,true,-3.6521,0.70947,2021-05-20T11:11:31.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04150,MIR2655,1287,MIR2655 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,122.0,122.0,121.7,This family represents the microRNA (miRNA) precursor MIR2655 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,25,445,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.42707,-14.08953,1600000,644474,414,267,true,-3.6535,0.70739,2021-05-20T11:12:01.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04151,MIR4387,1287,MIR4387 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.4,83.9,This family represents the microRNA (miRNA) precursor MIR4387 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,94,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.61661,-3.64222,1600000,163458,208,109,true,-3.0682,0.71869,2021-05-20T11:12:49.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04152,mir-3618,1287,mir-3618 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.8,82.6,This family represents the microRNA (miRNA) precursor mir-3618 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,114,0,0,Gene; miRNA;,Predicted; RNAalifold,108,0,,\"\",0.67022,-2.43983,1600000,221354,183,88,true,-3.6545,0.71879,2021-05-20T11:13:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04153,MIR856,1287,MIR856 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,297.0,297.8,193.1,This family represents the microRNA (miRNA) precursor MIR856 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.61922,-9.21451,1600000,311337,404,258,true,-4.0239,0.70814,2021-05-20T11:13:29.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04154,mir-1277,1287,mir-1277 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.5,74.6,This family represents the microRNA (miRNA) precursor mir-1277 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,28,0,0,Gene; miRNA;,Predicted; RNAalifold,31,0,,\"\",0.60103,-3.37962,1600000,229918,170,78,true,-3.7483,0.71898,2021-05-20T11:14:20.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04155,mir-9236,1287,mir-9236 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,98.0,98.0,97.9,This family represents the microRNA (miRNA) precursor mir-9236 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,209,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.60608,-2.35739,1600000,272616,176,83,true,-3.1727,0.71888,2021-05-20T11:14:54.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04156,MIR2670,1287,MIR2670 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,117.0,117.1,116.9,This family represents the microRNA (miRNA) precursor MIR2670 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,187,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58734,-7.95261,1600000,293494,328,200,true,-3.4691,0.71249,2021-05-20T11:15:15.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04157,MIR9486,1287,MIR9486 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,107.7,91.7,This family represents the microRNA (miRNA) precursor MIR9486 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62851,-3.70599,1600000,183025,221,116,true,-3.01,0.7185,2021-05-20T11:15:40.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04158,mir-568,1287,mir-568 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.9,66.2,This family represents the microRNA (miRNA) precursor mir-568 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,37,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.55196,-3.26262,1600000,168830,190,93,true,-3.6012,0.71884,2021-05-20T11:16:10.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04159,MIR1437,1287,MIR1437 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,168.0,168.0,138.6,This family represents the microRNA (miRNA) precursor MIR1437 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.50574,-6.96185,1600000,167776,321,192,true,-2.793,0.71341,2021-05-20T11:16:47.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04160,MIR868,1287,MIR868 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,200.0,209.6,111.2,This family represents the microRNA (miRNA) precursor MIR868 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.56397,-6.73001,1600000,242337,323,193,true,-3.517,0.71314,2021-05-20T11:17:01.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04161,mir-3688,1287,mir-3688 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.6,81.1,This family represents the microRNA (miRNA) precursor mir-3688 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,18,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.64087,-3.08883,1600000,222050,183,88,true,-3.3766,0.71921,2021-05-20T11:17:20.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04162,MIR9662,1287,MIR9662 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,108.4,69.6,This family represents the microRNA (miRNA) precursor MIR9662 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.53259,-4.58205,1600000,163531,202,100,true,-2.984,0.71882,2021-05-20T11:17:37.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04163,mir-8364,1287,mir-8364 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.0,66.9,This family represents the microRNA (miRNA) precursor mir-8364 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,2083,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.56691,-5.33356,1600000,360620,139,61,true,-3.5184,0.7193,2021-05-20T11:18:00.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04164,MIR1507,1287,MIR1507 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.2,65.1,This family represents the microRNA (miRNA) precursor MIR1507 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,22,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.70075,-2.16011,1600000,131799,204,107,true,-3.8264,0.71871,2021-05-20T11:18:33.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04165,mir-2483,1287,mir-2483 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.9,64.6,This family represents the microRNA (miRNA) precursor mir-2483 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,12,0,0,Gene; miRNA;,Predicted; RNAalifold,13,0,,\"\",0.618,-3.35603,1600000,238935,173,80,true,-3.5452,0.719,2021-05-20T11:18:58.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04166,mir-2278,1287,mir-2278 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.8,73.7,This family represents the microRNA (miRNA) precursor mir-2278 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.57939,-3.75508,1600000,160843,196,96,true,-3.448,0.71879,2021-05-20T11:19:18.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04167,mir-1972,1287,mir-1972 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.2,89.9,This family represents the microRNA (miRNA) precursor mir-1972 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.61661,-3.85981,1600000,257813,174,81,true,-2.6719,0.72375,2021-05-20T11:19:36.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04168,MIR839,1287,MIR839 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,200.0,267.0,199.6,This family represents the microRNA (miRNA) precursor MIR839 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.56458,-7.83123,1600000,276262,414,267,true,-4.141,0.70761,2021-05-20T11:19:51.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04169,mir-2235,1287,mir-2235 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,61.0,61.8,47.3,This family represents the microRNA (miRNA) precursor mir-2235 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,14,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.66116,-2.5255,1600000,198073,181,87,true,-3.9203,0.71882,2021-05-20T11:20:10.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04170,MIR7723,1287,MIR7723 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,200.0,203.0,145.0,This family represents the microRNA (miRNA) precursor MIR7723 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.56835,-7.3262,1600000,292133,370,235,true,-3.7372,0.70945,2021-05-20T11:20:31.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04171,MIR1509,1287,MIR1509 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,87.0,87.4,86.6,This family represents the microRNA (miRNA) precursor MIR1509 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,15,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.61515,-3.61849,1600000,226426,222,121,true,-2.9586,0.71842,2021-05-20T11:20:49.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04172,MIR8675,1287,MIR8675 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.8,90.8,90.7,This family represents the microRNA (miRNA) precursor MIR8675 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,32,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62766,-2.34116,1600000,177648,192,95,true,-3.4088,0.72028,2021-05-20T11:21:13.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04173,MIR847,1287,MIR847 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,184.0,184.8,95.1,This family represents the microRNA (miRNA) precursor MIR847 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.60763,-5.92778,1600000,242077,286,163,true,-3.4225,0.71671,2021-05-20T11:21:28.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04174,MIR4208,1287,MIR4208 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,140.0,141.1,137.9,This family represents the microRNA (miRNA) precursor MIR4208 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,7,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.48039,-7.63075,1600000,357488,323,191,true,-3.2406,0.71325,2021-05-20T11:21:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04175,mir-1260b,1287,mir-1260b microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,81.0,81.2,59.6,This family represents the microRNA (miRNA) precursor mir-1260b imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,28,0,0,Gene; miRNA;,Predicted; RNAalifold,30,0,,\"\",0.60293,-3.4488,1600000,131558,212,108,true,-3.4898,0.7187,2021-05-20T11:22:03.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04176,mir-3149,1287,mir-3149 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,112.2,89.0,This family represents the microRNA (miRNA) precursor mir-3149 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.59388,-4.04921,1600000,237107,178,83,true,-3.9623,0.71889,2021-05-20T11:22:45.000Z,2021-06-14T15:02:18.000Z\n",
            "RF04183,Bacteroidales_small_SRP,1315,Bacteroidales small SRP,\"Prezza, G, Ryan, D, Madler, G, Barquist, L, Westermann, A\",Published; PMID:PMC7366714;,55.2,55.2,54.6,The non-coding component of the signal recognition particle in Bacteroides [1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,31,611,0,0,Gene;,Published; PMID:PMC7366714,637,0,,\"\",0.58797,-5.33175,1600000,207735,206,107,true,-3.5718,0.71837,2021-05-24T14:08:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04185,MIR2118_2,1287,MIR2118_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,120.0,121.1,61.2,This family represents the microRNA (miRNA) precursor MIR2118_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60254,-4.36988,1600000,231093,210,106,true,-3.5144,0.71862,2021-05-27T11:10:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04186,mir-278_2,1287,mir-278_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,96.1,45.5,This family represents the microRNA (miRNA) precursor mir-278_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.66331,-2.7952,1600000,198061,172,79,true,-3.2364,0.72437,2021-05-27T14:17:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04187,MIR862,1287,MIR862 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,119.0,97.1,This family represents the microRNA (miRNA) precursor MIR862 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.64922,-2.87481,1600000,196840,215,110,true,-3.3009,0.71848,2021-05-27T14:51:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04190,Hairpin-meta1,1240,Hairpin ribozyme 1 from viruses-like metatranscriptomes,\"Eckert I, Weinberg Z\",Published; PMID:34096583,40.0,40.9,38.5,\"Hairpin-meta ribozymes were recognised in metatranscriptomic sequences of viruses-like organisms, it was proved by co-transcriptional assays that hairpin-meta1 and hairpin-meta2 non-coding RNA (ncRNAs) are catalytically active and it is suggested that they likely replicate as circular single-stranded RNAs [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,84,2,0,0,Gene; ribozyme;,Published; PMID:34096583,3,0,,\"\",0.51468,-6.95927,1600000,218734,462,118,true,-3.9037,0.71823,2021-10-26T12:44:42.000Z,2021-11-19T10:36:02.000Z\n",
            "RF04191,Hairpin-meta2,1240,Hairpin ribozyme 2 from viruses-like metatranscriptomes,\"Eckert I, Weinberg Z\",Published; PMID:34096583,210.0,212.6,35.7,\"Hairpin-meta ribozymes were recognised in metatranscriptomic sequences of viruses-like organisms, it was proved by co-transcriptional assays that hairpin-meta1 and hairpin-meta2 non-coding RNA (ncRNAs) are catalytically active and it is suggested that they likely replicate as circular single-stranded RNAs. [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,48,0,0,0,Gene; ribozyme;,Published; PMID:34096583,1,0,,\"\",0.71949,-8.91262,1600000,355941,678,269,true,-3.7143,0.70732,2021-10-26T14:43:41.000Z,2022-05-06T12:43:43.000Z\n",
            "RF04193,mir-51,1287,mir-51 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,78.1,62.5,This family represents the microRNA (miRNA) precursor mir-51 imported from miRBase,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,7,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.63863,-3.59502,1600000,244397,144,62,true,-3.4724,0.71928,2022-03-14T12:07:38.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04194,mir-57,1287,mir-57 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,44.0,44.2,43.7,This family represents the microRNA (miRNA) precursor mir-57 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,46,0,0,Gene; miRNA;,Predicted; RNAalifold,42,0,,\"\",0.63713,-3.41856,1600000,133956,172,84,true,-3.8266,0.71885,2022-03-14T17:06:08.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04195,MIR6217,1287,MIR6217 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,110.0,110.2,109.0,This family represents the microRNA (miRNA) precursor MIR6217 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,198,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.5969,-8.44703,1600000,322318,367,228,true,-3.1235,0.71006,2022-03-15T10:26:22.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04196,MIR5638,1287,MIR5638 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,500.0,510.4,236.2,This family represents the microRNA (miRNA) precursor MIR5638 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65664,-10.88808,1600000,438008,648,464,true,-4.5309,0.70145,2022-03-15T10:26:30.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04197,mir-9457,1287,mir-9457 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,93.8,64.8,This family represents the microRNA (miRNA) precursor mir-9457 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.61896,-3.98059,1600000,286388,157,68,true,-3.2985,0.72266,2022-03-15T10:26:39.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04198,MIR8670,1287,MIR8670 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,150.0,150.6,104.2,This family represents the microRNA (miRNA) precursor MIR8670 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57824,-4.67562,1600000,172011,258,142,true,-3.1927,0.71848,2022-03-15T10:26:45.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04199,MIR6027,1287,MIR6027 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,98.1,85.1,This family represents the microRNA (miRNA) precursor MIR6027 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,3,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.62394,-4.70716,1600000,190852,280,158,true,-2.6651,0.71751,2022-03-15T10:26:52.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04200,mir-9437,1287,mir-9437 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,82.3,82.3,82.2,This family represents the microRNA (miRNA) precursor mir-9437 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,277,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64682,-4.28367,1600000,258551,156,68,true,-3.6083,0.71901,2022-03-15T10:26:56.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04201,MIR7510,1287,MIR7510 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.0,92.6,This family represents the microRNA (miRNA) precursor MIR7510 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59903,-4.00033,1600000,186938,172,79,true,-2.9463,0.72375,2022-03-15T10:27:09.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04202,MIR5380,1287,MIR5380 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,150.0,153.5,129.8,This family represents the microRNA (miRNA) precursor MIR5380 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.60848,-4.08774,1600000,216042,253,137,true,-3.1305,0.71839,2022-03-15T10:27:16.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04203,MIR1144,1287,MIR1144 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,93.3,85.7,This family represents the microRNA (miRNA) precursor MIR1144 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.50564,-10.12667,1600000,283509,330,199,true,-3.4431,0.7126,2022-03-15T10:55:46.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04204,mir-3473,1287,mir-3473 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.8,71.8,71.7,This family represents the microRNA (miRNA) precursor mir-3473 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,181,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.62751,-3.9289,1600000,205718,168,78,true,-2.9531,0.71928,2022-03-15T12:18:45.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04205,MIR2606,1287,MIR2606 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,139.7,139.7,139.5,This family represents the microRNA (miRNA) precursor MIR2606 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,216,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.49396,-5.58651,1600000,250218,328,165,true,-3.0126,0.71646,2022-03-15T12:20:02.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04206,MIR9555,1287,MIR9555 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,119.0,119.0,118.9,This family represents the microRNA (miRNA) precursor MIR9555 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,30,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.57456,-4.16485,1600000,232252,250,137,true,-3.3111,0.71833,2022-03-15T12:20:57.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04207,MIR5048,1287,MIR5048 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,160.0,168.8,130.8,This family represents the microRNA (miRNA) precursor MIR5048 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,5,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.58719,-11.42266,1600000,603504,521,356,true,-4.1048,0.70399,2022-03-15T12:23:48.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04208,MIR8742,1287,MIR8742 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,130.0,132.6,87.3,This family represents the microRNA (miRNA) precursor MIR8742 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.64329,-3.58402,1600000,176555,242,131,true,-3.0948,0.71843,2022-03-15T12:23:51.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04209,MIR7533,1287,MIR7533 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,250.0,268.0,233.3,This family represents the microRNA (miRNA) precursor MIR7533 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.56049,-7.7673,1600000,205237,388,248,true,-3.6776,0.70875,2022-03-15T12:23:56.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04210,MIR8643,1287,MIR8643 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,135.0,135.2,134.7,This family represents the microRNA (miRNA) precursor MIR8643 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,250,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.61253,-4.59222,1600000,241795,270,150,true,-3.1158,0.71801,2022-03-15T12:23:59.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04211,MIR5185,1287,MIR5185 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,101.7,101.7,101.5,This family represents the microRNA (miRNA) precursor MIR5185 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,13,37,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58389,-4.23836,1600000,132340,213,111,true,-3.1972,0.7188,2022-03-15T12:27:48.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04212,mir-8904,1287,mir-8904 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.9,96.9,96.8,This family represents the microRNA (miRNA) precursor mir-8904 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57953,-3.77538,1600000,178461,200,99,true,-3.0725,0.71875,2022-03-15T12:30:55.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04213,MIR7807,1287,MIR7807 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,96.2,83.9,This family represents the microRNA (miRNA) precursor MIR7807 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62448,-4.13982,1600000,220602,243,133,true,-3.2051,0.71846,2022-03-15T12:38:54.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04214,MIR7526,1287,MIR7526 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,110.0,113.8,93.8,This family represents the microRNA (miRNA) precursor MIR7526 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,8,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63218,-3.42397,1600000,196088,308,113,true,-3.1577,0.71857,2022-03-15T13:10:11.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04215,mir-9214,1287,mir-9214 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,77.2,77.2,77.1,This family represents the microRNA (miRNA) precursor mir-9214 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,60,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.67186,-3.33857,1600000,261083,167,75,true,-3.5207,0.72094,2022-03-15T16:21:45.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04216,mir-509,1287,mir-509 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,64.5,64.5,63.7,This family represents the microRNA (miRNA) precursor mir-509 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 CM SEQDB,38,120,0,0,Gene; miRNA;,Predicted; RNAalifold,37,0,,\"\",0.66321,-2.74546,1600000,191534,169,82,true,-3.3353,0.71887,2022-03-16T12:26:43.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04217,mir-297,1287,mir-297 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,91.2,84.0,This family represents the microRNA (miRNA) precursor mir-297 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,11,4,0,0,Gene; miRNA;,Predicted; RNAalifold,6,0,,\"\",0.55642,-4.8458,1600000,271401,151,71,true,-3.8858,0.71922,2022-03-24T11:22:20.000Z,2022-05-06T12:43:49.000Z\n",
            "RF04218,HCV_5BSL1,2418,Hepatitis C virus stem-loop I,\"Lamkiewicz K, Triebel S, Marz M\",\"Kevin Lamkiewicz, Sandra Triebel, Manja Marz\",44.0,44.6,35.3,Genetic modification of RNA structures in Hepatitis C virus (HCV) genome affects replication and infectivity [1] Long distant RNA-RNA interactions in HCV RNAs are involved in translation to virion budding [2] It has been proposed that the 5'-3' UTR base-pairing in the HCV genome might play an important role in viral RNA replication [3],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,57,1,0,0,Cis-reg;,\"Predicted; BM VeGETA v. 0.4, LocARNA 2.0.0RC8, RNAfold\",21,0,,\"\",0.68415,-4.7908,1600000,476798,111,47,true,-3.7777,0.71941,2022-04-20T12:55:13.000Z,2023-11-06T16:44:07.000Z\n",
            "RF04219,HCV_J750,2418,J750 non-coding RNA (containing SLSL761 and SL783),\"Lamkiewicz K, Triebel S, Marz M\",\"Kevin Lamkiewicz, Sandra Triebel, Manja Marz\",90.0,91.5,39.9,Genetic modification of RNA structures in Hepatitis C virus (HCV) genome affects replication and infectivity [1] Long distant RNA-RNA interactions in HCV RNAs are involved in translation to virion budding [2] It has been proposed that the 5'-3' UTR base-pairing in the HCV genome might play an important role in viral RNA replication [3],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,57,1,0,0,Cis-reg;,\"Predicted; BM VeGETA v. 0.4, LocARNA 2.0.0RC8, RNAfold\",21,0,,\"\",0.54365,-7.5395,1600000,271760,215,113,true,-3.1471,0.71852,2022-04-20T13:10:40.000Z,2023-11-06T16:44:07.000Z\n",
            "RF04220,HCV_SL588,2418,SL588 non-coding RNA,\"Lamkiewicz K, Triebel S, Marz M\",\"Kevin Lamkiewicz, Sandra Triebel, Manja Marz\",79.0,79.4,48.7,Genetic modification of RNA structures in Hepatitis C virus (HCV) genome affects replication and infectivity [1] Long distant RNA-RNA interactions in HCV RNAs are involved in translation to virion budding [2] It has been proposed that the 5'-3' UTR base-pairing in the HCV genome might play an important role in viral RNA replication [3],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,57,1,0,0,Cis-reg;,\"Predicted; BM VeGETA v. 0.4, LocARNA 2.0.0RC8, RNAfold\",21,0,,\"\",0.6313,-3.14214,1600000,215707,172,81,true,-3.0745,0.71894,2022-04-20T13:20:35.000Z,2023-11-06T16:44:07.000Z\n",
            "RF04221,HCV_SL669,2418,SL669 non-coding RNA,\"Lamkiewicz K, Triebel S, Marz M\",\"Kevin Lamkiewicz, Sandra Triebel, Manja Marz\",80.0,80.2,38.6,Genetic modification of RNA structures in Hepatitis C virus (HCV) genome affects replication and infectivity [1] Long distant RNA-RNA interactions in HCV RNAs are involved in translation to virion budding [2] It has been proposed that the 5'-3' UTR base-pairing in the HCV genome might play an important role in viral RNA replication [3],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,57,1,0,0,Cis-reg;,\"Predicted; BM VeGETA v. 0.4, LocARNA 2.0.0RC8, RNAfold, by hand\",21,0,,\"\",0.62856,-3.59938,1600000,222596,170,80,true,-3.3152,0.71896,2022-04-20T16:23:42.000Z,2023-11-06T16:44:07.000Z\n",
            "RF04223,MIR2619,1287,MIR2619 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,113.0,113.0,112.9,This family represents the microRNA (miRNA) precursor MIR2619 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1173,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64157,-6.70192,1600000,227766,330,202,true,-3.4382,0.71225,2022-09-07T16:58:24.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04224,mir-9229,1287,mir-9229 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.0,70.9,This family represents the microRNA (miRNA) precursor mir-9229 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,11544,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.62767,-3.76607,1600000,272721,166,76,true,-3.69,0.71915,2022-09-08T10:11:19.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04225,MIR7502,1287,MIR7502 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,71.0,71.0,70.9,This family represents the microRNA (miRNA) precursor MIR7502 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,17675,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.41382,-6.21518,1600000,202747,249,143,true,-2.4635,0.71812,2022-09-08T10:36:08.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04226,mir-9186,1287,mir-9186 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.0,73.4,72.9,This family represents the microRNA (miRNA) precursor mir-9186 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,31,65,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.59521,-4.53352,1600000,240217,167,80,true,-3.561,0.71899,2022-09-08T11:32:41.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04227,mir-9215,1287,mir-9215 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.3,87.7,This family represents the microRNA (miRNA) precursor mir-9215 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.64782,-2.68392,1600000,183191,179,85,true,-3.3,0.71894,2022-09-09T15:38:37.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04228,MIR6140,1287,MIR6140 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,140.0,143.7,114.9,This family represents the microRNA (miRNA) precursor MIR6140 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.56798,-4.99518,1600000,196189,285,165,true,-3.1544,0.71642,2022-09-16T11:46:29.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04229,mir-9279,1287,mir-9279 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.2,47.2,This family represents the microRNA (miRNA) precursor mir-9279 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58389,-3.7871,1600000,203194,163,74,true,-3.3179,0.71912,2022-09-16T13:54:45.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04230,mir-9261,1287,mir-9261 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,86.0,97.2,85.5,This family represents the microRNA (miRNA) precursor mir-9261 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63375,-3.36758,1600000,237536,188,92,true,-3.154,0.71886,2022-09-16T13:58:02.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04231,mir-9191,1287,mir-9191 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,102.3,39.7,This family represents the microRNA (miRNA) precursor mir-9191 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62627,-3.34703,1600000,285388,164,73,true,-3.0611,0.72141,2022-09-16T14:04:44.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04232,mir-9318,1287,mir-9318 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,99.3,87.9,This family represents the microRNA (miRNA) precursor mir-9318 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.65006,-3.67477,1600000,228175,167,76,true,-3.2076,0.72175,2022-09-16T14:07:53.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04233,mir-680,1287,mir-680 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,104.0,104.3,103.9,This family represents the microRNA (miRNA) precursor mir-680 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,195,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59514,-3.76745,1600000,199265,203,101,true,-3.2805,0.71868,2022-10-18T12:39:38.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04234,mir-1421,1287,mir-1421 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,45.0,45.5,44.9,This family represents the microRNA (miRNA) precursor mir-1421 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.00 -Z 742849.287494 CM SEQDB,53,34,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.60604,-4.3671,1600000,246573,155,74,true,-3.6922,0.719,2022-10-19T21:30:34.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04235,mir-242_2,1287,microRNA mir-242_2,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.3,54.7,This family represents the microRNA (miRNA) precursor mir-1421 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.00 -Z 742849.287494 CM SEQDB,9,7,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.72229,-1.90634,1600000,134988,158,81,true,-4.292,0.71884,2022-10-19T23:18:40.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04236,mir-1285,1287,mir-1285 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.0,55.9,This family represents the microRNA (miRNA) precursor mir-1421 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.00 -Z 742849.287494 CM SEQDB,5,101,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.59214,-4.59783,1600000,208590,150,68,true,-3.4568,0.71909,2022-10-19T23:25:38.000Z,2022-10-29T17:42:07.000Z\n",
            "RF04237,mir-1490,1287,mir-1490 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.0,75.9,39.4,Family represents the microRNA (miRNA) precursor mir-1490 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.68782,-3.39857,1600000,174599,165,77,true,-3.5234,0.72237,2023-03-27T17:25:38.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04238,mir-4012,1287,mir-4012 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,87.5,48.4,Family represents the microRNA (miRNA) precursor mir-4012 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64412,-3.17651,1600000,319543,150,63,true,-3.4765,0.72546,2023-04-04T08:17:03.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04239,mir-4679,1287,mir-4679 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.2,73.2,Family represents the microRNA (miRNA) precursor mir-4679 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.60347,-3.4807,1600000,199019,167,75,true,-2.8894,0.72345,2023-04-04T08:32:48.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04240,mir-4716,1287,mir-4716 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.0,76.2,75.7,Family represents the microRNA (miRNA) precursor mir-4716 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.64177,-3.5477,1600000,231014,179,84,true,-3.7328,0.71895,2023-04-04T08:37:40.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04241,mir-5014,1287,mir-5014 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,101.0,101.2,99.9,Family represents the microRNA (miRNA) precursor mir-5014 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,16,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.63199,-3.81046,1600000,218767,231,123,true,-3.2869,0.71843,2023-04-04T08:41:54.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04242,mir-5595,1287,mir-5595 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.6,66.4,Family represents the microRNA (miRNA) precursor mir-5595 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.62368,-2.83275,1600000,174439,163,72,true,-3.3992,0.72336,2023-04-13T12:02:20.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04243,MIR169_3,1287,MIR169_3 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,116.2,56.5,Family represents the microRNA (miRNA) precursor MIR169_3 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.57445,-4.32783,1600000,187244,214,109,true,-3.2314,0.71859,2023-06-12T13:50:46.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04244,MIR169_6,1287,MIR169_6 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,54.0,54.9,53.6,Family represents the microRNA (miRNA) precursor MIR169_6 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,52,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.65068,-3.10375,1600000,182644,198,99,true,-3.2683,0.71872,2023-06-12T14:15:17.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04245,MIR169_7,1287,MIR169_7 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,44.0,44.9,42.4,Family represents the microRNA (miRNA) precursor MIR169_7 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,2,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.58057,-4.28284,1600000,243320,179,85,true,-3.2454,0.71879,2023-06-12T14:24:46.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04246,MIR169_8,1287,MIR169_8 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,107.0,107.3,71.8,Family represents the microRNA (miRNA) precursor MIR169_8 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.58215,-4.34404,1600000,202654,257,137,true,-3.0645,0.71817,2023-06-12T14:56:48.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04248,MIR7486,1287,MIR7486 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.0,66.9,This family represents the microRNA (miRNA) precursor MIR7486 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,4176,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.55384,-4.76338,1600000,232623,173,87,true,-3.4506,0.71898,2023-09-25T15:10:00.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04249,mir-310,1287,mir-310 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,48.0,48.4,47.9,This family represents the microRNA (miRNA) precursor mir-310 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,23,46,0,0,Gene; miRNA;,Predicted; RNAalifold,27,0,,\"\",0.74681,-0.42209,1600000,198105,152,75,true,-3.8829,0.71908,2023-09-26T11:47:45.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04250,mir-236,1287,mir-236 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,55.0,55.2,48.2,This family represents the microRNA (miRNA) precursor mir-236 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,35,0,0,Gene; miRNA;,Predicted; RNAalifold,32,0,,\"\",0.61337,-3.69312,1600000,215878,160,76,true,-3.4856,0.71915,2023-09-26T13:26:10.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04251,MIR5070,1287,MIR5070 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.0,99.9,This family represents the microRNA (miRNA) precursor MIR5070 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,236,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.59032,-4.03786,1600000,197112,222,116,true,-2.9784,0.71857,2023-09-26T15:50:01.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04252,mir-8186,1287,mir-8186 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,84.1,60.4,This family represents the microRNA (miRNA) precursor mir-8186 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.67702,-3.22378,1600000,276986,148,61,true,-2.926,0.72929,2023-09-27T10:06:44.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04254,mir-1677,1287,mir-1677 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,84.0,84.6,81.9,This family represents the microRNA (miRNA) precursor mir-1677 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,5,49,0,0,Gene; miRNA;,Predicted; RNAalifold,49,0,,\"\",0.6385,-2.85629,1600000,207266,160,76,true,-3.8963,0.7191,2023-09-27T13:25:14.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04255,MIR8001,1287,MIR8001 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.9,89.6,This family represents the microRNA (miRNA) precursor MIR8001 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,11,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.59817,-4.1018,1600000,237519,156,67,true,-2.968,0.72531,2023-09-27T14:11:25.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04256,mir-125_2,1287,mir-125 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.8,57.6,This family represents the microRNA (miRNA) precursor mir-125 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.6184,-3.64534,1600000,219504,178,84,true,-3.0168,0.72024,2023-09-27T15:18:40.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04257,mir-9230,1287,mir-9230 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,100.0,99.9,This family represents the microRNA (miRNA) precursor mir-9230 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,328,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.63876,-3.32011,1600000,204687,182,87,true,-3.5046,0.71888,2023-10-03T11:39:08.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04258,mir-994,1287,mir-994 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,93.0,93.5,86.8,This family represents the microRNA (miRNA) precursor mir-994 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,13,0,0,Gene; miRNA;,Predicted; RNAalifold,14,0,,\"\",0.6331,-3.43839,1600000,259257,175,81,true,-3.4838,0.71965,2023-10-03T11:48:26.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04259,mir-2003,1287,mir-2003 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,114.0,114.4,57.6,This family represents the microRNA (miRNA) precursor mir-2003 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.59941,-4.03032,1600000,223982,184,87,true,-3.4441,0.71897,2023-10-03T11:54:25.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04260,mir-2005,1287,mir-2005 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,61.4,61.4,61.2,This family represents the microRNA (miRNA) precursor mir-2005 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,2,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.59542,-3.64255,1600000,234050,190,93,true,-3.2359,0.71882,2023-10-03T11:58:39.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04261,MIR1861,1287,MIR1861 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,88.0,88.0,87.9,This family represents the microRNA (miRNA) precursor MIR1861 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,13,207,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.65943,-2.82395,1600000,203546,213,110,true,-3.3751,0.71844,2023-10-03T12:07:13.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04262,MIR1319,1287,MIR1319 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,116.6,116.6,114.3,This family represents the microRNA (miRNA) precursor MIR1319 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,10,0,0,Gene; miRNA;,Predicted; RNAalifold,7,0,,\"\",0.57622,-4.23902,1600000,215086,227,119,true,-2.8307,0.7186,2023-10-03T12:52:40.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04263,MIR2593,1287,MIR2593 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.2,96.2,96.1,This family represents the microRNA (miRNA) precursor MIR2593 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,855,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.58024,-2.59809,1600000,168904,217,116,true,-3.3326,0.71851,2023-10-03T13:06:31.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04264,mir-5408,1287,mir-5408 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,98.4,80.9,This family represents the microRNA (miRNA) precursor mir-5408 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,2,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.55964,-4.36467,1600000,224842,179,84,true,-3.0841,0.72544,2023-10-03T13:35:29.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04265,mir-4650,1287,mir-4650 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.7,80.7,75.9,This family represents the microRNA (miRNA) precursor mir-4650 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,11,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.67035,-2.56341,1600000,210953,196,97,true,-3.1699,0.71863,2023-10-03T13:56:11.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04266,mir-9412,1287,mir-9412 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,70.1,69.7,This family represents the microRNA (miRNA) precursor mir-9412 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,281,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60266,-4.35016,1600000,299507,150,64,true,-3.1192,0.71907,2023-10-03T15:00:52.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04267,mir-362,1287,mir-362 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,82.0,61.2,This family represents the microRNA (miRNA) precursor mir-362 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,14,34,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.69857,-2.75324,1600000,241726,151,65,true,-3.5103,0.71926,2023-10-03T15:18:44.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04268,mir-512,1287,mir-512 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,94.0,94.8,92.5,This family represents the microRNA (miRNA) precursor mir-512 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,16,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.63236,-3.04144,1600000,205668,177,84,true,-3.3659,0.71886,2023-10-03T16:31:30.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04269,mir-373,1287,mir-373 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,69.0,69.4,66.6,This family represents the microRNA (miRNA) precursor mir-373 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,9,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.59493,-4.71295,1600000,327676,157,69,true,-3.795,0.71914,2023-10-03T16:37:47.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04270,mir-743,1287,mir-743 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.1,65.9,This family represents the microRNA (miRNA) precursor mir-743 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,21,0,0,Gene; miRNA;,Predicted; RNAalifold,14,0,,\"\",0.64431,-3.2828,1600000,231975,155,75,true,-3.3423,0.71893,2023-10-16T13:21:21.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04271,mir-1244,1287,mir-1244 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,105.4,105.4,105.3,This family represents the microRNA (miRNA) precursor mir-1244 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,119,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.57569,-3.19686,1600000,220447,178,84,true,-3.4579,0.71888,2023-10-16T15:09:28.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04272,mir-8908,1287,mir-8908 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,74.0,74.2,73.9,This family represents the microRNA (miRNA) precursor mir-8908 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 70.00 -Z 742849.287494 CM SEQDB,36,68,0,0,Gene; miRNA;,Predicted; RNAalifold,12,0,,\"\",0.64114,-4.22048,1600000,215023,168,86,true,-3.4956,0.7187,2023-10-17T11:38:48.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04273,MIR827_2,1287,MIR827_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,67.0,67.0,59.6,This family represents the microRNA (miRNA) precursor MIR827_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,6,0,0,Gene; miRNA;,Predicted; RNAalifold,5,0,,\"\",0.55892,-3.97606,1600000,229887,184,89,true,-2.8137,0.72174,2023-10-17T11:56:17.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04274,mir-9223,1287,mir-9223 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,53.0,53.0,52.9,This family represents the microRNA (miRNA) precursor mir-9223 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,7781,0,0,Gene; miRNA;,Predicted; RNAalifold,9,0,,\"\",0.63778,-3.57427,1600000,218633,164,76,true,-3.5177,0.71921,2023-10-17T15:01:31.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04275,mir-2733,1287,mir-2733 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,72.9,72.9,72.8,This family represents the microRNA (miRNA) precursor mir-2733 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,11,22,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.66179,-2.11331,1600000,237314,163,81,true,-3.4956,0.71892,2023-10-20T13:34:19.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04276,MIR4371,1287,MIR4371 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,110.0,110.6,109.9,This family represents the microRNA (miRNA) precursor MIR4371 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,48,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.65525,-3.69108,1600000,199898,339,142,true,-3.028,0.71845,2023-10-20T16:02:29.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04277,mir-1268,1287,mir-1268 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,83.0,83.3,81.9,This family represents the microRNA (miRNA) precursor mir-1268 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 72.00 -Z 742849.287494 CM SEQDB,2,39,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.58372,-5.30503,1600000,413608,136,52,true,-2.8952,0.73054,2023-10-20T16:58:34.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04278,MIR4372,1287,MIR4372 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,100.0,126.1,93.7,This family represents the microRNA (miRNA) precursor MIR4372 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,2,0,,\"\",0.55113,-4.71818,1600000,223193,230,123,true,-3.0776,0.71829,2023-10-20T17:06:01.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04279,mir-H20,1287,mir-H20 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,99.0,99.0,53.7,This family represents the microRNA (miRNA) precursor mir-H20 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,2,4,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.55858,-5.26644,1600000,190539,175,82,true,-3.411,0.71972,2023-10-23T12:01:39.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04280,mir-890,1287,mir-890 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.0,55.9,This family represents the microRNA (miRNA) precursor mir-890 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,17,122,0,0,Gene; miRNA;,Predicted; RNAalifold,26,0,,\"\",0.67185,-3.29289,1600000,245143,156,74,true,-3.6485,0.71906,2023-10-23T12:23:46.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04281,mir-8791,1287,mir-8791 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,85.0,85.1,84.9,This family represents the microRNA (miRNA) precursor mir-8791 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,7,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.64424,-3.16698,1600000,239655,169,77,true,-2.6993,0.72415,2023-10-23T13:36:35.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04282,mir-507,1287,mir-507 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,79.0,79.0,78.5,This family represents the microRNA (miRNA) precursor mir-507 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,25,55,0,0,Gene; miRNA;,Predicted; RNAalifold,35,0,,\"\",0.64972,-3.41625,1600000,225978,168,82,true,-3.3017,0.71894,2023-10-23T15:30:21.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04283,mir-3135,1287,mir-3135 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,81.1,78.4,This family represents the microRNA (miRNA) precursor mir-3135 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 75.00 -Z 742849.287494 CM SEQDB,2,14,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.57896,-4.86504,1600000,288941,157,68,true,-2.8011,0.72724,2023-10-23T19:13:30.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04284,mir-532,1287,mir-532 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,75.2,75.2,48.4,This family represents the microRNA (miRNA) precursor mir-532 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,35,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.68063,-2.97903,1600000,204386,165,75,true,-3.7282,0.71902,2023-10-23T19:41:07.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04285,mir-660,1287,mir-660 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,73.3,73.3,49.8,This family represents the microRNA (miRNA) precursor mir-660 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,32,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.65675,-3.10305,1600000,234227,167,76,true,-3.7082,0.71897,2023-10-23T19:49:36.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04286,mir-513,1287,mir-513 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,72.1,72.1,71.8,This family represents the microRNA (miRNA) precursor mir-513 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,35,73,0,0,Gene; miRNA;,Predicted; RNAalifold,29,0,,\"\",0.64713,-2.96503,1600000,174726,172,82,true,-3.4942,0.71892,2023-10-23T20:01:34.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04287,mir-9195,1287,mir-9195 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.5,90.5,90.4,This family represents the microRNA (miRNA) precursor mir-9195 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,3,186,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.61768,-3.32675,1600000,173637,182,87,true,-2.9208,0.71885,2023-10-23T20:39:43.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04288,mir-9201,1287,mir-9201 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.5,95.5,95.4,This family represents the microRNA (miRNA) precursor mir-9201 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,6,0,0,0,Gene; miRNA;,Predicted; RNAalifold,1,0,,\"\",0.60322,-4.8533,1600000,175642,173,81,true,-3.7107,0.71895,2023-10-23T20:59:54.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04289,mir-3596,1287,mir-3596 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,96.0,96.4,95.4,This family represents the microRNA (miRNA) precursor mir-3596 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,58,0,0,Gene; miRNA;,Predicted; RNAalifold,34,0,,\"\",0.60562,-3.72645,1600000,212083,176,82,true,-3.1852,0.72195,2023-10-23T22:34:30.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04290,mir-1197,1287,mir-1197 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,80.0,89.3,57.5,This family represents the microRNA (miRNA) precursor mir-1197 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,36,0,0,Gene; miRNA;,Predicted; RNAalifold,39,0,,\"\",0.69753,-2.12329,1600000,184943,179,84,true,-3.6745,0.71899,2023-10-23T23:12:46.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04291,mir-368,1287,mir-368 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,58.0,58.4,57.8,This family represents the microRNA (miRNA) precursor mir-368 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 CM SEQDB,57,164,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.64591,-2.03752,1600000,198909,156,72,true,-3.2462,0.7192,2023-10-23T23:47:36.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04292,mir-379,1287,mir-379 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,56.0,56.1,55.5,This family represents the microRNA (miRNA) precursor mir-379 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 CM SEQDB,60,185,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.65463,-4.03869,1600000,232121,140,64,true,-3.8121,0.71919,2023-10-24T00:00:15.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04293,mir-889,1287,mir-889 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,95.0,96.3,91.9,This family represents the microRNA (miRNA) precursor mir-889 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,7,26,0,0,Gene; miRNA;,Predicted; RNAalifold,28,0,,\"\",0.63706,-2.81709,1600000,211971,172,79,true,-3.6033,0.71896,2023-10-24T00:14:39.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04294,mir-3578,1287,mir-3578 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,78.0,78.4,77.8,This family represents the microRNA (miRNA) precursor mir-3578 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,4,84,0,0,Gene; miRNA;,Predicted; RNAalifold,36,0,,\"\",0.61325,-3.04861,1600000,239731,170,82,true,-3.4384,0.71893,2023-10-24T00:22:25.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04295,mir-329,1287,mir-329 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,76.8,76.8,76.5,This family represents the microRNA (miRNA) precursor mir-329 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 70.00 -Z 742849.287494 CM SEQDB,46,129,0,0,Gene; miRNA;,Predicted; RNAalifold,41,0,,\"\",0.66337,-3.2082,1600000,219322,166,80,true,-3.5857,0.71882,2023-10-24T00:58:23.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04296,mir-485,1287,mir-485 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,70.0,71.6,56.4,This family represents the microRNA (miRNA) precursor mir-485 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 CM SEQDB,13,34,0,0,Gene; miRNA;,Predicted; RNAalifold,38,0,,\"\",0.67175,-3.07769,1600000,244409,163,73,true,-3.8153,0.71911,2023-10-24T01:09:46.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04297,mir-35_2,1287,mir-35_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,41.0,41.7,40.7,This family represents the microRNA (miRNA) precursor mir-35_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,9,25,0,0,Gene; miRNA;,Predicted; RNAalifold,3,0,,\"\",0.6742,-3.13074,1600000,247363,143,62,true,-3.7306,0.71933,2023-10-24T10:55:22.000Z,2023-11-06T17:21:46.000Z\n",
            "RF04298,mir-36_2,1287,mir-36_2 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,46.0,46.4,44.4,This family represents the microRNA (miRNA) precursor mir-36_2 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,19,0,0,Gene; miRNA;,Predicted; RNAalifold,8,0,,\"\",0.64385,-2.89485,1600000,200105,152,64,true,-2.5328,0.74909,2023-10-24T10:56:50.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04299,MIR814,1287,MIR814 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,90.0,90.1,89.9,This family represents the microRNA (miRNA) precursor MIR814 imported from miRBase.,MIR814_2;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 70.00 -Z 742849.287494 CM SEQDB,3,48,0,0,Gene; miRNA;,Predicted; RNAalifold,10,0,,\"\",0.5898,-3.71233,1600000,219783,176,83,true,-3.5402,0.71903,2023-10-24T12:12:18.000Z,2023-11-06T16:44:06.000Z\n",
            "RF04300,mir-39,1287,mir-39 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.3,56.2,This family represents the microRNA (miRNA) precursor mir-39 imported from miRBase.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,5,0,0,Gene; miRNA;,Predicted; RNAalifold,4,0,,\"\",0.58968,-4.07156,1600000,231323,159,74,true,-3.3292,0.71919,2023-10-24T15:20:31.000Z,2023-11-06T17:21:46.000Z\n",
            "RF00170,msr,2500,Retron msr RNA,Griffiths-Jones SR,Griffiths-Jones SR,52.0,52.1,50.3,\"Retron elements are found in a wide range of mainly Gram-negative bacteria. The retron encodes a reverse transcriptase that is responsible for the production of a small satellite DNA called msDNA (multi-copy single-stranded DNA). msDNA has both DNA (encoded by the retron msd gene) and RNA (encoded by the msr gene) components. The 5' end of msDNA is linked to an internal guanosine residue of the RNA strand by a 2'-5' linkage. The components of msDNA are poorly conserved in sequence between different organisms -- this family therefore represents only a subset of known msr genes. Despite considerable investigation, very little is known about the function of msDNA, the mobility of retron elements, or their effect on the host cell.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,15,0,0,Gene;,Predicted; PFOLD,21,0,,\"\",0.55457,-4.19244,1600000,214717,180,69,true,-3.8304,0.71899,2013-10-03T23:42:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02156,NPPA-AS1_1,2501,NPPA antisense RNA 1 conserved region 1,Eberhardt R,Predicted; CMfinder; Eberhardt R,45.0,47.2,39.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.25 -Z 549862.597050 CM SEQDB,19,26,0,0,Gene; lncRNA;,Predicted; CMfinder;,31,0,,\"\",0.45247,-14.47959,1600000,460612,288,176,false,-3.4499,0.71472,2013-10-04T03:23:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02157,NPPA-AS1_2,2501,NPPA antisense RNA 1 conserved region 2,Eberhardt R,Predicted; CMfinder; Eberhardt R,60.0,64.2,56.7,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.11 -Z 549862.597050 CM SEQDB,16,19,0,0,Gene; lncRNA;,Predicted; CMfinder;,23,0,,\"\",0.43814,-11.29614,1600000,275175,257,150,false,-3.2517,0.71776,2013-10-04T03:23:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02158,NPPA-AS1_3,2501,NPPA antisense RNA 1 conserved region 3,Eberhardt R,Predicted; CMfinder; Eberhardt R,55.0,55.7,43.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.11 -Z 549862.597050 CM SEQDB,24,32,0,0,Gene; lncRNA;,Predicted; CMfinder;,40,0,,\"\",0.61425,-8.18859,1600000,351221,197,108,false,-3.1193,0.71819,2013-10-04T03:23:34.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02197,TP73-AS1,2502,TP73 antisense RNA 1 conserved region,Eberhardt R,Predicted; CMfinder,75.0,82.5,56.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.98 -Z 549862.597050 CM SEQDB,4,9,0,0,Gene; lncRNA;,Predicted; CMfinder,10,0,,\"\",0.49436,-12.52544,1600000,327496,276,161,false,-3.1715,0.71666,2013-10-04T03:24:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02202,UCA1,2503,Urothelial cancer associated 1 conserved region,Eberhardt R,Predicted; CMfinder,80.0,93.5,32.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.22 -Z 549862.597050 CM SEQDB,5,8,0,0,Gene; lncRNA;,Predicted; CMfinder,8,0,,\"\",0.62144,-8.88875,1600000,388691,163,75,false,-3.5098,0.71869,2013-10-04T03:25:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02154,NCRUPAR_1,2504,\"Non-protein coding RNA, upstream of F2R/PAR1 conserved region 1\",Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,50.1,44.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 40.56 -Z 549862.597050 CM SEQDB,9,26,0,0,Gene; lncRNA;,Predicted; CMfinder;,29,0,,\"\",0.35473,-11.52162,1600000,307383,247,140,false,-2.5663,0.71788,2013-10-04T03:23:27.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02155,NCRUPAR_2,2504,\"Non-protein coding RNA, upstream of F2R/PAR1 conserved region 2\",Eberhardt R,Predicted; CMfinder; Eberhardt R,80.0,93.9,45.5,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 38.02 -Z 549862.597050 CM SEQDB,3,9,0,0,Gene; lncRNA;,Predicted; CMfinder;,9,0,,\"\",0.3757,-11.22416,1600000,315463,197,99,false,-2.7712,0.7183,2013-10-04T03:23:28.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00662,mir-132,2505,microRNA mir-132,Griffiths-Jones SR,Griffiths-Jones SR,52.0,52.1,51.8,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,50,146,0,0,Gene; miRNA;,Predicted; RNAalifold,68,0,,\"\",0.59362,-4.84845,1600000,117511,168,84,true,-4.1821,0.71884,2013-10-04T01:08:13.000Z,2023-11-06T16:44:07.000Z\n",
            "RF00057,RyhB,2506,RyhB RNA,\"Bateman A, Moxon SJ\",Bateman A,55.0,55.2,54.8,\"The RyhB 90-nt RNA down-regulates a set of iron-storage and iron-using proteins when iron is limiting; it is itself negatively regulated by the ferric uptake repressor protein, Fur (Ferric uptake regulator). RyhB RNA levels are inversely correlated with mRNA levels for the sdhCDAB operon, encoding succinate dehydrogenase, as well as five other genes previously shown to be positively regulated by Fur by an unknown mechanism. These include two other genes encoding enzymes in the tricarboxylic acid cycle, acnA and fumA, two ferritin genes, ftnA and bfr, and a gene for superoxide dismutase, sodB [1]. This ncRNA gene was also identified in a screen and called SraI and it was found to be expressed only in stationary phase [2].First sRNA shown to mediate persistence to antibiotics in E.coli.[5] It may act both as an RNA-RNA interaction based regulator and as a transcript encoding for a small protein.[6]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.95 -Z 549862.597050 CM SEQDB,26,261,0,0,Gene; sRNA;,Predicted; PFOLD; Moxon SJ,173,0,,\"\",0.54406,-6.58276,1600000,244407,143,67,true,-3.6066,0.71901,2013-10-03T23:22:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02098,DGCR5,2507,DiGeorge syndrome critical region gene 5 conserved region,Eberhardt R,Predicted; CMfinder; Eberhardt R,50.0,60.1,38.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.91 -Z 549862.597050 CM SEQDB,4,13,0,0,Gene; lncRNA;,Predicted; CMfinder;,10,0,,\"\",0.51705,-9.65794,1600000,472889,123,49,false,-3.2332,0.72014,2013-10-04T03:21:37.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00255,mir-218,2508,mir-218 microRNA precursor family,Griffiths-Jones SR,Griffiths-Jones SR,62.0,62.4,60.2,miR-218 appears to be a vertebrate specific microRNA and has now been been predicted and experimentally confirmed in a wide range of vertebrate species (MIR:MIPF0000026). The extents of the hairpin precursors are not known. In this case the mature sequence in excised from the 5'arm of the hairpin.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,74,231,0,0,Gene; miRNA;,Predicted; RNAalifold,123,0,,\"\",0.71328,-2.46295,1600000,182986,167,82,true,-3.8322,0.71891,2013-10-04T00:24:39.000Z,2021-11-19T10:36:03.000Z\n",
            "RF00360,snoZ107_R87,2509,Small nucleolar RNA Z107/R87,Moxon SJ,Moxon SJ,50.0,50.2,49.8,Z107 and R87 are members of the C/D class of snoRNA which contain the C (UGAUGA) and D (CUGA) box motifs. Most of the members of the box C/D family function in directing site-specific 2'-O-methylation of substrate RNA,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19 -Z 549862.597050 CM SEQDB,9,258,0,0,Gene; snRNA; snoRNA; CD-box;,\"Predicted; RNAfold; Moxon SJ, Daub J, Gardner PP\",76,0,,\"\",0.61981,-5.21848,1600000,279426,217,117,true,-3.2606,0.71808,2013-10-04T00:34:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02698,Avalong_thermometer,2510,Avalong 5' UTR thermometer,Argasinska J,Argasinska J,154.0,154.8,50.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Cis-reg; thermoregulator;,Predicted; PMID:24755616;,2,0,,\"\",0.60655,-4.41643,1600000,203277,244,131,true,-2.8435,0.71815,2017-02-06T11:16:00.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02699,Avashort_thermometer,2510,Avashort 5' UTR thermometer,Argasinska J,Argasinska J,88.0,88.6,41.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,0,0,0,Cis-reg; thermoregulator;,Predicted; PMID:24755616;,2,0,,\"\",0.54375,-8.3342,1600000,439107,145,59,true,-2.7417,0.72764,2017-02-06T11:16:32.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02704,LcrF_thermometer,2511,LcrF intergenic thermometer,Argasinska J,Argasinska J,169.0,169.6,43.5,Intergenic two hairpin RNA thermometer (usually RNAT are in the 5'UTR). Together with the termo-labile YmoA protein it activates synthesis of the most crucial virulence activator LcrF (VirF). The sequence is 100% identical in all human pathogenic Yersinia species suggesting the importance of this structure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Cis-reg; thermoregulator;,Predicted; PMID:22359501;,3,0,,\"\",0.5199,-6.02646,1600000,239780,247,133,true,-2.7877,0.7181,2017-02-20T10:33:34.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02705,B_rapa_snoR775,2512,Brassica rapa small nucleolar RNA 775,Argasinska J,Argasinska J,64.0,64.1,55.8,Name based on close proximity to miR775. Precursors of both are encoded by a single gene sno-miR775.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,20,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 26607788;,8,0,,\"\",0.46337,-7.45507,1600000,293961,239,127,true,-2.9664,0.71822,2017-02-28T14:28:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02706,TeloSII_ncR30,2513,TeloSII non coding RNA 30,Argasinska J,Argasinska J,86.0,86.8,41.3,Located downstream of the TeloSII elements. These elements have been shown to coordinate the expression of protein-coding gene related to ribosome biogenesis throughout the cell cycle in Arabidopsis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,14,0,0,Gene; sRNA;,Predicted; RNAfold,7,0,,\"\",0.58191,-10.61665,1600000,470911,364,226,true,-3.5201,0.71002,2017-02-28T14:33:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02707,TeloSII_ncR33,2513,TeloSII non coding RNA 33,Argasinska J,Argasinska J,105.0,105.3,72.0,Located downstream of the TeloSII elements. These elements have been shown to coordinate the expression of protein-coding gene related to ribosome biogenesis throughout the cell cycle in Arabidopsis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,18,0,0,Gene; sRNA;,Predicted; RNAfold,8,0,,\"\",0.57894,-9.29986,1600000,436454,450,297,true,-3.5158,0.7061,2017-02-28T14:34:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02708,TeloSII_ncR40,2513,TeloSII non coding RNA 40,Argasinska J,Argasinska J,186.0,186.5,40.3,Located downstream of the TeloSII elements. These elements have been shown to coordinate the expression of protein-coding gene related to ribosome biogenesis throughout the cell cycle in Arabidopsis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.57548,-8.04341,1600000,283881,285,162,true,-3.3154,0.71675,2017-02-28T14:42:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02709,TeloSII_ncR43,2513,TeloSII non coding RNA 43,Argasinska J,Argasinska J,298.0,298.4,41.9,Located downstream of the TeloSII elements. These elements have been shown to coordinate the expression of protein-coding gene related to ribosome biogenesis throughout the cell cycle in Arabidopsis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,2,0,0,Gene; sRNA;,Predicted; RNAfold,1,0,,\"\",0.6192,-9.71358,1600000,441333,448,277,true,-3.635,0.70695,2017-02-28T14:45:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02710,TeloSII_ncR45,2513,TeloSII non coding RNA 45,Argasinska J,Argasinska J,101.0,101.6,39.5,Located downstream of the TeloSII elements. These elements have been shown to coordinate the expression of protein-coding gene related to ribosome biogenesis throughout the cell cycle in Arabidopsis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,1,0,,\"\",0.58535,-3.52074,1600000,235219,159,69,true,-3.4466,0.72238,2017-02-28T14:45:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02711,TeloSII_ncR49,2513,TeloSII non coding RNA 49,Argasinska J,Argasinska J,55.0,55.1,53.6,Located downstream of the TeloSII elements. These elements have been shown to coordinate the expression of protein-coding gene related to ribosome biogenesis throughout the cell cycle in Arabidopsis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; RNAfold,2,0,,\"\",0.51393,-3.03199,1600000,172148,184,89,true,-3.4335,0.71864,2017-02-28T14:46:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02715,sno_ncR1,2513,Small nucleolar RNA ncR1,Argasinska J,Argasinska J,94.0,108.4,37.7,Type H/ACA. Orphan.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,7,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; RNAfold,2,0,,\"\",0.50223,-6.48594,1600000,199302,215,111,true,-3.3389,0.7184,2017-03-07T16:01:46.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02716,sno_ncR3,2513,Small nucleolar RNA ncR3,Argasinska J,Argasinska J,88.0,88.5,48.5,Type C/D. Target 18S.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,1,0,,\"\",0.63068,-5.50363,1600000,341680,163,75,true,-3.9019,0.71882,2017-03-07T16:20:09.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02717,sno_ncR4,2513,Small nucleolar RNA ncR4,Argasinska J,Argasinska J,103.0,103.9,34.1,Type C/D. Target 18S.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,1,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,2,0,,\"\",0.68848,-2.94481,1600000,246077,175,82,true,-3.5964,0.71881,2017-03-07T16:23:22.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02718,sca_ncR14,2513,Small Cajal body specific RNA ncR14,Argasinska J,Argasinska J,62.0,62.4,32.2,Type C/D. Putative target U6.26.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,12,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold,8,0,,\"\",0.67299,-4.03413,1600000,227993,242,131,true,-3.2003,0.71838,2017-03-07T16:24:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02719,sca_ncR20,2513,Small Cajal body specific RNA ncR20,Argasinska J,Argasinska J,127.0,127.0,37.2,Type C/D. Putative target U6-29.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold,1,0,,\"\",0.46069,-8.91491,1600000,373678,208,104,true,-3.3006,0.71848,2017-03-07T16:25:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02720,sca_ncR21,2513,Small Cajal body specific RNA ncR21,Argasinska J,Argasinska J,113.0,113.4,39.1,Type C/D. Putative target U2.4.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,3,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold,2,0,,\"\",0.5734,-3.3115,1600000,239849,216,111,true,-3.3784,0.71847,2017-03-07T16:25:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02721,sca_ncR26,2513,Small Cajal body specific RNA ncR26,Argasinska J,Argasinska J,37.0,37.2,36.6,Type C/D. Putative target U6.1.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,12,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold,9,0,,\"\",0.54944,-4.0389,1600000,232690,157,68,true,-3.0913,0.72636,2017-03-07T16:26:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02722,sca_ncR27,2513,Small Cajal body specific RNA ncR27,Argasinska J,Argasinska J,91.0,91.9,36.1,Type C/D. Putative target U6.1.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,3,0,0,Gene; snRNA; snoRNA; scaRNA;,Predicted; RNAfold,5,0,,\"\",0.58961,-5.42754,1600000,283346,174,83,true,-3.7329,0.71877,2017-03-07T16:26:30.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02728,HrrF,2514,Haemophilus regulatory RNA responsive to iron,Argasinska J,\"Santana et al, PMID:25157846\",77.0,77.0,45.1,\"Involved in iron homeostasis in Haemophilus species. Orthologues exist only among other Pasteurellacae. It is an analog to PrrF and RyhB RNAs. HrrF is maximally expressed when iron levels are low. Ferric uptake regulator (Fur) binds upstream of the hrrF promoter. HrrF stability is not dependant on the RNA chaperone Hfq. RNA-seq has shown that that HrrF targets are mRNAs of genes whose products are involved in molybdate uptake, deoxyribonucleotide synthesis, and amino acid synthesis.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,7,13,0,0,Gene; sRNA;,Predicted; CMfinder,17,0,,\"\",0.50168,-8.66264,1600000,356755,340,116,true,-3.1154,0.71848,2017-03-14T16:52:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02729,JA01,2515,Aggregatibacter sRNA JA01,Argasinska J,Argasinska J,111.0,111.8,42.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,4,0,,\"\",0.50922,-7.37652,1600000,308676,274,153,true,-2.733,0.71802,2017-03-14T16:53:47.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02730,JA02,2515,Aggregatibacter sRNA JA02,Argasinska J,Argasinska J,49.0,49.0,48.0,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,2,0,0,Gene; sRNA;,Predicted; RNAfold,6,0,,\"\",0.44911,-8.20066,1600000,358613,222,116,true,-2.7378,0.71829,2017-03-14T16:54:05.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02731,JA03,2515,Aggregatibacter sRNA JA03,Argasinska J,Argasinska J,44.0,44.3,43.6,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,13,0,0,Gene; sRNA;,Predicted; RNAfold,15,0,,\"\",0.62302,-8.00402,1600000,382318,423,274,true,-3.5742,0.70713,2017-03-14T16:54:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02732,JA04,2515,Aggregatibacter sRNA JA04,Argasinska J,Argasinska J,81.0,81.4,65.1,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,6,14,0,0,Gene; sRNA;,Predicted; RNAfold,14,0,,\"\",0.485,-6.49589,1600000,347056,171,81,true,-2.9743,0.71867,2017-03-14T16:56:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02734,Cgb105,2516,Corynebacterium sRNA 105,Argasinska J,Argasinska J,163.0,163.5,32.2,Absent under heat stress conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,2,0,,\"\",0.69046,-4.8879,1600000,225636,249,134,true,-3.3549,0.71838,2017-03-29T16:23:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02735,Sernc350,2517,Saccharopolyspora sRNA 350,Argasinska J,Argasinska J,50.0,50.6,39.3,Probably regulates terpene metabolism.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,463,0,0,Gene; antisense;,Predicted; RNAfold;,278,0,,\"\",0.64207,-9.1128,1600000,310051,698,508,true,-4.0272,0.70074,2017-03-29T16:39:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02743,Sernc389,2517,Saccharopolyspora sRNA 389,Argasinska J,Argasinska J,74.0,74.2,44.9,Probably regulates polyketide biosynthesis.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,2,33,0,0,Gene; antisense;,Predicted; RNAfold;,21,0,,\"\",0.65349,-8.95659,1600000,311262,801,595,true,-4.3344,0.69962,2017-04-24T11:08:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02736,Rev11,2518,Soft rot Enterobacteriaceae Rev 11 asRNA,Argasinska J,Argasinska J,91.3,91.3,91.2,From bacteria cultured under starvation conditions. Antisense to ECA3028.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,46,0,0,Gene; antisense;,Predicted; RNAfold,47,0,,\"\",0.7587,-7.5896,1600000,319266,593,417,true,-4.3477,0.70234,2017-04-18T12:32:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02737,Rev13,2518,Soft rot Enterobacteriaceae Rev 13 asRNA,Argasinska J,Argasinska J,242.0,242.6,50.9,From bacteria cultured under starvation conditions. Antisense to ECA0388.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; antisense;,Predicted; RNAfold,3,0,,\"\",0.68175,-7.64879,1600000,335908,515,354,true,-4.0993,0.70397,2017-04-18T12:52:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02738,Rev24,2518,Soft rot Enterobacteriaceae Rev 24 asRNA,Argasinska J,Argasinska J,246.0,246.5,230.3,From bacteria cultured under starvation conditions. Antisense to rcsC.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,7,0,0,Gene; antisense;,Predicted; RNAfold,9,0,,\"\",0.67172,-8.63001,1600000,328704,674,489,true,-4.4568,0.70098,2017-04-18T12:54:13.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02739,Rev41,2518,Soft rot Enterobacteriaceae Rev 41 sRNA,Argasinska J,Argasinska J,535.0,535.2,145.0,From bacteria cultured under starvation conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.71393,-8.71468,1600000,353446,667,480,true,-4.5213,0.70119,2017-04-18T13:59:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02740,Fwd6_3p_UTR,2518,Soft rot Enterobacteriaceae Fwd 6 3'UTR,Argasinska J,Argasinska J,79.0,79.9,38.8,From bacteria cultured under starvation conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,5,0,0,Cis-reg;,Predicted; RNAfold,7,0,,\"\",0.44366,-8.8652,1600000,363204,224,116,true,-2.931,0.71828,2017-04-18T14:27:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02741,Rev44,2518,Soft rot Enterobacteriaceae Rev 44 asRNA,Argasinska J,Argasinska J,178.0,178.1,33.5,From bacteria cultured under starvation conditions. Antisense to ECA0910.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,2,0,0,Gene; antisense;,Predicted; RNAfold,4,0,,\"\",0.68868,-8.86677,1600000,358632,498,336,true,-3.8248,0.7046,2017-04-18T14:30:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02742,Rev72,2518,Soft rot Enterobacteriaceae Rev 72 asRNA,Argasinska J,Argasinska J,289.0,289.6,63.3,From bacteria cultured under starvation conditions. Antisense to gudP.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,1,0,0,Gene; antisense;,Predicted; RNAfold,3,0,,\"\",0.72966,-8.44619,1600000,402990,602,426,true,-4.2183,0.70213,2017-04-18T14:31:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02744,Rev39_5p_UTR,2518,Soft rot Enterobacteriaceae Rev 39 5'UTR,Argasinska J,Argasinska J,88.0,88.6,86.3,From bacteria cultured under starvation conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,4,113,0,0,Cis-reg;,Predicted; RNAfold,113,0,,\"\",0.76526,-7.24521,1600000,353540,454,300,true,-3.9044,0.70596,2017-04-24T13:52:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02745,Rev42_5p_UTR,2518,Soft rot Enterobacteriaceae Rev 42 5'UTR,Argasinska J,Argasinska J,229.0,229.4,129.9,From bacteria cultured under starvation conditions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,3,3,0,0,Cis-reg;,Predicted; RNAfold,5,0,,\"\",0.73693,-7.83576,1600000,338752,921,443,true,-4.7016,0.70181,2017-04-24T13:55:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02746,AsrC,2519,antisense RNA of rseC mRNA,Argasinska J,Argasinska J,111.0,111.5,110.3,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,7,61,0,0,Gene; antisense;,Predicted; RNAfold,63,0,,\"\",0.75025,-8.09528,1600000,297923,1200,886,true,-5.3977,0.69748,2017-04-24T14:23:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02747,FtrA,2520,Francisella sRNA A,Argasinska J,Argasinska J,134.0,134.2,41.1,\"Deletion led to significant changes in the expression of several mRNAs. However, it did not alter Francisella's growth under stress condition or its ability to induce disease in a mouse model.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,1,0,0,Gene; sRNA;,Predicted; RNAfold,6,0,,\"\",0.46068,-8.03468,1600000,321304,217,111,true,-3.1176,0.71828,2017-04-24T14:52:09.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02748,FtrB,2520,Francisella sRNA B,Argasinska J,Argasinska J,66.0,66.0,53.7,\"Deletion led to significant changes in the expression of several mRNAs. However, it did not alter Francisella's growth under stress condition or its ability to induce disease in a mouse model.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 549862.597050 --mxsize 128 CM SEQDB,5,3,0,0,Gene; sRNA;,Predicted; RNAfold,6,0,,\"\",0.49689,-7.65678,1600000,296268,221,115,true,-3.1165,0.71834,2017-04-24T14:52:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02763,IsrM,2521,IsrM sRNA,Argasinska J,Argasinska J,376.0,376.2,115.4,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold,2,0,,\"\",0.71712,-8.53504,1600000,434805,488,329,true,-4.1717,0.70482,2017-05-17T10:28:12.000Z,2022-05-06T12:43:55.000Z\n",
            "RF00233,Tymo_tRNA-like,2522,Tymovirus/Pomovirus/Furovirus tRNA-like 3' UTR element,Moxon SJ,PMID:15033563,40.0,41.4,38.8,\"This family represents a tRNA-like structure found in the 3' UTR of Tymoviruses, Pomoviruses and Furovirus. This family (in conjunction with Rfam:RF00390 and a 5'-cap) are known to enhance translation [1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21 -Z 549862.597050 CM SEQDB,28,23,0,0,Cis-reg;,Published; PMID:15033563,24,0,,\"\",0.56863,-9.00191,1600000,403383,165,84,true,-3.5334,0.71876,2013-10-04T00:21:36.000Z,2019-01-04T15:01:52.000Z\n",
            "RF01408,sraL,2523,sraL Hfq binding RNA,\"Daub J, Gardner PP\",Predicted; ClustalW; Infernal,39.0,39.5,37.8,\"The SraL RNA ('sra' for small RNA), discovered in E. coli,[2][3] and later in Salmonella Tiphimurium.[5][6] It is expressed only in stationary phase.[2] It may possibly play a role in Salmonella virulence.[5] The major stationary phase regulator RpoS is transcriptionally regulating SraL and directly binds to the sraL gene promoter. SraL down-regulates the expression of of the ribosome-associated chaperone Trigger Factor (TF), which is involved in the folding of the newly synthesised cystolic proteins.[7]\",ryjA,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,6,43,0,0,Gene; sRNA;,Predicted; RNAalifold,44,0,,\"\",0.58821,-9.31554,1600000,366764,267,142,true,-2.9824,0.71822,2013-10-04T02:33:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02786,snoTBR2,2524,Small nucleolar RNA TBR2,Argasinska J,Argasinska J,62.0,62.7,56.2,\"TBR4 was one of seventeen snoRNAs identified in Trypanosoma brucei.[2] Together with TBR12, TBR6 and TBR4 it is a part of snoRNA gene cluster that is tandemly repeated. These 4 snoRNA genes are transcribed as a polycistronic RNA transcript.[1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,49,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,9,0,,\"\",0.57946,-5.13491,1600000,207666,207,109,true,-3.0772,0.7187,2017-10-04T14:46:19.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02787,snoTBR4,2525,Small nucleolar RNA TBR4,Argasinska J,Argasinska J,83.0,83.4,77.6,\"TBR4 was one of seventeen snoRNAs identified in Trypanosoma brucei.[2] Together with TBR12, TBR6 and TBR2 it is a part of snoRNA gene cluster that is tandemly repeated. These 4 snoRNA genes are transcribed as a polycistronic RNA transcript.[1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,24,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,5,0,,\"\",0.50076,-8.23112,1600000,288630,198,99,true,-3.5006,0.71858,2017-10-04T14:47:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02788,snoTBR6,2526,Small nucleolar RNA TBR6,Argasinska J,Argasinska J,37.6,37.6,37.1,\"TBR6 was one of seventeen snoRNAs identified in Trypanosoma brucei.[2] Together with TBR2, TBR12 and TBR4 it is a part of snoRNA gene cluster that is tandemly repeated. These 4 snoRNA genes are transcribed as a polycistronic RNA transcript.[1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 605271.01655 --mxsize 128 CM SEQDB,3,52,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,10,0,,\"\",0.50214,-8.32377,1600000,279410,196,83,true,-3.5826,0.7187,2017-10-04T14:47:25.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02789,snoTBR12,2527,Small nucleolar RNA TBR12,Argasinska J,Argasinska J,77.0,77.5,34.9,\"TBR12 was one of seventeen snoRNAs identified in Trypanosoma brucei.[2] Together with TBR2, TBR6 and TBR4 it is a part of snoRNA gene cluster that is tandemly repeated. These 4 snoRNA genes are transcribed as a polycistronic RNA transcript.[1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 451031.997884 --mxsize 128 CM SEQDB,3,31,0,0,Gene; snRNA; snoRNA; CD-box;,Predicted; RNAfold,6,0,,\"\",0.68049,-2.01241,1600000,166632,200,101,true,-3.6423,0.71864,2017-10-04T14:47:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02790,sodF_sRNA,2528,sodF sRNA,Argasinska J,Argasinska J,46.0,46.1,45.0,\"Identified in Streptomyces coelicolor. It is produced from sodF mRNA by cleavage of about 90 nucleotides from its 3'UTR. However, it does not affect the function of sodF mRNA, but acts on another mRNA called sodN. s-SodF RNA has a sequence complimentary to sodN mRNA from the 5'-end up to the ribosome binding site. It pairs with sodN mRNA, blocks its translation and facilitates sodN mRNA decay.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,6,169,0,0,Gene; sRNA;,Predicted; 24234448;,159,0,,\"\",0.46947,-9.36293,1600000,318078,185,90,true,-2.6612,0.71885,2017-10-11T14:22:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02792,sRNA162,2529,Archaeal Small RNA 162,Argasinska J,Argasinska J,78.0,78.4,57.3,Acts on cis and trans-encoded mRNAs.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 605271.01655 --mxsize 128 CM SEQDB,3,5,0,0,Gene; antisense;,Predicted; 22965121;,3,0,,\"\",0.52231,-8.86541,1600000,383063,314,190,true,-3.0982,0.71339,2017-10-16T10:51:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02793,asR3,2530,Pyrobaculum asRNA3,Argasinska J,Argasinska J,46.0,46.2,37.2,Binds to the 3'-end of the tpi gene (triose-phosphate-isomerse). The tpi mRNA has a conserved structural element located close to the stop codon. Binding of the asR3 may be able to compete against the formation of the tpi element structure and this may modulate the function of this highly conserved element.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,4,0,0,Gene; antisense;,Predicted; 22783241;,4,0,,\"\",0.4865,-10.32892,1600000,440082,138,57,true,-3.1532,0.72362,2017-10-16T10:55:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02794,Pab19,2531,Pab19 RNA,Argasinska J,Argasinska J,55.0,55.5,44.6,By using various computational and experimental approaches in three Pyrococcus genomes seven H/ACA sRNAs and 15 pseudouridine resides on rRNA were identified. One H/ACA motif was shown to guide up to three distinct pseudouridylations.[1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,12,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 18304947;,12,0,,\"\",0.52527,-6.32276,1600000,254485,203,102,true,-3.1454,0.71865,2017-10-18T12:38:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02795,Pab91,2531,Pab91 RNA,Argasinska J,Argasinska J,71.0,71.1,49.1,By using various computational and experimental approaches in three Pyrococcus genomes seven H/ACA sRNAs and 15 pseudouridine resides on rRNA were identified. One H/ACA motif was shown to guide up to three distinct pseudouridylations.[1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,7,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 18304947;,7,0,,\"\",0.57694,-5.60656,1600000,337540,139,57,true,-3.3795,0.71938,2017-10-18T12:40:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02796,Pab160,2531,Pab160 RNA,Argasinska J,Argasinska J,47.4,47.4,37.0,By using various computational and experimental approaches in three Pyrococcus genomes seven H/ACA sRNAs and 15 pseudouridine resides on rRNA were identified. One H/ACA motif was shown to guide up to three distinct pseudouridylations [1]. The structure of Pab160 RNA in complex with ribonucleoprotein pseudouridine synthase was reported in [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,5,9,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 18304947;,14,0,,\"\",0.52935,-5.7699,1600000,295457,151,68,true,-3.4642,0.71924,2017-10-18T12:41:15.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02801,PyrR201,2531,Pyrobaculum sRNA 201,Argasinska J,Argasinska J,49.0,49.1,37.0,Has weak conservation of the canonical pseudouridylation guide sRNA features among the Pyrobaculum species.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,8,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 22282340,11,0,,\"\",0.55999,-4.89966,1600000,258009,154,68,true,-3.26,0.71913,2017-10-31T16:23:02.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02802,PyrR204,2531,Pyrobaculum sRNA 204,Argasinska J,Argasinska J,71.0,71.4,37.7,Non canonical pseudouridylation guide sRNA features. Predicted to target 23S U2023 in Pyrobaculum.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 22282340,6,0,,\"\",0.6358,-4.71636,1600000,318894,134,57,true,-3.346,0.71929,2017-10-31T16:24:01.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02803,PyrR205,2531,Pyrobaculum sRNA 205,Argasinska J,Argasinska J,35.6,35.6,35.4,Non canonical pseudouridylation guide sRNA features. Predicted to target 23S U2029 in Pyrobaculum.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,35,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 22282340,8,0,,\"\",0.60741,-6.62628,1600000,285055,143,61,true,-3.4674,0.71915,2017-10-31T16:24:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02804,PyrR206,2531,Pyrobaculum sRNA 206,Argasinska J,Argasinska J,46.0,46.5,39.2,Non canonical pseudouridylation guide sRNA features. Predicted to target 23S U1902 in Pyrobaculum.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 22282340,6,0,,\"\",0.50154,-9.34219,1600000,407977,143,61,true,-3.3639,0.71922,2017-10-31T16:25:07.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02805,PyrR207,2531,Pyrobaculum sRNA 207,Argasinska J,Argasinska J,60.0,60.8,42.1,Non canonical pseudouridylation guide sRNA features. Predicted to target 23S U2597 in Pyrobaculum.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,3,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 22282340,5,0,,\"\",0.59432,-5.54977,1600000,354753,149,65,true,-3.1022,0.71916,2017-10-31T16:25:29.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02806,PyrR208,2531,Pyrobaculum sRNA 208,Argasinska J,Argasinska J,64.0,64.7,36.4,Non canonical pseudouridylation guide sRNA features. Predicted to target 23S U1978 in Pyrobaculum.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,2,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 22282340,5,0,,\"\",0.54119,-8.6122,1600000,277953,139,60,true,-3.4528,0.72101,2017-10-31T16:25:45.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02807,PyrR209,2531,Pyrobaculum sRNA 209,Argasinska J,Argasinska J,42.0,42.6,35.8,Non canonical pseudouridylation guide sRNA features. Predicted to target 16S U8 in Pyrobaculum.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,5,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 22282340,7,0,,\"\",0.56714,-7.07813,1600000,443888,146,65,true,-3.7787,0.71905,2017-10-31T16:28:10.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02808,PyrR210,2531,Pyrobaculum sRNA 210,Argasinska J,Argasinska J,59.0,59.6,33.6,Non canonical pseudouridylation guide sRNA features. Predicted to target tRNALys U54 in Pyrobaculum.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,6,4,0,0,Gene; snRNA; snoRNA; HACA-box;,Predicted; 22282340,8,0,,\"\",0.5379,-9.03253,1600000,410123,138,59,true,-3.2787,0.71919,2017-10-31T16:28:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF00511,IRES_KSHV,2532,Kaposi's sarcoma-associated herpesvirus internal ribosome entry site,Moxon SJ,\"Published; 11160685, INFERNAL\",100.0,317.1,30.1,\"This family represents the Kaposi's sarcoma-associated herpesvirus (KSHV) internal ribosome entry site (IRES) present in the vCyclin gene. The vCyclin and vFLIP coding sequences are present on a bicistronic transcript and it is thought the IRES may initiate translation of vFLIP from this bicistronic transcript [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -E 1000 -Z 549862.597050 CM SEQDB,5,1,0,0,Cis-reg; IRES;,Published; PMID:11160685,2,0,,\"\",0.69307,-8.84376,1600000,406567,393,248,true,-3.7486,0.70856,2013-10-04T00:52:21.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02809,RsmW,2534,RsmW RNA family,Argasinska J,Argasinska J,52.0,52.4,46.6,\"Specifically binds to RsmA protein in vitro, restores biofilm production (possibly due to the interaction with RsmA) and partially complements the loss of RsmY and RsmZ in rsmY/rsmZ double mutant in regards to their contribution to swarming. Compared to RsmY and RsmZ its production is induced in high temperatures and rsmW is not transcriptionally activated by GacA.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; 27430253,7,0,,\"\",0.47298,-11.18155,1600000,334419,227,124,true,-2.9843,0.71844,2017-11-07T16:08:44.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02810,Lst_thermometer,2535,Lst 5' UTR thermometer,Argasinska J,Argasinska J,126.0,126.4,45.3,\"Three independent RNA thermosensors were identified in the 5'UTRs of genes needed for: capsule biosynthesis (cssA), the expression of factor H binding protein (fHbp) and sialylation of lipopolysaccharide, which is essential for bacterial resistance against immune killing (lst). The very different nucleotide sequence and predicted inhibitory structures of the three RNATs indicate that they have evolved independently.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,2,0,0,Cis-reg; thermoregulator;,Predicted; 24067614,2,0,,\"\",0.60372,-4.2396,1600000,231782,222,115,true,-3.1753,0.71835,2017-11-20T10:17:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02811,FHbp_thermometer,2535,FHbp 5' UTR thermometer,Argasinska J,Argasinska J,104.0,104.8,37.6,\"Three independent RNA thermosensors were identified in the 5'UTRs of genes needed for: capsule biosynthesis (cssA), the expression of factor H binding protein (fHbp) and sialylation of lipopolysaccharide, which is essential for bacterial resistance against immune killing (lst). The very different nucleotide sequence and predicted inhibitory structures of the three RNATs indicate that they have evolved independently.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,2,0,0,Cis-reg; thermoregulator;,Predicted; 24067614,2,0,,\"\",0.52293,-5.55197,1600000,314089,164,73,true,-3.4811,0.72142,2017-11-20T10:18:56.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02812,asponA,2536,anti ponA RNA,Argasinska J,Argasinska J,113.0,113.6,85.7,Transcribed antisense to the penicillin-binding protein 1A gene called ponA. It was identified by RNAseq and the expression was validated by 5' and 3' RACE experiments. AsponA expression was up or down regulated under different antibiotic stress. Due to its location it may be able to prevent the transcription or translation of the opposite gene.[1] Study by Wurtzel et al. and Ferrara et al. also detected its expression.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,73,0,0,Gene; antisense;,Predicted; RNAfold,72,0,,\"\",0.74022,-8.34583,1600000,388314,447,295,true,-3.8781,0.70621,2017-11-20T10:22:53.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02814,Sso133,2537,Sulfolobus sRNA133,Argasinska J,Argasinska J,44.0,44.5,39.2,\"This RNA has structural homologues in 3 archaeon Sulfolobus genomes: tokodaii, acidocalarius and solfotaricus. It shows extensive compensatory base changes, possibly consistent with a functional role.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,15,0,0,Gene; sRNA;,Predicted; 15659164,7,0,,\"\",0.57812,-5.8566,1600000,246151,161,73,true,-3.4243,0.71908,2017-11-21T09:54:41.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02815,Lig_thermometer,2538,Lig 5' UTR thermometer,Argasinska J,Argasinska J,51.5,51.5,51.3,It controls ligA and ligB gene expression in Leptospira interrogans in response to temperature change. The lipoproteins LigA and LigB promote adhesion of Leptospira to host proteins.The RNA is composed of 175-nucleotide 5'UTR and the first six lig codons (identical in ligA and ligB). Lig expressions is limited by double-stranded RNA structure that occludes the ribosome-binding site. At higher temperatures the site is exposed to promote translation initiation.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,3,0,0,Cis-reg; thermoregulator;,Predicted; 24013626,3,0,,\"\",0.44306,-9.87069,1600000,376572,339,205,true,-3.0649,0.71173,2017-11-22T16:02:16.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02816,HBV_PRE_1151-1410,2539,Hepatitis B virus post-transcriptional regulatory element 1151-1410,Argasinska J,Argasinska J,82.0,82.7,40.0,\"Part of 500 base pair long HBV PRE, contains nucleotides 1151-1410. Enhances nuclear export of intronless transcripts and represses the splicing mechanism to a comparable degree to that of the full-length PRE. It was proposed to be the core HBV PRE element. contains 3 known regulatory elements: PRE SL-alpha (nucleotides 1292-1321), human La protein binding site (nucleotide 1275-1291) and SRE-1 (nucleotides 1252-1348).\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,5,0,0,Cis-reg;,Predicted; RNAfold,5,0,,\"\",0.7198,-7.90066,1600000,344528,406,259,true,-3.7418,0.70802,2017-11-22T16:26:13.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02818,V_AS5,2540,Vibrio RNA AS5,Argasinska J,Argasinska J,96.0,96.1,95.8,Identified by direct cloning an parallel sequencing experiments with 5S/tRNA depletion. Confirmed by Northern blot.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,52,0,0,Gene; antisense;,Predicted; RNAfold,8,0,,\"\",0.64216,-3.66891,1600000,193768,243,129,true,-2.8505,0.71841,2017-12-04T13:05:17.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02819,V_AS7,2540,Vibrio RNA AS7,Argasinska J,Argasinska J,90.0,91.4,89.3,Identified by direct cloning an parallel sequencing experiments with 5S/tRNA depletion. Confirmed by Northern blot.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,7,66,0,0,Gene; antisense;,Predicted; RNAfold,67,0,,\"\",0.53262,-9.53286,1600000,304689,290,169,true,-3.5142,0.71592,2017-12-04T13:07:06.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02820,V_AS9,2540,Vibrio RNA AS9,Argasinska J,Argasinska J,50.0,50.3,49.9,Identified by direct cloning an parallel sequencing experiments with 5S/tRNA depletion. Confirmed by Northern blot.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,126,0,0,Gene; antisense;,Predicted; RNAfold,125,0,,\"\",0.52048,-7.6946,1600000,287927,203,104,true,-3.3542,0.71866,2017-12-04T13:07:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02821,V_IGR5,2540,Vibrio RNA IGR5,Argasinska J,Argasinska J,75.0,75.6,45.8,Identified by direct cloning an parallel sequencing experiments with 5S/tRNA depletion. Confirmed by Northern blot.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,3,0,0,Gene; sRNA;,Predicted; RNAfold,3,0,,\"\",0.48509,-7.1894,1600000,247059,216,114,true,-2.9487,0.71855,2017-12-04T13:08:49.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02823,LPR69,2541,Legionella RNA 69,Argasinska J,Argasinska J,40.0,40.4,39.6,Identified by strand specific sequencing of the transcriptional response during broth growth and during infection inside the host. Infection specific expression.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,5,136,0,0,Gene; sRNA;,Predicted; RNAfold,6,0,,\"\",0.64369,-5.01812,1600000,247169,171,82,true,-2.9774,0.71879,2017-12-05T13:45:26.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02824,LPR10,2542,Legionella RNA 10,Argasinska J,Argasinska J,69.0,69.6,56.8,Identified by strand specific sequencing of the transcriptional response during broth growth and during infection inside the host. Up-regulated in late infection or broth growth time points.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,9,0,0,Gene; sRNA;,Predicted; RNAfold,10,0,,\"\",0.54762,-6.32901,1600000,276500,191,94,true,-3.3482,0.71849,2017-12-05T13:46:39.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02825,LPR17,2542,Legionella RNA 17,Argasinska J,Argasinska J,98.0,98.4,74.0,Identified by strand specific sequencing of the transcriptional response during broth growth and during infection inside the host.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,8,0,0,Gene; sRNA;,Predicted; RNAfold,8,0,,\"\",0.65755,-8.1089,1600000,404472,400,256,true,-3.5802,0.70817,2017-12-05T13:46:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02834,VqmR,2543,Vibrio RNA VqmR,Argasinska J,Argasinska J,76.0,76.4,42.3,\"Repress the expression of multiple mRNAs including the rtx (repeats in toxin) toxin genes and the vpsT, which is required for biofilm formation. In fact, VqmR which is highly conserved in vibrionaceae, was shown to strongly inhibit biofilm formation through the repression of vpsT gene and could be the link between biofilm formation and quorum sensing.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,16,0,0,Gene; sRNA;,Published; 25646441,20,0,,\"\",0.52051,-9.47517,1600000,334882,269,154,true,-3.5207,0.71806,2017-12-12T11:02:58.000Z,2022-05-06T13:10:07.000Z\n",
            "RF02856,30_292,2544,Leptospira sRNA 30_292,Argasinska J,Argasinska J,112.0,112.7,37.2,Identified by RNA seq and Northern blot on cells grown to exponential phase at 30 degree C.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,6,0,0,Gene; sRNA;,Predicted; RNAfold;,6,0,,\"\",0.53175,-7.51451,1600000,281013,221,115,true,-2.8017,0.71838,2018-02-06T11:05:54.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02857,30_255,2544,Leptospira sRNA 30_255,Argasinska J,Argasinska J,197.0,197.9,44.2,Identified by RNA seq and Northern blot on cells grown to exponential phase at 30 degree C.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,1,0,0,Gene; sRNA;,Predicted; RNAfold;,2,0,,\"\",0.45177,-10.55752,1600000,370208,349,213,true,-3.0296,0.71107,2018-02-06T11:07:00.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02858,Arrc08,2545,Actinobacillus sRNA 08,Argasinska J,Argasinska J,219.0,219.8,172.4,Arrc stands for Actinobacillus pleuropneumoniae Regulatory RNA Candidate. TargetRNA2 was used for the predictions. sufI gene target may be involved in stress resistance.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,5,0,0,Gene; sRNA;,Predicted; RNAfold,5,0,,\"\",0.7271,-7.29359,1600000,352726,417,270,true,-3.973,0.70743,2018-02-27T14:57:55.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02859,Arrc11,2545,Actinobacillus sRNA 11,Argasinska J,Argasinska J,52.0,52.2,42.6,Arrc stands for Actinobacillus pleuropneumoniae Regulatory RNA Candidate. TargetRNA2 was used for the predictions. radA gene target may be involved in stress resistance.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,7,0,0,Gene; sRNA;,Predicted; RNAfold,7,0,,\"\",0.48216,-9.14832,1600000,335531,297,173,true,-3.111,0.71531,2018-02-27T14:59:59.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02860,Arrc14,2545,Actinobacillus sRNA 14,Argasinska J,Argasinska J,99.0,99.3,55.4,Arrc stands for Actinobacillus pleuropneumoniae Regulatory RNA Candidate. May control the expression of ferric permease (gene afuB2) and ferredoxin (fdx). TargetRNA2 was used for the predictions.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,10,0,0,Gene; sRNA;,Predicted; RNAfold,10,0,,\"\",0.46544,-8.46604,1600000,314782,285,167,true,-3.105,0.71614,2018-02-27T15:00:24.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02861,IR_84,2546,Neisseria sRNA 84,Argasinska J,Argasinska J,51.0,51.5,50.2,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,11,0,0,Gene; sRNA;,Predicted; RNAfold;,10,0,,\"\",0.50156,-9.12421,1600000,373994,319,186,true,-3.0058,0.71379,2018-02-27T15:07:18.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02862,sRNA_1300,2547,Enterococcus sRNA 1300,Argasinska J,Argasinska J,48.8,48.8,48.2,Expressed in mid and late exponential phases of growth. Expression remained uniform under daptomycin exposure.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,48,0,0,Gene; sRNA;,Predicted; RNAfold;,46,0,,\"\",0.61596,-8.18298,1600000,317495,371,231,true,-3.5322,0.70987,2018-02-27T15:14:32.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02863,sRNA_2410,2547,Enterococcus sRNA 2410,Argasinska J,Argasinska J,61.0,61.6,59.7,\"Produced as 2 transcripts on Northern blot. Expressed in mid, late exponential and early stationary phases of growth. Expression remained uniform under daptomycin exposure.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,82,0,0,Gene; sRNA;,Predicted; RNAfold;,55,0,,\"\",0.48063,-9.5318,1600000,295226,326,196,true,-3.0018,0.71279,2018-02-27T15:15:11.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02864,sRNA_0030,2547,Enterococcus sRNA 30,Argasinska J,Argasinska J,123.0,123.2,71.6,\"Produced as 2 transcripts on Northern blot. Expressed in mid, late exponential and early stationary phases of growth. Expression fluctuated under daptomycin exposure. Expressed from an integrative transposon carrying the tet(M) gene, which confers tetacyclin resistance.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,47,0,0,Gene; sRNA;,Predicted; RNAfold;,46,0,,\"\",0.56387,-7.60282,1600000,314238,289,165,true,-3.3391,0.71643,2018-02-27T15:15:43.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02872,SrbA,2548,sRNA regulator of biofilms A,Argasinska J,Argasinska J,69.0,69.7,44.4,\"Important for biofilm formation and pathogenicity. Bacterial strain with deleted SrbA had reduced biofilm mass. As the ability to form biofilms can contribute to the ability a pathogen to thrive within the host, the C. elegans hosts infected with the srbA deleted strain displayed significantly lower mortality rate than the wild-type strain. However, the deletion of srbA had no effect on growth or antibiotic resistance in P. aeruginosa.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,58,0,0,Gene; sRNA;,Predicted; R-Scape;,61,0,,\"\",0.66167,-9.35928,1600000,450981,370,235,true,-3.8792,0.70947,2018-03-06T11:48:03.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02873,AS-traI,2549,Antisense to traI,Argasinska J,Argasinska J,99.0,99.9,36.2,\"The pHK01 produces plasmid-encoded small RNAs and mediates expression of host sRNAs. These sRNAs are antisense to genes involved in replication, conjugate transfer and plasmid stabilisation: AS-repA3 (CopA), AS-traI, AS-finO, AS-traG, AS-pc02. The overexpression of AS-traI shortened the log phase of host growth.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,5,0,0,Gene; antisense;,Predicted; RNAfold,7,0,,\"\",0.57246,-5.14945,1600000,238088,168,77,true,-3.1532,0.71895,2018-03-06T11:52:58.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02874,AS-traG,2549,Antisense to traG,Argasinska J,Argasinska J,58.0,58.6,35.7,\"The pHK01 produces plasmid-encoded small RNAs and mediates expression of host sRNAs. These sRNAs are antisense to genes involved in replication, conjugate transfer and plasmid stabilisation: AS-repA3 (CopA), AS-traI, AS-finO, AS-traG, AS-pc02. The overexpression of AS-traI shortened the log phase of host growth.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,13,0,0,Gene; antisense;,Predicted; RNAfold,13,0,,\"\",0.58414,-3.93265,1600000,191237,179,86,true,-3.8182,0.71877,2018-03-06T11:53:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02875,AS-pc01,2549,Antisense to pHK01_035,Argasinska J,Argasinska J,79.0,79.5,41.9,\"The pHK01 produces plasmid-encoded small RNAs and mediates expression of host sRNAs. These sRNAs are antisense to genes involved in replication, conjugate transfer and plasmid stabilisation: AS-repA3 (CopA), AS-traI, AS-finO, AS-traG, AS-pc02. The overexpression of AS-traI shortened the log phase of host growth.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,8,0,0,Gene; antisense;,Predicted; RNAfold,6,0,,\"\",0.51299,-8.21981,1600000,304439,196,99,true,-3.2348,0.71868,2018-03-06T11:54:48.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02876,AS-pc02,2549,Antisense to pHK01_099,Argasinska J,Argasinska J,40.0,40.5,36.9,\"The pHK01 produces plasmid-encoded small RNAs and mediates expression of host sRNAs. These sRNAs are antisense to genes involved in replication, conjugate transfer and plasmid stabilisation: AS-repA3 (CopA), AS-traI, AS-finO, AS-traG, AS-pc02. The overexpression of AS-traI shortened the log phase of host growth.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,4,0,0,Gene; antisense;,Predicted; RNAfold,5,0,,\"\",0.65612,-8.41031,1600000,397142,313,185,true,-3.44,0.71399,2018-03-06T11:55:14.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02877,NmsRb,2550,Neisseria metabolic switch regulator b (RcoF1/NgncR163),Argasinska J,Argasinska J,52.0,52.4,45.3,\"Sibling sRNA described and independently named in three publications. NgncR163 is N. gonorrhoeae while the others are N. meningitidis RNAs. RcoF1 stands for: RNA regulating colonization factor 1. NmsRb_RcoF1_NgncR163 was shown to be the predominant sibling. They are tandemly arranged, structurally nearly identical and and share 70% sequence identity. They translationally down-regulate genes involved in basic metabolic processes including tricarboxylic acid cycle enzymes and amino acid uptake and degradation. The targets include: fumC, sdhC, gltA, sucC, prpB and prpC [1][3] Their expression is under the control of RelA.[3] Furthermore, they interact with Hfq and target mRNA that encodes putative colonization factor of the human nasopharynx (PrpB).[2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,6,50,0,0,Gene; sRNA;,Predicted; RNAfold;,36,0,,\"\",0.47451,-8.82692,1600000,308981,183,91,true,-4.0121,0.71867,2018-03-06T12:23:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02878,MH_s7,2551,Mesorhizobail RNA 7,Argasinska J,Argasinska J,221.0,221.1,67.0,\"Existence predicted via genome wide comparisons with 4 related strains. Analysed further by deep sequencing and Northern blot under various stress conditions. Expressed only under rare growth conditions: symbiosis, micro-oxygen stress, and alkali stress.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,13,0,0,Gene; sRNA;,Predicted; RNAfold;,13,0,,\"\",0.69381,-8.73167,1600000,373659,465,312,true,-4.3331,0.70542,2018-03-06T13:35:44.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02879,MH_s10,2551,Mesorhizobail RNA 10,Argasinska J,Argasinska J,81.0,81.1,39.7,\"Existence predicted via genome wide comparisons with 4 related strains. Analysed further by deep sequencing and Northern blot under various stress conditions. Exhibited 2 bands. Strong signals in response to variety of stimuli. Highly abundant in symbiotic, micro-oxygen, oxidative and salt stress conditions.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,3,11,0,0,Gene; sRNA;,Predicted; R-scape;,11,0,,\"\",0.52767,-9.90818,1600000,401236,259,143,true,-2.6672,0.71815,2018-03-06T14:13:23.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02880,MH_s15,2551,Mesorhizobail RNA 15,Argasinska J,Argasinska J,123.0,123.2,34.7,\"Existence predicted via genome wide comparisons with 4 related strains. Analysed further by deep sequencing and Northern blot under various stress conditions. Widely expressed, accumulated heavily under most of the induction conditions, suggesting function in multiple cellular processes.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; RNAfold;,5,0,,\"\",0.59558,-9.67984,1600000,364528,410,266,true,-3.5568,0.70764,2018-03-06T14:17:03.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02881,MH_s25,2551,Mesorhizobail RNA 25,Argasinska J,Argasinska J,114.0,114.9,51.7,\"Existence predicted via genome wide comparisons with 4 related strains. Analysed further by deep sequencing and Northern blot under various stress conditions. Showed complex banding pattern. Highly abundant in symbiotic, micro-oxygen, oxidative and salt stress conditions. Widely expressed and as it accumulated heavily under most tested induction conditions it may function in many cellular processes.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; R-scape;,5,0,,\"\",0.48749,-11.03758,1600000,428913,332,203,true,-3.2123,0.71182,2018-03-06T15:19:50.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02882,MH_s36,2551,Mesorhizobail RNA 36,Argasinska J,Argasinska J,46.0,46.0,33.8,\"Existence predicted via genome wide comparisons with 4 related strains. Analysed further by deep sequencing and Northern blot under various stress conditions. Strong signals in response to variety of stimuli. Highly abundant in symbiotic, micro-oxygen, oxidative and salt stress conditions.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,2,10,0,0,Gene; sRNA;,Predicted; RNAfold;,10,0,,\"\",0.47737,-8.8507,1600000,320201,189,92,true,-2.8013,0.7187,2018-03-06T15:20:31.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02885,SAM_VI,2552,SAM-VI riboswitch,Argasinska J,Argasinska J,57.5,57.5,57.2,\"sixth member of the riboswitch family predominantly found in Bifidobacterium and exhibits some similarities to the SAM-III (Smk box) riboswitch class, but it lacks most of the highly-conserved nucleotides of SAM-III class. SAM-VI aptamers bind the cofactor S-adenosylmethinine SAM (a key metabolite in sulphur metabolism) and discriminate strongly against S-adenosylhomocysteine SAH. The class was discovered by further analysis of Bifido-meK motif RNAs[1].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,26,0,0,Cis-reg; riboswitch;,Predicted; RNAfold;,26,0,,\"\",0.60187,-4.61909,1600000,305168,238,62,true,-3.4244,0.71931,2018-03-07T12:24:52.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02887,mgtC_leader,2553,Salmonella mgtC leader RNA,Argasinska J,Argasinska J,48.0,48.5,42.1,\"From mgtCBR operon, decreases flagellin production during infection by directly base pairing with mRNAs of the fljB gene encoding flagellin and promotes fljBA mRNA degradation.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 604040.189692 --mxsize 128 CM SEQDB,4,26,0,0,Cis-reg; leader;,Predicted; Rscape;,26,0,,\"\",0.52675,-10.17504,1600000,459585,217,114,true,-3.1635,0.71819,2018-03-28T15:36:15.000Z,2019-01-04T15:01:52.000Z\n",
            "RF02888,BtsR1,2554,Bacillus sRNA 1,Argasinska J,Argasinska J,66.0,66.6,36.2,\"Allows avoidance of host behavioural defences by silencing the Cry toxin when outside the host via direct base pairing to the RBS site of the Cry5Ba toxin transcript and inhibiting its expression. The silencing results in increased ingestion by C. elegans and is relieved inside the host, resulting in host death.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,4,0,0,Gene; sRNA;,Predicted; Rscape;,7,0,,\"\",0.57327,-7.9053,1600000,529658,131,51,true,-3.5104,0.72359,2018-03-28T15:36:53.000Z,2022-05-06T13:10:08.000Z\n",
            "RF02889,Sr006,2555,Pseudomonas sRNA 6,Argasinska J,Argasinska J,100.0,106.7,40.1,\"Positively regulates (post-transcriptionally), without Hfq, the expression of PagL, an enzyme responsible for deacylation of lipid A. This reduces the pro-inflammatory property of lipid A. [1] Furthermore, similarly to Salmonella, Sr006 regulation of PagL expression was suggested to aid in polymyxin B resistance.[1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,0,0,0,Gene; sRNA;,Predicted; Rscape;,3,0,,\"\",0.4717,-10.00745,1600000,360017,229,124,true,-2.9112,0.71831,2018-03-28T15:37:23.000Z,2021-11-19T11:53:19.000Z\n",
            "RF02500,Atu_C6,2556,EcpR1,Eberhardt R,Predicted; WAR,60.0,60.7,57.7,\"Perviously named SmelC291, SmrC10, or Sra33.[3][4][5] According to its overproduction phenotype it was renamed Elongated Cell Phenotype RNA1.[2] Induced by stress,EcpR1 negatively regulates cell cycle master regulatory genes dnaA and gcrA at post-transcriptional level by base pairing between its strongly conserved GC-rich loop and the target mRNAs. It's regulation is hfq-independent but requires RNase E to regulate dnaA.[2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,32,166,0,0,Gene; sRNA;,Predicted; WAR,191,0,,\"\",0.42634,-11.10934,1600000,309026,225,113,true,-3.4944,0.71852,2013-10-04T03:36:59.000Z,2021-12-03T12:41:14.000Z\n",
            "RF02892,SR6_antitoxin,2557,Bacillus SR6 antitoxin,Argasinska J,Argasinska J,55.0,55.6,43.7,\"Overlaps 2 toxins: 3' end of yonT and yoyJ at its 5'end. In type I toxin-antitoxin (TA) systems the antitoxin is a small RNA that neutralizes a toxin protein. YonT/SR6 system is located on the SPÎ² prophage of the B. subtilis chromosome and it was shown to be multi-stress responsive. SR6 acts by promoting yonT mRNA degradation. It may regulate the second toxin, yoyJ by a different mechanism.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,3,3,0,0,Gene; antisense;,Predicted; RNAfold;,6,0,,\"\",0.52712,-6.27896,1600000,254544,206,104,true,-3.5583,0.71853,2018-05-23T12:03:39.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02893,RpsF_leader,2558,RpsF leader,Argasinska J,Argasinska J,37.0,37.0,36.6,\"Conserved and widely distributed to many bacterial species RNA structure. It precedes operon containing rpsF, which encodes ribosomal protein S6. The motif also precedes operon containing rpsR, which encodes ribosomal protein S18. The RNA was shown to interact with S6:S18 protein dimer in vitro.[1] It was shown that RpsF leader regulates gene expression in response to the S6:S18 complex, contributing to the regulation of ribosomal protein levels [2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,10,214,0,0,Cis-reg; leader;,Predicted; 24310371;,223,0,,\"\",0.65102,-3.609,1600000,166130,179,85,true,-3.7025,0.71869,2018-05-23T12:19:06.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02898,AaHKsRNA20,2559,Aggregatibacter sRNA 20,Argasinska J,Argasinska J,43.0,43.4,39.8,Validated by RT-PCR and Northern blot.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,6,26,0,0,Gene; sRNA;,Predicted; Rscape;,32,0,,\"\",0.56081,-2.74659,1600000,213054,163,77,true,-3.6655,0.71889,2018-06-18T16:11:04.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02899,AaHKsRNA54,2559,Aggregatibacter sRNA 54,Argasinska J,Argasinska J,37.0,37.1,36.2,\"Validated by RT-PCR, expression up-regulated during growth.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,6,18,0,0,Gene; sRNA;,Predicted; RNAfold;,24,0,,\"\",0.50362,-8.02735,1600000,302830,225,118,true,-3.1376,0.71846,2018-06-18T16:12:00.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02900,AaHKsRNA82,2559,Aggregatibacter sRNA 82,Argasinska J,Argasinska J,51.4,51.4,50.8,\"Validated by RT-PCR, expression down-regulated during growth. Blast search using Bacterial Small Regulatory RNA Database found isrK. Has Rho-independent terminator.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,4,10,0,0,Gene; sRNA;,Predicted; RNAfold;,12,0,,\"\",0.5186,-5.26773,1600000,353327,167,81,true,-3.4094,0.7188,2018-06-18T16:13:02.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02901,AaHKsRNA41,2559,Aggregatibacter sRNA 41,Argasinska J,Argasinska J,97.0,97.4,95.8,\"Validated by RT-PCR, expression up-regulated during growth.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,2,5,0,0,Gene; sRNA;,Predicted; RNAfold;,5,0,,\"\",0.61719,-7.90639,1600000,300433,302,180,true,-3.3041,0.71445,2018-06-18T16:13:36.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02902,AaHKsRNA22,2559,Aggregatibacter sRNA 22,Argasinska J,Argasinska J,39.8,39.8,39.6,Validated by RT-PCR.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,7,132,0,0,Gene; sRNA;,Predicted; RNAfold;,36,0,,\"\",0.52927,-8.97756,1600000,316515,195,101,true,-3.5851,0.71861,2018-06-18T16:14:18.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02903,AaHKsRNA96,2559,Aggregatibacter sRNA 96,Argasinska J,Argasinska J,43.0,43.8,40.9,Validated by RT-PCR. Antisense gene: HK1651_08600.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,3,11,0,0,Gene; antisense;,Predicted; RNAfold;,14,0,,\"\",0.51856,-6.31703,1600000,268643,222,117,true,-3.1241,0.71837,2018-06-18T16:15:06.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02904,AaHKsRNA69,2559,Aggregatibacter sRNA 69,Argasinska J,Argasinska J,43.0,43.7,40.9,Validated by RT-PCR. Has Rho-independent terminator.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,3,12,0,0,Gene; sRNA;,Predicted; RNAfold;,15,0,,\"\",0.51099,-5.62115,1600000,319417,172,83,true,-3.4421,0.71877,2018-06-18T16:15:34.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02905,sRNA41,2560,Archaeal Small RNA 41,Argasinska J,Argasinska J,45.0,45.2,44.0,Highly expressed during nitrogen sufficiency[1]. Capable to bind several ribosome binding sites independently within a polycistronic mRNA. Inhibits translation initiation of all ACDS (acetyl-CoA decarbonylase/ synthase complex) genes in N-dependent manner [2].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,9,0,0,Gene; sRNA;,Predicted; RNAfold;,4,0,,\"\",0.51575,-6.66217,1600000,301099,215,113,true,-2.9366,0.71843,2018-06-18T16:23:17.000Z,2022-05-06T13:10:08.000Z\n",
            "RF02906,sRNA154,2560,Archaeal Small RNA 154,Argasinska J,Argasinska J,88.0,88.4,35.9,\"Exclusively expressed under nitrogen deficiency, has a central regulatory role in nitrogen metabolism affecting nitrogenase and glutamine synthetase by masking the ribosome binding site or positively affecting transcript stability. Global nitrogen repressor NrpR direct binding.[2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,3,1,0,0,Gene; sRNA;,Predicted; RNAfold;,4,0,,\"\",0.60234,-5.74845,1600000,276180,245,134,true,-3.3136,0.71822,2018-06-18T16:29:43.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02911,Baculoviridae_NAE,2562,Baculoviridae Nucleocapsid Assembly essential Element,Argasinska J,Argasinska J,51.0,51.4,42.0,Internal cis-element within the ac83 gene. Ac83 is involved in nucleocapsid assembly via NAE (nucleocapsid assembly essential element). element). The nucleocapsid assembly is not dependent on the Ac83 protein product.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,6,24,0,0,Cis-reg;,Predicted; RNAfold;,30,0,,\"\",0.43827,-10.98331,1600000,407564,325,200,true,-3.3552,0.71219,2018-06-19T10:59:34.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02912,AAC_AAD_leader,2563,putative aminoglycoside riboswitch / attI site,Argasinska J,Argasinska J,56.0,56.2,50.7,\"Conserved RNA structure found upstream of the bacterial aminoglycosides antibiotic-resistant genes: aminoglycoside acetyl transferase (AAC) and aminoglycoside adenyl transferase (AAD). It has putative short open reading frame (ORF) that encode leader peptide upstream of the resistance gene. 75 nucleotide long sequence from the first ribosome binding site (SD1) up to the coding sequence was shown to have the regulatory role, and it was considered the minima functional RNA. Aminoglycosides bind to specific regions of AAC/AAD 5' leader and induce change in the RNA structure. The drug binding induces expression of the resistance protein. The leader RNA is widely distributed among antibiotic-resistant bacterial pathogens.[1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,25,0,0,Cis-reg; riboswitch;,Published; 23332747;,23,0,,\"\",0.6019,-6.63998,1600000,349391,164,75,true,-3.6281,0.71895,2018-06-19T11:03:38.000Z,2020-09-14T15:07:08.000Z\n",
            "RF02913,pemK,2564,pemK RNA,\"Argasinska J, Corbino K, Weinberg Z\",Weinberg Z,34.8,34.8,34.4,\"Firmicutes. Environment (Human gut, Qin et al., obese & healthy subjects)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,1542,750,0,0,Cis-reg;,Published; PMID:28977401;,477,0,,\"\",0.52357,-9.43446,1600000,497899,394,61,true,-3.2726,0.71898,2018-06-27T12:30:23.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02914,DUF805b,2565,DUF805b RNA,\"Argasinska J, Weinberg Z\",Weinberg Z,35.6,35.6,35.5,\"Bacteroidetes, Chlorobi (1 example), Proteobacteria. Lineage (Flavobacteria)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.93 -Z 742849.287494 --mxsize 128 CM SEQDB,1230,807,0,0,Cis-reg;,Published; PMID:28977401;,334,0,,\"\",0.79771,-5.02831,1600000,428378,79,36,true,-3.6142,0.72676,2018-06-27T12:38:12.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03081,DUF805,2565,DUF805 RNA,Weinberg Z,Weinberg Z,51.1,51.1,50.6,\"Bacteroidetes, Aeromonadales. Lineage (Bacteroidetes)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.97 -Z 742849.287494 --mxsize 128 CM SEQDB,46,235,0,0,Cis-reg;,Published; PMID:28977401;,161,0,,\"\",0.53494,-7.16345,1600000,313683,139,68,true,-4.0205,0.71889,2019-01-04T10:54:19.000Z,2021-07-05T14:43:57.000Z\n",
            "RF02915,DUF3800-VI,2566,DUF3800-VI RNA,Weinberg Z,Weinberg Z,35.0,35.0,34.5,\"Actinobacteria, Planctomycetes, Proteobacteria, Spirochaetes. Nearby to DUF3800\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.79 -Z 742849.287494 --mxsize 128 CM SEQDB,87,26,0,0,Cis-reg;,Published; PMID:28977401;,29,0,,\"\",0.61042,-4.99422,1600000,311329,106,52,true,-3.5351,0.72432,2018-07-02T14:22:42.000Z,2019-01-17T14:00:28.000Z\n",
            "RF02961,DUF3800-II,2566,DUF3800-II RNA,Weinberg Z,Weinberg Z,43.2,43.2,36.3,Environmental. Nearby to DUF3800,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.81 -Z 742849.287494 --mxsize 128 CM SEQDB,23,22,0,0,Gene; sRNA;,Published; PMID:28977401;,22,0,,\"\",0.61685,-4.27445,1600000,246851,140,69,true,-3.2162,0.71889,2018-12-13T11:24:30.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02962,DUF3800-III,2566,DUF3800-III RNA,Weinberg Z,Weinberg Z,43.8,43.8,33.6,Microgenomates. Nearby to DUF3800,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.83 -Z 742849.287494 --mxsize 128 CM SEQDB,55,18,0,0,Gene; sRNA;,Published; PMID:28977401;,17,0,,\"\",0.684,-3.13944,1600000,221823,387,108,true,-3.2455,0.7184,2018-12-13T11:27:25.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02963,DUF3800-IV,2566,DUF3800-IV RNA,Weinberg Z,Weinberg Z,60.7,60.7,47.4,Actinobacteria. Nearby to DUF3800,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.95 -Z 742849.287494 --mxsize 128 CM SEQDB,10,3,0,0,Gene; sRNA;,Published; PMID:28977401;,8,0,,\"\",0.52386,-5.24252,1600000,218850,183,91,true,-3.7451,0.71874,2018-12-13T11:34:45.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02964,DUF3800-V,2566,DUF3800-V RNA,Weinberg Z,Weinberg Z,37.4,37.4,34.9,Proteobacteria. Nearby to DUF3800,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.29 -Z 742849.287494 --mxsize 128 CM SEQDB,79,59,0,0,Gene; sRNA;,Published; PMID:28977401;,57,0,,\"\",0.64811,-1.95316,1600000,191974,259,96,true,-3.3638,0.71862,2018-12-13T11:41:51.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02967,DUF3800-VII,2566,DUF3800-VII RNA,Weinberg Z,Weinberg Z,36.2,36.2,33.7,\"Firmicutes, Proteobacteria. Nearby to DUF3800\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.68 -Z 742849.287494 --mxsize 128 CM SEQDB,139,21,0,0,Gene; sRNA;,Published; PMID:28977401;,33,0,,\"\",0.60541,-3.30996,1600000,164898,198,90,true,-3.5549,0.71855,2018-12-13T12:22:47.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02968,DUF3800-IX,2566,DUF3800-IX RNA,Weinberg Z,Weinberg Z,36.8,37.1,36.3,\"Actinobacteria, Firmicutes (1 example), Nitrospirae, Proteobacteria, Verrucomicrobia (1 example). Nearby to DUF3800\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,229,89,0,0,Gene; sRNA;,Published; PMID:28977401;,88,0,,\"\",0.63534,-4.02292,1600000,206788,132,67,true,-3.6497,0.7247,2018-12-13T13:01:12.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02969,DUF3800-I,2566,DUF3800-I RNA,Weinberg Z,Weinberg Z,44.2,44.3,43.4,\"Actinobacteria, Firmicutes, Fusobacteria, Proteobacteria (1 example), Tenericutes (2 examples). Terminator (Clostridia)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,321,44,0,0,Gene; sRNA;,Published; PMID:28977401;,77,0,,\"\",0.4706,-8.37837,1600000,308686,197,115,true,-3.9686,0.7183,2018-12-13T14:08:34.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02975,DUF3800-XI,2566,DUF3800-XI RNA,Weinberg Z,Weinberg Z,37.8,37.8,36.5,\"Actinobacteria, Firmicutes, Proteobacteria, Spirochaetes (1 example). Nearby to DUF3800\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.17 -Z 742849.287494 --mxsize 128 CM SEQDB,256,42,0,0,Gene; sRNA;,Published; PMID:28977401;,47,0,,\"\",0.60523,-9.22864,1600000,707545,97,48,true,-3.8733,0.71989,2018-12-13T14:34:41.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02984,DUF3800-X,2566,DUF3800-X RNA,Weinberg Z,Weinberg Z,38.2,40.9,35.7,Euryarchaeota (Archaea). Nearby to DUF3800,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.18 -Z 742849.287494 --mxsize 128 CM SEQDB,88,13,0,0,Gene; sRNA;,Published; PMID:28977401;,14,0,,\"\",0.6547,-2.74803,1600000,188506,271,73,true,-3.7351,0.71885,2018-12-13T16:49:44.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03075,DUF3800-VIII,2566,DUF3800-VIII RNA,Weinberg Z,Weinberg Z,40.2,40.2,37.8,\"Bacteroidetes, Cyanobacteria, Firmicutes, Planctomycetes (1 example), Proteobacteria, Tenericutes (1 example). Nearby to DUF3800\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.74 -Z 742849.287494 --mxsize 128 CM SEQDB,209,45,0,0,Gene; sRNA;,Published; PMID:28977401;,59,0,,\"\",0.54786,-5.15944,1600000,251218,253,64,true,-3.4356,0.71892,2019-01-03T14:33:53.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02916,atpB,2567,atpB RNA,\"Argasinska J, Weinberg Z\",Weinberg Z,37.4,37.4,35.0,Corynebacteriaceae (Actinobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.62 -Z 742849.287494 --mxsize 128 CM SEQDB,72,88,0,0,Cis-reg;,Published; PMID:28977401;,126,0,,\"\",0.58958,-9.26653,1600000,315314,237,128,true,-3.5944,0.71804,2018-07-03T14:33:52.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02917,Burkholderiales-2,2568,Burkholderiales-2 RNA,\"Argasinska J, Weinberg Z\",Weinberg Z,67.9,67.9,37.5,\"BetaProteobacteria, Bacteroidetes (suspicious). Big IGRs\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.61 -Z 742849.287494 --mxsize 128 CM SEQDB,19,8,0,0,Cis-reg;,Published; PMID:28977401;,13,0,,\"\",0.49269,-9.53477,1600000,512535,157,74,true,-2.9535,0.719,2018-07-03T14:39:15.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02918,MDR-NUDIX,2569,MDR-NUDIX RNA,Weinberg Z,Weinberg Z,54.9,54.9,36.2,\"TM7. Lineage (Saccharibacteria, aka TM7)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.65 -Z 742849.287494 --mxsize 128 CM SEQDB,15,9,0,0,Cis-reg;,Published; PMID:28977401;,7,0,,\"\",0.54183,-8.80383,1600000,473550,139,61,true,-3.6387,0.71928,2018-07-03T14:41:13.000Z,2019-01-17T14:00:28.000Z\n",
            "RF02919,ilvB-OMG,2570,ilvB-OMG RNA,Weinberg Z,Weinberg Z,95.9,95.9,67.2,OMG group (GammaProteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.71 -Z 742849.287494 --mxsize 128 CM SEQDB,28,6,0,0,Cis-reg;,Published; PMID:28977401;,7,0,,\"\",0.74219,-8.05743,1600000,436078,337,210,true,-3.6247,0.71143,2018-07-03T14:43:06.000Z,2019-01-17T14:00:28.000Z\n",
            "RF02920,Fusobacteriales-1,2571,Fusobacteriales-1 RNA,\"Corbino K, Weinberg Z\",Weinberg Z,52.1,52.1,49.5,Fusobacteria. Lineage (Fusobacteria),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.98 -Z 742849.287494 --mxsize 128 CM SEQDB,13,9,0,0,Cis-reg;,Published; PMID:28977401;,14,0,,\"\",0.41104,-10.77637,1600000,317555,212,114,true,-2.9485,0.71834,2018-07-03T14:47:50.000Z,2019-01-17T14:00:28.000Z\n",
            "RF02921,RT-14,2572,RT-14 RNA,Weinberg Z,Weinberg Z,48.1,48.1,46.3,\"Cloacimonetes, Firmicutes, Proteobacteria. Nearby to self-splicing intron genes\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.10 -Z 742849.287494 --mxsize 128 CM SEQDB,143,52,0,0,Gene; sRNA;,Published; PMID:28977401;,45,0,,\"\",0.42616,-10.38265,1600000,367526,235,144,true,-2.831,0.71806,2018-07-03T14:50:22.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03016,RT-12,2572,RT-12 RNA,Weinberg Z,Weinberg Z,44.2,44.2,42.6,Proteobacteria. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.51 -Z 742849.287494 --mxsize 128 CM SEQDB,42,6,0,0,Gene; sRNA;,Published; PMID:28977401;,38,0,,\"\",0.49881,-8.80532,1600000,261365,221,122,true,-3.2836,0.71823,2018-12-20T17:41:14.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03017,RT-13,2572,RT-13 RNA,Weinberg Z,Weinberg Z,69.1,69.1,50.6,Alphaproteobacteria. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.44 -Z 742849.287494 --mxsize 128 CM SEQDB,13,11,0,0,Gene; sRNA;,Published; PMID:28977401;,14,0,,\"\",0.58818,-5.81502,1600000,374600,146,67,true,-3.2898,0.71905,2018-12-20T17:43:17.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03018,RT-15,2572,RT-15 RNA,Weinberg Z,Weinberg Z,82.0,82.0,61.3,Proteobacteria. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.45 -Z 742849.287494 --mxsize 128 CM SEQDB,64,30,0,0,Gene; sRNA;,Published; PMID:28977401;,54,0,,\"\",0.61871,-9.2558,1600000,383129,344,193,true,-3.3706,0.71309,2018-12-20T17:45:40.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03019,RT-16,2572,RT-16 RNA,Weinberg Z,Weinberg Z,50.0,51.8,48.0,candidate division DUSEL4 (Archaea). Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,104,12,0,0,Gene; sRNA;,Published; PMID:28977401;,12,0,,\"\",0.47455,-7.27459,1600000,218063,384,158,true,-3.6326,0.71722,2018-12-20T17:49:37.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03020,RT-17,2572,RT-17 RNA,Weinberg Z,Weinberg Z,40.4,40.4,39.8,\"Bacteroidales (Bacteroidetes), Clostridiales (Firmicutes). Nearby to self- splicing intron genes\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.65 -Z 742849.287494 --mxsize 128 CM SEQDB,63,15,0,0,Gene; sRNA;,Published; PMID:28977401;,21,0,,\"\",0.57472,-5.77679,1600000,267200,173,95,true,-2.8879,0.71844,2018-12-20T17:52:08.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03021,RT-18,2572,RT-18 RNA,Weinberg Z,Weinberg Z,41.3,41.3,36.7,Clostridiales (Firmicutes). Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.08 -Z 742849.287494 --mxsize 128 CM SEQDB,141,21,0,0,Gene; sRNA;,Published; PMID:28977401;,21,0,,\"\",0.52419,-7.07766,1600000,367917,247,83,true,-3.057,0.71869,2018-12-20T17:53:29.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03022,RT-10,2572,RT-10 RNA,Weinberg Z,Weinberg Z,49.3,49.3,35.6,Proteobacteria. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.88 -Z 742849.287494 --mxsize 128 CM SEQDB,113,6,0,0,Gene; sRNA;,Published; PMID:28977401;,9,0,,\"\",0.55063,-9.47639,1600000,353828,218,127,true,-3.6623,0.71821,2018-12-20T17:56:16.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03025,RT-4,2572,RT-4 RNA,Weinberg Z,Weinberg Z,73.5,73.5,48.2,Clostridiales. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.68 -Z 742849.287494 --mxsize 128 CM SEQDB,47,3,0,0,Gene; sRNA;,Published; PMID:28977401;,8,0,,\"\",0.51535,-7.45306,1600000,263318,263,122,true,-3.0753,0.71826,2018-12-20T18:06:57.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03026,RT-5,2572,RT-5 RNA,Weinberg Z,Weinberg Z,45.7,45.7,40.0,Clostridium (Firmicutes). Environment (Cow rumen (57)),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.38 -Z 742849.287494 --mxsize 128 CM SEQDB,71,11,0,0,Gene; sRNA;,Published; PMID:28977401;,12,0,,\"\",0.56909,-6.46486,1600000,282848,275,103,true,-3.9959,0.7184,2018-12-20T18:09:04.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03027,RT-6,2572,RT-6 RNA,Weinberg Z,Weinberg Z,46.7,46.7,44.5,Firmicutes. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.53 -Z 742849.287494 --mxsize 128 CM SEQDB,59,7,0,0,Gene; sRNA;,Published; PMID:28977401;,16,0,,\"\",0.61996,-6.6588,1600000,289200,140,75,true,-3.8654,0.71873,2018-12-20T18:13:38.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03028,RT-7,2572,RT-7 RNA,Weinberg Z,Weinberg Z,116.8,116.8,66.5,Bacteroidales. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.04 -Z 742849.287494 --mxsize 128 CM SEQDB,54,4,0,0,Gene; sRNA;,Published; PMID:28977401;,10,0,,\"\",0.63242,-8.34376,1600000,252938,411,191,true,-3.6658,0.7132,2018-12-20T18:16:58.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03029,RT-8,2572,RT-8 RNA,Weinberg Z,Weinberg Z,87.0,87.0,37.8,Proteobacteria. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.55 -Z 742849.287494 --mxsize 128 CM SEQDB,9,2,0,0,Gene; sRNA;,Published; PMID:28977401;,6,0,,\"\",0.54535,-5.55607,1600000,257044,187,92,true,-3.4894,0.71858,2018-12-20T18:18:45.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03068,RT-3,2572,RT-3 RNA,Weinberg Z,Weinberg Z,40.5,49.2,37.4,Bacteroidetes. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,231,13,0,0,Gene; sRNA;,Published; PMID:28977401;,32,0,,\"\",0.6377,-8.47469,1600000,384891,249,148,true,-3.0405,0.71799,2019-01-03T12:20:09.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03073,RT-19,2572,RT-19 RNA,Weinberg Z,Weinberg Z,78.3,78.3,33.9,Actinobacteria. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.58 -Z 742849.287494 --mxsize 128 CM SEQDB,23,0,0,0,Gene; sRNA;,Published; PMID:28977401;,6,0,,\"\",0.55726,-7.42377,1600000,208772,230,99,true,-3.3171,0.71848,2019-01-03T14:23:46.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03077,RT-2,2572,RT-2 RNA,Weinberg Z,Weinberg Z,40.0,40.8,33.1,\"Coriobacteriales (Actinobacteria), Clostridiales (Firmicutes). Downstream gene\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,195,15,0,0,Gene; sRNA;,Published; PMID:28977401;,18,0,,\"\",0.61824,-9.22332,1600000,360722,353,222,true,-3.9817,0.71032,2019-01-04T09:23:46.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03080,RT-9,2572,RT-9 RNA,Weinberg Z,Weinberg Z,46.1,46.1,43.5,Firmicutes. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.76 -Z 742849.287494 --mxsize 128 CM SEQDB,48,17,0,0,Gene; sRNA;,Published; PMID:28977401;,14,0,,\"\",0.49663,-6.87959,1600000,283299,182,92,true,-3.4603,0.71852,2019-01-04T10:37:29.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03106,RT-11,2572,RT-11 RNA,Weinberg Z,Weinberg Z,143.5,143.5,33.1,Environmental. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.80 -Z 742849.287494 --mxsize 128 CM SEQDB,3,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.58037,-8.81764,1600000,358074,261,147,true,-3.1835,0.71807,2019-01-15T10:36:43.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03114,RT-1,2572,RT-1 RNA,Weinberg Z,Weinberg Z,106.4,106.4,38.1,Neisseria elongata (Betaproteobacteria). Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.90 -Z 742849.287494 --mxsize 128 CM SEQDB,5,1,0,0,Gene; sRNA;,Published; PMID:28977401;,3,0,,\"\",0.53902,-6.68471,1600000,269132,210,107,true,-3.2306,0.7186,2019-01-15T11:55:17.000Z,2019-01-17T14:00:28.000Z\n",
            "RF02923,HTH-XRE,2573,HTH-XRE RNA,Weinberg Z,Weinberg Z,49.8,49.8,39.8,Clostridiales. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.51 -Z 742849.287494 --mxsize 128 CM SEQDB,60,10,0,0,Cis-reg;,Published; PMID:28977401;,17,0,,\"\",0.59259,-4.93515,1600000,266737,162,78,true,-3.5223,0.71872,2018-07-03T16:27:14.000Z,2019-01-17T14:00:28.000Z\n",
            "RF02924,skipping-rope,2574,skipping-rope RNA,\"Argasinska J, Weinberg Z\",Weinberg Z,36.7,36.7,36.5,\"Firmicutes, Fusobacteria, AlphaProteobacteria, Spirochaetes. Downstream gene\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.15 -Z 742849.287494 --mxsize 128 CM SEQDB,1426,173,0,0,Gene; sRNA;,Published; PMID:28977401;,360,0,,\"\",0.61173,-4.55766,1600000,167702,417,85,true,-3.7064,0.71859,2018-07-04T15:28:53.000Z,2019-01-17T14:00:28.000Z\n",
            "RF02925,6A,2575,6A RNA,Weinberg Z,Weinberg Z,41.2,41.2,40.6,\"Actinobacteria (1 example), Firmicutes, Fusobacteria. Terminator (Clostridia)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.56 -Z 742849.287494 --mxsize 128 CM SEQDB,273,20,0,0,Gene; sRNA;,Published; PMID:28977401;,41,0,,\"\",0.60914,-4.16536,1600000,288869,329,52,true,-3.7535,0.71918,2018-11-26T15:26:14.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02926,DUF2693-FD,2576,DUF2693-FD RNA,Weinberg Z,Weinberg Z,46.7,46.7,33.5,Bacteroides. No recognized genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.93 -Z 742849.287494 --mxsize 128 CM SEQDB,91,5,0,0,Gene; sRNA;,Published; PMID:28977401;,17,0,,\"\",0.68179,-3.8612,1600000,365441,108,49,true,-3.9957,0.7193,2018-12-11T12:46:01.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02927,Actino-ugpB,2577,Actino-ugpB RNA,Weinberg Z,Weinberg Z,42.5,42.5,40.9,Actinobacteria. Lineage (Bifidobacteriales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.42 -Z 742849.287494 --mxsize 128 CM SEQDB,12,3,0,0,Gene; sRNA;,Published; PMID:28977401;,7,0,,\"\",0.51716,-8.61563,1600000,447456,368,61,true,-3.5953,0.71906,2018-12-12T10:23:43.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02928,Actinomyces-1,2578,Actinomyces-1 RNA,Weinberg Z,Weinberg Z,39.1,39.1,36.9,Actinomyces. Lineage (Actinomycetales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.02 -Z 742849.287494 --mxsize 128 CM SEQDB,236,49,0,0,Gene; sRNA;,Published; PMID:28977401;,18,0,,\"\",0.49786,-7.82161,1600000,327799,204,110,true,-3.192,0.7185,2018-12-12T10:40:24.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02929,algC,2579,algC RNA,Weinberg Z,Weinberg Z,33.8,33.8,33.6,Enterobacteriaceae. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.99 -Z 742849.287494 --mxsize 128 CM SEQDB,492,21,0,0,Cis-reg;,Published; PMID:28977401;,472,0,,\"\",0.65008,-3.80271,1600000,289956,117,52,true,-3.8584,0.7192,2018-12-12T10:50:55.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02930,aspS,2580,aspS RNA,Weinberg Z,Weinberg Z,42.5,42.5,36.4,Actinobacteria. Lineage (Coriobacterales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.47 -Z 742849.287494 --mxsize 128 CM SEQDB,58,24,0,0,Cis-reg;,Published; PMID:28977401;,26,0,,\"\",0.49953,-8.76822,1600000,424886,164,80,true,-3.4743,0.71901,2018-12-12T10:53:54.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02931,Bacilli-1,2581,Bacilli-1 RNA,Weinberg Z,Weinberg Z,55.6,55.6,36.6,Bacilli (Firmicutes). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.99 -Z 742849.287494 --mxsize 128 CM SEQDB,145,8,0,0,Gene; sRNA;,Published; PMID:28977401;,92,0,,\"\",0.68054,-4.78659,1600000,347854,143,65,true,-3.2928,0.71894,2018-12-12T10:58:30.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02932,Betaproteobacteria-1,2582,Betaproteobacteria-1 RNA,Weinberg Z,Weinberg Z,49.4,49.4,36.0,Betaproteobacteria. Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.11 -Z 742849.287494 --mxsize 128 CM SEQDB,102,27,0,0,Gene; sRNA;,Published; PMID:28977401;,33,0,,\"\",0.49792,-11.75606,1600000,546090,208,120,true,-3.2297,0.71817,2018-12-12T11:15:28.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02933,ARRPOF,2583,ARRPOF RNA,Weinberg Z,Weinberg Z,127.2,127.2,115.2,Fusobacteria. Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.82 -Z 742849.287494 --mxsize 128 CM SEQDB,35,1,0,0,Gene; sRNA;,Published; PMID:28977401;,3,0,,\"\",0.64508,-8.63235,1600000,385048,413,257,true,-3.4568,0.70799,2018-12-12T12:15:45.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02934,caiA,2584,caiA RNA,Weinberg Z,Weinberg Z,55.5,55.5,41.8,Clostridiales (Firmicutes). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.89 -Z 742849.287494 --mxsize 128 CM SEQDB,14,4,0,0,Gene; sRNA;,Published; PMID:28977401;,8,0,,\"\",0.5081,-6.74151,1600000,260649,159,78,true,-3.479,0.71871,2018-12-12T13:18:43.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02935,che1,2585,che1 RNA,Weinberg Z,Weinberg Z,54.1,54.1,49.3,Streptomyces. Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.87 -Z 742849.287494 --mxsize 128 CM SEQDB,98,186,0,0,Gene; sRNA;,Published; PMID:28977401;,237,0,,\"\",0.43807,-10.33526,1600000,230882,190,91,true,-2.9381,0.71872,2018-12-12T13:24:32.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02936,Chloroflexus-1,2586,Chloroflexus-1 RNA,Weinberg Z,Weinberg Z,181.5,181.5,46.6,Chloroflexus (Chloroflexi). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.13 -Z 742849.287494 --mxsize 128 CM SEQDB,4,1,0,0,Gene; sRNA;,Published; PMID:28977401;,4,0,,\"\",0.66266,-8.46099,1600000,363490,293,171,true,-3.3207,0.71568,2018-12-12T13:45:05.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02937,Clostridiales-2,2587,Clostridiales-2 RNA,Weinberg Z,Weinberg Z,58.6,58.6,58.2,Clostridiales. Environment (Chloroethene soil),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.76 -Z 742849.287494 --mxsize 128 CM SEQDB,44,110,0,0,Gene; sRNA;,Published; PMID:28977401;,13,0,,\"\",0.66744,-4.87234,1600000,220118,205,82,true,-3.8591,0.7187,2018-12-12T13:58:26.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02938,COG2827,2588,COG2827 RNA,Weinberg Z,Weinberg Z,49.1,49.1,38.5,Clostridiales. Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 742849.287494 --mxsize 128 CM SEQDB,48,20,0,0,Gene; sRNA;,Published; PMID:28977401;,30,0,,\"\",0.53014,-7.314,1600000,291260,206,80,true,-3.0375,0.71904,2018-12-12T14:12:05.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02939,COG2908,2589,COG2908 RNA,Weinberg Z,Weinberg Z,58.7,58.7,32.2,Freshwater metagenome only. Environment (Freshwater),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.80 -Z 742849.287494 --mxsize 128 CM SEQDB,9,0,0,0,Cis-reg;,Published; PMID:28977401;,1,0,,\"\",0.78954,-3.12525,1600000,343352,113,45,true,-4.0277,0.71953,2018-12-12T14:49:16.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02940,COG3860,2590,COG3860 RNA,Weinberg Z,Weinberg Z,42.5,42.5,41.6,Proteobacteria. Lineage (Rhizobiales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.88 -Z 742849.287494 --mxsize 128 CM SEQDB,42,122,0,0,Cis-reg;,Published; PMID:28977401;,134,0,,\"\",0.60949,-8.51983,1600000,441135,125,54,true,-3.7599,0.71921,2018-12-12T16:19:16.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02941,COG3943,2591,COG3943 RNA,Weinberg Z,Weinberg Z,71.8,71.8,36.3,cow rumen. Environment (Cow rumen (57)),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.52 -Z 742849.287494 --mxsize 128 CM SEQDB,27,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.52276,-8.01915,1600000,252317,202,108,true,-3.6438,0.71853,2018-12-12T16:23:38.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02942,Clostridiales-3,2592,Clostridiales-3 RNA,Weinberg Z,Weinberg Z,103.4,103.4,55.8,Clostridiales (Firmicutes). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.17 -Z 742849.287494 --mxsize 128 CM SEQDB,157,16,0,0,Gene; sRNA;,Published; PMID:28977401;,21,0,,\"\",0.67697,-8.987,1600000,408170,558,262,true,-3.713,0.70778,2018-12-12T16:34:45.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02943,Bacteroidales-2,2593,Bacteroidales-2 RNA,Weinberg Z,Weinberg Z,204.3,204.3,46.6,Bacteroidales. Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.63 -Z 742849.287494 --mxsize 128 CM SEQDB,43,3,0,0,Gene; sRNA;,Published; PMID:28977401;,11,0,,\"\",0.68634,-8.05042,1600000,383705,698,392,true,-4.1808,0.70288,2018-12-12T16:37:46.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02945,Corio-PBP,2594,Corio-PBP RNA,Weinberg Z,Weinberg Z,63.5,63.5,41.3,Actinobacteria. Lineage (Coriobacterales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.65 -Z 742849.287494 --mxsize 128 CM SEQDB,14,16,0,0,Gene; sRNA;,Published; PMID:28977401;,17,0,,\"\",0.59775,-4.70682,1600000,297772,321,74,true,-3.5267,0.71905,2018-12-12T16:50:43.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02946,cow-rumen-1,2595,cow-rumen-1 RNA,Weinberg Z,Weinberg Z,53.4,53.4,37.9,cow rumen metagenome. Environment (Cow rumen (57)),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.01 -Z 742849.287494 --mxsize 128 CM SEQDB,50,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.67235,-5.04147,1600000,333806,146,61,true,-3.6203,0.71905,2018-12-12T16:55:03.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02947,cow-rumen-2,2595,cow-rumen-2 RNA,Weinberg Z,Weinberg Z,66.0,66.6,36.8,cow rumen metagenome. Environment (Cow rumen (57)),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,32,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.5442,-6.75247,1600000,231072,193,106,true,-3.9191,0.71837,2018-12-12T16:56:58.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02948,cow-rumen-4,2595,cow-rumen-4 RNA,Weinberg Z,Weinberg Z,48.5,48.5,38.7,cow rumen metagenome. Environment (Cow rumen (57)),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.83 -Z 742849.287494 --mxsize 128 CM SEQDB,8,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.5205,-7.85552,1600000,709503,137,64,true,-3.4666,0.72292,2018-12-12T16:58:45.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03083,cow-rumen-3,2595,cow-rumen-3 RNA,Weinberg Z,Weinberg Z,86.7,86.7,35.9,cow rumen metagenome. Environment (Cow rumen (57)),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.47 -Z 742849.287494 --mxsize 128 CM SEQDB,8,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.61527,-8.68603,1600000,454988,248,141,true,-3.3036,0.71819,2019-01-04T11:26:09.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02949,Cupriavidus-1,2596,Cupriavidus-1 RNA,Weinberg Z,Weinberg Z,45.5,45.5,40.4,Betaproteobacteria. Lineage (Betaproteobacteria),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.11 -Z 742849.287494 --mxsize 128 CM SEQDB,5,9,0,0,Gene; sRNA;,Published; PMID:28977401;,9,0,,\"\",0.45106,-10.25106,1600000,428436,161,78,true,-3.545,0.71901,2018-12-12T17:01:06.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02950,D12-methyl,2597,D12-methyl RNA,Weinberg Z,Weinberg Z,57.0,57.0,37.2,Environmental. Terminator (Clostridia),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.44 -Z 742849.287494 --mxsize 128 CM SEQDB,19,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.51703,-9.44339,1600000,412742,147,64,true,-3.6882,0.71906,2018-12-12T17:05:26.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02951,DABA-DC-AT,2598,DABA-DC-AT RNA,Weinberg Z,Weinberg Z,56.1,56.1,36.1,Gammaproteobacteria. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.01 -Z 742849.287494 --mxsize 128 CM SEQDB,126,55,0,0,Cis-reg;,Published; PMID:28977401;,159,0,,\"\",0.63569,-6.93177,1600000,209961,443,151,true,-3.3907,0.71796,2018-12-12T17:08:10.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02952,dfrA-dnaX,2599,dfrA-dnaX RNA,Weinberg Z,Weinberg Z,48.6,48.6,45.1,Parvimonas (Firmicutes). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 39.34 -Z 742849.287494 --mxsize 128 CM SEQDB,138,20,0,0,Gene; sRNA;,Published; PMID:28977401;,5,0,,\"\",0.40238,-6.73727,1600000,243262,153,76,true,-3.1544,0.71873,2018-12-12T17:10:48.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02953,DUF1874,2600,DUF1874 RNA,Weinberg Z,Weinberg Z,44.8,44.8,39.3,\"Hot springs metagenome only. Environment (Bison Hot Springs, Yellowstone)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.63 -Z 742849.287494 --mxsize 128 CM SEQDB,76,0,0,0,Cis-reg;,Published; PMID:28977401;,1,0,,\"\",0.59269,-4.32523,1600000,199789,176,93,true,-3.5489,0.71865,2018-12-13T09:43:50.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02954,DUF3577,2601,DUF3577 RNA,Weinberg Z,Weinberg Z,42.6,42.6,36.6,Cardiobacterium (Gammaproteobacteria). No recognized genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.82 -Z 742849.287494 --mxsize 128 CM SEQDB,86,4,0,0,Cis-reg;,Published; PMID:28977401;,2,0,,\"\",0.52044,-7.5339,1600000,211367,197,105,true,-3.3936,0.71862,2018-12-13T09:52:33.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02955,EGFOA,2602,EGFOA RNA,Weinberg Z,Weinberg Z,37.7,37.7,36.5,\"Firmicutes, Actinobacteria. Big IGRs\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.00 -Z 742849.287494 --mxsize 128 CM SEQDB,73,26,0,0,Gene; sRNA;,Published; PMID:28977401;,26,0,,\"\",0.52314,-8.84692,1600000,299985,398,107,true,-3.0739,0.71834,2018-12-13T10:10:30.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02970,EGFOA-assoc-1,2602,EGFOA-assoc-1 RNA,Weinberg Z,Weinberg Z,224.5,224.5,34.1,Environmental. Nearby to EGFOA RNA,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.59 -Z 742849.287494 --mxsize 128 CM SEQDB,10,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.71505,-8.0083,1600000,392680,388,250,true,-3.4809,0.70841,2018-12-13T14:21:27.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02956,DUF2693,2603,DUF2693 RNA,Weinberg Z,Weinberg Z,81.3,81.3,36.4,Porphyromonas (Bacteroidetes). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.28 -Z 742849.287494 --mxsize 128 CM SEQDB,63,0,0,0,Gene; sRNA;,Published; PMID:28977401;,2,0,,\"\",0.51744,-10.59411,1600000,415946,366,165,true,-3.1054,0.71633,2018-12-13T10:28:23.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02957,EFASI,2604,EFASI RNA,Weinberg Z,Weinberg Z,51.2,51.2,43.8,Enterobacteriaceae (Gammaproteobacteria). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.25 -Z 742849.287494 --mxsize 128 CM SEQDB,448,105,0,0,Gene; sRNA;,Published; PMID:28977401;,332,0,,\"\",0.5667,-6.06468,1600000,193850,374,134,true,-3.3216,0.71822,2018-12-13T10:46:40.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02958,drum,2605,drum RNA,Weinberg Z,Weinberg Z,43.0,43.0,38.9,\"Firmicutes, Bacteroidetes (1 example), Proteobacteria (2 examples), Spirochaetes (2 examples). Terminator (Clostridia)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.05 -Z 742849.287494 --mxsize 128 CM SEQDB,464,54,0,0,Gene; sRNA;,Published; PMID:28977401;,72,0,,\"\",0.5297,-8.15317,1600000,269874,199,114,true,-3.8096,0.71824,2018-12-13T11:00:01.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02959,DUF3085,2606,DUF3085 RNA,Weinberg Z,Weinberg Z,40.6,40.6,37.2,Thauera (Betaproteobacteria). Environment (Sludge),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.18 -Z 742849.287494 --mxsize 128 CM SEQDB,16,15,0,0,Cis-reg;,Published; PMID:28977401;,2,0,,\"\",0.49606,-10.01901,1600000,365450,157,76,true,-3.1985,0.71893,2018-12-13T11:05:47.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02960,DUF2800,2607,DUF2800 RNA,Weinberg Z,Weinberg Z,60.2,60.2,39.9,\"Actinobacteria, Firmicutes, Synergistetes (1 example). Terminator (Clostridia)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.83 -Z 742849.287494 --mxsize 128 CM SEQDB,155,13,0,0,Cis-reg;,Published; PMID:28977401;,24,0,,\"\",0.51147,-10.49723,1600000,420854,262,158,true,-3.6337,0.71732,2018-12-13T11:22:20.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02965,CyVA-1,2608,CyVA-1 RNA,Weinberg Z,Weinberg Z,41.4,41.4,39.4,\"Cyanobacteria, Acidobacteria (1 example), Verrucomicrobia (1 example). Big IGRs\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.26 -Z 742849.287494 --mxsize 128 CM SEQDB,123,63,0,0,Gene; sRNA;,Published; PMID:28977401;,58,0,,\"\",0.65879,-3.86193,1600000,240038,147,77,true,-4.3154,0.71961,2018-12-13T11:57:47.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02966,DUF3268,2609,DUF3268 RNA,Weinberg Z,Weinberg Z,65.3,65.3,45.2,\"Firmicutes (Bacilli, Clostridia). Downstream gene\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.21 -Z 742849.287494 --mxsize 128 CM SEQDB,108,9,0,0,Gene; sRNA;,Published; PMID:28977401;,13,0,,\"\",0.56773,-6.01275,1600000,189976,290,120,true,-3.5691,0.71832,2018-12-13T12:01:48.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02971,emrB-Lactobacillus,2610,emrB-Lactobacillus RNA,Weinberg Z,Weinberg Z,60.8,60.8,37.2,Lactobacillus (Firmicutes). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.20 -Z 742849.287494 --mxsize 128 CM SEQDB,39,11,0,0,Gene; sRNA;,Published; PMID:28977401;,36,0,,\"\",0.51567,-5.14326,1600000,297020,145,65,true,-3.0313,0.71909,2018-12-13T14:23:43.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02972,engA,2611,engA RNA,Weinberg Z,Weinberg Z,46.0,46.0,38.4,Prevotella (Bacteroidetes). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.78 -Z 742849.287494 --mxsize 128 CM SEQDB,154,26,0,0,Cis-reg;,Published; PMID:28977401;,51,0,,\"\",0.54055,-8.97386,1600000,311107,188,101,true,-3.4064,0.71866,2018-12-13T14:25:24.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02973,Enterococcus-1,2612,Enterococcus-1 RNA,Weinberg Z,Weinberg Z,71.4,71.4,43.0,Enterococcus (Firmicutes). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.16 -Z 742849.287494 --mxsize 128 CM SEQDB,78,7,0,0,Gene; sRNA;,Published; PMID:28977401;,55,0,,\"\",0.57156,-7.14566,1600000,225638,252,152,true,-3.427,0.71802,2018-12-13T14:26:56.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02974,Fibro-purF,2613,Fibro-purF RNA,Weinberg Z,Weinberg Z,60.0,60.0,34.7,Fibrobacteres. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.28 -Z 742849.287494 --mxsize 128 CM SEQDB,12,7,0,0,Cis-reg;,Published; PMID:28977401;,8,0,,\"\",0.71317,-3.91438,1600000,278789,162,75,true,-3.6799,0.71888,2018-12-13T14:28:50.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02976,Flavobacterium-1,2614,Flavobacterium-1 RNA,Weinberg Z,Weinberg Z,58.9,58.9,34.5,Flavobacterium (Bacteroidetes). No recognized genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.87 -Z 742849.287494 --mxsize 128 CM SEQDB,210,0,0,0,Gene; sRNA;,Published; PMID:28977401;,2,0,,\"\",0.56797,-6.79773,1600000,246511,211,120,true,-3.7579,0.71836,2018-12-13T14:39:11.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02977,folE,2615,folE RNA,Weinberg Z,Weinberg Z,42.5,42.5,32.1,Alphaproteobacteria. Upstream of fluoride genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.71 -Z 742849.287494 --mxsize 128 CM SEQDB,45,39,0,0,Cis-reg;,Published; PMID:28977401;,56,0,,\"\",0.73426,-7.29939,1600000,886327,223,39,true,-4.0825,0.71957,2018-12-13T14:46:14.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02978,folP,2616,folP RNA,Weinberg Z,Weinberg Z,99.0,99.0,38.0,Dialister (Negativicutes). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.58 -Z 742849.287494 --mxsize 128 CM SEQDB,28,5,0,0,Cis-reg;,Published; PMID:28977401;,7,0,,\"\",0.62686,-5.17104,1600000,270070,199,102,true,-3.0416,0.7186,2018-12-13T14:48:03.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02979,freshwater-1,2617,freshwater-1 RNA,Weinberg Z,Weinberg Z,72.0,72.0,36.1,Freshwater and estuary metagenome. Environment (Freshwater),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.72 -Z 742849.287494 --mxsize 128 CM SEQDB,8,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.65333,-5.0209,1600000,307971,140,62,true,-3.7149,0.71914,2018-12-13T15:18:36.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02980,freshwater-2,2618,freshwater-2 RNA,Weinberg Z,Weinberg Z,49.9,49.9,48.4,\"Aquatic, esp. fresh water. No recognized genes\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.23 -Z 742849.287494 --mxsize 128 CM SEQDB,200,0,0,0,Cis-reg;,Published; PMID:28977401;,1,0,,\"\",0.46417,-10.0067,1600000,443328,163,85,true,-3.7033,0.71867,2018-12-13T15:20:42.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02981,FTHFS,2619,FTHFS RNA,Weinberg Z,Weinberg Z,72.6,72.6,38.8,human gut. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.17 -Z 742849.287494 --mxsize 128 CM SEQDB,27,4,0,0,Cis-reg;,Published; PMID:28977401;,5,0,,\"\",0.44256,-10.54046,1600000,347980,213,117,true,-3.2159,0.71833,2018-12-13T15:24:02.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02982,gltS,2620,gltS RNA,Weinberg Z,Weinberg Z,47.5,47.5,44.9,Vibrionaceae. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 33.08 -Z 742849.287494 --mxsize 128 CM SEQDB,108,38,0,0,Cis-reg;,Published; PMID:28977401;,129,0,,\"\",0.48008,-5.94363,1600000,294480,221,105,true,-3.2466,0.71831,2018-12-13T15:28:32.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02983,Fibrobacter-1,2621,Fibrobacter-1 RNA,Weinberg Z,Weinberg Z,77.9,77.9,35.4,Fibrobacter (Fibrobacteres). BLAST against metagenomes (Fibrobacteres),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.98 -Z 742849.287494 --mxsize 128 CM SEQDB,4,5,0,0,Gene; sRNA;,Published; PMID:28977401;,6,0,,\"\",0.60978,-5.72832,1600000,292095,200,102,true,-3.4892,0.71858,2018-12-13T15:29:55.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02985,ftsZ-DE,2622,ftsZ-DE RNA,Weinberg Z,Weinberg Z,72.2,72.2,41.8,Fibrobacter (Fibrobacteres). BLAST against metagenomes (Fibrobacteres),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.78 -Z 742849.287494 --mxsize 128 CM SEQDB,10,7,0,0,Gene; sRNA;,Published; PMID:28977401;,8,0,,\"\",0.53071,-9.07009,1600000,394098,153,69,true,-3.7151,0.71909,2018-12-20T11:15:33.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02986,FuFi-1,2623,FuFi-1 RNA,Weinberg Z,Weinberg Z,59.6,59.6,46.4,\"Firmicutes, Fusobacteria. Big IGRs\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.88 -Z 742849.287494 --mxsize 128 CM SEQDB,170,12,0,0,Gene; sRNA;,Published; PMID:28977401;,15,0,,\"\",0.61157,-9.17421,1600000,381695,1022,181,true,-4.0796,0.7142,2018-12-20T11:20:14.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02987,GA-cis,2624,GA-cis RNA,Weinberg Z,Weinberg Z,34.7,34.7,34.1,\"Firmicutes. Environment (Human gut, Qin et al., obese & healthy subjects)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,481,150,0,0,Gene; sRNA;,Published; PMID:28977401;,20,0,,\"\",0.61422,-6.11879,1600000,346020,156,88,true,-3.8899,0.71857,2018-12-20T11:28:42.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02988,GEBRO,2625,GEBRO RNA,Weinberg Z,Weinberg Z,284.7,284.7,36.4,Streptococcus (Firmicutes). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.02 -Z 742849.287494 --mxsize 128 CM SEQDB,42,0,0,0,Gene; sRNA;,Published; PMID:28977401;,7,0,,\"\",0.7184,-8.586,1600000,429386,498,350,true,-3.6194,0.70407,2018-12-20T11:34:47.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02989,gntR-DTE,2626,gntR-DTE RNA,Weinberg Z,Weinberg Z,63.6,63.6,37.7,Actinobacteria. Lineage (Actinomycetales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.19 -Z 742849.287494 --mxsize 128 CM SEQDB,7,23,0,0,Gene; sRNA;,Published; PMID:28977401;,25,0,,\"\",0.58249,-5.42777,1600000,215099,391,120,true,-3.2581,0.71838,2018-12-20T11:38:40.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02990,gut-2,2627,gut-2 RNA,Weinberg Z,Weinberg Z,39.8,39.8,36.1,gut metagenomes. Environment (Cow rumen (57)),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.53 -Z 742849.287494 --mxsize 128 CM SEQDB,73,8,0,0,Gene; sRNA;,Published; PMID:28977401;,9,0,,\"\",0.59744,-8.0536,1600000,336767,102,48,true,-3.7052,0.72989,2018-12-20T11:39:57.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02991,GP20-a,2628,GP20-a RNA,Weinberg Z,Weinberg Z,47.2,47.2,39.5,gut metagenomes. Environment (Cow rumen (57)),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.39 -Z 742849.287494 --mxsize 128 CM SEQDB,54,0,0,0,Cis-reg;,Published; PMID:28977401;,1,0,,\"\",0.52153,-6.2806,1600000,259049,176,97,true,-3.6465,0.7184,2018-12-20T11:42:40.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03003,GP20-b,2628,GP20-b RNA,Weinberg Z,Weinberg Z,36.2,36.6,35.6,Clostridia. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.51 -Z 742849.287494 --mxsize 128 CM SEQDB,210,48,0,0,Cis-reg;,Published; PMID:28977401;,60,0,,\"\",0.62401,-6.18865,1600000,449286,132,62,true,-3.0428,0.72224,2018-12-20T17:02:05.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02992,hya,2629,hya RNA,Weinberg Z,Weinberg Z,56.8,56.8,48.3,Actinobacteria. Lineage (Actinomycetales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.47 -Z 742849.287494 --mxsize 128 CM SEQDB,27,56,0,0,Cis-reg;,Published; PMID:28977401;,71,0,,\"\",0.50668,-8.29389,1600000,265153,156,72,true,-3.2561,0.71884,2018-12-20T11:54:48.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02993,ilvH,2630,ilvH RNA,Weinberg Z,Weinberg Z,55.1,55.1,36.0,Betaproteobacteria. Lineage (Betaproteobacteria),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.76 -Z 742849.287494 --mxsize 128 CM SEQDB,17,76,0,0,Cis-reg;,Published; PMID:28977401;,80,0,,\"\",0.61523,-4.78981,1600000,312656,151,68,true,-3.2736,0.71902,2018-12-20T15:36:50.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02994,IMPDH,2631,IMPDH RNA,Weinberg Z,Weinberg Z,95.4,95.4,36.7,Faecalibacterium (Firmicutes). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.60 -Z 742849.287494 --mxsize 128 CM SEQDB,81,8,0,0,Gene; sRNA;,Published; PMID:28977401;,9,0,,\"\",0.6171,-5.53751,1600000,257912,208,109,true,-3.1377,0.71832,2018-12-20T15:39:48.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02995,IMES-5,2632,IMES-5 RNA,Weinberg Z,Weinberg Z,96.1,96.1,41.5,Environmental. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.18 -Z 742849.287494 --mxsize 128 CM SEQDB,52,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.54091,-8.74009,1600000,344431,256,150,true,-3.1676,0.71808,2018-12-20T15:41:16.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02996,int-alpA,2633,int-alpA RNA,Weinberg Z,Weinberg Z,35.8,35.8,35.7,\"Acidobacteria (1 example), Planctomycetes (1 example), Proteobacteria. Environment (Lake Washington)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.91 -Z 742849.287494 CM SEQDB,167,407,0,0,Gene; sRNA;,Published; PMID:28977401;,344,0,,\"\",0.59411,-6.21441,1600000,218126,159,71,true,-3.4909,0.72486,2018-12-20T15:42:52.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02997,HOLDH,2634,HOLDH RNA,Weinberg Z,Weinberg Z,319.4,319.4,31.0,\"Environmental (human tongue dorsum, human supragingival plaque). Downstream gene\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.34 -Z 742849.287494 --mxsize 128 CM SEQDB,13,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.65067,-8.76661,1600000,362189,576,391,true,-4.2443,0.70292,2018-12-20T15:43:46.000Z,2019-01-10T15:16:03.000Z\n",
            "RF02998,IS605-orfB-II,2635,IS605-orfB-II RNA,Weinberg Z,Weinberg Z,88.0,88.0,41.0,Environmental. Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.64 -Z 742849.287494 --mxsize 128 CM SEQDB,56,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.46239,-10.13928,1600000,332540,264,154,true,-2.901,0.7179,2018-12-20T15:49:42.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03065,IS605-orfB-I,2635,IS605-orfB-I RNA,Weinberg Z,Weinberg Z,94.2,94.2,91.4,Enterococcus (Firmicutes). No recognized genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,259,1,0,0,Gene; sRNA;,Published; PMID:28977401;,2,0,,\"\",0.60897,-8.18893,1600000,410914,342,214,true,-3.3722,0.71099,2018-12-24T16:44:40.000Z,2022-10-29T17:55:17.000Z\n",
            "RF02999,ivy-DE,2636,ivy-DE RNA,Weinberg Z,Weinberg Z,41.4,41.4,39.0,Pseudomonas. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 742849.287494 --mxsize 128 CM SEQDB,248,56,0,0,Cis-reg;,Published; PMID:28977401;,231,0,,\"\",0.56075,-4.84011,1600000,213915,181,93,true,-3.2869,0.71869,2018-12-20T15:53:32.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03000,LOOT,2637,LOOT RNA,Weinberg Z,Weinberg Z,46.4,49.5,45.8,\"Latescibacteria, OD1/Parcubacteria, OP11, TM7. Lineage (Microgenomates, aka OP11)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,368,330,0,0,Gene; sRNA;,Published; PMID:28977401;,245,0,,\"\",0.53853,-9.19442,1600000,404553,194,121,true,-3.4212,0.71803,2018-12-20T16:06:27.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03001,leuA-Halobacteria,2638,leuA-Halobacteria RNA,Weinberg Z,Weinberg Z,47.9,47.9,32.3,Halobacteriaceae (Euryarchaeota). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.79 -Z 742849.287494 --mxsize 128 CM SEQDB,152,59,0,0,Cis-reg;,Published; PMID:28977401;,101,0,,\"\",0.56164,-8.58303,1600000,296745,122,53,true,-4.0805,0.71922,2018-12-20T16:42:00.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03002,lysM-Prevotella,2639,lysM-Prevotella RNA,Weinberg Z,Weinberg Z,66.6,66.6,40.5,Prevotella (Bacteroidetes). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.72 -Z 742849.287494 --mxsize 128 CM SEQDB,47,3,0,0,Cis-reg;,Published; PMID:28977401;,11,0,,\"\",0.52851,-4.67144,1600000,286063,145,66,true,-3.1991,0.71896,2018-12-20T16:50:42.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03005,lysM-TM7,2639,lysM-TM7 RNA,Weinberg Z,Weinberg Z,61.5,61.5,41.5,TM7. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.34 -Z 742849.287494 --mxsize 128 CM SEQDB,129,9,0,0,Cis-reg;,Published; PMID:28977401;,11,0,,\"\",0.53295,-8.1351,1600000,350321,292,140,true,-3.1005,0.71805,2018-12-20T17:11:50.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03090,lysM-Actino,2639,lysM-Actino RNA,Weinberg Z,Weinberg Z,153.8,153.8,64.1,Actinomycetales. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 21.00 -Z 742849.287494 --mxsize 128 CM SEQDB,56,2,0,0,Cis-reg;,Published; PMID:28977401;,7,0,,\"\",0.63128,-9.37751,1600000,458418,342,213,true,-3.3894,0.71116,2019-01-15T09:48:57.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03004,Lacto-phage-1,2640,Lacto-phage-1 RNA,Weinberg Z,Weinberg Z,50.0,69.1,35.0,Lactobacillales (Firmicutes). Lineage (Lactobacillaceae),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.67 -Z 742849.287494 --mxsize 128 CM SEQDB,9,1,0,0,Gene; sRNA;,Published; PMID:28977401;,4,0,,\"\",0.53672,-8.60377,1600000,359705,170,81,true,-3.4095,0.71874,2018-12-20T17:07:56.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03006,M23,2641,M23 RNA,Weinberg Z,Weinberg Z,41.2,41.2,40.6,Clostridia. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.59 -Z 742849.287494 --mxsize 128 CM SEQDB,31,5,0,0,Gene; sRNA;,Published; PMID:28977401;,7,0,,\"\",0.58891,-6.10368,1600000,495825,121,54,true,-3.3238,0.7193,2018-12-20T17:14:17.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03007,Mahella-1,2642,Mahella-1 RNA,Weinberg Z,Weinberg Z,70.7,70.7,34.4,Mahella australiensis (Firmicutes). Terminator (Clostridia),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.22 -Z 742849.287494 --mxsize 128 CM SEQDB,19,7,0,0,Gene; sRNA;,Published; PMID:28977401;,2,0,,\"\",0.54754,-8.75297,1600000,257993,209,114,true,-3.1211,0.71835,2018-12-20T17:15:40.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03008,malK-II,2643,malK-II RNA,Weinberg Z,Weinberg Z,52.9,52.9,39.2,Vibrio (Proteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.63 -Z 742849.287494 --mxsize 128 CM SEQDB,94,20,0,0,Cis-reg;,Published; PMID:28977401;,103,0,,\"\",0.50401,-8.80521,1600000,557174,209,50,true,-3.4928,0.71933,2018-12-20T17:16:49.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03009,malK-III,2643,malK-III RNA,Weinberg Z,Weinberg Z,51.9,51.9,34.9,Environmental. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 24.06 -Z 742849.287494 --mxsize 128 CM SEQDB,39,3,0,0,Cis-reg;,Published; PMID:28977401;,4,0,,\"\",0.63612,-4.45592,1600000,162956,341,95,true,-3.4538,0.71863,2018-12-20T17:18:17.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03069,malK-I,2643,malK-I RNA,Weinberg Z,Weinberg Z,60.3,60.3,44.3,Clostridia. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.97 -Z 742849.287494 --mxsize 128 CM SEQDB,322,27,0,0,Cis-reg;,Published; PMID:28977401;,33,0,,\"\",0.50852,-8.87241,1600000,294490,399,179,true,-3.8052,0.71455,2019-01-03T12:25:46.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03010,mcrA,2644,mcrA RNA,Weinberg Z,Weinberg Z,57.6,57.6,35.6,Listeria (Firmicutes). Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.08 -Z 742849.287494 --mxsize 128 CM SEQDB,16,6,0,0,Gene; sRNA;,Published; PMID:28977401;,20,0,,\"\",0.70606,-7.5931,1600000,660987,125,51,true,-3.9724,0.71919,2018-12-20T17:19:42.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03011,Methylophilales-1,2645,Methylophilales-1 RNA,Weinberg Z,Weinberg Z,79.9,79.9,78.6,\"Methylophilales, lake sediment metagenome. Environment (Lake Washington)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.14 -Z 742849.287494 --mxsize 128 CM SEQDB,14,11,0,0,Gene; sRNA;,Published; PMID:28977401;,12,0,,\"\",0.63592,-2.57064,1600000,182779,206,103,true,-3.5111,0.71856,2018-12-20T17:21:48.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03012,Mu-gpT-DE,2646,Mu-gpT-DE RNA,Weinberg Z,Weinberg Z,50.2,50.2,49.7,Environmental. No recognized genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 --mxsize 128 CM SEQDB,316,10,0,0,Gene; sRNA;,Published; PMID:28977401;,2,0,,\"\",0.65146,-6.50986,1600000,453227,118,53,true,-3.8955,0.71922,2018-12-20T17:29:53.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03013,nadA,2647,nadA RNA,Weinberg Z,Weinberg Z,39.6,39.6,37.8,Acidobacteria. Lineage (Acidobacteria),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,182,13,0,0,Gene; sRNA;,Published; PMID:28977401;,11,0,,\"\",0.67791,-4.69817,1600000,364751,239,66,true,-2.982,0.72283,2018-12-20T17:31:30.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03014,Transposase-1,2648,Transposase-1 RNA,Weinberg Z,Weinberg Z,42.8,42.8,40.8,\"Lactobacillales (Firmicutes). Environment (Human Microbiome Project, vagina)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,176,66,0,0,Gene; sRNA;,Published; PMID:28977401;,37,0,,\"\",0.66041,-3.87573,1600000,243483,120,59,true,-4.0516,0.71907,2018-12-20T17:38:08.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03015,Transposase-2,2649,Transposase-2 RNA,Weinberg Z,Weinberg Z,42.7,42.7,42.1,\"Actinobacteria, Firmicutes. Lineage (Bifidobacteriales)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,151,74,0,0,Gene; sRNA;,Published; PMID:28977401;,51,0,,\"\",0.57735,-8.42723,1600000,503431,138,69,true,-3.6346,0.71889,2018-12-20T17:39:29.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03023,rpfG,2650,rpfG RNA,Weinberg Z,Weinberg Z,75.9,75.9,34.4,Xanthomonas (Gammaproteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.54 -Z 742849.287494 --mxsize 128 CM SEQDB,28,3,0,0,Gene; sRNA;,Published; PMID:28977401;,30,0,,\"\",0.60371,-7.81696,1600000,359683,175,85,true,-3.1266,0.71869,2018-12-20T17:59:07.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03024,Rothia-sucC,2651,Rothia-sucC RNA,Weinberg Z,Weinberg Z,64.5,64.5,44.6,Rothia (Actinobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.11 -Z 742849.287494 --mxsize 128 CM SEQDB,28,2,0,0,Cis-reg;,Published; PMID:28977401;,8,0,,\"\",0.57071,-6.49905,1600000,261019,139,59,true,-3.4319,0.71906,2018-12-20T18:01:11.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03030,salivarius-1,2652,salivarius-1 RNA,Weinberg Z,Weinberg Z,55.3,55.3,40.5,Lactobacillus salivarius (Firmicutes). Lineage (Lactobacillaceae),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.17 -Z 742849.287494 --mxsize 128 CM SEQDB,19,5,0,0,Gene; sRNA;,Published; PMID:28977401;,5,0,,\"\",0.63916,-4.57871,1600000,390336,126,56,true,-3.8795,0.7193,2018-12-20T18:24:12.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03031,ssnA,2653,ssnA RNA,Weinberg Z,Weinberg Z,37.3,37.3,34.7,Clostridiales. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 20.89 -Z 742849.287494 --mxsize 128 CM SEQDB,118,18,0,0,Cis-reg;,Published; PMID:28977401;,22,0,,\"\",0.7716,-3.22567,1600000,260451,99,48,true,-4.5291,0.72089,2018-12-20T18:33:35.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03032,narK,2654,narK RNA,Weinberg Z,Weinberg Z,43.2,43.2,42.4,Betaproteobacteria and Gammaproteobacteria. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.05 -Z 742849.287494 --mxsize 128 CM SEQDB,308,89,0,0,Cis-reg;,Published; PMID:28977401;,239,0,,\"\",0.5412,-9.96034,1600000,627290,148,72,true,-3.5508,0.71877,2018-12-20T18:57:28.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03033,NLPC-P60,2655,NLPC-P60 RNA,Weinberg Z,Weinberg Z,53.7,53.7,35.0,Streptomyces (Actinobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.16 -Z 742849.287494 --mxsize 128 CM SEQDB,17,15,0,0,Cis-reg;,Published; PMID:28977401;,29,0,,\"\",0.49672,-8.03055,1600000,227436,204,97,true,-2.783,0.71855,2018-12-21T09:30:52.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03034,nrdJ,2656,nrdJ RNA,Weinberg Z,Weinberg Z,40.3,40.3,40.1,Streptomyces (Actinobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.64 -Z 742849.287494 --mxsize 128 CM SEQDB,73,151,0,0,Gene; sRNA;,Published; PMID:28977401;,190,0,,\"\",0.56852,-6.37616,1600000,306376,160,80,true,-3.3824,0.71886,2018-12-21T09:40:06.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03035,nqrA-Marinomonas,2657,nqrA-Marinomonas RNA,Weinberg Z,Weinberg Z,95.5,95.5,41.4,Marinomonas (Gammaproteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.01 -Z 742849.287494 --mxsize 128 CM SEQDB,5,7,0,0,Gene; sRNA;,Published; PMID:28977401;,9,0,,\"\",0.55334,-4.66267,1600000,265549,242,134,true,-3.265,0.71818,2018-12-21T09:40:59.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03036,osmY,2658,osmY RNA,Weinberg Z,Weinberg Z,46.9,46.9,40.0,Enterobacteriaceae (Gammaproteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.54 -Z 742849.287494 --mxsize 128 CM SEQDB,38,24,0,0,Gene; sRNA;,Published; PMID:28977401;,45,0,,\"\",0.64873,-6.85106,1600000,410076,148,72,true,-3.3872,0.71893,2018-12-21T09:48:00.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03037,PAGEV,2659,PAGEV RNA,Weinberg Z,Weinberg Z,61.1,61.1,38.2,Gammaproteobacteria. Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.37 -Z 742849.287494 --mxsize 128 CM SEQDB,86,6,0,0,Gene; sRNA;,Published; PMID:28977401;,29,0,,\"\",0.68577,-8.26233,1600000,364448,637,196,true,-3.5205,0.71259,2018-12-21T09:51:03.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03038,nhaA-II,2660,nhaA-II RNA,Weinberg Z,Weinberg Z,70.4,70.4,37.5,Caulobacterales (Alphaproteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 32.11 -Z 742849.287494 --mxsize 128 CM SEQDB,16,18,0,0,Cis-reg;,Published; PMID:28977401;,18,0,,\"\",0.44484,-10.08443,1600000,305519,195,84,true,-3.1781,0.71893,2018-12-21T09:57:10.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03039,Peptidase-S11,2661,Peptidase-S11 RNA,Weinberg Z,Weinberg Z,34.8,34.8,33.8,\"Enterobacteria. Environment (Termite gut, fungus grower)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.44 -Z 742849.287494 --mxsize 128 CM SEQDB,135,53,0,0,Cis-reg;,Published; PMID:28977401;,133,0,,\"\",0.55453,-4.91842,1600000,232144,161,84,true,-3.2561,0.71869,2018-12-21T10:00:45.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03040,PGK,2662,PGK RNA,Weinberg Z,Weinberg Z,50.8,50.8,43.8,mammalian gut metagenomes. Environment (Cow rumen (57)),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 37.81 -Z 742849.287494 --mxsize 128 CM SEQDB,18,8,0,0,Cis-reg;,Published; PMID:28977401;,8,0,,\"\",0.4287,-6.45379,1600000,374896,179,93,true,-3.0117,0.71856,2018-12-21T10:02:52.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03041,poplar-1,2663,poplar-1 RNA,Weinberg Z,Weinberg Z,110.5,110.5,36.0,\"metagenome only. Environment (Groundwater, Oak Ridge)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.53 -Z 742849.287494 --mxsize 128 CM SEQDB,22,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.60183,-8.74858,1600000,322413,246,141,true,-3.1146,0.7181,2018-12-21T10:05:01.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03042,porB,2664,porB RNA,Weinberg Z,Weinberg Z,35.1,35.1,33.9,Neisseria (Betaproteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.40 -Z 742849.287494 --mxsize 128 CM SEQDB,85,18,0,0,Gene; sRNA;,Published; PMID:28977401;,78,0,,\"\",0.50282,-8.61033,1600000,429185,138,61,true,-2.8442,0.71905,2018-12-21T10:08:00.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03043,Prevotella-2,2665,Prevotella-2 RNA,Weinberg Z,Weinberg Z,72.1,72.1,41.7,Prevotella. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.36 -Z 742849.287494 --mxsize 128 CM SEQDB,20,4,0,0,Gene; sRNA;,Published; PMID:28977401;,8,0,,\"\",0.43472,-8.94853,1600000,322689,216,116,true,-3.1618,0.71839,2018-12-21T10:10:31.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03044,Proteo-phage-1,2666,Proteo-phage-1 RNA,Weinberg Z,Weinberg Z,42.0,42.0,41.7,Proteobacteria. Environment (Dendroctonus ponderosae gallery),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.93 -Z 742849.287494 --mxsize 128 CM SEQDB,67,40,0,0,Gene; sRNA;,Published; PMID:28977401;,54,0,,\"\",0.52512,-8.03098,1600000,342306,174,82,true,-3.2889,0.71859,2018-12-21T10:14:03.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03045,proV,2667,proV RNA,Weinberg Z,Weinberg Z,88.5,88.5,34.0,Pseudomonas (Gammaproteobacteria). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.62 -Z 742849.287494 --mxsize 128 CM SEQDB,16,12,0,0,Gene; sRNA;,Published; PMID:28977401;,21,0,,\"\",0.58159,-9.75453,1600000,324080,257,149,true,-3.3418,0.71804,2018-12-21T10:16:50.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03046,Pseudomonadales-1,2668,Pseudomonadales-1 RNA,Weinberg Z,Weinberg Z,68.0,69.3,67.6,Pseudomonadales (Gammaproteobacteria). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,54,19,0,0,Gene; sRNA;,Published; PMID:28977401;,41,0,,\"\",0.62792,-8.19265,1600000,291651,280,164,true,-3.2992,0.71639,2018-12-21T10:18:34.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03053,OTKONC,2669,OTKONC RNA,Weinberg Z,Weinberg Z,141.4,141.4,39.2,Environmental. No recognized genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 31.61 -Z 742849.287494 --mxsize 128 CM SEQDB,43,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.46015,-9.4312,1600000,343105,283,166,true,-3.0339,0.71603,2018-12-21T10:52:10.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03054,Xanthine,2670,Xanthine riboswitch (NMT1 RNA),\"Weinberg Z, Ontiveros-Palacios N\",Weinberg Z,36.0,36.7,35.6,\"Xanthine riboswitch was first reported in Proteobacteria as NMT1 non-coding RNA (ncRNA) in [1]. Later it was reported as a ncRNA that recognised Xanthine [2]. More recently the structure of Xanthine riboswitch was reported in 7ELP, 7ELQ, 7ELR and 7ELS PDBs [3]\",NMT1;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,383,192,0,0,Cis-reg;,Published; PMID:28977401;,278,0,,\"\",0.63736,-4.62233,1600000,160049,151,56,true,-3.8795,0.72144,2018-12-21T10:55:41.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03057,nhaA-I,2671,nhaA-I RNA,Weinberg Z,Weinberg Z,38.7,38.8,38.5,\"Acidobacteria, Alphaproteobacteria, Betaproteobacteria and Gammaproteobacteria, Verrucomicrobia, cand. division NC10. Downstream gene\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.99 -Z 742849.287494 --mxsize 128 CM SEQDB,281,277,0,0,Cis-reg; riboswitch;,Published; PMID:28977401;,299,0,,\"\",0.59251,-9.15327,1600000,727800,131,56,true,-3.1855,0.72303,2018-12-21T11:55:37.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03058,sul1,2672,sul1 RNA,Weinberg Z,Weinberg Z,36.2,36.2,35.6,Alphaproteobacteria. Lineage (Rhizobiales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.81 -Z 742849.287494 --mxsize 128 CM SEQDB,54,289,0,0,Cis-reg; riboswitch;,Published; PMID:28977401;,268,0,,\"\",0.53073,-8.61133,1600000,313864,125,56,true,-3.8051,0.71923,2018-12-21T12:14:18.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03059,raiA-hairpin,2673,raiA-hairpin RNA,Weinberg Z,Weinberg Z,43.7,43.7,36.4,Streptomyces (Actinobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.57 -Z 742849.287494 --mxsize 128 CM SEQDB,80,163,0,0,Gene; sRNA;,Published; PMID:28977401;,203,0,,\"\",0.52168,-9.85954,1600000,649435,114,48,true,-3.755,0.71937,2018-12-22T22:53:55.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03060,uup,2674,uup RNA,Weinberg Z,Weinberg Z,36.8,36.8,35.5,\"Firmicutes, Gammaproteobacteria. Downstream gene\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.60 -Z 742849.287494 --mxsize 128 CM SEQDB,203,84,0,0,Cis-reg;,Published; PMID:28977401;,196,0,,\"\",0.55788,-8.06593,1600000,541234,103,48,true,-4.3012,0.71928,2018-12-22T22:55:57.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03061,uxuA,2675,uxuA RNA,Weinberg Z,Weinberg Z,66.8,66.8,34.0,Vibrio (Gammaproteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.82 -Z 742849.287494 --mxsize 128 CM SEQDB,33,3,0,0,Gene; sRNA;,Published; PMID:28977401;,33,0,,\"\",0.59425,-4.20745,1600000,218667,178,89,true,-3.4347,0.7186,2018-12-22T22:57:27.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03062,xerDC,2676,xerDC RNA,Weinberg Z,Weinberg Z,72.5,72.5,48.5,Clostridia. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 29.43 -Z 742849.287494 --mxsize 128 CM SEQDB,72,4,0,0,Cis-reg;,Published; PMID:28977401;,5,0,,\"\",0.46867,-10.56846,1600000,297713,259,130,true,-2.7214,0.71834,2018-12-22T22:58:45.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03063,Streptomyces-metK,2677,Streptomyces-metK RNA,Weinberg Z,Weinberg Z,44.2,44.2,41.0,Streptomyces (Actinobacteria). Lineage (Actinomycetales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.00 -Z 742849.287494 --mxsize 128 CM SEQDB,38,191,0,0,Gene; sRNA;,Published; PMID:28977401;,199,0,,\"\",0.53688,-9.27981,1600000,363053,350,117,true,-3.1727,0.71826,2018-12-22T23:01:10.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03076,Streptomyces-metH,2677,Streptomyces-metH RNA,Weinberg Z,Weinberg Z,50.0,50.0,36.1,Streptomyces (Actinobacteria). Lineage (Actinomycetales),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.30 -Z 742849.287494 --mxsize 128 CM SEQDB,43,194,0,0,Gene; sRNA;,Published; PMID:28977401;,203,0,,\"\",0.49015,-10.43005,1600000,377848,274,122,true,-3.6542,0.71821,2019-01-03T14:37:36.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03066,COG3610-DE,2678,COG3610-DE RNA,Weinberg Z,Weinberg Z,73.5,73.5,47.7,Lactobacillales (Firmicutes). Lineage (Lactobacillaceae),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.61 -Z 742849.287494 --mxsize 128 CM SEQDB,59,16,0,0,Gene; sRNA;,Published; PMID:28977401;,52,0,,\"\",0.5094,-9.70565,1600000,388641,216,116,true,-2.7828,0.71848,2019-01-02T10:35:09.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03067,terC,2679,terC RNA,Weinberg Z,Weinberg Z,39.9,39.9,39.5,\"Alphaproteobacteria, Pseudomonadales. Lineage (Rhizobiales)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 --mxsize 128 CM SEQDB,129,591,0,0,Cis-reg;,Published; PMID:28977401;,426,0,,\"\",0.53153,-5.2333,1600000,142878,194,60,true,-3.8992,0.71934,2019-01-03T09:15:05.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03071,Sodium,2681,Na+ riboswitch (DUF1646 RNA),Weinberg Z,Weinberg Z,40.5,40.6,40.4,\"Proteobacteria, NC10, Acidobacteria, Verrucomicrobia. Lineage (Clostridia) The riboswitch of Na+ was first reported in a comparative analysis of intergenic regions and called DUF1646 RNA, it was also suggested that it acts as cis-reg RNA present upstream of Na+ genes [1]. Afterward it was reported as a riboswitch that selectively senses Na+ [2].\",DUF1646;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,146,125,0,0,Cis-reg; riboswitch;,Published; PMID:28977401;,166,0,,\"\",0.58441,-8.67519,1600000,553595,107,55,true,-3.6149,0.72088,2019-01-03T13:59:50.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03072,raiA,2682,raiA RNA,Weinberg Z,Weinberg Z,101.3,101.4,87.7,\"Actinobacteria, Firmicutes. Lineage (Negativicutes)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,488,248,0,0,Cis-reg; riboswitch;,Published; PMID:28977401;,295,0,,\"\",0.64784,-8.26486,1600000,246043,542,209,true,-3.1599,0.71144,2019-01-03T14:12:32.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03074,Rhodo-rpoB,2683,Rhodo-rpoB RNA,Weinberg Z,Weinberg Z,33.6,33.6,32.2,Rhodobacterales (Alphaproteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.84 -Z 742849.287494 --mxsize 128 CM SEQDB,261,204,0,0,Cis-reg;,Published; PMID:28977401;,232,0,,\"\",0.45927,-11.44147,1600000,584573,160,93,true,-3.8625,0.71856,2019-01-03T14:31:26.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03078,chrB-a,2684,chrB-a RNA,Weinberg Z,Weinberg Z,43.9,43.9,43.6,Alphaproteobacteria. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.00 -Z 742849.287494 --mxsize 128 CM SEQDB,163,244,0,0,Cis-reg;,Published; PMID:28977401;,208,0,,\"\",0.50332,-9.18733,1600000,289902,253,81,true,-3.4044,0.71875,2019-01-04T09:43:52.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03086,chrB-b,2684,chrB-b RNA,Weinberg Z,Weinberg Z,36.5,36.5,36.4,\"Alphaproteobacteria, Betaproteobacteria. Environment (Rainforest soil)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.21 -Z 742849.287494 --mxsize 128 CM SEQDB,313,136,0,0,Cis-reg;,Published; PMID:28977401;,120,0,,\"\",0.55565,-9.9984,1600000,673901,309,51,true,-3.5121,0.71924,2019-01-04T13:13:22.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03079,MISL,2685,MISL RNA,Weinberg Z,Weinberg Z,274.6,274.6,22.4,Verrucomicrobia. Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 17.24 -Z 742849.287494 --mxsize 128 CM SEQDB,44,5,0,0,Gene; sRNA;,Published; PMID:28977401;,6,0,,\"\",0.73559,-8.28017,1600000,305107,4403,747,true,-4.9552,0.69827,2019-01-04T10:32:37.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03082,dinG,2686,dinG RNA,Weinberg Z,Weinberg Z,46.9,46.9,44.9,Clostridiales (Firmicutes). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.91 -Z 742849.287494 --mxsize 128 CM SEQDB,86,25,0,0,Gene; sRNA;,Published; PMID:28977401;,28,0,,\"\",0.49119,-9.71148,1600000,373637,214,86,true,-3.2764,0.71872,2019-01-04T11:21:05.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03084,DUF2815,2687,DUF2815 RNA,Weinberg Z,Weinberg Z,41.2,41.2,35.8,Environmental. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.94 -Z 742849.287494 --mxsize 128 CM SEQDB,22,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.59432,-5.40795,1600000,481605,131,63,true,-3.7283,0.71912,2019-01-04T12:25:27.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03085,abiF,2688,abiF RNA,Weinberg Z,Weinberg Z,37.0,37.0,36.8,\"Actinobacteria, Bacteroidetes, Clostridia, Proteobacteria, one Clostridia phage. Environment (Freshwater)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.18 -Z 742849.287494 --mxsize 128 CM SEQDB,126,186,0,0,Gene; sRNA;,Published; PMID:28977401;,187,0,,\"\",0.56786,-5.88456,1600000,542358,87,37,true,-4.0517,0.72531,2019-01-04T12:40:45.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03087,ROOL,2689,ROOL RNA,Weinberg Z,Weinberg Z,99.1,99.1,29.7,\"Firmicutes, Fusobacteria (1 example), Tenericutes (1 example). Environment (Cow rumen (57))\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.77 -Z 742849.287494 --mxsize 128 CM SEQDB,290,26,0,0,Gene; sRNA;,Published; PMID:28977401;,34,0,,\"\",0.53971,-9.65118,1600000,432902,1133,580,true,-4.5358,0.69972,2019-01-08T09:31:14.000Z,2019-01-10T15:16:03.000Z\n",
            "RF03088,Parabacteroides-1,2690,Parabacteroides-1 RNA,Weinberg Z,Weinberg Z,93.4,93.4,45.1,Parabacteroides (Bacteroidetes). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.60 -Z 742849.287494 --mxsize 128 CM SEQDB,15,1,0,0,Gene; sRNA;,Published; PMID:28977401;,3,0,,\"\",0.6076,-4.83112,1600000,230570,203,105,true,-3.4831,0.71857,2019-01-15T09:42:34.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03089,cold-seep-1,2691,cold-seep-1 RNA,Weinberg Z,Weinberg Z,125.8,125.8,39.4,\"Environmental, Norwegian cold seep. Environment (Cold seep)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.06 -Z 742849.287494 --mxsize 128 CM SEQDB,15,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.52728,-8.23494,1600000,259694,264,152,true,-3.0407,0.71803,2019-01-15T09:47:02.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03091,Clostridium-PBP,2692,Clostridium-PBP RNA,Weinberg Z,Weinberg Z,94.4,94.4,39.7,\"Environmental, Clostridium (Firmicute; 1 example). Downstream gene\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 34.82 -Z 742849.287494 --mxsize 128 CM SEQDB,11,0,0,0,Gene; sRNA;,Published; PMID:28977401;,3,0,,\"\",0.43984,-8.74065,1600000,450712,160,73,true,-3.3518,0.71872,2019-01-15T10:09:23.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03092,Ocean-VII,2693,Ocean-VII RNA,Weinberg Z,Weinberg Z,53.5,53.5,37.7,marine metagenomes. Environment (Puerto Rico trench),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 18.62 -Z 742849.287494 --mxsize 128 CM SEQDB,16,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.78996,-6.15163,1600000,679278,106,41,true,-3.8854,0.7197,2019-01-15T10:17:09.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03093,queA,2694,queA RNA,Weinberg Z,Weinberg Z,52.6,52.6,49.6,Environmental. Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.01 -Z 742849.287494 --mxsize 128 CM SEQDB,8,1,0,0,Gene; sRNA;,Published; PMID:28977401;,2,0,,\"\",0.51905,-10.30523,1600000,557044,164,59,true,-3.5868,0.71918,2019-01-15T10:19:19.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03094,LAGLIDADG-2,2695,LAGLIDADG-2 RNA,Weinberg Z,Weinberg Z,94.9,94.9,42.6,Halobacteria (Archaea). Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.56 -Z 742849.287494 --mxsize 128 CM SEQDB,25,0,0,0,Gene; sRNA;,Published; PMID:28977401;,3,0,,\"\",0.51825,-9.956,1600000,355936,235,128,true,-2.9522,0.71845,2019-01-15T10:22:25.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03107,saliva-tongue-1,2696,saliva-tongue-1 RNA,Weinberg Z,Weinberg Z,107.3,107.3,38.1,Human saliva and tongue dorsum. Terminator (Clostridia),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.09 -Z 742849.287494 --mxsize 128 CM SEQDB,10,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.51126,-8.81933,1600000,337053,213,109,true,-3.1776,0.71831,2019-01-15T10:37:46.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03108,Methylosinus-1,2697,Methylosinus-1 RNA,Weinberg Z,Weinberg Z,145.1,145.1,60.6,Methylosinus trichosporium (Alphaproteobacteria). Big IGRs,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 22.20 -Z 742849.287494 --mxsize 128 CM SEQDB,12,8,0,0,Gene; sRNA;,Published; PMID:28977401;,4,0,,\"\",0.57654,-10.79803,1600000,393530,414,212,true,-3.5688,0.71115,2019-01-15T10:44:09.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03109,Thermales-rpoB,2698,Thermales-rpoB RNA,Weinberg Z,Weinberg Z,77.3,77.3,34.1,Thermales (Deinococcus-Thermus). Lineage (Deinococcus-Thermus),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 23.87 -Z 742849.287494 --mxsize 128 CM SEQDB,14,5,0,0,Cis-reg;,Published; PMID:28977401;,12,0,,\"\",0.67453,-3.58503,1600000,238081,190,89,true,-3.0054,0.71861,2019-01-15T10:55:39.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03110,throat-1,2699,throat-1 RNA,Weinberg Z,Weinberg Z,124.4,124.4,33.3,\"throat and tongue metagenomes. Environment (Human Microbiome Project, throat)\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 19.40 -Z 742849.287494 --mxsize 128 CM SEQDB,27,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.69848,-7.78284,1600000,380357,482,334,true,-3.7447,0.70456,2019-01-15T10:57:20.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03111,Zeta-pan,2700,Zeta-pan RNA,Weinberg Z,Weinberg Z,56.7,56.7,35.8,Zetaproteobacteria. Lineage (Zetaproteobacteria),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.74 -Z 742849.287494 --mxsize 128 CM SEQDB,11,1,0,0,Gene; sRNA;,Published; PMID:28977401;,8,0,,\"\",0.53072,-8.40637,1600000,461981,160,50,true,-3.6631,0.7193,2019-01-15T10:58:14.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03112,Staphylococcus-1,2701,Staphylococcus-1 RNA,Weinberg Z,Weinberg Z,87.7,87.7,35.5,Firmicute (1 example). Nearby to self-splicing intron genes,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.83 -Z 742849.287494 --mxsize 128 CM SEQDB,21,1,0,0,Gene; sRNA;,Published; PMID:28977401;,2,0,,\"\",0.5806,-3.56845,1600000,177690,230,127,true,-3.5682,0.71816,2019-01-15T10:59:16.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03113,Poribacteria-1,2702,Poribacteria-1 RNA,Weinberg Z,Weinberg Z,74.2,74.2,34.6,Poribacteria. Lineage (Poribacteria),,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 28.47 -Z 742849.287494 --mxsize 128 CM SEQDB,6,0,0,0,Gene; sRNA;,Published; PMID:28977401;,1,0,,\"\",0.50243,-8.82066,1600000,295225,209,107,true,-3.4638,0.71835,2019-01-15T11:49:06.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03115,KDPG-aldolase,2703,KDPG-aldolase RNA,Weinberg Z,Weinberg Z,54.6,54.6,33.3,Enterobacteriaceae (Gammaproteobacteria). Downstream gene,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 26.46 -Z 742849.287494 --mxsize 128 CM SEQDB,43,14,0,0,Gene; sRNA;,Published; PMID:28977401;,44,0,,\"\",0.56076,-6.44358,1600000,221441,179,88,true,-3.3264,0.71868,2019-01-16T10:25:22.000Z,2019-01-17T14:00:28.000Z\n",
            "RF03070,ssNA-helicase,2704,ssNA-helicase RNA,Weinberg Z,Weinberg Z,41.5,41.5,41.0,\"Firmicutes, Actinobacteria (1 example). Downstream gene\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 27.93 -Z 742849.287494 --mxsize 128 CM SEQDB,104,10,0,0,Gene; sRNA;,Published; PMID:28977401;,16,0,,\"\",0.51582,-9.6771,1600000,573396,103,49,true,-4.1526,0.71938,2019-01-03T12:29:03.000Z,2020-04-23T12:36:30.000Z\n",
            "RF03116,aCoV-5UTR,2707,Alphacoronavirus 5'UTR,Lamkiewicz K,Lamkiewicz K,70.3,70.3,49.7,\"The first 150 to 200 nucleotides within the 5' UTR of alphacoronaviruses are highly structured and shown to be conserved on the structural level among alphacoronaviruses. The 5' UTR has been predicted to contain 3 conserved stem-loops: SL1, SL2 and SL4. SL1 is important for viral replication, most likely playing a role in the template-switch of viral subgenomic RNA transcription. Mutations of the upper part of SL1 seems to have an higher impact of viral replication level. SL2 is crucial for viral viability, nucleotides are interchangeable as long as the structure remains stable. Disrupting any G-C pairing causes major defects in viral replication. SL4 is hypothesised to play a role in directing subgenomic RNA synthesis during viral replication. Downstream of SL4 lies SL5, which overlaps with the first ORF of the viral genome. The 3 terminal loops of SL5 contain a conserved sequence 5â€²-UUCCGU-3â€² and are thought to act as the packaging signal [3]. This strong level of conservation has not been observed in other genera of coronaviruses yet.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,17,27,0,0,Cis-reg;,Predicted;,30,0,,\"\",0.60461,-9.90325,1600000,491527,544,324,true,-4.5962,0.70506,2020-04-15T14:24:12.000Z,2020-04-23T12:36:30.000Z\n",
            "RF03117,bCoV-5UTR,2707,Betacoronavirus 5'UTR,Lamkiewicz K,Lamkiewicz K,111.6,111.6,55.5,\"The first 150 to 200 nucleotides within the 5' UTR of betacoronaviruses are highly structured and shown to be conserved on a structure level among betacoronaviruses. It has been predicted to contain 3 conserved stem-loops (SL1, SL2 and SL4). SARS-CoV and BCoV have an additional stem-loop, called SL3, which contains the TRS-L sequence in its loop region. However, according to predictions, SL3 is not stable in other betacoronaviruses like MHV. SL1 is important for viral replication, most likely playing a role in the template-switch of viral subgenomic RNA transcription. SL2 is crucial for viral viability, however, the nucleotide sequence can be mutated and some residues, as long as the structure remains stable. SL4 is hypothesized to play a role in directing subgenomic RNA synthesis during viral replication.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,17,22,0,0,Cis-reg;,Predicted;,27,0,,\"\",0.64251,-9.88954,1600000,483492,562,310,true,-4.1423,0.70546,2020-04-15T16:35:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03118,gCoV-5UTR,2707,Gammacoronavirus 5'UTR,Lamkiewicz K,Lamkiewicz K,124.9,124.9,51.6,\"The 5' UTR of gammacoronaviruses is similar to the 5' UTRs of alpha- and betacoronaviruses, as they also contain 3 helices denoted as SL1, SL2 and SL4. Further, in a subset of gammacoronaviruses a third stem-loop, SL3, is observed. SL1 and SL2 have major impacts of the level of viral replication, whereas SL4 is hypothesized to play a role as a \\\"spacer\\\" during the template-switch of CoV sgRNA synthesis.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,3,6,0,0,Cis-reg;,Predicted;,8,0,,\"\",0.44149,-10.9394,1600000,454394,350,200,true,-3.4942,0.71233,2020-04-15T19:23:09.000Z,2020-04-23T12:36:30.000Z\n",
            "RF03119,dCoV-5UTR,2707,Deltacoronavirus 5'UTR,Lamkiewicz K,Lamkiewicz K,102.9,102.9,40.5,\"The 5'UTR of deltacoronaviruses is similar to the 5'UTRs of alpha- and betacoronaviruses, as they also contain three helices denoted as SL1, SL2 and SL4. SL1 and SL2 have major impacts of the level of viral replication, whereas SL4 is hypothesized to play a role as a \\\"spacer\\\" during the template-switch of CoV sgRNA synthesis. Further, predictions show a conserved fourth stem-loop (SL3) between SL2 and SL4, sometimes observed in beta- and gammacoronaviruses as well. SL3 usually exposes the TRS-L sequence, which is crucial for sgRNA synthesis during viral replication, in the loop region.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,7,7,0,0,Cis-reg;,Predicted;,11,0,,\"\",0.55585,-10.63893,1600000,543298,455,294,true,-3.5699,0.70618,2020-04-15T19:36:20.000Z,2020-04-23T12:36:30.000Z\n",
            "RF03120,Sarbecovirus-5UTR,2707,Sarbecovirus 5'UTR,Lamkiewicz K,Lamkiewicz K,150.0,150.1,73.4,\"The 5' UTR of Sarbecovirus consists of 4 distinct stem-loops, namely SL1, SL2, SL3 and SL4. Further, there is a larger structure, SL5, present, which includes the first ORFs of the polyprotein. Note, that SL3 is typically present in SARS coronaviruses, but not necessarily conserved among other betacoronaviruses. SL1 is important for viral replication, most likely playing a role in the template-switch of viral subgenomic RNA transcription. SL2 is crucial for viral viability, however, the nucleotide sequence can be mutated and some residues, as long as the structure remains stable. SL4 is hypothesized to play a role in directing subgenomic RNA synthesis during viral replication.\",SARS-CoV-2-5UTR;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 50.00 -Z 742849.287494 CM SEQDB,19,84,0,0,Cis-reg;,Predicted;,19,0,,\"\",0.72899,-8.16489,1600000,363609,451,298,true,-4.0775,0.70612,2020-04-15T19:39:09.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03121,aCoV-3UTR,2708,Alphacoronavirus 3'UTR,Lamkiewicz K,Kevin Lamkiewicz,85.0,85.0,53.6,\"The 3' UTR of alphacoronaviruses consists of two small hairpins (PK-SL2) which can form an alternate conformation (PK-SL1) where the loop region of the second hairpin interacts with the stem of the first hairpin. PK-SL2 has been confirmed in HCoV-229E and HCoV-NL63 by in vitro structure probing experiments. Downstream of this pseudoknot lies the hypervariable region (HVR), which is supported by many covarying basepairs in alphacoronaviruses. Further, a conserved octanucleotide sequence is present in the HVR.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,17,28,0,0,Cis-reg;,Predicted;,31,0,,\"\",0.6112,-10.5818,1600000,615718,502,305,true,-3.7214,0.70563,2020-04-15T20:41:15.000Z,2020-04-23T12:36:30.000Z\n",
            "RF03122,bCoV-3UTR,2708,Betacoronavirus 3'UTR,Lamkiewicz K,Lamkiewicz K,112.9,112.9,68.3,\"The 3' UTR of betacoronaviruses is approximately 300-500 nucleotides long. The most 5' structure is the bulged stem-loop (BSL), starting downstream of the N gene stop codon and is essential for viral replication. It is predicted to be conserved among betacoronaviruses. Downstream to the BSL lies the hypervariable bulged-stemloop region (HVR), containing a triple helix junction. The hairpin stem-loop arising from this junction participates in the pseudoknot pairing with the BSL region. The pseudoknot and the BSL partially overlap and therefore cannot be formed at the same time. It is proposed that the BSL and pseudoknot act as a molecular switch that may regulate a transition during viral RNA synthesis. Part of the HVR is the octanucleotide sequence '5-GGAAGAGC-3' which is conserved among alpha- and betacoronaviruses.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,17,22,0,0,Cis-reg;,Predicted;,27,0,,\"\",0.64324,-9.57939,1600000,459676,537,327,true,-4.073,0.70489,2020-04-15T20:50:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03123,gCoV-3UTR,2708,Gammacoronavirus 3'UTR,Lamkiewicz K,Lamkiewicz K,175.8,175.8,50.5,\"In gammacoronaviruses a stem-loop located at the 3' UTR is observed, which is vital for viral replication. Even though a nearby pseudoknot structure, which has been observed in other genera of coronaviruses, is present, its functional importance has not been established yet.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,3,7,0,0,Cis-reg;,Predicted;,9,0,,\"\",0.67718,-7.7436,1600000,415767,417,275,true,-3.294,0.70708,2020-04-15T20:56:52.000Z,2020-04-23T12:36:30.000Z\n",
            "RF03124,dCoV-3UTR,2708,Deltacoronavirus 3'UTR,Lamkiewicz K,Lamkiewicz K,174.8,174.8,76.5,There are two smaller hairpins and the hypervariable region (HVR) present in the 3' UTR of deltacoronaviruses. The pseudoknot structure which is typical for alpha- and betacoronaviruses has not been predicted in deltacoronaviruses thus far.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,7,7,0,0,Cis-reg;,Predicted;,11,0,,\"\",0.70088,-8.7986,1600000,368029,555,410,true,-4.0091,0.70254,2020-04-15T20:59:38.000Z,2020-04-23T12:36:30.000Z\n",
            "RF03125,Sarbecovirus-3UTR,2708,Sarbecovirus 3'UTR,Lamkiewicz K,Lamkiewicz K,179.7,179.7,132.8,\"The Sarbecovirus 3'UTR is similar to other betacoronaviruses and is approximately 300-500 nucleotides long. The most 5' structure is the bulged stem-loop (BSL), starting downstream of the N gene stop codon and is essential for viral replication. It is predicted to be conserved among betacoronaviruses. Downstream to the BSL lies the hypervariable bulged-stem loop region (HVR), containing a triple helix junction. The hairpin stem from this junction participates in the pseudoknot pairing with the BSL region. The pseudoknot and the BSL partially overlap and therefore cannot be formed at the same time. It is proposed that the BSL and pseudoknot act as a molecular switch that may regulate a transition during viral RNA synthesis. Part of the HVR is the octanucleotide sequence '5-GGAAGAGC-3' which is conserved among alpha- and betacoronaviruses.\",SARS-CoV-2-3UTR;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 765380.351698 CM SEQDB,11,86,0,0,Cis-reg;,Predicted;,15,0,,\"\",0.71285,-8.54678,1600000,368925,496,335,true,-4.1199,0.7046,2020-04-15T21:02:37.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03142,L2-Alphaproteobacteria,2724,L2-Alphaproteobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,37.8,37.8,37.6,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,452,170,0,0,Cis-reg; leader;,Published; PMID:32448158;,230,0,,\"\",0.67763,-8.15632,1600000,578466,405,69,true,-3.155,0.71887,2020-07-16T08:24:04.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03144,eL15-Euryarchaeota,2725,eL15-Euryarchaeota ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,33.7,33.7,33.6,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,539,103,0,0,Cis-reg; leader;,Published; PMID:32448158;,223,0,,\"\",0.5671,-6.02302,1600000,314917,153,48,true,-3.6453,0.73142,2020-07-16T10:43:12.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03128,L31-Coriobacteria,2726,L31-Coriobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,33.0,33.0,32.5,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,75,53,0,0,Cis-reg; leader;,Published; PMID:32448158;,55,0,,\"\",0.46579,-12.40537,1600000,1187323,86,33,true,-3.5581,0.73188,2020-07-10T10:01:18.000Z,2020-09-14T15:07:41.000Z\n",
            "RF03129,L31-Corynebacteriaceae,2726,L31-Corynebacteriaceae ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,36.0,36.6,35.4,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,113,55,0,0,Cis-reg; leader;,Published; PMID:32448158;,105,0,,\"\",0.5495,-8.53939,1600000,426470,332,73,true,-3.6833,0.71899,2020-07-10T10:27:40.000Z,2020-09-14T15:07:41.000Z\n",
            "RF03156,L31-Firmicutes,2726,L31-Firmicutes ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,33.3,33.3,33.2,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,1083,65,0,0,Cis-reg; leader;,Published; PMID:32448158;,681,0,,\"\",0.54028,-10.15967,1600000,821267,99,30,true,-2.8527,0.73866,2020-07-28T14:09:21.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03157,L31-Gammaproteobacteria,2726,L31-Gammaproteobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,33.2,33.2,33.1,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,733,319,0,0,Cis-reg; leader;,Published; PMID:32448158;,733,0,,\"\",0.63155,-7.64602,1600000,652766,177,52,true,-3.8052,0.72012,2020-07-29T14:55:31.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03158,L31-Actinobacteria,2726,L31-Actinobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,35.0,35.0,34.9,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,971,516,0,0,Cis-reg; leader;,Published; PMID:32448158;,972,0,,\"\",0.61677,-4.74631,1600000,444107,146,48,true,-3.2023,0.72115,2020-07-29T17:39:45.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03131,L25-Gammaproteobacteria,2727,L25-Gammaproteobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,34.1,34.3,34.0,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,381,125,0,0,Cis-reg; leader;,Published; PMID:32448158;,418,0,,\"\",0.68323,-3.65752,1600000,410532,70,26,true,-4.0589,0.73155,2020-07-10T12:56:43.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03132,L17-Actino-Proteobacteria,2728,L17-Actino-Proteobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,34.9,34.9,34.6,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,408,236,0,0,Cis-reg; leader;,Published; PMID:32448158;,516,0,,\"\",0.46495,-8.64158,1600000,481144,145,56,true,-3.2511,0.71931,2020-07-10T13:06:27.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03133,S4-Fusobacteriales,2729,S4-Fusobacteriales ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,48.8,48.8,38.3,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,84,18,0,0,Cis-reg; leader;,Published; PMID:32448158;,41,0,,\"\",0.58072,-4.95412,1600000,372404,247,62,true,-3.4186,0.71915,2020-07-10T14:20:45.000Z,2020-09-14T15:07:42.000Z\n",
            "RF03139,S4-Bacteroidia,2729,S4-Bacteroidia ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,34.7,34.7,34.6,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,225,63,0,0,Cis-reg; leader;,Published; PMID:32448158;,91,0,,\"\",0.64396,-7.9181,1600000,776078,226,39,true,-2.8687,0.73018,2020-07-10T17:08:17.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03140,S4-Clostridia,2729,S4-Clostridia ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,37.5,37.5,37.2,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,140,39,0,0,Cis-reg; leader;,Published; PMID:32448158;,73,0,,\"\",0.71631,-2.56467,1600000,296659,453,57,true,-3.5294,0.72365,2020-07-10T17:27:32.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03141,S4-Flavobacteria,2729,S4-Flavobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,35.2,35.2,35.1,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,260,176,0,0,Cis-reg; leader;,Published; PMID:32448158;,230,0,,\"\",0.64164,-2.281,1600000,316349,109,34,true,-4.0971,0.72747,2020-07-10T17:45:24.000Z,2020-09-14T15:07:42.000Z\n",
            "RF03134,S6-S18-Chlorobi,2730,S6-S18-Chlorobi ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,47.4,47.4,41.2,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,34,12,0,0,Cis-reg; leader;,Published; PMID:32448158;,15,0,,\"\",0.58068,-4.49978,1600000,217647,155,76,true,-3.606,0.71911,2020-07-10T14:24:06.000Z,2020-09-14T15:07:42.000Z\n",
            "RF03135,L4-Archaeoglobi,2731,L4-Archaeoglobi ribosomal protein leades,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,34.5,34.6,34.0,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 0 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,379,84,0,0,Cis-reg; leader;,Published; PMID:32448158;,161,0,,\"\",0.6111,-3.90897,1600000,268472,185,85,true,-3.3971,0.71872,2020-07-10T15:31:10.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03136,S10-Clostridia,2732,S10-Clostridia ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,35.1,35.1,34.9,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,199,470,0,0,Cis-reg; leader;,Published; PMID:32448158;,518,0,,\"\",0.55635,-7.7976,1600000,253731,288,81,true,-3.5232,0.71877,2020-07-10T16:20:21.000Z,2020-09-14T15:07:42.000Z\n",
            "RF03138,S16-Flavobacteria,2733,S16-Flavobacteria ribosomal protein leader,\"Weinberg Z, Eckert I\",Published; PMID:32448158;,35.7,36.0,35.6,Imported from the ZWD database.,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,130,151,0,0,Cis-reg; leader;,Published; PMID:32448158;,166,0,,\"\",0.4358,-5.86038,1600000,515866,84,35,true,-3.9321,0.72286,2020-07-10T16:51:51.000Z,2020-09-14T15:07:42.000Z\n",
            "RF03151,cyano-S1,2734,Cyanobacterial ribosomal protein S1 leader,Weinberg Z,Published; PMID:17621584;,39.9,39.9,37.7,\"cyano-S1 is a ncRNA recognized in the 5-prime region of genes encoding 30S ribosomal protein S1 [1], it has been proposed as an cis regulation element.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,26,16,0,0,Cis-reg; leader;,Published; PMID:17621584;,21,0,,\"\",0.63897,-3.70632,1600000,307042,153,81,true,-4.0514,0.71869,2020-07-20T11:41:27.000Z,2020-09-14T15:07:42.000Z\n",
            "RF03155,TD-1,2735,TD-1 RNA,Weinberg Z,Published; PMID:17621584;,64.5,64.5,64.4,non coding RNA found in Treponema denticola [1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,28,109,0,0,Cis-reg;,Published; PMID:17621584;,8,0,,\"\",0.47911,-8.5333,1600000,274141,1014,120,true,-3.3287,0.71833,2020-07-20T15:47:01.000Z,2020-09-14T15:07:43.000Z\n",
            "RF02837,anti-hemB,2736,Burkholderia RNA 7 (anti-hemB),\"Weinberg Z, Gore J\",Published; PMID:17621584,49.5,49.5,49.4,\"This RNA motif was identified by computational search in b-proteobacteria Burkholderia [1]. Co-purified with Hfq, expression confirmed by Northern blot. Putative functional homologue of SP-RydB and predicted target hemB [2]. Named anti-hemB in [1,3]. Named ncS04 and shown to be expressed in biofilm [4].\",Bcj7;,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,26,314,0,0,Gene; sRNA;,Published; PMID:17621584,305,0,,\"\",0.65066,-7.03273,1600000,435769,167,80,true,-3.1025,0.71887,2017-12-13T16:33:57.000Z,2020-09-14T15:07:43.000Z\n",
            "RF03162,TD-2,2737,TD-2 RNA,Weinberg Z,Published; PMID:17621584;,48.7,48.7,48.5,non coding RNA found in Treponema denticola [1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,36,236,0,0,Gene; sRNA;,Published; PMID:17621584;,8,0,,\"\",0.47306,-7.01772,1600000,266431,175,98,true,-3.7257,0.71865,2020-08-11T14:30:39.000Z,2020-09-14T15:07:43.000Z\n",
            "RF03530,bglG-cis-reg,2740,cis-regulatory element of the bglG/LicT operon,\"Secherre E, Boudhouche H, Bahroun R\",PMID:23675309,80.0,102.0,78.9,,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,0,0,0,Cis-reg;,Predicted; LocaRNA,6,0,,\"\",0.54459,-6.28171,1600000,253364,245,137,true,-3.1778,0.71845,2020-09-09T16:49:01.000Z,2020-09-14T15:07:43.000Z\n",
            "RF03531,n00280,2741,Clostridioides difficile sRNA included into helicase gene,\"Gobalakichenin A, Ibene M, Pohyer V\",Published; PMID:23675309,65.2,65.2,65.1,\"This element was identified by RNA deep sequencing of Clostridioides difficile 630 [1] where it is located within gene CD0749 (putative DNA helicase, UvrD/REP type) in the same direction as the gene. A strong transcription start site for this RNA was experimentally confirmed, in the 3â€™ region of gene CD0749. The 3â€™ end of the sRNA has not been confirmed and its length was arbitrarily set to 100nt. For inclusion criteria: Only sequences from order Clostridiales were included, mostly because the starting sequence belonged to this order and the large majority of sequences found by the different RFAM search iterations belonged to this order as well. Associated e-values did not exceed 1e-09. As the original RNA is part of a coding sequence (CDS), it is possible that homologies were detected due to selection pressure for the CDS rather than for a sRNA. R-scape identified only two significantly covarying pairs present in the structure (4-100 and 7-97). Thus evidence for structure is weak.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,24,38,0,0,Gene; sRNA;,Predicted; LocARNA;,56,0,,\"\",0.5303,-4.65586,1600000,272069,189,101,true,-3.7298,0.71845,2020-09-09T16:57:33.000Z,2020-09-14T15:07:43.000Z\n",
            "RF03532,SQ2397,2742,Cis-regulator of HTH transcription factor,\"SERIR R, SCHNEIDER T, SANGARANE V\",Published; PMID:23675309,100.0,100.9,99.6,\"This element was identified by RNA deep sequencing of Clostridioides difficile 630 [1] where it is located immediately upstream of an uncharacterized HTH-type Transcriptional regulator. Homologues are found widely across firmicutes, consistently located 5â€™ of transcriptional regulators of similar size and sequence, which strongly supports a cis-regulatory function. RNA is referred to as CD630_SQ2397 in ref [1]. Note: the sequence is highly conserved and shows no evidence for covariation as per R-scape analysis. The proposed secondary structure is therefore purely hypothetical.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,57,3,0,0,Cis-reg;,Predicted; LocARNA;,56,0,,\"\",0.47741,-10.81529,1600000,459796,391,243,true,-3.5327,0.70892,2020-09-09T19:18:02.000Z,2020-09-14T15:07:43.000Z\n",
            "RF03533,SQ1002_RNA,2744,Clostridioides difficile sRNA SQ1002,\"Rabearivelo N, Marty M, Sabban J\",Predicted; Blastn; LocARNA;,128.0,131.0,127.9,\"This 191nt small RNA was identified in Clostridioides difficile 630 by RNA deep sequencing and confirmed by Northern Blot and RT-PCR [1]. Extremities were confirmed by 5â€™ and 3â€™-RACE [1]. The element is found in several C. difficile strains in syntenic positions between a putative acyltransferase gene and a Ferrous iron transport gene. No homologues were found outside C. difficile, although a putative paralogue was found in C. difficile 630, located between a Tryptophanyl-tRNA synthetase gene and a pseudogene. Due to lack of sequence variation, the proposed secondary structure is not supported by any covariation and should be considered speculative\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,1,0,0,Gene; sRNA;,Predicted; R-scape;,1,0,,\"\",0.59452,-8.5016,1600000,354188,383,237,true,-3.6138,0.70937,2020-09-10T12:01:46.000Z,2023-11-06T17:21:47.000Z\n",
            "RF03534,sRNA71,2745,Staphylococcus sRNA71 small RNA,\"Ducos C, Michel M, Morice E\",Published; PMID:23733475;,50.0,76.9,41.9,\"sRNA71 is a Bacterial small RNA discovered by RNA-seq analysis of the multiresistant Staphylococcus aureus strain ST239 grown in the presence of antibiotics [1] and found throughout the Staphylococcus genus. It is thought to be a trans-encoded small RNA expressed in diverse conditions and media. The 5â€™ position is uncertain, this record adds 12 bases to the initial sRNA71 sequence to include additional conserved nucleotides.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,1,0,0,Gene; sRNA;,Predicted; LocARNA;,13,0,,\"\",0.46853,-6.69683,1600000,209950,204,107,true,-3.3184,0.71863,2020-09-10T12:04:45.000Z,2021-07-05T14:43:57.000Z\n",
            "RF03535,TEG147,2750,Staphylococcus aureus small RNA TEG147,\"Liger J, Minaud L, Poulain C, Vencic J\",Published; PMID:20505759;,56.1,56.1,55.2,\"TEG147 is an RNA element identified by RNA-seq in S. aureus N315 [1] with homologues in other Staphylococcus species. Other names: sRNA85, srn_0960, SAOUHSCs103.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,4,29,0,0,Cis-reg;,Predicted; LocARNA;,31,0,,\"\",0.61137,-4.09748,1600000,279910,141,60,true,-3.8322,0.71923,2020-09-11T15:16:51.000Z,2020-09-14T15:07:43.000Z\n",
            "RF03536,Flavi_TBFV_xrRNA,2754,Tick-borne Flavivirus exoribonuclease-resistant RNA element,Lamkiewicz K,Lamkiewicz K,46.5,46.5,35.9,\"The exoribonuclease-resistant RNA elements (xrRNA) are described throughout all groups of flaviviruses. Usually, each virus harbors two xrRNAs, xrRNA1 and xrRNA2, in the beginning of the 3' UTR [1]. The formation of these stem-loops, especially xrRNA1, is vital to ensure resistance against XRN1 activity [2]. The Y-shaped stem-loop is also termed SL II and SL IV, respectively. In order to function as xrRNA, the sequence downstream is needed as well, since the upper loop region forms a pseudoknot (PK) with the single-stranded region directly downstream to its respective hairpin. In some species, the region downstream also forms a small hairpin. In such cases, the PK interactions takes place between the two loop regions. Conserved formation of these structures were observed in mammalian cells but not in mosquito cells, suggesting this region has varying functions in different hosts[3][4]. In plant-viruses, xrRNA elements have been observed as well, showing some similarities to flaviviral xrRNAs[5][6]. However, plant-virus xrRNA and flaviviral xrRNA are distinguishable by their underlying three-dimensional folds[7].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,18,23,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,16,0,,\"\",0.67552,-3.04646,1600000,163479,173,90,true,-3.8643,0.71894,2020-09-11T18:05:34.000Z,2022-10-11T11:22:54.000Z\n",
            "RF03537,Flavi_TBFV_SL6,2754,Tick-borne Flavivirus short stem-loop SL6,\"Lamkiewicz K, Wolfinger MT\",Lamkiewicz K,38.3,38.3,38.0,\"In different studies, a short stem-loop, named SL6 has been observed in at least Tick-borne encephalitis virus, Langat virus and Omsk hemorrhagic fever [1][2]. SL6 shows a high heterogeneity among different tick-borne flaviviruses, but is structurally conserved supported by multiple covariation [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,9,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,13,0,,\"\",0.81352,-1.98823,1600000,407370,110,45,true,-2.9609,0.7475,2020-09-11T21:27:26.000Z,2022-10-11T11:23:50.000Z\n",
            "RF03538,Flavi_TBFV_CRE,2754,Tick-borne Flavivirus 3' UTR cis-acting replication element (CRE),Lamkiewicz K,Lamkiewicz K,60.5,67.5,60.4,\"Flavi_TBFV_CRE is a Tick Borne Flavivirus family. Flavivirus 3' UTR cis-acting replication element (CRE) plays a crucial role in the virus replication and is thought to be needed for the formation of a \\\"replication complex\\\" [1], it is composed of a small hairpin (sHP) and a larger structural element (3'SL). It has been shown that mutations in sHP are lethal for the virus [2]. CRE is involved in the 5'-3' UTR interation of flaviviruses [3]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,10,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,12,0,,\"\",0.4929,-6.64087,1600000,226107,207,106,true,-3.3535,0.71863,2020-09-11T21:28:47.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03539,Flavi_NKV_xrRNA,2754,No-known vector Flavivirus exoribonuclease-resistant RNA element,Lamkiewicz K,Lamkiewicz K,50.0,50.0,37.7,\"The exoribonuclease-resistant RNA elements (xrRNA) are described throughout all groups of flaviviruses. Usually, each virus harbors two xrRNAs, xrRNA1 and xrRNA2, in the beginning of the 3' UTR [1]. The formation of these stem-loops, especially xrRNA1, is vital to ensure resistance against XRN1 activity [2]. The Y-shaped stem-loop is also termed SL II and SL IV, respectively. In order to function as xrRNA, the sequence downstream is needed as well, since the upper loop region forms a pseudoknot (PK) with the single-stranded region directly downstream to its respective hairpin. In some species, the region downstream also forms a small hairpin. In such cases, the PK interactions takes place between the two loop regions. Conserved formation of these structures were observed in mammalian cells but not in mosquito cells, suggesting this region has varying functions in different hosts[3][4]. In plant-viruses, xrRNA elements have been observed as well, showing some similarities to flaviviral xrRNAs[5][6]. However, plant-virus xrRNA and flaviviral xrRNA are distinguishable by their underlying three-dimensional folds[7].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,3,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,4,0,,\"\",0.60548,-5.03539,1600000,247486,175,84,true,-2.8026,0.73195,2020-09-11T21:29:32.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03540,Flavi_NKV_CRE,2754,No-Known-Vector Flavivirus 3' UTR cis-acting replication element (CRE),Lamkiewicz K,Lamkiewicz K,50.0,83.9,40.6,\"Flavi_NKV_CRE is a Flavivirus family were the vector is not known. Flavivirus 3' UTR cis-acting replication element (CRE) plays a crucial role in the virus replication and is thought to be needed for the formation of a \\\"replication complex\\\" [1], it is composed of a small hairpin (sHP) and a larger structural element (3'SL). It has been shown that mutations in sHP are lethal for the virus [2]. CRE is involved in the 5'-3' UTR interation of flaviviruses [3]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,1,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,2,0,,\"\",0.54383,-5.42231,1600000,263768,194,96,true,-2.9697,0.72778,2020-09-11T21:54:05.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03541,Flavi_ISFV_xrRNA,2754,Insect-specific Flavivirus exoribonuclease-resistant RNA element,Lamkiewicz K,Lamkiewicz K,50.0,54.8,32.9,\"The exoribonuclease-resistant RNA elements (xrRNA) are described throughout all groups of flaviviruses. Usually, each virus harbors two xrRNAs, xrRNA1 and xrRNA2, in the beginning of the 3' UTR [1]. The formation of these stem-loops, especially xrRNA1, is vital to ensure resistance against XRN1 activity [2]. The Y-shaped stem-loop is also termed SL II and SL IV, respectively. In order to function as xrRNA, the sequence downstream is needed as well, since the upper loop region forms a pseudoknot (PK) with the single-stranded region directly downstream to its respective hairpin. In some species, the region downstream also forms a small hairpin. In such cases, the PK interactions takes place between the two loop regions. Conserved formation of these structures were observed in mammalian cells but not in mosquito cells, suggesting this region has varying functions in different hosts[3][4]. In plant-viruses, xrRNA elements have been observed as well, showing some similarities to flaviviral xrRNAs[5][6]. However, plant-virus xrRNA and flaviviral xrRNA are distinguishable by their underlying three-dimensional folds[7].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,9,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,5,0,,\"\",0.57956,-7.518,1600000,255500,173,89,true,-3.6531,0.7186,2020-09-11T21:54:39.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03542,Flavi_ISFV_repeat_Ra_Rb,2754,Insect-Specific Flavivirus 3' UTR repeats Ra and Rb elements,\"Lamkiewicz K, Wolfinger MT\",Lamkiewicz K,50.0,50.5,41.3,\"3' UTRs in insect specific flavivirus families present three to four repeats of conserved element Ra and Rb [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,12,40,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,4,0,,\"\",0.48191,-9.15151,1600000,280805,188,100,true,-3.6981,0.71863,2020-09-11T21:55:16.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03543,Flavi_ISFV_repeat_Rb,2754,Insect-Specific Flavivirus 3' UTR repeats Rb,\"Lamkiewicz K, Wolfinger MT\",Lamkiewicz K,43.0,43.0,37.9,\"3' UTRs in insect specific flavivirus families present three to four repeats of conserved element Rb [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,19,5,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,4,0,,\"\",0.63859,-4.95306,1600000,422355,115,47,true,-3.4905,0.71953,2020-09-12T11:32:22.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03544,Flavi_ISFV_repeat_Ra,2754,Insect-Specific Flavivirus 3' UTR repeats Ra,\"Lamkiewicz K, Wolfinger MT\",Lamkiewicz K,41.0,41.8,39.0,\"3' UTR in insect specific flavivirus families present three to four repeats of conserved element Ra [1,2].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,30,7,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,4,0,,\"\",0.53535,-6.78572,1600000,332951,125,58,true,-3.9426,0.71911,2020-09-12T11:32:54.000Z,2022-10-29T17:55:17.000Z\n",
            "RF03545,Flavi_ISFV_CRE,2754,Insect-specific Flavivirus 3' UTR cis-acting replication element (CRE),Lamkiewicz K,Lamkiewicz K,41.0,41.8,40.9,\"Flavi_ISFV_CRE is a Insect specific Flavivirus family. Flavivirus 3' UTR cis-acting replication element (CRE) plays a crucial role in the virus replication and is thought to be needed for the formation of a \\\"replication complex\\\" [1], it is composed of a small hairpin (sHP) and a larger structural element (3'SL). It has been shown that mutations in sHP are lethal for viruses [2]. CRE is involved in the 5'-3' UTR interaction of flaviviruses [3].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,16,4,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,20,0,,\"\",0.52608,-5.39216,1600000,229632,207,89,true,-3.0607,0.72988,2020-09-12T11:39:29.000Z,2022-10-29T17:42:08.000Z\n",
            "RF03547,Flavi_xrRNA,2754,General Flavivirus exoribonuclease-resistant RNA element,Lamkiewicz K,Lamkiewicz K,37.0,40.3,36.9,\"The exoribonuclease-resistant RNA elements (xrRNA) are described throughout all groups of flaviviruses. Usually, each virus harbors two xrRNAs, xrRNA1 and xrRNA2, in the beginning of the 3' UTR [1]. The formation of these stem-loops, especially xrRNA1, is vital to ensure resistance against XRN1 activity [2]. The Y-shaped stem-loop is also termed SL II and SL IV, respectively. In order to function as xrRNA, the sequence downstream is needed as well, since the upper loop region forms a pseudoknot (PK) with the single-stranded region directly downstream to its respective hairpin. In some species, the region downstream also forms a small hairpin. In such cases, the PK interactions takes place between the two loop regions. Conserved formation of these structures were observed in mammalian cells but not in mosquito cells, suggesting this region has varying functions in different hosts[3][4]. In plant-viruses, xrRNA elements have been observed as well, showing some similarities to flaviviral xrRNAs[5][6]. However, plant-virus xrRNA and flaviviral xrRNA are distinguishable by their underlying three-dimensional folds[7].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,48,22,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,29,0,,\"\",0.46945,-6.30662,1600000,205262,226,82,true,-1.9424,0.74559,2020-09-12T11:55:42.000Z,2022-10-29T17:55:17.000Z\n",
            "RF04222,PLRV_xrRNA,2754,Flavivirus Potato leading responsible virus exoribonuclease-resistant RNA,\"Steckelberg A, Vicens Q, Jeffrey K\",Vicens Q,32.4,32.4,32.1,\"Potato leading responsible virus exoribonuclease-resistant RNA (PLRV-xrRNA) is a non-coding RNA (ncRNA) that blocks the progression of 5' to 3' exoribonuclease using only a folded RNA element, these elements were first reported in Flavivirus. Insights into the mechanism of programmed exoribonuclease resistance in PLRV has been reported using X-ray crystallography and in vitro assays of exoribonuclease resistance [1,2]. The structure of PLRV_xrRNA is reported in PDB 7JJU [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 25.00 -Z 742849.287494 CM SEQDB,44,40,0,0,Cis-reg;,Published; PMID:32848042,61,0,,\"\",0.53031,-7.27708,1600000,271575,211,57,true,-3.3886,0.72516,2022-04-26T11:57:24.000Z,2023-11-06T16:44:07.000Z\n",
            "RF03546,Flavivirus-5UTR,2760,Flavivirus 5' UTR,Lamkiewicz K,Lamkiewicz K,80.0,83.6,79.3,\"The 5' UTR of flaviviruses are highly structured, has a length of approximately 100 nucleotides and harbors two conserved RNA secondary structures vital for the viral life cycle[1][2]. During replication, a long-range RNA-RNA interaction between 5' UTR and 3' UTR of the genome is formed to initiate synthesis of new viral replicates and viral protein translation. In direct adjacency to the 5' UTR, already part of the coding region, lies the cHP structure, which is essential for the viral replication[3][4]. The first structural element is termed 5'SLA and comprises three stems (S1, S2, S3) folded as Y-shaped-like stem structure, and a side structure domain (SSD). Its overall length is around 70 nucleotides. Viral replication is stopped by disruption of S1 and S2. Further, via RNA-protein interaction with NS5, SLA serves as a promoter for RNA synthesis during circularization of the viral genome[5][6]. Despite the diversity of SSD, its stable structure is essential for infectivity[3]. The second structural element is termed 5'SLB. At its top region of the stem loop lies the translation initiation codon. It further contains the 5'UAR (upstream AUG region), which is essential for the circularization of the genome[8]. The 5'UAR interacts with the 3'UAR, which is located at the 3' UTR of the genome to form the long-range RNA-RNA interaction[8]. At the start of the coding region lies the capsid-coding hairpin region (cHP), which is followed by the 5'CS (conserved sequence). The 5'CS forms an additional long-range RNA-RNA interaction with the 3' UTR (3'CS)[5]. The cHP further aids in the start codon recognition and thus in viral replication. Studies show that the function of cHP is sequence-independent but structure-dependent[9].\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,8,1,0,0,Cis-reg;,Predicted; LocARNA 2.0.0RC8,9,0,,\"\",0.56466,-7.90607,1600000,264399,264,156,true,-3.0405,0.71786,2020-09-12T11:51:34.000Z,2021-07-05T14:43:57.000Z\n",
            "RF04177,BTnc201,2761,Bacteroides sRNA BTnc201,\"Prezza, G, Ryan, D, Madler, G, Barquist, L, Westermann, A\",Published; PMID:PMC7366714;,105.0,105.0,79.1,An uncharacterized small RNA discovered in Bacteroides thetaiotaomicron [1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,2,8,0,0,Gene; sRNA;,Published; PMID:PMC7366714;,10,0,,\"\",0.53599,-6.59342,1600000,276764,200,99,true,-3.6113,0.7185,2021-05-21T15:25:35.000Z,2021-11-19T11:53:19.000Z\n",
            "RF04178,BTnc005,2761,Bacteroides sRNA BTnc005,\"Prezza, G, Ryan, D, Madler, G, Barquist, L, Westermann, A\",Published; PMID:32678091;,174.8,179.3,174.3,An uncharacterized small RNA discovered in Bacteroides thetaiotaomicron [1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,3,2,0,0,Gene; sRNA;,Published; PMID:32678091;,4,0,,\"\",0.61499,-8.58473,1600000,418549,336,205,true,-3.179,0.71185,2021-05-21T16:28:00.000Z,2021-11-19T11:53:19.000Z\n",
            "RF04179,BTnc049,2761,Bacteroides sRNA BTnc049,\"Prezza, G, Ryan, D, Madler, G, Barquist, L, Westermann, A\",Published; PMID:PMC7366714;,81.9,81.9,67.4,An uncharacterized small RNA discovered in Bacteroides thetaiotaomicron [1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,9,30,0,0,Gene; sRNA;,Published; PMID:PMC7366714;,38,0,,\"\",0.51364,-7.83296,1600000,349632,211,111,true,-3.5134,0.71839,2021-05-21T19:07:32.000Z,2021-11-19T11:53:19.000Z\n",
            "RF04180,BTnc231,2761,Bacteroides sRNA BTnc231,\"Prezza, G, Ryan, D, Madler, G, Barquist, L, Westermann, A\",Published; PMID:32678091;,80.4,80.4,60.0,An uncharacterized small RNA discovered in Bacteroides thetaiotaomicron [1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,7,21,0,0,Gene; sRNA;,Published; PMID:32678091;,28,0,,\"\",0.56862,-7.94235,1600000,301676,357,153,true,-3.2702,0.71805,2021-05-21T20:58:32.000Z,2021-11-19T11:53:19.000Z\n",
            "RF04181,rteR,2761,rteR sRNA,\"Prezza, G, Ryan, D, Madler, G, Barquist, L, Westermann, A\",Published; PMID:32678091; PMID:22821972;,72.0,72.0,58.0,\"A small RNA discovered in Bacteroides fragilis that has been shown to repress a transposon operon [1,2]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,10,16,0,0,Gene; sRNA;,Published; PMID:32678091; PMID:22821972;,23,0,,\"\",0.49212,-6.81156,1600000,295233,173,84,true,-3.3329,0.71888,2021-05-21T21:33:11.000Z,2021-11-19T11:53:19.000Z\n",
            "RF04182,GibS,2761,GibS sRNA,\"Prezza, G, Ryan, D, Madler, G, Barquist, L, Westermann, A\",Published; PMID:PMC7366714;,94.7,94.7,83.4,\"A small RNA with a conserved region in the 5' region, it appears to play a role in carbon metabolism [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,13,20,0,0,Gene; sRNA;,Published; PMID:PMC7366714;,29,0,,\"\",0.42228,-10.93694,1600000,370933,250,142,true,-2.8458,0.718,2021-05-24T12:36:10.000Z,2021-11-19T11:53:19.000Z\n",
            "RF04184,BTnc060,2761,Bacteroides sRNA BTnc060,\"Prezza, G, Ryan, D, Madler, G, Barquist, L, Westermann, A\",Published; PMID:32678091;,228.9,228.9,198.3,An uncharacterized small RNA discovered in Bacteroides thetaiotaomicron [1],,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,6,7,0,0,Gene; sRNA;,Published; PMID:32678091;,13,0,,\"\",0.46331,-10.39877,1600000,422115,348,213,true,-3.4898,0.71103,2021-05-24T14:57:40.000Z,2021-11-19T11:53:19.000Z\n",
            "RF04188,Hovlinc,2762,Hovlinc ribozyme (hominin vlincRNA-located),\"Fei Q, Kapranov P\",Published; PMID:33753927,42.3,42.3,36.2,\"Hovlinc is a recently evolved class of self-cleaving ribozyme found in a very long non coding RNA (vlincRNA) in hominins, including humans (genus Homo), and chimpanzees and bonobos (genus Pan), but missing in gorillas (genus Gorilla). [1]\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,74,7,0,0,Gene; ribozyme;,Published; PMID:33753927,75,0,,\"\",0.55829,-8.23714,1600000,343412,277,170,true,-3.2116,0.71559,2021-06-10T16:19:15.000Z,2021-12-03T12:41:14.000Z\n",
            "RF04189,icd-II,2763,icd-II ncRNA motif,\"Brewer, KI, Ontiveros-Palacios N\",Published;  PMID:33970790,99.0,99.2,34.0,\"icd-II ncRNA motif was located upstream of icd gene, which codes for NADP+-dependent isocitrate dehydrogenase (IDH) enzyme. icd-II has been proposed as a strong candidate riboswitch [1]. Part of the secondary structure of icd-II includes the ribosomal binding site (RBS) of the adjacent gene wish suggest that this element can be regulating translation initiation of its associated transcript.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,15,3,0,0,Cis-reg;,Published;  PMID:33970790,15,0,,\"\",0.53208,-6.87475,1600000,354440,241,133,true,-2.5472,0.71819,2021-10-20T12:22:47.000Z,2021-11-19T10:36:03.000Z\n",
            "RF04192,carA,2764,carA ncRNA motif,\"Brewer, KI, Ontiveros-Palacios N\",Published;  PMID:33970790,100.0,100.3,37.8,\"carA ncRNA motif was reported as a Strong Riboswitch Candidate [1] CarA ncRNA motif was located upstream of carA gene which codes for the small subunit of carbamoyl phosphate synthase, which is an enzyme that catalyses the first committed step in pyrimidine and arginine biosynthesis. It is proposed as a possible cis-regulatory element.\",,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,35,2,0,0,Cis-reg;,Published;  PMID:33970790,28,0,,\"\",0.50873,-10.54347,1600000,371574,325,202,true,-3.7185,0.71208,2021-11-01T11:25:47.000Z,2023-11-06T17:21:47.000Z\n",
            "RF04247,bZIP,2767,bZIP non canonical Hac1/Xbp1 intron,\"Hooks KB, Griffiths-Jones SR\",Griffiths-Jones SR,41.0,41.0,40.9,bZIP intron RNA is a non-canonical intron that was first reported in HAC1 gene in yeast and XBP1 gene in Metazoa [1].,,cmbuild -F CM SEED,cmcalibrate --mpi CM,cmsearch --cpu 4 --verbose --nohmmonly -T 30.00 -Z 742849.287494 CM SEQDB,112,393,0,0,Gene; sRNA;,Predicted; RNAfold,442,0,,\"\",0.60396,-3.29642,1600000,214983,139,72,true,-3.9575,0.72542,2023-08-08T12:08:58.000Z,2023-11-06T16:44:07.000Z\n"
          ]
        }
      ]
    }
  ]
}